##contig	feature	orfid	location	check	product	gene	hamap	nite_blast	migap_product	migap_gene	migap_note
##GOHSU01	contig00011				32932
GOHSU01	CDS	GOHSU_01_00010	<1..388	3x codon frame	probable glycerate kinase [nite]			RER_36060 :probable glycerate kinase : : 2.7.1.31 : Rhodococcus erythropolis PR4	glycerate kinase		G7GPS2_9ACTO Putative glycerate kinase OS=Gordonia amarae NBRC 15530 GN=GOAMR_40_00870 PE=4 SV=1;glycerate kinase [Arthrobacter phenanthrenivorans S phe3];PFAM: glycerate kinase family; TIGRFAM: glycerate kinase;Glycerate kinase family; cl00841
GOHSU01	CDS	GOHSU_01_00020	531..881		putative iron-sulfur cluster assembly protein IscA [nite]	 iscA [nite]		RER_36050 :putative iron-sulfur cluster assembly protein IscA : iscA : : Rhodococcus erythropolis PR4	fe-s cluster assembly protein		E4WIU3_RHOE1 Putative Fe-S cluster assembly protein OS=Rhodococcus equi (strain 103S) GN=REQ_28320 PE=4 SV=1;fe-s cluster assembly protein [Rhodococcus equi 103 S];Iron-sulphur cluster biosynthesis; cl00400
GOHSU01	CDS	GOHSU_01_00030	1046..1975		adenosine kinase [nite]	 adoK [nite]		ROP_08670 :adenosine kinase : adoK : 2.7.1.20 : Rhodococcus opacus B4	sugar kinase, ribokinase		F1YHH3_9ACTO PfkB domain-containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_06205 PE=4 SV=1;sugar kinase, ribokinase [Saccharomonospora marina XMU15];PFAM: pfkB family carbohydrate kinase;Sugar kinases, ribokinase family [Carbohydrate transport and metabolism]; COG0524;Ribokinase-like subgroup A.  Found in bacteria and archaea, this subgroup is part of the ribokinase/pfkB superfamily.  Its oligomerization state is unknown at this time; cd01942;substrate binding site [chemical binding];ATP binding site [chemical binding]
GOHSU01	CDS	GOHSU_01_00040	complement(2098..4023)		glutamine amidotransferase LtsA [nite]	 ltsA [nite]		ROP_08660 :glutamine amidotransferase LtsA : ltsA : : Rhodococcus opacus B4	asparagine synthase		F1YHH4_9ACTO Asparagine synthetase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_06210 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Asparagine synthase (glutamine-hydrolyzing) [Amino acid transport and metabolism]; COG0367;TIGRFAM: asparagine synthase (glutamine- hydrolyzing); PFAM: asparagine synthase; glutamine amidotransferase class-II; KEGG: dds:Ddes_2055 asparagine synthase (glutamine- hydrolyzing);Glutamine amidotransferases class-II (GATase) asparagine synthase_B type.  Asparagine synthetase B catalyses the ATP-dependent conversion of aspartate to asparagine. This enzyme is a homodimer, with each monomer composed of a  glutaminase domain and a...; cd00712;dimer interface [polypeptide binding];The C-terminal domain of Asparagine Synthase B. This domain is always found associated n-terminal amidotransferase domain. Family members that contain this domain catalyse the conversion of aspartate to asparagine. Asparagine synthetase B  catalyzes the...; cd01991;Ligand Binding Site [chemical binding];Molecular Tunnel
GOHSU01	CDS	GOHSU_01_00050	complement(4334..4585)	overlap	putative peptidase S33 family protein [migap]				putative peptidase S33 family protein		unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Alpha/beta hydrolase family; pfam12697
GOHSU01	CDS	GOHSU_01_00060	4503..5492	overlap	cytochrome c oxidase subunit II [nite]	 ctaC [nite]		RER_36020 :cytochrome c oxidase subunit II : ctaC : 1.9.3.1 : Rhodococcus erythropolis PR4	cytochrome c oxidase subunit 2	ctaC	F1YHH5_9ACTO Cytochrome c oxidase subunit II OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_06215 PE=4 SV=1;ctaC gene product [Gordonia polyisoprenivorans VH2] ;Heme/copper-type cytochrome/quinol oxidases, subunit 2 [Energy production and conversion]; COG1622;Cytochrome C oxidase subunit II, periplasmic domain; cl11412
GOHSU01	CDS	GOHSU_01_00070	5500..5916		putative cytochrome c oxidase subunit IV [nite]	 ctaF [nite]		RER_36010 :putative cytochrome c oxidase subunit IV : ctaF : 1.9.3.1 : Rhodococcus erythropolis PR4	integral membrane protein		F1YHH6_9ACTO Cytochrome-c oxidase OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_06220 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];Cytochrome c oxidase subunit IV; pfam12270;KEGG: sco:SCO2154 integral membrane protein
GOHSU01	CDS	GOHSU_01_00080	6339..6917		putative transposition helper protein [nite]			RER_pREL1-02030 :putative transposition helper protein : : : Rhodococcus erythropolis PR4	transposase		C7NGC6_KYTSD DNA replication protein OS=Kytococcus sedentarius (strain ATCC 14392 / DSM 20547 / CCM 314 / 541 ) GN=Ksed_25740 PE=4 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];PFAM: IstB domain protein ATP-binding protein; SMART: AAA ATPase; KEGG: sen:SACE_2604 transposase, ISlxx1;transposase; Provisional;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif
GOHSU01	CDS	GOHSU_01_00090	complement(6886..7236)	overlap	integrase catalytic protein [migap]			RER_pREL1-02040 :putative transposase (fragment) : : : Rhodococcus erythropolis PR4	integrase catalytic protein		C8XFE2_NAKMY Integrase catalytic region OS=Nakamure lla multipartita (strain ATCC 700099 / DSM 44233 / JCM 954 3 / Y-104) GN=Namu_3601 PE=4 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];Transposase and inactivated derivatives [DNA replication, recombination, and repair];PFAM: Integrase catalytic region; KEGG: sen:SACE_6452 IS21 family transposase
GOHSU01	CDS	GOHSU_01_00100	complement(7176..7610)	overlap	integrase catalytic protein [migap]				integrase catalytic protein		C8XFE2_NAKMY Integrase catalytic region OS=Nakamure lla multipartita (strain ATCC 700099 / DSM 44233 / JCM 954 3 / Y-104) GN=Namu_3601 PE=4 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];Transposase and inactivated derivatives [DNA replication, recombination, and repair];PFAM: Integrase catalytic region; Resolvase helix-turn-helix domain protein; KEGG: sen:SACE_2605 IS21 family transposase;Integrase core domain; cl15866
GOHSU01	CDS	GOHSU_01_00110	complement(8043..9668)		menaquinol-cytochrome c reductase cytochrome b subunit [nite]	 qcrB [nite]		RER_35980 :menaquinol-cytochrome c reductase cytochrome b subunit : qcrB : : Rhodococcus erythropolis PR4	cytochrome b/b6 domain-containing protein		F1YHH7_9ACTO Cytochrome b/b6 domain-containing prot ein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06225 P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: cytochrome b/b6 domain; KEGG: gme:Gmet_2119 cytochrome b/b6-like;Cytochrome b subunit of the bc complex [Energy production and conversion]; COG1290;Cytochrome b (N-terminus)/b6/petB:  Cytochrome b is a subunit of cytochrome bc1, an 11-subunit mitochondrial respiratory enzyme. Cytochrome b spans the mitochondrial membrane with 8 transmembrane helices (A-H) in eukaryotes. In plants and cyanobacteria; cl00859;heme bH binding site [chemical binding];intrachain domain interface;heme bL binding site [chemical binding];interchain domain interface [polypeptide binding];Qo binding site
GOHSU01	CDS	GOHSU_01_00120	complement(9665..10834)		menaquinol-cytochrome c reductase iron-sulfur protein [nite]	 qcrA [nite]		RER_35970 :menaquinol-cytochrome c reductase iron-sulfur protein : qcrA : : Rhodococcus erythropolis PR4	Rieske Fe-S protein		F1YHH8_9ACTO Rieske (2Fe-2S) iron-sulfur domain-con taining protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_06230 PE=4 SV=1;Rieske Fe-S protein [Saccharomonospora marina XMU15 ];PFAM: Rieske [2Fe-2S] domain;Rieske Fe-S protein [Energy production and conversion]; COG0723;Rieske domain; a [2Fe-2S] cluster binding domain commonly found in Rieske non-heme iron oxygenase (RO) systems such as naphthalene and biphenyl dioxygenases, as well as in plant/cyanobacterial chloroplast b6f and mitochondrial cytochrome bc(1) complexes; cd03467;iron-sulfur cluster [ion binding];[2Fe-2S] cluster binding site [ion binding]
GOHSU01	CDS	GOHSU_01_00130	complement(10831..11772)		menaquinol-cytochrome c reductase cytochrome c subunit [nite]	 qcrC [nite]		RER_35960 :menaquinol-cytochrome c reductase cytochrome c subunit : qcrC : : Rhodococcus erythropolis PR4	ubiquinol-cytochrome c reductase cytochrome c subunit QcrC	qcrC	H0QV26_9ACTO Menaquinol-cytochrome c reductase cyto chrome c subunit OS=Gordonia effusa NBRC 100432 GN=qcrC PE =4 SV=1;qcrC gene product [Gordonia polyisoprenivorans VH2] ;Cytochrome c; cl11414
GOHSU01	CDS	GOHSU_01_00140	complement(11822..12418)		cytochrome c oxidase subunit III [nite]	 ctaE [nite]		RER_35950 :cytochrome c oxidase subunit III : ctaE : 1.9.3.1 : Rhodococcus erythropolis PR4	cytochrome c oxidase subunit III		F1YHI1_9ACTO Cytochrome-c oxidase OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_06245 PE=3 SV=1;cytochrome c oxidase subunit III [Mycobacterium gil vum PYR- GCK];PFAM: cytochrome c oxidase, subunit III; KEGG: rha:RHA1_ro01130 cytochrome c oxidase;Heme-copper oxidase subunit III.  Heme-copper oxidases are transmembrane protein complexes in the respiratory chains of prokaryotes and mitochondria which couple the reduction of molecular oxygen to water to, proton pumping across the membrane. The...; cd00386;Subunit I/III interface [polypeptide binding]
GOHSU01	CDS	GOHSU_01_00150	12672..13085		hypothetical protein [nite]			RER_35940 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YHI2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06250 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU01	CDS	GOHSU_01_00160	complement(13069..13308)	overlap	not assigned						
GOHSU01	CDS	GOHSU_01_00170	13190..14161	overlap	anthranilate phosphoribosyltransferase [hamap]	trpD [hamap]	MF_00211:anthranilate phosphoribosyltransferase:trpD 	RER_35930 :anthranilate phosphoribosyltransferase : trpD : 2.4.2.18 : Rhodococcus erythropolis PR4	anthranilate phosphoribosyltransferase TrpD	trpD	F1YHI3_9ACTO Anthranilate phosphoribosyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=trpD PE=3 SV=1;trpD gene product [Gordonia polyisoprenivorans VH2] ;anthranilate phosphoribosyltransferase; Provisional; PRK00188;Glycosyl transferase family, helical bundle domain; pfam02885;Glycosyl transferase family, a/b domain; pfam00591
GOHSU01	CDS	GOHSU_01_00180	complement(14199..14477)		conserved hypothetical protein [nite]			RER_35920 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	AsnC-type transcriptional regulator		F1YHI6_9ACTO AsnC-type transcriptional regulator OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06270 PE=4 SV =1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: transcription regulator, AsnC-type-like; KEGG: dps:DP0751 transcription regulator;Helix-turn-helix domains; cl00088
GOHSU01	CDS	GOHSU_01_00190	complement(14474..16273)		putative DNA polymerase III epsilon subunit [nite]	 dnaQ [nite]		RER_35910 :putative DNA polymerase III epsilon subunit : dnaQ : 2.7.7.7 : Rhodococcus erythropolis PR4	DNA polymerase III subunit epsilon		F1YHI7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06275 PE=4 SV=1;unnamed protein product [Flavobacterium johnsoniae UW101];TIGRFAM: DNA polymerase III, epsilon subunit; PFAM: Excinuclease ABC, C subunit domain protein; Exonuclease, RNase T and DNA polymerase III; SMART: Exonuclease;DEDDh 3'-5' exonuclease domain family; cd06127;substrate binding site [chemical binding];catalytic site [active];Catalytic GIY-YIG domain of nucleotide excision repair endonucleases UvrC, Cho, and similar proteins; cd10434;GIY-YIG motif/motif A;putative DNA binding site [nucleotide binding];metal binding site [ion binding]
GOHSU01	CDS	GOHSU_01_00200	16467..16730		NIF3-like protein [migap]				NIF3-like protein		F9VPX4_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_002_01280 PE=4 SV= 1;unnamed protein product [Aggregatibacter actinomyce temcomitans ANH9381];Uncharacterized conserved protein [Function unknown];NIF3 (NGG1p interacting factor 3); cl15371
GOHSU01	CDS	GOHSU_01_00210	17039..17617		NlpC/P60 family protein [nite]			ROP_08530 :NlpC/P60 family protein : : : Rhodococcus opacus B4	NlpC/P60 family protein		F1YHI9_9ACTO NLP/P60 protein OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_06285 PE=4 SV=1;NlpC/P60 family protein [Rhodococcus opacus B4];NlpC/P60 family; cl11438
GOHSU01	CDS	GOHSU_01_00220	17733..18752		NlpC/P60 family protein [nite]			ROP_08520 :NlpC/P60 family protein : : : Rhodococcus opacus B4	NLP/P60 protein;hypothetical protein		F1YHJ0_9ACTO NLP/P60 protein OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_06290 PE=4 SV=1;NLP/P60 protein [Mycobacterium gilvum PYR- GCK];Signal predicted by SignalP 2.0 HMM (Signal peptide probability0.999) with cleavage site probability 0.972 at residue 34;PFAM: NLP/P60 protein; KEGG: mkm:Mkms_5661 NLP/P60 protein;NlpC/P60 family; cl11438
GOHSU01	CDS	GOHSU_01_00230	18754..20370		hypothetical protein [migap]				hypothetical protein		F1YHJ1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06295 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: ppd:Ppro_1566 hypotheticalprotein;Peptidase Gluzincin family (thermolysin-like proteinases, TLPs) includes peptidases M1, M2, M3, M4, M13, M32 and M36 (fungalysins); cl14813
GOHSU01	CDS	GOHSU_01_00240	20370..21488		mannosyltransferase PimB' [nite]	 pimB' [nite]		RER_35870 :mannosyltransferase PimB' : pimB' : 2.4.1.- : Rhodococcus erythropolis PR4	glycoside hydrolase family protein		F1YHJ2_9ACTO Glycosyl transferase, group 1 family p rotein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_0630 0 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Glycosyltransferase [Cell envelope biogenesis, outer membrane]; COG0438;identified by match to protein family HMM PF00534;This family is most closely related to the GT1 family of glycosyltransferases and named after YqgM in Bacillus licheniformis about which little is known. Glycosyltransferases catalyze the transfer of sugar moieties from activated donor molecules to...; cd03801
GOHSU01	CDS	GOHSU_01_00250	complement(21570..23381)		long-chain fatty-acid--CoA ligase FadD15 [nite]	 fadD15 [nite]		ROP_08490 :long-chain fatty-acid--CoA ligase FadD15 : fadD15 : 6.2.1.3 : Rhodococcus opacus B4	acyl-CoA ligase		F1YHJ3_9ACTO AMP-dependent synthetase and ligase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06305 PE=4 SV =1;acyl-CoA ligase/synthetase [Rhodococcus equi 103S];Long-chain acyl-CoA synthetases (AMP-forming) [Lipid metabolism]; COG1022;AMP-binding enzyme; cl15778
GOHSU01	CDS	GOHSU_01_00260	23500..23925		hypothetical protein [nite]			ROP_08480 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GPQ2_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_40_00670 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU01	CDS	GOHSU_01_00270	23918..24355		hypothetical protein [nite]			ROP_08470 :hypothetical protein : : : Rhodococcus opacus B4	cyclase/dehydrase		F1YHJ5_9ACTO Cyclase/dehydrase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_06315 PE=4 SV=1;cyclase/dehydrase [Mycobacterium sp. JLS];PFAM: cyclase/dehydrase; KEGG: mmc:Mmcs_3280 cyclase/dehydrase;Ligand-binding SRPBCC domain of an uncharacterized subfamily of proteins; cd07819;putative hydrophobic ligand binding site [chemical binding]
GOHSU01	CDS	GOHSU_01_00280	24409..25662		putative anion-transporting ATPase [nite]			ROP_08460 :putative anion-transporting ATPase : : 3.6.3.- : Rhodococcus opacus B4	chromosome partitioning ATPase		F1YHJ6_9ACTO Chromosome partitioning ATPase OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_06320 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Predicted ATPase involved in chromosome partitioning [Cell division and chromosome partitioning]; COG0003
GOHSU01	CDS	GOHSU_01_00290	25659..26000		hypothetical protein [migap]				hypothetical protein		F1YHJ7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06325 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU01	CDS	GOHSU_01_00300	26014..27018		glucokinase [nite]	 glk [nite]		ROP_08440 :glucokinase : glk : 2.7.1.2 : Rhodococcus opacus B4	glucokinase		H5UK37_9ACTO Glucokinase OS=Gordonia terrae NBRC 10 0016 GN=glk PE=4 SV=1;glucokinase [Rhodococcus jostii RHA1];ROK family protein (putative glucokinase); TIGR00744;FGGY family of carbohydrate kinases, N-terminal domain; cl09121
GOHSU01	CDS	GOHSU_01_00310	27082..27777		putative acyltransferase [nite]			RER_35800 :putative acyltransferase : : 2.3.1.- : Rhodococcus erythropolis PR4	acyltransferase		F1YHJ9_9ACTO Phospholipid/glycerol acyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06335 PE=4 SV=1;acyltransferase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: AGPAT-like; cd07989;putative acyl-acceptor binding pocket
GOHSU01	CDS	GOHSU_01_00320	complement(27749..29041)		mannosyltransferase [nite]			ROP_08420 :mannosyltransferase : : 2.4.1.- : Rhodococcus opacus B4	polyprenol-phosphate-mannose-dependent alpha-(1-2)-phosphatidylinositol mannoside mannosyltransferase		F1YHK0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06340 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF2029); cl09812
GOHSU01	CDS	GOHSU_01_00330	29131..29616		hypothetical protein [nite]			ROP_08410 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F4CL47_PSEUX Putative uncharacterized protein OS=Ps eudonocardia dioxanivorans (strain ATCC 55486 / DSM 44775 / JCM 13855 / CB1190) GN=Psed_2776 PE=4 SV=1;unnamed protein product [Dechlorosoma suillum PS]
GOHSU01	CDS	GOHSU_01_00340	29696..31069		3-deoxy-D-arabino-heptulosonate 7-phosphate synthase [nite]	 aroH [nite]		RER_35780 :3-deoxy-D-arabino-heptulosonate 7-phosphate synthase : aroH : 2.5.1.54 : Rhodococcus erythropolis PR4	3-deoxy-D-arabinoheptulosonate-7-phosphate synthase	aroG	F1YHK2_9ACTO Phospho-2-dehydro-3-deoxyheptonate ald olase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06350 PE=4 SV=1;3-deoxy-D-arabinoheptulosonate-7-phosphate synthase [Mycobacterium marinum M];involved in chorismate biosynthesis;Class-II DAHP synthetase family; cl03230
GOHSU01	CDS	GOHSU_01_00350	complement(31128..32444)		serine/threonine protein kinase [migap]	pknL [migap]			serine/threonine protein kinase	pknL	F1YHK3_9ACTO Serine/threonine protein kinase-like p rotein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_0635 5 PE=4 SV=1;serine/threonine protein kinase [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
GOHSU01	CDS	GOHSU_01_00360	32519..32887		hypothetical protein [nite]			ROP_08380 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YHK4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06360 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.828) with cleavage site probability 0.535 at residue 20;KEGG: asa:ASA_pAsa202 hypothetical protein;Helix-Turn-Helix DNA binding domain of transcription regulators from the MerR superfamily; cl02600
##GOHSU02	contig00012				279701
GOHSU02	CDS	GOHSU_02_00010	complement(<1..886)	3x codon	phytoene synthase [nite]	 crtB [nite]		ROP_08370 :phytoene synthase : crtB : 2.5.1.32 : Rhodococcus opacus B4	phytoene synthase	crtB3	F1YHK5_9ACTO Phytoene synthase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_06365 PE=4 SV=1;phytoene synthase [Streptomyces griseus subsp. gris eus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;Trans-Isoprenyl Diphosphate Synthases, head-to-head; cd00683;active site lid residues [active];substrate binding pocket [chemical binding];catalytic residues [active];substrate-Mg2+ binding site;aspartate-rich region 1;aspartate-rich region 2
GOHSU02	CDS	GOHSU_02_00020	complement(879..2426)		mannosyltransferase MptA [nite]	 mptA [nite]		RER_35720 :mannosyltransferase MptA : mptA : 2.4.1.- : Rhodococcus erythropolis PR4	predicted protein		D0LAU8_GORB4 Carotene biosynthesis associated membr ane protein OS=Gordonia bronchialis (strain ATCC 25592 / D SM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3034 PE=4 SV=1;predicted protein [Streptomyces sp. AA4];carotene biosynthesis associated membrane protein; TIGR03459
GOHSU02	CDS	GOHSU_02_00030	complement(2423..3949)		probable phytoene dehydrogenase [nite]	 crtI [nite]		RER_35710 :probable phytoene dehydrogenase : crtI : 1.14.99.- : Rhodococcus erythropolis PR4	phytoene dehydrogenase	crtI	F1YHK8_9ACTO Phytoene dehydrogenase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_06380 PE=4 SV=1;phytoene dehydrogenase [Streptomyces griseus subsp. griseus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;phytoene desaturase; TIGR02734
GOHSU02	CDS	GOHSU_02_00040	complement(3962..5056)		probable geranylgeranyl pyrophosphate synthase [nite]	 crtE [nite]		RER_35700 :probable geranylgeranyl pyrophosphate synthase : crtE : 2.5.1.29 : Rhodococcus erythropolis PR4	geranylgeranyl pyrophosphate synthase		F1YHK9_9ACTO Polyprenyl synthetase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_06385 PE=3 SV=1;geranylgeranyl pyrophosphate synthase [Corynebacter ium glutamicum ATCC 13032];Geranylgeranyl pyrophosphate synthase [Coenzyme metabolism]; COG0142;Trans-Isoprenyl Diphosphate Synthases, head-to-tail; cd00685;substrate binding pocket [chemical binding];chain length determination region;substrate-Mg2+ binding site;catalytic residues [active];aspartate-rich region 1;active site lid residues [active];aspartate-rich region 2
GOHSU02	CDS	GOHSU_02_00050	5105..6166		5,10-methylenetetrahydrofolate reductase [nite]	 metF [nite]		ROP_08330 :5,10-methylenetetrahydrofolate reductase : metF : 1.5.1.20 : Rhodococcus opacus B4	5,10-methylenetetrahydrofolate reductase	metF	F1YHL0_9ACTO Methylenetetrahydrofolate reductase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06390 PE=3 SV =1;5,10-methylenetetrahydrofolate reductase [Corynebac terium glutamicum ATCC 13032];IS fingerprint type 4-5;Methylenetetrahydrofolate reductase; pfam02219;Methylenetetrahydrofolate reductase (MTHFR). 5,10-Methylenetetrahydrofolate is reduced to 5-methyltetrahydrofolate by methylenetetrahydrofolate reductase, a cytoplasmic, NAD(P)-dependent enzyme. 5-methyltetrahydrofolate is utilized by methionine synthase...; cd00537;FAD binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_00060	complement(6227..7246)		hypothetical protein [migap]				hypothetical protein		F1YHL1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06395 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU02	CDS	GOHSU_02_00070	complement(7321..7965)		conserved hypothetical protein [nite]			RER_35660 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	GCN5-like N-acetyltransferase		F1YHL2_9ACTO GCN5-like N-acetyltransferase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_06400 PE=4 SV=1;GCN5-like N-acetyltransferase [Verrucosispora maris AB-18- 032];COG0456 Acetyltransferases;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU02	CDS	GOHSU_02_00080	8233..8679		hypothetical protein [migap]				hypothetical protein		F1YHL3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06405 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU02	CDS	GOHSU_02_00090	8821..9189		hypothetical membrane protein [nite]			ROP_08260 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H6MSN7_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c27840 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF3040); pfam11239
GOHSU02	CDS	GOHSU_02_00100	9800..10303		protein MraZ [hamap]	mraZ [hamap]	MF_01008:protein MraZ:mraZ 	RER_35610 :conserved hypothetical protein MraZ : mraZ : : Rhodococcus erythropolis PR4	cell division protein MraZ	mraZ	F1YHL5_9ACTO Protein MraZ OS=Gordonia neofelifaecis NRRL B-59395 GN=mraZ PE=3 SV=1;cell division protein MraZ [Corynebacterium glutami cum ATCC 13032];IS fingerprint type 4-5;cell division protein MraZ; Reviewed;MraZ protein; pfam02381;MraZ; UPF0040; crystal structure shows similarity to AbrB
GOHSU02	CDS	GOHSU_02_00110	10500..11567		ribosomal RNA small subunit methyltransferase H [hamap]	rsmH [hamap]	MF_01007:ribosomal RNA small subunit methyltransferase H:rsmH 	ROP_08240 :methyltransferase MraW : mraW : 2.1.1.- : Rhodococcus opacus B4	S-adenosyl-methyltransferase MraW	mraW	F1YHL6_9ACTO Ribosomal RNA small subunit methyltran sferase H OS=Gordonia neofelifaecis NRRL B-59395 GN=rsmH P E=3 SV=1;S-adenosyl-methyltransferase MraW [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Predicted S-adenosylmethionine-dependent methyltransferase involved in cell envelope biogenesis [Cell envelope biogenesis, outer membrane];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU02	CDS	GOHSU_02_00120	11564..12403		FHA domain-containing protein [migap]				FHA domain-containing protein		F1YHL7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06425 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mkm:Mkms_3325 FHA domain-containing protein;Merozoite surface protein 1 (MSP1) C-terminus; pfam07462
GOHSU02	CDS	GOHSU_02_00130	12415..14280		peptidoglycan synthase FtsI [nite]	 ftsI [nite]		ROP_08220 :peptidoglycan synthase FtsI : ftsI : 2.4.1.129 : Rhodococcus opacus B4	peptidoglycan glycosyltransferase		D0LAT7_GORB4 Peptidoglycan glycosyltransferase OS=G ordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3 198 / NCTC 10667) GN=Gbro_3023 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: penicillin-binding protein transpeptidase; Penicillin-binding protein dimerisation domain; KEGG: gur:Gura_3980 peptidoglycan glycosyltransferase;Cell division protein FtsI/penicillin-binding protein 2 [Cell envelope biogenesis, outer membrane]; COG0768;Penicillin-binding Protein dimerisation domain; pfam03717;Penicillin binding protein transpeptidase domain; cl01009
GOHSU02	CDS	GOHSU_02_00140	14379..16469	case	UDP-N-acetylmuramoyl-L-alanyl-D-glutamate--2,6-diaminopimelate ligase [hamap]	murE [hamap]	MF_00208:UDP-N-acetylmuramoyl-L-alanyl-D-glutamate--2,6-diaminopimelate ligase:murE 	ROP_08210 :UDP-N-acetylmuramoylalanyl-D-glutamate--2,6-diaminopimelate ligase : murE : 6.3.2.13 : Rhodococcus opacus B4	UDP-N-acetylmuramoylalanyl-D-glutamate--2, 6-diaminopimelate ligase	murE	F1YHL9_9ACTO UDP-N-acetylmuramoylalanyl-D-glutamate --2, 6-diaminopimelate ligase OS=Gordonia neofelifaecis NR RL B-59395 GN=murE PE=3 SV=1;UDP-N-acetylmuramoylalanyl-D-glutamate--2, 6-diamin opimelate ligase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;involved in cell wall formation; peptidoglycan synthesis; cytoplasmic enzyme; catalyzes the addition of meso-diaminopimelic acid to the nucleotide precursor UDP-N-aceylmuramoyl-l-alanyl-d-glutamate;UDP-N-acetylmuramoylalanyl-D-glutamate--2, 6-diaminopimelate ligase; Provisional; PRK00139;Mur ligase family, catalytic domain; pfam01225;Mur ligase middle domain; pfam08245;Mur ligase family, glutamate ligase domain; pfam02875
GOHSU02	CDS	GOHSU_02_00150	16594..17697	case	phospho-N-acetylmuramoyl-pentapeptide-transferase [hamap]	mraY [hamap]	MF_00038:phospho-N-acetylmuramoyl-pentapeptide-transferase:mraY 	ROP_08190 :phospho-N-acetylmuramoyl-pentapeptide-transferase : mraY : 2.7.8.13 : Rhodococcus opacus B4	phospho-N-acetylmuramoyl-pentapeptide- transferase	mraY	F1YHM0_9ACTO Phospho-N-acetylmuramoyl-pentapeptide- transferase OS=Gordonia neofelifaecis NRRL B-59395 GN=mraY PE=3 SV=1;phospho-N-acetylmuramoyl-pentapeptide- transferase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;First step of the lipid cycle reactions in the biosynthesis of the cell wall peptidoglycan;phospho-N-acetylmuramoyl-pentapeptide-transferase; TIGR00445;Phospho-N-acetylmuramoyl-pentapeptide-transferase (mraY) is an enzyme responsible for the formation of the first lipid intermediate in the synthesis of bacterial cell wall peptidoglycan. It catalyzes the formation of...; cd06852;Mg++ binding site [ion binding];putative catalytic motif [active];putative substrate binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_00160	17784..19919		cell division protein FtsW [migap]				cell division protein FtsW		H5TI00_9ACTO Cell division protein FtsW OS=Gordonia otitidis NBRC 100426 GN=ftsW PE=4 SV=1;cell division protein FtsW [Streptomyces sp. AA4];Cell cycle protein; cl00511
GOHSU02	CDS	GOHSU_02_00170	19916..21088		UDP-N-acetylglucosamine--N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetylglucosamine transferase [hamap]	murG [hamap]	MF_00033:UDP-N-acetylglucosamine--N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetylglucosamine transferase:murG 	RER_35520 :UDP-N-acetylglucosamine--N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetylglucosamine transferase : murG : 2.4.1.227 : Rhodococcus erythropolis PR4	UDP-N-acetylglucosamine-N-acetylmuramyl- (pentapeptide) pyrophosphoryl-undecaprenol N- acetylglucosamine transferase	murG	H5U039_9ACTO UDP-N-acetylglucosamine--N-acetylmuram yl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetylgluc osamine transferase OS=Gordonia sputi NBRC 100414 GN=murG PE=3 SV=1;murG gene product [Gordonia polyisoprenivorans VH2] ;undecaprenyldiphospho-muramoylpentapeptide beta-N- acetylglucosaminyltransferase; Provisional; PRK00726;MurG is an N-acetylglucosaminyltransferase, the last enzyme involved in the intracellular phase of peptidoglycan biosynthesis. It transfers N-acetyl-D-glucosamine (GlcNAc) from UDP-GlcNAc to the C4 hydroxyl of a lipid-linked N-acetylmuramoyl pentapeptide...; cd03785;homodimer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_00180	21085..22554		UDP-N-acetylmuramate--L-alanine ligase [hamap]	murC [hamap]	MF_00046:UDP-N-acetylmuramate--L-alanine ligase:murC 	ROP_08150 :UDP-N-acetylmuramate--L-alanine ligase : murC : 6.3.2.8 : Rhodococcus opacus B4	UDP-N-acetylmuramate--L-alanine ligase	murC	F1YHM4_9ACTO UDP-N-acetylmuramate--L-alanine ligase OS=Gordonia neofelifaecis NRRL B-59395 GN=murC PE=3 SV=1;UDP-N-acetylmuramate--L-alanine ligase [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;Catalyzes the formation of UDP-N-acetylmuramoyl-L-alanine from UDP-N-acetylmuramate and L-alanine in peptidoglycan synthesis;UDP-N-acetylmuramate--L-alanine ligase; Provisional; PRK00421;Mur ligase family, catalytic domain; pfam01225;Mur ligase middle domain; pfam08245;Mur ligase family, glutamate ligase domain; pfam02875
GOHSU02	CDS	GOHSU_02_00190	22551..23372		putative cell division protein FtsQ [nite]	 ftsQ [nite]		ROP_08140 :putative cell division protein FtsQ : ftsQ : : Rhodococcus opacus B4	polypeptide-transport-associated domain-containing protein		F1YHM5_9ACTO Cell division protein FtsQ OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06465 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.923) with cleavage site probability 0.761 at residue 30;PFAM: polypeptide-transport-associated domain protein FtsQ-type; KEGG: ppd:Ppro_3285 polypeptide-transport- associated domain-containing protein;cell division protein FtsQ; Provisional;POTRA domain, FtsQ-type; pfam08478
GOHSU02	CDS	GOHSU_02_00200	23547..24713		cell division protein FtsZ [nite]	 ftsZ [nite]		ROP_08130 :cell division protein FtsZ : ftsZ : : Rhodococcus opacus B4	cell division protein FtsZ		D0LAS9_GORB4 Cell division protein ftsZ OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / N CTC 10667) GN=Gbro_3014 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: cell division protein FtsZ; PFAM: Tubulin/FtsZ GTPase; Tubulin/FtsZ, 2-layer sandwich domain; KEGG: sat:SYN_00437 cell division protein FtsZ;cell division protein FtsZ; Validated;FtsZ is a GTPase that is similar to the eukaryotic tubulins and is essential for cell division in prokaryotes.  FtsZ is capable of polymerizing in a GTP-driven process into structures similar to those formed by tubulin. FtsZ forms a ring-shaped septum at...; cd02201;nucleotide binding site [chemical binding];SulA interaction site
GOHSU02	CDS	GOHSU_02_00210	24713..25459		hypothetical protein [nite]			ROP_08120 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YHM7_9ACTO Multicopper polyphenol oxidase OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_06475 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: mch:Mchl_3109 protein of unknown function DUF152;Multicopper polyphenol oxidase (laccase) [Secondary metabolites biosynthesis, transport and catabolism]; COG1496;Multi-copper polyphenol oxidoreductase laccase; cl00650
GOHSU02	CDS	GOHSU_02_00220	25456..26193		conserved hypothetical protein [nite]			RER_35470 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	pyridoxal phosphate enzyme, YggS family		H5U044_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_058_00550 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzymes; cl00261;pyridoxal 5'-phosphate (PLP) binding site [chemical binding];catalytic residue [active]
GOHSU02	CDS	GOHSU_02_00230	26248..26991		cell division protein SepF [hamap]	sepF [hamap]	MF_01197:cell division protein SepF:sepF 	ROP_08100 :hypothetical protein : : : Rhodococcus opacus B4	cell division protein sepF 2		F1YHM9_9ACTO Cell division protein sepF OS=Gordonia neofelifaecis NRRL B-59395 GN=sepF PE=3 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG1799 Uncharacterized protein conserved in bacteria;Protein of unknown function (DUF552); cl00775
GOHSU02	CDS	GOHSU_02_00240	27051..27383		hypothetical membrane protein [nite]			ROP_08090 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		G7GPL3_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_40_00280 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU02	CDS	GOHSU_02_00250	27642..28430		DivIVA family protein [nite]			ROP_08080 :DivIVA family protein : : : Rhodococcus opacus B4	DivIVA domain-containing protein		F1YHN1_9ACTO DivIVA domain-containing protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06495 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: DivIVA domain; PFAM: DivIVA family protein; KEGG: mxa:MXAN_5055 hypothetical protein;DivIVA domain; TIGR03544;Cell division initiation protein [Cell division and chromosome partitioning]; COG3599
GOHSU02	CDS	GOHSU_02_00260	28465..29505		hypothetical protein [migap]				hypothetical protein		F1YHN2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06500 PE=4 SV=1;integral membrane protein [Rhodococcus equi 103S]
GOHSU02	CDS	GOHSU_02_00270	29806..33018		isoleucyl-tRNA synthetase [hamap]	ileS [hamap]	MF_02003:isoleucyl-tRNA synthetase:ileS 	RER_35420 :isoleucyl-tRNA synthetase : ileS : 6.1.1.5 : Rhodococcus erythropolis PR4	isoleucyl-tRNA synthetase	ileS	F1YHN3_9ACTO Isoleucine--tRNA ligase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=ileS PE=3 SV=1;isoleucyl-tRNA synthetase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;IleRS; catalyzes the formation of isoleucyl-tRNA(Ile) from isoleucine and tRNA(Ile); since isoleucine and other amino acids such as valine are similar, there are additional editing function in this enzyme; one is involved in hydrolysis of activated valine-AMP and the other is involved in deacylation of mischarged Val-tRNA(Ile); there are two active sites, one for aminoacylation and one for editing; class-I aminoacyl-tRNA synthetase type 2 subfamily; some organisms carry two different copies of this enzyme; in some organisms, the type 2 subfamily is associated with resistance to the antibiotic pseudomonic acid (mupirocin);isoleucyl-tRNA synthetase; Reviewed; PRK06039;catalytic core domain of isoleucyl-tRNA synthetases; cd00818;HIGH motif;KMSKS motif;Anticodon-binding domain of archaeal, bacterial, and eukaryotic cytoplasmic isoleucyl tRNA synthetases; cd07961;tRNA binding surface [nucleotide binding];anticodon binding site
GOHSU02	CDS	GOHSU_02_00280	33015..33872		phosphoesterase PA-phosphatase related protein [migap]				phosphoesterase PA-phosphatase related protein		G7HVS6_9CORY PAP2 family protein OS=Corynebacterium casei UCMA 3821 GN=CCAS_03420 PE=4 SV=1;phosphoesterase PA-phosphatase related protein [Str eptomyces flavogriseus ATCC 33331];PFAM: phosphoesterase PA-phosphatase related; KEGG: sma:SAV_31 prepilin peptidase-like protein;PAP2_like proteins, a super-family of histidine phosphatases and vanadium haloperoxidases, includes type 2 phosphatidic acid phosphatase or lipid phosphate phosphatase (LPP), Glucose-6-phosphatase, Phosphatidylglycerophosphatase B and bacterial acid...; cl00474
GOHSU02	CDS	GOHSU_02_00290	complement(33877..34656)		conserved hypothetical protein [nite]			RER_21500 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	GDSL family lipase		F1YHN4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06510 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];PFAM: lipolytic enzyme, G-D-S-L family; KEGG: lxx:Lxx11940 hypothetical protein;Members of the SGNH-hydrolase superfamily, a diverse family of lipases and esterases. The tertiary fold of the enzyme is substantially different from that of the alpha/beta hydrolase family and unique among all known hydrolases; its active site closely...; cd01832;catalytic triad [active];oxyanion hole [active]
GOHSU02	CDS	GOHSU_02_00300	35016..35216		hypothetical protein [migap]				hypothetical protein		A1T6G1_MYCVP Putative uncharacterized protein OS=My cobacterium vanbaalenii (strain DSM 7251 / PYR-1) GN=Mvan_ 1941 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU02	CDS	GOHSU_02_00310	complement(35238..35936)		putative TetR family transcriptional regulator [nite]			ROP_34510 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		H0R2M2_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia effusa NBRC 100432 GN=GOEFS_081_00120 PE=4 SV=1;TetR family transcriptional regulator [Amycolatopsi s mediterranei U32];Helix-turn-helix domains; cl00088
GOHSU02	CDS	GOHSU_02_00320	36129..36563		hypothetical protein [nite]			YM304_01920 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	deazaflavin-dependent nitroreductase family protein		H0R2M1_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_081_00110 PE=4 SV=1;deazaflavin-dependent nitroreductase family protein [Saccharomonospora marina XMU15];PFAM: Domain of unknown function (DUF385); TIGRFAM: deazaflavin-dependent nitroreductase family protein;Domain of unknown function (DUF385); cl04387
GOHSU02	CDS	GOHSU_02_00330	36602..38011	case	DNA polymerase IV [hamap]	dinB [hamap]	MF_01113:DNA polymerase IV:dinB 	ROP_08050 :DNA polymerase IV : dinB : 2.7.7.7 : Rhodococcus opacus B4	DNA polymerase IV	dinP	F1YHN6_9ACTO DNA polymerase IV 1 OS=Gordonia neofel ifaecis NRRL B-59395 GN=dinB1 PE=3 SV=1;DNA polymerase IV [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;DNA polymerase IV; Provisional;involved in translesion DNA polymerization with beta clamp of polymerase III; belongs to Y family of polymerases; does not contain proofreading function;DNA Polymerase IV/Kappa; cd03586;DNA binding site [nucleotide binding]
GOHSU02	CDS	GOHSU_02_00340	complement(38065..39003)		putative L-asparaginase [nite]			RER_56400 :putative L-asparaginase : : 3.5.1.1 : Rhodococcus erythropolis PR4	L-asparaginase	ansA1	F1YHN7_9ACTO Asparaginase/glutaminase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_06525 PE=4 SV=1;L-asparaginase [Corynebacterium ulcerans BR- AD22];L-asparaginase/archaeal Glu-tRNAGln amidotransferase subunit D;Type II (periplasmic) bacterial L-asparaginase; cd08964;homotetramer interface [polypeptide binding];homodimer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_00350	39184..39810		hypothetical protein [nite]			ROP_08040 :hypothetical protein : : : Rhodococcus opacus B4	lipoprotein		F1YHN8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06530 PE=4 SV=1;lipoprotein [Rhodococcus equi 103S]
GOHSU02	CDS	GOHSU_02_00360	complement(39892..40851)		hypothetical protein [migap]				hypothetical protein		F1YHN9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06535 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.979) with cleavage site probability 0.595 at residue 33;KEGG: scl:sce2052 hypothetical protein
GOHSU02	CDS	GOHSU_02_00370	40959..41642		lipoprotein signal peptidase [hamap]	lspA [hamap]	MF_00161:lipoprotein signal peptidase:lspA 	RER_35350 :signal peptidase II : lspA : 3.4.23.36 : Rhodococcus erythropolis PR4	lipoprotein signal peptidase		F9VQ29_9ACTO Lipoprotein signal peptidase OS=Gordon ia alkanivorans NBRC 16433 GN=lspA PE=3 SV=1;lipoprotein signal peptidase [Streptomyces clavulig erus ATCC 27064];Signal peptidase (SPase) II; cl00458;lipoprotein signal peptidase; Provisional
GOHSU02	CDS	GOHSU_02_00380	41635..42561		putative pseudouridine synthase RluD [nite]	 rluD [nite]		RER_35340 :putative pseudouridine synthase RluD : rluD : : Rhodococcus erythropolis PR4	pseudouridine synthase		F1YHP1_9ACTO Pseudouridine synthase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_06545 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hha:Hhal_2235 RluA family pseudouridine synthase; TIGRFAM: pseudouridine synthase; PFAM: pseudouridine synthase; RNA-binding S4 domain protein; SMART: RNA-binding S4 domain protein;pseudouridine synthase, RluA family; TIGR00005;PseudoU_synth_RsuA/RluD: Pseudouridine synthase, RsuA/RluD family. This group is comprised of eukaryotic, bacterial and archeal proteins similar to eight site specific Escherichia coli pseudouridine synthases: RsuA, RluA, RluB, RluC, RluD, RluE, RluF and...; cd02869
GOHSU02	CDS	GOHSU_02_00390	42712..46251		DNA polymerase III alpha subunit [nite]	 dnaE [nite]		RER_35310 :DNA polymerase III alpha subunit : dnaE : 2.7.7.7 : Rhodococcus erythropolis PR4	DNA polymerase III subunit alpha	dnaE	F1YHP2_9ACTO DNA polymerase III subunit alpha OS=Go rdonia neofelifaecis NRRL B-59395 GN=dnaE PE=4 SV=1;DNA polymerase III subunit alpha [Corynebacterium g lutamicum ATCC 13032];catalyzes DNA-template-directed extension of the 3'- end of a DNA strand by one nucleotide at a time; main replicative polymerase;DNA polymerase III subunit alpha; Validated; PRK05673;Bacterial DNA polymerase III alpha subunit; pfam07733;DnaE_OBF: A subfamily of OB folds corresponding to the C-terminal OB-fold nucleic acid binding domain of Thermus aquaticus and Escherichia coli type C replicative DNA polymerase III alpha subunit (DnaE). The DNA polymerase holoenzyme of E. coli contains...; cd04485;generic binding surface II;generic binding surface I
GOHSU02	CDS	GOHSU_02_00400	46273..46890		hypothetical protein [nite]			KSE_52820 :hypothetical protein : : : Kitasatospora setae NBRC 14216	impact family member in pol 5'region		E8PP17_THESS Impact family member in pol 5'region O S=Thermus scotoductus (strain ATCC 700910 / SA-01) GN=TSC_ c09380 PE=4 SV=1;unnamed protein product [Thermus scotoductus SA- 01 ];fragment;Uncharacterized conserved protein [Function unknown];Uncharacterized protein family UPF0029; pfam01205;Domain of unknown function (DUF1949); pfam09186
GOHSU02	CDS	GOHSU_02_00410	46960..47223		hypothetical protein [migap]				hypothetical protein		F1YHP3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06555 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU02	CDS	GOHSU_02_00420	47192..48574		threonine dehydratase [nite]	 ilvA [nite]		ROP_07930 :threonine dehydratase : ilvA : 4.3.1.19 : Rhodococcus opacus B4	threonine dehydratase	ilvA	F1YHP4_9ACTO Threonine dehydratase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_06560 PE=4 SV=1;threonine dehydratase [Corynebacterium glutamicum A TCC 13032];IS fingerprint type 4-5;catalyzes the formation of 2-oxobutanoate from L-threonine; biosynthetic;threonine dehydratase; Validated;Threonine dehydratase: The first step in amino acid degradation is the removal of nitrogen. Although the nitrogen atoms of most amino acids are transferred to alpha-ketoglutarate before removal, the alpha-amino group of threonine can be directly...; cd01562;tetramer interface [polypeptide binding];pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active];Second of two tandem C-terminal ACT domains of threonine dehydratase I (ThrD-I; L-threonine hydrolyase); cd04907;putative Ile/Val binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_00430	48675..49451		NAD-dependent deacetylase [hamap]	cobB [hamap]	MF_01121:NAD-dependent deacetylase:cobB 	PH0947 :NAD-dependent deacetylase : npdA : 3.5.1.- : Pyrococcus horikoshii OT3	NAD-dependent deacetylase		H0IDX7_9MYCO NAD-dependent deacetylase (Regulatory protein Sir2-like protein) OS=Mycobacterium massiliense CC UG 48898 GN=MMAS_33140 PE=3 SV=1;NAD-dependent deacetylase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Modulates the activities of several enzymes which are inactive in their acetylated form;NAD-dependent deacetylase; Provisional;SIRT5_Af1_CobB: Eukaryotic, archaeal and prokaryotic group (class3) which includes human sirtuin SIRT5, Archaeoglobus fulgidus Sir2-Af1, and E. coli CobB; and are members of the SIR2 family of proteins, silent information regulator 2 (Sir2) enzymes which...; cd01412;NAD+ binding site [chemical binding];substrate binding site [chemical binding];Zn binding site [ion binding]
GOHSU02	CDS	GOHSU_02_00440	49727..50593		putative YrbE family protein [nite]	 yrbE4A [nite]		RER_17230 :putative YrbE family protein : yrbE4A : : Rhodococcus erythropolis PR4	YrbE family protein	yrbE6A	F1YHP5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06565 PE=4 SV=1;YrbE family protein [Nocardia farcinica IFM 10152];Permease; cl00510
GOHSU02	CDS	GOHSU_02_00450	50595..51461		putative YrbE family protein [nite]	 yrbE1B [nite]		ROP_16500 :putative YrbE family protein : yrbE1B : : Rhodococcus opacus B4	YrbE family protein	yrbE6B	F1YHP6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06570 PE=4 SV=1;YrbE family protein [Nocardia farcinica IFM 10152];Permease; cl00510
GOHSU02	CDS	GOHSU_02_00460	51468..52862		putative Mce family protein [nite]	 mce1A [nite]		RER_06770 :putative Mce family protein : mce1A : : Rhodococcus erythropolis PR4	putative virulence factor, Mce family protein		F1YHP7_9ACTO Mammalian cell entry related domain-co ntaining protein OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_06575 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU02	CDS	GOHSU_02_00470	52859..53896		putative Mce family protein [nite]	 mce1B [nite]		ROP_16480 :putative Mce family protein : mce1B : : Rhodococcus opacus B4	Mce family protein	mce4B	F1YHP8_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06580 PE=4 SV =1;Mce family protein [Nocardia farcinica IFM 10152];virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU02	CDS	GOHSU_02_00480	53896..54888		putative Mce family protein [nite]	 mce6C [nite]		ROP_61770 :putative Mce family protein : mce6C : : Rhodococcus opacus B4	virulence factor Mce family protein		H0R167_9ACTO Mce family protein OS=Gordonia effusa NBRC 100432 GN=mceC PE=4 SV=1;virulence factor Mce family protein [Saccharomonosp ora marina XMU15];Signal predicted by SignalP 3.0 HMM; IMG reference gene:2508554505_SP;PFAM: mce related protein; TIGRFAM: virulence factor Mce family protein; overlaps another CDS with the same product name;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU02	CDS	GOHSU_02_00490	54882..55991		putative Mce family protein [nite]	 mce4D [nite]		RER_17280 :putative Mce family protein : mce4D : : Rhodococcus erythropolis PR4	Mce family protein	mce6D	H5UF36_9ACTO Mce family protein OS=Gordonia terrae NBRC 100016 GN=mceD PE=4 SV=1;Mce family protein [Nocardia farcinica IFM 10152];mce related protein; pfam02470;virulence factor Mce family protein; TIGR00996
GOHSU02	CDS	GOHSU_02_00500	55988..57094		putative Mce family protein [nite]	 mce1E [nite]		RER_06730 :putative Mce family protein : mce1E : : Rhodococcus erythropolis PR4	Mce family protein	mce6E	F1YHQ1_9ACTO Mammalian cell entry related domain-co ntaining protein OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_06595 PE=4 SV=1;Mce family protein [Nocardia farcinica IFM 10152];virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU02	CDS	GOHSU_02_00510	57095..58252		putative Mce family protein [nite]	 mce4F [nite]		ROP_46860 :putative Mce family protein : mce4F : : Rhodococcus opacus B4	Mce family protein Mce5F	mce5F	F1YHQ2_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06600 PE=4 SV =1;Mce family protein Mce5F [Mycobacterium sp. JDM601] ;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU02	CDS	GOHSU_02_00520	58319..59149		hypothetical protein [migap]				hypothetical protein		F9VY61_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_084_00190 PE=4 SV= 1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU02	CDS	GOHSU_02_00530	complement(59188..61452)		malto-oligosyltrehalose synthase [nite]	 treY [nite]		RER_35070 :malto-oligosyltrehalose synthase : treY : 5.4.99.15 : Rhodococcus erythropolis PR4	maltooligosyl trehalose synthase	treY	F1YHQ4_9ACTO Malto-oligosyltrehalose synthase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06610 PE=4 SV=1;maltooligosyl trehalose synthase [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;Maltooligosyl trehalose synthase [Carbohydrate transport and metabolism]; COG3280;Alpha amylase catalytic domain found in maltooligosyl trehalose synthase (MTSase); cd11336;catalytic site [active]
GOHSU02	CDS	GOHSU_02_00540	complement(61521..62087)		putative TetR family transcriptional regulator [nite]			ROP_07710 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		G7GPI4_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia amarae NBRC 15530 GN=GOAMR_39_00020 P E=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: regulatory protein TetR; Tetracyclin repressor domain protein; KEGG: nfa:nfa18310  transcriptional regulator;tetracycline repressor protein TetR; Provisional;Helix-turn-helix domains; cl00088;Tetracyclin repressor, C-terminal all-alpha domain; pfam02909
GOHSU02	CDS	GOHSU_02_00550	62164..63081		putative esterase [nite]			YM304_23370 :putative esterase : : : Acidimicrobidae bacterium YM16-304	alpha/beta hydrolase fold protein		C0STW9_9NOCA Putative hydrolase OS=Rhodococcus sp. HI-31 GN=chnC2 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: pla:Plav_1674 alpha/beta hydrolase fold;short chain dehydrogenase; Validated;alpha/beta hydrolase fold; pfam00561;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU02	CDS	GOHSU_02_00560	63201..64520		adenosylmethionine-8-amino-7-oxononanoate aminotransferase [nite]	 bioA [nite]		ROP_07830 :adenosylmethionine-8-amino-7-oxononanoate aminotransferase : bioA : 2.6.1.62 : Rhodococcus opacus B4	adenosylmethionine-8-amino-7-oxononanoate aminotransferase		H5U077_9ACTO Adenosylmethionine-8-amino-7-oxononano ate aminotransferase OS=Gordonia sputi NBRC 100414 GN=bioA PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: adenosylmethionine-8-amino-7-oxononanoate aminotransferase; PFAM: aminotransferase class-III; KEGG: abo:ABO_0256 7,8-diaminononanoate transaminase;adenosylmethionine--8-amino-7-oxononanoate transaminase; Provisional;Acetyl ornithine aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). The major groups in this CD correspond to ornithine aminotransferase, acetylornithine aminotransferase; cd00610;inhibitor-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU02	CDS	GOHSU_02_00570	64517..65731		8-amino-7-oxononanoate synthase [nite]	 bioF [nite]		RER_35040 :8-amino-7-oxononanoate synthase : bioF : 2.3.1.47 : Rhodococcus erythropolis PR4	8-amino-7-oxononanoate synthase	bioF	D0LA17_GORB4 8-amino-7-oxononanoate synthase OS=Gor donia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 319 8 / NCTC 10667) GN=Gbro_2974 PE=4 SV=1;bioF gene product [Streptomyces cattleya NRRL 8057] ;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;Aminotransferase class I and II; pfam00155;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU02	CDS	GOHSU_02_00580	65728..66450		ATP-dependent dethiobiotin synthetase BioD [hamap]	bioD [hamap]	MF_00336:ATP-dependent dethiobiotin synthetase BioD:bioD 	RER_35030 :dethiobiotin synthetase : bioD : 6.3.3.3 : Rhodococcus erythropolis PR4	dethiobiotin synthetase	bioD	H5U079_9ACTO ATP-dependent dethiobiotin synthetase BioD OS=Gordonia sputi NBRC 100414 GN=bioD PE=3 SV=1;bioD gene product [Streptomyces cattleya NRRL 8057] ;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;AAA domain; pfam13500
GOHSU02	CDS	GOHSU_02_00590	66447..67481		biotin synthase [nite]	 bioB [nite]		ROP_07700 :biotin synthase : bioB : 2.8.1.6 : Rhodococcus opacus B4	biotin synthase	bioB	D5UNP6_TSUPD Biotin synthase OS=Tsukamurella paurom etabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 1 6120 / NCTC 13040) GN=Tpau_2002 PE=3 SV=1;biotin synthase [Corynebacterium glutamicum ATCC 13 032];IS fingerprint type 4-5;biotin synthase; Validated;catalyzes the formation of biotin from dethiobiotin and sulfur 2 S-adenosyl-L-methionine;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site;Biotin and Thiamin Synthesis associated domain; cl06149
GOHSU02	CDS	GOHSU_02_00600	67582..67818		hypothetical protein [nite]			ROP_07690 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		D0LA14_GORB4 Putative uncharacterized protein (Prec ursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43 247 / JCM 3198 / NCTC 10667) GN=Gbro_2971 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];NADH pyrophosphatase zinc ribbon domain; pfam09297
GOHSU02	CDS	GOHSU_02_00610	67815..68612		hypothetical protein [nite]			RER_34860 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0R1X3_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_073_00940 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.909 at residue 30;Protein of unknown function (DUF2567); pfam10821
GOHSU02	CDS	GOHSU_02_00620	complement(68575..69282)		conserved hypothetical protein [nite]			RER_34820 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	NUDIX hydrolase		F1YHR2_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_06650 PE=4 SV=1;unnamed protein product [Acaryochloris marina MBIC1 1017];type strain of Acaryochloris marina;Nudix hydrolase is a superfamily of enzymes found in all three kingdoms of life, and it catalyzes the hydrolysis of NUcleoside DIphosphates linked to other moieties, X. Enzymes belonging to this superfamily require a divalent cation, such as Mg2+ or Mn2+...; cl00447;nudix motif
GOHSU02	CDS	GOHSU_02_00630	69425..70492		quinolinate synthase A [hamap]	nadA [hamap]	MF_00568:quinolinate synthase A:nadA 	ROP_07630 :quinolinate synthetase A : nadA : : Rhodococcus opacus B4	quinolinate synthetase nadA		F1YHR3_9ACTO Quinolinate synthase A OS=Gordonia neo felifaecis NRRL B-59395 GN=nadA PE=3 SV=1;quinolinate synthetase nadA [Mycobacterium tubercul osis T17];Quinolinate synthetase A protein; cl00420
GOHSU02	CDS	GOHSU_02_00640	70497..71390		quinolinate phosphoribosyltransferase [nite]	 nadC [nite]		RER_34790 :quinolinate phosphoribosyltransferase : nadC : 2.4.2.19 : Rhodococcus erythropolis PR4	nicotinate-nucleotide pyrophosphorylase	nadC	F1YHR4_9ACTO Nicotinate-nucleotide pyrophosphorylas e OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06660 PE= 4 SV=1;nadC gene product [Gordonia polyisoprenivorans VH2] ;nicotinate-nucleotide pyrophosphorylase; Provisional;Quinolinate phosphoribosyl transferase (QAPRTase or QPRTase), also called nicotinate-nucleotide pyrophosphorylase, is involved in the de novo synthesis of NAD in both prokaryotes and eukaryotes. It catalyses the reaction of quinolinic acid (QA) with...; cd01572;dimerization interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_00650	71423..72307		hypothetical protein [migap]				hypothetical protein		H5TM18_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_112_00100 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU02	CDS	GOHSU_02_00660	72279..72887		not assigned						
GOHSU02	CDS	GOHSU_02_00670	complement(73045..74427)		hypothetical protein [nite]			ROP_14020 :hypothetical protein : : : Rhodococcus opacus B4	IS1096, tnpR protein		B8HJH8_ARTCA Plasmid pRiA4b ORF-3 family protein OS =Arthrobacter chlorophenolicus (strain A6 / ATCC 700700 / DSM 12829 / JCM 12360) GN=Achl_4625 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by similarity to GP:150004; match to protein family HMM PF07929;Plasmid pRiA4b ORF-3-like protein; pfam07929
GOHSU02	CDS	GOHSU_02_00680	complement(74512..75081)		not assigned						
GOHSU02	CDS	GOHSU_02_00690	75235..76557		histidinol dehydrogenase [hamap]	hisD [hamap]	MF_01024:histidinol dehydrogenase:hisD 	RER_34520 :histidinol dehydrogenase : hisD : 1.1.1.23 : Rhodococcus erythropolis PR4	histidinol dehydrogenase	hisD	F1YH09_9ACTO Histidinol dehydrogenase OS=Gordonia n eofelifaecis NRRL B-59395 GN=hisD PE=3 SV=1;histidinol dehydrogenase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;catalyzes the oxidation of L-histidinol to L-histidinaldehyde and then to L-histidine in histidine biosynthesis; functions as a dimer;Histidinol dehydrogenase; pfam00815;Histidinol dehydrogenase, HisD, E.C 1.1.1.23. Histidinol dehydrogenase catalyzes the last two steps in the L-histidine biosynthesis pathway, which is conserved in bacteria, archaea, fungi, and plants. These last two steps are (i) the NAD-dependent...; cd06572;NAD binding site [chemical binding];dimerization interface [polypeptide binding];product binding site;substrate binding site [chemical binding];zinc binding site [ion binding];catalytic residues [active]
GOHSU02	CDS	GOHSU_02_00700	76554..77678		histidinol-phosphate aminotransferase [hamap]	hisC [hamap]	MF_01023:histidinol-phosphate aminotransferase:hisC 	ROP_07560 :histidinol-phosphate aminotransferase : hisC : 2.6.1.9 : Rhodococcus opacus B4	histidinol-phosphate aminotransferase	hisC	D0LA02_GORB4 Histidinol-phosphate aminotransferase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / J CM 3198 / NCTC 10667) GN=hisC PE=3 SV=1;histidinol-phosphate aminotransferase [Corynebacter ium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of L-histidinol phosphate from imidazole-acetol phosphate and glutamate in histidine biosynthesis;histidinol-phosphate aminotransferase; Provisional;Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU02	CDS	GOHSU_02_00710	77675..78304		imidazoleglycerol-phosphate dehydratase [hamap]	hisB [hamap]	MF_00076:imidazoleglycerol-phosphate dehydratase:hisB 	ROP_07550 :imidazoleglycerol-phosphate dehydratase : hisB : 4.2.1.19 : Rhodococcus opacus B4	imidazoleglycerol-phosphate dehydratase		F1YH11_9ACTO Imidazoleglycerol-phosphate dehydratas e OS=Gordonia neofelifaecis NRRL B-59395 GN=hisB PE=3 SV=1 ;imidazoleglycerol-phosphate dehydratase [Mycobacter ium tuberculosis T92];Imidazoleglycerol-phosphate dehydratase; cd07914;imidazoleglycerol-phosphate dehydratase; Validated; PRK00951;4-fold oligomerization interface [polypeptide binding];putative active site pocket [active];metal binding residues [ion binding];3-fold/trimer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_00720	78301..78942		imidazole glycerol phosphate synthase subunit HisH [hamap]	hisH [hamap]	MF_00278:imidazole glycerol phosphate synthase subunit HisH:hisH 	KSE_20660 :putative imidazole glycerol phosphate synthase subunit HisH : hisH : 2.4.2.- : Kitasatospora setae NBRC 14216	imidazole glycerol phosphate synthase subunit HisH	hisH	F1YH12_9ACTO Imidazole glycerol phosphate synthase subunit HisH OS=Gordonia neofelifaecis NRRL B-59395 GN=his H PE=3 SV=1;imidazole glycerol phosphate synthase subunit HisH [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;with HisF IGPS catalyzes the conversion of phosphoribulosyl-formimino-5-aminoimidazole-4-carboxamide ribonucleotide phosphate and glutamine to imidazole-glycerol phosphate, 5-aminoimidazol-4-carboxamide ribonucleotide, and glutamate in histidine biosynthesis; the HisH subunit provides the glutamine amidotransferase activity that produces the ammonia necessary to HisF for the synthesis of imidazole-glycerol phosphate and 5-aminoimidazol-4-carboxamide ribonucleotide;imidazole glycerol phosphate synthase subunit HisH; Provisional; PRK13146;Type 1 glutamine amidotransferase (GATase1) domain found in imidazole glycerol phosphate synthase (IGPS); cd01748;putative active site [active];oxyanion strand;catalytic triad [active]
GOHSU02	CDS	GOHSU_02_00730	78939..79205		diguanylate cyclase/phosphodiesterase [migap]				diguanylate cyclase/phosphodiesterase		F1YH13_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06800 PE=4 SV=1;unnamed protein product [Burkholderia phymatum STM8 15];TIGRFAM: diguanylate cyclase; PFAM: GGDEF domain containing protein; EAL domain protein; KEGG: bxe:Bxe_B1411 putative diguanylate cyclase/phosphodiesterase (GGDEF & EAL domains);Predicted signal transduction protein containing a membrane domain, an EAL and a GGDEF domain [Signal transduction mechanisms]
GOHSU02	CDS	GOHSU_02_00740	79202..79963	case	1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino] imidazole-4-carboxamide isomerase [hamap]	hisA [hamap]	MF_01014:1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino] imidazole-4-carboxamide isomerase:hisA 	ROP_07500 :1-(5-phosphoribosyl)-5-[(5-phosphoribosylamino)methylideneamino] imidazole-4-carboxamide isomerase : hisA : 5.3.1.16 : Rhodococcus opacus B4	bifunctional HisA/TrpF protein		F1YH14_9ACTO 1-(5-phosphoribosyl)-5-[(5-phosphoribo sylamino)methylideneamino] imidazole-4-carboxamide isomera se OS=Gordonia neofelifaecis NRRL B-59395 GN=hisA PE=3 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: plu:plu0797 1-(5-phosphoribosyl)-5-[(5- phosphoribosylamino)methylideneamino] imidazole-4- carboxamide isomerase; TIGRFAM: bifunctional HisA/TrpF protein; PFAM: histidine biosynthesis protein;Phosphoribosylformimino-5-aminoimidazole carboxamide ribonucleotide (ProFAR) isomerase [Amino acid transport and metabolism]; COG0106;HisA.  Phosphoribosylformimino-5-aminoimidazole carboxamide ribonucleotide (ProFAR) isomerase catalyzes the fourth step in histidine biosynthesis, an isomerisation of the aminoaldose moiety of ProFAR to the aminoketose of PRFAR (N-(5'-phospho-D-1...; cd04732;catalytic residues [active]
GOHSU02	CDS	GOHSU_02_00750	79960..80802		putative inositol monophosphatase ImpA [nite]	 impA [nite]		ROP_07490 :putative inositol monophosphatase ImpA : impA : 3.1.3.25 : Rhodococcus opacus B4	monophosphatase		H5RKZ7_9NOCA Monophosphatase OS=Nocardia brasiliens is ATCC 700358 GN=O3I_01015 PE=4 SV=1;monophosphatase [Nocardia farcinica IFM 10152];Inositol-monophosphatase-like domains. This family of phosphatases is dependent on bivalent metal ions such as Mg++, and many members are inhibited by Li+ (which is thought to displace a bivalent ion in the active site). Substrates include fructose-1; cd01637
GOHSU02	CDS	GOHSU_02_00760	80878..81663		imidazole glycerol phosphate synthase subunit HisF [hamap]	hisF [hamap]	MF_01013:imidazole glycerol phosphate synthase subunit HisF:hisF 	ROP_07480 :imidazole glycerol phosphate synthase subunit HisF : hisF : 4.1.3.- : Rhodococcus opacus B4	imidazole glycerol phosphate synthase subunit HisF	hisF	H0R1V9_9ACTO Imidazole glycerol phosphate synthase subunit HisF OS=Gordonia effusa NBRC 100432 GN=hisF PE=3 S V=1;imidazole glycerol phosphate synthase subunit HisF [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the conversion of 5-[(5-phospho-1-deoxyribulos-1-ylamino)methylideneamino]- 1-(5-phosphoribosyl)imidazole-4-carboxamideand glutamine to imidazole-glycerol phosphate, 5-aminoimidazol-4-carboxamideribonucleotide and glutamate; the HisF subunit acts as a cyclase;The cyclase subunit of imidazoleglycerol phosphate synthase (HisF). Imidazole glycerol phosphate synthase (IGPS) catalyzes the fifth step of histidine biosynthesis, the formation of the imidazole ring. IGPS converts N1-(5'-phosphoribulosyl)...; cd04731;Histidine biosynthesis protein; pfam00977;substrate binding site [chemical binding];glutamase interaction surface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_00770	81751..82134		phosphoribosyl-AMP cyclohydrolase [hamap]	hisI [hamap]	MF_01021:phosphoribosyl-AMP cyclohydrolase:hisI 	RER_34390 :phosphoribosyl-AMP cyclohydrolase : hisI : 3.5.4.19 : Rhodococcus erythropolis PR4	phosphoribosyl-AMP cyclohydrolase HisI	hisI	F1YH17_9ACTO Phosphoribosyl-AMP cyclohydrolase OS=G ordonia neofelifaecis NRRL B-59395 GN=hisI PE=4 SV=1;hisI gene product [Gordonia polyisoprenivorans VH2] ;Phosphoribosyl-AMP cyclohydrolase; cl00344
GOHSU02	CDS	GOHSU_02_00780	82150..82329		putative tautomerase [nite]			ROP_07460 :putative tautomerase : : : Rhodococcus opacus B4	4-oxalocrotonate tautomerase	xylHI	F9VWP8_9ACTO Putative tautomerase OS=Gordonia alkan ivorans NBRC 16433 GN=GOALK_066_00110 PE=4 SV=1;4-oxalocrotonate tautomerase [Pseudomonas putida];4-Oxalocrotonate Tautomerase:  Catalyzes the isomerization of unsaturated ketones. The structure is a homohexamer that is arranged as a trimer of dimers. The hexamer contains six active sites, each formed by residues from three monomers, two from one...; cd00491;active site 1 [active];dimer interface [polypeptide binding];hexamer interface [polypeptide binding];active site 2 [active]
GOHSU02	CDS	GOHSU_02_00790	82326..83891		anthranilate synthase component I [nite]	 trpE [nite]		ROP_07420 :anthranilate synthase component I : trpE : 4.1.3.27 : Rhodococcus opacus B4	anthranilate synthase component 1	trpE	F1YH19_9ACTO Anthranilate synthase component I OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_06830 PE=4 SV=1 ;trpE gene product [Gordonia polyisoprenivorans VH2] ;anthranilate synthase component I; Provisional;Anthranilate synthase component I, N terminal region; pfam04715;chorismate binding enzyme; cl10555
GOHSU02	CDS	GOHSU_02_00800	83888..84616		hypothetical membrane protein [nite]			ROP_07410 :hypothetical membrane protein : : : Rhodococcus opacus B4	membrane protein		F1YH20_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06835 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];trp region conserved hypothetical membrane protein; TIGR02234;Tryptophan-associated transmembrane protein (Trp_oprn_chp); pfam09534
GOHSU02	CDS	GOHSU_02_00810	84691..85509		indole-3-glycerol-phosphate synthase [nite]	 trpC [nite]		RER_33540 :indole-3-glycerol-phosphate synthase : trpC : 4.1.1.48 : Rhodococcus erythropolis PR4	Indole-3-glycerol phosphate synthase	trpC	G7H2N7_9ACTO Indole-3-glycerol phosphate synthase O S=Gordonia araii NBRC 100433 GN=trpC PE=3 SV=1;Indole-3-glycerol phosphate synthase [Amycolicicocc us subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;Indole-3-glycerol phosphate synthase (IGPS); an enzyme in the tryptophan biosynthetic pathway, catalyzing the ring closure reaction of 1-(o-carboxyphenylamino)-1-deoxyribulose-5-phosphate (CdRP) to indole-3-glycerol phosphate (IGP), accompanied by the...; cd00331;ribulose/triose binding site [chemical binding];phosphate binding site [ion binding];substrate (anthranilate) binding pocket [chemical binding];product (indole) binding pocket [chemical binding]
GOHSU02	CDS	GOHSU_02_00820	85606..86937		tryptophan synthase beta chain [hamap]	trpB [hamap]	MF_00133:tryptophan synthase beta chain:trpB 	RER_33530 :tryptophan synthase beta chain : trpB : 4.2.1.20 : Rhodococcus erythropolis PR4	tryptophan synthase subunit beta		F1YH22_9ACTO Tryptophan synthase beta chain OS=Gord onia neofelifaecis NRRL B-59395 GN=trpB PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: tryptophan synthase subunit beta; PFAM: pyridoxal-5'-phosphate-dependent protein subunit beta; KEGG: mca:MCA2495 tryptophan synthase subunit beta;Tryptophan synthase-beta:  Trptophan synthase is a bifunctional enzyme that catalyses the last two steps in the biosynthesis of L-tryptophan via its alpha and beta reactions. In the alpha reaction, indole 3-glycerol phosphate is cleaved reversibly to...; cd06446;Pyridoxal-phosphate dependent enzyme; pfam00291;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU02	CDS	GOHSU_02_00830	86934..87740		tryptophan synthase alpha chain [hamap]	trpA [hamap]	MF_00131:tryptophan synthase alpha chain:trpA 	RER_33520 :tryptophan synthase alpha chain : trpA : 4.2.1.20 : Rhodococcus erythropolis PR4	tryptophan synthase subunit alpha		F1YH23_9ACTO Tryptophan synthase alpha chain OS=Gor donia neofelifaecis NRRL B-59395 GN=trpA PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: tryptophan synthase, subunit alpha; PFAM: tryptophan synthase alpha chain; KEGG: TRP5; likely tryptophan synthetase alpha chain similar to S. cerevisiae TRP5 (YGL026C) ; K01695 tryptophan synthase alpha chain;tryptophan synthase subunit alpha; Provisional; PRK13111;Ttryptophan synthase (TRPS) alpha subunit (TSA). TPRS is a bifunctional tetrameric enzyme (2 alpha and 2 beta subunits) that catalyzes the last two steps of L-tryptophan biosynthesis. Alpha and beta subunit catalyze two distinct reactions which are both...; cd04724;substrate binding site [chemical binding];catalytic residues [active];heterodimer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_00840	87763..89382		prolipoprotein diacylglyceryl transferase [hamap]	lgt [hamap]	MF_01147:prolipoprotein diacylglyceryl transferase:lgt 		prolipoprotein diacylglyceryl transferase	lgt	D0L9Y8_GORB4 Prolipoprotein diacylglyceryl transfer ase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_2945 PE=3 SV=1;prolipoprotein diacylglyceryl transferase [Rhodococ cus opacus B4];transfers the N-acyl diglyceride moiety to the prospective N-terminal cysteine in prolipoprotein;Prolipoprotein diacylglyceryl transferase; cl00478
GOHSU02	CDS	GOHSU_02_00850	89393..90826		pyruvate kinase [nite]	 pyk [nite]		RER_33460 :pyruvate kinase : pyk : 2.7.1.40 : Rhodococcus erythropolis PR4	pyruvate kinase	pyk1	F1YH25_9ACTO Pyruvate kinase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_06860 PE=3 SV=1;pyruvate kinase [Rhodococcus jostii RHA1];pyruvate kinase; Provisional;catalyzes the formation of phosphoenolpyruvate from pyruvate;Pyruvate kinase (PK):  Large allosteric enzyme that regulates glycolysis through binding of the substrate, phosphoenolpyruvate, and one or more allosteric effectors. Like other allosteric enzymes, PK has a high substrate affinity R state and a low...; cl09155;domain interfaces
GOHSU02	CDS	GOHSU_02_00860	90880..91809		acyl-CoA thioesterase [nite]	 tesB [nite]		ROP_07320 :acyl-CoA thioesterase : tesB : 3.1.2.- : Rhodococcus opacus B4	acyl-coenzyme A thioesterase Acot	acot	H6MRR4_GORPV Acyl-coenzyme A thioesterase Acot OS=G ordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=aco t PE=4 SV=1;acot gene product [Gordonia polyisoprenivorans VH2] ;Acyl-CoA thioesterase [Lipid metabolism]; COG1946;Thioesterase II (TEII) is thought to regenerate misprimed nonribosomal peptide synthetases (NRPSs) as well as modular polyketide synthases (PKSs) by hydrolyzing acetyl groups bound to the peptidyl carrier protein (PCP) and acyl carrier protein (ACP)...; cd03445;Thioesterase II (TEII) is thought to regenerate misprimed nonribosomal peptide synthetases (NRPSs) as well as modular polyketide synthases (PKSs) by hydrolyzing acetyl groups bound to the peptidyl carrier protein (PCP) and acyl carrier protein (ACP)...; cd03444;catalytic triad [active];dimer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_00870	91806..92399		hypothetical protein [migap]				hypothetical protein		F1YH27_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06870 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU02	CDS	GOHSU_02_00880	complement(92477..94588)		putative hydrolase [nite]			RER_47610 :putative hydrolase : : : Rhodococcus erythropolis PR4	putative peptidase		F1YH29_9ACTO Peptidase S15 OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_06880 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];putative hydrolase, CocE/NonD family; TIGR00976;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;X-Pro dipeptidyl-peptidase C-terminal non-catalytic domain; cl08482
GOHSU02	tRNA	GOHSU_02_t0010	94889..94970		not assigned						identified by tRNAscan-SE; putative
GOHSU02	CDS	GOHSU_02_00890	94984..95610		putative two-component response regulator [nite]			RER_33280 :putative two-component response regulator : : : Rhodococcus erythropolis PR4	response regulator receiver		F1YH30_9ACTO Response regulator receiver OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_06885 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: response regulator receiver; ANTAR domain protein; SMART: response regulator receiver; KEGG: gsu:GSU2822 response regulator;Response regulator with putative antiterminator output domain [Signal transduction mechanisms]; COG3707;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];ANTAR domain; cl04297
GOHSU02	CDS	GOHSU_02_00900	95688..98429		DNA polymerase I [nite]	 polA [nite]		RER_33220 :DNA polymerase I : polA : 2.7.7.7 : Rhodococcus erythropolis PR4	DNA polymerase I	polA	F1YH31_9ACTO DNA polymerase OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_06890 PE=3 SV=1;polA gene product [Gordonia polyisoprenivorans VH2] ;DNA polymerase I; Provisional;PIN domain of the 5'-3' exonuclease of Taq DNA polymerase I and homologs; cd09859;metal binding site 1 [ion binding];putative 5' ssDNA interaction site;metal binding site 3;metal binding site 2 [ion binding];H3TH domain of the 5'-3' exonuclease of Taq DNA polymerase I and homologs; cd09898;putative DNA binding site [nucleotide binding];putative metal binding site [ion binding];inactive DEDDy 3'-5' exonuclease domain of Bacillus stearothermophilus DNA polymerase I and similar family-A DNA polymerases; cd06140;Polymerase I functions primarily to fill DNA gaps that arise during DNA repair, recombination and replication; cd08637;DNA binding site [nucleotide binding];catalytic site [active]
GOHSU02	CDS	GOHSU_02_00910	98448..100013		serine/threonine protein kinase-like protein [migap]				serine/threonine protein kinase-like protein		D0L771_GORB4 Serine/threonine protein kinase-relate d protein (Precursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1582 P E=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: serine/threonine protein kinase-related; tyrosine protein kinase; SMART: serine/threonine protein kinase; tyrosine protein kinase; KEGG: mxa:MXAN_3338 serine/threonine protein kinase;Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
GOHSU02	CDS	GOHSU_02_00920	100078..101148		putative 3-isopropylmalate dehydrogenase [nite]			RER_33170 :putative 3-isopropylmalate dehydrogenase : : 1.1.1.85 : Rhodococcus erythropolis PR4	3-isopropylmalate dehydrogenase		F1YH32_9ACTO 3-isopropylmalate dehydrogenase OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_06895 PE=3 SV=1;3-isopropylmalate dehydrogenase [Starkeya novella D SM 506];KEGG: acr:Acry_1270 3-isopropylmalate dehydrogenase; PFAM: isocitrate/isopropylmalate dehydrogenase;Isocitrate/isopropylmalate dehydrogenase; cl00445;tartrate dehydrogenase; Provisional
GOHSU02	CDS	GOHSU_02_00930	101249..102124		putative amino acid ABC transporter amino acid-binding protein [nite]			RER_33160 :putative amino acid ABC transporter amino acid-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter substrate-binding protein		H0R1S2_9ACTO Putative amino acid ABC transporter am ino acid-binding protein OS=Gordonia effusa NBRC 100432 GN =GOEFS_073_00430 PE=4 SV=1;ABC transporter substrate-binding protein [Bifidoba cterium adolescentis ATCC 15703];COG family: ABC-type amino acid transport system_periplasmic component; Orthologue of BL1179_BL1178;Bacterial periplasmic transport systems use membrane-bound complexes and substrate-bound, membrane-associated, periplasmic binding proteins (PBPs) to transport a wide variety of  substrates, such as, amino acids, peptides, sugars, vitamins and inorganic...; cd00134;Bacterial extracellular solute-binding proteins, family 3; pfam00497;substrate binding pocket [chemical binding];membrane-bound complex binding site;hinge residues
GOHSU02	CDS	GOHSU_02_00940	102121..103044		putative amino acid ABC transporter permease protein [nite]			ROP_07120 :putative amino acid ABC transporter permease protein : : : Rhodococcus opacus B4	permease		Q5YYK5_NOCFA Putative ABC transporter permease OS=N ocardia farcinica (strain IFM 10152) GN=NFA_18900 PE=3 SV= 1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG0765 ABC-type amino acid transport system, permease component;ABC-type arginine transport system, permease component [Amino acid transport and metabolism]; COG4215;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU02	CDS	GOHSU_02_00950	103041..103823		putative amino acid ABC transporter ATP-binding protein [nite]			ROP_07110 :putative amino acid ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	ATP-binding protein		H0RN41_9ACTO Putative amino acid ABC transporter AT P-binding protein OS=Gordonia polyisoprenivorans NBRC 1632 0 GN=GOPIP_104_00320 PE=3 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00005;ABC-type polar amino acid transport system, ATPase component [Amino acid transport and metabolism]; COG1126;HisP and GlnQ are the ATP-binding components of the bacterial periplasmic histidine and glutamine permeases, repectively.  Histidine permease is a multisubunit complex containing the HisQ and HisM integral membrane subunits and two copies of HisP.  HisP...; cd03262;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU02	CDS	GOHSU_02_00960	complement(103968..104516)		hypothetical protein [nite]			MLP_04080 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YH37_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06920 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];KEGG: sco:SCO5445 hypothetical protein;Domain of unknown function (DUF1990); cl01969
GOHSU02	CDS	GOHSU_02_00970	complement(104537..105370)		putative methyltransferase [nite]			ROP_07100 :putative methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	methyltransferase type 11		F1YH38_9ACTO Type 11 methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06925 PE=4 SV=1;methyltransferase type 11 [Frankia sp. EuI1c];PFAM: methyltransferase type 11; KEGG: fal:FRAAL0751 SAM-dependent methyltransferase;SAM-dependent methyltransferases [Secondary metabolites biosynthesis, transport, and catabolism / General function prediction only]; COG0500;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_00980	105591..107066		probable 30S ribosomal protein S1 [nite]	 rpsA [nite]		RER_33120 :probable 30S ribosomal protein S1 : rpsA : : Rhodococcus erythropolis PR4	30S ribosomal protein S1	rpsA	F9VYP1_9ACTO 30S ribosomal protein S1 OS=Gordonia a lkanivorans NBRC 16433 GN=rpsA PE=3 SV=1;30S ribosomal protein S1 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;30S ribosomal protein S1; Reviewed; PRK07899;in Escherichia coli this protein is involved in binding to the leader sequence of mRNAs and is itself bound to the 30S subunit; autoregulates expression via a C-terminal domain; in most gram negative organisms this protein is composed of 6 repeats of the S1 domain while in gram positive there are 4 repeats; the S1 nucleic acid-binding domain is found associated with other proteins;S1_RPS1_repeat_ec1_hs1: Ribosomal protein S1 (RPS1) domain. RPS1 is a component of the small ribosomal subunit thought to be involved in the recognition and binding of mRNA's during translation initiation. The bacterial RPS1 domain architecture consists...; cd05687;RNA binding site [nucleotide binding];S1_RPS1_repeat_ec2_hs2: Ribosomal protein S1 (RPS1) domain. RPS1 is a component of the small ribosomal subunit thought to be involved in the recognition and binding of mRNA's during translation initiation. The bacterial RPS1 domain architecture consists...; cd04465;S1_like: Ribosomal protein S1-like RNA-binding domain. Found in a wide variety of RNA-associated proteins. Originally identified in S1 ribosomal protein. This superfamily also contains the Cold Shock Domain (CSD), which is a homolog of the S1 domain; cl09927
GOHSU02	CDS	GOHSU_02_00990	complement(107171..108103)		dephospho-CoA kinase [hamap]	coaE [hamap]	MF_00376:dephospho-CoA kinase:coaE 	ROP_07070 :dephospho-CoA kinase : coaE : 2.7.1.24 : Rhodococcus opacus B4	dephospho-CoA kinase		F1YH64_9ACTO Dephospho-CoA kinase/protein folding a ccessory domain-containing protein OS=Gordonia neofelifaec is NRRL B-59395 GN=coaE PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: dze:Dd1591_0623 dephospho-CoA kinase; TIGRFAM: dephospho-CoA kinase; PFAM: Dephospho-CoA kinase;Dephospho-coenzyme A kinase (DPCK, EC 2.7.1.24) catalyzes the phosphorylation of dephosphocoenzyme A (dCoA) to yield CoA, which is the final step in CoA biosynthesis; cd02022;dephospho-CoA kinase;CoA-binding site [chemical binding];ATP-binding [chemical binding]
GOHSU02	CDS	GOHSU_02_01000	complement(108187..108936)		putative IclR family transcriptional regulator [nite]			ROP_pROB02-02270 :putative IclR family transcriptional regulator : : : Rhodococcus opacus B4	IclR family transcriptional regulator		F1YH57_9ACTO IclR family transcriptional regulator OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07020 PE=4 SV=1;iclr family transcriptional regulator [Rhodococcus equi 103S];Transcriptional regulator [Transcription]; COG1414;Helix-turn-helix domains; cl00088;Bacterial transcriptional regulator; pfam01614
GOHSU02	CDS	GOHSU_02_01010	109006..109803		2-hydroxypenta-2,4-dienoate hydratase [nite]			RER_57560 :2-hydroxypenta-2,4-dienoate hydratase : : 4.2.1.80 : Rhodococcus erythropolis PR4	4-oxalocrotonate decarboxylase		F1YH58_9ACTO 2-oxopent-4-enoate hydratase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_07025 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];PFAM: Hydratase/decarboxylase; KEGG: nfa:nfa33180 putative hydratase;Fumarylacetoacetate (FAA) hydrolase family; cl11421
GOHSU02	CDS	GOHSU_02_01020	109813..110775		probable acetaldehyde dehydrogenase [nite]			RER_57570 :probable acetaldehyde dehydrogenase : : 1.2.1.10 : Rhodococcus erythropolis PR4	acetaldehyde dehydrogenase	mhpF2	F1YH59_9ACTO Acetaldehyde dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07030 PE=3 SV=1;acetaldehyde dehydrogenase [Cupriavidus necator N- 1];type strain of Cupriavidus necator;acetaldehyde dehydrogenase; Validated;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Prokaryotic acetaldehyde dehydrogenase, dimerisation; pfam09290
GOHSU02	CDS	GOHSU_02_01030	110772..111818		probable 4-hydroxy-2-oxovalerate aldolase [nite]			RER_57580 :probable 4-hydroxy-2-oxovalerate aldolase : : 4.1.3.- : Rhodococcus erythropolis PR4	4-hydroxy-2-ketovalerate aldolase	bphF3	F1YH60_9ACTO 4-hydroxy-2-oxovalerate aldolase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07035 PE=3 SV=1;4-hydroxy-2-ketovalerate aldolase [Rhodococcus jost ii RHA1];catalyzes the formation of pyruvate and acetaldehyde from 4-hydroxy-2-ketovaleric acid; involved in the degradation of phenylpropionate;4-hyroxy-2-oxovalerate/4-hydroxy-2-oxopentanoic acid aldolase,; Validated;4-hydroxy-2-oxovalerate aldolase, N-terminal catalytic TIM barrel domain; cd07943;catalytic residues [active];metal binding site [ion binding];DmpG-like communication domain; pfam07836
GOHSU02	CDS	GOHSU_02_01040	111979..112800		putative LuxR family transcriptional regulator [nite]			RER_25570 :putative LuxR family transcriptional regulator : : : Rhodococcus erythropolis PR4	response regulator		H6MS69_GORPV LuxR-family transcriptional regulator OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN =GPOL_c02540 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding];dimerization interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_01050	complement(112878..113063)		not assigned						
GOHSU02	CDS	GOHSU_02_01060	113026..114381		putative MFS transporter [nite]			RER_18170 :putative MFS transporter : : : Rhodococcus erythropolis PR4	major facilitator transporter		F9VW76_9ACTO Putative major facilitator superfamily transporter OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_ 060_00980 PE=4 SV=1;major facilitator transporter [Rhodococcus jostii R HA1];The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU02	CDS	GOHSU_02_01070	complement(114382..114915)		conserved hypothetical protein [nite]			RER_33090 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0RN51_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_104_00430 PE =4 SV=1;unnamed protein product [Deinococcus maricopensis D SM 21211];COGs: COG2306 conserved hypothetical protein; InterPro IPR007295; KEGG: ddr:Deide_08860 hypothetical protein; PFAM: Domain of unknown function DUF402; SPTR: Predicted RNA-binding protein associated with RNAse G/E; PFAM: Protein of unknown function (DUF402);Protein of unknown function (DUF402); cl00979
GOHSU02	CDS	GOHSU_02_01080	115068..116291		conserved hypothetical protein [nite]			RER_54290 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0L9R9_GORB4 Putative uncharacterized protein (Prec ursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43 247 / JCM 3198 / NCTC 10667) GN=Gbro_2871 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.565 at residue 30;KEGG: cko:CKO_00948 hypothetical protein
GOHSU02	CDS	GOHSU_02_01090	complement(116296..116733)		putative iron-sulfur protein [nite]			RER_pREL1-01130 :putative iron-sulfur protein : : : Rhodococcus erythropolis PR4	iron sulfur protein		B5H7W1_STRPR Iron sulfur protein OS=Streptomyces pr istinaespiralis ATCC 25486 GN=SSDG_01006 PE=4 SV=2;iron sulfur protein [Streptomyces pristinaespiralis ATCC 25486];Rieske domain; a [2Fe-2S] cluster binding domain commonly found in Rieske non-heme iron oxygenase (RO) systems such as naphthalene and biphenyl dioxygenases, as well as in plant/cyanobacterial chloroplast b6f and mitochondrial cytochrome bc(1) complexes; cd03467;iron-sulfur cluster [ion binding];[2Fe-2S] cluster binding site [ion binding]
GOHSU02	CDS	GOHSU_02_01100	116855..119032		UvrABC system protein B [hamap]	uvrB [hamap]	MF_00204:UvrABC system protein B:uvrB 	ROP_07050 :UvrABC system protein B : uvrB : : Rhodococcus opacus B4	excinuclease ABC subunit B	uvrB	F1YH76_9ACTO UvrABC system protein B OS=Gordonia ne ofelifaecis NRRL B-59395 GN=uvrB PE=3 SV=1;excinuclease ABC subunit B [Corynebacterium glutami cum ATCC 13032];IS fingerprint type 4-5;The UvrABC repair system catalyzes the recognition and processing of DNA lesions. The beta-hairpin of the Uvr-B subunit is inserted between the strands, where it probes for the presence of a lesion;excinuclease ABC subunit B; Provisional;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];Ultra-violet resistance protein B; pfam12344;UvrB/uvrC motif; pfam02151
GOHSU02	CDS	GOHSU_02_01110	119122..119568		conserved hypothetical protein [nite]			RER_33060 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	UspA domain-containing protein		F1YH77_9ACTO UspA domain-containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_07120 PE=4 SV=1;UspA domain-containing protein [Mycobacterium gilvu m PYR- GCK];PFAM: UspA domain protein; KEGG: mmc:Mmcs_2992 UspA;Usp: Universal stress protein family. The universal stress protein Usp is a small cytoplasmic bacterial protein whose expression is enhanced when the cell is exposed to stress agents. Usp enhances the rate of cell survival during prolonged exposure to...; cd00293;Ligand Binding Site [chemical binding]
GOHSU02	CDS	GOHSU_02_01120	complement(119615..121912)		hypothetical protein [nite]			ROP_07020 :hypothetical protein : : : Rhodococcus opacus B4	superfamily I DNA and RNA helicase-like protein		G7GSL3_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_56_00180 PE=4 SV=1;superfamily I DNA and RNA helicase-like protein [Ac tinosynnema mirum DSM 43827];Superfamily I DNA and RNA helicases [General function prediction only];KEGG: bcr:BCAH187_A2861 UvrD/Rep helicase family protein;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Family description; cl15862
GOHSU02	CDS	GOHSU_02_01130	complement(122021..123028)		hypothetical protein [nite]			RER_33030 :hypothetical protein : : : Rhodococcus erythropolis PR4	DoxX protein		F9W1F9_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_113_00250 PE=4 SV= 1;DoxX protein [Propionibacterium acnes SK137];identified by match to protein family HMM PF07681
GOHSU02	CDS	GOHSU_02_01140	complement(123178..123876)		conserved hypothetical protein [nite]			RER_33010 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	beta-lactamase		F1YH79_9ACTO Beta-lactamase domain-containing prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07130 PE =4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: beta-lactamase; KEGG: dol:Dole_2072 beta-lactamase domain- containing protein;Metallo-beta-lactamase superfamily; cl00446
GOHSU02	CDS	GOHSU_02_01150	124018..126891		UvrABC system protein A [hamap]	uvrA [hamap]	MF_00205:UvrABC system protein A:uvrA 	ROP_06990 :UvrABC system protein A : uvrA : : Rhodococcus opacus B4	excinuclease ABC subunit A	uvrA	F1YH80_9ACTO Excinuclease ABC subunit A OS=Gordonia neofelifaecis NRRL B-59395 GN=uvrA PE=3 SV=1;excinuclease ABC subunit A [Corynebacterium glutami cum ATCC 13032];IS fingerprint type 4-5;excinuclease ABC subunit A; Reviewed; PRK00349;The UvrABC repair system catalyzes the recognition and processing of DNA lesions. UvrA is an ATPase and a DNA-binding protein. A damage recognition complex composed of 2 uvrA and 2 uvrB subunits scans DNA for abnormalities. When the presence of a lesion has been verified by uvrB, the uvrA molecules dissociate;The excision repair protein UvrA domain I; Nucleotide excision repair in eubacteria is a process that repairs DNA damage by the removal of a 12-13-mer oligonucleotide containing the lesion.  Recognition and cleavage of the damaged DNA is a multistep...; cd03270;The excision repair protein UvrA domain II; Nucleotide excision repair in eubacteria is a process that repairs DNA damage by the removal of a 12-13-mer oligonucleotide containing the lesion.  Recognition and cleavage of the damaged DNA is a multistep...; cd03271
GOHSU02	CDS	GOHSU_02_01160	126940..127929		not assigned						C7QI42_CATAD Putative uncharacterized protein OS=Ca tenulispora acidiphila (strain DSM 44928 / NRRL B-24433 / NBRC 102108 / JCM 14897) GN=Caci_4223 PE=4 SV=1
GOHSU02	CDS	GOHSU_02_01170	127985..128296		hypothetical protein [nite]			ROP_06880 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YH82_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07145 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Phage shock protein A (IM30), suppresses sigma54-dependent transcription [Transcription / Signal transduction mechanisms]; COG1842
GOHSU02	CDS	GOHSU_02_01180	complement(128319..128699)		hypothetical protein [nite]			ROP_06950 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YH83_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07150 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU02	CDS	GOHSU_02_01190	129026..129505	case	translation initiation factor IF-3 [hamap]	infC [hamap]	MF_00080:translation initiation factor IF-3:infC 	ROP_06940 :translation initiation factor IF-3 : infC : : Rhodococcus opacus B4	translation initiation factor IF-3	infC	H0R1Q0_9ACTO Translation initiation factor IF-3 OS= Gordonia effusa NBRC 100432 GN=infC PE=3 SV=1;translation initiation factor IF-3 [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;IF-3 has several functions that are required and promote translation initiation including; preventing association of 70S by binding to 30S; monitoring codon-anticodon interactions by promoting disassociation of fMet-tRNA(fMet) from initiation complexes formed on leaderless mRNAs or incorrectly bound noninitiatior tRNAs and complexes with noncanonical start sites; stimulates codon-anticodon interactions at P-site; involved in moving mRNA to the P-site; and in recycling subunits;translation initiation factor IF-3; Reviewed; PRK00028;Translation initiation factor IF-3, N-terminal domain; pfam05198;Translation initiation factor IF-3, C-terminal domain; pfam00707
GOHSU02	CDS	GOHSU_02_01200	129599..129793	case	50S ribosomal protein L35 [hamap]	rpmI [hamap]	MF_00514:50S ribosomal protein L35:rpmI 	ROP_06930 :50S ribosomal protein L35 : rpmI : : Rhodococcus opacus B4	50S ribosomal protein L35		F1YH85_9ACTO 50S ribosomal protein L35 OS=Gordonia neofelifaecis NRRL B-59395 GN=rpmI PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Ribosomal protein L35; cl00392;TIGRFAM: ribosomal protein L35; PFAM: ribosomal protein L35; KEGG: sat:SYN_01719 50S ribosomal protein L35P
GOHSU02	CDS	GOHSU_02_01210	129832..130212	case	50S ribosomal protein L20 [hamap]	rplT [hamap]	MF_00382:50S ribosomal protein L20:rplT 	ROP_06920 :50S ribosomal protein L20 : rplT : : Rhodococcus opacus B4	50S ribosomal protein L20	rplT	F1YH86_9ACTO 50S ribosomal protein L20 OS=Gordonia neofelifaecis NRRL B-59395 GN=rplT PE=3 SV=1;50S ribosomal protein L20 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;ribosomal protein L20; CHL00068;binds directly to 23S ribosomal RNA prior to in vitro assembly of the 50S ribosomal subunit;Ribosomal protein L20; cd07026;23S rRNA binding site [nucleotide binding];L21 binding site [polypeptide binding];L13 binding site [polypeptide binding]
GOHSU02	CDS	GOHSU_02_01220	130254..131048		putative RNA methyltransferase [nite]			RER_32920 :putative RNA methyltransferase : : 2.1.1.- : Rhodococcus erythropolis PR4	tRNA/rRNA methyltransferase		H0RHF8_9ACTO Putative RNA methyltransferase OS=Gord onia polyisoprenivorans NBRC 16320 GN=GOPIP_069_00140 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];RNA 2'-O ribose methyltransferase substrate binding; cl06870;rRNA methylase, putative, group 3; TIGR00186;SpoU rRNA Methylase family; cl00362
GOHSU02	CDS	GOHSU_02_01230	complement(131234..133525)		serine/threonine protein kinase [migap]				serine/threonine protein kinase		G7H4L2_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_062_00550 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];SMART: serine/threonine protein kinase; KEGG: afw:Anae109_2802 regulator of chromosome condensation RCC1;Protein Kinases, catalytic domain; cl09925;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop);Regulator of chromosome condensation (RCC1) repeat; pfam00415;Regulator of chromosome condensation (RCC1) repeat; pfam13540
GOHSU02	CDS	GOHSU_02_01240	complement(133694..134926)		putative hydrolase [nite]			YM304_00800 :putative hydrolase : : : Acidimicrobidae bacterium YM16-304	amidohydrolase		F1YJ24_9ACTO Amidohydrolase OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_09491 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];PFAM: amidohydrolase; Amidohydrolase 3; KEGG: mjl:Mjls_3728 amidohydrolase;Imidazolonepropionase and related amidohydrolases [Secondary metabolites biosynthesis, transport, and catabolism]; COG1228;Metallo-dependent hydrolases, subgroup A is part of the superfamily of metallo-dependent hydrolases, a large group of proteins that show conservation in their 3-dimensional fold (TIM barrel) and in details of their active site. The vast majority of the...; cd01299
GOHSU02	CDS	GOHSU_02_01250	135182..136177		phenylalanyl-tRNA synthetase alpha chain [hamap]	pheS [hamap]	MF_00281:phenylalanyl-tRNA synthetase alpha chain:pheS 	ROP_06900 :phenylalanyl-tRNA synthetase alpha chain : pheS : 6.1.1.20 : Rhodococcus opacus B4	phenylalanyl-tRNA synthetase subunit alpha	pheS	H6MQT9_GORPV Phenylalanine--tRNA ligase alpha subun it OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=pheS PE=3 SV=1;phenylalanyl-tRNA synthetase subunit alpha [Coryneb acterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes a two-step reaction, first charging a phenylalanine molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA; forms a heterotetramer of alpha(2)beta(2); binds two magnesium ions per tetramer; type 1 subfamily;phenylalanyl-tRNA synthetase subunit alpha; Validated; PRK00488;Aminoacyl tRNA synthetase class II, N-terminal domain; pfam02912;Phenylalanyl-tRNA synthetase (PheRS) alpha chain catalytic core domain. PheRS belongs to class II aminoacyl-tRNA synthetases (aaRS) based upon its structure and the presence of three characteristic sequence motifs. This domain is primarily responsible...; cd00496;dimer interface [polypeptide binding];motif 1;motif 2;motif 3
GOHSU02	CDS	GOHSU_02_01260	136255..138747	case	phenylalanyl-tRNA synthetase beta chain [hamap]	pheT [hamap]	MF_00283:phenylalanyl-tRNA synthetase beta chain:pheT 	RER_32900 :phenylalanyl-tRNA synthetase beta chain : pheT : 6.1.1.20 : Rhodococcus erythropolis PR4	phenylalanyl-tRNA synthetase subunit beta	pheT	F1YH92_9ACTO Phenylalanine--tRNA ligase beta subuni t OS=Gordonia neofelifaecis NRRL B-59395 GN=pheT PE=3 SV=1 ;phenylalanyl-tRNA synthetase subunit beta [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;phenylalanyl-tRNA synthetase subunit beta; Reviewed; PRK00629;catalyzes a two-step reaction, first charging a phenylalanine molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA; forms a tetramer of alpha(2)beta(2); binds two magnesium ions per tetramer; type 2 subfamily;tRNA-binding-domain-containing prokaryotic phenylalanly tRNA synthetase (PheRS) beta chain.  PheRS aminoacylate phenylalanine transfer RNAs (tRNAphe). PheRSs belong structurally to class II aminoacyl tRNA synthetases (aaRSs) but, as they aminoacylate...; cd02796;putative tRNA-binding site [nucleotide binding];B3/4 domain; cl11458;tRNA synthetase B5 domain; cl08394;Phenylalanyl-tRNA synthetase (PheRS) beta chain core domain. PheRS belongs to class II aminoacyl-tRNA synthetases (aaRS) based upon its structure. While class II aaRSs generally aminoacylate the 3'-OH ribose of the appropriate tRNA,  PheRS is an...; cd00769;dimer interface [polypeptide binding];motif 1;motif 3;motif 2;Ferredoxin-fold anticodon binding domain; cl08386
GOHSU02	CDS	GOHSU_02_01270	complement(138877..139152)		hypothetical protein [nite]			KSE_36860 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		F8AAM6_THEID Putative uncharacterized protein OS=Th ermodesulfatator indicus (strain DSM 15286 / JCM 11887 / C IR29812) GN=Thein_0416 PE=4 SV=1;unnamed protein product [Haloarcula marismortui ATC C 43049]
GOHSU02	CDS	GOHSU_02_01280	139272..140423		conserved hypothetical membrane protein [nite]			RER_32190 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	metallophosphoesterase		G7H4P2_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_063_00160 PE=4 SV=1;metallophosphoesterase [Mycobacterium gilvum PYR- G CK];PFAM: metallophosphoesterase; KEGG: mmc:Mmcs_4782 metallophosphoesterase;Bacillus subtilis YkuE and related proteins, C-terminal metallophosphatase domain; cd07385;Calcineurin-like phosphoesterase; pfam00149;putative active site [active];putative metal binding site [ion binding]
GOHSU02	CDS	GOHSU_02_01290	complement(140457..140639)		not assigned						
GOHSU02	CDS	GOHSU_02_01300	140628..141524		N-acetyl-gamma-glutamyl-phosphate reductase [nite]	 argC [nite]		ROP_06870 :N-acetyl-gamma-glutamyl-phosphate reductase : argC : 1.2.1.38 : Rhodococcus opacus B4	N-acetyl-gamma-glutamyl-phosphate reductase	argC	F1YH94_9ACTO N-acetyl-gamma-glutamyl-phosphate redu ctase OS=Gordonia neofelifaecis NRRL B-59395 GN=argC PE=3 SV=1;N-acetyl-gamma-glutamyl-phosphate reductase [Coryne bacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the reduction of N-acetyl-5-glutamyl phosphate to N-acetyl-L-glutamate 5-semialdehyde in arginine biosynthesis and the reduction of N-acetyl-gamma-aminoadipyl-phosphate to N-acetyl-L-aminoadipate-semialdehyde in lysine biosynthesis; involved in both the arginine and lysine biosynthetic pathways; lysine is produced via the AAA pathway, lysine from alpha-aminoadipate;N-acetyl-gamma-glutamyl-phosphate reductase; Validated; PRK00436;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Glyceraldehyde 3-phosphate dehydrogenase, C-terminal domain; cl15856
GOHSU02	CDS	GOHSU_02_01310	141521..142789		arginine biosynthesis bifunctional protein ArgJ [hamap]	argJ [hamap]	MF_01106:arginine biosynthesis bifunctional protein ArgJ:argJ 	RER_32830 :arginine biosynthesis bifunctional protein ArgJ : argJ : 2.3.1.35/2.3.1.1 : Rhodococcus erythropolis PR4	bifunctional ornithine acetyltransferase/N-acetylglutamate synthase	argJ	F1YH95_9ACTO Bifunctional ornithine acetyltransfera se/N-acetylglutamate synthase protein OS=Gordonia neofelif aecis NRRL B-59395 GN=argJ PE=3 SV=1;bifunctional ornithine acetyltransferase/N-acetylgl utamate synthase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;bifunctional arginine biosynthesis protein ArgJ; functions at the 1st and 5th steps in arginine biosynthesis; involved in synthesis of acetylglutamate from glutamate and acetyl-CoA and ornithine by transacetylation between acetylornithine and glutmate;Ornithine acetyltransferase (OAT) family; also referred to as ArgJ. OAT catalyzes the first and fifth steps in arginine biosynthesis, coupling acetylation of glutamate with deacetylation of N-acetylornithine, which allows recycling of the acetyl group in...; cd02152;heterotetramer interface [polypeptide binding];active site pocket [active]
GOHSU02	CDS	GOHSU_02_01320	142786..143694		acetylglutamate kinase [nite]	 argB [nite]		RER_32820 :acetylglutamate kinase : argB : 2.7.2.8 : Rhodococcus erythropolis PR4	acetylglutamate kinase	argB	F1YH96_9ACTO Acetylglutamate kinase OS=Gordonia neo felifaecis NRRL B-59395 GN=argB PE=3 SV=1;acetylglutamate kinase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the phosphorylation of N-acetyl-L-glutamate to form N-acetyl-L-glutamate 5-phosphate;AAK_NAGK-C: N-Acetyl-L-glutamate kinase - cyclic (NAGK-C) catalyzes the phosphorylation of the gamma-COOH group of N-acetyl-L-glutamate (NAG) by ATP in the second step of arginine biosynthesis found in some bacteria and photosynthetic organisms using the...; cd04250;feedback inhibition sensing region;homohexameric interface [polypeptide binding];nucleotide binding site [chemical binding];N-acetyl-L-glutamate binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_01330	143691..144899	case	acetylornithine/succinyldiaminopimelate aminotransferase [hamap]	argD [hamap]	MF_01107:acetylornithine/succinyldiaminopimelate aminotransferase:argD 	RER_32810 :acetylornithine aminotransferase : argD : 2.6.1.11 : Rhodococcus erythropolis PR4	acetylornithine aminotransferase ArgD	argD	F1YH97_9ACTO Acetylornithine aminotransferase OS=Go rdonia neofelifaecis NRRL B-59395 GN=argD PE=3 SV=1;argD gene product [Gordonia polyisoprenivorans VH2] ;acetylornithine aminotransferase; Provisional; PRK03244;Acetyl ornithine aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). The major groups in this CD correspond to ornithine aminotransferase, acetylornithine aminotransferase; cd00610;inhibitor-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU02	CDS	GOHSU_02_01340	144899..145822	case	ornithine carbamoyltransferase [hamap]	argI [hamap]	MF_01109:ornithine carbamoyltransferase:argI 	RER_32800 :ornithine carbamoyltransferase : argF : 2.1.3.3 : Rhodococcus erythropolis PR4	ornithine carbamoyltransferase	argF	F1YH98_9ACTO Ornithine carbamoyltransferase OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_07225 PE=3 SV=1;ornithine carbamoyltransferase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of L-citrulline from carbamoyl phosphate and L-ornithine in arginine biosynthesis and degradation;ornithine carbamoyltransferase; Provisional;Aspartate/ornithine carbamoyltransferase, carbamoyl-P binding domain; pfam02729;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU02	CDS	GOHSU_02_01350	145831..146358		arginine repressor [hamap]	argR [hamap]	MF_00173:arginine repressor:argR 	ROP_06820 :arginine repressor ArgR : argR : : Rhodococcus opacus B4	arginine repressor ArgR	argR	F9W1E0_9ACTO Arginine repressor OS=Gordonia alkaniv orans NBRC 16433 GN=argR PE=3 SV=1;arginine repressor (partial match) [Saccharopolyspo ra erythraea NRRL 2338];arginine repressor; Provisional;Arginine repressor, DNA binding domain; pfam01316;Arginine repressor, C-terminal domain; pfam02863
GOHSU02	CDS	GOHSU_02_01360	146448..147902		argininosuccinate lyase [hamap]	argH [hamap]	MF_00006:argininosuccinate lyase:argH 	ROP_06800 :argininosuccinate lyase : argH : 4.3.2.1 : Rhodococcus opacus B4	argininosuccinate lyase	argH	F1YHA0_9ACTO Argininosuccinate lyase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=argH PE=3 SV=1;argininosuccinate lyase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of arginine from (N-L-arginino)succinate;argininosuccinate lyase; Provisional;Argininosuccinate lyase (argininosuccinase, ASAL); cd01359;active sites [active];tetramer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_01370	147951..151133		putative fatty-acid--CoA ligase [nite]			ROP_06790 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	putative AMP-dependent synthetase/ligase		F1YHA1_9ACTO Acyl-CoA synthetase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_07240 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];acyl-CoA synthetase; Validated;AMP-binding enzyme; cl15778
GOHSU02	CDS	GOHSU_02_01380	151130..151354		conserved hypothetical protein [nite]			RER_32750 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YHA2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07245 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Trm112p-like protein; cl01066
GOHSU02	CDS	GOHSU_02_01390	151351..152652		tyrosyl-tRNA synthetase [hamap]	tyrS [hamap]	MF_02006:tyrosyl-tRNA synthetase:tyrS 	ROP_06730 :tyrosyl-tRNA synthetase : tyrS : 6.1.1.1 : Rhodococcus opacus B4	tyrosyl-tRNA synthetase	tyrS	G7GQB6_9ACTO Tyrosine--tRNA ligase OS=Gordonia amar ae NBRC 15530 GN=tyrS PE=3 SV=1;tyrosyl-tRNA synthetase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;tyrosyl-tRNA synthetase; Validated;catalyzes the formation of tyrosyl-tRNA(Tyr) from tyrosine and tRNA(Tyr);catalytic core domain of tyrosinyl-tRNA synthetase; cd00805;HIGH motif;dimer interface [polypeptide binding];KMSKS motif;S4/Hsp/ tRNA synthetase RNA-binding domain; The domain surface is populated by conserved, charged residues that define a likely RNA-binding site;  Found in stress proteins, ribosomal proteins and tRNA synthetases; This may imply a hitherto unrecognized...; cl09940
GOHSU02	CDS	GOHSU_02_01400	152649..154385		hydrolase [migap]				hydrolase		F1YNH6_9ACTO HAD-superfamily hydrolase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_17600 PE=4 SV=1;hydrolase [Streptomyces clavuligerus ATCC 27064];COG:COG0647 Predicted sugar phosphatases of the HAD superfamily.;Predicted sugar phosphatases of the HAD superfamily [Carbohydrate transport and metabolism]; COG0647;Haloacid dehalogenase-like hydrolases. The haloacid dehalogenase-like (HAD) superfamily includes L-2-haloacid dehalogenase, epoxide hydrolase, phosphoserine phosphatase, phosphomannomutase, phosphoglycolate phosphatase, P-type ATPase, and many others; cd01427;motif I;motif II;Haloacid Dehalogenase-like Hydrolases; cl11391
GOHSU02	CDS	GOHSU_02_01410	154427..154630		hypothetical protein [migap]				hypothetical protein		F1YNH5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17595 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU02	CDS	GOHSU_02_01420	154630..155460		putative hemolysin [nite]	 tlyA [nite]		ROP_06690 :putative hemolysin : tlyA : : Rhodococcus opacus B4	cytotoxin/hemolysin	tlyA	F1YNH4_9ACTO Hemolysin A OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_17590 PE=4 SV=1;cytotoxin;Predicted rRNA methylase [Translation, ribosomal structure and biogenesis];S4/Hsp/ tRNA synthetase RNA-binding domain; The domain surface is populated by conserved, charged residues that define a likely RNA-binding site;  Found in stress proteins, ribosomal proteins and tRNA synthetases; This may imply a hitherto unrecognized...; cd00165;RNA binding surface [nucleotide binding];S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_01430	155573..156523		probable inorganic polyphosphate/ATP-NAD kinase [hamap]	ppnK [hamap]	MF_00361:probable inorganic polyphosphate/ATP-NAD kinase:ppnK 	RER_32660 :inorganic polyphosphate/ATP-NAD kinase : ppnK : 2.7.1.23 : Rhodococcus erythropolis PR4	inorganic polyphosphate/ATP-NAD kinase	ppnK	D0L950_GORB4 Probable inorganic polyphosphate/ATP-N AD kinase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=ppnK PE=3 SV=1;inorganic polyphosphate/ATP-NAD kinase [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the phosphorylation of NAD to NADP;Diacylglycerol kinase catalytic domain; cl01255
GOHSU02	CDS	GOHSU_02_01440	156527..158287		DNA repair protein RecN [nite]	 recN [nite]		ROP_06670 :DNA repair protein RecN : recN : : Rhodococcus opacus B4	DNA repair protein RecN		F1YNH2_9ACTO DNA repair protein RecN OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_17580 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];DNA repair protein RecN; TIGR00634;TIGRFAM: DNA repair protein RecN; PFAM: SMC domain protein; KEGG: afw:Anae109_0776 DNA repair protein RecN;RecN ATPase involved in DNA repair; ABC (ATP-binding cassette) transporter nucleotide-binding domain; ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds including sugars, ions, peptides, and...; cd03241;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU02	CDS	GOHSU_02_01450	158417..159640		conserved hypothetical protein [nite]			RER_32640 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	membrane protein		D0L948_GORB4 Uncharacterized membrane-anchored prot ein-like protein OS=Gordonia bronchialis (strain ATCC 2559 2 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_2825 PE=4 S V=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Uncharacterized membrane-anchored protein conserved in bacteria [Function unknown];Thiamin pyrophosphokinase, catalytic domain; cl09135;Thiamine pyrophosphokinase C terminal; pfam12555
GOHSU02	CDS	GOHSU_02_01460	159640..160554		putative channel protein [nite]			RER_32630 :putative channel protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YNH0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17570 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF3186); pfam11382
GOHSU02	CDS	GOHSU_02_01470	160605..162353		CTP synthase [hamap]	pyrG [hamap]	MF_01227:CTP synthase:pyrG 	RER_32620 :CTP synthase : pyrG : 6.3.4.2 : Rhodococcus erythropolis PR4	CTP synthetase	pyrG	F1YNG9_9ACTO CTP synthase OS=Gordonia neofelifaecis NRRL B-59395 GN=pyrG PE=3 SV=1;CTP synthetase [Corynebacterium glutamicum ATCC 130 32];IS fingerprint type 4-5;CTP synthase; cytidine triphosphate synthetase; catalyzes the ATP-dependent amination of UTP to CTP with either L-glutamine or ammonia as the source of nitrogen; in Escherichia coli this enzyme forms a homotetramer;CTP synthetase; Validated; PRK05380;CTP synthetase (CTPs) is a two-domain protein, which consists of an N-terminal synthetase domain and C-terminal glutaminase domain. The enzymes hydrolyze the amide bond of glutamine to ammonia and glutamate at the glutaminase domains and transfer nascent...; cd03113;Catalytic site [active];UTP binding site [chemical binding];Type 1 glutamine amidotransferase (GATase1) domain found in Cytidine Triphosphate Synthetase; cd01746;putative oxyanion hole;catalytic triad [active]
GOHSU02	CDS	GOHSU_02_01480	162384..163052		putative ADP-ribose pyrophosphatase [nite]	 nudF [nite]		ROP_06630 :putative ADP-ribose pyrophosphatase : nudF : 3.6.1.13 : Rhodococcus opacus B4	NUDIX hydrolase		H5U2N7_9ACTO ADP-ribose pyrophosphatase OS=Gordonia sputi NBRC 100414 GN=nudF PE=4 SV=1;NUDIX hydrolase [Frankia sp. EuI1c];PFAM: NUDIX hydrolase; KEGG: fal:FRAAL5178 ADP-ribose pyrophosphatase;ADP-ribose pyrophosphatase (ADPRase) catalyzes the hydrolysis of ADP-ribose and a variety of additional ADP-sugar conjugates to AMP and ribose-5-phosphate. Like other members of the Nudix hydrolase superfamily, it requires a divalent cation, such as Mg2+; cd03424;dimer interface [polypeptide binding];ADP-ribose binding site [chemical binding];nudix motif;metal binding site [ion binding]
GOHSU02	CDS	GOHSU_02_01490	163061..163987	case	tyrosine recombinase XerD/tyrosine recombinase XerC [hamap]	xerD/xerC [hamap]	MF_01807:tyrosine recombinase XerD:xerD MF_01808:tyrosine recombinase XerC:xerC 	RER_32600 :tyrosine recombinase XerD : xerD : : Rhodococcus erythropolis PR4	tyrosine recombinase XerD	xerD	H5U2N6_9ACTO Tyrosine recombinase xerD OS=Gordonia sputi NBRC 100414 GN=xerD PE=3 SV=1;xerD gene product [Gordonia polyisoprenivorans VH2] ;site-specific tyrosine recombinase XerD; Reviewed; PRK00283;DNA breaking-rejoining enzymes, C-terminal catalytic domain. The DNA breaking-rejoining enzyme superfamily includes type IB topoisomerases and tyrosine recombinases that share the same fold in their catalytic domain containing six conserved active site...; cl00213;DNA binding site [nucleotide binding];Int/Topo IB signature motif
GOHSU02	CDS	GOHSU_02_01500	complement(163995..165425)		probable serine/threonine protein kinase PknB [nite]	 pknB [nite]		RER_00280 :probable serine/threonine protein kinase PknB : pknB : 2.7.11.1 : Rhodococcus erythropolis PR4	serine/threonine protein kinase-like protein		D5UMH4_TSUPD Serine/threonine protein kinase OS=Tsu kamurella paurometabola (strain ATCC 8368 / DSM 20162 / JC M 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_3873 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: serine/threonine protein kinase-related; tyrosine protein kinase; SMART: serine/threonine protein kinase; tyrosine protein kinase; KEGG: mxa:MXAN_4591 serine/threonine-protein kinase PKN1;Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
GOHSU02	CDS	GOHSU_02_01510	complement(165660..167432)		hypothetical protein [nite]			ROP_37430 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0R5L8_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_115_00090 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF222); pfam02720;HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and...; cd00085
GOHSU02	CDS	GOHSU_02_01520	168025..169599		putative long-chain fatty-acid--CoA ligase [nite]			KSE_06930 :putative long-chain fatty-acid--CoA ligase : : 6.2.1.- : Kitasatospora setae NBRC 14216	AMP-dependent synthetase and ligase		F1YE43_9ACTO AMP-dependent synthetase and ligase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02125 PE=4 SV =1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];PFAM: AMP-dependent synthetase and ligase; KEGG: met:M446_0171 AMP-dependent synthetase and ligase;long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778
GOHSU02	CDS	GOHSU_02_01530	169771..170610		putative Soj/ParA-related protein [nite]			RER_32590 :putative Soj/ParA-related protein : : : Rhodococcus erythropolis PR4	chromosome partitioning ATPase	parA2	F1YNG6_9ACTO Soj/ParA-related protein OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_17550 PE=4 SV=1;chromosome partitioning ATPase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;ATPases involved in chromosome partitioning [Cell division and chromosome partitioning]; COG1192;ParA and ParB of Caulobacter crescentus belong to a conserved family of bacterial proteins implicated in chromosome segregation. ParB binds to DNA sequences adjacent to the origin of replication and localizes to opposite cell poles shortly following the...; cd02042;P-loop;Magnesium ion binding site [ion binding]
GOHSU02	CDS	GOHSU_02_01540	170617..171462		putative chromosome segregation and condensation protein ScpA [nite]	 scpA [nite]		RER_32580 :putative chromosome segregation and condensation protein ScpA : scpA : : Rhodococcus erythropolis PR4	chromosome segregation and condensation protein ScpA		F1YNG5_9ACTO Chromosome segregation and condensatio n protein ScpA OS=Gordonia neofelifaecis NRRL B-59395 GN=S CNU_17545 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: chromosome segregation and condensation protein ScpA; KEGG: nmu:Nmul_A2318 chromosome segregation and condensation protein ScpA;ScpA/B protein; cl00598
GOHSU02	CDS	GOHSU_02_01550	171459..172178		chromosome segregation and condensation protein ScpB [nite]	 scpB [nite]		ROP_06600 :chromosome segregation and condensation protein ScpB : scpB : : Rhodococcus opacus B4	segregation and condensation protein B	scpB	F9W1P3_9ACTO Chromosome segregation and condensatio n protein ScpB OS=Gordonia alkanivorans NBRC 16433 GN=scpB PE=3 SV=1;scpB gene product [Gordonia polyisoprenivorans VH2] ;Putative transcriptional regulators (Ypuh-like); cl00612
GOHSU02	CDS	GOHSU_02_01560	complement(172181..172552)		not assigned						
GOHSU02	CDS	GOHSU_02_01570	172655..173380		pseudouridine synthase [nite]			ROP_06590 :pseudouridine synthase : : : Rhodococcus opacus B4	pseudouridine synthase		F1YNG3_9ACTO Pseudouridine synthase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_17535 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: pseudouridine synthase; RNA-binding S4 domain protein; SMART: RNA-binding S4 domain protein; KEGG: gbm:Gbem_3089 pseudouridine synthase;16S rRNA uridine-516 pseudouridylate synthase and related pseudouridylate synthases [Translation, ribosomal structure and biogenesis]; COG1187;S4/Hsp/ tRNA synthetase RNA-binding domain; The domain surface is populated by conserved, charged residues that define a likely RNA-binding site;  Found in stress proteins, ribosomal proteins and tRNA synthetases; This may imply a hitherto unrecognized...; cd00165;RNA binding surface [nucleotide binding];Pseudouridine synthases  are responsible for the synthesis of pseudouridine from uracil in ribosomal RNA. The RsuA subfamily includes Pseudouridine Synthase similar to Ribosomal small subunit pseudouridine 516 synthase. Most of the proteins in this...; cd02870
GOHSU02	CDS	GOHSU_02_01580	173377..174078		cytidylate kinase [hamap]	cmk [hamap]	MF_00238:cytidylate kinase:cmk 	ROP_06580 :cytidylate kinase : cmk : 2.7.4.14 : Rhodococcus opacus B4	cytidylate kinase	cmk	H6N4M7_GORPV Cytidylate kinase OS=Gordonia polyisop renivorans (strain DSM 44266 / VH2) GN=cmk PE=3 SV=1;cytidylate kinase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Catalyzes the formation of (d)CDP from ATP and (d)CMP;cytidylate kinase; Provisional; PRK00023;Cytidine monophosphate kinase (CMPK) catalyzes the reversible phosphorylation of cytidine monophosphate (CMP) to produce cytidine diphosphate (CDP), using ATP as the preferred phosphoryl donor; cd02020;CMP-binding site;The sites determining sugar specificity
GOHSU02	CDS	GOHSU_02_01590	174071..175501		GTPase Der [hamap]	der [hamap]	MF_00195:GTPase Der:der 	RER_32540 :probable GTP-binding protein EngA : engA : : Rhodococcus erythropolis PR4	GTP-binding protein EngA	engA	F1YNG1_9ACTO GTPase Der OS=Gordonia neofelifaecis N RRL B-59395 GN=der PE=3 SV=1;GTP-binding protein EngA [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;synchronizes cellular events by interacting with multiple targets with tandem G-domains; overexpression in Escherichia coli suppresses rrmJ mutation; structural analysis of the Thermotoga maritima ortholog shows different nucleotide binding affinities in the two binding domains;GTP-binding protein Der; Reviewed;EngA1 GTPase contains the first domain of EngA; cd01894;G1 box;GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;Switch II region;G3 box;G4 box;G5 box;EngA2 GTPase contains the second domain of EngA; cd01895
GOHSU02	CDS	GOHSU_02_01600	175543..175842		hypothetical membrane protein [nite]			RER_53360 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	transglycosylase-associated protein		F1YNG0_9ACTO Integral membrane protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_17520 PE=4 SV=1;transglycosylase-associated protein [Geodermatophil us obscurus DSM 43160];PFAM: Transglycosylase-associated protein; KEGG: cmi:CMM_2668 hypothetical protein
GOHSU02	tRNA	GOHSU_02_t0020	175907..175983		not assigned						identified by tRNAscan-SE; putative
GOHSU02	CDS	GOHSU_02_01610	176235..176426		aromatic amino acid decarboxylase [migap]				aromatic amino acid decarboxylase		aromatic amino acid decarboxylase [Thalassiobium sp . R2A62];identified by match to protein family HMM PF00282;Glutamate decarboxylase and related PLP-dependent proteins [Amino acid transport and metabolism]; COG0076;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321
GOHSU02	CDS	GOHSU_02_01620	176854..177978		hypothetical protein [nite]			KRH_20250 :hypothetical protein : : : Kocuria rhizophila DC2201	type IV restriction endonuclease		B2GH06_KOCRD Putative uncharacterized protein OS=Ko curia rhizophila (strain ATCC 9341 / DSM 348 / NBRC 103217 / DC2201) GN=KRH_20250 PE=4 SV=1;type IV restriction endonuclease [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;Uncharacterized conserved protein [Function unknown];Type I restriction enzyme R protein N terminus (HSDR_N); cl14958
GOHSU02	CDS	GOHSU_02_01630	178386..179423		hypothetical protein [migap]				hypothetical protein		Q8FP47_COREF Putative uncharacterized protein OS=Co rynebacterium efficiens (strain DSM 44549 / YS-314 / AJ 12 310 / JCM 11189 / NBRC 100395) GN=CE1941 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];similar to AL646076-134|CAD17285.1| percent identity: 26 in 139 aa;Fusaric acid resistance protein-like; cl15844
GOHSU02	CDS	GOHSU_02_01640	complement(179427..180338)		putative esterase [nite]			YM304_23370 :putative esterase : : : Acidimicrobidae bacterium YM16-304	alpha/beta hydrolase fold protein		B1MH73_MYCA9 Probable lipase/esterase LipG OS=Mycob acterium abscessus (strain ATCC 19977 / DSM 44196) GN=MAB_ 3880 PE=4 SV=1;alpha/beta hydrolase fold protein [Segniliparus rot undus DSM 44985];type strain of Segniliparus rotundus;PFAM: alpha/beta hydrolase fold; KEGG: mmi:MMAR_0983 lipase/esterase LipG2;Alpha/beta hydrolase family; pfam12697;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU02	CDS	GOHSU_02_01650	180482..181522		acyl dehydratase [migap]				acyl dehydratase		G7H748_9ACTO Beta-hydroxyacyl-[acyl-carrier-protein ] dehydratase subunit HadB OS=Gordonia araii NBRC 100433 G N=hadB PE=4 SV=1;acyl dehydratase [Mycobacterium sp. Spyr1];PFAM: MaoC like domain;(R)-hydratase [(R)-specific enoyl-CoA hydratase]. Catalyzes the hydration of trans-2-enoyl CoA to (R)-3-hydroxyacyl-CoA as part of the PHA (polyhydroxyalkanoate) biosynthetic pathway.  The structure of the monomer includes a five-strand antiparallel...; cd03441;catalytic site [active];SAV4209_like.  Similar in sequence to the Streptomyces avermitilis SAV4209 protein, with a hot dog fold that is similar to those of (R)-specific enoyl-CoA hydratase, the peroxisomal Hydratase-Dehydrogenase-Epimerase (HDE) protein, and the fatty acid...; cd03453
GOHSU02	CDS	GOHSU_02_01660	181576..181995		hypothetical protein [nite]			MLP_06180 :hypothetical protein : : : Microlunatus phosphovorus NM-1	3-demethylubiquinone-9 3-methyltransferase		F1YMC3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15659 PE=4 SV=1;3-demethylubiquinone-9 3-methyltransferase [Chitino phaga pinensis DSM 2588];type strain of Chitinophaga pinensis;PFAM: 3-demethylubiquinone-9 3-methyltransferase; glyoxalase/bleomycin resistance protein/dioxygenase; KEGG: abc:ACICU_03092 hypothetical protein;Escherichia coli PhnB and similar proteins; the E. coli phnB gene is found next to an operon involved in the cleavage of carbon-phosphorus bonds in unactivated alkylphosphonates; cd06588;dimer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_01670	complement(182042..183214)		hypothetical protein [nite]			ROP_52590 :hypothetical protein : : : Rhodococcus opacus B4	type I phosphodiesterase/nucleotide pyrophosphatase		D0L511_GORB4 Type I phosphodiesterase/nucleotide py rophosphatase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1155 PE=4 SV=1 ;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: type I phosphodiesterase/nucleotide pyrophosphatase; KEGG: tgr:Tgr7_0170 hypothetical protein;Type I phosphodiesterase / nucleotide pyrophosphatase; pfam01663
GOHSU02	CDS	GOHSU_02_01680	183383..183586		cold shock protein [nite]			MLP_26620 :cold shock protein : : : Microlunatus phosphovorus NM-1	cold-shock DNA-binding domain-containing protein		F1YNA1_9ACTO Cold-shock protein DNA-binding protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16998 PE=3 SV=1;cold-shock DNA-binding domain-containing protein [M ycobacterium gilvum PYR- GCK];PFAM: Cold-shock protein, DNA-binding; SMART: Cold shock protein; KEGG: mpa:MAP0669 cold shock protein;Cold-Shock Protein (CSP) contains an S1-like cold-shock domain (CSD) that is found in eukaryotes, prokaryotes, and archaea.  CSP's include the major cold-shock proteins CspA and CspB in bacteria and the eukaryotic gene regulatory factor Y-box protein; cd04458;DNA-binding site [nucleotide binding];RNA-binding motif
GOHSU02	CDS	GOHSU_02_01690	183702..184466		hypothetical protein [nite]			MLP_33290 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		G7GP97_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_34_00890 PE=3 SV=1;unnamed protein product [Bordetella avium 197N];Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzymes, YBL036c-like proteins; cd00635;pyridoxal 5'-phosphate (PLP) binding site [chemical binding];catalytic residue [active]
GOHSU02	CDS	GOHSU_02_01700	184499..185338		phosphomethylpyrimidine kinase [nite]	 thiD [nite]		RER_32350 :phosphomethylpyrimidine kinase : thiD : 2.7.4.7 : Rhodococcus erythropolis PR4	phosphomethylpyrimidine kinase	thiD2	H5TFN8_9ACTO Phosphomethylpyrimidine kinase OS=Gord onia otitidis NBRC 100426 GN=thiD PE=4 SV=1;phosphomethylpyrimidine kinase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 4-amino-2-methyl-5-diphosphomethylpyrimidine;4-amino-5-hydroxymethyl-2-methyl-pyrimidine phosphate kinase (HMPP-kinase) catalyzes two consecutive phosphorylation steps in the thiamine phosphate biosynthesis pathway, leading to the synthesis of vitamin B1. The first step is the phosphorylation of...; cd01169;substrate binding site [chemical binding];dimer interface [polypeptide binding];ATP binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_01710	complement(185366..185758)		putative hydrolase [nite]			KSE_42960 :putative hydrolase : : : Kitasatospora setae NBRC 14216	NUDIX hydrolase		F1YNK5_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_17435 PE=3 SV=1;NUDIX hydrolase [Streptomyces sp. AA4];Members of the Nudix hydrolase superfamily catalyze the hydrolysis of NUcleoside DIphosphates linked to other moieties, X. Enzymes belonging to this superfamily require a divalent cation, such as Mg2+ or Mn2+, for their activity and contain a highly...; cd04673;nudix motif
GOHSU02	CDS	GOHSU_02_01720	185927..186301		sugar transporter [migap]				sugar transporter		B5ZCC0_GLUDA Sugar transporter OS=Gluconacetobacter diazotrophicus (strain ATCC 49037 / DSM 5601 / PAl5) GN=G dia_0388 PE=3 SV=1;sugar transporter [Gluconacetobacter diazotrophicus PAl 5];KEGG: gdi:GDI2169 putative galactose-proton symporter; TIGRFAM: sugar transporter; PFAM: General substrate transporter; major facilitator superfamily MFS_1;MFS transporter, sugar porter (SP) family; TIGR00879
GOHSU02	CDS	GOHSU_02_01730	complement(186258..186626)		hypothetical protein [nite]			MLP_32780 :hypothetical protein : : : Microlunatus phosphovorus NM-1	uroporphyrin-III C-methyltransferase		D9V2T4_9ACTO Putative uncharacterized protein OS=St reptomyces sp. AA4 GN=SSMG_04749 PE=4 SV=1;uroporphyrin-III C-methyltransferase [Brucella meli tensis bv. 1 str. 16M];Protein of unknown function, DUF488; cl01246
GOHSU02	CDS	GOHSU_02_01740	complement(186583..187356)		putative uracil-DNA glycosylase [nite]			ROP_60020 :putative uracil-DNA glycosylase : : 3.2.2.- : Rhodococcus opacus B4	Uracil-DNA glycosylase superfamily		F1YNJ7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17395 PE=4 SV=1;Uracil-DNA glycosylase superfamily [Cellulomonas fi mi ATCC 484];PFAM: Uracil-DNA glycosylase superfamily; KEGG: uracil-DNA glycosylase superfamily;Family-5 Uracil-DNA glycosylases (UDG), found in thermophilic organisms; cd10031;Fe-S cluster binding site [ion binding];DNA binding site [nucleotide binding]
GOHSU02	CDS	GOHSU_02_01750	complement(187389..188171)		putative ABC transporter ATP-binding protein [nite]			ROP_51180 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	ABC transporter ATP-binding protein		C7MDW9_BRAFD ABC-type molybdenum transport system, ATPase component/photorepair protein PhrA OS=Brachybacteri um faecium (strain ATCC 43885 / DSM 4810 / NCIB 9860) GN=B fae_19640 PE=4 SV=1;ABC transporter ATP-binding protein [Rhodococcus op acus B4];ABC-type cobalamin/Fe3+-siderophores transport systems, ATPase components [Inorganic ion transport and metabolism / Coenzyme metabolism]; COG1120;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU02	CDS	GOHSU_02_01760	188223..188768		peptidase M15B and M15C DD-carboxypeptidase VanY/endolysin [migap]				peptidase M15B and M15C DD-carboxypeptidase VanY/endolysin		F1YNJ1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17365 PE=4 SV=1;peptidase M15B and M15C DD-carboxypeptidase VanY/en dolysin [Kribbella flavida DSM 17836];PFAM: peptidase M15B and M15C DD-carboxypeptidase VanY/endolysin; KEGG: mxa:MXAN_0560 penicillin-resistant dd- carboxypeptidase;D-ala-D-ala dipeptidase; cl00813
GOHSU02	CDS	GOHSU_02_01770	188828..189487		hypothetical protein [migap]				hypothetical protein		H6RDL7_NOCCG Uncharacterized protein OS=Nocardia cy riacigeorgica (strain GUH-2) GN=NOCYR_4512 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mmi:MMAR_2661 hypothetical protein
GOHSU02	CDS	GOHSU_02_01780	189513..190511		conserved hypothetical protein [nite]			RER_41160 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	glutathione transferase		F1YNI9_9ACTO Glutathione transferase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_17355 PE=4 SV=1;glutathione transferase [Verrucosispora maris AB-18 - 032];COG0435 Predicted glutathione S-transferase;Predicted glutathione S-transferase [Posttranslational modification, protein turnover, chaperones]; COG0435;Protein Disulfide Oxidoreductases and Other Proteins with a Thioredoxin fold; cl00388;C-terminal, alpha helical domain of Class Omega-like Glutathione S-transferases; cd03190;putative dimer interface [polypeptide binding];N-terminal domain interface [polypeptide binding];putative substrate binding pocket (H-site) [chemical binding]
GOHSU02	CDS	GOHSU_02_01790	190600..190980		conserved hypothetical protein [nite]			RER_29730 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	alkylphosphonate utilization operon protein PhnA		G0CYN4_CORUB Putative uncharacterized protein OS=Co rynebacterium ulcerans (strain BR-AD22) GN=CULC22_00805 PE =4 SV=1;alkylphosphonate utilization operon protein PhnA [P sychrobacter cryohalolentis K5];Uncharacterized Zn-ribbon-containing protein involved in phosphonate metabolism [Inorganic ion transport and metabolism]; COG2824;PhnA Zinc-Ribbon; pfam08274;PhnA protein; pfam03831
GOHSU02	CDS	GOHSU_02_01800	190992..192296		putative major facilitator superfamily transporter [nite]			MLP_46220 :putative major facilitator superfamily transporter : : : Microlunatus phosphovorus NM-1	major facilitator superfamily protein		H0QX18_9ACTO Putative major facilitator superfamily transporter OS=Gordonia effusa NBRC 100432 GN=GOEFS_028_0 0330 PE=4 SV=1;major facilitator superfamily protein [Xylanimonas cellulosilytica DSM 15894];PFAM: major facilitator superfamily MFS_1; protein of unknown function DUF894 DitE; KEGG: bpy:Bphyt_5813 major facilitator superfamily MFS_1;H+ Antiporter protein; TIGR00900;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU02	CDS	GOHSU_02_01810	complement(192297..193001)		not assigned						
GOHSU02	CDS	GOHSU_02_01820	complement(193005..193277)	overlap	3-oxoacyl-ACP reductase [migap]				3-oxoacyl-ACP reductase		H8FSI0_RHOMO 3-oxoacyl-(Acyl-carrier-protein) reduc tase OS=Phaeospirillum molischianum DSM 120 GN=fabG PE=4 S V=1;3-oxoacyl-ACP reductase [Sphingobium chlorophenolic um L- 1];TIGRFAM: 3-oxoacyl-(acyl-carrier-protein) reductase; KEGG: sjp:SJA_C1-16060 3-oxoacyl-[acyl-carrier-protein] reductase; PFAM: Short-chain dehydrogenase/reductase SDR;3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05653;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU02	CDS	GOHSU_02_01830	193183..195552	overlap	transcriptional accessory protein [nite]			ROP_65920 :transcriptional accessory protein : : : Rhodococcus opacus B4	putative RNA-binding protein		H6N531_GORPV Putative RNA-binding protein OS=Gordon ia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_174 p00570 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transcriptional accessory protein [Transcription]; COG2183;Tex-like protein N-terminal domain; pfam09371;Holliday junction resolvases (HJRs) are endonucleases that specifically resolve Holliday junction DNA intermediates during homologous recombination.  HJR's occur in archaea, bacteria, and in the mitochondria of certain fungi, however this CD includes...; cl00243;S1_like: Ribosomal protein S1-like RNA-binding domain. Found in a wide variety of RNA-associated proteins. Originally identified in S1 ribosomal protein. This superfamily also contains the Cold Shock Domain (CSD), which is a homolog of the S1 domain; cl09927;RNA binding site [nucleotide binding]
GOHSU02	CDS	GOHSU_02_01840	complement(195732..196028)		hypothetical protein [nite]			KSE_53080 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		G0HBY8_CORVD Putative uncharacterized protein OS=Co rynebacterium variabile (strain DSM 44702 / JCM 12073 / NC IMB 30131) GN=CVAR_0494 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF503; KEGG: gme:Gmet_2411 hypothetical protein;Protein of unknown function (DUF503); cl00669
GOHSU02	CDS	GOHSU_02_01850	complement(196042..197271)		hypothetical protein [nite]			KRH_02000 :hypothetical protein : : : Kocuria rhizophila DC2201	hypothetical protein		D0L917_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_2791 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU02	CDS	GOHSU_02_01860	complement(197268..198122)		polyphosphate kinase 2 [nite]	 ppk2 [nite]		MLP_50300 :polyphosphate kinase 2 : ppk2 : 2.7.4.1 : Microlunatus phosphovorus NM-1	polyphosphate kinase 2 domain-containing protein		H6N541_GORPV Polyphosphate kinase 2 domain-containi ng protein OS=Gordonia polyisoprenivorans (strain DSM 4426 6 / VH2) GN=GPOL_174p00670 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Polyphosphate kinase 2 (PPK2); cl11971
GOHSU02	CDS	GOHSU_02_01870	complement(198203..199210)		malate dehydrogenase [hamap]	mdh [hamap]	MF_01517:malate dehydrogenase:mdh 	ROP_63030 :malate dehydrogenase : mdh : 1.1.1.37 : Rhodococcus opacus B4	malate dehydrogenase		H5UHU3_9ACTO Malate dehydrogenase OS=Gordonia terra e NBRC 100016 GN=mdh PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.831) with cleavage site probability 0.380 at residue 21;KEGG: cja:CJA_1983 malate dehydrogenase; TIGRFAM: malate dehydrogenase; PFAM: lactate/malate dehydrogenase;malate dehydrogenase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding];LDH/MDH dimer interface [polypeptide binding];substrate binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_01880	complement(199273..200022)		hypothetical protein [nite]			RER_23460 :hypothetical protein : : : Rhodococcus erythropolis PR4	twin-arginine translocation pathway signal		Q1GFD5_SILST Twin-arginine translocation pathway si gnal OS=Silicibacter sp. (strain TM1040) GN=TM1040_1898 PE =4 SV=1;twin-arginine translocation pathway signal [Ruegeri a sp. TM1040];TIGRFAM: Twin-arginine translocation pathway signal: (0.15); PFAM: protein of unknown function DUF306, Meta and HslJ: (9.4e-21); KEGG: sil:SPO2951 hypothetical protein, ev=9e-14, 41% identity;META domain; cl01245
GOHSU02	CDS	GOHSU_02_01890	200192..202483		protein translocase subunit SecA [hamap]	secA [hamap]	MF_01382:protein translocase subunit SecA:secA 	DMR_07450 :preprotein translocase SecA subunit : secA : : Desulfovibrio magneticus RS-1	preprotein translocase subunit SecA	secA2	F1YP53_9ACTO Protein translocase subunit SecA 2 OS= Gordonia neofelifaecis NRRL B-59395 GN=secA2 PE=3 SV=1;preprotein translocase subunit SecA [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;SecA2; functions in protein export; can interact with acidic membrane phospholipids and the SecYEG protein complex; binds to preproteins; binds to ATP and undergoes a conformational change to promote membrane insertion of SecA/bound preprotein; ATP hydrolysis appears to drive release of the preprotein from SecA and deinsertion of SecA from the membrane; additional proteins SecD/F/YajC aid SecA recycling; exists in an equilibrium between monomers and dimers; may possibly form higher order oligomers; proteins in this cluster correspond to SecA2; which is non-essential and seems to play a role in secretion of a subset of proteins;preprotein translocase subunit SecA; Reviewed;SecA preprotein cross-linking domain; cl03078;Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_01900	202480..203160		putative phosphatidylglycerophosphate synthase [nite]			RER_32290 :putative phosphatidylglycerophosphate synthase : : 2.7.8.5 : Rhodococcus erythropolis PR4	CDP-diacylglycerol-glycerol-3-phosphate 3-phosphatidyl-transferase		F1YP52_9ACTO CDP-diacylglycerol--glycerol-3-phospha te 3-phosphatidyltransferase OS=Gordonia neofelifaecis NRR L B-59395 GN=SCNU_18572 PE=3 SV=1;CDP-diacylglycerol-glycerol-3-phosphate 3-phosphati dyl-transferase [Streptomyces clavuligerus ATCC 27064];CDP-alcohol phosphatidyltransferase; cl00453
GOHSU02	CDS	GOHSU_02_01910	203246..203644	case	glycine cleavage system H protein [hamap]	gcvH [hamap]	MF_00272:glycine cleavage system H protein:gcvH 	ROP_06420 :glycine cleavage system H-protein : gcvH : : Rhodococcus opacus B4	glycine cleavage system protein H		F1YP51_9ACTO Glycine cleavage system H protein OS=G ordonia neofelifaecis NRRL B-59395 GN=gcvH PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: glycine cleavage system protein H; PFAM: glycine cleavage H-protein; KEGG: afw:Anae109_2523 glycine cleavage system protein H;Glycine cleavage H-protein. Glycine cleavage H-proteins are part of the glycine cleavage system (GCS) found in bacteria, archea and the mitochondria of eukaryotes. GCS is a multienzyme complex consisting of 4 different components (P-, H-, T- and...; cd06848;lipoyl attachment site [posttranslational modification]
GOHSU02	CDS	GOHSU_02_01920	203832..204275		putative signal transduction protein GarA [nite]	 garA [nite]		RER_32240 :putative signal transduction protein GarA : garA : : Rhodococcus erythropolis PR4	FHA domain-containing protein		H0R4P2_9ACTO Signal transduction protein GarA OS=Go rdonia effusa NBRC 100432 GN=garA PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Forkhead associated domain (FHA); found in eukaryotic and prokaryotic proteins. Putative nuclear signalling domain. FHA domains may bind phosphothreonine, phosphoserine and sometimes phosphotyrosine. In eukaryotes, many FHA domain-containing proteins...; cd00060;phosphopeptide binding site
GOHSU02	CDS	GOHSU_02_01930	204290..205021		putative MerR family transcriptional regulator [nite]			ROP_06400 :putative MerR family transcriptional regulator : : : Rhodococcus opacus B4	MerR family transcriptional regulator		F1YP49_9ACTO Regulatory protein MerR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_18557 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: MerR family transcriptional regulator; SMART: MerR family transcriptional regulator; KEGG: ret:RHE_CH00094 MerR family transcriptional regulator;Helix-Turn-Helix DNA binding domain of MerR-like transcription regulators; cd00592;MerR HTH family regulatory protein; pfam13411;DNA binding residues [nucleotide binding];dimer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_01940	205066..205566		hypothetical protein [nite]			ROP_06390 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YP48_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18552 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Bifunctional nuclease; cl00553
GOHSU02	CDS	GOHSU_02_01950	205961..206587		putative MerR family transcriptional regulator [nite]			ROP_06380 :putative MerR family transcriptional regulator : : : Rhodococcus opacus B4	transcriptional regulator		F9W1V2_9ACTO Putative MerR family transcriptional r egulator OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_117_ 00730 PE=4 SV=1;transcriptional regulator [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Helix-Turn-Helix DNA binding domain of transcription regulators from the MerR superfamily; cl02600;MerR HTH family regulatory protein; pfam13411;DNA binding residues [nucleotide binding]
GOHSU02	CDS	GOHSU_02_01960	206919..209804		glycine dehydrogenase [hamap]	gcvP [hamap]	MF_00711:glycine dehydrogenase:gcvP 	RER_32200 :glycine dehydrogenase : gcvP : 1.4.4.2 : Rhodococcus erythropolis PR4	glycine dehydrogenase	gcvP1	F1YP46_9ACTO Glycine dehydrogenase [decarboxylating ] OS=Gordonia neofelifaecis NRRL B-59395 GN=gcvP PE=3 SV=1 ;glycine dehydrogenase [Rhodococcus jostii RHA1];acts in conjunction with GvcH to form H-protein-S-aminomethyldihydrolipoyllysine from glycine;glycine dehydrogenase; Provisional;Glycine cleavage system P-protein, alpha- and beta-subunits. This family consists of Glycine cleavage system P-proteins EC:1.4.4.2 from bacterial, mammalian and plant sources. The P protein is part of the glycine decarboxylase multienzyme complex EC:2.1; cd00613;tetramer interface [polypeptide binding];pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU02	CDS	GOHSU_02_01970	complement(209865..211223)		flavin-containing amine oxidase [nite]			ROP_04530 :flavin-containing amine oxidase : : 1.4.3.- : Rhodococcus opacus B4	flavin-containing amine oxidase		H5RLL5_9NOCA Oxidoreductase OS=Nocardia brasiliensi s ATCC 700358 GN=O3I_02105 PE=4 SV=1;flavin-containing amine oxidase [Rhodococcus opacus B4];Monoamine oxidase [Amino acid transport and metabolism];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU02	CDS	GOHSU_02_01980	complement(211331..212452)		hypothetical protein [nite]			ROP_06350 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YP44_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18532 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Bacterial extracellular solute-binding protein; pfam13531
GOHSU02	CDS	GOHSU_02_01990	complement(212543..214759)		malate synthase G [hamap]	glcB [hamap]	MF_00641:malate synthase G:glcB 	RER_32170 :malate synthase : glcB : 2.3.3.9 : Rhodococcus erythropolis PR4	malate synthase G	glcB	G7H0R7_9ACTO Malate synthase G OS=Gordonia araii NB RC 100433 GN=glcB PE=3 SV=1;malate synthase G [Rhodococcus opacus B4];catalyzes the formation of malate from glyoxylate and acetyl-CoA;Pyruvate kinase (PK):  Large allosteric enzyme that regulates glycolysis through binding of the substrate, phosphoenolpyruvate, and one or more allosteric effectors. Like other allosteric enzymes, PK has a high substrate affinity R state and a low...; cl09155
GOHSU02	CDS	GOHSU_02_02000	complement(214862..215770)		conserved hypothetical protein [nite]			RER_30950 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5UHS9_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_123_00180 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU02	CDS	GOHSU_02_02010	complement(215770..216849)		hypothetical membrane protein [nite]			ROP_70250 :hypothetical membrane protein : : : Rhodococcus opacus B4	integral membrane protein		F1YP41_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18517 PE=4 SV=1;integral membrane protein [Streptomyces roseosporus NRRL 15998];overlaps another CDS with the same product name;Domain of unknown function DUF21; pfam01595;This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domains associated with the CorC_HlyC domain. CorC_HlyC is a transporter associated domain. This small domain is found in Na+/H+ antiporters, in proteins involved in...; cd04590
GOHSU02	CDS	GOHSU_02_02020	complement(216842..218266)		conserved hypothetical membrane protein [nite]			RER_30970 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	putative integral membrane protein		F1YP40_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18512 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG1253 Hemolysins and related proteins containing CBS domains;Hemolysins and related proteins containing CBS domains [General function prediction only]; COG1253;Domain of unknown function DUF21; pfam01595;This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domains associated with the CorC_HlyC domain. CorC_HlyC is a transporter associated domain. This small domain is found in Na+/H+ antiporters, in proteins involved in...; cd04590;Transporter associated domain; cl08393
GOHSU02	CDS	GOHSU_02_02030	complement(218277..219716)		putative inosine-5'-monophosphate dehydrogenase [nite]			ROP_70270 :putative inosine-5'-monophosphate dehydrogenase : : 1.1.1.205 : Rhodococcus opacus B4	IMP dehydrogenase		F1YP39_9ACTO Inosine 5-monophosphate dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18507 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];inosine 5-monophosphate dehydrogenase; Validated;KEGG: afw:Anae109_3133 inosine 5-monophosphate dehydrogenase; TIGRFAM: IMP dehydrogenase; PFAM: IMP dehydrogenase/GMP reductase; CBS domain containing protein;TIM barrel proteins share a structurally conserved phosphate binding motif and in general share an eight beta/alpha closed barrel structure. Specific for this family is the conserved phosphate binding site at the edges of strands 7 and 8. The phosphate...; cl09108;The CBS domain, named after human CBS, is a small domain originally identified in cystathionine beta-synthase and is subsequently found in a wide range of different proteins. CBS domains usually occur in tandem repeats. They associate to form a so-called...; cd02205;IMPDH: The catalytic domain of the inosine monophosphate dehydrogenase. IMPDH catalyzes the NAD-dependent oxidation of inosine 5'-monophosphate (IMP) to xanthosine 5' monophosphate (XMP). It is a rate-limiting step in the de novo synthesis of the guanine...; cd00381
GOHSU02	CDS	GOHSU_02_02040	complement(219753..220697)		hypothetical protein [nite]			ROP_70320 :hypothetical protein : : : Rhodococcus opacus B4	peptidase M48 Ste24p		F1YP38_9ACTO Peptidase M48 Ste24p OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_18502 PE=3 SV=1;peptidase M48 Ste24p [Catenulispora acidiphila DSM 44928];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.971) with cleavage site probability 0.885 at residue 27;PFAM: peptidase M48 Ste24p; KEGG: aau:AAur_1766 M48 family peptidase;Peptidase family M48; cl12018
GOHSU02	CDS	GOHSU_02_02050	complement(220844..221227)		putative BlaI family transcriptional regulator [nite]			ROP_70330 :putative BlaI family transcriptional regulator : : : Rhodococcus opacus B4	CopY family transcriptional regulator		F1YP37_9ACTO CopY family transcriptional regulator OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18497 PE=4 SV=1;CopY family transcriptional regulator [Mycobacteriu m sp. JLS];PFAM: Penicillinase repressor; KEGG: mmc:Mmcs_2810 transcriptional repressor, CopY family;Helix-turn-helix domains; cl00088
GOHSU02	CDS	GOHSU_02_02060	221517..221954		thioesterase [nite]			ROP_70420 :thioesterase : : 3.1.2.- : Rhodococcus opacus B4	thioesterase superfamily protein		F1YP36_9ACTO Thioesterase superfamily protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18492 PE=4 SV=1;thioesterase superfamily protein [Conexibacter woes ei DSM 14684];PFAM: thioesterase superfamily protein; KEGG: aav:Aave_0099 hypothetical protein;PaaI_thioesterase is a tetrameric acyl-CoA thioesterase with a hot dog fold and one of several proteins responsible for phenylacetic acid (PA) degradation in bacteria.  Although orthologs of PaaI exist in archaea and eukaryotes, their function has not...; cd03443;CoenzymeA binding site [chemical binding];subunit interaction site [polypeptide binding];PHB binding site
GOHSU02	CDS	GOHSU_02_02070	complement(222133..223656)		probable NADH dehydrogenase [nite]	 ndh [nite]		RER_31110 :probable NADH dehydrogenase : ndh : 1.6.99.3 : Rhodococcus erythropolis PR4	NADH dehydrogenase	ndh1	F1YP35_9ACTO NADH dehydrogenase (Ubiquinone) OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_18487 PE=4 SV=1;NADH dehydrogenase [Rhodococcus jostii RHA1];NADH dehydrogenase, FAD-containing subunit [Energy production and conversion]; COG1252;Pyridine nucleotide-disulphide oxidoreductase; pfam00070
GOHSU02	CDS	GOHSU_02_02080	complement(223749..224381)		putative acetyltransferase [nite]			KSE_15950 :putative acetyltransferase : : 2.3.1.- : Kitasatospora setae NBRC 14216	hypothetical protein		F1YP34_9ACTO GCN5-like N-acetyltransferase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_18482 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: hypothetical protein;Acetyltransferase (GNAT) domain; pfam13302;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU02	CDS	GOHSU_02_02090	complement(224439..224624)	overlap	not assigned						
GOHSU02	CDS	GOHSU_02_02100	224559..224885	overlap	hypothetical protein [nite]			KRH_11010 :hypothetical protein : : : Kocuria rhizophila DC2201	thioesterase superfamily protein		G7H028_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_033_00040 PE=4 SV=1;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;PFAM: thioesterase superfamily protein; KEGG: bcv:Bcav_2639 thioesterase superfamily protein;4-hydroxybenzoyl-CoA thioesterase (4HBT). Catalyzes the final step in the 4-chlorobenzoate degradation pathway in which 4-chlorobenzoate is converted to 4-hydroxybenzoate in certain soil-dwelling bacteria. 4HBT forms a homotetramer with four active sites; cd00586
GOHSU02	CDS	GOHSU_02_02110	224991..228326		exodeoxyribonuclease V gamma chain [nite]	 recC [nite]		RER_08570 :exodeoxyribonuclease V gamma chain : recC : 3.1.11.5 : Rhodococcus erythropolis PR4	exodeoxyribonuclease V gamma chain RecC	recC	F1YP32_9ACTO Exodeoxyribonuclease V, gamma subunit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18472 PE=4 SV=1;recC gene product [Gordonia polyisoprenivorans VH2] ;Exodeoxyribonuclease V, gamma subunit; pfam04257;exonuclease V subunit gamma; Provisional; PRK11069
GOHSU02	CDS	GOHSU_02_02120	228323..231661		exodeoxyribonuclease V beta chain [nite]	 recB [nite]		RER_08580 :exodeoxyribonuclease V beta chain : recB : 3.1.11.5 : Rhodococcus erythropolis PR4	exodeoxyribonuclease V		F1YP31_9ACTO Exodeoxyribonuclease V OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_18467 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: UvrD/REP helicase; KEGG: gur:Gura_0579 exodeoxyribonuclease V subunit beta;UvrD/REP helicase N-terminal domain; pfam00580;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;exodeoxyribonuclease V, beta subunit; TIGR00609;Family description; cl15862;CRISPR/Cas system-associated protein Cas4; cl00641
GOHSU02	CDS	GOHSU_02_02130	231658..233529		exodeoxyribonuclease V alpha chain [nite]	 recD [nite]		ROP_15180 :exodeoxyribonuclease V alpha chain : recD : 3.1.11.5 : Rhodococcus opacus B4	exodeoxyribonuclease V subunit alpha		F1YP30_9ACTO Exodeoxyribonuclease V subunit alpha O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18462 PE=4 S V=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: exodeoxyribonuclease V, subunit alpha; KEGG: lch:Lcho_1778 exodeoxyribonuclease V, subunit alpha;exodeoxyribonuclease V, alpha subunit; TIGR01447;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Family description; cl15862
GOHSU02	CDS	GOHSU_02_02140	complement(233580..234365)		N-acetyl-1-D-myo-Inosityl-2-amino-2-deoxy-alpha-D-glucopyranoside deacetylase MshB [nite]	 mshB [nite]		ROP_59960 :N-acetyl-1-D-myo-Inosityl-2-amino-2-deoxy-alpha-D-glucopyranoside deacetylase MshB : mshB : : Rhodococcus opacus B4	1D-myo-inosityl-2-acetamido-2-deoxy-alpha-D- glucopyranoside deacetylase		F1YP29_9ACTO LmbE family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_18457 PE=4 SV=1;1D-myo-inosityl-2-acetamido-2-deoxy-alpha-D- glucop yranoside deacetylase [Streptomyces viridochromogenes DSM 40736];GlcNAc-PI de-N-acetylase; cl00929
GOHSU02	CDS	GOHSU_02_02150	234458..236374		chaperone protein HtpG [hamap]	htpG [hamap]	MF_00505:chaperone protein HtpG:htpG 	RER_50170 :chaperone protein HtpG : htpG : : Rhodococcus erythropolis PR4	heat shock protein 90		F1YP28_9ACTO Chaperone protein htpG OS=Gordonia neo felifaecis NRRL B-59395 GN=htpG PE=3 SV=1;heat shock protein 90 [Methylobacillus flagellatus KT];molecular chaperone;heat shock protein 90; Provisional;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU02	CDS	GOHSU_02_02160	236382..236882		hypothetical protein [nite]			ROP_23940 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YP27_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18447 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF04075; match to protein family HMM TIGR00026;Domain of unknown function (DUF385); cl04387
GOHSU02	CDS	GOHSU_02_02170	complement(236895..237728)		putative enoyl-CoA isomerase [nite]			RER_06160 :putative enoyl-CoA isomerase : : 5.3.3.- : Rhodococcus erythropolis PR4	enoyl-CoA hydratase/isomerase		F1YP26_9ACTO Enoyl-CoA hydratase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_18442 PE=3 SV=1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];enoyl-CoA hydratase; Provisional;PFAM: enoyl-CoA hydratase/isomerase; KEGG: dol:Dole_3232 enoyl-CoA hydratase/isomerase;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU02	CDS	GOHSU_02_02180	complement(237811..239580)		hypothetical membrane protein [nite]			ROP_36300 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YP25_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18437 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Predicted membrane protein (DUF2319);Predicted membrane protein [Function unknown];Predicted membrane protein (DUF2319); pfam10081
GOHSU02	CDS	GOHSU_02_02190	239723..240463		glyoxalase/bleomycin resistance protein/dioxygenase [migap]				glyoxalase/bleomycin resistance protein/dioxygenase		F1YP24_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18432 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: glyoxalase/bleomycin resistance protein/dioxygenase; KEGG: mab:MAB_4629 hypothetical protein;Glyoxalase/Bleomycin resistance protein/Dioxygenase superfamily; pfam00903;This domain superfamily is found in a variety of structurally related metalloproteins, including the type I extradiol dioxygenases, glyoxalase I and a group of antibiotic resistance proteins; cd06587;metal binding site [ion binding]
GOHSU02	CDS	GOHSU_02_02200	240536..241444		hypothetical membrane protein [nite]			ROP_41050 :hypothetical membrane protein : : : Rhodococcus opacus B4	putative integral membrane protein		D0L449_GORB4 Integral membrane protein-like protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_0962 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted integral membrane protein (DUF2189); cl02291
GOHSU02	CDS	GOHSU_02_02210	complement(241489..241707)		putative ClC chloride channel [nite]			RER_55830 :putative ClC chloride channel : : : Rhodococcus erythropolis PR4	voltage-gated cloride-channel family protein		H0JMZ4_9NOCA Cl-channel voltage-gated family protei n OS=Rhodococcus pyridinivorans AK37 GN=AK37_04857 PE=4 SV =1;unnamed protein product [Gordonia polyisoprenivoran s VH2];ClC chloride channel EriC.  This domain is found in the EriC chloride transporters that mediate the extreme acid resistance response in eubacteria and archaea. This response allows bacteria to survive in the acidic environments by decarboxylation-linked...; cd01031
GOHSU02	CDS	GOHSU_02_02220	complement(241707..242879)		putative ClC chloride channel [nite]			RER_55830 :putative ClC chloride channel : : : Rhodococcus erythropolis PR4	voltage-gated cloride-channel family protein		H5TFX3_9ACTO Putative ClC chloride channel OS=Gordo nia otitidis NBRC 100426 GN=GOOTI_005_00590 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];ClC chloride channel EriC.  This domain is found in the EriC chloride transporters that mediate the extreme acid resistance response in eubacteria and archaea. This response allows bacteria to survive in the acidic environments by decarboxylation-linked...; cd01031;Voltage gated chloride channel; pfam00654;Cl- selectivity filter;Cl- binding residues [ion binding];pore gating glutamate residue;dimer interface [polypeptide binding];H+/Cl- coupling transport residue
GOHSU02	tRNA	GOHSU_02_t0030	complement(243000..243088)		not assigned						identified by tRNAscan-SE; putative
GOHSU02	CDS	GOHSU_02_02230	243265..244569		peptidase M20 family protein [nite]			ROP_06280 :peptidase M20 family protein : : : Rhodococcus opacus B4	peptidase M20		H5U9G9_9ACTO Peptidase M20 family protein OS=Gordon ia terrae NBRC 100016 GN=GOTRE_014_00720 PE=4 SV=1;peptidase M20 [Frankia sp. EuI1c];PFAM: peptidase M20; peptidase dimerisation domain protein; KEGG: fal:FRAAL4849 hypothetical protein;hypothetical protein; Provisional;M20 Peptidase, carboxypeptidase yscS-like; cd05675;putative metal binding site [ion binding]
GOHSU02	CDS	GOHSU_02_02240	244759..245448		hypothetical protein [migap]				hypothetical protein		Q172A7_AEDAE Thrombospondin OS=Aedes aegypti GN=AAE L007455 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GK14198 gene product from transcript GK14198- RA
GOHSU02	CDS	GOHSU_02_02250	245564..246106		hypothetical protein [nite]			ROP_06270 :hypothetical protein : : : Rhodococcus opacus B4	PEBP family protein		H5TWF3_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_022_01550 PE=4 SV=1;PEBP family protein [Actinosynnema mirum DSM 43827] ;PFAM: PEBP family protein; KEGG: PEBP family protein;PhosphatidylEthanolamine-Binding Protein (PEBP) domain present in bacteria and archaea; cd00865;substrate binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_02260	246271..246897		not assigned						
GOHSU02	CDS	GOHSU_02_02270	complement(247052..248182)		dihydroorotate dehydrogenase [hamap]	pyrD [hamap]	MF_00225:dihydroorotate dehydrogenase:pyrD 	RER_32110 :dihydroorotate dehydrogenase : pyrD : 1.3.3.1 : Rhodococcus erythropolis PR4	dihydroorotate dehydrogenase 2	pyrD	F1YGM7_9ACTO Dihydroorotate dehydrogenase (quinone) OS=Gordonia neofelifaecis NRRL B-59395 GN=pyrD PE=3 SV=1;dihydroorotate dehydrogenase 2 [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the conversion of dihydroorotate to orotate in the pyrimidine biosynthesis pathway; uses a flavin nucleotide as an essential cofactor; class 2 enzymes are monomeric and compared to the class 1 class 2 possess an extended N terminus, which plays a role in the membrane association of the enzyme and provides the binding site for the respiratory quinones that serve as physiological electron acceptors;Dihydroorotate dehydrogenase (DHOD) class 2. DHOD catalyzes the oxidation of (S)-dihydroorotate to orotate. This is the fourth step and the only redox reaction in the de novo biosynthesis of UMP, the precursor of all pyrimidine nucleotides. DHOD requires...; cd04738;quinone interaction residues [chemical binding];Dihydroorotate dehydrogenase [Nucleotide transport and metabolism]; COG0167;catalytic residues [active];FMN binding site [chemical binding];substrate binding site [chemical binding]
GOHSU02	CDS	GOHSU_02_02280	complement(248197..249063)	overlap	hypothetical protein [nite]			ROP_06240 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YGM8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04951 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: bra:BRADO4012  signal peptide
GOHSU02	CDS	GOHSU_02_02290	248977..249318	overlap	not assigned						
GOHSU02	CDS	GOHSU_02_02300	249311..250150	case	undecaprenyl-diphosphatase [hamap]	uppP [hamap]	MF_01006:undecaprenyl-diphosphatase:uppP 	ROP_06170 :undecaprenyl-diphosphatase : uppP : 3.6.1.27 : Rhodococcus opacus B4	undecaprenyl pyrophosphate phosphatase	bacA1	G7GJE8_9ACTO Undecaprenyl-diphosphatase OS=Gordonia amarae NBRC 15530 GN=uppP PE=3 SV=1;undecaprenyl pyrophosphate phosphatase [Corynebacte rium urealyticum DSM 7109];BacA; phosphatase activity in Escherichia coli not kinase; involved in bacitracin resistance as bacitracin supposedly sequesters undecaprenyl disphosphate which reduces the pool of lipid carrier available to the cell;Bacitracin resistance protein BacA; cl00858
GOHSU02	CDS	GOHSU_02_02310	250221..250910		conserved hypothetical protein [nite]			RER_32010 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	phosphoglycerate mutase		F1YGN0_9ACTO Phosphoglycerate mutase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_04961 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: phosphoglycerate mutase; KEGG: rec:RHECIAT_CH0002879  phosphoglycerate mutase protein;Histidine phosphatase domain found in phosphoglycerate mutases and related proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cd07067;catalytic core [active]
GOHSU02	CDS	GOHSU_02_02320	250969..251799		hypothetical protein [migap]				hypothetical protein		F1YGN1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04966 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 4 : Homologs of previously reported genes of unknown function;Protein of unknown function (DUF3090); cl12832
GOHSU02	CDS	GOHSU_02_02330	251796..252611		conserved hypothetical protein [nite]			RER_31990 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	phosphatidylinositol 3- and 4-kinase catalytic subunit		F1YGN2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04971 PE=4 SV=1;phosphatidylinositol 3- and 4-kinase catalytic subu nit [Mycobacterium vanbaalenii PYR- 1];KEGG: mmc:Mmcs_3155 phosphatidylinositol 3- and 4-kinase, catalytic;Phosphoinositide 3-kinase (PI3K)-like family, catalytic domain; The PI3K-like catalytic domain family is part of a larger superfamily that includes the catalytic domains of other kinases such as the typical serine/threonine/tyrosine protein kinases (PKs); cl00119
GOHSU02	CDS	GOHSU_02_02340	252634..253902		cysteine--1D-myo-inosityl 2-amino-2-deoxy-alpha-D-glucopyranoside ligase MshC [nite]	 mshC [nite]		ROP_06130 :cysteine--1D-myo-inosityl 2-amino-2-deoxy-alpha-D-glucopyranoside ligase MshC : mshC : : Rhodococcus opacus B4	cysteinyl-tRNA synthetase	cysS	G7GJG3_9ACTO L-cysteine:1D-myo-inositol 2-amino-2-d eoxy-alpha-D-glucopyranoside ligase OS=Gordonia amarae NBR C 15530 GN=mshC PE=3 SV=1;cysteinyl-tRNA synthetase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;catalyzes a two-step reaction; charges a cysteine by linking its carboxyl group to the alpha-phosphate of ATP then transfers the aminoacyl-adenylate to its tRNA;cysteine--1-D-myo-inosityl 2-amino-2-deoxy-alpha-D-glucopyranoside ligase; TIGR03447;nucleotidyl transferase superfamily; cl00015;HIGH motif;nucleotide binding site [chemical binding];KMSKS motif;Anticodon-binding domain of class Ia aminoacyl tRNA synthetases and similar domains; cl12020;tRNA binding surface [nucleotide binding];anticodon binding site
GOHSU02	CDS	GOHSU_02_02350	253912..254220		hypothetical protein [migap]				hypothetical protein		F1YGN4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04981 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU02	CDS	GOHSU_02_02360	complement(254249..255124)		conserved hypothetical protein [nite]			RER_31900 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GJG5_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_04_00570 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: protein of unknown function DUF75; KEGG: rha:RHA1_ro00860 hypothetical protein;PAC2 family; cl00847
GOHSU02	CDS	GOHSU_02_02370	255182..258784		methionine synthase [nite]	 metH [nite]		ROP_05970 :methionine synthase : metH : 2.1.1.13 : Rhodococcus opacus B4	methionine synthase		F9W0G0_9ACTO Methionine synthase OS=Gordonia alkani vorans NBRC 16433 GN=metH PE=3 SV=1;methionine synthase [Streptomyces clavuligerus ATCC 27064];Methionine synthase I (cobalamin-dependent), methyltransferase domain [Amino acid transport and metabolism]; cl14105;5-methyltetrahydrofolate--homocysteine methyltransferase; TIGR02082;MeTr subgroup of pterin binding enzymes. This family includes cobalamin-dependent methyltransferases such as methyltetrahydrofolate, corrinoid iron-sulfur protein methyltransferase (MeTr) and methionine synthase (MetH).  Cobalamin-dependent...; cd00740;substrate binding pocket [chemical binding];B12 binding domain of methionine synthase. This domain binds methylcobalamin, which it uses as an intermediate methyl carrier from methyltetrahydrofolate (CH3H4folate) to homocysteine (Hcy); cd02069;B12 binding site [chemical binding];cobalt ligand [ion binding];Vitamin B12 dependent methionine synthase, activation domain; pfam02965
GOHSU02	CDS	GOHSU_02_02380	258879..259841		conserved hypothetical membrane protein [nite]			RER_30990 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H6MXS1_GORPV Putative membrane protein OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c07730 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU02	CDS	GOHSU_02_02390	259855..260559		putative hydrolase [nite]			ROP_05960 :putative hydrolase : : : Rhodococcus opacus B4	phosphatase/phosphohexomutase		F1YGN7_9ACTO HAD-superfamily hydrolase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_05006 PE=4 SV=1;phosphatase/phosphohexomutase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;Predicted phosphatase/phosphohexomutase [General function prediction only];Haloacid dehalogenase-like hydrolases. The haloacid dehalogenase-like (HAD) superfamily includes L-2-haloacid dehalogenase, epoxide hydrolase, phosphoserine phosphatase, phosphomannomutase, phosphoglycolate phosphatase, P-type ATPase, and many others; cd01427;motif II
GOHSU02	CDS	GOHSU_02_02400	260586..260849		phosphoribosyl-ATP pyrophosphatase [hamap]	hisE [hamap]	MF_01020:phosphoribosyl-ATP pyrophosphatase:hisE 	ROP_05930 :phosphoribosyl-ATP pyrophosphatase : hisE : 3.6.1.31 : Rhodococcus opacus B4	phosphoribosyl-ATP pyrophosphatase	hisE	F1YGN8_9ACTO Phosphoribosyl-ATP pyrophosphatase OS= Gordonia neofelifaecis NRRL B-59395 GN=hisE PE=3 SV=1;phosphoribosyl-ATP pyrophosphatase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;MazG nucleotide pyrophosphohydrolase domain; cl00345;catalyzes the formation of 1-(5-phosphoribosyl)-AMP from 1-(5-phosphoribolsyl)-ATP in histidine biosynthesis
GOHSU02	CDS	GOHSU_02_02410	260865..261764		ATP phosphoribosyltransferase [hamap]	hisG [hamap]	MF_00079:ATP phosphoribosyltransferase:hisG 	RER_31840 :ATP phosphoribosyltransferase : hisG : 2.4.2.17 : Rhodococcus erythropolis PR4	ATP phosphoribosyltransferase	hisG	F1YGN9_9ACTO ATP phosphoribosyltransferase OS=Gordo nia neofelifaecis NRRL B-59395 GN=hisG PE=3 SV=1;ATP phosphoribosyltransferase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;ATP phosphoribosyltransferase; Reviewed; PRK00489;long form of enzyme; catalyzes the formation of N'-5'-phosphoribosyl-ATP from phosphoribosyl pyrophosphate; crucial role in histidine biosynthesis; forms active dimers and inactive hexamers which is dependent on concentration of substrates and inhibitors;ATP phosphoribosyltransferase; cl15266;HisG, C-terminal domain; cl06867
GOHSU02	CDS	GOHSU_02_02420	complement(261828..262691)		hypothetical protein [nite]			RER_31830 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YGP0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05021 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Thioesterase-like superfamily; pfam13622
GOHSU02	CDS	GOHSU_02_02430	262872..263702		putative tRNA methyltransferase [nite]			ROP_05890 :putative tRNA methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	SAM-dependent methyltransferase involved in tRNA-Met maturation	pimT	F1YGP1_9ACTO tRNA (Adenine-N(1)-)-methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_05026 PE=4 SV=1;SAM-dependent methyltransferase involved in tRNA-Me t maturation [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;tRNA(1-methyladenosine) methyltransferase and related methyltransferases [Translation, ribosomal structure and biogenesis]; COG2519;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440
GOHSU02	CDS	GOHSU_02_02440	263788..265476		ATPase ARC [nite]	 arc [nite]		ROP_05870 :ATPase ARC : arc : 3.6.-.- : Rhodococcus opacus B4	proteasome-associated ATPase Arc	arc	F1YGP2_9ACTO Proteasome-associated ATPase OS=Gordon ia neofelifaecis NRRL B-59395 GN=arc PE=3 SV=1;arc gene product [Gordonia polyisoprenivorans VH2];proteasome ATPase; TIGR03689;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger
GOHSU02	CDS	GOHSU_02_02450	265532..268108		hypothetical membrane protein [nite]			ROP_49820 :hypothetical membrane protein : : : Rhodococcus opacus B4	transmembrane ABC transporter ATP-binding protein		F1YGP3_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_05036 PE=4 SV=1;transmembrane ABC transporter ATP-binding protein [ Mycobacterium marinum M];thought to be involved in active transport of undeterminated substrate (possibly lipooligosaccharide) across the membrane. responsible for energy coupling to the transport system and for the translocation of the substrate across the membrane;Forkhead associated domain (FHA); found in eukaryotic and prokaryotic proteins. Putative nuclear signalling domain. FHA domains may bind phosphothreonine, phosphoserine and sometimes phosphotyrosine. In eukaryotes, many FHA domain-containing proteins...; cd00060;phosphopeptide binding site;ABCG transporters are involved in eye pigment (EP) precursor transport, regulation of lipid-trafficking mechanisms, and pleiotropic drug resistance (DR).  DR is a well-described phenomenon occurring in fungi and shares several similarities with processes...; cd03213;ABC-type phosphate/phosphonate transport system, ATPase component [Inorganic ion transport and metabolism];Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region;ABC-2 type transporter; cl11417
GOHSU02	CDS	GOHSU_02_02460	268177..269679		putative proteasome accessory factor [nite]			RER_31760 :putative proteasome accessory factor : : : Rhodococcus erythropolis PR4	proteasome component		F1YGP4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05041 PE=4 SV=1;unnamed protein product [Thermobifida fusca YX];Pup-ligase protein; cl15463
GOHSU02	CDS	GOHSU_02_02470	269745..269939		putative proteasome accessory factor [nite]			RER_31750 :putative proteasome accessory factor : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YGP5_9ACTO Prokaryotic ubiquitin-like protein Pup OS=Gordonia neofelifaecis NRRL B-59395 GN=pup PE=3 SV=1;unnamed protein product [Mycobacterium leprae TN];Pup-like protein; cl05289
GOHSU02	CDS	GOHSU_02_02480	269936..270790		20S proteasome beta-type subunit [nite]	 prcB [nite]		ROP_05820 :20S proteasome beta-type subunit : prcB : 3.4.25.1 : Rhodococcus opacus B4	20S proteasome beta-subunit		F9W0E0_9ACTO Proteasome subunit beta OS=Gordonia al kanivorans NBRC 16433 GN=prcB PE=3 SV=1;20S proteasome beta-subunit [Streptomyces clavulige rus ATCC 27064];20S proteasome, alpha and beta subunits [Posttranslational modification, protein turnover, chaperones]; COG0638;proteasome_protease_HslV. This group contains the eukaryotic proteosome alpha and beta subunits and the prokaryotic protease hslV subunit. Proteasomes are large multimeric self-compartmentalizing proteases, involved in the clearance of misfolded proteins; cd01906
GOHSU02	CDS	GOHSU_02_02490	270787..271581		20S proteasome alpha-type subunit [nite]	 prcA [nite]		RER_31730 :20S proteasome alpha-type subunit : prcA : 3.4.25.1 : Rhodococcus erythropolis PR4	proteasome subunit alpha	prcA	F9W0D9_9ACTO Proteasome subunit alpha OS=Gordonia a lkanivorans NBRC 16433 GN=prcA PE=3 SV=1;prcA gene product [Gordonia polyisoprenivorans VH2] ;proteasome, alpha subunit, bacterial type; TIGR03691;The Ntn hydrolases (N-terminal nucleophile) are a diverse superfamily of of enzymes that are activated autocatalytically via an N-terminally lcated nucleophilic amino acid.  N-terminal nucleophile (NTN-) hydrolase superfamily, which contains a...; cl00467
GOHSU02	CDS	GOHSU_02_02500	271587..272945		proteasome accessory factor A [nite]	 pafA [nite]		RER_31710 :proteasome accessory factor A : pafA : : Rhodococcus erythropolis PR4	hypothetical protein		F1YGP8_9ACTO Pup--protein ligase OS=Gordonia neofel ifaecis NRRL B-59395 GN=pafA PE=3 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];Pup-ligase protein; cl15463
GOHSU02	CDS	GOHSU_02_02510	272963..273979		proteasome accessory factor B [nite]	 pafB [nite]		ROP_05780 :proteasome accessory factor B : pafB : : Rhodococcus opacus B4	transcriptional regulator		D0LDT4_GORB4 Transcriptional regulator protein-like protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_2466 PE=4 SV=1;transcriptional regulator [Nocardia farcinica IFM 1 0152];Predicted transcriptional regulator [Transcription];WYL domain; cl14852
GOHSU02	CDS	GOHSU_02_02520	273976..274935		proteasome accessory factor C [nite]	 pafC [nite]		RER_31690 :proteasome accessory factor C : pafC : : Rhodococcus erythropolis PR4	putative transcriptional regulator		F1YGQ0_9ACTO Transcriptional regulator-like protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_05071 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted transcriptional regulator [Transcription];WYL domain; cl14852
GOHSU02	CDS	GOHSU_02_02530	274991..275242		Sec-independent protein translocase protein TatA/E homolog [hamap]	tatA [hamap]	MF_00236:Sec-independent protein translocase protein TatA/E homolog:tatA 	ROP_05750 :Sec-independent protein translocase protein TatA : tatA : : Rhodococcus opacus B4	twin-arginine translocation protein TatA		F9W0D5_9ACTO Sec-independent protein translocase pr otein TatA/E homolog OS=Gordonia alkanivorans NBRC 16433 G N=tatA PE=3 SV=1;twin-arginine translocation protein, TatA/E family subunit [Streptomyces flavogriseus ATCC 33331];KEGG: sgr:SGR_5870 twin arginine translocase A; TIGRFAM: twin-arginine translocation protein, TatA/E family subunit; PFAM: sec-independent translocation protein mttA/Hcf106;mttA/Hcf106 family; cl00788
GOHSU02	CDS	GOHSU_02_02540	275260..276267		putative Sec-independent protein translocase protein TatC [nite]	 tatC [nite]		RER_31670 :putative Sec-independent protein translocase protein TatC : tatC : : Rhodococcus erythropolis PR4	Sec-independent protein translocase subunit TatC		F1YGQ2_9ACTO Sec-independent protein translocase, T atC subunit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _05081 PE=3 SV=1;Sec-independent protein translocase subunit TatC [N akamurella multipartita DSM 44233];TIGRFAM: Sec-independent protein translocase, TatC subunit; PFAM: Sec-independent periplasmic protein translocase; KEGG: sfu:Sfum_0692 sec-independent protein translocase, TatC subunit;Sec-independent protein translocase protein (TatC); cl00521
GOHSU02	CDS	GOHSU_02_02550	276284..279094		putative helicase [nite]			RER_31660 :putative helicase : : 3.6.1.- : Rhodococcus erythropolis PR4	DEAD/DEAH box helicase		F1YGQ3_9ACTO DEAD/DEAH box helicase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 05086 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: DEAD/DEAH box helicase; DSH domain protein; helicase; SMART: DEAD/DEAH box helicase ; helicase domain protein; KEGG: predicted protein;Superfamily II RNA helicase [DNA replication, recombination, and repair];DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];DSHCT (NUC185) domain; pfam08148
GOHSU02	CDS	GOHSU_02_02560	279220..>279701	3x codon	not assigned						
##GOHSU03	contig00013				25212
GOHSU03	CDS	GOHSU_03_00010	<1..868	3x codon frame	predicted protein [migap]				predicted protein		H5U9C8_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_014_00310 PE=4 SV=1;predicted protein [Streptomyces sp. AA4];Domain of unknown function (DUF4333); pfam14230
GOHSU03	CDS	GOHSU_03_00020	complement(894..1832)		putative 5'-3' exonuclease [nite]			ROP_05710 :putative 5'-3' exonuclease : : 3.1.11.- : Rhodococcus opacus B4	putative 5'-3' exonuclease		F1YGQ5_9ACTO 5'-3' exonuclease OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_05096 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];5'-3' exonuclease; smart00475;PIN domain of the 5'-3' exonuclease of Taq DNA polymerase I and homologs; cd09859;metal binding site 1 [ion binding];putative 5' ssDNA interaction site;metal binding site 3;metal binding site 2 [ion binding];H3TH domain of the 5'-3' exonuclease of Taq DNA polymerase I and homologs; cd09898;putative DNA binding site [nucleotide binding];putative metal binding site [ion binding]
GOHSU03	CDS	GOHSU_03_00030	1931..3085		putative Xaa-Pro dipeptidase [nite]	 pepQ [nite]		ROP_05700 :putative Xaa-Pro dipeptidase : pepQ : 3.4.13.9 : Rhodococcus opacus B4	peptidase M24		G7GJJ6_9ACTO Xaa-Pro dipeptidase OS=Gordonia amarae NBRC 15530 GN=pepQ PE=3 SV=1;peptidase M24 [Catenulispora acidiphila DSM 44928];PFAM: peptidase M24; creatinase; KEGG: saq:Sare_2376 peptidase M24;Creatinase/Prolidase N-terminal domain; pfam01321;Xaa-Pro aminopeptidase [Amino acid transport and metabolism]; COG0006;Similar to Prolidase and Aminopeptidase P. The members of this subfamily presumably catalyse hydrolysis of Xaa-Pro dipeptides and/or release of any N-terminal amino acid, including proline, that is linked with proline; cd01092
GOHSU03	CDS	GOHSU_03_00040	complement(3103..3522)		hypothetical protein [nite]			ROP_05680 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F9W0C9_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_099_00580 PE=4 SV= 1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];Pyridoxine 5'-phosphate (PNP) oxidase-like proteins; cl00381
GOHSU03	CDS	GOHSU_03_00050	3605..4354		putative oxidoreductase [nite]			ROP_05670 :putative oxidoreductase : : : Rhodococcus opacus B4	short chain dehydrogenase		F1YGQ8_9ACTO Short-chain dehydrogenase/reductase SD R OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_05111 PE= 4 SV=1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;Short-chain dehydrogenases of various substrate specificities [General function prediction only]; COG0300;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU03	CDS	GOHSU_03_00060	4351..5574		precorrin-6Y C5,15-methyltransferase [nite]	 cobL [nite]		ROP_05660 :precorrin-6Y C5,15-methyltransferase : cobL : 2.1.1.132 : Rhodococcus opacus B4	precorrin-6Y C(5,15)-methyltransferase	cobL	F9W0C7_9ACTO Precorrin-6Y C5,15-methyltransferase O S=Gordonia alkanivorans NBRC 16433 GN=cobL PE=4 SV=1;cobL gene product [Gordonia polyisoprenivorans VH2] ;Tetrapyrrole (Corrin/Porphyrin) Methylases; pfam00590;Precorrin-3B methylase [Coenzyme metabolism]; cl00304;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cl15754
GOHSU03	CDS	GOHSU_03_00070	5571..6347		precorrin-4 C11-methyltransferase [nite]	 cobM [nite]		ROP_05650 :precorrin-4 C11-methyltransferase : cobM : 2.1.1.133 : Rhodococcus opacus B4	precorrin-4 C(11)-methyltransferase		F1YGR0_9ACTO Precorrin-4 C11-methyltransferase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_05121 PE=3 SV=1 ;unnamed protein product [Gordonia bronchialis DSM 4 3247];Precorrin-3B methylase [Coenzyme metabolism]; cl00304;KEGG: mlo:mlr1385 precorrin 3 methylase; TIGRFAM: precorrin-4 C11-methyltransferase; PFAM: uroporphyrin-III C/tetrapyrrole (Corrin/Porphyrin) methyltransferase;Tetrapyrrole (Corrin/Porphyrin) Methylases; pfam00590
GOHSU03	CDS	GOHSU_03_00080	6340..7086		putative precorrin-6X reductase [nite]	 cobK [nite]		KSE_64840 :putative precorrin-6X reductase : cobK : 1.3.1.54 : Kitasatospora setae NBRC 14216	precorrin-6A reductase CobK	cobK	F1YGR1_9ACTO Precorrin-6x reductase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_05126 PE=4 SV=1;cobK gene product [Gordonia polyisoprenivorans VH2] ;Precorrin-6x reductase CbiJ/CobK; cl00922;Precorrin-6x reductase [Coenzyme metabolism]; COG2099
GOHSU03	CDS	GOHSU_03_00090	complement(7099..8298)		not assigned						
GOHSU03	CDS	GOHSU_03_00100	complement(8337..9875)		precorrin-2 C(20)-methyltransferase/precorrin-3B C(17)-methyltransferase [nite]	 cobIJ [nite]		RER_31440 :precorrin-2 C(20)-methyltransferase/precorrin-3B C(17)-methyltransferase : cobIJ : 2.1.1.130/2.1.1.131 : Rhodococcus erythropolis PR4	precorrin-3 methyltransferase / precorrin-2 C20-methyltransferase		F1YGR3_9ACTO Precorrin-3B C17-methyltransferase OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_05136 PE=3 SV= 1;unnamed protein product [Nocardioides sp. JS614];TIGRFAM: precorrin-3B C17-methyltransferase; precorrin-2 C20-methyltransferase; PFAM: Uroporphyrin-III C/tetrapyrrole (Corrin/Porphyrin) methyltransferase; KEGG: tfu:Tfu_0316 precorrin-3B C17-methyltransferase region:precorrin-2 C20-methyltransferase;Tetrapyrrole (Corrin/Porphyrin) Methylases; pfam00590;Precorrin-3B methylase [Coenzyme metabolism]; cl00304
GOHSU03	CDS	GOHSU_03_00110	complement(9881..10504)		precorrin-8X methylmutase [nite]	 cobH [nite]		ROP_02930 :precorrin-8X methylmutase : cobH : 5.4.1.2 : Rhodococcus opacus B4	precorrin-8X methylmutase		F9W0C3_9ACTO Precorrin-8X methylmutase OS=Gordonia alkanivorans NBRC 16433 GN=cobH PE=4 SV=1;precorrin-8X methylmutase [Brucella pinnipedialis B 2 /94];Precorrin isomerase [Coenzyme metabolism]; cl00913;Precorrin-8X methylmutase; pfam02570
GOHSU03	CDS	GOHSU_03_00120	complement(10513..11676)		putative precorrin-3B synthase [nite]	 cobG [nite]		ROP_02920 :putative precorrin-3B synthase : cobG : 1.14.13.83 : Rhodococcus opacus B4	precorrin-3B synthase CobG	cobG	F1YGR5_9ACTO Precorrin-3B synthase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_05146 PE=4 SV=1;cobG gene product [Gordonia polyisoprenivorans VH2] ;precorrin-3B synthase; TIGR02435;Nitrite/Sulfite reductase ferredoxin-like half domain; pfam03460
GOHSU03	CDS	GOHSU_03_00130	11850..12473		hypothetical protein [nite]			ROP_11590 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G2P2K5_STRVO Putative uncharacterized protein OS=St reptomyces violaceusniger Tu 4113 GN=Strvi_2404 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.999) with cleavage site probability 0.782 at residue 24;KEGG: esa:ESA_00774 hypothetical protein;Putative neutral zinc metallopeptidase; cl00988
GOHSU03	CDS	GOHSU_03_00140	12733..16356		cobaltochelatase CobN subunit [nite]	 cobN [nite]		ROP_02910 :cobaltochelatase CobN subunit : cobN : 6.6.1.2 : Rhodococcus opacus B4	aerobic cobaltochelatase subunit CobN	cobN	F1YGR6_9ACTO Cobaltochelatase subunit CobN OS=Gordo nia neofelifaecis NRRL B-59395 GN=cobN PE=4 SV=1;cobN gene product [Gordonia polyisoprenivorans VH2] ;cobaltochelatase subunit CobN; Reviewed; PRK05989;CobN subunit of cobaltochelatase, bchH and chlH subunits of magnesium chelatases, and similar proteins; cd10150
GOHSU03	CDS	GOHSU_03_00150	16469..17056		hypothetical membrane protein [nite]			ROP_02650 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein	fxsA	F1YGR7_9ACTO FxsA cytoplasmic membrane protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_05156 PE=4 SV=1 ;fxsA gene product [Gordonia polyisoprenivorans VH2] ;FxsA cytoplasmic membrane protein; cl01148
GOHSU03	CDS	GOHSU_03_00160	17104..18729		conserved hypothetical protein [nite]			RER_31300 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YGR8_9ACTO Amidohydrolase 3 OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_05161 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];Predicted metal-dependent hydrolase with the TIM-barrel fold [General function prediction only];YtcJ_like metal dependent amidohydrolases. YtcJ is a Bacillus subtilis ORF of unknown function. The Arabidopsis homolog LAF3 has been identified as a factor required for photochrome A signalling; cd01300
GOHSU03	CDS	GOHSU_03_00170	18726..20417		apolipoprotein N-acyltransferase [hamap]	lnt [hamap]	MF_01148:apolipoprotein N-acyltransferase:lnt 	RER_31290 :putative apolipoprotein N-acyltransferase : ppm2 : 2.3.1.- : Rhodococcus erythropolis PR4	apolipoprotein N-acyltransferase		F1YGR9_9ACTO Apolipoprotein N-acyltransferase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05166 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.860) with cleavage site probability 0.848 at residue 23;TIGRFAM: apolipoprotein N-acyltransferase; PFAM: nitrilase/cyanide hydratase and apolipoprotein N-acyltransferase; KEGG: ppr:PBPRA2882 apolipoprotein N- acyltransferase;apolipoprotein N-acyltransferase; TIGR00546;Apolipoprotein N-acyl transferase (class 9 nitrilases); cd07571;putative active site [active];catalytic triad [active];putative dimer interface [polypeptide binding]
GOHSU03	CDS	GOHSU_03_00180	20407..21192		putative polyprenol-phosphate mannosyltransferase [nite]	 ppm1 [nite]		ROP_02620 :putative polyprenol-phosphate mannosyltransferase : ppm1 : 2.4.1.- : Rhodococcus opacus B4	dolichol-phosphate mannosyltransferase Dpm	dpm	F1YGS0_9ACTO Dolichyl-phosphate beta-D-mannosyltran sferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_051 71 PE=4 SV=1;dpm gene product [Gordonia polyisoprenivorans VH2];DPM1_like represents putative enzymes similar to eukaryotic DPM1; cd06442;Ligand binding site;Putative Catalytic site;DXD motif
GOHSU03	CDS	GOHSU_03_00190	complement(21271..21615)		hypothetical protein [nite]			ROP_02600 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GJL7_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_04_01100 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];KEGG: fal:FRAAL3771 conserved hypothetical protein; putative copper ion binding domain;Domain of unknown function (DUF4109); pfam13397
GOHSU03	CDS	GOHSU_03_00200	21785..22066		conserved hypothetical protein [nite]			RER_31240 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0LDV6_GORB4 Putative uncharacterized protein (Prec ursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43 247 / JCM 3198 / NCTC 10667) GN=Gbro_2488 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU03	CDS	GOHSU_03_00210	22063..23205		hypothetical membrane protein [nite]			ROP_21760 :hypothetical membrane protein : : : Rhodococcus opacus B4	integral membrane protein		C3JHF3_RHOER Integral membrane protein OS=Rhodococc us erythropolis SK121 GN=RHOER0001_1488 PE=4 SV=1;integral membrane protein [Streptomyces lividans TK 24];Integral membrane protein TerC family; cl10468
GOHSU03	CDS	GOHSU_03_00220	23262..24080		probable adenylate cyclase [nite]	 cya [nite]		RER_14860 :probable adenylate cyclase : cya : 4.6.1.1 : Rhodococcus erythropolis PR4	adenylyl cyclase class-3/4/guanylyl cyclase		F1YGS3_9ACTO Adenylyl cyclase class-3/4/guanylyl cy clase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_05186 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: adenylyl cyclase class-3/4/guanylyl cyclase; SMART: adenylyl cyclase class-3/4/guanylyl cyclase; KEGG: bja:bll8011  hydrolase;Adenylate cyclase, family 3 (some proteins contain HAMP domain) [Signal transduction mechanisms]; COG2114;cyclase homology domain; cd07302;nucleotidyl binding site;metal binding site [ion binding];dimer interface [polypeptide binding]
GOHSU03	CDS	GOHSU_03_00230	24091..24366		hypothetical protein [migap]				hypothetical protein		F1YGS4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05191 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU03	CDS	GOHSU_03_00240	complement(24476..24664)		not assigned						
GOHSU03	CDS	GOHSU_03_00250	24864..>25212	3x codon	hypothetical membrane protein [nite]			ROP_49820 :hypothetical membrane protein : : : Rhodococcus opacus B4	Fis family sigma-54 dependent transcriptional regulator		G7GJI1_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_04_00740 PE=4 SV=1;Fis family sigma-54 dependent transcriptional regul ator [Stigmatella aurantiaca DW4/3- 1];Forkhead associated domain (FHA); found in eukaryotic and prokaryotic proteins. Putative nuclear signalling domain. FHA domains may bind phosphothreonine, phosphoserine and sometimes phosphotyrosine. In eukaryotes, many FHA domain-containing proteins...; cd00060;phosphopeptide binding site;Response regulator containing CheY-like receiver, AAA-type ATPase, and DNA-binding domains [Signal transduction mechanisms]; COG2204
##GOHSU04	contig00021				245187
GOHSU04	tRNA	GOHSU_04_t0010	complement(241..313)		not assigned						identified by tRNAscan-SE; putative
GOHSU04	tRNA	GOHSU_04_t0020	complement(391..465)		not assigned						identified by tRNAscan-SE; putative
GOHSU04	CDS	GOHSU_04_00010	complement(612..2096)		glutamyl-tRNA synthetase [nite]	 gltX [nite]		RER_24040 :glutamyl-tRNA synthetase : gltX : 6.1.1.17 : Rhodococcus erythropolis PR4	glutamyl-tRNA synthetase	gltX	F1YEZ5_9ACTO Glutamate--tRNA ligase OS=Gordonia neo felifaecis NRRL B-59395 GN=gltX PE=3 SV=1;glutamyl-tRNA synthetase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;Charges one glutamine molecule and pairs it to its corresponding RNA trinucleotide during protein translation;glutamyl-tRNA synthetase; Reviewed; PRK01406;catalytic core domain of discriminating glutamyl-tRNA synthetase; cd00808;HIGH motif;KMSKS motif
GOHSU04	CDS	GOHSU_04_00020	2191..2550		putative GntR family transcriptional regulator [nite]			RER_29210 :putative GntR family transcriptional regulator : : : Rhodococcus erythropolis PR4	transcriptional regulator		D1BJU8_SANKS Transcriptional regulator, GntR family OS=Sanguibacter keddieii (strain ATCC 51767 / DSM 10542 / NCFB 3025 / ST-74) GN=Sked_24450 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: bsu:BSU30460 hypothetical protein;Winged helix-turn-helix (WHTH) DNA-binding domain of the GntR family of transcriptional regulators; cd07377;DNA-binding site [nucleotide binding]
GOHSU04	CDS	GOHSU_04_00030	2537..3592		hypothetical membrane protein [nite]			RER_29200 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H8E7K2_9MICO Putative uncharacterized protein (Prec ursor) OS=Microbacterium laevaniformans OR221 GN=OR221_282 9 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];Evidence 4 : Homologs of previously reported genes of unknown function
GOHSU04	CDS	GOHSU_04_00040	complement(3564..4400)		putative hydrolase [nite]			ROP_48110 :putative hydrolase : : : Rhodococcus opacus B4	alpha/beta hydrolase		F1YEZ3_9ACTO Alpha/beta hydrolase fold protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_01325 PE=4 SV=1 ;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: alpha/beta hydrolase fold;Alpha/beta hydrolase family; pfam12697;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU04	CDS	GOHSU_04_00050	complement(4397..5167)		putative fumarylacetoacetate hydrolase family protein [nite]			RER_24020 :putative fumarylacetoacetate hydrolase family protein : : : Rhodococcus erythropolis PR4	5-oxopent-3-ene-1,2,5-tricarboxylate decarboxylase		F1YEZ2_9ACTO 5-carboxymethyl-2-hydroxymuconate delt a-isomerase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _01320 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF2437); pfam10370;2-keto-4-pentenoate hydratase/2-oxohepta-3-ene-1,7-dioic acid hydratase (catechol pathway) [Secondary metabolites biosynthesis, transport, and catabolism]; COG0179;Fumarylacetoacetate (FAA) hydrolase family; cl11421
GOHSU04	CDS	GOHSU_04_00060	5312..6466		putative MFS transporter [nite]			ROP_65440 :putative MFS transporter : : : Rhodococcus opacus B4	major facilitator superfamily protein		D0LC39_GORB4 Major facilitator superfamily MFS_1 (P recursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3221 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.982) with cleavage site probability 0.491 at residue 32;PFAM: major facilitator superfamily MFS_1; KEGG: vap:Vapar_3886 major facilitator superfamily MFS_1;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;Major Facilitator Superfamily; pfam07690;putative substrate translocation pore
GOHSU04	CDS	GOHSU_04_00070	complement(6487..7140)		hypothetical membrane protein [nite]			RER_47840 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F9VWJ1_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_061_00360 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU04	CDS	GOHSU_04_00080	complement(7221..8231)		3-isopropylmalate dehydrogenase [hamap]	leuB [hamap]	MF_01035:3-isopropylmalate dehydrogenase:leuB 	ROP_65430 :3-isopropylmalate dehydrogenase : leuB : 1.1.1.85 : Rhodococcus opacus B4	3-isopropylmalate dehydrogenase	leuB	F1YEY9_9ACTO 3-isopropylmalate dehydrogenase OS=Gor donia neofelifaecis NRRL B-59395 GN=leuB PE=3 SV=1;3-isopropylmalate dehydrogenase [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;Isocitrate/isopropylmalate dehydrogenase; cl00445;catalyzes the oxidation of 3-isopropylmalate to 3-carboxy-4-methyl-2-oxopentanoate in leucine biosynthesis;tartrate dehydrogenase; Provisional
GOHSU04	CDS	GOHSU_04_00090	complement(8228..9823)		D-3-phosphoglycerate dehydrogenase [nite]	 serA [nite]		ROP_65420 :D-3-phosphoglycerate dehydrogenase : serA : 1.1.1.95 : Rhodococcus opacus B4	D-3-phosphoglycerate dehydrogenase	serA	F1YEY8_9ACTO D-3-phosphoglycerate dehydrogenase OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01300 PE=3 SV= 1;D-3-phosphoglycerate dehydrogenase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 3-phosphonooxypyruvate from 3-phospho-D-glycerate in serine biosynthesis; can also reduce alpha ketoglutarate to form 2-hydroxyglutarate;D-3-phosphoglycerate dehydrogenase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;C-terminal ACT (regulatory) domain of D-3-phosphoglycerate dehydrogenase (3PGDH); cd04902;putative L-serine binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_00100	10096..10773		putative S1 family peptidase [nite]			RER_50550 :putative S1 family peptidase : : 3.4.21.- : Rhodococcus erythropolis PR4	S1 family peptidase		F1YEY7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01295 PE=3 SV=1;S1 family peptidase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887
GOHSU04	CDS	GOHSU_04_00110	complement(10872..11480)		hypothetical protein [nite]			MLP_03000 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hydroxyacylglutathione hydrolase		D1BFV6_SANKS Zn-dependent hydrolase, glyoxylase OS= Sanguibacter keddieii (strain ATCC 51767 / DSM 10542 / NCF B 3025 / ST-74) GN=Sked_15320 PE=4 SV=1;hydroxyacylglutathione hydrolase [Saccharopolyspora erythraea NRRL 2338];Metallo-beta-lactamase superfamily; cl00446
GOHSU04	CDS	GOHSU_04_00120	complement(11477..12595)		aryl-alcohol dehydrogenase AdhC [nite]	 adhC [nite]		ROP_23120 :aryl-alcohol dehydrogenase AdhC : adhC : 1.1.1.90 : Rhodococcus opacus B4	mycothiol-dependent formaldehyde dehydrogenase		C7QHY4_CATAD Mycothiol-dependent formaldehyde dehyd rogenase OS=Catenulispora acidiphila (strain DSM 44928 / N RRL B-24433 / NBRC 102108 / JCM 14897) GN=Caci_4165 PE=3 S V=1;mycothiol-dependent formaldehyde dehydrogenase [Str eptomyces violaceusniger Tu 4113];manually curated; TIGRFAM: Formaldehyde dehydrogenase, mycothiol-dependent; KEGG: sgr:SGR_5824  zinc-containing dehydrogenase; PFAM: Alcohol dehydrogenase, zinc-binding; Alcohol dehydrogenase GroES-like;mycothiol-dependent formaldehyde dehydrogenase; TIGR03451;Class III alcohol dehydrogenase; cd08279;NAD binding site [chemical binding];catalytic Zn binding site [ion binding];substrate binding site [chemical binding];structural Zn binding site [ion binding]
GOHSU04	CDS	GOHSU_04_00130	complement(12624..13064)		hypothetical protein [nite]			ROP_20550 :hypothetical protein : : : Rhodococcus opacus B4	polyketide cyclase/dehydrase		F1YEY5_9ACTO Polyketide cyclase/dehydrase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_01285 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: polyketide cyclase/dehydrase; KEGG: tgr:Tgr7_2654 cyclase/dehydrase;START/RHO_alpha_C/PITP/Bet_v1/CoxG/CalC (SRPBCC) ligand-binding domain superfamily; cd07812;hydrophobic ligand binding site
GOHSU04	CDS	GOHSU_04_00140	complement(13223..14236)		ketol-acid reductoisomerase [hamap]	ilvC [hamap]	MF_00435:ketol-acid reductoisomerase:ilvC 	RER_23990 :ketol-acid reductoisomerase : ilvC : 1.1.1.86 : Rhodococcus erythropolis PR4	ketol-acid reductoisomerase	ilvC	G7H4C7_9ACTO Ketol-acid reductoisomerase OS=Gordoni a araii NBRC 100433 GN=ilvC PE=3 SV=1;ketol-acid reductoisomerase [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of (R)-2,3-dihydroxy-3-methylbutanoate from (S)-2-hydroxy-2-methyl-3-oxobutanoate in valine and isoleucine biosynthesis;ketol-acid reductoisomerase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Acetohydroxy acid isomeroreductase, catalytic domain; pfam01450
GOHSU04	CDS	GOHSU_04_00150	complement(14315..14815)		acetolactate synthase small subunit [nite]	 ilvN [nite]		RER_23980 :acetolactate synthase small subunit : ilvN : 2.2.1.6 : Rhodococcus erythropolis PR4	acetolactate synthase small subunit IlvH	ilvH	F1YEY3_9ACTO Acetolactate synthase 3 regulatory sub unit OS=Gordonia neofelifaecis NRRL B-59395 GN=ilvH PE=4 S V=1;ilvH gene product [Gordonia polyisoprenivorans VH2] ;acetolactate synthase 3 regulatory subunit; Reviewed; PRK11895;N-terminal ACT domain of the Escherichia coli IlvH-like regulatory subunit of acetohydroxyacid synthase (AHAS); cd04878;putative valine binding site [chemical binding];dimer interface [polypeptide binding];Small subunit of acetolactate synthase; pfam10369
GOHSU04	CDS	GOHSU_04_00160	complement(14821..16722)		acetolactate synthase large subunit [nite]	 ilvB [nite]		ROP_65390 :acetolactate synthase large subunit : ilvB : 2.2.1.6 : Rhodococcus opacus B4	acetolactate synthase 1 catalytic subunit	ilvB	F1YEY2_9ACTO Acetolactate synthase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_01270 PE=3 SV=1;acetolactate synthase 1 catalytic subunit [Rhodococ cus opacus B4];acetolactate synthase large subunit; catalyzes the formation of 2-acetolactate from pyruvate;acetolactate synthase 1 catalytic subunit; Validated;Pyrimidine (PYR) binding domain of POX and related proteins; cd07035;PYR/PP interface [polypeptide binding];dimer interface [polypeptide binding];TPP binding site [chemical binding];Thiamine pyrophosphate enzyme, central domain; cl00815;Thiamine pyrophosphate (TPP) family, Acetohydroxyacid synthase (AHAS) subfamily, TPP-binding module; composed of proteins similar to the large catalytic subunit of AHAS. AHAS catalyzes the condensation of two molecules of pyruvate to give the...; cd02015;TPP-binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_00170	17001..17426		hypothetical membrane protein [nite]			ROP_65380 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H5UA08_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_018_00930 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF2581); pfam10756
GOHSU04	CDS	GOHSU_04_00180	17491..18828		magnesium transporter [nite]	 mgtE [nite]		OBV_06160 :magnesium transporter : mgtE : : Oscillibacter valericigenes Sjm18-20	Mg2+ transporter	mgtE2	F1YEY0_9ACTO Mg2+ transporter OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_01260 PE=4 SV=1;Mg2+ transporter [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;Mg/Co/Ni transporter MgtE (contains CBS domain) [Inorganic ion transport and metabolism]; COG2239;MgtE intracellular N domain; cl15244;This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domain in the magnesium transporter, MgtE.  MgtE and its homologs are found in eubacteria, archaebacteria, and eukaryota. Members of this family transport Mg2+ or other...; cd04606;Divalent cation transporter; cl00786
GOHSU04	CDS	GOHSU_04_00190	19161..21005		dihydroxy-acid dehydratase [hamap]	ilvD [hamap]	MF_00012:dihydroxy-acid dehydratase:ilvD 	RER_23950 :dihydroxy-acid dehydratase : ilvD : 4.2.1.9 : Rhodococcus erythropolis PR4	dihydroxy-acid dehydratase	ilvD	F1YEX9_9ACTO Dihydroxy-acid dehydratase OS=Gordonia neofelifaecis NRRL B-59395 GN=ilvD PE=3 SV=1;dihydroxy-acid dehydratase [Corynebacterium glutami cum ATCC 13032];IS fingerprint type 4-5;catalyzes the dehydration of 2,3-dihydroxy-3-methylbutanoate to 3-methyl-2-oxobutanoate in valine and isoleucine biosynthesis;Dehydratase family; cl00340
GOHSU04	CDS	GOHSU_04_00200	complement(21126..22163)		DoxX family protein [migap]				DoxX family protein		F1YEX8_9ACTO DoxX family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_01245 PE=4 SV=1;DoxX family protein [Mycobacterium sp. JLS];PFAM: DoxX family protein; KEGG: mmc:Mmcs_1904 DoxX
GOHSU04	CDS	GOHSU_04_00210	22245..23393		conserved hypothetical protein [nite]			RER_23920 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	glucose dehydrogenase		F1YEX7_9ACTO Glucose/sorbosone dehydrogenase-like p rotein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_0124 0 PE=4 SV=1;glucose dehydrogenase [Rhodococcus jostii RHA1];Glucose/sorbosone dehydrogenases [Carbohydrate transport and metabolism];dehydrogenase, PQQ-dependent, s-GDH family; TIGR03606
GOHSU04	CDS	GOHSU_04_00220	complement(23416..24921)		aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit B [hamap]	gatB [hamap]	MF_00121:aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit B:gatB 	ROP_65330 :aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit B : gatB : 6.3.5.6/6.3.5.7 : Rhodococcus opacus B4	aspartyl/glutamyl-tRNA amidotransferase subunit B	gatB	F1YEX6_9ACTO Aspartyl/glutamyl-tRNA amidotransferas e subunit B OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _01235 PE=3 SV=1;aspartyl/glutamyl-tRNA amidotransferase subunit B [ Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;allows the formation of correctly charged Asn-tRNA(Asn) or Gln-tRNA(Gln) through the transamidation of misacylated Asp-tRNA(Asn) or Glu-tRNA(Gln) in organisms which lack either or both of asparaginyl-tRNA or glutaminyl-tRNA synthetases; reaction takes place in the presence of glutamine and ATP through an activated phospho-Asp-tRNA(Asn) or phospho-Glu-tRNA;aspartyl/glutamyl-tRNA amidotransferase subunit B; Validated; PRK05477;GatB/GatE catalytic domain; pfam02934;GatB domain; cl11497
GOHSU04	CDS	GOHSU_04_00230	25024..25230		conserved hypothetical protein [nite]			RER_48860 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	phage shock protein PspC		H0R6S2_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_132_00040 PE=4 SV=1;phage shock protein PspC [Chloroflexus sp. Y-400- f l];PFAM: PspC domain protein; KEGG: cau:Caur_0706 PspC domain protein;PspC domain; cl00864
GOHSU04	CDS	GOHSU_04_00240	complement(25238..26269)		6-phosphofructokinase [hamap]	pfkA [hamap]	MF_00339:6-phosphofructokinase:pfkA 	KRH_05770 :6-phosphofructokinase : : 2.7.1.- : Kocuria rhizophila DC2201	6-phosphofructokinase		F1YEX4_9ACTO Phosphofructokinase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_01225 PE=3 SV=1;6-phosphofructokinase [Streptomyces himastatinicus ATCC 53653];Phosphofructokinase, a key regulatory enzyme in glycolysis, catalyzes the phosphorylation of fructose-6-phosphate to fructose-1,6-biphosphate. The members belong to PFK family that includes ATP- and pyrophosphate (PPi)- dependent phosphofructokinases; cd00363;ADP/pyrophosphate binding site [chemical binding];dimerization interface [polypeptide binding];allosteric effector site;fructose-1,6-bisphosphate binding site
GOHSU04	CDS	GOHSU_04_00250	complement(26383..27882)		glutamyl-tRNA(Gln) amidotransferase subunit A [hamap]	gatA [hamap]	MF_00120:glutamyl-tRNA(Gln) amidotransferase subunit A:gatA 	ROP_65270 :aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit A : gatA : 6.3.5.6/6.3.5.7 : Rhodococcus opacus B4	aspartyl/glutamyl-tRNA amidotransferase subunit A	gatA	F1YEW9_9ACTO Aspartyl/glutamyl-tRNA amidotransferas e subunit A OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _01200 PE=3 SV=1;aspartyl/glutamyl-tRNA amidotransferase subunit A [ Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;allows the formation of correctly charged Asn-tRNA(Asn) or Gln-tRNA(Gln) through the transamidation of misacylated Asp-tRNA(Asn) or Glu-tRNA(Gln) in organisms which lack either or both of asparaginyl-tRNA or glutaminyl-tRNA synthetases; reaction takes place in the presence of glutamine and ATP through an activated phospho-Asp-tRNA(Asn) or phospho-Glu-tRNA;Asp-tRNAAsn/Glu-tRNAGln amidotransferase A subunit and related amidases [Translation, ribosomal structure and biogenesis]; COG0154;Amidase; cl11426
GOHSU04	CDS	GOHSU_04_00260	complement(27885..28196)		aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit C [hamap]	gatC [hamap]	MF_00122:aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit C:gatC 	RER_23830 :aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit C : gatC : 6.3.5.6/6.3.5.7 : Rhodococcus erythropolis PR4	aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit C	gatC	F1YEW8_9ACTO Aspartyl/glutamyl-tRNA amidotransferas e subunit C OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _01195 PE=3 SV=1;gatC gene product [Gordonia polyisoprenivorans VH2] ;Glu-tRNAGln amidotransferase C subunit; cl00495
GOHSU04	CDS	GOHSU_04_00270	28440..29108		conserved hypothetical protein [nite]			RER_23820 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	amino acid-binding ACT domain-containing protein		F1YEW6_9ACTO Amino acid-binding ACT domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 01185 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: amino acid-binding ACT domain protein;ACT domains are commonly involved in specifically binding an amino acid or other small ligand leading to regulation of the enzyme; cl09141
GOHSU04	CDS	GOHSU_04_00280	complement(29119..31188)		DNA ligase [hamap]	ligA [hamap]	MF_01588:DNA ligase:ligA 	RER_23800 :NAD(+)-dependent DNA ligase : ligA : 6.5.1.2 : Rhodococcus erythropolis PR4	NAD-dependent DNA ligase LigA	ligA	D0LCW2_GORB4 DNA ligase OS=Gordonia bronchialis (st rain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=li gA PE=3 SV=1;NAD-dependent DNA ligase LigA [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;this protein catalyzes the formation of phosphodiester linkages between 5'-phosphoryl and 3'-hydroxyl groups in double-stranded DNA using NAD as a coenzyme and as the energy source for the reaction; essential for DNA replication and repair of damaged DNA; similar to ligase LigB;NAD-dependent DNA ligase LigA; Validated; PRK07956;NAD+ dependent DNA ligase adenylation domain. DNA ligases catalyze the crucial step of joining the breaks in duplex DNA during DNA replication, repair and recombination, utilizing either ATP or NAD(+) as a cofactor, but using the same basic reaction...; cd00114;nucleotide binding pocket [chemical binding];K-X-D-G motif;catalytic site [active];NAD-dependent DNA ligase OB-fold domain; pfam03120;NAD-dependent DNA ligase C4 zinc finger domain; pfam03119;Breast Cancer Suppressor Protein (BRCA1), carboxy-terminal domain. The BRCT domain is found within many DNA damage repair and cell cycle checkpoint proteins. The unique diversity of this domain superfamily allows BRCT modules to interact forming homo...; cd00027;Dimer interface [polypeptide binding]
GOHSU04	CDS	GOHSU_04_00290	31257..31643		hypothetical protein [nite]			YM304_04470 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	hypothetical protein		F1YEW4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01175 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF419); cl15265
GOHSU04	CDS	GOHSU_04_00300	complement(31649..32470)		putative thiol-disulfide oxidoreductase [nite]			RER_pREL1-02330 :putative thiol-disulfide oxidoreductase : : 1.8.-.- : Rhodococcus erythropolis PR4	cytochrome c biogenesis protein transmembrane region		Q3L9D7_RHOE4 Putative thiol-disulfide oxidoreductas e OS=Rhodococcus erythropolis (strain PR4 / NBRC 100887) G N=RER_pREL1-02330 PE=4 SV=1;cytochrome c biogenesis protein transmembrane regio n [Streptomyces griseus XylebKG- 1];Signal predicted by SignalP 3.0 HMM (Signal peptide probabilty 0.620) with cleavage site probability 0.207 at residue 25;PFAM: Cytochrome c assembly protein, transmembrane domain; KEGG: sgr:SGR_3886 putative integral membrane cytochrome biogenesis protein;Cytochrome C biogenesis protein transmembrane region; cl15788
GOHSU04	CDS	GOHSU_04_00310	complement(32487..33218)		putative thiol-disulfide oxidoreductase [nite]			RER_pREL1-02320 :putative thiol-disulfide oxidoreductase : : 1.8.-.- : Rhodococcus erythropolis PR4	DSBA-like thioredoxin domain-containing protein		G4J7K6_9PSEU DSBA oxidoreductase (Precursor) OS=Sac charomonospora paurometabolica YIM 90007 GN=SacpaDRAFT_423 1 PE=4 SV=1;DSBA-like thioredoxin domain-containing protein [St igmatella aurantiaca DW4/3- 1];Protein Disulfide Oxidoreductases and Other Proteins with a Thioredoxin fold; cl00388;Predicted dithiol-disulfide isomerase involved in polyketide biosynthesis [Secondary metabolites biosynthesis, transport, and catabolism]; COG2761
GOHSU04	CDS	GOHSU_04_00320	complement(33215..34501)		conserved hypothetical membrane protein [nite]			BBR47_51790 :conserved hypothetical membrane protein : : : Brevibacillus brevis NBRC 100599	permease	ydeG	E8VB38_BACST Putative permease OS=Bacillus subtilis (strain BSn5) GN=BSn5_14105 PE=4 SV=1;permease [Bacillus subtilis subsp. subtilis str. 16 8];Evidence 3: Function proposed based on presence of conserved amino acid motif, structural feature or limited homology; PubMedId: 15849754, 16850406; Product type pt: transporter;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU04	CDS	GOHSU_04_00330	34562..35212		putative GntR family transcriptional regulator [nite]			RER_05370 :putative GntR family transcriptional regulator : : : Rhodococcus erythropolis PR4	GntR family transcriptional regulator		I0H5E5_ACTMI Putative GntR-family transcriptional r egulator OS=Actinoplanes missouriensis 431 GN=AMIS_30120 P E=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];Transcriptional regulators [Transcription]; COG2186;Winged helix-turn-helix (WHTH) DNA-binding domain of the GntR family of transcriptional regulators; cd07377;DNA-binding site [nucleotide binding];FCD domain; cl11656
GOHSU04	CDS	GOHSU_04_00340	complement(35255..36259)		conserved hypothetical protein [nite]			RER_23790 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	vitamin-B12 independent methionine synthase		F1YEW3_9ACTO Methionine synthase vitamin-B12 indepe ndent OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01170 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];CIMS - Cobalamine-independent methonine synthase, or MetE. Many members have been characterized as 5-methyltetrahydropteroyltriglutamate-homocysteine methyltransferases, EC:2.1.1.14, mostly from bacteria and plants. This enzyme catalyses the last step in...; cd03310
GOHSU04	CDS	GOHSU_04_00350	36311..38242		N-6 DNA methylase [migap]				N-6 DNA methylase		F4GZG9_CELFA N-6 DNA methylase OS=Cellulomonas fimi (strain ATCC 484 / DSM 20113 / JCM 1341 / NBRC 15513 / NC IMB 8980 / NCTC 7547) GN=Celf_0750 PE=4 SV=1;N-6 DNA methylase [Cellulomonas fimi ATCC 484];PFAM: N-6 DNA methylase; KEGG: aat:D11S_1858 putative N-6 DNA methylase;Predicted DNA modification methylase [DNA replication, recombination, and repair];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU04	CDS	GOHSU_04_00360	complement(38239..39438)		tRNA-specific 2-thiouridylase MnmA [hamap]	mnmA [hamap]	MF_00144:tRNA-specific 2-thiouridylase MnmA:mnmA 	RER_23780 :probable tRNA uridine-sulfurtransferase : : 2.8.1.- : Rhodococcus erythropolis PR4	tRNA-specific 2-thiouridylase MnmA	mnmA	D0LCW6_GORB4 tRNA-specific 2-thiouridylase MnmA OS= Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=mnmA PE=3 SV=1;tRNA-specific 2-thiouridylase MnmA [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes a sulfuration reaction to synthesize 2-thiouridine at the U34 position of tRNAs;tRNA methyl transferase. This family represents tRNA(5-methylaminomethyl-2-thiouridine)-methyltransferase which is involved in the biosynthesis of the modified nucleoside 5-methylaminomethyl-2-thiouridine present in the wobble position of some tRNAs; cd01998
GOHSU04	CDS	GOHSU_04_00370	complement(39438..40607)	overlap	probable cysteine desulfurase [nite]	 iscS [nite]		RER_23770 :probable cysteine desulfurase : iscS : 2.8.1.7 : Rhodococcus erythropolis PR4	class V aminotransferase		G7GVG8_9ACTO Cysteine desulfurase OS=Gordonia amara e NBRC 15530 GN=iscS PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: aminotransferase class V; aromatic amino acid beta-eliminating lyase/threonine aldolase; KEGG: gur:Gura_1493 aminotransferase, class V;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;cysteine desulfurase DndA; TIGR03235;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU04	CDS	GOHSU_04_00380	complement(40544..41464)	overlap	putative 1-acylglycerol-3-phosphate O-acyltransferase [nite]	 plsC [nite]		RER_23760 :putative 1-acylglycerol-3-phosphate O-acyltransferase : plsC : 2.3.1.51 : Rhodococcus erythropolis PR4	1-acyl-sn-glycerol-3-phosphate acyltransferase		F1YEW0_9ACTO Lyso-ornithine lipid acyltransferase O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01155 PE=4 S V=1;1-acyl-sn-glycerol-3-phosphate acyltransferase [Sac charomonospora marina XMU15];Signal predicted by SignalP 3.0 HMM; IMG reference gene:2508557703_SP;PFAM: Acyltransferase; TIGRFAM: 1-acyl-sn-glycerol-3-phosphate acyltransferases;Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: AGPAT-like; cd07989;putative acyl-acceptor binding pocket
GOHSU04	CDS	GOHSU_04_00390	complement(41465..42313)		hypothetical protein [nite]			ROP_65170 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YEV9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01150 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU04	CDS	GOHSU_04_00400	complement(42475..43431)		putative electron transfer flavoprotein alpha subunit [nite]	 etfA [nite]		RER_23740 :putative electron transfer flavoprotein alpha subunit : etfA : : Rhodococcus erythropolis PR4	electron transfer flavoprotein		H0RJ13_9ACTO Electron transfer flavoprotein alpha s ubunit OS=Gordonia polyisoprenivorans NBRC 16320 GN=etfA P E=4 SV=1;electron transfer flavoprotein [Streptomyces sp. SP B78];Electron transfer flavoprotein, alpha subunit [Energy production and conversion]; COG2025;The electron transfer flavoprotein (ETF) serves as a specific electron acceptor for various mitochondrial dehydrogenases. ETF transfers electrons to the main respiratory chain via ETF-ubiquinone oxidoreductase. ETF is an heterodimer that consists of an...; cd01715;Electron transfer flavoprotein FAD-binding domain; pfam00766
GOHSU04	CDS	GOHSU_04_00410	complement(43470..44255)		electron transfer flavoprotein beta subunit [nite]	 etfB [nite]		ROP_65150 :electron transfer flavoprotein beta subunit : etfB : : Rhodococcus opacus B4	electron transfer flavoprotein beta subunit		F1YEV6_9ACTO Electron transfer flavoprotein beta-su bunit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01135 PE=4 SV=1;electron transfer flavoprotein beta subunit [Strept omyces griseus subsp. griseus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;The electron transfer flavoprotein (ETF) serves as a specific electron acceptor for various mitochondrial dehydrogenases. ETF transfers electrons to the main respiratory chain via ETF-ubiquinone oxidoreductase. ETF is an heterodimer that consists of an...; cd01714;Ligand binding site [chemical binding];Electron transfer flavoprotein domain; pfam01012
GOHSU04	CDS	GOHSU_04_00420	44499..45260		putative methyltransferase [nite]			RER_23720 :putative methyltransferase : : 2.1.1.- : Rhodococcus erythropolis PR4	3-demethylubiquinone-9 3-methyltransferase		F1YEV5_9ACTO Type 11 methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01130 PE=4 SV=1;3-demethylubiquinone-9 3-methyltransferase [Rhodoco ccus jostii RHA1];S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_00430	45257..46798		hypothetical protein [nite]			ROP_65130 :hypothetical protein : : : Rhodococcus opacus B4	glycoside hydrolase		F1YEV4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01125 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Uncharacterized conserved protein [Function unknown];Domain of unknown function (DUF1957); pfam09210
GOHSU04	CDS	GOHSU_04_00440	46795..48078		probable glucosyltransferase [nite]			RER_23700 :probable glucosyltransferase : : 2.4.1.- : Rhodococcus erythropolis PR4	group 1 glycosyl transferase		H6MVH6_GORPV Glycosyl transferase OS=Gordonia polyi soprenivorans (strain DSM 44266 / VH2) GN=GPOL_c30550 PE=4 SV=1;group 1 glycosyl transferase [Catenulispora acidiph ila DSM 44928];PFAM: glycosyl transferase group 1; Starch synthase catalytic domain protein; KEGG: sen:SACE_6200 transferase;Glycosyltransferase [Cell envelope biogenesis, outer membrane]; COG0438;This family is most closely related to the GT1 family of glycosyltransferases and named after YqgM in Bacillus licheniformis about which little is known. Glycosyltransferases catalyze the transfer of sugar moieties from activated donor molecules to...; cd03801
GOHSU04	CDS	GOHSU_04_00450	complement(48105..48869)		conserved hypothetical protein [nite]			RER_23680 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hexapeptide transferase		G7GVF7_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_69_01010 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00132;Acetyltransferase (isoleucine patch superfamily) [General function prediction only]; COG0110;Maltose O-acyltransferase (MAT)-like: This family is composed of maltose O-acetyltransferase, galactoside O-acetyltransferase (GAT), xenobiotic acyltransferase (XAT) and similar proteins. MAT and GAT catalyze the CoA-dependent acetylation of the...; cd04647;trimer interface [polypeptide binding];substrate binding site [chemical binding];CoA binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_00460	49058..50335		conserved hypothetical protein [nite]			RER_23670 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5U9X8_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_018_00630 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];FOG: WD40-like repeat [Function unknown];PQQ-dependent dehydrogenases and related proteins; cl11493
GOHSU04	CDS	GOHSU_04_00470	complement(50351..51586)		conserved hypothetical protein [nite]			RER_23660 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	SAM-dependent methyltransferase		D0LCX8_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_3266 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU04	CDS	GOHSU_04_00480	complement(51615..52625)		putative methyltransferase [nite]			ROP_65070 :putative methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	UbiE/COQ5 family methyltransferase		F1YEU3_9ACTO Type 11 methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01070 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_00490	complement(52639..53427)		putative enoyl-CoA hydratase [nite]			RER_23640 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase/carnithine racemase		G7H490_9ACTO Putative enoyl-CoA hydratase OS=Gordon ia araii NBRC 100433 GN=GOARA_061_01040 PE=4 SV=1;unnamed protein product [Desulfosporosinus orientis DSM 765];3-hydroxybutyryl-CoA dehydratase; Validated;PFAM: Enoyl-CoA hydratase/isomerase family;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU04	CDS	GOHSU_04_00500	53552..54022		hypothetical protein [nite]			KSE_64600 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		B2HEW7_MYCMM Conserved protein OS=Mycobacterium mar inum (strain ATCC BAA-535 / M) GN=MMAR_2946 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];Also detected in the extracellular matrix and membrane fraction by proteomics.; cytoplasmic protein
GOHSU04	CDS	GOHSU_04_00510	complement(54246..55076)		conserved hypothetical protein [nite]			RER_23630 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	NUDIX hydrolase		F1YEU0_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_01055 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: NUDIX hydrolase; KEGG: nfa:nfa42900 hypothetical protein;Nudix hydrolase is a superfamily of enzymes found in all three kingdoms of life, and it catalyzes the hydrolysis of NUcleoside DIphosphates linked to other moieties, X. Enzymes belonging to this superfamily require a divalent cation, such as Mg2+ or Mn2+...; cl00447;nudix motif
GOHSU04	CDS	GOHSU_04_00520	complement(55101..55934)		putative ABC transporter ATP-binding protein [nite]			ROP_65040 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	molybdenum ABC transporter ATPase		F1YET9_9ACTO ABC transporter related protein OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_01050 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00005;ABC-type Mn/Zn transport systems, ATPase component [Inorganic ion transport and metabolism]; COG1121;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU04	CDS	GOHSU_04_00530	56023..57096		conserved hypothetical protein [nite]			RER_15500 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	luciferase family oxidoreductase, group 1		F1YET8_9ACTO Luciferase family oxidoreductase, grou p 1 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01045 P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: luciferase family oxidoreductase, group 1; PFAM: Luciferase-like, subgroup; KEGG: scl:sce1809 hypothetical protein;Flavin-utilizing monoxygenases; cl07892;Coenzyme F420-dependent N5,N10-methylene tetrahydromethanopterin reductase and related flavin-dependent oxidoreductases [Energy production and conversion]
GOHSU04	CDS	GOHSU_04_00540	complement(57093..57785)		hypothetical protein [nite]			ROP_65010 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YET7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01040 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Peptidyl-tRNA hydrolase, type 2 (PTH2)_like . Peptidyl-tRNA hydrolase activity releases tRNA from the premature translation termination product peptidyl-tRNA. Two structurally different enzymes have been reported to encode such activity, Pth present in...; cl00869;dimer interface [polypeptide binding];putative active site [active]
GOHSU04	CDS	GOHSU_04_00550	57879..59066		hypothetical protein [nite]			KSE_59850 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		Q5YVB3_NOCFA Putative uncharacterized protein OS=No cardia farcinica (strain IFM 10152) GN=NFA_30310 PE=4 SV=1 ;unnamed protein product [Streptomyces bingchenggens is BCW- 1];Catalytic NodB homology domain of the carbohydrate esterase 4 superfamily; cl15692
GOHSU04	CDS	GOHSU_04_00560	complement(59126..60367)		phosphoserine phosphatase [nite]	 serB [nite]		RER_23580 :phosphoserine phosphatase : serB : 3.1.3.3 : Rhodococcus erythropolis PR4	phosphoserine phosphatase	serB	F1YET6_9ACTO Phosphoserine phosphatase SerB OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_01035 PE=4 SV=1;serB gene product [Mycobacterium smegmatis str. MC2 155];identified by match to protein family HMM PF00702; match to protein family HMM PF01842; match to protein family HMM TIGR00338; match to protein family HMM TIGR01488;CT domains found N-terminal of phosphoserine phosphatase (PSP, SerB); cd04870;ACT domains found N-terminal of phosphoserine phosphatase (PSP, SerB); cd04871;phosphoserine phosphatase SerB; TIGR00338;Haloacid dehalogenase-like hydrolases. The haloacid dehalogenase-like (HAD) superfamily includes L-2-haloacid dehalogenase, epoxide hydrolase, phosphoserine phosphatase, phosphomannomutase, phosphoglycolate phosphatase, P-type ATPase, and many others; cd01427;motif II
GOHSU04	CDS	GOHSU_04_00570	complement(60471..62183)		cytochrome c oxidase subunit I [nite]	 ctaD [nite]		RER_23570 :cytochrome c oxidase subunit I : ctaD : 1.9.3.1 : Rhodococcus erythropolis PR4	cytochrome-c oxidase		F1YET5_9ACTO Cytochrome c oxidase subunit I OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_01030 PE=3 SV=1;cytochrome-c oxidase [Mycobacterium gilvum PYR- GCK ];PFAM: cytochrome c oxidase, subunit I; KEGG: mmc:Mmcs_1852 cytochrome-c oxidase;Heme/copper-type cytochrome/quinol oxidases, subunit 1 [Energy production and conversion]; COG0843;Ubiquinol oxidase subunit I.  Ubiquinol oxidase, the terminal oxidase in the respiratory chains of aerobic bacteria, is a multi-chain transmembrane protein located in the cell membrane.  It catalyzes the reduction of O2 and simultaneously pumps protons...; cd01662;D-pathway;Putative ubiquinol binding site [chemical binding];Low-spin heme (heme b) binding site [chemical binding];Putative water exit pathway;Binuclear center (heme o3/CuB) [ion binding];K-pathway;Putative proton exit pathway
GOHSU04	CDS	GOHSU_04_00580	62422..63450		putative ABC transporter substrate-binding protein [nite]			RER_23550 :putative ABC transporter substrate-binding protein : : : Rhodococcus erythropolis PR4	periplasmic binding protein		F1YET4_9ACTO Periplasmic-binding protein OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_01025 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF01497;Helical backbone metal receptor (TroA-like domain). These proteins have been shown to function in the ABC transport of ferric siderophores and metal ions such as Mn2+, Fe3+, Cu2+ and/or Zn2+.  Their ligand binding site is formed in the interface between...; cl00262;Periplasmic binding protein; pfam01497;intersubunit interface [polypeptide binding]
GOHSU04	CDS	GOHSU_04_00590	63466..66450		putative helicase [nite]			ROP_12610 :putative helicase : : 3.6.1.- : Rhodococcus opacus B4	SNF2 family helicase		F1YET3_9ACTO SNF2-like protein OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_01020 PE=4 SV=1;unnamed protein product [Acaryochloris marina MBIC1 1017];type strain of Acaryochloris marina;SNF2 family N-terminal domain; pfam00176;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_00600	66450..67217		hypothetical protein [nite]			RER_39650 :hypothetical protein : : : Rhodococcus erythropolis PR4	predicted protein		F1YET2_9ACTO Zinc finger SWIM domain-containing pro tein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01015 PE=4 SV=1;predicted protein [Streptomyces sp. AA4];SWIM zinc finger; cl15408
GOHSU04	CDS	GOHSU_04_00610	complement(67219..68637)		hypothetical protein [nite]			MLP_05440 :hypothetical protein : : : Microlunatus phosphovorus NM-1	membrane protein		Q8NSM8_CORGL Membrane protein OS=Corynebacterium gl utamicum (strain ATCC 13032 / DSM 20300 / JCM 1318 / LMG 3 730 / NCIMB 10025) GN=cg0740 PE=4 SV=1;membrane protein [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;Predicted membrane protein [Function unknown];Bacterial PH domain; cl01348
GOHSU04	CDS	GOHSU_04_00620	complement(68634..69131)		conserved hypothetical protein [nite]			RER_41850 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	membrane-flanked domain-containing protein		E2MXC3_9CORY Membrane-flanked domain protein OS=Cor ynebacterium amycolatum SK46 GN=CORAM0001_1551 PE=4 SV=1;membrane-flanked domain-containing protein [Xylanim onas cellulosilytica DSM 15894];PFAM: membrane-flanked domain; KEGG: eli:ELI_13595 hypothetical protein;Bacterial PH domain; cl01348
GOHSU04	CDS	GOHSU_04_00630	69444..71438		hypothetical protein [migap]				hypothetical protein		H0R2V0_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_083_00330 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GH24424 gene product from transcript GH24424- RA;Domain of unknown function (DUF222); pfam02720
GOHSU04	CDS	GOHSU_04_00640	71634..72575		hypothetical protein [nite]			ROP_52740 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YET0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01005 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cl00264
GOHSU04	CDS	GOHSU_04_00650	72572..73267		hypothetical protein [migap]				hypothetical protein		G7H1Z7_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_048_00740 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU04	CDS	GOHSU_04_00660	complement(73347..74348)		ribonucleoside-diphosphate reductase beta chain [nite]	 nrdF [nite]		ROP_64950 :ribonucleoside-diphosphate reductase beta chain : nrdF : 1.17.4.1 : Rhodococcus opacus B4	ribonucleotide-diphosphate reductase subunit beta	nrdF2	G7H475_9ACTO Ribonucleoside-diphosphate reductase b eta chain OS=Gordonia araii NBRC 100433 GN=nrdF PE=4 SV=1;nrdF2 gene product [Mycobacterium smegmatis str. MC 2 155];ribonucleotide-diphosphate reductase subunit beta; Provisional; PRK13966;B2 or R2 protein; type 1b enzyme; catalyzes the rate-limiting step in dNTP synthesis; converts nucleotides to deoxynucleotides; forms a homodimer and then a multimeric complex with NrdE;Ribonucleotide Reductase, R2/beta subunit, ferritin-like diiron-binding domain; cd01049;dimer interface [polypeptide binding];putative radical transfer pathway;diiron center [ion binding];tyrosyl radical
GOHSU04	CDS	GOHSU_04_00670	74595..75212		conserved hypothetical protein [nite]			RER_52640 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	peptidase S16		H0R6Z2_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_132_00740 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;ATP-dependent protease La (LON) domain; pfam02190;KEGG: rer:RER_52640 hypothetical protein; PFAM: peptidase S16 lon domain protein; SMART: peptidase S16
GOHSU04	CDS	GOHSU_04_00680	complement(75252..77300)		putative phosphatase [nite]			RER_59640 :putative phosphatase : : : Rhodococcus erythropolis PR4	twin-arginine translocation pathway signal		H0R8N5_9ACTO Putative phosphatase OS=Gordonia polyi soprenivorans NBRC 16320 GN=GOPIP_008_00080 PE=4 SV=1;twin-arginine translocation pathway signal [Sacchar opolyspora erythraea NRRL 2338];Bacterial protein of unknown function (DUF839); pfam05787
GOHSU04	CDS	GOHSU_04_00690	77477..78241		conserved hypothetical membrane protein [nite]			RER_42050 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	abortive infection protein		H5TGV4_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_024_00160 PE=4 SV=1;abortive infection protein [Nakamurella multipartit a DSM 44233];PFAM: hypothetical protein; KEGG: pmy:Pmen_2685 abortive infection protein;CAAX protease self-immunity; cl00558
GOHSU04	CDS	GOHSU_04_00700	complement(78332..80563)		ribonucleoside-diphosphate reductase alpha chain [nite]	 nrdE [nite]		ROP_64940 :ribonucleoside-diphosphate reductase alpha chain : nrdE : 1.17.4.1 : Rhodococcus opacus B4	ribonucleotide-diphosphate reductase subunit alpha		H5U0U7_9ACTO Ribonucleoside-diphosphate reductase O S=Gordonia sputi NBRC 100414 GN=nrdE PE=3 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];ribonucleotide-diphosphate reductase subunit alpha; Validated;Catalyzes the rate-limiting step in dNTP synthesis;Ribonucleotide reductase N-terminal; pfam08343;Class I ribonucleotide reductase; cd01679;dimer interface [polypeptide binding];catalytic residues [active];effector binding site;R2 peptide binding site
GOHSU04	CDS	GOHSU_04_00710	complement(80632..81141)		protein NrdI [hamap]	nrdI [hamap]	MF_00128:protein NrdI:nrdI 	RER_23510 :NrdI protein : nrdI : : Rhodococcus erythropolis PR4	NrdI protein		F1YES3_9ACTO Protein nrdI OS=Gordonia neofelifaecis NRRL B-59395 GN=nrdI PE=3 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rer:RER_23510 NrdI protein; TIGRFAM: NrdI protein; PFAM: NrdI family protein;NADPH-dependent FMN reductase; cl00438
GOHSU04	CDS	GOHSU_04_00720	complement(81165..81398)		NrdH-redoxin [nite]	 nrdH [nite]		ROP_64920 :NrdH-redoxin : nrdH : : Rhodococcus opacus B4	glutaredoxin-like protein NrdH		H6MVK9_GORPV Glutaredoxin-like protein NrdH OS=Gord onia polyisoprenivorans (strain DSM 44266 / VH2) GN=nrdH P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: glutaredoxin-like protein NrdH; PFAM: glutaredoxin; glutaredoxin 2; KEGG: rlt:Rleg2_3500 glutaredoxin-like protein NrdH;NrdH-redoxin (NrdH) family; NrdH is a small monomeric protein with a conserved redox active CXXC motif within a TRX fold, characterized by a glutaredoxin (GRX)-like sequence and TRX-like activity profile. In vitro, it displays protein disulfide reductase...; cd02976;catalytic residues [active]
GOHSU04	CDS	GOHSU_04_00730	81745..82368		putative TetR family transcriptional regulator [nite]			ROP_64890 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		F1YES0_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_00955 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG1309 Transcriptional regulator;Helix-turn-helix domains; cl00088
GOHSU04	CDS	GOHSU_04_00740	complement(82407..83780)		hypothetical protein [nite]			ROP_62200 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		D0LD06_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_3295 PE=4 SV=1;unnamed protein product [Mycobacterium tuberculosis KZN 4207];drug-susceptible;Protein of unknown function (DUF690); cl04939
GOHSU04	CDS	GOHSU_04_00750	83929..85254		hypothetical protein [nite]			RER_18870 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5U0U0_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_063_00120 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein;Protein of unknown function (DUF2984); cl12752
GOHSU04	CDS	GOHSU_04_00760	complement(85274..86515)		mannose-6-phosphate isomerase [nite]	 manA [nite]		ROP_63800 :mannose-6-phosphate isomerase : manA : 5.3.1.8 : Rhodococcus opacus B4	mannose-6-phosphate isomerase	manA2	H0R700_9ACTO Mannose-6-phosphate isomerase OS=Gordo nia effusa NBRC 100432 GN=manA PE=3 SV=1;mannose-6-phosphate isomerase [Rhodococcus jostii R HA1];mannose-6-phosphate isomerase; Provisional;Phosphomannose isomerase type I; cl15429
GOHSU04	CDS	GOHSU_04_00770	complement(86518..87648)		hypothetical protein [nite]			ROP_63790 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YER6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00935 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GJ17040 gene product from transcript GJ17040- RA
GOHSU04	CDS	GOHSU_04_00780	complement(87676..88074)		hypothetical protein [nite]			ROP_63770 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YER5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00930 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3499); pfam12005
GOHSU04	CDS	GOHSU_04_00790	88230..88724		hypothetical protein [nite]			ROP_63760 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YER4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00925 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Possibl zinc metallo-peptidase; cl01577
GOHSU04	CDS	GOHSU_04_00800	complement(88841..89182)		WhiB family regulatory protein [nite]			RER_21540 :WhiB family regulatory protein : : : Rhodococcus erythropolis PR4	transcription factor WhiB		H5U9U8_9ACTO Putative WhiB family regulatory protei n OS=Gordonia terrae NBRC 100016 GN=GOTRE_018_00330 PE=4 S V=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: transcription factor WhiB;Transcription factor WhiB; pfam02467
GOHSU04	CDS	GOHSU_04_00810	89450..90469		probable LPPG--FO 2-phopspho-L-lactate transferase [nite]	 fbiA [nite]		RER_21530 :probable LPPG--FO 2-phopspho-L-lactate transferase : fbiA : : Rhodococcus erythropolis PR4	LPPG:FO 2-phospho-L-lactate transferase		H0R7M3_9ACTO LPPG--FO 2-phopspho-L-lactate transfer ase OS=Gordonia polyisoprenivorans NBRC 16320 GN=fbiA PE=3 SV=1;LPPG:FO 2-phospho-L-lactate transferase [Streptomyc es clavuligerus ATCC 27064];Family of CofD-like proteins and proteins related to YvcK; cl00425;phosphate binding site [ion binding];dimer interface [polypeptide binding]
GOHSU04	CDS	GOHSU_04_00820	90466..91809		F420-0:gamma-glutamyl ligase [hamap]	cofE [hamap]	MF_01259:F420-0:gamma-glutamyl ligase:cofE 	RER_21520 :probable coenzyme F420--glutamate ligase : fbiB : : Rhodococcus erythropolis PR4	F420-0:gamma-glutamyl ligase CofE	cofE	F9VXJ9_9ACTO F420-0--gamma-glutamyl ligase OS=Gordo nia alkanivorans NBRC 16433 GN=cofE PE=3 SV=1;cofE gene product [Gordonia polyisoprenivorans VH2] ;F420-0--gamma-glutamyl ligase; Provisional;F420-0:Gamma-glutamyl ligase; cl00644;NAD(P)H:flavin oxidoreductase-like family 1.  A subfamily of the nitroreductase family containing uncharacterized proteins that are similar to nitroreductase. Nitroreductase catalyzes the reduction of nitroaromatic compounds such as nitrotoluenes; cd02150;putative FMN binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_00830	91806..92390		conserved hypothetical protein [nite]			RER_21510 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	NTP pyrophosphohydrolase		F1YER0_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_00905 PE=4 SV=1;NTP pyrophosphohydrolase [Saccharomonospora cyanea NA- 134];PFAM: NUDIX domain;Members of the Nudix hydrolase superfamily catalyze the hydrolysis of NUcleoside DIphosphates linked to other moieties, X. Enzymes belonging to this superfamily require a divalent cation, such as Mg2+ or Mn2+, for their activity. They also contain a...; cd03674;nudix motif
GOHSU04	CDS	GOHSU_04_00840	complement(92415..93785)		not assigned						
GOHSU04	CDS	GOHSU_04_00850	complement(93839..95053)		glucose-1-phosphate adenylyltransferase [hamap]	glgC [hamap]	MF_00624:glucose-1-phosphate adenylyltransferase:glgC 	ROP_60350 :glucose-1-phosphate adenylyltransferase : glgC : 2.7.7.27 : Rhodococcus opacus B4	glucose-1-phosphate adenylyltransferase	glgC	F1YEQ9_9ACTO Glucose-1-phosphate adenylyltransferas e OS=Gordonia neofelifaecis NRRL B-59395 GN=glgC PE=3 SV=1 ;glucose-1-phosphate adenylyltransferase [Corynebact erium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of ADP-glucose and diphosphate from ATP and alpha-D-glucose 1-phosphate;glucose-1-phosphate adenylyltransferase; Provisional; PRK00844;ADP-glucose pyrophosphorylase is involved in the biosynthesis of glycogen or starch; cd02508;ligand binding site;oligomer interface;Glucose-1-phosphate adenylyltransferase, C-terminal Left-handed parallel beta helix (LbH) domain: Glucose-1-phosphate adenylyltransferase is also known as ADP-glucose synthase or ADP-glucose pyrophosphorylase. It catalyzes the first committed and...; cd04651;sulfate 1 binding site
GOHSU04	CDS	GOHSU_04_00860	complement(95146..95970)		putative methyltransferase [nite]			RER_41620 :putative methyltransferase : : 2.1.1.- : Rhodococcus erythropolis PR4	SAM dependent methyltransferase		H0R710_9ACTO Putative methyltransferase OS=Gordonia effusa NBRC 100432 GN=GOEFS_132_00920 PE=4 SV=1;SAM dependent methyltransferase [Rhodococcus equi 1 03S];23S rRNA methyltransferase A; Provisional; PRK11088;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU04	CDS	GOHSU_04_00870	complement(96003..96170)		hypothetical protein [nite]			ROP_60320 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YEQ7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00890 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3117); pfam11314
GOHSU04	CDS	GOHSU_04_00880	complement(96239..96565)		hypothetical protein [nite]			ROP_60300 :hypothetical protein : : : Rhodococcus opacus B4	DivIVA domain-containing protein		F1YEQ6_9ACTO DivIVA domain-containing protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00885 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;TIGRFAM: DivIVA domain; KEGG: rer:RER_41650 hypothetical protein;DivIVA domain; TIGR03544
GOHSU04	CDS	GOHSU_04_00890	complement(96570..97526)		putative glucosyl-3-phosphoglycerate synthase [nite]	 gpgS [nite]		RER_41660 :putative glucosyl-3-phosphoglycerate synthase : gpgS : 2.4.-.- : Rhodococcus erythropolis PR4	glycosyl transferase family protein		H5U3J1_9ACTO Glucosyl-3-phosphoglycerate synthase O S=Gordonia sputi NBRC 100414 GN=gpgS PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];putative glucosyl-3-phosphoglycerate synthase; Provisional;DPM_DPG-synthase_like is a member of the Glycosyltransferase 2 superfamily; cd04179;Ligand binding site;Putative Catalytic site;DXD motif
GOHSU04	CDS	GOHSU_04_00900	complement(97523..98389)		putative dihydropteroate synthase [nite]			RER_41670 :putative dihydropteroate synthase : : 2.5.1.15 : Rhodococcus erythropolis PR4	dihydropteroate synthase		H6MWB7_GORPV Dihydropteroate synthase FolP OS=Gordo nia polyisoprenivorans (strain DSM 44266 / VH2) GN=folP1 P E=4 SV=1;dihydropteroate synthase [Streptomyces lividans TK2 4];DHPS subgroup of Pterin binding enzymes. DHPS (dihydropteroate synthase), a functional homodimer, catalyzes the condensation of p-aminobenzoic acid (pABA) in the de novo biosynthesis of folate, which is an essential cofactor in both nucleic acid and...; cd00739;dihydropteroate synthase; TIGR01496;substrate binding pocket [chemical binding];dimer interface [polypeptide binding];inhibitor binding site
GOHSU04	CDS	GOHSU_04_00910	complement(98392..100191)		putative fatty-acid--CoA ligase [nite]			ROP_60270 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	AMP-dependent synthetase and ligase		F1YEQ3_9ACTO Long-chain-acyl-CoA synthetase OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_00870 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: AMP-dependent synthetase and ligase; KEGG: abo:ABO_1663 acyl-CoA synthetase;long-chain-acyl-CoA synthetase; Validated;AMP-binding enzyme; cl15778
GOHSU04	CDS	GOHSU_04_00920	complement(100230..100808)		conserved hypothetical protein [nite]			RER_41690 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YEQ2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00865 PE=4 SV=1;unnamed protein product [Pseudovibrio sp. FO- BEG1] ;isolated with Beggiatoa sp.;Possible lysine decarboxylase; cl00695
GOHSU04	CDS	GOHSU_04_00930	complement(100798..101592)		conserved hypothetical protein [nite]			RER_41700 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0LD25_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_3314 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: conserved hypothetical protein; KEGG: sfu:Sfum_1863 conserved hypothetical protein 730;Possible lysine decarboxylase; cl00695
GOHSU04	CDS	GOHSU_04_00940	complement(101598..102731)		succinyl-diaminopimelate desuccinylase [nite]	 dapE [nite]		RER_41710 :succinyl-diaminopimelate desuccinylase : dapE : 3.5.1.18 : Rhodococcus erythropolis PR4	succinyl-diaminopimelate desuccinylase	dapE	F1YEP7_9ACTO Succinyl-diaminopimelate desuccinylase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_00840 PE=4 SV=1;succinyl-diaminopimelate desuccinylase [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;Acetylornithine deacetylase/Succinyl-diaminopimelate desuccinylase and related deacylases [Amino acid transport and metabolism]; COG0624;catalyzes the formation of succinate and diaminoheptanedioate from succinyldiaminoheptanedioate;M20 Peptidase actinobacterial DapE encoded N-succinyl-L,L-diaminopimelic acid desuccinylase; cd05647;metal binding site [ion binding];putative dimer interface [polypeptide binding]
GOHSU04	CDS	GOHSU_04_00950	102777..103730		tetrahydrodipicolinate N-succinyltransferase [nite]	 dapD [nite]		RER_41720 :tetrahydrodipicolinate N-succinyltransferase : dapD : 2.3.1.117 : Rhodococcus erythropolis PR4	tetrahydrodipicolinate N-succinyltransferase	dapD	F1YEP6_9ACTO 2,3,4,5-tetrahydropyridine-2,6-dicarbo xylateN-su ccinyltransferase OS=Gordonia neofelifaecis NRR L B-59395 GN=SCNU_00835 PE=4 SV=1;tetrahydrodipicolinate N-succinyltransferase [Amyco licicoccus subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase; TIGR03535;Putative 2,3,4,5-tetrahydropyridine-2,6-dicarboxylate (THDP) N-succinyltransferase (THP succinyltransferase), C-terminal left-handed parallel alpha-helix (LbH) domain: This group is composed of mostly uncharacterized proteins containing an N-terminal...; cd04649;putative trimer interface [polypeptide binding];putative CoA binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_00960	complement(104006..105115)		succinyldiaminopimelate aminotransferase [nite]	 dapC [nite]		RER_41800 :succinyldiaminopimelate aminotransferase : dapC : 2.6.1.17 : Rhodococcus erythropolis PR4	N-succinyldiaminopimelate aminotransferase	dapC	H6MWC7_GORPV LL-diaminopimelate aminotransferase Da pL OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=dapL PE=4 SV=1;N-succinyldiaminopimelate aminotransferase [Coryneb acterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of N-succinyl-LL-2,6-diaminopimelate from N-succinyl-L-2-amino-6-oxopimelate in lysine biosynthesis;succinyldiaminopimelate transaminase; TIGR03539;Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU04	CDS	GOHSU_04_00970	complement(105116..105442)		7Fe ferredoxin [nite]			RER_41810 :7Fe ferredoxin : : : Rhodococcus erythropolis PR4	7Fe ferredoxin		F1YEP3_9ACTO 4Fe-4S ferredoxin iron-sulfur-binding domain-containing protein OS=Gordonia neofelifaecis NRRL B -59395 GN=SCNU_00820 PE=4 SV=1;7Fe ferredoxin [Rhodococcus opacus B4];[FeFe] hydrogenase, group B1/B3; TIGR04105;4Fe-4S binding domain; cl02805
GOHSU04	CDS	GOHSU_04_00980	105704..107371		putative coenzyme A-disulfide reductase [nite]	 cdr [nite]		TEH_20260 :putative coenzyme A-disulfide reductase : cdr : 1.8.1.14 : Tetragenococcus halophilus NBRC 12172	FAD-dependent pyridine nucleotide-disulfide oxidoreductase		F1YEP2_9ACTO FAD-dependent pyridine nucleotide-disu lfide oxidoreductase OS=Gordonia neofelifaecis NRRL B-5939 5 GN=SCNU_00815 PE=4 SV=1;FAD-dependent pyridine nucleotide-disulfide oxidore ductase [Jonesia denitrificans DSM 20603];type strain of Jonesia denitrificans;Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.820) with cleavage site probability 0.694 at residue 30;PFAM: FAD-dependent pyridine nucleotide-disulphide oxidoreductase; pyridine nucleotide-disulphide oxidoreductase dimerisation region; SMART: Rhodanese domain protein; KEGG: dal:Dalk_0198 FAD-dependent pyridine nucleotide-disulphide oxidoreductase;CoA-disulfide reductase; TIGR03385;Pyridine nucleotide-disulphide oxidoreductase; pfam00070;Rhodanese Homology Domain (RHOD); an alpha beta fold domain found duplicated in the rhodanese protein. The cysteine containing enzymatically active version of the domain is also found in the Cdc25 class of protein phosphatases and a variety of proteins...; cl00125;active site residue [active]
GOHSU04	CDS	GOHSU_04_00990	107450..107608		hypothetical protein [nite]			RER_pREL1-02260 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G6WWC7_CORGT Putative uncharacterized protein OS=Co rynebacterium glutamicum ATCC 14067 GN=KIQ_07347 PE=4 SV=1 ;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;KEGG: mva:Mvan_5833 hypothetical protein
GOHSU04	CDS	GOHSU_04_01000	complement(107709..110234)		FO synthase subunit 1/FO synthase subunit 2 [hamap]	cofG/cofH [hamap]	MF_01611:FO synthase subunit 1:cofG MF_01612:FO synthase subunit 2:cofH 	ROP_60060 :7,8-didemethyl-8-hydroxy-5-deazariboflavine synthase : fbiC : : Rhodococcus opacus B4	FO synthase	fbiC	F1YEN9_9ACTO FO synthase OS=Gordonia neofelifaecis NRRL B-59395 GN=fbiC PE=3 SV=1;fbiC gene product [Nocardioides sp. JS614];FO synthase; Reviewed; PRK09234;7,8-didemethyl-8-hydroxy-5-deazariboflavin synthase; catalyzes radical-mediated transfer of hydroxybenzyl group from 4-hydroxyphenylpyruvate (HPP) to 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione to form 7,8-didemethyl-8-hydroxy-5-deazariboflavin (FO); functions in F420 biosynthesis;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site
GOHSU04	CDS	GOHSU_04_01010	complement(110323..110664)		hypothetical protein [migap]				hypothetical protein		F1YEN8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00795 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];bifunctional preprotein translocase subunit SecD/SecF; Reviewed
GOHSU04	CDS	GOHSU_04_01020	complement(110661..111533)		N-acetyl-1-D-myo-Inosityl-2-amino-2-deoxy-alpha-D-glucopyranoside deacetylase MshB [nite]	 mshB [nite]		ROP_59960 :N-acetyl-1-D-myo-Inosityl-2-amino-2-deoxy-alpha-D-glucopyranoside deacetylase MshB : mshB : : Rhodococcus opacus B4	LmbE-like protein		G7GV96_9ACTO 1D-myo-inositol 2-acetamido-2-deoxy-al pha-D-glucopyranoside deacetylase OS=Gordonia amarae NBRC 15530 GN=mshB PE=3 SV=1;LmbE-like protein [Frankia sp. CcI3];GlcNAc-PI de-N-acetylase; cl00929
GOHSU04	CDS	GOHSU_04_01030	complement(111518..113419)		conserved hypothetical protein [nite]			RER_41910 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	extracellular solute-binding protein family 5 protein		F1YEN6_9ACTO Extracellular solute-binding protein f amily 5 protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_00785 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.998) with cleavage site probability 0.435 at residue 46;PFAM: extracellular solute-binding protein family 5; KEGG: mes:Meso_1835 extracellular solute-binding protein;The substrate-binding domain of mycobacterial lipoprotein Lpqw contains type 2 periplasmic binding fold; cd08501;Bacterial extracellular solute-binding proteins, family 5 Middle; pfam00496
GOHSU04	CDS	GOHSU_04_01040	complement(113437..115344)		probable GTP-binding protein TypA [nite]	 typA [nite]		RER_41920 :probable GTP-binding protein TypA : typA : : Rhodococcus erythropolis PR4	GTP-binding protein TypA		F1YEN5_9ACTO GTP-binding protein TypA OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_00780 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Tyrosine phosphorylated protein A (TypA)/BipA family belongs to ribosome-binding GTPases; cd01891;GTP-binding protein TypA/BipA; TIGR01394;G1 box;putative GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;BipA_TypA_II: domain II of BipA (also called TypA) having homology to domain II of the elongation factors (EFs) EF-G and EF-Tu.  BipA is a highly conserved protein with global regulatory properties in Escherichia coli. BipA is phosphorylated on a...; cd03691;BipA_TypA_C: a C-terminal portion of BipA or TypA having homology to the C terminal domains of the elongation factors EF-G and EF-2. A member of the ribosome binding GTPase superfamily, BipA is widely distributed in bacteria and plants.  BipA is a highly...; cd03710
GOHSU04	CDS	GOHSU_04_01050	115504..115704		hypothetical protein [nite]			ROP_59930 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7H434_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_061_00480 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU04	CDS	GOHSU_04_01060	complement(116001..116414)		NTP pyrophosphohydrolase [nite]			RER_41940 :NTP pyrophosphohydrolase : : 3.6.1.- : Rhodococcus erythropolis PR4	putative NUDIX hydrolase		G7H433_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia araii NBRC 100433 GN=GOARA_061_00470 PE =3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];The MutT pyrophosphohydrolase is a prototypical Nudix hydrolase that catalyzes the hydrolysis of nucleoside and deoxynucleoside triphosphates (NTPs and dNTPs) by substitution at a beta-phosphorus to yield a nucleotide monophosphate (NMP) and inorganic...; cd03425;8-oxo-dGMP binding site [chemical binding];nudix motif;metal binding site [ion binding]
GOHSU04	CDS	GOHSU_04_01070	complement(116411..117445)		ABC transporter ATP binding protein [nite]			BBR47_45290 :ABC transporter ATP binding protein : : : Brevibacillus brevis NBRC 100599	spermidine/putrescine ABC transporter ATPase		H5TMA2_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia otitidis NBRC 100426 GN=GOOTI_114_00390 PE=3 SV=1;spermidine/putrescine ABC transporter ATPase [Sinor hizobium medicae WSM419];ABC-type spermidine/putrescine transport systems, ATPase components [Amino acid transport and metabolism]; COG3842;TIGRFAM: spermidine/putrescine ABC transporter ATPase subunit; PFAM: ABC transporter related; TOBE domain protein; Transport-associated OB domain protein; SMART: AAA ATPase; KEGG: sme:SMc01965 putative spermidine/putrescine transport ATP-binding ABC transporter protein;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region;TOBE domain; cl01440
GOHSU04	CDS	GOHSU_04_01080	complement(117447..119084)		putative ABC transporter permease protein [nite]			ANT_06710 :putative ABC transporter permease protein : : : Anaerolinea thermophila UNI-1	ABC transporter integral membrane protein		F1YEN1_9ACTO ABC-type Fe3+ transport system, permea se component OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_00760 PE=3 SV=1;ABC transporter integral membrane protein [Streptom yces clavuligerus ATCC 27064];Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU04	CDS	GOHSU_04_01090	complement(119081..120106)		putative ABC transporter substrate binding protein [nite]			ANT_06700 :putative ABC transporter substrate binding protein : : : Anaerolinea thermophila UNI-1	ABC transporter substrate-binding protein		F1YEN0_9ACTO ABC transporter periplasmic-binding pr otein, thiB subfamily OS=Gordonia neofelifaecis NRRL B-593 95 GN=SCNU_00755 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];ABC-type thiamine transport system, periplasmic component [Coenzyme metabolism]; COG4143;The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cl11398
GOHSU04	CDS	GOHSU_04_01100	120321..121193		conserved hypothetical protein [nite]			RER_41960 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YEM9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00750 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: bbr:BB1832 hypothetical protein
GOHSU04	CDS	GOHSU_04_01110	121444..122502		hypothetical protein [migap]				hypothetical protein		H6MWE7_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c31370 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU04	CDS	GOHSU_04_01120	122554..123711		hypothetical protein [nite]			ROP_22260 :hypothetical protein : : : Rhodococcus opacus B4	putative secretory lipase		F1YEM7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00740 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Dipeptidyl aminopeptidases/acylaminoacyl-peptidases [Amino acid transport and metabolism]; COG1506;Secretory lipase; pfam03583
GOHSU04	CDS	GOHSU_04_01130	complement(123726..124226)		hypothetical protein [migap]				hypothetical protein		H0R1J1_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_070_00260 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU04	CDS	GOHSU_04_01140	124269..124835		hypothetical protein [nite]			ROP_41240 :hypothetical protein : : : Rhodococcus opacus B4	HhH-GPD family protein		F9VRQ0_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_026_00950 PE=4 SV= 1;unnamed protein product [Frankia sp. EAN1pec];PFAM: HhH-GPD family protein; KEGG: fal:FRAAL0558 conserved hypothetical protein; putative DNA-glycosylase domain;endonuclease III; includes endonuclease III (DNA-(apurinic or apyrimidinic site) lyase), alkylbase DNA glycosidases (Alka-family) and other DNA glycosidases; cl14786
GOHSU04	CDS	GOHSU_04_01150	124880..125956		hypothetical protein [nite]			ROP_48310 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		A7S724_NEMVE Predicted protein (Fragment) OS=Nemato stella vectensis GN=v1g107225 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];Detected in the membrane fraction by proteomics.; membrane protein
GOHSU04	CDS	GOHSU_04_01160	complement(125962..128649)		aldehyde dehydrogenase [migap]				aldehyde dehydrogenase		G7H2Q0_9ACTO Aldehyde dehydrogenase OS=Gordonia ara ii NBRC 100433 GN=GOARA_052_00240 PE=4 SV=1;aldehyde dehydrogenase [Kocuria rhizophila DC2201];Aldehyde dehydrogenase family; pfam00171;Mycobacterium tuberculosis succinate-semialdehyde dehydrogenase 1-like; cd07100;NAD(P) binding site [chemical binding];catalytic residues [active]
GOHSU04	CDS	GOHSU_04_01170	128722..131097		acetyl-CoA acetyltransferase [migap]				acetyl-CoA acetyltransferase		F1YEM0_9ACTO Enoyl-CoA hydratase/isomerase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_00705 PE=3 SV=1;acetyl-CoA acetyltransferase [Mycobacterium sp. KMS ];Catalyzes the synthesis of acetoacetyl coenzyme A from two molecules of acetyl coenzyme A. It can also act as a thiolase, catalyzing the reverse reaction and generating two-carbon units from the four-carbon product of fatty acid oxidation;acetyl-CoA acetyltransferase; Validated;Thiolase domain associated with sterol carrier protein (SCP)-x isoform and related proteins; SCP-2  has multiple roles in intracellular lipid circulation and metabolism. The N-terminal presequence in the SCP-x isoform represents a peroxisomal...; cd00829
GOHSU04	CDS	GOHSU_04_01180	131211..131420		hypothetical protein [nite]			ROP_40870 :hypothetical protein : : : Rhodococcus opacus B4	CsbD family protein		E5XNI5_9ACTO CsbD protein OS=Segniliparus rugosus A TCC BAA-974 GN=HMPREF9336_01014 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: CsbD family protein; KEGG: mab:MAB_1241c hypothetical protein;CsbD-like; cl15799
GOHSU04	CDS	GOHSU_04_01190	complement(131417..131731)		putative quaternary ammonium compound-resistance protein [nite]			RER_09840 :putative quaternary ammonium compound-resistance protein : : : Rhodococcus erythropolis PR4	SugE1	sugE1	G0HBH0_CORVD Putative membrane protein OS=Corynebac terium variabile (strain DSM 44702 / JCM 12073 / NCIMB 301 31) GN=CVAR_1731 PE=3 SV=1;SugE1 [Escherichia coli];identified by Glimmer 2; sequence similarity to pSN254_0085; blastp_file = YP_209396.1; quaternary ammonium compound-resistance protein;Small Multidrug Resistance protein; cl00910
GOHSU04	CDS	GOHSU_04_01200	131894..132277		hypothetical protein [nite]			ROP_02740 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5TM69_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_114_00060 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU04	CDS	GOHSU_04_01210	complement(132616..132951)		hypothetical protein [migap]				hypothetical protein		G2QPJ9_THIHA Putative uncharacterized protein OS=Th ielavia heterothallica (strain ATCC 42464 / BCRC 31852 / D SM 1799) GN=MYCTH_2311727 PE=4 SV=1;hypothetical protein Kfla_0682 [Kribbella flavida D SM 17836];PFAM: protein of unknown function DUF305; KEGG: met:M446_1820 hypothetical protein;Domain of unknown function (DUF305); cl15795
GOHSU04	CDS	GOHSU_04_01220	complement(133069..133554)		putative hydroperoxide peroxidase [nite]			ROP_07430 :putative hydroperoxide peroxidase : : 1.11.1.- : Rhodococcus opacus B4	alkyl hydroperoxide reductase/ thiol specific antioxidant/ Mal allergen		F1YEK7_9ACTO Alkyl hydroperoxide reductase/ Thiol s pecific antioxidant/ Mal allergen OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_00645 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: alkyl hydroperoxide reductase/ thiol specific antioxidant/ Mal allergen; redoxin; KEGG: mav:MAV_3178 antioxidant, AhpC/TSA family protein;Peroxiredoxin (PRX) family, Bacterioferritin comigratory protein (BCP) subfamily; composed of thioredoxin-dependent thiol peroxidases, widely expressed in pathogenic bacteria, that protect cells against toxicity from reactive oxygen species by reducing...; cd03017;catalytic triad [active]
GOHSU04	CDS	GOHSU_04_01230	complement(133567..134358)		putative enoyl-CoA hydratase [nite]			RER_52130 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase		F1YEK9_9ACTO Enoyl-CoA hydratase/isomerase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_00655 PE=4 SV=1;enoyl-CoA hydratase [Rhodococcus jostii RHA1];enoyl-CoA hydratase; Validated;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU04	CDS	GOHSU_04_01240	134545..134904		conserved hypothetical protein [nite]			SLG_05660 :conserved hypothetical protein : : : Sphingobium sp. SYK-6	hypothetical protein		Q8P546_XANCP Putative uncharacterized protein OS=Xa nthomonas campestris pv. campestris (strain ATCC 33913 / N CPPB 528 / LMG 568) GN=XCC3501 PE=4 SV=1;unnamed protein product [Stenotrophomonas maltophil ia JV3];PFAM: Protein of unknown function DUF454; KEGG: sml:Smlt4081  transmembrane protein;Protein of unknown function (DUF454); cl01063
GOHSU04	CDS	GOHSU_04_01250	134927..135367		type 11 methyltransferase [migap]				type 11 methyltransferase		G7GZZ7_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_030_00120 PE=4 SV=1;type 11 methyltransferase [Micromonospora sp. L5];PFAM: Methyltransferase type 11; KEGG: cmi:CMM_1481 SAM-dependant methyltransferase;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440
GOHSU04	CDS	GOHSU_04_01260	complement(135587..137821)		isocitrate dehydrogenase [nite]	 icd [nite]		ROP_47620 :isocitrate dehydrogenase : icd : 1.1.1.42 : Rhodococcus opacus B4	isocitrate dehydrogenase, NADP-dependent		F1YHZ9_9ACTO Isocitrate dehydrogenase, NADP-depende nt OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07568 PE =3 SV=1;isocitrate dehydrogenase, NADP-dependent [Mycobacte rium avium 104];identified by match to protein family HMM PF03971; match to protein family HMM TIGR00178;Monomeric isocitrate dehydrogenase; cl15383
GOHSU04	CDS	GOHSU_04_01270	138101..139228		hypothetical protein [nite]			MLP_13760 :hypothetical protein : : : Microlunatus phosphovorus NM-1	putative phosphatase		H0RJA7_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_077_00840 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];PAP2_like proteins, a super-family of histidine phosphatases and vanadium haloperoxidases, includes type 2 phosphatidic acid phosphatase or lipid phosphate phosphatase (LPP), Glucose-6-phosphatase, Phosphatidylglycerophosphatase B and bacterial acid...; cl00474
GOHSU04	CDS	GOHSU_04_01280	complement(139338..140285)		hypothetical protein [nite]			RER_02740 :hypothetical protein : : : Rhodococcus erythropolis PR4	glycoside hydrolase		G7GT96_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_59_01220 PE=4 SV=1;glycoside hydrolase [Streptomyces violaceusniger Tu 4113];PFAM: Glycoside hydrolase, family 43; KEGG: scb:SCAB_19571 glycosyl hydrolase;Glycosyl hydrolase families: GH43, GH62, GH32, GH68; cl14647
GOHSU04	CDS	GOHSU_04_01290	140432..141025		hypothetical protein [nite]			ROP_64870 :hypothetical protein : : : Rhodococcus opacus B4	nucleosidase		F1YEG6_9ACTO Nucleosidase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_00435 PE=4 SV=1;nucleosidase [Corynebacterium glutamicum ATCC 13032 ];IS fingerprint type 4-5;Catalyzes hydrolysis of n-ribosyl-purine into a purine and d-ribose;nucleosidase; Provisional;Phosphorylase superfamily; cl00303
GOHSU04	CDS	GOHSU_04_01300	141081..142352		putative drug resistance efflux protein [nite]			RER_25380 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	major facilitator superfamily multidrug resistance protein		H6MWH4_GORPV Drug resistance transporter OS=Gordoni a polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c316 40 PE=4 SV=1;major facilitator superfamily multidrug resistance protein [Rhodococcus jostii RHA1];tc: 2.A.1.2;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;bicyclomycin/multidrug efflux system; Provisional;putative substrate translocation pore
GOHSU04	CDS	GOHSU_04_01310	complement(142300..143319)		putative pseudouridine synthase [nite]			RER_42220 :putative pseudouridine synthase : : : Rhodococcus erythropolis PR4	pseudouridine synthase		H0JM53_9NOCA Pseudouridine synthase OS=Rhodococcus pyridinivorans AK37 GN=AK37_03358 PE=3 SV=1;pseudouridine synthase [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;PseudoU_synth:  Pseudouridine synthases catalyze the isomerization of specific uridines in an RNA molecule to pseudouridines (5-ribosyluracil, psi). Pseudouridine synthases contains the RsuA/RluD, TruA, TruB and TruD families.  This group consists of...; cl00130
GOHSU04	CDS	GOHSU_04_01320	143764..145224		sulfide-quinone oxidoreductase [nite]	 sqr [nite]		ACMV_14370 :sulfide-quinone oxidoreductase : sqr : : Acidiphilium multivorum AIU301	sulfide-quinone reductase		D5PGQ8_9MYCO Sulfide-quinone reductase OS=Mycobacte rium parascrofulaceum ATCC BAA-614 GN=HMPREF0591_5352 PE=4 SV=1;sulfide-quinone reductase [Chlorobium luteolum DSM 273];Uncharacterized NAD(FAD)-dependent dehydrogenases [General function prediction only]; COG0446
GOHSU04	CDS	GOHSU_04_01330	145242..145415		hypothetical protein [migap]				hypothetical protein		B2HD40_MYCMM Putative uncharacterized protein OS=My cobacterium marinum (strain ATCC BAA-535 / M) GN=MMAR_2735 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];cytoplasmic protein
GOHSU04	CDS	GOHSU_04_01340	145451..145894		hypothetical protein [nite]			KSE_43940 :hypothetical protein : : : Kitasatospora setae NBRC 14216	protein chain release factor B		C7MER3_BRAFD Protein chain release factor B OS=Brac hybacterium faecium (strain ATCC 43885 / DSM 4810 / NCIB 9 860) GN=Bfae_22660 PE=4 SV=1;protein chain release factor B [Brachybacterium fae cium DSM 4810];type strain of Brachybacterium faecium DSM 4810;PFAM: RF-1 domain;RF-1 domain; cl02875
GOHSU04	CDS	GOHSU_04_01350	complement(145920..146291)		hypothetical protein [nite]			BBR47_13230 :hypothetical protein : : : Brevibacillus brevis NBRC 100599	Carboxymuconolactone decarboxylase		Q8FQN1_COREF Putative uncharacterized protein OS=Co rynebacterium efficiens (strain DSM 44549 / YS-314 / AJ 12 310 / JCM 11189 / NBRC 100395) GN=CE1088 PE=4 SV=1;unnamed protein product [Eubacterium limosum KIST61 2];Carboxymuconolactone decarboxylase family; cl00460
GOHSU04	CDS	GOHSU_04_01360	complement(146297..146737)		hypothetical protein [migap]				hypothetical protein		Q8FQN2_COREF Putative uncharacterized protein OS=Co rynebacterium efficiens (strain DSM 44549 / YS-314 / AJ 12 310 / JCM 11189 / NBRC 100395) GN=CE1087 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];similar to AP001518-86|BAB06974.1| percent identity: 40 in 146 aa;Protein of unknown function (DUF2872); cl11924
GOHSU04	CDS	GOHSU_04_01370	complement(146865..148001)		L-asparaginase/archaeal Glu-tRNAGln amidotransferase subunit D [migap]				L-asparaginase/archaeal Glu-tRNAGln amidotransferase subunit D		B9Z8W1_9NEIS Asparaginase OS=Pseudogulbenkiania fer rooxidans 2002 GN=FuraDRAFT_3798 PE=4 SV=1;L-asparaginase/archaeal Glu-tRNAGln amidotransferas e subunit D [Moorea producta 3L];Type I (cytosolic) bacterial L-asparaginase; cd08963;homodimer interface [polypeptide binding]
GOHSU04	CDS	GOHSU_04_01380	148419..148682		putative transposase for insertion sequence element [nite]			ROP_71920 :putative transposase for insertion sequence element : : : Rhodococcus opacus B4	transposase, Mutator family protein		H0JUE4_9NOCA Transposase for insertion sequence ele ment OS=Rhodococcus pyridinivorans AK37 GN=AK37_16590 PE=4 SV=1;transposase, Mutator family protein [Mycobacterium avium 104];identified by match to protein family HMM PF00872;Transposase, Mutator family; pfam00872
GOHSU04	CDS	GOHSU_04_01390	complement(148898..149170)		putative methyltransferase [nite]			ROP_04930 :putative methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	methyltransferase		Q0SFX9_RHOSR Possible methyltransferase OS=Rhodococ cus sp. (strain RHA1) GN=RHA1_ro01744 PE=4 SV=1;methyltransferase [Streptomyces clavuligerus ATCC 2 7064]
GOHSU04	CDS	GOHSU_04_01400	complement(149322..149624)		putative Xre family DNA-binding protein [nite]			KRH_07230 :putative Xre family DNA-binding protein : : : Kocuria rhizophila DC2201	XRE family transcriptional regulator		C7R1W3_JONDD Plasmid maintenance system antidote pr otein, XRE family OS=Jonesia denitrificans (strain ATCC 14 870 / DSM 20603 / CIP 55134) GN=Jden_0768 PE=4 SV=1;XRE family transcriptional regulator [Jonesia denit rificans DSM 20603];type strain of Jonesia denitrificans;KEGG: hha:Hhal_1672 plasmid maintenance system antidote protein, XRE family; TIGRFAM: addiction module antidote protein, HigA family; PFAM: helix-turn-helix domain protein; SMART: helix-turn-helix domain protein;Helix-turn-helix XRE-family like proteins. Prokaryotic DNA binding proteins belonging to the xenobiotic response element family of transcriptional regulators; cd00093;sequence-specific DNA binding site [nucleotide binding];salt bridge
GOHSU04	CDS	GOHSU_04_01410	complement(149611..149892)		putative toxin [nite]			ROP_30860 :putative toxin : : : Rhodococcus opacus B4	plasmid maintenance system killer protein		C7R1W2_JONDD Plasmid maintenance system killer OS=J onesia denitrificans (strain ATCC 14870 / DSM 20603 / CIP 55134) GN=Jden_0767 PE=4 SV=1;plasmid maintenance system killer protein [Oligotro pha carboxidovorans OM5];type strain of Oligotropha carboxidovorans;Plasmid maintenance system killer protein; cl01422
GOHSU04	CDS	GOHSU_04_01420	150080..151177		beta-lactamase domain-containing protein [migap]				beta-lactamase domain-containing protein		G2NZP1_STRVO Putative uncharacterized protein OS=St reptomyces violaceusniger Tu 4113 GN=Strvi_4006 PE=4 SV=1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];PFAM: beta-lactamase domain protein; KEGG: dvl:Dvul_0948 beta-lactamase domain-containing protein;Metallo-beta-lactamase superfamily; cl00446
GOHSU04	CDS	GOHSU_04_01430	151225..151788		hypothetical protein [nite]			ACMV_30450 :hypothetical protein : : : Acidiphilium multivorum AIU301	putative ester cyclase		G2NZP0_STRVO Putative uncharacterized protein OS=St reptomyces violaceusniger Tu 4113 GN=Strvi_4005 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: SnoaL-like polyketide cyclase; TIGRFAM: conserved hypothetical protein, steroid delta-isomerase-related;Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109
GOHSU04	CDS	GOHSU_04_01440	151812..153431		zinc-containing alcohol dehydrogenase superfamily protein [migap]				zinc-containing alcohol dehydrogenase superfamily protein		D5P618_9MYCO Alcohol dehydrogenase OS=Mycobacterium parascrofulaceum ATCC BAA-614 GN=qor PE=4 SV=1;zinc-containing alcohol dehydrogenase superfamily p rotein [Ralstonia eutropha JMP134];putative NAD(P)H quinone oxidoreductase, PIG3 family; TIGR02824;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_01450	153428..154234		hypothetical protein [nite]			ROP_33910 :hypothetical protein : : : Rhodococcus opacus B4	beta-lactamase class B		F9W225_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_118_00030 PE=4 SV= 1;beta-lactamase class B [Amycolatopsis mediterranei U32];Metallo-beta-lactamase superfamily; cl00446
GOHSU04	CDS	GOHSU_04_01460	154231..154884		conserved hypothetical protein [nite]	 yheG [nite]		BBR47_59260 :conserved hypothetical protein : yheG : : Brevibacillus brevis NBRC 100599	hypothetical protein		F9W226_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_118_00040 PE=4 SV= 1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: scl:sce9254  secreted protein;biliverdin IX beta reductase (BVR-B, aka flavin reductase)-like proteins; atypical (a) SDRs; cd05244;NADH(P)-binding; pfam13460;NAD binding site [chemical binding];substrate binding site [chemical binding];putative active site [active]
GOHSU04	CDS	GOHSU_04_01470	complement(154999..155673)		putative TetR family transcriptional regulator [nite]			ROP_29890 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		G2NZP2_STRVO Regulatory protein TetR OS=Streptomyce s violaceusniger Tu 4113 GN=Strvi_4007 PE=4 SV=1;TetR family transcriptional regulator [Streptomyces violaceusniger Tu 4113];PFAM: Transcriptional regulator, TetR-like, DNA-binding, bacterial/archaeal; Tetracycline transcriptional regulator, TetR, C-terminal; KEGG: ami:Amir_4056 TetR family transcriptional regulator;tetracycline repressor protein TetR; Provisional;Helix-turn-helix domains; cl00088;Tetracyclin repressor, C-terminal all-alpha domain; pfam02909
GOHSU04	CDS	GOHSU_04_01480	complement(155722..156300)		putative sodium/proton antiporter [nite]	 nhaA [nite]		KSE_36000 :putative sodium/proton antiporter : nhaA : : Kitasatospora setae NBRC 14216	Na+/H+ antiporter NhaA		D7AY20_NOCDD Na(+)/H(+) antiporter nhaA OS=Nocardio psis dassonvillei (strain ATCC 23218 / DSM 43111 / IMRU 50 9 / JCM 7437 / NCTC 10488) GN=nhaA PE=3 SV=1;Na+/H+ antiporter NhaA [Saccharomonospora cyanea NA - 134];PFAM: Na+/H+ antiporter 1; TIGRFAM: Na+/H+ antiporter NhaA;Na+/H+ antiporter 1; cl01133
GOHSU04	CDS	GOHSU_04_01490	complement(156364..156900)		Na(+)/H(+) antiporter [nite]	 nhaA [nite]		ROP_00220 :Na(+)/H(+) antiporter : nhaA : : Rhodococcus opacus B4	Na+/H+ antiporter NhaA		H5XPH0_9PSEU Na(+)/H(+) antiporter nhaA 3 (Precurso r) OS=Saccharomonospora cyanea NA-134 GN=nhaA3 PE=3 SV=1;Na+/H+ antiporter NhaA [Saccharomonospora cyanea NA - 134];Signal predicted by SignalP 3.0 HMM; IMG reference gene:2508560719_SP;PFAM: Na+/H+ antiporter 1; TIGRFAM: Na+/H+ antiporter NhaA;Na+/H+ antiporter 1; cl01133
GOHSU04	CDS	GOHSU_04_01500	complement(157410..158501)		putative GTP-binding protein [nite]			ROP_59410 :putative GTP-binding protein : : : Rhodococcus opacus B4	GTP-dependent nucleic acid-binding protein EngD		H5U4H0_9ACTO Putative GTP-binding protein OS=Gordon ia sputi NBRC 100414 GN=GOSPT_113_00150 PE=4 SV=1;GTP-dependent nucleic acid-binding protein EngD [Co rynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;GTP-binding protein YchF; Reviewed;translation-associated GTPase; the crystal structure of the Haemophilus influenzae YchF protein showed similarity to the yeast structure (PDB: 1NI3); fluorescence spectroscopy revealed nucleic acid binding; the yeast protein YBR025c interacts with the translation elongation factor eEF1;YchF GTPase; cd01900;G1 box;GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;Switch II region;G3 box;G4 box;G5 box;TGS_YchF_C: This subfamily represents TGS domain-containing YchF GTP-binding protein, a universally conserved GTPase whose function is unknown. The N-terminal domain of the YchF protein belongs to the Obg-like family of GTPases, and some members of the...; cd04867
GOHSU04	CDS	GOHSU_04_01510	complement(158527..159867)		hypothetical membrane protein [nite]			RER_42300 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YEH2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00465 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];pheromone autoinducer 2 transporter; Reviewed; cl00465
GOHSU04	CDS	GOHSU_04_01520	159924..161039		hypothetical protein [nite]			MLP_43620 :hypothetical protein : : : Microlunatus phosphovorus NM-1	DNA recombination protein RmuC	rmuC	F1YEG8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00445 PE=4 SV=1;rmuC gene product [Gordonia polyisoprenivorans VH2] ;RmuC family; pfam02646
GOHSU04	CDS	GOHSU_04_01530	161123..162241		hypothetical membrane protein [nite]			ROP_59330 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H5TRI6_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_195_00270 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: similar to coiled-coil domain containing 40;alpha-ketoglutarate decarboxylase; Reviewed; PRK12270
GOHSU04	CDS	GOHSU_04_01540	complement(162296..163264)		putative oxidoreductase [nite]			RER_43800 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short-chain dehydrogenase/reductase SDR		F1YEG5_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_00430 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: short-chain dehydrogenase/reductase SDR; KR domain protein; KEGG: dol:Dole_2823 short chain dehydrogenase;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU04	CDS	GOHSU_04_01550	complement(163302..164372)		hypothetical protein [nite]			ROP_56510 :hypothetical protein : : : Rhodococcus opacus B4	cell wall hydrolase/autolysin		F1YEG4_9ACTO Cell wall hydrolase/autolysin OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_00425 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rer:RER_03890 hydrolase; PFAM: cell wall hydrolase/autolysin; SMART: cell wall hydrolase/autolysin;N-acetylmuramoyl-l-alanine amidase I; Provisional;N-acetylmuramoyl-L-alanine amidase or MurNAc-LAA (also known as peptidoglycan aminohydrolase, NAMLA amidase, NAMLAA, Amidase 3, and peptidoglycan amidase; EC 3.5.1.28) is an autolysin that hydrolyzes the amide bond between N-acetylmuramoyl and L-amino...; cd02696;metal binding site [ion binding]
GOHSU04	CDS	GOHSU_04_01560	complement(164497..165540)		hypothetical protein [nite]			ROP_64770 :hypothetical protein : : : Rhodococcus opacus B4	GTPase		F1YEG3_9ACTO Cobalamin synthesis protein P47K OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00420 PE=4 SV=1;GTPase [Vibrio vulnificus CMCP6];COG0523;Putative GTPases (G3E family) [General function prediction only];P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Cobalamin synthesis protein cobW C-terminal domain; cl08458
GOHSU04	CDS	GOHSU_04_01570	complement(165519..166325)	case	UPF0176 protein yceA [hamap]	yceA [hamap]	MF_00469:UPF0176 protein yceA:yceA 	ROP_05950 :hypothetical protein : : : Rhodococcus opacus B4	rhodanese		H5U4F8_9ACTO UPF0176 protein GOSPT_113_00030 OS=Gor donia sputi NBRC 100414 GN=GOSPT_113_00030 PE=3 SV=1;rhodanese [Paenibacillus sp. JDR- 2];PFAM: rhodanese; SMART: Rhodanese domain protein; KEGG: bsu:BSU02330 hypothetical protein;putative rhodanese-related sulfurtransferase; Provisional;Member of the Rhodanese Homology Domain superfamily. This CD includes Escherichia coli YceA, Bacillus subtilis YbfQ, and similar uncharacterized proteins; cd01518;active site residue [active]
GOHSU04	CDS	GOHSU_04_01580	166327..167073		hypothetical protein [migap]				hypothetical protein		H5UIY2_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_128_00660 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.605) with cleavage site probability 0.349 at residue 41;KEGG: hypothetical protein
GOHSU04	CDS	GOHSU_04_01590	167189..167509		LSR2-like protein [nite]			KRH_19160 :LSR2-like protein : : : Kocuria rhizophila DC2201	lysyl tRNA synthetase-like protein		F1YEG1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00410 PE=4 SV=1;lysyl tRNA synthetase-like protein [Amycolatopsis m editerranei U32];Lsr2; pfam11774
GOHSU04	CDS	GOHSU_04_01600	168041..169399		magnesium transporter MgtE [nite]	 mgtE [nite]		TEH_21150 :magnesium transporter MgtE : mgtE : : Tetragenococcus halophilus NBRC 12172	magnesium transporter		Q3JA90_NITOC Divalent cation transporter OS=Nitroso coccus oceani (strain ATCC 19707 / NCIMB 11848) GN=Noc_178 5 PE=4 SV=1;magnesium transporter [Desulfurispirillum indicum S 5];KEGG: dak:DaAHT2_2440 magnesium transporter; TIGRFAM: magnesium transporter; PFAM: MgtE intracellular region; CBS domain containing protein; MgtE integral membrane region;Mg/Co/Ni transporter MgtE (contains CBS domain) [Inorganic ion transport and metabolism]; COG2239;MgtE intracellular N domain; cl15244;This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domain in the magnesium transporter, MgtE.  MgtE and its homologs are found in eubacteria, archaebacteria, and eukaryota. Members of this family transport Mg2+ or other...; cd04606;Divalent cation transporter; cl00786
GOHSU04	CDS	GOHSU_04_01610	169480..171213		putative ABC transporter substrate-binding/permease protein [nite]			ROP_32620 :putative ABC transporter substrate-binding/permease protein : : : Rhodococcus opacus B4	amino acid ABC transporter		C3JW30_RHOER Polar amino acid ABC transporter, inne r membrane subunit OS=Rhodococcus erythropolis SK121 GN=RH OER0001_3326 PE=4 SV=1;amino acid ABC transporter [Arthrobacter phenanthre nivorans Sphe3];PFAM: Binding-protein-dependent transporter inner membrane component; TIGRFAM: amine acid ABC transporter permease, 3-TM region, His/Glu/Gln/Arg/opine family;ABC-type arginine transport system, permease component [Amino acid transport and metabolism]; COG4215;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU04	CDS	GOHSU_04_01620	171215..171973		putative ABC transporter ATP-binding protein [nite]			ROP_32630 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	amino acid ABC transporter ATP-binding protein		F1YEF9_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_00400 PE=3 SV=1;amino acid ABC transporter ATP-binding protein [Art hrobacter phenanthrenivorans Sphe3];PFAM: ABC transporter;ABC-type polar amino acid transport system, ATPase component [Amino acid transport and metabolism]; COG1126;HisP and GlnQ are the ATP-binding components of the bacterial periplasmic histidine and glutamine permeases, repectively.  Histidine permease is a multisubunit complex containing the HisQ and HisM integral membrane subunits and two copies of HisP.  HisP...; cd03262;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU04	CDS	GOHSU_04_01630	complement(171995..172927)		hypothetical protein [migap]				hypothetical protein		F1YEF8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00395 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: abo:ABO_2692 hypothetical protein
GOHSU04	CDS	GOHSU_04_01640	complement(172978..173703)		hypothetical membrane protein [nite]			ROP_41170 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YEF7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00390 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF1275); cl01453
GOHSU04	CDS	GOHSU_04_01650	complement(173700..174023)	overlap	not assigned						F8CIG5_MYXFH Putative uncharacterized protein OS=My xococcus fulvus (strain ATCC BAA-855 / HW-1) GN=LILAB_1564 0 PE=4 SV=1
GOHSU04	CDS	GOHSU_04_01660	173929..175002	overlap	conserved hypothetical protein [nite]			BBR47_46370 :conserved hypothetical protein : : : Brevibacillus brevis NBRC 100599	capsule biosynthesis protein CapA	capA	D0LB55_GORB4 Capsule synthesis protein, CapA (Precu rsor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 432 47 / JCM 3198 / NCTC 10667) GN=Gbro_0138 PE=4 SV=1;capsule biosynthesis protein CapA [Bacillus megater ium QM B1551];Bacterial capsule synthesis protein PGA_cap; smart00854;CapA and related proteins, metallophosphatase domain; cd07381;putative active site [active];putative metal binding site [ion binding]
GOHSU04	CDS	GOHSU_04_01670	complement(175009..176145)		hypothetical protein [nite]			MLP_06340 :hypothetical protein : : : Microlunatus phosphovorus NM-1			F1YNP2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17707 PE=4 SV=1
GOHSU04	CDS	GOHSU_04_01680	complement(176213..177040)		putative sugar-phosphate phosphatase [nite]			TEH_10990 :putative sugar-phosphate phosphatase : : 3.1.3.- : Tetragenococcus halophilus NBRC 12172	hydrolase		F1YEF6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00385 PE=4 SV=1;hydrolase [Vibrio sp. Ex25];haloacid dehalogenase-like hydrolase; pfam08282;Haloacid Dehalogenase-like Hydrolases; cl11391;motif I;motif II
GOHSU04	CDS	GOHSU_04_01690	complement(177112..177894)		hypothetical protein [nite]			RER_11000 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YL04_9ACTO Ferredoxin-like protein OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_13543 PE=4 SV=1;unnamed protein product [Azospirillum lipoferum 4B] ;Evidence 4 : Homologs of previously reported genes of unknown function;Thioredoxin (TRX)-like [2Fe-2S] Ferredoxin (Fd) family; composed of [2Fe-2S] Fds with a TRX fold (TRX-like Fds) and proteins containing domains similar to TRX-like Fd including formate dehydrogenases, NAD-reducing hydrogenases and the subunit E of NADH:...; cd02980;dimer interface [polypeptide binding];[2Fe-2S] cluster binding site [ion binding]
GOHSU04	CDS	GOHSU_04_01700	complement(177891..178745)		putative ABC transporter ATP-binding protein [nite]			RER_11910 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	Iron-chelate-transporting ATPase		F1YL03_9ACTO Abc transporter atpase subunit OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_13538 PE=3 SV=1;Iron-chelate-transporting ATPase [Streptomyces gris eus XylebKG- 1];ABC-type cobalamin/Fe3+-siderophores transport systems, ATPase components [Inorganic ion transport and metabolism / Coenzyme metabolism]; COG1120;PFAM: ABC transporter-like; KEGG: sgr:SGR_271 putative iron uptake ABC transporter ATP-binding protein; SMART: ATPase, AAA+ type, core;ABC transporters, involved in the uptake of siderophores, heme, and vitamin B12, are widely conserved in bacteria and archaea.  Only very few species lack representatives of the siderophore family transporters. The E. coli BtuCD protein is an ABC...; cd03214;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU04	CDS	GOHSU_04_01710	complement(178742..179740)		putative ABC transporter permease protein [nite]			ROP_30430 :putative ABC transporter permease protein : : : Rhodococcus opacus B4	iron complex transport system permease		F1YL02_9ACTO Abc transporter integral membrane subu nit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13533 P E=4 SV=1;iron complex transport system permease [Enterobacte r cloacae subsp. cloacae ATCC 13047];type strain of Enterobacter cloacae subsp. cloacae;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;dimer interface [polypeptide binding];putative PBP binding regions;ABC-ATPase subunit  interface
GOHSU04	CDS	GOHSU_04_01720	complement(179791..180756)		ABC transporter substrate binding protein [nite]			BBR47_20140 :ABC transporter substrate binding protein : : : Brevibacillus brevis NBRC 100599	ABC transporter substrate-binding protein		F1YL01_9ACTO Abc transporter substrate binding lipo protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_135 28 PE=4 SV=1;ABC transporter substrate-binding protein [Rhodococ cus opacus B4];proposed F420-0 ABC transporter, periplasmic F420-0 binding protein; TIGR03868;Metal binding protein TroA_a.  These proteins are predicted to function as initial receptors in ABC transport of metal ions in eubacteria. They belong to the TroA superfamily of helical backbone metal receptor proteins that share a distinct fold and...; cd01148;putative ligand binding residues [chemical binding]
GOHSU04	CDS	GOHSU_04_01730	181030..182052		conserved hypothetical protein [nite]			BBR47_03020 :conserved hypothetical protein : : : Brevibacillus brevis NBRC 100599	F420-dependent methylene-tetrahydromethanopterin reductase		F1YP59_9ACTO Putative monooxygenase/oxidoreductase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18327 PE=4 SV=1;F420-dependent methylene-tetrahydromethanopterin re ductase [Sanguibacter keddieii DSM 10542];PFAM: Luciferase-like monooxygenase;putative luciferase-like monooxygenase, FMN-dependent, CE1758 family; TIGR04036;Flavin-utilizing monoxygenases; cl07892
GOHSU04	CDS	GOHSU_04_01740	complement(182078..183721)		putative ABC transporter ATP-binding protein [nite]			KSE_19050 :putative ABC transporter ATP-binding protein : : : Kitasatospora setae NBRC 14216	ABC transporter--like protein		G7GUD2_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia amarae NBRC 15530 GN=GOAMR_63_00540 PE= 3 SV=1;ABC transporter--like protein [Mycobacterium vanbaa lenii PYR- 1];PFAM: ABC transporter related; SMART: AAA ATPase; KEGG: mmc:Mmcs_5299 ABC transporter related;ATPase components of ABC transporters with duplicated ATPase domains [General function prediction only]; COG0488;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABCF_EF-3  Elongation factor 3 (EF-3) is a cytosolic protein required by fungal ribosomes for in vitro protein synthesis and for in vivo growth.  EF-3 stimulates the binding of the EF-1: GTP: aa-tRNA ternary complex to the ribosomal A site by facilitated...; cd03221;ABC transporter; pfam12848
GOHSU04	CDS	GOHSU_04_01750	complement(183718..184995)		glutaminase [hamap]	glsA [hamap]	MF_00313:glutaminase:glsA 	ROP_68270 :glutaminase : glsA : 3.5.1.2 : Rhodococcus opacus B4	glutaminase		G7H335_9ACTO Glutaminase OS=Gordonia araii NBRC 100 433 GN=glsA PE=3 SV=1;glutaminase [Streptomyces griseus subsp. griseus NB RC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;catalyzes the formation of glutamate from glutamine;Glutaminase; cl00907;Sulphate Transporter and Anti-Sigma factor antagonist domain of SulP-like sulfate transporters, plays a role in the function and regulation of the transport activity, proposed general NTP binding function; cd07042
GOHSU04	CDS	GOHSU_04_01760	complement(185027..186130)		conserved hypothetical protein [nite]			RER_00120 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D4YJT9_9MICO Possible cytoplasmic protein OS=Brevib acterium mcbrellneri ATCC 49030 GN=HMPREF0183_0199 PE=4 SV =1;unnamed protein product [Anabaena variabilis ATCC 2 9413]
GOHSU04	tRNA	GOHSU_04_t0030	complement(186409..186481)		not assigned						identified by tRNAscan-SE; putative
GOHSU04	CDS	GOHSU_04_01770	complement(186634..187377)		hypothetical membrane protein [nite]			ROP_64730 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YE91_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00060 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: afr:AFE_1872 thiol:disulfide interchange protein;Protein-disulfide isomerase [Posttranslational modification, protein turnover, chaperones]; COG1651;DsbA family; consists of DsbA and DsbA-like proteins, including DsbC, DsbG, glutathione (GSH) S-transferase kappa (GSTK), 2-hydroxychromene-2-carboxylate (HCCA) isomerase, an oxidoreductase (FrnE) presumed to be involved in frenolicin biosynthesis, a...; cd02972;catalytic residues [active]
GOHSU04	CDS	GOHSU_04_01780	complement(187439..189076)		phosphoglucomutase [nite]	 pgm [nite]		RER_23280 :phosphoglucomutase : pgm : 5.4.2.2 : Rhodococcus erythropolis PR4	phosphoglucomutase	pgm	F1YE90_9ACTO Phosphoglucomutase OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_00055 PE=3 SV=1;phosphoglucomutase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the interconversion of alpha-D-glucose 1-phosphate to alpha-D-glucose 6-phosphate;phosphoglucomutase; Validated;This bacterial PGM-like (phosphoglucomutase-like) protein of unknown function belongs to the alpha-D-phosphohexomutase superfamily. The alpha-D-phosphohexomutases include several related enzymes that catalyze a reversible intramolecular phosphoryl...; cd05801;substrate binding site [chemical binding];metal binding site [ion binding]
GOHSU04	CDS	GOHSU_04_01790	189173..189760		conserved hypothetical protein [nite]			RER_23200 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YE89_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00050 PE=4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Alpha/beta hydrolase family; pfam12695;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU04	CDS	GOHSU_04_01800	complement(189786..191885)		putative acid--CoA ligase [nite]			ROP_64530 :putative acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	acyl-CoA synthetase		G7GTI7_9ACTO Putative acid--CoA ligase OS=Gordonia amarae NBRC 15530 GN=GOAMR_61_00070 PE=4 SV=1;acyl-CoA synthetase [Nocardia farcinica IFM 10152];activates fatty acids by binding to coenzyme A;acyl-CoA synthetase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU04	CDS	GOHSU_04_01810	complement(191882..192751)		putative acyltransferase [nite]			ROP_17290 :putative acyltransferase : : 2.3.1.- : Rhodococcus opacus B4	phospholipid/glycerol acyltransferase		G7GTI6_9ACTO Putative acyltransferase OS=Gordonia a marae NBRC 15530 GN=GOAMR_61_00060 PE=4 SV=1;phospholipid/glycerol acyltransferase [Mycobacteriu m sp. JLS];PFAM: phospholipid/glycerol acyltransferase; KEGG: mmc:Mmcs_2873 phospholipid/glycerol acyltransferase;Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: MGAT-like; cd07987;putative acyl-acceptor binding pocket
GOHSU04	CDS	GOHSU_04_01820	192865..193728		putative 3-hydroxybutyryl-CoA dehydrogenase [nite]	 hbd [nite]		RER_56680 :putative 3-hydroxybutyryl-CoA dehydrogenase : hbd : 1.1.1.157 : Rhodococcus erythropolis PR4	3-hydroxyacyl-CoA dehydrogenase	hbd-4	D7W160_9FLAO Possible 3-hydroxybutyryl-CoA dehydrog enase OS=Chryseobacterium gleum ATCC 35910 GN=HMPREF0204_1 3144 PE=4 SV=1;3-hydroxyacyl-CoA dehydrogenase [Archaeoglobus fulg idus DSM 4304];3-hydroxyacyl-CoA dehydrogenase [Lipid metabolism]; COG1250;identified by sequence similarity;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;3-hydroxyacyl-CoA dehydrogenase, C-terminal domain; pfam00725
GOHSU04	CDS	GOHSU_04_01830	193893..195539		fatty-acid--CoA ligase [nite]	 fadD [nite]		RER_56210 :fatty-acid--CoA ligase : fadD : 6.2.1.- : Rhodococcus erythropolis PR4	long-chain-fatty-acid--CoA ligase	fadD14_1	F1YE81_9ACTO Long-chain-fatty-acid--CoA ligase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_00010 PE=4 SV=1 ;long-chain-fatty-acid--CoA ligase [Mycobacterium ma rinum M];long-chain-fatty-acid--CoA ligase; Validated;activates fatty acids by binding to coenzyme A;AMP-binding enzyme; cl15778
GOHSU04	CDS	GOHSU_04_01840	195558..197138		hypothetical membrane protein [nite]			DMR_24520 :hypothetical membrane protein : : : Desulfovibrio magneticus RS-1	YidE/YbjL duplication		E8JG39_9ACTO Putative uncharacterized protein OS=Ac tinomyces sp. oral taxon 178 str. F0338 GN=HMPREF9005_0486 PE=3 SV=1;YidE/YbjL duplication [Arcanobacterium haemolyticum DSM 20595];COGs: COG2985 permease; InterPro IPR006037:IPR006512:IPR016040; KEGG: car:cauri_1117  transporter protein; PFAM: YidE/YbjL duplication domain protein; TrkA-C domain protein; SPTR: C0W2B0 YidE/YbjL family protein; TIGRFAM: YidE/YbjL duplication; PFAM: Predicted Permease membrane Region; TrkA-C domain; TIGRFAM: YidE/YbjL duplication;Predicted permease [General function prediction only];Anion permease ArsB/NhaD.  These permeases have been shown to translocate sodium, arsenate, antimonite, sulfate and organic anions across biological membranes in all three kingdoms of life.  A typical anion permease contains 8-13 transmembrane helices...; cl15357;Predicted Permease Membrane Region; cl06061
GOHSU04	CDS	GOHSU_04_01850	197446..198696		not assigned						A0LR57_ACIC1 Putative uncharacterized protein (Prec ursor) OS=Acidothermus cellulolyticus (strain ATCC 43068 / 11B) GN=Acel_0142 PE=4 SV=1
GOHSU04	CDS	GOHSU_04_01860	198788..199447		hypothetical protein [nite]			ROP_58800 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5U857_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_001_01640 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Thioesterase superfamily; manually curated;PaaI_thioesterase is a tetrameric acyl-CoA thioesterase with a hot dog fold and one of several proteins responsible for phenylacetic acid (PA) degradation in bacteria.  Although orthologs of PaaI exist in archaea and eukaryotes, their function has not...; cd03443;CoenzymeA binding site [chemical binding];subunit interaction site [polypeptide binding];PHB binding site
GOHSU04	CDS	GOHSU_04_01870	199575..201119		putative fatty-acid--CoA ligase [nite]			ROP_12890 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	AMP-dependent synthetase and ligase		F1YGC2_9ACTO Acyl-CoA synthetase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_04766 PE=4 SV=1;AMP-dependent synthetase and ligase [Frankia sp. Eu I1c];PFAM: AMP-dependent synthetase and ligase; KEGG: rop:ROP_38320  fatty-acid--CoA ligase;long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778
GOHSU04	CDS	GOHSU_04_01880	201177..202298		putative oxidoreductase [nite]			ROP_50960 :putative oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase		G8SKN3_ACTS5 FAD-dependent pyridine nucleotide-disu lfide oxidoreductase OS=Actinoplanes sp. (strain ATCC 3104 4 / CBS 674.73 / SE50/110) GN=ACPL_4859 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU04	CDS	GOHSU_04_01890	complement(202328..202732)		putative cadmium-induced protein CadI [nite]	 cadI [nite]		ROP_40640 :putative cadmium-induced protein CadI : cadI : : Rhodococcus opacus B4	glyoxalase/bleomycin resistance protein/dioxygenase		G2NWD2_STRVO Glyoxalase/bleomycin resistance protei n /dioxygenase OS=Streptomyces violaceusniger Tu 4113 GN=S trvi_7465 PE=4 SV=1;glyoxalase/bleomycin resistance protein/dioxygenase [Mycobacterium sp. JLS];PFAM: Glyoxalase/bleomycin resistance protein/dioxygenase; KEGG: rha:RHA1_ro04132 possible lactoylglutathione lyase;This conserved domain belongs to a superfamily including the bleomycin resistance protein, glyoxalase I, and type I ring-cleaving dioxygenases; cd07254;Glyoxalase/Bleomycin resistance protein/Dioxygenase superfamily; pfam00903;putative metal binding site [ion binding]
GOHSU04	CDS	GOHSU_04_01900	complement(202748..203893)		putative FAD-dependent oxidoreductase [nite]			RER_pREL1-01630 :putative FAD-dependent oxidoreductase : : : Rhodococcus erythropolis PR4	NADH binding oxidoreductase		Q3L9K7_RHOE4 Putative FAD-dependent oxidoreductase OS=Rhodococcus erythropolis (strain PR4 / NBRC 100887) GN= RER_pREL1-01630 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU04	CDS	GOHSU_04_01910	complement(203890..204939)		putative arsenite resistance protein [nite]	 arsB [nite]		RER_pREL1-01590 :putative arsenite resistance protein : arsB : : Rhodococcus erythropolis PR4	arsenic resistance protein		Q3L9L1_RHOE4 Putative arsenite resistance protein O S=Rhodococcus erythropolis (strain PR4 / NBRC 100887) GN=a rsB PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: arsenic resistance protein; PFAM: Bile acid:sodium symporter; KEGG: oan:Oant_3198 arsenic resistance protein;Arsenite efflux pump ACR3 and related permeases [Inorganic ion transport and metabolism]; COG0798;Membrane transport protein; cl09117
GOHSU04	CDS	GOHSU_04_01920	complement(204936..205295)		putative arsenic resistance operon repressor ArsR [nite]	 arsR [nite]		RER_35210 :putative arsenic resistance operon repressor ArsR : arsR : : Rhodococcus erythropolis PR4	ArsR family transcriptional regulator		C1A0U4_RHOE4 Putative arsenic resistance operon rep ressor ArsR OS=Rhodococcus erythropolis (strain PR4 / NBRC 100887) GN=arsR PE=4 SV=1;ArsR family transcriptional regulator [Catenulispor a acidiphila DSM 44928];PFAM: ArsR family transcriptional regulator; SMART: ArsR family transcriptional regulator; KEGG: sgr:SGR_1747 ArsR family transcriptional regulator;Arsenical Resistance Operon Repressor and similar prokaryotic, metal regulated homodimeric repressors. ARSR subfamily of helix-turn-helix bacterial transcription regulatory proteins (winged helix topology). Includes several proteins that appear to...; cd00090;dimerization interface [polypeptide binding];putative DNA binding site [nucleotide binding];putative Zn2+ binding site [ion binding]
GOHSU04	CDS	GOHSU_04_01930	complement(205368..205862)		not assigned						H0R672_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_120_00330 PE=4 SV=1
GOHSU04	CDS	GOHSU_04_01940	complement(206643..207116)	case	SsrA-binding protein [hamap]	smpB [hamap]	MF_00023:SsrA-binding protein:smpB 	ROP_64660 :SsrA-binding protein : smpB : : Rhodococcus opacus B4	ssra-binding protein		F1YJ70_9ACTO SsrA-binding protein OS=Gordonia neofe lifaecis NRRL B-59395 GN=smpB PE=3 SV=1;ssra-binding protein [Micromonospora sp. L5];KEGG: stp:Strop_0984 SsrA-binding protein; TIGRFAM: SsrA-binding protein; PFAM: SmpB protein;Small protein B (SmpB) is a component of the trans-translation system in prokaryotes for releasing stalled ribosome from damaged messenger RNAs; cd09294;SmpB-tmRNA interface
GOHSU04	CDS	GOHSU_04_01950	complement(207185..208006)		putative cell division protein FtsX [nite]	 ftsX [nite]		RER_23110 :putative cell division protein FtsX : ftsX : : Rhodococcus erythropolis PR4	cell division protein	ftsX	H0R0I8_9ACTO Cell division protein FtsX OS=Gordonia effusa NBRC 100432 GN=ftsX PE=4 SV=1;cell division protein [Corynebacterium glutamicum A TCC 13032];IS fingerprint type 4-5;Cell division protein [Cell division and chromosome partitioning]; COG2177;FtsX-like permease family; cl15850
GOHSU04	CDS	GOHSU_04_01960	complement(208016..208702)		putative cell division protein FtsE [nite]	 ftsE [nite]		RER_23100 :putative cell division protein FtsE : ftsE : : Rhodococcus erythropolis PR4	cell division ATP-binding protein FtsE	ftsE	F1YJ68_9ACTO Cell division ATP-binding protein FtsE OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10434 PE=3 SV=1;ftsE gene product [Gordonia polyisoprenivorans VH2] ;Predicted ATPase involved in cell division [Cell division and chromosome partitioning]; COG2884;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU04	CDS	GOHSU_04_01970	complement(208750..209259)		not assigned						F9VRV3_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_028_00200 PE=4 SV= 1
GOHSU04	CDS	GOHSU_04_01980	complement(209256..210179)		putative MscS family protein [nite]			RER_23080 :putative MscS family protein : : : Rhodococcus erythropolis PR4	small-conductance mechanosensitive channel		D0LE57_GORB4 MscS Mechanosensitive ion channel OS=G ordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3 198 / NCTC 10667) GN=Gbro_3455 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Mechanosensitive ion channel;Mechanosensitive ion channel; pfam00924
GOHSU04	CDS	GOHSU_04_01990	complement(210166..211284)		peptide chain release factor 2 [hamap]	prfB [hamap]	MF_00094:peptide chain release factor 2:prfB 	RER_23070 :peptide chain release factor 2 : prfB : : Rhodococcus erythropolis PR4	peptide chain release factor 2	prfB	F1YJ65_9ACTO Peptide chain release factor 2 OS=Gord onia neofelifaecis NRRL B-59395 GN=prfB PE=3 SV=1;peptide chain release factor 2 [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;peptide chain release factor 2; Validated; PRK00578;recognizes the termination signals UGA and UAA during protein translation a specificity which is dependent on amino acid residues residing in loops of the L-shaped tRNA-like molecule of RF2; in some organisms control of PrfB protein levels is maintained through a +1 ribosomal frameshifting mechanism; this protein is similar to release factor 1;RF-1 domain; cl02875
GOHSU04	CDS	GOHSU_04_02000	complement(211316..211672)		hypothetical protein [migap]				hypothetical protein		F1YJ64_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10414 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU04	CDS	GOHSU_04_02010	211692..212519		histidinol-phosphate phosphatase [nite]	 hisN [nite]		ROP_64590 :histidinol-phosphate phosphatase : hisN : 3.1.3.15 : Rhodococcus opacus B4	histidinol-phosphate phosphatase		F9VRU9_9ACTO Histidinol-phosphate phosphatase OS=Go rdonia alkanivorans NBRC 16433 GN=hisN PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: histidinol-phosphate phosphatase; PFAM: inositol monophosphatase; KEGG: afw:Anae109_2478 inositol monophosphatase;Predominantly bacterial family of Mg++ dependend phosphatases, related to inositol monophosphatases. These enzymes may dephosphorylate fructose-1,6-bisphosphate, inositol monophospate, 3'-phosphoadenosine-5'-phosphate, or similar substrates; cd01641
GOHSU04	CDS	GOHSU_04_02020	213037..213390		hypothetical protein [nite]			KSE_61070 :hypothetical protein : : : Kitasatospora setae NBRC 14216	glyoxalase/bleomycin resistance protein/dioxygenase		I0RDK2_MYCPH Glyoxalase/bleomycin resistance protei n /dioxygenase OS=Mycobacterium phlei RIVM601174 GN=MPHLEI _24686 PE=4 SV=1;glyoxalase/bleomycin resistance protein/dioxygenase [Mycobacterium vanbaalenii PYR- 1];PFAM: Glyoxalase/bleomycin resistance protein/dioxygenase; KEGG: mpa:MAP0136 hypothetical protein;This domain superfamily is found in a variety of structurally related metalloproteins, including the type I extradiol dioxygenases, glyoxalase I and a group of antibiotic resistance proteins; cd06587;Glyoxalase/Bleomycin resistance protein/Dioxygenase superfamily; pfam00903;metal binding site [ion binding]
GOHSU04	CDS	GOHSU_04_02030	complement(213398..214126)		purine nucleoside phosphorylase [nite]	 deoD [nite]		KRH_06640 :purine nucleoside phosphorylase : deoD : 2.4.2.1 : Kocuria rhizophila DC2201	purine nucleoside phosphorylase		G7GPH1_9ACTO Purine nucleoside phosphorylase deoD-t ype OS=Gordonia amarae NBRC 15530 GN=deoD PE=3 SV=1;unnamed protein product [Staphylococcus aureus subs p. aureus M013];Phosphorylase superfamily; cl00303;Uridine phosphorylase [Nucleotide transport and metabolism]; COG2820
GOHSU04	CDS	GOHSU_04_02040	complement(214197..215867)		putative phosphoglucomutase [nite]			KRH_18890 :putative phosphoglucomutase : : 5.4.2.2 : Kocuria rhizophila DC2201	phosphomannomutase		H0RM66_9ACTO Putative phosphoglucomutase OS=Gordoni a polyisoprenivorans NBRC 16320 GN=GOPIP_092_00850 PE=3 SV =1;unnamed protein product [Gordonia polyisoprenivoran s VH2];phosphoglucomutase-2-like protein; Provisional;This CD includes PGM2 (phosphoglucomutase 2) and PGM2L1 (phosphoglucomutase 2-like 1). The mammalian PGM2 is thought to be a phosphopentomutase that catalyzes the conversion of the nucleoside breakdown products, ribose-1-phosphate and...; cd05799;substrate binding site [chemical binding];metal binding site [ion binding]
GOHSU04	CDS	GOHSU_04_02050	complement(215891..216553)		conserved hypothetical protein [nite]			RER_10450 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	beta-lactamase		F1YJ61_9ACTO Beta-lactamase domain-containing prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10399 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Metallo-beta-lactamase superfamily; cl00446
GOHSU04	CDS	GOHSU_04_02060	216666..218057		putative acyl-CoA dehydrogenase [nite]			RER_23010 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	Acyl-CoA dehydrogenase, C-terminal domain protein		G7H7P8_9ACTO Putative acyl-CoA dehydrogenase OS=Gor donia araii NBRC 100433 GN=GOARA_090_00050 PE=3 SV=1;Acyl-CoA dehydrogenase, C-terminal domain protein [ Alcanivorax sp. DG881];[I] COG1960 Acyl-CoA dehydrogenases;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU04	CDS	GOHSU_04_02070	218057..219271		putative acyl-CoA dehydrogenase [nite]			RER_23000 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase		G7H7P7_9ACTO Putative acyl-CoA dehydrogenase OS=Gor donia araii NBRC 100433 GN=GOARA_090_00040 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU04	CDS	GOHSU_04_02080	219487..220893		putative lipase [nite]			ROP_48190 :putative lipase : : 3.1.1.- : Rhodococcus opacus B4	lipase		H0R0K1_9ACTO Putative lipase OS=Gordonia effusa NBR C 100432 GN=GOEFS_059_00370 PE=4 SV=1;lipase [Rhodococcus opacus B4];Secretory lipase; pfam03583
GOHSU04	CDS	GOHSU_04_02090	220927..221547		putative TetR family transcriptional regulator [nite]			RER_57170 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		G7GLC0_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia amarae NBRC 15530 GN=GOAMR_19_00820 P E=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m sp. JLS];PFAM: regulatory protein, TetR; KEGG: mmc:Mmcs_3200 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309
GOHSU04	CDS	GOHSU_04_02100	complement(221569..223476)		propionate--CoA ligase [nite]	 prpE [nite]		ROP_50660 :propionate--CoA ligase : prpE : 6.2.1.17 : Rhodococcus opacus B4	acyl-CoA synthetase/AMP-acid ligase		F1YJ53_9ACTO Acyl-CoA ligase/synthetase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10359 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function (DUF3448); AMP-binding enzyme;propionyl-CoA synthetase; Provisional; PRK10524;Domain of unknown function (DUF3448); pfam11930;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU04	CDS	GOHSU_04_02110	223727..224053		putative resuscitation-promoting factor [nite]			RER_22320 :putative resuscitation-promoting factor : : : Rhodococcus erythropolis PR4	transglycosylase domain-containing protein;hypothetical protein		G7H3W8_9ACTO Putative resuscitation-promoting facto r OS=Gordonia araii NBRC 100433 GN=GOARA_059_00090 PE=4 SV =1;transglycosylase domain-containing protein [Mycobac terium sp. JLS];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 1.000) with cleavage site probability 0.996 at residue 42;PFAM: Transglycosylase domain protein; KEGG: mmc:Mmcs_3564 transglycosylase-like protein;lysozyme_like domain.  This contains several members including Soluble Lytic Transglycosylases (SLT), Goose Egg-White Lysozymes (GEWL), Hen Egg-White Lysozymes (HEWL), chitinases, bacteriophage lambda lysozymes, endolysins, autolysins, and chitosanases; cl00222
GOHSU04	CDS	GOHSU_04_02120	224129..225520		ribonuclease [nite]			SLG_p_00930 :ribonuclease : : : Sphingobium sp. SYK-6	beta-lactamase domain-containing protein		F1YJ51_9ACTO Beta-lactamase domain-containing prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10349 PE =4 SV=1;beta-lactamase domain-containing protein [Acidovora x sp. JS42];Predicted exonuclease of the beta-lactamase fold involved in RNA processing [Translation, ribosomal structure and biogenesis]; COG1236;PFAM: beta-lactamase domain protein; RNA-metabolising metallo-beta-lactamase; KEGG: pol:Bpro_1210 beta-lactamase-like;Metallo-beta-lactamase superfamily; cl00446;Beta-Casp domain; cl12567;RNA-metabolising metallo-beta-lactamase; pfam07521
GOHSU04	CDS	GOHSU_04_02130	complement(225553..225990)		conserved hypothetical membrane protein [nite]			RER_44470 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJ50_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10344 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU04	CDS	GOHSU_04_02140	226086..226685		putative TetR family transcriptional regulator [nite]			ROP_04540 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	regulator		F1YFE9_9ACTO TetR family transcriptional regulator OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02752 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];transcriptional regulator BetI; Validated;Helix-turn-helix domains; cl00088;Bacterial transcriptional repressor; cl07106
GOHSU04	CDS	GOHSU_04_02150	226811..227311		ferritin [nite]	 ftnA [nite]		CSE_12380 :ferritin : ftnA : 1.16.3.1 : Caldisericum exile AZM16c01	ferritin		F1YK41_9ACTO Ferritin Dps family protein OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_11790 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: ferritin; KEGG: bfa:Bfae_30880 ferritin-like protein;nonheme-containing ferritins; cd01055;Ferritin-like domain; pfam00210;ferroxidase diiron center [ion binding]
GOHSU04	CDS	GOHSU_04_02160	complement(227786..228349)		putative TetR family transcriptional regulator [nite]			RER_07990 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		F9VRF6_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_026_ 00010 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m sp. KMS];KEGG: mmc:Mmcs_1595 putative transcriptional regulator, TetR family;Helix-turn-helix domains; cl00088
GOHSU04	CDS	GOHSU_04_02170	complement(228353..229177)		putative triacylglycerol lipase [nite]			ROP_43380 :putative triacylglycerol lipase : : 3.1.1.3 : Rhodococcus opacus B4			E3UUE9_RHORH Putative lipase OS=Rhodococcus rhodoch rous PE=4 SV=1
GOHSU04	CDS	GOHSU_04_02180	229302..231155		putative multicopper oxidase [nite]			RER_03790 :putative multicopper oxidase : : : Rhodococcus erythropolis PR4	multicopper oxidase		F9VRF7_9ACTO Putative multicopper oxidase OS=Gordon ia alkanivorans NBRC 16433 GN=GOALK_026_00020 PE=4 SV=1;multicopper oxidase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;TAT (twin-arginine translocation) pathway signal sequence; pfam10518;Cytochrome C oxidase subunit II, periplasmic domain; cl11412
GOHSU04	CDS	GOHSU_04_02190	231224..231970		putative YrbE family protein [nite]	 yrbE3A [nite]		ROP_25240 :putative YrbE family protein : yrbE3A : : Rhodococcus opacus B4	organic solvents resistance ABC transporter permease		C1B3S3_RHOOB Putative YrbE family protein OS=Rhodoc occus opacus (strain B4) GN=yrbE3A PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF140; TIGRFAM: conserved hypothetical integral membrane protein;Permease; cl00510
GOHSU04	CDS	GOHSU_04_02200	231967..232818		putative YrbE family protein [nite]	 yrbE5B [nite]		RER_33700 :putative YrbE family protein : yrbE5B : : Rhodococcus erythropolis PR4	organic solvents resistance ABC transporter permease		C1A0E3_RHOE4 Putative YrbE family protein OS=Rhodoc occus erythropolis (strain PR4 / NBRC 100887) GN=yrbE5B PE =4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF140; TIGRFAM: conserved hypothetical integral membrane protein;Permease; cl00510
GOHSU04	CDS	GOHSU_04_02210	232822..233991		putative Mce family protein [nite]	 mce5A [nite]		ROP_47970 :putative Mce family protein : mce5A : : Rhodococcus opacus B4	virulence factor Mce family protein;hypothetical protein		C1BBJ1_RHOOB Putative Mce family protein OS=Rhodoco ccus opacus (strain B4) GN=mce5A PE=4 SV=1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.992) with cleavage site probability 0.378 at residue 35;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_0275 mammalian cell entry;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU04	CDS	GOHSU_04_02220	233988..235004		putative Mce family protein [nite]	 mce3B [nite]		RER_09610 :putative Mce family protein : mce3B : : Rhodococcus erythropolis PR4	virulence factor Mce family protein		A4T972_MYCGI Virulence factor Mce family protein (P recursor) OS=Mycobacterium gilvum (strain PYR-GCK) GN=Mflv _2102 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: mce related protein; TIGRFAM: virulence factor Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU04	CDS	GOHSU_04_02230	235001..236026		putative Mce family protein [nite]	 mce5C [nite]		ROP_47990 :putative Mce family protein : mce5C : : Rhodococcus opacus B4	virulence factor Mce family protein;hypothetical protein		I0WA26_9NOCA MCE family protein OS=Rhodococcus imte chensis RKJ300 GN=W59_34108 PE=4 SV=1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.985) with cleavage site probability 0.609 at residue 28;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_1656 mammalian cell entry;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU04	CDS	GOHSU_04_02240	236023..237234		putative Mce family protein [nite]	 mce5D [nite]		ROP_48000 :putative Mce family protein : mce5D : : Rhodococcus opacus B4	virulence factor Mce family protein;hypothetical protein		C1BBJ4_RHOOB Putative Mce family protein OS=Rhodoco ccus opacus (strain B4) GN=mce5D PE=4 SV=1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.993) with cleavage site probability 0.503 at residue 38;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_4614 mammalian cell entry;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU04	CDS	GOHSU_04_02250	237234..238358		putative Mce family protein [nite]	 mce3E [nite]		RER_09640 :putative Mce family protein : mce3E : : Rhodococcus erythropolis PR4	virulence factor Mce family protein;hypothetical protein		C0ZRC2_RHOE4 Putative Mce family protein OS=Rhodoco ccus erythropolis (strain PR4 / NBRC 100887) GN=mce3E PE=4 SV=1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.959) with cleavage site probability 0.533 at residue 23;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mtc:MT2022 virulence factor mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU04	CDS	GOHSU_04_02260	238355..239782		putative Mce family protein [nite]	 mce5F [nite]		ROP_48020 :putative Mce family protein : mce5F : : Rhodococcus opacus B4	virulence factor Mce family protein		C1BBJ6_RHOOB Putative Mce family protein OS=Rhodoco ccus opacus (strain B4) GN=mce5F PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: mce related protein; TIGRFAM: virulence factor Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU04	CDS	GOHSU_04_02270	complement(239811..240560)		hypothetical protein [migap]				hypothetical protein		A3Q458_MYCSJ Putative uncharacterized protein OS=My cobacterium sp. (strain JLS) GN=Mjls_4163 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU04	CDS	GOHSU_04_02280	complement(240557..241126)		hypothetical membrane protein [nite]			ROP_15970 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		E6TC97_MYCSR Putative uncharacterized protein OS=My cobacterium sp. (strain Spyr1) GN=Mspyr1_19530 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU04	CDS	GOHSU_04_02290	241291..242448		hypothetical protein [nite]			ROP_34400 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5UCC5_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_039_02240 PE=4 SV=1;unnamed protein product [Sphingomonas wittichii RW1 ]
GOHSU04	CDS	GOHSU_04_02300	242555..243850		hypothetical protein [migap]				hypothetical protein		F9W1R7_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_117_00380 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Uncharacterized protein conserved in bacteria (DUF2236); cl01472
GOHSU04	CDS	GOHSU_04_02310	243863..245164		putative dioxygenase [nite]			ROP_20230 :putative dioxygenase : : : Rhodococcus opacus B4	homogentisate 1,2-dioxygenase		D5ZRB6_9ACTO Homogentisate 1,2-dioxygenase OS=Strep tomyces ghanaensis ATCC 14672 GN=hmgA PE=3 SV=1;homogentisate 1,2-dioxygenase [Burkholderia xenovor ans LB400];An oxygenase that acts to open the ring of homogentisate formingmaleylacetoacetate as part of the catabolism of L-tyrosine and L-phenylalanine;Cupin domain; cl09118
##GOHSU05	contig00022				34770
GOHSU05	CDS	GOHSU_05_00010	80..1558		aldehyde dehydrogenase [nite]			ROP_25550 :aldehyde dehydrogenase : : 1.2.1.3 : Rhodococcus opacus B4	aldehyde dehydrogenase		C1B3V4_RHOOB Aldehyde dehydrogenase OS=Rhodococcus opacus (strain B4) GN=ROP_25550 PE=3 SV=1;unnamed protein product [Kyrpidia tusciae DSM 2912] ;PFAM: Aldehyde Dehydrogenase; KEGG: gwc:GWCH70_0886 aldehyde dehydrogenase;NAD(P)+-dependent aldehyde dehydrogenase superfamily; cl11961;Aldehyde dehydrogenase family; pfam00171;NAD(P) binding site [chemical binding];catalytic residues [active]
GOHSU05	CDS	GOHSU_05_00020	1548..2813		conserved hypothetical protein [nite]			RER_04240 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	formyl-CoA transferase		B2HGF9_MYCMM Conserved protein OS=Mycobacterium mar inum (strain ATCC BAA-535 / M) GN=MMAR_1494 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: L-carnitine dehydratase/bile acid-inducible protein F; KEGG: bpt:Bpet2785 acyl-CoA transferase/carnitine dehydratase;Predicted acyl-CoA transferases/carnitine dehydratase [Energy production and conversion]; COG1804;CoA-transferase family III; pfam02515
GOHSU05	CDS	GOHSU_05_00030	2954..4018		hypothetical protein [migap]				hypothetical protein		H5S997_9CHLR Hypothetical conserved protein OS=uncu ltured Chloroflexi bacterium GN=HGMM_F03B08C11 PE=4 SV=1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];KEGG: mpe:MYPE5150 hypothetical protein;Hydroxyneurosporene synthase (CrtC); cl12101
GOHSU05	CDS	GOHSU_05_00040	4221..4643		hypothetical protein [migap]				hypothetical protein		H5TVH3_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_007_00920 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU05	tRNA	GOHSU_05_t0010	complement(4696..4772)		not assigned						identified by tRNAscan-SE; putative
GOHSU05	CDS	GOHSU_05_00050	5089..5376		hypothetical protein [migap]				hypothetical protein		F1YJ44_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10314 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.854) with cleavage site probability 0.645 at residue 44
GOHSU05	CDS	GOHSU_05_00060	complement(5544..8534)		UPF0182 protein <locus_tag> [hamap]		MF_01600:UPF0182 protein <locus_tag>: 	RER_22310 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TXF9_9ACTO UPF0182 protein GOSPT_034_00340 OS=Gor donia sputi NBRC 100414 GN=GOSPT_034_00340 PE=3 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Uncharacterized conserved protein [Function unknown]; cl09133;Differs from Mb0065 by 1aa, T435I, and from Rv0064 by 4aa, T435I, D457G, F550V, P906R in BCG Pasteur, BCG Tokyo and BCG Mexico;Uncharacterized protein family (UPF0182); pfam03699
GOHSU05	CDS	GOHSU_05_00070	8621..9112		hypothetical protein [nite]			ROP_64490 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YJ42_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10304 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rop:ROP_64490 hypothetical protein
GOHSU05	CDS	GOHSU_05_00080	complement(9159..10157)		hypothetical protein [nite]			ROP_64480 :hypothetical protein : : : Rhodococcus opacus B4	secreted protein containing a PDZ domain-like protein		G7GT10_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_59_00350 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted secreted protein containing a PDZ domain [Signal transduction mechanisms]; COG3480;PDZ domain of tryspin-like serine proteases, such as DegP/HtrA, which are oligomeric proteins involved in heat-shock response, chaperone function, and apoptosis. May be responsible for substrate recognition and/or binding, as most PDZ domains bind...; cd00987;Subunit ChlI of Mg-chelatase; cl12214
GOHSU05	CDS	GOHSU_05_00090	10255..11703		hypothetical protein [nite]			ROP_64470 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0RM70_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_092_00890 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Uncharacterized conserved protein (DUF2342); cl02183
GOHSU05	CDS	GOHSU_05_00100	11735..12868	overlap	conserved hypothetical protein [nite]			RER_22270 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0QUG0_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_001_00060 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein;MoeZ/MoeB domain; cl14816
GOHSU05	CDS	GOHSU_05_00110	12753..14087	overlap	conserved hypothetical protein [nite]			RER_22260 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	ABC transporter ATP-binding protein		F1YJ37_9ACTO ABC-1 domain-containing protein OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_10279 PE=4 SV=1;ABC transporter ATP-binding protein [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q73US1_MYCPA e-146;Predicted unusual protein kinase [General function prediction only]; COG0661;Protein Kinases, catalytic domain; cl09925;ATP binding site [chemical binding]
GOHSU05	CDS	GOHSU_05_00120	complement(14281..14592)		putative WhiB family regulatory protein [nite]			RER_22250 :putative WhiB family regulatory protein : : : Rhodococcus erythropolis PR4	WhiB-family transcriptional regulator		Q8VJ53_MYCTU Putative uncharacterized protein OS=My cobacterium tuberculosis GN=MT3290.1 PE=4 SV=1;WhiB-family transcriptional regulator [Streptomyces clavuligerus ATCC 27064];Transcription factor WhiB; pfam02467
GOHSU05	CDS	GOHSU_05_00130	complement(14982..16994)		probable ATP-dependent DNA helicase UvrD2 [nite]	 uvrD2 [nite]		RER_22240 :probable ATP-dependent DNA helicase UvrD2 : uvrD2 : 3.6.1.- : Rhodococcus erythropolis PR4	ATP-dependent DNA helicase		F1YJ35_9ACTO UvrD/REP helicase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_10269 PE=4 SV=1;ATP-dependent DNA helicase [Streptomyces clavuliger us ATCC 27064];Superfamily I DNA and RNA helicases [DNA replication, recombination, and repair]; COG0210;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Family description; cl15862;HRDC domain; cl02578
GOHSU05	CDS	GOHSU_05_00140	complement(17083..17331)		conserved hypothetical protein [nite]			RER_22230 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	Glutaredoxin-like protein		F1YJ34_9ACTO Glutaredoxin-like protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10264 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Glutaredoxin; TIGRFAM: Glutaredoxin-like protein;NrdH-redoxin (NrdH) family; NrdH is a small monomeric protein with a conserved redox active CXXC motif within a TRX fold, characterized by a glutaredoxin (GRX)-like sequence and TRX-like activity profile. In vitro, it displays protein disulfide reductase...; cd02976;catalytic residues [active]
GOHSU05	CDS	GOHSU_05_00150	complement(17364..18059)		not assigned						H1Q8I8_9ACTO Putative uncharacterized protein OS=St reptomyces coelicoflavus ZG0656 GN=SMCF_1176 PE=4 SV=1
GOHSU05	CDS	GOHSU_05_00160	complement(18035..18979)		putative NADH pyrophosphatase [nite]	 nudC [nite]		ROP_64410 :putative NADH pyrophosphatase : nudC : 3.6.1.22 : Rhodococcus opacus B4	NADH pyrophosphatase	nudC	F1YJL5_9ACTO NADH pyrophosphatase OS=Gordonia neofe lifaecis NRRL B-59395 GN=nudC PE=3 SV=1;NADH pyrophosphatase [Rhodococcus jostii RHA1];can catalyze hydrolysis of broad range of dinucleotide pyrophosphates but prefers reduced form of NADH; requires divalent metal ions such as magnesium and manganese and produces two mononucleoside 5'-phosphates;NADH pyrophosphatase; Reviewed; PRK00241;NADH pyrophosphatase-like rudimentary NUDIX domain; pfam09296;NADH pyrophosphatase, a member of the Nudix hydrolase superfamily, catalyzes the cleavage of NADH into reduced nicotinamide mononucleotide (NMNH) and AMP. Like other members of the Nudix family, it requires a divalent cation, such as Mg2+ or Mn2+, for...; cd03429;putative NADH binding site [chemical binding];putative active site [active];nudix motif;putative metal binding site [ion binding]
GOHSU05	CDS	GOHSU_05_00170	complement(18983..20068)		hypothetical membrane protein [nite]			ROP_64400 :hypothetical membrane protein : : : Rhodococcus opacus B4	ion channel membrane protein		F1YJL4_9ACTO TrkA-N domain-containing protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10254 PE=4 SV=1;ion channel membrane protein [Streptomyces clavulig erus ATCC 27064];Ion channel; cl11596;Kef-type K+ transport systems, predicted NAD-binding component [Inorganic ion transport and metabolism]; COG1226;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU05	CDS	GOHSU_05_00180	complement(20096..23458)		putative ATP-dependent DNA helicase [nite]			ROP_64380 :putative ATP-dependent DNA helicase : : 3.6.1.- : Rhodococcus opacus B4	DNA helicase II	uvrD2	F1YJL3_9ACTO UvrD/REP helicase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_10249 PE=4 SV=1;uvrD2 gene product [Gordonia polyisoprenivorans VH2 ];Superfamily I DNA and RNA helicases [DNA replication, recombination, and repair]; COG0210;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;exodeoxyribonuclease V, beta subunit; TIGR00609;CRISPR/Cas system-associated protein Cas4; cl00641
GOHSU05	CDS	GOHSU_05_00190	complement(23451..26747)		putative ATP-dependent DNA helicase [nite]			ROP_64370 :putative ATP-dependent DNA helicase : : 3.6.1.- : Rhodococcus opacus B4	UvrD/REP helicase		F1YJL2_9ACTO UvrD/REP helicase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_10244 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: UvrD/REP helicase; KEGG: glo:Glov_2069 UvrD/REP helicase;Superfamily I DNA and RNA helicases [DNA replication, recombination, and repair]; COG0210;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;ATP-dependent exoDNAse (exonuclease V) beta subunit (contains helicase and exonuclease domains) [DNA replication, recombination, and repair]; COG1074;CRISPR/Cas system-associated protein Cas4; cl00641
GOHSU05	CDS	GOHSU_05_00200	26837..27622		putative hydrolase [nite]			ROP_64360 :putative hydrolase : : : Rhodococcus opacus B4	putative hydrolase		F1YJL1_9ACTO Alpha/beta hydrolase fold protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_10239 PE=4 SV=1 ;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG0596 Predicted hydrolases or acyltransferases (alpha/beta hydrolase superfamily);Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697
GOHSU05	CDS	GOHSU_05_00210	27624..27914		hypothetical protein [nite]			ROP_64350 :hypothetical protein : : : Rhodococcus opacus B4	putative DNA-methyltransferase		G7H2E5_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_050_00830 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Differs from Rv3204 by 1aa, A34T in Mycobacterium tuberculosis CDC1551, Mycobacterium bovis, Mycobacterium bovis BCG Pasteur, BCG Tokyo and BCG Mexico;The DNA repair protein O6-alkylguanine-DNA alkyltransferase (ATase; also known as AGT, AGAT and MGMT) reverses O6-alkylation DNA damage by transferring O6-alkyl adducts to an active site cysteine irreversibly, without inducing DNA strand breaks. ATases...; cd06445;DNA binding site [nucleotide binding]
GOHSU05	CDS	GOHSU_05_00220	complement(28207..29067)		hypothetical protein [nite]			ROP_64230 :hypothetical protein : : : Rhodococcus opacus B4	predicted protein		H0QUH3_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_001_00190 PE=4 SV=1;predicted protein [Streptomyces sp. AA4];TIGR02569 family protein
GOHSU05	CDS	GOHSU_05_00230	complement(29192..30367)		putative molybdopterin synthase sulfurylase MoeB [nite]	 moeB [nite]		RER_22020 :putative molybdopterin synthase sulfurylase MoeB : moeB : : Rhodococcus erythropolis PR4	UBA/THIF-type NAD/FAD binding protein		H0RAE3_9ACTO Molybdopterin synthase sulfurylase Moe B OS=Gordonia polyisoprenivorans NBRC 16320 GN=moeB PE=4 S V=1;UBA/THIF-type NAD/FAD binding protein [Streptomyces clavuligerus ATCC 27064];molybdopterin biosynthesis-like protein MoeZ; Validated;ThiF_MoeB_HesA. Family of E1-like enzymes involved in molybdopterin and thiamine biosynthesis family. The common reaction mechanism catalyzed by MoeB and ThiF, like other E1 enzymes, begins with a nucleophilic attack of the C-terminal carboxylate of MoaD...; cd00757;ATP binding site [chemical binding];substrate interface [chemical binding];Rhodanese Homology Domain (RHOD); an alpha beta fold domain found duplicated in the rhodanese protein. The cysteine containing enzymatically active version of the domain is also found in the Cdc25 class of protein phosphatases and a variety of proteins...; cd00158;active site residue [active]
GOHSU05	CDS	GOHSU_05_00240	complement(30364..31275)		conserved hypothetical protein [nite]			RER_22010 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	lipoprotein		H0QUY2_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_008_00420 PE=4 SV=1;lipoprotein [Micromonospora sp. L5];KEGG: amd:AMED_1034 lipoprotein;Protein of unknown function (DUF3152); pfam11350
GOHSU05	CDS	GOHSU_05_00250	31476..32189		putative TetR family transcriptional regulator [nite]			RER_22000 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR-family transcriptional regulator		H5TQI2_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia otitidis NBRC 100426 GN=GOOTI_182_001 10 PE=4 SV=1;TetR-family transcriptional regulator [Streptomyces clavuligerus ATCC 27064];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU05	CDS	GOHSU_05_00260	complement(32267..32500)		hypothetical protein [nite]			ROP_64190 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YJK3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10199 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3107); pfam11305
GOHSU05	CDS	GOHSU_05_00270	complement(32532..33191)		conserved hypothetical protein [nite]			RER_21980 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJK2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10194 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cd00657;dinuclear metal binding motif [ion binding]
GOHSU05	CDS	GOHSU_05_00280	33378..>34770	3x codon	putative ATP-dependent RNA helicase [nite]			ROP_64170 :putative ATP-dependent RNA helicase : : 3.6.1.- : Rhodococcus opacus B4	putative cold-shock DEAD-box protein		F1YJK1_9ACTO DEAD/DEAH box helicase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 10189 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];DEAD-box helicases. A diverse family of proteins involved in ATP-dependent RNA unwinding, needed in a variety of cellular processes including splicing, ribosome biogenesis and RNA degradation. The name derives from the sequence of the Walker  B motif; cd00268;DEAD-like helicases superfamily; smart00487;ATP binding site [chemical binding];Mg++ binding site [ion binding];motif III;Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding]
##GOHSU06	contig00031				92799
GOHSU06	CDS	GOHSU_06_00010	complement(<1..939)	codon	hypothetical protein [migap]				hypothetical protein		G7GQH2_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_45_00450 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GJ17774 gene product from transcript GJ17774- RA
GOHSU06	CDS	GOHSU_06_00020	complement(936..1241)		hypothetical protein [nite]			ROP_68860 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5U594_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_117_00250 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU06	CDS	GOHSU_06_00030	complement(1344..1679)		hypothetical protein [migap]				hypothetical protein		F1YJF5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09959 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];YbaB/EbfC DNA-binding family; cl00494
GOHSU06	CDS	GOHSU_06_00040	complement(1688..2080)		hypothetical protein [migap]				hypothetical protein		H5U592_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_117_00230 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU06	CDS	GOHSU_06_00050	2302..3771		hypothetical protein [nite]			ROP_17050 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F9VZP2_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_097_01090 PE=4 SV= 1;unnamed protein product [Mycobacterium tuberculosis KZN 4207];drug-susceptible;ParA and ParB of Caulobacter crescentus belong to a conserved family of bacterial proteins implicated in chromosome segregation. ParB binds to DNA sequences adjacent to the origin of replication and localizes to opposite cell poles shortly following the...; cd02042;AAA domain; pfam13614;P-loop;Magnesium ion binding site [ion binding];Type 1 glutamine amidotransferase (GATase1)-like domain; cl00020
GOHSU06	CDS	GOHSU_06_00060	3833..4948		putative hydrolase [nite]			ROP_67890 :putative hydrolase : : : Rhodococcus opacus B4	alpha/beta hydrolase		H6N0W3_GORPV Putative hydrolase, alpha/beta fold fa mily OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH 2) GN=GPOL_c35140 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: alpha/beta hydrolase fold;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697
GOHSU06	CDS	GOHSU_06_00070	complement(5010..5849)		tRNA pseudouridine synthase A [hamap]	truA [hamap]	MF_00171:tRNA pseudouridine synthase A:truA 	ROP_62170 :tRNA pseudouridine synthase TruA : truA : 5.4.99.12 : Rhodococcus opacus B4	tRNA pseudouridine synthase A	truA	F1YJD7_9ACTO Pseudouridine synthase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_09869 PE=3 SV=1;tRNA pseudouridine synthase A [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;mediates pseudouridylation (positions 38, 39, 40) at the tRNA anticodon region which contributes to the structural stability;tRNA pseudouridine synthase A; Validated; PRK00021;PseudoU_synth_EcTruA: Pseudouridine synthase, Escherichia coli TruA like. This group consists of eukaryotic and bacterial pseudouridine synthases similar to E.  coli TruA, Pseudomonas aeruginosa truA and human pseudouridine synthase-like 1 (PUSL1); cd02570;dimerization interface 3.5A [polypeptide binding]
GOHSU06	CDS	GOHSU_06_00080	complement(5974..6570)	case	50S ribosomal protein L17 [hamap]	rplQ [hamap]	MF_01368:50S ribosomal protein L17:rplQ 	RER_18820 :50S ribosomal protein L17 : rplQ : : Rhodococcus erythropolis PR4	50S ribosomal protein L17	rplQ	H5U585_9ACTO 50S ribosomal protein L17 OS=Gordonia sputi NBRC 100414 GN=rplQ PE=3 SV=1;50S ribosomal protein L17 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;is a component of the macrolide binding site in the peptidyl transferase center;Ribosomal protein L17; cl00356
GOHSU06	CDS	GOHSU_06_00090	complement(6631..7683)	case	DNA-directed RNA polymerase subunit alpha [hamap]	rpoA [hamap]	MF_00059:DNA-directed RNA polymerase subunit alpha:rpoA 	ROP_62150 :DNA-directed RNA polymerase alpha subunit : rpoA : 2.7.7.6 : Rhodococcus opacus B4	DNA-directed RNA polymerase subunit alpha	rpoA	H0R4B0_9ACTO DNA-directed RNA polymerase subunit al pha OS=Gordonia effusa NBRC 100432 GN=rpoA PE=3 SV=1;DNA-directed RNA polymerase subunit alpha [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;DNA-directed RNA polymerase subunit alpha; Provisional;catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates. Dimerization of the alpha subunit is the first step in the sequential assembly of subunits to form the holoenzyme;N-terminal domain of the Alpha subunit of Bacterial RNA polymerase; cd06928;alphaNTD - beta interaction site [polypeptide binding];alphaNTD homodimer interface [polypeptide binding];alphaNTD - beta' interaction site [polypeptide binding];Bacterial RNA polymerase, alpha chain C terminal domain; cl11613
GOHSU06	CDS	GOHSU_06_00100	complement(7762..8367)		30S ribosomal protein S4 [nite]	 rpsD [nite]		RER_18800 :30S ribosomal protein S4 : rpsD : : Rhodococcus erythropolis PR4	30S ribosomal protein S4	rpsD	F1YJD4_9ACTO 30S ribosomal protein S4 OS=Gordonia n eofelifaecis NRRL B-59395 GN=rpsD PE=3 SV=1;30S ribosomal protein S4 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;30S ribosomal protein S4; Validated; PRK05327;primary rRNA binding protein; nucleates 30S assembly; involved in translational accuracy with proteins S5 and S12; interacts with protein S5; involved in autogeneously regulating ribosomal proteins by binding to pseudoknot structures in the polycistronic mRNA; interacts with transcription complex and functions similar to protein NusA in antitermination;Ribosomal protein S4/S9 N-terminal domain; pfam00163;S4/Hsp/ tRNA synthetase RNA-binding domain; The domain surface is populated by conserved, charged residues that define a likely RNA-binding site;  Found in stress proteins, ribosomal proteins and tRNA synthetases; This may imply a hitherto unrecognized...; cd00165;RNA binding surface [nucleotide binding]
GOHSU06	CDS	GOHSU_06_00110	complement(8392..8799)		30S ribosomal protein S11 [nite]	 rpsK [nite]		RER_18790 :30S ribosomal protein S11 : rpsK : : Rhodococcus erythropolis PR4	30S ribosomal protein S11		F1YJD3_9ACTO 30S ribosomal protein S11 OS=Gordonia neofelifaecis NRRL B-59395 GN=rpsK PE=3 SV=1;30S ribosomal protein S11 [Streptomyces clavuligeru s ATCC 27064];Ribosomal protein S11; cl00332
GOHSU06	CDS	GOHSU_06_00120	complement(8799..9167)		30S ribosomal protein S13 [nite]	 rpsM [nite]		ROP_62120 :30S ribosomal protein S13 : rpsM : : Rhodococcus opacus B4	30S ribosomal protein S13		F1YJD2_9ACTO 30S ribosomal protein S13 OS=Gordonia neofelifaecis NRRL B-59395 GN=rpsM PE=3 SV=1;30S ribosomal protein S13 [Streptomyces clavuligeru s ATCC 27064];Ribosomal protein S13/S18; cl00331;Ribosomal protein S13 [Translation, ribosomal structure and biogenesis]; COG0099
GOHSU06	CDS	GOHSU_06_00130	complement(9541..9762)	case	translation initiation factor IF-1 [hamap]	infA [hamap]	MF_00075:translation initiation factor IF-1:infA 	RER_18760 :translation initiation factor IF-1 : infA : : Rhodococcus erythropolis PR4	translation initiation factor IF-1	infA	A5WT35_MYCTF Translation initiation factor IF-1 OS= Mycobacterium tuberculosis (strain F11) GN=infA PE=3 SV=1;translation initiation factor IF-1 [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;stimulates the activities of the other two initiation factors, IF-2 and IF-3;S1_IF1: Translation Initiation Factor IF1, S1-like RNA-binding domain. IF1 contains an S1-like RNA-binding domain, which is found in a wide variety of RNA-associated proteins. Translation initiation includes a number of interrelated steps preceding the...; cd04451;rRNA binding site [nucleotide binding];predicted 30S ribosome binding site
GOHSU06	CDS	GOHSU_06_00140	9886..10914		not assigned						
GOHSU06	CDS	GOHSU_06_00150	10954..11361		hypothetical protein [nite]			MLP_02990 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YJC9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09829 PE=4 SV=1;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;Membrane protein of unknown function; cl00850;PFAM: membrane protein of unknown function; KEGG: mjl:Mjls_4717 membrane protein of unknown function
GOHSU06	CDS	GOHSU_06_00160	11394..12275	case	protein FdhD [hamap]	fdhD [hamap]	MF_00187:protein FdhD:fdhD 	ROP_23090 :putative FdhD protein homolog : : : Rhodococcus opacus B4	formate dehydrogenase accessory protein	fdhD	D0L2G4_GORB4 Protein FdhD homolog OS=Gordonia bronc hialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10 667) GN=fdhD PE=3 SV=1;fdhD gene product [Mycobacterium smegmatis str. MC2 155];FdhD/NarQ family; cl00659;involved in the production or activity of formate dehydrogenase-H which is active when nitrate is not present during anaerobic growth
GOHSU06	CDS	GOHSU_06_00170	12275..12724		D-tyrosyl-tRNA(Tyr) deacylase [hamap]	dtd [hamap]	MF_00518:D-tyrosyl-tRNA(Tyr) deacylase:dtd 	KSE_43730 :putative D-tyrosyl-tRNA(Tyr) deacylase : dtd : : Kitasatospora setae NBRC 14216	D-tyrosyl-tRNA(Tyr) deacylase		H5U574_9ACTO D-tyrosyl-tRNA(Tyr) deacylase OS=Gordo nia sputi NBRC 100414 GN=dtd PE=3 SV=1;D-tyrosyl-tRNA(Tyr) deacylase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;D-Tyrosyl-tRNAtyr deacylases; a class of tRNA-dependent hydrolases which are capable of hydrolyzing the ester bond of D-Tyrosyl-tRNA reducing the level of cellular D-Tyrosine while recycling the peptidyl-tRNA; found in bacteria and in eukaryotes but not...; cd00563;hydrolyzes D-tyrosyl-tRNA(Tyr) into D-tyrosine and free tRNA(Tyr); possible defense mechanism against a harmful effect of D-tyrosine;putative active site [active];dimerization interface [polypeptide binding];putative tRNAtyr binding site [nucleotide binding]
GOHSU06	CDS	GOHSU_06_00180	complement(12741..13541)		probable methionine aminopeptidase [nite]	 map [nite]		RER_18750 :probable methionine aminopeptidase : map : 3.4.11.18 : Rhodococcus erythropolis PR4	methionine aminopeptidase	map1	C0ZW48_RHOE4 Methionine aminopeptidase OS=Rhodococc us erythropolis (strain PR4 / NBRC 100887) GN=map PE=3 SV= 1;methionine aminopeptidase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;catalyzes the removal of N-terminal amino acids from peptides and arylamides; generally Co(II) however activity has been shown for some methionine aminopeptidases with Zn, Fe, or Mn;Methionine Aminopeptidase 1. E.C. 3.4.11.18. Also known as methionyl aminopeptidase and Peptidase M. Catalyzes release of N-terminal amino acids, preferentially methionine, from peptides and arylamides; cd01086
GOHSU06	CDS	GOHSU_06_00190	complement(13549..14097)		adenylate kinase [hamap]	adk [hamap]	MF_00235:adenylate kinase:adk 	ROP_62070 :adenylate kinase : adk : 2.7.4.3 : Rhodococcus opacus B4	adenylate kinase	adk	F1YJC2_9ACTO Adenylate kinase OS=Gordonia neofelifa ecis NRRL B-59395 GN=adk PE=3 SV=1;adenylate kinase [Rhodococcus opacus B4];adenylate kinase; Reviewed; PRK00279;essential enzyme that recycles AMP in active cells; converts ATP and AMP to two molecules of ADP;Adenylate kinase (ADK) catalyzes the reversible phosphoryl transfer from adenosine triphosphates (ATP) to adenosine monophosphates (AMP) and to yield adenosine diphosphates (ADP). This enzyme is required for the biosynthesis of ADP and is essential for...; cd01428;AMP-binding site [chemical binding];ATP-AMP (Ap5A)-binding site [chemical binding]
GOHSU06	CDS	GOHSU_06_00200	complement(14094..15428)		preprotein translocase SecY subunit [nite]	 secY [nite]		RER_18730 :preprotein translocase SecY subunit : secY : : Rhodococcus erythropolis PR4	preprotein translocase subunit SecY	secY	F1YJC1_9ACTO Protein translocase subunit SecY OS=Go rdonia neofelifaecis NRRL B-59395 GN=secY PE=3 SV=1;preprotein translocase subunit SecY [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;preprotein translocase subunit SecY; Reviewed; PRK09204;forms heterotrimeric complex in the membrane; in bacteria the complex consists of SecY which forms the channel pore and SecE and SecG; the SecG subunit is not essential; in bacteria translocation is driven via the SecA ATPase;SecY translocase; pfam00344
GOHSU06	CDS	GOHSU_06_00210	complement(15671..16114)		50S ribosomal protein L15 [nite]	 rplO [nite]		ROP_62050 :50S ribosomal protein L15 : rplO : : Rhodococcus opacus B4	50S ribosomal protein L15		F9VZR9_9ACTO 50S ribosomal protein L15 OS=Gordonia alkanivorans NBRC 16433 GN=rplO PE=3 SV=1;50S ribosomal protein L15 [Streptomyces clavuligeru s ATCC 27064];Ribosomal protein L18e/L15; cl12022
GOHSU06	CDS	GOHSU_06_00220	complement(16111..16296)		50S ribosomal protein L30 [nite]	 rpmD [nite]		ROP_62040 :50S ribosomal protein L30 : rpmD : : Rhodococcus opacus B4	50S ribosomal protein L30		F1YJB9_9ACTO 50S ribosomal protein L30 OS=Gordonia neofelifaecis NRRL B-59395 GN=rpmD PE=3 SV=1;50S ribosomal protein L30 [Streptomyces clavuligeru s ATCC 27064];Ribosomal protein L30, which is found in eukaryotes and prokaryotes but not in archaea, is one of the smallest ribosomal proteins with a molecular mass of about 7kDa. L30 binds the 23SrRNA as well as the 5S rRNA and is one of five ribosomal proteins that...; cd01658;23S rRNA binding site [nucleotide binding]
GOHSU06	CDS	GOHSU_06_00230	complement(16300..16965)		30S ribosomal protein S5 [nite]	 rpsE [nite]		ROP_62030 :30S ribosomal protein S5 : rpsE : : Rhodococcus opacus B4	30S ribosomal protein S5	rpsE	D0L2H0_GORB4 30S ribosomal protein S5 OS=Gordonia b ronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCT C 10667) GN=rpsE PE=3 SV=1;30S ribosomal protein S5 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;located at the back of the 30S subunit body where it stabilizes the conformation of the head with respect to the body; contacts S4 and S8; with S4 and S12 plays a role in translational accuracy; mutations in this gene result in spectinomycin resistance;30S ribosomal protein S5; Validated; PRK00550;Ribosomal protein S5, N-terminal domain; pfam00333;Ribosomal protein S5, C-terminal domain; pfam03719
GOHSU06	CDS	GOHSU_06_00240	complement(17012..17410)		50S ribosomal protein L18 [nite]	 rplR [nite]		ROP_62020 :50S ribosomal protein L18 : rplR : : Rhodococcus opacus B4	50S ribosomal protein L18	rplR	H5ULR7_9ACTO 50S ribosomal protein L18 OS=Gordonia terrae NBRC 100016 GN=rplR PE=3 SV=1;50S ribosomal protein L18 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;binds 5S rRNA along with protein L5 and L25;Ribosomal L18/L5e:  L18 (L5e) is a ribosomal protein found in the central protuberance (CP) of the large subunit. L18 binds 5S rRNA and induces a conformational change that stimulates the binding of L5 to 5S rRNA. Association of 5S rRNA with 23S rRNA...; cd00432;5S rRNA interface [nucleotide binding];L27 interface [polypeptide binding];23S rRNA interface [nucleotide binding];L5 interface [polypeptide binding]
GOHSU06	CDS	GOHSU_06_00250	complement(17414..17950)		50S ribosomal protein L6 [nite]	 rplF [nite]		ROP_62010 :50S ribosomal protein L6 : rplF : : Rhodococcus opacus B4	50S ribosomal protein L6	rplF	F9VZS3_9ACTO 50S ribosomal protein L6 OS=Gordonia a lkanivorans NBRC 16433 GN=rplF PE=3 SV=1;50S ribosomal protein L6 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;50S ribosomal protein L6; Validated; PRK05498;ribosomal protein L6 appears to have arisen as a result of an ancient gene duplication as based on structural comparison of the Bacillus stearothermophilus protein; RNA-binding appears to be in the C-terminal domain; mutations in the L6 gene confer resistance to aminoglycoside antibiotics such as gentamicin and these occur in truncations of the C-terminal domain; it has been localized to a region between the base of the L7/L12 stalk and the central protuberance;Ribosomal protein L6; pfam00347
GOHSU06	CDS	GOHSU_06_00260	complement(17966..18364)		30S ribosomal protein S8 [nite]	 rpsH [nite]		RER_18670 :30S ribosomal protein S8 : rpsH : : Rhodococcus erythropolis PR4	30S ribosomal protein S8	rpsH	F1YJB5_9ACTO 30S ribosomal protein S8 OS=Gordonia n eofelifaecis NRRL B-59395 GN=rpsH PE=3 SV=1;30S ribosomal protein S8 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;binds directly to 16S rRNA central domain where it helps coordinate assembly of the platform of the 30S subunit;Ribosomal protein S8; cl00330
GOHSU06	CDS	GOHSU_06_00270	18752..19852		hypothetical protein [nite]			MLP_46990 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		D0L2J0_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_3709 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU06	CDS	GOHSU_06_00280	19968..20723		hypothetical protein [nite]			RER_55620 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GM75_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_20_02500 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];Evidence 4 : Homologs of previously reported genes of unknown function;Predicted membrane protein (DUF2127); cl01785
GOHSU06	CDS	GOHSU_06_00290	complement(20889..21920)		hypothetical membrane protein [nite]			ROP_19650 :hypothetical membrane protein : : : Rhodococcus opacus B4	C4-dicarboxylate transporter/malic acid transport protein		H5U606_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_122_00190 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];PFAM: C4-dicarboxylate transporter/malic acid transport protein; KEGG: rfr:Rfer_4140 C4-dicarboxylate transporter/malic acid transport protein;The Tellurite-resistance/Dicarboxylate Transporter (TDT) family; cd09319;gating phenylalanine in ion channel
GOHSU06	CDS	GOHSU_06_00300	22045..25770		nitrate reductase alpha subunit [nite]	 narG [nite]		MLP_46640 :nitrate reductase alpha subunit : narG : 1.7.99.4 : Microlunatus phosphovorus NM-1	nitrate reductase subunit alpha		G7GYL4_9ACTO Nitrate reductase alpha subunit OS=Gor donia araii NBRC 100433 GN=narG PE=4 SV=1;nitrate reductase subunit alpha [Cellulomonas fimi ATCC 484];TIGRFAM: nitrate reductase, alpha subunit; KEGG: art:Arth_0314 respiratory nitrate reductase alpha subunit apoprotein; PFAM: molybdopterin oxidoreductase; molydopterin dinucleotide-binding region;Nitrate reductase alpha subunit [Energy production and conversion]; COG5013;Respiratory nitrate reductase A (NarGHI), alpha chain (NarG) and related proteins. Under anaerobic conditions in the presence of nitrate, E. coli synthesizes the cytoplasmic membrane-bound quinol-nitrate oxidoreductase (NarGHI), which reduces nitrate to...; cd02750;[4Fe-4S] binding site [ion binding];Molybdopterin-Binding (MopB) domain of the MopB superfamily of proteins, a  large, diverse, heterogeneous superfamily of enzymes that, in general, bind molybdopterin as a cofactor. The MopB domain is found in a wide variety of molybdenum- and...; cl09928;Respiratory nitrate reductase A (NarGHI), alpha chain (NarG) and related proteins. Under anaerobic conditions in the presence of nitrate, E. coli synthesizes the cytoplasmic membrane-bound quinol-nitrate oxidoreductase (NarGHI), which reduces nitrate to...; cd02776;molybdopterin cofactor binding site
GOHSU06	CDS	GOHSU_06_00310	25770..27425		nitrate reductase beta subunit [nite]	 narH [nite]		MLP_46650 :nitrate reductase beta subunit : narH : 1.7.99.4 : Microlunatus phosphovorus NM-1	nitrate reductase subunit beta		G7GYL5_9ACTO Nitrate reductase beta subunit OS=Gord onia araii NBRC 100433 GN=narH PE=4 SV=1;nitrate reductase subunit beta [Cellulomonas fimi A TCC 484];Nitrate reductase beta subunit [Energy production and conversion]; COG1140;TIGRFAM: nitrate reductase, beta subunit; KEGG: art:Arth_0315 respiratory nitrate reductase beta subunit
GOHSU06	CDS	GOHSU_06_00320	27425..28135		nitrate reductase delta subunit [nite]	 narJ [nite]		MLP_46660 :nitrate reductase delta subunit : narJ : 1.7.99.4 : Microlunatus phosphovorus NM-1	nitrate reductase subunit delta protein		G7GYL6_9ACTO Nitrate reductase delta subunit OS=Gor donia araii NBRC 100433 GN=narJ PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: azc:AZC_1427 nitrate reductase subunit delta protein;Nitrate reductase delta subunit; cl00958
GOHSU06	CDS	GOHSU_06_00330	28143..28889		nitrate reductase gamma subunit [nite]	 narI [nite]		MLP_46670 :nitrate reductase gamma subunit : narI : 1.7.99.4 : Microlunatus phosphovorus NM-1	respiratory nitrate reductase subunit gamma		F1YJA8_9ACTO Respiratory nitrate reductase subunit gamma OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09724 PE=4 SV=1;respiratory nitrate reductase subunit gamma [Nakamu rella multipartita DSM 44233];KEGG: hch:HCH_04008 respiratory nitrate reductase, gamma subunit; TIGRFAM: respiratory nitrate reductase, subunit gamma; PFAM: nitrate reductase subunit gamma;Nitrate reductase gamma subunit; cl00959
GOHSU06	CDS	GOHSU_06_00340	28886..30118		nitrate/nitrite transporter [nite]			MLP_46680 :nitrate/nitrite transporter : : : Microlunatus phosphovorus NM-1	major facilitator superfamily protein		G7GYL8_9ACTO Nitrate/nitrite transporter OS=Gordoni a araii NBRC 100433 GN=GOARA_015_00310 PE=4 SV=1;major facilitator superfamily protein [Nakamurella multipartita DSM 44233];Nitrate/nitrite transporter [Inorganic ion transport and metabolism]; COG2223;PFAM: major facilitator superfamily MFS_1; KEGG: bvi:Bcep1808_5661 major facilitator transporter;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU06	CDS	GOHSU_06_00350	30123..30467		hypothetical protein [nite]			MLP_32730 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YJA6_9ACTO Cupin 2 conserved barrel domain-contai ning protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_09714 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Cupin domain; cl09118
GOHSU06	CDS	GOHSU_06_00360	complement(30413..30631)		not assigned						
GOHSU06	CDS	GOHSU_06_00370	complement(30645..32969)		probable pyruvate, water dikinase [nite]	 ppsA [nite]		RER_33970 :probable pyruvate, water dikinase : ppsA : 2.7.9.2 : Rhodococcus erythropolis PR4	pyruvate water dikinase	ppsA	H0JWK0_9NOCA Pyruvate water dikinase OS=Rhodococcus pyridinivorans AK37 GN=AK37_20564 PE=4 SV=1;pyruvate water dikinase [Rhodococcus equi 103S];phosphoenolpyruvate synthase; Validated;Pyruvate phosphate dikinase, PEP/pyruvate binding domain; pfam01326;PEP-utilising enzyme, mobile domain; cl01586;Pyruvate kinase (PK):  Large allosteric enzyme that regulates glycolysis through binding of the substrate, phosphoenolpyruvate, and one or more allosteric effectors. Like other allosteric enzymes, PK has a high substrate affinity R state and a low...; cl09155
GOHSU06	CDS	GOHSU_06_00380	complement(33096..33818)		membrane protein [migap]				membrane protein		C0WGN1_9CORY Putative uncharacterized protein OS=Co rynebacterium accolens ATCC 49725 GN=HMPREF0276_0569 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];Evidence 3 : Function proposed based on presence of conserved amino acid motif, structural feature or limited homology
GOHSU06	CDS	GOHSU_06_00390	complement(33822..34460)		putative cholesterol esterase [nite]	 che1 [nite]		KSE_50590 :putative cholesterol esterase : che1 : 3.1.1.13 : Kitasatospora setae NBRC 14216	cholesterol esterase		G0HCN0_CORVD Putative secreted protein OS=Corynebac terium variabile (strain DSM 44702 / JCM 12073 / NCIMB 301 31) GN=CVAR_3059 PE=4 SV=1;cholesterol esterase [Saccharopolyspora erythraea N RRL 2338]
GOHSU06	CDS	GOHSU_06_00400	34749..34985		hypothetical protein [nite]			MLP_17170 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		G7GRY9_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_51_00200 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU06	CDS	GOHSU_06_00410	35054..35605		hypothetical protein [nite]			ROP_04130 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		C1AYN7_RHOOB Putative uncharacterized protein OS=Rh odococcus opacus (strain B4) GN=ROP_15680 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU06	CDS	GOHSU_06_00420	35669..35842		hypothetical protein [migap]				hypothetical protein		D5UVK9_TSUPD Putative uncharacterized protein OS=Ts ukamurella paurometabola (strain ATCC 8368 / DSM 20162 / J CM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_3204 PE=4 SV=1 ;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola
GOHSU06	CDS	GOHSU_06_00430	complement(35950..36135)		30S ribosomal protein S14 type Z [nite]	 rpsZ [nite]		RER_18650 :30S ribosomal protein S14 type Z : rpsZ : : Rhodococcus erythropolis PR4	30S ribosomal protein S14		G7GNG9_9ACTO 30S ribosomal protein S14 type Z OS=Go rdonia amarae NBRC 15530 GN=rpsZ PE=3 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Ribosomal protein S14p/S29e;Ribosomal protein S14p/S29e; cl00355
GOHSU06	CDS	GOHSU_06_00440	complement(36137..36715)		50S ribosomal protein L5 [nite]	 rplE [nite]		ROP_61980 :50S ribosomal protein L5 : rplE : : Rhodococcus opacus B4	50S ribosomal protein L5	rplE	E4WKV4_RHOE1 50S ribosomal protein L5 OS=Rhodococcu s equi (strain 103S) GN=rplE PE=3 SV=1;50S ribosomal protein L5 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;part of 50S and 5S/L5/L18/L25 subcomplex; contacts 5S rRNA and P site tRNA; forms a bridge to the 30S subunit in the ribosome by binding to S13;50S ribosomal protein L5; Validated; PRK00010;Ribosomal protein L5; pfam00281;ribosomal L5P family C-terminus; pfam00673
GOHSU06	CDS	GOHSU_06_00450	complement(36718..37035)		50S ribosomal protein L24 [nite]	 rplX [nite]		RER_18630 :50S ribosomal protein L24 : rplX : : Rhodococcus erythropolis PR4	50S ribosomal protein L24	rplX	H6N121_GORPV 50S ribosomal protein L24 OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=rplX PE=3 S V=1;50S ribosomal protein L24 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;50S ribosomal protein L24; Reviewed; PRK00004;KOW motif; cl00354;assembly initiator protein; binds to 5' end of 23S rRNA and nucleates assembly of the 50S; surrounds polypeptide exit tunnel
GOHSU06	CDS	GOHSU_06_00460	complement(37037..37405)		50S ribosomal protein L14 [nite]	 rplN [nite]		RER_18620 :50S ribosomal protein L14 : rplN : : Rhodococcus erythropolis PR4	50S ribosomal protein L14	rplN	F1YJ88_9ACTO 50S ribosomal protein L14 OS=Gordonia neofelifaecis NRRL B-59395 GN=rplN PE=3 SV=1;50S ribosomal protein L14 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosomal protein L14p/L23e; cl00328;binds to the 23S rRNA between the centers for peptidyl transferase and GTPase
GOHSU06	CDS	GOHSU_06_00470	complement(37687..38763)		metal-dependent hydrolase [migap]				metal-dependent hydrolase		Q5YVK9_NOCFA Putative uncharacterized protein OS=No cardia farcinica (strain IFM 10152) GN=NFA_29350 PE=4 SV=1 ;metal-dependent hydrolase [Thermofilum pendens Hrk 5];type strain of Thermofilum pendens Hrk 5;catalyzes the opening and hydrolysis of the beta-lactam ring of beta-lactam antibiotics such as penicillins and cephalosporins;Metallo-beta-lactamase superfamily; cl00446
GOHSU06	CDS	GOHSU_06_00480	complement(38878..39246)		LuxR family transcriptional regulator [migap]				LuxR family transcriptional regulator		D0L2K3_GORB4 Regulatory protein LuxR OS=Gordonia br onchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3722 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: LuxR family transcriptional regulator; SMART: LuxR family transcriptional regulator; KEGG: rlt:Rleg2_5436 LuxR family transcriptional regulator;C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding];dimerization interface [polypeptide binding]
GOHSU06	CDS	GOHSU_06_00490	complement(39557..40594)		hypothetical membrane protein [nite]			RER_04270 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJ87_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09619 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU06	CDS	GOHSU_06_00500	complement(40591..41826)		putative two-component histidine kinase [nite]			RER_04280 :putative two-component histidine kinase : : 2.7.13.- : Rhodococcus erythropolis PR4	ATP-binding protein		H5UD81_9ACTO Putative two-component histidine kinas e OS=Gordonia terrae NBRC 100016 GN=GOTRE_049_00270 PE=4 S V=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ATP-binding region ATPase domain protein; KEGG: rpa:RPA1490 bacteriophytochrome PhyB;Histidine kinase-like ATPase domain; pfam13581;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU06	CDS	GOHSU_06_00510	complement(41839..42528)		putative NarL family two-component response regulator [nite]			RER_04300 :putative NarL family two-component response regulator : : : Rhodococcus erythropolis PR4	two component LuxR family transcriptional regulator		F1YJ84_9ACTO Response regulator receiver OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_09604 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];PFAM: regulatory protein LuxR; response regulator receiver; KEGG: fal:FRAAL4532 putative two component response regulator;Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197;Helix-turn-helix domains; cl00088
GOHSU06	CDS	GOHSU_06_00520	42759..43289		hypothetical protein [migap]				hypothetical protein		F9VZW1_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_097_01780 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.625 at residue 25;KEGG: smt:Smal_2957 hypothetical protein
GOHSU06	CDS	GOHSU_06_00530	complement(43405..44526)		hypothetical membrane protein [nite]			RER_04270 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		C3JP00_RHOER Putative uncharacterized protein OS=Rh odococcus erythropolis SK121 GN=RHOER0001_3724 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU06	CDS	GOHSU_06_00540	complement(44523..45677)		hypothetical membrane protein [nite]			KRH_03490 :hypothetical membrane protein : : : Kocuria rhizophila DC2201	ATP-binding protein		F1YJ80_9ACTO ATP-binding region ATPase domain-conta ining protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SC NU_09584 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ATP-binding region ATPase domain protein; KEGG: rpa:RPA1490 bacteriophytochrome PhyB;Histidine kinase-like ATPase domain; pfam13581;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU06	CDS	GOHSU_06_00550	complement(45825..46130)		hypothetical protein [migap]				hypothetical protein		H5U636_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_122_00490 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];acyl-CoA synthetase; Validated
GOHSU06	CDS	GOHSU_06_00560	complement(46383..46616)		conserved hypothetical protein [nite]			RER_09800 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	MbtH domain-containing protein		H0R425_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_096_00030 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];MbtH-like protein; cl01279;PFAM: MbtH domain protein; KEGG: bbt:BBta_4111 hypothetical protein
GOHSU06	CDS	GOHSU_06_00570	complement(46616..57133)		non-ribosomal peptide synthetase [nite]			ROP_02580 :non-ribosomal peptide synthetase : : : Rhodococcus opacus B4	non-ribosomal peptide synthetase		F9VZW6_9ACTO Putative non-ribosomal peptide synthet ase OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_097_01830 PE=4 SV=1;non-ribosomal peptide synthetase [Rhodococcus josti i RHA1];peptide synthase; Provisional;AMP-binding enzyme; cl15778;Phosphopantetheine attachment site; cl09936;non-ribosomal peptide synthase domain TIGR01720;HxxPF-repeated domain; pfam13745
GOHSU06	CDS	GOHSU_06_00580	complement(57311..72331)		putative non-ribosomal peptide synthetase [nite]			ROP_33190 :putative non-ribosomal peptide synthetase : : : Rhodococcus opacus B4	non-ribosomal peptide synthetase		H5RY15_9NOCA Non-ribosomal peptide synthetase OS=No cardia brasiliensis ATCC 700358 GN=O3I_23869 PE=4 SV=1;non-ribosomal peptide synthetase [Nocardia farcinic a IFM 10152];Phosphopantetheine attachment site; cl09936;non-ribosomal peptide synthase domain TIGR01720;HxxPF-repeated domain; pfam13745;AMP-binding enzyme; cl15778;peptide synthase; Provisional
GOHSU06	CDS	GOHSU_06_00590	75081..75437		not assigned						C3JH46_RHOER Antigenic cell wall protein MP2 (Fragm ent) OS=Rhodococcus erythropolis SK121 GN=RHOER0001_5899 P E=4 SV=1
GOHSU06	CDS	GOHSU_06_00600	76883..77863		not assigned						Q3JSH2_BURP1 Putative uncharacterized protein OS=Bu rkholderia pseudomallei (strain 1710b) GN=BURPS1710b_2087 PE=4 SV=1
GOHSU06	CDS	GOHSU_06_00610	79359..79697		not assigned						Q172Z2_AEDAE Putative uncharacterized protein (Frag ment) OS=Aedes aegypti GN=AAEL007242 PE=4 SV=1
GOHSU06	CDS	GOHSU_06_00620	80810..81298		not assigned						
GOHSU06	CDS	GOHSU_06_00630	81335..81799		not assigned						
GOHSU06	CDS	GOHSU_06_00640	81932..82510		not assigned						E3A2M5_PSEAI Linear gramicidin synthetase subunit D OS=Pseudomonas aeruginosa 39016 GN=PA39016_002620002 PE=4 SV=1
GOHSU06	CDS	GOHSU_06_00650	83453..84121		not assigned						G2U6V5_PSEAI Probable non-ribosomal peptide synthet ase OS=Pseudomonas aeruginosa NCMG1179 GN=NCGM1179_2820 PE =4 SV=1
GOHSU06	CDS	GOHSU_06_00660	84417..84629		not assigned						
GOHSU06	CDS	GOHSU_06_00670	85337..86293		not assigned						
GOHSU06	CDS	GOHSU_06_00680	86321..87502		not assigned						H0BIZ8_9ACTO Linear gramicidin synthetase subunit D OS=Streptomyces sp. W007 GN=SPW_5236 PE=4 SV=1
GOHSU06	CDS	GOHSU_06_00690	87674..89020		hypothetical protein [migap]				hypothetical protein		Q08UN2_STIAD Putative uncharacterized protein (Frag ment) OS=Stigmatella aurantiaca (strain DW4/3-1) GN=STIAU_ 8750 PE=4 SV=1;hypothetical protein BURPS1710b_2089 [Burkholderia pseudomallei 1710b]
GOHSU06	CDS	GOHSU_06_00700	90227..90787		not assigned						Q3JSH2_BURP1 Putative uncharacterized protein OS=Bu rkholderia pseudomallei (strain 1710b) GN=BURPS1710b_2087 PE=4 SV=1
GOHSU06	CDS	GOHSU_06_00710	92343..92708		not assigned						C3JH46_RHOER Antigenic cell wall protein MP2 (Fragm ent) OS=Rhodococcus erythropolis SK121 GN=RHOER0001_5899 P E=4 SV=1
##GOHSU07	contig00032				31614
GOHSU07	CDS	GOHSU_07_00010	complement(<1..3764)	3x codon	putative non-ribosomal peptide synthetase [nite]			RER_18370 :putative non-ribosomal peptide synthetase : : : Rhodococcus erythropolis PR4	non-ribosomal peptide synthetase		F9VZW6_9ACTO Putative non-ribosomal peptide synthet ase OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_097_01830 PE=4 SV=1;non-ribosomal peptide synthetase [Rhodococcus josti i RHA1];D-alanine--poly(phosphoribitol) ligase subunit 1; Provisional;AMP-binding enzyme; cl15778;Phosphopantetheine attachment site; cl09936;peptide synthase; Provisional;HxxPF-repeated domain; pfam13745
GOHSU07	CDS	GOHSU_07_00020	complement(3761..11872)		putative non-ribosomal peptide synthetase [nite]			RER_18370 :putative non-ribosomal peptide synthetase : : : Rhodococcus erythropolis PR4	non-ribosomal peptide synthetase		H5RY15_9NOCA Non-ribosomal peptide synthetase OS=No cardia brasiliensis ATCC 700358 GN=O3I_23869 PE=4 SV=1;non-ribosomal peptide synthetase [Rhodococcus josti i RHA1];enterobactin synthase subunit F; Provisional; PRK10252;AMP-binding enzyme; cl15778;Phosphopantetheine attachment site; cl09936;peptide synthase; Provisional;HxxPF-repeated domain; pfam13745
GOHSU07	CDS	GOHSU_07_00030	complement(12125..12967)		putative ABC transporter permease protein [nite]			RER_18420 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	ABC transporter		F1YPI1_9ACTO ABC-2 type transporter OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_19180 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ABC transporter; KEGG: aeh:Mlg_2716 ABC-2 type transporter;ABC-2 type transporter; cl11417
GOHSU07	CDS	GOHSU_07_00040	complement(12964..13800)		putative ABC transporter permease protein [nite]			ROP_61820 :putative ABC transporter permease protein : : : Rhodococcus opacus B4	ABC transporter		H0RFF4_9ACTO Putative ABC transporter permease prot ein OS=Gordonia polyisoprenivorans NBRC 16320 GN=GOPIP_057 _01360 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ABC transporter; KEGG: mes:Meso_2100 ABC-2 type transporter;ABC-2 type transporter; cl11417
GOHSU07	CDS	GOHSU_07_00050	complement(13797..15359)		ABC transporter ATPase [migap]				ABC transporter ATPase		F1YPI3_9ACTO Daunorubicin resistance ABC transporte r ATPase subunit OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_19190 PE=3 SV=1;ABC transporter ATPase [Rhodococcus equi 103S];P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;daunorubicin resistance ABC transporter ATP-binding subunit; TIGR01188;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU07	CDS	GOHSU_07_00060	complement(15572..16009)		conserved hypothetical protein [nite]			RER_45260 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative polyketide cyclase/dehydrase		H5U7B7_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_141_00230 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];START/RHO_alpha_C/PITP/Bet_v1/CoxG/CalC (SRPBCC) ligand-binding domain superfamily; cl14643
GOHSU07	CDS	GOHSU_07_00070	complement(16122..16403)		30S ribosomal protein S17 [nite]	 rpsQ [nite]		ROP_61950 :30S ribosomal protein S17 : rpsQ : : Rhodococcus opacus B4	30S ribosomal protein S17		F1YPI4_9ACTO 30S ribosomal protein S17 OS=Gordonia neofelifaecis NRRL B-59395 GN=rpsQ PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: 30S ribosomal protein S17; PFAM: ribosomal protein S17; KEGG: afw:Anae109_1920 ribosomal protein S17;Ribosomal protein S17; cl00351
GOHSU07	CDS	GOHSU_07_00080	complement(16400..16633)		50S ribosomal protein L29 [nite]	 rpmC [nite]		RER_18600 :50S ribosomal protein L29 : rpmC : : Rhodococcus erythropolis PR4	50S ribosomal protein L29	rpmC	H5TG17_9ACTO 50S ribosomal protein L29 OS=Gordonia otitidis NBRC 100426 GN=rpmC PE=3 SV=1;50S ribosomal protein L29 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;one of the stabilizing components for the large ribosomal subunit;Ribosomal L29 protein/HIP.  L29 is a protein of the large ribosomal Subunit. A homolog, called heparin/heparan sulfate interacting protein (HIP), has also been identified in mammals.  L29 is located on the surface of the large ribosomal subunit, where it...; cd00427;putative translocon interaction site;23S rRNA interface [nucleotide binding];signal recognition particle (SRP54) interaction site;L23 interface [polypeptide binding];trigger factor interaction site
GOHSU07	CDS	GOHSU_07_00090	complement(16633..17049)	case	50S ribosomal protein L16 [hamap]	rplP [hamap]	MF_01342:50S ribosomal protein L16:rplP 	RER_18590 :50S ribosomal protein L16 : rplP : : Rhodococcus erythropolis PR4	50S ribosomal protein L16		H6N1U5_GORPV 50S ribosomal protein L16 OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=rplP PE=3 S V=1;50S ribosomal protein L16 [Streptomyces clavuligeru s ATCC 27064];Ribosomal_L16_L10e: L16 is an essential protein in the large ribosomal subunit of bacteria, mitochondria, and chloroplasts. Large subunits that lack L16 are defective in peptidyl transferase activity, peptidyl-tRNA hydrolysis activity, association with...; cd01433;23S rRNA interface [nucleotide binding];5S rRNA interface [nucleotide binding];putative antibiotic binding site [chemical binding];L25 interface [polypeptide binding];L27 interface [polypeptide binding]
GOHSU07	CDS	GOHSU_07_00100	complement(17053..17898)		30S ribosomal protein S3 [nite]	 rpsC [nite]		RER_18580 :30S ribosomal protein S3 : rpsC : : Rhodococcus erythropolis PR4	30S ribosomal protein S3		F5YYL3_MYCSD 30S ribosomal protein S3 OS=Mycobacter ium sp. (strain JDM601) GN=rpsC PE=3 SV=1;30S ribosomal protein S3 [Streptomyces clavuligerus ATCC 27064];ribosomal protein S3, bacterial type; TIGR01009;K homology RNA-binding (KH) domain of the prokaryotic 30S small ribosomal subunit protein S3. S3  is part of the head region of the 30S ribosomal subunit and is believed to interact with mRNA as it threads its way from the latch into the channel.  The KH...; cd02412;G-X-X-G motif;Ribosomal protein S3, C-terminal domain; pfam00189
GOHSU07	CDS	GOHSU_07_00110	complement(17902..18312)		50S ribosomal protein L22 [nite]	 rplV [nite]		RER_18570 :50S ribosomal protein L22 : rplV : : Rhodococcus erythropolis PR4	50S ribosomal protein L22		H5TXW2_9ACTO 50S ribosomal protein L22 OS=Gordonia sputi NBRC 100414 GN=rplV PE=3 SV=1;50S ribosomal protein L22 [Streptomyces clavuligeru s ATCC 27064];Ribosomal protein L22/L17e.  L22 (L17 in eukaryotes) is a core protein of the large ribosomal subunit.  It is the only ribosomal protein that interacts with all six domains of 23S rRNA, and is one of the proteins important for directing the proper...; cd00336;putative translocon binding site;protein-rRNA interface [nucleotide binding]
GOHSU07	CDS	GOHSU_07_00120	complement(18315..18596)		30S ribosomal protein S19 [nite]	 rpsS [nite]		RER_18560 :30S ribosomal protein S19 : rpsS : : Rhodococcus erythropolis PR4	30S ribosomal protein S19	rpsS	H5UD87_9ACTO 30S ribosomal protein S19 OS=Gordonia terrae NBRC 100016 GN=rpsS PE=3 SV=1;30S ribosomal protein S19 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosomal protein S19; cl00350;protein S19 forms a complex with S13 that binds strongly to the 16S ribosomal RNA
GOHSU07	CDS	GOHSU_07_00130	complement(18614..19450)		50S ribosomal protein L2 [nite]	 rplB [nite]		RER_18550 :50S ribosomal protein L2 : rplB : : Rhodococcus erythropolis PR4	50S ribosomal protein L2	rplB	F1YPJ0_9ACTO 50S ribosomal protein L2 OS=Gordonia n eofelifaecis NRRL B-59395 GN=rplB PE=3 SV=1;50S ribosomal protein L2 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;50S ribosomal protein L2; Validated; PRK09374;one of the primary rRNA-binding proteins; required for association of the 30S and 50S subunits to form the 70S ribosome, for tRNA binding and peptide bond formation;Ribosomal Proteins L2, RNA binding domain; pfam00181;Ribosomal Proteins L2, C-terminal domain; pfam03947
GOHSU07	CDS	GOHSU_07_00140	complement(19483..19788)		50S ribosomal protein L23 [nite]	 rplW [nite]		RER_18540 :50S ribosomal protein L23 : rplW : : Rhodococcus erythropolis PR4	50S ribosomal protein L23	rplW	F1YPJ1_9ACTO 50S ribosomal protein L23 OS=Gordonia neofelifaecis NRRL B-59395 GN=rplW PE=3 SV=1;50S ribosomal protein L23 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;binds third domain of 23S rRNA and protein L29; part of exit tunnel;Ribosomal protein L23; cl00326
GOHSU07	CDS	GOHSU_07_00150	complement(19790..20449)		50S ribosomal protein L4 [nite]	 rplD [nite]		ROP_61870 :50S ribosomal protein L4 : rplD : : Rhodococcus opacus B4	50S ribosomal protein L4	rplD	F1YPJ2_9ACTO 50S ribosomal protein L4 OS=Gordonia n eofelifaecis NRRL B-59395 GN=rplD PE=3 SV=1;50S ribosomal protein L4 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;L4 is important during the early stages of 50S assembly; it initially binds near the 5' end of the 23S rRNA;Ribosomal protein L4/L1 family; cl00325
GOHSU07	CDS	GOHSU_07_00160	complement(20446..21102)		50S ribosomal protein L3 [nite]	 rplC [nite]		ROP_61860 :50S ribosomal protein L3 : rplC : : Rhodococcus opacus B4	50S ribosomal protein L3		F1YPJ3_9ACTO 50S ribosomal protein L3 OS=Gordonia n eofelifaecis NRRL B-59395 GN=rplC PE=3 SV=1;50S ribosomal protein L3 [Streptomyces clavuligerus ATCC 27064];Ribosomal protein L3; cl00324
GOHSU07	CDS	GOHSU_07_00170	complement(21117..21422)	case	30S ribosomal protein S10 [hamap]	rpsJ [hamap]	MF_00508:30S ribosomal protein S10:rpsJ 	RER_18510 :30S ribosomal protein S10 : rpsJ : : Rhodococcus erythropolis PR4	30S ribosomal protein S10	rpsJ	F8E2C4_CORRG 30S ribosomal protein S10 OS=Corynebac terium resistens (strain DSM 45100 / JCM 12819 / GTC 2026) GN=rpsJ PE=3 SV=1;30S ribosomal protein S10 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosomal protein S10p/S20e; cl00314;NusE; involved in assembly of the 30S subunit; in the ribosome, this protein is involved in the binding of tRNA; in Escherichia coli this protein was also found to be involved in transcription antitermination; NusB/S10 heterodimers bind boxA sequences in the leader RNA of rrn operons which is required for antitermination; binding of NusB/S10 to boxA nucleates assembly of the antitermination complex
GOHSU07	CDS	GOHSU_07_00180	21916..23559		putative acid--CoA ligase [nite]			ROP_64530 :putative acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	acyl-CoA synthetase		H5UCZ7_9ACTO Putative fatty-acid--CoA ligase OS=Gor donia terrae NBRC 100016 GN=GOTRE_044_01090 PE=4 SV=1;acyl-CoA synthetase [Nocardia farcinica IFM 10152];activates fatty acids by binding to coenzyme A;acyl-CoA synthetase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU07	CDS	GOHSU_07_00190	23916..24899		hypothetical protein [nite]			RER_16860 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7H4D3_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_061_01480 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF4095); pfam13338
GOHSU07	CDS	GOHSU_07_00200	25183..25935		putative succinate dehydrogenase cytochrome b subunit [nite]			RER_55300 :putative succinate dehydrogenase cytochrome b subunit : : 1.3.99.1 : Rhodococcus erythropolis PR4	fumarate reductase, cytochrome B subunit		H5TG07_9ACTO Succinate dehydrogenase cytochrome b s ubunit OS=Gordonia otitidis NBRC 100426 GN=sdhC PE=4 SV=1;fumarate reductase, cytochrome B subunit [Actinomyc es viscosus C505];Succinate:quinone oxidoreductase (SQR)-like Type B subfamily 2, transmembrane subunit; composed of proteins with similarity to the SQRs of Geobacter metallireducens and Corynebacterium glutamicum. SQR catalyzes the oxidation of succinate to fumarate...; cd03498;putative Iron-sulfur protein interface [polypeptide binding];proximal heme binding site [chemical binding];distal heme binding site [chemical binding];putative dimer interface [polypeptide binding]
GOHSU07	CDS	GOHSU_07_00210	25996..27933		putative succinate dehydrogenase flavoprotein subunit [nite]			RER_55310 :putative succinate dehydrogenase flavoprotein subunit : : 1.3.99.1 : Rhodococcus erythropolis PR4	succinate dehydrogenase flavoprotein subunit	sdhA	H6N1V6_GORPV Succinate dehydrogenase, flavoprotein subunit SdhA OS=Gordonia polyisoprenivorans (strain DSM 44 266 / VH2) GN=sdhA2 PE=4 SV=1;succinate dehydrogenase flavoprotein subunit [Coryn ebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;part of four member succinate dehydrogenase enzyme complex that forms a trimeric complex (trimer of tetramers); SdhA/B are the catalytic subcomplex and can exhibit succinate dehydrogenase activity in the absence of SdhC/D which are the membrane components and form cytochrome b556; SdhC binds ubiquinone; oxidizes succinate to fumarate while reducing ubiquinone to ubiquinol;succinate dehydrogenase flavoprotein subunit; Reviewed; PRK07573;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Fumarate reductase flavoprotein C-term; pfam02910
GOHSU07	CDS	GOHSU_07_00220	27948..28694		putative succinate dehydrogenase iron-sulfur protein [nite]			RER_55320 :putative succinate dehydrogenase iron-sulfur protein : : 1.3.99.1 : Rhodococcus erythropolis PR4	succinate dehydrogenase/fumarate reductase iron-sulfur subunit	sdhB	H6N1V7_GORPV Succinate dehydrogenase, iron-sulfur s ubunit SdhB OS=Gordonia polyisoprenivorans (strain DSM 442 66 / VH2) GN=sdhB2 PE=4 SV=1;succinate dehydrogenase/fumarate reductase iron-sul fur subunit [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;succinate dehydrogenase/fumarate reductase iron-sulfur subunit; Validated;catalyzes the reversible formation of fumarate and ubiquinol from succinate and ubiquinone;2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cl00159
GOHSU07	CDS	GOHSU_07_00230	28713..29189		hypothetical protein [nite]			ROP_61840 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5UD93_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_049_00390 PE=4 SV=1;unnamed protein product [Pseudomonas aeruginosa LES B58];PaaI_thioesterase is a tetrameric acyl-CoA thioesterase with a hot dog fold and one of several proteins responsible for phenylacetic acid (PA) degradation in bacteria.  Although orthologs of PaaI exist in archaea and eukaryotes, their function has not...; cd03443;CoenzymeA binding site [chemical binding];subunit interaction site [polypeptide binding];PHB binding site
GOHSU07	CDS	GOHSU_07_00240	29335..30102		conserved hypothetical protein [nite]			RER_23310 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	DSBA oxidoreductase		F1YPJ8_9ACTO DSBA oxidoreductase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_19265 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: DSBA oxidoreductase; KEGG: aav:Aave_0501 hypothetical protein;Protein Disulfide Oxidoreductases and Other Proteins with a Thioredoxin fold; cl00388;DSBA-like thioredoxin domain; pfam01323
GOHSU07	CDS	GOHSU_07_00250	30099..30842		hypothetical membrane protein [nite]			ROP_pROB01-02870 :hypothetical membrane protein : : : Rhodococcus opacus B4	vitamin K epoxide reductase		F1YPJ9_9ACTO Vitamin K epoxide reductase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_19270 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: Vitamin K epoxide reductase; SMART: Vitamin K epoxide reductase; KEGG: scl:sce9005 hypothetical protein;Vitamin K epoxide reductase family; cl01729
GOHSU07	CDS	GOHSU_07_00260	complement(30854..31612)		glucose-methanol-choline oxidoreductase [migap]				glucose-methanol-choline oxidoreductase		F1YQ29_9ACTO Glucose-methanol-choline oxidoreductas e OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_20206 PE= 4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: glucose-methanol-choline oxidoreductase; FAD dependent oxidoreductase; GMC oxidoreductase; KEGG: bur:Bcep18194_A4914 glucose-methanol-choline oxidoreductase;dehydrogenase, Rv0697 family; TIGR03970;GMC oxidoreductase; pfam05199
##GOHSU08	contig00033				136259
GOHSU08	CDS	GOHSU_08_00010	complement(<1..553)	3x codon	putative oxidoreductase [nite]			YM304_23760 :putative oxidoreductase : : : Acidimicrobidae bacterium YM16-304	dehydrogenase		F1YQ29_9ACTO Glucose-methanol-choline oxidoreductas e OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_20206 PE= 4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: GMC oxidoreductase; TIGRFAM: dehydrogenase, Rv0697 family; manually curated;dehydrogenase, Rv0697 family; TIGR03970
GOHSU08	CDS	GOHSU_08_00020	complement(550..2034)		conserved hypothetical protein [nite]			RER_18070 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	mycofactocin system glycosyltransferase		H5UDA5_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_049_00510 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Glycosyl transferase family 2; TIGRFAM: mycofactocin system glycosyltransferase;mycofactocin system glycosyltransferase; TIGR03965;Subfamily of Glycosyltransferase Family GT2 of unknown function; cd04186;Probable Catalytic site
GOHSU08	CDS	GOHSU_08_00030	complement(2031..2756)		creatininase family protein [nite]			ROP_61350 :creatininase family protein : : : Rhodococcus opacus B4	creatinine amidohydrolase family protein, mycofactocin system		Q0S3N2_RHOSR Putative uncharacterized protein OS=Rh odococcus sp. (strain RHA1) GN=RHA1_ro06077 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Creatinine amidohydrolase; TIGRFAM: creatinine amidohydrolase family protein, mycofactocin system;Creatinine amidohydrolase; cl00618
GOHSU08	CDS	GOHSU_08_00040	2939..4051		probable alpha-methylacyl-CoA racemase [nite]	 mcr [nite]		RER_14330 :probable alpha-methylacyl-CoA racemase : mcr : 5.1.99.4 : Rhodococcus erythropolis PR4	alpha-methylacyl-CoA racemase	mcr	G7GXT1_9ACTO Alpha-methylacyl-CoA racemase OS=Gordo nia araii NBRC 100433 GN=mcr PE=4 SV=1;alpha-methylacyl-CoA racemase [Rhodococcus opacus B 4];Predicted acyl-CoA transferases/carnitine dehydratase [Energy production and conversion]; COG1804;CoA-transferase family III; pfam02515
GOHSU08	CDS	GOHSU_08_00050	4211..5011		oxidoreductase [nite]			ROP_61330 :oxidoreductase : : : Rhodococcus opacus B4	short chain dehydrogenase		F1YNU5_9ACTO 3-ketoacyl-(Acyl-carrier-protein) redu ctase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_17972 PE=3 SV=1;short chain dehydrogenase [Rhodococcus equi 103S];3-ketoacyl-(acyl-carrier-protein) reductase; Reviewed;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU08	CDS	GOHSU_08_00060	complement(5170..6081)		hypothetical protein [migap]				hypothetical protein		F1YFG1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02812 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Domain of unknown function (DUF4095); pfam13338
GOHSU08	CDS	GOHSU_08_00070	complement(6253..7470)		putative oxidoreductase [nite]			RER_18000 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	heme/flavin dehydrogenase, mycofactocin system		H0R393_9ACTO Putative oxidoreductase OS=Gordonia ef fusa NBRC 100432 GN=GOEFS_091_00360 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: FMN-dependent dehydrogenase; TIGRFAM: heme/flavin dehydrogenase, mycofactocin system;heme/flavin dehydrogenase, mycofactocin system; TIGR03966;TIM barrel proteins share a structurally conserved phosphate binding motif and in general share an eight beta/alpha closed barrel structure. Specific for this family is the conserved phosphate binding site at the edges of strands 7 and 8. The phosphate...; cl09108;phosphate binding site [ion binding]
GOHSU08	CDS	GOHSU_08_00080	complement(7546..8850)		conserved hypothetical protein [nite]			RER_17990 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	radical SAM peptide maturase, mycofactocin system		F1YNV9_9ACTO Radical SAM domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18042 PE=3 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Radical SAM superfamily; TIGRFAM: radical SAM peptide maturase, mycofactocin system; radical SAM additional 4Fe4S-binding domain;mycofactocin radical SAM maturase; TIGR03962;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site;Domain of unknown function (DUF4008); cl14869
GOHSU08	CDS	GOHSU_08_00090	complement(8855..9178)		hypothetical protein [nite]			ROP_61290 :hypothetical protein : : : Rhodococcus opacus B4	mycofactocin system RPExFGAL protein		F1YNW0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18047 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];TIGRFAM: mycofactocin system RPExFGAL protein;mycofactocin system RPExFGAL protein; TIGR03967
GOHSU08	CDS	GOHSU_08_00100	9360..9977		putative TetR family transcriptional regulator [nite]			ROP_61280 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		F1YNW2_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_18057 PE=4 SV=1;TetR family transcriptional regulator [Pseudonocard ia dioxanivorans CB1190];PFAM: Transcription regulator, TetR-like, DNA-binding, bacterial/archaeal; KEGG: nml:Namu_2350 TetR family transcriptional regulator;mycofactocin system transcriptional regulator; TIGR03968;Helix-turn-helix domains; cl00088
GOHSU08	CDS	GOHSU_08_00110	9974..11032		3-ketosteroid 9alpha-hydroxylase component KshB [nite]	 kshB [nite]		RER_17750 :3-ketosteroid 9alpha-hydroxylase component KshB : kshB : : Rhodococcus erythropolis PR4	putative ferredoxin oxidoreductase		G7GXR6_9ACTO 3-ketosteroid 9alpha-hydroxylase compo nent KshB OS=Gordonia araii NBRC 100433 GN=kshB PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];phenylacetate-CoA oxygenase/reductase, PaaK subunit; TIGR02160;NAD(P) binding domain of ferredoxin reductase like phenylacetic acid (PA) degradation oxidoreductase. PA oxidoreductases of E. coli hydroxylate PA-CoA in the second step of PA degradation. Members of this group typically fuse a ferredoxin reductase-like...; cd06214;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding];beta-alpha-beta structure motif;NAD(p) ribose binding residues [chemical binding];NAD binding pocket [chemical binding];NAD(P)-pyrophosphate-nicotinamide binding residues [chemical binding];2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cd00207;catalytic loop [active];iron binding site [ion binding]
GOHSU08	CDS	GOHSU_08_00120	11035..12246		ferredoxin reductase [nite]			ROP_61090 :ferredoxin reductase : : : Rhodococcus opacus B4	ferredoxin reductase		D0L334_GORB4 FAD-dependent pyridine nucleotide-disu lphide oxidoreductase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3785 P E=4 SV=1;ferredoxin reductase [Rhodococcus opacus B4];Pyridine nucleotide-disulphide oxidoreductase; pfam07992;Pyridine nucleotide-disulphide oxidoreductase; pfam00070
GOHSU08	CDS	GOHSU_08_00130	12348..12665		hypothetical protein [nite]			SLG_23440 :hypothetical protein : : : Sphingobium sp. SYK-6	helix-turn-helix domain-containing protein		F1YLK4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13979 PE=4 SV=1;helix-turn-helix domain-containing protein [Brevund imonas diminuta ATCC 11568];[K] COG1396 Predicted transcriptional regulators
GOHSU08	CDS	GOHSU_08_00140	complement(12699..13088)		hypothetical membrane protein [nite]			ROP_64390 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YNW4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18067 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Predicted membrane protein [Function unknown]
GOHSU08	CDS	GOHSU_08_00150	complement(13657..14163)		hypothetical protein [nite]			ROP_15960 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YNW7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18082 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU08	CDS	GOHSU_08_00160	complement(14172..14774)		hypothetical protein [migap]				hypothetical protein		F1YNW8_9ACTO Putative membrane glycoprotein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_18087 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU08	CDS	GOHSU_08_00170	complement(14771..15346)		hypothetical protein [migap]				hypothetical protein		F1YNW9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18092 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU08	CDS	GOHSU_08_00180	complement(15343..15936)		hypothetical membrane protein [nite]			ROP_15970 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YNX0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18097 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU08	CDS	GOHSU_08_00190	complement(15967..16833)		putative oxidoreductase [nite]			RER_55960 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	putative oxidoreductase		H0R379_9ACTO Putative oxidoreductase OS=Gordonia ef fusa NBRC 100432 GN=GOEFS_091_00220 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU08	CDS	GOHSU_08_00200	complement(16873..17463)		putative TetR family transcriptional regulator [nite]			ROP_57830 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		H0R378_9ACTO Transcriptional regulator KstR OS=Gord onia effusa NBRC 100432 GN=kstR PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m sp. JLS];PFAM: regulatory protein, TetR; KEGG: mmc:Mmcs_0746 transcriptional regulator, TetR family;Helix-turn-helix domains; cl00088
GOHSU08	CDS	GOHSU_08_00210	complement(17460..18803)		putative cytochrome P450 [nite]			ROP_52600 :putative cytochrome P450 : : : Rhodococcus opacus B4	cytochrome P450		F1YNX3_9ACTO Cytochrome P450 OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_18112 PE=3 SV=1;cytochrome P450 [Mycobacterium sp. KMS];PFAM: cytochrome P450; KEGG: mmc:Mmcs_3188 cytochrome P450;Cytochrome P450 [Secondary metabolites biosynthesis, transport, and catabolism]; cl12078;Cytochrome P450; pfam00067
GOHSU08	CDS	GOHSU_08_00220	complement(18908..19492)		not assigned						A3Q0A9_MYCSJ Putative uncharacterized protein (Prec ursor) OS=Mycobacterium sp. (strain JLS) GN=Mjls_2806 PE=4 SV=1
GOHSU08	CDS	GOHSU_08_00230	complement(19596..20633)		probable o-succinylbenzoate synthase [nite]	 menC [nite]		RER_17000 :probable o-succinylbenzoate synthase : menC : 4.2.1.- : Rhodococcus erythropolis PR4	O-succinylbenzoate synthase	menC	F1YNX4_9ACTO O-succinylbenzoate synthase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_18117 PE=4 SV=1;O-succinylbenzoate synthase [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;O-succinylbenzoate synthase; Provisional;catalyzes the dehydration of 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylic acid to form O-succinylbenzoate;o-Succinylbenzoate synthase (OSBS) catalyzes the conversion of 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate (SHCHC) to 4-(2'-carboxyphenyl)-4-oxobutyrate (o-succinylbenzoate or OSB), a reaction in the menaquinone biosynthetic pathway; cd03320
GOHSU08	CDS	GOHSU_08_00240	20728..21639		hypothetical protein [nite]			ROP_23910 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GZB2_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_022_00130 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];cytoplasmic protein;Uncharacterized protein conserved in bacteria (DUF2236); cl01472
GOHSU08	CDS	GOHSU_08_00250	21733..22671		hypothetical protein [nite]			MLP_02110 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YIG5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08398 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF559);Domain of unknown function (DUF4095); pfam13338;Superfamily of nucleases including Short Patch Repair (Vsr) Endonucleases, archaeal Holliday junction resolvases, MutH methy-directed DNA mismatch-repair endonucleases, and catalytic domains of many restriction endonucleases, such as EcoRI, BamHI, and...; cl00277;putative active site [active]
GOHSU08	CDS	GOHSU_08_00260	complement(22756..23400)		putative TetR family transcriptional regulator [nite]			YM304_01210 :putative TetR family transcriptional regulator : : : Acidimicrobidae bacterium YM16-304	TetR family transcriptional regulator		G4REW8_PELHB Transcriptional regulator, TetR family OS=Pelagibacterium halotolerans (strain JCM 15775 / CGMCC 1.7692 / B2) GN=KKY_1929 PE=4 SV=1;TetR family transcriptional regulator [Rhizobium le guminosarum bv. trifolii WSM1325];PFAM: regulatory protein TetR; KEGG: rec:RHECIAT_CH0002885 probable transcriptional regulator protein, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088;AefR-like transcriptional repressor, C-terminal region; pfam14246
GOHSU08	CDS	GOHSU_08_00270	complement(23503..24252)		putative hydrolase [nite]			RER_19560 :putative hydrolase : : : Rhodococcus erythropolis PR4	hydrolase		H0R374_9ACTO Putative hydrolase OS=Gordonia effusa NBRC 100432 GN=GOEFS_091_00170 PE=4 SV=1;hydrolase [Nocardia farcinica IFM 10152];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697
GOHSU08	CDS	GOHSU_08_00280	24303..25262		hypothetical protein [nite]			MLP_04100 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		H0R372_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_091_00150 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];Signal predicted by SignalP 2.0 HMM (Signal peptide probability 0.998) with cleavage site probability 0.979 at residue 31;PFAM: protein of unknown function DUF6, transmembrane; KEGG: mmc:Mmcs_1118 protein of unknown function DUF6, transmembrane
GOHSU08	CDS	GOHSU_08_00290	complement(25305..26558)		not assigned						
GOHSU08	CDS	GOHSU_08_00300	complement(26609..27226)		hypothetical protein [nite]			MLP_07500 :hypothetical protein : : : Microlunatus phosphovorus NM-1	von Willebrand factor A		D0L4S2_GORB4 von Willebrand factor type A (Precurso r) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_4136 PE=4 SV=1;von Willebrand factor A [Mycobacterium sp. JDM601];Von Willebrand factor type A (vWA) domain was originally found in the blood coagulation protein von Willebrand factor (vWF). Typically, the vWA domain is made up of approximately 200 amino acid residues folded into a classic a/b para-rossmann type of...; cl00057;metal ion-dependent adhesion site (MIDAS)
GOHSU08	CDS	GOHSU_08_00310	27366..27845		hypothetical protein [nite]			ROP_57150 :hypothetical protein : : : Rhodococcus opacus B4	fatty acid binding protein		F1YM03_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14736 PE=4 SV=1;fatty acid binding protein [Streptomyces sp. C];nitrobindin heme-binding domain; cd07828;heme-binding site [chemical binding]
GOHSU08	CDS	GOHSU_08_00320	complement(27983..29173)		elongation factor Tu [nite]	 tuf [nite]		RER_17640 :elongation factor Tu : tuf : 3.6.5.3 : Rhodococcus erythropolis PR4	elongation factor Tu	tuf	F9W025_9ACTO Elongation factor Tu OS=Gordonia alkan ivorans NBRC 16433 GN=tuf PE=3 SV=1;elongation factor Tu [Corynebacterium glutamicum AT CC 13032];IS fingerprint type 4-5;elongation factor Tu; Reviewed;EF-Tu; promotes GTP-dependent binding of aminoacyl-tRNA to the A-site of ribosomes during protein biosynthesis; when the tRNA anticodon matches the mRNA codon, GTP hydrolysis results; the inactive EF-Tu-GDP leaves the ribosome and release of GDP is promoted by elongation factor Ts; many prokaryotes have two copies of the gene encoding EF-Tu;Elongation Factor Tu (EF-Tu) GTP-binding proteins; cd01884;G1 box;GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;EFTU_II: Elongation factor Tu domain II. Elongation factors Tu (EF-Tu) are three-domain GTPases with an essential function in the elongation phase of mRNA translation. The GTPase center of EF-Tu is in the N-terminal domain (domain I), also known as the...; cd03697;Domain III of elongation factor (EF) Tu. Ef-Tu consists of three structural domains, designated I, II and III. Domain III adopts a beta barrel structure. Domain III is involved in binding to both charged tRNA and binding to elongation factor Ts (EF-Ts); cd03707;Antibiotic Binding Site [chemical binding]
GOHSU08	CDS	GOHSU_08_00330	complement(29285..31390)		elongation factor G [nite]	 fusA [nite]		ROP_16050 :elongation factor G : fusA : 3.6.5.3 : Rhodococcus opacus B4	elongation factor G	fusA	F1YM06_9ACTO Elongation factor G OS=Gordonia neofel ifaecis NRRL B-59395 GN=fusA PE=3 SV=1;elongation factor G [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;elongation factor G; Reviewed;EF-G; promotes GTP-dependent translocation of the ribosome during translation; many organisms have multiple copies of this gene;Elongation factor G (EF-G) family involved in both the elongation and ribosome recycling phases of protein synthesis; cd01886;G1 box;putative GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;EFG_mtEFG_II: this subfamily represents the domain II of elongation factor G (EF-G) in bacteria and, the C-terminus of mitochondrial Elongation factor G1 (mtEFG1) and G2 (mtEFG2)_like proteins found in eukaryotes. During the process of peptide synthesis...; cd04088;EFG_mtEFG1_IV: domains similar to domain IV of the bacterial translational elongation factor (EF) EF-G. Included in this group is a domain of mitochondrial Elongation factor G1 (mtEFG1) proteins homologous to domain IV of EF-G. Eukaryotic cells harbor 2...; cd01434;EFG_mtEFG_C: domains similar to the C-terminal domain of the bacterial translational elongation factor (EF) EF-G.  Included in this group is the C-terminus of mitochondrial Elongation factor G1 (mtEFG1) and G2 (mtEFG2) proteins. Eukaryotic cells harbor 2...; cd03713
GOHSU08	CDS	GOHSU_08_00340	complement(31496..31966)		30S ribosomal protein S7 [nite]	 rpsG [nite]		RER_17620 :30S ribosomal protein S7 : rpsG : : Rhodococcus erythropolis PR4	30S ribosomal protein S7		G7GWU6_9ACTO 30S ribosomal protein S7 OS=Gordonia a marae NBRC 15530 GN=rpsG PE=3 SV=1;30S ribosomal protein S7 [Streptomyces clavuligerus ATCC 27064];Ribosomal protein S7p/S5e; cl00313
GOHSU08	CDS	GOHSU_08_00350	complement(31972..32343)		30S ribosomal protein S12 [nite]	 rpsL [nite]		ROP_16070 :30S ribosomal protein S12 : rpsL : : Rhodococcus opacus B4	30S ribosomal protein S12	rpsL	G7GZB9_9ACTO 30S ribosomal protein S12 OS=Gordonia araii NBRC 100433 GN=rpsL PE=3 SV=1;30S ribosomal protein S12 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;interacts with and stabilizes bases of the 16S rRNA that are involved in tRNA selection in the A site and with the mRNA backbone; located at the interface of the 30S and 50S subunits, it traverses the body of the 30S subunit contacting proteins on the other side; mutations in the S12 gene confer streptomycin resistance;S12-like family, 30S ribosomal protein S12 subfamily; S12 is located at the interface of the large and small ribosomal subunits of prokaryotes, chloroplasts and mitochondria, where it plays an important role in both tRNA and ribosomal subunit...; cd03368;S17 interaction site [polypeptide binding];S8 interaction site;16S rRNA interaction site [nucleotide binding];streptomycin interaction site [chemical binding];23S rRNA interaction site [nucleotide binding];aminoacyl-tRNA interaction site (A-site) [nucleotide binding]
GOHSU08	CDS	GOHSU_08_00360	32810..33970		putative MFS transporter [nite]			ROP_64250 :putative MFS transporter : : : Rhodococcus opacus B4	MFS superfamily transporter		F1YM10_9ACTO Major facilitator superfamily MFS_1 OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14771 PE=4 SV =1;MFS superfamily transporter [Arthrobacter arilaiten sis Re117];major facilitator superfamily (TC 2.A.1.y.z). Identified by match to protein family PF07690 (Major Facilitator Superfamily);The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;Major Facilitator Superfamily; pfam07690;putative substrate translocation pore
GOHSU08	CDS	GOHSU_08_00370	34300..35916		putative transcriptional regulator [nite]			RER_18190 :putative transcriptional regulator : : : Rhodococcus erythropolis PR4	transcriptional regulator		F1YM25_9ACTO Helix-turn-helix domain-containing pro tein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14846 PE=4 SV=1;transcriptional regulator [Rhodococcus jostii RHA1] ;Transcriptional activator of acetoin/glycerol metabolism [Secondary metabolites biosynthesis, transport, and catabolism / Transcription]; COG3284;Helix-turn-helix domains; cl00088
GOHSU08	CDS	GOHSU_08_00380	36131..37627		aldehyde dehydrogenase [nite]			ROP_22240 :aldehyde dehydrogenase : : 1.2.1.3 : Rhodococcus opacus B4	aldehyde dehydrogenase		F1YM26_9ACTO Aldehyde dehydrogenase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_14851 PE=3 SV=1;aldehyde dehydrogenase [Rhodococcus jostii RHA1];NAD(P)+-dependent aldehyde dehydrogenase superfamily; cl11961;Aldehyde dehydrogenase family; pfam00171;NAD(P) binding site [chemical binding];catalytic residues [active]
GOHSU08	CDS	GOHSU_08_00390	37682..38800		putative zinc-containing alcohol dehydrogenase [nite]			RER_18210 :putative zinc-containing alcohol dehydrogenase : : 1.1.1.- : Rhodococcus erythropolis PR4	oxidoreductase, Rxyl_3153 family		F1YM27_9ACTO Zinc-containing alcohol dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14856 PE=3 SV=1;oxidoreductase, Rxyl_3153 family [Saccharomonospora marina XMU15];PFAM: Alcohol dehydrogenase GroES-like domain; Zinc-binding dehydrogenase; TIGRFAM: oxidoreductase, Rxyl_3153 family;oxidoreductase, Rxyl_3153 family; TIGR03989;Class III alcohol dehydrogenase; cd08279;NAD binding site [chemical binding];catalytic Zn binding site [ion binding];substrate binding site [chemical binding];structural Zn binding site [ion binding]
GOHSU08	CDS	GOHSU_08_00400	complement(38931..39815)		putative exodeoxyribonuclease III [nite]	 xthA [nite]		KSE_63990 :putative exodeoxyribonuclease III : xthA : 3.1.11.2 : Kitasatospora setae NBRC 14216	exodeoxyribonuclease III Xth		F1YM24_9ACTO Exodeoxyribonuclease III Xth OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_14841 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Escherichia coli exonuclease III (ExoIII) and Neisseria meningitides NExo-like subfamily of the ExoIII family purinic/apyrimidinic (AP) endonucleases; cd09086;TIGRFAM: exodeoxyribonuclease III Xth; exodeoxyribonuclease III; PFAM: endonuclease/exonuclease/phosphatase; KEGG: afr:AFE_0178 exodeoxyribonuclease III;putative phosphate binding site [ion binding];putative catalytic site [active];metal binding site A [ion binding];DNA binding site [nucleotide binding];putative AP binding site [nucleotide binding];putative metal binding site B [ion binding]
GOHSU08	CDS	GOHSU_08_00410	complement(39907..41457)		putative acyl-CoA carboxylase beta chain [nite]			YM304_22210 :putative acyl-CoA carboxylase beta chain : : 6.4.1.- : Acidimicrobidae bacterium YM16-304	pyruvate carboxylase., propionyl-CoA carboxylase		A1T167_MYCVP Propionyl-CoA carboxylase OS=Mycobacte rium vanbaalenii (strain DSM 7251 / PYR-1) GN=Mvan_0066 PE =4 SV=1;unnamed protein product [Ralstonia eutropha JMP134] ;Acetyl-CoA carboxylase, carboxyltransferase component (subunits alpha and beta) [Lipid metabolism];Acetyl co-enzyme A carboxylase carboxyltransferase alpha subunit; cl15772
GOHSU08	CDS	GOHSU_08_00420	complement(41529..42134)		putative TetR family transcriptional regulator [nite]			RER_17580 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	transcriptional regulator		F1YM32_9ACTO Tetr family transcriptional regulator OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14881 PE=4 SV=1;transcriptional regulator [Saccharomonospora marina XMU15];PFAM: Bacterial regulatory proteins, tetR family; manually curated;transcriptional repressor BetI; TIGR03384;Helix-turn-helix domains; cl00088
GOHSU08	CDS	GOHSU_08_00430	complement(42131..42931)		putative enoyl-CoA hydratase [nite]			RER_17570 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase		F1YM33_9ACTO Enoyl-CoA hydratase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_14886 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;enoyl-CoA hydratase; Provisional;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU08	CDS	GOHSU_08_00440	complement(42945..44102)		putative acyl-CoA dehydrogenase [nite]			RER_17560 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase		D0L375_GORB4 Acyl-CoA dehydrogenase domain protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / J CM 3198 / NCTC 10667) GN=Gbro_3827 PE=3 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Acyl-CoA dehydrogenase, C-terminal domain; Acyl-CoA dehydrogenase, middle domain; Acyl-CoA dehydrogenase, N-terminal domain;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cd00567
GOHSU08	CDS	GOHSU_08_00450	complement(44115..46181)		probable acyl-CoA carboxylase alpha chain [nite]	 accA [nite]		RER_17550 :probable acyl-CoA carboxylase alpha chain : accA : 6.4.1.- : Rhodococcus erythropolis PR4	carbamoyl-phosphate synthase ATP-binding subunit L		H5TFS2_9ACTO Acyl-CoA carboxylase alpha chain OS=Go rdonia otitidis NBRC 100426 GN=accA PE=4 SV=1;carbamoyl-phosphate synthase ATP-binding subunit L [Mycobacterium vanbaalenii PYR- 1];Acetyl/propionyl-CoA carboxylase, alpha subunit [Lipid metabolism];PFAM: biotin/lipoyl attachment domain-containing protein; Carbamoyl-phosphate synthase L chain, ATP-binding; Carbamoyl-phosphate synthetase large chain domain protein; biotin carboxylase domain protein; KEGG: mmc:Mmcs_4307 pyruvate carboxylase;Carbamoyl-phosphate synthase L chain, N-terminal domain; pfam00289;ATP-grasp domain; cl03087;Biotin carboxylase C-terminal domain; cl08365;The biotinyl-domain or biotin carboxyl carrier protein (BCCP) domain is present in all biotin-dependent enzymes, such as acetyl-CoA carboxylase, pyruvate carboxylase, propionyl-CoA carboxylase, methylcrotonyl-CoA carboxylase, geranyl-CoA carboxylase; cd06850;carboxyltransferase (CT) interaction site;biotinylation site [posttranslational modification]
GOHSU08	CDS	GOHSU_08_00460	complement(46189..47787)		acyl-CoA carboxylase beta chain [nite]	 accD [nite]		RER_17540 :acyl-CoA carboxylase beta chain : accD : 6.4.1.- : Rhodococcus erythropolis PR4	acetyl-CoA carboxylase carboxyltransferase		F1YM36_9ACTO Methylcrotonoyl-CoA carboxylase OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_14901 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF01039;3-methylcrotonyl-CoA carboxylase, beta chain;Acetyl co-enzyme A carboxylase carboxyltransferase alpha subunit; cl15772
GOHSU08	CDS	GOHSU_08_00470	complement(47784..48857)		acyl-CoA dehydrogenase [nite]	 fadE [nite]		ROP_16150 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		H0QX72_9ACTO Acyl-CoA dehydrogenase OS=Gordonia eff usa NBRC 100432 GN=fadE PE=3 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;PFAM: Acyl-CoA dehydrogenase, C-terminal domain; Acyl-CoA dehydrogenase, middle domain; Acyl-CoA dehydrogenase, N-terminal domain;Acyl-CoA dehydrogenase; cl09933
GOHSU08	CDS	GOHSU_08_00480	complement(48941..50638)		hypothetical protein [nite]			ROP_16160 :hypothetical protein : : : Rhodococcus opacus B4	Protein of unknown function (DUF1446)		H0QX71_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_032_00180 PE=4 SV=1;Protein of unknown function (DUF1446) [Saccharomono spora marina XMU15];PFAM: Protein of unknown function (DUF1446);Protein of unknown function (DUF1446); pfam07287
GOHSU08	CDS	GOHSU_08_00490	complement(50635..51459)		hypothetical protein [nite]			ROP_16170 :hypothetical protein : : : Rhodococcus opacus B4	TIGR03084 family protein		G7H3S4_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_057_00230 PE=4 SV=1;TIGR03084 family protein [Saccharomonospora marina XMU15];PFAM: Wyosine base formation; Mycothiol maleylpyruvate isomerase N-terminal domain; TIGRFAM: TIGR03084 family protein; uncharacterized Actinobacterial protein TIGR03083;TIGR03084 family protein;Mycothiol maleylpyruvate isomerase N-terminal domain; cl00986;Wyosine base formation; pfam08608
GOHSU08	CDS	GOHSU_08_00500	complement(51742..52623)		hypothetical protein [nite]			RER_49580 :hypothetical protein : : : Rhodococcus erythropolis PR4	putative thioesterase		F1YM40_9ACTO Thioesterase superfamily protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14921 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];PaaI_thioesterase is a tetrameric acyl-CoA thioesterase with a hot dog fold and one of several proteins responsible for phenylacetic acid (PA) degradation in bacteria.  Although orthologs of PaaI exist in archaea and eukaryotes, their function has not...; cd03443;CoenzymeA binding site [chemical binding];subunit interaction site [polypeptide binding];PHB binding site
GOHSU08	CDS	GOHSU_08_00510	52670..53335		hypothetical protein [nite]			RER_50990 :hypothetical protein : : : Rhodococcus erythropolis PR4	thioesterase superfamily protein		F1YM52_9ACTO Thioesterase superfamily protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14981 PE=4 SV=1;thioesterase superfamily protein [Mycobacterium gil vum PYR- GCK];PFAM: thioesterase superfamily protein; KEGG: mmc:Mmcs_2763 thioesterase superfamily;PaaI_thioesterase is a tetrameric acyl-CoA thioesterase with a hot dog fold and one of several proteins responsible for phenylacetic acid (PA) degradation in bacteria.  Although orthologs of PaaI exist in archaea and eukaryotes, their function has not...; cd03443;CoenzymeA binding site [chemical binding];subunit interaction site [polypeptide binding];PHB binding site
GOHSU08	CDS	GOHSU_08_00520	53389..54267		conserved hypothetical protein [nite]			RER_51760 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	amidohydrolase 2		F1YM57_9ACTO Amidohydrolase 2 OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_15006 PE=4 SV=1;amidohydrolase 2 [Streptomyces sp. AA4];Superfamily of metallo-dependent hydrolases (also called amidohydrolase superfamily) is a large group of proteins that show conservation in their 3-dimensional fold (TIM barrel) and in details of their active site. The vast majority of the members have a...; cd01292
GOHSU08	CDS	GOHSU_08_00530	complement(54329..54556)		response regulator receiver [migap]				response regulator receiver		F4G547_ALIDK Putative uncharacterized protein OS=Al icycliphilus denitrificans (strain DSM 14773 / CIP 107495 / K601) GN=Alide2_4074 PE=4 SV=1;response regulator receiver [Alicycliphilus denitri ficans BC];KEGG: dac:Daci_1875 response regulator receiver protein; PFAM: response regulator receiver; SMART: response regulator receiver;Hexameric tyrosine-coordinated heme protein (HTHP); pfam11534
GOHSU08	CDS	GOHSU_08_00540	complement(54685..55653)		putative oxidoreductase [nite]			RER_17420 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	NADPH:quinone reductase		H0QY61_9ACTO Putative oxidoreductase OS=Gordonia ef fusa NBRC 100432 GN=GOEFS_038_00290 PE=4 SV=1;NADPH:quinone reductase [Rhodococcus jostii RHA1];putative NAD(P)H quinone oxidoreductase, PIG3 family; TIGR02824;Quinone oxidoreductase (QOR); cd08241;NAD(P) binding site [chemical binding]
GOHSU08	CDS	GOHSU_08_00550	55812..56141		hypothetical protein [migap]				hypothetical protein		H5U6B0_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_125_00350 PE=4 SV=1;unnamed protein product [Propionibacterium acnes Ty peIA2 P.acn33]
GOHSU08	CDS	GOHSU_08_00560	56141..56455		not assigned						F1QSC3_DANRE Uncharacterized protein (Fragment) OS= Danio rerio GN=si:ch211-149m3.3 PE=4 SV=1
GOHSU08	CDS	GOHSU_08_00570	complement(56443..57318)		hypothetical protein [nite]			ROP_47530 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GZM0_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_027_00080 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00296;probable F420-dependent oxidoreductase, Rv2161c family; TIGR03619;Flavin-utilizing monoxygenases; cl07892
GOHSU08	CDS	GOHSU_08_00580	complement(57329..58996)		probable ferredoxin--NADP(+) reductase [nite]			RER_49700 :probable ferredoxin--NADP(+) reductase : : 1.18.1.2 : Rhodococcus erythropolis PR4	ferredoxin--NADP(+) reductase		H5TGT7_9ACTO Putative ferredoxin--NADP(+) reductase OS=Gordonia otitidis NBRC 100426 GN=GOOTI_023_00390 PE=4 SV=1;ferredoxin--NADP(+) reductase [Rhodococcus erythrop olis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Ferredoxin [Energy production and conversion];4Fe-4S binding domain; cl02805;ferredoxin-NADP+ reductase
GOHSU08	CDS	GOHSU_08_00590	59093..59950		hypothetical protein [nite]			ROP_16330 :hypothetical protein : : : Rhodococcus opacus B4	acyl-ACP thioesterase		F1YM71_9ACTO Acyl-ACP thioesterase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_15076 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: acyl-ACP thioesterase; KEGG: nfa:nfa50880 hypothetical protein;Acyl-ACP thioesterase; pfam01643;The hotdog fold was initially identified in the E. coli FabA (beta-hydroxydecanoyl-acyl carrier protein (ACP)-dehydratase) structure and subsequently in 4HBT (4-hydroxybenzoyl-CoA thioesterase) from Pseudomonas. A number of other seemingly unrelated...; cl00509;active site 2 [active];active site 1 [active]
GOHSU08	CDS	GOHSU_08_00600	complement(59964..60509)		hypothetical protein [nite]			ROP_16080 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YM72_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15081 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.999) with cleavage site probability 0.716 at residue 32;Protein of unknown function (DUF3558); pfam12079
GOHSU08	CDS	GOHSU_08_00610	complement(60518..61399)		RNA-binding S4 domain-containing protein [migap]				RNA-binding S4 domain-containing protein		D0L3C9_GORB4 RNA-binding S4 domain protein (Precurs or) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3883 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: RNA-binding S4 domain protein; SMART: RNA-binding S4 domain protein; KEGG: mei:Msip34_1728 RNA-binding S4 domain protein;S4/Hsp/ tRNA synthetase RNA-binding domain; The domain surface is populated by conserved, charged residues that define a likely RNA-binding site;  Found in stress proteins, ribosomal proteins and tRNA synthetases; This may imply a hitherto unrecognized...; cd00165;RNA binding surface [nucleotide binding];Protein of unknown function (DUF3558); pfam12079
GOHSU08	CDS	GOHSU_08_00620	complement(61559..61945)		hypothetical protein [nite]			ROP_41480 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YM74_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15091 PE=4 SV=1;integral membrane protein [Rhodococcus equi 103S];Phospholipase_D-nuclease N-terminal; pfam13396;Short C-terminal domain; cl01373
GOHSU08	CDS	GOHSU_08_00630	62259..62711		putative FMN-binding protein [nite]			ROP_14720 :putative FMN-binding protein : : : Rhodococcus opacus B4	flavodoxin/nitric oxide synthase		G7GZQ4_9ACTO Putative FMN-binding protein OS=Gordon ia araii NBRC 100433 GN=GOARA_027_00420 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: flavodoxin/nitric oxide synthase; KEGG: eca:ECA2072 flavodoxin;NADPH-dependent FMN reductase; cl00438
GOHSU08	CDS	GOHSU_08_00640	complement(62727..63641)		conserved hypothetical membrane protein [nite]			RER_30990 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0QY38_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_038_00060 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU08	CDS	GOHSU_08_00650	63722..64219		putative deaminase [nite]			RER_06470 :putative deaminase : : : Rhodococcus erythropolis PR4	cytidine/deoxycytidylate deaminase		D5NW49_CORAM Cytidine/deoxycytidylate deaminase, zi nc-binding region OS=Corynebacterium ammoniagenes DSM 2030 6 GN=HMPREF0281_00808 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG0590 Cytosine/adenosine deaminases;Nucleoside deaminases include adenosine, guanine and cytosine deaminases. These enzymes are Zn dependent and catalyze the deamination of nucleosides. The zinc ion in the active site plays a central role in the proposed catalytic mechanism, activating a...; cd01285;nucleoside/Zn binding site;dimer interface [polypeptide binding];catalytic motif [active]
GOHSU08	CDS	GOHSU_08_00660	64275..64601		hypothetical protein [migap]				hypothetical protein		F1YM78_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15111 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU08	CDS	GOHSU_08_00670	64598..65089		hypothetical protein [nite]			ROP_16360 :hypothetical protein : : : Rhodococcus opacus B4	phosphohistidine phosphatase SixA		H5TGR4_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_023_00160 PE=4 SV=1;phosphohistidine phosphatase SixA [Nakamurella mult ipartita DSM 44233];PFAM: phosphoglycerate mutase; KEGG: mrd:Mrad2831_1113  phosphohistidine phosphatase, SixA;Histidine phosphatase domain found in phosphoglycerate mutases and related proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cd07067;catalytic core [active]
GOHSU08	CDS	GOHSU_08_00680	complement(65219..67666)		hypothetical protein [migap]				hypothetical protein		G0HAS1_CORVD Putative secreted protein OS=Corynebac terium variabile (strain DSM 44702 / JCM 12073 / NCIMB 301 31) GN=CVAR_0149 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: hypothetical protein
GOHSU08	CDS	GOHSU_08_00690	complement(67909..71061)		hypothetical protein [migap]				hypothetical protein		G0HAS1_CORVD Putative secreted protein OS=Corynebac terium variabile (strain DSM 44702 / JCM 12073 / NCIMB 301 31) GN=CVAR_0149 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: hypothetical protein
GOHSU08	CDS	GOHSU_08_00700	71633..71920		esterase [migap]				esterase		G7GRX7_9ACTO Putative mycolyltransferase OS=Gordoni a amarae NBRC 15530 GN=GOAMR_51_00080 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Predicted esterase [General function prediction only]
GOHSU08	CDS	GOHSU_08_00710	71917..72192		putative mycolyltransferase [nite]			RER_14670 :putative mycolyltransferase : : 2.3.1.- : Rhodococcus erythropolis PR4	mycolyltransferase		E4W8Y5_RHOE1 Putative secreted esterase OS=Rhodococ cus equi (strain 103S) GN=REQ_39250 PE=4 SV=1;mycolyltransferase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Predicted esterase [General function prediction only]
GOHSU08	CDS	GOHSU_08_00720	72241..72603		putative mycolyltransferase [nite]			RER_14670 :putative mycolyltransferase : : 2.3.1.- : Rhodococcus erythropolis PR4	mycolyltransferase		H6MRT4_GORPV Esterase OS=Gordonia polyisoprenivoran s (strain DSM 44266 / VH2) GN=GPOL_c27190 PE=4 SV=1;mycolyltransferase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Predicted esterase [General function prediction only]
GOHSU08	CDS	GOHSU_08_00730	complement(72676..73692)		hypothetical protein [nite]			MLP_42990 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YI33_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07738 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF4095); pfam13338
GOHSU08	CDS	GOHSU_08_00740	complement(74128..78111)		DNA-directed RNA polymerase subunit beta' [hamap]	rpoC [hamap]	MF_01322:DNA-directed RNA polymerase subunit beta':rpoC 	ROP_16390 :DNA-directed RNA polymerase beta' subunit : rpoC : 2.7.7.6 : Rhodococcus opacus B4	DNA-directed RNA polymerase subunit beta'	rpoC	F1YII6_9ACTO DNA-directed RNA polymerase subunit be ta' OS=Gordonia neofelifaecis NRRL B-59395 GN=rpoC PE=3 SV =1;DNA-directed RNA polymerase subunit beta' [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates. Subunit beta' binds to sigma factor allowing it to bind to the -10 region of the promoter;RNA polymerase Rpb1, domain 1; pfam04997;ligand binding domain of Tar- and Tsr-related chemoreceptors; cl00144;RNA polymerase Rpb1, domain 2; cl11591;RNA polymerase Rpb1, domain 3; pfam04983;DNA-directed RNA polymerase subunit beta'; Provisional; PRK02597;RNA polymerase Rpb1, domain 4; pfam05000;Largest subunit (beta') of Bacterial DNA-dependent RNA polymerase (RNAP), C-terminal domain; cd02655;Rpb1 (beta') - Rpb6 (omega) interaction site [polypeptide binding];G-loop;DNA binding site [nucleotide binding];Rpb1 (beta') - Rpb2 (beta) interaction site [polypeptide binding]
GOHSU08	CDS	GOHSU_08_00750	complement(78177..81689)	case	DNA-directed RNA polymerase subunit beta [hamap]	rpoB [hamap]	MF_01321:DNA-directed RNA polymerase subunit beta:rpoB 	ROP_16400 :DNA-directed RNA polymerase beta subunit : rpoB : 2.7.7.6 : Rhodococcus opacus B4	DNA-directed RNA polymerase subunit beta	rpoB	D0L4Z4_GORB4 DNA-directed RNA polymerase subunit be ta OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=rpoB PE=3 SV=1;DNA-directed RNA polymerase subunit beta [Nocardia farcinica IFM 10152];DNA-dependent RNA polymerase catalyzes the transcription of DNA into RNA using the four ribonucleoside triphosphates as substrates; beta subunit is part of the catalytic core which binds with a sigma factor to produce the holoenzyme;DNA-directed RNA polymerase subunit beta; Reviewed; PRK00405;RNA polymerase beta subunit. RNA polymerases catalyse the DNA dependent polymerization of RNA. Prokaryotes contain a single RNA polymerase compared to three in eukaryotes (not including mitochondrial. and chloroplast polymerases). Each RNA polymerase...; cd00653;RPB12 interaction site [polypeptide binding];RPB1 interaction site [polypeptide binding];RPB10 interaction site [polypeptide binding];RPB11 interaction site [polypeptide binding];RPB3 interaction site [polypeptide binding]
GOHSU08	CDS	GOHSU_08_00760	complement(82072..82959)		conserved hypothetical protein [nite]			RER_17320 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YII8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08551 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109
GOHSU08	CDS	GOHSU_08_00770	complement(82959..83984)		hypothetical protein [migap]				hypothetical protein		F1YII9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08556 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU08	CDS	GOHSU_08_00780	complement(84081..85334)		putative Mce family protein [nite]	 mce4F [nite]		RER_17300 :putative Mce family protein : mce4F : : Rhodococcus erythropolis PR4	Mce family protein	mce4F	H0R5E3_9ACTO Mce family protein OS=Gordonia effusa NBRC 100432 GN=mceF PE=4 SV=1;Mce family protein [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU08	CDS	GOHSU_08_00790	complement(85324..86676)		putative Mce family protein [nite]	 mce4E [nite]		RER_17290 :putative Mce family protein : mce4E : : Rhodococcus erythropolis PR4	ABC transporter substrate-binding protein		F1YIJ1_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08566 PE=4 SV =1;ABC transporter substrate-binding protein [Amycolat opsis mediterranei U32];virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU08	CDS	GOHSU_08_00800	complement(86676..87944)		putative Mce family protein [nite]	 mce1D [nite]		ROP_16460 :putative Mce family protein : mce1D : : Rhodococcus opacus B4	Mce family protein	mce1D	F1YIJ2_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08571 PE=4 SV =1;Mce family protein [Rhodococcus opacus B4];mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU08	CDS	GOHSU_08_00810	complement(87944..89032)		putative Mce family protein [nite]	 mce4C [nite]		RER_17270 :putative Mce family protein : mce4C : : Rhodococcus erythropolis PR4	virulence factor Mce family protein;hypothetical protein		F1YIJ3_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08576 PE=4 SV =1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.993) with cleavage site probability 0.746 at residue 37;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_1526 mammalian cell entry;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU08	CDS	GOHSU_08_00820	complement(89035..90144)		putative Mce family protein [nite]	 mce4B [nite]		RER_17260 :putative Mce family protein : mce4B : : Rhodococcus erythropolis PR4	virulence factor Mce family protein		H0R5D9_9ACTO Mce family protein OS=Gordonia effusa NBRC 100432 GN=mceB PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF3407); mce related protein; TIGRFAM: virulence factor Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU08	CDS	GOHSU_08_00830	complement(90141..91460)		putative Mce family protein [nite]	 mce4A [nite]		RER_17250 :putative Mce family protein : mce4A : : Rhodococcus erythropolis PR4	Mce family protein	mce1A	F1YIJ4_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08591 PE=4 SV =1;Mce family protein [Rhodococcus opacus B4];mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU08	CDS	GOHSU_08_00840	complement(91468..92340)		putative YrbE family protein [nite]	 yrbE1B [nite]		ROP_16500 :putative YrbE family protein : yrbE1B : : Rhodococcus opacus B4	organic solvents resistance ABC transporter permease		F1YIJ5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08596 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF140; TIGRFAM: conserved hypothetical integral membrane protein;Permease; cl00510
GOHSU08	CDS	GOHSU_08_00850	complement(92343..93149)		putative YrbE family protein [nite]	 yrbE1A [nite]		ROP_16510 :putative YrbE family protein : yrbE1A : : Rhodococcus opacus B4	YrbE family protein	yrbE1A	F1YIJ6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08601 PE=4 SV=1;YrbE family protein [Rhodococcus opacus B4];Permease; cl00510
GOHSU08	CDS	GOHSU_08_00860	complement(93146..94393)		putative ABC transporter ATP-binding protein [nite]			ROP_46830 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	ABC transporter ATP-binding protein		F1YIJ7_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_08606 PE=3 SV=1;ABC transporter ATP-binding protein [Rhodococcus op acus B4];ABC-type transport system involved in resistance to organic solvents, ATPase component [Secondary metabolites biosynthesis, transport, and catabolism]; COG1127;ABC (ATP-binding cassette) transport system involved in resistant to organic solvents; ABC transporters are a large family of proteins involved in the transport of a wide variety of different compounds, like sugars, ions, peptides, and more complex...; cd03261;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU08	CDS	GOHSU_08_00870	complement(94777..95169)	case	50S ribosomal protein L7/L12 [hamap]	rplL [hamap]	MF_00368:50S ribosomal protein L7/L12:rplL 	ROP_16530 :50S ribosomal protein L7/L12 : rplL : : Rhodococcus opacus B4	50S ribosomal protein L7/L12	rplL	G7H7G9_9ACTO 50S ribosomal protein L7/L12 OS=Gordon ia araii NBRC 100433 GN=rplL PE=3 SV=1;50S ribosomal protein L7/L12 [Corynebacterium gluta micum ATCC 13032];IS fingerprint type 4-5;present in two forms; L12 is normal, while L7 is aminoacylated at the N-terminal serine; the only multicopy ribosomal protein; 4:1 ratio of L7/L12 per ribosome; two L12 dimers bind L10; critically important for translation efficiency and fidelity; stimulates GTPase activity of translation factors;Ribosomal protein L7/L12. Ribosomal protein L7/L12 refers to the large ribosomal subunit proteins L7 and L12, which are identical except that L7 is acetylated at the N terminus. It is a component of the L7/L12 stalk, which is located at the surface of...; cd00387;core dimer interface [polypeptide binding];peripheral dimer interface [polypeptide binding];L10 interface [polypeptide binding];L11 interface [polypeptide binding];putative EF-Tu interaction site [polypeptide binding];putative EF-G interaction site [polypeptide binding]
GOHSU08	CDS	GOHSU_08_00880	complement(95269..95796)	case	50S ribosomal protein L10 [hamap]	rplJ [hamap]	MF_00362:50S ribosomal protein L10:rplJ 	ROP_16540 :50S ribosomal protein L10 : rplJ : : Rhodococcus opacus B4	50S ribosomal protein L10	rplJ	H0R5D3_9ACTO 50S ribosomal protein L10 OS=Gordonia effusa NBRC 100432 GN=rplJ PE=3 SV=1;50S ribosomal protein L10 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;binds the two ribosomal protein L7/L12 dimers and anchors them to the large ribosomal subunit;Ribosomal protein L10 family, L10 subfamily; composed of bacterial 50S ribosomal protein and eukaryotic mitochondrial 39S ribosomal protein, L10. L10 occupies the L7/L12 stalk of the ribosome. The N-terminal domain (NTD) of L10 interacts with L11 protein...; cd05797;23S rRNA interface [nucleotide binding];Interface with L7/L12 ribosomal proteins [polypeptide binding]
GOHSU08	CDS	GOHSU_08_00890	96083..97282		conserved hypothetical protein [nite]			RER_14050 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	glycosyl hydrolase family protein		F1YIK0_9ACTO Putative glycosyl hydrolase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_08621 PE=4 SV=1;glycosyl hydrolase family protein [Paenibacillus sp . JDR- 2];PFAM: glycoside hydrolase family 76; KEGG: hypothetical protein;Glycosyl hydrolase family 76; cl01996
GOHSU08	CDS	GOHSU_08_00900	97397..99469		putative hydrolase [nite]			ROP_49710 :putative hydrolase : : : Rhodococcus opacus B4	X-Pro dipeptidyl-peptidase domain-containing protein		F1YIK1_9ACTO X-Pro dipeptidyl-peptidase domain-cont aining protein OS=Gordonia neofelifaecis NRRL B-59395 GN=S CNU_08626 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: X-Pro dipeptidyl-peptidase domain protein; peptidase S15; KEGG: rpf:Rpic12D_1770 peptidase S15;putative hydrolase, CocE/NonD family; TIGR00976;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;X-Pro dipeptidyl-peptidase C-terminal non-catalytic domain; cl08482
GOHSU08	CDS	GOHSU_08_00910	complement(99497..100474)		carboxylesterase [nite]			ROP_13060 :carboxylesterase : : 3.1.1.1 : Rhodococcus opacus B4	lipase	lipU	I0WP40_9NOCA Carboxylesterase OS=Rhodococcus imtech ensis RKJ300 GN=W59_20158 PE=4 SV=1;lipase [Mycobacterium sp. JDM601];alpha/beta hydrolase fold; pfam07859;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU08	CDS	GOHSU_08_00920	complement(100615..100971)	overlap	flavin-containing monooxygenase [nite]			ROP_42130 :flavin-containing monooxygenase : : : Rhodococcus opacus B4	putative flavoprotein involved in K+ transport		F5YZ67_MYCSD Monooxygenase OS=Mycobacterium sp. (st rain JDM601) GN=JDM601_1559 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Flavin-binding monooxygenase-like;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU08	CDS	GOHSU_08_00930	complement(100883..102178)	overlap	putative flavin-containing monooxygenase [nite]			RER_46620 :putative flavin-containing monooxygenase : : : Rhodococcus erythropolis PR4	monooxygenase		F1YHF5_9ACTO FAD-dependent pyridine nucleotide-disu lfide oxidoreductase OS=Gordonia neofelifaecis NRRL B-5939 5 GN=SCNU_06115 PE=4 SV=1;monooxygenase [Mycobacterium sp. JDM601];Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU08	CDS	GOHSU_08_00940	102269..102751		hypothetical protein [nite]			ROP_32510 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		A3Q581_MYCSJ Putative uncharacterized protein OS=My cobacterium sp. (strain JLS) GN=Mjls_4539 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU08	CDS	GOHSU_08_00950	complement(102834..104426)		putative fatty-acid--CoA ligase [nite]			ROP_38360 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	long-chain-fatty-acid--CoA ligase		F8AXI7_FRADG Long-chain-fatty-acid--CoA ligase OS=F rankia symbiont subsp. Datisca glomerata GN=FsymDg_2050 PE =4 SV=1;unnamed protein product [Sulfobacillus acidophilus DSM 10332];acyl-CoA synthetase; Validated;PFAM: AMP-binding enzyme; COGs: COG0318 Acyl-CoA synthetase (AMP-forming)/AMP-acid ligase II; InterPro IPR000873; KEGG: bts:Btus_2642 AMP-dependent synthetase and ligase; PFAM: AMP-dependent synthetase/ligase; PRIAM: Long-chain-fatty-acid--CoA ligase; SPTR: AMP-dependent synthetase and ligase;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU08	CDS	GOHSU_08_00960	complement(104546..105406)		hypothetical protein [nite]			ROP_32500 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F9VTZ2_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_048_00430 PE=4 SV= 1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Protein of unknown function (DUF2786); pfam10979
GOHSU08	CDS	GOHSU_08_00970	complement(105503..106459)		L-lactate dehydrogenase [hamap]	ldh [hamap]	MF_00488:L-lactate dehydrogenase:ldh 	RER_42180 :L-lactate dehydrogenase : ldh : 1.1.1.27 : Rhodococcus erythropolis PR4	L-lactate dehydrogenase		D0L4X2_GORB4 L-lactate dehydrogenase (Precursor) OS =Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=ldh PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.978) with cleavage site probability 0.873 at residue 24;TIGRFAM: L-lactate dehydrogenase; PFAM: lactate/malate dehydrogenase; KEGG: scl:sce5245 L-lactate dehydrogenase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;L-lactate dehydrogenase; TIGR01771;NAD(P) binding site [chemical binding];LDH/MDH dimer interface [polypeptide binding];substrate binding site [chemical binding]
GOHSU08	CDS	GOHSU_08_00980	complement(106617..107333)		50S ribosomal protein L1 [nite]	 rplA [nite]		RER_17180 :50S ribosomal protein L1 : rplA : : Rhodococcus erythropolis PR4	50S ribosomal protein L1	rplA	H5U765_9ACTO 50S ribosomal protein L1 OS=Gordonia s puti NBRC 100414 GN=rplA PE=3 SV=1;50S ribosomal protein L1 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;in Escherichia coli and Methanococcus, this protein autoregulates expression; the binding site in the mRNA mimics the binding site in the 23S rRNA;Ribosomal protein L1.  The L1 protein, located near the E-site of the ribosome, forms part of the L1 stalk along with 23S rRNA.  In bacteria and archaea, L1 functions both as a ribosomal protein that binds rRNA, and as a translation repressor that binds...; cd00403;mRNA/rRNA interface [nucleotide binding]
GOHSU08	CDS	GOHSU_08_00990	complement(107476..107910)		50S ribosomal protein L11 [nite]	 rplK [nite]		ROP_16570 :50S ribosomal protein L11 : rplK : : Rhodococcus opacus B4	50S ribosomal protein L11	rplK	H5UGL6_9ACTO 50S ribosomal protein L11 OS=Gordonia terrae NBRC 100016 GN=rplK PE=3 SV=1;50S ribosomal protein L11 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;binds directly to 23S ribosomal RNA;50S ribosomal protein L11; Validated; PRK00140;Ribosomal protein L11. Ribosomal protein L11, together with proteins L10 and L7/L12, and 23S rRNA, form the L7/L12 stalk on the surface of the large subunit of the ribosome. The homologous eukaryotic cytoplasmic protein is also called 60S ribosomal...; cd00349;23S rRNA interface [nucleotide binding];L7/L12 interface [polypeptide binding];putative thiostrepton binding site;L25 interface [polypeptide binding]
GOHSU08	CDS	GOHSU_08_01000	complement(108026..108853)		transcription antitermination protein NusG [nite]	 nusG [nite]		ROP_16580 :transcription antitermination protein NusG : nusG : : Rhodococcus opacus B4	transcription antitermination protein, NusG		G7H7F1_9ACTO Transcription antitermination protein nusG OS=Gordonia araii NBRC 100433 GN=nusG PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];transcription antitermination protein NusG; Validated; PRK05609;Bacterial N-Utilization Substance G (NusG) N-terminal (NGN) domain, subgroup 1; cd09891;putative homodimer interface [polypeptide binding]
GOHSU08	CDS	GOHSU_08_01010	complement(108927..109328)		probable preprotein translocase SecE subunit [nite]	 secE [nite]		RER_17150 :probable preprotein translocase SecE subunit : secE : : Rhodococcus erythropolis PR4	preprotein translocase subunit SecE		H5TNC5_9ACTO Preprotein translocase SecE subunit OS =Gordonia otitidis NBRC 100426 GN=secE PE=4 SV=1;preprotein translocase subunit SecE [Bifidobacteriu m longum subsp. longum JCM 1217];type strain of Bifidobacterium longum subsp. longum;SecE/Sec61-gamma subunits of protein translocation complex; cl00481
GOHSU08	tRNA	GOHSU_08_t0010	complement(109417..109489)		not assigned						identified by tRNAscan-SE; putative
GOHSU08	CDS	GOHSU_08_01020	complement(109718..110812)		MaoC like domain-containing protein [migap]				MaoC like domain-containing protein		F9VTX6_9ACTO Beta-hydroxyacyl-[acyl-carrier-protein ] dehydratase subunit HadA OS=Gordonia alkanivorans NBRC 1 6433 GN=hadA PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF01575;(R)-hydratase [(R)-specific enoyl-CoA hydratase]. Catalyzes the hydration of trans-2-enoyl CoA to (R)-3-hydroxyacyl-CoA as part of the PHA (polyhydroxyalkanoate) biosynthetic pathway.  The structure of the monomer includes a five-strand antiparallel...; cd03441;catalytic site [active];SAV4209_like.  Similar in sequence to the Streptomyces avermitilis SAV4209 protein, with a hot dog fold that is similar to those of (R)-specific enoyl-CoA hydratase, the peroxisomal Hydratase-Dehydrogenase-Epimerase (HDE) protein, and the fatty acid...; cd03453
GOHSU08	CDS	GOHSU_08_01030	complement(111030..111197)	case	50S ribosomal protein L33 [hamap]	rpmG [hamap]	MF_00294:50S ribosomal protein L33:rpmG 	ROP_16620 :50S ribosomal protein L33 : rpmG : : Rhodococcus opacus B4	50S ribosomal protein L33	rpmG	G7GQV1_9ACTO 50S ribosomal protein L33 2 OS=Gordoni a amarae NBRC 15530 GN=rpmG PE=3 SV=1;rpmG gene product [Mycobacterium smegmatis str. MC2 155];in Escherichia coli BM108, a mutation that results in lack of L33 synthesis had no effect on ribosome synthesis or function; there are paralogous genes in several bacterial genomes, and a CXXC motif for zinc binding and an upstream regulation region of the paralog lacking this motif that are regulated by zinc similar to other ribosomal proteins like L31; the proteins in this group have the CXXC motif;Ribosomal protein L33; cl00383
GOHSU08	tRNA	GOHSU_08_t0020	complement(111309..111385)		not assigned						identified by tRNAscan-SE; putative
GOHSU08	tRNA	GOHSU_08_t0030	complement(111428..111500)		not assigned						identified by tRNAscan-SE; putative
GOHSU08	tRNA	GOHSU_08_t0040	complement(111613..111696)		not assigned						identified by tRNAscan-SE; putative
GOHSU08	CDS	GOHSU_08_01040	111829..112320	case	UPF0234 protein yajQ [hamap]	yajQ [hamap]	MF_00632:UPF0234 protein yajQ:yajQ 	ROP_16630 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YIL1_9ACTO UPF0234 protein SCNU_08676 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08676 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF520); cl00723
GOHSU08	CDS	GOHSU_08_01050	complement(112307..113509)		alpha/beta hydrolase fold domain protein [migap]				alpha/beta hydrolase fold domain protein		H5UB97_9ACTO Putative esterase OS=Gordonia terrae N BRC 100016 GN=GOTRE_036_00110 PE=4 SV=1;alpha/beta hydrolase fold domain protein [Alcanivor ax sp. DG881];[I] COG0657 Esterase/lipase;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;alpha/beta hydrolase fold; pfam07859
GOHSU08	CDS	GOHSU_08_01060	complement(113566..115281)		putative fatty-acid--CoA ligase [nite]			RER_44820 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus erythropolis PR4	fatty-acid--CoA ligase		E2S805_9ACTO AMP-dependent synthetase and ligase OS =Aeromicrobium marinum DSM 15272 GN=HMPREF0063_10162 PE=4 SV=1;fatty-acid--CoA ligase [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;protein synonym:acyl-CoA synthetase;Acyl-CoA synthetases (AMP-forming)/AMP-acid ligases II [Lipid metabolism / Secondary metabolites biosynthesis, transport, and catabolism]; COG0318;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU08	CDS	GOHSU_08_01070	complement(115274..116239)		2-nitropropane dioxygenase family protein [nite]			SLG_36540 :2-nitropropane dioxygenase family protein : : : Sphingobium sp. SYK-6	2-nitropropane dioxygenase		H5UII6_9ACTO Putative 2-nitropropane dioxygenase OS =Gordonia terrae NBRC 100016 GN=GOTRE_125_01810 PE=4 SV=1;2-nitropropane dioxygenase [Sphingobium japonicum U T26S];Dioxygenases related to 2-nitropropane dioxygenase [General function prediction only];dioxygenase related to 2-nitropropane dioxygenase;2-Nitropropane dioxygenase (NPD), one of the nitroalkane oxidizing enzyme families, catalyzes oxidative denitrification of nitroalkanes to their corresponding carbonyl compounds and nitrites. NDP is a member of the NAD(P)H-dependent flavin oxidoreductase...; cd04730;FMN binding site [chemical binding];substrate binding site [chemical binding];putative catalytic residue [active]
GOHSU08	CDS	GOHSU_08_01080	complement(116236..117756)		putative dioxygenase [nite]			RER_44930 :putative dioxygenase : : : Rhodococcus erythropolis PR4	lignostilbene-alpha,beta-dioxygenase-like protein		G7GV34_9ACTO Putative dioxygenase OS=Gordonia amara e NBRC 15530 GN=GOAMR_68_00150 PE=4 SV=1;lignostilbene-alpha,beta-dioxygenase-like protein [ Mycobacterium sp. Spyr1];PFAM: Retinal pigment epithelial membrane protein;Lignostilbene-alpha,beta-dioxygenase and related enzymes [Secondary metabolites biosynthesis, transport, and catabolism];Retinal pigment epithelial membrane protein; cl10080
GOHSU08	CDS	GOHSU_08_01090	complement(117821..119032)		conserved hypothetical protein [nite]			RER_44940 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	acetyl-CoA acetyltransferase		F9QP77_9MYCO Acetyl-CoA acetyltransferase OS=Mycoba cterium colombiense CECT 3035 GN=MCOL_16641 PE=3 SV=1;acetyl-CoA acetyltransferase [Mycobacterium vanbaal enii PYR- 1];Catalyzes the synthesis of acetoacetyl coenzyme A from two molecules of acetyl coenzyme A. It can also act as a thiolase, catalyzing the reverse reaction and generating two-carbon units from the four-carbon product of fatty acid oxidation;acetyl-CoA acetyltransferase; Provisional;Thiolase domain associated with sterol carrier protein (SCP)-x isoform and related proteins; SCP-2  has multiple roles in intracellular lipid circulation and metabolism. The N-terminal presequence in the SCP-x isoform represents a peroxisomal...; cd00829
GOHSU08	CDS	GOHSU_08_01100	119173..120423		putative acetyl-CoA acyltransferase [nite]			RER_44950 :putative acetyl-CoA acyltransferase : : 2.3.1.- : Rhodococcus erythropolis PR4	acetyl-CoA acetyltransferase		E2S7X9_9ACTO Acetyl-CoA acetyltransferase OS=Aeromi crobium marinum DSM 15272 GN=HMPREF0063_10136 PE=3 SV=1;acetyl-CoA acetyltransferase [Sphingobium chlorophe nolicum L- 1];acetyl-CoA acetyltransferase; Validated;KEGG: ccs:CCNA_00075 acetyl-CoA acetyltransferase; TIGRFAM: Thiolase;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU08	CDS	GOHSU_08_01110	120433..121383		putative HxlR family transcriptional regulator [nite]			RER_44990 :putative HxlR family transcriptional regulator : : : Rhodococcus erythropolis PR4	MarR family transcriptional regulator		E2S7Y2_9ACTO Transcriptional regulator family prote in OS=Aeromicrobium marinum DSM 15272 GN=HMPREF0063_10139 PE=4 SV=1;unnamed protein product [Caulobacter crescentus NA1 000];CC_3112;Helix-turn-helix domains; cl00088
GOHSU08	CDS	GOHSU_08_01120	complement(121423..122304)	case	protease HtpX [hamap]	htpX [hamap]	MF_00188:protease HtpX:htpX 	ROP_16640 :putative protease HtpX : htpX : 3.4.24.- : Rhodococcus opacus B4	peptidase M48 Ste24p		F1YIL4_9ACTO Protease HtpX homolog OS=Gordonia neof elifaecis NRRL B-59395 GN=htpX PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.969) with cleavage site probability 0.534 at residue 34;PFAM: peptidase M48 Ste24p; KEGG: bxe:Bxe_A4413 M48 family peptidase;Peptidase family M48; cl12018
GOHSU08	CDS	GOHSU_08_01130	complement(122419..123435)		putative polyprenyl diphosphate synthase [nite]			ROP_16650 :putative polyprenyl diphosphate synthase : : 2.5.1.- : Rhodococcus opacus B4	trans-hexaprenyltranstransferase		F1YIL5_9ACTO Trans-hexaprenyltranstransferase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08696 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: polyprenyl synthetase; KEGG: xcv:XCV3050 octaprenyl-diphosphate synthase;Geranylgeranyl pyrophosphate synthase [Coenzyme metabolism]; COG0142;Trans-Isoprenyl Diphosphate Synthases, head-to-tail; cd00685;substrate binding pocket [chemical binding];chain length determination region;substrate-Mg2+ binding site;catalytic residues [active];aspartate-rich region 1;active site lid residues [active];aspartate-rich region 2
GOHSU08	CDS	GOHSU_08_01140	123570..124832		putative oxidoreductase [nite]			RER_17080 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	putative FAD-dependent oxidoreductase		F1YIL6_9ACTO Geranylgeranyl reductase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_08701 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];geranylgeranyl reductase family; TIGR02032
GOHSU08	CDS	GOHSU_08_01150	complement(124867..125562)	case	ubiquinone/menaquinone biosynthesis methyltransferase UbiE [hamap]	ubiE [hamap]	MF_01813:ubiquinone/menaquinone biosynthesis methyltransferase UbiE:ubiE 	ROP_16670 :menaquinone biosynthesis methyltransferase MenH : menH : 2.1.1.- : Rhodococcus opacus B4	ubiquinone/menaquinone biosynthesis methyltransferase	ubiE	H6N2C2_GORPV Demethylmenaquinone methyltransferase OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN =ubiE PE=3 SV=1;ubiquinone/menaquinone biosynthesis methyltransfera se [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Methylase involved in ubiquinone/menaquinone biosynthesis [Coenzyme metabolism]; COG2226;Catalyzes the carbon methylation reaction in the biosynthesis of ubiquinone;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU08	CDS	GOHSU_08_01160	complement(125559..126023)		conserved hypothetical protein [nite]			RER_17050 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YIL8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08711 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3592); pfam12158
GOHSU08	CDS	GOHSU_08_01170	complement(126050..127729)		2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylate synthase [hamap]	menD [hamap]	MF_01659:2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1-carboxylate synthase:menD 	ROP_16710 :menaquinone biosynthesis protein MenD : menD : 2.5.1.- : Rhodococcus opacus B4	2-succinyl-5-enolpyruvyl-6-hydroxy-3- cyclohexene-1-carboxylate synthase	menD	D0L4U8_GORB4 2-succinyl-5-enolpyruvyl-6-hydroxy-3-c yclohexene-1-carboxylate synthase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN =menD PE=3 SV=1;2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1- carboxylate synthase [Corynebacterium glutamicum ATCC 130 32];IS fingerprint type 4-5;2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexene-1- carboxylate synthase; Validated;SEPHCHC synthase; forms 5-enolpyruvoyl-6-hydroxy-2-succinyl-cyclohex-3-ene-1- carboxylate from 2-oxoglutarate and isochorismate in menaquinone biosynthesis;Pyrimidine (PYR) binding domain of 2-succinyl-5-enolpyruvyl-6-hydroxy-3-cyclohexadiene-1- carboxylate synthase (MenD) and related proteins; cd07037;dimer interface [polypeptide binding];tetramer interface [polypeptide binding];PYR/PP interface [polypeptide binding];TPP binding site [chemical binding];Thiamine pyrophosphate (TPP) family, SHCHC synthase subfamily, TPP-binding module; composed of proteins similar to Escherichia coli 2-succinyl-6-hydroxyl-2,4-cyclohexadiene-1-carboxylic acid (SHCHC) synthase (also called MenD). SHCHC synthase plays a key...; cd02009;TPP-binding site
GOHSU08	CDS	GOHSU_08_01180	127828..128445		hypothetical protein [nite]			ROP_38770 :hypothetical protein : : : Rhodococcus opacus B4	methyltransferase type 11		F1YIM0_9ACTO Type 11 methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08721 PE=4 SV=1;methyltransferase type 11 [Frankia sp. EuI1c];PFAM: methyltransferase type 11; KEGG: fal:FRAAL0865 hypothetical protein;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU08	CDS	GOHSU_08_01190	128711..129958		hypothetical protein [nite]			ROP_06320 :hypothetical protein : : : Rhodococcus opacus B4	saccharopine dehydrogenase		F1YIM1_9ACTO Saccharopine dehydrogenase (NAD(+), L- glutamate-forming) OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08726 PE=4 SV=1;saccharopine dehydrogenase [Alcanivorax sp. DG881];[S] COG3268 Uncharacterized conserved protein;Uncharacterized conserved protein [Function unknown];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU08	CDS	GOHSU_08_01200	129975..130553		galactoside O-acetyltransferase [nite]	 lacA [nite]		MLP_29770 :galactoside O-acetyltransferase : lacA : 2.3.1.18 : Microlunatus phosphovorus NM-1	maltose O-acetyltransferase		D5UUR0_TSUPD Putative sugar acetyltransferase OS=Ts ukamurella paurometabola (strain ATCC 8368 / DSM 20162 / J CM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_3034 PE=4 SV=1 ;maltose O-acetyltransferase [Clostridium botulinum B str. Eklund 17B];identified by match to protein family HMM PF00132;maltose O-acetyltransferase; Provisional;Maltose O-acetyltransferase (MAT) and Galactoside O-acetyltransferase (GAT): MAT and GAT catalyze the CoA-dependent acetylation of the 6-hydroxyl group of their respective sugar substrates. MAT acetylates maltose and glucose exclusively at the C6...; cd03357;substrate binding site [chemical binding];trimer interface [polypeptide binding];CoA binding site [chemical binding]
GOHSU08	CDS	GOHSU_08_01210	complement(130802..131461)		hypothetical protein [nite]			ROP_16760 :hypothetical protein : : : Rhodococcus opacus B4	thioesterase		F1YIM2_9ACTO Thioesterase superfamily protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08731 PE=4 SV=1;thioesterase [Mycobacterium avium 104];identified by match to protein family HMM PF03061;PaaI_thioesterase is a tetrameric acyl-CoA thioesterase with a hot dog fold and one of several proteins responsible for phenylacetic acid (PA) degradation in bacteria.  Although orthologs of PaaI exist in archaea and eukaryotes, their function has not...; cd03443;CoenzymeA binding site [chemical binding];subunit interaction site [polypeptide binding];PHB binding site
GOHSU08	CDS	GOHSU_08_01220	131560..132504		naphthoate synthase [nite]	 menB [nite]		RER_16980 :naphthoate synthase : menB : 4.1.3.36 : Rhodococcus erythropolis PR4	naphthoate synthase	menB	F1YIM7_9ACTO Naphthoate synthase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_08756 PE=3 SV=1;naphthoate synthase [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;catalyzes the formation of 1,4-dihydroxy-2-naphthoate from O-succinylbenzoyl-CoA;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;Enoyl-CoA hydratase/isomerase family; pfam00378;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU08	CDS	GOHSU_08_01230	complement(132767..133810)		putative oxidoreductase [nite]			ROP_19710 :putative oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase		F1YIN0_9ACTO Putative F420-dependent oxidoreductase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08771 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00296;probable F420-dependent oxidoreductase, Rv3520c family; TIGR03559;N5,N10-methylenetetrahydromethanopterin reductase (Mer) catalyzes the reduction of N5,N10-methylenetetrahydromethanopterin with reduced coenzyme F420 to N5-methyltetrahydromethanopterin and oxidized coenzyme F420; cd01097
GOHSU08	CDS	GOHSU_08_01240	134069..135331		inorganic phosphate transporter [nite]	 pit [nite]		KRH_23060 :inorganic phosphate transporter : pit : : Kocuria rhizophila DC2201	phosphate transporter		F1YIN1_9ACTO Phosphate transporter OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_08776 PE=4 SV=1;phosphate transporter [Streptomyces violaceusniger Tu 4113];PFAM: Phosphate transporter; KEGG: amd:AMED_1273 inorganic phosphate transporter PiT family;Phosphate transporter family; cl00396
GOHSU08	CDS	GOHSU_08_01250	135360..135644		hypothetical protein [nite]			KRH_23050 :hypothetical protein : : : Kocuria rhizophila DC2201	hypothetical protein		H5U6D8_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_125_00630 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU08	CDS	GOHSU_08_01260	135681..135995		conserved hypothetical protein [nite]			RER_36330 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YIN3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08786 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF3349);Protein of unknown function (DUF3349); pfam11829
GOHSU08	CDS	GOHSU_08_01270	135992..>136259	3x codon	putative o-succinylbenzoate--CoA ligase [nite]	 menE [nite]		RER_16870 :putative o-succinylbenzoate--CoA ligase : menE : 6.2.1.26 : Rhodococcus erythropolis PR4	acyl-CoA synthetase		I0WRF8_9NOCA O-succinylbenzoic acid--CoA ligase OS= Rhodococcus imtechensis RKJ300 GN=W59_16321 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: AMP-binding enzyme;AMP-binding enzyme; cl15778;O-succinylbenzoic acid--CoA ligase; Provisional
##GOHSU09	contig00034				6336
GOHSU09	CDS	GOHSU_09_00010	<1..929	3x codon frame	putative o-succinylbenzoate--CoA ligase [nite]	 menE [nite]		RER_16870 :putative o-succinylbenzoate--CoA ligase : menE : 6.2.1.26 : Rhodococcus erythropolis PR4	O-succinylbenzoic acid--CoA ligase	menE	F1YIN5_9ACTO O-succinylbenzoic acid--CoA ligase OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08796 PE=4 SV= 1;O-succinylbenzoic acid--CoA ligase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;AMP-binding enzyme; cl15778;O-succinylbenzoic acid--CoA ligase; Provisional
GOHSU09	CDS	GOHSU_09_00020	999..2189		conserved hypothetical protein [nite]			BBR47_46760 :conserved hypothetical protein : : : Brevibacillus brevis NBRC 100599	hypothetical protein		H0RAG4_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_021_00240 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];hypothetical protein; Provisional;ATP-grasp domain; cl03087
GOHSU09	CDS	GOHSU_09_00030	complement(2258..2560)		Fe-S protein [migap]				Fe-S protein		G4I907_MYCRH Putative uncharacterized protein (Prec ursor) OS=Mycobacterium rhodesiae JS60 GN=MycrhDRAFT_6570 PE=4 SV=1;Fe-S protein [Microbacterium testaceum StLB037]
GOHSU09	CDS	GOHSU_09_00040	complement(2706..4163)		hypothetical protein [migap]				hypothetical protein		F1YIN6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08801 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];PE-PPE domain; pfam08237;DNA polymerase III subunits gamma and tau; Provisional
GOHSU09	CDS	GOHSU_09_00050	4460..5332		1,4-dihydroxy-2-naphthoate octaprenyltransferase [nite]	 menA [nite]		RER_16820 :1,4-dihydroxy-2-naphthoate octaprenyltransferase : menA : 2.5.1.- : Rhodococcus erythropolis PR4	1,4-dihydroxy-2-naphthoate octaprenyltransferase	menA	H5UBK4_9ACTO 1,4-dihydroxy-2-naphthoate octaprenylt ransferase OS=Gordonia terrae NBRC 100016 GN=menA PE=4 SV= 1;1,4-dihydroxy-2-naphthoate octaprenyltransferase [C orynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of dimethylmenaquinone from 1,4-dihydroxy-2-naphthoate and octaprenyl diphosphate;UbiA prenyltransferase family; cl00337
GOHSU09	CDS	GOHSU_09_00060	5359..6324		not assigned						
##GOHSU10	contig00041				3661
GOHSU10	CDS	GOHSU_10_00010	<1..295	3x codon frame	not assigned						H5U4Z6_9ACTO Putative tRNA (cytidine(34)-2'-O)-meth yltransferase OS=Gordonia sputi NBRC 100414 GN=GOSPT_115_0 0610 PE=3 SV=1
GOHSU10	CDS	GOHSU_10_00020	292..480		not assigned						
GOHSU10	CDS	GOHSU_10_00030	456..1406		hypothetical protein [nite]			ROP_62360 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YFI8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02957 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU10	CDS	GOHSU_10_00040	1418..3280		glucosamine--fructose-6-phosphate aminotransferase [isomerizing] [hamap]	glmS [hamap]	MF_00164:glucosamine--fructose-6-phosphate aminotransferase [isomerizing]:glmS 	ROP_62370 :glucosamine-6-phosphate synthase : glmS : 2.6.1.16 : Rhodococcus opacus B4	L-glutamine:D-fructose-6-phosphate aminotransferase	glmS	F1YFI9_9ACTO Glucosamine--fructose-6-phosphate amin otransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_02962 PE=3 SV=1;L-glutamine-D-fructose-6-phosphate amidotransferase [Microlunatus phosphovorus NM- 1];strain coidentity: NBRC 101784;glucosamine--fructose-6-phosphate aminotransferase; Reviewed;Glutamine amidotransferases class-II (Gn-AT)_GFAT-type. This domain is found at the N-terminus of glucosamine-6P synthase (GlmS, or GFAT in humans).  The glutaminase domain catalyzes amide nitrogen transfer from glutamine to the appropriate substrate. In...; cd00714;glutaminase active site [active];SIS (Sugar ISomerase) domain repeat 1 found in Glucosamine 6-phosphate synthase (GlmS) and Glucosamine-6-phosphate deaminase (GlmD). The SIS domain is found in many phosphosugar isomerases and phosphosugar binding proteins. GlmS contains a N-terminal...; cd05008;dimer interface [polypeptide binding];SIS (Sugar ISomerase) domain repeat 2 found in Glucosamine 6-phosphate synthase (GlmS) and Glucosamine-6-phosphate deaminase (GlmD). The SIS domain is found in many phosphosugar isomerases and phosphosugar binding proteins. GlmS contains a N-terminal...; cd05009
GOHSU10	CDS	GOHSU_10_00050	3284..>3661	codon	conserved hypothetical protein [nite]			RER_19130 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative carbohydrate kinase, YjeF family protein		F1YFJ0_9ACTO Carbohydrate kinase, YjeF related prot ein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02967 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];YjeF-related protein N-terminus; cl00318
##GOHSU11	contig00042				953
##GOHSU12	contig00043				186019
GOHSU12	CDS	GOHSU_12_00010	117..1259		alanine racemase [hamap]	alr [hamap]	MF_01201:alanine racemase:alr 	RER_19140 :alanine racemase : alr : 5.1.1.1 : Rhodococcus erythropolis PR4	alanine racemase		H5TH50_9ACTO Alanine racemase OS=Gordonia otitidis NBRC 100426 GN=alr PE=3 SV=1;alanine racemase [Nocardiopsis dassonvillei subsp. dassonvillei DSM 43111];COGs: COG0787 Alanine racemase; InterPro IPR009006:IPR000821:IPR001608:IPR011079; KEGG: fal:FRAAL1122 alanine racemase; PFAM: alanine racemase domain protein; PRIAM: Alanine racemase; SPTR: C1YUH1 Alanine racemase; TIGRFAM: alanine racemase; PFAM: Alanine racemase, C-terminal domain; Alanine racemase, N-terminal domain; TIGRFAM: alanine racemase;alanine racemase; Reviewed; PRK00053;Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzyme Alanine Racemase; cd00430;pyridoxal 5'-phosphate (PLP) binding site [chemical binding];dimer interface [polypeptide binding];substrate binding site [chemical binding];catalytic residues [active]
GOHSU12	CDS	GOHSU_12_00020	1361..2413		putative hydrolase [nite]			RER_19150 :putative hydrolase : : : Rhodococcus erythropolis PR4	hydrolase		F1YFJ2_9ACTO Alpha/beta hydrolase fold protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_02977 PE=4 SV=1 ;hydrolase [Rhodococcus opacus B4];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697
GOHSU12	CDS	GOHSU_12_00030	2422..2862		putative ATPase [nite]			ROP_62420 :putative ATPase : : 3.6.-.- : Rhodococcus opacus B4	hypothetical protein		D0L7W2_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_1713 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function UPF0079; KEGG: glo:Glov_2530 protein of unknown function UPF0079;Uncharacterized P-loop hydrolase UPF0079; cl00520
GOHSU12	CDS	GOHSU_12_00040	2855..3529		conserved hypothetical protein [nite]			RER_19170 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	peptidase M22 glycoprotease		F1YFJ4_9ACTO Peptidase M22 glycoprotease OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_02987 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: peptidase M22 glycoprotease; KEGG: dps:DP1162 glycoprotein endopeptidase;Inactive homolog of metal-dependent proteases, putative molecular chaperone [Posttranslational modification, protein turnover, chaperones]; cl14000
GOHSU12	CDS	GOHSU_12_00050	3530..4015		putative ribosomal-protein-alanine acetyltransferase [nite]			ROP_62440 :putative ribosomal-protein-alanine acetyltransferase : : 2.3.1.128 : Rhodococcus opacus B4	ribosomal-protein-alanine acetyltransferase		F1YFQ6_9ACTO Ribosomal-protein-alanine acetyltransf erase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02992 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];ribosomal-protein-alanine acetyltransferase; TIGR01575;N-Acyltransferase superfamily: Various enzymes that characteristically catalyze the transfer of an acyl group to a substrate; cd04301;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357;Coenzyme A binding pocket [chemical binding]
GOHSU12	CDS	GOHSU_12_00060	4012..5040		probable tRNA threonylcarbamoyladenosine biosynthesis protein Gcp [hamap]	gcp [hamap]	MF_01445:probable tRNA threonylcarbamoyladenosine biosynthesis protein Gcp:gcp 	ROP_62450 :O-sialoglycoprotein endopeptidase : gcp : 3.4.24.57 : Rhodococcus opacus B4	DNA-binding/iron metalloprotein/AP endonuclease	gcp	F1YFQ7_9ACTO Probable tRNA threonylcarbamoyladenosi ne biosynthesis protein Gcp OS=Gordonia neofelifaecis NRRL B-59395 GN=gcp PE=3 SV=1;DNA-binding/iron metalloprotein/AP endonuclease [Co rynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;UGMP family protein; Validated;in most organisms, only the N-terminal domain is present in a single polypeptide; in some archaea this domain is fused to a kinase domain; this gene is essential for growth in Escherichia coli and Bacillus subtilis; the glycoprotease from Pasteurella haemolytica showed specificity for O-sialoglycosylated proteins; the Pyrococcus structure shows DNA-binding properties, iron-binding, ATP-binding, and AP endonuclease activity
GOHSU12	CDS	GOHSU_12_00070	complement(5027..6217)		putative siderophore esterase [nite]			RER_10270 :putative siderophore esterase : : : Rhodococcus erythropolis PR4	IroD	iroD	G7GZR6_9ACTO Putative non-ribosomal peptide synthet ase OS=Gordonia araii NBRC 100433 GN=GOARA_028_00020 PE=4 SV=1;IroD [Escherichia coli];avian pathogenic strain;Domain of unknown function (DUF3327); pfam11806;Enterochelin esterase and related enzymes [Inorganic ion transport and metabolism]; COG2382;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU12	CDS	GOHSU_12_00080	6377..6676		10 kDa chaperonin [hamap]	groS [hamap]	MF_00580:10 kDa chaperonin:groS 	RER_19230 :10 kDa chaperonin : groES : : Rhodococcus erythropolis PR4	co-chaperonin GroES	groES	G7GZS4_9ACTO 10 kDa chaperonin OS=Gordonia araii NB RC 100433 GN=groS PE=3 SV=1;co-chaperonin GroES [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;10 kDa chaperonin; Cpn10; GroES; forms homoheptameric ring; binds to one or both ends of the GroEL double barrel in the presence of adenine nucleotides capping it; folding of unfolded substrates initiates in a GroEL-substrate bound and capped by GroES; release of the folded substrate is dependent on ATP binding and hydrolysis in the trans ring;Chaperonin 10 Kd subunit (cpn10 or GroES); Cpn10 cooperates with chaperonin 60 (cpn60 or GroEL), an ATPase, to assist the folding and assembly of proteins and is found in eubacterial cytosol, as well as in the matrix of mitochondria and chloroplasts. It...; cd00320;oligomerisation interface [polypeptide binding];mobile loop;roof hairpin
GOHSU12	CDS	GOHSU_12_00090	complement(6631..7008)		WhiB family regulatory protein [nite]			ROP_62530 :WhiB family regulatory protein : : : Rhodococcus opacus B4	WhiB family transcriptional regulator		F1YFQ9_9ACTO WhiB family regulatory protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_03007 PE=4 SV=1;WhiB family transcriptional regulator [Rhodococcus jostii RHA1];Transcription factor WhiB; pfam02467
GOHSU12	CDS	GOHSU_12_00100	7298..8074		hypothetical protein [nite]			ROP_62550 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YFR0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03012 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU12	CDS	GOHSU_12_00110	8111..8698		RNA polymerase ECF-type sigma factor SigD [nite]	 sigD [nite]		ROP_62560 :RNA polymerase ECF-type sigma factor SigD : sigD : : Rhodococcus opacus B4	ECF subfamily RNA polymerase sigma-24 subunit		F1YFR1_9ACTO RNA polymerase sigma factor SigD OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03017 PE=4 SV=1;ECF subfamily RNA polymerase sigma-24 subunit [Acti nosynnema mirum DSM 43827];RNA polymerase sigma factor SigD; Reviewed;TIGRFAM: RNA polymerase sigma factor, sigma-70 family; PFAM: sigma-70 region 2 domain protein; Sigma-70 region 4 type 2; sigma-70 region 4 domain protein; KEGG: mno:Mnod_5360 RNA polymerase, sigma-24 subunit, ECF subfamily;Sigma-70 region 2; pfam04542;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU12	CDS	GOHSU_12_00120	8695..9633		hypothetical protein [migap]				hypothetical protein		H0R3R5_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_094_00150 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: MUC2; mucin 2, oligomeric mucus/gel-forming ; K10955    intestinal mucin-2
GOHSU12	CDS	GOHSU_12_00130	complement(9693..10082)		conserved hypothetical protein [nite]			RER_19320 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GNJ3_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_33_00050 PE=4 SV=1;unnamed protein product [Corynebacterium pseudotube rculosis FRC41]
GOHSU12	CDS	GOHSU_12_00140	10157..11668		inosine-5'-monophosphate dehydrogenase [nite]	 guaB [nite]		RER_19330 :inosine-5'-monophosphate dehydrogenase : guaB : 1.1.1.205 : Rhodococcus erythropolis PR4	inosine 5'-monophosphate dehydrogenase	guaB2	D7PDZ4_9ACTO Inosine-5'-monophosphate dehydrogenase OS=Gordonia cholesterolivorans GN=guaB PE=3 SV=1;inosine 5'-monophosphate dehydrogenase [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the synthesis of xanthosine monophosphate by the NAD+ dependent oxidation of inosine monophosphate;inosine 5'-monophosphate dehydrogenase; Reviewed;TIM barrel proteins share a structurally conserved phosphate binding motif and in general share an eight beta/alpha closed barrel structure. Specific for this family is the conserved phosphate binding site at the edges of strands 7 and 8. The phosphate...; cl09108;This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domains in the inosine 5' monophosphate dehydrogenase (IMPDH) protein. IMPDH is an essential enzyme that catalyzes the first step unique to GTP synthesis, playing a key...; cd04601;IMPDH: The catalytic domain of the inosine monophosphate dehydrogenase. IMPDH catalyzes the NAD-dependent oxidation of inosine 5'-monophosphate (IMP) to xanthosine 5' monophosphate (XMP). It is a rate-limiting step in the de novo synthesis of the guanine...; cd00381
GOHSU12	CDS	GOHSU_12_00150	11684..12850		IMP dehydrogenase family protein [nite]			ROP_62600 :IMP dehydrogenase family protein : : : Rhodococcus opacus B4	inosine 5-monophosphate dehydrogenase	guaB3	D7PDZ5_9ACTO Inosine-5-monophosphate dehydrogenase OS=Gordonia cholesterolivorans PE=3 SV=1;inosine 5-monophosphate dehydrogenase [Corynebacter ium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the synthesis of xanthosine monophosphate by the NAD+ dependent oxidation of inosine monophosphate;IMP dehydrogenase family protein; TIGR01304;TIM barrel proteins share a structurally conserved phosphate binding motif and in general share an eight beta/alpha closed barrel structure. Specific for this family is the conserved phosphate binding site at the edges of strands 7 and 8. The phosphate...; cl09108;phosphate binding site [ion binding]
GOHSU12	CDS	GOHSU_12_00160	12885..13379		hypothetical membrane protein [nite]			ROP_60200 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F9VS17_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_030_00030 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF1622); cl01991
GOHSU12	CDS	GOHSU_12_00170	13429..15189		conserved hypothetical protein [nite]			RER_19350 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative cholesterol oxidase		F1YFR8_9ACTO FAD dependent oxidoreductase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_03052 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Choline dehydrogenase and related flavoproteins [Amino acid transport and metabolism]; COG2303;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;GMC oxidoreductase; pfam05199
GOHSU12	CDS	GOHSU_12_00180	15294..16646		probable C4-dicarboxylate transporter [nite]	 dcu [nite]		RER_21000 :probable C4-dicarboxylate transporter : dcu : : Rhodococcus erythropolis PR4	anaerobic c4-dicarboxylate antiporter, Dcu family		H5U1H9_9ACTO C4-dicarboxylate transporter OS=Gordon ia sputi NBRC 100414 GN=dcu PE=4 SV=1;anaerobic c4-dicarboxylate antiporter, Dcu family [ Shewanella baltica OS183];Anaerobic c4-dicarboxylate membrane transporter; cl01040;KEGG: son:SO_4417 anaerobic C4-dicarboxylate transporter; TIGRFAM: Anaerobic c4-dicarboxylate membrane transporter; PFAM: Anaerobic c4-dicarboxylate membrane transporter
GOHSU12	CDS	GOHSU_12_00190	16687..17898		putative amidohydrolase [nite]			ROP_26330 :putative amidohydrolase : : : Rhodococcus opacus B4	peptidase		H0TZ12_9BRAD Hippurate hydrolase (Hippuricase) (Ben zoylglycine amidohydrolase) OS=Bradyrhizobium sp. STM 3843 GN=hipO PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];similar to AP001512-200|BAB05332.1| percent identity: 36 in 391 aa;M20 Peptidase Aminoacylase 1 subfamily; cd05664;Metal-dependent amidase/aminoacylase/carboxypeptidase [General function prediction only]; COG1473;metal binding site [ion binding];putative dimer interface [polypeptide binding]
GOHSU12	CDS	GOHSU_12_00200	complement(17934..19442)		putative drug resistance efflux protein [nite]			RER_33930 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	EmrB/QacA subfamily drug resistance transporter		H5TMU9_9ACTO Putative drug resistance transporter O S=Gordonia otitidis NBRC 100426 GN=GOOTI_123_00210 PE=4 SV =1;EmrB/QacA subfamily drug resistance transporter [Fr ankia sp. EuI1c];KEGG: fal:FRAAL4746  integral membrane efflux protein; TIGRFAM: drug resistance transporter EmrB/QacA subfamily; PFAM: major facilitator superfamily MFS_1;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;Major Facilitator Superfamily; pfam07690;putative substrate translocation pore
GOHSU12	CDS	GOHSU_12_00210	complement(19451..20050)		serine acetyltransferase [nite]	 cysE [nite]		RER_10920 :serine acetyltransferase : cysE : 2.3.1.30 : Rhodococcus erythropolis PR4	serine O-acetyltransferase		F1YFS7_9ACTO Serine acetyltransferase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_03097 PE=3 SV=1;serine O-acetyltransferase [Sanguibacter keddieii D SM 10542];PFAM: Bacterial transferase hexapeptide (three repeats); TIGRFAM: serine O-acetyltransferase;serine O-acetyltransferase; TIGR01172;Serine acetyltransferase (SAT): SAT catalyzes the CoA-dependent acetylation of the side chain hydroxyl group of L-serine to form O-acetylserine, as the first step of a two-step biosynthetic pathway in bacteria and plants leading to the formation of...; cd03354;trimer interface [polypeptide binding];substrate binding site [chemical binding];CoA binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_00220	complement(20051..20989)		cysteine synthase [nite]	 cysK [nite]		RER_10910 :cysteine synthase : cysK : 2.5.1.47 : Rhodococcus erythropolis PR4	cysteine synthase A		F1YFS8_9ACTO Cysteine synthase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_03102 PE=3 SV=1;unnamed protein product [Clostridium clariflavum DS M 19732];PFAM: Pyridoxal-phosphate dependent enzyme; TIGRFAM: cysteine synthase A; cysteine synthases;Cysteine synthase [Amino acid transport and metabolism]; COG0031;CBS_like: This subgroup includes Cystathionine beta-synthase (CBS) and Cysteine synthase. CBS is a unique heme-containing enzyme that catalyzes a pyridoxal 5'-phosphate (PLP)-dependent condensation of serine and homocysteine to give cystathionine; cd01561;dimer interface [polypeptide binding];pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU12	CDS	GOHSU_12_00230	complement(21036..21638)		putative ABC transporter permease protein [nite]			RER_27440 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	cobalt transport protein		F1YFS9_9ACTO Cobalt transport protein OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_03107 PE=4 SV=1;cobalt transport protein [Rhodospirillum rubrum ATC C 11170];Cobalt transport protein; cl00463
GOHSU12	CDS	GOHSU_12_00240	complement(21649..22335)		putative ABC transporter ATP-binding protein [nite]			RER_27450 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter		F1YFT0_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_03112 PE=3 SV=1;ABC transporter [Rhodospirillum rubrum ATCC 11170];ABC-type cobalt transport system, ATPase component [Inorganic ion transport and metabolism]; COG1122;Domain I of the ABC component of a cobalt transport family found in bacteria, archaea, and eukaryota.  The transition metal cobalt is an essential component of many enzymes and must be transported into cells in appropriate amounts when needed.  This ABC...; cd03225;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU12	CDS	GOHSU_12_00250	complement(22332..22919)		putative BioY family protein [nite]			RER_27460 :putative BioY family protein : : : Rhodococcus erythropolis PR4	BioY family protein		D0L8H7_GORB4 BioY protein OS=Gordonia bronchialis ( strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN= Gbro_1732 PE=4 SV=1;BioY family protein [Rhodococcus opacus B4];ECF-type riboflavin transporter, S component; cl01942
GOHSU12	CDS	GOHSU_12_00260	complement(22955..23629)		hypothetical protein [nite]			RER_08830 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D5UYT6_TSUPD Putative uncharacterized protein OS=Ts ukamurella paurometabola (strain ATCC 8368 / DSM 20162 / J CM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_3813 PE=4 SV=1 ;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rer:RER_08830 hypothetical protein
GOHSU12	CDS	GOHSU_12_00270	complement(23684..24670)		meso-diaminopimelate dehydrogenase [nite]	 ddh [nite]		TEH_03760 :meso-diaminopimelate dehydrogenase : ddh : 1.4.1.16 : Tetragenococcus halophilus NBRC 12172	diaminopimelate dehydrogenase	ddh	E1VRX6_ARTAR Diaminopimelate dehydrogenase OS=Arthr obacter arilaitensis (strain DSM 16368 / CIP 108037 / JCM 13566 / Re117) GN=ddh PE=4 SV=1;diaminopimelate dehydrogenase [Arthrobacter arilait ensis Re117];catalyses the following reaction: L-2-amino-6- oxoheptanedioate + NH(3) + NADPH <=> meso-2,6- diaminoheptanedioate + H(2)O + NADP(+). Involved in lysine biosynthesis;diaminopimelate dehydrogenase; TIGR01921;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU12	CDS	GOHSU_12_00280	24834..25478		conserved hypothetical protein [nite]			RER_54560 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	NADH-flavin reductase		F6FSM1_ISOV2 NADH-flavin reductase OS=Isoptericola variabilis (strain 225) GN=Isova_2472 PE=4 SV=1;NADH-flavin reductase [Isoptericola variabilis 225] ;KEGG: mts:MTES_2737 NADH-flavin reductase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NADH(P)-binding; pfam13460;NAD(P) binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_00290	complement(25485..25919)		hypothetical protein [nite]			KSE_56350 :hypothetical protein : : : Kitasatospora setae NBRC 14216	OsmC family protein		A4AA45_9GAMM OsmC-like protein OS=Congregibacter li toralis KT71 GN=KT71_08279 PE=4 SV=1;OsmC family protein [Arthrobacter chlorophenolicus A6];PFAM: OsmC family protein; KEGG: aau:AAur_2390  OsmC-like protein;OsmC-like protein; cl00767
GOHSU12	CDS	GOHSU_12_00300	25976..26698		conserved hypothetical protein [nite]			RER_pREL1-02760 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	transglutaminase domain-containing protein		E6J790_9ACTO Putative uncharacterized protein OS=Di etzia cinnamea P4 GN=ES5_05425 PE=4 SV=1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];PFAM: transglutaminase domain protein; KEGG: eba:ebA2531 hypothetical protein;Transglutaminase-like superfamily; cl10469
GOHSU12	CDS	GOHSU_12_00310	complement(26890..27708)		putative oxidoreductase [nite]			RER_52650 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	carveol dehydrogenase		F9VYP9_9ACTO Putative oxidoreductase OS=Gordonia al kanivorans NBRC 16433 GN=GOALK_092_00020 PE=3 SV=1;carveol dehydrogenase [Mycobacterium avium 104];identified by match to protein family HMM PF00106;3-ketoacyl-(acyl-carrier-protein) reductase; Reviewed;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_00320	complement(27760..28467)		not assigned						
GOHSU12	CDS	GOHSU_12_00330	complement(28585..30180)		hypothetical protein [nite]			RER_29110 :hypothetical protein : : : Rhodococcus erythropolis PR4	13e12 repeat-containing protein		F1YE04_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01930 PE=4 SV=1;13e12 repeat-containing protein [Mycobacterium mari num M];function unknown. thought to be regulated by LexA;Domain of unknown function (DUF222); pfam02720;HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and...; cd00085
GOHSU12	CDS	GOHSU_12_00340	complement(30328..31506)		hypothetical protein [migap]				hypothetical protein		I0WCW5_9NOCA Uncharacterized protein OS=Rhodococcus imtechensis RKJ300 GN=W59_31139 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];similar to AJ414142-106|CAC89255.1| percent identity: 22 in 367 aa;Uncharacterized protein conserved in bacteria N-term (DUF3322); pfam11795;Uncharacterized protein conserved in bacteria [Function unknown];Topoisomerase-primase domain. This is a nucleotidyl transferase/hydrolase domain found in type IA, type IIA and type IIB topoisomerases, bacterial DnaG-type primases, small primase-like proteins from bacteria and archaea, OLD family nucleases from...; cl00718
GOHSU12	CDS	GOHSU_12_00350	complement(31493..34780)		hypothetical protein [migap]				hypothetical protein		I0WCW6_9NOCA Uncharacterized protein OS=Rhodococcus imtechensis RKJ300 GN=W59_31144 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];Uncharacterized protein conserved in bacteria [Function unknown];similar to AE004080-1|AAF85519.1| percent identity: 27 in 1087 aa;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Putative exonuclease SbcCD, C subunit; pfam13558
GOHSU12	CDS	GOHSU_12_00360	complement(34777..35466)		hypothetical protein [migap]				hypothetical protein		H0JYQ0_9NOCA Putative uncharacterized protein OS=Rh odococcus pyridinivorans AK37 GN=AK37_24476 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];Domain of unknown function (DUF4194); pfam13835
GOHSU12	CDS	GOHSU_12_00370	complement(35463..36965)		hypothetical protein [migap]				hypothetical protein		I0WCW8_9NOCA Uncharacterized protein OS=Rhodococcus imtechensis RKJ300 GN=W59_31154 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];similar to AE004079-9|AAF85518.1| percent identity: 27 in 460 aa;Protein of unknown function (DUF3375); pfam11855
GOHSU12	CDS	GOHSU_12_00380	37235..38134		conserved hypothetical protein [nite]			RER_24540 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	LppW protein		C0ZXS7_RHOE4 Putative uncharacterized protein OS=Rh odococcus erythropolis (strain PR4 / NBRC 100887) GN=RER_2 4540 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Penicillin binding protein transpeptidase domain; cl01009
GOHSU12	CDS	GOHSU_12_00390	38180..39766		GMP synthase [glutamine-hydrolyzing] [hamap]	guaA [hamap]	MF_00344:GMP synthase [glutamine-hydrolyzing]:guaA 	ROP_62620 :GMP synthetase : guaA : 6.3.5.2 : Rhodococcus opacus B4	GMP synthase	guaA	F1YFT6_9ACTO GMP synthase [glutamine-hydrolyzing] O S=Gordonia neofelifaecis NRRL B-59395 GN=guaA PE=3 SV=1;GMP synthase [Corynebacterium glutamicum ATCC 13032 ];IS fingerprint type 4-5;contains glutamine-hydrolyzing domain and glutamine amidotransferase; GMP-binding domain; functions to produce GMP from XMP in the IMP pathway;GMP synthase; Reviewed; PRK00074;Type 1 glutamine amidotransferase (GATase1) domain found in GMP synthetase; cd01742;AMP/PPi binding site [chemical binding];candidate oxyanion hole;catalytic triad [active];potential glutamine specificity residues [chemical binding];The C-terminal domain of GMP synthetase. It contains two subdomains; the ATP pyrophosphatase domain which closes to the N-termial and the dimerization domain at C-terminal end. The ATP-PPase is a twisted, five-stranded parallel beta-sheet sandwiched...; cd01997;ATP Binding subdomain [chemical binding];Ligand Binding sites [chemical binding];Dimerization subdomain
GOHSU12	CDS	GOHSU_12_00400	39806..40153		hypothetical protein [nite]			MLP_27740 :hypothetical protein : : : Microlunatus phosphovorus NM-1	serine/threonine-protein phosphatase 1 regulatory subunit 10		E6J955_9ACTO Putative uncharacterized protein OS=Di etzia cinnamea P4 GN=ES5_08781 PE=4 SV=1;serine/threonine-protein phosphatase 1 regulatory s ubunit 10 [Actinomyces viscosus C505]
GOHSU12	CDS	GOHSU_12_00410	complement(40194..40943)		conserved hypothetical protein [nite]			RER_47940 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	ADP-L-glycero-D-manno-heptose-6-epimeraseADP-L- glycero-beta-D-manno-heptose-6-epimerase; ADP-glyceromanno-heptose 6-epimerase; ADP-hep 6-epimerase; AGME		G7GZ49_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_021_01110 PE=4 SV=1;unnamed protein product [Sinorhizobium fredii HH103 ];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NADH(P)-binding; pfam13460;NAD(P) binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_00420	complement(40983..41171)		hypothetical protein [migap]				hypothetical protein		D0L8H9_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_1734 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU12	CDS	GOHSU_12_00430	complement(41171..42460)		hypothetical membrane protein [nite]			ROP_62640 :hypothetical membrane protein : : : Rhodococcus opacus B4	phage shock protein PspC		F1YFT9_9ACTO PspC domain-containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_03157 PE=4 SV=1;phage shock protein PspC [Actinosynnema mirum DSM 4 3827];PFAM: PspC domain protein; KEGG: aba:Acid345_1324 phage shock protein C, PspC;PspC domain; cl00864;Cell wall-active antibiotics response protein (DUF2154); pfam09922
GOHSU12	CDS	GOHSU_12_00440	42578..43849		putative two-component histidine kinase [nite]			ROP_62650 :putative two-component histidine kinase : : 2.7.13.- : Rhodococcus opacus B4	PspC domain-containing protein		F1YFU0_9ACTO PspC domain-containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_03162 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: PspC domain protein; ATP-binding region ATPase domain protein; KEGG: gsu:GSU1292 sensory box histidine kinase;Signal transduction histidine kinase [Signal transduction mechanisms];Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU12	CDS	GOHSU_12_00450	43846..44502		putative NarL family two-component response regulator [nite]			ROP_62700 :putative NarL family two-component response regulator : : : Rhodococcus opacus B4	putative two component transcriptional regulator, LuxR family		F1YFU1_9ACTO Regulatory protein LuxR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_03167 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding]
GOHSU12	CDS	GOHSU_12_00460	complement(44530..45222)		not assigned						H0R3G0_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_092_00360 PE=4 SV=1
GOHSU12	CDS	GOHSU_12_00470	45301..47625		serine/threonine protein kinase [migap]				serine/threonine protein kinase		G7H4L2_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_062_00550 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];SMART: serine/threonine protein kinase; KEGG: afw:Anae109_2802 regulator of chromosome condensation RCC1;Protein Kinases, catalytic domain; cl09925;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop);Regulator of chromosome condensation (RCC1) repeat; pfam00415;Regulator of chromosome condensation (RCC1) repeat; pfam13540
GOHSU12	CDS	GOHSU_12_00480	47770..48498		hypothetical protein [nite]			RER_pREC1-00980 :hypothetical protein : : : Rhodococcus erythropolis PR4	predicted protein		F1YFU6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03192 PE=4 SV=1;predicted protein [Streptomyces sp. AA4]
GOHSU12	CDS	GOHSU_12_00490	48495..50138		hypothetical protein [nite]			ROP_62730 :hypothetical protein : : : Rhodococcus opacus B4	DNA polymerase		F1YFU7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03197 PE=4 SV=1;DNA polymerase [Amycolatopsis mediterranei U32];Y-family of DNA polymerases; cl12025;DNA binding site [nucleotide binding]
GOHSU12	CDS	GOHSU_12_00500	50208..51182		putative oxidoreductase [nite]			ROP_38230 :putative oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase		I0H951_ACTMI Putative dioxygenase OS=Actinoplanes m issouriensis 431 GN=AMIS_43180 PE=4 SV=1;oxidoreductase [Rhodococcus opacus B4];Dioxygenases related to 2-nitropropane dioxygenase [General function prediction only];2-Nitropropane dioxygenase (NPD), one of the nitroalkane oxidizing enzyme families, catalyzes oxidative denitrification of nitroalkanes to their corresponding carbonyl compounds and nitrites. NDP is a member of the NAD(P)H-dependent flavin oxidoreductase...; cd04730;FMN binding site [chemical binding];substrate binding site [chemical binding];putative catalytic residue [active]
GOHSU12	CDS	GOHSU_12_00510	complement(51179..52321)		hypothetical protein [nite]			ANT_27270 :hypothetical protein : : : Anaerolinea thermophila UNI-1	alanine racemase domain-containing protein		F1YFU8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03202 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];PFAM: alanine racemase domain protein; KEGG: cgb:cg2890 putative amino acid processing enzyme;Predicted amino acid aldolase or racemase [Amino acid transport and metabolism];Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzymes, Low Specificity D-Threonine Aldolase-like; cd06820;dimer interface [polypeptide binding];pyridoxal 5'-phosphate (PLP) binding site [chemical binding];substrate binding site [chemical binding];catalytic residue [active]
GOHSU12	CDS	GOHSU_12_00520	complement(52318..54246)		putative oxidoreductase [nite]			RER_19720 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YFU9_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03207 PE=3 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: Male sterility domain; KR domain protein; 3- beta hydroxysteroid dehydrogenase/isomerase; dTDP-4- dehydrorhamnose reductase; short-chain dehydrogenase/reductase SDR; NAD-dependent epimerase/dehydratase; KEGG: rha:RHA1_ro06221 short chain dehydrogenase;short chain dehydrogenase; Provisional;Pseudomonas fluorescens MupV-like, extended (e) SDRs; cd05263;putative NAD(P) binding site [chemical binding];putative substrate binding site [chemical binding];classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_00530	54318..57581		error-prone DNA polymerase [hamap]	dnaE2 [hamap]	MF_01902:error-prone DNA polymerase:dnaE2 	RER_19790 :error-prone DNA polymerase : dnaE2 : 2.7.7.7 : Rhodococcus erythropolis PR4	DNA polymerase III subunit alpha	dnaE	F1YFV0_9ACTO Error-prone DNA polymerase OS=Gordonia neofelifaecis NRRL B-59395 GN=dnaE2 PE=3 SV=1;DNA polymerase III subunit alpha [Amycolatopsis med iterranei U32];error-prone DNA polymerase; Validated; PRK05672;Bacterial DNA polymerase III alpha subunit; pfam07733;DnaE_OBF: A subfamily of OB folds corresponding to the C-terminal OB-fold nucleic acid binding domain of Thermus aquaticus and Escherichia coli type C replicative DNA polymerase III alpha subunit (DnaE). The DNA polymerase holoenzyme of E. coli contains...; cd04485;generic binding surface II;generic binding surface I
GOHSU12	CDS	GOHSU_12_00540	57638..57874	case	UPF0235 protein yggU [hamap]	yggU [hamap]	MF_00634:UPF0235 protein yggU:yggU 	ANT_18430 :hypothetical protein : : : Anaerolinea thermophila UNI-1	hypothetical protein		H6MU87_GORPV UPF0235 protein GPOL_c16340 OS=Gordoni a polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c163 40 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Uncharacterized ACR, YggU family COG1872; cl00811
GOHSU12	CDS	GOHSU_12_00550	57987..58739		glucosamine-6-phosphate deaminase [nite]	 nagB [nite]		RER_12190 :glucosamine-6-phosphate deaminase : nagB : 3.5.99.6 : Rhodococcus erythropolis PR4	glucosamine-6-phosphate deaminase	nagB	F1YFV1_9ACTO Glucosamine-6-phosphate deaminase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_03217 PE=4 SV=1 ;glucosamine-6-phosphate deaminase [Rhodococcus eryt hropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;protein synonym:glucosamine-6-phosphate isomerase;GlcN6P_deaminase: Glucosamine-6-phosphate (GlcN6P) deaminase subfamily; GlcN6P deaminase catalyzes the reversible conversion of GlcN6P to D-fructose-6-phosphate (Fru6P) and ammonium. The reaction is an aldo-keto isomerization coupled with an amination or...; cd01399;trimer interface [polypeptide binding];allosteric site;active site lid [active];hexamer (dimer of trimers) interface [polypeptide binding]
GOHSU12	CDS	GOHSU_12_00560	complement(58790..59275)		putative RNA methyltransferase [nite]			ROP_62910 :putative RNA methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	tRNA (cytidine/uridine-2'-O-)-methyltransferase TrmL	trmL	F1YFV5_9ACTO Putative tRNA (cytidine(34)-2'-O)-meth yltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SC NU_03237 PE=3 SV=1;trmL gene product [Gordonia polyisoprenivorans VH2] ;SpoU rRNA Methylase family; cl00362
GOHSU12	CDS	GOHSU_12_00570	complement(59294..59791)		hemerythrin HHE cation binding domain-containing protein [migap]				hemerythrin HHE cation binding domain-containing protein		D5NVP2_CORAM Putative hemerythrin HHE cation bindin g domain protein OS=Corynebacterium ammoniagenes DSM 20306 GN=HMPREF0281_00648 PE=4 SV=1;hemerythrin HHE cation binding domain-containing pr otein [Mycobacterium sp. JDM601];Hemerythrin (Hr) is a non-heme diiron oxygen transport protein found in four marine invertebrate phyla including priapulida, brachiopoda, sipunculida, and annelida, as well as in protozoa. Myohemerythrin (Mhr), a hemerythrin homolog, is found in the...; cl15774
GOHSU12	CDS	GOHSU_12_00580	complement(59845..60081)		hypothetical membrane protein [nite]			ROP_pROB01-03340 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F9VX96_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_072_00380 PE=4 SV= 1;unnamed protein product [Maricaulis maris MCS10];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.966) with cleavage site probability 0.488 at residue 25;KEGG: sil:SPO0443 hypothetical protein;Protein of unknown function (DUF2892); pfam11127
GOHSU12	CDS	GOHSU_12_00590	60228..61109		bifunctional protein FolD [hamap]	folD [hamap]	MF_01576:bifunctional protein FolD:folD 	RER_19980 :putative methylenetetrahydrofolate dehydrogenase/methenyltetrahydrofolate cyclohydrolase : folD : 1.5.1.5/3.5.4.9 : Rhodococcus erythropolis PR4	bifunctional 5,10-methylene-tetrahydrofolate dehydrogenase/ 5,10-methylene-tetrahydrofolate cyclohydrolase	folD	G7GSE7_9ACTO Bifunctional protein FolD OS=Gordonia amarae NBRC 15530 GN=folD PE=3 SV=1;bifunctional 5,10-methylene-tetrahydrofolate dehydr ogenase/ 5,10-methylene-tetrahydrofolate cyclohydrolase [C orynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;bifunctional 5,10-methylene-tetrahydrofolate dehydrogenase/ 5,10-methylene-tetrahydrofolate cyclohydrolase; Provisional;catalyzes the formation of 5,10-methenyltetrahydrofolate from 5,10-methylenetetrahydrofolate and subsequent formation of 10-formyltetrahydrofolate from 5,10-methenyltetrahydrofolate;Tetrahydrofolate dehydrogenase/cyclohydrolase, catalytic domain; pfam00763;NADP binding domain of methylene-tetrahydrofolate dehydrogenase/cyclohydrolase; cd01080;homodimer interface [polypeptide binding];NADP binding site [chemical binding];substrate binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_00600	61106..61459		hypothetical membrane protein [nite]			ROP_62940 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		D0L8K0_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_1758 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Protein of unknown function (DUF3017); pfam11222
GOHSU12	CDS	GOHSU_12_00610	complement(61504..62637)		homoserine O-acetyltransferase [hamap]	metX [hamap]	MF_00296:homoserine O-acetyltransferase:metX 	RER_20020 :homoserine O-acetyltransferase : metX : 2.3.1.31 : Rhodococcus erythropolis PR4	homoserine O-acetyltransferase	metX	F1YFW2_9ACTO Homoserine O-acetyltransferase OS=Gord onia neofelifaecis NRRL B-59395 GN=metX PE=3 SV=1;homoserine O-acetyltransferase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Homoserine acetyltransferase [Amino acid transport and metabolism]; COG2021;Catalyzes the conversion of acetyl-CoA and L-homoserine to CoA and O-acetyl-L-homoserine
GOHSU12	CDS	GOHSU_12_00620	complement(62671..63972)		O-acetylhomoserine (thiol)-lyase [nite]	 metY [nite]		ROP_51310 :O-acetylhomoserine (thiol)-lyase : metY : 2.5.1.49 : Rhodococcus opacus B4	O-acetylhomoserine aminocarboxypropyltransferase	metY	F1YFW3_9ACTO O-acetylhomoserine aminocarboxypropylt ransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 03277 PE=3 SV=1;O-acetylhomoserine aminocarboxypropyltransferase [R hodococcus opacus B4];O-acetylhomoserine aminocarboxypropyltransferase; Validated;catalyzes the formation of L-methionine and acetate from O-acetyl-L-homoserine and methanethiol;CGS_like: Cystathionine gamma-synthase is a PLP dependent enzyme and catalyzes the committed step of methionine biosynthesis. This pathway is unique to microorganisms and plants, rendering the enzyme an attractive target for the development of...; cd00614;homodimer interface [polypeptide binding];substrate-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU12	CDS	GOHSU_12_00630	64431..66116		hypothetical protein [migap]				hypothetical protein		D0L8K4_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_1762 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: YALI0B09867p;PE-PPE domain; pfam08237
GOHSU12	CDS	GOHSU_12_00640	66222..66656		hypothetical protein [nite]			ROP_55810 :hypothetical protein : : : Rhodococcus opacus B4	cation transport regulator		Q0S599_RHOSR Possible cation transport regulator OS =Rhodococcus sp. (strain RHA1) GN=RHA1_ro05507 PE=4 SV=1;cation transport regulator [Rhodococcus jostii RHA1 ];ChaB; cl01887;Rho termination factor, N-terminal domain; cl06505
GOHSU12	CDS	GOHSU_12_00650	complement(66693..68000)		guanine deaminase [nite]	 guaD [nite]		ACMV_07480 :guanine deaminase : guaD : 3.5.4.3 : Acidiphilium multivorum AIU301	guanine deaminase	guaD	E2Q197_STRCL Guanine deaminase OS=Streptomyces clav uligerus ATCC 27064 GN=guaD PE=4 SV=1;guanine deaminase [Gluconacetobacter diazotrophicus PAl 5];guanine deaminase; Provisional;Guanine deaminase (GDEase). Guanine deaminase is an aminohydrolase responsible for the conversion of guanine to xanthine and ammonia, the first step to utilize guanine as a nitrogen source. This reaction also removes the guanine base from the pool and...; cd01303
GOHSU12	CDS	GOHSU_12_00660	68108..68938		exodeoxyribonuclease [nite]	 exoA [nite]		ROP_62990 :exodeoxyribonuclease : exoA : 3.1.11.2 : Rhodococcus opacus B4	exodeoxyribonuclease III Xth	xth1	F1YFW9_9ACTO Exodeoxyribonuclease III OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_03307 PE=4 SV=1;xth1 gene product [Gordonia polyisoprenivorans VH2] ;Exonuclease-Endonuclease-Phosphatase (EEP) domain superfamily; cl00490;putative catalytic site [active];putative metal binding site [ion binding];putative phosphate binding site [ion binding]
GOHSU12	CDS	GOHSU_12_00670	68973..70007		tryptophanyl-tRNA synthetase [nite]	 trpS [nite]		RER_20100 :tryptophanyl-tRNA synthetase : trpS : 6.1.1.2 : Rhodococcus erythropolis PR4	tryptophanyl-tRNA synthetase	trpS	G7GYP1_9ACTO Tryptophanyl-tRNA synthetase OS=Gordon ia araii NBRC 100433 GN=trpS PE=3 SV=1;tryptophanyl-tRNA synthetase [Corynebacterium gluta micum ATCC 13032];IS fingerprint type 4-5;catalyzes a two-step reaction, first charging a tryptophan molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA;tryptophanyl-tRNA synthetase II; Reviewed;catalytic core domain of tryptophanyl-tRNA synthetase; cd00806;HIGH motif;dimer interface [polypeptide binding];KMSKS motif
GOHSU12	CDS	GOHSU_12_00680	70018..71091		putative ribonuclease [nite]			ROP_63010 :putative ribonuclease : : 3.1.-.- : Rhodococcus opacus B4	predicted protein		F9VWR9_9ACTO Putative ribonuclease OS=Gordonia alka nivorans NBRC 16433 GN=GOALK_067_00170 PE=4 SV=1;predicted protein [Streptomyces sp. AA4];Virulence factor BrkB; cl07918
GOHSU12	CDS	GOHSU_12_00690	complement(71152..72438)		putative D-alanyl-D-alanine carboxypeptidase [nite]	 dac [nite]		ROP_63020 :putative D-alanyl-D-alanine carboxypeptidase : dac : 3.4.16.4 : Rhodococcus opacus B4	D-alanyl-D-alanine carboxypeptidase		F1YFX4_9ACTO Peptidase S11 D-alanyl-D-alanine carbo xypeptidase 1 OS=Gordonia neofelifaecis NRRL B-59395 GN=SC NU_03332 PE=3 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00768;D-alanyl-D-alanine carboxypeptidase; pfam00768;Penicillin binding protein transpeptidase domain; cl01009
GOHSU12	CDS	GOHSU_12_00700	complement(72528..73310)		succinate dehydrogenase iron-sulfur protein [nite]	 sdhB [nite]		ROP_63040 :succinate dehydrogenase iron-sulfur protein : sdhB : 1.3.99.1 : Rhodococcus opacus B4	succinate dehydrogenase iron-sulfur subunit	sdhB	F1YFX5_9ACTO Succinate dehydrogenase iron-sulfur su bunit OS=Gordonia neofelifaecis NRRL B-59395 GN=sdhB PE=4 SV=1;succinate dehydrogenase iron-sulfur subunit [Rhodoc occus opacus B4];part of four member succinate dehydrogenase enzyme complex that forms a trimeric complex (trimer of tetramers); SdhA/B are the catalytic subcomplex and can exhibit succinate dehydrogenase activity in the absence of SdhC/D which are the membrane components and form cytochrome b556; SdhC binds ubiquinone; oxidizes succinate to fumarate while reducing ubiquinone to ubiquinol; the catalytic subunits are similar to fumarate reductase;succinate dehydrogenase iron-sulfur subunit; Reviewed; PRK05950;2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cl00159
GOHSU12	CDS	GOHSU_12_00710	complement(73310..75064)		succinate dehydrogenase flavoprotein subunit [nite]	 sdhA [nite]		ROP_63050 :succinate dehydrogenase flavoprotein subunit : sdhA : 1.3.99.1 : Rhodococcus opacus B4	succinate dehydrogenase, flavoprotein subunit SdhA	sdhA1	F1YFX6_9ACTO Succinate dehydrogenase flavoprotein s ubunit OS=Gordonia neofelifaecis NRRL B-59395 GN=sdhA PE=4 SV=1;sdhA1 gene product [Gordonia polyisoprenivorans VH2 ];succinate dehydrogenase flavoprotein subunit; Reviewed; PRK08205;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Archaeal flagella protein; cl01322;Fumarate reductase flavoprotein C-term; pfam02910
GOHSU12	CDS	GOHSU_12_00720	complement(75081..75515)		succinate dehydrogenase hydrophobic membrane anchor protein [nite]	 sdhD [nite]		ROP_63060 :succinate dehydrogenase hydrophobic membrane anchor protein : sdhD : : Rhodococcus opacus B4	succinate dehydrogenase	sdhD_3	F1YFX7_9ACTO Succinate dehydrogenase hydrophobic me mbrane anchor protein OS=Gordonia neofelifaecis NRRL B-593 95 GN=SCNU_03347 PE=4 SV=1;succinate dehydrogenase [Mycobacterium bovis BCG st r. Tokyo 172];Differs from Rv3317 by 1aa, L54V in Mycobacterium bovis, Mycobacterium bovis BCG Pasteur, and BCG Tokyo; hydrophobic membrane anchor subunit;Succinate:quinone oxidoreductase (SQR) Type A subfamily, Succinate dehydrogenase D (SdhD)-like subunit; SQR catalyzes the oxidation of succinate to fumarate coupled to the reduction of quinone to quinol. Members of this subfamily reduce low potential...; cd03500;putative Iron-sulfur protein interface [polypeptide binding];putative proximal heme binding site [chemical binding];putative SdhC-like subunit interface [polypeptide binding];putative distal heme binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_00730	complement(75531..75941)		probable succinate dehydrogenase cytochrome b subunit [nite]	 sdhC [nite]		RER_20450 :probable succinate dehydrogenase cytochrome b subunit : sdhC : 1.3.99.1 : Rhodococcus erythropolis PR4	succinate dehydrogenase	sdhC_3	H0R7B1_9ACTO Succinate dehydrogenase cytochrome b s ubunit OS=Gordonia polyisoprenivorans NBRC 16320 GN=sdhC P E=4 SV=1;succinate dehydrogenase [Mycobacterium bovis BCG st r. Tokyo 172];Succinate:quinone oxidoreductase (SQR) Type A subfamily, Succinate dehydrogenase C (SdhC)-like subunit; SQR catalyzes the oxidation of succinate to fumarate coupled to the reduction of quinone to quinol. Members of this subfamily reduce low potential...; cd03501;putative Iron-sulfur protein interface [polypeptide binding];cytochrome b-556 subunit;putative proximal heme binding site [chemical binding];putative SdhD-like interface [polypeptide binding];putative distal heme binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_00740	76135..76545		cytidine deaminase [nite]	 cdd [nite]		ROP_63080 :cytidine deaminase : cdd : 3.5.4.5 : Rhodococcus opacus B4	cytidine deaminase	cdd_3	G7GSG3_9ACTO Cytidine deaminase OS=Gordonia amarae NBRC 15530 GN=cdd PE=4 SV=1;cytidine deaminase [Mycobacterium bovis BCG str. To kyo 172];Reclaims exogenous and endogenous cytidine and 2'-deoxycytidine molecules for UMP synthesis;Cytidine deaminase zinc-binding domain. These enzymes are Zn dependent. The zinc ion in the active site plays a central role in the proposed catalytic mechanism, activating a water molecule to form a hydroxide ion that performs a nucleophilic attack on...; cd01283;catalytic motif [active];Zn binding site [ion binding]
GOHSU12	CDS	GOHSU_12_00750	76542..77852		pyrimidine nucleoside phosphorylase [nite]	 pdp [nite]		RER_20470 :pyrimidine nucleoside phosphorylase : pdp : 2.4.2.2 : Rhodococcus erythropolis PR4	thymidine phosphorylase	deoA	H0R0L4_9ACTO Pyrimidine nucleoside phosphorylase OS =Gordonia effusa NBRC 100432 GN=pdp PE=4 SV=1;thymidine phosphorylase [Mycobacterium bovis BCG st r. Tokyo 172];Catalyzes the reversible phosphorolysis of thymidine, deoxyuridine and their analogues to their respective bases and 2-deoxyribose 1-phosphate;thymidine phosphorylase; Reviewed; PRK05820;Glycosyl transferase family, helical bundle domain; pfam02885;Glycosyl transferase family, a/b domain; pfam00591;Pyrimidine nucleoside phosphorylase C-terminal domain; cl06734
GOHSU12	CDS	GOHSU_12_00760	complement(77849..78646)		hypothetical membrane protein [nite]			ROP_41050 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YFY1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03367 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted integral membrane protein (DUF2189); cl02291
GOHSU12	CDS	GOHSU_12_00770	78712..79830		adenosine deaminase [hamap]	add [hamap]	MF_00540:adenosine deaminase:add 	ROP_63100 :adenosine deaminase : add : 3.5.4.4 : Rhodococcus opacus B4	adenosine deaminase	add_3	H0R7A8_9ACTO Adenosine deaminase OS=Gordonia polyis oprenivorans NBRC 16320 GN=add PE=3 SV=1;adenosine deaminase [Mycobacterium bovis BCG str. T okyo 172];catalyzes the formation of inosine from adenosine;Adenosine deaminase [Nucleotide transport and metabolism]; COG1816;Superfamily of metallo-dependent hydrolases (also called amidohydrolase superfamily) is a large group of proteins that show conservation in their 3-dimensional fold (TIM barrel) and in details of their active site. The vast majority of the members have a...; cl00281
GOHSU12	CDS	GOHSU_12_00780	complement(79831..81579)		ATPase [migap]				ATPase		G7GYB4_9ACTO Putative ATP-dependent Clp protease AT P-binding subunit OS=Gordonia araii NBRC 100433 GN=GOARA_0 13_00330 PE=4 SV=1;ATPase [Thermotoga lettingae TMO];PFAM: AAA ATPase central domain protein; Clp domain protein; ATPase associated with various cellular activities AAA_5; ATPase AAA-2 domain protein; SMART: AAA ATPase; KEGG: tma:TM1391 ATP-dependent Clp protease, ATPase subunit;ATP-binding subunits of Clp protease and DnaK/DnaJ chaperones [Posttranslational modification, protein turnover, chaperones]; COG0542;arginine finger;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;C-terminal, D2-small domain, of ClpB protein; cl11083
GOHSU12	CDS	GOHSU_12_00790	complement(81771..83030)		hypothetical protein [nite]			ROP_63110 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YFY4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03382 PE=4 SV=1;unnamed protein product [Corynebacterium diphtheria e 241]
GOHSU12	CDS	GOHSU_12_00800	complement(83107..84012)		hypothetical protein [nite]			RER_20710 :hypothetical protein : : : Rhodococcus erythropolis PR4	NlpC/P60 family protein		D0L8M4_GORB4 NLP/P60 protein OS=Gordonia bronchiali s (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1783 PE=4 SV=1;NlpC/P60 family protein [Rhodococcus opacus B4];NlpC/P60 family; cl11438
GOHSU12	CDS	GOHSU_12_00810	complement(84012..84326)		hypothetical protein [migap]				hypothetical protein		H5U4V6_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_115_00210 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU12	CDS	GOHSU_12_00820	84385..85029		uracil phosphoribosyltransferase [nite]	 upp [nite]		RER_20730 :uracil phosphoribosyltransferase : upp : 2.4.2.9 : Rhodococcus erythropolis PR4	uracil phosphoribosyltransferase	upp	H6MUC6_GORPV Uracil phosphoribosyltransferase Upp O S=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN= upp PE=3 SV=1;uracil phosphoribosyltransferase [Mycobacterium bov is BCG str. Tokyo 172];Catalyzes the formation of uracil and 5-phospho-alpha-D-ribosy 1-diphosphate from UMP and diphosphate;Phosphoribosyl transferase (PRT)-type I domain; cd06223
GOHSU12	CDS	GOHSU_12_00830	85121..85888		not assigned						A1T419_MYCVP Putative uncharacterized protein (Prec ursor) OS=Mycobacterium vanbaalenii (strain DSM 7251 / PYR -1) GN=Mvan_1082 PE=4 SV=1
GOHSU12	CDS	GOHSU_12_00840	complement(85920..86735)		protein kinase [migap]				protein kinase		F1YFZ3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03427 PE=4 SV=1;protein kinase [Rhodococcus jostii RHA1]
GOHSU12	CDS	GOHSU_12_00850	86845..87540		putative enoyl-CoA hydratase [nite]			RER_20760 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase/isomerase family protein		D0L8N0_GORB4 Enoyl-CoA hydratase/isomerase OS=Gordo nia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1789 PE=4 SV=1;enoyl-CoA hydratase/isomerase family protein [Rhodo coccus opacus B4];Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU12	CDS	GOHSU_12_00860	87540..88751		putative amidohydrolase [nite]			ROP_63210 :putative amidohydrolase : : : Rhodococcus opacus B4	N-acyl-L-amino acid amidohydrolase	amiA1_3	F1YFZ7_9ACTO Amidohydrolase OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_03447 PE=4 SV=1;N-acyl-L-amino acid amidohydrolase [Mycobacterium b ovis BCG str. Tokyo 172];Metal-dependent amidase/aminoacylase/carboxypeptidase [General function prediction only]; COG1473;M20 Peptidase Aminoacylase 1 subfamily; cd08014;metal binding site [ion binding];putative dimer interface [polypeptide binding]
GOHSU12	CDS	GOHSU_12_00870	complement(88786..89277)		hypothetical protein [nite]			ROP_63220 :hypothetical protein : : : Rhodococcus opacus B4	AIG2-like family protein		F1YFZ8_9ACTO AIG2 family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_03452 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: AIG2-like family;GGCT-like domains, also called AIG2-like family. Gamma-glutamyl cyclotransferase (GGCT) catalyzes the formation of pyroglutamic acid (5-oxoproline) from dipeptides containing gamma-glutamyl, and is a dimeric protein. In Homo sapiens, the protein is...; cd06661;putative active site pocket [active];dimerization interface [polypeptide binding];putative catalytic residue [active]
GOHSU12	CDS	GOHSU_12_00880	89363..90781		putative oxidoreductase [nite]			RER_20810 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	flavoprotein disulfide reductase	lpdA_3	F1YFZ9_9ACTO Flavoprotein disulfide reductase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03457 PE=4 SV=1;flavoprotein disulfide reductase [Mycobacterium bov is BCG str. Tokyo 172];catalyzes the reduction of nonspecific electron acceptors such as 2,6-dimethyl-1,4-benzoquinone and 5-hydroxy-1,4-naphthaquinone; does not have lipoamide dehydrogenase activity;flavoprotein disulfide reductase; Reviewed;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Pyridine nucleotide-disulphide oxidoreductase; pfam00070;Pyridine nucleotide-disulphide oxidoreductase, dimerisation domain; pfam02852
GOHSU12	CDS	GOHSU_12_00890	90862..92250		putative drug resistance efflux protein [nite]			RER_55050 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	major facilitator family transporter		F1YG00_9ACTO Multidrug ABC transporter OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03462 PE=4 SV=1;unnamed protein product [Xenorhabdus bovienii SS- 2 004];The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU12	CDS	GOHSU_12_00900	complement(92225..93748)		glycerol kinase [hamap]	glpK [hamap]	MF_00186:glycerol kinase:glpK 	RER_20870 :glycerol kinase : glpK : 2.7.1.30 : Rhodococcus erythropolis PR4	glycerol kinase	glpK	F1YG03_9ACTO Glycerol kinase OS=Gordonia neofelifae cis NRRL B-59395 GN=glpK PE=3 SV=1;glycerol kinase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;glycerol kinase; Provisional; PRK00047;protein synonym:glycerokinase;Cellulomonas sp. glycerol kinase-like proteins; belongs to the FGGY family of carbohydrate kinases; cd07789;N- and C-terminal domain interface [polypeptide binding];putative active site [active];putative MgATP binding site [chemical binding];catalytic site [active];metal binding site [ion binding];putative homotetramer interface [polypeptide binding];putative glycerol binding site [chemical binding];homodimer interface [polypeptide binding]
GOHSU12	CDS	GOHSU_12_00910	93798..95552		putative glycerol-3-phosphate dehydrogenase [nite]	 glpD [nite]		ROP_63260 :putative glycerol-3-phosphate dehydrogenase : glpD : 1.1.99.5 : Rhodococcus opacus B4	glycerol-3-phosphate dehydrogenase	glpD2_3	F1YG04_9ACTO FAD dependent oxidoreductase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_03482 PE=3 SV=1;glycerol-3-phosphate dehydrogenase [Mycobacterium b ovis BCG str. Tokyo 172];Glycerol-3-phosphate dehydrogenase [Energy production and conversion]; COG0578
GOHSU12	CDS	GOHSU_12_00920	complement(95584..96099)		hypothetical protein [nite]			MLP_01950 :hypothetical protein : : : Microlunatus phosphovorus NM-1	amino acid-binding ACT domain-containing protein		F1YPU7_9ACTO Amino acid-binding ACT domain protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_19807 PE=4 SV=1;amino acid-binding ACT domain-containing protein [S hewanella halifaxensis HAW- EB4];PFAM: amino acid-binding ACT domain protein; KEGG: slo:Shew_1697 amino acid-binding ACT domain protein;Glycine cleavage system regulatory protein [Amino acid transport and metabolism]; COG2716;ACT domains are commonly involved in specifically binding an amino acid or other small ligand leading to regulation of the enzyme; cl09141
GOHSU12	CDS	GOHSU_12_00930	96284..96700		hypothetical protein [nite]			YM304_17310 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	flavin-nucleotide-binding protein		H5UGE4_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_072_00250 PE=4 SV=1;flavin-nucleotide-binding protein [Arthrobacter phe nanthrenivorans Sphe3];Pyridoxine 5'-phosphate (PNP) oxidase-like proteins; cl00381
GOHSU12	CDS	GOHSU_12_00940	complement(96823..97680)		hypothetical protein [migap]				hypothetical protein		E2MYS9_9CORY Putative uncharacterized protein OS=Co rynebacterium amycolatum SK46 GN=CORAM0001_1648 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];Trypsin-like serine protease; Many of these are synthesized as inactive precursor zymogens that are cleaved during limited proteolysis to generate their active forms. Alignment contains also inactive enzymes that have substitutions of the catalytic triad...; cl00149
GOHSU12	CDS	GOHSU_12_00950	complement(97770..99239)		not assigned						
GOHSU12	CDS	GOHSU_12_00960	complement(99366..100181)		putative DNA glycosylase [nite]			RER_20980 :putative DNA glycosylase : : 3.2.2.- : Rhodococcus erythropolis PR4	endonuclease VIII nei	nei_1	F1YG10_9ACTO Formamidopyrimidine-DNA glycosylase ca talytic domain-containing protein OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_03512 PE=4 SV=1;endonuclease VIII nei [Mycobacterium bovis BCG str. Pasteur 1173P2];Formamidopyrimidine-DNA glycosylase [DNA replication, recombination, and repair]; COG0266;N-terminal domain of the actinomycetal Nei2 and related DNA glycosylases; cd08971;putative DNA binding site [nucleotide binding];catalytic residue [active];putative H2TH interface [polypeptide binding];putative catalytic residues [active];Formamidopyrimidine-DNA glycosylase H2TH domain; pfam06831
GOHSU12	CDS	GOHSU_12_00970	complement(100174..104832)		putative ATP-dependent helicase [nite]			ROP_63320 :putative ATP-dependent helicase : : 3.6.1.- : Rhodococcus opacus B4	DEAD/DEAH box helicase		F1YG11_9ACTO DEAD/H associated domain-containing pr otein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03517 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: DEAD/H associated domain protein; DEAD/DEAH box helicase; helicase; SMART: DEAD/DEAH box helicase ; helicase; KEGG: eca:ECA1589  ATP-dependent helicase Lhr;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;putative ATP-dependent helicase Lhr; Provisional;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];DEAD/H associated; pfam08494;Protein of unknown function (DUF1006); cl15826
GOHSU12	CDS	GOHSU_12_00980	105062..105313		hypothetical protein [nite]			MLP_03160 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		H0R0Q3_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_060_00470 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU12	CDS	GOHSU_12_00990	105338..106165		hypothetical protein [nite]			KSE_69320 :hypothetical protein : : : Kitasatospora setae NBRC 14216	putative transmembrane protein		F1YG13_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03527 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Probable cobalt transporter subunit (CbtA); cl02266
GOHSU12	CDS	GOHSU_12_01000	106173..106706		phosphoglycerate mutase family protein [nite]			YM304_18980 :phosphoglycerate mutase family protein : : : Acidimicrobidae bacterium YM16-304	phosphoglycerate mutase		D0L8P6_GORB4 Phosphoglycerate mutase OS=Gordonia br onchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1805 PE=4 SV=1;phosphoglycerate mutase [Streptomyces griseoflavus Tu4000];Histidine phosphatase domain found in a functionally diverse set of proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cl11399
GOHSU12	CDS	GOHSU_12_01010	106818..107135		OmpA/MotB domain-containing protein [migap]				OmpA/MotB domain-containing protein		F8A6L2_CELGA Putative uncharacterized protein OS=Ce llvibrio gilvus (strain ATCC 13127 / NRRL B-14078) GN=Celg i_3008 PE=4 SV=1;OmpA/MotB domain-containing protein [Rhodopseudomon as palustris BisA53];PFAM: OmpA/MotB domain protein; KEGG: rpb:RPB_3590 OmpA/MotB
GOHSU12	CDS	GOHSU_12_01020	107334..107570		predicted protein [migap]				predicted protein		predicted protein [Streptomyces sp. AA4];ABC-2 type transporter; cl11417
GOHSU12	CDS	GOHSU_12_01030	complement(107810..109096)		putative hydrolase [nite]			SLG_20070 :putative hydrolase : : : Sphingobium sp. SYK-6	beta-lactamase		G7CCP8_MYCTH Beta-lactamase OS=Mycobacterium thermo resistibile ATCC 19527 GN=KEK_03722 PE=4 SV=1;beta-lactamase [Mycobacterium sp. JLS];PFAM: beta-lactamase; KEGG: mmc:Mmcs_5183 beta-lactamase;Beta-lactamase class C and other penicillin binding proteins [Defense mechanisms]; COG1680;Penicillin binding protein transpeptidase domain; cl01009
GOHSU12	CDS	GOHSU_12_01040	109182..110378		putative acyl-CoA dehydrogenase [nite]			RER_48380 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase	fadE11	F1YG16_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 03542 PE=3 SV=1;acyl-CoA dehydrogenase [Nocardia farcinica IFM 1015 2];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU12	CDS	GOHSU_12_01050	complement(110383..111897)		putative acyl-CoA synthetase [nite]			KSE_74040 :putative acyl-CoA synthetase : : 6.2.1.- : Kitasatospora setae NBRC 14216	substrate-CoA ligase		D9V7A5_9ACTO O-succinylbenzoate-CoA ligase OS=Strep tomyces sp. AA4 GN=SSMG_02969 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU12	CDS	GOHSU_12_01060	complement(111989..113512)		sodium/proline symporter [nite]			KRH_05020 :sodium/proline symporter : : : Kocuria rhizophila DC2201	sodium/proline symporter		G7H3X6_9ACTO Sodium/proline symporter OS=Gordonia a raii NBRC 100433 GN=GOARA_060_00020 PE=3 SV=1;sodium/proline symporter [Bacillus thuringiensis BM B171];Bacillus thuringiensis serovar kurstaki;Sodium:solute symporter family; cl00456
GOHSU12	CDS	GOHSU_12_01070	complement(113571..115082)		monooxygenase [nite]			ROP_26440 :monooxygenase : : : Rhodococcus opacus B4	monooxygenase		H0R0Q7_9ACTO Putative monooxygenase OS=Gordonia eff usa NBRC 100432 GN=GOEFS_060_00510 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: nfa:nfa12780  monooxygenase;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU12	CDS	GOHSU_12_01080	115223..116476		putative CdaR family transcriptional regulator [nite]			ROP_01020 :putative CdaR family transcriptional regulator : : : Rhodococcus opacus B4	hypothetical protein		G7H6K3_9ACTO Putative CdaR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_073_00100 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Regulator of polyketide synthase expression [Signal transduction mechanisms / Secondary metabolites biosynthesis, transport, and catabolism];Helix-turn-helix domains; cl00088
GOHSU12	CDS	GOHSU_12_01090	complement(116461..117891)		putative Xre family DNA-binding protein [nite]			ROP_30480 :putative Xre family DNA-binding protein : : : Rhodococcus opacus B4	XRE family transcriptional regulator		G7GR70_9ACTO Putative Xre family DNA-binding protei n OS=Gordonia amarae NBRC 15530 GN=GOAMR_48_00040 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted transcriptional regulators [Transcription]; COG1396;Helix-turn-helix XRE-family like proteins. Prokaryotic DNA binding proteins belonging to the xenobiotic response element family of transcriptional regulators; cd00093;non-specific DNA binding site [nucleotide binding];salt bridge;sequence-specific DNA binding site [nucleotide binding];Predicted transcriptional regulator [General function prediction only];Domain of unknown function (DUF955); cl01076;Predicted transcriptional regulator (DUF2083); pfam09856
GOHSU12	CDS	GOHSU_12_01100	118010..119512		2-methylcitrate dehydratase [nite]	 prpD [nite]		ROP_30470 :2-methylcitrate dehydratase : prpD : 4.2.1.79 : Rhodococcus opacus B4	2-methylcitrate dehydratase	prpD	F1YF66_9ACTO 2-methylcitrate dehydratase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_03602 PE=4 SV=1;2-methylcitrate dehydratase [Rhodococcus opacus B4] ;MmgE/PrpD family; cl00912
GOHSU12	CDS	GOHSU_12_01110	119516..120442		probable 2-methylisocitrate lyase [nite]	 prpB [nite]		RER_50300 :probable 2-methylisocitrate lyase : prpB : 4.1.3.30 : Rhodococcus erythropolis PR4	methylisocitric acid lyase	prpB1	H5ULP3_9ACTO 2-methylisocitrate lyase OS=Gordonia t errae NBRC 100016 GN=prpB PE=4 SV=1;methylisocitric acid lyase [Corynebacterium glutami cum ATCC 13032];IS fingerprint type 4-5;PEP phosphonomutase and related enzymes [Carbohydrate transport and metabolism]; COG2513;Members of the ICL/PEPM enzyme family catalyze either P-C or C-C bond formation/cleavage. Known members are phosphoenolpyruvate mutase (PEPM), phosphonopyruvate hydrolase (PPH), carboxyPEP mutase (CPEP mutase), oxaloacetate hydrolase (OAH), isocitrate...; cd00377;tetramer interface [polypeptide binding];Mg2+/Mn2+ binding site [ion binding]
GOHSU12	CDS	GOHSU_12_01120	120446..121561		2-methylcitrate synthase [nite]	 prpC [nite]		ROP_30450 :2-methylcitrate synthase : prpC : 2.3.3.5 : Rhodococcus opacus B4	citrate synthase	prpC2	H0R6R1_9ACTO 2-methylcitrate synthase/citrate synth ase OS=Gordonia effusa NBRC 100432 GN=prpC PE=3 SV=1;citrate synthase [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;citrate synthase; Provisional;catalyzes the formation of citrate from acetyl-CoA and oxaloacetate;Cold-active citrate synthase (CS) from an Antarctic bacterial strain DS2-3R (Ds)-like. CS catalyzes the condensation of acetyl coenzyme A (AcCoA) and oxalacetate (OAA) to form citrate and coenzyme A (CoA), the first step in the citric acid cycle (TCA or...; cd06111;dimer interface [polypeptide binding];citrylCoA binding site [chemical binding];oxalacetate/citrate binding site [chemical binding];coenzyme A binding site [chemical binding];catalytic triad [active]
GOHSU12	CDS	GOHSU_12_01130	121611..125000		pyruvate carboxylase [nite]	 pyc [nite]		ROP_05140 :pyruvate carboxylase : pyc : 6.4.1.1 : Rhodococcus opacus B4	pyruvate carboxylase	pyc	D0L8Q6_GORB4 Pyruvate carboxylase OS=Gordonia bronc hialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10 667) GN=Gbro_1816 PE=3 SV=1;pyc gene product [Mycobacterium smegmatis str. MC2 155];pyruvate carboxylase; Reviewed;biotin-containing enzyme that catalyzes a two step carboxylation of pyruvate to oxaloacetate;Carbamoyl-phosphate synthase L chain, N-terminal domain; pfam00289;ATP-grasp domain; cl03087;Biotin carboxylase C-terminal domain; cl08365;Pyruvate carboxylase and Transcarboxylase 5S, carboxyltransferase domain; cd07937;catalytic residues [active];metal binding site [ion binding];homodimer binding site [polypeptide binding];The biotinyl-domain or biotin carboxyl carrier protein (BCCP) domain is present in all biotin-dependent enzymes, such as acetyl-CoA carboxylase, pyruvate carboxylase, propionyl-CoA carboxylase, methylcrotonyl-CoA carboxylase, geranyl-CoA carboxylase; cd06850;carboxyltransferase (CT) interaction site;biotinylation site [posttranslational modification]
GOHSU12	CDS	GOHSU_12_01140	125023..125625		conserved hypothetical protein [nite]			RER_32300 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	NUDIX hydrolase		F1YF70_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_03622 PE=3 SV=1;NUDIX hydrolase [Brucella pinnipedialis B2/94];Coenzyme A pyrophosphatase (CoAse), a member of the Nudix hydrolase superfamily, functions to catalyze the elimination of oxidized inactive CoA, which can inhibit CoA-utilizing enzymes. The need of CoAses mainly arises under conditions of oxidative...; cd03426;putative active site [active];putative CoA binding site [chemical binding];nudix motif;metal binding site [ion binding]
GOHSU12	CDS	GOHSU_12_01150	125893..126846		hypothetical protein [nite]			MLP_42990 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YPJ5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19250 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF4095); pfam13338
GOHSU12	CDS	GOHSU_12_01160	complement(127017..129287)		5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase [hamap]	metE [hamap]	MF_00172:5-methyltetrahydropteroyltriglutamate--homocysteine methyltransferase:metE 	RER_47140 :cobalamin-independent methionine synthase : metE : 2.1.1.14 : Rhodococcus erythropolis PR4	5-methyltetrahydropteroyltriglutamate-- homocysteine S-methyltransferase	metE	F1YF72_9ACTO 5-methyltetrahydropteroyltriglutamate- -homocysteine methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=metE PE=3 SV=1;5-methyltetrahydropteroyltriglutamate/homocysteine S-methyltransferase [Saccharopolyspora erythraea NRRL 2338 ];catalyzes the transfer of a methyl group from 5-methyltetrahydrofolate to homocysteine to form methionine;5-methyltetrahydropteroyltriglutamate--homocysteine S-methyltransferase; Provisional;CIMS - Cobalamine-independent methonine synthase, or MetE, N-terminal domain_like. Many members have been characterized as 5-methyltetrahydropteroyltriglutamate-homocysteine methyltransferases, EC:2.1.1.14, mostly from bacteria and plants. This enzyme...; cd03312;THF binding site;CIMS - Cobalamine-independent methonine synthase, or MetE, C-terminal domain_like. Many members have been characterized as 5-methyltetrahydropteroyltriglutamate-homocysteine methyltransferases, EC:2.1.1.14, mostly from bacteria and plants. This enzyme...; cd03311;substrate binding site [chemical binding];zinc-binding site [ion binding]
GOHSU12	CDS	GOHSU_12_01170	129834..130415		hypothetical protein [nite]			ROP_44210 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YF73_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03637 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: pau:PA14_61110 hypothetical protein;GT_2_like_f is a subfamily of the glycosyltransferase family 2 (GT-2) with unknown function; cd04182;Ligand binding site
GOHSU12	CDS	GOHSU_12_01180	complement(130638..136100)		acyl-CoA carboxylase [nite]			MLP_10160 :acyl-CoA carboxylase : : : Microlunatus phosphovorus NM-1	carbamoyl-phosphate synthase L chain ATP- binding		G7H1P9_9ACTO Acyl-CoA carboxylase OS=Gordonia araii NBRC 100433 GN=accA PE=4 SV=1;carbamoyl-phosphate synthase L chain ATP- binding [ Catenulispora acidiphila DSM 44928];Carbamoyl-phosphate synthase L chain, N-terminal domain; pfam00289;PFAM: carbamoyl-phosphate synthase L chain ATP- binding; biotin carboxylase domain protein; ATP-dependent carboxylate-amine ligase domain protein ATP-grasp; carboxyl transferase; biotin/lipoyl attachment domain- containing protein; carbamoyl-phosphate synthetase large chain domain protein; KEGG: tfu:Tfu_1530 acetyl/propionyl-CoA carboxylase subunit alpha;acetyl-CoA carboxylase biotin carboxylase subunit; Validated;ATP-grasp domain; cl03087;Biotin carboxylase C-terminal domain; cl08365;The biotinyl-domain or biotin carboxyl carrier protein (BCCP) domain is present in all biotin-dependent enzymes, such as acetyl-CoA carboxylase, pyruvate carboxylase, propionyl-CoA carboxylase, methylcrotonyl-CoA carboxylase, geranyl-CoA carboxylase; cd06850;carboxyltransferase (CT) interaction site;biotinylation site [posttranslational modification];Acetyl-CoA carboxylase, central region; pfam08326;Acetyl-CoA carboxylase, carboxyltransferase component (subunits alpha and beta) [Lipid metabolism]
GOHSU12	CDS	GOHSU_12_01190	complement(136362..136766)		hypothetical protein [nite]			ROP_63430 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H6R2K8_NOCCG Uncharacterized protein OS=Nocardia cy riacigeorgica (strain GUH-2) GN=NOCYR_1050 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU12	CDS	GOHSU_12_01200	complement(136781..138577)		acyl-CoA carboxylase alpha chain [nite]	 accA [nite]		RER_21210 :acyl-CoA carboxylase alpha chain : accA : 6.4.1.- : Rhodococcus erythropolis PR4	acyl-CoA carboxylase alpha chain	accA	G7GYT3_9ACTO Acyl-CoA carboxylase alpha chain OS=Go rdonia araii NBRC 100433 GN=accA PE=4 SV=1;acyl-CoA carboxylase alpha chain [Rhodococcus opacu s B4];protein synonym:biotin carboxylase/biotin carboxyl carrier protein;acetyl-CoA carboxylase biotin carboxylase subunit; Validated;Carbamoyl-phosphate synthase L chain, N-terminal domain; pfam00289;ATP-grasp domain; cl03087;Biotin carboxylase C-terminal domain; cl08365;The biotinyl-domain or biotin carboxyl carrier protein (BCCP) domain is present in all biotin-dependent enzymes, such as acetyl-CoA carboxylase, pyruvate carboxylase, propionyl-CoA carboxylase, methylcrotonyl-CoA carboxylase, geranyl-CoA carboxylase; cd06850;carboxyltransferase (CT) interaction site;biotinylation site [posttranslational modification]
GOHSU12	CDS	GOHSU_12_01210	complement(138738..140141)		conserved hypothetical protein [nite]			RER_21220 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	condensation domain-containing protein		F1YF77_9ACTO Condensation domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03657 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Condensation domain; pfam00668
GOHSU12	CDS	GOHSU_12_01220	complement(140141..140563)		SufE family protein [nite]			RER_21230 :SufE family protein : : : Rhodococcus erythropolis PR4	Fe-S metabolism associated SufE		F9VWZ2_9ACTO Putative SufE family protein OS=Gordon ia alkanivorans NBRC 16433 GN=GOALK_067_00900 PE=4 SV=1;Fe-S metabolism associated SufE [Rhodothermus marin us SG0.5JP17- 172];PFAM: Fe-S metabolism associated SufE; KEGG: rmr:Rmar_0984 Fe-S metabolism associated SufE;Fe-S metabolism associated domain; cl00951
GOHSU12	CDS	GOHSU_12_01230	complement(140567..141451)		putative thiosulfate sulfurtransferase [nite]			RER_21240 :putative thiosulfate sulfurtransferase : : 2.8.1.1 : Rhodococcus erythropolis PR4	thiosulfate sulfurtransferase	thtR	H6MV68_GORPV Sulfurtransferase OS=Gordonia polyisop renivorans (strain DSM 44266 / VH2) GN=GPOL_c17260 PE=4 SV =1;thiosulfate sulfurtransferase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;Rhodanese-related sulfurtransferase [Inorganic ion transport and metabolism]; COG2897;Thiosulfate sulfurtransferase (TST), N-terminal, inactive domain. TST contains 2 copies of the Rhodanese Homology Domain; this is the 1st repeat, which does not contain the catalytically active Cys residue. The role of the 1st repeat is uncertain, but it...; cd01448;active site residue [active];Thiosulfate sulfurtransferase (TST), C-terminal, catalytic domain. TST contains 2 copies of the Rhodanese Homology Domain; this is the second repeat. Only the second repeat contains the catalytically active Cys residue; cd01449
GOHSU12	CDS	GOHSU_12_01240	complement(141590..142426)		hypothetical protein [nite]			ROP_16330 :hypothetical protein : : : Rhodococcus opacus B4	acyl-ACP thioesterase		D5UXM1_TSUPD Acyl-ACP thioesterase OS=Tsukamurella paurometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_1488 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: acyl-ACP thioesterase; KEGG: mtb:TBMG_01985 hypothetical protein;Acyl-ACP thioesterase [Lipid metabolism]; COG3884
GOHSU12	CDS	GOHSU_12_01250	complement(142430..143077)	case	septum formation protein Maf [hamap]	maf [hamap]	MF_00528:septum formation protein Maf:maf 	RER_21290 :putative Maf-like protein : : : Rhodococcus erythropolis PR4	Maf-like protein	maf	F1YF81_9ACTO Maf-like protein SCNU_03677 OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_03677 PE=3 SV=1;Maf-like protein [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;Maf-like protein; pfam02545;Maf; overexpression in Bacillus subtilis inhibits septation in the dividing cell;Nucleotide binding protein Maf. Maf has been implicated in inhibition of septum formation in eukaryotes, bacteria and archaea, but homologs in B.subtilis and S.cerevisiae are nonessential for cell division. Maf has been predicted to be a nucleotide- or...; cd00555;dimer interface [polypeptide binding]
GOHSU12	CDS	GOHSU_12_01260	complement(143079..143297)		hypothetical protein [migap]				hypothetical protein		H5TTA4_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_236_00060 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Acyl-CoA carboxylase epsilon subunit; pfam13822
GOHSU12	CDS	GOHSU_12_01270	complement(143294..144895)		propionyl-CoA carboxylase beta chain [nite]	 pccB [nite]		ROP_63560 :propionyl-CoA carboxylase beta chain : pccB : 6.4.1.3 : Rhodococcus opacus B4	propionyl-CoA carboxylase beta chain	pccB	H6MV72_GORPV Propionyl-CoA carboxylase beta chain O S=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN= GPOL_c17300 PE=4 SV=1;propionyl-CoA carboxylase beta chain [Rhodococcus o pacus B4];protein synonym:carboxyltransferase;Acetyl-CoA carboxylase, carboxyltransferase component (subunits alpha and beta) [Lipid metabolism];Acetyl co-enzyme A carboxylase carboxyltransferase alpha subunit; cl15772
GOHSU12	CDS	GOHSU_12_01280	144997..145815		biotin protein ligase [nite]	 birA [nite]		RER_21320 :biotin protein ligase : birA : 6.3.4.15 : Rhodococcus erythropolis PR4	biotin--protein ligase		F1YF84_9ACTO Biotin/acetyl-CoA-carboxylase ligase O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03692 PE=4 S V=1;biotin--protein ligase [Corynebacterium glutamicum ATCC 13032];catalyzes the formation of biotinyl-5'-AMP, also acts as a transcriptional repressor of the biotin operon;Biotin/lipoate A/B protein ligase family; cl14057;Biotin-(acetyl-CoA carboxylase) ligase [Coenzyme metabolism]; COG0340
GOHSU12	CDS	GOHSU_12_01290	complement(145825..146487)		hypothetical protein [migap]				hypothetical protein		F1YF85_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03697 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU12	CDS	GOHSU_12_01300	146667..147245		conserved hypothetical protein [nite]			RER_21340 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	membrane-flanked domain-containing protein		F1YF86_9ACTO Membrane-flanked domain-containing pro tein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03702 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: membrane-flanked domain; KEGG: cbg:CbuG_0240 hypothetical membrane- associated protein;Endoplasmic reticulum-based factor for assembly of V-ATPase; pfam11712;Bacterial PH domain; cl01348
GOHSU12	CDS	GOHSU_12_01310	147242..147928		putative OmpR family two-component response regulator [nite]			RER_21350 :putative OmpR family two-component response regulator : : : Rhodococcus erythropolis PR4	response regulator with CheY-like receiver domain-containing protein and winged-helix DNA-binding domain-containing protein		F1YF87_9ACTO Response regulator receiver OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_03707 PE=4 SV=1;response regulator with CheY-like receiver domain-c ontaining protein and winged-helix DNA-binding domain-cont aining protein [Saccharomonospora viridis DSM 43017];type strain of Saccharomonospora viridis DSM 43017;Response regulators consisting of a CheY-like receiver domain and a winged-helix DNA-binding domain [Signal transduction mechanisms / Transcription]; COG0745;PFAM: Transcriptional regulatory protein, C terminal; Response regulator receiver domain;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];Effector domain of response regulator. Bacteria and certain eukaryotes like protozoa and higher plants use two-component signal transduction systems to detect and respond to changes in the environment. The system consists of a sensor histidine kinase and...; cd00383;DNA binding site [nucleotide binding]
GOHSU12	CDS	GOHSU_12_01320	147933..149231		putative two-component histidine kinase [nite]			ROP_63610 :putative two-component histidine kinase : : 2.7.13.- : Rhodococcus opacus B4	histidine kinase		F1YF88_9ACTO ATP-binding region ATPase domain-conta ining protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SC NU_03712 PE=4 SV=1;histidine kinase [Pseudonocardia dioxanivorans CB11 90];KEGG: sen:SACE_6490 sensor kinase, two-component system; PFAM: ATPase-like, ATP-binding domain; HAMP linker domain; Signal transduction histidine kinase, subgroup 1, dimerisation/phosphoacceptor domain; SMART: ATPase-like, ATP-binding domain; Signal transduction histidine kinase, subgroup 1, dimerisation/phosphoacceptor domain;sensory histidine kinase CreC; Provisional;Histidine Kinase A (dimerization/phosphoacceptor) domain; Histidine Kinase A dimers are formed through parallel association of 2 domains creating 4-helix bundles; usually these domains contain a conserved His residue and are activated via...; cl00080;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU12	CDS	GOHSU_12_01330	complement(149203..149763)		hypothetical membrane protein [nite]			ROP_63620 :hypothetical membrane protein : : : Rhodococcus opacus B4	GtrA-like protein		F1YF89_9ACTO GtrA family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_03717 PE=4 SV=1;GtrA-like protein [Amycolatopsis mediterranei U32];GtrA-like protein; cl00971
GOHSU12	CDS	GOHSU_12_01340	149921..151117		phosphoribosylaminoimidazole carboxylase ATPase subunit [nite]	 purK [nite]		RER_21380 :phosphoribosylaminoimidazole carboxylase ATPase subunit : purK : 4.1.1.21 : Rhodococcus erythropolis PR4	phosphoribosylaminoimidazole carboxylase ATPase subunit		F1YF90_9ACTO Phosphoribosylaminoimidazole carboxyla se ATPase subunit OS=Gordonia neofelifaecis NRRL B-59395 G N=SCNU_03722 PE=4 SV=1;phosphoribosylaminoimidazole carboxylase ATPase sub unit [Corynebacterium glutamicum ATCC 13032];phosphoribosylaminoimidazole carboxylase ATPase subunit; Reviewed;With PurE catalyzes the conversion of aminoimidazole ribonucleotide to carboxyaminoimidazole ribonucleotide in the de novo purine nucleotide biosynthetic pathway;ATP-grasp domain; cl03087
GOHSU12	CDS	GOHSU_12_01350	151117..151626		phosphoribosylaminoimidazole carboxylase catalytic subunit [nite]	 purE [nite]		RER_21390 :phosphoribosylaminoimidazole carboxylase catalytic subunit : purE : 4.1.1.21 : Rhodococcus erythropolis PR4	phosphoribosylaminoimidazole carboxylase catalytic subunit	purE	F1YF91_9ACTO N5-carboxyaminoimidazole ribonucleotid e mutase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03 727 PE=3 SV=1;phosphoribosylaminoimidazole carboxylase catalytic subunit [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Catalyzes a step in the de novo purine nucleotide biosynthetic pathway;AIR carboxylase; cl00310
GOHSU12	CDS	GOHSU_12_01360	151630..152151		hypothetical protein [migap]				hypothetical protein		D5UPH6_TSUPD Putative uncharacterized protein OS=Ts ukamurella paurometabola (strain ATCC 8368 / DSM 20162 / J CM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_2121 PE=4 SV=1 ;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: sen:SACE_3425 hypothetical protein;DinB superfamily; pfam12867;Mycothiol maleylpyruvate isomerase N-terminal domain; cl00986
GOHSU12	CDS	GOHSU_12_01370	complement(152177..153190)		putative 2-nitropropane dioxygenase [nite]			ROP_05570 :putative 2-nitropropane dioxygenase : : 1.13.11.32 : Rhodococcus opacus B4	2-nitropropane dioxygenase		F1YF92_9ACTO 2-nitropropane dioxygenase NPD OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_03732 PE=4 SV=1;2-nitropropane dioxygenase [Rhodococcus jostii RHA1 ];2-Nitropropane dioxygenase (NPD), one of the nitroalkane oxidizing enzyme families, catalyzes oxidative denitrification of nitroalkanes to their corresponding carbonyl compounds and nitrites. NDP is a member of the NAD(P)H-dependent flavin oxidoreductase...; cd04730;Nitronate monooxygenase; pfam03060;FMN binding site [chemical binding];substrate binding site [chemical binding];putative catalytic residue [active]
GOHSU12	CDS	GOHSU_12_01380	complement(153303..154022)		conserved hypothetical protein [nite]			RER_40750 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	N-acetyltransferase GCN5		D5UU83_TSUPD GCN5-related N-acetyltransferase OS=Ts ukamurella paurometabola (strain ATCC 8368 / DSM 20162 / J CM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_2990 PE=4 SV=1 ;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: GCN5-related N-acetyltransferase; KEGG: rha:RHA1_ro01753 acetyltransferase;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU12	CDS	GOHSU_12_01390	complement(154019..154840)		conserved hypothetical protein [nite]			RER_36120 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	dienelactone hydrolase		F1YF94_9ACTO Dienelactone hydrolase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_03742 PE=4 SV=1;dienelactone hydrolase [Streptomyces sp. C];Dienelactone hydrolase and related enzymes [Secondary metabolites biosynthesis, transport, and catabolism]
GOHSU12	CDS	GOHSU_12_01400	complement(154837..155217)		hypothetical protein [nite]			ROP_05850 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0R6N4_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_128_00030 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF04075; match to protein family HMM TIGR00026;Domain of unknown function (DUF385); cl04387
GOHSU12	CDS	GOHSU_12_01410	complement(155280..155924)		hypothetical protein [nite]			RER_40030 :hypothetical protein : : : Rhodococcus erythropolis PR4	peptidase E		H0RID5_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_072_00620 PE =4 SV=1;peptidase E [Corynebacterium glutamicum ATCC 13032] ;IS fingerprint type 4-5;Type 1 glutamine amidotransferase (GATase1)-like domain found in peptidase E_like proteins; cd03129;catalytic triad [active];active site nucleophile [active]
GOHSU12	CDS	GOHSU_12_01420	155992..156573		putative TetR family transcriptional regulator [nite]			RER_50050 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	putative TetR family transcriptional regulator		G7GYR3_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_019_00200 PE=4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;mycofactocin system transcriptional regulator; TIGR03968;Helix-turn-helix domains; cl00088
GOHSU12	CDS	GOHSU_12_01430	156673..157836		acyl-CoA dehydrogenase [nite]	 fadE [nite]		ROP_51610 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		F1YF97_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 03757 PE=3 SV=1;acyl-CoA dehydrogenase [Amycolicicoccus subflavus D QS3- 9A1];type strain of Amycolicicoccus subflavus;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Short chain acyl-CoA dehydrogenases and eukaryotic short/branched chain acyl-CoA dehydrogenases; cd01158;FAD binding site [chemical binding];homotetramer interface [polypeptide binding];substrate binding pocket [chemical binding];catalytic base [active]
GOHSU12	CDS	GOHSU_12_01440	complement(157964..158656)		conserved hypothetical protein [nite]			RER_21450 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TJ89_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_066_00060 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];AMP-binding enzyme; cl15778
GOHSU12	CDS	GOHSU_12_01450	complement(158656..160131)		putative LytR family regulatory protein [nite]			ROP_63680 :putative LytR family regulatory protein : : : Rhodococcus opacus B4	LytR family transcriptional regulator		F1YF99_9ACTO Cell envelope-like function transcript ional attenuator OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_03767 PE=4 SV=1;LytR family transcriptional regulator [Amycolatopsi s mediterranei U32];Cell envelope-related transcriptional attenuator domain; cl00581;LytR cell envelope-related transcriptional attenuator; pfam13399
GOHSU12	CDS	GOHSU_12_01460	160712..161608		dTDP-6-deoxy-L-lyxo-4-hexulose reductase [nite]	 rmlD [nite]		RER_21470 :dTDP-6-deoxy-L-lyxo-4-hexulose reductase : rmlD : : Rhodococcus erythropolis PR4	dTDP-4-dehydrorhamnose reductase RmlD	rmlD	H6MV85_GORPV dTDP-4-dehydrorhamnose reductase RmlD OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN =rmlD PE=4 SV=1;rmlD gene product [Gordonia polyisoprenivorans VH2] ;dTDP-6-deoxy-L-lyxo-4-hexulose reductase and related proteins, extended (e) SDRs; cd05254;NADP binding site [chemical binding];putative substrate binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_01470	161637..162596		dTDP-rhamnose--alpha-D-N-acetylglucosamine-diphosphoryl polyprenol, alpha-3-L-rhamnosyl transferase [nite]	 wbbL [nite]		ROP_63700 :dTDP-rhamnose--alpha-D-N-acetylglucosamine-diphosphoryl polyprenol, alpha-3-L-rhamnosyl transferase : wbbL : 2.4.-.- : Rhodococcus opacus B4	glycosyl transferase family protein		F1YFA1_9ACTO dTDP-rhamnose--alpha-D-N-acetylglucosa mine-diphosphoryl polyprenol, alpha-3-L-rhamnosyl transfer ase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03777 P E=4 SV=1;glycosyl transferase family protein [Roseiflexus sp . RS- 1];PFAM: glycosyl transferase, family 2;Predicted glycosyltransferases [General function prediction only];Subfamily of Glycosyltransferase Family GT2 of unknown function; cd04186;Probable Catalytic site
GOHSU12	CDS	GOHSU_12_01480	162602..163693		probable mannose-1-phosphate guanylyltransferase [nite]			RER_21490 :probable mannose-1-phosphate guanylyltransferase : : 2.7.7.13 : Rhodococcus erythropolis PR4	mannose-1-phosphate guanylyltransferase		H0QVW0_9ACTO Putative mannose-1-phosphate guanylylt ransferase OS=Gordonia effusa NBRC 100432 GN=GOEFS_017_007 70 PE=4 SV=1;mannose-1-phosphate guanylyltransferase [Rhodococcu s jostii RHA1];Nucleoside-diphosphate-sugar pyrophosphorylase involved in lipopolysaccharide biosynthesis/translation initiation factor 2B, gamma/epsilon subunits (eIF-2Bgamma/eIF-2Bepsilon) [Cell envelope biogenesis, outer membrane / Translation, ribosomal structure...; COG1208;NTP_transferases catalyze the transfer of nucleotides onto phosphosugars; cd04181;Substrate binding site;Mg++ binding site;Glucose-1-phosphate adenylyltransferase, C-terminal Left-handed parallel beta helix (LbH) domain: Glucose-1-phosphate adenylyltransferase is also known as ADP-glucose synthase or ADP-glucose pyrophosphorylase. It catalyzes the first committed and...; cd04651;N-terminal domain interface [polypeptide binding];sulfate 1 binding site
GOHSU12	CDS	GOHSU_12_01490	complement(163759..164421)		putative O-methyltransferase [nite]			ROP_60360 :putative O-methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	putative O-methyltransferase		F1YFA3_9ACTO Methyltransferase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_03787 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted O-methyltransferase [General function prediction only];S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_01500	164606..165220		RNA polymerase ECF-type sigma factor SigE [nite]	 sigE [nite]		ROP_60370 :RNA polymerase ECF-type sigma factor SigE : sigE : : Rhodococcus opacus B4	RNA polymerase sigma factor SigE	sigE	F1YFA4_9ACTO RNA polymerase sigma factor SigE OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03792 PE=4 SV=1;RNA polymerase sigma factor SigE [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;Member of the extracytoplasmic function sigma factors which are active under specific conditions; binds with the catalytic core of RNA polymerase to produce the holoenzyme and directs bacterial core RNA polymerase to specific promoter elements to initiate transcription; in M. tuberculosis this protein is involved in heat shock, oxidative stress and virulence;RNA polymerase sigma factor SigE; Reviewed;Sigma-70 region 2; pfam04542;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU12	CDS	GOHSU_12_01510	165364..165663		hypothetical protein [nite]			ROP_60380 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GYQ1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_019_00080 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Putative zinc-finger; cl15806
GOHSU12	CDS	GOHSU_12_01520	165667..167139		S1C family peptidase [nite]			ROP_60390 :S1C family peptidase : : 3.4.21.- : Rhodococcus opacus B4	peptidase S1 and S6 chymotrypsin/Hap		H5TK50_9ACTO Peptidase S1 family protein OS=Gordoni a otitidis NBRC 100426 GN=GOOTI_085_00080 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: peptidase S1 and S6 chymotrypsin/Hap; PDZ/DHR/GLGF domain protein; SMART: PDZ/DHR/GLGF domain protein; KEGG: bbt:BBta_3032 serine protease do-like precursor;Trypsin-like serine protease; Many of these are synthesized as inactive precursor zymogens that are cleaved during limited proteolysis to generate their active forms. Alignment contains also inactive enzymes that have substitutions of the catalytic triad...; cl00149;PDZ domain of tryspin-like serine proteases, such as DegP/HtrA, which are oligomeric proteins involved in heat-shock response, chaperone function, and apoptosis. May be responsible for substrate recognition and/or binding, as most PDZ domains bind...; cd00987;protein binding site [polypeptide binding]
GOHSU12	CDS	GOHSU_12_01530	167151..167657		Sec-independent protein translocase protein TatB homolog [hamap]	tatB [hamap]	MF_00237:Sec-independent protein translocase protein TatB homolog:tatB 		sec-independent translocase	tatB	H5U0F4_9ACTO Sec-independent protein translocase pr otein TatB OS=Gordonia sputi NBRC 100414 GN=tatB PE=3 SV=1 ;sec-independent translocase [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;sec-independent translocase; Provisional; PRK00182;mttA/Hcf106 family; cl00788
GOHSU12	CDS	GOHSU_12_01540	complement(167710..168852)		putative Mrp family protein [nite]			ROP_60410 :putative Mrp family protein : : : Rhodococcus opacus B4	ATP-binding protein		F1YFA8_9ACTO ATPase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03812 PE=4 SV=1;ATP-binding protein [Streptomyces clavuligerus ATCC 27064];Domain of unknown function DUF59; cl00941;ATPases involved in chromosome partitioning [Cell division and chromosome partitioning]; COG0489;MRP (Multiple Resistance and pH adaptation) is a homologue of the Fer4_NifH superfamily. Like the other members of the superfamily, MRP contains a ATP-binding domain at the N-termini. It is found in bacteria as a membrane-spanning protein and functions...; cd02037
GOHSU12	CDS	GOHSU_12_01550	complement(168888..170039)		membrane-bound lytic murein transglycosylase B- like protein [migap]				membrane-bound lytic murein transglycosylase B- like protein		F1YFA9_9ACTO Membrane-bound lytic murein transglyco sylase B-like protein OS=Gordonia neofelifaecis NRRL B-593 95 GN=SCNU_03817 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.996) with cleavage site probability 0.827 at residue 52;KEGG: dvm:DvMF_2682 membrane-bound lytic murein transglycosylase B-like protein;Membrane-bound lytic murein transglycosylase B [Cell envelope biogenesis, outer membrane]; COG2951;Lytic Transglycosylase (LT)  and Goose Egg White Lysozyme (GEWL) domain. Members include the soluble and insoluble membrane-bound LTs in bacteria, the LTs in bacteriophage lambda, as well as, the eukaryotic 'goose-type' lysozymes (GEWL).  LTs catalyze...; cd00254;N-acetyl-D-glucosamine binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_01560	complement(170179..170718)		conserved hypothetical protein [nite]			RER_41510 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YFB0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03822 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 4 : Homologs of previously reported genes of unknown function;Protein of unknown function (DUF1003); cl01831
GOHSU12	CDS	GOHSU_12_01570	complement(170715..171998)		conserved hypothetical protein [nite]			RER_41500 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	MgtE integral membrane protein		F1YFB1_9ACTO MgtE intracellular region OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03827 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Mg/Co/Ni transporter MgtE (contains CBS domain) [Inorganic ion transport and metabolism]; COG2239;PFAM: MgtE intracellular region; CBS domain containing protein; KEGG: gur:Gura_2041 MgtE intracellular region;Photosynthetic reaction center (RC) complex, subunit H;  RC is an integral membrane protein-pigment complex which catalyzes light-induced reduction of ubiquinone to ubiquinol, generating a transmembrane electrochemical gradient of protons used to produce...; cl09959;MgtE intracellular N domain; cl15244;This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domain in the magnesium transporter, MgtE.  MgtE and its homologs are found in eubacteria, archaebacteria, and eukaryota. Members of this family transport Mg2+ or other...; cd04606
GOHSU12	CDS	GOHSU_12_01580	172060..172542		hypothetical membrane protein [nite]			ROP_60470 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H6MV97_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c17550 P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU12	CDS	GOHSU_12_01590	172740..174212		putative ABC transporter substrate-binding protein [nite]			ROP_60480 :putative ABC transporter substrate-binding protein : : : Rhodococcus opacus B4	family 1 extracellular solute-binding protein		F1YFB3_9ACTO Extracellular solute-binding protein f amily 1 protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_03837 PE=4 SV=1;family 1 extracellular solute-binding protein [Stac kebrandtia nassauensis DSM 44728];PFAM: extracellular solute-binding protein family 1; KEGG: noc:Noc_0282 extracellular solute-binding protein;The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cl11398;Bacterial extracellular solute-binding protein; pfam13416
GOHSU12	CDS	GOHSU_12_01600	174209..175474		putative ABC transporter permease protein [nite]			RER_41460 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	sugar ABC transporter permease		F1YFB4_9ACTO Binding-protein-dependent transporter inner membrane component OS=Gordonia neofelifaecis NRRL B- 59395 GN=SCNU_03842 PE=3 SV=1;sugar ABC transporter permease [Rhodococcus jostii RHA1];tc: 3.A.1.1;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU12	CDS	GOHSU_12_01610	175490..176512		sugar ABC transporter permease [migap]				sugar ABC transporter permease		H5U0E5_9ACTO Putative ABC transporter permease prot ein OS=Gordonia sputi NBRC 100414 GN=GOSPT_059_00650 PE=3 SV=1;sugar ABC transporter permease [Rhodococcus jostii RHA1];tc: 3.A.1.1;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU12	CDS	GOHSU_12_01620	176517..177686		putative ABC transporter ATP-binding protein [nite]			RER_41440 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter		F1YFM3_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_03852 PE=3 SV=1;ABC transporter [Conexibacter woesei DSM 14684];glycerol-3-phosphate transporter ATP-binding subunit; Provisional; PRK11650;PFAM: ABC transporter; transport-associated OB domain protein; SMART: ATPase AAA; KEGG: noc:Noc_0279 ABC transporter ATPase subunit;The N-terminal ATPase domain of the maltose transporter, MalK.  ATP binding cassette (ABC) proteins function from bacteria to human, mediating the translocation of substances into and out of cells or organelles.  ABC transporters contain two...; cd03301;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU12	CDS	GOHSU_12_01630	177709..178302		conserved hypothetical protein [nite]			RER_41380 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YFM4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03857 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Suppressor of fused protein (SUFU); pfam05076
GOHSU12	CDS	GOHSU_12_01640	178368..179231		putative ABC transporter substrate-binding protein [nite]			ROP_60650 :putative ABC transporter substrate-binding protein : : : Rhodococcus opacus B4	ABC transporter substrate-binding protein		F1YFM7_9ACTO Substrate-binding region of ABC-type g lycine betaine transporter OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03872 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];NMT1-like family; cl15260
GOHSU12	CDS	GOHSU_12_01650	complement(179322..180152)		putative alcohol dehydrogenase [nite]			RER_41250 :putative alcohol dehydrogenase : : 1.1.1.1 : Rhodococcus erythropolis PR4	short-chain dehydrogenase/reductase		F1YFM8_9ACTO Short-chain dehydrogenase/reductase SD R OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03877 PE= 3 SV=1;short-chain dehydrogenase/reductase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;hypothetical protein; Provisional;Similar to Q73WX3_MYCPA e-114;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU12	CDS	GOHSU_12_01660	complement(180202..181737)		flavin-containing monooxygenase [nite]			ROP_60680 :flavin-containing monooxygenase : : : Rhodococcus opacus B4	monooxygenase		F1YFM9_9ACTO Cyclohexanone monooxygenase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_03882 PE=4 SV=1;monooxygenase [Rhodococcus jostii RHA1];Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU12	CDS	GOHSU_12_01670	181837..182463		putative TetR family transcriptional regulator [nite]			RER_41230 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		H5TKT1_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia otitidis NBRC 100426 GN=GOOTI_093_001 50 PE=4 SV=1;TetR family transcriptional regulator [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU12	CDS	GOHSU_12_01680	complement(182548..>186019)	3x codon frame	2-oxoglutarate dehydrogenase E1 component [nite]	 sucA [nite]		ROP_60710 :2-oxoglutarate dehydrogenase E1 component : sucA : 1.2.4.2 : Rhodococcus opacus B4	alpha-ketoglutarate decarboxylase	kgd	F1YFN2_9ACTO Alpha-ketoglutarate decarboxylase OS=G ordonia neofelifaecis NRRL B-59395 GN=kgd PE=4 SV=1;alpha-ketoglutarate decarboxylase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;kgd; produces succinic semialdehyde; part of alternative pathway from alpha-ketoglutarate to succinate; essential for normal growth;alpha-ketoglutarate decarboxylase; Reviewed; PRK12270;2-oxoacid dehydrogenases acyltransferase (catalytic domain); cl02008;Thiamine pyrophosphate (TPP) family, E1 of OGDC-like subfamily, TPP-binding module; composed of proteins similar to the E1 component of the 2-oxoglutarate dehydrogenase multienzyme complex (OGDC). OGDC catalyzes the oxidative decarboxylation of...; cd02016;TPP-binding site [chemical binding];Pyrimidine (PYR) binding domain of thiamine pyrophosphate (TPP)-dependent enzymes; cl11410
##GOHSU13	contig00044				59668
GOHSU13	CDS	GOHSU_13_00010	complement(<1..255)	codon	putative 2-oxoglutarate dehydrogenase E1 component [nite]	 sucA [nite]		YM304_23290 :putative 2-oxoglutarate dehydrogenase E1 component : sucA : 1.2.4.2 : Acidimicrobidae bacterium YM16-304	2-oxoglutarate dehydrogenase, E1 subunit		H5UEY0_9ACTO 2-oxoglutarate dehydrogenase E1 compon ent OS=Gordonia terrae NBRC 100016 GN=sucA PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: 2-oxoglutarate dehydrogenase, E1 subunit; PFAM: transketolase; dehydrogenase E1 component; catalytic domain of components of various dehydrogenase complexes; KEGG: sfu:Sfum_3547 alpha-ketoglutarate decarboxylase;alpha-ketoglutarate decarboxylase; Reviewed; PRK12270
GOHSU13	CDS	GOHSU_13_00020	complement(383..4153)		putative ABC transporter permease/ATP-binding protein [nite]			RER_41210 :putative ABC transporter permease/ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter-like protein		F1YFN3_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_03902 PE=3 SV=1;ABC transporter-like protein [Streptomyces sp. Sire xAA- E];KEGG: sgr:SGR_4791 putative ABC transporter ATPase and permease component; PFAM: ABC transporter related; ABC transporter transmembrane region; SMART: AAA ATPase;ABC-type multidrug transport system, ATPase and permease components [Defense mechanisms]; COG1132;ABC transporter transmembrane region; cl00549;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU13	CDS	GOHSU_13_00030	complement(4150..5013)		conserved hypothetical protein [nite]			RER_41180 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0QVS8_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_017_00450 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU13	CDS	GOHSU_13_00040	5256..6752		putative drug resistance efflux protein [nite]			RER_41130 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	drug-transport integral membrane protein		F1YFN5_9ACTO EmrB/QacA subfamily drug resistance tr ansporter OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_0 3912 PE=4 SV=1;drug-transport integral membrane protein [Mycobacte rium marinum M];thought to be involved in transport of drug across the membrane (export): drug resistance by an export mechanism (conferes resistance to toxic compounds by removing them for the cells). responsible for the translocation of the substrate across the membrane;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;drug resistance transporter, EmrB/QacA subfamily; TIGR00711;putative substrate translocation pore
GOHSU13	CDS	GOHSU_13_00050	6881..8122		putative hydrolase [nite]			OBV_13210 :putative hydrolase : : : Oscillibacter valericigenes Sjm18-20	DNA repair exonuclease		F1YFN7_9ACTO DNA repair exonuclease OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_03922 PE=4 SV=1;DNA repair exonuclease [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;DNA repair exonuclease [DNA replication, recombination, and repair]; COG0420;Mre11 nuclease, N-terminal metallophosphatase domain; cd00840;metal binding site [ion binding];DNA binding site [nucleotide binding]
GOHSU13	CDS	GOHSU_13_00060	8119..10734		hypothetical protein [migap]				hypothetical protein		F1YFN8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_03927 PE=4 SV=1;unnamed protein product [Mycobacterium leprae TN];P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Uncharacterized conserved protein [Function unknown]
GOHSU13	tRNA	GOHSU_13_t0010	10802..10874		not assigned						identified by tRNAscan-SE; putative
GOHSU13	CDS	GOHSU_13_00070	11024..11725		hypothetical protein [nite]			RER_11810 :hypothetical protein : : : Rhodococcus erythropolis PR4	transcription antitermination regulator		F1YMK4_9ACTO PAS fold-3 domain-containing protein O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15444 PE=4 S V=1;transcription antitermination regulator [Nocardia f arcinica IFM 10152];PAS fold; pfam08447;ANTAR domain; cl04297
GOHSU13	CDS	GOHSU_13_00080	11854..12474		putative TetR family transcriptional regulator [nite]			RER_pREL1-02430 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		G7H583_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_064_00100 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m sp. JLS];PFAM: regulatory protein, TetR; KEGG: mmc:Mmcs_4837 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU13	CDS	GOHSU_13_00090	12540..15608		MmpL family protein [migap]				MmpL family protein		F1YL29_9ACTO MMPL domain-containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_12933 PE=4 SV=1;MmpL family protein [Mycobacterium abscessus ATCC 1 9977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q265T1_MYCVN 0.0;Transport protein; TIGR00833;Protein export membrane protein; cl14618
GOHSU13	CDS	GOHSU_13_00100	complement(15648..15947)		hypothetical protein [migap]				hypothetical protein		G7CD71_MYCTH Putative uncharacterized protein OS=My cobacterium thermoresistibile ATCC 19527 GN=KEK_04597 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU13	CDS	GOHSU_13_00110	16140..17057		putative oxidoreductase [nite]			ROP_53620 :putative oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase		D0L5V0_GORB4 Short-chain dehydrogenase/reductase SD R OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_4291 PE=3 SV=1;oxidoreductase [Gloeobacter violaceus PCC 7421];3-oxoacyl-(acyl-carrier-protein) reductase; TIGR01830;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU13	CDS	GOHSU_13_00120	17488..18183		putative RNA polymerase sigma factor [nite]	 sigH [nite]		KSE_49450 :putative RNA polymerase sigma factor : sigH : : Kitasatospora setae NBRC 14216	sigma factor		F1YPQ4_9ACTO Putative sigma factor OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_19557 PE=4 SV=1;sigma factor [Nocardia farcinica IFM 10152];RNA polymerase sigma-70 factor, sigma-B/F/G subfamily; TIGR02980;Sigma-70 region 2; pfam04542;Sigma-70 region 3; pfam04539;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU13	CDS	GOHSU_13_00130	18429..19019		hypothetical protein [nite]			ROP_36770 :hypothetical protein : : : Rhodococcus opacus B4	transcription antitermination regulator		F1YMK4_9ACTO PAS fold-3 domain-containing protein O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15444 PE=4 S V=1;transcription antitermination regulator [Nocardia f arcinica IFM 10152];PAS fold; pfam08447;ANTAR domain; cl04297
GOHSU13	CDS	GOHSU_13_00140	19072..20082		hypothetical protein [nite]			KSE_07380 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		D0L5V6_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_4297 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU13	CDS	GOHSU_13_00150	complement(20317..21024)		hypothetical protein [nite]			ROP_pROB01-03430 :hypothetical protein : : : Rhodococcus opacus B4	methyltransferase		H0QY33_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_038_00010 PE=4 SV=1;methyltransferase [Streptomyces sp. Mg1];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;putative methyltransferase; Provisional
GOHSU13	CDS	GOHSU_13_00160	complement(21024..21350)		hypothetical protein [migap]				hypothetical protein		F1YM85_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15469 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU13	CDS	GOHSU_13_00170	complement(21347..21601)		hypothetical protein [nite]			KSE_61640 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		F1YMK5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15449 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: afw:Anae109_1004 hypothetical protein
GOHSU13	CDS	GOHSU_13_00180	complement(21681..21881)		stress-induced acidophilic repeat motif-containing protein [migap]				stress-induced acidophilic repeat motif-containing protein		F1YM82_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15454 PE=4 SV=1;stress-induced acidophilic repeat motif-containing protein [Brachybacterium faecium DSM 4810];type strain of Brachybacterium faecium DSM 4810;PFAM: Stress-induced bacterial acidophilic repeat motif
GOHSU13	CDS	GOHSU_13_00190	complement(21955..22977)		hypothetical protein [migap]				hypothetical protein		G7H3S0_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_057_00190 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];KEGG: nfa:nfa19270 hypothetical protein
GOHSU13	CDS	GOHSU_13_00200	complement(22983..23267)		hypothetical protein [migap]				hypothetical protein		H5TQL3_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_182_00430 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];KEGG: fal:FRAAL1616 conserved hypothetical protein; putative protein kinase domain
GOHSU13	CDS	GOHSU_13_00210	23238..24146		putative shikimate dehydrogenase [nite]			ROP_12660 :putative shikimate dehydrogenase : : 1.1.1.25 : Rhodococcus opacus B4	shikimate dehydrogenase substrate binding domain-containing protein		F1YFP5_9ACTO Shikimate 5-dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_03962 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: shikimate dehydrogenase substrate binding domain protein; shikimate/quinate 5-dehydrogenase; KEGG: rha:RHA1_ro01564 shikimate 5-dehydrogenase;shikimate 5-dehydrogenase; Reviewed;Shikimate dehydrogenase substrate binding domain; pfam08501;NAD(P) binding domain of Shikimate dehydrogenase; cd01065;shikimate binding site;NAD(P) binding site [chemical binding]
GOHSU13	CDS	GOHSU_13_00220	24293..25960		arginyl-tRNA synthetase [hamap]	argS [hamap]	MF_00123:arginyl-tRNA synthetase:argS 	RER_39240 :arginyl-tRNA synthetase : argS : 6.1.1.19 : Rhodococcus erythropolis PR4	arginyl-tRNA synthetase	argS	F1YFP9_9ACTO Arginine--tRNA ligase OS=Gordonia neof elifaecis NRRL B-59395 GN=argS PE=3 SV=1;arginyl-tRNA synthetase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes a two-step reaction, first charging an arginine molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA; class-I aminoacyl-tRNA synthetase;arginyl-tRNA synthetase; Reviewed; PRK01611;Arginyl tRNA synthetase N terminal domain; cl04104;catalytic core domain of arginyl-tRNA synthetases; cd00671;HIGH motif;KMSK motif region;Anticodon-binding domain of class Ia aminoacyl tRNA synthetases and similar domains; cl12020;tRNA binding surface [nucleotide binding];anticodon binding site
GOHSU13	CDS	GOHSU_13_00230	26050..27390		diaminopimelate decarboxylase [nite]	 lysA [nite]		RER_39230 :diaminopimelate decarboxylase : lysA : 4.1.1.20 : Rhodococcus erythropolis PR4	diaminopimelate decarboxylase	lysA	F1YFQ0_9ACTO Diaminopimelate decarboxylase OS=Gordo nia neofelifaecis NRRL B-59395 GN=lysA PE=3 SV=1;diaminopimelate decarboxylase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;diaminopimelate decarboxylase; TIGR01048;Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzyme Diaminopimelate Decarboxylase; cd06828;pyridoxal 5'-phosphate (PLP) binding site [chemical binding];substrate binding site [chemical binding];catalytic residues [active];dimer interface [polypeptide binding]
GOHSU13	CDS	GOHSU_13_00240	27383..28702		homoserine dehydrogenase [nite]	 hom [nite]		RER_39220 :homoserine dehydrogenase : hom : 1.1.1.3 : Rhodococcus erythropolis PR4	homoserine dehydrogenase	hom	F1YFQ1_9ACTO Homoserine dehydrogenase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_03992 PE=3 SV=1;homoserine dehydrogenase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of L-aspartate 4-semialdehyde from L-homoserine;homoserine dehydrogenase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Homoserine dehydrogenase; pfam00742;ACT_HSDH_Hom CD includes the C-terminal ACT domain of the NAD(P)H-dependent, homoserine dehydrogenase (HSDH) and related domains; cd04881
GOHSU13	CDS	GOHSU_13_00250	28695..29777		threonine synthase [nite]	 thrC [nite]		RER_39210 :threonine synthase : thrC : 4.2.3.1 : Rhodococcus erythropolis PR4	threonine synthase		G7GRG9_9ACTO Threonine synthase OS=Gordonia amarae NBRC 15530 GN=thrC PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: threonine synthase; PFAM: pyridoxal-5'-phosphate-dependent protein subunit beta; KEGG: abo:ABO_0807 threonine synthase;threonine synthase; Reviewed;Threonine synthase is a pyridoxal phosphate (PLP) dependent enzyme that catalyses the last reaction in the synthesis of  threonine from aspartate. It proceeds by converting O-phospho-L-homoserine (OPH) into threonine and inorganic phosphate. In plants; cd01563;homodimer interface [polypeptide binding];pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU13	CDS	GOHSU_13_00260	29787..30743		homoserine kinase [hamap]	thrB [hamap]	MF_00384:homoserine kinase:thrB 	RER_39200 :homoserine kinase : thrB : 2.7.1.39 : Rhodococcus erythropolis PR4	homoserine kinase	thrB	F1YFQ3_9ACTO Homoserine kinase OS=Gordonia neofelif aecis NRRL B-59395 GN=thrB PE=3 SV=1;homoserine kinase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of O-phospho-L-homoserine from L-homoserine in threonine biosynthesis from asparate;homoserine kinase; Provisional
GOHSU13	CDS	GOHSU_13_00270	30957..32810		transcription termination factor Rho [nite]	 rho [nite]		ROP_11950 :transcription termination factor Rho : rho : : Rhodococcus opacus B4	transcription termination factor Rho	rho	G7GRH1_9ACTO Transcription termination factor Rho O S=Gordonia amarae NBRC 15530 GN=rho PE=3 SV=1;transcription termination factor Rho [Corynebacteri um glutamicum ATCC 13032];IS fingerprint type 4-5;An RNA-DNA helicase that actively releases nascent mRNAs from paused transcription complexes;transcription termination factor Rho; Provisional;S1_like: Ribosomal protein S1-like RNA-binding domain. Found in a wide variety of RNA-associated proteins. Originally identified in S1 ribosomal protein. This superfamily also contains the Cold Shock Domain (CSD), which is a homolog of the S1 domain; cl09927;RNA binding site [nucleotide binding];Transcription termination factor rho is a bacterial ATP-dependent RNA/DNA helicase. It is a homohexamer. Each monomer consists of an N-terminal domain of the OB fold, which is responsible for binding to cysteine rich nucleotides. This alignment is of the...; cd01128;multimer interface [polypeptide binding];Walker A motif;ATP binding site [chemical binding];Walker B motif
GOHSU13	CDS	GOHSU_13_00280	32860..33402		putative methylated-DNA--protein-cysteine methyltransferase [nite]	 ogt3 [nite]		KSE_58530 :putative methylated-DNA--protein-cysteine methyltransferase : ogt3 : 2.1.1.63 : Kitasatospora setae NBRC 14216	methylated DNA-protein cysteine methyltransferase-like protein		E3NVI7_CAERE Putative uncharacterized protein OS=Ca enorhabditis remanei GN=CRE_15855 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: bac:BamMC406_3089 AraC family transcriptional regulator;6-O-methylguanine DNA methyltransferase, ribonuclease-like domain; pfam02870;The DNA repair protein O6-alkylguanine-DNA alkyltransferase (ATase; also known as AGT, AGAT and MGMT) reverses O6-alkylation DNA damage by transferring O6-alkyl adducts to an active site cysteine irreversibly, without inducing DNA strand breaks. ATases...; cd06445;DNA binding site [nucleotide binding]
GOHSU13	CDS	GOHSU_13_00290	33399..34001		3-methyladenine-DNA glycosylase I [nite]	 tag [nite]		KRH_04180 :3-methyladenine-DNA glycosylase I : tag : 3.2.2.20 : Kocuria rhizophila DC2201	DNA-3-methyladenine glycosylase I		F1YHZ5_9ACTO DNA-3-methyladenine glycosylase I OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_07548 PE=4 SV=1 ;DNA-3-methyladenine glycosylase I [Kytococcus seden tarius DSM 20547];PFAM: Methyladenine glycosylase; TIGRFAM: DNA-3-methyladenine glycosylase I;Methyladenine glycosylase; cl01059
GOHSU13	CDS	GOHSU_13_00300	complement(34008..35180)		putative NADPH--sulfite reductase flavoprotein alpha-component [nite]	 cysJ [nite]		ROP_14710 :putative NADPH--sulfite reductase flavoprotein alpha-component : cysJ : 1.8.1.2 : Rhodococcus opacus B4	sulfite reductase		F1YFQ5_9ACTO FAD-binding domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04012 PE=4 SV=1;sulfite reductase [Mycobacterium abscessus ATCC 199 77];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q8UIZ5_AGRT5 0.0;sulfite reductase [NADPH] flavoprotein, alpha-component; TIGR01931;Cytochrome p450- like alpha subunits of E. coli sulfite reductase (SiR) multimerize with beta subunits to catalyze the NADPH dependent reduction of sulfite to sulfide. Beta subunits have an Fe4S4 cluster and a siroheme, while the alpha subunits (cysJ...; cd06199;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];catalytic residues [active];NAD binding pocket [chemical binding];phosphate binding motif [ion binding];beta-alpha-beta structure motif
GOHSU13	CDS	GOHSU_13_00310	35381..35593	case	50S ribosomal protein L31 [hamap]	rpmE [hamap]	MF_00501:50S ribosomal protein L31:rpmE 	RER_39180 :50S ribosomal protein L31 : rpmE : : Rhodococcus erythropolis PR4	50S ribosomal protein L31	rpmE	H5U0A9_9ACTO 50S ribosomal protein L31 OS=Gordonia sputi NBRC 100414 GN=rpmE PE=3 SV=1;50S ribosomal protein L31 [Bifidobacterium animalis subsp. lactis AD011];Ribosomal protein L31; cl00377;RpmE; there appears to be two types of ribosomal proteins L31 in bacterial genomes; some contain a CxxC motif while others do not; Bacillus subtilis has both types; the proteins in this cluster have the CXXC motif; RpmE is found in exponentially growing Bacilli while YtiA was found after exponential growth; expression of ytiA is controlled by a zinc-specific transcriptional repressor; RpmE contains one zinc ion and a CxxC motif is responsible for this binding; forms an RNP particle along with proteins L5, L18, and L25 and 5S rRNA; found crosslinked to L2 and L25 and EF-G; may be near the peptidyltransferase site of the 50S ribosome
GOHSU13	CDS	GOHSU_13_00320	35726..36799		peptide chain release factor 1 [hamap]	prfA [hamap]	MF_00093:peptide chain release factor 1:prfA 	RER_39170 :peptide chain release factor 1 : prfA : : Rhodococcus erythropolis PR4	peptide chain release factor 1	prfA	H0QVQ0_9ACTO Peptide chain release factor 1 OS=Gord onia effusa NBRC 100432 GN=prfA PE=3 SV=1;peptide chain release factor 1 [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;peptide chain release factor 1; Validated; PRK00591;recognizes the termination signals UAG and UAA during protein translation a specificity which is dependent on amino acid residues residing in loops of the L-shaped tRNA-like molecule of RF1; this protein is similar to release factor 2;RF-1 domain; cl02875
GOHSU13	CDS	GOHSU_13_00330	36796..37692		protein methyltransferase HemK [nite]	 hemK [nite]		ROP_11920 :protein methyltransferase HemK : hemK : 2.1.1.- : Rhodococcus opacus B4	rRNA or tRNA methylase		F1YFB8_9ACTO N5-glutamine S-adenosyl-L-methionine-d ependent methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04027 PE=4 SV=1;rRNA or tRNA methylase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;N5-glutamine S-adenosyl-L-methionine-dependent methyltransferase; Provisional;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding];RRM (RNA recognition motif), also known as RBD (RNA binding domain) or RNP (ribonucleoprotein domain), is a highly abundant domain in eukaryotes found in proteins involved in post-transcriptional gene expression processes including mRNA and rRNA...; cl02586
GOHSU13	CDS	GOHSU_13_00340	37689..38348		conserved hypothetical protein [nite]			RER_39150 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	translation factor (SUA5)		F9W0R3_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_100_00650 PE=4 SV= 1;translation factor (SUA5) [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Telomere recombination; cl00305
GOHSU13	CDS	GOHSU_13_00350	38345..39583		serine hydroxymethyltransferase [nite]	 glyA [nite]		RER_39140 :serine hydroxymethyltransferase : glyA : 2.1.2.1 : Rhodococcus erythropolis PR4	glycine hydroxymethyltransferase		F1YFC0_9ACTO Glycine hydroxymethyltransferase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04037 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: glycine hydroxymethyltransferase; aromatic amino acid beta-eliminating lyase/threonine aldolase; KEGG: pen:PSEEN3863 serine hydroxymethyltransferase;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;serine hydroxymethyltransferase; Reviewed; PRK00011;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU13	CDS	GOHSU_13_00360	39597..40817		putative UDP-N-acetylglucosamine--undecaprenyl-phosphate N-acetylglucosamine-1-phosphate transferase [nite]	 wecA [nite]		ROP_11900 :putative UDP-N-acetylglucosamine--undecaprenyl-phosphate N-acetylglucosamine-1-phosphate transferase : wecA : 2.7.8.- : Rhodococcus opacus B4	glycosyl transferase, family 4, conserved region		F1YFC1_9ACTO Glycosyl transferase, family 4, conser ved region OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 04042 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: glycosyl transferase, family 4, conserved region; KEGG: dma:DMR_04940  glycosyltransferase;UDP-N-acetylmuramyl pentapeptide phosphotransferase/UDP-N- acetylglucosamine-1-phosphate transferase [Cell envelope biogenesis, outer membrane]; COG0472;This subfamily contains Escherichia coli WecA, Bacillus subtilis TagO and related proteins. WecA is an UDP-N-acetylglucosamine (GlcNAc):undecaprenyl-phosphate (Und-P) GlcNAc-1-phosphate transferase that catalyzes the formation of a phosphodiester bond...; cd06853;Mg++ binding site [ion binding];putative catalytic motif [active];substrate binding site [chemical binding]
GOHSU13	CDS	GOHSU_13_00370	40828..41307		conserved hypothetical membrane protein [nite]			RER_39120 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YFC2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04047 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314]
GOHSU13	CDS	GOHSU_13_00380	41686..42468	case	ATP synthase subunit a [hamap]	atpB [hamap]	MF_01393:ATP synthase subunit a:atpB 	RER_39110 :ATP synthase subunit a : atpB : 3.6.3.14 : Rhodococcus erythropolis PR4	ATP synthase F0F1 subunit A	atpB	C3JI57_RHOER ATP synthase subunit a OS=Rhodococcus erythropolis SK121 GN=atpB PE=3 SV=1;F0F1 ATP synthase subunit A [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;Produces ATP from ADP in the presence of a proton gradient across the membrane. Subunit A is part of the membrane proton channel F0;ATP synthase A chain; cl00413
GOHSU13	CDS	GOHSU_13_00390	42550..42798	case	ATP synthase subunit c [hamap]	atpH [hamap]	MF_01396:ATP synthase subunit c:atpH 	ROP_11870 :ATP synthase subunit c : atpE : 3.6.3.14 : Rhodococcus opacus B4	ATP synthase F0F1 subunit C	atpE	F1YFC4_9ACTO F0F1 ATP synthase subunit C OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_04057 PE=3 SV=1;F0F1 ATP synthase subunit C [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;ATP synthase subunit C; cl00466;Produces ATP from ADP in the presence of a proton gradient across the membrane. Subunit C is part of the membrane proton channel F0
GOHSU13	CDS	GOHSU_13_00400	42813..43370	case	ATP synthase subunit b [hamap]	atpF [hamap]	MF_01398:ATP synthase subunit b:atpF 	ROP_11860 :ATP synthase subunit b : atpF : 3.6.3.14 : Rhodococcus opacus B4	ATP synthase F0F1 subunit B	atpF	F1YFC5_9ACTO ATP synthase subunit b OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_04062 PE=3 SV=1;F0F1 ATP synthase subunit B [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;Produces ATP from ADP in the presence of a proton gradient across the membrane. Subunit B is part of the membrane proton channel.;Plant ATP synthase F0; cl07975;F0F1-type ATP synthase, subunit b [Energy production and conversion]; COG0711
GOHSU13	CDS	GOHSU_13_00410	43394..44287	case	ATP synthase subunit delta [hamap]	atpD [hamap]	MF_01416:ATP synthase subunit delta:atpD 	ROP_11850 :ATP synthase delta chain : atpH : 3.6.3.14 : Rhodococcus opacus B4	ATP synthase F0F1 subunit delta	atpH	F1YFC6_9ACTO F0F1 ATP synthase subunit delta OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_04067 PE=3 SV=1;F0F1 ATP synthase subunit delta [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;Plant ATP synthase F0; cl07975;produces ATP from ADP in the presence of a proton gradient across the membrane; the delta subunit is part of the catalytic core of the ATP synthase complex;ATP synthase delta (OSCP) subunit; pfam00213
GOHSU13	CDS	GOHSU_13_00420	44349..45983	case	ATP synthase subunit alpha [hamap]	atpA [hamap]	MF_01346:ATP synthase subunit alpha:atpA 	ROP_11840 :ATP synthase alpha chain : atpA : 3.6.3.14 : Rhodococcus opacus B4	ATP synthase F0F1 subunit alpha	atpA	F1YFC7_9ACTO F0F1 ATP synthase subunit alpha OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_04072 PE=3 SV=1;F0F1 ATP synthase subunit alpha [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;produces ATP from ADP in the presence of a proton gradient across the membrane; the alpha chain is a catalytic subunit;F0F1 ATP synthase subunit alpha; Validated;ATP synthase alpha/beta family, beta-barrel domain; cl15275;F1 ATP synthase alpha, central domain. The F-ATPase is found in bacterial plasma membranes, mitochondrial inner membranes and in chloroplast thylakoid membranes. It has also been found in the archaea Methanosarcina barkeri. It uses a proton gradient to...; cd01132;beta subunit interaction interface [polypeptide binding];Walker A motif;ATP binding site [chemical binding];Walker B motif;ATP synthase alpha/beta chain, C terminal domain; pfam00306
GOHSU13	CDS	GOHSU_13_00430	45986..46951	case	ATP synthase gamma chain [hamap]	atpG [hamap]	MF_00815:ATP synthase gamma chain:atpG 	ROP_11830 :ATP synthase gamma chain : atpG : 3.6.3.14 : Rhodococcus opacus B4	ATP synthase F0F1 subunit gamma	atpG	F1YFC8_9ACTO ATP synthase gamma chain OS=Gordonia n eofelifaecis NRRL B-59395 GN=atpG PE=3 SV=1;F0F1 ATP synthase subunit gamma [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;ATP synthase; cl00365;F0F1 ATP synthase subunit gamma; Provisional;Produces ATP from ADP in the presence of a proton gradient across the membrane. The gamma chain is a regulatory subunit
GOHSU13	CDS	GOHSU_13_00440	46976..48421	case	ATP synthase subunit beta [hamap]	atpB [hamap]	MF_01347:ATP synthase subunit beta:atpB 	ROP_11820 :ATP synthase beta chain : atpD : 3.6.3.14 : Rhodococcus opacus B4	ATP synthase F0F1 subunit beta	atpD	F1YFC9_9ACTO F0F1 ATP synthase subunit beta OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_04082 PE=3 SV=1;F0F1 ATP synthase subunit beta [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;Produces ATP from ADP in the presence of a proton gradient across the membrane. The beta chain is a regulatory subunit;F0F1 ATP synthase subunit beta; Validated;ATP synthase alpha/beta family, beta-barrel domain; cl15275;F1 ATP synthase beta subunit, nucleotide-binding domain. The F-ATPase is found in bacterial plasma membranes, mitochondrial inner membranes and in chloroplast thylakoid membranes. It has also been found in the archaea Methanosarcina barkeri. It uses a...; cd01133;alpha subunit interaction interface [polypeptide binding];Walker A motif;ATP binding site [chemical binding];Walker B motif;inhibitor binding site;ATP synthase alpha/beta chain, C terminal domain; pfam00306
GOHSU13	CDS	GOHSU_13_00450	48431..48805	case	ATP synthase epsilon chain [hamap]	atpC [hamap]	MF_00530:ATP synthase epsilon chain:atpC 	ROP_11810 :ATP synthase epsilon chain : atpC : 3.6.3.14 : Rhodococcus opacus B4	ATP synthase F1 subunit epsilon		F1YFD0_9ACTO ATP synthase epsilon chain OS=Gordonia neofelifaecis NRRL B-59395 GN=atpC PE=3 SV=1;ATP synthase F1 subunit epsilon [Nakamurella multip artita DSM 44233];ATP synthase, Delta/Epsilon chain, beta-sandwich domain; cl10033;TIGRFAM: ATP synthase F1 subunit epsilon; PFAM: H+transporting two-sector ATPase delta/subunit epsilon; KEGG: abo:ABO_2725 ATP synthase, subunit epsilon
GOHSU13	CDS	GOHSU_13_00460	49032..49364		hypothetical protein [nite]			ROP_11800 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5UF96_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_064_00250 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF2550;Protein of unknown function (DUF2550); pfam10739
GOHSU13	CDS	GOHSU_13_00470	complement(49386..49976)		PduO-type ATP--cob(I)alamin adenosyltransferase [nite]			ROP_11790 :PduO-type ATP--cob(I)alamin adenosyltransferase : : 2.5.1.17 : Rhodococcus opacus B4	ATP/cobalamin adenosyltransferase		F1YFD2_9ACTO ATP/cobalamin adenosyltransferase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_04097 PE=4 SV=1 ;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: ATP/cobalamin adenosyltransferase; PFAM: cobalamin adenosyltransferase; KEGG: pla:Plav_1609 ATP--cobalamin adenosyltransferase;Cobalamin adenosyltransferase; cl00920
GOHSU13	CDS	GOHSU_13_00480	50021..51277		UDP-N-acetylglucosamine 1-carboxyvinyltransferase [hamap]	murA [hamap]	MF_00111:UDP-N-acetylglucosamine 1-carboxyvinyltransferase:murA 	ROP_11780 :UDP-N-acetylglucosamine 1-carboxyvinyltransferase : murA : 2.5.1.7 : Rhodococcus opacus B4	UDP-N-acetylglucosamine 1-carboxyvinyltransferase	murA2	H5U093_9ACTO UDP-N-acetylglucosamine 1-carboxyvinyl transferase OS=Gordonia sputi NBRC 100414 GN=murA PE=3 SV= 1;UDP-N-acetylglucosamine 1-carboxyvinyltransferase [ Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;adds enolpyruvyl to UDP-N-acetylglucosamine as a component of cell wall formation; gram-positive bacteria have 2 copies of MurA which are active;UDP-N-acetylglucosamine 1-carboxyvinyltransferase; TIGR01072;UDP-N-acetylglucosamine enolpyruvyl transferase catalyzes enolpyruvyl transfer as part of the first step in the biosynthesis of peptidoglycan, a component of the bacterial cell wall. The reaction is phosphoenolpyruvate + UDP-N-acetyl-D-glucosamine =...; cd01555;hinge
GOHSU13	CDS	GOHSU_13_00490	51438..52709		putative N,N'-dimethyl-4-nitrosoaniline-dependent alcohol oxidoreductase [nite]			ROP_61160 :putative N,N'-dimethyl-4-nitrosoaniline-dependent alcohol oxidoreductase : : 1.1.99.- : Rhodococcus opacus B4	class IV alcohol dehydrogenase		F1YFD4_9ACTO N,N'-dimethyl-4-nitrosoaniline-depende nt alcohol oxidoreductase OS=Gordonia neofelifaecis NRRL B -59395 GN=SCNU_04107 PE=4 SV=1;class IV alcohol dehydrogenase [Pelobacter carbinol icus DSM 2380];Lactadehyde:propanediol oxidoreductase (LPO) catalyzes the interconversion between L-lactaldehyde and L-1,2-propanediol in Escherichia coli and other enterobacteria; cd08176;putative alcohol dehydrogenase; Provisional;dimer interface [polypeptide binding];metal binding site [ion binding]
GOHSU13	CDS	GOHSU_13_00500	52794..53864		hypothetical protein [nite]			ROP_61170 :hypothetical protein : : : Rhodococcus opacus B4	AAA ATPase		H5TSZ9_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_230_00320 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];ATPases associated with a variety of cellular activities; smart00382;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger
GOHSU13	CDS	GOHSU_13_00510	53867..55420		hypothetical protein [nite]			ROP_61180 :hypothetical protein : : : Rhodococcus opacus B4	VWA domain-containing CoxE-like protein		F1YFD6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04117 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein containing von Willebrand factor type A (vWA) domain [General function prediction only]; COG3552;Von Willebrand factor type A (vWA) domain was originally found in the blood coagulation protein von Willebrand factor (vWF). Typically, the vWA domain is made up of approximately 200 amino acid residues folded into a classic a/b para-rossmann type of...; cl00057
GOHSU13	CDS	GOHSU_13_00520	55489..56667		iron-containing alcohol dehydrogenase [nite]			ROP_61190 :iron-containing alcohol dehydrogenase : : 1.1.1.- : Rhodococcus opacus B4	iron-containing alcohol dehydrogenase		F1YFD7_9ACTO Iron-containing alcohol dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04122 PE=4 SV=1;iron-containing alcohol dehydrogenase [Geobacter ur aniireducens Rf4];PFAM: iron-containing alcohol dehydrogenase;Iron-containing alcohol dehydrogenases-like; cd08188;Alcohol dehydrogenase, class IV [Energy production and conversion]; COG1454;putative active site [active];metal binding site [ion binding]
GOHSU13	CDS	GOHSU_13_00530	56670..57977		putative two-component histidine kinase [nite]			ROP_61200 :putative two-component histidine kinase : : 2.7.13.- : Rhodococcus opacus B4	histidine kinase		F1YFD8_9ACTO Histidine kinase dimerization and phos phoacceptor region OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04127 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: sus:acid_2189 histidine kinase;Signal transduction histidine kinase [Signal transduction mechanisms];Histidine kinase; pfam07730;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU13	CDS	GOHSU_13_00540	57974..58633		putative NarL family two-component response regulator [nite]			RER_17850 :putative NarL family two-component response regulator : : : Rhodococcus erythropolis PR4	response regulator receiver protein		F9W0T3_9ACTO Putative NarL family two-component res ponse regulator OS=Gordonia alkanivorans NBRC 16433 GN=GOA LK_100_00850 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: baa:BA_1979 REC, CheY-homologous receiver domain;Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding]
GOHSU13	CDS	GOHSU_13_00550	complement(58659..59495)		hypothetical protein [nite]			ROP_61270 :hypothetical protein : : : Rhodococcus opacus B4	putative SAM dependent methyltransferase		H5UFA5_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_064_00340 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
##GOHSU14	contig00051				172453
GOHSU14	CDS	GOHSU_14_00010	complement(63..935)		Clp domain-containing protein [migap]				Clp domain-containing protein		H0RLI2_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_088_00430 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_00020	complement(928..1260)		hypothetical protein [migap]				hypothetical protein		H5U696_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_125_00210 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];Evidence 4 : Homologs of previously reported genes of unknown function
GOHSU14	CDS	GOHSU_14_00030	1401..2399		not assigned						
GOHSU14	CDS	GOHSU_14_00040	2429..3400		not assigned						
GOHSU14	CDS	GOHSU_14_00050	complement(3496..6075)		hypothetical membrane protein [nite]			ANT_26430 :hypothetical membrane protein : : : Anaerolinea thermophila UNI-1	putative transporter permease		H0RLI4_9ACTO Putative ABC transporter permease prot ein OS=Gordonia polyisoprenivorans NBRC 16320 GN=GOPIP_088 _00450 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];MacB-like periplasmic core domain; pfam12704;FtsX-like permease family; cl15850
GOHSU14	CDS	GOHSU_14_00060	complement(6050..6913)		putative ABC transporter ATP-binding protein [nite]			ANT_27060 :putative ABC transporter ATP-binding protein : : : Anaerolinea thermophila UNI-1	antimicrobial peptide ABC transporter ATPase		F1YIP2_9ACTO ABC transporter ATPase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_08831 PE=3 SV=1;unnamed protein product [Clostridium clariflavum DS M 19732];PFAM: ABC transporter;This family is comprised of MJ0796 ATP-binding cassette, macrolide-specific ABC-type efflux carrier (MacAB), and proteins involved in cell division (FtsE), and release of liporoteins from the cytoplasmic membrane (LolCDE).  They are clustered together...; cd03255;ABC-type antimicrobial peptide transport system, ATPase component [Defense mechanisms]; COG1136;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU14	CDS	GOHSU_14_00070	complement(6920..7240)		hypothetical membrane protein [nite]			ROP_17020 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H0QYI9_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_041_00440 PE=4 SV=1;integral membrane protein [Rhodococcus equi 103S]
GOHSU14	CDS	GOHSU_14_00080	7436..7642		hypothetical protein [nite]			ROP_17030 :hypothetical protein : : : Rhodococcus opacus B4	excisionase family DNA binding domain-containing protein		F1YIP4_9ACTO DNA-binding domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08841 PE=4 SV=1;excisionase family DNA binding domain-containing pr otein [Nocardiopsis dassonvillei subsp. dassonvillei DSM 4 3111];InterPro IPR010093:IPR009061:IPR000551; KEGG: tfu:Tfu_2699 excisionase/Xis, DNA-binding; PFAM: regulatory protein MerR; SPTR: Q47LE0 Excisionase/Xis, DNA-binding; TIGRFAM: DNA binding domain protein, excisionase family; PFAM: MerR family regulatory protein; TIGRFAM: DNA binding domain, excisionase family;Helix-Turn-Helix DNA binding domain of truncated MerR-like proteins; cd04762;DNA binding residues [nucleotide binding]
GOHSU14	CDS	GOHSU_14_00090	7664..7906		hypothetical protein [migap]				hypothetical protein		F1YIP5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08846 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_00100	complement(7988..8836)		3-hydroxyacyl-CoA dehydrogenase [nite]			ROP_51600 :3-hydroxyacyl-CoA dehydrogenase : : 1.1.1.35 : Rhodococcus opacus B4	NAD-binding protein 3-hydroxyacyl-CoA dehydrogenase		G7H6A8_9ACTO 3-hydroxyacyl-CoA dehydrogenase OS=Gor donia araii NBRC 100433 GN=GOARA_068_00420 PE=4 SV=1;NAD-binding protein 3-hydroxyacyl-CoA dehydrogenase [Streptomyces violaceusniger Tu 4113];PFAM: 3-hydroxyacyl-CoA dehydrogenase, NAD binding; 3-hydroxyacyl-CoA dehydrogenase, C-terminal; KEGG: afo:Afer_1527 3-hydroxybutyryl-CoA dehydrogenase;3-hydroxybutyryl-CoA dehydrogenase; Validated;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;3-hydroxyacyl-CoA dehydrogenase, C-terminal domain; pfam00725
GOHSU14	CDS	GOHSU_14_00110	complement(8884..9891)		cytochrome c biogenesis protein ResC [nite]	 resC [nite]		RER_16710 :cytochrome c biogenesis protein ResC : resC : : Rhodococcus erythropolis PR4	cytochrome c assembly protein		F1YIP6_9ACTO Cytochrome c-type biogenesis protein C csB OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08851 P E=4 SV=1;cytochrome c assembly protein [Mycobacterium gilvum PYR- GCK];PFAM: cytochrome c assembly protein; KEGG: mva:Mvan_0869 cytochrome c assembly protein;Cytochrome C assembly protein; cl00504
GOHSU14	CDS	GOHSU_14_00120	complement(9928..11559)		putative cytochrome c biogenesis protein ResB [nite]	 resB [nite]		RER_16700 :putative cytochrome c biogenesis protein ResB : resB : : Rhodococcus erythropolis PR4	ResB family protein		F1YIP7_9ACTO ResB family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_08856 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ResB family protein; KEGG: dol:Dole_1104 ResB family protein;ResB-like family; pfam05140;ResB protein required for cytochrome c biosynthesis [Posttranslational modification, protein turnover, chaperones]; cl09125
GOHSU14	CDS	GOHSU_14_00130	complement(11556..12359)		cytochrome c biogenesis protein CcdA [nite]	 ccdA [nite]		RER_16690 :cytochrome c biogenesis protein CcdA : ccdA : 1.8.-.- : Rhodococcus erythropolis PR4	cytochrome c biogenesis protein, transmembrane region;hypothetical protein		F1YIP8_9ACTO Cytochrome c-type biosynthesis protein CCDA OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08861 PE=4 SV=1;cytochrome c biogenesis protein, transmembrane regi on [Mycobacterium gilvum PYR- GCK];Signal predicted by SignalP 2.0 HMM (Signal peptide probability0.963) with cleavage site probability 0.961 at residue 32;PFAM: cytochrome c biogenesis protein, transmembrane region; KEGG: mva:Mvan_0867 cytochrome c biogenesis protein, transmembrane region;Cytochrome C biogenesis protein transmembrane region; cl15788
GOHSU14	CDS	GOHSU_14_00140	complement(12356..12961)		cytochrome c biogenesis protein ResA [nite]	 resA [nite]		ROP_17090 :cytochrome c biogenesis protein ResA : resA : 1.8.-.- : Rhodococcus opacus B4	alkyl hydroperoxide reductase/ Thiol specific antioxidant/ Mal allergen;hypothetical protein		F1YIP9_9ACTO Redoxin domain-containing protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_08866 PE=4 SV=1 ;alkyl hydroperoxide reductase/ Thiol specific antio xidant/ Mal allergen [Mycobacterium gilvum PYR- GCK];Signal predicted by SignalP 2.0 HMM (Signal peptide probability0.992) with cleavage site probability 0.564 at residue 24;PFAM: alkyl hydroperoxide reductase/ Thiol specific antioxidant/ Mal allergen; Redoxin domain protein; KEGG: nfa:nfa51580 putative cytochrome c biogenesis protein;TlpA-like family; composed of  TlpA, ResA, DsbE and similar proteins. TlpA, ResA and DsbE are bacterial protein disulfide reductases with important roles in cytochrome maturation. They are membrane-anchored proteins with a soluble TRX domain containing a...; cd02966;catalytic residues [active]
GOHSU14	CDS	GOHSU_14_00150	complement(12967..13596)		phosphoglycerate mutase family protein [nite]			ROP_17100 :phosphoglycerate mutase family protein : : : Rhodococcus opacus B4	phosphoglycerate mutase		G7H1C8_9ACTO Putative phosphoglycerate mutase famil y protein OS=Gordonia araii NBRC 100433 GN=GOARA_045_00070 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: phosphoglycerate mutase; KEGG: aav:Aave_4233 phosphoglycerate mutase;Histidine phosphatase domain found in phosphoglycerate mutases and related proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cd07067;catalytic core [active]
GOHSU14	CDS	GOHSU_14_00160	complement(13601..14935)		glutamate-1-semialdehyde aminotransferase [nite]	 hemL [nite]		RER_16660 :glutamate-1-semialdehyde aminotransferase : hemL : 5.4.3.8 : Rhodococcus erythropolis PR4	glutamate-1-semialdehyde aminotransferase	hemL	F1YIQ1_9ACTO Glutamate-1-semialdehyde aminotransfer ase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08876 P E=3 SV=1;glutamate-1-semialdehyde aminotransferase [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;Converts (S)-4-amino-5-oxopentanoate to 5-aminolevulinate during the porphyrin biosynthesis pathway;glutamate-1-semialdehyde aminotransferase; Provisional;Acetyl ornithine aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). The major groups in this CD correspond to ornithine aminotransferase, acetylornithine aminotransferase; cd00610;inhibitor-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU14	CDS	GOHSU_14_00170	complement(15047..15892)		putative esterase [nite]			KSE_25240 :putative esterase : : : Kitasatospora setae NBRC 14216	putative hydrolase		G7GR30_9ACTO Putative esterase OS=Gordonia amarae N BRC 15530 GN=GOAMR_46_01530 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];SEST_like. A family of secreted SGNH-hydrolases similar to Streptomyces scabies esterase (SEST), a causal agent of the potato scab disease, which hydrolyzes a specific ester bond in suberin, a plant lipid. The tertiary fold of this enzyme is...; cd01823;catalytic triad [active];oxyanion hole [active]
GOHSU14	CDS	GOHSU_14_00180	complement(15889..16521)		hypothetical membrane protein [nite]			ROP_17200 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YIQ3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08886 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU14	CDS	GOHSU_14_00190	complement(16518..16979)		hypothetical protein [migap]				hypothetical protein		F1YIQ4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08891 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: scl:sce1521 hypothetical protein
GOHSU14	CDS	GOHSU_14_00200	complement(16976..17479)		adenosylmethionine-8-amino-7-oxononanoate aminotransferase [migap]				adenosylmethionine-8-amino-7-oxononanoate aminotransferase		F1YIQ5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08896 PE=4 SV=1;adenosylmethionine-8-amino-7-oxononanoate aminotran sferase [Desulfovibrio vulgaris DP4];TIGRFAM: adenosylmethionine-8-amino-7-oxononanoate aminotransferase; PFAM: aminotransferase class-III; KEGG: dvu:DVU2559 adenosylmethionine--8-amino-7-oxononanoate aminotransferase;Acetyl ornithine aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). The major groups in this CD correspond to ornithine aminotransferase, acetylornithine aminotransferase; cd00610;adenosylmethionine-8-amino-7-oxononanoate transaminase; TIGR00508;catalytic residue [active]
GOHSU14	CDS	GOHSU_14_00210	complement(17599..18549)		delta-aminolevulinic acid dehydratase [nite]	 hemB [nite]		RER_16440 :delta-aminolevulinic acid dehydratase : hemB : 4.2.1.24 : Rhodococcus erythropolis PR4	delta-aminolevulinic acid dehydratase	hemB	F1YIQ6_9ACTO Delta-aminolevulinic acid dehydratase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08901 PE=3 SV=1;delta-aminolevulinic acid dehydratase [Corynebacter ium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of porphobilinogen from 5-aminolevulinate;Porphobilinogen synthase (PBGS), which is also called delta-aminolevulinic acid dehydratase (ALAD), catalyzes the condensation of two 5-aminolevulinic acid (ALA) molecules to form the pyrrole porphobilinogen (PBG), which is the second step in the...; cd00384;dimer interface [polypeptide binding];Schiff base residues
GOHSU14	CDS	GOHSU_14_00220	18661..19236		hypothetical membrane protein [nite]			RER_00790 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0L543_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_1191 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU14	CDS	GOHSU_14_00230	complement(19259..20878)		putative uroporphyrinogen-III methyltransferase/uroporphyrinogen-III synthase [nite]	 cysGA/hemD [nite]		ROP_17230 :putative uroporphyrinogen-III methyltransferase/uroporphyrinogen-III synthase : cysGA/hemD : 2.1.1.107/4.2.1.75 : Rhodococcus opacus B4	putative uroporphyrin-III C-methyltransferase/uroporphyrinogen-III synthase		F1YIQ7_9ACTO Uroporphyrin-III C-methyltransferase H emD OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08906 P E=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Precorrin-3B methylase [Coenzyme metabolism]; cl00304;Uroporphyrinogen-III synthase (HemD) catalyzes the asymmetrical cyclization of tetrapyrrole (linear) to uroporphyrinogen-III, the fourth step in the biosynthesis of heme. This ubiquitous enzyme is present in eukaryotes, bacteria and archaea. Mutations in...; cd06578;Uroporphyrinogen-III synthase HemD; pfam02602
GOHSU14	CDS	GOHSU_14_00240	complement(20950..21894)	case	porphobilinogen deaminase [hamap]	hemC [hamap]	MF_00260:porphobilinogen deaminase:hemC 	RER_16420 :porphobilinogen deaminase : hemC : 2.5.1.61 : Rhodococcus erythropolis PR4	porphobilinogen deaminase	hemC	F1YIQ8_9ACTO Porphobilinogen deaminase OS=Gordonia neofelifaecis NRRL B-59395 GN=hemC PE=3 SV=1;porphobilinogen deaminase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;porphobilinogen deaminase; Reviewed; PRK00072;transformation of porphobilinogen to hydroxymethylbilane in porphyrin biosynthesis;Hydroxymethylbilane synthase (HMBS), also known as porphobilinogen deaminase (PBGD), is an intermediate enzyme in the biosynthetic pathway of tetrapyrrolic ring systems, such as heme, chlorophylls, and vitamin B12. HMBS catalyzes the conversion of...; cl03189;domain interfaces
GOHSU14	CDS	GOHSU_14_00250	complement(22011..23348)		glutamyl-tRNA reductase [hamap]	hemA [hamap]	MF_00087:glutamyl-tRNA reductase:hemA 	ROP_17250 :putative glutamyl-tRNA reductase : hemA : 1.2.1.70 : Rhodococcus opacus B4	glutamyl-tRNA reductase	hemA	F1YIQ9_9ACTO Glutamyl-tRNA reductase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=hemA PE=3 SV=1;glutamyl-tRNA reductase [Corynebacterium glutamicum ATCC 13032];catalyzes the formation of glutamate-1-semialdehyde from glutamyl-tRNA(Glu) and NADPH; the second step of the pathway is catalyzed by glutamate-1-semialdehyde aminomutase which results in the formation of 5-aminolevulinic acid; functions in porphyrin (tetrapyrroles) biosynthesis; the crystal structure showed a C-terminal dimerization domain that appears to be absent in Chlamydial proteins;glutamyl-tRNA reductase; Reviewed; PRK00045;NADP-binding domain of glutamyl-tRNA reductase; cd05213;tRNA;putative tRNA binding site [nucleotide binding];putative NADP binding site [chemical binding];Glutamyl-tRNAGlu reductase, dimerisation domain; pfam00745
GOHSU14	CDS	GOHSU_14_00260	complement(23457..23741)		conserved hypothetical protein [nite]			RER_16390 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	glutaredoxin		F1YIR0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08921 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.746) with cleavage site probability 0.736 at residue 20;PFAM: glutaredoxin 2; KEGG: bph:Bphy_0838 glutaredoxin 2;Protein Disulfide Oxidoreductases and Other Proteins with a Thioredoxin fold; cl00388
GOHSU14	CDS	GOHSU_14_00270	23883..25067		putative hydrolase [nite]			RER_16380 :putative hydrolase : : : Rhodococcus erythropolis PR4	HAD-superfamily hydrolase		F1YIR1_9ACTO HAD-superfamily hydrolase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08926 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: HAD-superfamily subfamily IB hydrolase, TIGR01490; HAD-superfamily hydrolase, subfamily IB (PSPase- like); PFAM: haloacid dehalogenase; KEGG: afw:Anae109_1214 HAD family hydrolase;Phosphoserine phosphatase [Amino acid transport and metabolism]; COG0560;Haloacid Dehalogenase-like Hydrolases; cl11391
GOHSU14	CDS	GOHSU_14_00280	complement(25120..26193)		conserved hypothetical protein [nite]			RER_16370 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	phospholipid/glycerol acyltransferase		F1YIR4_9ACTO Phospholipid/glycerol acyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08941 PE=4 SV=1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];PFAM: phospholipid/glycerol acyltransferase; KEGG: sco:SCO3325 hypothetical protein;Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: MGAT-like; cd07987;putative acyl-acceptor binding pocket
GOHSU14	CDS	GOHSU_14_00290	complement(26310..27365)		putative nucleotide-sugar epimerase [nite]			RER_16360 :putative nucleotide-sugar epimerase : : 5.1.3.- : Rhodococcus erythropolis PR4	NAD-dependent epimerase/dehydratase		F1YIR5_9ACTO NAD-dependent epimerase/dehydratase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08946 PE=4 SV =1;NAD-dependent epimerase/dehydratase [Streptomyces c lavuligerus ATCC 27064];KEGG: Amino sugar and nucleotide sugar metabolism Galactose metabolism Metabolic pathways. COG: COG0451 Nucleoside-diphosphate-sugar epimerases.;Nucleoside-diphosphate-sugar epimerases [Cell envelope biogenesis, outer membrane / Carbohydrate transport and metabolism]; COG0451;UDP-glucose 4 epimerase (G4E), subgroup 3, extended (e) SDRs; cd05240;putative NAD(P) binding site [chemical binding];putative substrate binding site [chemical binding]
GOHSU14	CDS	GOHSU_14_00300	complement(27749..28030)		hypothetical protein [nite]			ROP_17310 :hypothetical protein : : : Rhodococcus opacus B4	excisionase		F1YIR7_9ACTO DNA-binding domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08956 PE=4 SV=1;excisionase DNA binding domain-containing protein [ Pseudonocardia dioxanivorans CB1190];KEGG: svi:Svir_02760 DNA-binding protein, excisionase; TIGRFAM: Excisionase/Xis, DNA-binding; PFAM: Prophage CP4-57 regulatory;Helix-Turn-Helix DNA binding domain of transcription regulators from the MerR superfamily; cl02600
GOHSU14	CDS	GOHSU_14_00310	complement(28178..29131)		pyrroline-5-carboxylate reductase [nite]	 proC [nite]		ROP_17320 :pyrroline-5-carboxylate reductase : proC : 1.5.1.2 : Rhodococcus opacus B4	pyrroline-5-carboxylate reductase	proC	F1YIR8_9ACTO Pyrroline-5-carboxylate reductase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_08961 PE=3 SV=1 ;pyrroline-5-carboxylate reductase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;pyrroline-5-carboxylate reductase; Reviewed;catalyzes the formation of L-proline from pyrroline-5-carboxylate;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU14	CDS	GOHSU_14_00320	complement(29155..30036)		hypothetical protein [nite]			ROP_17330 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5TYU5_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_048_00330 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];cytoplasmic protein;Thioesterase-like superfamily; pfam13622;The hotdog fold was initially identified in the E. coli FabA (beta-hydroxydecanoyl-acyl carrier protein (ACP)-dehydratase) structure and subsequently in 4HBT (4-hydroxybenzoyl-CoA thioesterase) from Pseudomonas. A number of other seemingly unrelated...; cl00509
GOHSU14	CDS	GOHSU_14_00330	complement(30017..30964)		conserved hypothetical protein [nite]			RER_16260 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	xylose isomerase		F1YIS0_9ACTO Xylose isomerase domain-containing pro tein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08971 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: xylose isomerase; KEGG: gur:Gura_0975 xylose isomerase domain- containing protein;Sugar phosphate isomerases/epimerases [Carbohydrate transport and metabolism]; COG1082;AP endonuclease family 2; These endonucleases play a role in DNA repair. Cleave phosphodiester bonds at apurinic or apyrimidinic sites; the alignment also contains hexulose-6-phosphate isomerases, enzymes that catalyze the epimerization of...; cl12060;AP (apurinic/apyrimidinic) site pocket;Metal-binding active site
GOHSU14	CDS	GOHSU_14_00340	complement(30992..31915)		not assigned						F1YIS1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08976 PE=4 SV=1
GOHSU14	CDS	GOHSU_14_00350	complement(31979..32929)		hypothetical protein [nite]			ROP_17370 :hypothetical protein : : : Rhodococcus opacus B4	exopolyphosphatase	ppx1	B1MHK6_MYCA9 Putative uncharacterized protein OS=My cobacterium abscessus (strain ATCC 19977 / DSM 44196) GN=M AB_4013c PE=4 SV=1;exopolyphosphatase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Ppx/GppA phosphatase family; cl15449
GOHSU14	CDS	GOHSU_14_00360	32993..33835		conserved hypothetical protein [nite]			RER_16230 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TPY4_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_170_00090 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_00370	complement(33870..34559)		two-component response regulator RegX3 [nite]	 regX3 [nite]		RER_16220 :two-component response regulator RegX3 : regX3 : : Rhodococcus erythropolis PR4	putative sensory transduction protein		F1YIT5_9ACTO Response regulator receiver OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_08991 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Response regulators consisting of a CheY-like receiver domain and a winged-helix DNA-binding domain [Signal transduction mechanisms / Transcription]; COG0745;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];Effector domain of response regulator. Bacteria and certain eukaryotes like protozoa and higher plants use two-component signal transduction systems to detect and respond to changes in the environment. The system consists of a sensor histidine kinase and...; cd00383;DNA binding site [nucleotide binding]
GOHSU14	CDS	GOHSU_14_00380	complement(34556..35764)		two-component histidine kinase SenX3 [nite]	 senX3 [nite]		ROP_17410 :two-component histidine kinase SenX3 : senX3 : 2.7.13.- : Rhodococcus opacus B4	putative signal-transduction histidine kinase		F1YIT6_9ACTO ATP-binding region ATPase domain-conta ining protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SC NU_08996 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Histidine Kinase A (dimerization/phosphoacceptor) domain; Histidine Kinase A dimers are formed through parallel association of 2 domains creating 4-helix bundles; usually these domains contain a conserved His residue and are activated via...; cd00082;dimer interface [polypeptide binding];phosphorylation site [posttranslational modification];Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU14	CDS	GOHSU_14_00390	complement(35880..36203)		conserved hypothetical protein [nite]			RER_28270 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	GNAT family acetyltransferase		F1YIT7_9ACTO Acetyltransferase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_09001 PE=4 SV=1;GNAT family acetyltransferase [Rhodococcus equi 103 S]
GOHSU14	CDS	GOHSU_14_00400	complement(36307..37071)		2,3-bisphosphoglycerate-dependent phosphoglycerate mutase [hamap]	gpmA [hamap]	MF_01039:2,3-bisphosphoglycerate-dependent phosphoglycerate mutase:gpmA 	ROP_17420 :2,3-bisphosphoglycerate-dependent phosphoglycerate mutase : gpmA : 5.4.2.1 : Rhodococcus opacus B4	phosphoglyceromutase	gpmA	F1YIT8_9ACTO 2,3-bisphosphoglycerate-dependent phos phoglycerate mutase OS=Gordonia neofelifaecis NRRL B-59395 GN=gpmA PE=3 SV=1;phosphoglyceromutase [Corynebacterium glutamicum AT CC 13032];IS fingerprint type 4-5;2,3-bisphosphoglycerate-dependent; catalyzes the interconversion of 2-phosphoglycerate to 3-phosphoglycerate;Histidine phosphatase domain found in phosphoglycerate mutases and related proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cd07067;catalytic core [active]
GOHSU14	CDS	GOHSU_14_00410	complement(37132..37593)		hypothetical protein [migap]				hypothetical protein		H0RBR0_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_031_01290 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_00420	complement(37683..39287)		4-coumarate--CoA ligase [nite]			ROP_17430 :4-coumarate--CoA ligase : : 6.2.1.12 : Rhodococcus opacus B4	acyl-CoA synthetase	fadD7	F9VUU4_9ACTO Putative 4-coumarate--CoA ligase OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_050_02380 PE=4 SV= 1;acyl-CoA synthetase [Corynebacterium variabile DSM 44702];Acyl-CoA synthetases (AMP-forming)/AMP-acid ligases II;4-coumarate--CoA ligase;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU14	CDS	GOHSU_14_00430	complement(39338..39946)		conserved hypothetical protein [nite]			RER_16180 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YIT9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09011 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF2596;Putative bacterial sensory transduction regulator; cl15834
GOHSU14	CDS	GOHSU_14_00440	complement(39943..41259)		putative glycosyltransferase MshA [nite]	 mshA [nite]		ROP_17450 :putative glycosyltransferase MshA : mshA : 2.4.-.- : Rhodococcus opacus B4	glycosyl transferase		F1YIU0_9ACTO UDP-N-acetylglucosamine OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_09016 PE=3 SV=1;glycosyl transferase [Streptomyces sp. Mg1];This family is most closely related to the GT1 family of glycosyltransferases. The sucrose-phosphate synthases in this family may be unique to plants and photosynthetic bacteria. This enzyme catalyzes the synthesis of sucrose 6-phosphate from fructose...; cd03800;UDP-N-acetylglucosamine: 1L-myo-inositol-1-phosphate 1-alpha-D-N-acetylglucosaminyltransferase; TIGR03449;putative ADP-binding pocket [chemical binding]
GOHSU14	CDS	GOHSU_14_00450	41344..42312		putative ABC transporter substrate-binding protein [nite]			ROP_44070 :putative ABC transporter substrate-binding protein : : : Rhodococcus opacus B4	periplasmic solute binding protein		F1YIU1_9ACTO Periplasmic solute-binding protein OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09021 PE=4 SV= 1;periplasmic solute binding protein [Burkholderia ph ytofirmans PsJN];PFAM: periplasmic solute binding protein; KEGG: acr:Acry_2369 periplasmic solute binding protein;Helical backbone metal receptor (TroA-like domain). These proteins have been shown to function in the ABC transport of ferric siderophores and metal ions such as Mn2+, Fe3+, Cu2+ and/or Zn2+.  Their ligand binding site is formed in the interface between...; cl00262;Periplasmic solute binding protein family; pfam01297;intersubunit interface [polypeptide binding]
GOHSU14	CDS	GOHSU_14_00460	42309..43127		putative ABC transporter ATP-binding protein [nite]			RER_07020 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter		D0L472_GORB4 ABC transporter related protein OS=Gor donia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 319 8 / NCTC 10667) GN=Gbro_0985 PE=3 SV=1;ABC transporter [Burkholderia phytofirmans PsJN];PFAM: ABC transporter; SMART: ATPase AAA; KEGG: bxe:Bxe_A0716 ABC Mn2+/Zn2+ transporter ATPase;ABC-type Mn/Zn transport systems, ATPase component [Inorganic ion transport and metabolism]; COG1121;ABC component of the metal-type transporters.  This family includes transporters involved in the uptake of various metallic cations such as iron, manganese, and zinc.  The ATPases of this group of transporters are very similar to members of...; cd03235
GOHSU14	CDS	GOHSU_14_00470	43124..43978		putative ABC transporter permease protein [nite]			RER_07010 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	putative ABC metal ion transporter, permease component		F1YIU3_9ACTO ABC-3 protein OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_09031 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cl00462;ABC-ATPase subunit  interface;putative PBP binding regions
GOHSU14	CDS	GOHSU_14_00480	complement(44017..44508)		hypothetical protein [migap]				hypothetical protein		H0R089_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_055_00030 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF2599); pfam10783
GOHSU14	CDS	GOHSU_14_00490	complement(44577..46241)		putative carboxylesterase [nite]			ROP_56300 :putative carboxylesterase : : 3.1.1.1 : Rhodococcus opacus B4	carboxylesterase LipT		F1YIU5_9ACTO Carboxylesterase OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_09041 PE=4 SV=1;carboxylesterase LipT [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q272Q3_MYCFV e-149;Carboxylesterase type B [Lipid metabolism]; COG2272;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;substrate binding pocket [chemical binding];catalytic triad [active]
GOHSU14	CDS	GOHSU_14_00500	46248..47000		conserved hypothetical protein [nite]			RER_07000 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	LamB/YcsF family protein		D0L462_GORB4 LamB/YcsF family protein OS=Gordonia b ronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCT C 10667) GN=Gbro_0975 PE=4 SV=1;LamB/YcsF family protein [Arthrobacter chlorophenol icus A6];LamB/YcsF family; cl00664;PFAM: LamB/YcsF family protein; KEGG: art:Arth_0093 LamB/YcsF family protein
GOHSU14	CDS	GOHSU_14_00510	47031..47981		hypothetical membrane protein [nite]			KRH_05470 :hypothetical membrane protein : : : Kocuria rhizophila DC2201	permease		E6SAK1_INTC7 RarD protein, DMT superfamily transpor ter OS=Intrasporangium calvum (strain ATCC 23552 / DSM 430 43 / JCM 3097 / NBRC 12989 / 7 KIP) GN=Intca_0707 PE=4 SV= 1;permease [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Predicted permeases [General function prediction only]; COG2962;EamA-like transporter family; cl01037
GOHSU14	CDS	GOHSU_14_00520	complement(48019..48711)		not assigned						
GOHSU14	CDS	GOHSU_14_00530	complement(48777..49742)		putative RNA methyltransferase [nite]			RER_06900 :putative RNA methyltransferase : : 2.1.1.- : Rhodococcus erythropolis PR4	TrmH family RNA methyltransferase		D0L461_GORB4 RNA methyltransferase, TrmH family, gr oup 3 OS=Gordonia bronchialis (strain ATCC 25592 / DSM 432 47 / JCM 3198 / NCTC 10667) GN=Gbro_0974 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: RNA methyltransferase, TrmH family, group 3; PFAM: tRNA/rRNA methyltransferase (SpoU); RNA 2-O ribose methyltransferase substrate binding; KEGG: afw:Anae109_2230 RNA methyltransferase;rRNA methylases [Translation, ribosomal structure and biogenesis]; COG0566;RNA 2'-O ribose methyltransferase substrate binding; cl06870;SpoU rRNA Methylase family; cl00362
GOHSU14	CDS	GOHSU_14_00540	complement(49745..51181)		cysteinyl-tRNA synthetase [nite]	 cysS [nite]		ROP_43800 :cysteinyl-tRNA synthetase : cysS : 6.1.1.16 : Rhodococcus opacus B4	cysteinyl-tRNA synthetase	cysS	F1YIV0_9ACTO Cysteine--tRNA ligase OS=Gordonia neof elifaecis NRRL B-59395 GN=cysS PE=3 SV=1;cysteinyl-tRNA synthetase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;cysteinyl-tRNA synthetase; Validated; PRK00260;catalyzes a two-step reaction; charges a cysteine by linking its carboxyl group to the alpha-phosphate of ATP then transfers the aminoacyl-adenylate to its tRNA;nucleotidyl transferase superfamily; cl00015;HIGH motif;nucleotide binding site [chemical binding];catalytic core domain of cysteinyl tRNA synthetase; cd00672;KMSKS motif;Anticodon-binding domain of class Ia aminoacyl tRNA synthetases and similar domains; cl12020
GOHSU14	CDS	GOHSU_14_00550	complement(51193..51672)		2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase [hamap]	ispF [hamap]	MF_00107:2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase:ispF 	RER_06880 :2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase : ispF : 4.6.1.12 : Rhodococcus erythropolis PR4	2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase	ispF	G4HRG6_MYCRH 2-C-methyl-D-erythritol 2,4-cyclodipho sphate synthase OS=Mycobacterium rhodesiae JS60 GN=ispF PE =3 SV=1;2-C-methyl-D-erythritol 2,4-cyclodiphosphate syntha se [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the conversion of 4-diphosphocytidyl-2-C-methyl-D-erythritol 2-phosphate into 2-C-methyl-D-erythritol 2,4-cyclodiphosphate;MECDP_synthase (2-C-methyl-D-erythritol-2,4-cyclodiphosphate synthase), encoded by the ispF gene, catalyzes the formation of 2-C-methyl-D-erythritol 2,4-cyclodiphosphate (MEC) in the non-mevalonate deoxyxylulose (DOXP) pathway for isoprenoid biosynthesis; cd00554;homotrimer interaction site [polypeptide binding];zinc binding site [ion binding];CDP-binding sites
GOHSU14	CDS	GOHSU_14_00560	complement(51669..52337)		2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase [hamap]	ispD [hamap]	MF_00108:2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase:ispD 	ROP_43780 :2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase : ispD : 2.7.7.60 : Rhodococcus opacus B4	2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase	ispD	F1YIV2_9ACTO 2-C-methyl-D-erythritol 4-phosphate cy tidylyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=ispD PE=3 SV=1;2-C-methyl-D-erythritol 4-phosphate cytidylyltransf erase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;4-diphosphocytidyl-2C-methyl-D-erythritol synthase; MEP cytidylyltransferase; MCT; catalyzes the formation of 4-diphosphocytidyl-2-C-methyl-D-erythritol from CTP and 2-C-methyl-D-erythritol 4-phosphate; involved in isoprenoid and isopentenyl-PP biosynthesis; forms homodimers;CDP-ME synthetase is involved in mevalonate-independent isoprenoid production; cd02516;substrate binding site;dimer interface
GOHSU14	CDS	GOHSU_14_00570	complement(52388..52876)		conserved hypothetical protein [nite]			RER_06860 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	CarD family transcriptional regulator		G7H1H7_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_045_00560 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];CarD-like/TRCF domain; cl00588
GOHSU14	CDS	GOHSU_14_00580	53209..53877		hypothetical protein [nite]			RER_06850 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YIV4_9ACTO Copper(I)-binding protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09086 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.998) with cleavage site probability 0.728 at residue 24
GOHSU14	CDS	GOHSU_14_00590	54038..55387		putative DNA repair protein RadA [nite]	 radA [nite]		RER_06840 :putative DNA repair protein RadA : radA : : Rhodococcus erythropolis PR4	DNA repair protein RadA	radA	F1YIV5_9ACTO DNA repair protein radA OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_09091 PE=3 SV=1;DNA repair protein RadA [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;DNA repair protein RadA; Provisional;Sms; stabilizes the strand-invasion intermediate during the DNA repair; involved in recombination of donor DNA and plays an important role in DNA damage repair after exposure to mutagenic agents;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A motif;ATP binding site [chemical binding];Walker B motif;Subunit ChlI of Mg-chelatase; cl12214
GOHSU14	CDS	GOHSU_14_00600	complement(55409..55942)		conserved hypothetical protein [nite]			RER_06830 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	uncharacterized protein, possibly involved in aromatic compounds catabolism		F1YIV6_9ACTO Thioesterase superfamily protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09096 PE=4 SV=1;uncharacterized protein, possibly involved in aroma tic compounds catabolism [Saccharomonospora marina XMU15];PFAM: Thioesterase superfamily; manually curated;PaaI_thioesterase is a tetrameric acyl-CoA thioesterase with a hot dog fold and one of several proteins responsible for phenylacetic acid (PA) degradation in bacteria.  Although orthologs of PaaI exist in archaea and eukaryotes, their function has not...; cd03443;CoenzymeA binding site [chemical binding];subunit interaction site [polypeptide binding];PHB binding site
GOHSU14	CDS	GOHSU_14_00610	56051..57208		DNA integrity scanning protein DisA [hamap]	disA [hamap]	MF_01438:DNA integrity scanning protein DisA:disA 	RER_06820 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	DNA integrity scanning protein DisA		H0R0A0_9ACTO DNA integrity scanning protein DisA OS =Gordonia effusa NBRC 100432 GN=disA PE=3 SV=1;DNA integrity scanning protein DisA [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;non-specific DNA-binding; scans chromosomes during sporulation for DNA-damage; delays initiation of sporulation; participates in a checkpoint signaling cascade for cell-cycle progression and DNA repair;DNA integrity scanning protein DisA; Provisional;DisA bacterial checkpoint controller nucleotide-binding; pfam02457;DisA bacterial checkpoint controller linker region; pfam10635
GOHSU14	CDS	GOHSU_14_00620	complement(57244..57999)		hypothetical protein [nite]			RER_06810 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TYZ6_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_049_00300 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU14	CDS	GOHSU_14_00630	complement(58112..58780)		carbonic anhydrase [nite]			RER_06800 :carbonic anhydrase : : 4.2.1.1 : Rhodococcus erythropolis PR4	carbonate dehydratase		H5UE96_9ACTO Carbonic anhydrase OS=Gordonia terrae NBRC 100016 GN=GOTRE_057_00270 PE=3 SV=1;carbonate dehydratase [Rhodococcus jostii RHA1];Carbonic anhydrases (CA) are zinc-containing enzymes that catalyze the reversible hydration of carbon dioxide in a two-step mechanism in which the nucleophilic attack of a zinc-bound hydroxide ion on carbon dioxide is followed by the regeneration of an...; cd03378;active site clefts [active];zinc binding site [ion binding];dimer interface [polypeptide binding]
GOHSU14	CDS	GOHSU_14_00640	58779..59741		putative adenine glycosylase [nite]	 mutY [nite]		RER_06700 :putative adenine glycosylase : mutY : 3.2.2.- : Rhodococcus erythropolis PR4	HhH-GPD family protein		F1YIW0_9ACTO HhH-GPD family protein OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_09116 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: HhH-GPD family protein; helix-hairpin-helix motif; iron-sulfur cluster loop; SMART: HhH-GPD family protein; KEGG: pla:Plav_0701 A/G-specific adenine glycosylase;endonuclease III; includes endonuclease III (DNA-(apurinic or apyrimidinic site) lyase), alkylbase DNA glycosidases (Alka-family) and other DNA glycosidases; cd00056;endonuclease III; smart00478;minor groove reading motif;helix-hairpin-helix signature motif;substrate binding pocket [chemical binding];Iron-sulfur binding domain of endonuclease III; cl15304
GOHSU14	CDS	GOHSU_14_00650	complement(59766..61958)		protease II [nite]	 ptrB [nite]		ROP_49030 :protease II : ptrB : 3.4.21.83 : Rhodococcus opacus B4	oligopeptidase B		F1YLF8_9ACTO Oligopeptidase B OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_13749 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protease II [Amino acid transport and metabolism]; COG1770;PFAM: peptidase S9A prolyl oligopeptidase domain protein beta-propeller; peptidase S9 prolyl oligopeptidase active site domain protein; KEGG: ppg:PputGB1_4105 oligopeptidase B;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU14	CDS	GOHSU_14_00660	complement(61951..62862)		phosphoribosylaminoimidazole-succinocarboxamide synthase [hamap]	purC [hamap]	MF_00137:phosphoribosylaminoimidazole-succinocarboxamide synthase:purC 	ROP_49020 :phosphoribosylaminoimidazole-succinocarboxamide synthase : purC : 6.3.2.6 : Rhodococcus opacus B4	phosphoribosylaminoimidazole-succinocarboxamide synthase purc		F1YLF9_9ACTO Phosphoribosylaminoimidazole-succinoca rboxamide synthase OS=Gordonia neofelifaecis NRRL B-59395 GN=purC PE=3 SV=1;phosphoribosylaminoimidazole-succinocarboxamide syn thase purc [Mycobacterium tuberculosis T92];non-metazoan 5-aminoimidazole-4-(N-succinylcarboxamide) ribonucleotide (SAICAR) synthase; cd01414;ATP binding site [chemical binding];substrate binding site [chemical binding]
GOHSU14	CDS	GOHSU_14_00670	complement(62892..64328)		adenylosuccinate lyase [nite]	 purB [nite]		RER_10250 :adenylosuccinate lyase : purB : 4.3.2.2 : Rhodococcus erythropolis PR4	adenylosuccinate lyase	purB	F1YLG0_9ACTO Adenylosuccinate lyase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_13759 PE=4 SV=1;adenylosuccinate lyase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Catalyzes two discrete reactions in the de novo synthesis of purines: the cleavage of adenylosuccinate and succinylaminoimidazole carboxamide ribotide;adenylosuccinate lyase; TIGR00928;Lyase class I_like superfamily: contains the lyase class I family, histidine ammonia-lyase and phenylalanine ammonia-lyase, which catalyze similar beta-elimination reactions; cl00013;tetramer interface [polypeptide binding]
GOHSU14	CDS	GOHSU_14_00680	complement(64384..65319)		hypothetical protein [nite]			RER_59270 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0QVT5_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_017_00520 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.996) with cleavage site probability 0.483 at residue 29;KEGG: aav:Aave_0116 IclR family transcriptional regulator;SMP-30/Gluconolaconase/LRE-like region; pfam08450
GOHSU14	CDS	GOHSU_14_00690	65516..67714		conserved hypothetical membrane protein [nite]			RER_pREL1-02160 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	RND superfamily drug exporter protein	mmpL2	F9VX98_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_072_00400 PE=4 SV= 1;RND superfamily drug exporter protein [Corynebacter ium variabile DSM 44702];Predicted drug exporters of the RND superfamily;Predicted drug exporters of the RND superfamily [General function prediction only]
GOHSU14	CDS	GOHSU_14_00700	67760..68038		conserved hypothetical protein [nite]			RER_pREL1-01420 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H6R8Y8_NOCCG Uncharacterized protein OS=Nocardia cy riacigeorgica (strain GUH-2) GN=NOCYR_1571 PE=4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Transcriptional regulators CsoR (copper-sensitive operon repressor), RcnR, and FrmR, and related domains; this domain superfamily was previously known as DUF156; cd10148;putative homotetramer interface [polypeptide binding];putative homodimer interface [polypeptide binding];putative allosteric switch controlling residues;putative metal binding site [ion binding];putative homodimer-homodimer interface [polypeptide binding]
GOHSU14	CDS	GOHSU_14_00710	68058..68453		hypothetical protein [nite]			ROP_00370 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0JPS2_9NOCA Putative uncharacterized protein OS=Rh odococcus pyridinivorans AK37 GN=AK37_08697 PE=4 SV=1;unnamed protein product [Hyphomicrobium sp. MC1];Evidence 4 : Homologs of previously reported genes of unknown function;Domain of unknown function DUF302; cl01364
GOHSU14	CDS	GOHSU_14_00720	68492..69373		hypothetical protein [migap]				hypothetical protein		F1YLG4_9ACTO Putative esterase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_13779 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Differs from Rv0774c by 1aa A219S and from Mb0797c by 1aa D142A in BCG Pasteur,  BCG Tokyo and BCG Mexico;Alpha/beta hydrolase family; pfam12695;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU14	CDS	GOHSU_14_00730	complement(69513..70766)		phosphoribosylamine--glycine ligase [hamap]	purD [hamap]	MF_00138:phosphoribosylamine--glycine ligase:purD 	RER_10110 :phosphoribosylamine--glycine ligase : purD : 6.3.4.13 : Rhodococcus erythropolis PR4	phosphoribosylamine--glycine ligase	purD	F9VUE9_9ACTO Phosphoribosylamine--glycine ligase OS =Gordonia alkanivorans NBRC 16433 GN=purD PE=3 SV=1;phosphoribosylamine--glycine ligase [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;phosphoribosylamine--glycine ligase; Provisional;Phosphoribosylglycinamide synthetase, N domain; pfam02844;catalyzes the formation of N(1)-(5-phospho-D-ribosyl)glycinamide from 5-phospho-D-ribosylamine and glycine in purine biosynthesis;ATP-grasp domain; cl03087;Phosphoribosylglycinamide synthetase, C domain; pfam02843
GOHSU14	CDS	GOHSU_14_00740	complement(70777..71970)		hypothetical membrane protein [nite]			ROP_58670 :hypothetical membrane protein : : : Rhodococcus opacus B4	integral membrane protein		F1YLG7_9ACTO Low temperature requirement A OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_13794 PE=4 SV=1;integral membrane protein [Streptomyces clavuligeru s ATCC 27064];Bacterial low temperature requirement A protein (LtrA); cl01754
GOHSU14	CDS	GOHSU_14_00750	complement(72054..74210)		putative glycosyltransferase [nite]			ROP_48370 :putative glycosyltransferase : : 2.4.-.- : Rhodococcus opacus B4	putative integral membrane protein		F1YLG8_9ACTO Glycosyl transferase family 39 protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13799 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Dolichyl-phosphate-mannose-protein mannosyltransferase; cl15769
GOHSU14	CDS	GOHSU_14_00760	complement(74207..75421)		putative glycosyltransferase [nite]			ROP_48360 :putative glycosyltransferase : : 2.4.-.- : Rhodococcus opacus B4	family 2 glycosyl transferase		F1YLG9_9ACTO Glycosyl transferase family 2 protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13804 PE=4 SV=1;family 2 glycosyl transferase [Actinosynnema mirum DSM 43827];PFAM: glycosyl transferase family 2; GtrA family protein; KEGG: ppd:Ppro_2693 glycosyl transferase family protein;Glycosyltransferase family A (GT-A) includes diverse families of glycosyl transferases with a common GT-A type structural fold; cl11394;GtrA-like protein; cl00971
GOHSU14	CDS	GOHSU_14_00770	75648..77759		putative ATP-dependent DNA helicase [nite]			MLP_46900 :putative ATP-dependent DNA helicase : : 3.6.1.- : Microlunatus phosphovorus NM-1	ATP-dependent DNA helicase		F1YLH0_9ACTO DEAD/DEAH box helicase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 13809 PE=4 SV=1;ATP-dependent DNA helicase [Streptomyces clavuliger us ATCC 27064];DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ERCC4-like helicases [DNA replication, recombination, and repair]; COG1111;ATP-binding site [chemical binding]
GOHSU14	CDS	GOHSU_14_00780	complement(77789..78226)		HIT family protein [nite]			ROP_48350 :HIT family protein : : : Rhodococcus opacus B4	histidine triad (HIT)-like protein		F1YLH1_9ACTO Histidine triad (HIT) protein OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_13814 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];HIT family: HIT (Histidine triad) proteins, named for a motif related to the sequence HxHxH/Qxx (x, a hydrophobic amino acid), are a superfamily of nucleotide hydrolases and transferases, which act on the alpha-phosphate of ribonucleotides. On the basis...; cl00228;nucleotide binding site/active site [active];HIT family signature motif;catalytic residue [active]
GOHSU14	CDS	GOHSU_14_00790	complement(78236..79687)		two-component histidine kinase PhoR [nite]	 phoR [nite]		ROP_48340 :two-component histidine kinase PhoR : phoR : 2.7.13.- : Rhodococcus opacus B4	ATP-binding protein		H0R0Y1_9ACTO Two-component histidine kinase PhoR OS =Gordonia effusa NBRC 100432 GN=phoR PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.612) with cleavage site probability 0.496 at residue 51;PFAM: ATP-binding region ATPase domain protein; histidine kinase HAMP region domain protein; histidine kinase A domain protein; SMART: ATP-binding region ATPase domain protein; histidine kinase A domain protein; histidine kinase HAMP region domain protein; KEGG: gsu:GSU0452 sensor histidine kinase;Histidine kinase, Adenylyl cyclase, Methyl-accepting protein, and Phosphatase (HAMP) domain. HAMP is a signaling domain which occurs in a wide variety of signaling proteins, many of which are bacterial. The HAMP domain consists of two alpha helices...; cl01054;dimerization interface [polypeptide binding];Histidine Kinase A (dimerization/phosphoacceptor) domain; Histidine Kinase A dimers are formed through parallel association of 2 domains creating 4-helix bundles; usually these domains contain a conserved His residue and are activated via...; cd00082;dimer interface [polypeptide binding];phosphorylation site [posttranslational modification];Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU14	CDS	GOHSU_14_00800	complement(79689..80393)		two-component response regulator PhoP [nite]	 phoP [nite]		ROP_48330 :two-component response regulator PhoP : phoP : : Rhodococcus opacus B4	response regulator receiver protein		F1YLH3_9ACTO Response regulator receiver OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_13824 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: dat:HRM2_18670 two component transcriptional regulator (winged helix family protein);Response regulators consisting of a CheY-like receiver domain and a winged-helix DNA-binding domain [Signal transduction mechanisms / Transcription]; COG0745;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];Effector domain of response regulator. Bacteria and certain eukaryotes like protozoa and higher plants use two-component signal transduction systems to detect and respond to changes in the environment. The system consists of a sensor histidine kinase and...; cd00383;DNA binding site [nucleotide binding]
GOHSU14	CDS	GOHSU_14_00810	complement(80486..82063)		putative carboxylesterase [nite]			RER_57050 :putative carboxylesterase : : 3.1.1.1 : Rhodococcus erythropolis PR4	type B carboxylesterase		F1YLH7_9ACTO Para-nitrobenzyl esterase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13844 PE=4 SV=1;type B carboxylesterase [Mycobacterium gilvum PYR- GCK];PFAM: Carboxylesterase, type B; KEGG: mmc:Mmcs_2865 carboxylesterase, type B;Carboxylesterase type B [Lipid metabolism]; COG2272;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;substrate binding pocket [chemical binding];catalytic triad [active]
GOHSU14	CDS	GOHSU_14_00820	82385..83659		hypothetical protein [nite]			RER_49350 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5U7D3_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_142_00020 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.967 at residue 37
GOHSU14	CDS	GOHSU_14_00830	83656..85059		hypothetical protein [nite]			RER_49340 :hypothetical protein : : : Rhodococcus erythropolis PR4	putative secreted lipase		G7H728_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_082_00410 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Secretory lipase; pfam03583
GOHSU14	CDS	GOHSU_14_00840	85164..85646		transcriptional regulator, effector-binding domain/component [migap]				transcriptional regulator, effector-binding domain/component		F1YLH8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13849 PE=4 SV=1;transcriptional regulator, effector-binding domain/ component [Arthrobacter phenanthrenivorans Sphe3];PFAM: Bacterial transcription activator, effector binding domain;GyrI-like small molecule binding domain; cl01368
GOHSU14	CDS	GOHSU_14_00850	85766..85981		oxidoreductase [migap]				oxidoreductase		Q99PY4_STRCO Putative oxidoreductase OS=Streptomyce s coelicolor (strain ATCC BAA-471 / A3(2) / M145) GN=SCP1. 310c PE=4 SV=1;oxidoreductase [Streptomyces coelicolor A3(2)];possible oxidoreductase, len: 338aa; similar to many eg. TR:Q9RJW9 (EMBL:AL133210)  reductase (fragment) from Streptomyces coelicolor (311 aa) fasta scores; opt: 795, z-score: 852.7, E(): 0, 44.0% identity in 293 aa overlap and SW:O82864 (AHPF_PSEPU) alkyl hydroperoxide reductase subunit from Pseudomonas putida (520 aa) fasta scores; opt: 301, z-score: 325.3, E(): 1.2e-10, 25.6% identity in 301 aa overlap.;Thioredoxin reductase [Posttranslational modification, protein turnover, chaperones]; COG0492
GOHSU14	tRNA	GOHSU_14_t0010	complement(86267..86341)		not assigned						identified by tRNAscan-SE; putative
GOHSU14	CDS	GOHSU_14_00860	86342..87757		threonine export protein [nite]	 thrE [nite]		ROP_48160 :threonine export protein : thrE : : Rhodococcus opacus B4	hypothetical protein		G7GZF5_9ACTO Threonine export protein OS=Gordonia a raii NBRC 100433 GN=thrE PE=4 SV=1;unnamed protein product [Corynebacterium pseudotube rculosis FRC41];Uncharacterized conserved protein [Function unknown];Protein of unknown function (DUF1212); cl01118
GOHSU14	CDS	GOHSU_14_00870	87833..88006		hypothetical protein [nite]			ROP_48070 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5U7D6_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_142_00050 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU14	CDS	GOHSU_14_00880	88013..89416		putative alpha,alpha-trehalose-phosphate synthase [nite]	 otsA [nite]		ROP_48060 :putative alpha,alpha-trehalose-phosphate synthase : otsA : 2.4.1.15 : Rhodococcus opacus B4	alpha,alpha-trehalose-phosphate synthase (UDP- forming)		F1YLI4_9ACTO Alpha,alpha-trehalose-phosphate syntha se (UDP-forming) OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_13879 PE=4 SV=1;alpha,alpha-trehalose-phosphate synthase (UDP- form ing) [Catenulispora acidiphila DSM 44928];PFAM: glycosyl transferase family 20; KEGG: sen:SACE_0462 trehalose-6-phosphate synthase;Glycosyltransferase family 20; pfam00982;Trehalose-6-Phosphate Synthase (TPS) is a glycosyltransferase that catalyses the synthesis of alpha,alpha-1,1-trehalose-6-phosphate from glucose-6-phosphate using a UDP-glucose donor. It is a key enzyme in the trehalose synthesis pathway. Trehalose is a...; cd03788;homotetramer interface [polypeptide binding]
GOHSU14	CDS	GOHSU_14_00890	89481..90260		putative trehalose-phosphatase [nite]	 otsB [nite]		ROP_44090 :putative trehalose-phosphatase : otsB : 3.1.3.12 : Rhodococcus opacus B4	trehalose-phosphatase	otsB	F1YLI5_9ACTO HAD family hydrolase OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_13884 PE=4 SV=1;trehalose-phosphatase [Arthrobacter aurescens TC1];identified by match to protein family HMM PF02358; match to protein family HMM TIGR00685; match to protein family HMM TIGR01484;HAD-superfamily hydrolase, subfamily IIB; TIGR01484;Haloacid Dehalogenase-like Hydrolases; cl11391
GOHSU14	CDS	GOHSU_14_00900	complement(90288..91427)		putative NagC family transcriptional regulator [nite]			ROP_17530 :putative NagC family transcriptional regulator : : : Rhodococcus opacus B4	ROK family protein		F1YLI6_9ACTO ROK family protein OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_13889 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ROK family protein; KEGG: avn:Avin_03610 ROK (repressor, ORF, kinase) domain protein;Helix-turn-helix domains; cl00088;Transcriptional regulator/sugar kinase [Transcription / Carbohydrate transport and metabolism]; COG1940;FGGY family of carbohydrate kinases, N-terminal domain; cl09121
GOHSU14	CDS	GOHSU_14_00910	91471..92223		putative oxidoreductase [nite]			RER_16060 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	oxidoreductase		F1YLI7_9ACTO Short-chain dehydrogenase/reductase SD R OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13894 PE= 3 SV=1;oxidoreductase [Streptomyces coelicolor A3(2)];SC4A9.06c, possible oxidoreductase, len: 259 aa. Highly similar to Streptomyces coelicolor TR:Q9X890(EMBL:AL049707)  oxidoreductase, SCE15.15 (260 aa), fasta scores opt: 1088 z-score: 1221.9 E(): 0 66.9% identity in 254 aa overlap. Also similar to Streptomyces griseus subsp. griseus TR:Q9XDF5(EMBL:AF074603) ketoacyl reductase (271 aa), fasta scores opt: 421 z-score: 479.1 E(): 2.8e-19 36.7% identity in 256 aa overlap. Contains a Prosite hit to PS00061 Short-chain dehydrogenases/reductases family signature and a Pfam match to entry PF00106 adh_short, short chain dehydrogenase.;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU14	CDS	GOHSU_14_00920	complement(92259..92957)		putative S1 family peptidase [nite]			RER_50550 :putative S1 family peptidase : : 3.4.21.- : Rhodococcus erythropolis PR4	S1 family peptidase		H0RDE0_9ACTO Peptidase S1 family protein OS=Gordoni a polyisoprenivorans NBRC 16320 GN=GOPIP_039_00380 PE=4 SV =1;S1 family peptidase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887
GOHSU14	CDS	GOHSU_14_00930	93302..93871		RNA polymerase ECF-type sigma factor SigM [nite]	 sigM [nite]		ROP_34650 :RNA polymerase ECF-type sigma factor SigM : sigM : : Rhodococcus opacus B4	RNA polymerase sigma factor SigM		H5UEG8_9ACTO RNA polymerase ECF-type sigma factor S igM OS=Gordonia terrae NBRC 100016 GN=sigM PE=4 SV=1;RNA polymerase sigma factor SigM [Streptomyces clav uligerus ATCC 27064];RNA polymerase sigma factor SigM; Reviewed;Sigma-70 region 2; pfam04542;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU14	CDS	GOHSU_14_00940	complement(93884..94558)		deoxyribose-phosphate aldolase [hamap]	deoC [hamap]	MF_00114:deoxyribose-phosphate aldolase:deoC 	RER_15930 :deoxyribose-phosphate aldolase : deoC : 4.1.2.4 : Rhodococcus erythropolis PR4	deoxyribose-phosphate aldolase	deoC	F9VUC1_9ACTO Deoxyribose-phosphate aldolase OS=Gord onia alkanivorans NBRC 16433 GN=deoC PE=3 SV=1;deoC gene product [Mycobacterium smegmatis str. MC2 155];catalyzes the formation of D-glyceraldehyde 3-phosphate and acetaldehyde from 2-deoxy-D-ribose-5-phosphate;2-deoxyribose-5-phosphate aldolase (DERA) of the DeoC family; cd00959;intersubunit interface [polypeptide binding];catalytic residue [active]
GOHSU14	CDS	GOHSU_14_00950	94508..95497		formyltetrahydrofolate deformylase [nite]	 purU [nite]		ROP_17670 :formyltetrahydrofolate deformylase : purU : 3.5.1.10 : Rhodococcus opacus B4	formyltetrahydrofolate deformylase	purU	F1YLI9_9ACTO Formyltetrahydrofolate deformylase OS= Gordonia neofelifaecis NRRL B-59395 GN=purU PE=4 SV=1;formyltetrahydrofolate deformylase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;produces formate from formyl-tetrahydrofolate which is the major source of formate for PurT in de novo purine nucleotide biosynthesis; has a role in one-carbon metabolism; forms a homohexamer; activated by methionine and inhibited by glycine;formyltetrahydrofolate deformylase; Reviewed; PRK06027;N-terminal ACT domain of formyltetrahydrofolate deformylase (F4HF-DF; formyltetrahydrofolate hydrolase); cd04875;Formyltetrahydrofolate deformylase (Formyl-FH4 hydrolase), C-terminal hydrolase domain; cd08648;putative active site [active];putative substrate binding site [chemical binding];putative cosubstrate binding site;catalytic site [active]
GOHSU14	CDS	GOHSU_14_00960	complement(95514..96746)		hypothetical protein [migap]				hypothetical protein		F1YLJ1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13914 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_00970	complement(96756..97571)		putative hydrolase [nite]			MLP_36520 :putative hydrolase : : : Microlunatus phosphovorus NM-1	nitrilase/cyanide hydratase and apolipoprotein N-acyltransferase		G7GN80_9ACTO Putative hydrolase OS=Gordonia amarae NBRC 15530 GN=GOAMR_26_00280 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];Predicted amidohydrolase [General function prediction only];COG0388 Predicted amidohydrolase;Uncharacterized subgroup of the nitrilase superfamily (putative class 13 nitrilases); cd07581;putative active site [active];catalytic triad [active];putative dimer interface [polypeptide binding]
GOHSU14	CDS	GOHSU_14_00980	complement(97621..98607)		predicted protein [migap]				predicted protein		F1YLJ3_9ACTO Pyrrolo-quinoline quinone beta-propell er repeat-containing protein OS=Gordonia neofelifaecis NRR L B-59395 GN=SCNU_13924 PE=4 SV=1;predicted protein [Bifidobacterium longum subsp. in fantis CCUG 52486]
GOHSU14	CDS	GOHSU_14_00990	complement(98636..99433)		hypothetical protein [nite]			ROP_17610 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GN81_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_26_00300 PE=4 SV=1;unnamed protein product [Corynebacterium diphtheria e VA01];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU14	CDS	GOHSU_14_01000	complement(99430..99942)		hypothetical protein [nite]			ROP_17600 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLJ5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13934 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Protein of unknown function (DUF2505); pfam10698
GOHSU14	CDS	GOHSU_14_01010	complement(99968..100459)		hypothetical protein [nite]			ROP_17590 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLJ6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13939 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF2505); pfam10698
GOHSU14	CDS	GOHSU_14_01020	100484..101587		UDP-N-acetylenolpyruvoylglucosamine reductase [hamap]	murB [hamap]	MF_00037:UDP-N-acetylenolpyruvoylglucosamine reductase:murB 	ROP_17580 :UDP-N-acetylenolpyruvoylglucosamine reductase : murB : 1.1.1.158 : Rhodococcus opacus B4	UDP-N-acetylenolpyruvoylglucosamine reductase	murB	F1YLJ7_9ACTO UDP-N-acetylenolpyruvoylglucosamine re ductase OS=Gordonia neofelifaecis NRRL B-59395 GN=murB PE= 3 SV=1;UDP-N-acetylenolpyruvoylglucosamine reductase [Cory nebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the reduction of UDP-N-acetylglucosamine enolpyruvate to form UDP-N-acetylmuramate in peptidoglycan biosynthesis;UDP-N-acetylenolpyruvoylglucosamine reductase; Provisional; PRK13903;FAD binding domain; pfam01565;UDP-N-acetylenolpyruvoylglucosamine reductase, C-terminal domain; pfam02873
GOHSU14	CDS	GOHSU_14_01030	complement(101651..102706)		putative hydrolase [nite]			RER_15910 :putative hydrolase : : : Rhodococcus erythropolis PR4	hydrolase		F1YLJ8_9ACTO Alpha/beta hydrolase fold protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_13949 PE=4 SV=1 ;hydrolase [Nocardia farcinica IFM 10152];Alpha/beta hydrolase family; pfam12697;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU14	CDS	GOHSU_14_01040	complement(102699..103433)		putative MerR family transcriptional regulator [nite]			ROP_18210 :putative MerR family transcriptional regulator : : : Rhodococcus opacus B4	MerR family transcriptional regulator		H5TN68_9ACTO Putative MerR family transcriptional r egulator OS=Gordonia otitidis NBRC 100426 GN=GOOTI_130_001 90 PE=4 SV=1;MerR family transcriptional regulator [Streptomyces sp. C];Helix-Turn-Helix DNA binding domain of transcription regulators from the MerR superfamily; cl02600;MerR HTH family regulatory protein; pfam13411;DNA binding residues [nucleotide binding]
GOHSU14	CDS	GOHSU_14_01050	complement(103430..103792)		conserved hypothetical protein [nite]			RER_15900 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7H705_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_082_00170 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.655) with cleavage site probability 0.462 at residue 24;Protein of unknown function (DUF2516); pfam10724
GOHSU14	CDS	GOHSU_14_01060	complement(103844..104308)		putative Xre family DNA-binding protein [nite]			RER_15880 :putative Xre family DNA-binding protein : : : Rhodococcus erythropolis PR4	XRE family transcriptional regulator		H6MZM4_GORPV Helix-turn-helix domain-containing pro tein OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH 2) GN=GPOL_c09480 PE=4 SV=1;XRE family transcriptional regulator [Nakamurella m ultipartita DSM 44233];PFAM: helix-turn-helix domain protein; SMART: helix-turn-helix domain protein; KEGG: azc:AZC_2073 XRE family-like protein;Helix-turn-helix XRE-family like proteins. Prokaryotic DNA binding proteins belonging to the xenobiotic response element family of transcriptional regulators; cd00093;non-specific DNA binding site [nucleotide binding];salt bridge;sequence-specific DNA binding site [nucleotide binding]
GOHSU14	CDS	GOHSU_14_01070	complement(104400..105713)		conserved hypothetical protein [nite]			RER_15870 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLK2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13969 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF445; KEGG: glo:Glov_3026 protein of unknown function DUF445;Protein of unknown function (DUF445); pfam04286
GOHSU14	CDS	GOHSU_14_01080	105712..106551		putative TetR family transcriptional regulator [nite]			ROP_18260 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		F1YLK3_9ACTO TetR family transcriptional regulator OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13974 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m sp. KMS];PFAM: regulatory protein, TetR; KEGG: mmc:Mmcs_0640 transcriptional regulator, TetR family;transcriptional repressor BetI; TIGR03384;Helix-turn-helix domains; cl00088
GOHSU14	CDS	GOHSU_14_01090	106803..107444		conserved hypothetical protein [nite]			RER_15850 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	polyphosphate:nucleotide phosphotransferase		H6MZM0_GORPV Putative polyphosphate kinase OS=Gordo nia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c0 9440 PE=4 SV=1;polyphosphate:nucleotide phosphotransferase [Pseudo nocardia dioxanivorans CB1190];KEGG: nml:Namu_3539 hypothetical protein; TIGRFAM: Polyphosphate:nucleotide phosphotransferase, PPK2; PFAM: Polyphosphate kinase-2-related;Polyphosphate kinase 2 (PPK2); cl11971
GOHSU14	CDS	GOHSU_14_01100	complement(107538..108287)		putative heparin-binding hemagglutinin [nite]			RER_15890 :putative heparin-binding hemagglutinin : : : Rhodococcus erythropolis PR4	hypothetical protein		F9VU98_9ACTO Putative heparin-binding hemagglutinin OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_050_00410 PE =4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: abo:ABO_1439 hypothetical protein
GOHSU14	CDS	GOHSU_14_01110	complement(108585..109880)		isocitrate lyase [nite]	 aceA [nite]		ROP_18410 :isocitrate lyase : aceA : 4.1.3.1 : Rhodococcus opacus B4	isocitrate lyase	aceA	H6R488_NOCCG Isocitrate lyase OS=Nocardia cyriacige orgica (strain GUH-2) GN=aceA PE=4 SV=1;isocitrate lyase [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;Catalyzes the first step in the glyoxalate cycle, which converts lipids to carbohydrates;Members of the ICL/PEPM enzyme family catalyze either P-C or C-C bond formation/cleavage. Known members are phosphoenolpyruvate mutase (PEPM), phosphonopyruvate hydrolase (PPH), carboxyPEP mutase (CPEP mutase), oxaloacetate hydrolase (OAH), isocitrate...; cd00377;tetramer interface [polypeptide binding];Mg2+/Mn2+ binding site [ion binding]
GOHSU14	CDS	GOHSU_14_01120	complement(110239..110967)		hypothetical protein [migap]				hypothetical protein		F1YIS6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09401 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_01130	111052..111585		conserved hypothetical protein [nite]			RER_15470 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	carboxymuconolactone decarboxylase		F1YIS8_9ACTO Alkylhydroperoxidase like protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_09411 PE=3 SV=1 ;carboxymuconolactone decarboxylase [Frankia sp. EuI 1c];PFAM: carboxymuconolactone decarboxylase; KEGG: mav:MAV_1392 hypothetical protein;Carboxymuconolactone decarboxylase family; cl00460
GOHSU14	CDS	GOHSU_14_01140	111598..112245		menaquinone biosynthesis methyltransferase MenH [nite]	 menH [nite]		TEH_05900 :menaquinone biosynthesis methyltransferase MenH : menH : 2.1.1.- : Tetragenococcus halophilus NBRC 12172	methyltransferase		F1YIS9_9ACTO Type 11 methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09416 PE=4 SV=1;methyltransferase [Mycobacterium marinum M];thought to cause methylation;Methyltransferase domain; pfam13847;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU14	CDS	GOHSU_14_01150	112472..113284		hypothetical protein [nite]			MLP_32770 :hypothetical protein : : : Microlunatus phosphovorus NM-1	SAM-dependent methyltransferase		F9VU87_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_050_00300 PE=4 SV= 1;SAM-dependent methyltransferase [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;Methyltransferase domain; pfam13847;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU14	CDS	GOHSU_14_01160	113357..113602		cytoplasmic protein [migap]				cytoplasmic protein		F1YIT0_9ACTO Cupin 2 conserved barrel domain-contai ning protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_09421 PE=4 SV=1;cytoplasmic protein [Syntrophus aciditrophicus SB];Cupin domain; cl09118
GOHSU14	CDS	GOHSU_14_01170	113599..114258		hypothetical protein [nite]			ROP_00890 :hypothetical protein : : : Rhodococcus opacus B4	transcriptional regulator		F9VU85_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_050_00280 PE=4 SV= 1;transcriptional regulator [Mycobacterium sp. Spyr1] ;PFAM: IclR helix-turn-helix domain;Predicted transcriptional regulator [Transcription];Ribonuclease R winged-helix domain; pfam08461
GOHSU14	CDS	GOHSU_14_01180	114298..114771		hypothetical protein [migap]				hypothetical protein		F9VU84_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_050_00270 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.971 at residue 27
GOHSU14	CDS	GOHSU_14_01190	114791..115537		putative ABC transporter ATP-binding protein [nite]			RER_41750 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter ATP-binding protein		F9VU83_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_050_00 260 PE=3 SV=1;ABC transporter ATP-binding protein [Rhodococcus er ythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;ABC-type multidrug transport system, ATPase component [Defense mechanisms]; COG1131;The BcrA subfamily represents ABC transporters involved in peptide antibiotic resistance.  Bacitracin is a dodecapeptide antibiotic produced by B. licheniformis and B. subtilis.  The synthesis of bacitracin is non-ribosomally catalyzed by a multienzyme...; cd03268;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU14	CDS	GOHSU_14_01200	115534..116355		hypothetical protein [migap]				hypothetical protein		F9VU82_9ACTO Putative ABC transporter permease prot ein OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_050_00250 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU14	CDS	GOHSU_14_01210	116368..117000		hypothetical protein [migap]				hypothetical protein		unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 5 : No homology to any previously reported sequences
GOHSU14	CDS	GOHSU_14_01220	117024..118394		putative FAD-linked oxidase [nite]			RER_57260 :putative FAD-linked oxidase : : : Rhodococcus erythropolis PR4	FAD-linked oxidase		E9SX59_COREQ Oxidoreductase OS=Rhodococcus equi ATC C 33707 GN=HMPREF0724_10710 PE=4 SV=1;FAD-linked oxidase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;FAD/FMN-containing dehydrogenases [Energy production and conversion]; COG0277;FAD binding domain; pfam01565
GOHSU14	CDS	GOHSU_14_01230	118571..119689		putative peptidase C59 family protein [nite]			KSE_12560 :putative peptidase C59 family protein : : : Kitasatospora setae NBRC 14216	choloylglycine hydrolase		H5TZ25_9ACTO Putative hydrolase OS=Gordonia sputi N BRC 100414 GN=GOSPT_050_00150 PE=4 SV=1;choloylglycine hydrolase [Bacteroides sp. 1_1_6];Choloylglycine hydrolase (CGH) is a bile salt-modifying enzyme that hydrolyzes non-peptide carbon-nitrogen bonds in choloylglycine and choloyltaurine, both of which are present in bile.  CGH is present in a number of probiotic microbial organisms that...; cd01902;Linear amide C-N hydrolases, choloylglycine hydrolase family; pfam02275
GOHSU14	CDS	GOHSU_14_01240	119816..121063		hypothetical protein [nite]			KSE_68690 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		C5C299_BEUC1 Putative uncharacterized protein OS=Be utenbergia cavernae (strain ATCC BAA-8 / DSM 12333 / NBRC 16432) GN=Bcav_3482 PE=4 SV=1;unnamed protein product [Bacillus pseudofirmus OF4] ;COG2966 Uncharacterized conserved protein;Uncharacterized conserved protein [Function unknown];Protein of unknown function (DUF1212); cl01118
GOHSU14	CDS	GOHSU_14_01250	complement(121071..122372)		conserved hypothetical protein [nite]			RER_28450 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	diacylglycerol O-acyltransferase		H8IT53_MYCIA Putative uncharacterized protein OS=My cobacterium intracellulare (strain ATCC 13950 / DSM 43223 / JCM 6384 / NCTC 13025 / 3600) GN=OCU_31850 PE=4 SV=1;diacylglycerol O-acyltransferase [Mycobacterium sp. KMS];PFAM: protein of unknown function UPF0089; KEGG: mmc:Mmcs_0793 protein of unknown function UPF0089;acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007
GOHSU14	CDS	GOHSU_14_01260	complement(122564..123829)		dihydrolipoamide dehydrogenase [nite]	 lpd [nite]		RER_15390 :dihydrolipoamide dehydrogenase : lpd : 1.8.1.4 : Rhodococcus erythropolis PR4	dihydrolipoamide dehydrogenase	lpd	F1YJ26_9ACTO Dihydrolipoyl dehydrogenase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_09501 PE=3 SV=1;dihydrolipoamide dehydrogenase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;dihydrolipoamide dehydrogenase; Reviewed;E3 component of alpha keto acid dehydrogenase complexes LpdC; forms a homodimer; binds one molecule of FAD monomer; catalyzes NAD+-dependent oxidation of dihydrolipoyl cofactors that are covalently linked to the E2 component;Pyridine nucleotide-disulphide oxidoreductase; pfam00070;Pyridine nucleotide-disulphide oxidoreductase, dimerisation domain; pfam02852
GOHSU14	CDS	GOHSU_14_01270	124205..125638		putative proline/betaine transporter [nite]	 proP [nite]		ROP_67810 :putative proline/betaine transporter : proP : : Rhodococcus opacus B4	general substrate transporter		G7GY20_9ACTO Proline/betaine transporter OS=Gordoni a araii NBRC 100433 GN=proP PE=4 SV=1;general substrate transporter [Burkholderia cenocep acia MC0- 3];PFAM: General substrate transporter; major facilitator superfamily MFS_1; KEGG: bam:Bamb_6031 general substrate transporter;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;metabolite-proton symporter; TIGR00883;putative substrate translocation pore
GOHSU14	CDS	GOHSU_14_01280	125734..126378		putative TetR family transcriptional regulator [nite]			KSE_54850 :putative TetR family transcriptional regulator : : : Kitasatospora setae NBRC 14216	TetR family transcriptional regulator		F1YJ27_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_09506 PE=4 SV=1;TetR family transcriptional regulator [Streptomyces sp. SirexAA- E];Transcriptional regulator [Transcription]; COG1309;PFAM: regulatory protein TetR; KEGG: mva:Mvan_2880 TetR family transcriptional regulator;Helix-turn-helix domains; cl00088
GOHSU14	CDS	GOHSU_14_01290	126375..127628		putative oxidoreductase [nite]			ROP_27700 :putative oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase		G7GQ45_9ACTO Putative oxidoreductase OS=Gordonia am arae NBRC 15530 GN=GOAMR_43_00130 PE=4 SV=1;oxidoreductase [Rhodococcus opacus B4];Old yellow enzyme (OYE)-like FMN binding domain. OYE was the first flavin-dependent enzyme identified, however its true physiological role remains elusive to this day.  Each monomer of OYE contains FMN as a non-covalently bound cofactor, uses NADPH as a...; cd02803;FMN binding site [chemical binding];substrate binding site [chemical binding];putative catalytic residue [active]
GOHSU14	CDS	GOHSU_14_01300	127646..129724		putative S9A family peptidase [nite]			ROP_18590 :putative S9A family peptidase : : 3.4.-.- : Rhodococcus opacus B4	prolyl oligopeptidase		F1YJ29_9ACTO Prolyl oligopeptidase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_09526 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: peptidase S9 prolyl oligopeptidase active site domain protein; peptidase S9A prolyl oligopeptidase domain protein beta-propeller; KEGG: xac:XAC0628 prolyl oligopeptidase;Serine proteases of the peptidase family S9A [Amino acid transport and metabolism];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU14	CDS	GOHSU_14_01310	complement(129679..130635)		putative transmembrane protein [migap]				putative transmembrane protein		F1YNE7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17228 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_01320	complement(130638..131591)		putative polyphosphate kinase 2 [nite]			RER_15330 :putative polyphosphate kinase 2 : : 2.7.4.1 : Rhodococcus erythropolis PR4	polyphosphate kinase 2		F1YNE6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17223 PE=4 SV=1;polyphosphate kinase 2 [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Polyphosphate kinase 2 (PPK2); cl11971
GOHSU14	CDS	GOHSU_14_01330	complement(131661..132104)		hypothetical protein [nite]			ROP_67300 :hypothetical protein : : : Rhodococcus opacus B4	oligoketide cyclase/lipid transport protein		H5TR21_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_188_00030 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Polyketide cyclase / dehydrase and lipid transport;Ligand-binding SRPBCC domain of an uncharacterized subfamily of proteins; cd07819;putative hydrophobic ligand binding site [chemical binding]
GOHSU14	CDS	GOHSU_14_01340	complement(132130..132942)		putative protein-tyrosine-phosphatase [nite]			ROP_63410 :putative protein-tyrosine-phosphatase : : 3.1.3.48 : Rhodococcus opacus B4	protein-tyrosine-phosphatase		H6MZI1_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c09040 P E=4 SV=1;protein-tyrosine-phosphatase [Rhodococcus erythropo lis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;protein synonym:phosphotyrosine phosphatase;Protein tyrosine/serine phosphatase [Signal transduction mechanisms];Protein tyrosine phosphatases (PTP) catalyze the dephosphorylation of phosphotyrosine peptides; they regulate phosphotyrosine levels in signal transduction pathways. The depth of the active site cleft renders the enzyme specific for phosphorylated Tyr; cl00053;Tyrosine phosphatase family C-terminal region; pfam13348
GOHSU14	CDS	GOHSU_14_01350	complement(132939..133748)		putative protein-tyrosine-phosphatase [nite]			ROP_63410 :putative protein-tyrosine-phosphatase : : 3.1.3.48 : Rhodococcus opacus B4	protein-tyrosine-phosphatase		H5U8W6_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_010_00080 PE=4 SV=1;protein-tyrosine-phosphatase [Rhodococcus erythropo lis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;protein synonym:phosphotyrosine phosphatase;Protein tyrosine/serine phosphatase [Signal transduction mechanisms];Protein tyrosine phosphatases (PTP) catalyze the dephosphorylation of phosphotyrosine peptides; they regulate phosphotyrosine levels in signal transduction pathways. The depth of the active site cleft renders the enzyme specific for phosphorylated Tyr; cl00053;Tyrosine phosphatase family C-terminal region; pfam13348
GOHSU14	CDS	GOHSU_14_01360	complement(133849..135471)	case	60 kDa chaperonin [hamap]	groL [hamap]	MF_00600:60 kDa chaperonin:groL 	RER_15260 :60 kDa chaperonin : groEL : : Rhodococcus erythropolis PR4	60 kDa chaperonin	groL1	F1YND1_9ACTO 60 kDa chaperonin OS=Gordonia neofelif aecis NRRL B-59395 GN=groEL PE=3 SV=1;groL1 gene product [Gordonia polyisoprenivorans VH2 ];chaperonin GroEL; Reviewed; PRK12849;GroEL_like type I chaperonin. Chaperonins are involved in productive folding of proteins. They share a common general morphology, a double toroid of 2 stacked rings, each composed of 7-9 subunits. The symmetry of type I is seven-fold and they are found...; cd03344;ring oligomerisation interface [polypeptide binding];ATP/Mg binding site [chemical binding];stacking interactions;hinge regions
GOHSU14	CDS	GOHSU_14_01370	complement(135718..136608)		putative polyphosphate kinase [nite]			ACMV_03860 :putative polyphosphate kinase : : 2.7.4.1 : Acidiphilium multivorum AIU301	polyphosphate:nucleotide phosphotransferase		D3D565_9ACTO Putative uncharacterized protein OS=Fr ankia sp. EUN1f GN=FrEUN1fDRAFT_4937 PE=4 SV=1;polyphosphate:nucleotide phosphotransferase [Pseudo nocardia dioxanivorans CB1190];KEGG: mar:MAE_61320 hypothetical protein; TIGRFAM: Polyphosphate:nucleotide phosphotransferase, PPK2; PFAM: Polyphosphate kinase-2-related;Polyphosphate kinase 2 (PPK2); cl11971
GOHSU14	CDS	GOHSU_14_01380	complement(136797..138467)		fatty-acid--CoA ligase [nite]	 fadD [nite]		RER_00850 :fatty-acid--CoA ligase : fadD : 6.2.1.- : Rhodococcus erythropolis PR4	AMP-dependent synthetase and ligase		H6MYR3_GORPV Putative long-chain-fatty-acid--CoA li gase OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH 2) GN=GPOL_c08760 PE=4 SV=1;AMP-dependent synthetase and ligase [Thermaerobacte r marianensis DSM 12885];COGs: COG0318 acyl-CoA synthetase (AMP-forming)/AMP-acid ligase II; InterPro IPR020845: IPR000873; KEGG: tcu:Tcur_1238 AMP-dependent synthetase and ligase; PFAM: AMP-dependent synthetase and ligase; SPTR: AMP-dependent synthetase and ligase; PFAM: AMP-binding enzyme;long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU14	CDS	GOHSU_14_01390	complement(138464..139420)		conserved hypothetical protein [nite]			RER_00860 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H6MYR4_GORPV Putative cyclase OS=Gordonia polyisopr enivorans (strain DSM 44266 / VH2) GN=GPOL_c08770 PE=4 SV= 1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];Putative cyclase; cl00814
GOHSU14	CDS	GOHSU_14_01400	complement(139584..140423)		putative TetR family transcriptional regulator [nite]			RER_00870 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		F9VYI4_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_087_ 00710 PE=4 SV=1;TetR family transcriptional regulator [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Helix-turn-helix XRE-family like proteins. Prokaryotic DNA binding proteins belonging to the xenobiotic response element family of transcriptional regulators; cd00093;non-specific DNA binding site [nucleotide binding];salt bridge;sequence-specific DNA binding site [nucleotide binding];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU14	CDS	GOHSU_14_01410	140477..142141		long-chain fatty-acid--CoA ligase [nite]	 fadD [nite]		ACMV_12070 :long-chain fatty-acid--CoA ligase : fadD : 6.2.1.3 : Acidiphilium multivorum AIU301	long-chain-fatty-acid--CoA ligase	fadD3	H6MYR6_GORPV Putative long-chain-fatty-acid--CoA li gase OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH 2) GN=GPOL_c08790 PE=4 SV=1;long-chain-fatty-acid--CoA ligase [Carboxydothermus hydrogenoformans Z- 2901];identified by similarity to SP:P29212; match to protein family HMM PF00501;long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU14	CDS	GOHSU_14_01420	142227..143456		putative two-component histidine kinase [nite]			RER_45320 :putative two-component histidine kinase : : 2.7.13.- : Rhodococcus erythropolis PR4	signal transduction histidine kinase LytS		F1YNC7_9ACTO Histidine kinase internal region OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17128 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: dde:Dde_0160 ATP-binding region, ATPase-like;Putative regulator of cell autolysis [Signal transduction mechanisms]; COG3275;Histidine kinase; pfam06580;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU14	CDS	GOHSU_14_01430	143453..144253		putative LytT family two-component response regulator [nite]			RER_45310 :putative LytT family two-component response regulator : : : Rhodococcus erythropolis PR4	response regulator of the LytR/AlgR family		F1YNC6_9ACTO Response regulator receiver OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_17123 PE=4 SV=1;response regulator of the LytR/AlgR family [Sacchar omonospora marina XMU15];PFAM: Response regulator receiver domain; LytTr DNA-binding domain;Response regulator of the LytR/AlgR family [Transcription / Signal transduction mechanisms]; COG3279;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];LytTr DNA-binding domain; cl04498
GOHSU14	CDS	GOHSU_14_01440	144269..144652		conserved hypothetical protein [nite]			RER_45300 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GWE4_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_75_00790 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021]
GOHSU14	CDS	GOHSU_14_01450	144646..146379		putative SSS family transporter [nite]			ROP_51070 :putative SSS family transporter : : : Rhodococcus opacus B4	transporter		F1YNC4_9ACTO Na+/solute symporter OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_17113 PE=3 SV=1;transporter [Nocardia farcinica IFM 10152];Sodium:solute symporter family; cl00456
GOHSU14	CDS	GOHSU_14_01460	146436..147050		hypothetical protein [migap]				hypothetical protein		G7GJG2_9ACTO DivIVA family protein OS=Gordonia amar ae NBRC 15530 GN=GOAMR_04_00540 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];DivIVA domain; TIGR03544;DivIVA domain repeat protein; TIGR03543
GOHSU14	CDS	GOHSU_14_01470	147261..147668		hypothetical membrane protein [nite]			ROP_51080 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YNC3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17108 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];similar to AL132644-8|CAB59439.1| percent identity: 50 in 109 aa;Protein of unknown function, DUF485; cl01231
GOHSU14	CDS	GOHSU_14_01480	147665..149356		putative acetate permease [nite]	 actP [nite]		RER_45270 :putative acetate permease : actP : : Rhodococcus erythropolis PR4	SSS family solute/sodium (Na+) symporter		H5UJQ3_9ACTO Acetate permease OS=Gordonia terrae NB RC 100016 GN=actP PE=3 SV=1;SSS family solute/sodium (Na+) symporter [Saccharop olyspora erythraea NRRL 2338];Predicted symporter [General function prediction only]; COG4147;Sodium:solute symporter family; cl00456
GOHSU14	CDS	GOHSU_14_01490	149796..150110		transcriptional regulator [migap]				transcriptional regulator		G4HTN4_MYCRH Putative uncharacterized protein (Prec ursor) OS=Mycobacterium rhodesiae JS60 GN=MycrhDRAFT_0574 PE=4 SV=1;transcriptional regulator [Rhodococcus jostii RHA1] 
GOHSU14	CDS	GOHSU_14_01500	complement(150138..151511)		putative amidase [nite]			ROP_38890 :putative amidase : : 3.5.1.- : Rhodococcus opacus B4	amidase		D0L2N9_GORB4 Amidase OS=Gordonia bronchialis (strai n ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_ 0692 PE=4 SV=1;amidase [Desulfococcus oleovorans Hxd3];catalyzes the hydrolysis of a monocarboxylic acid amid to form a monocarboxylate and ammonia;amidase; Provisional;Amidase; cl11426
GOHSU14	CDS	GOHSU_14_01510	151603..152766		putative acyl-CoA dehydrogenase [nite]			ROP_15910 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		D0L2N7_GORB4 Acyl-CoA dehydrogenase domain protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / J CM 3198 / NCTC 10667) GN=Gbro_0690 PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;pimeloyl-CoA dehydrogenase, large subunit; TIGR03204;Acyl-CoA dehydrogenase; cl09933
GOHSU14	CDS	GOHSU_14_01520	152799..153917		putative acyl-CoA dehydrogenase [nite]			ROP_15900 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		F1YNB8_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 17083 PE=4 SV=1;unnamed protein product [Ralstonia eutropha JMP134] ;pimeloyl-CoA dehydrogenase, small subunit; TIGR03203;Acyl-CoA dehydrogenase; cd00567
GOHSU14	CDS	GOHSU_14_01530	153980..154435		conserved hypothetical membrane protein [nite]			RER_52210 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0RKX0_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_087_01120 PE =4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae
GOHSU14	CDS	GOHSU_14_01540	154538..154996		hypothetical protein [nite]			ROP_18720 :hypothetical protein : : : Rhodococcus opacus B4	polyketide cyclase/dehydrase		F1YNB7_9ACTO Polyketide cyclase/dehydrase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_17078 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: polyketide cyclase/dehydrase; KEGG: rop:ROP_18720 hypothetical protein;START/RHO_alpha_C/PITP/Bet_v1/CoxG/CalC (SRPBCC) ligand-binding domain superfamily; cd07812;hydrophobic ligand binding site
GOHSU14	CDS	GOHSU_14_01550	155004..155714		phosphatidylserine decarboxylase proenzyme [hamap]	psd [hamap]	MF_00664:phosphatidylserine decarboxylase proenzyme:psd 	RER_15170 :putative phosphatidylserine decarboxylase : psd : 4.1.1.65 : Rhodococcus erythropolis PR4	phosphatidylserine decarboxylase		F1YNB6_9ACTO Phosphatidylserine decarboxylase proen zyme OS=Gordonia neofelifaecis NRRL B-59395 GN=psd PE=3 SV =1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: bja:blr3796 phosphatidylserine decarboxylase; TIGRFAM: phosphatidylserine decarboxylase related protein; PFAM: phosphatidylserine decarboxylase-related;Phosphatidylserine decarboxylase; cl03656
GOHSU14	CDS	GOHSU_14_01560	155711..156703		probable phosphatidylserine synthase [nite]	 pssA [nite]		RER_15160 :probable phosphatidylserine synthase : pssA : 2.7.8.8 : Rhodococcus erythropolis PR4	CDP-diacylglycerol--serine O-phosphatidyltransferase		F9VTK5_9ACTO Phosphatidylserine synthase OS=Gordoni a alkanivorans NBRC 16433 GN=pssA PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: CDP-alcohol phosphatidyltransferase; KEGG: bbt:BBta_3506 phosphatidylserine synthase (CDP-diacylglycerol--serine O-phosphatidyltransferase);Phosphatidylserine synthase [Lipid metabolism]; COG1183;CDP-alcohol phosphatidyltransferase; cl00453
GOHSU14	CDS	GOHSU_14_01570	complement(156740..157102)		not assigned						
GOHSU14	CDS	GOHSU_14_01580	complement(157359..157763)		hypothetical protein [migap]				hypothetical protein		F1YJR9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10746 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_01590	157857..160763		putative Na(+)/H(+) antiporter subunit A/B [nite]	 mrpA/mrpB [nite]		RER_15030 :putative Na(+)/H(+) antiporter subunit A/B : mrpA/mrpB : : Rhodococcus erythropolis PR4	monovalent cation/H+ antiporter subunit A	mrpA	F1YJS0_9ACTO Putative monovalent cation/H+ antiport er subunit A OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_10751 PE=3 SV=1;monovalent cation/H+ antiporter subunit A [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;subunit A of antiporter complex involved in resistance to high concentrations of Na+, K+, Li+ and/or alkali;putative monovalent cation/H+ antiporter subunit A; Reviewed;NADH-Ubiquinone/plastoquinone (complex I), various chains; cl14187;Domain of unknown function (DUF4040); cl00676
GOHSU14	CDS	GOHSU_14_01600	160760..161689		monovalent cation/H+ antiporter subunit C [migap]	mrpC [migap]			monovalent cation/H+ antiporter subunit C	mrpC	F1YJS1_9ACTO Putative monovalent cation/H+ antiport er subunit C OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_10756 PE=4 SV=1;monovalent cation/H+ antiporter subunit C [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;NADH-ubiquinone/plastoquinone oxidoreductase chain 4L; cl00492;subunit C of antiporter complex involved in resistance to high concentrations of Na+, K+, Li+ and/or alkali
GOHSU14	CDS	GOHSU_14_01610	161686..163308		putative Na(+)/H(+) antiporter subunit D [nite]	 mrpD [nite]		ROP_18830 :putative Na(+)/H(+) antiporter subunit D : mrpD : : Rhodococcus opacus B4	monovalent cation/H+ antiporter subunit D		H6MYL2_GORPV Putative monovalent cation/H+ antiport er subunit D OS=Gordonia polyisoprenivorans (strain DSM 44 266 / VH2) GN=GPOL_c08230 PE=3 SV=1;monovalent cation/H+ antiporter subunit D [Coryneba cterium glutamicum ATCC 13032];subunit D of antiporter complex involved in resistance to high concentrations of Na+, K+, Li+ and/or alkali; contains an oxidoreductase domain; catalyzes the transfer of electrons from NADH to ubiquinone;putative monovalent cation/H+ antiporter subunit D; Reviewed;NADH-Ubiquinone/plastoquinone (complex I), various chains; cl14187
GOHSU14	CDS	GOHSU_14_01620	163305..164117		putative Na(+)/H(+) antiporter subunit E [nite]	 mrpE [nite]		KRH_03840 :putative Na(+)/H(+) antiporter subunit E : mrpE : : Kocuria rhizophila DC2201	cation antiporter		F1YJS3_9ACTO Putative monovalent cation/H+ antiport er subunit E OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_10766 PE=4 SV=1;cation antiporter [Micromonospora sp. L5];PFAM: cation antiporter; KEGG: cfl:Cfla_0211 multisubunit Na+/H+ antiporter MnhE subunit;Na+/H+ ion antiporter subunit; cl00807
GOHSU14	CDS	GOHSU_14_01630	164114..164374		putative Na(+)/H(+) antiporter subunit F [nite]	 mrpF [nite]		ROP_18850 :putative Na(+)/H(+) antiporter subunit F : mrpF : : Rhodococcus opacus B4	monovalent cation/H+ antiporter subunit F	mrpF	F1YJS4_9ACTO Putative monovalent cation/H+ antiport er subunit F OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_10771 PE=4 SV=1;monovalent cation/H+ antiporter subunit F [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;subunit F of antiporter complex involved in resistance to high concentrations of Na+, K+, Li+ and/or alkali;Multiple resistance and pH regulation protein F (MrpF / PhaF); cl09154
GOHSU14	CDS	GOHSU_14_01640	164371..164721		putative Na(+)/H(+) antiporter subunit G [nite]	 mrpG [nite]		ROP_18860 :putative Na(+)/H(+) antiporter subunit G : mrpG : : Rhodococcus opacus B4	monovalent cation/H+ antiporter subunit G	mrpG	F1YJS5_9ACTO Putative monovalent cation/H+ antiport er subunit G OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_10776 PE=4 SV=1;monovalent cation/H+ antiporter subunit G [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;Na+/H+ antiporter subunit; cl00583;subunit G of antiporter complex involved in resistance to high concentrations of Na+, K+, Li+ and/or alkali
GOHSU14	CDS	GOHSU_14_01650	complement(164757..165896)	case	carboxylate-amine ligase YbdK [hamap]	ybdK [hamap]	MF_01609:carboxylate-amine ligase YbdK:ybdK 	ROP_18880 :putative carboxylate-amine ligase : : 6.3.-.- : Rhodococcus opacus B4	carboxylate-amine ligase		F1YJS6_9ACTO Carboxylate-amine ligase SCNU_10781 OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10781 PE=3 SV =1;carboxylate-amine ligase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;Glutamate-cysteine ligase family 2(GCS2); cl00954;ATP-dependent carboxylate-amine ligase;carboxylate-amine ligase, YbdK family; TIGR02050
GOHSU14	CDS	GOHSU_14_01660	complement(165911..166513)		probable superoxide dismutase [nite]	 sodC [nite]		RER_14950 :probable superoxide dismutase : sodC : 1.15.1.1 : Rhodococcus erythropolis PR4	superoxide dismutase copper/zinc-binding protein		F1YJS7_9ACTO Superoxide dismutase [Cu-Zn] OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_10786 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.997) with cleavage site probability 0.861 at residue 29;PFAM: superoxide dismutase copper/zinc binding; KEGG: aav:Aave_2599 superoxide dismutase, copper/zinc binding;Cu/Zn superoxide dismutase [Inorganic ion transport and metabolism]; COG2032;Copper/zinc superoxide dismutase (SOD). superoxide dismutases catalyse the conversion of superoxide radicals to molecular oxygen. Three evolutionarily distinct families of SODs are known, of which the copper/zinc-binding family is one. Defects in the...; cd00305;E-class dimer interface [polypeptide binding];P-class dimer interface [polypeptide binding];Cu2+ binding site [ion binding]
GOHSU14	CDS	GOHSU_14_01670	complement(166577..167092)		conserved hypothetical protein [nite]			RER_14940 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5U263_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_080_00370 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];LytR cell envelope-related transcriptional attenuator; pfam13399
GOHSU14	CDS	GOHSU_14_01680	complement(167138..167455)		conserved hypothetical protein [nite]			RER_14930 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0QXP2_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_036_00320 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3263); pfam11662
GOHSU14	CDS	GOHSU_14_01690	167650..168240		peptide deformylase [hamap]	def [hamap]	MF_00163:peptide deformylase:def 	ROP_18920 :peptide deformylase : def : 3.5.1.88 : Rhodococcus opacus B4	peptide deformylase	def	F1YJT2_9ACTO Peptide deformylase OS=Gordonia neofel ifaecis NRRL B-59395 GN=def PE=3 SV=1;peptide deformylase [Rhodococcus jostii RHA1];cleaves off formyl group from N-terminal methionine residues of newly synthesized proteins; binds iron(2+);Polypeptide or peptide deformylase; a family of metalloenzymes that catalyzes the removal of the N-terminal formyl group in a growing polypeptide chain following translation initiation during protein synthesis in prokaryotes. These enzymes utilize Fe(II)...; cd00487;catalytic residues [active];metal binding site [ion binding]
GOHSU14	CDS	GOHSU_14_01700	168248..168934		putative acetyltransferase [migap]				putative acetyltransferase		F1YJT3_9ACTO GCN5-like N-acetyltransferase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_10816 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU14	CDS	GOHSU_14_01710	168921..169493		guanyl-specific ribonuclease Sa3 [migap]				guanyl-specific ribonuclease Sa3		G7GUL9_9ACTO Putative guanyl-specific ribonuclease OS=Gordonia amarae NBRC 15530 GN=GOAMR_63_01410 PE=4 SV=1;guanyl-specific ribonuclease Sa3 [Streptomyces sp. C];RNase_Sa. Ribonucleases first isolated from Streptomyces aureofaciens. In general, ribonucleases cleave phosphodiester bonds in RNA and are essential  for both non-specific RNA degradation and for numerous forms of RNA processing. RNAse Sa is a guanylate...; cd00607;barstar interaction site
GOHSU14	CDS	GOHSU_14_01720	169490..170029		hypothetical protein [nite]			RER_20090 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GXZ1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_012_00160 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: xca:xccb100_1898 hypothetical protein;Barstar is an intracellular inhibitor of barnase, an extracellular ribonuclease of Bacillus amyloliquefaciens. Barstar binds tightly to the barnase active site and sterically blocks it, thus inhibiting its potentially lethal RNase activity inside the...; cl01048;RNAase interaction site [polypeptide binding]
GOHSU14	CDS	GOHSU_14_01730	170034..170840		putative exodeoxyribonuclease III [nite]	 xthA [nite]		RER_14900 :putative exodeoxyribonuclease III : xthA : 3.1.11.2 : Rhodococcus erythropolis PR4	exodeoxyribonuclease III protein XthA		F1YJT6_9ACTO Exodeoxyribonuclease III Xth OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_10831 PE=4 SV=1;exodeoxyribonuclease III protein XthA [Mycobacteriu m tuberculosis 94_M4241A];Escherichia coli exonuclease III (ExoIII) and Neisseria meningitides NExo-like subfamily of the ExoIII family purinic/apyrimidinic (AP) endonucleases; cd09086;putative phosphate binding site [ion binding];putative catalytic site [active];metal binding site A [ion binding];DNA binding site [nucleotide binding];putative AP binding site [nucleotide binding];putative metal binding site B [ion binding]
GOHSU14	CDS	GOHSU_14_01740	170907..171893		putative ABC transporter substrate-binding protein [nite]			ROP_23260 :putative ABC transporter substrate-binding protein : : : Rhodococcus opacus B4	ABC transporter substrate-binding protein		F1YNM8_9ACTO ABC transporter substrate-binding prot ein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_17637 P E=4 SV=1;ABC transporter substrate-binding protein [Rhodococ cus opacus B4];Helical backbone metal receptor (TroA-like domain). These proteins have been shown to function in the ABC transport of ferric siderophores and metal ions such as Mn2+, Fe3+, Cu2+ and/or Zn2+.  Their ligand binding site is formed in the interface between...; cl00262;Periplasmic binding protein; pfam01497;intersubunit interface [polypeptide binding]
GOHSU14	CDS	GOHSU_14_01750	complement(171979..172452)		hypothetical protein [migap]				hypothetical protein		F1YNS1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17837 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
##GOHSU15	contig00052				45294
GOHSU15	CDS	GOHSU_15_00010	complement(<1..348)	codon	hypothetical protein [migap]				hypothetical protein		unnamed protein product [Candidatus Solibacter usit atus Ellin6076];PFAM: Planctomycete cytochrome C; protein of unknown function DUF1549; KEGG: rba:RB7229 hypothetical protein-signal peptide and transmembrane prediction;Laminin G domain; Laminin G-like domains are usually Ca++ mediated receptors that can have binding sites for steroids, beta1 integrins, heparin, sulfatides, fibulin-1, and alpha-dystroglycans. Proteins that contain LamG domains serve a variety of...; cl00102
GOHSU15	CDS	GOHSU_15_00020	523..864		hypothetical protein [nite]			ROP_18950 :hypothetical protein : : : Rhodococcus opacus B4	acetyltransferase		F1YJT7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10836 PE=4 SV=1;acetyltransferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;N-Acyltransferase superfamily: Various enzymes that characteristically catalyze the transfer of an acyl group to a substrate; cd04301;Coenzyme A binding pocket [chemical binding]
GOHSU15	CDS	GOHSU_15_00030	1016..2632		phosphomethylpyrimidine synthase [hamap]	thiC [hamap]	MF_00089:phosphomethylpyrimidine synthase:thiC 	ROP_18970 :thiamine biosynthesis protein ThiC : thiC : : Rhodococcus opacus B4	thiamine biosynthesis protein ThiC	thiC	H5U271_9ACTO Phosphomethylpyrimidine synthase OS=Go rdonia sputi NBRC 100414 GN=thiC PE=3 SV=1;thiamine biosynthesis protein ThiC [Bacillus lichen iformis ATCC 14580];required for the synthesis of the hydromethylpyrimidine moiety of thiamine;thiamine biosynthesis protein ThiC; Provisional;ThiC-associated domain; pfam13667;ThiC family; cl08031
GOHSU15	CDS	GOHSU_15_00040	2632..3462		phosphomethylpyrimidine kinase [nite]	 thiD [nite]		RER_14870 :phosphomethylpyrimidine kinase : thiD : 2.7.4.7 : Rhodococcus erythropolis PR4	phosphomethylpyrimidine kinase		F9VQ72_9ACTO Phosphomethylpyrimidine kinase OS=Gord onia alkanivorans NBRC 16433 GN=thiD PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: msm:MSMEG_0825 phosphomethylpyrimidine kinase; TIGRFAM: phosphomethylpyrimidine kinase; PFAM: phosphomethylpyrimidine kinase type-1;4-amino-5-hydroxymethyl-2-methyl-pyrimidine phosphate kinase (HMPP-kinase) catalyzes two consecutive phosphorylation steps in the thiamine phosphate biosynthesis pathway, leading to the synthesis of vitamin B1. The first step is the phosphorylation of...; cd01169;dimer interface [polypeptide binding];substrate binding site [chemical binding];ATP binding site [chemical binding]
GOHSU15	CDS	GOHSU_15_00050	complement(3816..4424)		not assigned						
GOHSU15	CDS	GOHSU_15_00060	complement(4402..5118)		hypothetical membrane protein [nite]			ROP_14580 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YJU3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10866 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU15	CDS	GOHSU_15_00070	complement(5204..6574)		glutamate decarboxylase [nite]			RER_35380 :glutamate decarboxylase : : 4.1.1.15 : Rhodococcus erythropolis PR4	glutamate decarboxylase		F1YJU4_9ACTO Glutamate decarboxylase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_10871 PE=3 SV=1;glutamate decarboxylase [Pseudonocardia dioxanivora ns CB1190];TIGRFAM: Glutamate decarboxylase; KEGG: nfa:nfa35400  glutamate decarboxylase; PFAM: Pyridoxal phosphate-dependent decarboxylase;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU15	CDS	GOHSU_15_00080	complement(6627..8507)		conserved hypothetical membrane protein [nite]			RER_11270 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	ABC transporter transmembrane protein		H5U394_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_091_00060 PE=4 SV=1;ABC transporter transmembrane protein [Rhodococcus equi 103S];YhgE/Pip N-terminal domain; TIGR03061;X-X-X-Leu-X-X-Gly heptad repeats; TIGR03057;Predicted membrane protein [Function unknown];ABC-2 type transporter; cl11417
GOHSU15	CDS	GOHSU_15_00090	complement(8510..9277)		hypothetical protein [nite]			RER_11280 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5U395_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_091_00070 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: bav:BAV1814 ABC transporter ATP-binding protein
GOHSU15	CDS	GOHSU_15_00100	complement(9435..10682)		hypothetical protein [nite]			ROP_22260 :hypothetical protein : : : Rhodococcus opacus B4	exported lipase		F9VVJ9_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_056_00610 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.960 at residue 25;KEGG: sml:Smlt3749  exported lipase;Dipeptidyl aminopeptidases/acylaminoacyl-peptidases [Amino acid transport and metabolism]; COG1506
GOHSU15	CDS	GOHSU_15_00110	10824..11366		not assigned						
GOHSU15	CDS	GOHSU_15_00120	complement(11405..12430)		F420-dependent glucose-6-phosphate dehydrogenase [nite]	 fdg [nite]		ROP_19180 :F420-dependent glucose-6-phosphate dehydrogenase : fdg : : Rhodococcus opacus B4	F420-dependent oxidoreductase		F1YJU7_9ACTO Glucose-6-phosphate dehydrogenase, F42 0-dependent OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _10886 PE=4 SV=1;F420-dependent oxidoreductase [Conexibacter woesei DSM 14684];TIGRFAM: F420-dependent oxidoreductase, G6PDH family; PFAM: Luciferase-like, subgroup; KEGG: sml:Smlt2638  dehydrogenase;glucose-6-phosphate dehydrogenase (coenzyme-F420); TIGR03554;N5,N10-methylenetetrahydromethanopterin reductase (Mer) catalyzes the reduction of N5,N10-methylenetetrahydromethanopterin with reduced coenzyme F420 to N5-methyltetrahydromethanopterin and oxidized coenzyme F420; cd01097
GOHSU15	CDS	GOHSU_15_00130	complement(12508..13740)		putative glutaryl-CoA dehydrogenase [nite]	 gcdH [nite]		ROP_20160 :putative glutaryl-CoA dehydrogenase : gcdH : 1.3.99.7 : Rhodococcus opacus B4	glutaryl-CoA dehydrogenase		G7GIN9_9ACTO Glutaryl-CoA dehydrogenase OS=Gordonia amarae NBRC 15530 GN=gcdH PE=3 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00441; match to protein family HMM PF02770; match to protein family HMM PF02771; match to protein family HMM PF08028;Acyl-CoA dehydrogenase; cl09933;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960
GOHSU15	CDS	GOHSU_15_00140	13810..17703		putative non-ribosomal peptide synthetase [nite]			ROP_42870 :putative non-ribosomal peptide synthetase : : : Rhodococcus opacus B4	non-ribosomal peptide synthetase	cps	F1YJU8_9ACTO Non-ribosomal peptide synthetase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10891 PE=4 SV=1;non-ribosomal peptide synthetase [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;D-alanine--poly(phosphoribitol) ligase, subunit 1; TIGR01734;AMP-binding enzyme; cl15778;Phosphopantetheine attachment site; cl09936;non-ribosomal peptide synthetase terminal domain of unknown function; TIGR02353;Left-handed parallel beta-Helix (LbetaH or LbH) domain: The alignment contains 5 turns, each containing three imperfect tandem repeats of a hexapeptide repeat motif (X-[STAV]-X-[LIV]-[GAED]-X). Proteins containing hexapeptide repeats are often enzymes...; cl00160;putative CoA binding site [chemical binding];putative trimer interface [polypeptide binding]
GOHSU15	CDS	GOHSU_15_00150	17783..19138		putative aminopeptidase [nite]			ROP_42860 :putative aminopeptidase : : 3.4.11.- : Rhodococcus opacus B4	metallopeptidase		F1YJV2_9ACTO Peptidase M1, membrane alanine aminope ptidase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_109 11 PE=4 SV=1;metallopeptidase [Streptomyces sviceus ATCC 29083];Peptidase M1 family Aminopeptidase N; cd09603;Zn binding site [ion binding]
GOHSU15	CDS	GOHSU_15_00160	complement(19187..19810)		dTDP-6-deoxy-D-xylo-4-hexulose 3,5-epimerase [nite]	 rmlC [nite]		RER_02480 :dTDP-6-deoxy-D-xylo-4-hexulose 3,5-epimerase : rmlC : 5.1.3.13 : Rhodococcus erythropolis PR4	dTDP-4-dehydrorhamnose 3,5-epimerase		D0LEK7_GORB4 dTDP-4-dehydrorhamnose 3,5-epimerase O S=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JC M 3198 / NCTC 10667) GN=Gbro_0597 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: dTDP-4-dehydrorhamnose 35-epimerase related; KEGG: bch:Bcen2424_6650 dTDP-4-dehydrorhamnose 3,5- epimerase;Cupin domain; cl09118
GOHSU15	CDS	GOHSU_15_00170	complement(19818..20693)		glucose-1-phosphate thymidylyltransferase [nite]	 rmlA [nite]		MLP_22460 :glucose-1-phosphate thymidylyltransferase : rmlA : 2.7.7.24 : Microlunatus phosphovorus NM-1	glucose-1-phosphate thymidylyltransferase		F1YJV6_9ACTO Glucose-1-phosphate thymidylyltransfer ase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10931 P E=3 SV=1;glucose-1-phosphate thymidylyltransferase [Mycobact erium sp. Spyr1];G1P_TT_short is the short form of glucose-1-phosphate thymidylyltransferase; cd02538;PFAM: nucleotidyl transferase; TIGRFAM: glucose-1-phosphate thymidylyltransferase, short form;glucose-1-phosphate thymidylyltransferase, short form; TIGR01207;substrate binding site;tetramer interface
GOHSU15	CDS	GOHSU_15_00180	20755..21756		dTDP-glucose 4,6-dehydratase [nite]	 rmlB [nite]		RER_02500 :dTDP-glucose 4,6-dehydratase : rmlB : 4.2.1.46 : Rhodococcus erythropolis PR4	dTDP-glucose 4,6-dehydratase		F1YJV7_9ACTO dTDP-glucose 4,6-dehydratase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_10936 PE=3 SV=1;dTDP-glucose 4,6-dehydratase [Mycobacterium gilvum PYR- GCK];dTDP-D-glucose 4,6-dehydratase, extended (e) SDRs; cd05246;TIGRFAM: dTDP-glucose 4,6-dehydratase; PFAM: NAD-dependent epimerase/dehydratase; 3-beta hydroxysteroid dehydrogenase/isomerase; polysaccharide biosynthesis protein CapD; dTDP-4-dehydrorhamnose reductase; Male sterility C-terminal domain; KEGG: mmc:Mmcs_1066 dTDP-glucose 4,6-dehydratase;dTDP-glucose 4,6-dehydratase; TIGR01181;NAD binding site [chemical binding];substrate binding site [chemical binding];homodimer interface [polypeptide binding]
GOHSU15	CDS	GOHSU_15_00190	21795..22811		UDP-glucose 4-epimerase [nite]	 galE [nite]		MLP_29640 :UDP-glucose 4-epimerase : galE : 5.1.3.2 : Microlunatus phosphovorus NM-1	UDP-glucose 4-epimerase		F1YJV8_9ACTO UDP-glucose 4-epimerase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_10941 PE=3 SV=1;UDP-glucose 4-epimerase [Vibrio sp. Ex25];UDP-glucose 4 epimerase, subgroup 1, extended (e) SDRs; cd05247;UDP-glucose 4-epimerase;NAD binding site [chemical binding];homodimer interface [polypeptide binding];substrate binding site [chemical binding]
GOHSU15	CDS	GOHSU_15_00200	complement(22825..24075)		putative MFS transporter [nite]			ROP_42850 :putative MFS transporter : : : Rhodococcus opacus B4	MFS transporter		F1YJW0_9ACTO Major facilitator superfamily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10951 PE=4 SV=1;MFS transporter [Rhodococcus opacus B4];The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174
GOHSU15	CDS	GOHSU_15_00210	complement(24072..24707)		putative GntR family transcriptional regulator [nite]			RER_05370 :putative GntR family transcriptional regulator : : : Rhodococcus erythropolis PR4	GntR family transcriptional regulator		H5UHQ4_9ACTO Putative GntR family transcriptional r egulator OS=Gordonia terrae NBRC 100016 GN=GOTRE_121_00580 PE=4 SV=1;GntR family transcriptional regulator [Rhodococcus jostii RHA1];Transcriptional regulators [Transcription]; COG2186;Winged helix-turn-helix (WHTH) DNA-binding domain of the GntR family of transcriptional regulators; cd07377;DNA-binding site [nucleotide binding];FCD domain; cl11656
GOHSU15	CDS	GOHSU_15_00220	24869..26062		malate dehydrogenase [nite]			RER_41320 :malate dehydrogenase : : 1.1.1.40 : Rhodococcus erythropolis PR4	malate dehydrogenase		G7GY75_9ACTO Malate dehydrogenase OS=Gordonia araii NBRC 100433 GN=GOARA_012_01010 PE=3 SV=1;malate dehydrogenase [Rhodococcus opacus B4];protein synonym:malic enzyme;Malic enzyme, N-terminal domain; pfam00390;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding pocket [chemical binding]
GOHSU15	tRNA	GOHSU_15_t0010	26138..26210		not assigned						identified by tRNAscan-SE; putative
GOHSU15	CDS	GOHSU_15_00230	26428..26946		not assigned						H0RKR7_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_087_00590 PE =4 SV=1
GOHSU15	CDS	GOHSU_15_00240	complement(27005..28405)		putative cytochrome P450 [nite]			BBR47_09990 :putative cytochrome P450 : : : Brevibacillus brevis NBRC 100599	cytochrome P450		G7GYX1_9ACTO Putative cytochrome P450 OS=Gordonia a raii NBRC 100433 GN=GOARA_021_00330 PE=3 SV=1;cytochrome P450 [Conexibacter woesei DSM 14684];PFAM: cytochrome P450; KEGG: scl:sce6424 cytochrome P450 CYP110J1;Cytochrome P450 [Secondary metabolites biosynthesis, transport, and catabolism]; cl12078
GOHSU15	CDS	GOHSU_15_00250	28374..28550		putative nitrite reductase, copper-dependent [migap]				putative nitrite reductase, copper-dependent		unnamed protein product [Propionibacterium acnes Ty peIA2 P.acn17];COG2132 Putative multicopper oxidases
GOHSU15	CDS	GOHSU_15_00260	28583..29605		conserved hypothetical protein [nite]			RER_29600 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	radical SAM domain-containing protein		H5UGJ6_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_072_00770 PE=4 SV=1;radical SAM domain-containing protein [Thermobispor a bispora DSM 43833];type strain of Thermobispora bispora;PFAM: Radical SAM domain protein; KEGG: aba:Acid345_0620 Fe-S protein, radical SAM family;DNA repair photolyase [DNA replication, recombination, and repair]; COG1533;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site
GOHSU15	CDS	GOHSU_15_00270	complement(29650..33501)		putative WD-40 repeat protein [nite]			RER_58350 :putative WD-40 repeat protein : : : Rhodococcus erythropolis PR4	XRE family transcriptional regulator		H6N2Q2_GORPV WD-40 repeat-containing protein OS=Gor donia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_ c36670 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG2319 FOG: WD40 repeat;Helix-turn-helix XRE-family like proteins. Prokaryotic DNA binding proteins belonging to the xenobiotic response element family of transcriptional regulators; cl15761;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;WD40 domain, found in a number of eukaryotic proteins that cover a wide variety of functions including adaptor/regulatory modules in signal transduction, pre-mRNA processing and cytoskeleton assembly; typically contains a GH dipeptide 11-24 residues from...; cd00200;structural tetrad;WD40 domain, found in a number of eukaryotic proteins that cover a wide variety of functions including adaptor/regulatory modules in signal transduction, pre-mRNA processing and cytoskeleton assembly; typically contains a GH dipeptide 11-24 residues from...; cl02567
GOHSU15	CDS	GOHSU_15_00280	33639..33965		putative LuxR family transcriptional regulator [nite]			ROP_pKNR-00920 :putative LuxR family transcriptional regulator : : : Rhodococcus opacus B4	LuxR family transcriptional regulator		D0L2K3_GORB4 Regulatory protein LuxR OS=Gordonia br onchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3722 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: LuxR family transcriptional regulator; SMART: LuxR family transcriptional regulator; KEGG: rlt:Rleg2_5436 LuxR family transcriptional regulator;C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding];dimerization interface [polypeptide binding]
GOHSU15	CDS	GOHSU_15_00290	34053..34211		hypothetical protein [migap]				hypothetical protein		F1YJX1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11006 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU15	CDS	GOHSU_15_00300	34377..35150		putative oxidoreductase [nite]			ACMV_16210 :putative oxidoreductase : : : Acidiphilium multivorum AIU301	short-chain dehydrogenase/reductase SDR		G7GV53_9ACTO Putative oxidoreductase OS=Gordonia am arae NBRC 15530 GN=GOAMR_68_00350 PE=3 SV=1;short-chain dehydrogenase/reductase SDR [Variovorax paradoxus S110];gluconate 5-dehydrogenase; Provisional;PFAM: short-chain dehydrogenase/reductase SDR; KEGG: cak:Caul_3063 short-chain dehydrogenase/reductase SDR;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU15	CDS	GOHSU_15_00310	complement(35165..35875)		putative TetR family transcriptional regulator [nite]			ROP_20590 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		H5UPL7_9MICO Putative TetR family transcriptional r egulator OS=Mobilicoccus pelagius NBRC 104925 GN=MOPEL_023 _00150 PE=4 SV=1;TetR family transcriptional regulator [Frankia sp. EuI1c];PFAM: regulatory protein TetR; KEGG: sna:Snas_2904 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU15	CDS	GOHSU_15_00320	complement(35981..37318)		DNA polymerase III delta' subunit [nite]	 holB [nite]		ROP_42640 :DNA polymerase III delta' subunit : holB : 2.7.7.7 : Rhodococcus opacus B4	DNA polymerase III subunit delta'	dnaX	F1YJX3_9ACTO DNA polymerase III subunit delta' OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_11016 PE=4 SV=1 ;DNA polymerase III subunit delta' [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the DNA-template-directed extension of the 3'-end of a DNA strand; the delta' subunit seems to interact with the gamma subunit to transfer the beta subunit on the DNA;DNA polymerase III subunit delta'; Validated;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger
GOHSU15	CDS	GOHSU_15_00330	37427..38944		putative adenylate cyclase [nite]			RER_10750 :putative adenylate cyclase : : 4.6.1.1 : Rhodococcus erythropolis PR4	adenylate cyclase		F1YJX4_9ACTO Adenylyl cyclase class-3/4/guanylyl cy clase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11021 PE=4 SV=1;adenylate cyclase [Amycolicicoccus subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;Histidine kinase, Adenylyl cyclase, Methyl-accepting protein, and Phosphatase (HAMP) domain. HAMP is a signaling domain which occurs in a wide variety of signaling proteins, many of which are bacterial. The HAMP domain consists of two alpha helices...; cd06225;dimerization interface [polypeptide binding];Adenylate cyclase, family 3 (some proteins contain HAMP domain) [Signal transduction mechanisms]; COG2114;cyclase homology domain; cd07302;nucleotidyl binding site;metal binding site [ion binding];dimer interface [polypeptide binding]
GOHSU15	CDS	GOHSU_15_00340	complement(38984..42034)		DNA topoisomerase I [nite]	 topA [nite]		RER_05130 :DNA topoisomerase I : topA : 5.99.1.2 : Rhodococcus erythropolis PR4	DNA topoisomerase I	topA	F1YJX5_9ACTO DNA topoisomerase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_11026 PE=3 SV=1;DNA topoisomerase I [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;catalyzes the ATP-dependent breakage of single-stranded DNA followed by passage and rejoining, maintains net negative superhelicity;DNA topoisomerase I subunit omega; Validated;TOPRIM_TopoIA_TopoI: The topoisomerase-primase (TORPIM) domain found in members of the type IA family of DNA topoisomerases (Topo IA) similar to Escherichia coli DNA topoisomerase I.   Type IA DNA topoisomerases remove (relax) negative supercoils in the...; cd03363;interdomain interaction site;putative metal-binding site [ion binding];nucleotide binding site [chemical binding];DNA Topoisomerase, subtype IA; DNA-binding, ATP-binding and catalytic domain of bacterial DNA topoisomerases I and III, and eukaryotic DNA topoisomerase III and eubacterial and archael reverse gyrases. Topoisomerases clevage single or double stranded DNA...; cl11986;domain I;phosphate binding site [ion binding];DNA Topoisomerase, subtype IA; DNA-binding, ATP-binding and catalytic domain of bacterial DNA topoisomerases I and III, and eukaryotic DNA topoisomerase III and eubacterial and archael reverse gyrases. Topoisomerases clevage single or double stranded DNA...; cd00186;domain II;domain III;DNA binding groove [nucleotide binding];catalytic site [active];domain IV;Topoisomerase C-terminal repeat; pfam13368
GOHSU15	CDS	GOHSU_15_00350	complement(42193..42765)		hypothetical protein [nite]			ROP_42620 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YJX6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11031 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU15	CDS	GOHSU_15_00360	42860..45184		putative ATP-dependent helicase [nite]			RER_05100 :putative ATP-dependent helicase : : 3.6.1.- : Rhodococcus erythropolis PR4	DEAD/DEAH box helicase		F1YJX8_9ACTO DEAD/DEAH box helicase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 11041 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: DEAD/DEAH box helicase; protein of unknown function DUF1998; helicase; SMART: DEAD/DEAH box helicase ; helicase; KEGG: dma:DMR_45970  helicase;helicase/secretion neighborhood putative DEAH-box helicase; TIGR03817;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];Domain of unknown function (DUF1998); pfam09369
##GOHSU16	contig00061				165103
GOHSU16	CDS	GOHSU_16_00010	complement(<1..709)	3x codon	probable glycerate kinase [nite]			RER_36060 :probable glycerate kinase : : 2.7.1.31 : Rhodococcus erythropolis PR4	glycerate kinase		H5TJ47_9ACTO Putative glycerate kinase OS=Gordonia otitidis NBRC 100426 GN=GOOTI_065_01100 PE=4 SV=1;glycerate kinase [Mycobacterium vanbaalenii PYR- 1] ;PFAM: glycerate kinase; KEGG: mbo:Mb2228c glycerate kinase;Glycerate kinase family; cl00841
GOHSU16	CDS	GOHSU_16_00020	816..1493		conserved hypothetical protein [nite]			RER_36070 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F9VPV7_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_002_01110 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: similar to trichohyalin;Protein of unknown function (DUF3043); pfam11241
GOHSU16	CDS	GOHSU_16_00030	1501..3270		nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase [hamap]	cobT [hamap]	MF_00230:nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase:cobT 		nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferase	cobT	G7GPS4_9ACTO Nicotinate-nucleotide--dimethylbenzimi dazole phosphoribosyltransferase OS=Gordonia amarae NBRC 1 5530 GN=cobT PE=3 SV=1;nicotinate-nucleotide--dimethylbenzimidazole phosph oribosyltransferase [Corynebacterium glutamicum ATCC 13032 ];IS fingerprint type 4-5;catalyzes the synthesis of alpha-ribazole-5'-phosphate from nicotinate mononucleotide and 5,6-dimethylbenzimidazole;Phosphoribosyltransferase; pfam02277;Nicotinate-nucleotide-dimethylbenzimidazole phosphoribosyltransferase (DMB-PRT), also called CobT; cd02439;putative dimer interface [polypeptide binding];active site pocket [active];putative cataytic base [active]
GOHSU16	CDS	GOHSU_16_00040	3245..4114		nicotinate-nucleotide--dimethylbenzimidazole phosphoribosyltransferasecobalamin synthase [hamap]	cobTcobS [hamap]	MF_00719:cobalamin synthase:cobS 	RER_36100 :cobalamin (5'-phosphate) synthase : cobV : 2.7.8.26 : Rhodococcus erythropolis PR4	cobalamin synthase	cobS	H5U6Z4_9ACTO Cobalamin synthase OS=Gordonia sputi N BRC 100414 GN=cobS PE=3 SV=1;cobalamin synthase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of adenosylcobalamin from Ado-cobinamide-GDP and alpha-ribazole;Cobalamin-5-phosphate synthase; cl00415
GOHSU16	CDS	GOHSU_16_00050	complement(4153..5253)		branched-chain amino acid aminotransferase [nite]	 ilvE [nite]		ROP_08740 :branched-chain amino acid aminotransferase : ilvE : 2.6.1.42 : Rhodococcus opacus B4	branched-chain amino acid aminotransferase	ilvE	F1YHG6_9ACTO Branched-chain-amino-acid aminotransfe rase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06170 PE=3 SV=1;branched-chain amino acid aminotransferase [Coryneb acterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the transamination of the branched-chain amino acids to their respective alpha-keto acids;Branched-chain amino acid aminotransferase; pfam12427;branched-chain amino acid aminotransferase; Provisional;BCAT_beta_family: Branched-chain aminotransferase catalyses the transamination of the branched-chain amino acids  leusine, isoleucine and valine to their respective alpha-keto acids, alpha-ketoisocaproate, alpha-keto-beta-methylvalerate and...; cd01557;homodimer interface [polypeptide binding];substrate-cofactor binding pocket;catalytic residue [active]
GOHSU16	CDS	GOHSU_16_00060	complement(5309..6418)	case	aminomethyltransferase [hamap]	gcvT [hamap]	MF_00259:aminomethyltransferase:gcvT 	RER_36140 :aminomethyltransferase : gcvT : 2.1.2.10 : Rhodococcus erythropolis PR4	glycine cleavage system aminomethyltransferase T	gcvT	H5U1J3_9ACTO Aminomethyltransferase OS=Gordonia spu ti NBRC 100414 GN=gcvT PE=3 SV=1;glycine cleavage system aminomethyltransferase T [R hodococcus opacus B4];catalyzes the transfer of a methylene carbon from the methylamine-loaded GcvH protein to tetrahydrofolate, causing the release of ammonia and the generation of reduced GcvH protein;glycine cleavage system aminomethyltransferase T; Reviewed; PRK00389;Glycine cleavage T-protein C-terminal barrel domain; pfam08669
GOHSU16	CDS	GOHSU_16_00070	6481..8058		probable cytosol aminopeptidase [hamap]	pepA [hamap]	MF_00181:probable cytosol aminopeptidase:pepA 	ROP_08760 :aminopeptidase A : pepA : 3.4.11.1 : Rhodococcus opacus B4	leucyl aminopeptidase	pepB	F1YHG4_9ACTO Probable cytosol aminopeptidase OS=Gor donia neofelifaecis NRRL B-59395 GN=pepA PE=3 SV=1;leucyl aminopeptidase [Corynebacterium glutamicum A TCC 13032];IS fingerprint type 4-5;catalyzes the removal of N-terminal amino acids preferably leucine from various peptides;Cytosol aminopeptidase family, N-terminal and catalytic domains.  Family M17 contains zinc- and manganese-dependent exopeptidases ( EC  3.4.11.1), including leucine aminopeptidase. They catalyze removal of amino acids from the N-terminus of a protein and...; cd00433;multifunctional aminopeptidase A; Provisional;interface (dimer of trimers) [polypeptide binding];Substrate-binding/catalytic site;Zn-binding sites [ion binding]
GOHSU16	CDS	GOHSU_16_00080	8198..9958		dihydrolipoamide acyltransferase [nite]			ROP_08780 :dihydrolipoamide acyltransferase : : 2.3.1.- : Rhodococcus opacus B4	dihydrolipoamide acetyltransferase	aceF	H5UK70_9ACTO Putative dihydrolipoamide acyltransfer ase OS=Gordonia terrae NBRC 100016 GN=GOTRE_145_00580 PE=3 SV=1;dihydrolipoamide acetyltransferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;pyruvate dehydrogenase dihydrolipoyltransacetylase; Validated; PRK11854;E2 component of pyruvate dehydrogenase multienzyme complex; in Escherichia coli AceF contains three N-terminal lipoyl domains;Lipoyl domain of the dihydrolipoyl acyltransferase component (E2) of 2-oxo acid dehydrogenases. 2-oxo acid dehydrogenase multienzyme complexes, like pyruvate dehydrogenase (PDH), 2-oxoglutarate dehydrogenase (OGDH) and branched-chain 2-oxo acid...; cd06849;E3 interaction surface;lipoyl attachment site [posttranslational modification];e3 binding domain; pfam02817;2-oxoacid dehydrogenases acyltransferase (catalytic domain); cl02008
GOHSU16	CDS	GOHSU_16_00090	10001..10882		hypothetical protein [nite]			ROP_08790 :hypothetical protein : : : Rhodococcus opacus B4	NAD-dependent nucleoside-diphosphate sugar epimerase		F1YHG2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06150 PE=4 SV=1;NAD-dependent nucleoside-diphosphate sugar epimeras e [Amycolatopsis mediterranei U32];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;TIGR01777 family protein;NAD(P) binding site [chemical binding]
GOHSU16	CDS	GOHSU_16_00100	10875..11582	case	octanoyltransferase [hamap]	lipB [hamap]	MF_00013:octanoyltransferase:lipB 	RER_36190 :putative lipoate-protein ligase B : lipB : 2.3.1.181 : Rhodococcus erythropolis PR4	lipoate-protein ligase B	lipB	F9VPU9_9ACTO Octanoyltransferase OS=Gordonia alkani vorans NBRC 16433 GN=lipB PE=3 SV=1;lipoate-protein ligase B [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;lipoyl/octanoyltransferase; catalyzes the transfer of the lipoyl/octanoyl moiety of lipoyl/octanoyl-ACP onto lipoate-dependent enzymes like pyruvate dehydrogenase and the glycine cleavage system H protein;Biotin/lipoate A/B protein ligase family; cl14057
GOHSU16	CDS	GOHSU_16_00110	11610..12638	case	lipoyl synthase [hamap]	lipA [hamap]	MF_00206:lipoyl synthase:lipA 	ROP_08810 :lipoyl synthase : lipA : 2.8.1.8 : Rhodococcus opacus B4	lipoyl synthase		G7GPT5_9ACTO Lipoyl synthase OS=Gordonia amarae NBR C 15530 GN=lipA PE=3 SV=1;lipoyl synthase [Streptomyces sp. AA4];lipoyl synthase; Provisional;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site
GOHSU16	CDS	GOHSU_16_00120	12666..13433		conserved hypothetical protein [nite]			RER_36210 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	membrane protein		F1YHF9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06135 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF4191); pfam13829
GOHSU16	CDS	GOHSU_16_00130	complement(13535..13921)		hypothetical protein [nite]			ROP_15510 :hypothetical protein : : : Rhodococcus opacus B4	patatin		H0QUZ7_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_009_00150 PE=4 SV=1;patatin [Frankia sp. EuI1c];PFAM: patatin; KEGG: fal:FRAAL1315 hypothetical protein
GOHSU16	CDS	GOHSU_16_00140	complement(13960..14376)		patatin [migap]				patatin		H0QUZ7_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_009_00150 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: patatin; KEGG: rer:RER_48540 hypothetical protein;Patatins and Phospholipases; cl11396
GOHSU16	CDS	GOHSU_16_00150	complement(14373..14996)		putative oxidoreductase [nite]			RER_48560 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		F1YHF3_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06105 PE=3 SV=1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU16	CDS	GOHSU_16_00160	complement(15390..16067)		conserved hypothetical protein [nite]			RER_36220 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	RDD domain-containing protein		F1YHF1_9ACTO RDD domain containing protein OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_06095 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: RDD domain containing protein;RDD family; cl00746
GOHSU16	CDS	GOHSU_16_00170	16326..17759		glutamine synthetase I [nite]	 glnA [nite]		ROP_08840 :glutamine synthetase I : glnA : 6.3.1.2 : Rhodococcus opacus B4	glutamine synthetase	glnA1	G7GZK3_9ACTO Glutamine synthetase OS=Gordonia araii NBRC 100433 GN=glnA PE=3 SV=1;glutamine synthetase [Mycobacterium sp. JDM601];glutamine synthetase, type I; TIGR00653;Glutamine synthetase, beta-Grasp domain; pfam03951;Glutamine synthetase, catalytic domain; pfam00120
GOHSU16	CDS	GOHSU_16_00180	17918..18835		hypothetical protein [migap]				hypothetical protein		F1YHE9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06085 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: similar to nucleoporin 62kDa
GOHSU16	CDS	GOHSU_16_00190	18832..19569		peptidyl-prolyl cis-trans isomerase domain-containing protein [migap]				peptidyl-prolyl cis-trans isomerase domain-containing protein		F1YHE6_9ACTO Peptidyl-prolyl cis-trans isomerase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06070 PE=4 SV =1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00254;FKBP-type peptidyl-prolyl cis-trans isomerase; cl11587
GOHSU16	CDS	GOHSU_16_00200	complement(19681..20058)		protein CrcB homolog [hamap]	crcB [hamap]	MF_00454:protein CrcB homolog:crcB 	ROP_00280 :CrcB protein homolog : : : Rhodococcus opacus B4	camphor resistance protein CrcB	crcB	C1AS26_RHOOB Protein CrcB homolog 1 OS=Rhodococcus opacus (strain B4) GN=crcB1 PE=3 SV=1;camphor resistance protein CrcB [Oligotropha carbox idovorans OM5];type strain of Oligotropha carboxidovorans;CrcB-like protein; cl09114;may be involved in chromosome condensation; overexpression in Escherichia coli protects against decondensation by camphor; overexpressing the protein results in an increase in supercoiling
GOHSU16	CDS	GOHSU_16_00210	complement(20055..20510)		protein CrcB homolog [hamap]	crcB [hamap]	MF_00454:protein CrcB homolog:crcB 	ROP_pROB01-01790 :CrcB protein homolog : : : Rhodococcus opacus B4	camphor resistance protein		H5TT52_9ACTO Protein CrcB homolog 1 OS=Gordonia oti tidis NBRC 100426 GN=crcB PE=3 SV=1;camphor resistance protein [Streptomyces ghanaensis ATCC 14672];CrcB-like protein; cl09114
GOHSU16	CDS	GOHSU_16_00220	complement(20507..23512)		glutamate-ammonia-ligase adenylyltransferase [hamap]	glnE [hamap]	MF_00802:glutamate-ammonia-ligase adenylyltransferase:glnE 	RER_36480 :glutamine-synthetase adenylyltransferase : glnE : 2.7.7.42 : Rhodococcus erythropolis PR4	bifunctional glutamine-synthetase adenylyltransferase/deadenyltransferase	glnE	F1YHE5_9ACTO Glutamate-ammonia-ligase adenylyltrans ferase OS=Gordonia neofelifaecis NRRL B-59395 GN=glnE PE=3 SV=1;bifunctional glutamine-synthetase adenylyltransfera se/deadenyltransferase [Corynebacterium glutamicum ATCC 13 032];IS fingerprint type 4-5;catalyzes the ATP-dependent addition of AMP to a subunit of glutamine synthetase; also catalyzes the reverse reaction - deadenylation; adenylation/deadenylation of glutamine synthetase subunits is important for the regulation of this enzyme;bifunctional glutamine-synthetase adenylyltransferase/deadenyltransferase; Provisional;Nucleotidyltransferase (NT) domain of Escherichia coli adenylyltransferase (GlnE), Escherichia coli uridylyl transferase (GlnD), and similar proteins; cd05401;metal binding triad;GlnD PII-uridylyltransferase; pfam08335
GOHSU16	CDS	GOHSU_16_00230	complement(23551..24888)		glutamine synthetase I [nite]	 glnA [nite]		RER_36490 :glutamine synthetase I : glnA : 6.3.1.2 : Rhodococcus erythropolis PR4	glutamine synthetase	glnA2	H5UK83_9ACTO Glutamine synthetase I OS=Gordonia ter rae NBRC 100016 GN=glnA PE=3 SV=1;glnA2 gene product [Gordonia polyisoprenivorans VH2 ];Glutamine synthetase [Amino acid transport and metabolism]; COG0174;Glutamine synthetase, beta-Grasp domain; pfam03951;Glutamine synthetase, catalytic domain; pfam00120
GOHSU16	CDS	GOHSU_16_00240	25064..25909		3-methyl-2-oxobutanoate hydroxymethyltransferase [hamap]	panB [hamap]	MF_00156:3-methyl-2-oxobutanoate hydroxymethyltransferase:panB 	RER_36510 :3-methyl-2-oxobutanoate hydroxymethyltransferase : panB : 2.1.2.11 : Rhodococcus erythropolis PR4	3-methyl-2-oxobutanoate hydroxymethyltransferase	panB	F1YHE3_9ACTO 3-methyl-2-oxobutanoate hydroxymethylt ransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=panB PE=3 SV=1;panB gene product [Gordonia polyisoprenivorans VH2] ;Ketopantoate hydroxymethyltransferase (KPHMT) is the first enzyme in the pantothenate biosynthesis pathway. Ketopantoate hydroxymethyltransferase (KPHMT) catalyzes the first committed step in the biosynthesis of pantothenate (vitamin B5), which is a...; cd06557;oligomerization interface [polypeptide binding];metal binding site [ion binding]
GOHSU16	CDS	GOHSU_16_00250	25994..26983		prolyl aminopeptidase [nite]	 pip [nite]		ROP_08970 :prolyl aminopeptidase : pip : 3.4.11.5 : Rhodococcus opacus B4	proline iminopeptidase		F1YHE2_9ACTO Proline iminopeptidase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_06050 PE=3 SV=1;proline iminopeptidase [Gluconacetobacter diazotrop hicus PAl 5];KEGG: gdi:GDI2355 putative proline iminopeptidase; TIGRFAM: proline iminopeptidase; PFAM: alpha/beta hydrolase fold;proline iminopeptidase, Neisseria-type subfamily; TIGR01249
GOHSU16	CDS	GOHSU_16_00260	26980..28482		conserved hypothetical protein [nite]			RER_47630 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	CHAD domain-containing protein		F1YHE1_9ACTO CHAD domain containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_06045 PE=4 SV=1;CHAD domain-containing protein [Nakamurella multipa rtita DSM 44233];PFAM: CHAD domain containing protein; adenylate cyclase; KEGG: ccs:CCNA_03637  adenylate cyclase family;CYTH-like (also known as triphosphate tunnel metalloenzyme (TTM)-like) Phosphatases; cd07374;putative active site;putative metal binding residues [ion binding];signature motif;putative triphosphate binding site [ion binding];CHAD domain; cl10506
GOHSU16	CDS	GOHSU_16_00270	28573..29553		putative ABC transporter substrate-binding protein [nite]			KRH_06710 :putative ABC transporter substrate-binding protein : : : Kocuria rhizophila DC2201	periplasmic binding protein		D5USJ2_TSUPD Periplasmic binding protein OS=Tsukamu rella paurometabola (strain ATCC 8368 / DSM 20162 / JCM 10 117 / NBRC 16120 / NCTC 13040) GN=Tpau_2663 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: periplasmic binding protein; KEGG: cjk:jk1887  iron ABC transporter, solute-binding protein;ABC-type Fe3+-hydroxamate transport system, periplasmic component [Inorganic ion transport and metabolism]; COG0614;Helical backbone metal receptor (TroA-like domain). These proteins have been shown to function in the ABC transport of ferric siderophores and metal ions such as Mn2+, Fe3+, Cu2+ and/or Zn2+.  Their ligand binding site is formed in the interface between...; cl00262;intersubunit interface [polypeptide binding]
GOHSU16	CDS	GOHSU_16_00280	29554..30522		putative ABC transporter permease protein [nite]			KRH_06720 :putative ABC transporter permease protein : : : Kocuria rhizophila DC2201	transporter permease		D5USJ3_TSUPD Transport system permease protein OS=T sukamurella paurometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_2664 PE=4 SV= 1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: transporter permease; KEGG: car:cauri_0742 iron ABC transporter, permease;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;ABC-ATPase subunit  interface;dimer interface [polypeptide binding];putative PBP binding regions
GOHSU16	CDS	GOHSU_16_00290	30509..31543		putative ABC transporter permease protein [nite]			KRH_06730 :putative ABC transporter permease protein : : : Kocuria rhizophila DC2201	transporter permease		D5USJ4_TSUPD Transport system permease protein OS=T sukamurella paurometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_2665 PE=4 SV= 1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: transporter permease; KEGG: kra:Krad_3704 transporter permease;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;ABC-ATPase subunit  interface;dimer interface [polypeptide binding];putative PBP binding regions
GOHSU16	CDS	GOHSU_16_00300	31540..32301		putative ABC transporter ATP-binding protein [nite]			KRH_06740 :putative ABC transporter ATP-binding protein : : : Kocuria rhizophila DC2201	ABC transporter		D5USJ5_TSUPD ABC transporter related protein OS=Tsu kamurella paurometabola (strain ATCC 8368 / DSM 20162 / JC M 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_2666 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: ypb:YPTS_4078 ABC transporter; PFAM: ABC transporter related; SMART: ATPase AAA;ABC-type enterochelin transport system, ATPase component [Inorganic ion transport and metabolism]; COG4604;ABC transporters, involved in the uptake of siderophores, heme, and vitamin B12, are widely conserved in bacteria and archaea.  Only very few species lack representatives of the siderophore family transporters. The E. coli BtuCD protein is an ABC...; cd03214;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU16	CDS	GOHSU_16_00310	complement(32824..33468)		ribonuclease H/acid phosphatase [nite]			RER_36580 :ribonuclease H/acid phosphatase : : 3.1.26.4/3.1.3.2 : Rhodococcus erythropolis PR4	bifunctional RNase H/acid phosphatase		F1YHD8_9ACTO Bifunctional RNase H/acid phosphatase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_06030 PE=4 SV=1;bifunctional RNase H/acid phosphatase [Corynebacter ium glutamicum ATCC 13032];IS fingerprint type 4-5;bifunctional RNase H/acid phosphatase; Provisional;Histidine phosphatase domain found in phosphoglycerate mutases and related proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cd07067;catalytic core [active]
GOHSU16	CDS	GOHSU_16_00320	complement(33465..34226)		conserved hypothetical protein [nite]			RER_36590 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YHD7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06025 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Zn-ribbon protein, possibly nucleic acid-binding [General function prediction only];Putative zinc ribbon domain; pfam02591
GOHSU16	CDS	GOHSU_16_00330	complement(34279..35400)		conserved hypothetical protein [nite]			RER_36600 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YHD6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06020 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF34; KEGG: pca:Pcar_2677 hypothetical protein;NIF3 (NGG1p interacting factor 3); cl15371
GOHSU16	CDS	GOHSU_16_00340	complement(35430..36515)		putative aminotransferase [nite]			RER_36610 :putative aminotransferase : : 2.6.1.- : Rhodococcus erythropolis PR4	class I/II aminotransferase		F1YHD5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06015 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: aminotransferase class I and II; KEGG: gbm:Gbem_3537 L-threonine-O-3-phosphate decarboxylase;hypothetical protein; Provisional;Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU16	CDS	GOHSU_16_00350	36538..37254		putative hydrolase [nite]			ROP_09100 :putative hydrolase : : : Rhodococcus opacus B4	HAD hydrolase		H5UK92_9ACTO Putative hydrolase OS=Gordonia terrae NBRC 100016 GN=GOTRE_145_00800 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted phosphatases [General function prediction only]; COG0546;Haloacid dehalogenase-like hydrolases. The haloacid dehalogenase-like (HAD) superfamily includes L-2-haloacid dehalogenase, epoxide hydrolase, phosphoserine phosphatase, phosphomannomutase, phosphoglycolate phosphatase, P-type ATPase, and many others; cd01427;motif II
GOHSU16	CDS	GOHSU_16_00360	37319..37738		protein-tyrosine-phosphatase [nite]	 ptpA [nite]		RER_36630 :protein-tyrosine-phosphatase : ptpA : 3.1.3.48 : Rhodococcus erythropolis PR4	protein-tyrosine-phosphatase		F1YHD3_9ACTO Protein-tyrosine phosphatase, low mole cular weight OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_06005 PE=4 SV=1;protein-tyrosine-phosphatase [Corynebacterium gluta micum ATCC 13032];Low molecular weight phosphatase family; cd00115;Low molecular weight phosphotyrosine protein phosphatase; pfam01451
GOHSU16	CDS	GOHSU_16_00370	37783..38751		hypothetical membrane protein [nite]			ROP_09120 :hypothetical membrane protein : : : Rhodococcus opacus B4	SURF1-like protein		F1YHD2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_06000 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];SURF1 superfamily. Surf1/Shy1 has been implicated in the posttranslational steps of the biogenesis of the mitochondrially-encoded Cox1 subunit of cytochrome c oxidase (complex IV). Cytochrome c oxidase (complex IV), the terminal electron-transferring...; cd06662
GOHSU16	CDS	GOHSU_16_00380	complement(38810..39481)		hypothetical protein [migap]				hypothetical protein		H0R020_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_054_00340 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU16	CDS	GOHSU_16_00390	complement(39642..40580)		putative adenosylcobinamide-phosphate synthase [nite]	 cobD [nite]		RER_36650 :putative adenosylcobinamide-phosphate synthase : cobD : 6.3.1.10 : Rhodococcus erythropolis PR4	cobalamin biosynthesis protein CobD		F1YHC9_9ACTO Cobalamin biosynthesis protein OS=Gord onia neofelifaecis NRRL B-59395 GN=cobD PE=3 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: CobD/Cbib protein; TIGRFAM: cobalamin biosynthesis protein CobD;CobD/Cbib protein; cl00561
GOHSU16	CDS	GOHSU_16_00400	40555..41397		conserved hypothetical protein [nite]			RER_36470 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YHC8_9ACTO Spermidine synthase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_05980 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];cluster 0773 with SCAB77671;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cl15754
GOHSU16	CDS	GOHSU_16_00410	complement(41958..42824)		triacylglycerol lipase [nite]	 lip [nite]		ROP_55260 :triacylglycerol lipase : lip : 3.1.1.3 : Rhodococcus opacus B4	lipase		H5UHP4_9ACTO Triacylglycerol lipase OS=Gordonia ter rae NBRC 100016 GN=lip PE=4 SV=1;lipase [Rhodococcus jostii RHA1];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU16	CDS	GOHSU_16_00420	complement(42869..44338)		succinate-semialdehyde dehydrogenase [nite]	 gabD [nite]		ROP_56610 :succinate-semialdehyde dehydrogenase : gabD : 1.2.1.16 : Rhodococcus opacus B4	succinate-semialdehyde dehydrogenase (NAD(P)+)	gabD5	E9SW28_COREQ Succinate-semialdehyde dehydrogenase O S=Rhodococcus equi ATCC 33707 GN=gabD PE=3 SV=1;succinate-semialdehyde dehydrogenase (NAD(P)+) [Rho dococcus jostii RHA1];Aldehyde dehydrogenase family; pfam00171;Mitochondrial succinate-semialdehyde dehydrogenase and ALDH family members 5A1 and 5F1-like; cd07103;tetramerization interface [polypeptide binding];NAD(P) binding site [chemical binding];catalytic residues [active]
GOHSU16	tRNA	GOHSU_16_t0010	44546..44620		not assigned						identified by tRNAscan-SE; putative
GOHSU16	CDS	GOHSU_16_00430	44840..45652		hypothetical protein [nite]			ROP_12900 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GL98_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_19_00600 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];uncharacterized Actinobacterial protein TIGR03083;Mycothiol maleylpyruvate isomerase N-terminal domain; cl00986
GOHSU16	CDS	GOHSU_16_00440	45742..46599		hypothetical protein [migap]				hypothetical protein		D0LB27_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_3114 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];SGNH_hydrolase, or GDSL_hydrolase, is a diverse family of lipases and esterases. The tertiary fold of the enzyme is substantially different from that of the alpha/beta hydrolase family and unique among all known hydrolases; its active site closely...; cl01053;catalytic triad [active];oxyanion hole [active]
GOHSU16	CDS	GOHSU_16_00450	46749..49220		acyltransferase [migap]				acyltransferase		D0LB28_GORB4 Acyltransferase 3 OS=Gordonia bronchia lis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667 ) GN=Gbro_3115 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: acyltransferase; KEGG: scl:sce8980 hypothetical protein;OpgC protein; cl00792;Acyltransferase family; pfam01757
GOHSU16	CDS	GOHSU_16_00460	complement(49263..49733)		peroxiredoxin AhpE [nite]	 ahpE [nite]		ROP_09170 :peroxiredoxin AhpE : ahpE : 1.11.1.15 : Rhodococcus opacus B4	alkyl hydroperoxide reductase/thiol specific antioxidant	ahpC	F1YHC5_9ACTO Alkyl hydroperoxide reductase/ Thiol s pecific antioxidant/ Mal allergen OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_05965 PE=4 SV=1;alkyl hydroperoxide reductase/thiol specific antiox idant [Amycolatopsis mediterranei U32];Peroxiredoxin (PRX) family, AhpE-like subfamily; composed of proteins similar to Mycobacterium tuberculosis AhpE. AhpE is described as a 1-cys PRX because of the absence of a resolving cysteine. The structure and sequence of AhpE, however, show greater...; cd03018;dimer interface [polypeptide binding];catalytic triad [active]
GOHSU16	CDS	GOHSU_16_00470	complement(49733..50146)		hypothetical protein [nite]			ROP_09180 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5TIZ6_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_065_00590 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF3052); pfam11253
GOHSU16	CDS	GOHSU_16_00480	complement(50297..50797)		hypothetical protein [migap]				hypothetical protein		F1YHC3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05955 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU16	CDS	GOHSU_16_00490	51049..53820		pyruvate dehydrogenase E1 component [nite]	 aceE [nite]		RER_36700 :pyruvate dehydrogenase E1 component : aceE : 1.2.4.1 : Rhodococcus erythropolis PR4	pyruvate dehydrogenase subunit E1	aceE	F1YHC2_9ACTO Pyruvate dehydrogenase E1 component OS =Gordonia neofelifaecis NRRL B-59395 GN=aceE PE=3 SV=1;pyruvate dehydrogenase subunit E1 [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;E1 component; part of pyruvate dehydrogenase; forms a complex with DlaT and LpdC;pyruvate dehydrogenase subunit E1; Reviewed; PRK09405;Thiamine pyrophosphate (TPP) family, E1 of E. coli PDC-like subfamily, TPP-binding module; composed of proteins similar to the E1 component of the Escherichia coli pyruvate dehydrogenase multienzyme complex (PDC). PDC catalyzes the oxidative...; cd02017;dimer interface [polypeptide binding];TPP-binding site [chemical binding]
GOHSU16	CDS	GOHSU_16_00500	53869..55137		putative CdaR family transcriptional regulator [nite]			ROP_09210 :putative CdaR family transcriptional regulator : : : Rhodococcus opacus B4	hypothetical protein		H6MTM8_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c28840 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Regulator of polyketide synthase expression [Signal transduction mechanisms / Secondary metabolites biosynthesis, transport, and catabolism];Helix-turn-helix domains; cl00088
GOHSU16	CDS	GOHSU_16_00510	55329..56255		malonyl-CoA--acyl carrier protein transacylase [nite]	 fabD [nite]		ROP_09220 :malonyl-CoA--acyl carrier protein transacylase : fabD : 2.3.1.39 : Rhodococcus opacus B4	acyl transferase		H0R2D9_9ACTO Malonyl-CoA--acyl carrier protein tran sacylase OS=Gordonia effusa NBRC 100432 GN=fabD PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Acyl transferase domain; cl08282;malonyl CoA-acyl carrier protein transacylase; TIGR00128;PFAM: acyl transferase; KEGG: dze:Dd1591_1622 malonyl CoA-acyl carrier protein transacylase
GOHSU16	CDS	GOHSU_16_00520	56394..56696		acyl carrier protein [hamap]	acpP [hamap]	MF_01217:acyl carrier protein:acpP 	ROP_09230 :acyl carrier protein : acpM : : Rhodococcus opacus B4	phosphopantetheine-binding protein		Q2V0N9_RHOER Acyl carrier protein OS=Rhodococcus er ythropolis GN=acpM PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: phosphopantetheine-binding; KEGG: nis:NIS_0314 acyl carrier protein ACP;Phosphopantetheine attachment site; cl09936
GOHSU16	CDS	GOHSU_16_00530	56696..57964		3-oxoacyl-[acyl-carrier-protein] synthase II [nite]	 fabF [nite]		ROP_09240 :3-oxoacyl-[acyl-carrier-protein] synthase II : fabF : 2.3.1.179 : Rhodococcus opacus B4	3-oxoacyl-(acyl carrier protein) synthase II		H5TIZ0_9ACTO 3-oxoacyl-[acyl-carrier-protein] synth ase II OS=Gordonia otitidis NBRC 100426 GN=fabF PE=3 SV=1;3-oxoacyl-(acyl carrier protein) synthase II [Mycob acterium vanbaalenii PYR- 1];3-oxoacyl-(acyl carrier protein) synthase II; Reviewed;Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.807) with cleavage site probability 0.806 at residue 27;FabF; beta-ketoacyl-ACP synthase II, KASII; catalyzes a condensation reaction in fatty acid biosynthesis: addition of an acyl acceptor of two carbons from malonyl-ACP; required for the elongation of short-chain unsaturated acyl-ACP;Beta-ketoacyl-acyl carrier protein (ACP) synthase (KAS), type I and II. KASs are responsible for the elongation steps in fatty acid biosynthesis. KASIII catalyses the initial condensation and KAS I and II catalyze further elongation steps by Claisen...; cd00834;dimer interface [polypeptide binding]
GOHSU16	CDS	GOHSU_16_00540	57999..59450		putative propionyl-CoA carboxylase beta chain [nite]			RER_36750 :putative propionyl-CoA carboxylase beta chain : : 6.4.1.3 : Rhodococcus erythropolis PR4	protionyl-CoA carboxylase		H0R2E2_9ACTO Propionyl-CoA carboxylase beta chain O S=Gordonia effusa NBRC 100432 GN=pccB PE=4 SV=1;unnamed protein product [Frankia symbiont of Datisc a glomerata];Acetyl-CoA carboxylase, carboxyltransferase component (subunits alpha and beta) [Lipid metabolism];KEGG: nml:Namu_2037 carboxyl transferase; PFAM: Carboxyl transferase;Acetyl co-enzyme A carboxylase carboxyltransferase alpha subunit; cl15772
GOHSU16	CDS	GOHSU_16_00550	60078..60596		hypothetical protein [nite]			ROP_09260 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YHB6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05920 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3145); pfam11343
GOHSU16	CDS	GOHSU_16_00560	complement(60581..61138)		conjugal transfer nickase/helicase TraI [migap]	traI [migap]			conjugal transfer nickase/helicase TraI	traI	G7GZG4_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_026_00190 PE=4 SV=1;conjugal transfer nickase/helicase TraI [Escherichi a coli];cleaves origin of transfer and unwinds; oriT relaxase;conjugal transfer nickase/helicase TraI; Provisional
GOHSU16	CDS	GOHSU_16_00570	complement(61331..62404)		hypothetical protein [nite]			MLP_25610 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YHB5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05915 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU16	CDS	GOHSU_16_00580	complement(62513..63337)		conserved hypothetical protein [nite]			RER_36770 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	penicillin-binding protein, beta-lactamase class C		H5U1J8_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_074_00030 PE=4 SV=1;penicillin-binding protein, beta-lactamase class C [Saccharomonospora viridis DSM 43017];type strain of Saccharomonospora viridis DSM 43017;PFAM: Beta-lactamase;Penicillin binding protein transpeptidase domain; cl01009
GOHSU16	CDS	GOHSU_16_00590	complement(63386..64276)		hypothetical protein [migap]				hypothetical protein		F1YHB3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05905 PE=4 SV=1;unnamed protein product [Mesorhizobium loti MAFF303 099]
GOHSU16	CDS	GOHSU_16_00600	complement(64339..65469)		zinc-containing alcohol dehydrogenase [nite]			ROP_09280 :zinc-containing alcohol dehydrogenase : : 1.1.1.- : Rhodococcus opacus B4	oxidoreductase, Rxyl_3153 family		F1YHB1_9ACTO Alcohol dehydrogenase GroES domain-con taining protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_05895 PE=3 SV=1;oxidoreductase, Rxyl_3153 family [Saccharomonospora marina XMU15];PFAM: Alcohol dehydrogenase GroES-like domain; Zinc-binding dehydrogenase; TIGRFAM: oxidoreductase, Rxyl_3153 family;oxidoreductase, Rxyl_3153 family; TIGR03989;Class III alcohol dehydrogenase; cd08279;NAD binding site [chemical binding];catalytic Zn binding site [ion binding];substrate binding site [chemical binding];structural Zn binding site [ion binding]
GOHSU16	CDS	GOHSU_16_00610	complement(65588..67183)		putative fatty-acid--CoA ligase [nite]			RER_06480 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus erythropolis PR4	acyl-CoA synthetase (AMP-forming)/AMP-acid ligase II		H6MTN7_GORPV Long-chain-fatty-acid--CoA ligase LcfB OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) G N=lcfB3 PE=4 SV=1;acyl-CoA synthetase (AMP-forming)/AMP-acid ligase I I [Saccharomonospora marina XMU15];PFAM: AMP-binding enzyme; TIGRFAM: O-succinylbenzoate-CoA ligase;long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU16	CDS	GOHSU_16_00620	complement(67275..68075)		putative enoyl-CoA hydratase [nite]			RER_06490 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase		F1YHA9_9ACTO Enoyl-CoA hydratase/isomerase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_05885 PE=4 SV=1;enoyl-CoA hydratase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Catalyzes the reversible hydration of unsaturated fatty acyl-CoA to beta-hydroxyacyl-CoA;enoyl-CoA hydratase; Provisional;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU16	CDS	GOHSU_16_00630	68247..69443		putative TetR family transcriptional regulator [nite]			RER_06510 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		F1YHA8_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_05880 PE=4 SV=1;TetR family transcriptional regulator [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU16	CDS	GOHSU_16_00640	complement(69495..70385)		hypothetical protein [migap]				hypothetical protein		F1YHA7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05875 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU16	CDS	GOHSU_16_00650	70494..71633		putative acyl-CoA dehydrogenase [nite]			ROP_61670 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	butyryl-CoA dehydrogenase		F1YHA6_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 05870 PE=3 SV=1;butyryl-CoA dehydrogenase [Alicycliphilus denitrifi cans K601];KEGG: bur:Bcep18194_C7492 acyl-CoA dehydrogenase-like; PFAM: Acyl-CoA oxidase/dehydrogenase, type 1; Acyl-CoA dehydrogenase, N-terminal; Acyl-CoA oxidase/dehydrogenase, central domain;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU16	CDS	GOHSU_16_00660	71623..72297		putative GntR family transcriptional regulator [nite]			RER_55100 :putative GntR family transcriptional regulator : : : Rhodococcus erythropolis PR4	putative GntR family transcriptional regulator		G7GZF9_9ACTO Putative GntR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_026_00130 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transcriptional regulators [Transcription]; COG1802;Winged helix-turn-helix (WHTH) DNA-binding domain of the GntR family of transcriptional regulators; cd07377;DNA-binding site [nucleotide binding];FCD domain; cl11656
GOHSU16	tRNA	GOHSU_16_t0020	complement(72383..72458)		not assigned						identified by tRNAscan-SE; putative
GOHSU16	CDS	GOHSU_16_00670	complement(72546..73448)		hypothetical protein [migap]				hypothetical protein		G0PSY3_STRGR Putative uncharacterized protein OS=St reptomyces griseus XylebKG-1 GN=SACT1_6582 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rsa:RSal33209_0057 hypothetical protein
GOHSU16	CDS	GOHSU_16_00680	complement(73634..74083)	overlap	conserved hypothetical protein [nite]			RER_48210 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	MOSC domain-containing protein		F1YLK7_9ACTO MOSC domain containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_13994 PE=4 SV=1;MOSC domain-containing protein [Catenulispora acidi phila DSM 44928];PFAM: MOSC domain containing protein; KEGG: sco:SCO4765 hypothetical protein;Uncharacterized protein conserved in bacteria [Function unknown];MOSC domain; pfam03473
GOHSU16	CDS	GOHSU_16_00690	73964..74173	overlap	not assigned						
GOHSU16	CDS	GOHSU_16_00700	74181..74774		putative TetR family transcriptional regulator [nite]			KRH_06930 :putative TetR family transcriptional regulator : : : Kocuria rhizophila DC2201	TetR family transcriptional regulator		G7H3P3_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_056_02160 PE=4 SV=1;TetR family transcriptional regulator [Kocuria rhiz ophila DC2201];mycofactocin system transcriptional regulator; TIGR03968;Helix-turn-helix domains; cl00088
GOHSU16	CDS	GOHSU_16_00710	74771..76474		drug resistance efflux protein [nite]			KRH_06920 :drug resistance efflux protein : : : Kocuria rhizophila DC2201	major facilitator superfamily permease		G7H3P4_9ACTO Drug resistance efflux protein OS=Gord onia araii NBRC 100433 GN=GOARA_056_02170 PE=4 SV=1;major facilitator superfamily permease [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;Major Facilitator Superfamily; pfam07690;putative substrate translocation pore
GOHSU16	CDS	GOHSU_16_00720	complement(76624..77241)		peptidase M23 family protein [nite]			ROP_pROB01-02420 :peptidase M23 family protein : : : Rhodococcus opacus B4	peptidase M23 family protein		F1YEC3_9ACTO Peptidase M23 OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_00220 PE=4 SV=1;peptidase M23 family protein [Rhodococcus opacus B4 ];Peptidase family M23; pfam01551
GOHSU16	tRNA	GOHSU_16_t0030	complement(77511..77587)		not assigned						identified by tRNAscan-SE; putative
GOHSU16	CDS	GOHSU_16_00730	complement(77672..80308)		putative oxidoreductase [nite]			MLP_11520 :putative oxidoreductase : : : Microlunatus phosphovorus NM-1	putative multicopper oxidase		H5WXY4_9PSEU Putative multicopper oxidase OS=Saccha romonospora marina XMU15 GN=SacmaDRAFT_4677 PE=4 SV=1;unnamed protein product [Corynebacterium diphtheria e HC02];Cytochrome C oxidase subunit II, periplasmic domain; cl11412;nitrite reductase, copper-containing; TIGR02376
GOHSU16	CDS	GOHSU_16_00740	complement(80305..81552)		hypothetical protein [nite]			MLP_11530 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		H8JF07_MYCIT Putative uncharacterized protein OS=My cobacterium intracellulare MOTT-64 GN=OCQ_18180 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];KEGG: pac:PPA0093 hypothetical membrane protein
GOHSU16	CDS	GOHSU_16_00750	complement(81549..82301)		hypothetical protein [nite]			MLP_11540 :hypothetical protein : : : Microlunatus phosphovorus NM-1	transcriptional regulator		H5WXY2_9PSEU Putative transcriptional regulator (Pr ecursor) OS=Saccharomonospora marina XMU15 GN=SacmaDRAFT_4 675 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];PFAM: LexA DNA-binding domain protein; KEGG: mpa:MAP2725c hypothetical protein;Helix-turn-helix domains; cl00088;Predicted transcriptional regulator [Transcription]
GOHSU16	CDS	GOHSU_16_00760	82433..84070		hypothetical protein [nite]			MLP_26290 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		C7R2G9_JONDD Putative uncharacterized protein OS=Jo nesia denitrificans (strain ATCC 14870 / DSM 20603 / CIP 5 5134) GN=Jden_0878 PE=4 SV=1;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;PFAM: protein of unknown function DUF1212; KEGG: lxx:Lxx19660 hypothetical protein;Protein of unknown function (DUF1212); cl01118
GOHSU16	CDS	GOHSU_16_00770	complement(84067..85983)		putative cation-transporting ATPase [nite]			TEH_05200 :putative cation-transporting ATPase : : 3.6.3.- : Tetragenococcus halophilus NBRC 12172	heavy metal translocating P-type ATPase		C5BVR5_BEUC1 Heavy metal translocating P-type ATPas e OS=Beutenbergia cavernae (strain ATCC BAA-8 / DSM 12333 / NBRC 16432) GN=Bcav_0240 PE=3 SV=1;heavy metal translocating P-type ATPase [Lactobacil lus buchneri NRRL B- 30929];TIGRFAM: ATPase, P-type, heavy metal translocating; ATPase, P-type, heavy metal-(Cd/Co/Hg/Pb/Zn)-translocating; ATPase, P-type, K/Mg/Cd/Cu/Zn/Na/Ca/Na/H-transporter; KEGG: lbr:LVIS_0392 cation transport ATPase; PFAM: ATPase, P-type, ATPase-associated domain; Haloacid dehalogenase-like hydrolase;heavy metal-(Cd/Co/Hg/Pb/Zn)-translocating P-type ATPase; TIGR01512;E1-E2 ATPase; pfam00122;Protein of unknown function (DUF3515); pfam12028;Haloacid Dehalogenase-like Hydrolases; cl11391
GOHSU16	CDS	GOHSU_16_00780	complement(86095..88092)		putative transporter [nite]			PH0361 :putative transporter : : : Pyrococcus horikoshii OT3	oligopeptide transporter OPT family protein		F1YE31_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02065 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF03169; match to protein family HMM TIGR00728; match to protein family HMM TIGR00733;oligopeptide transporters, OPT superfamily; TIGR00728;OPT oligopeptide transporter protein; cl14607
GOHSU16	CDS	GOHSU_16_00790	complement(88436..90352)		DNA primase [nite]	 dnaG [nite]		RER_36980 :DNA primase : dnaG : 2.7.7.- : Rhodococcus erythropolis PR4	DNA primase	dnaG	H0QWY1_9ACTO DNA primase OS=Gordonia effusa NBRC 10 0432 GN=dnaG PE=3 SV=1;DNA primase [Corynebacterium glutamicum ATCC 13032] ;IS fingerprint type 4-5;synthesizes RNA primers at the replication forks;DNA primase; Validated; PRK05667;CHC2 zinc finger; cl15369;DNA primase catalytic core, N-terminal domain; pfam08275;TOPRIM_DnaG_primases: The topoisomerase-primase (TORPIM) nucleotidyl transferase/hydrolase domain found in the active site regions of proteins similar to Escherichia coli DnaG. Primases synthesize RNA primers for the initiation of DNA replication. DnaG...; cd03364;metal binding site [ion binding];interdomain interaction site;DnaB-helicase binding domain of primase; pfam10410;DNA primase DnaG DnaB-binding; cl07879
GOHSU16	CDS	GOHSU_16_00800	complement(90576..90698)	short	not assigned						
GOHSU16	CDS	GOHSU_16_00810	90909..92066		cutinase;hypothetical protein [migap]				cutinase;hypothetical protein		G0HAX7_CORVD Putative secreted protein OS=Corynebac terium variabile (strain DSM 44702 / JCM 12073 / NCIMB 301 31) GN=CVAR_0444 PE=4 SV=1;cutinase [Mycobacterium vanbaalenii PYR- 1];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 1.000) with cleavage site probability 0.897 at residue 30;PFAM: cutinase; KEGG: mmc:Mmcs_0988 cutinase;Cutinase; pfam01083
GOHSU16	CDS	GOHSU_16_00820	92185..92661		hypothetical protein [nite]			YM304_23810 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	hypothetical protein		F1YE28_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02050 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: similar to conserved hypothetical protein;Domain of unknown function (DUF4262); pfam14081
GOHSU16	CDS	GOHSU_16_00830	92758..96105		putative helicase [nite]			ROP_13480 :putative helicase : : 3.6.1.- : Rhodococcus opacus B4	helicase		F1YDX8_9ACTO SNF2-like protein OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_01790 PE=4 SV=1;helicase [Microlunatus phosphovorus NM- 1];strain coidentity: NBRC 101784;Superfamily II DNA/RNA helicases, SNF2 family [Transcription / DNA replication, recombination, and repair]; COG0553;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding]
GOHSU16	CDS	GOHSU_16_00840	complement(96111..96365)		not assigned						A3VCK3_9RHOB Putative uncharacterized protein OS=Ma ritimibacter alkaliphilus HTCC2654 GN=RB2654_12389 PE=4 SV =1
GOHSU16	CDS	GOHSU_16_00850	96495..97325		putative epoxide hydrolase [nite]			RER_39420 :putative epoxide hydrolase : : 3.3.2.- : Rhodococcus erythropolis PR4	alpha/beta hydrolase fold protein		F1YE20_9ACTO Alpha/beta fold family hydrolase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02010 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;short chain dehydrogenase; Validated;PFAM: alpha/beta hydrolase fold; KEGG: bte:BTH_II1459 alpha/beta fold family hydrolase
GOHSU16	CDS	GOHSU_16_00860	complement(97367..98137)		hypothetical protein [migap]				hypothetical protein		D5P479_9MYCO Putative uncharacterized protein OS=My cobacterium parascrofulaceum ATCC BAA-614 GN=HMPREF0591_09 73 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 5 : No homology to any previously reported sequences
GOHSU16	CDS	GOHSU_16_00870	complement(98134..98637)		putative hydrolase [nite]			RER_56890 :putative hydrolase : : : Rhodococcus erythropolis PR4	Appr-1-p processing domain-containing protein		F1YE17_9ACTO Appr-1-p processing domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_019 95 PE=4 SV=1;Appr-1-p processing domain-containing protein [Micr omonospora aurantiaca ATCC 27029];KEGG: sro:Sros_0952 hypothetical protein; PFAM: Appr-1-p processing domain protein; SMART: Appr-1-p processing domain protein;Macro domain, Appr-1'-pase_like family. The macro domain is a high-affinity ADP-ribose binding module found in a variety of proteins as a stand-alone domain or in combination with other domains like in histone macroH2A and some PARPs (poly ADP-ribose...; cd02908;putative ADP-ribose binding site [chemical binding];putative active site [active]
GOHSU16	CDS	GOHSU_16_00880	98751..99446		hypothetical protein [nite]			RER_08830 :hypothetical protein : : : Rhodococcus erythropolis PR4			C0ZQB3_RHOE4 Putative uncharacterized protein OS=Rh odococcus erythropolis (strain PR4 / NBRC 100887) GN=RER_0 8830 PE=4 SV=1
GOHSU16	CDS	GOHSU_16_00890	99516..99896		hypothetical protein [nite]			ROP_27740 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7H3Y1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_060_00070 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Protein of unknown function (DUF3423); pfam11903
GOHSU16	CDS	GOHSU_16_00900	99893..100477		hypothetical protein [nite]			ROP_27750 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7H3Y2_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_060_00080 PE=4 SV=1;unnamed protein product [Candidatus Accumulibacter phosphatis clade IIA str. UW- 1];KEGG: gsu:GSU0887 hypothetical protein;Zeta toxin; pfam06414;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099
GOHSU16	CDS	GOHSU_16_00910	complement(100509..100829)		putative oxidoreductase [nite]			MLP_02960 :putative oxidoreductase : : : Microlunatus phosphovorus NM-1	antibiotic biosynthesis monooxygenase		G7GT45_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_59_00710 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Antibiotic biosynthesis monooxygenase; cl10022;identified by match to protein family HMM PF03992
GOHSU16	CDS	GOHSU_16_00920	complement(100866..101243)		acetyl-CoA carboxylase, biotin carboxylase [migap]				acetyl-CoA carboxylase, biotin carboxylase		D0LHN6_HALO1 Acetyl-CoA carboxylase, biotin carboxy lase OS=Haliangium ochraceum (strain DSM 14365 / JCM 11303 / SMP-2) GN=Hoch_0257 PE=4 SV=1;acetyl-CoA carboxylase, biotin carboxylase [Haliang ium ochraceum DSM 14365];acetyl-CoA carboxylase biotin carboxylase subunit; Validated;TIGRFAM: acetyl-CoA carboxylase, biotin carboxylase; PFAM: carbamoyl-phosphate synthase L chain ATP- binding; biotin carboxylase domain protein; ATP-dependent carboxylate-amine ligase domain protein ATP-grasp; RimK domain protein ATP-grasp; carbamoyl-phosphate synthetase large chain domain protein; KEGG: mxa:MXAN_5767 acetyl-CoA carboxylase, biotin carboxylase;Biotin carboxylase C-terminal domain; cl08365
GOHSU16	CDS	GOHSU_16_00930	101328..101846		hypothetical protein [nite]			ROP_14010 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5TJW3_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_082_00010 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU16	CDS	GOHSU_16_00940	complement(101861..102445)		phosphoesterase or phosphohydrolase [migap]				phosphoesterase or phosphohydrolase		A0R497_MYCS2 Serine/threonine specific protein phos phatase OS=Mycobacterium smegmatis (strain ATCC 700084 / m c(2)155) GN=MSMEG_5753 PE=4 SV=1;unnamed protein product [Methylobacterium extorquen s AM1];Evidence 3 : Function proposed based on presence of conserved amino acid motif, structural feature or limited homology; Product type pe :  enzyme;Aquifex aeolicus AQ1575 and related proteins, metallophosphatase domain; cd07390;putative active site [active];putative metal binding site [ion binding]
GOHSU16	CDS	GOHSU_16_00950	complement(102490..103080)		hypothetical protein [migap]				hypothetical protein		B4VYL5_9CYAN Putative uncharacterized protein OS=Mi crocoleus chthonoplastes PCC 7420 GN=MC7420_2644 PE=4 SV=1 ;unnamed protein product [Ruegeria pomeroyi DSS- 3];identified by similarity to PIR:AD1992
GOHSU16	CDS	GOHSU_16_00960	103304..104266		not assigned						H1QNK5_9ACTO Putative uncharacterized protein (Frag ment) OS=Streptomyces coelicoflavus ZG0656 GN=SMCF_6570 PE =4 SV=1
GOHSU16	CDS	GOHSU_16_00970	104307..105560		oxidoreductase [nite]			MLP_12440 :oxidoreductase : : : Microlunatus phosphovorus NM-1	L-lactate dehydrogenase	lldD1	G7GLY0_9ACTO Putative oxidoreductase OS=Gordonia am arae NBRC 15530 GN=GOAMR_20_01530 PE=4 SV=1;L-lactate dehydrogenase [Corynebacterium variabile DSM 44702];L-lactate dehydrogenase (FMN-dependent) and related alpha-hydroxy acid dehydrogenases;Isopentenyl diphosphate isomerase (BS_ypgA, MTH48 and related proteins) [Coenzyme transport and metabolism]; COG1304;Family of homologous FMN-dependent alpha-hydroxyacid oxidizing enzymes. This family occurs in both prokaryotes and eukaryotes. Members of this family include flavocytochrome b2 (FCB2), glycolate oxidase (GOX), lactate monooxygenase (LMO), mandelate...; cd02809;substrate binding site [chemical binding];FMN binding site [chemical binding];putative catalytic residues [active]
GOHSU16	CDS	GOHSU_16_00980	105686..106654		putative dioxygenase [nite]			MLP_33270 :putative dioxygenase : : : Microlunatus phosphovorus NM-1	intradiol ring-cleavage dioxygenase		H0JKM0_9NOCA Protocatechuate 3,4-dioxygenase subuni t beta OS=Rhodococcus pyridinivorans AK37 GN=AK37_00597 PE =4 SV=1;unnamed protein product [Frankia sp. EAN1pec];PFAM: intradiol ring-cleavage dioxygenase; KEGG: cmi:CMM_0747 putative dioxygenase;Intradiol dioxygenase supgroup. Intradiol dioxygenases catalyze the critical ring-cleavage step in the conversion of catecholate derivatives to citric acid cycle intermediates. They break the catechol C1-C2 bond and utilize Fe3+, as opposed to  the...; cd03457;putative active site [active]
GOHSU16	CDS	GOHSU_16_00990	complement(106730..107200)		appr-1-p processing domain-containing protein [migap]				appr-1-p processing domain-containing protein		H1K7W6_9MYCO Appr-1-p processing domain protein OS= Mycobacterium tusciae JS617 GN=MyctuDRAFT_5769 PE=4 SV=1;appr-1-p processing domain-containing protein [Myco bacterium gilvum PYR- GCK];PFAM: Appr-1-p processing domain protein; KEGG: nfa:nfa36990 hypothetical protein
GOHSU16	CDS	GOHSU_16_01000	107525..108106		hypothetical protein [migap]				hypothetical protein		F1YE32_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02070 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU16	CDS	GOHSU_16_01010	complement(108434..108592)		uracil-DNA glycosylase superfamily protein [migap]				uracil-DNA glycosylase superfamily protein		G9AP42_PANAN Uracil-DNA glycosylase superfamily pro tein OS=Pantoea ananatis LMG 5342 GN=PANA5342_3942 PE=4 SV =1;unnamed protein product [Pantoea ananatis LMG 5342] ;Uracil-DNA glycosylases (UDG) and related enzymes; cl00483;ligand binding site [chemical binding]
GOHSU16	CDS	GOHSU_16_01020	complement(108692..109945)		hypothetical protein [migap]				hypothetical protein		H6N489_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c01630 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU16	CDS	GOHSU_16_01030	complement(110178..112529)		ATPase AAA [migap]				ATPase AAA		E6TFT7_MYCSR Putative uncharacterized protein OS=My cobacterium sp. (strain Spyr1) GN=Mspyr1_45720 PE=4 SV=1;ATPase AAA [Mycobacterium gilvum PYR- GCK];SMART: AAA ATPase; KEGG: mmc:Mmcs_0861 ATPas;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;PIF1-like helicase; pfam05970;Family description; cl15862;DnaQ-like (or DEDD) 3'-5' exonuclease domain superfamily; cl10012;catalytic site [active];DNA ligase, NAD-dependent; TIGR00575
GOHSU16	CDS	GOHSU_16_01040	complement(112813..114102)		deoxyguanosinetriphosphate triphosphohydrolase-like protein [hamap]		MF_01212:deoxyguanosinetriphosphate triphosphohydrolase-like protein: 	RER_37000 :putative deoxyguanosinetriphosphate triphosphohydrolase : dgt : 3.1.5.1 : Rhodococcus erythropolis PR4	deoxyguanosinetriphosphate triphosphohydrolase-like protein	dgt	F1YDX5_9ACTO Deoxyguanosinetriphosphate triphosphoh ydrolase-like protein OS=Gordonia neofelifaecis NRRL B-593 95 GN=SCNU_01775 PE=3 SV=1;deoxyguanosinetriphosphate triphosphohydrolase-like protein [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;deoxyguanosinetriphosphate triphosphohydrolase-like protein; Provisional;dGTPase type 2 subfamily; presumably hydrolyzes dGTP to deoxyguanosine and triphosphate;Metal dependent phosphohydrolases with conserved 'HD' motif; cd00077;Zn2+ binding site [ion binding];Mg2+ binding site [ion binding];Phosphohydrolase-associated domain; pfam13286
GOHSU16	CDS	GOHSU_16_01050	complement(114202..114747)		cold shock protein CspA [migap]	csp [migap]			cold shock protein CspA	csp	F1YDX4_9ACTO Cold-shock protein DNA-binding protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01770 PE=3 SV=1;csp gene product [Halomonas elongata DSM 2581];Cold shock-like protein cspA - Stigmatella aurantiaca,InterPro: Cold-shock DNA-binding domain, High confidence in function and specificity;Cold-Shock Protein (CSP) contains an S1-like cold-shock domain (CSD) that is found in eukaryotes, prokaryotes, and archaea.  CSP's include the major cold-shock proteins CspA and CspB in bacteria and the eukaryotic gene regulatory factor Y-box protein; cd04458;DNA-binding site [nucleotide binding];RNA-binding motif
GOHSU16	CDS	GOHSU_16_01060	complement(114872..116212)		methylated-DNA-(protein)-cysteine S-methyltransferase DNA binding protein [migap]				methylated-DNA-(protein)-cysteine S-methyltransferase DNA binding protein		E3IWJ5_FRASU Methylated-DNA-(Protein)-cysteine S-me thyltransferase DNA binding protein OS=Frankia sp. (strain EuI1c) GN=FraEuI1c_2136 PE=4 SV=1;methylated-DNA-(protein)-cysteine S-methyltransfera se DNA binding protein [Frankia sp. EuI1c];PFAM: methylated-DNA-[protein]-cysteine S-methyltransferase DNA binding; KEGG: esa:ESA_00616 hypothetical protein;Protein of unknown function DUF91; cl00709;The DNA repair protein O6-alkylguanine-DNA alkyltransferase (ATase; also known as AGT, AGAT and MGMT) reverses O6-alkylation DNA damage by transferring O6-alkyl adducts to an active site cysteine irreversibly, without inducing DNA strand breaks. ATases...; cl01497
GOHSU16	CDS	GOHSU_16_01070	116314..117948		von Willebrand factor A [migap]				von Willebrand factor A		H5U448_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_103_00830 PE=4 SV=1;von Willebrand factor A [Mycobacterium sp. JDM601];Von Willebrand factor type A (vWA) domain was originally found in the blood coagulation protein von Willebrand factor (vWF). Typically, the vWA domain is made up of approximately 200 amino acid residues folded into a classic a/b para-rossmann type of...; cl00057;metal ion-dependent adhesion site (MIDAS)
GOHSU16	CDS	GOHSU_16_01080	118211..118828		conserved hypothetical protein [nite]			RER_51820 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	pyridoxamine 5'-phosphate oxidase-like FMN-binding protein		F1YDW8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01740 PE=4 SV=1;pyridoxamine 5'-phosphate oxidase-like FMN-binding protein [Terriglobus saanensis SP1PR4];PFAM: pyridoxamine 5'-phosphate oxidase-related FMN-binding; KEGG: sus:Acid_3477 pyridoxamine 5'-phosphate oxidase-related, FMN-binding;Pyridoxine 5'-phosphate (PNP) oxidase-like proteins; cl00381
GOHSU16	CDS	GOHSU_16_01090	118985..119218		alpha-mannosidase [migap]				alpha-mannosidase		E1VRF8_ARTAR Putative uncharacterized protein OS=Ar throbacter arilaitensis (strain DSM 16368 / CIP 108037 / J CM 13566 / Re117) GN=AARI_pI00160 PE=4 SV=1;alpha-mannosidase [Rothia mucilaginosa DY- 18]
GOHSU16	CDS	GOHSU_16_01100	119299..119916		hypothetical protein [nite]			RER_29560 :hypothetical protein : : : Rhodococcus erythropolis PR4	Fic family protein	fic	G9PDD9_9ACTO Putative uncharacterized protein OS=Ac tinomyces graevenitzii C83 GN=HMPREF0045_00022 PE=4 SV=1;Fic family protein [Arthrobacter arilaitensis Re117 ];identified by match to PF02661: Fic protein family. cAMP may be a regulation factor in cell division of some bacteria. The Fic (filamentation induced by cAMP) protein is involved in the synthesis of PAB or folate. It would appear that the Fic protein and cAMP are involved in a regulatory mechanism of cell division via folate metabolism and in these organisms cell division could be controlled by coordination of cAMP, Fic and Fts proteins;Fic/DOC family; cl00960
GOHSU16	CDS	GOHSU_16_01110	119992..121740		pyruvate dehydrogenase [nite]	 poxB [nite]		KRH_17600 :pyruvate dehydrogenase : poxB : 1.2.2.2 : Kocuria rhizophila DC2201	pyruvate dehydrogenase		F1YKC1_9ACTO Thiamine pyrophosphate protein TPP-bin ding domain-containing protein OS=Gordonia neofelifaecis N RRL B-59395 GN=SCNU_11380 PE=3 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];pyruvate dehydrogenase; Provisional;catalyzes the formation of acetate from pyruvate;Pyrimidine (PYR) binding domain of POX; cd07039;PYR/PP interface [polypeptide binding];tetramer interface [polypeptide binding];dimer interface [polypeptide binding];TPP binding site [chemical binding];Thiamine pyrophosphate enzyme, central domain; cl00815;Thiamine pyrophosphate (TPP) family, Pyruvate oxidase (POX) subfamily, TPP-binding module; composed of proteins similar to Lactobacillus plantarum POX, which plays a key role in controlling acetate production under aerobic conditions. POX decarboxylates...; cd02014;TPP-binding site [chemical binding]
GOHSU16	CDS	GOHSU_16_01120	121828..122373		hypothetical protein [nite]			RER_51640 :hypothetical protein : : : Rhodococcus erythropolis PR4	OsmC family protein		F1YM77_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15106 PE=4 SV=1;OsmC family protein [Pseudonocardia dioxanivorans C B1190];PFAM: Peroxiredoxin, OsmC-like protein; KEGG: nfa:nfa53120 hypothetical protein;OsmC-like protein; cl00767
GOHSU16	CDS	GOHSU_16_01130	122538..123941		hypothetical protein [nite]			RER_48710 :hypothetical protein : : : Rhodococcus erythropolis PR4	putative HNH endonuclease		F1YN44_9ACTO HNH endonuclease OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_16758 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF222); pfam02720;HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and...; cd00085
GOHSU16	CDS	GOHSU_16_01140	complement(124228..124383)		two-component system sensor histidine kinase [migap]				two-component system sensor histidine kinase		F1YDW6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01730 PE=4 SV=1;two-component system sensor histidine kinase [Bacte roides vulgatus ATCC 8482]
GOHSU16	CDS	GOHSU_16_01150	complement(124469..125860)		glycyl-tRNA synthetase [nite]	 glyQS [nite]		ROP_09410 :glycyl-tRNA synthetase : glyQS : 6.1.1.14 : Rhodococcus opacus B4	glycyl-tRNA synthetase	glyS	F1YDW4_9ACTO Glycine--tRNA ligase OS=Gordonia neofe lifaecis NRRL B-59395 GN=glyQS PE=3 SV=1;glycyl-tRNA synthetase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Catalyzes a two-step reaction, first charging a glycine molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA;glycyl-tRNA synthetase; Provisional;Class II tRNA amino-acyl synthetase-like catalytic core domain. Class II amino acyl-tRNA synthetases (aaRS) share a common fold and generally attach an amino acid to the 3' OH of ribose of the appropriate tRNA.   PheRS is an exception in that it attaches...; cl00268;motif 1;Glycyl-tRNA synthetase (GlyRS)-like class II core catalytic domain. GlyRS functions as a homodimer in eukaryotes, archaea and some bacteria and as a heterotetramer in the remainder of prokaryotes. It is responsible for the attachment of glycine to the 3'...; cd00774;motif 2;motif 3;GlyRS Glycyl-anticodon binding domain. GlyRS belongs to class II aminoacyl-tRNA synthetases (aaRS). This alignment contains the anticodon binding domain, which is responsible for specificity in tRNA-binding, so that the activated amino acid is...; cd00858;anticodon binding site
GOHSU16	CDS	GOHSU_16_01160	126069..126401		putative ArsR family transcriptional regulator [nite]			ROP_09420 :putative ArsR family transcriptional regulator : : : Rhodococcus opacus B4	transcriptional regulator		F1YDW1_9ACTO Putative transcriptional regulator OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01705 PE=4 SV= 1;transcriptional regulator [Nocardia farcinica IFM 1 0152];Arsenical Resistance Operon Repressor and similar prokaryotic, metal regulated homodimeric repressors. ARSR subfamily of helix-turn-helix bacterial transcription regulatory proteins (winged helix topology). Includes several proteins that appear to...; cd00090;dimerization interface [polypeptide binding];putative DNA binding site [nucleotide binding];putative Zn2+ binding site [ion binding]
GOHSU16	CDS	GOHSU_16_01170	126398..126802		putative Fur family transcriptional regulator [nite]			RER_37100 :putative Fur family transcriptional regulator : : : Rhodococcus erythropolis PR4	Fur family transcriptional regulator		F1YDW0_9ACTO Ferric-uptake regulator OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_01700 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Ferric uptake regulator(Fur) and related metalloregulatory proteins; typically iron-dependent, DNA-binding repressors and activators; cd07153;metal binding site 2 [ion binding];putative DNA binding helix;metal binding site 1 [ion binding];dimer interface [polypeptide binding];structural Zn2+ binding site [ion binding]
GOHSU16	CDS	GOHSU_16_01180	126911..129034		putative inner membrane protein translocase component YidC [migap]				putative inner membrane protein translocase component YidC		H5UEF8_9ACTO OxaA family protein OS=Gordonia terrae NBRC 100016 GN=GOTRE_060_00230 PE=3 SV=1;OxaA family protein [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;60Kd inner membrane protein; cl00489;functions to insert inner membrane proteins into the IM in Escherichia coli; interacts with transmembrane segments; functions in both Sec-dependent and -independent membrane insertion; similar to Oxa1p in mitochondria
GOHSU16	CDS	GOHSU_16_01190	complement(129072..129470)		metal ABC transporter periplasmic protein [migap]				metal ABC transporter periplasmic protein		C2CQI5_CORST Putative uncharacterized protein OS=Co rynebacterium striatum ATCC 6940 GN=HMPREF0308_1664 PE=4 S V=1;metal ABC transporter periplasmic protein [Microbac terium testaceum StLB037]
GOHSU16	CDS	GOHSU_16_01200	complement(129529..130332)		undecaprenyl pyrophosphate synthase [hamap]	uppS [hamap]	MF_01139:undecaprenyl pyrophosphate synthase:uppS 	ROP_09450 :undecaprenyl pyrophosphate synthase : uppS : 2.5.1.31 : Rhodococcus opacus B4	undecaprenyl pyrophosphate synthase	uppS2	H0R0T9_9ACTO Undecaprenyl pyrophosphate synthase OS =Gordonia effusa NBRC 100432 GN=uppS PE=3 SV=1;undecaprenyl pyrophosphate synthase [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of undecaprenyl pyrophosphate from isopentenyl pyrophosphate;Cis (Z)-Isoprenyl Diphosphate Synthases (cis-IPPS); homodimers which catalyze the successive 1'-4 condensation of the isopentenyl diphosphate (IPP) molecule to trans,trans-farnesyl diphosphate (FPP) or to cis,trans-FPP to form long-chain polyprenyl...; cd00475;Putative undecaprenyl diphosphate synthase; pfam01255;catalytic residue [active];putative FPP diphosphate  binding site;putative FPP binding hydrophobic cleft;dimer interface [polypeptide binding];putative IPP diphosphate binding site
GOHSU16	CDS	GOHSU_16_01210	130565..131650		enoyl-CoA hydratase/isomerase family protein [nite]			ROP_12430 :enoyl-CoA hydratase/isomerase family protein : : : Rhodococcus opacus B4	3-hydroxyisobutyryl-CoA hydrolase		H5UI30_9ACTO Putative enoyl-CoA hydratase OS=Gordon ia terrae NBRC 100016 GN=GOTRE_125_00250 PE=4 SV=1;3-hydroxyisobutyryl-CoA hydrolase [Rhodococcus opac us B4];catalyzes the formation of 3-hydroxy-2-methylpropanoate from 3-hydroxy-2-methylpropanoyl-CoA;3-hydroxyisobutyryl-CoA hydrolase; Provisional;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding];Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cl15762
GOHSU16	CDS	GOHSU_16_01220	131720..132496		enoyl-CoA hydratase [nite]	 echA [nite]		ROP_12410 :enoyl-CoA hydratase : echA : 4.2.1.17 : Rhodococcus opacus B4	enoyl-CoA hydratase	echA	F9VTQ9_9ACTO Enoyl-CoA hydratase OS=Gordonia alkani vorans NBRC 16433 GN=echA PE=3 SV=1;enoyl-CoA hydratase [Rhodococcus opacus B4];enoyl-CoA hydratase; Provisional;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU16	CDS	GOHSU_16_01230	complement(132597..133343)		DNA repair protein RecO [hamap]	recO [hamap]	MF_00201:DNA repair protein RecO:recO 	ROP_09460 :DNA repair protein RecO : recO : : Rhodococcus opacus B4	DNA repair protein RecO	recO	F1YDV6_9ACTO DNA repair protein RecO OS=Gordonia ne ofelifaecis NRRL B-59395 GN=recO PE=3 SV=1;recO gene product [Gordonia polyisoprenivorans VH2] ;DNA repair protein RecO; Reviewed; PRK00085;Recombination protein O N terminal; cl15812;Recombination protein O C terminal; pfam02565
GOHSU16	CDS	GOHSU_16_01240	complement(133402..133848)		hypothetical protein [nite]			MLP_43670 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F5XSX3_MICPN Putative uncharacterized protein OS=Mi crolunatus phosphovorus (strain ATCC 700054 / DSM 10555 / JCM 9379 / NBRC 101784 / NCIMB 13414 / VKM Ac-1990 / NM-1) GN=MLP_43670 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU16	CDS	GOHSU_16_01250	complement(133954..134907)		GTPase Era [hamap]	era [hamap]	MF_00367:GTPase Era:era 	RER_37310 :probable GTP-binding protein Era : era : : Rhodococcus erythropolis PR4	GTP-binding protein Era		F9VPK3_9ACTO GTPase Era OS=Gordonia alkanivorans NB RC 16433 GN=era PE=3 SV=1;GTP-binding protein Era [Streptomyces clavuligerus ATCC 27064];GTPase Era; Reviewed; PRK00089;E. coli Ras-like protein (Era) is a multifunctional GTPase; cd04163;G1 box;GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;Switch II region;G3 box;G4 box;G5 box;K homology RNA-binding domain, type I.  KH binds single-stranded RNA or DNA. It is found in a wide variety of proteins including ribosomal proteins, transcription factors and post-transcriptional modifiers of mRNA. There are two different KH domains that...; cl00098
GOHSU16	CDS	GOHSU_16_01260	complement(134900..136345)		hypothetical membrane protein [nite]			ROP_09620 :hypothetical membrane protein : : : Rhodococcus opacus B4	CBS domain-containing protein		F1YDV2_9ACTO CBS domain containing protein OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_01660 PE=4 SV=1;CBS domain-containing protein [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;Hemolysins and related proteins containing CBS domains [General function prediction only]; COG1253;Domain of unknown function DUF21; pfam01595;This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domains associated with the CorC_HlyC domain. CorC_HlyC is a transporter associated domain. This small domain is found in Na+/H+ antiporters, in proteins involved in...; cd04590;Transporter associated domain; cl08393
GOHSU16	CDS	GOHSU_16_01270	complement(136350..136895)	case	probable rRNA maturation factor YbeY [hamap]	ybeY [hamap]	MF_00009:probable rRNA maturation factor YbeY:ybeY 	RER_37330 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	metalloprotease		F1YDV1_9ACTO Probable rRNA maturation factor OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_01655 PE=3 SV=1;metalloprotease [Corynebacterium glutamicum ATCC 13 032];IS fingerprint type 4-5;Uncharacterized protein family UPF0054; cl00402
GOHSU16	CDS	GOHSU_16_01280	complement(136892..138016)		PhoH-like protein [nite]			ROP_09640 :PhoH-like protein : : : Rhodococcus opacus B4	PhoH family protein		F1YDV0_9ACTO PhoH family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_01650 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: PhoH family protein; KEGG: mxa:MXAN_4738 PhoH family protein;Phosphate starvation-inducible protein PhoH, predicted ATPase [Signal transduction mechanisms]; COG1702;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099
GOHSU16	CDS	GOHSU_16_01290	complement(138013..138822)		hypothetical protein [nite]			ROP_09650 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YDU9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01645 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mxa:MXAN_0913 hypothetical protein;Esterase/lipase [Lipid metabolism]; COG0657;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU16	CDS	GOHSU_16_01300	complement(138828..139580)		hypothetical protein [nite]			ROP_09660 :hypothetical protein : : : Rhodococcus opacus B4	16S ribosomal RNA methyltransferase RsmE		F1YDU8_9ACTO 16S ribosomal RNA methyltransferase Rs mE OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01640 PE =4 SV=1;16S ribosomal RNA methyltransferase RsmE [Corynebac terium glutamicum ATCC 13032];IS fingerprint type 4-5;Uncharacterized protein conserved in bacteria [Function unknown]; cl00611;in Escherichia coli RsmE methylates the N3 position of the U1498 base in 16S rRNA; cells lacking this function can grow, but are outcompeted by wild-type; SAM-dependent m(3)U1498 methyltransferase;RNA methyltransferase; pfam04452
GOHSU16	CDS	GOHSU_16_01310	complement(139577..140725)		chaperone protein DnaJ [hamap]	dnaJ [hamap]	MF_01152:chaperone protein DnaJ:dnaJ 	RER_37400 :chaperone protein DnaJ : dnaJ : : Rhodococcus erythropolis PR4	molecular chaperone DnaJ	dnaJ2	G4I131_MYCRH Chaperone protein DnaJ 2 OS=Mycobacter ium rhodesiae JS60 GN=dnaJ2 PE=3 SV=1;chaperone protein DnaJ [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;chaperone protein DnaJ; Provisional;chaperone Hsp40; co-chaperone with DnaK; Participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stress-denatured proteins and by disaggregating proteins, also in an autonomous, dnaK-independent fashion;DnaJ domain or J-domain.  DnaJ/Hsp40 (heat shock protein 40) proteins are highly conserved and play crucial roles in protein translation, folding, unfolding, translocation, and degradation. They act primarily by stimulating the ATPase activity of Hsp70s; cd06257;HSP70 interaction site [polypeptide binding];Zinc finger domain of DnaJ and HSP40; cd10719;Zn binding sites [ion binding];C-terminal substrate binding domain of DnaJ and HSP40; cd10747;dimer interface [polypeptide binding]
GOHSU16	CDS	GOHSU_16_01320	complement(140752..141789)		heat-inducible transcription repressor HrcA [hamap]	hrcA [hamap]	MF_00081:heat-inducible transcription repressor HrcA:hrcA 	RER_37410 :heat-inducible transcriptional repressor HrcA : hrcA : : Rhodococcus erythropolis PR4	heat-inducible transcription repressor	hrcA	F9VWM3_9ACTO Heat-inducible transcription repressor hrcA OS=Gordonia alkanivorans NBRC 16433 GN=hrcA PE=3 SV= 1;heat-inducible transcription repressor [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;heat-inducible transcription repressor; Provisional; PRK00082;Negative regulator of class I heat shock genes (grpE-dnaK-dnaJ and groELS operons). Prevents heat-shock induction of these operons;Helix-turn-helix domains; cl00088;HrcA protein C terminal domain; pfam01628
GOHSU16	CDS	GOHSU_16_01330	141872..142327		hypothetical protein [migap]				hypothetical protein		H0R0S7_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_061_00170 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Uncharacterized protein conserved in bacteria (DUF2191); cl02270
GOHSU16	CDS	GOHSU_16_01340	complement(142579..143778)		coproporphyrinogen III oxidase [nite]	 hemN [nite]		RER_37440 :coproporphyrinogen III oxidase : hemN : 1.3.99.22 : Rhodococcus erythropolis PR4	coproporphyrinogen III oxidase	hemN	F1YDU4_9ACTO Coproporphyrinogen III oxidase OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_01620 PE=4 SV=1;coproporphyrinogen III oxidase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the oxygen-independent formation of protoporphyrinogen-IX from coproporphyrinogen-III;coproporphyrinogen III oxidase; Validated;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site;HemN C-terminal domain; pfam06969
GOHSU16	CDS	GOHSU_16_01350	144113..145870		sulfite reductase [nite]	 sirA [nite]		RER_37460 :sulfite reductase : sirA : 1.8.7.1 : Rhodococcus erythropolis PR4	ferredoxin--nitrite reductase		F1YDU3_9ACTO Putative sulfite reductase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01615 PE=4 SV=1;ferredoxin--nitrite reductase [Rhodococcus jostii R HA1];Sulfite reductase, beta subunit (hemoprotein) [Inorganic ion transport and metabolism]; COG0155;Nitrite/Sulfite reductase ferredoxin-like half domain; pfam03460
GOHSU16	CDS	GOHSU_16_01360	145883..146611		adenosine 5'-phosphosulfate reductase [nite]	 cysH [nite]		ROP_09720 :adenosine 5'-phosphosulfate reductase : cysH : 1.8.4.10 : Rhodococcus opacus B4	phosphoadenosine phosphosulfate reductase	cysH	G7H4J5_9ACTO Adenosine 5'-phosphosulfate reductase OS=Gordonia araii NBRC 100433 GN=cysH PE=3 SV=1;phosphoadenosine phosphosulfate reductase [Mycobact erium marinum M];catalyzes the reduction of 3'-phosphoadenylyl sulfate into sulfite;This domain is found in phosphoadenosine phosphosulphate (PAPS) reductase enzymes or PAPS sulphotransferase. PAPS reductase is part of the adenine nucleotide alpha hydrolases superfamily also including N type ATP PPases and ATP sulphurylases. A highly...; cd01713;Active Sites [active]
GOHSU16	CDS	GOHSU_16_01370	146608..147564		sulfate adenylyltransferase subunit 2 [nite]	 cysD [nite]		ROP_09730 :sulfate adenylyltransferase subunit 2 : cysD : 2.7.7.4 : Rhodococcus opacus B4	sulfate adenylyltransferase subunit 2		F1YDU1_9ACTO Sulfate adenylyltransferase subunit 2 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01605 PE=4 SV=1;sulfate adenylyltransferase subunit 2 [Catenulispor a acidiphila DSM 44928];with CysN catalyzes the formation of adenylylsulfate from sulfate and ATP;sulfate adenylyltransferase, small subunit; TIGR02039;This domain is found in phosphoadenosine phosphosulphate (PAPS) reductase enzymes or PAPS sulphotransferase. PAPS reductase is part of the adenine nucleotide alpha hydrolases superfamily also including N type ATP PPases and ATP sulphurylases. A highly...; cd01713;Active Sites [active]
GOHSU16	CDS	GOHSU_16_01380	147564..148898		sulfate adenylyltransferase subunit 1 [nite]	 cysN [nite]		ROP_09740 :sulfate adenylyltransferase subunit 1 : cysN : 2.7.7.4 : Rhodococcus opacus B4	sulfate adenylyltransferase, large subunit		H5UJT7_9ACTO Sulfate adenylyltransferase subunit 1 OS=Gordonia terrae NBRC 100016 GN=cysN PE=3 SV=1;sulfate adenylyltransferase, large subunit [Catenul ispora acidiphila DSM 44928];KEGG: sen:SACE_1473 sulfate adenylyltransferase subunit 1 / adenylylsulfate kinase; TIGRFAM: sulfate adenylyltransferase, large subunit; PFAM: protein synthesis factor GTP-binding;sulfate adenylyltransferase, large subunit; TIGR02034;CysN, together with protein CysD, forms the ATP sulfurylase (ATPS) complex; cd04166;CysD dimerization site [polypeptide binding];G1 box;putative GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;CysN_NodQ_II: This subfamily represents the domain II of the large subunit of ATP sulfurylase (ATPS): CysN or the N-terminal portion of NodQ, found mainly in proteobacteria and homologous to the domain II of EF-Tu. Escherichia coli ATPS consists of CysN...; cd03695;TCysN_NoDQ_II: This subfamily represents the domain II of the large subunit of ATP sulfurylase (ATPS): CysN or the N-terminal portion of NodQ, found mainly in proteobacteria and homologous to the domain II of EF-Tu. Escherichia coli ATPS consists of CysN...; cd04095
GOHSU16	CDS	GOHSU_16_01390	148952..149710		putative ferrochelatase [nite]			ROP_09750 :putative ferrochelatase : : 4.99.1.- : Rhodococcus opacus B4	secreted protein		F1YDT9_9ACTO Cobalamin (Vitamin B12) biosynthesis C biX protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _01595 PE=4 SV=1;secreted protein [Streptomyces griseoflavus Tu4000] ;Sirohydrochlorin ferrochelatase [Inorganic ion transport and metabolism];Sirohydrochlorin cobalt chelatase (CbiX) and sirohydrochlorin iron chelatase (SirB), N-terminal domain. SirB catalyzes the ferro-chelation of sirohydrochlorin to siroheme, the prosthetic group of sulfite and nitrite reductases. CbiX is a cobaltochelatase; cd03416;putative active site [active];Sirohydrochlorin cobalt chelatase (CbiX) and sirohydrochlorin iron chelatase (SirB), C-terminal domain. SirB catalyzes the ferro-chelation of sirohydrochlorin to siroheme, the prosthetic group of sulfite and nitrite reductases. CbiX is a cobaltochelatase; cd03414
GOHSU16	CDS	GOHSU_16_01400	complement(149721..149993)		not assigned						
GOHSU16	CDS	GOHSU_16_01410	150297..150533		CsbD family protein [migap]				CsbD family protein		A0QJZ0_MYCA1 Putative uncharacterized protein OS=My cobacterium avium (strain 104) GN=MAV_4077 PE=4 SV=1;CsbD family protein [Mycobacterium sp. JDM601];CsbD-like; cl15799
GOHSU16	CDS	GOHSU_16_01420	150630..151199		DNA-binding protein Dps [nite]	 dps [nite]		ROP_02190 :DNA-binding protein Dps : dps : : Rhodococcus opacus B4	ferritin Dps family protein		I0H5E8_ACTMI Uncharacterized protein OS=Actinoplane s missouriensis 431 GN=AMIS_30150 PE=4 SV=1;ferritin Dps family protein [Kineococcus radiotoler ans SRS30216];type strain of Kineococcus radiotolerans;PFAM: Ferritin Dps family protein; KEGG: sco:SCO0596  DNA-binding protein;Ferritin-like domain; pfam00210;DPS protein, ferritin-like diiron-binding domain; cd01043;dimerization interface [polypeptide binding];DPS ferroxidase diiron center [ion binding];ion pore
GOHSU16	CDS	GOHSU_16_01430	151186..151809		putative peptidase C56 family protein [nite]			KSE_07450 :putative peptidase C56 family protein : : : Kitasatospora setae NBRC 14216	intracellular protease, PfpI family		D0L7E8_GORB4 Intracellular protease, PfpI family OS =Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_4608 PE=4 SV=1;intracellular protease, PfpI family [Variovorax par adoxus S110];TIGRFAM: intracellular protease, PfpI family; PFAM: ThiJ/PfpI domain protein; KEGG: rce:RC1_2891 intracellular protease, PFpI family protein;A type 1 glutamine amidotransferase (GATase1)-like domain found in PfpI from Pyrococcus furiosus; cd03134;intracellular protease, PfpI family; TIGR01382;proposed catalytic triad [active];conserved cys residue [active]
GOHSU16	CDS	GOHSU_16_01440	complement(151868..153502)		putative fatty-acid--CoA ligase [nite]			RER_50020 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus erythropolis PR4	o-succinylbenzoate--CoA ligase		F1YDT7_9ACTO AMP-binding enzyme OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_01585 PE=4 SV=1;o-succinylbenzoate--CoA ligase [Pseudonocardia diox anivorans CB1190];KEGG: rha:RHA1_ro10402 long-chain-fatty-acid--CoA ligase; PFAM: AMP-dependent synthetase/ligase;acyl-CoA synthetase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU16	CDS	GOHSU_16_01450	153702..155159		serine/threonine protein kinase PknB [nite]	 pknB [nite]		ROP_35110 :serine/threonine protein kinase PknB : pknB : 2.7.11.1 : Rhodococcus opacus B4	serine/threonine protein kinase-like protein		G7GSN2_9ACTO Putative serine/threonine protein kina se OS=Gordonia amarae NBRC 15530 GN=GOAMR_56_00380 PE=4 SV =1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: serine/threonine protein kinase-related; tyrosine protein kinase; SMART: serine/threonine protein kinase; tyrosine protein kinase; KEGG: rfr:Rfer_3942 serine/threonine protein kinase with CHASE2 sensor;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
GOHSU16	CDS	GOHSU_16_01460	complement(155164..156159)		sulfate ABC transporter ATP-binding protein [nite]	 cysA [nite]		RER_37530 :sulfate ABC transporter ATP-binding protein : cysA : 3.6.3.25 : Rhodococcus erythropolis PR4	sulfate transport system ATP-binding protein		F1YDT6_9ACTO Sulfate ABC transporter ATPase subunit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01580 PE=3 SV=1;sulfate transport system ATP-binding protein [Micro cystis aeruginosa NIES- 843];Part of the ABC transporter complex cysAWTP involved in sulfate import.  Responsible for energy coupling to the transport system.  The complex is composed of two ATP-binding proteins (cysA), two transmembrane proteins (cysT and cysW), and a...; cd03296;sulfate ABC transporter, ATP-binding protein; TIGR00968;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region;TOBE domain; cl01440
GOHSU16	CDS	GOHSU_16_01470	complement(156160..157056)		sulfate ABC transporter permease protein [nite]	 cysW [nite]		RER_37540 :sulfate ABC transporter permease protein : cysW : : Rhodococcus erythropolis PR4	sulfate ABC transporter permease		D0LBZ0_GORB4 Sulfate ABC transporter, inner membran e subunit CysW OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3172 PE=3 SV =1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: sulfate ABC transporter inner membrane protein; sulfate ABC transporter inner membrane subunit; PFAM: binding-protein-dependent transporters inner membrane component; KEGG: reh:H16_A2238 ABC-type transporter permease: SulT family;sulfate transport protein; Provisional; CHL00187;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU16	CDS	GOHSU_16_01480	complement(157053..157922)		sulfate ABC transporter permease protein [nite]	 cysT [nite]		RER_37550 :sulfate ABC transporter permease protein : cysT : : Rhodococcus erythropolis PR4	sulfate transport system permease CysT	cysT	F9VX69_9ACTO Sulfate ABC transporter permease prote in OS=Gordonia alkanivorans NBRC 16433 GN=cysT PE=3 SV=1;cysT gene product [Gordonia polyisoprenivorans VH2] ;sulfate transport protein; Provisional; CHL00187;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU16	CDS	GOHSU_16_01490	complement(157919..158986)		sulfate ABC transporter sulfate-binding protein [nite]	 subI [nite]		ROP_09930 :sulfate ABC transporter sulfate-binding protein : subI : : Rhodococcus opacus B4	sulfate ABC transporter substrate-binding protein		F1YDT4_9ACTO Sulfate ABC transporter periplasmic su lfate--binding protein OS=Gordonia neofelifaecis NRRL B-59 395 GN=SCNU_01560 PE=4 SV=1;sulfate ABC transporter substrate-binding protein [ Cyanothece sp. PCC 7822];KEGG: ava:Ava_4715 thiosulphate-binding protein; TIGRFAM: sulfate ABC transporter, periplasmic sulfate-binding protein; PFAM: extracellular solute-binding protein family 1;The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cl11398;Bacterial extracellular solute-binding protein; pfam13531
GOHSU16	CDS	GOHSU_16_01500	complement(159155..159355)		hypothetical membrane protein [nite]			ROP_09940 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YDT3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01555 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU16	CDS	GOHSU_16_01510	complement(159407..161524)		peptidyl-dipeptidase Dcp [nite]	 dcp [nite]		MLP_12170 :peptidyl-dipeptidase Dcp : dcp : 3.4.15.5 : Microlunatus phosphovorus NM-1	peptidyl-dipeptidase Dcp		F1YDT2_9ACTO Peptidyl-dipeptidase Dcp OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_01550 PE=3 SV=1;peptidyl-dipeptidase Dcp [Opitutus terrae PB90- 1];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.997) with cleavage site probability 0.476 at residue 29;PFAM: peptidase M3A and M3B thimet/oligopeptidase F; KEGG: xac:XAC1456 peptidyl-dipeptidase;Zn-dependent oligopeptidases [Amino acid transport and metabolism]; COG0339;Peptidase family M3 dipeptidyl carboxypeptidase (DCP); cd06456;Zn binding site [ion binding]
GOHSU16	CDS	GOHSU_16_01520	complement(161530..163425)		elongation factor 4 [hamap]	lepA [hamap]	MF_00071:elongation factor 4:lepA 	ROP_09980 :elongation factor LepA : lepA : : Rhodococcus opacus B4	GTP-binding protein LepA	lepA	H5UJW4_9ACTO Elongation factor 4 OS=Gordonia terrae NBRC 100016 GN=lepA PE=3 SV=1;GTP-binding protein LepA [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;binds to the ribosome on the universally-conserved alpha-sarcin loop;GTP-binding protein LepA; Provisional;LepA also known as Elongation Factor 4 (EF4); cd01890;G1 box;putative GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;lepA_II: This subfamily represents the domain II of LepA, a GTP-binding protein localized in the cytoplasmic membrane. The N-terminal domain of LepA shares regions of homology to translation factors. In terms of interaction with the ribosome, EF-G, EF-Tu...; cd03699;lepA_C: This family represents the C-terminal region of LepA, a GTP-binding protein localized in the cytoplasmic membrane.   LepA is ubiquitous in Bacteria and Eukaryota (e.g. Saccharomyces cerevisiae GUF1p), but is missing from Archaea. LepA exhibits...; cd03709;GTP-binding protein LepA C-terminus; pfam06421
GOHSU16	CDS	GOHSU_16_01530	163509..164207		hypothetical protein [nite]			ROP_51970 :hypothetical protein : : : Rhodococcus opacus B4	alpha/beta hydrolase		F1YDT0_9ACTO Hydrolase of the alpha/beta-hydrolase fold-like protein OS=Gordonia neofelifaecis NRRL B-59395 G N=SCNU_01540 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: maq:Maqu_1783 alpha/beta fold family hydrolase;Predicted hydrolase of the alpha/beta-hydrolase fold [General function prediction only]
GOHSU16	CDS	GOHSU_16_01540	164204..164623		conserved hypothetical protein [nite]			RER_37620 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G8RKI9_MYCRN PemK-like protein OS=Mycobacterium rho desiae (strain NBB3) GN=MycrhN_4279 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];PemK-like protein; cl00995
GOHSU16	CDS	GOHSU_16_01550	164706..165095		hypothetical protein [migap]				hypothetical protein		G7H4I1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_062_00240 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.991 at residue 31;KEGG: rpa:RPA3794 hypothetical protein;Domain of unknown function (DUF4189); pfam13827
##GOHSU17	contig00062				47381
GOHSU17	CDS	GOHSU_17_00010	complement(<1..1147)	3x codon	hypothetical protein [migap]				hypothetical protein		G7H4H9_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_062_00220 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614]
GOHSU17	CDS	GOHSU_17_00020	1357..1617	case	30S ribosomal protein S20 [hamap]	rpsT [hamap]	MF_00500:30S ribosomal protein S20:rpsT 	ROP_10050 :30S ribosomal protein S20 : rpsT : : Rhodococcus opacus B4	30S ribosomal protein S20	rpsT	F1YDS7_9ACTO 30S ribosomal protein S20 OS=Gordonia neofelifaecis NRRL B-59395 GN=rpsT PE=3 SV=1;30S ribosomal protein S20 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosomal protein S20; cl00384;binds directly to the 16S rRNA and is involved in post-translational inhibition of arginine and ornithine decarboxylase
GOHSU17	CDS	GOHSU_17_00030	complement(1681..2643)		putative DNA polymerase III delta subunit [nite]	 holA [nite]		ROP_10060 :putative DNA polymerase III delta subunit : holA : 2.7.7.7 : Rhodococcus opacus B4	DNA polymerase III subunit delta		F1YDS6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01520 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];hypothetical protein; Reviewed;DNA polymerase III, delta subunit; pfam06144
GOHSU17	CDS	GOHSU_17_00040	2724..3800		putative homoserine O-acetyltransferase [nite]	 metX [nite]		BBR47_03140 :putative homoserine O-acetyltransferase : metX : 2.3.1.31 : Brevibacillus brevis NBRC 100599	alpha/beta hydrolase fold protein		C8X630_NAKMY Alpha/beta hydrolase fold protein OS=N akamurella multipartita (strain ATCC 700099 / DSM 44233 / JCM 9543 / Y-104) GN=Namu_0373 PE=4 SV=1;alpha/beta hydrolase fold protein [Geodermatophilus obscurus DSM 43160];Homoserine acetyltransferase [Amino acid transport and metabolism]; COG2021;PFAM: alpha/beta hydrolase fold; KEGG: rha:RHA1_ro10186 hypothetical protein;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU17	CDS	GOHSU_17_00050	3894..5171		two-component histidine kinase [nite]			ROP_38400 :two-component histidine kinase : : 2.7.13.- : Rhodococcus opacus B4	histidine kinase		Q0S9L9_RHOSR Sensor kinase, two-component system OS =Rhodococcus sp. (strain RHA1) GN=RHA1_ro03970 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: reh:H16_A1985 signal transduction histidine kinase containing PAS/PAC sensor domain;Histidine kinase; pfam07730;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU17	CDS	GOHSU_17_00060	5168..5794		probable NarL family two-component response regulator [nite]			RER_00980 :probable NarL family two-component response regulator : : : Rhodococcus erythropolis PR4	NarL family two-component response regulator		H6QZ93_NOCCG Response regulator, two-component syst em OS=Nocardia cyriacigeorgica (strain GUH-2) GN=NOCYR_206 5 PE=4 SV=1;NarL family two-component response regulator [Rhodo coccus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding]
GOHSU17	CDS	GOHSU_17_00070	5931..8048		conserved hypothetical membrane protein [nite]			RER_31970 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	MMPL domain-containing protein		D6EN65_STRLI Export protein OS=Streptomyces lividan s TK24 GN=SSPG_01824 PE=4 SV=1;MMPL domain-containing protein [Kribbella flavida D SM 17836];PFAM: MMPL domain protein; KEGG: hypothetical protein;Transport protein; TIGR00833
GOHSU17	CDS	GOHSU_17_00080	8497..8679		hypothetical protein [migap]				hypothetical protein		Q8FUJ6_COREF Putative uncharacterized protein OS=Co rynebacterium efficiens (strain DSM 44549 / YS-314 / AJ 12 310 / JCM 11189 / NBRC 100395) GN=CE0022 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314]
GOHSU17	CDS	GOHSU_17_00090	complement(8742..10181)		putative transposase [nite]			MLP_01030 :putative transposase : : : Microlunatus phosphovorus NM-1	integrase catalytic subunit		A1SCS8_NOCSJ Integrase, catalytic region OS=Nocardi oides sp. (strain BAA-499 / JS614) GN=Noca_0065 PE=4 SV=1;integrase catalytic subunit [Methylobacterium sp. 4 - 46];PFAM: Integrase catalytic region; KEGG: sen:SACE_5091 transposase;Helix-turn-helix domains; cl00088;Transposase and inactivated derivatives, IS30 family [DNA replication, recombination, and repair]; COG2826;Integrase core domain; cl01316
GOHSU17	CDS	GOHSU_17_00100	10246..11580		aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit A [nite]	 gatA [nite]		BBR47_06940 :aspartyl/glutamyl-tRNA(Asn/Gln) amidotransferase subunit A : gatA : 6.3.5.- : Brevibacillus brevis NBRC 100599	glutamyl-tRNA(gln) amidotransferase subunit a		A4QI09_CORGB Putative uncharacterized protein OS=Co rynebacterium glutamicum (strain R) GN=cgR_2856 PE=4 SV=1;glutamyl-tRNA(gln) amidotransferase subunit a [Paen ibacillus polymyxa SC2];Asp-tRNAAsn/Glu-tRNAGln amidotransferase A subunit and related amidases [Translation, ribosomal structure and biogenesis]; COG0154;Amidase; cl11426
GOHSU17	CDS	GOHSU_17_00110	11667..12278		putative amidase substrates transport protein [nite]	 amiS [nite]		RER_04930 :putative amidase substrates transport protein : amiS : : Rhodococcus erythropolis PR4	AmiS/UreI transporter		D2KYA8_CORGT Putative uncharacterized protein OS=Co rynebacterium glutamicum PE=4 SV=1;AmiS/UreI transporter [Oligotropha carboxidovorans OM5];type strain of Oligotropha carboxidovorans;AmiS/UreI family transporter; pfam02293
GOHSU17	CDS	GOHSU_17_00120	12302..13501		putative arsenical pump membrane protein [nite]			KSE_01500 :putative arsenical pump membrane protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		F1YLZ6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14701 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314];similar to M86824-3|AAA25637.1| percent identity: 25 in 381 aa;Anion permease ArsB/NhaD.  These permeases have been shown to translocate sodium, arsenate, antimonite, sulfate and organic anions across biological membranes in all three kingdoms of life.  A typical anion permease contains 8-13 transmembrane helices...; cl15357
GOHSU17	CDS	GOHSU_17_00130	complement(13496..14566)		putative aliphatic amidase regulator [nite]	 amiC [nite]		RER_04940 :putative aliphatic amidase regulator : amiC : : Rhodococcus erythropolis PR4	regulatory protein		A4QI10_CORGB Putative uncharacterized protein OS=Co rynebacterium glutamicum (strain R) GN=cgR_2857 PE=4 SV=1;regulatory protein [Pseudonocardia dioxanivorans CB 1190];KEGG: sma:SAV_1189 regulatory protein;Periplasmic binding protein; pfam13458;Type I periplasmic-binding protein of the nitrile hydratase (NHase) system that selectively converts nitriles to corresponding amides; cd06358;putative ligand binding site [chemical binding]
GOHSU17	CDS	GOHSU_17_00140	14710..15174		MarR family transcriptional regulator [nite]			ACMV_21320 :MarR family transcriptional regulator : : : Acidiphilium multivorum AIU301	MarR family transcriptional regulator		A4QI11_CORGB Putative uncharacterized protein OS=Co rynebacterium glutamicum (strain R) GN=cgR_2858 PE=4 SV=1;MarR family transcriptional regulator [Rhodococcus jostii RHA1];helix_turn_helix multiple antibiotic resistance protein; smart00347;Helix-turn-helix domains; cl00088
GOHSU17	CDS	GOHSU_17_00150	complement(15191..15577)		predicted protein [migap]				predicted protein		G2PC11_STRVO Putative uncharacterized protein OS=St reptomyces violaceusniger Tu 4113 GN=Strvi_1646 PE=4 SV=1;predicted protein [Streptomyces sp. C];TIGR02679 family protein;Protein of unknown function N-terminus (DUF3323); pfam11796
GOHSU17	CDS	GOHSU_17_00160	complement(15905..16171)		luciferase-like monooxygenase [migap]				luciferase-like monooxygenase		F9VSL0_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_033_01320 PE=4 SV= 1;luciferase-like monooxygenase [Thermobispora bispor a DSM 43833];type strain of Thermobispora bispora;PFAM: Luciferase-like monooxygenase; KEGG: tgr:Tgr7_0734 luciferase-like monooxygenase;Coenzyme F420-dependent N5,N10-methylene tetrahydromethanopterin reductase and related flavin-dependent oxidoreductases [Energy production and conversion]
GOHSU17	CDS	GOHSU_17_00170	complement(16428..16775)		PemK-like protein [nite]			SLG_24180 :PemK-like protein : : : Sphingobium sp. SYK-6	transcriptional modulator of MazE/toxin MazF		D0WLM8_9ACTO Toxin-antitoxin system, toxin componen t, MazF family OS=Actinomyces sp. oral taxon 848 str. F033 2 GN=HMPREF0972_00683 PE=4 SV=1;unnamed protein product [Frankia symbiont of Datisc a glomerata];PFAM: PemK-like protein; KEGG: mar:MAE_62380 PemK-like protein;PemK-like protein; cl00995
GOHSU17	CDS	GOHSU_17_00180	complement(16762..16908)	short	not assigned						
GOHSU17	CDS	GOHSU_17_00190	complement(17000..18496)		putative ComEC/Rec2-related protein [nite]			ROP_10130 :putative ComEC/Rec2-related protein : : : Rhodococcus opacus B4	ComEC/Rec2-like protein		F1YDS5_9ACTO ComEC/Rec2-like protein OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_01515 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.764) with cleavage site probability 0.746 at residue 29;TIGRFAM: ComEC/Rec2-related protein; PFAM: ComEC/Rec2-related protein; KEGG: ank:AnaeK_2593 DNA internalization-related competence protein ComEC/Rec2;Competence protein; cl00471
GOHSU17	CDS	GOHSU_17_00200	complement(18493..19620)		competence protein ComEA [migap]				competence protein ComEA		F1YDS4_9ACTO Competence protein ComEA helix-hairpin -helix repeat-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01510 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: aha:AHA_3498 ComEA protein; TIGRFAM: competence protein ComEA helix-hairpin- helix repeat protein; PFAM: Soluble ligand binding domain; helix-hairpin- helix motif; SMART: Helix-hairpin-helix DNA-binding class 1;Helix-hairpin-helix motif; pfam12836;endonuclease III; includes endonuclease III (DNA-(apurinic or apyrimidinic site) lyase), alkylbase DNA glycosidases (Alka-family) and other DNA glycosidases; cl14786
GOHSU17	CDS	GOHSU_17_00210	complement(19750..20598)		hypothetical protein [nite]			ROP_10150 :hypothetical protein : : : Rhodococcus opacus B4	DegV family protein		F1YDS1_9ACTO DegV family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_01495 PE=4 SV=1;DegV family protein [Amycolatopsis mediterranei U32 ];Uncharacterized protein, DegV family COG1307; cl00577
GOHSU17	CDS	GOHSU_17_00220	complement(20643..21428)		hypothetical protein [nite]			ROP_10160 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YDS0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01490 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];SGNH_hydrolase, or GDSL_hydrolase, is a diverse family of lipases and esterases. The tertiary fold of the enzyme is substantially different from that of the alpha/beta hydrolase family and unique among all known hydrolases; its active site closely...; cl01053
GOHSU17	CDS	GOHSU_17_00230	complement(21428..22141)		phosphoglycerate mutase family protein [nite]			ROP_10170 :phosphoglycerate mutase family protein : : : Rhodococcus opacus B4	fructose-2,6-bisphosphatase		F1YDR9_9ACTO Phosphoglycerate mutase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_01485 PE=4 SV=1;fructose-2,6-bisphosphatase [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;Histidine phosphatase domain found in phosphoglycerate mutases and related proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cd07067;catalytic core [active]
GOHSU17	CDS	GOHSU_17_00240	complement(22138..22530)		conserved hypothetical protein [nite]			RER_37750 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	iojap family protein		H5RSM6_9NOCA Putative uncharacterized protein OS=No cardia brasiliensis ATCC 700358 GN=O3I_15834 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: iojap-like protein; PFAM: Iojap-related protein; KEGG: pmy:Pmen_3800 iojap-like protein;Oligomerisation domain; cl00519
GOHSU17	CDS	GOHSU_17_00250	complement(22527..23177)		probable nicotinate-nucleotide adenylyltransferase [hamap]	nadD [hamap]	MF_00244:probable nicotinate-nucleotide adenylyltransferase:nadD 	RER_37760 :nicotinate-nucleotide adenylyltransferase : nadD : 2.7.7.18 : Rhodococcus erythropolis PR4	nicotinic acid mononucleotide adenylyltransferase	nadD	F1YDR7_9ACTO Probable nicotinate-nucleotide adenyly ltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=nad D PE=3 SV=1;nicotinic acid mononucleotide adenylyltransferase [ Corynebacterium glutamicum ATCC 13032];transfers an adenyl group from ATP to NaMN to form nicotinic acid adenine dinucleotide (NaAD) which is then converted to the ubiquitous compound NAD by NAD synthetase; essential enzyme in bacteria;nicotinic acid mononucleotide adenylyltransferase; Provisional; PRK00071;Nicotinamide/nicotinate mononucleotide adenylyltransferase; cd02165;(T/H)XGH motif
GOHSU17	CDS	GOHSU_17_00260	complement(23202..24650)		hypothetical protein [nite]			ROP_10200 :hypothetical protein : : : Rhodococcus opacus B4	VWA domain-containing CoxE-like protein		F1YDR6_9ACTO VWA containing CoxE family protein OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01470 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Von Willebrand factor type A (vWA) domain was originally found in the blood coagulation protein von Willebrand factor (vWF). Typically, the vWA domain is made up of approximately 200 amino acid residues folded into a classic a/b para-rossmann type of...; cl00057
GOHSU17	CDS	GOHSU_17_00270	complement(24655..25584)		hypothetical protein [nite]			ROP_10210 :hypothetical protein : : : Rhodococcus opacus B4	MoxR-like ATPase		F1YDR5_9ACTO ATPase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_01465 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: ATPase family associated with various cellular activities (AAA);ATPases associated with a variety of cellular activities; smart00382;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger
GOHSU17	CDS	GOHSU_17_00280	complement(25581..26852)		gamma-glutamyl phosphate reductase [hamap]	proA [hamap]	MF_00412:gamma-glutamyl phosphate reductase:proA 	RER_37790 :gamma-glutamyl phosphate reductase : proA : 1.2.1.41 : Rhodococcus erythropolis PR4	gamma-glutamyl phosphate reductase	proA	F1YDR4_9ACTO Gamma-glutamyl phosphate reductase OS= Gordonia neofelifaecis NRRL B-59395 GN=proA PE=3 SV=1;gamma-glutamyl phosphate reductase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Catalyzes the phosphorylation of L-glutamate during the proline biosynthesis pathway;Gamma-glutamyl phosphate reductase (GPR), aldehyde dehydrogenase families 18 and 19; cd07079;gamma-glutamyl phosphate reductase; TIGR00407;putative catalytic cysteine [active]
GOHSU17	CDS	GOHSU_17_00290	26947..27804		hypothetical protein [migap]				hypothetical protein		G7GU01_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_61_01740 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: fumarate reductase
GOHSU17	CDS	GOHSU_17_00300	complement(27842..28621)		hypothetical protein [nite]			ROP_52660 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YDQ9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01435 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];KEGG: cef:CE2653 hypothetical protein;Protein of unknown function DUF262; cl14890
GOHSU17	CDS	GOHSU_17_00310	complement(28709..29638)		putative oxidoreductase [nite]			RER_01680 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	luciferase family protein		G7GYX6_9ACTO Putative oxidoreductase OS=Gordonia ar aii NBRC 100433 GN=GOARA_021_00380 PE=4 SV=1;luciferase family protein [Mycobacterium vanbaaleni i PYR- 1];PFAM: luciferase family protein; KEGG: mmc:Mmcs_0451 luciferase-like protein;probable F420-dependent oxidoreductase, Rv1855c family; TIGR03560;Flavin-utilizing monoxygenases; cl07892
GOHSU17	CDS	GOHSU_17_00320	complement(29681..31918)		ATP-dependent DNA helicase RecG [nite]	 recG [nite]		RER_24240 :ATP-dependent DNA helicase RecG : recG : 3.6.1.- : Rhodococcus erythropolis PR4	ATP-dependent DNA helicase RecG	recG	F1YDQ7_9ACTO ATP-dependent DNA helicase RecG OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_01425 PE=4 SV=1;ATP-dependent DNA helicase RecG [Streptomyces clavu ligerus ATCC 27064];KEGG: Homologous recombination. COG: COG1200 RecG-like helicase.;ATP-dependent DNA helicase RecG; Provisional;RecG_wedge_OBF: A subfamily of OB folds corresponding to the OB fold found in the N-terminal (wedge) domain of Escherichia coli RecG. RecG is a branched-DNA-specific helicase, which catalyzes the interconversion of a DNA replication fork to a...; cd04488;generic binding surface II;ssDNA binding site;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding]
GOHSU17	CDS	GOHSU_17_00330	complement(31922..33568)		hypothetical protein [nite]			ROP_65660 :hypothetical protein : : : Rhodococcus opacus B4	DAK2 domain fusion protein YloV		F1YDQ6_9ACTO DAK2 domain fusion protein YloV OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_01420 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: DAK2 domain fusion protein YloV; PFAM: Dak phosphatase; KEGG: dvm:DvMF_0600 DegV family protein;DAK2 domain fusion protein YloV; TIGR03599;DAK2 domain; cl03685;Uncharacterized protein, DegV family COG1307; cl00577
GOHSU17	CDS	GOHSU_17_00340	33800..33991	case	50S ribosomal protein L28 [hamap]	rpmB [hamap]	MF_00373:50S ribosomal protein L28:rpmB 	RER_24220 :probable 50S ribosomal protein L28 : rpmB : : Rhodococcus erythropolis PR4	50S ribosomal protein L28		D0LC20_GORB4 50S ribosomal protein L28 2 OS=Gordoni a bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=rpmB2 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Ribosomal L28 family; cl00367;TIGRFAM: ribosomal protein L28; PFAM: ribosomal protein L28; KEGG: nam:NAMH_1582 ribosomal protein L28
GOHSU17	CDS	GOHSU_17_00350	34070..34858		putative enoyl-CoA hydratase [nite]			RER_24210 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase/isomerase		F1YDQ4_9ACTO Enoyl-CoA hydratase/isomerase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_01410 PE=4 SV=1;enoyl-CoA hydratase/isomerase [Sphingobium chloroph enolicum L- 1];enoyl-CoA hydratase; Provisional;KEGG: swi:Swit_3600 enoyl-CoA hydratase/isomerase; manually curated; PFAM: Crotonase, core;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU17	CDS	GOHSU_17_00360	complement(34862..35359)		NADPH:adrenodoxin oxidoreductase FprB [migap]	fprB [migap]			NADPH:adrenodoxin oxidoreductase FprB	fprB	fprB gene product [Mycobacterium ulcerans Agy99];cytoplasmic protein; serves as the first electron transfer protein in all the P450 systems [catalytic activity: reduced adrenodoxin + NADP+ = oxidized adrenodoxin + NADPH];ferredoxin-NADP+ reductase
GOHSU17	CDS	GOHSU_17_00370	complement(35356..36033)		uracil-DNA glycosylase [hamap]	ung [hamap]	MF_00148:uracil-DNA glycosylase:ung 	RER_24200 :uracil-DNA glycosylase : ung : 3.2.2.- : Rhodococcus erythropolis PR4	uracil-DNA glycosylase	ung	F1YF10_9ACTO Uracil-DNA glycosylase OS=Gordonia neo felifaecis NRRL B-59395 GN=ung PE=3 SV=1;ung gene product [Gordonia polyisoprenivorans VH2];Family 1 of Uracil-DNA glycosylase (UDG) enzymes; cd10027;ligand binding site [chemical binding];UGI interface [polypeptide binding];catalytic site [active]
GOHSU17	CDS	GOHSU_17_00380	complement(36042..36572)		hypothetical protein [nite]			ROP_65620 :hypothetical protein : : : Rhodococcus opacus B4	Bacterial transferase hexapeptide repeat protein		F1YF09_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01400 PE=4 SV=1;Bacterial transferase hexapeptide repeat protein [A ciduliprofundum boonei T469];[R] COG0663 Carbonic anhydrases/acetyltransferases, isoleucine patch superfamily;Carbonic anhydrases/acetyltransferases, isoleucine patch superfamily [General function prediction only]; COG0663;Gamma carbonic anhydrase-like: This family is composed of gamma carbonic anhydrase (CA), Ferripyochelin Binding Protein (FBP), E. coli paaY protein, and similar proteins. CAs are zinc-containing enzymes that catalyze the reversible hydration of carbon...; cd04645;trimer interface [polypeptide binding];putative metal binding site [ion binding]
GOHSU17	CDS	GOHSU_17_00390	complement(36590..37150)	overlap	hypothetical membrane protein [nite]			ROP_65590 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H6MUL9_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c29860 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF3515); pfam12028
GOHSU17	CDS	GOHSU_17_00400	37044..38198	overlap	thiamine-monophosphate kinase [nite]	 thiL [nite]		RER_24180 :thiamine-monophosphate kinase : thiL : 2.7.4.16 : Rhodococcus erythropolis PR4	thiamine monophosphate kinase		F1YF08_9ACTO Thiamine-monophosphate kinase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_01395 PE=4 SV=1;thiamine monophosphate kinase [Corynebacterium glut amicum ATCC 13032];catalyzes the formation of thiamine diphosphate from thiamine phosphate ant ATP;thiamine monophosphate kinase; Provisional;ThiL (Thiamine-monophosphate kinase) plays a dual role in de novo biosynthesis and in salvage of exogenous thiamine. Thiamine salvage occurs in two steps, with thiamine kinase catalyzing the formation of thiamine phosphate, and ThiL catalyzing the...; cd02194;ATP binding site [chemical binding];dimerization interface [polypeptide binding]
GOHSU17	CDS	GOHSU_17_00410	complement(38214..39212)		NAD(P)H-dependent glycerol-3-phosphate dehydrogenase [hamap]	gpsA [hamap]	MF_00394:NAD(P)H-dependent glycerol-3-phosphate dehydrogenase:gpsA 	ROP_65560 :NAD(P)H-dependent glycerol-3-phosphate dehydrogenase : gpsA : 1.1.1.94 : Rhodococcus opacus B4	NAD(P)H-dependent glycerol-3-phosphate dehydrogenase	gpsA	F1YF06_9ACTO Glycerol-3-phosphate dehydrogenase [NA D(P)+] OS=Gordonia neofelifaecis NRRL B-59395 GN=gpsA PE=3 SV=1;NAD(P)H-dependent glycerol-3-phosphate dehydrogenas e [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;NAD(P)H-dependent glycerol-3-phosphate dehydrogenase; Validated; PRK00094;catalyzes the NAD(P)H-dependent reduction of glycerol 3-phosphate to glycerone phosphate;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD-dependent glycerol-3-phosphate dehydrogenase C-terminus; pfam07479
GOHSU17	CDS	GOHSU_17_00420	39359..40048		hypothetical protein [nite]			KSE_51470 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		H5UJZ9_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_145_02180 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];KEGG: sco:SCO5557a hypothetical protein;Glycosyltransferase family A (GT-A) includes diverse families of glycosyl transferases with a common GT-A type structural fold; cl11394
GOHSU17	CDS	GOHSU_17_00430	40050..42251		polyphosphate kinase [hamap]	ppk [hamap]	MF_00347:polyphosphate kinase:ppk 	RER_24120 :polyphosphate kinase : ppk : 2.7.4.1 : Rhodococcus erythropolis PR4	polyphosphate kinase Ppk	ppk	F1YF04_9ACTO Polyphosphate kinase OS=Gordonia neofe lifaecis NRRL B-59395 GN=ppk PE=3 SV=1;ppk gene product [Gordonia polyisoprenivorans VH2];polyphosphate kinase; Provisional;Polyphosphate kinase N-terminal domain; pfam13089;Polyphosphate kinase middle domain; pfam02503;Catalytic C-terminal domain, first repeat, of Pseudomonas aeruginosa polyphosphate kinase 1 and similar proteins; cd09165;putative domain interface [polypeptide binding];putative active site [active];catalytic site [active];Catalytic C-terminal domain, second repeat, of Pseudomonas aeruginosa polyphosphate kinase 1 and similar proteins; cd09168
GOHSU17	CDS	GOHSU_17_00440	42248..43165		NTP pyrophosphohydrolase MutT [nite]	 mutT [nite]		ROP_65530 :NTP pyrophosphohydrolase MutT : mutT : 3.6.1.- : Rhodococcus opacus B4	NUDIX hydrolase		F1YF03_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_01370 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: NUDIX hydrolase; phosphoglycerate mutase; KEGG: bra:BRADO5692  NUDIX hydrolase (modular protein);Diadenosine hexaphosphate (Ap6A) hydrolase is a member of the Nudix hydrolase superfamily. Ap6A hydrolase specifically hydrolyzes diadenosine polyphosphates, but not ATP or diadenosine triphosphate, and it generates ATP as the product. Ap6A, the most...; cd03673;Ap6A binding site [chemical binding];nudix motif;metal binding site [ion binding];Histidine phosphatase domain found in phosphoglycerate mutases and related proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cd07067;catalytic core [active]
GOHSU17	CDS	GOHSU_17_00450	complement(43188..43817)		DNA-binding protein HU [migap]				DNA-binding protein HU		E2U177_MYCTU Putative uncharacterized protein (Frag ment) OS=Mycobacterium tuberculosis SUMu004 GN=TMDG_00186 PE=3 SV=1;DNA-binding protein HU [Synechococcus sp. PCC 7335] ;[L] COG0776 Bacterial nucleoid DNA-binding protein;Integration host factor (IHF) and HU are small heterodimeric members of the DNABII protein family that bind and bend DNA, functioning as architectural factors in many cellular processes including transcription, site-specific recombination, and...; cd00591;IHF dimer interface [polypeptide binding];IHF - DNA interface [nucleotide binding]
GOHSU17	CDS	GOHSU_17_00460	complement(43924..44514)		3-isopropylmalate dehydratase small subunit [hamap]	leuD [hamap]	MF_01031:3-isopropylmalate dehydratase small subunit:leuD 	RER_24090 :3-isopropylmalate dehydratase small subunit : leuD : 4.2.1.33 : Rhodococcus erythropolis PR4	isopropylmalate isomerase small subunit	leuD	F1YF01_9ACTO 3-isopropylmalate dehydratase small su bunit OS=Gordonia neofelifaecis NRRL B-59395 GN=leuD PE=3 SV=1;isopropylmalate isomerase small subunit [Corynebact erium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate in leucine biosynthesis; forms a heterodimer of LeuC/D;Aconatase-like swivel domain of 3-isopropylmalate dehydratase and related uncharacterized proteins. 3-isopropylmalate dehydratase catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate, via the formation of 2-isopropylmaleate...; cd01577;substrate binding site [chemical binding]
GOHSU17	CDS	GOHSU_17_00470	complement(44545..45993)		3-isopropylmalate dehydratase large subunit [hamap]	leuC [hamap]	MF_01026:3-isopropylmalate dehydratase large subunit:leuC 	RER_24080 :3-isopropylmalate dehydratase large subunit : leuC : 4.2.1.33 : Rhodococcus erythropolis PR4	3-isopropylmalate dehydratase large subunit		F1YEZ9_9ACTO 3-isopropylmalate dehydratase large su bunit 1 OS=Gordonia neofelifaecis NRRL B-59395 GN=leuC1 PE =3 SV=1;3-isopropylmalate dehydratase large subunit [Strept omyces clavuligerus ATCC 27064];3-isopropylmalate dehydratase large subunit; Reviewed;3-isopropylmalate dehydratase catalyzes the isomerization between 2-isopropylmalate and 3-isopropylmalate; cd01583;substrate binding site [chemical binding];ligand binding site [chemical binding]
GOHSU17	CDS	GOHSU_17_00480	46105..46794		putative IclR family transcriptional regulator [nite]			ROP_65490 :putative IclR family transcriptional regulator : : : Rhodococcus opacus B4	IclR family transcriptional regulator		F1YEZ8_9ACTO Transcriptional regulator IclR OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_01350 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transcriptional regulator [Transcription]; COG1414;Helix-turn-helix domains; cl00088;Bacterial transcriptional regulator; pfam01614
GOHSU17	CDS	GOHSU_17_00490	46835..47332		conserved hypothetical protein [nite]			RER_24060 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	FMN-binding pyridoxamine 5'-phosphate oxidase-like protein		H0QZY5_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_053_00160 PE=4 SV=1;FMN-binding pyridoxamine 5'-phosphate oxidase-like protein [Streptomyces violaceusniger Tu 4113];PFAM: Pyridoxamine 5'-phosphate oxidase-like, FMN-binding domain; KEGG: sgr:SGR_6175  pyridoxamine 5'-phosphate oxidase-related protein;Pyridoxine 5'-phosphate (PNP) oxidase-like proteins; cl00381
##GOHSU18	contig00071				177930
GOHSU18	CDS	GOHSU_18_00010	complement(481..1143)		hypothetical protein [migap]				hypothetical protein		G7H590_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_064_00170 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mab:MAB_2560 hypothetical protein;LGFP repeat; pfam08310
GOHSU18	CDS	GOHSU_18_00020	complement(1253..2119)		conserved hypothetical protein [nite]			RER_44670 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	universal stress protein		F1YPP8_9ACTO UspA domain-containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_19527 PE=4 SV=1;universal stress protein [Rhodococcus jostii RHA1];Usp: Universal stress protein family. The universal stress protein Usp is a small cytoplasmic bacterial protein whose expression is enhanced when the cell is exposed to stress agents. Usp enhances the rate of cell survival during prolonged exposure to...; cd00293;Ligand Binding Site [chemical binding]
GOHSU18	CDS	GOHSU_18_00030	complement(2232..2939)		hypothetical protein [migap]				hypothetical protein		D0LEP1_GORB4 Putative uncharacterized protein (Prec ursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43 247 / JCM 3198 / NCTC 10667) GN=Gbro_2524 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein;LGFP repeat; pfam08310
GOHSU18	CDS	GOHSU_18_00040	3117..3632		not assigned						F1YP13_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18377 PE=4 SV=1
GOHSU18	CDS	GOHSU_18_00050	3698..4981		conserved hypothetical protein [nite]	 yuaG [nite]		BBR47_48420 :conserved hypothetical protein : yuaG : : Brevibacillus brevis NBRC 100599	hypothetical protein		F1YP14_9ACTO Band 7 protein OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_18382 PE=4 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];PFAM: band 7 protein; KEGG: sco:SCO3607 putative secreted protein;Band_7_flotillin: a subgroup of the band 7 domain of flotillin (reggie) like proteins. This subgroup contains proteins similar to stomatin, prohibitin, flotillin, HlfK/C and podicin.  These two proteins are lipid raft-associated.  Individual proteins of...; cd03399
GOHSU18	CDS	GOHSU_18_00060	5651..6055		putative NADH-dependent flavin oxidoreductase YqiG [migap]	yqiG [migap]			putative NADH-dependent flavin oxidoreductase YqiG	yqiG	H0RLQ9_9ACTO Putative oxidoreductase OS=Gordonia po lyisoprenivorans NBRC 16320 GN=GOPIP_091_00030 PE=4 SV=1;yqiG gene product [Gordonia polyisoprenivorans VH2] ;Old yellow enzyme (OYE)-like FMN binding domain. OYE was the first flavin-dependent enzyme identified, however its true physiological role remains elusive to this day.  Each monomer of OYE contains FMN as a non-covalently bound cofactor, uses NADPH as a...; cd02803;FMN binding site [chemical binding]
GOHSU18	CDS	GOHSU_18_00070	complement(6102..6386)		predicted protein [migap]				predicted protein		F3NFI6_9ACTO Integral membrane protein OS=Streptomy ces griseoaurantiacus M045 GN=SGM_1885 PE=4 SV=1;predicted protein [Streptomyces sp. SPB78]
GOHSU18	CDS	GOHSU_18_00080	6598..7971		major facilitator superfamily transporter [nite]			OBV_44490 :major facilitator superfamily transporter : : : Oscillibacter valericigenes Sjm18-20	major facilitator superfamily protein		D5UIF0_CELFN Major facilitator superfamily MFS_1 OS =Cellulomonas flavigena (strain ATCC 482 / DSM 20109 / NCI B 8073 / NRS 134) GN=Cfla_0536 PE=4 SV=1;unnamed protein product [Sphaerochaeta globus str. Buddy];PFAM: Major facilitator superfamily MFS-1; KEGG: elm:ELI_1208 macrolide-efflux protein;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;H+ Antiporter protein; TIGR00900;putative substrate translocation pore
GOHSU18	CDS	GOHSU_18_00090	complement(8032..8997)		hypothetical protein [nite]			ROP_16270 :hypothetical protein : : : Rhodococcus opacus B4	pirin		D5UXX4_TSUPD Pirin domain protein OS=Tsukamurella p aurometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / N BRC 16120 / NCTC 13040) GN=Tpau_3633 PE=3 SV=1;pirin [Streptomyces flavogriseus ATCC 33331];PFAM: Pirin domain protein; KEGG: sco:SCO0468 hypothetical protein;Pirin-related protein [General function prediction only];Cupin domain; cl09118;Pirin C-terminal cupin domain; pfam05726
GOHSU18	CDS	GOHSU_18_00100	complement(9356..10354)		GTPase [nite]			ROP_70190 :GTPase : : 3.6.1.- : Rhodococcus opacus B4	arginine/ornithine transport system ATPase		F1YG37_9ACTO LAO/AO transporter ATPase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04861 PE=4 SV=1;arginine/ornithine transport system ATPase [Coryneb acterium glutamicum ATCC 13032];IS fingerprint type 4-5;functions in transport of arginine/ornithine; inner membrane ATPase that cleaves ATP and phosphorylates two periplasmic proteins that function as two distinct transport systems, the AO (arginine and ornithine) and LAO (lysine, arginine, and ornithine) periplasmic binding proteins;membrane ATPase/protein kinase; Provisional;The function of this protein family is unkown. The protein sequences are similar to the ArgK protein in E. coli. ArgK protein is a membrane ATPase which is required for transporting arginine, ornithine and lysine into the cells by the arginine and...; cd03114;Walker A
GOHSU18	CDS	GOHSU_18_00110	complement(10416..12686)		methylmalonyl-CoA mutase large subunit [nite]	 mutB [nite]		ROP_70180 :methylmalonyl-CoA mutase large subunit : mutB : 5.4.99.2 : Rhodococcus opacus B4	methylmalonyl-CoA mutase	mcmB	F1YG36_9ACTO Methylmalonyl-CoA mutase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_04856 PE=4 SV=1;methylmalonyl-CoA mutase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;methylmalonyl-CoA mutase; Reviewed;MDM; functions in conversion of succinate to propionate;Coenzyme B12-dependent-methylmalonyl coenzyme A (CoA) mutase (MCM) family, Alpha subunit-like subfamily; contains proteins similar to the alpha subunit of Propionbacterium shermanni MCM, as well as human and E. coli MCM. Members of this subfamily contain...; cd03679;substrate binding site [chemical binding];coenzyme B12 binding site [chemical binding];methylmalonyl CoA mutase B12 binding domain. This domain binds to B12 (adenosylcobamide), which initiates the conversion of succinyl CoA and methylmalonyl CoA by forming an adenosyl radical, which then undergoes a rearrangement exchanging a hydrogen atom...; cd02071;B12 binding site [chemical binding];cobalt ligand [ion binding]
GOHSU18	CDS	GOHSU_18_00120	complement(12693..14594)		methylmalonyl-CoA mutase small subunit [nite]	 mutA [nite]		ROP_70170 :methylmalonyl-CoA mutase small subunit : mutA : 5.4.99.2 : Rhodococcus opacus B4	methylmalonyl-CoA mutase small subunit MutA	mutA	H5TWC6_9ACTO Methylmalonyl-CoA mutase small subunit OS=Gordonia sputi NBRC 100414 GN=mutA PE=4 SV=1;mutA gene product [Gordonia polyisoprenivorans VH2] ;methylmalonyl-CoA mutase, heterodimeric type, beta chain; TIGR00642;Coenzyme B12-dependent-methylmalonyl coenzyme A (CoA) mutase (MCM) family, Beta subunit-like subfamily; contains bacterial proteins similar to the beta subunit of MCMs from Propionbacterium shermanni and Streptomyces cinnamonensis, which are alpha/beta...; cd03677;heterodimer interface [polypeptide binding];substrate interaction site [chemical binding];B12 binding domain (B12-BD). Most of the members bind different cobalamid derivates, like B12 (adenosylcobamide) or methylcobalamin or methyl-Co(III) 5-hydroxybenzimidazolylcobamide. This domain is found in several enzymes, such as glutamate mutase; cl00293
GOHSU18	CDS	GOHSU_18_00130	14681..15475		hypothetical membrane protein [nite]			ROP_70150 :hypothetical membrane protein : : : Rhodococcus opacus B4	DedA family protein		F1YG34_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04846 PE=4 SV=1;DedA family protein [Clostridium botulinum A3 str. Loch Maree];identified by match to protein family HMM PF09335;SNARE associated Golgi protein; cl00429
GOHSU18	CDS	GOHSU_18_00140	15601..16185		hypothetical protein [nite]			RER_30860 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0LDN6_GORB4 Putative uncharacterized protein (Prec ursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43 247 / JCM 3198 / NCTC 10667) GN=Gbro_2417 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];KEGG: lxx:Lxx06130 hypothetical protein
GOHSU18	CDS	GOHSU_18_00150	16182..16742		hypothetical protein [migap]				hypothetical protein		F1YG32_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04836 PE=4 SV=1;protein [Clavibacter michiganensis subsp. sepedonic us]
GOHSU18	CDS	GOHSU_18_00160	complement(16782..17612)		hypothetical protein [nite]			ROP_70050 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YG30_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04826 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3097); pfam11296
GOHSU18	CDS	GOHSU_18_00170	17678..18478		hypothetical protein [nite]			ROP_70040 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YG31_9ACTO Serine/threonine protein kinase OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_04831 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU18	CDS	GOHSU_18_00180	complement(18507..19580)		ferrochelatase [hamap]	hemH [hamap]	MF_00323:ferrochelatase:hemH 	RER_30760 :ferrochelatase : hemH : 4.99.1.1 : Rhodococcus erythropolis PR4	ferrochelatase	hemH	F1YG29_9ACTO Ferrochelatase OS=Gordonia neofelifaec is NRRL B-59395 GN=hemH PE=3 SV=1;ferrochelatase [Corynebacterium glutamicum ATCC 130 32];IS fingerprint type 4-5;protoheme ferro-lyase; catalyzes the insertion of a ferrous ion into protoporphyrin IX to form protoheme; involved in protoheme biosynthesis; in some organisms this protein is membrane-associated while in others it is cytosolic;ferrochelatase; Reviewed; PRK00035;Ferrochelatase, N-terminal domain: Ferrochelatase (protoheme ferrolyase or HemH) is the terminal enzyme of the heme biosynthetic pathway. It catalyzes the insertion of ferrous iron into the protoporphyrin IX ring yielding protoheme. This enzyme is...; cd03411;C-terminal domain interface [polypeptide binding];Ferrochelatase, C-terminal domain: Ferrochelatase (protoheme ferrolyase or HemH) is the terminal enzyme of the heme biosynthetic pathway. It catalyzes the insertion of ferrous iron into the protoporphyrin IX ring yielding protoheme. This enzyme is...; cd00419;N-terminal domain interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00190	complement(19585..20418)		enoyl-[acyl-carrier-protein] reductase [nite]	 fabI [nite]		ROP_70020 :enoyl-[acyl-carrier-protein] reductase : fabI : 1.3.1.9 : Rhodococcus opacus B4	enoyl-ACP reductase	fabI2	H5TNH4_9ACTO Enoyl-[acyl-carrier-protein] reductase [NADH] OS=Gordonia otitidis NBRC 100426 GN=fabI PE=3 SV=1 ;enoyl-ACP reductase [Streptomyces avermitilis MA- 4 680];enoyl-(acyl carrier protein) reductase; Provisional;Catalyzes a key regulatory step in fatty acid biosynthesis;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU18	CDS	GOHSU_18_00200	complement(20468..21196)		3-oxoacyl-[acyl-carrier-protein] reductase [nite]	 fabG [nite]		ROP_70010 :3-oxoacyl-[acyl-carrier-protein] reductase : fabG : 1.1.1.100 : Rhodococcus opacus B4	3-oxoacyl-ACP reductase	fabG1	H6N1Q1_GORPV 3-oxoacyl-[acyl-carrier-protein] reduc tase FabG OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=fabG1 PE=3 SV=1;fabG1 gene product [Gordonia polyisoprenivorans VH2 ];beta-Keto acyl carrier protein reductase (BKR), involved in Type II FAS, classical (c) SDRs; cd05333;3-oxoacyl-(acyl-carrier-protein) reductase; TIGR01830;NAD(P) binding site [chemical binding];homotetramer interface [polypeptide binding];homodimer interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00210	complement(21213..22205)		conserved hypothetical membrane protein [nite]			RER_30730 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	von Willebrand factor type A		F1YG26_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04806 PE=4 SV=1;von Willebrand factor type A [Meiothermus ruber DSM 1279];PFAM: von Willebrand factor type A; SMART: von Willebrand factor type A; InterPro IPR002035; KEGG: azo:azo2864 hypothetical protein; SPTR: A2SQ24 von Willebrand factor, type A; PFAM: von Willebrand factor type A domain;Von Willebrand factor type A (vWA) domain was originally found in the blood coagulation protein von Willebrand factor (vWF). Typically, the vWA domain is made up of approximately 200 amino acid residues folded into a classic a/b para-rossmann type of...; cl00057;metal ion-dependent adhesion site (MIDAS)
GOHSU18	CDS	GOHSU_18_00220	complement(22205..23137)		conserved hypothetical protein [nite]			RER_30720 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TWB5_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_022_01170 PE=4 SV=1;unnamed protein product [Salinispora tropica CNB- 4 40];PFAM: protein of unknown function DUF58;Uncharacterized conserved protein (some members contain a von Willebrand factor type A (vWA) domain) [General function prediction only];Von Willebrand factor type A (vWA) domain was originally found in the blood coagulation protein von Willebrand factor (vWF). Typically, the vWA domain is made up of approximately 200 amino acid residues folded into a classic a/b para-rossmann type of...; cl00057
GOHSU18	CDS	GOHSU_18_00230	complement(23124..24287)		conserved hypothetical protein [nite]			RER_30710 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	ATPase		F1YG24_9ACTO ATPase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04796 PE=4 SV=1;ATPase [Pseudonocardia dioxanivorans CB1190];PFAM: ATPase, AAA-3; KEGG: amd:AMED_4951 MoxR-like ATPase;MoxR-like ATPases [General function prediction only];The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger
GOHSU18	CDS	GOHSU_18_00240	complement(24402..25847)		NLP/P60 protein [migap]				NLP/P60 protein		F1YG23_9ACTO NLP/P60 protein OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_04791 PE=4 SV=1;NLP/P60 protein [Mycobacterium sp. JLS];PFAM: NLP/P60 protein; KEGG: mmc:Mmcs_1440 NLP/P60;Cell wall-associated hydrolases (invasion-associated proteins) [Cell envelope biogenesis, outer membrane]; COG0791;NlpC/P60 family; cl11438
GOHSU18	CDS	GOHSU_18_00250	complement(26126..26749)		hypothetical protein [migap]				hypothetical protein		F1YGC6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04786 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU18	CDS	GOHSU_18_00260	27052..29847		aconitase [nite]	 acn [nite]		RER_30680 :aconitase : acn : 4.2.1.3 : Rhodococcus erythropolis PR4	aconitate hydratase	acn	G7H5M0_9ACTO Aconitase OS=Gordonia araii NBRC 10043 3 GN=acn PE=4 SV=1;aconitate hydratase [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;Catalyzes the conversion of citrate to isocitrate;aconitate hydratase; Validated;Aconitase catalytic domain; Aconitase catalyzes the reversible isomerization of citrate and isocitrate as part of the TCA cycle; cl00285;substrate binding site [chemical binding];ligand binding site [chemical binding];Aconitase A swivel domain. This is the major form of the TCA cycle enzyme aconitate hydratase, also known as aconitase and citrate hydro-lyase. It includes bacterial and archaeal aconitase A, and the eukaryotic cytosolic form of aconitase. This group...; cd01580
GOHSU18	CDS	GOHSU_18_00270	29879..30454		putative TetR family transcriptional regulator [nite]			ROP_69940 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	regulatory protein TetR		F1YGC4_9ACTO TetR family transcriptional regulator OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04776 PE=4 SV=1;unnamed protein product [Granulicella mallensis MP5 ACTX8];KEGG: sco:SCO4303 transcriptional regulator; manually curated; PFAM: Transcription regulator, TetR-like, DNA-binding, bacterial/archaeal;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU18	CDS	GOHSU_18_00280	complement(30563..32122)		putative ABC transporter ATP-binding protein [nite]			ROP_69920 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	putative ABC transporter ATP binding protein		F1YGB6_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_04726 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];ATPase components of ABC transporters with duplicated ATPase domains [General function prediction only]; COG0488;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region;ABCF_EF-3  Elongation factor 3 (EF-3) is a cytosolic protein required by fungal ribosomes for in vitro protein synthesis and for in vivo growth.  EF-3 stimulates the binding of the EF-1: GTP: aa-tRNA ternary complex to the ribosomal A site by facilitated...; cd03221
GOHSU18	CDS	GOHSU_18_00290	32309..33082		putative oxidoreductase [nite]			RER_30650 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short-chain dehydrogenase/reductase SDR		F1YGB5_9ACTO 3-ketoacyl-(Acyl-carrier-protein) redu ctase OS=Gordonia neofelifaecis NRRL B-59395 GN=fabG PE=4 SV=1;short-chain dehydrogenase/reductase SDR [Streptomyc es violaceusniger Tu 4113];PFAM: Short-chain dehydrogenase/reductase SDR; KEGG: sco:SCO1345 3-ketoacyl-(acyl-carrier-protein) reductase;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU18	CDS	GOHSU_18_00300	33079..34260		lycopene beta-cyclase [nite]	 crtL [nite]		RER_30640 :lycopene beta-cyclase : crtL : 1.14.-.- : Rhodococcus erythropolis PR4	lycopene cyclase		G7GIT4_9ACTO Lycopene beta-cyclase OS=Gordonia amar ae NBRC 15530 GN=crtL PE=4 SV=1;lycopene cyclase [Nocardia farcinica IFM 10152];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU18	CDS	GOHSU_18_00310	complement(34335..34706)		thioredoxin [nite]	 trxA [nite]		ROP_07180 :thioredoxin : trxA : : Rhodococcus opacus B4	thioredoxin		H0R603_9ACTO Thioredoxin OS=Gordonia effusa NBRC 10 0432 GN=trxA PE=3 SV=1;thioredoxin [Rhodococcus jostii RHA1];TRX family; composed of two groups: Group I, which includes proteins that exclusively encode a TRX domain; and Group II, which are composed of fusion proteins of TRX and additional domains. Group I TRX is a small ancient protein that alter the redox...; cd02947;catalytic residues [active]
GOHSU18	CDS	GOHSU_18_00320	complement(34775..35143)		hypothetical protein [nite]			ROP_69890 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		D0LDK7_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_2388 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function DUF59; cl00941
GOHSU18	CDS	GOHSU_18_00330	complement(35189..35608)		putative iron-sulfur cluster assembly protein IscU [nite]	 iscU [nite]		ROP_69880 :putative iron-sulfur cluster assembly protein IscU : iscU : : Rhodococcus opacus B4	SUF system FeS assembly protein		H5TU38_9ACTO Iron-sulfur cluster assembly protein I scU OS=Gordonia otitidis NBRC 100426 GN=iscU PE=4 SV=1;SUF system FeS assembly protein [Streptomyces clavu ligerus ATCC 27064];NifU homolog involved in Fe-S cluster formation [Energy production and conversion]; COG0822;Iron-sulfur cluster scaffold-like proteins; cd06664;trimerization site [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00340	complement(35626..36906)		probable cysteine desulfurase/selenocysteine lyase [nite]	 sufS [nite]		RER_30580 :probable cysteine desulfurase/selenocysteine lyase : sufS : 2.8.1.7/4.4.1.16 : Rhodococcus erythropolis PR4	SufS subfamily cysteine desulfurase		F1YGA3_9ACTO Cysteine desulfurase OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_04661 PE=3 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rop:ROP_69870 cysteine desulfurase/selenocysteine lyase; TIGRFAM: cysteine desulfurase, SufS subfamily; PFAM: aminotransferase class V;Selenocysteine lyase/Cysteine desulfurase [Posttranslational modification, protein turnover, chaperones]; COG0520;Cysteine desulfurase (SufS)-like. This family belongs to the pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). The major groups in this CD correspond to cysteine desulfurase (SufS) and selenocysteine lyase. SufS...; cd06453;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU18	CDS	GOHSU_18_00350	complement(36911..37600)		putative iron-sulfur cluster assembly protein SufC [nite]	 sufC [nite]		ROP_69860 :putative iron-sulfur cluster assembly protein SufC : sufC : : Rhodococcus opacus B4	ABC transporter ATP-binding subunit		F1YGA2_9ACTO FeS assembly ATPase SufC OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_04656 PE=4 SV=1;ABC transporter ATP-binding subunit [Streptomyces c lavuligerus ATCC 27064];ABC-type transport system involved in Fe-S cluster assembly, ATPase component.  Biosynthesis of iron-sulfur clusters (Fe-S) depends on multiprotein systems.  The SUF system of E. coli and Erwinia chrysanthemi is important for Fe-S biogenesis under...; cd03217;FeS assembly ATPase SufC; TIGR01978;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU18	CDS	GOHSU_18_00360	complement(37716..38909)		putative iron-sulfur cluster assembly protein SufD [nite]	 sufD [nite]		ROP_69850 :putative iron-sulfur cluster assembly protein SufD : sufD : : Rhodococcus opacus B4	FeS assembly protein SufD	sufD	F1YGA1_9ACTO FeS assembly protein SufD OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04651 PE=4 SV=1;sufD gene product [Gordonia polyisoprenivorans VH2] ;Uncharacterized protein family (UPF0051); cl03223
GOHSU18	CDS	GOHSU_18_00370	complement(38909..40360)		putative iron-sulfur cluster assembly protein SufB [nite]	 sufB [nite]		RER_30550 :putative iron-sulfur cluster assembly protein SufB : sufB : : Rhodococcus erythropolis PR4	FeS assembly protein SufB		F1YGH2_9ACTO FeS assembly protein SufB OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04646 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: FeS assembly protein SufB; PFAM: SufBD protein; KEGG: afw:Anae109_0903 cysteine desulfurase activator complex subunit SufB;Uncharacterized protein family (UPF0051); cl03223;FeS assembly protein SufB; TIGR01980
GOHSU18	CDS	GOHSU_18_00380	complement(40357..41157)		putative SufR family transcriptional regulator [nite]			ROP_69830 :putative SufR family transcriptional regulator : : : Rhodococcus opacus B4	transcriptional regulator		F1YGH1_9ACTO Regulatory protein ArsR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_04641 PE=4 SV=1;transcriptional regulator [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Predicted transcriptional regulator [Transcription];Arsenical Resistance Operon Repressor and similar prokaryotic, metal regulated homodimeric repressors. ARSR subfamily of helix-turn-helix bacterial transcription regulatory proteins (winged helix topology). Includes several proteins that appear to...; cd00090;putative DNA binding site [nucleotide binding];putative Zn2+ binding site [ion binding]
GOHSU18	CDS	GOHSU_18_00390	41297..43057		mannosyltransferase MptB [nite]	 mptB [nite]		RER_30530 :mannosyltransferase MptB : mptB : 2.4.1.- : Rhodococcus erythropolis PR4	hypothetical protein		F1YGH0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04636 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: dat:HRM2_13880 hypothetical protein;carotene biosynthesis associated membrane protein; TIGR03459
GOHSU18	CDS	GOHSU_18_00400	43100..44059		putative ABC transporter ATP-binding protein [nite]			RER_30520 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter ATP-binding protein		F1YGG9_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_04631 PE=3 SV=1;ABC transporter ATP-binding protein [Rhodococcus er ythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;ABC-type multidrug transport system, ATPase component [Defense mechanisms]; COG1131;This family of ATP-binding proteins belongs to a multisubunit transporter involved in drug resistance (BcrA and DrrA), nodulation, lipid transport, and lantibiotic immunity.  In bacteria and archaea, these transporters usually include an ATP-binding...; cd03230;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU18	CDS	GOHSU_18_00410	44056..44865		putative ABC transporter permease protein [nite]			RER_30510 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	ABC transporter permease protein		F1YGG8_9ACTO Putative ABC transporter membrane prot ein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04626 P E=4 SV=1;ABC transporter permease protein [Streptomyces clav uligerus ATCC 27064];ABC-2 type transporter; cl11417
GOHSU18	CDS	GOHSU_18_00420	44862..45842		putative cytochrome oxidase assembly protein [nite]			MLP_14430 :putative cytochrome oxidase assembly protein : : : Microlunatus phosphovorus NM-1	cytochrome oxidase assembly		F1YGG7_9ACTO Cytochrome oxidase assembly OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_04621 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: cytochrome oxidase assembly; KEGG: bte:BTH_I0434 cytochrome C assembly family protein;Uncharacterized protein required for cytochrome oxidase assembly [Posttranslational modification, protein turnover, chaperones]; COG1612;Cytochrome oxidase assembly protein; cl15852
GOHSU18	CDS	GOHSU_18_00430	complement(45964..46347)		hypothetical protein [nite]			KSE_55530 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		G7H0N9_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_036_01460 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU18	CDS	GOHSU_18_00440	46380..47468		NADPH--quinone reductase [nite]	 qor [nite]		ROP_69690 :NADPH--quinone reductase : qor : 1.6.5.5 : Rhodococcus opacus B4	NADPH--quinone reductase	qor	H6N1J8_GORPV Quinone oxidoreductase Qor OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=qor PE=4 S V=1;NADPH--quinone reductase [Rhodococcus opacus B4];protein synonym:quinone oxidoreductase;Quinone oxidoreductase (QOR); cd05286;quinone oxidoreductase, NADPH-dependent; Provisional;NADP binding site [chemical binding];dimer interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00450	complement(47549..48535)	case	protoheme IX farnesyltransferase [hamap]	cyoE [hamap]	MF_00154:protoheme IX farnesyltransferase:cyoE 	RER_30460 :putative protoheme IX farnesyltransferase : : 2.5.1.- : Rhodococcus erythropolis PR4	protoheme IX farnesyltransferase	ctaB	F1YGG2_9ACTO Protoheme IX farnesyltransferase OS=Go rdonia neofelifaecis NRRL B-59395 GN=ctaB PE=3 SV=1;protoheme IX farnesyltransferase [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;converts protoheme IX and farnesyl diphosphate to heme O;UbiA prenyltransferase family; cl00337
GOHSU18	CDS	GOHSU_18_00460	48806..50902		transketolase [nite]	 tkt [nite]		ROP_69670 :transketolase : tkt : 2.2.1.1 : Rhodococcus opacus B4	transketolase		F1YGG1_9ACTO Transketolase OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_04591 PE=3 SV=1;transketolase [Nakamurella multipartita DSM 44233];catalyzes the formation of ribose 5-phosphate and xylulose 5-phosphate from sedoheptulose 7-phosphate and glyceraldehyde 3-phosphate; can transfer ketol groups between several groups; in Escherichia coli there are two tkt genes, tktA expressed during exponential growth and the tktB during stationary phase;transketolase; Reviewed;Thiamine pyrophosphate (TPP) family, Transketolase (TK) subfamily, TPP-binding module; TK catalyzes the transfer of a two-carbon unit from ketose phosphates to aldose phosphates. In heterotrophic organisms, TK provides a link between glycolysis and the...; cd02012;TPP-binding site [chemical binding];dimer interface [polypeptide binding];Pyrimidine (PYR) binding domain of 1-deoxy-D-xylulose-5-phosphate synthase (DXS), transketolase (TK), and related proteins; cd07033;PYR/PP interface [polypeptide binding];TPP binding site [chemical binding];Transketolase, C-terminal domain; pfam02780
GOHSU18	CDS	GOHSU_18_00470	50972..52090		transaldolase [hamap]	tal [hamap]	MF_00493:transaldolase:tal 	RER_30440 :probable transaldolase : tal : 2.2.1.2 : Rhodococcus erythropolis PR4	transaldolase	tal	F1YGG0_9ACTO Transaldolase OS=Gordonia neofelifaeci s NRRL B-59395 GN=tal PE=3 SV=1;tal gene product [Mycobacterium smegmatis str. MC2 155];catalyzes the reversible formation of D-erythrose 4-phosphate and D-fructose 6-phosphate from sedoheptulose 7-phosphate and D-glyceraldehyde 3-phosphate;Transaldolase-like proteins from plants and bacteria; cd00955;putative active site [active];transaldolase; Provisional;catalytic residue [active]
GOHSU18	CDS	GOHSU_18_00480	52101..53645		glucose-6-phosphate 1-dehydrogenase [nite]	 zwf [nite]		RER_30430 :glucose-6-phosphate 1-dehydrogenase : zwf : 1.1.1.49 : Rhodococcus erythropolis PR4	glucose-6-phosphate 1-dehydrogenase	zwf	H5TP06_9ACTO Glucose-6-phosphate 1-dehydrogenase OS =Gordonia otitidis NBRC 100426 GN=zwf PE=3 SV=1;glucose-6-phosphate 1-dehydrogenase [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of D-glucono-1,5-lactone 6-phosphate from D-glucose 6-phosphate;glucose-6-phosphate 1-dehydrogenase; Validated;Glucose-6-phosphate dehydrogenase, NAD binding domain; pfam00479;Glucose-6-phosphate dehydrogenase, C-terminal domain; pfam02781
GOHSU18	CDS	GOHSU_18_00490	53656..54573		putative OxPP cycle protein OpcA [nite]	 opcA [nite]		ROP_69640 :putative OxPP cycle protein OpcA : opcA : : Rhodococcus opacus B4	oxppcycle protein		F1YGF8_9ACTO Glucose-6-phosphate dehydrogenase subu nit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04576 P E=4 SV=1;oxppcycle protein [Streptomyces sp. Mg1];opcA protein; TIGR00534;Glucose-6-phosphate dehydrogenase subunit; cl01359
GOHSU18	CDS	GOHSU_18_00500	54570..55307		6-phosphogluconolactonase [nite]	 pgl [nite]		RER_30410 :6-phosphogluconolactonase : pgl : 3.1.1.31 : Rhodococcus erythropolis PR4	6-phosphogluconolactonase	pgl	H5TP04_9ACTO 6-phosphogluconolactonase OS=Gordonia otitidis NBRC 100426 GN=pgl PE=4 SV=1;pgl gene product [Gordonia polyisoprenivorans VH2];6PGL: 6-Phosphogluconolactonase (6PGL) subfamily; 6PGL catalyzes the second step of the oxidative phase of the pentose phosphate pathway, the hydrolyzation of 6-phosphoglucono-1,5-lactone (delta form) to 6-phosphogluconate. 6PGL is thought to guard...; cd01400;putative active site [active]
GOHSU18	CDS	GOHSU_18_00510	complement(55330..57003)		putative solute/hydrogen antiporter [nite]			MLP_24880 :putative solute/hydrogen antiporter : : : Microlunatus phosphovorus NM-1	sodium/hydrogen exchanger		B1VH02_CORU7 Putative potassium-efflux system prote in OS=Corynebacterium urealyticum (strain ATCC 43042 / DSM 7109) GN=kefB PE=4 SV=1;unnamed protein product [Magnetococcus sp. MC- 1];PFAM: TrkA-N domain protein; sodium/hydrogen exchanger; KEGG: pha:PSHAa0856 glutathione-regulated potassium-efflux system protein ( K(+)/H(+) antiporter);Sodium/hydrogen exchanger family; pfam00999;Kef-type K+ transport systems, membrane components [Inorganic ion transport and metabolism]; cl10482;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU18	CDS	GOHSU_18_00520	complement(57082..57900)		hypothetical protein [migap]				hypothetical protein		H5U3U0_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_099_00470 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GA21679 gene product from transcript GA21679- RA
GOHSU18	CDS	GOHSU_18_00530	complement(57999..60827)		phosphoenolpyruvate carboxylase [hamap]	ppc [hamap]	MF_00595:phosphoenolpyruvate carboxylase:ppc 	ROP_69620 :phosphoenolpyruvate carboxylase : ppc : 4.1.1.31 : Rhodococcus opacus B4	phosphoenolpyruvate carboxylase	ppc	F1YGF0_9ACTO Phosphoenolpyruvate carboxylase OS=Gor donia neofelifaecis NRRL B-59395 GN=ppc PE=3 SV=1;phosphoenolpyruvate carboxylase [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;Phosphoenolpyruvate carboxylase; cl14656;Phosphoenolpyruvate carboxylase [Energy production and conversion]; COG2352;catalyzes the formation of oxaloacetate from phosphoenolpyruvate
GOHSU18	CDS	GOHSU_18_00540	complement(60879..61877)		putative ABC transporter substrate-binding protein [nite]			ROP_08900 :putative ABC transporter substrate-binding protein : : : Rhodococcus opacus B4	glycine betaine ABC transporter substrate-binding protein		H5TJS4_9ACTO Putative ABC transporter substrate bin ding protein OS=Gordonia otitidis NBRC 100426 GN=GOOTI_078 _00100 PE=4 SV=1;glycine betaine ABC transporter substrate-binding p rotein [Chromohalobacter salexigens DSM 3043];type strain of Chromohalobacter salexigens;NMT1-like family; cl15260;Substrate binding domain of ABC-type glycine betaine transport system; pfam04069
GOHSU18	CDS	GOHSU_18_00550	complement(61874..62638)		putative ABC transporter permease protein [nite]			RER_49060 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	OpuCB protein		F9W2S1_9ACTO Putative ABC transporter permease prot ein OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_125_00050 PE=3 SV=1;unnamed protein product [Bacillus atrophaeus 1942];COG1174 ABC-type proline/glycine betaine transport systems, permease component;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU18	CDS	GOHSU_18_00560	complement(62635..63837)		putative ABC transporter ATP-binding protein [nite]			RER_49050 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter ATP-binding protein		H0QVA6_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia effusa NBRC 100432 GN=GOEFS_013_00210 P E=3 SV=1;ABC transporter ATP-binding protein [Nocardia farci nica IFM 10152];ABC-type proline/glycine betaine transport systems, ATPase components [Amino acid transport and metabolism]; COG1125;OpuCA is a the ATP binding component of a bacterial solute transporter that serves a protective role to cells growing in a hyperosmolar environment.  ABC (ATP-binding cassette) transporter nucleotide-binding domain; ABC transporters are a large family of...; cd03295;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region;The CBS domain, named after human CBS, is a small domain originally identified in cystathionine beta-synthase and is subsequently found in a wide range of different proteins. CBS domains usually occur in tandem repeats. They associate to form a so-called...; cd02205
GOHSU18	CDS	GOHSU_18_00570	complement(63834..64478)		putative ABC transporter permease protein [nite]			ROP_08870 :putative ABC transporter permease protein : : : Rhodococcus opacus B4	binding-protein-dependent transporter inner membrane component		D0L746_GORB4 Binding-protein-dependent transport sy stems inner membrane component OS=Gordonia bronchialis (st rain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gb ro_1556 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cl00427;PFAM: binding-protein-dependent transporters inner membrane component; KEGG: spe:Spro_1446 binding-protein-dependent transport systems inner membrane component
GOHSU18	CDS	GOHSU_18_00580	complement(64759..64992)		protein-export membrane protein SecG [nite]	 secG [nite]		RER_30330 :protein-export membrane protein SecG : secG : : Rhodococcus erythropolis PR4	preprotein translocase subunit SecG	secG	G7CFZ8_MYCTH Preprotein translocase subunit SecG OS =Mycobacterium thermoresistibile ATCC 19527 GN=KEK_09597 P E=4 SV=1;preprotein translocase subunit SecG [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;Preprotein translocase SecG subunit; cl09123
GOHSU18	CDS	GOHSU_18_00590	complement(65081..65890)		triose-phosphate isomerase [nite]	 tpiA [nite]		RER_30310 :triose-phosphate isomerase : tpiA : 5.3.1.1 : Rhodococcus erythropolis PR4	triosephosphate isomerase	tpiA	F1YGE8_9ACTO Triosephosphate isomerase OS=Gordonia neofelifaecis NRRL B-59395 GN=tpiA PE=3 SV=1;triosephosphate isomerase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Reversibly isomerizes the ketone sugar dihydroxyacetone phosphate to the aldehyde sugar glyceraldehyde-3-phosphate;Triosephosphate isomerase (TIM) is a glycolytic enzyme that catalyzes the interconversion of dihydroxyacetone phosphate and D-glyceraldehyde-3-phosphate. The reaction is very efficient and requires neither cofactors nor metal ions. TIM, usually...; cd00311;substrate binding site [chemical binding];dimer interface [polypeptide binding];catalytic triad [active]
GOHSU18	CDS	GOHSU_18_00600	complement(65894..67111)		phosphoglycerate kinase [hamap]	pgk [hamap]	MF_00145:phosphoglycerate kinase:pgk 	RER_30300 :phosphoglycerate kinase : pgk : 2.7.2.3 : Rhodococcus erythropolis PR4	phosphoglycerate kinase	pgk	F1YGE7_9ACTO Phosphoglycerate kinase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=pgk PE=3 SV=1;phosphoglycerate kinase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Converts 3-phospho-D-glycerate to 3-phospho-D-glyceroyl phosphate during the glycolysis pathway;Phosphoglycerate kinase; pfam00162;Phosphoglycerate kinase (PGK) is a monomeric enzyme which catalyzes the transfer of the high-energy phosphate group of 1,3-bisphosphoglycerate to ADP, forming ATP and 3-phosphoglycerate. This reaction represents the first of the two substrate-level...; cd00318;substrate binding site [chemical binding];hinge regions;ADP binding site [chemical binding];catalytic site [active]
GOHSU18	CDS	GOHSU_18_00610	complement(67114..68130)		glyceraldehyde-3-phosphate dehydrogenase [nite]	 gap [nite]		ROP_69580 :glyceraldehyde-3-phosphate dehydrogenase : gap : 1.2.1.12 : Rhodococcus opacus B4	glyceraldehyde-3-phosphate dehydrogenase, type I		D0LCT4_GORB4 Glyceraldehyde-3-phosphate dehydrogena se, type I OS=Gordonia bronchialis (strain ATCC 25592 / DS M 43247 / JCM 3198 / NCTC 10667) GN=Gbro_2363 PE=3 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Glyceraldehyde 3-phosphate dehydrogenase, C-terminal domain; Glyceraldehyde 3-phosphate dehydrogenase, NAD binding domain; TIGRFAM: glyceraldehyde-3-phosphate dehydrogenase, type I;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;glyceraldehyde-3-phosphate dehydrogenase, type I; TIGR01534;Glyceraldehyde 3-phosphate dehydrogenase, C-terminal domain; cl15856
GOHSU18	CDS	GOHSU_18_00620	complement(68239..69222)		putative sporulation transcription regulator WhiA [hamap]	whiA [hamap]	MF_01420:putative sporulation transcription regulator WhiA:whiA 	RER_30280 :WhiA-like protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YGE5_9ACTO Putative sporulation transcription reg ulator WhiA OS=Gordonia neofelifaecis NRRL B-59395 GN=whiA PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Uncharacterized protein conserved in bacteria [Function unknown];PFAM: protein of unknown function DUF199; sporulation regulator WhiA-like; KEGG: scl:sce5694 cell division protein MukB;Sporulation Regulator WhiA N terminal; pfam10298
GOHSU18	CDS	GOHSU_18_00630	complement(69299..70315)		hypothetical protein [nite]			ROP_69560 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F9VR13_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_017_00360 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];family of mostly uncharacterized proteins similar to B.subtilis YvcK; cd07187;Uncharacterized protein family UPF0052; pfam01933;phosphate binding site [ion binding];putative substrate binding pocket [chemical binding];dimer interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00640	complement(70312..71205)	case	UPF0042 nucleotide-binding protein yhbJ [hamap]	yhbJ [hamap]	MF_00636:UPF0042 nucleotide-binding protein yhbJ:yhbJ 	RER_30260 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative nucleotide-binding protein		F1YGE3_9ACTO GlmZ(SRNA)-inactivating NTPase OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_04501 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];P-loop ATPase protein family; cl15794;Predicted P-loop-containing kinase [General function prediction only]
GOHSU18	CDS	GOHSU_18_00650	complement(71216..73240)		UvrABC system protein C [hamap]	uvrC [hamap]	MF_00203:UvrABC system protein C:uvrC 	RER_30250 :UvrABC system protein C : uvrC : : Rhodococcus erythropolis PR4	excinuclease ABC subunit C	uvrC	F1YGE2_9ACTO UvrABC system protein C OS=Gordonia ne ofelifaecis NRRL B-59395 GN=uvrC PE=3 SV=1;excinuclease ABC subunit C [Corynebacterium glutami cum ATCC 13032];IS fingerprint type 4-5;The UvrABC repair system catalyzes the recognition and processing of DNA lesions. UvrC both incises the 5' and 3' sides of the lesion. The N-terminal half is responsible for the 3' incision and the C-terminal half is responsible for the 5' incision;excinuclease ABC subunit C; Validated; PRK00558;Catalytic GIY-YIG domain of nucleotide excision repair endonucleases UvrC, Cho, and similar proteins; cd10434;GIY-YIG motif/motif A;catalytic site [active];putative DNA binding site [nucleotide binding];metal binding site [ion binding];UvrB/uvrC motif; pfam02151;UvrC Helix-hairpin-helix N-terminal; pfam08459
GOHSU18	CDS	GOHSU_18_00660	complement(73301..73816)		hypothetical membrane protein [nite]			ROP_69530 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YGE1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04491 PE=4 SV=1;unnamed protein product [Mycobacterium tuberculosis KZN 4207];drug-susceptible;Protein of unknown function (DUF2581); pfam10756
GOHSU18	CDS	GOHSU_18_00670	complement(73813..74289)		6,7-dimethyl-8-ribityllumazine synthase [hamap]	ribH [hamap]	MF_00178:6,7-dimethyl-8-ribityllumazine synthase:ribH 	RER_30230 :6,7-dimethyl-8-ribityllumazine synthase : ribH : 2.5.1.9 : Rhodococcus erythropolis PR4	6,7-dimethyl-8-ribityllumazine synthase	ribH	F1YGE0_9ACTO 6,7-dimethyl-8-ribityllumazine synthas e OS=Gordonia neofelifaecis NRRL B-59395 GN=ribH PE=3 SV=1 ;6,7-dimethyl-8-ribityllumazine synthase [Corynebact erium glutamicum ATCC 13032];IS fingerprint type 4-5;RibE; 6,7-diimethyl-8-ribityllumazine synthase; DMRL synthase; lumazine synthase; beta subunit of riboflavin synthase; condenses 5-amino-6-(1'-D)-ribityl-amino-2,4(1H,3H)-pyrimidinedione with L-3,4-dihydrohy-2-butanone-4-phosphate to generate 6,6-dimethyl-8-lumazine (DMRL); riboflavin synthase then uses 2 molecules of DMRL to produce riboflavin (vitamin B12); involved in the last steps of riboflavin biosynthesis; forms a 60mer (icosahedral shell) in both Bacillus subtilis and Escherichia coli; in Bacillus subtilis this 60mer is associated with the riboflavin synthase subunit (alpha) while in Escherichia coli it is not;lumazine synthase (6,7-dimethyl-8-ribityllumazine synthase, LS), catalyzes the penultimate step in the biosynthesis of riboflavin (vitamin B2); type-I; cd09209;homopentamer interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00680	complement(74286..75566)		GTP cyclohydrolase II/3,4-dihydroxy-2-butanone 4-phosphate synthase/riboflavin biosynthesis protein RibBA [hamap]	ribA/ribB/ribBA [hamap]	MF_00179:GTP cyclohydrolase II:ribA MF_00180:3,4-dihydroxy-2-butanone 4-phosphate synthase:ribB MF_01283:riboflavin biosynthesis protein RibBA:ribBA 	RER_30220 :riboflavin biosynthesis bifunctional protein RibA : ribA : 3.5.4.25 : Rhodococcus erythropolis PR4	bifunctional 3,4-dihydroxy-2-butanone 4-phosphate synthase/GTP cyclohydrolase II protein	ribA	G7GIY0_9ACTO Riboflavin biosynthesis protein RibBA OS=Gordonia amarae NBRC 15530 GN=ribBA PE=3 SV=1;bifunctional 3,4-dihydroxy-2-butanone 4-phosphate s ynthase/GTP cyclohydrolase II protein [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;bifunctional enzyme DHBP synthase/GTP cyclohydrolase II; functions in riboflavin synthesis; converts GTP to 2,5-diamino-6-hydroxy-4-(5-phosphoribosylamino)pyrimidine; converts ribulose 5-phopshate to 3,4-dihydroxy-2-butanone 4-phosphate;bifunctional 3,4-dihydroxy-2-butanone 4-phosphate synthase/GTP cyclohydrolase II protein; Provisional;3,4-dihydroxy-2-butanone 4-phosphate synthase; cl00336;GTP cyclohydrolase II (RibA).  GTP cyclohydrolase II catalyzes the conversion of GTP to 2,5-diamino-6-ribosylamino-4(3H)-pyrimidinone 5' phosphate, formate, pyrophosphate (APy), and GMP in the biosynthetic pathway of riboflavin. Riboflavin is the...; cd00641;dimerization interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00690	complement(75553..76194)		riboflavin synthase alpha chain [nite]	 ribB [nite]		ROP_69500 :riboflavin synthase alpha chain : ribB : 2.5.1.9 : Rhodococcus opacus B4	riboflavin synthase subunit alpha		F1YGD8_9ACTO Riboflavin synthase subunit alpha OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_04476 PE=4 SV=1 ;riboflavin synthase subunit alpha [Streptomyces cla vuligerus ATCC 27064];riboflavin synthase subunit alpha; Provisional;Lumazine binding domain; pfam00677
GOHSU18	CDS	GOHSU_18_00700	76582..76836		hypothetical protein [migap]				hypothetical protein		D1BKJ1_SANKS Putative uncharacterized protein OS=Sa nguibacter keddieii (strain ATCC 51767 / DSM 10542 / NCFB 3025 / ST-74) GN=Sked_05070 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU18	CDS	GOHSU_18_00710	complement(76933..77955)		diaminohydroxyphosphoribosylaminopyrimidine deaminase/5-amino-6-(5-phosphoribosylamino)uracil reductase [nite]	 ribG [nite]		ROP_69490 :diaminohydroxyphosphoribosylaminopyrimidine deaminase/5-amino-6-(5-phosphoribosylamino)uracil reductase : ribG : 3.5.4.26/1.1.1.193 : Rhodococcus opacus B4	riboflavin biosynthesis protein RibD		F1YGD7_9ACTO Riboflavin biosynthesis protein RibD O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04471 PE=4 S V=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: riboflavin biosynthesis protein RibD; PFAM: bifunctional deaminase-reductase domain protein; CMP/dCMP deaminase zinc-binding; KEGG: scl:sce3811 diaminohydroxyphosphoribosylaminopyrimidine deaminase / 5- amino-6-(5-phosphoribosylamino)uracil reductase;riboflavin biosynthesis protein RibD; TIGR00326;Riboflavin-specific deaminase. Riboflavin biosynthesis protein RibD (Diaminohydroxyphosphoribosylaminopyrimidine deaminase) catalyzes the deamination of 2,5-diamino-6-ribosylamino-4(3H)-pyrimidinone 5'-phosphate, which is an intermediate step in the...; cd01284;catalytic motif [active];Zn binding site [ion binding];Dihydrofolate reductase (DHFR). Reduces 7,8-dihydrofolate to 5,6,7,8-tetrahydrofolate with NADPH as a cofactor. This is an essential step in the biosynthesis of deoxythymidine phosphate since 5,6,7,8-tetrahydrofolate is required to regenerate 5; cl00161
GOHSU18	CDS	GOHSU_18_00720	complement(78181..78891)		ribulose-5-phosphate 3-epimerase [nite]	 rpe [nite]		RER_30190 :ribulose-5-phosphate 3-epimerase : rpe : 5.1.3.1 : Rhodococcus erythropolis PR4	ribulose-phosphate 3-epimerase		F1YGD6_9ACTO Ribulose-phosphate 3-epimerase OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_04466 PE=3 SV=1;ribulose-phosphate 3-epimerase [Saccharomonospora m arina XMU15];Ribulose-5-phosphate 3-epimerase (RPE). This enzyme catalyses the interconversion of D-ribulose 5-phosphate (Ru5P) into D-xylulose 5-phosphate, as part of the Calvin cycle (reductive pentose phosphate pathway) in chloroplasts and in the oxidative pentose...; cd00429;PFAM: Ribulose-phosphate 3 epimerase family; TIGRFAM: ribulose-phosphate 3-epimerase;ribulose-phosphate 3-epimerase; TIGR01163;substrate binding site [chemical binding];hexamer interface [polypeptide binding];metal binding site [ion binding]
GOHSU18	CDS	GOHSU_18_00730	complement(79214..79699)		not assigned						
GOHSU18	CDS	GOHSU_18_00740	complement(79759..81144)		putative rRNA methyltransferase [nite]			ROP_69450 :putative rRNA methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	putative RNA-binding Sun protein		F1YGD5_9ACTO Fmu (Sun) domain-containing protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04461 PE=4 SV =1;unnamed protein product [Gordonia polyisoprenivoran s VH2];RNA binding domain of NusB (N protein-Utilization Substance B) and Sun (also known as RrmB or Fmu) proteins. This family includes two orthologous groups exemplified by the transcription termination factor NusB and the N-terminal domain of the...; cl00223;putative RNA binding site [nucleotide binding];16S rRNA methyltransferase B; Provisional;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU18	CDS	GOHSU_18_00750	complement(81141..82097)		methionyl-tRNA formyltransferase [hamap]	fmt [hamap]	MF_00182:methionyl-tRNA formyltransferase:fmt 	RER_30160 :methionyl-tRNA formyltransferase : fmt : 2.1.2.9 : Rhodococcus erythropolis PR4	methionyl-tRNA formyltransferase	fmt	F1YGD4_9ACTO Methionyl-tRNA formyltransferase OS=Go rdonia neofelifaecis NRRL B-59395 GN=fmt PE=3 SV=1;methionyl-tRNA formyltransferase [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;methionyl-tRNA formyltransferase; Reviewed; PRK00005;Methionyl-tRNA formyltransferase, N-terminal hydrolase domain; cd08646;modifies the free amino group of the aminoacyl moiety of methionyl-tRNA(fMet) which is important in translation initiation; inactivation of this gene in Escherichia coli severely impairs growth;putative active site [active];substrate binding site [chemical binding];putative cosubstrate binding site;catalytic site [active];C-terminal domain of Formyltransferase and other enzymes; cd08704
GOHSU18	CDS	GOHSU_18_00760	complement(82099..84165)		primosomal protein n' [nite]	 priA [nite]		ROP_69400 :primosomal protein n' : priA : 3.6.1.- : Rhodococcus opacus B4	primosome assembly protein PriA	priA	F1YGD3_9ACTO Primosome assembly protein PriA OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_04451 PE=4 SV=1;primosome assembly protein PriA [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;primosome assembly protein PriA; Provisional;binding of PriA to forked DNA starts the assembly of the primosome, also possesses 3'-5' helicase activity
GOHSU18	CDS	GOHSU_18_00770	complement(84319..85539)		S-adenosylmethionine synthase [hamap]	metK [hamap]	MF_00086:S-adenosylmethionine synthase:metK 	RER_30110 :S-adenosylmethionine synthetase : metK : 2.5.1.6 : Rhodococcus erythropolis PR4	S-adenosylmethionine synthetase		F1YGD2_9ACTO S-adenosylmethionine synthase OS=Gordo nia neofelifaecis NRRL B-59395 GN=metK PE=3 SV=1;S-adenosylmethionine synthetase [Saccharomonospora cyanea NA- 134];PFAM: S-adenosylmethionine synthetase, C-terminal domain; S-adenosylmethionine synthetase, N-terminal domain; S-adenosylmethionine synthetase, central domain; TIGRFAM: S-adenosylmethionine synthetase;S-adenosylmethionine synthetase; Validated;S-adenosylmethionine synthetase, N-terminal domain; pfam00438;S-adenosylmethionine synthetase, central domain; pfam02772;S-adenosylmethionine synthetase, C-terminal domain; pfam02773
GOHSU18	CDS	GOHSU_18_00780	complement(85610..86872)		probable coenzyme A biosynthesis bifunctional protein CoaBC [nite]	 coaBC [nite]		RER_30100 :probable coenzyme A biosynthesis bifunctional protein CoaBC : coaBC : 4.1.1.36/6.3.2.5 : Rhodococcus erythropolis PR4	bifunctional phosphopantothenoylcysteine decarboxylase/phosphopantothenate synthase	dfp	D0LCR8_GORB4 Phosphopantothenoylcysteine decarboxyl ase/phosphopantothenate/cysteine ligase OS=Gordonia bronch ialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 106 67) GN=Gbro_2347 PE=4 SV=1;bifunctional phosphopantothenoylcysteine decarboxyl ase/phosphopantothenate synthase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;catalyzes the conjugation of cysteine to 4'-phosphopantothenate to form 4-phosphopantothenoylcysteine, which is then decarboxylated to form 4'-phosphopantotheine;bifunctional phosphopantothenoylcysteine decarboxylase/phosphopantothenate synthase; Validated;Flavoprotein; cl08021;DNA / pantothenate metabolism flavoprotein; cl04410
GOHSU18	CDS	GOHSU_18_00790	complement(86880..87176)	case	DNA-directed RNA polymerase subunit omega [hamap]	rpoZ [hamap]	MF_00366:DNA-directed RNA polymerase subunit omega:rpoZ 	RER_30090 :DNA-directed RNA polymerase omega chain : rpoZ : 2.7.7.6 : Rhodococcus erythropolis PR4	DNA-directed RNA polymerase subunit omega		H0JPB8_9NOCA DNA-directed RNA polymerase subunit om ega OS=Rhodococcus pyridinivorans AK37 GN=rpoZ PE=3 SV=1;DNA-directed RNA polymerase subunit omega [Coryneba cterium glutamicum ATCC 13032];IS fingerprint type 4-5;Promotes RNA polymerase assembly. Latches the N- and C-terminal regions of the beta' subunit thereby facilitating its interaction with the beta and alpha subunits;RNA polymerase Rpb6; cl14651
GOHSU18	CDS	GOHSU_18_00800	complement(87250..87789)		guanylate kinase [hamap]	gmk [hamap]	MF_00328:guanylate kinase:gmk 	ROP_69360 :guanylate kinase : gmk : 2.7.4.8 : Rhodococcus opacus B4	guanylate kinase	gmk	F1YGC9_9ACTO Guanylate kinase OS=Gordonia neofelifa ecis NRRL B-59395 GN=gmk PE=3 SV=1;guanylate kinase [Corynebacterium glutamicum ATCC 1 3032];Essential for recycling GMP and indirectly, cGMP;Guanylate kinase [Nucleotide transport and metabolism]; COG0194;Guanosine monophosphate kinase (GMPK, EC 2.7.4.8), also known as guanylate kinase (GKase), catalyzes the reversible phosphoryl transfer from adenosine triphosphate (ATP) to guanosine monophosphate (GMP) to yield adenosine diphosphate (ADP) and guanosine...; cd00071;catalytic site [active];G-X2-G-X-G-K
GOHSU18	CDS	GOHSU_18_00810	complement(87831..88145)		conserved hypothetical protein [nite]			RER_30070 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	integration host factor		H5UHB5_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_081_00650 PE=4 SV=1;integration host factor [Rhodococcus jostii RHA1]
GOHSU18	CDS	GOHSU_18_00820	88369..89451		putative hydrolase [nite]			ROP_43060 :putative hydrolase : : : Rhodococcus opacus B4	alpha-beta fold family hydrolase		F1YGC7_9ACTO Putative hydrolase OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_04421 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Poly(3-hydroxyalkanoate) synthetase [Lipid metabolism]; COG3243
GOHSU18	CDS	GOHSU_18_00830	complement(89543..90415)		orotidine 5'-phosphate decarboxylase [nite]	 pyrF [nite]		RER_30050 :orotidine 5'-phosphate decarboxylase : pyrF : 4.1.1.23 : Rhodococcus erythropolis PR4	orotidine 5'-phosphate decarboxylase	pyrF	F1YGA0_9ACTO Orotidine 5'-phosphate decarboxylase O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04416 PE=3 S V=1;orotidine 5'-phosphate decarboxylase [Corynebacteri um glutamicum ATCC 13032];IS fingerprint type 4-5;type 2 subfamily; involved in last step of pyrimidine biosynthesis; converts orotidine 5'-phosphate to UMP and carbon dioxide; OMP decarboxylase; OMPDCase; OMPdecase;Orotidine 5'-phosphate decarboxylase (ODCase) is a dimeric enzyme that decarboxylates orotidine 5'-monophosphate (OMP) to form uridine 5'-phosphate (UMP), an essential step in the pyrimidine biosynthetic pathway. In mammals, UMP synthase contains two...; cd04725;dimer interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00840	complement(90412..93762)		carbamoyl-phosphate synthase large chain [nite]	 carB [nite]		RER_30040 :carbamoyl-phosphate synthase large chain : carB : 6.3.5.5 : Rhodococcus erythropolis PR4	carbamoyl phosphate synthase large subunit	carB	F1YG99_9ACTO Carbamoyl-phosphate synthase large cha in OS=Gordonia neofelifaecis NRRL B-59395 GN=carB PE=3 SV= 1;carbamoyl phosphate synthase large subunit [Coryneb acterium glutamicum ATCC 13032];IS fingerprint type 4-5;carbamoyl phosphate synthase large subunit; Reviewed; PRK05294;four CarB-CarA dimers form the carbamoyl phosphate synthetase holoenzyme that catalyzes the production of carbamoyl phosphate; CarB is responsible for the amidotransferase activity;Carbamoyl-phosphate synthase L chain, N-terminal domain; pfam00289;ATP-grasp domain; cl03087;Carbamoyl-phosphate synthetase large chain, oligomerisation domain; pfam02787;Methylglyoxal synthase-like domain from type II glutamine-dependent carbamoyl phosphate synthetase (CSP). CSP, a CarA and CarB heterodimer, catalyzes the production of carbamoyl phosphate which is subsequently employed in the metabolic pathways...; cd01424;IMP binding site;dimer interface [polypeptide binding];interdomain contacts;partial ornithine binding site
GOHSU18	CDS	GOHSU_18_00850	complement(93762..94973)		carbamoyl-phosphate synthase small chain [nite]	 carA [nite]		ROP_69320 :carbamoyl-phosphate synthase small chain : carA : 6.3.5.5 : Rhodococcus opacus B4	carbamoyl-phosphate synthase small subunit	carA	F1YG98_9ACTO Carbamoyl-phosphate synthase small cha in OS=Gordonia neofelifaecis NRRL B-59395 GN=carA PE=3 SV= 1;carA gene product [Gordonia polyisoprenivorans VH2] ;carbamoyl phosphate synthase small subunit; Reviewed;Carbamoyl-phosphate synthase small chain, CPSase domain; cl03056;Small chain of the glutamine-dependent form of carbamoyl phosphate synthase, CPSase II; cd01744;catalytic site [active];subunit interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_00860	complement(94970..95647)		conserved hypothetical protein [nite]			RER_30020 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YG97_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04401 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU18	CDS	GOHSU_18_00870	complement(95644..96930)		dihydroorotase [nite]	 pyrC [nite]		ROP_69300 :dihydroorotase : pyrC : 3.5.2.3 : Rhodococcus opacus B4	dihydroorotase	pyrC	F1YG96_9ACTO Dihydroorotase OS=Gordonia neofelifaec is NRRL B-59395 GN=pyrC PE=3 SV=1;dihydroorotase [Corynebacterium glutamicum ATCC 130 32];IS fingerprint type 4-5;catalyzes the formation of N-carbamoyl-L-aspartate from (S)-dihydroorotate in pyrimidine biosynthesis;dihydroorotase; Validated; PRK09357;Superfamily of metallo-dependent hydrolases (also called amidohydrolase superfamily) is a large group of proteins that show conservation in their 3-dimensional fold (TIM barrel) and in details of their active site. The vast majority of the members have a...; cl00281;Dihydroorotase (DHOase), subgroup IIa; DHOases catalyze the reversible interconversion of carbamoyl aspartate to dihydroorotate, a key reaction in pyrimidine biosynthesis. This subgroup also contains proteins that lack the active site, like unc-33, a C; cd01317
GOHSU18	CDS	GOHSU_18_00880	complement(96930..97928)		aspartate carbamoyltransferase [hamap]	pyrB [hamap]	MF_00001:aspartate carbamoyltransferase:pyrB 	ROP_69290 :aspartate carbamoyltransferase : pyrB : 2.1.3.2 : Rhodococcus opacus B4	aspartate carbamoyltransferase	pyrB	F1YG95_9ACTO Aspartate carbamoyltransferase OS=Gord onia neofelifaecis NRRL B-59395 GN=pyrB PE=3 SV=1;aspartate carbamoyltransferase catalytic subunit [C orynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;aspartate carbamoyltransferase catalytic subunit; Provisional; PRK00856;catalyzes the transfer of the carbamoyl moiety from carbamoyl phosphate to L- aspartate in pyrimidine biosynthesis;Aspartate/ornithine carbamoyltransferase, carbamoyl-P binding domain; pfam02729;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU18	CDS	GOHSU_18_00890	complement(97925..98551)		bifunctional protein PyrR [hamap]	pyrR [hamap]	MF_01219:bifunctional protein PyrR:pyrR 	RER_29990 :bifunctional protein PyrR : pyrR : 2.4.2.9 : Rhodococcus erythropolis PR4	bifunctional pyrimidine regulatory protein PyrR uracil phosphoribosyltransferase	pyrR	F9VQY2_9ACTO Bifunctional protein pyrR OS=Gordonia alkanivorans NBRC 16433 GN=pyrR PE=3 SV=1;bifunctional pyrimidine regulatory protein PyrR ura cil phosphoribosyltransferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;regulates pyrimidine biosynthesis by binding to the mRNA of the pyr genes, also has been shown to have uracil phosphoribosyltransferase activity;Phosphoribosyl transferase (PRT)-type I domain; cd06223
GOHSU18	CDS	GOHSU_18_00900	complement(98611..99018)		N utilization substance protein B homolog [hamap]	nusB [hamap]	MF_00073:N utilization substance protein B homolog:nusB 	RER_29980 :transcription antitermination protein NusB : nusB : : Rhodococcus erythropolis PR4	N utilization substance protein B	nusB	F1YG93_9ACTO N utilization substance protein B homo log OS=Gordonia neofelifaecis NRRL B-59395 GN=nusB PE=3 SV =1;nusB gene product [Gordonia polyisoprenivorans VH2] ;RNA binding domain of NusB (N protein-Utilization Substance B) and Sun (also known as RrmB or Fmu) proteins. This family includes two orthologous groups exemplified by the transcription termination factor NusB and the N-terminal domain of the...; cl00223
GOHSU18	CDS	GOHSU_18_00910	complement(99065..99628)		elongation factor P [hamap]	efp [hamap]	MF_00141:elongation factor P:efp 	ROP_69260 :elongation factor P : efp : : Rhodococcus opacus B4	elongation factor P	efp	H0JPA7_9NOCA Elongation factor P OS=Rhodococcus pyr idinivorans AK37 GN=efp PE=3 SV=1;elongation factor P [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;elongation factor P; Validated;Involved in peptide bond synthesis; alters the affinity of the ribosome for aminoacyl-tRNA;Elongation factor P (EF-P) KOW-like domain; pfam08207;S1_EF-P_repeat_1: Translation elongation factor P (EF-P), S1-like RNA-binding domain, repeat 1. EF-P stimulates the peptidyltransferase activity in the prokaryotic 70S ribosome. EF-P enhances the synthesis of certain dipeptides with...; cd04470;RNA binding site [nucleotide binding];S1_EF-P_repeat_2: Translation elongation factor P (EF-P), S1-like RNA-binding domain, repeat 1. EF-P stimulates the peptidyltransferase activity in the prokaryotic 70S ribosome. EF-P enhances the synthesis of certain dipeptides with...; cd05794
GOHSU18	CDS	GOHSU_18_00920	complement(99664..100788)	overlap	putative aminopeptidase [nite]			RER_29960 :putative aminopeptidase : : 3.4.-.- : Rhodococcus erythropolis PR4	peptidase M24		F1YG91_9ACTO X-Pro dipeptidase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_04371 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: peptidase M24; creatinase; KEGG: gsu:GSU2021 xaa-pro dipeptidase;Xaa-Pro aminopeptidase [Amino acid transport and metabolism]; COG0006;Creatinase/Prolidase N-terminal domain; pfam01321;Similar to Prolidase and Aminopeptidase P. The members of this subfamily presumably catalyse hydrolysis of Xaa-Pro dipeptides and/or release of any N-terminal amino acid, including proline, that is linked with proline; cd01092
GOHSU18	CDS	GOHSU_18_00930	100714..101334	overlap	hypothetical protein [migap]				hypothetical protein		F1YG90_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04366 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU18	CDS	GOHSU_18_00940	101607..102368		hypothetical protein [nite]			ROP_60980 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0RD07_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_032_00630 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU18	CDS	GOHSU_18_00950	complement(102547..104211)	case	shikimate kinase/3-dehydroquinate synthase [hamap]	aroK/aroB [hamap]	MF_00109:shikimate kinase:aroK MF_00110:3-dehydroquinate synthase:aroB 	ACMV_15130 :shikimate kinase/3-dehydroquinate synthase : aroK/aroB : 2.7.1.71/4.2.3.4 : Acidiphilium multivorum AIU301	3-dehydroquinate synthase		F1YG89_9ACTO 3-dehydroquinate synthase OS=Gordonia neofelifaecis NRRL B-59395 GN=aroB PE=3 SV=1;3-dehydroquinate synthase [Streptomyces clavuligeru s ATCC 27064];Dehydroquinate synthase (DHQS) catalyzes the conversion of DAHP to DHQ in shikimate pathway for aromatic compounds synthesis; cd08195;dimer interface [polypeptide binding];metal binding site [ion binding]
GOHSU18	CDS	GOHSU_18_00960	complement(104208..105353)		chorismate synthase [nite]	 aroC [nite]		RER_29910 :chorismate synthase : aroC : 4.2.3.5 : Rhodococcus erythropolis PR4	chorismate synthase	aroC	F9VQ91_9ACTO Chorismate synthase OS=Gordonia alkani vorans NBRC 16433 GN=aroC PE=3 SV=1;chorismate synthase [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;catalyzes the formation of chorismate from 5-O-(1-carboxyvinyl)-3-phosphoshikimate in aromatic amino acid biosynthesis;Chorismase synthase, the enzyme catalyzing the final step of the shikimate pathway; cd07304;Tetramer interface [polypeptide binding];FMN-binding site [chemical binding]
GOHSU18	CDS	GOHSU_18_00970	complement(105421..105795)		prepilin peptidase [migap]				prepilin peptidase		H6N0Q3_GORPV Peptidase A24 (Type IV leader peptidas e) family protein OS=Gordonia polyisoprenivorans (strain D SM 44266 / VH2) GN=GPOL_c22330 PE=4 SV=1;unnamed protein product [Klebsiella oxytoca KCTC 16 86];COG1989 Type II secretory pathway, prepilin signal peptidase PulO and related peptidases;Type IV leader peptidase family; cl02077
GOHSU18	CDS	GOHSU_18_00980	complement(105919..106752)		shikimate dehydrogenase [nite]	 aroE [nite]		RER_29880 :shikimate dehydrogenase : aroE : 1.1.1.25 : Rhodococcus erythropolis PR4	shikimate 5-dehydrogenase	aroE	G7GJ19_9ACTO Shikimate dehydrogenase OS=Gordonia am arae NBRC 15530 GN=aroE PE=4 SV=1;shikimate 5-dehydrogenase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;AroE; catalyzes the conversion of shikimate to 3-dehydroshikimate;shikimate 5-dehydrogenase; Reviewed; PRK00258;Shikimate dehydrogenase substrate binding domain; pfam08501;NAD(P) binding domain of Shikimate dehydrogenase; cd01065;shikimate binding site;NAD(P) binding site [chemical binding]
GOHSU18	CDS	GOHSU_18_00990	complement(106752..108653)		aminodeoxychorismate lyase [migap]				aminodeoxychorismate lyase		F1YG84_9ACTO Aminodeoxychorismate lyase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04336 PE=4 SV=1;aminodeoxychorismate lyase [Mycobacterium sp. JLS];PFAM: aminodeoxychorismate lyase; KEGG: mmc:Mmcs_2348 aminodeoxychorismate lyase;YceG-like family; pfam02618;proteins similar to Escherichia coli yceG; cd08010;dimerization interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_01000	complement(108646..109098)	case	putative Holliday junction resolvase [hamap]	yqgF [hamap]	MF_00651:putative Holliday junction resolvase:yqgF 	ROP_69170 :putative Holliday junction resolvase : : 3.1.-.- : Rhodococcus opacus B4	Holliday junction resolvase-like protein		H0REV3_9ACTO Putative Holliday junction resolvase O S=Gordonia polyisoprenivorans NBRC 16320 GN=GOPIP_056_0049 0 PE=3 SV=1;Holliday junction resolvase-like protein [Corynebac terium glutamicum ATCC 13032];IS fingerprint type 4-5;similar to RuvC resolvase with substantial differences; NMR structural information suggests this protein is monomeric; unknown cellular function;Holliday junction resolvases (HJRs) are endonucleases that specifically resolve Holliday junction DNA intermediates during homologous recombination.  HJR's occur in archaea, bacteria, and in the mitochondria of certain fungi, however this CD includes...; cl00243
GOHSU18	CDS	GOHSU_18_01010	complement(109095..111761)		alanyl-tRNA synthetase [nite]	 alaS [nite]		RER_29850 :alanyl-tRNA synthetase : alaS : 6.1.1.7 : Rhodococcus erythropolis PR4	alanyl-tRNA synthetase	alaS	F1YG82_9ACTO Alanyl-tRNA synthetase OS=Gordonia neo felifaecis NRRL B-59395 GN=alaS PE=3 SV=1;alanyl-tRNA synthetase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;alanyl-tRNA synthetase; Reviewed; PRK00252;Catalyzes a two-step reaction, first charging an alanyl molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA;Alanyl-tRNA synthetase (AlaRS) class II core catalytic domain. AlaRS is a homodimer. It is responsible for the attachment of alanine to the 3' OH group of ribose of the appropriate tRNA. This domain is primarily responsible for ATP-dependent formation of...; cd00673;motif 1;motif 2;motif 3;Threonyl and Alanyl tRNA synthetase second additional domain; cl08469;DHHA1 domain; pfam02272
GOHSU18	CDS	GOHSU_18_01020	complement(111810..113111)		putative AAA family ATPase [nite]			ROP_69150 :putative AAA family ATPase : : 3.6.-.- : Rhodococcus opacus B4	recombination factor protein RarA		F1YG81_9ACTO Recombination factor protein RarA OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_04321 PE=4 SV=1 ;recombination factor protein RarA [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;recombination factor protein RarA; Reviewed;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;MgsA AAA+ ATPase C terminal; pfam12002
GOHSU18	CDS	GOHSU_18_01030	complement(113246..114361)		hypothetical protein [nite]			RER_29820 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YG80_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04316 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein Kinases, catalytic domain; cl09925
GOHSU18	CDS	GOHSU_18_01040	complement(114376..116199)		aspartyl-tRNA synthetase [nite]	 aspS [nite]		ROP_69100 :aspartyl-tRNA synthetase : aspS : 6.1.1.12 : Rhodococcus opacus B4	aspartyl-tRNA synthetase	aspS	F1YG79_9ACTO Aspartate--tRNA ligase OS=Gordonia neo felifaecis NRRL B-59395 GN=aspS PE=3 SV=1;aspartyl-tRNA synthetase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;aspartyl-tRNA synthetase; Validated; PRK00476;catalyzes a two-step reaction, first charging an aspartate molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA; contains discriminating and non-discriminating subtypes;EcAspRS_like_N: N-terminal, anticodon recognition domain of the type found in Escherichia coli aspartyl-tRNA synthetase (AspRS), the human mitochondrial (mt) AspRS-2, the discriminating (D) Thermus thermophilus AspRS-1, and the nondiscriminating (ND)...; cd04317;dimer interface [polypeptide binding];anticodon binding site;Asp tRNA synthetase (aspRS) class II core domain. Class II assignment is based upon its structure and the presence of three characteristic sequence motifs. AspRS is a homodimer, which attaches a specific amino acid to the 3' OH group of ribose of the...; cd00777;homodimer interface [polypeptide binding];motif 1;motif 2;GAD domain; pfam02938;Class II tRNA amino-acyl synthetase-like catalytic core domain. Class II amino acyl-tRNA synthetases (aaRS) share a common fold and generally attach an amino acid to the 3' OH of ribose of the appropriate tRNA.   PheRS is an exception in that it attaches...; cl00268;motif 3
GOHSU18	CDS	GOHSU_18_01050	116386..117270		conserved hypothetical protein [nite]			RER_29790 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	metalloprotease		F1YG78_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_04306 PE=4 SV=1;metalloprotease [Corynebacterium glutamicum ATCC 13 032];Putative neutral zinc metallopeptidase; cl00988
GOHSU18	CDS	GOHSU_18_01060	complement(117292..118560)	case	histidyl-tRNA synthetase [hamap]	hisS [hamap]	MF_00127:histidyl-tRNA synthetase:hisS 	RER_29350 :histidyl-tRNA synthetase : hisS : 6.1.1.21 : Rhodococcus erythropolis PR4	histidyl-tRNA synthetase	hisS	F1YG77_9ACTO Histidine--tRNA ligase OS=Gordonia neo felifaecis NRRL B-59395 GN=hisS PE=3 SV=1;histidyl-tRNA synthetase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;catalyzes a two-step reaction, first charging a histidine molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA; class II aminoacyl-tRNA synthetase; forms homodimers; some organisms have a paralogous gene, hisZ, that is similar to hisS and produces a protein that performs the first step in histidine biosynthesis along with HisG;histidyl-tRNA synthetase; Reviewed; PRK00037;Class II Histidinyl-tRNA synthetase (HisRS)-like catalytic core domain. HisRS is a homodimer. It is responsible for the attachment of histidine to the 3' OH group of ribose of the appropriate tRNA. This domain is primarily responsible for ATP-dependent...; cd00773;dimer interface [polypeptide binding];motif 1;motif 2;motif 3;HisRS Histidyl-anticodon binding domain. HisRS belongs to class II aminoacyl-tRNA synthetases (aaRS). This alignment contains the anticodon binding domain, which is responsible for specificity in tRNA-binding, so that the activated amino acid is...; cd00859;anticodon binding site
GOHSU18	CDS	GOHSU_18_01070	complement(118557..119258)		hypothetical protein [nite]			ROP_68820 :hypothetical protein : : : Rhodococcus opacus B4	putative Zn-dependent hydrolase		F1YG76_9ACTO Beta-lactamase domain-containing prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04296 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Zn-dependent hydrolases, including glyoxylases [General function prediction only]; COG0491;Metallo-beta-lactamase superfamily; cl00446
GOHSU18	CDS	GOHSU_18_01080	119443..120417		putative peptidyl-prolyl cis-trans isomerase [nite]			ROP_68810 :putative peptidyl-prolyl cis-trans isomerase : : 5.2.1.8 : Rhodococcus opacus B4	cyclophilin type peptidyl-prolyl cis-trans isomerase		F9VQA7_9ACTO Putative peptidyl-prolyl cis-trans iso merase OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_014_00 190 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: peptidyl-prolyl cis-trans isomerase cyclophilin type; KEGG: hpy:HP1441 peptidyl-prolyl cis-trans isomerase B, cyclosporin-type rotamase (ppi);cyclophilin: cyclophilin-type peptidylprolyl cis- trans isomerases. This family contains eukaryotic, bacterial and archeal proteins which exhibit a peptidylprolyl cis- trans isomerases activity (PPIase, Rotamase) and in addition bind the...; cd00317;peptidyl-prolyl cis-trans isomerase B (rotamase B); Provisional
GOHSU18	CDS	GOHSU_18_01090	complement(120516..122822)		GTP pyrophosphokinase/guanosine-3',5'-bis(diphosphate) 3-pyrophosphohydrolase [nite]	 relA [nite]		ROP_68790 :GTP pyrophosphokinase/guanosine-3',5'-bis(diphosphate) 3-pyrophosphohydrolase : relA : 2.7.6.5/3.1.7.2 : Rhodococcus opacus B4	GTP pyrophosphokinase RelA	relA	F1YG73_9ACTO RelA/SpoT family protein OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_04281 PE=3 SV=1;relA gene product [Gordonia polyisoprenivorans VH2] ;(p)ppGpp synthetase, RelA/SpoT family; TIGR00691;Metal dependent phosphohydrolases with conserved 'HD' motif; cl00076;Nucleotidyltransferase (NT) domain of RelA- and SpoT-like ppGpp synthetases and hydrolases; cd05399;synthetase active site [active];NTP binding site [chemical binding];metal binding site [ion binding];TGS_RelA_SpoT: The RelA (SpoT) protein, also referred to as ppGpp hydrolase/synthetase, is a ribosome-associated protein that is activated during amino acid starvation and thought to mediate the stringent response. RelA contains a TGS domain, named after...; cd01668;ACT  domain found C-terminal of the RelA/SpoT domains; cd04876
GOHSU18	CDS	GOHSU_18_01100	complement(122866..123438)		adenine phosphoribosyltransferase [hamap]	apt [hamap]	MF_00004:adenine phosphoribosyltransferase:apt 	ROP_68780 :adenine phosphoribosyltransferase : apt : 2.4.2.7 : Rhodococcus opacus B4	adenine phosphoribosyltransferase	apt	F1YG72_9ACTO Adenine phosphoribosyltransferase OS=G ordonia neofelifaecis NRRL B-59395 GN=apt PE=3 SV=1;adenine phosphoribosyltransferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes a salvage reaction resulting in the formation of AMP which is metabolically less costly than a de novo synthesis;Adenine/guanine phosphoribosyltransferases and related PRPP-binding proteins [Nucleotide transport and metabolism]; COG0503;Phosphoribosyl transferase (PRT)-type I domain; cd06223
GOHSU18	CDS	GOHSU_18_01110	complement(123423..125135)		conserved hypothetical protein [nite]			RER_29290 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	extracellular solute-binding protein family 5 protein		F1YG71_9ACTO Extracellular solute-binding protein f amily 5 protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_04271 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.646 at residue 30;PFAM: extracellular solute-binding protein family 5; KEGG: mes:Meso_1835 extracellular solute-binding protein;ABC-type dipeptide transport system, periplasmic component [Amino acid transport and metabolism]; COG0747;The substrate-binding domain of an ABC-type nickel/oligopeptide-like import system contains the type 2 periplasmic binding fold; cl01709
GOHSU18	CDS	GOHSU_18_01120	complement(125132..126229)		protein-export membrane protein SecF [nite]	 secF [nite]		RER_29280 :protein-export membrane protein SecF : secF : : Rhodococcus erythropolis PR4	preprotein translocase subunit SecF	secF	G7H5G0_9ACTO Protein translocase subunit SecF OS=Go rdonia araii NBRC 100433 GN=secF PE=3 SV=1;preprotein translocase subunit SecF [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;forms a complex with SecD and YajC; SecDFyajC stimulates the proton motive force-driven protein translocation; seems to modulate the cycling of SecA by stabilizing its membrane-inserted state and appears to be required for the release of mature proteins from the extracytoplasmic side of the membrane; in some organisms, such as Bacillus subtilis, SecD is fused to SecF;preprotein translocase subunit SecF; Reviewed; PRK13022;Protein export membrane protein; cl14618
GOHSU18	CDS	GOHSU_18_01130	complement(126226..127830)		protein-export membrane protein SecD [nite]	 secD [nite]		RER_29270 :protein-export membrane protein SecD : secD : : Rhodococcus erythropolis PR4	preprotein translocase subunit SecD	secD	F1YG69_9ACTO Protein translocase subunit SecD OS=Go rdonia neofelifaecis NRRL B-59395 GN=secD PE=3 SV=1;preprotein translocase subunit SecD [Corynebacteriu m glutamicum ATCC 13032];part of the preprotein secretory system; when complexed with proteins SecF and YajC, SecDFyajC stimulates the proton motive force-driven protein translocation, and appears to be required for the release of mature proteins from the extracytoplasmic side of the membrane;Preprotein translocase subunit SecD [Intracellular trafficking and secretion]; COG0342;Protein export membrane protein; cl14618
GOHSU18	CDS	GOHSU_18_01140	complement(127842..128231)		preprotein translocase subunit YajC [migap]				preprotein translocase subunit YajC		F1YG68_9ACTO Preprotein translocase, YajC subunit O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04256 PE=4 S V=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Preprotein translocase subunit; cl00806;TIGRFAM: preprotein translocase subunit YajC; PFAM: YajC family protein; KEGG: aph:APH_1090 preprotein translocase subunit YajC
GOHSU18	CDS	GOHSU_18_01150	complement(128287..129375)		Holliday junction ATP-dependent DNA helicase RuvB [hamap]	ruvB [hamap]	MF_00016:Holliday junction ATP-dependent DNA helicase RuvB:ruvB 	RER_29250 :Holliday junction DNA helicase RuvB : ruvB : 3.6.1.- : Rhodococcus erythropolis PR4	Holliday junction DNA helicase RuvB	ruvB	H0QZL5_9ACTO Holliday junction ATP-dependent DNA he licase RuvB OS=Gordonia effusa NBRC 100432 GN=ruvB PE=3 SV =1;Holliday junction DNA helicase RuvB [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;promotes strand exchange during homologous recombination; RuvAB complex promotes branch migration; RuvABC complex scans the DNA during branch migration and resolves Holliday junctions at consensus sequences; forms hexameric rings around opposite DNA arms; requires ATP for branch migration and orientation of RuvAB complex determines direction of migration;Holliday junction DNA helicase RuvB; Reviewed; PRK00080;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger;Holliday junction DNA helicase ruvB C-terminus; pfam05491
GOHSU18	CDS	GOHSU_18_01160	complement(129372..129968)		Holliday junction ATP-dependent DNA helicase RuvA [hamap]	ruvA [hamap]	MF_00031:Holliday junction ATP-dependent DNA helicase RuvA:ruvA 	ROP_68720 :Holliday junction DNA helicase RuvA : ruvA : 3.6.1.- : Rhodococcus opacus B4	Holliday junction DNA helicase RuvA	ruvA	F1YG66_9ACTO Holliday junction ATP-dependent DNA he licase RuvA OS=Gordonia neofelifaecis NRRL B-59395 GN=ruvA PE=3 SV=1;Holliday junction DNA helicase RuvA [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;Holliday junction DNA helicase RuvA; Reviewed; PRK00116;RuvA N terminal domain; pfam01330;plays an essential role in ATP-dependent branch migration of the Holliday junction;RuvA, C-terminal domain; pfam07499
GOHSU18	CDS	GOHSU_18_01170	complement(129965..130552)		crossover junction endodeoxyribonuclease RuvC [hamap]	ruvC [hamap]	MF_00034:crossover junction endodeoxyribonuclease RuvC:ruvC 	ROP_68710 :Holliday junction endonuclease RuvC : ruvC : 3.1.22.4 : Rhodococcus opacus B4	Holliday junction resolvase	ruvC	F1YG65_9ACTO Crossover junction endodeoxyribonuclea se RuvC OS=Gordonia neofelifaecis NRRL B-59395 GN=ruvC PE= 3 SV=1;Holliday junction resolvase [Corynebacterium glutam icum ATCC 13032];endonuclease; resolves Holliday structures; forms a complex of RuvABC; the junction binding protein RuvA forms a hexameric ring along with the RuvB helicase and catalyzes branch migration; RuvC then interacts with RuvAB to resolve the Holliday junction by nicking DNA strands of like polarity;Holliday junction resolvases (HJRs) are endonucleases that specifically resolve Holliday junction DNA intermediates during homologous recombination.  HJR's occur in archaea, bacteria, and in the mitochondria of certain fungi, however this CD includes...; cd00529;putative DNA-binding cleft [nucleotide binding];dimer interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_01180	complement(130658..131413)	case	UPF0082 protein yeeN [hamap]	yeeN [hamap]	MF_00693:UPF0082 protein yeeN:yeeN 	ROP_68700 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7H5G6_9ACTO Probable transcriptional regulatory pr otein GOARA_064_00930 OS=Gordonia araii NBRC 100433 GN=GOA RA_064_00930 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transcriptional regulator; cl00361
GOHSU18	CDS	GOHSU_18_01190	complement(131420..132049)		glutamine amidotransferase subunit PdxT [hamap]	pdxT [hamap]	MF_01615:glutamine amidotransferase subunit PdxT:pdxT 	ROP_68670 :pyridoxal phosphate synthase component Pdx2 : pdx2 : : Rhodococcus opacus B4	SNO glutamine amidotransferase		F1YG63_9ACTO Glutamine amidotransferase subunit Pdx T OS=Gordonia neofelifaecis NRRL B-59395 GN=pdxT PE=3 SV=1 ;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: SNO glutamine amidotransferase; CobB/CobQ domain protein glutamine amidotransferase; KEGG: dds:Ddes_1897 SNO glutamine amidotransferase;Glutamine Amidotransferase (GATase_I) involved in pyridoxine biosynthesis; cd01749;predicted active site [active];catalytic triad [active]
GOHSU18	CDS	GOHSU_18_01200	complement(132046..132894)		acyl-CoA thioesterase [nite]	 tesB [nite]		ROP_68660 :acyl-CoA thioesterase : tesB : 3.1.2.- : Rhodococcus opacus B4	acyl-CoA thioesterase	tesB	H5TNK1_9ACTO Acyl-CoA thioesterase OS=Gordonia otit idis NBRC 100426 GN=tesB PE=4 SV=1;acyl-CoA thioesterase [Rhodococcus opacus B4];Acyl-CoA thioesterase [Lipid metabolism]; COG1946;protein synonym:thioesterase II;Thioesterase II (TEII) is thought to regenerate misprimed nonribosomal peptide synthetases (NRPSs) as well as modular polyketide synthases (PKSs) by hydrolyzing acetyl groups bound to the peptidyl carrier protein (PCP) and acyl carrier protein (ACP)...; cd03445;Thioesterase II (TEII) is thought to regenerate misprimed nonribosomal peptide synthetases (NRPSs) as well as modular polyketide synthases (PKSs) by hydrolyzing acetyl groups bound to the peptidyl carrier protein (PCP) and acyl carrier protein (ACP)...; cd03444;catalytic triad [active];dimer interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_01210	132948..134915		putative acyl-CoA oxidase [nite]			ROP_17180 :putative acyl-CoA oxidase : : 1.3.3.6 : Rhodococcus opacus B4	acyl-CoA oxidase domain-containing protein		D5UT28_TSUPD Acyl-CoA oxidase domain protein OS=Tsu kamurella paurometabola (strain ATCC 8368 / DSM 20162 / JC M 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_0678 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: acyl-CoA oxidase domain protein; acyl-CoA dehydrogenase domain protein; KEGG: mab:MAB_3588  acyl-CoA oxidase;Acyl-CoA dehydrogenase; cl09933
GOHSU18	CDS	GOHSU_18_01220	complement(134950..135732)		putative serine/threonine protein phosphatase [nite]			YM304_02150 :putative serine/threonine protein phosphatase : : 3.1.3.16 : Acidimicrobidae bacterium YM16-304	protein serine/threonine phosphatase		F1YG58_9ACTO Protein phosphatase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_04206 PE=4 SV=1;protein serine/threonine phosphatase [Beutenbergia cavernae DSM 12333];type strain of Beutenbergia cavernae;PFAM: Protein phosphatase 2C;Serine/threonine phosphatases, family 2C, catalytic domain; The protein architecture and deduced catalytic mechanism of PP2C phosphatases are similar to the PP1, PP2A, PP2B family of protein Ser/Thr phosphatases, with which PP2C shares no sequence...; cd00143
GOHSU18	CDS	GOHSU_18_01230	complement(135776..136681)		pyridoxal biosynthesis lyase PdxS [hamap]	pdxS [hamap]	MF_01824:pyridoxal biosynthesis lyase PdxS:pdxS 	RER_29090 :probable pyridoxal phosphate synthase component Pdx1 : pdx1 : : Rhodococcus erythropolis PR4	pyridoxine biosynthesis protein		F1YG56_9ACTO Pyridoxal biosynthesis lyase PdxS OS=G ordonia neofelifaecis NRRL B-59395 GN=pdxS PE=3 SV=1;pyridoxine biosynthesis protein [Rhodococcus equi 1 03S];pyridoxal 5'-phosphate synthase, synthase subunit Pdx1;PdxS is a subunit of the pyridoxal 5'-phosphate (PLP) synthase, an important enzyme in deoxyxylulose 5-phosphate (DXP)-independent pathway for de novo biosynthesis of PLP,  present in some eubacteria, in archaea, fungi, plants, plasmodia, and some...; cd04727;multimer interface [polypeptide binding]
GOHSU18	CDS	GOHSU_18_01240	complement(136785..137297)		conserved hypothetical protein [nite]			RER_29080 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GL65_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_19_00270 PE=4 SV=1;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;KEGG: hypothetical protein
GOHSU18	CDS	GOHSU_18_01250	complement(137305..138429)		mannosyltransferase PimA [nite]	 pimA [nite]		RER_29070 :mannosyltransferase PimA : pimA : 2.4.1.- : Rhodococcus erythropolis PR4	glycosyl transferase group 1 protein		F1YG54_9ACTO Glycosyl transferase group 1 protein O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04186 PE=4 S V=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Glycosyltransferase [Cell envelope biogenesis, outer membrane]; COG0438;PFAM: glycosyl transferase group 1; KEGG: acp:A2cp1_4300 phosphatidylinositol alpha- mannosyltransferase;This family is most closely related to the GT1 family of glycosyltransferases and named after YqgM in Bacillus licheniformis about which little is known. Glycosyltransferases catalyze the transfer of sugar moieties from activated donor molecules to...; cd03801
GOHSU18	CDS	GOHSU_18_01260	complement(138426..139376)		probable acyltransferase [nite]			RER_29060 :probable acyltransferase : : 2.3.1.- : Rhodococcus erythropolis PR4	lipid A biosynthesis lauroyl acyltransferase		F1YG53_9ACTO Lipid A biosynthesis lauroyl acyltrans ferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_0418 1 PE=4 SV=1;lipid A biosynthesis lauroyl acyltransferase [Nocar dia farcinica IFM 10152];Acylates the intermediate (KDO)2-lipid IVA to form (KDO)2-(lauroyl)-lipid IVA;Bacterial lipid A biosynthesis acyltransferase; pfam03279;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357;Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: LABLAT-like; cd07984;putative acyl-acceptor binding pocket
GOHSU18	CDS	GOHSU_18_01270	complement(139378..140040)		phosphatidylinositol synthase [nite]			ROP_68590 :phosphatidylinositol synthase : : 2.7.8.11 : Rhodococcus opacus B4	phosphatidylglycerophosphate synthase	pgsA1	F1YG52_9ACTO CDP-alcohol phosphatidyltransferase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_04176 PE=3 SV =1;phosphatidylglycerophosphate synthase [Corynebacter ium glutamicum ATCC 13032];IS fingerprint type 4-5;CDP-alcohol phosphatidyltransferase; cl00453
GOHSU18	CDS	GOHSU_18_01280	complement(140037..140573)		HIT family protein [nite]			RER_29040 :HIT family protein : : : Rhodococcus erythropolis PR4	HIT family protein		F1YG51_9ACTO Histidine triad (HIT) protein OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_04171 PE=4 SV=1;HIT family protein [Streptomyces clavuligerus ATCC 27064];FHIT (fragile histidine family): FHIT proteins, related to the HIT family carry a motif HxHxH/Qxx (x, is a hydrophobic amino acid), On the basis of sequence, substrate specificity, structure, evolution and mechanism, HIT proteins are classified into...; cd01275;nucleotide binding site/active site [active];HIT family signature motif;catalytic residue [active]
GOHSU18	CDS	GOHSU_18_01290	complement(140592..142673)		threonyl-tRNA synthetase [nite]	 thrS [nite]		RER_29030 :threonyl-tRNA synthetase : thrS : 6.1.1.3 : Rhodococcus erythropolis PR4	threonyl-tRNA synthetase	thrS	G7H5H8_9ACTO Threonine--tRNA ligase OS=Gordonia ara ii NBRC 100433 GN=thrS PE=3 SV=1;threonyl-tRNA synthetase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;catalyzes a two-step reaction, first charging a threonine molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA; catalyzes the formation of threonyl-tRNA(Thr) from threonine and tRNA(Thr);threonyl-tRNA synthetase; Reviewed; PRK12305;Threonyl and Alanyl tRNA synthetase second additional domain; cl08469;Threonyl-tRNA synthetase (ThrRS) class II core catalytic domain. ThrRS is a homodimer. It is responsible for the attachment of threonine to the 3' OH group of ribose of the appropriate tRNA. This domain is primarily responsible for ATP-dependent...; cd00771;dimer interface [polypeptide binding];motif 1;motif 2;motif 3;ThrRS Threonyl-anticodon binding domain. ThrRS belongs to class II aminoacyl-tRNA synthetases (aaRS). This alignment contains the anticodon binding domain, which is responsible for specificity in tRNA-binding, so that the activated amino acid is...; cd00860;anticodon binding site
GOHSU18	CDS	GOHSU_18_01300	complement(142717..144072)		putative glycosyltransferase [nite]			RER_03630 :putative glycosyltransferase : : 2.4.-.- : Rhodococcus erythropolis PR4	hypothetical protein		G7H1G3_9ACTO Putative glycosyltransferase OS=Gordon ia araii NBRC 100433 GN=GOARA_045_00420 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF2029; KEGG: dol:Dole_2388 hypothetical protein;Protein of unknown function (DUF2029); cl09812
GOHSU18	CDS	GOHSU_18_01310	complement(144166..144606)		hypothetical membrane protein [nite]			ROP_68560 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		D0LCL0_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_2286 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function, PGPGW, transmembrane; KEGG: geo:Geob_2847 hypothetical protein;Putative transmembrane protein (PGPGW); cl09870
GOHSU18	CDS	GOHSU_18_01320	complement(144748..145089)		aldose 1-epimerase [migap]				aldose 1-epimerase		E5XUA9_9ACTO Putative uncharacterized protein OS=Se gniliparus rugosus ATCC BAA-974 GN=HMPREF9336_03081 PE=4 S V=1;aldose 1-epimerase [Citreicella sp. SE45];identified by match to protein family HMM PF01263;Aldose 1-epimerase, similar to Lactococcus lactis lacX; cd09024;Galactose mutarotase and related enzymes [Carbohydrate transport and metabolism]; COG2017
GOHSU18	tRNA	GOHSU_18_t0010	complement(145254..145325)		not assigned						identified by tRNAscan-SE; putative
GOHSU18	tRNA	GOHSU_18_t0020	145500..145572		not assigned						identified by tRNAscan-SE; putative
GOHSU18	tRNA	GOHSU_18_t0030	145629..145702		not assigned						identified by tRNAscan-SE; putative
GOHSU18	tRNA	GOHSU_18_t0040	145728..145802		not assigned						identified by tRNAscan-SE; putative
GOHSU18	tRNA	GOHSU_18_t0050	146396..146468		not assigned						identified by tRNAscan-SE; putative
GOHSU18	tRNA	GOHSU_18_t0060	146525..146598		not assigned						identified by tRNAscan-SE; putative
GOHSU18	tRNA	GOHSU_18_t0070	146624..146695		not assigned						identified by tRNAscan-SE; putative
GOHSU18	CDS	GOHSU_18_01330	146833..148062		hypothetical protein [nite]			ROP_59630 :hypothetical protein : : : Rhodococcus opacus B4	PGAP1 family protein		G7H646_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_067_00270 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: PGAP1 family protein; KEGG: rop:ROP_59630 hypothetical protein;2-succinyl-6-hydroxy-2, 4-cyclohexadiene-1-carboxylate synthase; TIGR03695;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU18	CDS	GOHSU_18_01340	complement(148137..149291)		putative zinc-containing alcohol dehydrogenase [nite]			YM304_05230 :putative zinc-containing alcohol dehydrogenase : : 1.1.1.- : Acidimicrobidae bacterium YM16-304	alcohol dehydrogenase		E9URE1_9ACTO Zinc-binding dehydrogenase OS=Nocardio idaceae bacterium Broad-1 GN=NBCG_01304 PE=4 SV=1;alcohol dehydrogenase [Mycobacterium sp. KMS];Threonine dehydrogenase and related Zn-dependent dehydrogenases [Amino acid transport and metabolism / General function prediction only]; COG1063;Alcohol dehydrogenases of the MDR family; cd08262;PFAM: Alcohol dehydrogenase, zinc-binding domain protein; Alcohol dehydrogenase GroES domain protein; KEGG: mmc:Mmcs_2286 alcohol dehydrogenase GroES-like protein;putative NAD(P) binding site [chemical binding];catalytic Zn binding site [ion binding]
GOHSU18	CDS	GOHSU_18_01350	complement(149680..151122)		hypothetical protein [nite]			ROP_55160 :hypothetical protein : : : Rhodococcus opacus B4	putative HNH endonuclease		F1YN44_9ACTO HNH endonuclease OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_16758 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF222); pfam02720;HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and...; cd00085
GOHSU18	CDS	GOHSU_18_01360	151306..154779		putative helicase [nite]			MLP_22380 :putative helicase : : 3.6.1.- : Microlunatus phosphovorus NM-1	helicase domain-containing protein		F1YMK7_9ACTO Helicase domain-containing protein OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15764 PE=4 SV= 1;helicase domain-containing protein [Truepera radiov ictrix DSM 17093];COGs: COG0553 Superfamily II DNA/RNA helicase SNF2 family; InterProIPR014001:IPR001650:IPR014021:IPR010916:IPR 000330; KEGG: sat:SYN_00017 SNF2 family DNA/RNA helicase; PFAM: helicase domain protein; SNF2-related protein; SMART: DEAD-like helicase ; helicase domain protein; SPTR: A1HMN5 Helicase domain protein; PFAM: Helicase conserved C-terminal domain; SNF2 family N-terminal domain;DEAD-like helicases superfamily; smart00487;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];Domain of unknown function (DUF3883); pfam13020
GOHSU18	CDS	GOHSU_18_01370	154874..157651		hypothetical protein [nite]			MLP_22330 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YMK9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15774 PE=4 SV=1;unnamed protein product [Anabaena variabilis ATCC 2 9413];Adenine-specific DNA methylase containing a Zn-ribbon [DNA replication, recombination, and repair];Protein of unknown function (DUF1156); pfam06634;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU18	CDS	GOHSU_18_01380	157749..161138		hypothetical protein [nite]			MLP_22300 :hypothetical protein : : : Microlunatus phosphovorus NM-1	ATPase of the AAA+ family protein		F1YML0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15779 PE=4 SV=1;ATPase of the AAA+ family protein [Actinomyces visc osus C505];Protein of unknown function (DUF499); pfam04465
GOHSU18	CDS	GOHSU_18_01390	161154..163034		hypothetical protein [nite]			MLP_22320 :hypothetical protein : : : Microlunatus phosphovorus NM-1			G7GZ24_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_021_00860 PE=4 SV=1
GOHSU18	CDS	GOHSU_18_01400	163047..163619		conserved hypothetical protein [nite]			RER_28590 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	N-acetyltransferase GCN5		F1YML2_9ACTO GCN5-like N-acetyltransferase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_15789 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: GCN5-related N-acetyltransferase; KEGG: pen:PSEEN3468 acetyltransferase;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU18	CDS	GOHSU_18_01410	complement(163633..164658)		hypothetical protein [nite]			ROP_68510 :hypothetical protein : : : Rhodococcus opacus B4	putative ATP-binding protein		F1YML4_9ACTO AFG1-family ATPase OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_15799 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];AFG1-like ATPase; pfam03969;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099
GOHSU18	CDS	GOHSU_18_01420	164695..165486		hypothetical protein [nite]			ROP_68500 :hypothetical protein : : : Rhodococcus opacus B4	bifunctional deaminase-reductase domain-containing protein		F1YML5_9ACTO Bifunctional deaminase-reductase domai n-containing protein OS=Gordonia neofelifaecis NRRL B-5939 5 GN=SCNU_15804 PE=4 SV=1;bifunctional deaminase-reductase domain-containing protein [Nocardiopsis dassonvillei subsp. dassonvillei DSM 43111];COGs: COG1985 Pyrimidine reductase riboflavin biosynthesis; InterPro IPR002734; KEGG: sro:Sros_6709 pyrimidine reductase riboflavin biosynthesis-like protein; PFAM: bifunctional deaminase-reductase domain protein; SPTR: C1YK26 Pyrimidine reductase, riboflavin biosynthesis; PFAM: RibD C-terminal domain;Dihydrofolate reductase (DHFR). Reduces 7,8-dihydrofolate to 5,6,7,8-tetrahydrofolate with NADPH as a cofactor. This is an essential step in the biosynthesis of deoxythymidine phosphate since 5,6,7,8-tetrahydrofolate is required to regenerate 5; cl00161
GOHSU18	CDS	GOHSU_18_01430	165497..167053		putative hydrolase [nite]			ROP_68490 :putative hydrolase : : : Rhodococcus opacus B4	hypothetical protein		F1YML6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15809 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.996) with cleavage site probability 0.939 at residue 21;KEGG: xac:XAC4055 cysteine proteinase;Alpha/beta hydrolase family; pfam12697;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU18	CDS	GOHSU_18_01440	167086..168309		arabinosyltransferase AftC [nite]	 aftC [nite]		RER_28550 :arabinosyltransferase AftC : aftC : 2.4.2.- : Rhodococcus erythropolis PR4	hypothetical protein		F1YML7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15814 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF2029; KEGG: reh:H16_A0724 hypothetical membrane protein;Protein of unknown function (DUF2029); cl09812
GOHSU18	CDS	GOHSU_18_01450	complement(168359..169150)		hypothetical protein [nite]			KRH_03140 :hypothetical protein : : : Kocuria rhizophila DC2201	hypothetical protein		G2EQ97_CORGT Putative uncharacterized protein OS=Co rynebacterium glutamicum S9114 GN=CgS9114_13391 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: bcv:Bcav_1778 hypothetical protein;Thioesterase-like superfamily; pfam13622
GOHSU18	CDS	GOHSU_18_01460	169223..169666		peptide methionine sulfoxide reductase MsrB [nite]	 msrB [nite]		KRH_14090 :peptide methionine sulfoxide reductase MsrB : msrB : 1.8.4.12 : Kocuria rhizophila DC2201	oxidoreductase		H6MZV0_GORPV Methionine-R-sulfoxide reductase OS=Go rdonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL _c21620 PE=4 SV=1;oxidoreductase [Streptomyces clavuligerus ATCC 2706 4];methionine sulfoxide reductase B; Provisional; cl15841;SelR domain; pfam01641
GOHSU18	CDS	GOHSU_18_01470	complement(169695..170402)		conserved hypothetical protein [nite]			RER_28490 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	chlorite dismutase		G7H5I6_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_064_01130 PE=4 SV=1;chlorite dismutase [Rhodococcus jostii RHA1];Chlorite dismutase; cl01280
GOHSU18	CDS	GOHSU_18_01480	complement(170475..171857)		protoporphyrinogen oxidase [nite]	 hemG [nite]		ROP_68440 :protoporphyrinogen oxidase : hemG : 1.3.3.4 : Rhodococcus opacus B4	protoporphyrinogen oxidase		F1YMM0_9ACTO Protoporphyrinogen oxidase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15829 PE=4 SV=1;protoporphyrinogen oxidase [Corynebacterium glutami cum ATCC 13032];catalyzes the formation of protoporphyrin IX from protoporphyrinogen IX;protoporphyrinogen oxidase; Reviewed
GOHSU18	CDS	GOHSU_18_01490	complement(171854..172879)		uroporphyrinogen decarboxylase [hamap]	hemE [hamap]	MF_00218:uroporphyrinogen decarboxylase:hemE 	RER_28470 :uroporphyrinogen decarboxylase : hemE : 4.1.1.37 : Rhodococcus erythropolis PR4	uroporphyrinogen decarboxylase	hemE	F1YMM1_9ACTO Uroporphyrinogen decarboxylase OS=Gord onia neofelifaecis NRRL B-59395 GN=hemE PE=3 SV=1;uroporphyrinogen decarboxylase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of coproporphyrinogen from uroporphyrinogen III;Uroporphyrinogen decarboxylase (URO-D) is a dimeric cytosolic enzyme that decarboxylates the four acetate side chains of uroporphyrinogen III (uro-III) to create coproporphyrinogen III, without requiring any prosthetic groups or cofactors. This reaction...; cd00717;substrate binding site [chemical binding]
GOHSU18	CDS	GOHSU_18_01500	173401..174021		conserved hypothetical protein [nite]			RER_28460 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMM2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15839 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF3000); pfam11452
GOHSU18	CDS	GOHSU_18_01510	174090..175280		putative ribonuclease D [nite]	 rnd [nite]		RER_28440 :putative ribonuclease D : rnd : 3.1.13.5 : Rhodococcus erythropolis PR4	putative ribonuclease D		F1YMM3_9ACTO 3'-5' exonuclease OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_15844 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Ribonuclease D [Translation, ribosomal structure and biogenesis]; COG0349;DEDDy 3'-5' exonuclease domain of Ribonuclease D and similar proteins; cd06142;putative active site [active];catalytic site [active];putative substrate binding site [chemical binding];HRDC domain; cl02578
GOHSU18	CDS	GOHSU_18_01520	complement(175343..177214)		1-deoxy-D-xylulose-5-phosphate synthase [hamap]	dxs [hamap]	MF_00315:1-deoxy-D-xylulose-5-phosphate synthase:dxs 	ROP_68280 :1-deoxy-D-xylulose-5-phosphate synthase : dxs : 2.2.1.7 : Rhodococcus opacus B4	1-deoxy-D-xylulose-5-phosphate synthase	dxs	F1YMM5_9ACTO 1-deoxy-D-xylulose-5-phosphate synthas e OS=Gordonia neofelifaecis NRRL B-59395 GN=dxs PE=3 SV=1;1-deoxy-D-xylulose-5-phosphate synthase [Corynebact erium glutamicum ATCC 13032];IS fingerprint type 4-5;1-deoxy-D-xylulose-5-phosphate synthase; Provisional;catalyzes the formation of 1-deoxy-D-xylulose 5-phosphate from pyruvate and D-glyceraldehyde 3-phosphate;Thiamine pyrophosphate (TPP) family, DXS subfamily, TPP-binding module; 1-Deoxy-D-xylulose-5-phosphate synthase (DXS) is a regulatory enzyme of the mevalonate-independent pathway involved in terpenoid biosynthesis. Terpeniods are plant natural products...; cd02007;TPP-binding site;Pyrimidine (PYR) binding domain of 1-deoxy-D-xylulose-5-phosphate synthase (DXS), transketolase (TK), and related proteins; cd07033;PYR/PP interface [polypeptide binding];dimer interface [polypeptide binding];TPP binding site [chemical binding];Transketolase, C-terminal domain; pfam02780
GOHSU18	CDS	GOHSU_18_01530	complement(177282..>177930)	3x codon frame	putative RNA methyltransferase [nite]			RER_28310 :putative RNA methyltransferase : : 2.1.1.- : Rhodococcus erythropolis PR4	SAM-dependent methyltransferase		F1YMM6_9ACTO SAM-dependent methyltransferase OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_15859 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: sat:SYN_01023 SAM-dependent methyltransferase related to tRNA (uracil-5-)-methyltransferase;SAM-dependent methyltransferases related to tRNA (uracil-5-)-methyltransferase [Translation, ribosomal structure and biogenesis]; COG2265;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cl15754
##GOHSU19	contig00081				66609
GOHSU19	CDS	GOHSU_19_00010	<1..143	short	not assigned						
GOHSU19	CDS	GOHSU_19_00020	complement(295..645)		not assigned						
GOHSU19	CDS	GOHSU_19_00030	complement(1190..1456)		shikimate 5-dehydrogenase [migap]				shikimate 5-dehydrogenase		G4MI52_9BURK Shikimate dehydrogenase OS=Candidatus Burkholderia kirkii UZHbot1 GN=aroE PE=3 SV=1;unnamed protein product [Burkholderia sp. YI23];shikimate 5-dehydrogenase; Reviewed; PRK00258;NAD(P) binding domain of Shikimate dehydrogenase; cd01065;shikimate binding site;NAD(P) binding site [chemical binding]
GOHSU19	CDS	GOHSU_19_00040	complement(1453..1758)		HpcH/HpaI aldolase [migap]				HpcH/HpaI aldolase		HpcH/HpaI aldolase [Methylobacterium sp. 4- 46];PFAM: HpcH/HpaI aldolase; KEGG: mex:Mext_3917 HpcH/HpaI aldolase;Pyruvate kinase (PK):  Large allosteric enzyme that regulates glycolysis through binding of the substrate, phosphoenolpyruvate, and one or more allosteric effectors. Like other allosteric enzymes, PK has a high substrate affinity R state and a low...; cl09155
GOHSU19	CDS	GOHSU_19_00050	complement(1755..1880)	short	not assigned						
GOHSU19	tRNA	GOHSU_19_t0010	complement(2044..2115)		not assigned						identified by tRNAscan-SE; putative
GOHSU19	CDS	GOHSU_19_00060	complement(2230..2985)		3-oxoacyl-[acyl-carrier-protein] reductase [nite]	 fabG [nite]		ROP_20570 :3-oxoacyl-[acyl-carrier-protein] reductase : fabG : 1.1.1.100 : Rhodococcus opacus B4	3-oxoacyl-ACP reductase		E3BBM8_9MICO Oxidoreductase, short chain dehydrogen ase/reductase family protein OS=Dermacoccus sp. Ellin185 G N=HMPREF0321_1218 PE=3 SV=1;3-oxoacyl-ACP reductase [Rhodococcus jostii RHA1];3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05653;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU19	CDS	GOHSU_19_00070	3145..4539		bifunctional protein GlmU [hamap]	glmU [hamap]	MF_01631:bifunctional protein GlmU:glmU 	RER_43100 :bifunctional protein GlmU : glmU : 2.7.7.23/2.3.1.157 : Rhodococcus erythropolis PR4	bifunctional N-acetylglucosamine-1-phosphate uridyltransferase/glucosamine-1-phosphate acetyltransferase	glmU	G7GP14_9ACTO Bifunctional protein GlmU OS=Gordonia amarae NBRC 15530 GN=glmU PE=3 SV=1;bifunctional N-acetylglucosamine-1-phosphate uridyl transferase/glucosamine-1-phosphate acetyltransferase [Cor ynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;forms a homotrimer; catalyzes the acetylation of glucosamine-1-phosphate and uridylation of N-acetylglucosamine-1-phosphate to produce UDP-GlcNAc; function in cell wall synthesis;bifunctional N-acetylglucosamine-1-phosphate uridyltransferase/glucosamine-1-phosphate acetyltransferase; Provisional; PRK14352;N-terminal domain of bacterial GlmU; cd02540;Substrate binding site;Mg++ binding site;N-acetyl-glucosamine-1-phosphate uridyltransferase (GlmU), C-terminal left-handed beta-helix (LbH) acetyltransferase domain: GlmU is also known as UDP-N-acetylglucosamine pyrophosphorylase. It is a bifunctional bacterial enzyme that catalyzes two...; cd03353;substrate binding site [chemical binding];CoA binding site [chemical binding]
GOHSU19	CDS	GOHSU_19_00080	4536..5513		phosphoribosylpyrophosphate synthase [nite]	 prs [nite]		ROP_57880 :phosphoribosylpyrophosphate synthase : prs : 2.7.6.1 : Rhodococcus opacus B4	ribose-phosphate pyrophosphokinase	prsA	F1YKX6_9ACTO Ribose-phosphate pyrophosphokinase OS= Gordonia neofelifaecis NRRL B-59395 GN=prs PE=3 SV=1;ribose-phosphate pyrophosphokinase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 5-phospho-alpha-D-ribose 1-phosphate from D-ribose 5-phosphate and ATP;N-terminal domain of ribose phosphate pyrophosphokinase; pfam13793;ribose-phosphate pyrophosphokinase; Provisional;Phosphoribosyl transferase (PRT)-type I domain; cd06223
GOHSU19	CDS	GOHSU_19_00090	complement(5521..6078)		hypothetical protein [nite]			BBR47_47640 :hypothetical protein : : : Brevibacillus brevis NBRC 100599	hypothetical protein		G7GP16_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_34_00070 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.638) with cleavage site probability 0.330 at residue 35;NTR_like domain; a beta barrel with an oligosaccharide/oligonucleotide-binding fold found in netrins, complement proteins, tissue inhibitors of metalloproteases (TIMP), and procollagen C-proteinase enhancers (PCOLCE), amongst others. In netrins, the...; cl02512
GOHSU19	CDS	GOHSU_19_00100	6114..6461		putative arsenate reductase [nite]	 arsC [nite]		RER_43750 :putative arsenate reductase : arsC : 1.20.4.1 : Rhodococcus erythropolis PR4	arsenate reductase		G7H5Y5_9ACTO Arsenate reductase OS=Gordonia araii N BRC 100433 GN=arsC PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: arsenate reductase; PFAM: arsenate reductase and related; KEGG: bbr:BB2882  arsenate reductase;Arsenate Reductase (ArsC) family, ArsC subfamily; arsenic reductases similar to that encoded by arsC on the R733 plasmid of Escherichia coli. E. coli ArsC catalyzes the reduction of arsenate [As(V)] to arsenite [As(III)], the first step in the...; cd03034;ArsC family; pfam03960;catalytic residues [active]
GOHSU19	CDS	GOHSU_19_00110	complement(6501..6998)		conserved hypothetical protein [nite]			RER_42470 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	limonene-12-epoxide hydrolase		F1YKX2_9ACTO Limonene-12-epoxide hydrolase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_12797 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: Limonene-12-epoxide hydrolase; KEGG: mmi:MMAR_1973 hypothetical protein;Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109
GOHSU19	CDS	GOHSU_19_00120	7154..7777		50S ribosomal protein L25 [hamap]	rplY [hamap]	MF_01334:50S ribosomal protein L25:rplY 	ROP_57640 :50S ribosomal protein L25 : rplY : : Rhodococcus opacus B4	50S ribosomal protein L25	rplY	H5UJ52_9ACTO 50S ribosomal protein L25 OS=Gordonia terrae NBRC 100016 GN=rplY PE=3 SV=1;50S ribosomal protein L25/general stress protein Ct c [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;the Ctc family of proteins consists of two types, one that contains the N-terminal ribosomal protein L25 domain only which in Escherichia coli binds the 5S rRNA while a subset of proteins contain a C-terminal extension that is involved in the stress response;50S ribosomal protein L25/general stress protein Ctc; Reviewed;Ribosomal L25/TL5/CTC N-terminal 5S rRNA binding domain; cd00495;5S rRNA interface [nucleotide binding];CTC domain interface [polypeptide binding];L16 interface [polypeptide binding]
GOHSU19	CDS	GOHSU_19_00130	7849..8346		peptidyl-tRNA hydrolase [nite]	 pth [nite]		ROP_57630 :peptidyl-tRNA hydrolase : pth : 3.1.1.29 : Rhodococcus opacus B4	peptidyl-tRNA hydrolase	pth1	F1YKX0_9ACTO Peptidyl-tRNA hydrolase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=pth PE=3 SV=1;peptidyl-tRNA hydrolase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Enables the recycling of peptidyl-tRNAs produced at termination of translation;Peptidyl-tRNA hydrolase (PTH) is a monomeric protein that cleaves the ester bond linking the nascent peptide and tRNA when peptidyl-tRNA is released prematurely from the ribosome. This ensures the recycling of peptidyl-tRNAs into tRNAs produced through...; cd00462;putative active site [active]
GOHSU19	CDS	GOHSU_19_00140	complement(8412..9200)	overlap	putative esterase [nite]			KSE_25240 :putative esterase : : : Kitasatospora setae NBRC 14216	putative secreted hydrolase		H0IS84_MYCAB Putative secreted hydrolase OS=Mycobac terium abscessus subsp. bolletii BD GN=MBOL_31950 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];SEST_like. A family of secreted SGNH-hydrolases similar to Streptomyces scabies esterase (SEST), a causal agent of the potato scab disease, which hydrolyzes a specific ester bond in suberin, a plant lipid. The tertiary fold of this enzyme is...; cd01823;catalytic triad [active];oxyanion hole [active]
GOHSU19	CDS	GOHSU_19_00150	9094..9447	overlap	not assigned						H3HEM7_STRPU Uncharacterized protein OS=Strongyloce ntrotus purpuratus GN=SPU_000053gn PE=4 SV=1
GOHSU19	CDS	GOHSU_19_00160	9600..10721		3-oxoacyl-[acyl-carrier-protein] synthase III [hamap]	fabH [hamap]	MF_01815:3-oxoacyl-[acyl-carrier-protein] synthase III:fabH 	RER_11360 :3-oxoacyl-[acyl-carrier-protein] synthase III : fabH : 2.3.1.180 : Rhodococcus erythropolis PR4	3-oxoacyl-ACP synthase	fabH3	F1YKW7_9ACTO 3-oxoacyl-[acyl-carrier-protein] synth ase 3 2 OS=Gordonia neofelifaecis NRRL B-59395 GN=fabH2 PE =3 SV=1;3-oxoacyl-ACP synthase [Nocardia farcinica IFM 1015 2];3-oxoacyl-(acyl-carrier-protein) synthase III; TIGR00747;Ketoacyl-acyl carrier protein synthase III (KASIII) initiates the elongation in type II fatty acid synthase systems. It is found in bacteria and plants. Elongation of fatty acids in the type II systems occurs by Claisen condensation of malonyl-acyl...; cd00830;dimer interface [polypeptide binding];CoA binding pocket [chemical binding]
GOHSU19	CDS	GOHSU_19_00170	complement(10754..11065)		putative thioredoxin [nite]	 trxA1 [nite]		KSE_39090 :putative thioredoxin : trxA1 : : Kitasatospora setae NBRC 14216	thioredoxin		G7GKQ1_9ACTO Thioredoxin OS=Gordonia amarae NBRC 15 530 GN=trxA PE=3 SV=1;thioredoxin [Thermoanaerobacterium thermosaccharoly ticum DSM 571];KEGG: tpd:Teth39_0995 thioredoxin; TIGRFAM: thioredoxin; PFAM: thioredoxin;TRX family; composed of two groups: Group I, which includes proteins that exclusively encode a TRX domain; and Group II, which are composed of fusion proteins of TRX and additional domains. Group I TRX is a small ancient protein that alter the redox...; cd02947;catalytic residues [active]
GOHSU19	CDS	GOHSU_19_00180	11141..12160		3-oxoacyl-[acyl-carrier-protein] synthase III [hamap]	fabH [hamap]	MF_01815:3-oxoacyl-[acyl-carrier-protein] synthase III:fabH 	ROP_52560 :3-oxoacyl-[acyl-carrier-protein] synthase III : fabH : 2.3.1.180 : Rhodococcus opacus B4	3-oxoacyl-(acyl-carrier-protein) synthase III		H5U348_9ACTO 3-oxoacyl-[acyl-carrier-protein] synth ase 3 1 OS=Gordonia sputi NBRC 100414 GN=fabH PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: dia:Dtpsy_2633 3-oxoacyl-(acyl-carrier- protein) synthase III; TIGRFAM: 3-oxoacyl-(acyl-carrier-protein) synthase III; PFAM: 3-Oxoacyl-[acyl-carrier-protein (ACP)] synthase III domain protein; 3-Oxoacyl-[acyl-carrier- protein (ACP)] synthase III;3-oxoacyl-(acyl carrier protein) synthase III; Reviewed;Ketoacyl-acyl carrier protein synthase III (KASIII) initiates the elongation in type II fatty acid synthase systems. It is found in bacteria and plants. Elongation of fatty acids in the type II systems occurs by Claisen condensation of malonyl-acyl...; cd00830;dimer interface [polypeptide binding];CoA binding pocket [chemical binding]
GOHSU19	CDS	GOHSU_19_00190	12222..13841		peptide chain release factor 3 [nite]	 prfC [nite]		RER_43280 :peptide chain release factor 3 : prfC : : Rhodococcus erythropolis PR4	peptide chain release factor 3		F1YKW2_9ACTO Peptide chain release factor 3 OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_12737 PE=4 SV=1;peptide chain release factor 3 [Corynebacterium glu tamicum ATCC 13032];GTPase;Peptide chain release factor RF-3 [Translation, ribosomal structure and biogenesis]; COG4108;Release Factor 3 (RF3) protein involved in the terminal step of translocation in bacteria; cd04169;G1 box;putative GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;Translation_Factor_II_like: Elongation factor Tu (EF-Tu) domain II-like proteins. Elongation factor Tu consists of three structural domains, this family represents the second domain. Domain II adopts a beta barrel structure and is involved in binding to...; cl02787
GOHSU19	CDS	GOHSU_19_00200	13841..14536		hypothetical protein [migap]				hypothetical protein		F1YKW1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12732 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU19	CDS	GOHSU_19_00210	complement(14554..15558)		hypothetical protein [nite]			ROP_72050 :hypothetical protein : : : Rhodococcus opacus B4	3-hydroxyisobutyryl-CoA hydrolase		H5UJ43_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_130_00080 PE=4 SV=1;3-hydroxyisobutyryl-CoA hydrolase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;3-hydroxyisobutyryl-CoA hydrolase; Provisional;catalyzes the formation of 3-hydroxy-2-methylpropanoate from 3-hydroxy-2-methylpropanoyl-CoA;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding];Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cl15762
GOHSU19	CDS	GOHSU_19_00220	15736..16623	overlap	putative esterase [nite]			ROP_23390 :putative esterase : : : Rhodococcus opacus B4	hypothetical protein		D0L712_GORB4 Putative uncharacterized protein (Prec ursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43 247 / JCM 3198 / NCTC 10667) GN=Gbro_1522 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];SEST_like. A family of secreted SGNH-hydrolases similar to Streptomyces scabies esterase (SEST), a causal agent of the potato scab disease, which hydrolyzes a specific ester bond in suberin, a plant lipid. The tertiary fold of this enzyme is...; cd01823;catalytic triad [active];oxyanion hole [active]
GOHSU19	CDS	GOHSU_19_00230	16557..17831	overlap	membrane-anchored adenylyl cyclase Cya [migap]	cya_1 [migap]			membrane-anchored adenylyl cyclase Cya	cya_1	G7H5S1_9ACTO Putative adenylate cyclase OS=Gordonia araii NBRC 100433 GN=GOARA_064_01980 PE=3 SV=1;membrane-anchored adenylyl cyclase Cya [Mycobacteri um marinum M];involved in camp synthesis [catalytic activity: ATP = 3',5'-cyclic AMP + diphosphate];Heme NO binding associated; pfam07701;Adenylyl- / guanylyl cyclase, catalytic domain; smart00044;cyclase homology domain; cd07302;nucleotidyl binding site;metal binding site [ion binding];dimer interface [polypeptide binding]
GOHSU19	CDS	GOHSU_19_00240	complement(17839..19704)		putative ABC transporter ATP-binding protein [nite]			ROP_57460 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	ABC transporter ATP-binding subunit		G7H640_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia araii NBRC 100433 GN=GOARA_067_00200 PE =3 SV=1;ABC transporter ATP-binding subunit [Streptomyces c lavuligerus ATCC 27064];ATPase components of ABC transporters with duplicated ATPase domains [General function prediction only]; COG0488;ABCF_EF-3  Elongation factor 3 (EF-3) is a cytosolic protein required by fungal ribosomes for in vitro protein synthesis and for in vivo growth.  EF-3 stimulates the binding of the EF-1: GTP: aa-tRNA ternary complex to the ribosomal A site by facilitated...; cd03221;ABC transporter; pfam12848
GOHSU19	CDS	GOHSU_19_00250	complement(19757..20674)		4-diphosphocytidyl-2-C-methyl-D-erythritol kinase [hamap]	ispE [hamap]	MF_00061:4-diphosphocytidyl-2-C-methyl-D-erythritol kinase:ispE 	ROP_57450 :4-diphosphocytidyl-2-C-methyl-D-erythritol kinase : ispE : 2.7.1.148 : Rhodococcus opacus B4	4-diphosphocytidyl-2-C-methyl-D-erythritol kinase	ispE	F1YKV7_9ACTO 4-diphosphocytidyl-2-C-methyl-D-erythr itol kinase OS=Gordonia neofelifaecis NRRL B-59395 GN=ispE PE=3 SV=1;4-diphosphocytidyl-2-C-methyl-D-erythritol kinase [ Streptomyces clavuligerus ATCC 27064];COG function: Lipid transport and metabolism. KEGG: Metabolic pathways Terpenoid backbone biosynthesis. COG: COG1947 4-diphosphocytidyl-2C-methyl-D-erythritol 2-phosphate synthase.;4-diphosphocytidyl-2-C-methyl-D-erythritol kinase; Provisional
GOHSU19	CDS	GOHSU_19_00260	complement(20671..21597)	case	ribosomal RNA small subunit methyltransferase A [hamap]	rsmA [hamap]	MF_00607:ribosomal RNA small subunit methyltransferase A:rsmA 	RER_43600 :dimethyladenosine transferase : ksgA : 2.1.1.- : Rhodococcus erythropolis PR4	dimethyladenosine transferase	ksgA	F1YKV6_9ACTO Ribosomal RNA small subunit methyltran sferase A OS=Gordonia neofelifaecis NRRL B-59395 GN=rsmA P E=3 SV=1;dimethyladenosine transferase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;catalyzes the transfer of a total of four methyl groups from S-adenosyl-l-methionine (S-AdoMet) to two adjacent adenosine bases A1518 and A1519 in 16S rRNA; mutations in ksgA causes resistance to the translation initiation inhibitor kasugamycin;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU19	CDS	GOHSU_19_00270	complement(21622..22734)		resuscitation-promoting factor [nite]	 rpf [nite]		ROP_57310 :resuscitation-promoting factor : rpf : : Rhodococcus opacus B4	transglycosylase		F1YKV5_9ACTO Transglycosylase domain-containing pro tein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12702 PE=4 SV=1;transglycosylase [Nakamurella multipartita DSM 4423 3];PFAM: transglycosylase domain protein; protein of unknown function DUF348; G5 domain protein; KEGG: AGAP008244-PA;Domain of unknown function (DUF348); pfam03990;G5 domain; pfam07501;lysozyme_like domain.  This contains several members including Soluble Lytic Transglycosylases (SLT), Goose Egg-White Lysozymes (GEWL), Hen Egg-White Lysozymes (HEWL), chitinases, bacteriophage lambda lysozymes, endolysins, autolysins, and chitosanases; cl00222
GOHSU19	CDS	GOHSU_19_00280	complement(22810..23715)		putative deoxyribonuclease [nite]			RER_43660 :putative deoxyribonuclease : : 3.1.-.- : Rhodococcus erythropolis PR4	TatD family hydrolase		F1YKV4_9ACTO Hydrolase, TatD family protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_12697 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: hydrolase, TatD family; PFAM: TatD-related deoxyribonuclease; KEGG: pca:Pcar_1695 Mg-dependent DNase;TatD like proteins;  E.coli TatD is a cytoplasmic protein, shown to have magnesium dependent DNase activity; cd01310
GOHSU19	CDS	GOHSU_19_00290	complement(23712..24611)		putative phenazine biosynthesis protein [nite]			KRH_04450 :putative phenazine biosynthesis protein : : : Kocuria rhizophila DC2201	phenazine biosynthesis PhzC/PhzF protein		F1YKV3_9ACTO PhzF family phenazine biosynthesis pro tein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12692 PE=4 SV=1;phenazine biosynthesis PhzC/PhzF protein [Burkholde ria cenocepacia AU 1054];Predicted epimerase, PhzC/PhzF homolog [General function prediction only];Diaminopimelate epimerase; cl14668;PFAM: Phenazine biosynthesis PhzC/PhzF protein; KEGG: atc:AGR_L_1187 hypothetical protein PA1367
GOHSU19	CDS	GOHSU_19_00300	24663..25367		putative hydrolase [nite]			ROP_54730 :putative hydrolase : : : Rhodococcus opacus B4	beta-phosphoglucomutase family hydrolase		F1YKV0_9ACTO Beta-phosphoglucomutase family hydrola se OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12677 PE =4 SV=1;beta-phosphoglucomutase family hydrolase [Arthrobac ter chlorophenolicus A6];TIGRFAM: HAD-superfamily hydrolase, subfamily IA, variant 3; beta-phosphoglucomutase family hydrolase; PFAM: Haloacid dehalogenase domain protein hydrolase; KEGG: art:Arth_0307 beta-phosphoglucomutase family hydrolase;beta-phosphoglucomutase family hydrolase; TIGR02009;Haloacid dehalogenase-like hydrolases. The haloacid dehalogenase-like (HAD) superfamily includes L-2-haloacid dehalogenase, epoxide hydrolase, phosphoserine phosphatase, phosphomannomutase, phosphoglycolate phosphatase, P-type ATPase, and many others; cd01427;motif II
GOHSU19	CDS	GOHSU_19_00310	25402..27840		putative glycosidase [nite]			ROP_54720 :putative glycosidase : : 3.2.1.- : Rhodococcus opacus B4	glycoside hydrolase family protein		F1YKU9_9ACTO Glycoside hydrolase family 65 central catalytic OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1 2672 PE=4 SV=1;glycoside hydrolase family protein [Geodermatophilu s obscurus DSM 43160];PFAM: glycoside hydrolase family 65 central catalytic; glycoside hydrolase family 65 domain protein; KEGG: rop:ROP_54720 putative glycosidase;Trehalose and maltose hydrolases (possible phosphorylases) [Carbohydrate transport and metabolism]; COG1554;Glycosyl hydrolase family 65, N-terminal domain; pfam03636;Glycosyl hydrolase family 65 central catalytic domain; pfam03632;Glycosyl hydrolase family 65, C-terminal domain; pfam03633
GOHSU19	CDS	GOHSU_19_00320	27865..29403		methionyl-tRNA synthetase [hamap]	metG [hamap]	MF_01228:methionyl-tRNA synthetase:metG 	ROP_57250 :methionyl-tRNA synthetase : metG : 6.1.1.10 : Rhodococcus opacus B4	methionyl-tRNA synthetase		G7GP59_9ACTO Methionyl-tRNA synthetase OS=Gordonia amarae NBRC 15530 GN=metG PE=3 SV=1;methionyl-tRNA synthetase [Brucella pinnipedialis B 2 /94];methionyl-tRNA synthetase; Reviewed;catalytic core domain of methioninyl-tRNA synthetases; cd00814;HIGH motif;KMSKS motif;Anticodon-binding domain of methionyl tRNA synthetases; cd07957;tRNA binding surface [nucleotide binding];anticodon binding site
GOHSU19	CDS	GOHSU_19_00330	29414..30592		para-aminobenzoate synthase [nite]	 pabAB [nite]		RER_18480 :para-aminobenzoate synthase : pabAB : 2.6.1.85 : Rhodococcus erythropolis PR4	para-aminobenzoate synthase component I		H5U316_9ACTO Putative para-aminobenzoate synthase c omponent I OS=Gordonia sputi NBRC 100414 GN=GOSPT_089_0009 0 PE=4 SV=1;para-aminobenzoate synthase component I [Mycobacter ium tuberculosis 94_M4241A];truncated CDS;chorismate binding enzyme; cl10555;aminodeoxychorismate synthase, component I, clade 2; TIGR01824
GOHSU19	CDS	GOHSU_19_00340	complement(30649..32274)		hypothetical protein [nite]			ROP_37430 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YHW7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07408 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF222;Domain of unknown function (DUF222); pfam02720;HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and...; cd00085
GOHSU19	CDS	GOHSU_19_00350	complement(32377..33213)		conserved hypothetical protein [nite]			RER_43690 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	tetrapyrrole methylase		F1YKU5_9ACTO Uroporphyrin-III C/tetrapyrrole (Corri n /Porphyrin) methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12652 PE=4 SV=1;tetrapyrrole methylase [Streptomyces clavuligerus A TCC 27064];COG:COG0313 Predicted methyltransferases.;Precorrin-3B methylase [Coenzyme metabolism]; cl00304;Predicted methyltransferases [General function prediction only]
GOHSU19	CDS	GOHSU_19_00360	33311..34834		protein O-mannosyltransferase [nite]	 pmt [nite]		RER_43700 :protein O-mannosyltransferase : pmt : 2.4.1.- : Rhodococcus erythropolis PR4	glycosyl transferase family protein		F1YKU4_9ACTO Glycosyl transferase family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12647 PE=4 SV =1;unnamed protein product [Streptomyces scabiei 87.22 ];Dolichyl-phosphate-mannose-protein mannosyltransferase; cl15769;Dolichyl-phosphate-mannose--protein O-mannosyl transferase [Posttranslational modification, protein turnover, chaperones]; COG1928
GOHSU19	CDS	GOHSU_19_00370	complement(34847..35542)		hypothetical membrane protein [nite]			RER_52170 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4			H5TTM7_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_240_00150 PE=4 SV=1
GOHSU19	CDS	GOHSU_19_00380	35563..37086		hypothetical protein [nite]			SLG_25290 :hypothetical protein : : : Sphingobium sp. SYK-6	aminoglycoside phosphotransferase		H5U412_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_103_00470 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: aminoglycoside phosphotransferase; KEGG: mlo:mll6593 hypothetical protein;Uncharacterized protein conserved in bacteria [Function unknown];Predicted kinase [General function prediction only]
GOHSU19	CDS	GOHSU_19_00390	complement(37150..37473)		hypothetical protein [migap]				hypothetical protein		F1YKU1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12632 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU19	CDS	GOHSU_19_00400	37616..38485		putative Usp family protein [nite]			ROP_19720 :putative Usp family protein : : : Rhodococcus opacus B4	universal stress protein		F1YKU2_9ACTO UspA domain-containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_12637 PE=4 SV=1;universal stress protein [Rhodococcus jostii RHA1];Adenine nucleotide alpha hydrolases superfamily including N type ATP PPases, ATP sulphurylases Universal Stress Response protein and electron transfer flavoprotein (ETF). The domain forms a apha/beta/apha fold which  binds to Adenosine nucleotide; cl00292;Usp: Universal stress protein family. The universal stress protein Usp is a small cytoplasmic bacterial protein whose expression is enhanced when the cell is exposed to stress agents. Usp enhances the rate of cell survival during prolonged exposure to...; cd00293;Ligand Binding Site [chemical binding]
GOHSU19	CDS	GOHSU_19_00410	38538..39743		putative MFS transporter [nite]			RER_23290 :putative MFS transporter : : : Rhodococcus erythropolis PR4	MFS transporter		F1YKU3_9ACTO Major facilitator superfamily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12642 PE=4 SV=1;MFS transporter [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;Major Facilitator Superfamily; pfam07690;putative substrate translocation pore
GOHSU19	CDS	GOHSU_19_00420	complement(39754..41469)		putative solute/hydrogen antiporter [nite]			MLP_45150 :putative solute/hydrogen antiporter : : : Microlunatus phosphovorus NM-1	putative sodium/hydrogen exchanger		F1YKU0_9ACTO Sodium/hydrogen exchanger OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12627 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Na+/H+ antiporter, bacterial form; TIGR00831
GOHSU19	CDS	GOHSU_19_00430	41630..42643		conserved hypothetical protein [nite]			RER_56300 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0QUS2_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_006_00260 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mab:MAB_0929 hypothetical protein;Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cl00264
GOHSU19	CDS	GOHSU_19_00440	42842..44296		putative ferredoxin--NADP(+) reductase [nite]			RER_56310 :putative ferredoxin--NADP(+) reductase : : 1.18.1.2 : Rhodococcus erythropolis PR4	putative ferredoxin--NADP(+) reductase		F1YKT8_9ACTO 4Fe-4S ferredoxin iron-sulfur binding domain-containing protein OS=Gordonia neofelifaecis NRRL B -59395 GN=SCNU_12617 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Ferredoxin [Energy production and conversion];4Fe-4S binding domain; cl02805;ferredoxin-NADP+ reductase
GOHSU19	CDS	GOHSU_19_00450	44592..44879		degV family protein [migap]				degV family protein		G7H167_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_043_00020 PE=4 SV=1;degV family protein [Clostridium phytofermentans IS Dg];TIGRFAM: degV family protein; PFAM: DegV family protein; KEGG: ctc:CTC02002 DegV protein;Uncharacterized protein, DegV family COG1307; cl00577
GOHSU19	CDS	GOHSU_19_00460	44876..46531		protein of unknown function DUF1023 [migap]				protein of unknown function DUF1023		F1YKT6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12607 PE=4 SV=1;protein of unknown function DUF1023 [Streptomyces g riseus XylebKG- 1];PFAM: Protein of unknown function DUF1023; KEGG: sgr:SGR_5126 putative secreted protein;Alpha/beta hydrolase; pfam06259
GOHSU19	CDS	GOHSU_19_00470	46613..47698		hypothetical protein [nite]			ROP_52660 :hypothetical protein : : : Rhodococcus opacus B4	lipoprotein		F1YKT4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12597 PE=4 SV=1;lipoprotein [Rhodococcus equi 103S];Protein of unknown function DUF262; cl14890;Excalibur calcium-binding domain; cl05460
GOHSU19	tRNA	GOHSU_19_t0020	complement(48293..48368)		not assigned						identified by tRNAscan-SE; putative
GOHSU19	CDS	GOHSU_19_00480	48462..49202		hypothetical protein [nite]			RER_41820 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YKS7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12562 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU19	CDS	GOHSU_19_00490	complement(49199..49378)		conserved hypothetical protein [nite]			RER_36550 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7H010_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_031_00100 PE=4 SV=1;unnamed protein product [Corynebacterium diphtheria e INCA 402]
GOHSU19	CDS	GOHSU_19_00500	complement(49389..50489)		hypothetical protein [migap]				hypothetical protein		F9VST0_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_038_00200 PE=4 SV= 1;unnamed protein product [Mycobacterium ulcerans Agy 99];membrane protein
GOHSU19	CDS	GOHSU_19_00510	50676..51395		putative ABC transporter ATP-binding protein [nite]			ROP_06750 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	multidrug ABC transporter ATPase		H0QWC4_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia effusa NBRC 100432 GN=GOEFS_019_00380 P E=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: ABC transporter;ABC-type Mn/Zn transport systems, ATPase component [Inorganic ion transport and metabolism]; COG1121;This family of ATP-binding proteins belongs to a multisubunit transporter involved in drug resistance (BcrA and DrrA), nodulation, lipid transport, and lantibiotic immunity.  In bacteria and archaea, these transporters usually include an ATP-binding...; cd03230;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU19	CDS	GOHSU_19_00520	51392..52150		putative ABC transporter permease protein [nite]			ROP_06760 :putative ABC transporter permease protein : : : Rhodococcus opacus B4	ABC transporter efflux protein, DrrB family		E4WHW1_RHOE1 Putative ABC-2 type transporter integr al membrane subunit OS=Rhodococcus equi (strain 103S) GN=R EQ_24790 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: ABC-2 type transporter; TIGRFAM: ABC transporter efflux protein, DrrB family;ABC-2 type transporter; cl11417
GOHSU19	CDS	GOHSU_19_00530	complement(52173..52829)		putative acetyltransferase [nite]			RER_43870 :putative acetyltransferase : : 2.3.1.- : Rhodococcus erythropolis PR4	acetyltransferase		H6N3F1_GORPV Putative ribosomal-protein-alanine N-a cetyltransferase OS=Gordonia polyisoprenivorans (strain DS M 44266 / VH2) GN=GPOL_c12520 PE=4 SV=1;acetyltransferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU19	CDS	GOHSU_19_00540	complement(52841..54145)		molybdopterin biosynthesis protein MoeA [nite]	 moeA [nite]		RER_43880 :molybdopterin biosynthesis protein MoeA : moeA : : Rhodococcus erythropolis PR4	molybdenum cofactor synthesis domain-containing protein		F1YKS3_9ACTO Molybdopterin biosynthesis protein Moe A1 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12542 PE =4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: molybdenum cofactor synthesis domain protein; PFAM: MoeA domain protein domain I and II; MoeA domain protein domain IV; molybdopterin binding domain; KEGG: hha:Hhal_0242 molybdenum cofactor synthesis domain-containing protein;Molybdopterin biosynthesis enzyme [Coenzyme metabolism]; COG0303;MoeA family. Members of this family are involved in biosynthesis of the molybdenum cofactor (MoCF), an essential cofactor of a diverse group of redox enzymes. MoCF biosynthesis is an evolutionarily conserved pathway present in eubacteria, archaea and...; cd00887;dimer interface [polypeptide binding];putative functional site;putative MPT binding site
GOHSU19	CDS	GOHSU_19_00550	complement(54252..55145)		UTP--glucose-1-phosphate uridylyltransferase [nite]	 galU [nite]		RER_43890 :UTP--glucose-1-phosphate uridylyltransferase : galU : 2.7.7.9 : Rhodococcus erythropolis PR4	UTP--glucose-1-phosphate uridylyltransferase	galU	H6N3E9_GORPV Nucleotidyl transferase OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c12500 P E=4 SV=1;UTP--glucose-1-phosphate uridylyltransferase [Rhodo coccus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;protein synonym:UDP-glucose pyrophosphorylase;Prokaryotic UGPase catalyses the synthesis of UDP-glucose; cd02541;tetramer interface
GOHSU19	CDS	GOHSU_19_00560	55207..55788		conserved hypothetical protein [nite]			RER_43900 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	5-formyltetrahydrofolate cyclo-ligase		F1YKS2_9ACTO 5-formyltetrahydrofolate cyclo-ligase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12527 PE=4 SV=1;5-formyltetrahydrofolate cyclo-ligase [Corynebacter ium glutamicum ATCC 13032];IS fingerprint type 4-5;5-formyltetrahydrofolate cyclo-ligase family; cl00360
GOHSU19	CDS	GOHSU_19_00570	complement(55792..56142)		regulatory protein, FmdB family [migap]				regulatory protein, FmdB family		F1YKS1_9ACTO Regulatory protein, FmdB family OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_12522 PE=4 SV=1;regulatory protein, FmdB family [[Cellvibrio] gilvu s ATCC 13127];Zinc ribbon domain; cl00993;KEGG: bcv:Bcav_0902 regulatory protein, FmdB family; TIGRFAM: regulatory protein, FmdB family; PFAM: Putative regulatory protein FmdB
GOHSU19	CDS	GOHSU_19_00580	56258..57817		putative sodium/alanine symporter [nite]			OBV_41010 :putative sodium/alanine symporter : : : Oscillibacter valericigenes Sjm18-20	amino acid carrier protein		G7H3P5_9ACTO Sodium/amino acid symporter OS=Gordoni a araii NBRC 100433 GN=GOARA_056_02180 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Na+/alanine symporter [Amino acid transport and metabolism]; COG1115;identified by match to protein family HMM PF01235; match to protein family HMM TIGR00835;Sodium:alanine symporter family; cl00548
GOHSU19	CDS	GOHSU_19_00590	complement(57807..59042)		Na(+)/H(+) antiporter NhaA [hamap]	nhaA [hamap]	MF_01844:Na(+)/H(+) antiporter NhaA:nhaA 	ROP_00220 :Na(+)/H(+) antiporter : nhaA : : Rhodococcus opacus B4	Na+/H+ antiporter NhaA		F1YKS0_9ACTO Na(+)/H(+) antiporter nhaA 1 OS=Gordon ia neofelifaecis NRRL B-59395 GN=nhaA1 PE=3 SV=1;Na+/H+ antiporter NhaA [Saccharomonospora cyanea NA - 134];Signal predicted by SignalP 3.0 HMM; IMG reference gene:2508561055_SP;PFAM: Na+/H+ antiporter 1; TIGRFAM: Na+/H+ antiporter NhaA;Na+/H+ antiporter 1; cl01133
GOHSU19	CDS	GOHSU_19_00600	59314..59862		conserved hypothetical protein [nite]			RER_43920 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	SAF domain-containing protein		F1YKR8_9ACTO SAF domain-containing protein OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_12507 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: SAF domain protein; KEGG: ade:Adeh_2828 Flp pilus assembly CpaB;SAF domain; cl00555
GOHSU19	CDS	GOHSU_19_00610	59877..60464		large-conductance mechanosensitive channel [nite]	 mscL [nite]		ROP_56930 :large-conductance mechanosensitive channel : mscL : : Rhodococcus opacus B4	putative large conductance mechanosensitive channel protein		H0RJY1_9ACTO Large-conductance mechanosensitive cha nnel OS=Gordonia polyisoprenivorans NBRC 16320 GN=mscL PE= 4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Large-conductance mechanosensitive channel, MscL; cl00860;large-conductance mechanosensitive channel; Reviewed; PRK00567
GOHSU19	CDS	GOHSU_19_00620	60590..61267		putative heme oxygenase [nite]	 hmuO [nite]		RER_32040 :putative heme oxygenase : hmuO : 1.14.99.3 : Rhodococcus erythropolis PR4	heme oxygenase		E9V246_9ACTO Heme oxygenase OS=Nocardioidaceae bact erium Broad-1 GN=NBCG_05143 PE=4 SV=1;unnamed protein product [Nostoc sp. PCC 7120];ORF_ID:alr3125;Heme oxygenase catalyzes the rate limiting step in the degradation of heme to bilirubin, it is essential for recycling of iron from heme. Heme is used as a substrate and cofactor for its own degradation to biliverdin, iron, and carbon monoxide. This...; cd00232;heme binding pocket [chemical binding];heme ligand [chemical binding]
GOHSU19	CDS	GOHSU_19_00630	complement(61284..62375)		putative lyase [nite]			MLP_22980 :putative lyase : : : Microlunatus phosphovorus NM-1	pyridoxal-phosphate dependent enzyme		F1YPM4_9ACTO Pyridoxal-phosphate dependent enzyme O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_19405 PE=4 S V=1;pyridoxal-phosphate dependent enzyme [Arthrobacter aurescens TC1];identified by match to protein family HMM PF00291;Cysteine synthase [Amino acid transport and metabolism]; COG0031;Tryptophan synthase beta superfamily (fold type II); this family of pyridoxal phosphate (PLP)-dependent enzymes catalyzes beta-replacement and beta-elimination reactions. This CD corresponds to aminocyclopropane-1-carboxylate deaminase (ACCD), tryptophan...; cl00342;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU19	CDS	GOHSU_19_00640	complement(62430..63137)		hypothetical protein [nite]			ROP_56900 :hypothetical protein : : : Rhodococcus opacus B4	MspA family protein		F1YKR5_9ACTO MspA family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_12492 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.999) with cleavage site probability 0.379 at residue 28;PFAM: MspA family protein;MspA; pfam09203
GOHSU19	CDS	GOHSU_19_00650	complement(63144..63827)		hypothetical protein [nite]			ROP_56900 :hypothetical protein : : : Rhodococcus opacus B4	MspA family protein		F1YKR5_9ACTO MspA family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_12492 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.999) with cleavage site probability 0.379 at residue 28;PFAM: MspA family protein;MspA; pfam09203
GOHSU19	CDS	GOHSU_19_00660	complement(63854..64477)		hypothetical protein [nite]			ROP_30150 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YKR4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12487 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: bbr:BB0450 autotransporter;MspA; pfam09203
GOHSU19	CDS	GOHSU_19_00670	complement(64692..64892)		predicted protein [migap]				predicted protein		C8X7S3_NAKMY Putative uncharacterized protein OS=Na kamurella multipartita (strain ATCC 700099 / DSM 44233 / J CM 9543 / Y-104) GN=Namu_4649 PE=4 SV=1;predicted protein [Streptomyces sp. AA4]
GOHSU19	CDS	GOHSU_19_00680	complement(64882..65412)		putative molybdenum cofactor biosynthesis protein [nite]			ROP_56880 :putative molybdenum cofactor biosynthesis protein : : : Rhodococcus opacus B4	molybdenum cofactor synthesis domain-containing protein		G7GKB2_9ACTO Putative molybdenum cofactor biosynthe sis protein OS=Gordonia amarae NBRC 15530 GN=GOAMR_10_0014 0 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: molybdenum cofactor synthesis domain protein; PFAM: molybdopterin binding domain; KEGG: sat:SYN_02931  cytoplasmic protein;MogA_MoaB family. Members of this family are involved in biosynthesis of the molybdenum cofactor (MoCF) an essential cofactor of a diverse group of redox enzymes. MoCF biosynthesis is an evolutionarily conserved pathway present in eubacteria, archaea; cd00886;MPT binding site;trimer interface [polypeptide binding]
GOHSU19	CDS	GOHSU_19_00690	complement(65430..>66609)	3x codon frame	putative S1 family peptidase [nite]			RER_43960 :putative S1 family peptidase : : 3.4.21.- : Rhodococcus erythropolis PR4	trypsin-like serine protease with C-terminal PDZ domain		F1YKR2_9ACTO Peptidase S1 and S6 chymotrypsin/Hap O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12477 PE=4 S V=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Trypsin;Trypsin-like serine protease; Many of these are synthesized as inactive precursor zymogens that are cleaved during limited proteolysis to generate their active forms. Alignment contains also inactive enzymes that have substitutions of the catalytic triad...; cl00149;PDZ domain, also called DHR (Dlg homologous region) or GLGF (after a conserved sequence motif). Many PDZ domains bind C-terminal polypeptides, though binding to internal (non-C-terminal) polypeptides and even to lipids has been demonstrated; cl00117
##GOHSU20	contig00082				48096
GOHSU20	CDS	GOHSU_20_00010	complement(<1..143)	short	not assigned						
GOHSU20	CDS	GOHSU_20_00020	complement(223..1638)		two-component histidine kinase MprB [nite]	 mprB [nite]		ROP_56860 :two-component histidine kinase MprB : mprB : 2.7.13.- : Rhodococcus opacus B4	histidine kinase		H6N3D8_GORPV Two-component system sensor kinase OS= Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GP OL_c12390 PE=4 SV=1;histidine kinase [Catenulispora acidiphila DSM 4492 8];PFAM: ATP-binding region ATPase domain protein; histidine kinase A domain protein; histidine kinase HAMP region domain protein; SMART: ATP-binding region ATPase domain protein; histidine kinase HAMP region domain protein; histidine kinase A domain protein; KEGG: ace:Acel_1594 periplasmic sensor signal transduction histidine kinase;Histidine kinase, Adenylyl cyclase, Methyl-accepting protein, and Phosphatase (HAMP) domain. HAMP is a signaling domain which occurs in a wide variety of signaling proteins, many of which are bacterial. The HAMP domain consists of two alpha helices...; cd06225;dimerization interface [polypeptide binding];Histidine Kinase A (dimerization/phosphoacceptor) domain; Histidine Kinase A dimers are formed through parallel association of 2 domains creating 4-helix bundles; usually these domains contain a conserved His residue and are activated via...; cd00082;dimer interface [polypeptide binding];phosphorylation site [posttranslational modification];Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU20	CDS	GOHSU_20_00030	complement(1645..2331)		two-component response regulator MprA [nite]	 mprA [nite]		RER_43980 :two-component response regulator MprA : mprA : : Rhodococcus erythropolis PR4	response regulator with CheY-like receiver domain and winged-helix DNA-binding domain		H8ITN1_MYCIA Two component response transcriptional regulatory protein MprA OS=Mycobacterium intracellulare ( strain ATCC 13950 / DSM 43223 / JCM 6384 / NCTC 13025 / 36 00) GN=OCU_09660 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Response regulators consisting of a CheY-like receiver domain and a winged-helix DNA-binding domain [Signal transduction mechanisms / Transcription]; COG0745;PFAM: Response regulator receiver domain; Transcriptional regulatory protein, C terminal; manually curated;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];Effector domain of response regulator. Bacteria and certain eukaryotes like protozoa and higher plants use two-component signal transduction systems to detect and respond to changes in the environment. The system consists of a sensor histidine kinase and...; cd00383;DNA binding site [nucleotide binding]
GOHSU20	CDS	GOHSU_20_00040	2330..2554		not assigned						
GOHSU20	CDS	GOHSU_20_00050	complement(2617..4209)		probable ferredoxin--NADP(+) reductase [nite]			RER_49700 :probable ferredoxin--NADP(+) reductase : : 1.18.1.2 : Rhodococcus erythropolis PR4	ferredoxin--NADP(+) reductase		G7GX51_9ACTO Putative ferredoxin--NADP(+) reductase OS=Gordonia araii NBRC 100433 GN=GOARA_004_00160 PE=4 SV= 1;ferredoxin--NADP(+) reductase [Rhodococcus erythrop olis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Ferredoxin [Energy production and conversion];4Fe-4S binding domain; cl02805;ferredoxin-NADP+ reductase
GOHSU20	CDS	GOHSU_20_00060	complement(4219..5160)		hypothetical protein [nite]			ROP_19170 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GX50_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_004_00150 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cl00264
GOHSU20	CDS	GOHSU_20_00070	5310..6080		putative oxidoreductase [nite]			RER_14020 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		F1YKQ7_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12452 PE=3 SV=1;short chain dehydrogenase [Mycobacterium avium 104] ;identified by match to protein family HMM PF00106;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Short-chain alcohol dehydrogenase of unknown specificity [General function prediction only];NAD(P) binding site [chemical binding]
GOHSU20	CDS	GOHSU_20_00080	complement(6177..6350)	case	50S ribosomal protein L32 [hamap]	rpmF [hamap]	MF_00340:50S ribosomal protein L32:rpmF 	ROP_04720 :50S ribosomal protein L32 : rpmF : : Rhodococcus opacus B4	50S ribosomal protein L32	rpmF	F1YKQ3_9ACTO 50S ribosomal protein L32 OS=Gordonia neofelifaecis NRRL B-59395 GN=rpmF PE=3 SV=1;50S ribosomal protein L32 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosomal L32p protein family; cl09115;some L32 proteins have zinc finger motifs consisting of CXXC while others do not
GOHSU20	CDS	GOHSU_20_00090	6504..7436		putative AEC family transporter [nite]			ROP_56750 :putative AEC family transporter : : : Rhodococcus opacus B4	permease		F1YKQ2_9ACTO Auxin Efflux Carrier OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_12427 PE=4 SV=1;permease [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Membrane transport protein; cl09117
GOHSU20	CDS	GOHSU_20_00100	complement(7502..8164)		hypothetical protein [nite]			MLP_05460 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YKQ1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12422 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: ilo:IL0624 hypothetical protein;Domain of unknown function (DUF1707); pfam08044;Cell wall-active antibiotics response protein (DUF2154); pfam09922
GOHSU20	CDS	GOHSU_20_00110	complement(8195..10180)		hypothetical protein [nite]			ROP_56550 :hypothetical protein : : : Rhodococcus opacus B4	von Willebrand factor type A		F1YKP8_9ACTO von Willebrand factor type A OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_12407 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];SMART: von Willebrand factor type A; KEGG: myosin heavy chain;Uncharacterized protein with a von Willebrand factor type A (vWA) domain [General function prediction only];Von Willebrand factor type A (vWA) domain was originally found in the blood coagulation protein von Willebrand factor (vWF). Typically, the vWA domain is made up of approximately 200 amino acid residues folded into a classic a/b para-rossmann type of...; cd00198;metal ion-dependent adhesion site (MIDAS)
GOHSU20	CDS	GOHSU_20_00120	complement(10177..11637)		hypothetical protein [nite]			ROP_56540 :hypothetical protein : : : Rhodococcus opacus B4	Mg-chelatase subunit ChlI		F1YKP7_9ACTO Mg-chelatase subunit ChlI-like protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12402 PE=4 SV=1;Mg-chelatase subunit ChlI [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Mg-chelatase subunit ChlI [Coenzyme metabolism]; COG1239
GOHSU20	CDS	GOHSU_20_00130	11691..12272		hypothetical protein [nite]			RER_03520 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YKP6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12397 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGR03086 family protein
GOHSU20	CDS	GOHSU_20_00140	complement(12460..12741)		conserved hypothetical protein [nite]			RER_40910 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	sec-C motif domain-containing protein		F1YKN7_9ACTO Sec-C motif domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12352 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: lhk:LHK_01817 sec-C motif domain protein;Uncharacterized protein conserved in bacteria [Function unknown]
GOHSU20	CDS	GOHSU_20_00150	13009..13530		hypothetical protein [nite]			KRH_17610 :hypothetical protein : : : Kocuria rhizophila DC2201	MutT/NUDIX-family protein		A7A8J8_BIFAD Putative uncharacterized protein OS=Bi fidobacterium adolescentis L2-32 GN=BIFADO_02189 PE=3 SV=1 ;MutT/NUDIX-family protein [Streptomyces clavuligeru s ATCC 27064];Members of the Nudix hydrolase superfamily catalyze the hydrolysis of NUcleoside DIphosphates linked to other moieties, X. Enzymes belonging to this superfamily require a divalent cation, such as Mg2+ or Mn2+, for their activity and contain a highly...; cd04676;nudix motif
GOHSU20	CDS	GOHSU_20_00160	13622..14104		conserved hypothetical protein [nite]			RER_15190 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H6RA27_NOCCG Uncharacterized protein OS=Nocardia cy riacigeorgica (strain GUH-2) GN=NOCYR_1643 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];Evidence 4 : Homologs of previously reported genes of unknown function
GOHSU20	CDS	GOHSU_20_00170	complement(14204..15148)		hypothetical protein [nite]			MLP_42990 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YI33_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07738 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF4095); pfam13338
GOHSU20	CDS	GOHSU_20_00180	complement(15288..16481)		conserved hypothetical protein [nite]			RER_38450 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YKK2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12177 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mxa:MXAN_4606 carbohydrate-binding protein;Domain of unknown function (DUF4185); pfam13810
GOHSU20	CDS	GOHSU_20_00190	complement(16543..17229)		hypothetical membrane protein [nite]			ROP_52100 :hypothetical membrane protein : : : Rhodococcus opacus B4	transporter		H5TPU5_9ACTO Putative uncharacterized protein (Frag ment) OS=Gordonia otitidis NBRC 100426 GN=GOOTI_169_00010 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF01925;Sulfite exporter TauE/SafE; pfam01925
GOHSU20	CDS	GOHSU_20_00200	complement(17279..18853)		bifunctional purine biosynthesis protein PurH [hamap]	purH [hamap]	MF_00139:bifunctional purine biosynthesis protein PurH:purH 	ROP_56480 :phosphoribosylaminoimidazolecarboxamide formyltransferase/IMP cyclohydrolase : purH : 2.1.2.3/3.5.4.10 : Rhodococcus opacus B4	bifunctional phosphoribosylaminoimidazolecarboxamide formyltransferase/IMP cyclohydrolase	purH	F1YKN3_9ACTO Bifunctional purine biosynthesis prote in PurH OS=Gordonia neofelifaecis NRRL B-59395 GN=purH PE= 3 SV=1;bifunctional phosphoribosylaminoimidazolecarboxamid e formyltransferase/IMP cyclohydrolase [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;involved in de novo purine biosynthesis;bifunctional phosphoribosylaminoimidazolecarboxamide formyltransferase/IMP cyclohydrolase; Provisional; PRK00881;Inosine monophosphate cyclohydrolase domain. This is the N-terminal domain in the purine biosynthesis pathway protein ATIC (purH). The bifunctional ATIC protein contains a C-terminal  ATIC formylase domain that formylates...; cd01421;purine monophosphate binding site [chemical binding];dimer interface [polypeptide binding];putative catalytic residues [active];AICARFT/IMPCHase bienzyme; cl03362
GOHSU20	CDS	GOHSU_20_00210	complement(18865..19485)		glycinamide ribonucleotide transformylase PurN [nite]	 purN [nite]		RER_44240 :glycinamide ribonucleotide transformylase PurN : purN : 2.1.2.2 : Rhodococcus erythropolis PR4	phosphoribosylglycinamide formyltransferase		D0L693_GORB4 Phosphoribosylglycinamide formyltransf erase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 432 47 / JCM 3198 / NCTC 10667) GN=Gbro_1366 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: phosphoribosylglycinamide formyltransferase; PFAM: formyl transferase; KEGG: dvm:DvMF_2769 phosphoribosylglycinamide formyltransferase;phosphoribosylglycinamide formyltransferase; Reviewed; PRK05647;Phosphoribosylglycinamide formyltransferase (GAR transformylase, GART); cd08645;substrate binding site [chemical binding];cosubstrate binding site;catalytic site [active]
GOHSU20	CDS	GOHSU_20_00220	complement(19495..21018)		hypothetical membrane protein [nite]			RER_44250 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YKN0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12317 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU20	CDS	GOHSU_20_00230	complement(21154..22221)		hypothetical protein [migap]				hypothetical protein		H5ULM9_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_175_01180 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: eba:ebA2155 hypothetical protein
GOHSU20	CDS	GOHSU_20_00240	complement(22294..24228)		acetoacetyl-CoA synthetase [nite]	 acsA [nite]		ROP_61570 :acetoacetyl-CoA synthetase : acsA : 6.2.1.16 : Rhodococcus opacus B4	acetoacetyl-CoA synthetase		F1YKM8_9ACTO Acetoacetyl-CoA synthase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_12307 PE=4 SV=1;acetoacetyl-CoA synthetase [Rhodococcus jostii RHA1 ];AcsA; in Sinorhizobium meliloti this enzyme is required for acetoacetate activation; similar to acetyl-CoA synthase;acetoacetyl-CoA synthetase; Provisional;Domain of unknown function (DUF3448); pfam11930;AMP-binding enzyme; cl15778
GOHSU20	CDS	GOHSU_20_00250	complement(24215..25867)		putative fatty-acid--CoA ligase [nite]			KRH_22800 :putative fatty-acid--CoA ligase : : 6.2.1.- : Kocuria rhizophila DC2201	acyl-CoA synthetase		F1YKM7_9ACTO AMP-dependent synthetase and ligase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12302 PE=4 SV =1;unnamed protein product [Mycobacterium rhodesiae NB B3];AMP-binding domain protein; Validated;PFAM: AMP-binding enzyme;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU20	CDS	GOHSU_20_00260	complement(25909..26754)		putative citrate lyase beta subunit [nite]			RER_18340 :putative citrate lyase beta subunit : : : Rhodococcus erythropolis PR4	citrate lyase subunit beta-like protein		H6N2E5_GORPV Citrate lyase subunit beta-like protei n OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c12000 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Pyruvate kinase (PK):  Large allosteric enzyme that regulates glycolysis through binding of the substrate, phosphoenolpyruvate, and one or more allosteric effectors. Like other allosteric enzymes, PK has a high substrate affinity R state and a low...; cl09155
GOHSU20	CDS	GOHSU_20_00270	complement(26754..27248)		conserved hypothetical protein [nite]			RER_18330 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7H3J1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_056_01640 PE=4 SV=1;unnamed protein product [Thermobifida fusca YX];FkbR2 is a Streptomyces hygroscopicus protein with a hot dog fold that belongs to a conserved family of proteins found in prokaryotes and archaea but not in eukaryotes. FkbR2  has sequence similarity to (R)-specific enoyl-CoA hydratase, the peroxisomal...; cd03451;putative active site [active];putative catalytic site [active]
GOHSU20	CDS	GOHSU_20_00280	complement(27245..28393)		acyl-CoA dehydrogenase [nite]	 fadE [nite]		ROP_61540 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase domain-containing protein		F1YKM4_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 12287 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;PFAM: acyl-CoA dehydrogenase domain protein; acyl- CoA dehydrogenase type 2 domain; KEGG: dat:HRM2_10010 Acd3;Acyl-CoA dehydrogenase; cl09933
GOHSU20	CDS	GOHSU_20_00290	complement(28411..30411)		probable acyl-CoA carboxylase alpha chain [nite]	 accA [nite]		RER_18310 :probable acyl-CoA carboxylase alpha chain : accA : 6.4.1.- : Rhodococcus erythropolis PR4	acetyl-/propionyl-coenzyme A carboxylase subunit alpha	accC	G7H3I9_9ACTO Acyl-CoA carboxylase alpha chain OS=Go rdonia araii NBRC 100433 GN=accA PE=4 SV=1;accC gene product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 2b : Function of strongly homologous gene; PubMedId : 7592499; Product type e : enzyme;Acetyl/propionyl-CoA carboxylase, alpha subunit [Lipid metabolism];Carbamoyl-phosphate synthase L chain, N-terminal domain; pfam00289;ATP-grasp domain; cl03087;Biotin carboxylase C-terminal domain; cl08365;The biotinyl-domain or biotin carboxyl carrier protein (BCCP) domain is present in all biotin-dependent enzymes, such as acetyl-CoA carboxylase, pyruvate carboxylase, propionyl-CoA carboxylase, methylcrotonyl-CoA carboxylase, geranyl-CoA carboxylase; cd06850;carboxyltransferase (CT) interaction site;biotinylation site [posttranslational modification]
GOHSU20	CDS	GOHSU_20_00300	complement(30468..32048)		acyl-CoA carboxylase beta chain [nite]	 accD [nite]		ROP_61520 :acyl-CoA carboxylase beta chain : accD : 6.4.1.- : Rhodococcus opacus B4	methylcrotonyl-CoA carboxylase subunit beta	accD	G7H3I8_9ACTO Acyl-CoA carboxylase beta chain OS=Gor donia araii NBRC 100433 GN=accD PE=4 SV=1;accD gene product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;3-methylcrotonyl-CoA carboxylase, beta chain;Evidence 2b : Function of strongly homologous gene; Product type e : enzyme;Acetyl co-enzyme A carboxylase carboxyltransferase alpha subunit; cl15772
GOHSU20	CDS	GOHSU_20_00310	complement(32135..32737)		putative TetR family transcriptional regulator [nite]			ROP_61510 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR-family transcriptional regulator		D0L680_GORB4 Regulatory protein TetR OS=Gordonia br onchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1353 PE=4 SV=1;TetR-family transcriptional regulator [Streptomyces clavuligerus ATCC 27064];Helix-turn-helix domains; cl00088
GOHSU20	CDS	GOHSU_20_00320	complement(32734..33261)		hypothetical protein [migap]				hypothetical protein		F1YKL6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12247 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU20	CDS	GOHSU_20_00330	complement(33384..33806)		hypothetical membrane protein [nite]			ROP_35210 :hypothetical membrane protein : : : Rhodococcus opacus B4	RDD domain-containing protein		H5UC12_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_039_01100 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];RDD family; cl00746
GOHSU20	CDS	GOHSU_20_00340	complement(33803..34261)		hypothetical protein [nite]			ROP_35200 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		C8XC65_NAKMY Putative uncharacterized protein OS=Na kamurella multipartita (strain ATCC 700099 / DSM 44233 / J CM 9543 / Y-104) GN=Namu_5192 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU20	CDS	GOHSU_20_00350	complement(34508..35410)		succinyl-CoA synthetase alpha subunit [nite]	 sucD [nite]		RER_44280 :succinyl-CoA synthetase alpha subunit : sucD : 6.2.1.5 : Rhodococcus erythropolis PR4	succinyl-CoA synthetase subunit alpha		G7H3I1_9ACTO Succinyl-CoA ligase [ADP-forming] subu nit alpha OS=Gordonia araii NBRC 100433 GN=sucD PE=3 SV=1;succinyl-CoA synthetase subunit alpha [Mycobacteriu m abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;succinyl-CoA synthetase subunit alpha; Validated;Catalyzes the only substrate-level phosphorylation in the TCA cycle;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;CoA-ligase; cl02894
GOHSU20	CDS	GOHSU_20_00360	complement(35425..36588)		succinyl-CoA ligase [ADP-forming] subunit beta [hamap]	sucC [hamap]	MF_00558:succinyl-CoA ligase [ADP-forming] subunit beta:sucC 	RER_44290 :succinyl-CoA synthetase beta subunit : sucC : 6.2.1.5 : Rhodococcus erythropolis PR4	succinyl-CoA synthetase, beta subunit		H5U3F0_9ACTO Succinyl-CoA ligase [ADP-forming] subu nit beta OS=Gordonia sputi NBRC 100414 GN=sucC PE=3 SV=1;succinyl-CoA synthetase, beta subunit [Saccharomono spora marina XMU15];succinyl-CoA synthetase subunit beta; Provisional; PRK00696;PFAM: CoA-ligase; ATP-grasp domain; TIGRFAM: succinyl-CoA synthetase, beta subunit;ATP-grasp domain; cl03087;CoA-ligase; cl02894
GOHSU20	CDS	GOHSU_20_00370	complement(36938..37249)	overlap	not assigned						F1YNC6_9ACTO Response regulator receiver OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_17123 PE=4 SV=1
GOHSU20	CDS	GOHSU_20_00380	37131..37886	overlap	peptidase M23 family protein [nite]			ROP_56400 :peptidase M23 family protein : : : Rhodococcus opacus B4	peptidase M23 family protein		H0R1M2_9ACTO Peptidase M23 family protein OS=Gordon ia effusa NBRC 100432 GN=GOEFS_071_00170 PE=4 SV=1;peptidase M23 family protein [Rhodococcus opacus B4 ];Peptidase family M23; pfam01551
GOHSU20	CDS	GOHSU_20_00390	complement(37923..40337)		ATP-dependent DNA helicase UvrD1 [nite]	 uvrD1 [nite]		RER_44320 :ATP-dependent DNA helicase UvrD1 : uvrD1 : 3.6.1.- : Rhodococcus erythropolis PR4	ATP-dependent DNA helicase PcrA	pcrA	F1YKL0_9ACTO ATP-dependent DNA helicase PcrA OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_12217 PE=4 SV=1;pcrA gene product [Gordonia polyisoprenivorans VH2] ;ATP-dependent DNA helicase PcrA; TIGR01073;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Family description; cl15862
GOHSU20	CDS	GOHSU_20_00400	complement(40376..42181)		hypothetical membrane protein [nite]			ROP_39000 :hypothetical membrane protein : : : Rhodococcus opacus B4	putative transmembrane protein		F1YKK9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12212 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU20	CDS	GOHSU_20_00410	42245..42568		probable chorismate mutase [nite]			RER_44330 :probable chorismate mutase : : 5.4.99.5 : Rhodococcus erythropolis PR4	chorismate mutase		H0R1M6_9ACTO Putative chorismate mutase OS=Gordonia effusa NBRC 100432 GN=GOEFS_071_00210 PE=4 SV=1;chorismate mutase [Rhodococcus equi 103S];Chorismate mutase type II; cl00693
GOHSU20	CDS	GOHSU_20_00420	complement(42599..44248)	case	glucose-6-phosphate isomerase [hamap]	pgi [hamap]	MF_00473:glucose-6-phosphate isomerase:pgi 	RER_44350 :glucose-6-phosphate isomerase : pgi : 5.3.1.9 : Rhodococcus erythropolis PR4	glucose-6-phosphate isomerase	pgi	F1YKK5_9ACTO Glucose-6-phosphate isomerase OS=Gordo nia neofelifaecis NRRL B-59395 GN=pgi PE=3 SV=1;glucose-6-phosphate isomerase [Rhodococcus jostii R HA1];glucose-6-phosphate isomerase; Reviewed; PRK00179;functions in sugar metabolism in glycolysis and the Embden-Meyerhof pathways (EMP) and in gluconeogenesis; catalyzes reversible isomerization of glucose-6-phosphate to fructose-6-phosphate; member of PGI family;Phosphoglucose isomerase (PGI) contains two SIS (Sugar ISomerase) domains. This classification is based on the alignment of the first SIS domain. PGI is a multifunctional enzyme which as an intracellular dimer catalyzes the reversible isomerization of...; cd05015;dimer interface [polypeptide binding];Phosphoglucose isomerase (PGI) contains two SIS (Sugar ISomerase) domains. This classification is based on the alignment of the second SIS domain. PGI is a multifunctional enzyme which as an intracellular dimer catalyzes the reversible isomerization of...; cd05016
GOHSU20	CDS	GOHSU_20_00430	complement(44298..45038)		putative oxidoreductase [nite]			RER_44430 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		F1YKK4_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12187 PE=3 SV=1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;short chain dehydrogenase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU20	CDS	GOHSU_20_00440	45397..46290		conserved hypothetical protein [nite]			RER_54720 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YKK3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12182 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mxa:MXAN_4606 carbohydrate-binding protein;Domain of unknown function (DUF4185); pfam13810
GOHSU20	CDS	GOHSU_20_00450	complement(46336..47244)		putative DNA glycosylase [nite]			ROP_19600 :putative DNA glycosylase : : 3.2.2.- : Rhodococcus opacus B4	DNA-formamidopyrimidine glycosylase		F1YKK1_9ACTO DNA-formamidopyrimidine glycosylase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12172 PE=3 SV =1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Formamidopyrimidine-DNA glycosylase [DNA replication, recombination, and repair]; COG0266;Uncharacterized bacterial subgroup of the N-terminal domain of Fpg (formamidopyrimidine-DNA glycosylase, MutM)_Nei  (endonuclease VIII) base-excision repair DNA glycosylases; cd08973;PFAM: formamidopyrimidine-DNA glycosylase catalytic domain protein; DNA glycosylase/AP lyase, H2TH DNA- binding; KEGG: hch:HCH_02013 formamidopyrimidine-DNA glycosylase;putative DNA binding site [nucleotide binding];catalytic residue [active];putative H2TH interface [polypeptide binding];putative catalytic residues [active];Formamidopyrimidine-DNA glycosylase H2TH domain; pfam06831;Zinc finger found in FPG and IleRS; pfam06827
GOHSU20	CDS	GOHSU_20_00460	47292..48038		putative phosphatase [nite]			SLG_22420 :putative phosphatase : : : Sphingobium sp. SYK-6	inositol monophosphatase		F1YKK0_9ACTO Inositol monophosphatase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_12167 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: inositol monophosphatase; KEGG: cvi:CV_1577 inositol monophosphatase family protein;CysQ, a 3'-Phosphoadenosine-5'-phosphosulfate (PAPS) 3'-phosphatase, is a bacterial member of the inositol monophosphatase family. It has been proposed that CysQ helps control intracellular levels of PAPS, which is an intermediate in cysteine...; cd01638
##GOHSU21	contig00083				30550
GOHSU21	CDS	GOHSU_21_00010	<1..238	3x codon	putative 3'-phosphoadenosine-5'-phosphatase [nite]	 cysQ [nite]		RER_39350 :putative 3'-phosphoadenosine-5'-phosphatase : cysQ : 3.1.3.- : Rhodococcus erythropolis PR4	inositol monophosphatase		F1YKK0_9ACTO Inositol monophosphatase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_12167 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: inositol monophosphatase; KEGG: cvi:CV_1577 inositol monophosphatase family protein;CysQ, a 3'-Phosphoadenosine-5'-phosphosulfate (PAPS) 3'-phosphatase, is a bacterial member of the inositol monophosphatase family. It has been proposed that CysQ helps control intracellular levels of PAPS, which is an intermediate in cysteine...; cd01638
GOHSU21	CDS	GOHSU_21_00020	complement(276..464)		not assigned						B6K9B2_TOXGO WW domain-containing protein OS=Toxopl asma gondii GN=TGME49_114360 PE=4 SV=1
GOHSU21	CDS	GOHSU_21_00030	complement(499..1236)		precorrin-6A synthase [nite]	 cobF [nite]		ROP_19610 :precorrin-6A synthase : cobF : 2.1.1.152 : Rhodococcus opacus B4	precorrin 6A synthase		F1YKJ9_9ACTO Precorrin 6A synthase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_12162 PE=4 SV=1;precorrin 6A synthase [Streptomyces clavuligerus AT CC 27064];Precorrin-3B methylase [Coenzyme metabolism]; cl00304;Tetrapyrrole (Corrin/Porphyrin) Methylases; pfam00590
GOHSU21	CDS	GOHSU_21_00040	complement(1248..2075)		oxidoreductase [nite]			CSE_14360 :oxidoreductase : : : Caldisericum exile AZM16c01	short chain dehydrogenase		F9VYE5_9ACTO Putative oxidoreductase OS=Gordonia al kanivorans NBRC 16433 GN=GOALK_087_00320 PE=3 SV=1;short chain dehydrogenase [Rhizobium etli CIAT 652] ;ypch00333;short chain dehydrogenase; Provisional;17beta hydroxysteroid dehydrogenase-like, classical (c) SDRs; cd05374;NADP binding site [chemical binding];steroid binding site
GOHSU21	CDS	GOHSU_21_00050	complement(2152..3471)		LGFP repeat-containing protein [migap]				LGFP repeat-containing protein		G7H3G9_9ACTO 50S ribosomal protein L4 OS=Gordonia a raii NBRC 100433 GN=rplD PE=4 SV=1;LGFP repeat-containing protein [Streptomyces pristi naespiralis ATCC 25486];LGFP repeat; pfam08310
GOHSU21	tRNA	GOHSU_21_t0010	complement(3853..3925)		not assigned						identified by tRNAscan-SE; putative
GOHSU21	CDS	GOHSU_21_00060	complement(4032..4247)		not assigned						
GOHSU21	CDS	GOHSU_21_00070	4309..4812		putative oxidoreductase [nite]			RER_48560 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		Q5YXP5_NOCFA Putative short chain dehydrogenase OS= Nocardia farcinica (strain IFM 10152) GN=NFA_21990 PE=3 SV =1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU21	CDS	GOHSU_21_00080	4896..5588		putative TetR family transcriptional regulator [nite]			ROP_51270 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		Q0S6J0_RHOSR Possible transcriptional regulator, Te tR family protein OS=Rhodococcus sp. (strain RHA1) GN=RHA1 _ro05065 PE=4 SV=1;TetR family transcriptional regulator [Rhodococcus jostii RHA1];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU21	CDS	GOHSU_21_00090	complement(5639..6076)		conserved hypothetical protein [nite]			RER_45260 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	polyketide cyclase/dehydrase and lipid transport		H5U7B7_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_141_00230 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Polyketide cyclase / dehydrase and lipid transport;START/RHO_alpha_C/PITP/Bet_v1/CoxG/CalC (SRPBCC) ligand-binding domain superfamily; cd07812;hydrophobic ligand binding site
GOHSU21	CDS	GOHSU_21_00100	6075..6653		putative 3-methyladenine DNA glycosylase [nite]			MLP_04930 :putative 3-methyladenine DNA glycosylase : : 3.2.2.- : Microlunatus phosphovorus NM-1	DNA-3-methyladenine glycosylase		F1YKI7_9ACTO Putative 3-methyladenine DNA glycosyla se OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12102 PE =3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: DNA-3-methyladenine glycosylase; PFAM: methylpurine-DNA glycosylase (MPG); KEGG: acp:A2cp1_4387 DNA-3-methyladenine glycosylase;Alkyladenine DNA glycosylase catalyzes the first step in base excision repair; cd00540;DNA binding site [nucleotide binding]
GOHSU21	CDS	GOHSU_21_00110	complement(6680..9196)		putative cation-transporting ATPase [nite]			RER_45380 :putative cation-transporting ATPase : : 3.6.3.- : Rhodococcus erythropolis PR4	integral membrane ATPase		F1YLU3_9ACTO ATPase, P-type (Transporting), HAD sup erfamily, subfamily IC OS=Gordonia neofelifaecis NRRL B-59 395 GN=SCNU_14436 PE=3 SV=1;integral membrane ATPase [Streptomyces coelicolor A 3(2)];SCD12A.15c, possible integral membrane ATPase, len: 802 aa; similar to TR:Q9Z4W5 (EMBL:AL035654) Streptomyces coelicolor  integral membrane ATPase SCE8.09, 796 aa; fasta scores: opt: 3320 z-score: 3591.6 E(): 0; 67.0% identity in 796 aa overlap and to TR:O93862 (EMBL:AF043332) Aspergillus nidulans plasma membrane H(+)ATPase PmaA, 990 aa; fasta scores: opt: 504 z-score: 543.3 E(): 8.4e-23; 25.4% identity in 808 aa overlap. Contains 2x Pfam match to entry PF00122 E1-E2_ATPase, E1-E2 ATPase and match to Prosite entry PS00154 E1-E2 ATPases phosphorylation site. Contains also possible hydrophobic membrane spanning regions;Cation transport ATPase [Inorganic ion transport and metabolism]; COG0474;E1-E2 ATPase; pfam00122;Haloacid Dehalogenase-like Hydrolases; cl11391;Cation transporting ATPase, C-terminus; pfam00689
GOHSU21	CDS	GOHSU_21_00120	complement(9193..10422)		hypothetical protein [nite]			ROP_50980 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLU4_9ACTO Zn-dependent hydrolase of the beta-lac tamase fold-like protein OS=Gordonia neofelifaecis NRRL B- 59395 GN=SCNU_14441 PE=4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Metallo-beta-lactamase superfamily; cl00446
GOHSU21	CDS	GOHSU_21_00130	10425..11702		hypothetical protein [nite]			ROP_60900 :hypothetical protein : : : Rhodococcus opacus B4	alanine racemase		F1YLU5_9ACTO Alanine racemase domain-containing pro tein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14446 PE=4 SV=1;alanine racemase [Streptomyces sp. e14];Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzymes; cl00261;Predicted amino acid aldolase or racemase [Amino acid transport and metabolism];pyridoxal 5'-phosphate (PLP) binding site [chemical binding];catalytic residue [active]
GOHSU21	CDS	GOHSU_21_00140	11699..13012		putative FAD-linked oxidase [nite]			ROP_60880 :putative FAD-linked oxidase : : : Rhodococcus opacus B4	oxidoreductase, FAD-binding		H0R9B4_9ACTO Putative FAD-linked oxidase OS=Gordoni a polyisoprenivorans NBRC 16320 GN=GOPIP_011_00560 PE=4 SV =1;oxidoreductase, FAD-binding [Streptomyces sp. e14];FAD-linked oxidoreductase; TIGR01679;FAD binding domain; pfam01565;D-arabinono-1,4-lactone oxidase; cl04370
GOHSU21	CDS	GOHSU_21_00150	complement(13029..13220)		not assigned						
GOHSU21	CDS	GOHSU_21_00160	complement(13266..13997)		putative enoyl-CoA hydratase [nite]			RER_45400 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase		H5TMV5_9ACTO Putative enoyl-CoA hydratase OS=Gordon ia otitidis NBRC 100426 GN=GOOTI_123_00270 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Enoyl-CoA hydratase/carnithine racemase [Lipid metabolism]; COG1024;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU21	CDS	GOHSU_21_00170	complement(14041..14400)		FKBP-type peptidyl-prolyl cis-trans isomerase [nite]			RER_46110 :FKBP-type peptidyl-prolyl cis-trans isomerase : : 5.2.1.8 : Rhodococcus erythropolis PR4	peptidyl prolyl isomerase		E4WDU2_RHOE1 Peptidyl-prolyl cis-trans isomerase OS =Rhodococcus equi (strain 103S) GN=REQ_09930 PE=4 SV=1;peptidyl prolyl isomerase [Rhodococcus equi 103S];FKBP-type peptidyl-prolyl cis-trans isomerase; cl11587
GOHSU21	CDS	GOHSU_21_00180	complement(14403..15650)		citrate synthase [nite]	 gltA [nite]		ROP_50580 :citrate synthase : gltA : 2.3.3.1 : Rhodococcus opacus B4	citrate synthase		D0L5N3_GORB4 Citrate synthase OS=Gordonia bronchial is (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1270 PE=3 SV=1;citrate synthase [Intrasporangium calvum DSM 43043] ;COGs: COG0372 citrate synthase; InterPro IPR010953: IPR002020: IPR019810; KEGG: rsa:RSal33209_1176 citrate synthase; PFAM: citrate synthase; PRIAM: citrate (Si)-synthase; SPTR: citrate synthase; TIGRFAM: citrate synthase I; PFAM: citrate synthase; TIGRFAM: citrate synthase I (hexameric type);Escherichia coli (Ec) citrate synthase (CS) GltA_like. CS catalyzes the condensation of acetyl coenzyme A (AcCoA) and oxalacetate (OAA) to form citrate and coenzyme A (CoA), the first step in the citric acid cycle (TCA or Krebs cycle). The overall CS...; cd06114;dimer interface [polypeptide binding];Citrate synthase; pfam00285;citrylCoA binding site [chemical binding];NADH binding [chemical binding];cationic pore residues;oxalacetate/citrate binding site [chemical binding];coenzyme A binding site [chemical binding];catalytic triad [active]
GOHSU21	CDS	GOHSU_21_00190	15864..16520		hypothetical protein [nite]			ROP_32660 :hypothetical protein : : : Rhodococcus opacus B4	allophanate hydrolase subunit 1		F1YLV5_9ACTO Allophanate hydrolase subunit 1 OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_14496 PE=4 SV=1;allophanate hydrolase subunit 1 [Rhodococcus jostii RHA1];Allophanate hydrolase subunit 1; pfam02682;Allophanate hydrolase subunit 1 [Amino acid transport and metabolism]; cl00896
GOHSU21	CDS	GOHSU_21_00200	16643..17371		hypothetical protein [nite]			ROP_32650 :hypothetical protein : : : Rhodococcus opacus B4	allophanate hydrolase		F1YLV6_9ACTO Urea amidolyase related protein OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_14501 PE=4 SV=1;allophanate hydrolase [Amycolatopsis mediterranei U 32];Allophanate hydrolase subunit 2; cl00865
GOHSU21	CDS	GOHSU_21_00210	complement(17381..18667)		putative drug resistance efflux protein [nite]			RER_46140 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLW0_9ACTO Major facilitator superfamily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14521 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: lpf:lpl1388 hypothetical protein;H+ Antiporter protein; TIGR00900;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU21	CDS	GOHSU_21_00220	complement(18668..19363)		pyridoxine/pyridoxamine 5'-phosphate oxidase [hamap]	pdxH [hamap]	MF_01629:pyridoxine/pyridoxamine 5'-phosphate oxidase:pdxH 	RER_46150 :pyridoxamine 5'-phosphate oxidase : pdxH : 1.4.3.5 : Rhodococcus erythropolis PR4	pyridoxine/pyridoxamine 5'-phosphate oxidase PdxH	pdxH	H5TMT2_9ACTO Pyridoxine/pyridoxamine 5'-phosphate o xidase OS=Gordonia otitidis NBRC 100426 GN=pdxH PE=3 SV=1;pdxH gene product [Gordonia polyisoprenivorans VH2] ;pyridoxamine 5'-phosphate oxidase; Provisional;Pyridoxine 5'-phosphate (PNP) oxidase-like proteins; cl00381;Pyridoxine 5'-phosphate oxidase C-terminal dimerisation region; pfam10590
GOHSU21	CDS	GOHSU_21_00230	complement(19427..21022)		hypothetical membrane protein [nite]			ROP_26770 :hypothetical membrane protein : : : Rhodococcus opacus B4	anibiotic ABC transporter efflux pump		H0R986_9ACTO Putative ABC transporter permease prot ein OS=Gordonia polyisoprenivorans NBRC 16320 GN=GOPIP_011 _00280 PE=4 SV=1;anibiotic ABC transporter efflux pump [Rhodococcus jostii RHA1];ABC-2 type transporter; cl11417
GOHSU21	CDS	GOHSU_21_00240	complement(21019..21924)		putative multidrug ABC transporter ATP-binding protein [nite]			KSE_56320 :putative multidrug ABC transporter ATP-binding protein : : : Kitasatospora setae NBRC 14216	multidrug ABC transporter ATPase		H0R985_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia polyisoprenivorans NBRC 16320 GN=GOPIP_ 011_00270 PE=3 SV=1;multidrug ABC transporter ATPase [Kytococcus sedent arius DSM 20547];PFAM: ABC transporter;ABC-type multidrug transport system, ATPase component [Defense mechanisms]; COG1131;This family of ATP-binding proteins belongs to a multisubunit transporter involved in drug resistance (BcrA and DrrA), nodulation, lipid transport, and lantibiotic immunity.  In bacteria and archaea, these transporters usually include an ATP-binding...; cd03230;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU21	CDS	GOHSU_21_00250	22003..23136		phosphoserine aminotransferase [hamap]	serC [hamap]	MF_00160:phosphoserine aminotransferase:serC 	RER_46170 :phosphoserine aminotransferase : serC : 2.6.1.52 : Rhodococcus erythropolis PR4	phosphoserine aminotransferase	serC	F1YLW7_9ACTO Phosphoserine aminotransferase OS=Gord onia neofelifaecis NRRL B-59395 GN=serC PE=3 SV=1;phosphoserine aminotransferase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 3-phosphonooxypyruvate and glutamate from O-phospho-L-serine and 2-oxoglutarate;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;Serine-pyruvate aminotransferase/archaeal aspartate aminotransferase [Amino acid transport and metabolism];pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU21	CDS	GOHSU_21_00260	23294..24418		hypothetical protein [migap]				hypothetical protein		H6MR51_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c39440 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF3071); pfam11268
GOHSU21	CDS	GOHSU_21_00270	complement(24429..25277)		putative RNA methyltransferase [nite]			ROP_50480 :putative RNA methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	tRNA/rRNA methyltransferase SpoU		F1YLX1_9ACTO tRNA/rRNA methyltransferase SpoU OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14576 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: tRNA/rRNA methyltransferase (SpoU); KEGG: mes:Meso_2208 tRNA/rRNA methyltransferase (SpoU);rRNA methylases [Translation, ribosomal structure and biogenesis]; COG0566;SpoU rRNA Methylase family; cl00362
GOHSU21	CDS	GOHSU_21_00280	25304..26536		conserved hypothetical protein [nite]			RER_50830 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLX2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14581 PE=4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;hypothetical protein; Provisional;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU21	CDS	GOHSU_21_00290	complement(26619..28100)		putative hypoxanthine/guanine permease [nite]			ROP_50390 :putative hypoxanthine/guanine permease : : : Rhodococcus opacus B4	xanthine/uracil permease		H0QY12_9ACTO Hypoxanthine/guanine permease OS=Gordo nia effusa NBRC 100432 GN=pbuG PE=4 SV=1;xanthine/uracil permease [Arthrobacter aurescens TC 1];identified by match to protein family HMM PF00860;Xanthine/uracil/vitamin C permease [Nucleotide transport and    metabolism]
GOHSU21	CDS	GOHSU_21_00300	28183..28437		hypothetical protein [nite]			RER_46270 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5UC94_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_039_01930 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF2530); pfam10745
GOHSU21	CDS	GOHSU_21_00310	complement(28434..29030)		hypothetical membrane protein [nite]			RER_46290 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	PA-phosphatase-like phosphoesterase		F1YLX5_9ACTO Phosphoesterase PA-phosphatase related protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14 596 PE=4 SV=1;PA-phosphatase-like phosphoesterase [Nostoc punctif orme PCC 73102];PFAM: phosphoesterase, PA-phosphatase related; KEGG: ana:all7623 hypothetical protein;PAP2_like_2 proteins. PAP2 is a super-family of phosphatases and haloperoxidases. This subgroup, which is specific to bacteria, lacks functional characterization and may act as a membrane-associated lipid phosphatase; cd03392
GOHSU21	CDS	GOHSU_21_00320	complement(29036..>30550)	codon	conserved hypothetical protein [nite]			RER_46300 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GS69_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_52_00300 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: scl:sce3972 hypothetical protein
##GOHSU22	contig00091				132658
GOHSU22	CDS	GOHSU_22_00010	complement(<1..610)	3x codon	Rne/Rng family ribonuclease [migap]				Rne/Rng family ribonuclease		G7GV11_9ACTO Putative ribonuclease OS=Gordonia amar ae NBRC 15530 GN=GOAMR_67_00080 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;TIGRFAM: ribonuclease, Rne/Rng family; KEGG: rop:ROP_10360  ribonuclease; PFAM: RNA-binding protein AU-1/ribonuclease E/G; RNA binding S1 domain protein; translation initiation factor IF-2 domain protein;Translation initiation factor IF-2, N-terminal region; pfam04760
GOHSU22	CDS	GOHSU_22_00020	complement(902..1318)		nucleoside diphosphate kinase [hamap]	ndk [hamap]	MF_00451:nucleoside diphosphate kinase:ndk 	RER_37970 :nucleoside diphosphate kinase : ndk : 2.7.4.6 : Rhodococcus erythropolis PR4	nucleoside diphosphate kinase	ndk	F1YI58_9ACTO Nucleoside diphosphate kinase OS=Gordo nia neofelifaecis NRRL B-59395 GN=ndk PE=3 SV=1;nucleoside diphosphate kinase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of nucleoside triphosphate from ATP and nucleoside diphosphate;Nucleoside diphosphate kinase Group I (NDPk_I)-like: NDP kinase domains are present in a large family of structurally and functionally conserved proteins from bacteria to humans that generally catalyze the transfer of gamma-phosphates of a nucleoside...; cd04413;multimer interface [polypeptide binding]
GOHSU22	CDS	GOHSU_22_00030	complement(1385..1927)		not assigned						F1YI57_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07858 PE=4 SV=1
GOHSU22	CDS	GOHSU_22_00040	complement(1979..2377)		hypothetical membrane protein [nite]			ROP_10380 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YI56_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07853 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF4233); pfam14017
GOHSU22	CDS	GOHSU_22_00050	complement(2374..3774)		folylpolyglutamate synthase [nite]	 folC [nite]		ROP_10390 :folylpolyglutamate synthase : folC : 6.3.2.17 : Rhodococcus opacus B4	folylpolyglutamate synthase	folC	H5TSG0_9ACTO Folylpolyglutamate synthase OS=Gordoni a otitidis NBRC 100426 GN=folC PE=4 SV=1;folylpolyglutamate synthase [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;Folylpolyglutamate synthase [Coenzyme metabolism]; COG0285;Mur ligase middle domain; pfam08245
GOHSU22	CDS	GOHSU_22_00060	complement(3771..6425)		valyl-tRNA synthetase [hamap]	valS [hamap]	MF_02004:valyl-tRNA synthetase:valS 	ROP_10400 :valyl-tRNA synthetase : valS : 6.1.1.9 : Rhodococcus opacus B4	valyl-tRNA synthetase	valS	F1YI54_9ACTO Valine--tRNA ligase OS=Gordonia neofel ifaecis NRRL B-59395 GN=valS PE=3 SV=1;valyl-tRNA synthetase [Corynebacterium glutamicum A TCC 13032];IS fingerprint type 4-5;valine--tRNA ligase; ValRS; converts valine ATP and tRNA(Val) to AMP PPi and valyl-tRNA(Val); class-I aminoacyl-tRNA synthetase type 1 subfamily; has a posttransfer editing process to hydrolyze mischarged Thr-tRNA(Val) which is done by the editing domain;valyl-tRNA synthetase; Reviewed; PRK05729;nucleotidyl transferase superfamily; cl00015;HIGH motif;nucleotide binding site [chemical binding];Leucyl-tRNA synthetase, Domain 2; pfam13603;catalytic core domain of valyl-tRNA synthetases; cd00817;KMSKS motif;Anticodon-binding domain of valyl tRNA synthetases; cd07962;tRNA binding surface [nucleotide binding];anticodon binding site;Valyl tRNA synthetase tRNA binding arm; pfam10458
GOHSU22	CDS	GOHSU_22_00070	complement(6599..6796)		hypothetical protein [migap]				hypothetical protein		F6ERW1_AMYSD Putative uncharacterized protein OS=Am ycolicicoccus subflavus (strain DSM 45089 / DQS3-9A1) GN=A S9A_2329 PE=4 SV=1;hypothetical protein AS9A_2329 [Amycolicicoccus sub flavus DQS3- 9A1];type strain of Amycolicicoccus subflavus
GOHSU22	CDS	GOHSU_22_00080	complement(7058..7690)	case	molybdopterin-guanine dinucleotide biosynthesis protein A [hamap]	mobA [hamap]	MF_00316:molybdopterin-guanine dinucleotide biosynthesis protein A:mobA 	PH0980 :molybdopterin-guanine dinucleotide biosynthesis protein MobA : mobA : : Pyrococcus horikoshii OT3	molybdopterin guanine dinucleotide synthase		F1YI50_9ACTO Probable molybdenum cofactor guanylylt ransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=mobA PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: xac:XAC2033 molybdopterin guanine dinucleotide synthase;MobA catalyzes the formation of molybdopterin guanine dinucleotide; cd02503;molybdopterin-guanine dinucleotide biosynthesis protein MobA; Reviewed; PRK00317;GTP binding site
GOHSU22	CDS	GOHSU_22_00090	complement(7713..8423)		not assigned						H1QNK5_9ACTO Putative uncharacterized protein (Frag ment) OS=Streptomyces coelicoflavus ZG0656 GN=SMCF_6570 PE =4 SV=1
GOHSU22	CDS	GOHSU_22_00100	complement(8471..9088)		superoxide dismutase [nite]	 sodC [nite]		ROP_18890 :superoxide dismutase : sodC : 1.15.1.1 : Rhodococcus opacus B4	superoxide dismutase, Cu-Zn	sodC	I0RD34_MYCXE Superoxide dismutase, Cu-Zn OS=Mycobac terium xenopi RIVM700367 GN=MXEN_21402 PE=4 SV=1;superoxide dismutase, Cu-Zn [Mycobacterium avium 10 4];identified by similarity to SP:P53635; match to protein family HMM PF00080;Copper/zinc superoxide dismutase (SOD). superoxide dismutases catalyse the conversion of superoxide radicals to molecular oxygen. Three evolutionarily distinct families of SODs are known, of which the copper/zinc-binding family is one. Defects in the...; cd00305;E-class dimer interface [polypeptide binding];P-class dimer interface [polypeptide binding];Cu2+ binding site [ion binding];Zn2+ binding site [ion binding]
GOHSU22	CDS	GOHSU_22_00110	complement(9159..10433)		ATP-dependent Clp protease ATP-binding subunit ClpX [hamap]	clpX [hamap]	MF_00175:ATP-dependent Clp protease ATP-binding subunit ClpX:clpX 	ROP_10860 :ATP-dependent Clp protease ATP-binding subunit ClpX : clpX : : Rhodococcus opacus B4	ATP-dependent Clp protease ATP-binding subunit clpX		H5TSF7_9ACTO ATP-dependent Clp protease ATP-binding subunit ClpX OS=Gordonia otitidis NBRC 100426 GN=clpX PE= 3 SV=1;ATP-dependent Clp protease ATP-binding subunit clpX [Streptomyces clavuligerus ATCC 27064];ATP-dependent protease ATP-binding subunit ClpX; Provisional; PRK05342;ClpX C4-type zinc finger; cl05964;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;C-terminal, D2-small domain, of ClpB protein; cl11083
GOHSU22	CDS	GOHSU_22_00120	complement(10616..11293)		ATP-dependent Clp protease proteolytic subunit [hamap]	clpP [hamap]	MF_00444:ATP-dependent Clp protease proteolytic subunit:clpP 	RER_38140 :ATP-dependent Clp protease proteolytic subunit ClpP : clpP : 3.4.21.92 : Rhodococcus erythropolis PR4	ATP-dependent Clp protease proteolytic subunit	clpP2	F1YI47_9ACTO ATP-dependent Clp protease proteolytic subunit 2 OS=Gordonia neofelifaecis NRRL B-59395 GN=clpP2 PE=3 SV=1;ATP-dependent Clp protease proteolytic subunit [Cor ynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;hydrolyzes proteins to small peptides; with the ATPase subunits ClpA or ClpX, ClpP degrades specific substrates;ATP-dependent Clp protease proteolytic subunit; Reviewed;Caseinolytic protease (ClpP) is an ATP-dependent, highly conserved serine protease; cd07017;oligomer interface [polypeptide binding];active site residues [active]
GOHSU22	CDS	GOHSU_22_00130	complement(11341..11946)		ATP-dependent Clp protease proteolytic subunit [hamap]	clpP [hamap]	MF_00444:ATP-dependent Clp protease proteolytic subunit:clpP 	ROP_10880 :ATP-dependent Clp protease proteolytic subunit : clpP : 3.4.21.92 : Rhodococcus opacus B4	ATP-dependent Clp protease proteolytic subunit	clpP	F1YI46_9ACTO ATP-dependent Clp protease proteolytic subunit 1 OS=Gordonia neofelifaecis NRRL B-59395 GN=clpP PE=3 SV=1;ATP-dependent Clp protease proteolytic subunit [Cor ynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;hydrolyzes proteins to small peptides; with the ATPase subunits ClpA or ClpX, ClpP degrades specific substrates;Clp protease; pfam00574;Caseinolytic protease (ClpP) is an ATP-dependent, highly conserved serine protease; cd07017;oligomer interface [polypeptide binding];active site residues [active]
GOHSU22	CDS	GOHSU_22_00140	complement(12215..13570)		trigger factor [hamap]	tig [hamap]	MF_00303:trigger factor:tig 	RER_38160 :trigger factor : tig : : Rhodococcus erythropolis PR4	trigger factor	tig	F1YI45_9ACTO Trigger factor OS=Gordonia neofelifaec is NRRL B-59395 GN=tig PE=3 SV=1;trigger factor [Corynebacterium glutamicum ATCC 130 32];IS fingerprint type 4-5;trigger factor; Provisional; PRK01490;Tig; RopA; peptidyl-prolyl cis/trans isomerase; promotes folding of newly synthesized proteins; binds ribosomal 50S subunit; forms a homodimer;FKBP-type peptidyl-prolyl cis-trans isomerase; cl11587;Bacterial trigger factor protein (TF) C-terminus; pfam05698
GOHSU22	tRNA	GOHSU_22_t0010	complement(13613..13689)		not assigned						identified by tRNAscan-SE; putative
GOHSU22	tRNA	GOHSU_22_t0020	complement(13734..13807)		not assigned						identified by tRNAscan-SE; putative
GOHSU22	CDS	GOHSU_22_00150	14260..14955		YkuD domain-containing protein [migap]				YkuD domain-containing protein		H5U4K6_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_114_00060 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];L,D-transpeptidase catalytic domain; cl15405
GOHSU22	CDS	GOHSU_22_00160	complement(15013..15855)		hypothetical protein [nite]			RER_42190 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YI40_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07773 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU22	CDS	GOHSU_22_00170	complement(15969..16784)		putative DNA glycosylase [nite]			ROP_10910 :putative DNA glycosylase : : 3.2.2.- : Rhodococcus opacus B4	formamidopyrimidine-DNA glycosylase	mutM2	H5U4K9_9ACTO Putative DNA glycosylase OS=Gordonia s puti NBRC 100414 GN=GOSPT_114_00090 PE=3 SV=1;formamidopyrimidine-DNA glycosylase [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;Formamidopyrimidine-DNA glycosylase [DNA replication, recombination, and repair]; COG0266;N-terminal domain of the actinomycetal Nei1 and related DNA glycosylases; cd08970;putative DNA binding site [nucleotide binding];catalytic residue [active];putative H2TH interface [polypeptide binding];putative catalytic residues [active];Formamidopyrimidine-DNA glycosylase H2TH domain; pfam06831
GOHSU22	CDS	GOHSU_22_00180	complement(16810..17280)		ribose-5-phosphate isomerase B [nite]	 rpiB [nite]		ROP_10920 :ribose-5-phosphate isomerase B : rpiB : 5.3.1.6 : Rhodococcus opacus B4	ribose-5-phosphate isomerase B	rpi	E2MXC9_9CORY Ribose 5-phosphate isomerase OS=Coryne bacterium amycolatum SK46 GN=CORAM0001_1145 PE=4 SV=1;ribose-5-phosphate isomerase B [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;Ribose/Galactose Isomerase; cl00485;catalyzes the interconversion of ribose 5-phosphate to ribulose 5-phosphate; enzyme from E. coli shows allose 6-phosphate isomerase activity
GOHSU22	CDS	GOHSU_22_00190	17304..18317		conserved hypothetical protein [nite]			RER_38200 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	peptidase M48 Ste24p		F9VZ91_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_095_00140 PE=3 SV= 1;peptidase M48 Ste24p [Nocardiopsis dassonvillei sub sp. dassonvillei DSM 43111];InterPro IPR001915; KEGG: tfu:Tfu_2436 hypothetical protein; PFAM: peptidase M48 Ste24p; SPTR: C1YHB8 Zn-dependent protease with chaperone function; PFAM: peptidase family M48;Peptidase family M48; cl12018
GOHSU22	CDS	GOHSU_22_00200	complement(18393..19001)		hypothetical protein [nite]			ROP_10940 :hypothetical protein : : : Rhodococcus opacus B4	DSBA-like thioredoxin domain-containing protein		F9VZ92_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_095_00150 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];DSBA-like thioredoxin domain; pfam01323;DsbA family; consists of DsbA and DsbA-like proteins, including DsbC, DsbG, glutathione (GSH) S-transferase kappa (GSTK), 2-hydroxychromene-2-carboxylate (HCCA) isomerase, an oxidoreductase (FrnE) presumed to be involved in frenolicin biosynthesis, a...; cd02972;catalytic residues [active]
GOHSU22	CDS	GOHSU_22_00210	19094..21679		aminopeptidase N [nite]	 pepN [nite]		ROP_10960 :aminopeptidase N : pepN : 3.4.11.2 : Rhodococcus opacus B4	aminopeptidase	pepN	H5U4L3_9ACTO Aminopeptidase N OS=Gordonia sputi NBR C 100414 GN=pepN PE=4 SV=1;aminopeptidase N [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;aminopeptidase N, Streptomyces lividans type; TIGR02412;Peptidase M1 family containing Aminopeptidase N; cd09602;Zn binding site [ion binding]
GOHSU22	CDS	GOHSU_22_00220	complement(21756..22256)		hypothetical protein [nite]			ROP_11000 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YI32_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07733 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Repair protein; cl01535
GOHSU22	CDS	GOHSU_22_00230	complement(22243..22491)		hypothetical protein [migap]				hypothetical protein		F1YI31_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07728 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU22	CDS	GOHSU_22_00240	22653..23036		2-on-2 hemoglobin [nite]	 glbO [nite]		RER_38260 :2-on-2 hemoglobin : glbO : : Rhodococcus erythropolis PR4	globin		F1YI30_9ACTO Globin protein OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_07723 PE=4 SV=1;globin [Nocardia farcinica IFM 10152];Truncated hemoglobins (trHbs) are a family of oxygen-binding heme proteins found in cyanobacteria, eubacteria, unicellular eukaryotes, and plants. The truncated hemoglobins have a characteristic two-over-two alpha helical folding pattern that is distinct...; cd00454;apolar tunnel;heme binding site [chemical binding];dimerization interface [polypeptide binding]
GOHSU22	CDS	GOHSU_22_00250	23061..24698		alpha-glucosidase [nite]	 aglA [nite]		RER_38280 :alpha-glucosidase : aglA : 3.2.1.20 : Rhodococcus erythropolis PR4	alpha amylase		F1YI29_9ACTO Alpha amylase catalytic region OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_07718 PE=4 SV=1;alpha amylase [Catenulispora acidiphila DSM 44928];PFAM: alpha amylase catalytic region; SMART: alpha amylase catalytic sub domain; KEGG: sen:SACE_4257 alpha-glucosidase;alpha,alpha-phosphotrehalase; TIGR02403;Alpha amylase catalytic domain found in oligo-1,6-glucosidase (also called isomaltase; sucrase-isomaltase; alpha-limit dextrinase), trehalose synthase (also called maltose alpha-D-glucosyltransferase), and related proteins; cd11332;catalytic site [active];Domain of unknown function (DUF3459); pfam11941
GOHSU22	CDS	GOHSU_22_00260	complement(24757..25428)		conserved hypothetical protein [nite]			RER_38290 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YI28_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07713 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU22	CDS	GOHSU_22_00270	complement(25425..25898)		conserved hypothetical protein [nite]			RER_38300 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative thioesterase		F1YI27_9ACTO Thioesterase superfamily protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07708 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];4-hydroxybenzoyl-CoA thioesterase (4HBT). Catalyzes the final step in the 4-chlorobenzoate degradation pathway in which 4-chlorobenzoate is converted to 4-hydroxybenzoate in certain soil-dwelling bacteria. 4HBT forms a homotetramer with four active sites; cd00586
GOHSU22	CDS	GOHSU_22_00280	complement(25900..26325)	overlap	NAD-dependent glutamate dehydrogenase [nite]	 gdh [nite]		ROP_11070 :NAD-dependent glutamate dehydrogenase : gdh : 1.4.1.2 : Rhodococcus opacus B4	NAD-dependent glutamate dehydrogenase	gdh	H5U4M5_9ACTO NAD-dependent glutamate dehydrogenase OS=Gordonia sputi NBRC 100414 GN=gdh PE=4 SV=1;NAD-dependent glutamate dehydrogenase [Rhodococcus opacus B4];Bacterial NAD-glutamate dehydrogenase; pfam05088
GOHSU22	CDS	GOHSU_22_00290	complement(26243..30550)	overlap	NAD-dependent glutamate dehydrogenase [nite]	 gdh [nite]		RER_38310 :NAD-dependent glutamate dehydrogenase : gdh : 1.4.1.2 : Rhodococcus erythropolis PR4	NAD-dependent glutamate dehydrogenase	gdh	F1YI26_9ACTO NAD-glutamate dehydrogenase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_07703 PE=4 SV=1;NAD-dependent glutamate dehydrogenase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Bacterial NAD-glutamate dehydrogenase; pfam05088
GOHSU22	CDS	GOHSU_22_00300	complement(30618..31595)		AraC family transcriptional regulator [migap]				AraC family transcriptional regulator		H5TLH4_9ACTO Putative AraC family transcriptional r egulator OS=Gordonia otitidis NBRC 100426 GN=GOOTI_103_001 70 PE=4 SV=1;AraC family transcriptional regulator [Arthrobacter aurescens TC1];identified by match to protein family HMM PF00165;DNA-binding transcriptional activator FeaR; Provisional
GOHSU22	CDS	GOHSU_22_00310	complement(31588..32616)		hypothetical protein [migap]				hypothetical protein		H5U391_9ACTO Putative AraC family transcriptional r egulator OS=Gordonia sputi NBRC 100414 GN=GOSPT_091_00030 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU22	CDS	GOHSU_22_00320	complement(32613..34292)		putative ABC transporter ATP-binding protein [nite]			ROP_11090 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	ABC transporter ATP-binding protein		H0QYZ0_9ACTO Putative ABC transporter ATP-binding p rotein OS=Gordonia effusa NBRC 100432 GN=GOEFS_045_00310 P E=3 SV=1;ABC transporter ATP-binding protein [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;putative ABC transporter ATP-binding protein; Reviewed;ChvD; in Agrobacterium tumefaciens, mutations in both Walker boxes were found to affect virulence;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region;ABC transporter; pfam12848;ABCF_EF-3  Elongation factor 3 (EF-3) is a cytosolic protein required by fungal ribosomes for in vitro protein synthesis and for in vivo growth.  EF-3 stimulates the binding of the EF-1: GTP: aa-tRNA ternary complex to the ribosomal A site by facilitated...; cd03221
GOHSU22	CDS	GOHSU_22_00330	complement(34508..34852)		glutamate synthase (NADPH) large subunit [migap]				glutamate synthase (NADPH) large subunit		E4TWH1_SULKY Glutamate synthase (NADPH) large subun it OS=Sulfuricurvum kujiense (strain ATCC BAA-921 / DSM 16 994 / JCM 11577 / YK-1) GN=Sulku_1126 PE=4 SV=1;glutamate synthase (NADPH) large subunit [Sulfuricu rvum kujiense DSM 16994];COGs: COG0069 Glutamate synthase domain 2; InterProIPR017932: IPR000583: IPR006982: IPR002932: IPR 002489; KEGG: tdn:Suden_1176 glutamate synthase (NADPH) large subunit; PFAM: ferredoxin-dependent glutamate synthase; glutamine amidotransferase class-II; glutamate synthase; glutamate synthase alpha subunit domain protein; PRIAM: Glutamate synthase (ferredoxin); SPTR: Glutamate synthase (NADPH) large subunit; PFAM: Conserved region in glutamate synthase; GXGXG motif; Glutamate synthase central domain; Glutamine amidotransferases class-II;glutamate synthase subunit alpha; Provisional; PRK11750;substrate binding site [chemical binding]
GOHSU22	CDS	GOHSU_22_00340	complement(35072..35284)		UvrA-like protein [nite]			MLP_35210 :UvrA-like protein : : : Microlunatus phosphovorus NM-1	excinuclease ATPase subunit		C0W4B4_9ACTO Putative uncharacterized protein OS=Ac tinomyces urogenitalis DSM 15434 GN=HMPREF0058_0708 PE=4 S V=1;excinuclease ATPase subunit [Slackia heliotriniredu cens DSM 20476];type strain of Slackia heliotrinireducens DSM 20476;PFAM: ABC transporter; TIGRFAM: excinuclease ABC, A subunit;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding]
GOHSU22	CDS	GOHSU_22_00350	35419..36204		hypothetical protein [nite]			ROP_23910 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7H131_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_043_00670 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];cytoplasmic protein;Uncharacterized protein conserved in bacteria (DUF2236); cl01472
GOHSU22	CDS	GOHSU_22_00360	36357..36683		hypothetical membrane protein [nite]			ROP_68560 :hypothetical membrane protein : : : Rhodococcus opacus B4	putative transmembrane protein		F1YI24_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07693 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Putative transmembrane protein (PGPGW); cl09870
GOHSU22	CDS	GOHSU_22_00370	complement(36688..37236)		putative single-stranded DNA-binding protein [nite]	 ssb2 [nite]		KSE_54570 :putative single-stranded DNA-binding protein : ssb2 : : Kitasatospora setae NBRC 14216	single-strand binding protein		H0QYY8_9ACTO Single-stranded DNA-binding protein OS =Gordonia effusa NBRC 100432 GN=ssb PE=4 SV=1;single-strand binding protein [Acidothermus cellulo lyticus 11B];type strain of Acidothermus cellulolyticus 11B;TIGRFAM: single-strand binding protein; PFAM: single-strand binding protein/Primosomal replication protein n; KEGG: tfu:Tfu_3094 single-strand binding protein;SSB_OBF: A subfamily of OB folds similar to the OB fold of ssDNA-binding protein (SSB). SSBs bind with high affinity to ssDNA. They bind to and protect ssDNA intermediates during DNA metabolic pathways. All bacterial and eukaryotic SSBs studied to date...; cd04496;dimer interface [polypeptide binding];ssDNA binding site [nucleotide binding];tetramer (dimer of dimers) interface [polypeptide binding]
GOHSU22	CDS	GOHSU_22_00380	complement(37401..39473)		conserved hypothetical membrane protein [nite]			RER_38380 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	copper resistance D domain-containing protein;hypothetical protein		F1YI23_9ACTO Cytochrome c oxidase caa3-type, assemb ly factor CtaG-like protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07688 PE=4 SV=1;copper resistance D domain-containing protein [Myco bacterium gilvum PYR- GCK];Signal predicted by SignalP 2.0 HMM (Signal peptide probability0.968) with cleavage site probability 0.356 at residue 49;PFAM: copper resistance D domain protein; KEGG: mmc:Mmcs_3617 copper resistance D;Copper resistance protein D; cl00563;Cytochrome c oxidase caa3 assembly factor (Caa3_CtaG); cl09173
GOHSU22	tRNA	GOHSU_22_t0030	39531..39607		not assigned						identified by tRNAscan-SE; putative
GOHSU22	CDS	GOHSU_22_00390	39753..40337	overlap	hypothetical protein [nite]			KSE_07560 :hypothetical protein : : : Kitasatospora setae NBRC 14216	bifunctional deaminase-reductase domain-containing protein		H6QYY2_NOCCG Putative dihydrofolate reductase OS=No cardia cyriacigeorgica (strain GUH-2) GN=NOCYR_0723 PE=4 S V=1;bifunctional deaminase-reductase domain-containing protein [Beutenbergia cavernae DSM 12333];type strain of Beutenbergia cavernae;PFAM: RibD C-terminal domain;Dihydrofolate reductase (DHFR). Reduces 7,8-dihydrofolate to 5,6,7,8-tetrahydrofolate with NADPH as a cofactor. This is an essential step in the biosynthesis of deoxythymidine phosphate since 5,6,7,8-tetrahydrofolate is required to regenerate 5; cl00161
GOHSU22	CDS	GOHSU_22_00400	complement(40266..40640)	overlap	putative AraC family transcriptional regulator [nite]			MLP_05900 :putative AraC family transcriptional regulator : : : Microlunatus phosphovorus NM-1	AraC family transcriptional regulator		D9XMX1_9ACTO AraC family transcriptional regulator OS=Streptomyces griseoflavus Tu4000 GN=SSRG_06124 PE=4 SV= 1;AraC family transcriptional regulator [Burkholderia multivorans ATCC 17616];PFAM: helix-turn-helix- domain containing protein AraC type; ThiJ/PfpI domain protein; KEGG: bch:Bcen2424_5153 transcriptional regulator, AraC family;Transcriptional regulator containing an amidase domain and an AraC-type DNA-binding HTH domain [Transcription];Bacterial regulatory helix-turn-helix proteins, AraC family; pfam00165
GOHSU22	CDS	GOHSU_22_00410	complement(40960..41487)		hypothetical protein [nite]			ROP_00800 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G8RLB3_MYCRN Putative uncharacterized protein OS=My cobacterium rhodesiae (strain NBB3) GN=MycrhN_1764 PE=4 SV =1;unnamed protein product [Gluconacetobacter xylinus NBRC 3288];Protein of unknown function, DUF488; cl01246
GOHSU22	CDS	GOHSU_22_00420	complement(41484..41699)		hypothetical protein [nite]			KRH_23140 :hypothetical protein : : : Kocuria rhizophila DC2201	hypothetical protein		F4GBM4_ALIDK Putative uncharacterized protein OS=Al icycliphilus denitrificans (strain DSM 14773 / CIP 107495 / K601) GN=Alide2_3542 PE=4 SV=1;unnamed protein product [Burkholderia cenocepacia H I2424];KEGG: bcn:Bcen_4925 hypothetical protein;Protein of unknown function (DUF2945); pfam11160
GOHSU22	CDS	GOHSU_22_00430	complement(41696..42799)		putative oxidoreductase [nite]			MLP_50690 :putative oxidoreductase : : : Microlunatus phosphovorus NM-1	2-polyprenyl-6-methoxyphenol 4-hydroxylase		G8RJF3_MYCRN 2-polyprenyl-6-methoxyphenol hydroxyla se-like oxidoreductase OS=Mycobacterium rhodesiae (strain NBB3) GN=MycrhN_4202 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];FAD binding domain; pfam01494;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU22	CDS	GOHSU_22_00440	complement(42819..43616)		putative enoyl-CoA hydratase [nite]			ROP_16190 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus opacus B4	enoyl-CoA hydratase		F1YI20_9ACTO Enoyl-CoA hydratase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_07673 PE=3 SV=1;enoyl-CoA hydratase [Rhodococcus jostii RHA1];enoyl-CoA hydratase; Provisional;Catalyzes the reversible hydration of unsaturated fatty acyl-CoA to beta-hydroxyacyl-CoA;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU22	CDS	GOHSU_22_00450	complement(43613..44746)		putative acetyl-CoA acyltransferase [nite]			RER_51940 :putative acetyl-CoA acyltransferase : : 2.3.1.- : Rhodococcus erythropolis PR4	acetyl-CoA acetyltransferase		H0QYX9_9ACTO Putative acetyl-CoA acyltransferase OS =Gordonia effusa NBRC 100432 GN=GOEFS_045_00200 PE=3 SV=1;acetyl-CoA acetyltransferase [Frankia sp. EuI1c];TIGRFAM: acetyl-CoA acetyltransferase; KEGG: rop:ROP_26220  acetyl-CoA acyltransferase; PFAM: Thiolase-like;acetyl-CoA acetyltransferase; Provisional;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU22	CDS	GOHSU_22_00460	44841..46577		putative molybdenum cofactor biosynthesis protein [nite]			RER_22150 :putative molybdenum cofactor biosynthesis protein : : : Rhodococcus erythropolis PR4	moea domain-containing protein domain i and ii		H0QYX7_9ACTO Putative molybdenum cofactor biosynthe sis protein OS=Gordonia effusa NBRC 100432 GN=GOEFS_045_00 180 PE=4 SV=1;moea domain-containing protein domain i and ii [Mic romonospora sp. L5];PFAM: MoeA domain protein domain I and II; molybdopterin binding domain; MoeA domain protein domain IV; KEGG: nda:Ndas_1112 MoeA domain protein domain I and II;MoeA family. Members of this family are involved in biosynthesis of the molybdenum cofactor (MoCF), an essential cofactor of a diverse group of redox enzymes. MoCF biosynthesis is an evolutionarily conserved pathway present in eubacteria, archaea and...; cd00887;dimer interface [polypeptide binding];putative functional site;putative MPT binding site
GOHSU22	CDS	GOHSU_22_00470	complement(46643..47464)		putative molybdate ABC transporter permease protein [nite]	 modB [nite]		KSE_62340 :putative molybdate ABC transporter permease protein : modB : : Kitasatospora setae NBRC 14216	ABC transporter membrane subunit		G7H115_9ACTO Molybdate ABC transporter permease pro tein OS=Gordonia araii NBRC 100433 GN=modB PE=3 SV=1;ABC transporter membrane subunit [Streptomyces coel icolor A3(2)];SCH35.19c, ABC transporter membrane subunit, len: 283aa; similar to many egs. TR:O31230 (EMBL:Y10817) from Arthrobacter nicotinovorans (239 aa) fasta scores; opt: 745, z-score: 854.5, E(): 0, (52.8% identity in 229 aa overlap) and SW:MODB_ECOLI molybdate transport permease from Escherichia coli (229 aa) fasta scores; opt: 639, z-score: 734.6, E(): 0, (44.7% identity in 219 aa overlap). Contains Pfam match to entry PF00528 BPD_transp, Binding-protein-dependent transport systems inner membrane component and Prosite match to PS00402 Binding-protein-dependent transport systems inner membrane comp sign. Contains possible membrane spanning hydrophobic regions.;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU22	CDS	GOHSU_22_00480	complement(47495..48235)		putative molybdate ABC transporter molybdate-binding protein [nite]	 modA [nite]		KRH_01720 :putative molybdate ABC transporter molybdate-binding protein : modA : : Kocuria rhizophila DC2201	molybdenum ABC transporter substrate-binding protein		G7H114_9ACTO Molybdate ABC transporter substrate bi nding protein OS=Gordonia araii NBRC 100433 GN=modA PE=4 S V=1;molybdenum ABC transporter substrate-binding protei n [Thermomonospora curvata DSM 43183];TIGRFAM: molybdenum ABC transporter, periplasmic molybdate-binding protein; PFAM: extracellular solute-binding protein family 1; KEGG: scl:sce1135 ABC-type molybdate transporter, substrate-binding protein;The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cl11398
GOHSU22	CDS	GOHSU_22_00490	complement(48250..48507)		putative molybdopterin synthase small subunit MoaD [nite]	 moaD [nite]		RER_23420 :putative molybdopterin synthase small subunit MoaD : moaD : : Rhodococcus erythropolis PR4	sulfur carrier protein ThiS		H0QYX1_9ACTO Molybdopterin synthase small subunit M oaD OS=Gordonia effusa NBRC 100432 GN=moaD PE=4 SV=1;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;PFAM: thiamineS protein; KEGG: mav:MAV_0997 ThiS family protein;Ubiquitin domain of MoaD-like proteins; cd00754;MoaE interaction surface [polypeptide binding];MoeB interaction surface [polypeptide binding];thiocarboxylated glycine
GOHSU22	CDS	GOHSU_22_00500	complement(48500..49510)		molybdenum cofactor biosynthesis protein MoaA [nite]	 moaA [nite]		RER_23410 :molybdenum cofactor biosynthesis protein MoaA : moaA : : Rhodococcus erythropolis PR4	molybdenum cofactor biosynthesis protein A	moaA	G7H112_9ACTO Molybdenum cofactor biosynthesis prote in MoaA OS=Gordonia araii NBRC 100433 GN=moaA PE=3 SV=1;molybdenum cofactor biosynthesis protein MoaA [Rhod ococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;together with moaC, is involved in the conversion of a guanosine derivative (GXP) into molybdopterin precursor Z;molybdenum cofactor biosynthesis protein A; Reviewed; PRK00164;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site;Molybdenum Cofactor Synthesis C; pfam06463
GOHSU22	CDS	GOHSU_22_00510	49606..50796		molybdopterin biosynthesis protein MoeA [nite]	 moeA [nite]		ROP_64820 :molybdopterin biosynthesis protein MoeA : moeA : : Rhodococcus opacus B4	molybdenum cofactor synthesis domain-containing protein		H0QYW9_9ACTO Molybdopterin biosynthesis protein Moe A OS=Gordonia effusa NBRC 100432 GN=moeA PE=4 SV=1;molybdenum cofactor synthesis domain-containing pro tein [Geobacter sp. M18];Molybdopterin biosynthesis enzyme [Coenzyme metabolism]; COG0303;TIGRFAM: molybdenum cofactor synthesis domain protein; PFAM: MoeA domain protein domain I and II; MoeA domain protein domain IV; molybdopterin binding domain; KEGG: gbm:Gbem_1349 molybdenum cofactor synthesis domain protein; SMART: molybdopterin binding domain;MoeA family. Members of this family are involved in biosynthesis of the molybdenum cofactor (MoCF), an essential cofactor of a diverse group of redox enzymes. MoCF biosynthesis is an evolutionarily conserved pathway present in eubacteria, archaea and...; cd00887;dimer interface [polypeptide binding];putative functional site;putative MPT binding site
GOHSU22	CDS	GOHSU_22_00520	50781..51845		molybdenum cofactor biosynthesis bifunctional protein MoaCB [nite]	 moaCB [nite]		ROP_64810 :molybdenum cofactor biosynthesis bifunctional protein MoaCB : moaCB : : Rhodococcus opacus B4	bifunctional molybdenum cofactor biosynthesis protein MoaC/MogA	moaC	G7H110_9ACTO Molybdenum cofactor biosynthesis bifun ctional protein MoaCB OS=Gordonia araii NBRC 100433 GN=moa CB PE=3 SV=1;bifunctional molybdenum cofactor biosynthesis prote in MoaC/MogA [Rhodococcus jostii RHA1];MoaC; along with MoaA is involved in conversion of a guanosine derivative into molybdopterin precursor Z; involved in molybdenum cofactor biosynthesis;bifunctional molybdenum cofactor biosynthesis protein MoaC/MogA; Provisional; PRK03604;MoaC family, prokaryotic and eukaryotic. Members of this family are involved in molybdenum cofactor (Moco) biosynthesis, an essential cofactor of a diverse group of redox enzymes. MoaC, a small hexameric protein, converts, together with MoaA, a guanosine...; cd01420;trimer interface [polypeptide binding];dimer interface [polypeptide binding];putative active site [active];MogA_MoaB family. Members of this family are involved in biosynthesis of the molybdenum cofactor (MoCF) an essential cofactor of a diverse group of redox enzymes. MoCF biosynthesis is an evolutionarily conserved pathway present in eubacteria, archaea; cd00886;MPT binding site
GOHSU22	CDS	GOHSU_22_00530	51838..52242		molybdopterin synthase large subunit MoaE [nite]	 moaE [nite]		ROP_64800 :molybdopterin synthase large subunit MoaE : moaE : : Rhodococcus opacus B4	molybdopterin biosynthesis protein E		H0QYW7_9ACTO Molybdopterin synthase large subunit M oaE OS=Gordonia effusa NBRC 100432 GN=moaE PE=4 SV=1;molybdopterin biosynthesis protein E [Streptomyces albus J1074];MoaE family. Members of this family are involved in biosynthesis of the molybdenum cofactor (Moco), an essential cofactor for a diverse group of redox enzymes. Moco biosynthesis is an evolutionarily conserved pathway present in eubacteria, archaea and...; cd00756;MoaE homodimer interface [polypeptide binding];MoaD interaction [polypeptide binding];active site residues [active]
GOHSU22	CDS	GOHSU_22_00540	complement(52253..53122)		hypothetical protein [nite]			ROP_06470 :hypothetical protein : : : Rhodococcus opacus B4	ABC transporter permease		F1YI10_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07623 PE=4 SV=1;unnamed protein product [Bradyrhizobium japonicum U SDA 110];ABC transporter permease;Strictosidine synthase; pfam03088
GOHSU22	CDS	GOHSU_22_00550	53301..53864		putative PadR family transcriptional regulator [nite]			ROP_11250 :putative PadR family transcriptional regulator : : : Rhodococcus opacus B4	PadR-like family transcriptional regulator		F1YI09_9ACTO Transcriptional regulator PadR-like pr otein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07618 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Helix-turn-helix domains; cl00088;Virulence activator alpha C-term; pfam10400
GOHSU22	CDS	GOHSU_22_00560	53977..56040		2,4-dienoyl-CoA reductase [nite]	 fadH [nite]		ROP_11260 :2,4-dienoyl-CoA reductase : fadH : 1.3.1.34 : Rhodococcus opacus B4	NADH:flavin oxidoreductase		F1YI08_9ACTO NADH:flavin oxidoreductase/NADH oxidas e OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07613 PE= 4 SV=1;NADH:flavin oxidoreductase [Marinobacter aquaeolei VT8];PFAM: NADH:flavin oxidoreductase/NADH oxidase; FAD-dependent pyridine nucleotide-disulphide oxidoreductase; KEGG: rha:RHA1_ro01415 2,4-dienoyl-CoA reductase (NADPH);2,4-dienoyl-CoA reductase (DCR) FMN-binding domain. DCR in E. coli  is an iron-sulfur flavoenzyme which contains FMN, FAD, and a 4Fe-4S cluster. It is also a monomer, unlike that of its eukaryotic counterparts which form homotetramers and lack the...; cd02930;FMN binding site [chemical binding];2,4-decadienoyl-CoA binding site;catalytic residue [active];4Fe-4S cluster binding site [ion binding];dihydropyrimidine dehydrogenase subunit A; Provisional
GOHSU22	CDS	GOHSU_22_00570	complement(56054..56470)		myeloperoxidase, thyroid peroxidase-like protein [migap]				myeloperoxidase, thyroid peroxidase-like protein		Q4W2V7_PLARE Merzoite surface protein 1 OS=Plasmodi um reichenowi GN=msp1 PE=4 SV=1;myeloperoxidase, thyroid peroxidase-like protein [S tigmatella aurantiaca DW4/3- 1];Uncharacterized bacterial family of heme peroxidases; cd09819
GOHSU22	CDS	GOHSU_22_00580	complement(56800..57258)		not assigned						G7GRN0_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_50_00270 PE=4 SV=1
GOHSU22	CDS	GOHSU_22_00590	complement(57367..59886)		putative LuxR family transcriptional regulator [nite]			ROP_55170 :putative LuxR family transcriptional regulator : : : Rhodococcus opacus B4	hypothetical protein		G7GNG1_9ACTO Putative LuxR family transcriptional r egulator OS=Gordonia amarae NBRC 15530 GN=GOAMR_28_00230 P E=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF07721;Putative Zn-dependent protease, contains TPR repeats [General function prediction only]
GOHSU22	CDS	GOHSU_22_00600	complement(59903..62434)		regulatory protein [migap]				regulatory protein		G7GNG0_9ACTO Putative LuxR family transcriptional r egulator OS=Gordonia amarae NBRC 15530 GN=GOAMR_28_00220 P E=4 SV=1;regulatory protein [Streptomyces viridochromogenes DSM 40736];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding]
GOHSU22	CDS	GOHSU_22_00610	62519..63331		putative oxidoreductase [nite]			RER_38590 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		F1YI06_9ACTO Short-chain dehydrogenase/reductase SD R OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07603 PE= 3 SV=1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;Short-chain dehydrogenases of various substrate specificities [General function prediction only]; COG0300;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU22	CDS	GOHSU_22_00620	63368..63931		oligoribonuclease [nite]	 orn [nite]		ROP_11290 :oligoribonuclease : orn : 3.1.-.- : Rhodococcus opacus B4	oligoribonuclease	ornA	E9T4L2_COREQ Oligoribonuclease OS=Rhodococcus equi ATCC 33707 GN=orn PE=4 SV=1;oligoribonuclease [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;3'-5' exoribonuclease specific for small oligoribonuclotides;DEDDh 3'-5' exonuclease domain of oligoribonuclease and similar proteins; cd06135;putative active site [active];putative substrate binding site [chemical binding];catalytic site [active];dimer interface [polypeptide binding]
GOHSU22	tRNA	GOHSU_22_t0040	64032..64107		not assigned						identified by tRNAscan-SE; putative
GOHSU22	CDS	GOHSU_22_00630	complement(64204..65331)		hypothetical membrane protein [nite]			ROP_19650 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical membrane protein		A1SNU1_NOCSJ C4-dicarboxylate transporter/malic aci d transport protein OS=Nocardioides sp. (strain BAA-499 / JS614) GN=Noca_3979 PE=4 SV=1;unnamed protein product [Mycobacterium tuberculosis KZN 4207];drug-susceptible;The Tellurite-resistance/Dicarboxylate Transporter (TDT) family; cd09319;gating phenylalanine in ion channel
GOHSU22	CDS	GOHSU_22_00640	complement(65385..66566)		putative hydrolase [nite]			MLP_07840 :putative hydrolase : : : Microlunatus phosphovorus NM-1	arginase		F4CXC2_PSEUX Arginase/agmatinase/formiminoglutamase OS=Pseudonocardia dioxanivorans (strain ATCC 55486 / DSM 44775 / JCM 13855 / CB1190) GN=Psed_4337 PE=4 SV=1;Arginase/agmatinase/formiminoglutamase [Pseudonocar dia dioxanivorans CB1190];PFAM: Ureohydrolase; KEGG: pdi:BDI_1757 arginase;Arginase family; cl00306
GOHSU22	CDS	GOHSU_22_00650	66670..67710		hypothetical membrane protein [nite]			ROP_38290 :hypothetical membrane protein : : : Rhodococcus opacus B4	ribonuclease		D0LEP7_GORB4 Ribonuclease BN OS=Gordonia bronchiali s (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_2530 PE=3 SV=1;ribonuclease [Rhodococcus jostii RHA1]
GOHSU22	CDS	GOHSU_22_00660	complement(67691..69271)		major facilitator superfamily protein [nite]			SLG_06310 :major facilitator superfamily protein : : : Sphingobium sp. SYK-6	MFS permease		H5TU81_9ACTO Putative major facilitator superfamily protein OS=Gordonia otitidis NBRC 100426 GN=GOOTI_256_000 60 PE=4 SV=1;unnamed protein product [Bradyrhizobium japonicum U SDA 110];MFS permease;Transmembrane secretion effector; pfam05977;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174
GOHSU22	CDS	GOHSU_22_00670	69421..71328		hypothetical protein [nite]			RER_40520 :hypothetical protein : : : Rhodococcus erythropolis PR4	RNA polymerase sigma factor, sigma-70 family		I1CXV7_9PSEU RNA polymerase sigma factor, sigma-70 family OS=Saccharomonospora glauca K62 GN=SacglDRAFT_00581 PE=4 SV=1;RNA polymerase sigma factor, sigma-70 family [Sacch aromonospora cyanea NA- 134];PFAM: Sigma-70, region 4;alternative sigma factor RpoH; TIGR02392
GOHSU22	CDS	GOHSU_22_00680	71347..72099		hypothetical protein [nite]			RER_50560 :hypothetical protein : : : Rhodococcus erythropolis PR4	translation initiation factor		G7H7Q0_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_090_00070 PE=4 SV=1;translation initiation factor [Rhodococcus jostii R HA1]
GOHSU22	CDS	GOHSU_22_00690	complement(72118..72624)		putative acetyltransferase [nite]			ANT_12640 :putative acetyltransferase : : 2.3.1.- : Anaerolinea thermophila UNI-1	diamine N-acetyltransferase		F1YHW3_9ACTO Diamine N-acetyltransferase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_07388 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: nfa:nfa27360  acetyltransferase; PFAM: GCN5-related N-acetyltransferase;Acetyltransferases, including N-acetylases of ribosomal proteins [Translation, ribosomal structure and biogenesis]; COG1670;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU22	CDS	GOHSU_22_00700	72679..73077		hypothetical protein [nite]			ROP_59700 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YHY8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07513 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Response regulators consisting of a CheY-like receiver domain and a winged-helix DNA-binding domain [Signal transduction mechanisms / Transcription]; COG0745
GOHSU22	CDS	GOHSU_22_00710	73187..73549	case	NAD(P)H-quinone oxidoreductase subunit 3, chloroplastic [hamap]	ndhC [hamap]	MF_01394:NAD(P)H-quinone oxidoreductase subunit 3, chloroplastic:ndhC 	ROP_59710 :NADH-quinone oxidoreductase chain A : nuoA : 1.6.99.5 : Rhodococcus opacus B4	NADH:ubiquinone oxidoreductase subunit 3 (chain A)		H0R518_9ACTO NADH-quinone oxidoreductase subunit OS =Gordonia effusa NBRC 100432 GN=nuoA PE=3 SV=1;NADH:ubiquinone oxidoreductase subunit 3 (chain A) [Saccharomonospora marina XMU15];Signal predicted by SignalP 3.0 HMM; IMG reference gene:2508555813_SP;PFAM: NADH-ubiquinone/plastoquinone oxidoreductase, chain 3;NADH-ubiquinone/plastoquinone oxidoreductase, chain 3; cl00535
GOHSU22	CDS	GOHSU_22_00720	73540..74094	case	NAD(P)H-quinone oxidoreductase subunit K, chloroplastic [hamap]	ndhK [hamap]	MF_01356:NAD(P)H-quinone oxidoreductase subunit K, chloroplastic:ndhK 	ROP_59720 :NADH-quinone oxidoreductase chain B : nuoB : 1.6.99.5 : Rhodococcus opacus B4	NADH-quinone oxidoreductase, B subunit		H0R519_9ACTO NADH-quinone oxidoreductase subunit B OS=Gordonia effusa NBRC 100432 GN=nuoB PE=3 SV=1;NADH-quinone oxidoreductase, B subunit [Saccharomon ospora marina XMU15];PFAM: NADH ubiquinone oxidoreductase, 20 Kd subunit; TIGRFAM: NADH-quinone oxidoreductase, B subunit;NADH ubiquinone oxidoreductase, 20 Kd subunit; cl15792
GOHSU22	CDS	GOHSU_22_00730	74091..74753	case	NAD(P)H-quinone oxidoreductase subunit J, chloroplastic [hamap]	ndhJ [hamap]	MF_01357:NAD(P)H-quinone oxidoreductase subunit J, chloroplastic:ndhJ 	KSE_45170 :putative NADH-quinone oxidoreductase subunit C : nuoC : 1.6.99.5 : Kitasatospora setae NBRC 14216	NADH/F420H2 dehydrogenase, subunit C		H2JQE9_STRHJ NADH-quinone oxidoreductase subunit C OS=Streptomyces hygroscopicus subsp. jinggangensis (strain 5008) GN=nuoC PE=3 SV=1;NADH/F420H2 dehydrogenase, subunit C [Saccharomonos pora marina XMU15];PFAM: Respiratory-chain NADH dehydrogenase, 30 Kd subunit; TIGRFAM: NADH (or F420H2) dehydrogenase, subunit C;Respiratory-chain NADH dehydrogenase, 30 Kd subunit; cl15771
GOHSU22	CDS	GOHSU_22_00740	74750..76045	case	NAD(P)H-quinone oxidoreductase subunit H, chloroplastic [hamap]	ndhH [hamap]	MF_01358:NAD(P)H-quinone oxidoreductase subunit H, chloroplastic:ndhH 	ROP_59740 :NADH-quinone oxidoreductase chain D : nuoD : 1.6.99.5 : Rhodococcus opacus B4	NAD(P)H-quinone oxidoreductase subunit H		F1YHY0_9ACTO NADH dehydrogenase subunit D OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_07473 PE=3 SV=1;NAD(P)H-quinone oxidoreductase subunit H [Streptomy ces violaceusniger Tu 4113];TIGRFAM: NADH dehydrogenase I, D subunit; HAMAP: NAD(P)H-quinone oxidoreductase subunit H; KEGG: sgr:SGR_2981 NADH dehydrogenase subunit D; PFAM: NADH-quinone oxidoreductase, subunit D;Nickel-dependent hydrogenase; cl00417;Respiratory-chain NADH dehydrogenase, 49 Kd subunit; pfam00346
GOHSU22	CDS	GOHSU_22_00750	76042..76677		putative NADH-quinone oxidoreductase subunit E [nite]	 nuoE [nite]		KSE_45190 :putative NADH-quinone oxidoreductase subunit E : nuoE : 1.6.99.5 : Kitasatospora setae NBRC 14216	NADH-quinone oxidoreductase subunit F		F1YHX9_9ACTO NADH dehydrogenase subunit E OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_07468 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: NADH-quinone oxidoreductase, F subunit; NADH-quinone oxidoreductase, E subunit; PFAM: Respiratory-chain NADH dehydrogenase domain 51 kDa subunit; NADH dehydrogenase (ubiquinone) 24 kDa subunit; Soluble ligand binding domain; NADH ubiquinone oxidoreductase, F subunit, iron sulphur binding; KEGG: rhi:NGR_c22070 NADH dehydrogenase I chain F1;Respiratory-chain NADH dehydrogenase 24 Kd subunit; pfam01257;TRX-like [2Fe-2S] Ferredoxin (Fd) family, NADH:ubiquinone oxidoreductase (Nuo) subunit E subfamily; Nuo, also called respiratory chain Complex 1, is the entry point for electrons into the respiratory chains of bacteria and the mitochondria of eukaryotes; cd03064;putative dimer interface [polypeptide binding];[2Fe-2S] cluster binding site [ion binding]
GOHSU22	CDS	GOHSU_22_00760	76674..77975		NADH-quinone oxidoreductase chain F [nite]	 nuoF [nite]		ROP_59760 :NADH-quinone oxidoreductase chain F : nuoF : 1.6.99.5 : Rhodococcus opacus B4	NADH-quinone oxidoreductase subunit F	nuoF1	F1YHX8_9ACTO NADH-quinone oxidoreductase subunit F OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07463 PE=4 SV=1;NADH-quinone oxidoreductase subunit F [Streptomyces clavuligerus ATCC 27064];COG function: Energy production and conversion. KEGG: Metabolic pathways Oxidative phosphorylation. COG: COG1894 NADH:ubiquinone oxidoreductase, NADH-binding 51 kD subunit.;NADH-quinone oxidoreductase, F subunit; TIGR01959;Respiratory-chain NADH dehydrogenase 51 Kd subunit; pfam01512;SLBB domain; pfam10531;NADH-ubiquinone oxidoreductase-F iron-sulfur binding region; cl11211
GOHSU22	CDS	GOHSU_22_00770	77972..80398		NADH-quinone oxidoreductase chain G [nite]	 nuoG [nite]		ROP_59770 :NADH-quinone oxidoreductase chain G : nuoG : 1.6.99.5 : Rhodococcus opacus B4	NADH-quinone oxidoreductase subunit G		F1YHX7_9ACTO NADH dehydrogenase subunit G OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_07458 PE=4 SV=1;NADH-quinone oxidoreductase subunit G [Streptomyces clavuligerus ATCC 27064];NADH dehydrogenase subunit G; Validated;2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cd00207;catalytic loop [active];iron binding site [ion binding];NADH-ubiquinone oxidoreductase-G iron-sulfur binding region; cl11210;Molybdopterin-Binding (MopB) domain of the MopB superfamily of proteins, a  large, diverse, heterogeneous superfamily of enzymes that, in general, bind molybdopterin as a cofactor. The MopB domain is found in a wide variety of molybdenum- and...; cl09928;molybdopterin cofactor binding site;Molybdopterin-Binding, C-terminal (MopB_CT) domain of the MopB superfamily of proteins, a  large, diverse, heterogeneous superfamily of enzymes that, in general, bind molybdopterin as a cofactor. The MopB domain is found in a wide variety of molybdenum-...; cl09929
GOHSU22	CDS	GOHSU_22_00780	80398..81702		NADH-quinone oxidoreductase chain H [nite]	 nuoH [nite]		RER_27200 :NADH-quinone oxidoreductase chain H : nuoH : 1.6.99.5 : Rhodococcus erythropolis PR4	NADH-quinone oxidoreductase subunit H		H0R525_9ACTO NADH-quinone oxidoreductase subunit H OS=Gordonia effusa NBRC 100432 GN=nuoH PE=3 SV=1;NADH-quinone oxidoreductase subunit H [Streptomyces clavuligerus ATCC 27064];NADH dehydrogenase; cl00469
GOHSU22	CDS	GOHSU_22_00790	81695..82198	case	NAD(P)H-quinone oxidoreductase subunit I, chloroplastic [hamap]	ndhI [hamap]	MF_01351:NAD(P)H-quinone oxidoreductase subunit I, chloroplastic:ndhI 	ROP_59790 :NADH-quinone oxidoreductase chain I : nuoI : 1.6.99.5 : Rhodococcus opacus B4	NADH dehydrogenase subunit I	nuoI_2	I0WF14_9NOCA NADH dehydrogenase subunit I OS=Rhodoc occus imtechensis RKJ300 GN=W59_29650 PE=4 SV=1;NADH dehydrogenase subunit I [Mycobacterium avium s ubsp. paratuberculosis K- 10];NADH dehydrogenase subunit I; Provisional;Catalyzes the transfer of electrons from NADH to quinone;4Fe-4S binding domain; cl02805
GOHSU22	CDS	GOHSU_22_00800	82195..82953		NADH-quinone oxidoreductase chain J [nite]	 nuoJ [nite]		ROP_59800 :NADH-quinone oxidoreductase chain J : nuoJ : 1.6.99.5 : Rhodococcus opacus B4	NADH-ubiquinone/plastoquinone oxidoreductase chain 6		F1YHX4_9ACTO NADH:ubiquinone oxidoreductase subunit J OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07443 PE =3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: NADH-ubiquinone/plastoquinone oxidoreductase chain 6; KEGG: reu:Reut_A0970 NADH dehydrogenase subunit J;NADH:ubiquinone oxidoreductase subunit J; Provisional
GOHSU22	CDS	GOHSU_22_00810	82950..83249		NADH-quinone oxidoreductase chain K [nite]	 nuoK [nite]		RER_27170 :NADH-quinone oxidoreductase chain K : nuoK : 1.6.99.5 : Rhodococcus erythropolis PR4	NADH:ubiquinone oxidoreductase subunit 11 or 4L (chain K)		H8ILK3_MYCIA NADH dehydrogenase subunit K OS=Mycoba cterium intracellulare (strain ATCC 13950 / DSM 43223 / JC M 6384 / NCTC 13025 / 3600) GN=OCU_38920 PE=4 SV=1;NADH:ubiquinone oxidoreductase subunit 11 or 4L (ch ain K) [Saccharomonospora marina XMU15];PFAM: NADH-ubiquinone/plastoquinone oxidoreductase chain 4L;NADH-ubiquinone/plastoquinone oxidoreductase chain 4L; cl00492
GOHSU22	CDS	GOHSU_22_00820	83255..85144		NADH-quinone oxidoreductase chain L [nite]	 nuoL [nite]		RER_27160 :NADH-quinone oxidoreductase chain L : nuoL : 1.6.99.5 : Rhodococcus erythropolis PR4	NADH-quinone oxidoreductase subunit L		F1YHX2_9ACTO NADH:ubiquinone oxidoreductase subunit L OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07433 PE =3 SV=1;NADH-quinone oxidoreductase subunit L [Streptomyces clavuligerus ATCC 27064];NADH:ubiquinone oxidoreductase subunit L; Reviewed;NADH-Ubiquinone oxidoreductase (complex I), chain 5 N-terminus; pfam00662;NADH-Ubiquinone/plastoquinone (complex I), various chains; cl14187
GOHSU22	CDS	GOHSU_22_00830	85141..86655		NADH-quinone oxidoreductase chain M [nite]	 nuoM [nite]		RER_27150 :NADH-quinone oxidoreductase chain M : nuoM : 1.6.99.5 : Rhodococcus erythropolis PR4	NADH-quinone oxidoreductase subunit M	nuoM	H0R530_9ACTO NADH-quinone oxidoreductase subunit M OS=Gordonia effusa NBRC 100432 GN=nuoM PE=3 SV=1;nuoM gene product [Gordonia polyisoprenivorans VH2] ;NADH:ubiquinone oxidoreductase subunit M; Reviewed;NADH-Ubiquinone/plastoquinone (complex I), various chains; cl14187
GOHSU22	CDS	GOHSU_22_00840	86652..88307		NADH-quinone oxidoreductase chain N [nite]	 nuoN [nite]		RER_27140 :NADH-quinone oxidoreductase chain N : nuoN : 1.6.99.5 : Rhodococcus erythropolis PR4	NADH-quinone oxidoreductase subunit N	nuoN	F1YHX0_9ACTO NADH:ubiquinone oxidoreductase subunit N OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07423 PE =3 SV=1;nuoN gene product [Gordonia polyisoprenivorans VH2] ;NADH:ubiquinone oxidoreductase subunit N; Provisional;NADH-Ubiquinone/plastoquinone (complex I), various chains; cl14187
GOHSU22	CDS	GOHSU_22_00850	88375..89373		conserved hypothetical protein [nite]			RER_58460 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	dioxygenase, isopenicillin N synthase		G7GLA2_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_19_00640 PE=3 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Isopenicillin N synthase and related dioxygenases [General function prediction only]; COG3491;PFAM: 2OG-Fe(II) oxygenase superfamily;non-haem dioxygenase in morphine synthesis N-terminal; cl14733;2OG-Fe(II) oxygenase superfamily; cl15773
GOHSU22	CDS	GOHSU_22_00860	complement(89416..90183)		enoyl-CoA hydratase/isomerase family protein [nite]			ROP_41680 :enoyl-CoA hydratase/isomerase family protein : : : Rhodococcus opacus B4	enoyl-CoA hydratase		F1YHW2_9ACTO Enoyl-CoA hydratase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_07383 PE=3 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];Catalyzes the reversible hydration of unsaturated fatty acyl-CoA to beta-hydroxyacyl-CoA;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU22	CDS	GOHSU_22_00870	complement(90180..91706)		putative fatty-acid--CoA ligase [nite]			RER_46870 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus erythropolis PR4	AMP-dependent synthetase and ligase		F1YHW1_9ACTO Fatty-acid--CoA ligase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_07378 PE=4 SV=1;AMP-dependent synthetase and ligase [Rhodopseudomon as palustris BisB5];acyl-CoA synthetase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU22	CDS	GOHSU_22_00880	91778..92341		putative TetR family transcriptional regulator [nite]			ROP_42120 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	transcriptional regulator		F1YHW0_9ACTO TetR family transcriptional regulator OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07373 PE=4 SV=1;transcriptional regulator [Nocardia farcinica IFM 1 0152];Transcriptional regulator [Transcription]; COG1309
GOHSU22	CDS	GOHSU_22_00890	complement(92355..92693)		not assigned						D7GCX2_PROFC Putative uncharacterized protein OS=Pr opionibacterium freudenreichii subsp. shermanii (strain AT CC 9614 / CIP 103027 / CIRM-BIA1) GN=PFREUD_08560 PE=4 SV= 1
GOHSU22	CDS	GOHSU_22_00900	complement(92863..93882)		hypothetical protein [nite]			MLP_51600 :hypothetical protein : : : Microlunatus phosphovorus NM-1	putative monooxygenase		G7GMC0_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_20_02960 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Luciferase-like monooxygenase; pfam00296;Flavin-utilizing monoxygenases; cl07892
GOHSU22	CDS	GOHSU_22_00910	94015..94719		hypothetical protein [migap]				hypothetical protein		F9VSF0_9ACTO Putative Mpr protein OS=Gordonia alkan ivorans NBRC 16433 GN=GOALK_033_00720 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein;Domain of unknown function (DUF4352); pfam11611
GOHSU22	CDS	GOHSU_22_00920	94848..96746		putative sphingolipid ceramide N-deacylase [nite]			KSE_62600 :putative sphingolipid ceramide N-deacylase : : : Kitasatospora setae NBRC 14216	hypothetical protein		G7H309_9ACTO Putative sphingolipid ceramide N-deacy lase OS=Gordonia araii NBRC 100433 GN=GOARA_054_00100 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mxa:MXAN_3638 M19 family peptidase;Superfamily of metallo-dependent hydrolases (also called amidohydrolase superfamily) is a large group of proteins that show conservation in their 3-dimensional fold (TIM barrel) and in details of their active site. The vast majority of the members have a...; cl00281
GOHSU22	CDS	GOHSU_22_00930	96743..98068		putative MFS transporter [nite]			RER_34660 :putative MFS transporter : : : Rhodococcus erythropolis PR4	major facilitator superfamily protein		F1YHU3_9ACTO Major facilitator superfamily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07288 PE=4 SV=1;major facilitator superfamily protein [Micromonospo ra sp. L5];PFAM: major facilitator superfamily MFS_1; KEGG: stp:Strop_0388 major facilitator transporter;MFS/sugar transport protein; pfam13347;melibiose:sodium symporter; Provisional; cl15392
GOHSU22	CDS	GOHSU_22_00940	98065..99546		putative lyase [nite]			RER_34650 :putative lyase : : : Rhodococcus erythropolis PR4	lyase		F1YHU2_9ACTO Pyridoxal-dependent decarboxylase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_07283 PE=3 SV=1 ;lyase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;tyrosine decarboxylase MnfA; TIGR03812;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU22	CDS	GOHSU_22_00950	99658..100104		hemerythrin HHE cation-binding domain-containing protein [migap]				hemerythrin HHE cation-binding domain-containing protein		H5UF14_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_063_00120 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: hemerythrin HHE cation binding domain protein;Hemerythrin (Hr) is a non-heme diiron oxygen transport protein found in four marine invertebrate phyla including priapulida, brachiopoda, sipunculida, and annelida, as well as in protozoa. Myohemerythrin (Mhr), a hemerythrin homolog, is found in the...; cl15774
GOHSU22	CDS	GOHSU_22_00960	complement(100131..101567)		hypothetical protein [nite]			ROP_57120 :hypothetical protein : : : Rhodococcus opacus B4	acyltransferase, WS/DGAT/MGAT		H5TP63_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_152_00200 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Uncharacterised protein family (UPF0089); TIGRFAM: acyltransferase, WS/DGAT/MGAT;acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007
GOHSU22	CDS	GOHSU_22_00970	complement(101557..103065)		putative 1-acylglycerol-3-phosphate O-acyltransferase [nite]	 plsC [nite]		ROP_57100 :putative 1-acylglycerol-3-phosphate O-acyltransferase : plsC : 2.3.1.51 : Rhodococcus opacus B4	HAD-superfamily hydrolase		F1YHU5_9ACTO HAD-superfamily hydrolase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07298 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein; K00655 1-acyl-sn- glycerol-3-phosphate acyltransferase; TIGRFAM: HAD-superfamily subfamily IB hydrolase, TIGR01490; 1-acyl-sn-glycerol-3-phosphate acyltransferase; HAD-superfamily hydrolase, subfamily IB (PSPase-like); PFAM: phospholipid/glycerol acyltransferase; Haloacid dehalogenase domain protein hydrolase; SMART: phospholipid/glycerol acyltransferase;Phosphoserine phosphatase [Amino acid transport and metabolism]; COG0560;Haloacid Dehalogenase-like Hydrolases; cl11391;Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: AGPAT-like; cd07989;putative acyl-acceptor binding pocket
GOHSU22	CDS	GOHSU_22_00980	complement(103062..105341)		putative glycerol-3-phosphate O-acyltransferase [nite]	 plsB [nite]		ROP_57110 :putative glycerol-3-phosphate O-acyltransferase : plsB : 2.3.1.15 : Rhodococcus opacus B4	glycerol-3-phosphate acyltransferase	plsB1	F1YHU4_9ACTO Glycerol-3-phosphate acyltransferase O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07293 PE=4 S V=1;glycerol-3-phosphate acyltransferase [Mycobacterium avium subsp. paratuberculosis K- 10];PlsB; catalyzes the formation of 1-acyl-sn-glycerol 3-phosphate by transfering the acyl moiety from acyl-CoA;glycerol-3-phosphate acyltransferase; Reviewed;Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: GPAT-like; cd07993;putative acyl-acceptor binding pocket
GOHSU22	CDS	GOHSU_22_00990	complement(105588..106727)		acyl-CoA dehydrogenase [nite]	 fadE [nite]		RER_pREL1-02720 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase	fadE	F1YHU1_9ACTO Acyl-CoA dehydrogenase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_07278 PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU22	CDS	GOHSU_22_01000	complement(106826..108049)		conserved hypothetical protein [nite]			RER_38660 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	ErfK/YbiS/YcfS/YnhG family protein		F1YHU0_9ACTO ErfK/YbiS/YcfS/YnhG family protein OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_07273 PE=4 SV= 1;ErfK/YbiS/YcfS/YnhG family protein [Mycobacterium g ilvum PYR- GCK];PFAM: ErfK/YbiS/YcfS/YnhG family protein; KEGG: mmc:Mmcs_3641 ErfK/YbiS/YcfS/YnhG;Uncharacterized protein conserved in bacteria [Function unknown]; COG1376;Bacterial Ig-like domain; cl01012;L,D-transpeptidase catalytic domain; cl15405
GOHSU22	tRNA	GOHSU_22_t0050	complement(108288..108363)		not assigned						identified by tRNAscan-SE; putative
GOHSU22	CDS	GOHSU_22_01010	complement(108456..108680)		hypothetical protein [nite]			ROP_11310 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7H186_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_043_00210 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3618); pfam12277
GOHSU22	CDS	GOHSU_22_01020	108794..109267		probable hydroperoxide peroxidase Bcp [nite]	 bcp [nite]		RER_38690 :probable hydroperoxide peroxidase Bcp : bcp : 1.11.1.- : Rhodococcus erythropolis PR4	Alkyl hydroperoxide reductase/ Thiol specific antioxidant/ Mal allergen		F1YQ22_9ACTO Alkyl hydroperoxide reductase/ Thiol s pecific antioxidant/ Mal allergen OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_20167 PE=4 SV=1;Alkyl hydroperoxide reductase/ Thiol specific antio xidant/ Mal allergen [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q26ZQ4_MYCFV 5e-69;Peroxiredoxin (PRX) family, Bacterioferritin comigratory protein (BCP) subfamily; composed of thioredoxin-dependent thiol peroxidases, widely expressed in pathogenic bacteria, that protect cells against toxicity from reactive oxygen species by reducing...; cd03017;catalytic triad [active]
GOHSU22	CDS	GOHSU_22_01030	109288..109938		putative TetR family transcriptional regulator [nite]			ROP_11330 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		F1YQ21_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_20162 PE=4 SV=1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];PFAM: regulatory protein TetR; KEGG: maq:Maqu_1734 TetR family transcriptional regulator;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU22	CDS	GOHSU_22_01040	complement(109973..110368)		holo-[acyl-carrier-protein] synthase [hamap]	acpS [hamap]	MF_00101:holo-[acyl-carrier-protein] synthase:acpS 	ROP_11340 :holo-[acyl-carrier-protein] synthase : acpS : 2.7.8.7 : Rhodococcus opacus B4	4'-phosphopantetheinyl transferase	acpS	F1YQ20_9ACTO Holo-[acyl-carrier-protein] synthase O S=Gordonia neofelifaecis NRRL B-59395 GN=acpS PE=3 SV=1;4'-phosphopantetheinyl transferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;4'-phosphopantetheinyl transferase superfamily; cl00500;Catalyzes the formation of holo-ACP, which mediates the essential transfer of acyl fatty acid intermediates during the biosynthesis of fatty acids and lipids
GOHSU22	CDS	GOHSU_22_01050	complement(110365..119634)		fatty-acid synthase I [nite]	 fas1 [nite]		ROP_11350 :fatty-acid synthase I : fas1 : 2.3.1.85 : Rhodococcus opacus B4	fatty acid synthase	fas-IA	F1YQ19_9ACTO Fatty acid synthase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_20152 PE=4 SV=1;fatty acid synthase [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;Acyl transferase domain; cl08282;Enoyl reductase domain of yeast-type FAS1 [Lipid metabolism];TIM barrel proteins share a structurally conserved phosphate binding motif and in general share an eight beta/alpha closed barrel structure. Specific for this family is the conserved phosphate binding site at the edges of strands 7 and 8. The phosphate...; cl09108;phosphate binding site [ion binding];Domain of unknown function (DUF1729); pfam08354;The hotdog fold was initially identified in the E. coli FabA (beta-hydroxydecanoyl-acyl carrier protein (ACP)-dehydratase) structure and subsequently in 4HBT (4-hydroxybenzoyl-CoA thioesterase) from Pseudomonas. A number of other seemingly unrelated...; cl00509;active site 2 [active];active site 1 [active];malonyl CoA-acyl carrier protein transacylase; TIGR00128;3-oxoacyl-[acyl-carrier protein];ketoacyl reductase (KR) domain of fungal-type fatty acid synthase (fFAS), classical (c)-like SDRs; cd08950;putative NAD(P) binding site [chemical binding];'elongating' condensing enzymes are a subclass of decarboxylating condensing enzymes, including beta-ketoacyl [ACP] synthase, type I and II and polyketide synthases.They are characterized by the utlization of acyl carrier protein (ACP) thioesters as...; cd00828;3-oxoacyl-(acyl-carrier-protein) synthase [Lipid metabolism / Secondary metabolites biosynthesis, transport, and catabolism]; COG0304
GOHSU22	CDS	GOHSU_22_01060	120058..121899		conserved hypothetical membrane protein [nite]			RER_54790 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	ABC transporter transmembrane protein		G7H1H9_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_045_00580 PE=4 SV=1;ABC transporter transmembrane protein [Rhodococcus equi 103S];YhgE/Pip N-terminal domain; TIGR03061;Predicted membrane protein [Function unknown];X-X-X-Leu-X-X-Gly heptad repeats; TIGR03057;ABC-2 type transporter; cl11417
GOHSU22	CDS	GOHSU_22_01070	121994..122410		HIT family protein [nite]			RER_06660 :HIT family protein : : : Rhodococcus erythropolis PR4	HIT family protein		F9VRD6_9ACTO Putative HIT family protein OS=Gordoni a alkanivorans NBRC 16433 GN=GOALK_023_00170 PE=4 SV=1;HIT family protein [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;HINT (histidine triad nucleotide-binding protein) subgroup: Members of this CD belong to the superfamily of histidine triad hydrolases that act on alpha-phosphate of ribonucleotides. This subgroup includes members from all three forms of cellular life; cd01277;HIT family signature motif;catalytic residue [active]
GOHSU22	CDS	GOHSU_22_01080	complement(122411..124159)		ATP-dependent DEAD-box RNA helicase DeaD [nite]	 deaD [nite]		ROP_60970 :ATP-dependent DEAD-box RNA helicase DeaD : deaD : 3.6.1.- : Rhodococcus opacus B4	putative cold-shock DEAD-box protein		G7H178_9ACTO ATP-dependent DEAD-box RNA helicase De aD OS=Gordonia araii NBRC 100433 GN=deaD PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Superfamily II DNA and RNA helicases [DNA replication, recombination, and repair / Transcription / Translation, ribosomal structure and biogenesis]; COG0513;DEAD-box helicases. A diverse family of proteins involved in ATP-dependent RNA unwinding, needed in a variety of cellular processes including splicing, ribosome biogenesis and RNA degradation. The name derives from the sequence of the Walker  B motif; cd00268;ATP binding site [chemical binding];Mg++ binding site [ion binding];motif III;Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];DbpA RNA binding domain; pfam03880
GOHSU22	CDS	GOHSU_22_01090	124314..126101		probable long-chain fatty-acid--CoA ligase [nite]	 fadD [nite]		RER_40980 :probable long-chain fatty-acid--CoA ligase : fadD : 6.2.1.3 : Rhodococcus erythropolis PR4	AMP-dependent synthetase and ligase		F1YPB2_9ACTO AMP-dependent synthetase and ligase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18823 PE=4 SV =1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: AMP-dependent synthetase and ligase; KEGG: mxa:MXAN_2959  long-chain-fatty-acid-- CoA ligase;Long-chain acyl-CoA synthetases (AMP-forming) [Lipid metabolism]; COG1022;AMP-binding enzyme; cl15778
GOHSU22	CDS	GOHSU_22_01100	126181..127443		hypothetical protein [migap]				hypothetical protein		H5U4R2_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_114_00640 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];OpgC protein; cl00792
GOHSU22	CDS	GOHSU_22_01110	127415..128569		probable methionine gamma-lyase [nite]	 megL [nite]		BBR47_55420 :probable methionine gamma-lyase : megL : 4.4.1.11 : Brevibacillus brevis NBRC 100599	Cys/Met metabolism pyridoxal-phosphate-dependent protein		G7GMQ5_9ACTO Putative methionine gamma-lyase OS=Gor donia amarae NBRC 15530 GN=GOAMR_24_00030 PE=3 SV=1;Cys/Met metabolism pyridoxal-phosphate-dependent pr otein [Caldithrix abyssi DSM 13497];PFAM: Cys/Met metabolism PLP-dependent enzyme; TIGRFAM: methionine gamma-lyase; COGs: COG0626 Cystathionine beta-lyase/cystathionine gamma-synthase; InterPro IPR000277; KEGG: toc:Toce_1559 methionine gamma-lyase; PFAM: Cys/Met metabolism, pyridoxal phosphate-dependent enzyme; SPTR: Methionine gamma-lyase;Cys/Met metabolism PLP-dependent enzyme; pfam01053;CGS_like: Cystathionine gamma-synthase is a PLP dependent enzyme and catalyzes the committed step of methionine biosynthesis. This pathway is unique to microorganisms and plants, rendering the enzyme an attractive target for the development of...; cd00614;homodimer interface [polypeptide binding];substrate-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU22	CDS	GOHSU_22_01120	complement(128566..130068)		hypothetical protein [nite]			ROP_67990 :hypothetical protein : : : Rhodococcus opacus B4	rRNA or tRNA methyltransferase		F1YPB5_9ACTO Methyltransferase small OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_18848 PE=4 SV=1;rRNA or tRNA methyltransferase [Streptomyces clavul igerus ATCC 27064];S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU22	CDS	GOHSU_22_01130	130112..131818		hypothetical protein [migap]				hypothetical protein		H5U4R5_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_114_00670 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU22	CDS	GOHSU_22_01140	131829..132275		predicted protein [migap]				predicted protein		I0H967_ACTMI Putative LuxR-family transcriptional r egulator OS=Actinoplanes missouriensis 431 GN=AMIS_43340 P E=4 SV=1;predicted protein [Streptomyces sp. SPB78]
GOHSU22	tRNA	GOHSU_22_t0060	132303..132387		not assigned						identified by tRNAscan-SE; putative
##GOHSU23	contig00101				61951
GOHSU23	CDS	GOHSU_23_00010	complement(48..368)		hypothetical protein [nite]			KRH_19420 :hypothetical protein : : : Kocuria rhizophila DC2201	rhodanese-related sulfurtransferase		F1YM94_9ACTO Rhodanese domain-containing protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15514 PE=4 SV =1;rhodanese-related sulfurtransferase [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;Rhodanese Homology Domain (RHOD); an alpha beta fold domain found duplicated in the rhodanese protein. The cysteine containing enzymatically active version of the domain is also found in the Cdc25 class of protein phosphatases and a variety of proteins...; cl00125;active site residue [active]
GOHSU23	CDS	GOHSU_23_00020	complement(858..1934)		hypothetical protein [nite]			ROP_60530 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YM96_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15524 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];Protein of unknown function (DUF1006); cl15826
GOHSU23	CDS	GOHSU_23_00030	complement(1943..2641)		peptide methionine sulfoxide reductase MsrA [hamap]	msrA [hamap]	MF_01401:peptide methionine sulfoxide reductase MsrA:msrA 	RER_01660 :peptide methionine sulfoxide reductase MsrA : msrA : 1.8.4.11 : Rhodococcus erythropolis PR4	peptide methionine sulfoxide reductase MsrA	msrA	F1YM97_9ACTO Peptide methionine sulfoxide reductase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15529 PE=3 SV=1;msrA gene product [Gordonia polyisoprenivorans VH2] ;Peptide methionine sulfoxide reductase; cl00366
GOHSU23	CDS	GOHSU_23_00040	2778..2930		not assigned						
GOHSU23	CDS	GOHSU_23_00050	3047..3673		superoxide dismutase [nite]	 sodA [nite]		RER_48670 :superoxide dismutase : sodA : 1.15.1.1 : Rhodococcus erythropolis PR4	superoxide dismutase	sodA	G7H210_9ACTO Superoxide dismutase OS=Gordonia araii NBRC 100433 GN=sodA PE=3 SV=1;superoxide dismutase [Rhodococcus erythropolis PR4] ;This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Superoxide dismutase [Inorganic ion transport and metabolism]; COG0605;protein synonym:Fe/Mn-SOD;Iron/manganese superoxide dismutases, alpha-hairpin domain; pfam00081;Iron/manganese superoxide dismutases, C-terminal domain; pfam02777
GOHSU23	CDS	GOHSU_23_00060	complement(3737..4630)		manganese ABC transporter permease protein [nite]	 mntB [nite]		ROP_70550 :manganese ABC transporter permease protein : mntB : : Rhodococcus opacus B4	chelated iron transport system membrane protein yfeC		G7GZT9_9ACTO Manganese ABC transporter permease pro tein OS=Gordonia araii NBRC 100433 GN=mntB PE=3 SV=1;unnamed protein product [Xenorhabdus bovienii SS- 2 004];Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;ABC-ATPase subunit  interface;dimer interface [polypeptide binding];putative PBP binding regions
GOHSU23	CDS	GOHSU_23_00070	complement(4632..5381)		manganese ABC transporter ATP-binding protein [nite]	 mntA [nite]		ROP_70560 :manganese ABC transporter ATP-binding protein : mntA : : Rhodococcus opacus B4	metal ABC transporter ATP-binding protein		F9W1K8_9ACTO Manganese ABC transporter ATP-binding protein OS=Gordonia alkanivorans NBRC 16433 GN=mntA PE=3 S V=1;metal ABC transporter ATP-binding protein [Rhodococ cus jostii RHA1];tc: 3.A.1.15;ABC-type Mn/Zn transport systems, ATPase component [Inorganic ion transport and metabolism]; COG1121;ABC component of the metal-type transporters.  This family includes transporters involved in the uptake of various metallic cations such as iron, manganese, and zinc.  The ATPases of this group of transporters are very similar to members of...; cd03235
GOHSU23	CDS	GOHSU_23_00080	complement(5378..6295)		manganese ABC transporter manganese-binding protein [nite]	 mntC [nite]		RER_59770 :manganese ABC transporter manganese-binding protein : mntC : : Rhodococcus erythropolis PR4	iron/manganese/zinc ABC transporter substrate-binding protein		F9W1K9_9ACTO Manganese ABC transporter manganese-bi nding protein OS=Gordonia alkanivorans NBRC 16433 GN=mntC PE=3 SV=1;iron/manganese/zinc ABC transporter substrate-bindi ng protein [Arthrobacter arilaitensis Re117];TCDB: ATP-binding cassette (ABC) superfamily, Manganese/Zinc/Iron Chelate Uptake Transporter (MZT) Family (TC 3.A.1.15.z). ABCISSE: ABC transporter, binding protein (BP), MET-family. This family is comprised of systems involved in the uptake of various metallic cations such as iron, manganese and zinc;Metal binding protein PsaA.  These proteins have been shown to function as initial receptors in ABC transport of Mn2+ and as surface adhesins in some eubacterial species.  They belong to the TroA superfamily of periplasmic metal binding proteins that...; cd01137;Periplasmic solute binding protein family; pfam01297;metal binding site [ion binding]
GOHSU23	CDS	GOHSU_23_00090	complement(6385..7137)		putative ABC transporter permease protein [nite]			RER_26880 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	cobalt transporter		D0LB75_GORB4 Cobalt transport protein OS=Gordonia b ronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCT C 10667) GN=Gbro_0158 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: cobalt transporter; KEGG: bpa:BPP3347  cobalt transporter;Cobalt transport protein; cl00463
GOHSU23	CDS	GOHSU_23_00100	complement(7134..9197)		putative ABC transporter ATP-binding protein [nite]			RER_26890 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter-like protein;hypothetical protein		F1YMA4_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_15564 PE=3 SV=1;ABC transporter-like protein [Mycobacterium sp. KMS ];Signal predicted by SignalP 2.0 HMM (Signal peptide probability 0.741) with cleavage site probability 0.502 at residue 45;PFAM: ABC transporter related; SMART: AAA ATPase; KEGG: mmc:Mmcs_3431 ABC transporter related;Domain I of the ABC component of a cobalt transport family found in bacteria, archaea, and eukaryota.  The transition metal cobalt is an essential component of many enzymes and must be transported into cells in appropriate amounts when needed.  This ABC...; cd03225;cobalt transport protein ATP-binding subunit; TIGR01166;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region;ABC-type cobalt transport system, ATPase component [Inorganic ion transport and metabolism]; COG1122
GOHSU23	CDS	GOHSU_23_00110	9324..10079		conserved hypothetical membrane protein [nite]			RER_01580 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H6MT40_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c03330 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted membrane protein [Function unknown];Uncharacterized protein family UPF0016; pfam01169
GOHSU23	CDS	GOHSU_23_00120	complement(10096..11763)		conserved hypothetical protein [nite]			RER_01340 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	recombinase B		F9VVV4_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_056_01660 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: RecB family nuclease, KEGG: sml:Smlt2488 hypothetical protein;RecB family nuclease, putative, TM0106 family;DnaQ-like (or DEDD) 3'-5' exonuclease domain superfamily; cl10012
GOHSU23	CDS	GOHSU_23_00130	11885..12610		conserved hypothetical protein [nite]			RER_01330 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMA8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15584 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GI21568 gene product from transcript GI21568- RA
GOHSU23	CDS	GOHSU_23_00140	12626..13174		transcription termination factor Rho [migap]				transcription termination factor Rho		F0ZL72_DICPU Putative uncharacterized protein OS=Di ctyostelium purpureum GN=DICPUDRAFT_78981 PE=4 SV=1;transcription termination factor Rho [Rothia dentoc ariosa M567]
GOHSU23	CDS	GOHSU_23_00150	complement(13623..15176)		putative aldehyde dehydrogenase [nite]			ROP_38130 :putative aldehyde dehydrogenase : : 1.2.1.- : Rhodococcus opacus B4	succinate-semialdehyde dehydrogenase		H0RBQ3_9ACTO Putative aldehyde dehydrogenase OS=Gor donia polyisoprenivorans NBRC 16320 GN=GOPIP_031_01220 PE= 3 SV=1;succinate-semialdehyde dehydrogenase [Amycolicicocc us subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;Aldehyde dehydrogenase family; pfam00171;Mycobacterium tuberculosis succinate-semialdehyde dehydrogenase 2-like; cd07101;NAD(P) binding site [chemical binding];catalytic residues [active]
GOHSU23	CDS	GOHSU_23_00160	complement(15215..15700)		regulator of ribonuclease activity A [hamap]	rraA [hamap]	MF_00471:regulator of ribonuclease activity A:rraA 	RER_00730 :putative ribonuclease regulator : rraA : : Rhodococcus erythropolis PR4	ribonuclease activity regulator protein RraA	menG	F1YMB6_9ACTO Ribonuclease activity regulator protei n RraA OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1562 4 PE=3 SV=1;ribonuclease activity regulator protein RraA [Coryn ebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;regulator of RNase E; increases half-life and abundance of RNAs; interacts with RNase E possibly inhibiting catalytic activity;Demethylmenaquinone methyltransferase; cl00480
GOHSU23	CDS	GOHSU_23_00170	15836..16087		hypothetical protein [nite]			ROP_35520 :hypothetical protein : : : Rhodococcus opacus B4	RNA binding protein		E4WBB6_RHOE1 RNA binding protein OS=Rhodococcus equ i (strain 103S) GN=REQ_00610 PE=4 SV=1;RNA binding protein [Rhodococcus equi 103S];S4/Hsp/ tRNA synthetase RNA-binding domain; The domain surface is populated by conserved, charged residues that define a likely RNA-binding site;  Found in stress proteins, ribosomal proteins and tRNA synthetases; This may imply a hitherto unrecognized...; cd00165;RNA binding surface [nucleotide binding]
GOHSU23	CDS	GOHSU_23_00180	16089..16391		hypothetical protein [nite]			KRH_21060 :hypothetical protein : : : Kocuria rhizophila DC2201	hypothetical protein		G0HAS6_CORVD Putative uncharacterized protein OS=Co rynebacterium variabile (strain DSM 44702 / JCM 12073 / NC IMB 30131) GN=CVAR_0154 PE=4 SV=1;unnamed protein product [Nitrobacter winogradskyi N b- 255];Bacteriocin-protection, YdeI or OmpD-Associated; pfam13376
GOHSU23	CDS	GOHSU_23_00190	16789..18549		serine/threonine protein kinase-like protein [migap]				serine/threonine protein kinase-like protein		D0L2M3_GORB4 Serine/threonine protein kinase-relate d protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3745 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: serine/threonine protein kinase-related; tyrosine protein kinase; SMART: serine/threonine protein kinase; tyrosine protein kinase; KEGG: mxa:MXAN_4591 serine/threonine-protein kinase PKN1;Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop);Domain of unknown function (DUF4190); pfam13828
GOHSU23	CDS	GOHSU_23_00200	18549..19529		putative esterase [nite]			KSE_25240 :putative esterase : : : Kitasatospora setae NBRC 14216	putative hydrolase		G7GR30_9ACTO Putative esterase OS=Gordonia amarae N BRC 15530 GN=GOAMR_46_01530 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];SEST_like. A family of secreted SGNH-hydrolases similar to Streptomyces scabies esterase (SEST), a causal agent of the potato scab disease, which hydrolyzes a specific ester bond in suberin, a plant lipid. The tertiary fold of this enzyme is...; cd01823;catalytic triad [active];oxyanion hole [active]
GOHSU23	CDS	GOHSU_23_00210	complement(19626..20414)		putative lipase [nite]			KSE_62770 :putative lipase : : : Kitasatospora setae NBRC 14216	hydrolase		Q5YZU8_NOCFA Putative uncharacterized protein OS=No cardia farcinica (strain IFM 10152) GN=NFA_14480 PE=4 SV=1 ;hydrolase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;SEST_like. A family of secreted SGNH-hydrolases similar to Streptomyces scabies esterase (SEST), a causal agent of the potato scab disease, which hydrolyzes a specific ester bond in suberin, a plant lipid. The tertiary fold of this enzyme is...; cd01823;catalytic triad [active];oxyanion hole [active]
GOHSU23	CDS	GOHSU_23_00220	complement(20624..21910)		putative acyltransferase [nite]			KSE_76010t :putative acyltransferase : : 2.3.1.- : Kitasatospora setae NBRC 14216	putative acyltransferase		Q5YZU7_NOCFA Putative acyltransferase OS=Nocardia f arcinica (strain IFM 10152) GN=NFA_14490 PE=4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;OpgC protein; cl00792
GOHSU23	CDS	GOHSU_23_00230	complement(22191..22517)		N-acetylmuramoyl-L-alanine amidase [migap]				N-acetylmuramoyl-L-alanine amidase		E1NJD8_9LACO F5/8 type C domain protein (Fragment) OS=Lactobacillus iners LactinV 09V1-c GN=HMPREF9213_0990 P E=4 SV=1;N-acetylmuramoyl-L-alanine amidase [Finegoldia magn a ATCC 29328]
GOHSU23	CDS	GOHSU_23_00240	complement(22620..24107)		glutamate synthase small subunit [nite]	 gltD [nite]		ROP_35360 :glutamate synthase small subunit : gltD : 1.4.1.13 : Rhodococcus opacus B4	glutamate synthase, NADH/NADPH small subunit		F9VVW7_9ACTO Glutamate synthase small subunit OS=Go rdonia alkanivorans NBRC 16433 GN=gltD PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];glutamate synthase subunit beta; Reviewed; PRK12810;TIGRFAM: glutamate synthase, NADH/NADPH, small subunit; PFAM: FAD-dependent pyridine nucleotide-disulphide oxidoreductase; KEGG: scl:sce5061 glutamate synthase (NADPH);Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU23	CDS	GOHSU_23_00250	complement(24100..28674)		glutamate synthase large subunit [nite]	 gltB [nite]		ROP_35350 :glutamate synthase large subunit : gltB : 1.4.1.13 : Rhodococcus opacus B4	glutamate synthase (NADPH/NADH) large chain	gltB	F1YL23_9ACTO Glutamate synthase OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_13638 PE=4 SV=1;glutamate synthase (NADPH/NADH) large chain [Amycol atopsis mediterranei U32];glutamate synthase subunit alpha; Provisional; PRK11750;Glutamine amidotransferases class-II (Gn-AT), glutamate synthase (GltS)-type. GltS is a homodimer that synthesizes L-glutamate from 2-oxoglutarate and L-glutamine, an important step in ammonia assimilation in bacteria, cyanobacteria and plants. The...; cd00713;dimer interface [polypeptide binding];Glutamate synthase central domain; pfam04898;Glutamate synthase (GltS) FMN-binding domain.  GltS is a complex iron-sulfur flavoprotein that catalyzes the reductive synthesis of L-glutamate from 2-oxoglutarate and L-glutamine via intramolecular channelling of ammonia, a reaction in the plant, yeast...; cd02808;FMN binding site [chemical binding];substrate binding site [chemical binding];3Fe-4S cluster binding site [ion binding];gltb_C. This domain is found at the C-terminus of the large subunit (gltB) of glutamate synthase (GltS). GltS encodes a complex iron-sulfur flavoprotein that catalyzes the synthesis of L-glutamate from L-glutamine and 2-oxoglutarate. It requires the...; cd00982;domain interface
GOHSU23	CDS	GOHSU_23_00260	28906..29910		putative esterase [nite]			ACMV_12400 :putative esterase : : : Acidiphilium multivorum AIU301	esterase/lipase		F1YL22_9ACTO Alpha/beta hydrolase fold-3 domain-con taining protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_13633 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: alpha/beta hydrolase fold;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;alpha/beta hydrolase fold; pfam07859
GOHSU23	CDS	GOHSU_23_00270	30020..30622		conserved hypothetical protein [nite]			RER_47870 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0LB35_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_0116 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF2302; KEGG: sml:Smlt1384 hypothetical protein;Bacterial protein of unknown function (DUF937); cl01528
GOHSU23	CDS	GOHSU_23_00280	30661..32484		hypothetical protein [nite]			MLP_32850 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YL20_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13623 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Predicted membrane protein (DUF2207); pfam09972
GOHSU23	CDS	GOHSU_23_00290	32551..33126		LemA family protein [nite]			CSE_06300 :LemA family protein : : : Caldisericum exile AZM16c01	LemA family protein		F1YL19_9ACTO LemA family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_13618 PE=4 SV=1;LemA family protein [Thermoanaerobacter pseudethano licus ATCC 33223];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.784) with cleavage site probability 0.645 at residue 24;PFAM: LemA family protein;LemA family; cl00742
GOHSU23	CDS	GOHSU_23_00300	complement(33138..33386)		hypothetical protein [migap]				hypothetical protein		F1YL18_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13613 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU23	CDS	GOHSU_23_00310	complement(33383..35182)		putative sulfate transporter [nite]			MLP_40960 :putative sulfate transporter : : : Microlunatus phosphovorus NM-1	sulfate transporter		F1YL16_9ACTO Sulphate transporter OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_13603 PE=4 SV=1;sulfate transporter [Microlunatus phosphovorus NM- 1];strain coidentity: NBRC 101784;Sulfate transporter family; cl15842;Sulfate permease and related transporters (MFS superfamily) [Inorganic ion transport and metabolism]; COG0659;Sulphate Transporter and Anti-Sigma factor antagonist domain of SulP-like sulfate transporters, plays a role in the function and regulation of the transport activity, proposed general NTP binding function; cd07042
GOHSU23	CDS	GOHSU_23_00320	complement(35255..35797)		protein p97; cilium adhesin [migap]	P97 [migap]			protein p97; cilium adhesin	P97	G1QEV8_MYOLU Uncharacterized protein OS=Myotis luci fugus GN=PCLO PE=4 SV=1;protein p97; cilium adhesin [Mycoplasma hyopneumoni ae 232]
GOHSU23	CDS	GOHSU_23_00330	complement(35944..37092)		putative N-acetylglucosamine-6-phosphate deacetylase [nite]	 nagA [nite]		KSE_43990 :putative N-acetylglucosamine-6-phosphate deacetylase : nagA : 3.5.1.25 : Kitasatospora setae NBRC 14216	N-acetylglucosamine-6-phosphate deacetylase		F1YL12_9ACTO N-acetylglucosamine-6-phosphate deacet ylase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13583 PE=3 SV=1;N-acetylglucosamine-6-phosphate deacetylase [Kineoc occus radiotolerans SRS30216];type strain of Kineococcus radiotolerans;TIGRFAM: N-acetylglucosamine-6-phosphate deacetylase; PFAM: amidohydrolase; KEGG: sco:SCO4284  N-acetylglucosamine-6-phosphate deacetylase;N-acetylglucosamine-6-phosphate deacetylase [Carbohydrate transport and metabolism]; COG1820;Superfamily of metallo-dependent hydrolases (also called amidohydrolase superfamily) is a large group of proteins that show conservation in their 3-dimensional fold (TIM barrel) and in details of their active site. The vast majority of the members have a...; cl00281
GOHSU23	CDS	GOHSU_23_00340	37268..37609		deoxyribodipyrimidine photo-lyase type I [migap]				deoxyribodipyrimidine photo-lyase type I		F1YL11_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13578 PE=4 SV=1;deoxyribodipyrimidine photo-lyase type I [Arthrobac ter phenanthrenivorans Sphe3];PFAM: FAD binding domain of DNA photolyase; DNA photolyase;Deoxyribodipyrimidine photolyase [DNA replication, recombination, and repair]; COG0415;FAD binding domain of DNA photolyase; pfam03441
GOHSU23	CDS	GOHSU_23_00350	37731..39074		NADP-specific glutamate dehydrogenase [nite]	 gdhA [nite]		MLP_32750 :NADP-specific glutamate dehydrogenase : gdhA : 1.4.1.4 : Microlunatus phosphovorus NM-1	glutamate dehydrogenase	gdh	F1YL07_9ACTO Glutamate dehydrogenase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_13558 PE=3 SV=1;glutamate dehydrogenase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;glutamate dehydrogenase; Provisional;converts 2-oxoglutarate to glutamate; in Escherichia coli this enzyme plays a role in glutamate synthesis when the cell is under energy restriction; uses NADPH; forms a homohexamer;Glu/Leu/Phe/Val dehydrogenase, dimerisation domain; pfam02812;NAD(P) binding domain of glutamate dehydrogenase, subgroup 2; cd05313;NAD(P) binding site [chemical binding]
GOHSU23	CDS	GOHSU_23_00360	complement(39130..40656)		putative flavin-containing monooxygenase [nite]			RER_34410 :putative flavin-containing monooxygenase : : : Rhodococcus erythropolis PR4	monooxygenase		H0EBB9_9ACTN Cyclohexanone monooxygenase OS=Patulib acter sp. I11 GN=PAI11_41480 PE=4 SV=1;monooxygenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to P71662_MYCTU e-135;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU23	CDS	GOHSU_23_00370	complement(40675..41028)		flavin-binding family monooxygenase [migap]				flavin-binding family monooxygenase		F2RR28_TRIT1 Monooxygenase OS=Trichophyton tonsuran s (strain CBS 112818) GN=TESG_01311 PE=4 SV=1;flavin-binding family monooxygenase [Parvibaculum l avamentivorans DS- 1];KEGG: bja:bll5942 flavin-binding family monooxygenase;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU23	CDS	GOHSU_23_00380	complement(41025..41951)		hypothetical protein [migap]				hypothetical protein		F6EER2_AMYSD Putative uncharacterized protein OS=Am ycolicicoccus subflavus (strain DSM 45089 / DQS3-9A1) GN=A S9A_2415 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cl00264
GOHSU23	CDS	GOHSU_23_00390	complement(41948..42847)		putative oxidoreductase [nite]			YM304_24280 :putative oxidoreductase : : : Acidimicrobidae bacterium YM16-304	oxidoreductase, short chain dehydrogenase/reductase family		E2SCH6_9ACTO Short chain dehydrogenase/reductase fa mily oxidoreductase OS=Aeromicrobium marinum DSM 15272 GN= HMPREF0063_12138 PE=3 SV=1;oxidoreductase, short chain dehydrogenase/reductase family [Alcanivorax sp. DG881];[IQR] COG1028 Dehydrogenases with different specificities (related to short-chain alcohol dehydrogenases);classical (c) SDRs; cd05233;short chain dehydrogenase; Provisional;NAD(P) binding site [chemical binding]
GOHSU23	CDS	GOHSU_23_00400	complement(42844..43578)		putative hydrolase [nite]			ANT_01270 :putative hydrolase : : : Anaerolinea thermophila UNI-1	esterase/lipase		H5RSK6_9NOCA Alpha/beta hydrolase fold-3 protein do main-containing protein OS=Nocardia brasiliensis ATCC 7003 58 GN=O3I_15734 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: alpha/beta hydrolase fold;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;alpha/beta hydrolase fold; pfam07859
GOHSU23	CDS	GOHSU_23_00410	complement(43615..43851)		3-octaprenyl-4-hydroxybenzoate carboxy-lyase [migap]				3-octaprenyl-4-hydroxybenzoate carboxy-lyase		Q74BN7_GEOSL 3-octaprenyl-4-hydroxybenzoate carboxy -lyase family protein OS=Geobacter sulfurreducens (strain ATCC 51573 / DSM 12127 / PCA) GN=GSU2004 PE=4 SV=1;3-octaprenyl-4-hydroxybenzoate carboxy-lyase [Geoba cter sulfurreducens PCA];identified by match to protein family HMM PF01977; match to protein family HMM TIGR00148
GOHSU23	CDS	GOHSU_23_00420	43909..44583		putative TetR family transcriptional regulator [nite]			RER_50040 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		Q1B729_MYCSS Transcriptional regulator, TetR family OS=Mycobacterium sp. (strain MCS) GN=Mmcs_3198 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m gilvum PYR- GCK];PFAM: regulatory protein, TetR; KEGG: mmc:Mmcs_3198 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU23	CDS	GOHSU_23_00430	complement(44580..45845)		putative CdaR family transcriptional regulator [nite]			RER_55770 :putative CdaR family transcriptional regulator : : : Rhodococcus erythropolis PR4	hypothetical protein		H0QWK1_9ACTO Putative CdaR family transcriptional r egulator OS=Gordonia effusa NBRC 100432 GN=GOEFS_021_00290 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];cytoplasmic protein; function unknown but domain identity suggests transcriptional regulator;Helix-turn-helix domains; cl00088
GOHSU23	CDS	GOHSU_23_00440	complement(45851..47272)		conserved hypothetical protein [nite]			RER_55780 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	dehydrogenase		H5TZR8_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_054_00080 PE=4 SV=1;dehydrogenase [Mycobacterium marinum M];Phytoene dehydrogenase and related proteins [Secondary metabolites biosynthesis, transport, and catabolism];function unknown, probably involved in cellular metabolism;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU23	CDS	GOHSU_23_00450	complement(47269..49086)		conserved hypothetical membrane protein [nite]			RER_55790 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0RLU6_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_091_00400 PE =4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];Transmembrane protein of unknown function (DUF3556); pfam12077;membrane protein
GOHSU23	CDS	GOHSU_23_00460	complement(49271..50725)		putative acid--CoA ligase [nite]			ROP_32590 :putative acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	long-chain-fatty-acid--CoA ligase	fadD3	H5TZR6_9ACTO Putative fatty-acid--CoA ligase OS=Gor donia sputi NBRC 100414 GN=GOSPT_054_00060 PE=4 SV=1;long-chain-fatty-acid--CoA ligase [Carboxydothermus hydrogenoformans Z- 2901];identified by similarity to SP:P29212; match to protein family HMM PF00501;long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU23	CDS	GOHSU_23_00470	complement(50858..51655)		hypothetical protein [nite]			RER_27980 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YL06_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13553 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mab:MAB_3767c hypothetical protein
GOHSU23	CDS	GOHSU_23_00480	complement(51684..52508)		hypothetical protein [nite]			RER_38550 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GX62_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_004_00270 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Thioesterase-like superfamily; pfam13622
GOHSU23	CDS	GOHSU_23_00490	52533..53321		hypothetical protein [migap]				hypothetical protein		G7H240_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_048_01170 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mab:MAB_1644 hypothetical protein
GOHSU23	CDS	GOHSU_23_00500	53498..53707		transposase [migap]				transposase		F5XHN2_MICPN Putative transposase OS=Microlunatus p hosphovorus (strain ATCC 700054 / DSM 10555 / JCM 9379 / N BRC 101784 / NCIMB 13414 / VKM Ac-1990 / NM-1) GN=MLP_0163 0 PE=4 SV=1;transposase [Microlunatus phosphovorus NM- 1];strain coidentity: NBRC 101784;fragment
GOHSU23	CDS	GOHSU_23_00510	complement(53724..54377)		not assigned						
GOHSU23	tRNA	GOHSU_23_t0010	complement(54517..54602)		not assigned						identified by tRNAscan-SE; putative
GOHSU23	CDS	GOHSU_23_00520	54855..56225		conserved hypothetical protein [nite]			RER_00340 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	FHA domain-containing protein		H5UL02_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_152_00040 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rer:RER_00340 hypothetical protein; PFAM: Forkhead-associated protein; SMART: Forkhead-associated protein;Protein of unknown function (DUF2662); pfam12401;Forkhead associated domain (FHA); found in eukaryotic and prokaryotic proteins. Putative nuclear signalling domain. FHA domains may bind phosphothreonine, phosphoserine and sometimes phosphotyrosine. In eukaryotes, many FHA domain-containing proteins...; cd00060;phosphopeptide binding site
GOHSU23	CDS	GOHSU_23_00530	56318..56776		conserved hypothetical protein [nite]			RER_00330 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	forkhead-associated protein		F1YLC7_9ACTO Forkhead-associated protein OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_13433 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: Forkhead-associated protein; SMART: Forkhead-associated protein; KEGG: scl:sce2902 GGDEF domain-containing protein;Forkhead associated domain (FHA); found in eukaryotic and prokaryotic proteins. Putative nuclear signalling domain. FHA domains may bind phosphothreonine, phosphoserine and sometimes phosphotyrosine. In eukaryotes, many FHA domain-containing proteins...; cd00060;phosphopeptide binding site
GOHSU23	CDS	GOHSU_23_00540	56773..58206		probable serine/threonine protein phosphatase PstP [nite]	 pstP [nite]		RER_00320 :probable serine/threonine protein phosphatase PstP : pstP : 3.1.3.16 : Rhodococcus erythropolis PR4	phosphoprotein phosphatase		H5U0L0_9ACTO Serine/threonine protein phosphatase P stP OS=Gordonia sputi NBRC 100414 GN=pstP PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: sfu:Sfum_2540 protein serine/threonine phosphatases; PFAM: protein phosphatase 2C-like; SMART: protein phosphatase 2C domain protein;Serine/threonine phosphatases, family 2C, catalytic domain; The protein architecture and deduced catalytic mechanism of PP2C phosphatases are similar to the PP1, PP2A, PP2B family of protein Ser/Thr phosphatases, with which PP2C shares no sequence...; cd00143;Sigma factor PP2C-like phosphatases; smart00331
GOHSU23	CDS	GOHSU_23_00550	58206..59600		putative cell division protein RodA [nite]	 rodA [nite]		ROP_35140 :putative cell division protein RodA : rodA : : Rhodococcus opacus B4	cell division protein FtsW		F1YLC5_9ACTO Cell cycle protein OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_13423 PE=4 SV=1;cell division protein FtsW [Streptomyces sp. AA4];Cell cycle protein; cl00511
GOHSU23	CDS	GOHSU_23_00560	59597..61084		penicillin-binding protein PbpA [nite]	 pbpA [nite]		RER_00300 :penicillin-binding protein PbpA : pbpA : : Rhodococcus erythropolis PR4	penicillin-binding protein transpeptidase		F1YLC4_9ACTO Penicillin-binding protein transpeptid ase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13418 P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.998) with cleavage site probability 0.666 at residue 24;PFAM: penicillin-binding protein transpeptidase; KEGG: ppd:Ppro_2545 peptidoglycan glycosyltransferase;penicillin-binding protein 2; TIGR03423;Penicillin binding protein transpeptidase domain; cl01009
GOHSU23	CDS	GOHSU_23_00570	61081..>61951	3x codon	probable serine/threonine protein kinase PknA [nite]	 pknA [nite]		RER_00290 :probable serine/threonine protein kinase PknA : pknA : 2.7.11.1 : Rhodococcus erythropolis PR4	PASTA sensor-containing serine/threonine protein kinase		F1YLC3_9ACTO Serine/threonine protein kinase OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_13413 PE=4 SV=1;PASTA sensor-containing serine/threonine protein ki nase [Geodermatophilus obscurus DSM 43160];PFAM: Serine/threonine protein kinase-related; tyrosine protein kinase; PASTA domain containing protein; SMART: serine/threonine protein kinase; tyrosine protein kinase; KEGG: nfa:nfa810 putative serine/threonine protein kinase;Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
##GOHSU24	contig00102				58037
GOHSU24	CDS	GOHSU_24_00010	complement(<1..470)	3x codon	hypothetical protein [migap]				hypothetical protein		unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 5 : No homology to any previously reported sequences
GOHSU24	CDS	GOHSU_24_00020	469..2364		probable serine/threonine protein kinase PknB [nite]	 pknB [nite]		RER_00280 :probable serine/threonine protein kinase PknB : pknB : 2.7.11.1 : Rhodococcus erythropolis PR4	serine/threonine-protein kinase PknB	pknB	F1YLC2_9ACTO Serine/threonine protein kinase-like p rotein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1340 8 PE=3 SV=1;pknB gene product [Gordonia polyisoprenivorans VH2] ;Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop);PASTA domain of bacterial serine/threonine kinase pknB-like proteins. PknB is a member of a group of related transmembrane sensor kinases present in many gram positive bacteria, which has been shown to regulate cell shape in Mycobacterium tubercolosis; cd06577
GOHSU24	CDS	GOHSU_24_00030	complement(2372..3022)		anthranilate synthase component II [nite]	 trpG [nite]		RER_00270 :anthranilate synthase component II : trpG : 4.1.3.27 : Rhodococcus erythropolis PR4	para-aminobenzoate synthase component II	trpG_1	F1YLC0_9ACTO Para-aminobenzoate synthase component II OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13398 PE =4 SV=1;para-aminobenzoate synthase component II [Mycobacte rium bovis BCG str. Pasteur 1173P2];aminodeoxychorismate synthase subunit PabA; with PabB catalyzes the formation of 4-amino-4-deoxychorismate from chorismate and glutamine in para-aminobenzoate synthesis; PabA provides the glutamine amidotransferase activity;Type 1 glutamine amidotransferase (GATase1) domain found in Anthranilate synthase; cd01743;Glutamine amidotransferase class-I; pfam00117;glutamine binding [chemical binding];catalytic triad [active]
GOHSU24	CDS	GOHSU_24_00040	3077..3373		hypothetical membrane protein [nite]			ROP_35080 :hypothetical membrane protein : : : Rhodococcus opacus B4	septation inhibitor protein		G7H077_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_035_00070 PE=4 SV=1;septation inhibitor protein [Mycobacterium bovis BC G str. Pasteur 1173P2];Uncharacterized protein family (UPF0233); cl11506;integral membrane protein involved in inhibition of the Z-ring formation
GOHSU24	CDS	GOHSU_24_00050	3486..3938		hypothetical membrane protein [nite]			ROP_35070 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLB8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13388 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU24	CDS	GOHSU_24_00060	complement(3975..4709)		conserved hypothetical membrane protein [nite]			RER_00180 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	rhomboid family protein		F1YLB7_9ACTO Rhomboid family protein OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_13383 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: rhomboid family protein; KEGG: rso:RS04802 hypothetical protein;Rhomboid family; cl11446
GOHSU24	CDS	GOHSU_24_00070	complement(4835..5353)		peptidyl-prolyl cis-trans isomerase [nite]			ROP_35050 :peptidyl-prolyl cis-trans isomerase : : 5.2.1.8 : Rhodococcus opacus B4	cyclophilin type peptidyl-prolyl cis-trans isomerase		E6TGR9_MYCSR Peptidyl-prolyl cis-trans isomerase OS =Mycobacterium sp. (strain Spyr1) GN=Mspyr1_00150 PE=3 SV= 1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;protein synonym:PPIase; protein synonym:rotamase;cyclophilin: cyclophilin-type peptidylprolyl cis- trans isomerases. This family contains eukaryotic, bacterial and archeal proteins which exhibit a peptidylprolyl cis- trans isomerases activity (PPIase, Rotamase) and in addition bind the...; cd00317
GOHSU24	CDS	GOHSU_24_00080	5634..6911		not assigned						
GOHSU24	CDS	GOHSU_24_00090	6908..7627		hypothetical protein [nite]			RER_pREL1-00150 :hypothetical protein : : : Rhodococcus erythropolis PR4	serine esterase, cutinase		D0L5W8_GORB4 Cutinase (Precursor) OS=Gordonia bronc hialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10 667) GN=Gbro_4311 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF01083;Cutinase; pfam01083
GOHSU24	CDS	GOHSU_24_00100	7715..8881		putative esterase [nite]			ACMV_08050 :putative esterase : : : Acidiphilium multivorum AIU301	alpha/beta hydrolase domain-containing protein		F1YLA7_9ACTO Alpha/beta hydrolase fold-3 domain-con taining protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_13333 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;alpha/beta hydrolase fold; pfam07859
GOHSU24	CDS	GOHSU_24_00110	8878..9582		serine esterase, cutinase [migap]				serine esterase, cutinase		D0L5W8_GORB4 Cutinase (Precursor) OS=Gordonia bronc hialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10 667) GN=Gbro_4311 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF01083;Cutinase; pfam01083
GOHSU24	CDS	GOHSU_24_00120	9711..10796		hypothetical protein [nite]			RER_53080 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YKZ3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12902 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rer:RER_53080 hypothetical protein
GOHSU24	CDS	GOHSU_24_00130	complement(10819..11205)		integrase catalytic subunit [migap]				integrase catalytic subunit		Q8VL29_9MICC Putative transposase OS=Micrococcus sp . 28 PE=4 SV=1;unnamed protein product [Methylomonas methanica MC0 9];PFAM: Integrase, catalytic core; KEGG: ppr:PBPRA2371  transposase;Helix-turn-helix domains; cl00088;Transposase and inactivated derivatives [DNA replication, recombination, and repair]; COG2801
GOHSU24	CDS	GOHSU_24_00140	complement(11202..11633)		hypothetical protein [nite]			ROP_pROB02-00840 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		Q8VL91_9MICC Putative transposase OS=Micrococcus sp . 28 PE=4 SV=1;unnamed protein product [Methylomicrobium alcaliphi lum];Evidence 5 : No homology to any previously reported sequences;Helix-turn-helix domains; cl00088;rod shape-determining protein MreC; Provisional
GOHSU24	tRNA	GOHSU_24_t0010	complement(11869..11944)		not assigned						identified by tRNAscan-SE; putative
GOHSU24	tRNA	GOHSU_24_t0020	complement(11971..12047)		not assigned						identified by tRNAscan-SE; putative
GOHSU24	CDS	GOHSU_24_00150	complement(12116..12643)		hypothetical protein [nite]			RER_00110 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLA5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13323 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: CG10555 gene product from transcript CG10555- RA;Transmembrane domain of unknown function (DUF3566); pfam12089
GOHSU24	CDS	GOHSU_24_00160	complement(12624..13094)		not assigned						
GOHSU24	CDS	GOHSU_24_00170	complement(13147..15678)		DNA gyrase subunit A [nite]	 gyrA [nite]		RER_00100 :DNA gyrase subunit A : gyrA : 5.99.1.3 : Rhodococcus erythropolis PR4	DNA gyrase subunit A	gyrA_1	F1YLA4_9ACTO DNA gyrase subunit A OS=Gordonia neofe lifaecis NRRL B-59395 GN=gyrA PE=3 SV=1;DNA gyrase subunit A [Mycobacterium bovis BCG str. Pasteur 1173P2];negatively supercoils closed circular double-stranded DNA;DNA gyrase subunit A; Validated;DNA Topoisomerase, subtype IIA; domain A'; bacterial DNA topoisomerase IV (C subunit, ParC), bacterial DNA gyrases (A subunit, GyrA),mammalian DNA toposiomerases II. DNA topoisomerases are essential enzymes that regulate the conformational changes in DNA...; cd00187;CAP-like domain;primary dimer interface [polypeptide binding];DNA gyrase C-terminal domain, beta-propeller; pfam03989
GOHSU24	CDS	GOHSU_24_00180	complement(15756..17816)		DNA gyrase subunit B [nite]	 gyrB [nite]		ROP_34910 :DNA gyrase subunit B : gyrB : 5.99.1.3 : Rhodococcus opacus B4	DNA gyrase subunit B	gyrB	G7H7A8_9ACTO DNA gyrase subunit B OS=Gordonia araii NBRC 100433 GN=gyrB PE=3 SV=1;gyrB gene product [Mycobacterium smegmatis str. MC2 155];negatively supercoils closed circular double-stranded DNA;DNA gyrase subunit B; Validated; PRK05644;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;Mg2+ binding site [ion binding];G-X-G motif;TopoIIA_Trans_DNA_gyrase: Transducer domain, having a ribosomal S5 domain 2-like fold, of the type found in proteins of the type IIA family of DNA topoisomerases similar to the B subunits of E. coli DNA gyrase and E. coli Topoisomerase IV which are; cd00822;anchoring element;dimer interface [polypeptide binding];ATP binding site [chemical binding];TOPRIM_TopoIIA_GyrB: topoisomerase-primase (TOPRIM) nucleotidyl transferase/hydrolase domain of the type found in proteins of the type IIA family of DNA topoisomerases similar to the Escherichia coli GyrB subunit. TopoIIA enzymes cut both strands of the...; cd03366;putative metal-binding site [ion binding];DNA gyrase B subunit, carboxyl terminus; pfam00986
GOHSU24	CDS	GOHSU_24_00190	complement(17998..18537)		UPF0232 protein <locus_tag> [hamap]		MF_00630:UPF0232 protein <locus_tag>: 	RER_00050 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	Tra3-like protein		H5UEF1_9ACTO UPF0232 protein GOTRE_060_00160 OS=Gor donia terrae NBRC 100016 GN=GOTRE_060_00160 PE=3 SV=1;Tra3-like protein [Streptomyces griseus subsp. gris eus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;Protein of unknown function (DUF721); cl02324
GOHSU24	CDS	GOHSU_24_00200	complement(18534..19751)		DNA replication and repair protein RecF [hamap]	recF [hamap]	MF_00365:DNA replication and repair protein RecF:recF 	ROP_34860 :DNA replication and repair protein RecF : recF : : Rhodococcus opacus B4	recombination protein F	recF	F1YLA1_9ACTO DNA replication and repair protein Rec F OS=Gordonia neofelifaecis NRRL B-59395 GN=recF PE=3 SV=1 ;recombination protein F [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;recombination protein F; Reviewed; PRK00064;Required for DNA replication; binds preferentially to single-stranded, linear DNA;RecF is a recombinational DNA repair ATPase that maintains replication in the presence of DNA damage.  When replication is prematurely disrupted by DNA damage, several recF pathway gene products play critical roles processing the arrested replication...; cd03242;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU24	CDS	GOHSU_24_00210	complement(19751..20671)		putative 6-phosphogluconate dehydrogenase [nite]			ROP_34850 :putative 6-phosphogluconate dehydrogenase : : 1.1.1.44 : Rhodococcus opacus B4	6-phosphogluconate dehydrogenase	gnd2	F1YLA0_9ACTO 6-phosphogluconate dehydrogenase-like protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_132 98 PE=4 SV=1;6-phosphogluconate dehydrogenase [Rhodococcus josti i RHA1];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;6-phosphogluconate dehydrogenase-like protein; Reviewed;similar to full-length Gnd, these proteins seems to have a truncated C-terminal 6PGD domainin; in Methylobacillus flagellatus this gene is essential for NAD+-dependent oxidation of 6-phosphogluconate
GOHSU24	CDS	GOHSU_24_00220	complement(20718..21920)		DNA polymerase III beta subunit [nite]	 dnaN [nite]		RER_00020 :DNA polymerase III beta subunit : dnaN : 2.7.7.7 : Rhodococcus erythropolis PR4	DNA polymerase III subunit beta	dnaN	F1YL99_9ACTO DNA polymerase III subunit beta OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_13293 PE=3 SV=1;DNA polymerase III subunit beta [Nocardia farcinica IFM 10152];DNA polymerase III subunit beta; Validated;Beta clamp domain.  The beta subunit (processivity factor) of DNA polymerase III holoenzyme, refered to as the beta clamp, forms a ring shaped dimer that encircles dsDNA (sliding clamp) in bacteria.  The beta-clamp is structurally similar to the trimeric...; cd00140;binds the polymerase to DNA and acts as a sliding clamp;putative DNA binding surface [nucleotide binding];dimer interface [polypeptide binding];beta-clamp/clamp loader binding surface;beta-clamp/translesion DNA polymerase binding surface
GOHSU24	CDS	GOHSU_24_00230	complement(22419..23891)		chromosomal replication initiator protein DnaA [hamap]	dnaA [hamap]	MF_00377:chromosomal replication initiator protein DnaA:dnaA 	ROP_34830 :chromosomal replication initiator protein DnaA : dnaA : : Rhodococcus opacus B4	chromosomal replication initiation protein	dnaA	D0LA33_GORB4 Chromosomal replication initiator prot ein DnaA OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=dnaA PE=3 SV=1;chromosomal replication initiation protein [Mycobac terium bovis BCG str. Pasteur 1173P2];binds to the dnaA-box as an ATP-bound complex at the origin of replication during the initiation of chromosomal replication; can also affect transcription of multiple genes including itself.;chromosomal replication initiation protein; Reviewed; PRK00149;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger;C-terminal domain of bacterial DnaA proteins. The DNA-binding C-terminal domain of DnaA contains a helix-turn-helix motif that specifically interacts with the DnaA box, a 9-mer motif that occurs repetitively in the replication origin oriC. Multiple...; cd06571;DnaA box-binding interface [nucleotide binding]
GOHSU24	CDS	GOHSU_24_00240	complement(23888..24076)		drug-transport transmembrane ATP-binding protein ABC transporter [migap]				drug-transport transmembrane ATP-binding protein ABC transporter		drug-transport transmembrane ATP-binding protein AB C transporter [Mycobacterium tuberculosis 02_1987];Siderophore-interacting protein [Inorganic ion transport and metabolism]; COG2375;truncated CDS;Siderophore interacting proteins share the domain structure of the ferredoxin reductase like family. Siderophores are produced in various bacteria (and some plants) to extract iron from hosts. Binding constants are high, so iron can be pilfered from...; cd06193;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding]
GOHSU24	CDS	GOHSU_24_00250	24231..24374	short	50S ribosomal protein L34 [hamap]	rpmH [hamap]	MF_00391:50S ribosomal protein L34:rpmH 	ROP_34820 :50S ribosomal protein L34 : rpmH : : Rhodococcus opacus B4	50S ribosomal protein L34	rpmH	D5UPA2_TSUPD 50S ribosomal protein L34 OS=Tsukamure lla paurometabola (strain ATCC 8368 / DSM 20162 / JCM 1011 7 / NBRC 16120 / NCTC 13040) GN=rpmH PE=3 SV=1;50S ribosomal protein L34 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;in Escherichia coli transcription of this gene is enhanced by polyamines
GOHSU24	CDS	GOHSU_24_00260	24400..24822		ribonuclease P protein component [hamap]	rnpA [hamap]	MF_00227:ribonuclease P protein component:rnpA 	RER_60330 :ribonuclease P protein component : rnpA : 3.1.26.5 : Rhodococcus erythropolis PR4	ribonuclease P	rnpA	F1YL96_9ACTO Ribonuclease P protein component OS=Go rdonia neofelifaecis NRRL B-59395 GN=rnpA PE=3 SV=1;ribonuclease P [Corynebacterium glutamicum ATCC 130 32];IS fingerprint type 4-5;Ribonuclease P; cl00457;protein component of RNaseP which catalyzes the removal of the 5'-leader sequence from pre-tRNA to produce the mature 5'terminus; this enzyme also cleaves other RNA substrates
GOHSU24	CDS	GOHSU_24_00270	25113..27101		YidC/Oxa1 family membrane protein insertase [migap]				YidC/Oxa1 family membrane protein insertase		F1YL94_9ACTO Putative inner membrane protein transl ocase component YidC OS=Gordonia neofelifaecis NRRL B-5939 5 GN=SCNU_13268 PE=3 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rer:RER_60310 OxaA family protein; TIGRFAM: membrane protein insertase, YidC/Oxa1 family; PFAM: 60 kDa inner membrane insertion protein;60Kd inner membrane protein; cl00489
GOHSU24	CDS	GOHSU_24_00280	27119..27655		conserved hypothetical protein [nite]			RER_60300 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	RNA-binding protein Jag		G7H799_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_087_00170 PE=4 SV=1;RNA-binding protein Jag [Rhodococcus jostii RHA1];jag_K homology RNA-binding domain. The KH domain is found in proteins homologous to the Bacillus subtilis protein Jag, which is associated with SpoIIIJ and is necessary for the third stage of sporulation.  The KH motif is a beta-alpha-alpha-beta-beta...; cd02414;G-X-X-G motif;R3H domain found in proteins homologous to Bacillus subtilus Jag, which is associated with SpoIIIJ. SpoIIIJ is necessary for the third stage of sporulation. The name of the R3H domain comes from the characteristic spacing of the most conserved arginine...; cd02644;RxxxH motif
GOHSU24	CDS	GOHSU_24_00290	28806..29540	case	ribosomal RNA small subunit methyltransferase G [hamap]	rsmG [hamap]	MF_00074:ribosomal RNA small subunit methyltransferase G:rsmG 	KSE_39130 :putative ribosomal RNA small subunit methyltransferase G : rsmG : 2.1.1.- : Kitasatospora setae NBRC 14216	16S rRNA methyltransferase GidB	gidB	G7H798_9ACTO Ribosomal RNA small subunit methyltran sferase G OS=Gordonia araii NBRC 100433 GN=rsmG PE=3 SV=1;16S rRNA methyltransferase GidB [Corynebacterium gl utamicum ATCC 13032];glucose-inhibited division protein B; SAM-dependent methyltransferase; methylates the N7 position of guanosine in position 527 of 16S rRNA;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cl15754;Methyltransferase domain; pfam13847
GOHSU24	CDS	GOHSU_24_00300	29573..30544		chromosome partitioning protein ParA [nite]	 parA [nite]		ROP_34750 :chromosome partitioning protein ParA : parA : : Rhodococcus opacus B4	chromosome partitioning protein parA	parA_2	D0L9C6_GORB4 Cobyrinic acid ac-diamide synthase OS= Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_4858 PE=4 SV=1;chromosome partitioning protein parA [Mycobacterium bovis BCG str. Pasteur 1173P2];ATPases involved in chromosome partitioning [Cell division and chromosome partitioning]; COG1192;ParA and ParB of Caulobacter crescentus belong to a conserved family of bacterial proteins implicated in chromosome segregation. ParB binds to DNA sequences adjacent to the origin of replication and localizes to opposite cell poles shortly following the...; cd02042;P-loop;Magnesium ion binding site [ion binding]
GOHSU24	CDS	GOHSU_24_00310	30541..31554		probable chromosome partitioning protein ParB [nite]	 parB [nite]		RER_60240 :probable chromosome partitioning protein ParB : parB : : Rhodococcus erythropolis PR4	chromosome partitioning protein parB	parB_2	F1YL90_9ACTO ParB-like partition protein OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_13248 PE=4 SV=1;chromosome partitioning protein parB [Mycobacterium bovis BCG str. Pasteur 1173P2];ParB-like partition proteins; TIGR00180;ParB-like nuclease domain; cl02129
GOHSU24	CDS	GOHSU_24_00320	31689..32387		hypothetical protein [nite]			ROP_34730 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YL89_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13243 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU24	CDS	GOHSU_24_00330	complement(32534..33694)		putative N-acetylmuramoyl-L-alanine amidase [nite]			RER_60220 :putative N-acetylmuramoyl-L-alanine amidase : : 3.5.1.28 : Rhodococcus erythropolis PR4	hydrolase		F1YL88_9ACTO Cell wall hydrolase/autolysin OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_13238 PE=4 SV=1;hydrolase [Mycobacterium bovis BCG str. Pasteur 117 3P2];Putative peptidoglycan-binding domain-containing protein [Cell envelope biogenesis, outer membrane];Putative peptidoglycan binding domain; cl15437;N-acetylmuramoyl-L-alanine amidase [Cell envelope biogenesis, outer membrane]; COG0860;N-acetylmuramoyl-L-alanine amidase or MurNAc-LAA (also known as peptidoglycan aminohydrolase, NAMLA amidase, NAMLAA, Amidase 3, and peptidoglycan amidase; EC 3.5.1.28) is an autolysin that hydrolyzes the amide bond between N-acetylmuramoyl and L-amino...; cd02696;metal binding site [ion binding]
GOHSU24	CDS	GOHSU_24_00340	complement(33765..34103)		thioredoxin [nite]	 trxA [nite]		ROP_34710 :thioredoxin : trxA : : Rhodococcus opacus B4	putative thioredoxin		G7H793_9ACTO Thioredoxin OS=Gordonia araii NBRC 100 433 GN=trxA PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];TRX family; composed of two groups: Group I, which includes proteins that exclusively encode a TRX domain; and Group II, which are composed of fusion proteins of TRX and additional domains. Group I TRX is a small ancient protein that alter the redox...; cd02947;catalytic residues [active]
GOHSU24	CDS	GOHSU_24_00350	complement(34168..35109)		thioredoxin reductase [nite]	 trxB [nite]		RER_60200 :thioredoxin reductase : trxB : 1.8.1.9 : Rhodococcus erythropolis PR4	thioredoxin reductase	trxB	F1YL86_9ACTO Thioredoxin reductase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_13228 PE=3 SV=1;thioredoxin reductase [Rhodococcus erythropolis PR4 ];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;thioredoxin-disulfide reductase; TIGR01292;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Pyridine nucleotide-disulphide oxidoreductase; pfam00070
GOHSU24	CDS	GOHSU_24_00360	complement(35148..35402)		hypothetical protein [nite]			RER_60170 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H6N333_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c50180 P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: BAT2D1; BAT2 domain containing 1
GOHSU24	CDS	GOHSU_24_00370	35502..35810		hypothetical protein [migap]				hypothetical protein		F1YL83_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13213 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola
GOHSU24	CDS	GOHSU_24_00380	complement(35836..36534)		putative TetR family transcriptional regulator [nite]			ROP_54410 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		F1YL82_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_13208 PE=4 SV=1;TetR family transcriptional regulator [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Transcriptional regulator [Transcription]; COG1309
GOHSU24	CDS	GOHSU_24_00390	36602..37510		conserved hypothetical membrane protein [nite]			RER_03360 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	C-5 sterol desaturase		F1YL81_9ACTO Fatty acid hydroxylase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_13203 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Sterol desaturase [Lipid metabolism]; COG3000;Fatty acid hydroxylase superfamily; cl01132
GOHSU24	CDS	GOHSU_24_00400	complement(37545..41267)		conserved hypothetical membrane protein [nite]			RER_60150 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	integral membrane protein MviN		F1YL80_9ACTO Virulence factor MVIN family protein O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13198 PE=4 S V=1;integral membrane protein MviN [Streptomyces sp. AA 4];MviN-like protein; pfam03023
GOHSU24	CDS	GOHSU_24_00410	complement(41264..43735)		conserved hypothetical protein [nite]			RER_60140 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YL79_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13193 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU24	CDS	GOHSU_24_00420	complement(43725..44702)		nudix hydrolase [migap]				nudix hydrolase		F1YL78_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_13188 PE=3 SV=1;nudix hydrolase [Oceanithermus profundus DSM 14977] ;type strain of Oceanithermus profundus;COGs: COG1051 ADP-ribose pyrophosphatase; InterPro IPR000086: IPR020476: IPR020084; KEGG: msv:Mesil_3106 NUDIX hydrolase; PFAM: NUDIX hydrolase; SPTR: NUDIX hydrolase; PFAM: NUDIX domain;Diadenosine hexaphosphate (Ap6A) hydrolase is a member of the Nudix hydrolase superfamily. Ap6A hydrolase specifically hydrolyzes diadenosine polyphosphates, but not ATP or diadenosine triphosphate, and it generates ATP as the product. Ap6A, the most...; cd03673;Ap6A binding site [chemical binding];nudix motif;metal binding site [ion binding]
GOHSU24	CDS	GOHSU_24_00430	44752..46215		tRNA nucleotidyltransferase [nite]			ROP_34610 :tRNA nucleotidyltransferase : : 2.7.7.- : Rhodococcus opacus B4	tRNA adenylyltransferase		F1YL77_9ACTO tRNA adenylyltransferase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_13183 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein; TIGRFAM: tRNA adenylyltransferase; metal dependent phophohydrolase; PFAM: polynucleotide adenylyltransferase region; metal-dependent phosphohydrolase HD sub domain; SMART: metal-dependent phosphohydrolase HD region;tRNA adenylyltransferase; TIGR02692;Nucleotidyltransferase (NT) domain of ClassII CCA-adding enzymes; cd05398;NTP binding site [chemical binding];metal binding triad [ion binding];Probable RNA and SrmB- binding site of polymerase A; pfam12627;Metal dependent phosphohydrolases with conserved 'HD' motif; cd00077;Zn2+ binding site [ion binding];Mg2+ binding site [ion binding]
GOHSU24	CDS	GOHSU_24_00440	46225..47316		putative aminotransferase [nite]			RER_43220 :putative aminotransferase : : 2.6.1.- : Rhodococcus erythropolis PR4	class V aminotransferase		H5UEJ3_9ACTO Putative aminotransferase OS=Gordonia terrae NBRC 100016 GN=GOTRE_060_00590 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: aminotransferase class V; KEGG: scl:sce8911 class-V aminotransferase;Selenocysteine lyase/Cysteine desulfurase [Posttranslational modification, protein turnover, chaperones]; COG0520;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU24	CDS	GOHSU_24_00450	47367..47747	overlap	hypothetical protein [migap]				hypothetical protein		D7WC72_9CORY Putative uncharacterized protein OS=Co rynebacterium genitalium ATCC 33030 GN=HMPREF0291_11081 PE =4 SV=1;unnamed protein product [Zobellia galactanivorans];Would contain ten type-3 Thrombospondin repeats; Signal peptide cleaved between the residues 24 and 25; ly localized in the outer membrane
GOHSU24	CDS	GOHSU_24_00460	complement(47684..49030)	overlap	conserved hypothetical membrane protein [nite]			RER_60110 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	major facilitator superfamily protein		F1YL73_9ACTO Major facilitator superfamily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13163 PE=4 SV=1;major facilitator superfamily protein [Pseudonocard ia dioxanivorans CB1190];PFAM: Major facilitator superfamily MFS-1; KEGG: ami:Amir_7077 major facilitator superfamily MFS_1
GOHSU24	CDS	GOHSU_24_00470	49040..49627	case	UPF0301 protein yqgE [hamap]	yqgE [hamap]	MF_00758:UPF0301 protein yqgE:yqgE 	ROP_34500 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YL74_9ACTO UPF0301 protein SCNU_13168 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13168 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF179; KEGG: mca:MCA0413 hypothetical protein;Uncharacterized ACR, COG1678; cl00731
GOHSU24	CDS	GOHSU_24_00480	49629..50288		hypothetical membrane protein [nite]			DMR_06120 :hypothetical membrane protein : : : Desulfovibrio magneticus RS-1	DedA-like membrane protein	dedA	H5TR86_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_191_00020 PE=4 SV=1;DedA-like membrane protein [Amycolatopsis mediterra nei U32];SNARE associated Golgi protein; cl00429
GOHSU24	CDS	GOHSU_24_00490	complement(50326..51150)		putative oxidoreductase [nite]			ROP_57260 :putative oxidoreductase : : : Rhodococcus opacus B4	short chain dehydrogenase		Q0S4U3_RHOSR Dehydrogenase OS=Rhodococcus sp. (stra in RHA1) GN=RHA1_ro05664 PE=3 SV=1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU24	CDS	GOHSU_24_00500	complement(51222..52220)		conserved hypothetical membrane protein [nite]			RER_60020 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	putative transporter, sodium bile acid symporter family		H0QWM1_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_022_00020 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted Na+-dependent transporter [General function prediction only];Membrane transport protein; cl09117
GOHSU24	CDS	GOHSU_24_00510	complement(52236..52802)		hypothetical membrane protein [nite]			RER_59980 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YL70_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13148 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.998) with cleavage site probability 0.974 at residue 25;KEGG: hypothetical protein;SdpI/YhfL protein family; pfam13630
GOHSU24	CDS	GOHSU_24_00520	52826..55720		leucyl-tRNA synthetase [nite]	 leuS [nite]		ROP_34430 :leucyl-tRNA synthetase : leuS : 6.1.1.4 : Rhodococcus opacus B4	leucyl-tRNA synthetase	leuS	H5U4D3_9ACTO Leucine--tRNA ligase OS=Gordonia sputi NBRC 100414 GN=leuS PE=3 SV=1;leucyl-tRNA synthetase [Corynebacterium glutamicum ATCC 13032];leucine--tRNA ligase; LeuRS; class-I aminoacyl-tRNA synthetase; charges leucine by linking carboxyl group to alpha-phosphate of ATP and then transfers aminoacyl-adenylate to its tRNA; due to the large number of codons that tRNA(Leu) recognizes, the leucyl-tRNA synthetase does not recognize the anticodon loop of the tRNA, but instead recognition is dependent on a conserved discriminator base A37 and a long arm; an editing domain hydrolyzes misformed products; in Methanothermobacter thermautotrophicus this enzyme associates with prolyl-tRNA synthetase;leucyl-tRNA synthetase; Validated; PRK00390;catalytic core domain of leucyl-tRNA synthetases; cd00812;HIGH motif;Leucyl-tRNA synthetase, Domain 2; pfam13603;nucleotidyl transferase superfamily; cl00015;KMSKS motif;Anticodon-binding domain of bacterial and eukaryotic mitochondrial leucyl tRNA synthetases; cd07958;tRNA binding surface [nucleotide binding]
GOHSU24	CDS	GOHSU_24_00530	complement(55760..56860)		hypothetical protein [nite]			ANT_22380 :hypothetical protein : : : Anaerolinea thermophila UNI-1			F1YL69_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13133 PE=4 SV=1
GOHSU24	CDS	GOHSU_24_00540	complement(56891..57877)	overlap	integral membrane protein [migap]				integral membrane protein		F9W037_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_097_02620 PE=4 SV= 1;integral membrane protein [Streptomyces sviceus ATC C 29083];Fusaric acid resistance protein family; pfam04632;Fusaric acid resistance protein-like; cl15844
GOHSU24	CDS	GOHSU_24_00550	57759..>58037	codon; overlap	not assigned						
##GOHSU25	contig00111				90524
GOHSU25	CDS	GOHSU_25_00010	<1..327	codon	not assigned						C0BUS9_9BIFI Putative uncharacterized protein OS=Bi fidobacterium pseudocatenulatum DSM 20438 = JCM 1200 GN=BI FPSEUDO_04167 PE=4 SV=1
GOHSU25	CDS	GOHSU_25_00020	342..1457		hypothetical membrane protein [nite]			ROP_49660 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		D0L6Q6_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_4483 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: conserved hypothetical protein; KEGG: psa:PST_3725 Mg2+ transport ATPase;Predicted integral membrane protein [Function unknown];Uncharacterized protein family (UPF0104); cl04219
GOHSU25	CDS	GOHSU_25_00030	1484..1903		hypothetical membrane protein [nite]			ROP_52120 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H5TL26_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_096_00240 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF3054); pfam11255
GOHSU25	CDS	GOHSU_25_00040	complement(1888..3519)		putative penicillin-binding protein [nite]			ROP_52090 :putative penicillin-binding protein : : : Rhodococcus opacus B4	NTF2 domain-containing protein transpeptidase		F1YKE3_9ACTO NTF2 domain-containing protein transpe ptidase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_115 00 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.998) with cleavage site probability 0.370 at residue 45;PFAM: NTF2 domain protein transpeptidase; KEGG: bch:Bcen2424_6865 YadA domain-containing protein;NTF2-like N-terminal transpeptidase domain; pfam05223;Cell division protein FtsI/penicillin-binding protein 2 [Cell envelope biogenesis, outer membrane]; COG0768
GOHSU25	CDS	GOHSU_25_00050	3620..4384		conserved hypothetical protein [nite]			RER_49570 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	pirin		F1YKE4_9ACTO Pirin-like protein OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_11505 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: pirin; KEGG: bxe:Bxe_A3389 hypothetical protein;Cupin domain; cl09118;Pirin-related protein [General function prediction only]
GOHSU25	CDS	GOHSU_25_00060	4418..6400		metalloendopeptidase [nite]			ROP_51780 :metalloendopeptidase : : 3.4.24.- : Rhodococcus opacus B4	neprilysin		F9VQL0_9ACTO Putative metalloendopeptidase OS=Gordo nia alkanivorans NBRC 16433 GN=GOALK_016_00220 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: peptidase M13; peptidase M13, neprilysin- like; KEGG: she:Shewmr4_3179 endothelin-converting protein 1;Predicted metalloendopeptidase [Posttranslational modification, protein turnover, chaperones]; COG3590;Peptidase family M13 includes neprilysin, endothelin-converting enzyme I; cd08662;Zn binding site [ion binding]
GOHSU25	CDS	GOHSU_25_00070	6397..8493		conserved hypothetical membrane protein [nite]			RER_31970 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	MMPL domain-containing protein		F1YKE6_9ACTO Putative drug exporter of the RND supe rfamily OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_115 15 PE=4 SV=1;MMPL domain-containing protein [Kribbella flavida D SM 17836];PFAM: MMPL domain protein; KEGG: MmpL efflux pump;Predicted drug exporters of the RND superfamily [General function prediction only]
GOHSU25	CDS	GOHSU_25_00080	8585..9121		hypothetical membrane protein [nite]			ROP_62470 :hypothetical membrane protein : : : Rhodococcus opacus B4	Ion transport 2 domain-containing protein		H5U8Q2_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_008_00020 PE=4 SV=1;Ion transport 2 domain-containing protein [Pseudono cardia dioxanivorans CB1190];PFAM: Ion transport 2; KEGG: sma:SAV_3759 hypothetical protein;Ion channel; cl11596
GOHSU25	CDS	GOHSU_25_00090	complement(9158..10015)		putative hydrolase [nite]			RER_47760 :putative hydrolase : : : Rhodococcus erythropolis PR4	alpha/beta hydrolase		F1YKF1_9ACTO Putative hydrolase, alpha/beta fold pr otein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11540 PE=4 SV=1;alpha/beta hydrolase [Rhodococcus equi 103S];2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate hydrolase; TIGR03343;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU25	CDS	GOHSU_25_00100	10069..10365		hypothetical membrane protein [nite]			KRH_05810 :hypothetical membrane protein : : : Kocuria rhizophila DC2201	integral membrane protein		H5TX12_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_025_00540 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 3 : Function proposed based on presence of conserved amino acid motif, structural feature or limited homology
GOHSU25	CDS	GOHSU_25_00110	complement(10457..11332)		putative inositol catabolism protein IolH [nite]	 iolH [nite]		RER_20150 :putative inositol catabolism protein IolH : iolH : : Rhodococcus erythropolis PR4	IolH protein	iolH	G7GUT6_9ACTO Inositol catabolism protein IolH OS=Go rdonia amarae NBRC 15530 GN=iolH PE=4 SV=1;IolH protein [Acidiphilium multivorum AIU301];strain coidentity: NBRC 100883;Sugar phosphate isomerases/epimerases [Carbohydrate transport and metabolism]; COG1082;AP endonuclease family 2; These endonucleases play a role in DNA repair. Cleave phosphodiester bonds at apurinic or apyrimidinic sites; the alignment also contains hexulose-6-phosphate isomerases, enzymes that catalyze the epimerization of...; cl12060;DNA interaction;Metal-binding active site
GOHSU25	CDS	GOHSU_25_00120	complement(11383..12420)	case	inositol 2-dehydrogenase/D-chiro-inositol 3-dehydrogenase [hamap]	iolG [hamap]	MF_01671:inositol 2-dehydrogenase/D-chiro-inositol 3-dehydrogenase:iolG 	ACMV_P1_01120 :inositol 2-dehydrogenase : iolG : 1.1.1.18 : Acidiphilium multivorum AIU301	Inositol 2-dehydrogenase/D-chiro-inositol 3-dehydrogenase		G7GUT5_9ACTO Inositol 2-dehydrogenase OS=Gordonia a marae NBRC 15530 GN=iolG PE=3 SV=1;Inositol 2-dehydrogenase/D-chiro-inositol 3-dehydro genase [Streptomyces violaceusniger Tu 4113];KEGG: scb:SCAB_9601 oxidoreductase; HAMAP: Inositol 2-dehydrogenase/D-chiro-inositol 3-dehydrogenase; PFAM: Oxidoreductase, N-terminal; Oxidoreductase, C-terminal;Predicted dehydrogenases and related proteins [General function prediction only]; COG0673;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU25	CDS	GOHSU_25_00130	complement(12497..13420)		putative 2-keto-myo-inositol dehydratase [nite]	 iolE [nite]		MLP_23520 :putative 2-keto-myo-inositol dehydratase : iolE : 4.2.1.44 : Microlunatus phosphovorus NM-1	phosphate isomerases/epimerase	iolE	G7GUT4_9ACTO 2-keto-myo-inositol dehydratase OS=Gor donia amarae NBRC 15530 GN=iolE PE=4 SV=1;phosphate isomerases/epimerase [Saccharopolyspora e rythraea NRRL 2338];Sugar phosphate isomerases/epimerases [Carbohydrate transport and metabolism]; COG1082
GOHSU25	CDS	GOHSU_25_00140	complement(13456..15384)		putative 3D-(3,5/4)-trihydroxycyclohexane-1,2-dione hydrolase [nite]	 iolD [nite]		RER_20180 :putative 3D-(3,5/4)-trihydroxycyclohexane-1,2-dione hydrolase : iolD : 3.7.1.- : Rhodococcus erythropolis PR4	3D-(3,5/4)-trihydroxycyclohexane-1,2-dione hydrolase	iolD	G7GUT3_9ACTO 3D-(3,5/4)-trihydroxycyclohexane-1,2-d ione hydrolase OS=Gordonia amarae NBRC 15530 GN=iolD PE=3 SV=1;3D-(3,5/4)-trihydroxycyclohexane-1,2-dione hydrolas e [Microlunatus phosphovorus NM- 1];strain coidentity: NBRC 101784;Acetolactate synthase [Amino acid transport and metabolism];Pyrimidine (PYR) binding domain of POX and related proteins; cd07035;PYR/PP interface [polypeptide binding];dimer interface [polypeptide binding];TPP binding site [chemical binding];Thiamine pyrophosphate enzyme, central domain; cl00815;Thiamine pyrophosphate (TPP) family, IolD subfamily, TPP-binding module; composed of proteins similar to Rhizobium leguminosarum bv. viciae IolD. IolD plays an important role in myo-inositol catabolism; cd02003;TPP-binding site
GOHSU25	CDS	GOHSU_25_00150	complement(15381..16337)		5-deoxy-glucuronate isomerase [nite]	 iolB [nite]		MLP_39390 :5-deoxy-glucuronate isomerase : iolB : 5.3.1.- : Microlunatus phosphovorus NM-1	5-deoxy-glucuronate isomerase	iolB	G7GUT2_9ACTO 5-deoxy-glucuronate isomerase OS=Gordo nia amarae NBRC 15530 GN=iolB PE=4 SV=1;5-deoxy-glucuronate isomerase [Microlunatus phospho vorus NM- 1];strain coidentity: NBRC 101784;KduI/IolB family; cl01508
GOHSU25	CDS	GOHSU_25_00160	complement(16423..17922)		methylmalonate-semialdehyde dehydrogenase [nite]	 mmsA [nite]		MLP_24060 :methylmalonate-semialdehyde dehydrogenase : mmsA : 1.2.1.27 : Microlunatus phosphovorus NM-1	methylmalonate-semialdehyde dehydrogenase	mmsA	D5UWX3_TSUPD Methylmalonate-semialdehyde dehydrogen ase OS=Tsukamurella paurometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_1367 PE=3 SV=1;methylmalonate-semialdehyde dehydrogenase [Rhodococ cus opacus B4];Methylmalonate semialdehyde dehydrogenase and ALDH family members 6A1 and 6B2; cd07085;Aldehyde dehydrogenase family; pfam00171;tetrameric interface [polypeptide binding];NAD binding site [chemical binding];catalytic residues [active]
GOHSU25	CDS	GOHSU_25_00170	complement(17963..18952)	overlap	hypothetical protein [nite]			MLP_39400 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		D5UWX2_TSUPD Putative aldolase OS=Tsukamurella paur ometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_1366 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: sus:acid_1482 hypothetical protein
GOHSU25	CDS	GOHSU_25_00180	complement(18889..19845)	overlap	putative 5-dehydro-2-deoxygluconokinase [nite]	 iolC [nite]		ROP_10660 :putative 5-dehydro-2-deoxygluconokinase : iolC : 2.7.1.92 : Rhodococcus opacus B4	ribokinase sugar kinase		D5UWX1_TSUPD PfkB domain protein OS=Tsukamurella pa urometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NB RC 16120 / NCTC 13040) GN=Tpau_1365 PE=4 SV=1;ribokinase sugar kinase [Corynebacterium glutamicum ATCC 13032];2-keto-3-deoxygluconate kinase (KdgK) phosphorylates 2-keto-3-deoxygluconate (KDG) to form 2-keto-3-deoxy-6-phosphogluconate (KDGP). KDG is the common intermediate product, that allows organisms to channel D-glucuronate and/or D-galacturinate into the...; cd01166;Sugar kinases, ribokinase family [Carbohydrate transport and metabolism]; COG0524;substrate binding site [chemical binding];ATP binding site [chemical binding]
GOHSU25	CDS	GOHSU_25_00190	20082..20831		putative GntR family transcriptional regulator [nite]			RER_20220 :putative GntR family transcriptional regulator : : : Rhodococcus erythropolis PR4	GntR family transcriptional regulator		G7GUS9_9ACTO Putative GntR family transcriptional r egulator OS=Gordonia amarae NBRC 15530 GN=GOAMR_64_00370 P E=4 SV=1;GntR family transcriptional regulator [Kribbella fl avida DSM 17836];PFAM: UbiC transcription regulator-associated domain protein; regulatory protein GntR HTH; SMART: regulatory protein GntR HTH; KEGG: gsu:GSU0267 GntR family transcriptional regulator;Transcriptional regulators [Transcription]; COG2188;Winged helix-turn-helix (WHTH) DNA-binding domain of the GntR family of transcriptional regulators; cd07377;DNA-binding site [nucleotide binding];UTRA domain; cl01230
GOHSU25	CDS	GOHSU_25_00200	complement(20852..23545)		putative acetate--CoA ligase [nite]			RER_28030 :putative acetate--CoA ligase : : 6.2.1.13 : Rhodococcus erythropolis PR4	CoA-binding domain-containing protein		G7GTD0_9ACTO Putative acetate--CoA ligase OS=Gordon ia amarae NBRC 15530 GN=GOAMR_59_01560 PE=4 SV=1;CoA-binding domain-containing protein [Actinosynnem a mirum DSM 43827];PFAM: CoA-binding domain protein; GCN5-related N- acetyltransferase; KEGG: mgm:Mmc1_1098 CoA-binding domain protein;Acetyltransferase (GNAT) domain; pfam13302;acetyl coenzyme A synthetase (ADP forming), alpha domain; TIGR02717;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;CoA-ligase; cl02894;ATP-grasp domain; cl03087
GOHSU25	CDS	GOHSU_25_00210	complement(23538..24755)		putative acetoin utilization protein [nite]			ROP_67960 :putative acetoin utilization protein : : : Rhodococcus opacus B4	acetoin dehydrogenase		Q5YT65_NOCFA Putative acetoin dehydrogenase OS=Noca rdia farcinica (strain IFM 10152) GN=NFA_37780 PE=4 SV=1;acetoin dehydrogenase [Nocardia farcinica IFM 10152 ];Deacetylases, including yeast histone deacetylase and acetoin utilization protein [Chromatin structure and dynamics / Secondary metabolites biosynthesis, transport, and catabolism]; COG0123;Histone deacetylase domain; cl02986
GOHSU25	CDS	GOHSU_25_00220	complement(24882..25904)		putative LacI family transcriptional regulator [nite]			ROP_37570 :putative LacI family transcriptional regulator : : : Rhodococcus opacus B4	LacI family transcriptional regulator		G7GUS8_9ACTO Putative LacI family transcriptional r egulator OS=Gordonia amarae NBRC 15530 GN=GOAMR_64_00360 P E=4 SV=1;LacI family transcriptional regulator [Arthrobacter chlorophenolicus A6];PFAM: regulatory protein LacI; periplasmic binding protein/LacI transcriptional regulator; KEGG: aau:AAur_1987 LacI family transcriptional regulator;Transcriptional regulators [Transcription]; COG1609;Helix-turn-helix (HTH) DNA binding domain of the LacI family of transcriptional regulators; cd01392;DNA binding site [nucleotide binding];domain linker motif;Ligand binding domain of the LacI tanscriptional regulator family belonging to the type I periplasmic-binding fold protein superfamily; cd06267;ligand binding site [chemical binding];dimerization interface [polypeptide binding]
GOHSU25	CDS	GOHSU_25_00230	26038..26274		Oxidoreductase family, NAD-binding Rossmann fold protein [migap]				Oxidoreductase family, NAD-binding Rossmann fold protein		C3UMZ9_9MICO Putative oxioreductase OS=Microbacteri um sp. MA1 PE=4 SV=1;Oxidoreductase family, NAD-binding Rossmann fold pr otein [Octadecabacter antarcticus 307];[R] COG0673 Predicted dehydrogenases and related proteins;Predicted dehydrogenases and related proteins [General function prediction only]; COG0673;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU25	CDS	GOHSU_25_00240	26771..27235		putative oxidoreductase [nite]			MLP_28230 :putative oxidoreductase : : : Microlunatus phosphovorus NM-1	oxidoreductase family, NAD-binding Rossmann fold domain protein		G7GUS7_9ACTO Putative oxidoreductase OS=Gordonia am arae NBRC 15530 GN=GOAMR_64_00350 PE=4 SV=1;oxidoreductase family, NAD-binding Rossmann fold do main protein [Streptomyces himastatinicus ATCC 53653];Predicted dehydrogenases and related proteins [General function prediction only]; COG0673
GOHSU25	CDS	GOHSU_25_00250	27285..28301		hypothetical protein [nite]			MLP_28220 :hypothetical protein : : : Microlunatus phosphovorus NM-1	xylose isomerase domain-containing protein		G7GUS6_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_64_00340 PE=4 SV=1;xylose isomerase domain-containing protein [Bacillu s megaterium DSM 319];Sugar phosphate isomerases/epimerases [Carbohydrate transport and metabolism]; COG1082;AP endonuclease family 2; These endonucleases play a role in DNA repair. Cleave phosphodiester bonds at apurinic or apyrimidinic sites; the alignment also contains hexulose-6-phosphate isomerases, enzymes that catalyze the epimerization of...; cl12060;DNA interaction;Metal-binding active site;AP endonuclease family 2 C terminus; pfam07582
GOHSU25	CDS	GOHSU_25_00260	complement(28331..29518)		putative oxidoreductase [nite]			RER_20230 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	oxidoreductase		D5UWW4_TSUPD Oxidoreductase domain protein OS=Tsuka murella paurometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_1357 PE=4 SV=1;oxidoreductase [Rhodococcus opacus B4];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Oxidoreductase family, C-terminal alpha/beta domain; cl11611
GOHSU25	CDS	GOHSU_25_00270	complement(29566..29922)		hypothetical protein [nite]			MLP_23600 :hypothetical protein : : : Microlunatus phosphovorus NM-1	universal stress protein		Q0SCD4_RHOSR Universal stress protein OS=Rhodococcu s sp. (strain RHA1) GN=RHA1_ro02999 PE=4 SV=1;universal stress protein [Rhodococcus jostii RHA1];Usp: Universal stress protein family. The universal stress protein Usp is a small cytoplasmic bacterial protein whose expression is enhanced when the cell is exposed to stress agents. Usp enhances the rate of cell survival during prolonged exposure to...; cd00293;Ligand Binding Site [chemical binding]
GOHSU25	CDS	GOHSU_25_00280	complement(30034..31413)		putative myo-inositol transporter IolT [nite]	 iolT [nite]		ROP_37590 :putative myo-inositol transporter IolT : iolT : : Rhodococcus opacus B4	major facilitator superfamily sugar transporter		H0RCL5_9ACTO Myo-inositol transporter IolT OS=Gordo nia polyisoprenivorans NBRC 16320 GN=iolT PE=3 SV=1;major facilitator superfamily sugar transporter [Rh odococcus jostii RHA1];tc: 2.A.1.1;Sugar (and other) transporter; pfam00083;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU25	CDS	GOHSU_25_00290	complement(31739..32356)		putative TetR family transcriptional regulator [nite]			ROP_pROB01-04390 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		G7H1Q8_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_047_00180 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m gilvum PYR- GCK];PFAM: regulatory protein, TetR; KEGG: mmc:Mmcs_1335 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU25	CDS	GOHSU_25_00300	complement(32375..33574)		rubredoxin reductase [nite]	 rubB [nite]		ROP_pROB01-04380 :rubredoxin reductase : rubB : 1.18.1.- : Rhodococcus opacus B4	FAD-dependent pyridine nucleotide-disulfide oxidoreductase		F1YKF3_9ACTO Rubredoxin reductase OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_11550 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Pyridine nucleotide-disulphide oxidoreductase; pfam00070;Pyridine nucleotide-disulphide oxidoreductase; pfam07992;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU25	CDS	GOHSU_25_00310	complement(33571..33762)		rubredoxin [nite]	 rubA [nite]		ROP_22590 :rubredoxin : rubA : : Rhodococcus opacus B4	rubredoxin	rubA	F5YT11_MYCSD Rubredoxin OS=Mycobacterium sp. (strai n JDM601) GN=rubB PE=3 SV=1;rubredoxin [Rhodococcus opacus B4];Rubredoxin [Energy production and conversion];Rubredoxin; nonheme iron binding domains containing a [Fe(SCys)4] center. Rubredoxins are small nonheme iron proteins. The iron atom is coordinated by four cysteine residues (Fe(S-Cys)4), but iron can also be replaced by cobalt, nickel or zinc. They are...; cd00730;iron binding site [ion binding]
GOHSU25	CDS	GOHSU_25_00320	complement(33764..35296)		alkane-1-monooxygenase [nite]	 alkB [nite]		ROP_22570 :alkane-1-monooxygenase : alkB : 1.14.15.3 : Rhodococcus opacus B4	alkane-1-monooxygenase	alkB	F1YKF6_9ACTO Alkane 1-monooxygenase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_11565 PE=4 SV=1;alkane-1-monooxygenase [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Alkane hydroxylase is a bacterial, integral-membrane di-iron enzyme that shares a requirement for iron and oxygen for activity similar to that of the non-heme integral-membrane acyl coenzyme A (CoA) desaturases and acyl lipid desaturases. The alk genes...; cd03512;Fatty acid desaturase; pfam00487;Di-iron ligands [ion binding]
GOHSU25	CDS	GOHSU_25_00330	35422..36243		hypothetical protein [nite]			MLP_51610 :hypothetical protein : : : Microlunatus phosphovorus NM-1	GDSL family lipase		F1YKF7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11570 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];PFAM: lipolytic enzyme, G-D-S-L family; KEGG: lxx:Lxx11940 hypothetical protein;Members of the SGNH-hydrolase superfamily, a diverse family of lipases and esterases. The tertiary fold of the enzyme is substantially different from that of the alpha/beta hydrolase family and unique among all known hydrolases; its active site closely...; cd01832;catalytic triad [active];oxyanion hole [active]
GOHSU25	CDS	GOHSU_25_00340	complement(36248..36751)		not assigned						
GOHSU25	CDS	GOHSU_25_00350	complement(36847..37131)		hypothetical protein [nite]			YM304_40350 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	rhodanese-like protein		C7R5I0_JONDD Rhodanese domain protein (Precursor) O S=Jonesia denitrificans (strain ATCC 14870 / DSM 20603 / C IP 55134) GN=Jden_1605 PE=4 SV=1;rhodanese-like protein [Haliscomenobacter hydrossis DSM 1100];collected before 1973;COGs: COG0607 Rhodanese-related sulfurtransferase; InterPro IPR001763; KEGG: dly:Dehly_1001 rhodanese domain-containing protein; PFAM: Rhodanese-like; SMART: Rhodanese-like; SPTR:  uncharacterized protein; IMG reference gene:2504771905; PFAM: Rhodanese-like domain;Rhodanese Homology Domain (RHOD); an alpha beta fold domain found duplicated in the rhodanese protein. The cysteine containing enzymatically active version of the domain is also found in the Cdc25 class of protein phosphatases and a variety of proteins...; cd00158;active site residue [active]
GOHSU25	CDS	GOHSU_25_00360	complement(37262..38053)		conserved hypothetical membrane protein [nite]			RER_pREL1-02230 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	permease		I0UYE4_9PSEU Putative permease (Precursor) OS=Sacch aromonospora xinjiangensis XJ-54 GN=SacxiDRAFT_0625 PE=4 S V=1;permease [Mycobacterium sp. Spyr1];PFAM: Sulfite exporter TauE/SafE;Sulfite exporter TauE/SafE; pfam01925;Predicted permeases [General function prediction only]; cl00498
GOHSU25	CDS	GOHSU_25_00370	complement(38079..39497)		conserved hypothetical protein [nite]			RER_pREL1-02200 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	metallo-beta-lactamase superfamily protein		F0M6X2_ARTPP Zn-dependent hydrolase, glyoxylase OS= Arthrobacter phenanthrenivorans (strain DSM 18606 / JCM 16 027 / LMG 23796 / Sphe3) GN=Asphe3_36680 PE=4 SV=1;metallo-beta-lactamase superfamily protein [Arthrob acter aurescens TC1];identified by match to protein family HMM PF00753;Metallo-beta-lactamase superfamily; cl00446;Rhodanese Homology Domain (RHOD); an alpha beta fold domain found duplicated in the rhodanese protein. The cysteine containing enzymatically active version of the domain is also found in the Cdc25 class of protein phosphatases and a variety of proteins...; cd00158;active site residue [active]
GOHSU25	CDS	GOHSU_25_00380	complement(39533..39718)		hypothetical protein [nite]			RER_pREL1-02260 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		A7B8Z9_9ACTO Putative uncharacterized protein OS=Ac tinomyces odontolyticus ATCC 17982 GN=ACTODO_00099 PE=4 SV =1;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;KEGG: jde:Jden_0156 hypothetical protein
GOHSU25	CDS	GOHSU_25_00390	complement(39958..42159)		fatty oxidation complex alpha subunit [nite]	 fadB [nite]		ROP_41320 :fatty oxidation complex alpha subunit : fadB : : Rhodococcus opacus B4	3-hydroxyacyl-CoA dehydrogenase NAD-binding protein		G7H6W9_9ACTO Fatty oxidation complex alpha subunit OS=Gordonia araii NBRC 100433 GN=fadB PE=3 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: 3-hydroxyacyl-CoA dehydrogenase NAD-binding; enoyl-CoA hydratase/isomerase; 3-hydroxyacyl-CoA dehydrogenase domain protein; KEGG: mab:MAB_0851 fatty oxidation protein FadB;multifunctional fatty acid oxidation complex subunit alpha; Reviewed; PRK11730;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;3-hydroxyacyl-CoA dehydrogenase, C-terminal domain; pfam00725
GOHSU25	CDS	GOHSU_25_00400	complement(42194..43405)		putative acetyl-CoA acyltransferase [nite]	 fadA [nite]		RER_03800 :putative acetyl-CoA acyltransferase : fadA : 2.3.1.16 : Rhodococcus erythropolis PR4	acetyl-CoA acetyltransferase		G7H6X0_9ACTO Acetyl-CoA acyltransferase OS=Gordonia araii NBRC 100433 GN=fadA PE=3 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;acetyl-CoA acetyltransferase; Validated;TIGRFAM: acetyl-CoA acetyltransferase; KEGG: svi:Svir_30250 acetyl-CoA acetyltransferase; PFAM: thiolase;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU25	CDS	GOHSU_25_00410	43530..44717		putative TetR family transcriptional regulator [nite]			ROP_61700 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	transcriptional regulator		F1YKG1_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_11590 PE=4 SV=1;transcriptional regulator [Saccharomonospora marina XMU15];PFAM: Bacterial regulatory proteins, tetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU25	CDS	GOHSU_25_00420	44710..45786		adenylate cyclase [nite]	 cya [nite]		ROP_51450 :adenylate cyclase : cya : 4.6.1.1 : Rhodococcus opacus B4	adenylate cyclase	cya	F1YKG2_9ACTO Adenylyl cyclase class-3/4/guanylyl cy clase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11595 PE=4 SV=1;adenylate cyclase [Rhodococcus opacus B4];protein synonym:adenylyl cyclase;cyclase homology domain; cd07302;nucleotidyl binding site;metal binding site [ion binding];dimer interface [polypeptide binding]
GOHSU25	CDS	GOHSU_25_00430	complement(46001..47269)		putative NagC family transcriptional regulator [nite]			ROP_17530 :putative NagC family transcriptional regulator : : : Rhodococcus opacus B4	ROK family protein		F1YKG4_9ACTO ROK family protein OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_11605 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transcriptional regulator/sugar kinase [Transcription / Carbohydrate transport and metabolism]; COG1940;FGGY family of carbohydrate kinases, N-terminal domain; cl09121
GOHSU25	CDS	GOHSU_25_00440	complement(47541..48455)		conserved hypothetical protein [nite]			RER_04400 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	peptidase S15		D2SE65_GEOOG Peptidase S15 OS=Geodermatophilus obsc urus (strain ATCC 25078 / DSM 43160 / JCM 3152 / G-20) GN= Gobs_1832 PE=4 SV=1;peptidase S15 [Conexibacter woesei DSM 14684];PFAM: peptidase S15; KEGG: mpt:Mpe_A2886 hypothetical protein;Hydrolases of the alpha/beta superfamily [General function prediction only];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU25	CDS	GOHSU_25_00450	48602..49549		putative AraC family transcriptional regulator [nite]			RER_59180 :putative AraC family transcriptional regulator : : : Rhodococcus erythropolis PR4	AraC family transcriptional regulator		C8X8B4_NAKMY Transcriptional regulator, AraC family OS=Nakamurella multipartita (strain ATCC 700099 / DSM 442 33 / JCM 9543 / Y-104) GN=Namu_0675 PE=4 SV=1;AraC family transcriptional regulator [Pseudonocard ia dioxanivorans CB1190];KEGG: nml:Namu_0675 AraC family transcriptional regulator; PFAM: HTH transcriptional regulator, AraC; SMART: Helix-turn-helix, AraC type, DNA binding domain;DNA-binding transcriptional activator FeaR; Provisional;Bacterial regulatory helix-turn-helix proteins, AraC family; pfam00165
GOHSU25	CDS	GOHSU_25_00460	complement(49602..50024)		conserved hypothetical protein [nite]			RER_35290 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	deazaflavin-dependent nitroreductase family protein		F1YKH7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11670 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function (DUF385); TIGRFAM: deazaflavin-dependent nitroreductase family protein;Domain of unknown function (DUF385); cl04387
GOHSU25	CDS	GOHSU_25_00470	complement(50095..50763)		phosphoglycerate mutase family protein [nite]			YM304_00160 :phosphoglycerate mutase family protein : : : Acidimicrobidae bacterium YM16-304	phosphoglycerate mutase family protein		H5U8V2_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_009_00440 PE=4 SV=1;phosphoglycerate mutase family protein [Burkholderi a mallei JHU];Histidine phosphatase domain found in a functionally diverse set of proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cl11399;identified by match to protein family HMM PF00300
GOHSU25	CDS	GOHSU_25_00480	complement(50767..51507)		putative oxidoreductase [nite]			ROP_46500 :putative oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase		F9VT81_9ACTO Putative oxidoreductase OS=Gordonia al kanivorans NBRC 16433 GN=GOALK_045_00150 PE=4 SV=1;oxidoreductase [Rhodococcus opacus B4];Short-chain alcohol dehydrogenase of unknown specificity [General function prediction only];classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU25	CDS	GOHSU_25_00490	complement(51504..52520)		putative phosphotransferase [nite]			ROP_20520 :putative phosphotransferase : : 2.7.-.- : Rhodococcus opacus B4	phosphotransferase		G7H6X9_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_078_00530 PE=4 SV=1;phosphotransferase [Rhodococcus jostii RHA1];Predicted aminoglycoside phosphotransferase [General function prediction only];Acyl-CoA dehydrogenase (ACAD) 10 and 11, N-terminal domain, and similar proteins. This subfamily is part of a larger superfamily that includes the catalytic domains of other kinases, such as the typical serine/threonine/tyrosine protein kinases (PKs); cd05154;putative active site [active];putative substrate binding site [chemical binding];ATP binding site [chemical binding]
GOHSU25	CDS	GOHSU_25_00500	complement(52513..53730)		putative acyl-CoA dehydrogenase [nite]			ROP_20530 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		H0R177_9ACTO Putative acyl-CoA dehydrogenase OS=Gor donia effusa NBRC 100432 GN=GOEFS_066_00020 PE=3 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 3 : Function proposed based on presence of conserved amino acid motif, structural feature or limited homology; Product type e : enzyme;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase, N-terminal domain; pfam02771;Acyl-CoA dehydrogenase; cd00567
GOHSU25	CDS	GOHSU_25_00510	complement(53794..54396)		putative TetR family transcriptional regulator [nite]			RER_52540 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		H5U8V6_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia terrae NBRC 100016 GN=GOTRE_009_00480 PE=4 SV=1;TetR family transcriptional regulator [Rhodococcus opacus B4];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU25	CDS	GOHSU_25_00520	complement(54407..55060)		putative TetR family transcriptional regulator [nite]			ROP_54410 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		F1YKG9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11630 PE=4 SV=1;TetR family transcriptional regulator [Amycolicicoc cus subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU25	CDS	GOHSU_25_00530	55160..56050		conserved hypothetical protein [nite]			RER_55080 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YKH0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11635 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Uncharacterized protein conserved in bacteria (DUF2236); cl01472
GOHSU25	CDS	GOHSU_25_00540	56262..57827		hypothetical protein [nite]			MLP_36960 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YHW7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07408 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF222;Domain of unknown function (DUF222); pfam02720;HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and...; cd00085
GOHSU25	CDS	GOHSU_25_00550	complement(57897..59183)		putative FMN-dependent dehydrogenase [nite]			ROP_52640 :putative FMN-dependent dehydrogenase : : : Rhodococcus opacus B4	FMN-dependent alpha-hydroxy acid dehydrogenase		F1YKH1_9ACTO Lactate 2-monooxygenase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_11640 PE=4 SV=1;FMN-dependent alpha-hydroxy acid dehydrogenase [Str eptomyces violaceusniger Tu 4113];PFAM: FMN-dependent dehydrogenase; KEGG: sma:SAV_7482 L-lactate 2-monooxygenase;L-Lactate 2-monooxygenase (LMO) FMN-binding domain. LMO is a FMN-containing enzyme that catalyzes the conversion of L-lactate and oxygen to acetate, carbon dioxide, and water. LMO is a member of the family of alpha-hydroxy acid oxidases.  It is thought...; cd03332;FMN-dependent dehydrogenase; pfam01070;putative active site [active];putative substrate binding site [chemical binding];putative FMN binding site [chemical binding];putative catalytic residues [active]
GOHSU25	CDS	GOHSU_25_00560	complement(59184..59675)		hypothetical protein [nite]			ROP_49560 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0R182_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_066_00070 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];GXWXG protein; pfam14231;Domain of unknown function (DUF4334); pfam14232
GOHSU25	CDS	GOHSU_25_00570	complement(59672..61165)		monooxygenase [nite]			RER_48790 :monooxygenase : : : Rhodococcus erythropolis PR4	flavoprotein involved in K+ transport		G7GLX8_9ACTO Putative monooxygenase OS=Gordonia ama rae NBRC 15530 GN=GOAMR_20_01510 PE=4 SV=1;flavoprotein involved in K+ transport [Mycobacteriu m sp. Spyr1];PFAM: Flavin-binding monooxygenase-like;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU25	CDS	GOHSU_25_00580	complement(61183..62106)		DNA end-binding protein Ku [nite]			ROP_69900 :DNA end-binding protein Ku : : : Rhodococcus opacus B4	DNA end-binding protein Ku		G7GRN4_9ACTO Probable DNA repair protein GOAMR_50_0 0310 OS=Gordonia amarae NBRC 15530 GN=GOAMR_50_00310 PE=3 SV=1;DNA end-binding protein Ku [Amycolatopsis mediterra nei U32];Ku protein, prokaryotic; TIGR02772;Ku-core domain, Ku-like subfamily; composed of prokaryotic homologs of the eukaryotic DNA binding protein Ku. The alignment includes the core domain shared by the prokaryotic YkoV-like proteins and the eukaryotic Ku70 and Ku80. The prokaryotic Ku...; cd00789;putative DNA binding site [nucleotide binding];putative homodimer interface [polypeptide binding]
GOHSU25	CDS	GOHSU_25_00590	complement(62130..62669)		hypothetical protein [migap]				hypothetical protein		H0R3D0_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_092_00060 PE=4 SV=1;unnamed protein product [Ralstonia eutropha JMP134] 
GOHSU25	CDS	GOHSU_25_00600	62774..64771		DNA polymerase LigD [migap]				DNA polymerase LigD		F1YKH6_9ACTO ATP-dependent DNA ligase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_11665 PE=4 SV=1;DNA polymerase LigD [Stackebrandtia nassauensis DSM 44728];TIGRFAM: DNA polymerase LigD, polymerase domain protein; PFAM: DNA primase small subunit; KEGG: acp:A2cp1_1020 DNA polymerase LigD, polymerase domain protein;LigD_Pol_like: Polymerase (Pol) domain of bacterial LigD proteins similar to Pseudomonas aeruginosa (Pae) LigD. The LigD Pol domain belongs to the archaeal/eukaryal primase (AEP) superfamily. In prokaryotes, LigD along with Ku is required for...; cd04861;nucleotide binding site [chemical binding]
GOHSU25	CDS	GOHSU_25_00610	complement(64779..66614)		putative oxidoreductase [nite]			RER_06610 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	flavoprotein involved in K+ transport-like protein		H5TLV3_9ACTO Putative oxidoreductase OS=Gordonia ot itidis NBRC 100426 GN=GOOTI_109_00220 PE=4 SV=1;flavoprotein involved in K+ transport-like protein [Acidiphilium cryptum JF- 5];Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU25	CDS	GOHSU_25_00620	66795..67997		conserved hypothetical protein [nite]			RER_06580 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	GAF domain-containing protein		G7GXB8_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_006_00020 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: GAF domain protein; KEGG: dar:Daro_1017 helix-turn-helix, fis-type;Transcriptional activator of acetoin/glycerol metabolism [Secondary metabolites biosynthesis, transport, and catabolism / Transcription]; COG3284;Helix-turn-helix domains; cl00088
GOHSU25	CDS	GOHSU_25_00630	68052..68828		putative 3',5'-cyclic-nucleotide phosphodiesterase [nite]			RER_40650 :putative 3',5'-cyclic-nucleotide phosphodiesterase : : 3.1.4.17 : Rhodococcus erythropolis PR4	metallophosphoesterase		F1YKI1_9ACTO Metallophosphoesterase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_11690 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: rpc:RPC_1269 metallophosphoesterase;Enterobacter aerogenes GpdQ and related proteins, metallophosphatase domain; cd07402;Calcineurin-like phosphoesterase; pfam00149;metal binding site [ion binding];hexamer interface [polypeptide binding]
GOHSU25	CDS	GOHSU_25_00640	complement(68992..70446)		putative pyridine nucleotide transhydrogenase beta subunit [nite]	 pntB [nite]		KSE_59690 :putative pyridine nucleotide transhydrogenase beta subunit : pntB : 1.6.1.2 : Kitasatospora setae NBRC 14216	NAD(P) transhydrogenase subunit beta	pntB	G7H694_9ACTO NAD(P) transhydrogenase subunit beta O S=Gordonia araii NBRC 100433 GN=pntB PE=3 SV=1;pntB gene product [Mycobacterium smegmatis str. MC2 155];identified by similarity to SP:P07002; match to protein family HMM PF02233;NAD(P) transhydrogenase beta subunit; pfam02233;NAD/NADP transhydrogenase beta subunit [Energy production and conversion]; cl00566
GOHSU25	CDS	GOHSU_25_00650	complement(70450..71982)		putative pyridine nucleotide transhydrogenase alpha subunit [nite]	 pntA [nite]		KSE_59700 :putative pyridine nucleotide transhydrogenase alpha subunit : pntA : 1.6.1.2 : Kitasatospora setae NBRC 14216	NAD(P) transhydrogenase subunit alpha	pntA	G7H695_9ACTO Pyridine nucleotide transhydrogenase a lpha subunit OS=Gordonia araii NBRC 100433 GN=pntA PE=4 SV =1;pntA gene product [Mycobacterium smegmatis str. MC2 155];forms a tetramer composed of 2 subunit alphas and 2 beta subunits in the inner membrane; involved in catalyzing transfer of hydride ion equivalents between NAD and NADP; stereospecific (AB-specific); functions as a proton pump by translocating protons from cytoplasm to periplasm;NAD(P) transhydrogenase subunit alpha; Provisional; PRK09424;Alanine dehydrogenase/PNT, N-terminal domain; cl15379;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Domain of unknown function (DUF3814); pfam12769
GOHSU25	CDS	GOHSU_25_00660	complement(72241..73731)		hypothetical protein [nite]			RER_33860 :hypothetical protein : : : Rhodococcus erythropolis PR4	fumarate reductase/succinate dehydrogenase		F1YK27_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11720 PE=4 SV=1;fumarate reductase/succinate dehydrogenase [Rhodoco ccus equi 103S];hypothetical protein; Validated
GOHSU25	CDS	GOHSU_25_00670	complement(73787..74803)		not assigned						
GOHSU25	CDS	GOHSU_25_00680	complement(74842..76428)		L-aspartate oxidase [nite]	 nadB [nite]		ROP_07620 :L-aspartate oxidase : nadB : 1.4.3.16 : Rhodococcus opacus B4	L-aspartate oxidase		F1YK47_9ACTO L-aspartate oxidase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_11820 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: L-aspartate oxidase; PFAM: fumarate reductase/succinate dehydrogenase flavoprotein domain protein; FAD dependent oxidoreductase; KEGG: mxa:MXAN_4629 L-aspartate oxidase;L-aspartate oxidase; TIGR00551;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Fumarate reductase flavoprotein C-term; pfam02910
GOHSU25	CDS	GOHSU_25_00690	76761..78137		hypothetical protein [nite]			ROP_21660 :hypothetical protein : : : Rhodococcus opacus B4	protein kinase	AarF	F1YK48_9ACTO ABC-1 domain-containing protein OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_11825 PE=4 SV=1;protein kinase [Thermoanaerobacter tengcongensis MB 4];type strain of Thermoanaerobacter tengcongensis MB4;Predicted unusual protein kinase [General function prediction only]; COG0661;Best Blastp hit =  gi|10175329|dbj|BAB06427.1| (AP001516) ABC transporter [Bacillus halodurans], score 318, E-value 1.00E-85;Protein Kinases, catalytic domain; cl09925;Aminoglycoside 3'-phosphotransferase (APH) and Choline Kinase (ChoK) family. The APH/ChoK family is part of a larger superfamily that includes the catalytic domains of other kinases, such as the typical serine/threonine/tyrosine protein kinases (PKs); cd05120;ATP binding site [chemical binding]
GOHSU25	CDS	GOHSU_25_00700	78250..79812		hypothetical membrane protein [nite]			ROP_52400 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F9VTD7_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_045_00710 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.998) with cleavage site probability 0.709 at residue 23;KEGG: OGFR; opioid growth factor receptor;Protein of unknown function (DUF3068); pfam11271
GOHSU25	CDS	GOHSU_25_00710	complement(79919..81592)		conserved hypothetical protein [nite]			RER_53440 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative FAD-binding dehydrogenase		F1YK50_9ACTO Putative FAD-binding dehydrogenase OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11835 PE=4 SV= 1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG3573 Predicted oxidoreductase;putative FAD-binding dehydrogenase; Reviewed
GOHSU25	CDS	GOHSU_25_00720	81875..82816		putative mycolyltransferase [nite]			ROP_41180 :putative mycolyltransferase : : 2.3.1.- : Rhodococcus opacus B4	mycolyltransferase		F1YK52_9ACTO Putative esterase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_11845 PE=4 SV=1;mycolyltransferase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Predicted esterase [General function prediction only];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU25	CDS	GOHSU_25_00730	82952..83974		conserved hypothetical protein [nite]			RER_59910 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	dyp-type peroxidase family protein		H6MU82_GORPV Dyp-type peroxidase family protein OS= Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GP OL_c04890 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Dyp-type peroxidase family; cl01067
GOHSU25	CDS	GOHSU_25_00740	84042..84845		probable bacteriocin [nite]			RER_59900 :probable bacteriocin : : : Rhodococcus erythropolis PR4	Linocin_M18 bacteriocin protein		G7GRR2_9ACTO Putative bacteriocin OS=Gordonia amara e NBRC 15530 GN=GOAMR_50_00590 PE=4 SV=1;Linocin_M18 bacteriocin protein [Streptomyces sp. A A4];Uncharacterized protein, linocin/CFP29 homolog [Function unknown];Encapsulating protein for peroxidase; cl12047
GOHSU25	CDS	GOHSU_25_00750	complement(84907..85368)		not assigned						G7GM08_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_20_01810 PE=4 SV=1
GOHSU25	CDS	GOHSU_25_00760	85461..87194		putative ABC transporter permease/ATP-binding protein [nite]			RER_03410 :putative ABC transporter permease/ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter ATP-binding protein/permease		F1YK55_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_11860 PE=3 SV=1;ABC transporter permease/ATP-binding protein [Rhodo coccus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;ABC-type multidrug transport system, ATPase and permease components [Defense mechanisms]; COG1132;ABC transporter transmembrane region; cl00549;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU25	CDS	GOHSU_25_00770	87191..89125		putative ABC transporter permease/ATP-binding protein [nite]			RER_03420 :putative ABC transporter permease/ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter ATP-binding protein/permease		F1YK56_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_11865 PE=3 SV=1;ABC transporter permease/ATP-binding protein [Rhodo coccus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;ABC transporter transmembrane region; cl00549;ABC-type multidrug transport system, ATPase and permease components [Defense mechanisms]; COG1132;Glucan exporter ATP-binding protein.  In A. tumefaciens cyclic beta-1, 2-glucan must be transported into the periplasmic space to exert its action as a virluence factor.  This subfamily belongs to the MRP-like family and is involved in drug, peptide, and...; cd03254;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU25	CDS	GOHSU_25_00780	89193..>90524	codon	putative oxidoreductase [nite]			MLP_11520 :putative oxidoreductase : : : Microlunatus phosphovorus NM-1	putative nitrite reductase, copper-dependent		C0E510_9CORY Putative uncharacterized protein OS=Co rynebacterium matruchotii ATCC 33806 GN=CORMATOL_02085 PE= 4 SV=1;unnamed protein product [Propionibacterium acnes Ty peIA2 P.acn33];COG2132 Putative multicopper oxidases
##GOHSU26	contig00112				16594
GOHSU26	CDS	GOHSU_26_00010	<1..1388	3x codon frame	putative oxidoreductase [nite]			MLP_11520 :putative oxidoreductase : : : Microlunatus phosphovorus NM-1	oxidoreductase		F5XNQ2_MICPN Putative oxidoreductase OS=Microlunatu s phosphovorus (strain ATCC 700054 / DSM 10555 / JCM 9379 / NBRC 101784 / NCIMB 13414 / VKM Ac-1990 / NM-1) GN=MLP_ 11520 PE=4 SV=1;oxidoreductase [Microlunatus phosphovorus NM- 1];strain coidentity: NBRC 101784;nitrite reductase, copper-containing; TIGR02376;Cytochrome C oxidase subunit II, periplasmic domain; cl11412
GOHSU26	CDS	GOHSU_26_00020	1539..1874		NADPH:quinone reductase-like Zn-dependent oxidoreductase [migap]				NADPH:quinone reductase-like Zn-dependent oxidoreductase		NADPH:quinone reductase-like Zn-dependent oxidoredu ctase [Lactobacillus rhamnosus Lc 705];alcohol dehydrogenase and quinone reductase-like medium chain degydrogenases/reductases; cd05289;NADPH:quinone reductase and related Zn-dependent oxidoreductases [Energy production and conversion / General function prediction only]; COG0604;putative NAD(P) binding site [chemical binding]
GOHSU26	CDS	GOHSU_26_00030	complement(1953..2405)		ribonuclease H [hamap]	rnhA [hamap]	MF_00042:ribonuclease H:rnhA 	SLG_05680 :ribonuclease H : rnhA : 3.1.26.4 : Sphingobium sp. SYK-6	ribonuclease H		H0RC62_9ACTO Ribonuclease H OS=Gordonia polyisopren ivorans NBRC 16320 GN=rnhA PE=3 SV=1;ribonuclease H [Hirschia baltica ATCC 49814];PFAM: ribonuclease H; KEGG: bpe:BP3211 ribonuclease H;RNA/DNA hybrid binding site [nucleotide binding]
GOHSU26	CDS	GOHSU_26_00040	2440..2982		hypothetical protein [nite]			MLP_45470 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YK58_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11875 PE=4 SV=1;unnamed protein product [Sinorhizobium fredii HH103 ];Protein of unknown function (DUF1697); cl01557
GOHSU26	CDS	GOHSU_26_00050	complement(2987..4195)		conserved hypothetical protein [nite]			RER_51910 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	L-carnitine dehydratase/bile acid-inducible protein F		F1YK75_9ACTO L-carnitine dehydratase/bile acid-indu cible protein F OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_11960 PE=4 SV=1;L-carnitine dehydratase/bile acid-inducible protein F [Novosphingobium sp. PP1Y];KEGG: sal:Sala_2895;Predicted acyl-CoA transferases/carnitine dehydratase [Energy production and conversion]; COG1804;CoA-transferase family III; pfam02515
GOHSU26	CDS	GOHSU_26_00060	complement(4210..5445)		acetyl-CoA acetyltransferase [nite]			ROP_22440 :acetyl-CoA acetyltransferase : : 2.3.1.9 : Rhodococcus opacus B4	acetyl-CoA acetyltransferase		G7H670_9ACTO Acetyl-CoA acetyltransferase OS=Gordon ia araii NBRC 100433 GN=GOARA_068_00030 PE=3 SV=1;acetyl-CoA acetyltransferase [Rhodococcus opacus B4 ];Catalyzes the synthesis of acetoacetyl coenzyme A from two molecules of acetyl coenzyme A. It can also act as a thiolase, catalyzing the reverse reaction and generating two-carbon units from the four-carbon product of fatty acid oxidation;acetyl-CoA acetyltransferase; Provisional;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU26	CDS	GOHSU_26_00070	5504..6697		putative acyl-CoA dehydrogenase [nite]	 fadE6 [nite]		KSE_56470 :putative acyl-CoA dehydrogenase : fadE6 : 1.3.99.- : Kitasatospora setae NBRC 14216	acyl-CoA dehydrogenase		G7H671_9ACTO Acyl-CoA dehydrogenase OS=Gordonia ara ii NBRC 100433 GN=fadE PE=3 SV=1;acyl-CoA dehydrogenase [Mycobacterium avium 104];identified by match to protein family HMM PF00441; match to protein family HMM PF02770; match to protein family HMM PF02771; match to protein family HMM PF08028;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU26	CDS	GOHSU_26_00080	6709..7953		probable acyl-CoA dehydrogenase [nite]	 fadE [nite]		RER_52110 :probable acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus erythropolis PR4	cyclohexanecarboxyl-CoA dehydrogenase		F1YK79_9ACTO Acyl-CoA dehydrogenase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_11980 PE=3 SV=1;cyclohexanecarboxyl-CoA dehydrogenase [Acinetobacte r radioresistens SH164];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU26	CDS	GOHSU_26_00090	7950..8693		3-oxoacyl-[acyl-carrier-protein] reductase [nite]	 fabG [nite]		RER_52100 :3-oxoacyl-[acyl-carrier-protein] reductase : fabG : 1.1.1.100 : Rhodococcus erythropolis PR4	3-ketoacyl-ACP reductase	fabG	A0QUH2_MYCS2 3-oxoacyl-(Acyl-carrier-protein) reduc tase OS=Mycobacterium smegmatis (strain ATCC 700084 / mc(2 )155) GN=fabG PE=3 SV=1;fabG gene product [Mycobacterium smegmatis str. MC2 155];Catalyzes the first of the two reduction steps in the elongation cycle of fatty acid synthesis;3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05653;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU26	CDS	GOHSU_26_00100	8677..9579		hypothetical protein [nite]			ACMV_29040 :hypothetical protein : : : Acidiphilium multivorum AIU301	hypothetical protein		G7H674_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_068_00070 PE=4 SV=1;unnamed protein product [Burkholderia sp. YI23];Uncharacterized conserved protein [Function unknown];The hotdog fold was initially identified in the E. coli FabA (beta-hydroxydecanoyl-acyl carrier protein (ACP)-dehydratase) structure and subsequently in 4HBT (4-hydroxybenzoyl-CoA thioesterase) from Pseudomonas. A number of other seemingly unrelated...; cl00509;active site 2 [active];active site 1 [active]
GOHSU26	CDS	GOHSU_26_00110	complement(9604..10761)		acyl-CoA dehydrogenase [nite]	 fadE [nite]		ROP_31510 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		G7H679_9ACTO Acyl-CoA dehydrogenase OS=Gordonia ara ii NBRC 100433 GN=fadE PE=3 SV=1;acyl-CoA dehydrogenase [Mycobacterium avium 104];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;identified by match to protein family HMM PF00441; match to protein family HMM PF02770; match to protein family HMM PF02771; match to protein family HMM PF08028;Acyl-CoA dehydrogenase; cl09933
GOHSU26	CDS	GOHSU_26_00120	complement(10828..11229)		hypothetical membrane protein [nite]			ROP_19940 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YN34_9ACTO Integral membrane protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16708 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rha:RHA1_ro00326 hypothetical protein
GOHSU26	CDS	GOHSU_26_00130	11325..11798		putative MarR family transcriptional regulator [nite]			RER_45540 :putative MarR family transcriptional regulator : : : Rhodococcus erythropolis PR4	MarR family transcriptional regulator		F1YN35_9ACTO Regulatory protein MarR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_16713 PE=4 SV=1;MarR family transcriptional regulator [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Helix-turn-helix domains; cl00088;Transcriptional regulators [Transcription]; COG1846
GOHSU26	CDS	GOHSU_26_00140	11836..12213		hypothetical membrane protein [nite]			ROP_19190 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		G7H5R9_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_064_01960 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];PFAM: protein of unknown function DUF202; KEGG: cgb:cg0135 putative inner membrane protein;Domain of unknown function (DUF202); cl09954
GOHSU26	CDS	GOHSU_26_00150	complement(12398..12907)		conserved hypothetical membrane protein [nite]			RER_pREL1-02310 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	integral membrane protein		E6JCC1_9ACTO Putative uncharacterized protein OS=Di etzia cinnamea P4 GN=ES5_14443 PE=4 SV=1;integral membrane protein [Streptomyces ghanaensis ATCC 14672]
GOHSU26	CDS	GOHSU_26_00160	complement(12907..13743)		putative oxidoreductase [nite]			ANT_02360 :putative oxidoreductase : : : Anaerolinea thermophila UNI-1	short-chain dehydrogenase/reductase SDR		F1YFI0_9ACTO Short-chain dehydrogenase/reductase SD R OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02917 PE= 3 SV=1;short-chain dehydrogenase/reductase SDR [Cupriavidu s metallidurans CH34];17beta hydroxysteroid dehydrogenase-like, classical (c) SDRs; cd05374;short chain dehydrogenase; Provisional;NADP binding site [chemical binding];steroid binding site
GOHSU26	CDS	GOHSU_26_00170	complement(13872..14366)		hypothetical protein [nite]			ROP_15780 :hypothetical protein : : : Rhodococcus opacus B4	patatin		F1YFL5_9ACTO Lipid acyl hydrolase OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_02562 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];Predicted esterase of the alpha-beta hydrolase superfamily [General function prediction only]; COG1752;PFAM: Patatin; KEGG: rha:RHA1_ro01898 possible lipid acyl hydrolase;Hypothetical patatin similar to Z1214 protein of Escherichia coli; cd07209
GOHSU26	CDS	GOHSU_26_00180	14359..14490	short	putative transposase orfA for insertion sequence element [nite]			ROP_pROB01-04670 :putative transposase orfA for insertion sequence element : : : Rhodococcus opacus B4	transposase IS3/IS911 family protein		F1YPR7_9ACTO Putative transposase orfA for insertio n sequence element OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_19622 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Helix-turn-helix domains; cl00088;PFAM: transposase IS3/IS911 family protein; KEGG: ajs:Ajs_0906 transposase IS3/IS911 family protein
GOHSU26	CDS	GOHSU_26_00190	14487..15158		putative transposase orfB for insertion sequence element [nite]			ROP_pROB01-04660 :putative transposase orfB for insertion sequence element : : : Rhodococcus opacus B4	IS6110 transposase		E8WGS9_STRFA Integrase catalytic region OS=Streptom yces flavogriseus (strain ATCC 33331 / DSM 40990 / IAF-45C D) GN=Sfla_6481 PE=4 SV=1;IS6110 transposase [Mycobacterium tuberculosis H37R a];putative transposase OrfB; Reviewed;HTH-like domain; pfam13276;Integrase core domain; cl01316
GOHSU26	CDS	GOHSU_26_00200	15185..15412		putative transposase orfB for insertion sequence element [nite]			ROP_pROB01-04660 :putative transposase orfB for insertion sequence element : : : Rhodococcus opacus B4	IS629 ORF2		G7GX34_9ACTO Putative transposase OS=Gordonia araii NBRC 100433 GN=GOARA_003_00010 PE=4 SV=1;unnamed protein product [Shigella flexneri 2a str. 301]
GOHSU26	CDS	GOHSU_26_00210	complement(15722..16060)		type I restriction-modification system modification subunit [nite]	 hsdM [nite]		MLP_45890 :type I restriction-modification system modification subunit : hsdM : 2.1.1.- : Microlunatus phosphovorus NM-1	type I restriction-modification system, M subunit		E6TKG0_MYCSR Type I restriction-modification system methyltransferase subunit OS=Mycobacterium sp. (strain Sp yr1) GN=Mspyr1_37750 PE=4 SV=1;type I restriction-modification system, M subunit [ Bacillus coagulans 36D1];KEGG: bld:BLi04318 hypothetical protein; TIGRFAM: Restriction endonuclease, type I, EcoRI, M subunit; PFAM: DNA methylase, adenine-specific;HsdM N-terminal domain; pfam12161
##GOHSU27	contig00121				84503
GOHSU27	CDS	GOHSU_27_00010	complement(<1..313)	3x codon	conserved hypothetical protein [nite]			RER_07070 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase		F1YN51_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 16793 PE=3 SV=1;acyl-CoA dehydrogenase [Mycobacterium sp. Spyr1];PFAM: acyl-CoA dehydrogenase, C-terminal domain; acyl-CoA dehydrogenase, middle domain; acyl-CoA dehydrogenase, N-terminal domain;pimeloyl-CoA dehydrogenase, small subunit; TIGR03203;Acyl-CoA dehydrogenase; cl09933
GOHSU27	CDS	GOHSU_27_00020	441..1094		putative TetR family transcriptional regulator [nite]			RER_07060 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		F1YN52_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_16798 PE=4 SV=1;TetR family transcriptional regulator [Rhodococcus jostii RHA1];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU27	CDS	GOHSU_27_00030	complement(1133..2044)		putative mycolyltransferase [nite]			ROP_18610 :putative mycolyltransferase : : 2.3.1.- : Rhodococcus opacus B4	mycolyltransferase		F1YPG7_9ACTO Putative esterase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_19110 PE=4 SV=1;mycolyltransferase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Predicted esterase [General function prediction only];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU27	CDS	GOHSU_27_00040	complement(2311..3240)		hypothetical protein [migap]				hypothetical protein		F1YPH0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19125 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU27	CDS	GOHSU_27_00050	complement(3421..5961)		putative ATP-dependent Clp protease ATP-binding subunit ClpC [nite]	 clpC [nite]		RER_06460 :putative ATP-dependent Clp protease ATP-binding subunit ClpC : clpC : : Rhodococcus erythropolis PR4	ATP-dependent chaperone ClpB		H5TR41_9ACTO ATP-dependent Clp protease ATP-binding subunit ClpC OS=Gordonia otitidis NBRC 100426 GN=clpC PE= 4 SV=1;ATP-dependent chaperone ClpB [Streptomyces sp. AA4] ;Clp protease ATP binding subunit; CHL00095;Clp amino terminal domain; pfam02861;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger;UvrB/uvrC motif; pfam02151;C-terminal, D2-small domain, of ClpB protein; cl11083
GOHSU27	CDS	GOHSU_27_00060	6201..7502		hypothetical protein [nite]			ROP_13160 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLE2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13669 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU27	CDS	GOHSU_27_00070	complement(7698..8975)		probable M18 family aminopeptidase 2 [hamap]	apeB [hamap]	MF_00467:probable M18 family aminopeptidase 2:apeB 	RER_48340 :probable M18 family aminopeptidase : : 3.4.11.- : Rhodococcus erythropolis PR4	aminopeptidase 2		F1YLE5_9ACTO M18 family aminopeptidase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13684 PE=3 SV=1;aminopeptidase 2 [Streptomyces griseus subsp. grise us NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;catalyzes the removal of amino acids from the N termini of peptides;M18 Peptidase Aspartyl aminopeptidase; cd05658;Aminopeptidase I zinc metalloprotease (M18); pfam02127;metal binding site [ion binding]
GOHSU27	CDS	GOHSU_27_00080	complement(8998..9678)		phosphoribosylformylglycinamidine synthase I [hamap]	purQ [hamap]	MF_00421:phosphoribosylformylglycinamidine synthase I:purQ 	ROP_49120 :phosphoribosylformylglycinamidine synthase I : purQ : 6.3.5.3 : Rhodococcus opacus B4	phosphoribosylformylglycinamidine synthase I	purQ	F1YLE6_9ACTO Phosphoribosylformylglycinamidine synt hase 1 OS=Gordonia neofelifaecis NRRL B-59395 GN=purQ PE=3 SV=1;phosphoribosylformylglycinamidine synthase I [Coryn ebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 2-(formamido)-N1-(5-phospho-D-ribosyl)acetamidine from N2-formyl-N1-(5-phospho-D-ribosyl)glycinamide and L-glutamine in purine biosynthesis;Type 1 glutamine amidotransferase (GATase1)-like domain found in Formylglycinamide ribonucleotide amidotransferase; cd01740;putative active site [active];catalytic triad [active]
GOHSU27	CDS	GOHSU_27_00090	complement(9682..9897)		phosphoribosylformylglycinamidine synthase PurS component [nite]	 purS [nite]		ROP_49110 :phosphoribosylformylglycinamidine synthase PurS component : purS : : Rhodococcus opacus B4	phosphoribosylformylglycinamidine synthase subunit PurS	purS	D0L3G0_GORB4 Phosphoribosylformylglycinamidine synt hase, purS OS=Gordonia bronchialis (strain ATCC 25592 / DS M 43247 / JCM 3198 / NCTC 10667) GN=Gbro_0846 PE=4 SV=1;phosphoribosylformylglycinamidine synthase subunit PurS [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;With PurL and PurQ catalyzes the conversion of formylglycinamide ribonucleotide, ATP, and glutamine to formylglycinamidine ribonucleotide, ADP, and glutamate in the fourth step of the purine biosynthetic pathway;Phosphoribosylformylglycinamidine (FGAM) synthase; cl00789
GOHSU27	CDS	GOHSU_27_00100	complement(9956..10444)		putative glutathione peroxidase [nite]			ROP_49080 :putative glutathione peroxidase : : 1.11.1.9 : Rhodococcus opacus B4	putative glutathione peroxidase		F1YLE8_9ACTO Glutathione peroxidase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_13699 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Glutathione (GSH) peroxidase family; tetrameric selenoenzymes that catalyze the reduction of a variety of hydroperoxides including lipid peroxidases, using GSH as a specific electron donor substrate. GSH peroxidase contains one selenocysteine residue per...; cd00340;catalytic residues [active];dimer interface [polypeptide binding]
GOHSU27	CDS	GOHSU_27_00110	10528..10995		putative MarR family transcriptional regulator [nite]			ROP_16310 :putative MarR family transcriptional regulator : : : Rhodococcus opacus B4	MarR family transcriptional regulator		F1YLE9_9ACTO Regulatory protein MarR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_13704 PE=4 SV=1;MarR family transcriptional regulator [Nakamurella multipartita DSM 44233];PFAM: regulatory protein MarR; SMART: regulatory protein MarR; KEGG: sbp:Sbal223_0569 MarR family transcriptional regulator;helix_turn_helix multiple antibiotic resistance protein; smart00347;Helix-turn-helix domains; cl00088
GOHSU27	CDS	GOHSU_27_00120	11072..12631		putative drug resistance efflux protein [nite]			RER_17440 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	EmrB/QacA family drug resistance transporter		F1YLF0_9ACTO EmrB/QacA subfamily drug resistance tr ansporter OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1 3709 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];TIGRFAM: drug resistance transporter, EmrB/QacA subfamily; PFAM: major facilitator superfamily MFS_1; KEGG: fal:FRAAL1973 putative multidrug export protein;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;Major Facilitator Superfamily; pfam07690;putative substrate translocation pore
GOHSU27	CDS	GOHSU_27_00130	complement(12642..13427)		putative hydrolase [nite]			RER_06650 :putative hydrolase : : : Rhodococcus erythropolis PR4	hydrolase		F1YIW1_9ACTO Alpha/beta hydrolase fold protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_09121 PE=4 SV=1 ;hydrolase [Streptomyces griseus subsp. griseus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697
GOHSU27	CDS	GOHSU_27_00140	13608..13913		hypothetical protein [nite]			ROP_43520 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YIW2_9ACTO Antibiotic biosynthesis monooxygenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09126 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Antibiotic biosynthesis monooxygenase; cl10022;Evidence 4 : Homologs of previously reported genes of unknown function
GOHSU27	CDS	GOHSU_27_00150	13983..15443		aldehyde dehydrogenase [nite]			ROP_52320 :aldehyde dehydrogenase : : 1.2.1.- : Rhodococcus opacus B4	aldehyde dehydrogenase		F1YIW3_9ACTO Betaine-aldehyde dehydrogenase OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_09131 PE=3 SV=1;aldehyde dehydrogenase [Frankia sp. EuI1c];PFAM: aldehyde dehydrogenase; KEGG: mjl:Mjls_3731 aldehyde dehydrogenase;Mycobacterium tuberculosis aldehyde dehydrogenase AldA-like; cd07139;Aldehyde dehydrogenase family; pfam00171;NAD binding site [chemical binding];catalytic residues [active]
GOHSU27	CDS	GOHSU_27_00160	complement(15499..16128)		hypothetical membrane protein [nite]			ROP_pROB01-03910 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YIW4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09136 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.970) with cleavage site probability 0.925 at residue 32;KEGG: hypothetical protein
GOHSU27	CDS	GOHSU_27_00170	complement(16125..17090)		hypothetical protein [migap]				hypothetical protein		F1YIW5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09141 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU27	CDS	GOHSU_27_00180	complement(17054..18475)		putative Mce family protein [nite]	 mce3F [nite]		RER_09650 :putative Mce family protein : mce3F : : Rhodococcus erythropolis PR4	virulence factor Mce family protein;hypothetical protein		F1YIW6_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09146 PE=4 SV =1;virulence factor Mce family protein [Mycobacterium gilvum PYR- GCK];Signal predicted by SignalP 2.0 HMM (Signal peptide probability0.720) with cleavage site probability 0.315 at residue 42;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_3666 mammalian cell entry;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU27	CDS	GOHSU_27_00190	complement(18475..19791)		putative Mce family protein [nite]	 mce3E [nite]		RER_09640 :putative Mce family protein : mce3E : : Rhodococcus erythropolis PR4	virulence factor Mce family protein;hypothetical protein		F1YIW7_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09151 PE=4 SV =1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.996) with cleavage site probability 0.361 at residue 31;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_1658 mammalian cell entry;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU27	CDS	GOHSU_27_00200	complement(19791..21092)		putative Mce family protein [nite]	 mce3D [nite]		RER_09630 :putative Mce family protein : mce3D : : Rhodococcus erythropolis PR4	virulence factor Mce family protein		F1YIW8_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09156 PE=4 SV =1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: mce related protein; TIGRFAM: virulence factor Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU27	CDS	GOHSU_27_00210	complement(21094..22341)		putative Mce family protein [nite]	 mce5C [nite]		ROP_47990 :putative Mce family protein : mce5C : : Rhodococcus opacus B4	virulence factor Mce family protein		F1YIW9_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09161 PE=4 SV =1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_4615 mammalian cell entry;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU27	CDS	GOHSU_27_00220	complement(22338..23432)		putative Mce family protein [nite]	 mce5B [nite]		ROP_47980 :putative Mce family protein : mce5B : : Rhodococcus opacus B4	virulence factor Mce family protein;hypothetical protein		F1YIX0_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09166 PE=4 SV =1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];Signal predicted by SignalP 2.0 HMM (Signal peptide probabilty 0.950) with cleavage site probability 0.779 at residue 33;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_1655 mammalian cell entry;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU27	CDS	GOHSU_27_00230	complement(23425..24594)		putative Mce family protein [nite]	 mce5A [nite]		ROP_47970 :putative Mce family protein : mce5A : : Rhodococcus opacus B4	virulence factor Mce family protein		F1YIX1_9ACTO Virulence factor Mce family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09171 PE=4 SV =1;virulence factor Mce family protein [Mycobacterium vanbaalenii PYR- 1];TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_1524 mammalian cell entry;ABC-type transport system involved in resistance to organic solvents, periplasmic component [Secondary metabolites biosynthesis, transport, and catabolism]; COG1463;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU27	CDS	GOHSU_27_00240	complement(24599..25450)		putative YrbE family protein [nite]	 yrbE5B [nite]		ROP_47960 :putative YrbE family protein : yrbE5B : : Rhodococcus opacus B4	organic solvents resistance ABC transporter permease		H0QV79_9ACTO YrbE family protein OS=Gordonia effusa NBRC 100432 GN=yrbEB PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF140; TIGRFAM: conserved hypothetical integral membrane protein;Permease; cl00510
GOHSU27	CDS	GOHSU_27_00250	complement(25455..26195)		putative YrbE family protein [nite]	 yrbE5A [nite]		ROP_47950 :putative YrbE family protein : yrbE5A : : Rhodococcus opacus B4	organic solvents resistance ABC transporter permease		G7GKG1_9ACTO YrbE family protein OS=Gordonia amarae NBRC 15530 GN=yrbEA PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF140; TIGRFAM: conserved hypothetical integral membrane protein;Permease; cl00510
GOHSU27	CDS	GOHSU_27_00260	complement(26300..27184)		putative oxidoreductase [nite]			RER_09570 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		H5TQ36_9ACTO Putative oxidoreductase OS=Gordonia ot itidis NBRC 100426 GN=GOOTI_170_00610 PE=3 SV=1;short chain dehydrogenase [Rhodococcus equi 103S];3-ketoacyl-(acyl-carrier-protein) reductase; Provisional; PRK07792;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU27	CDS	GOHSU_27_00270	complement(27278..27466)		3Fe-4S ferredoxin [nite]			ROP_47930 :3Fe-4S ferredoxin : : : Rhodococcus opacus B4	ferredoxin		G7H1K1_9ACTO 3Fe-4S ferredoxin OS=Gordonia araii NB RC 100433 GN=GOARA_045_00800 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Protein of unknown function (DUF1271); cl01443;Ferredoxin [Energy production and conversion]; COG1141;PFAM: Protein of unknown function (DUF1271)
GOHSU27	CDS	GOHSU_27_00280	27562..28731		putative acyl-CoA dehydrogenase [nite]			ROP_47920 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		F1YIX6_9ACTO Acyl-CoA dehydrogenase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_09196 PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus equi 103S];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Putative acyl-CoA dehydrogenases similar to fadE6, fadE17, and fadE26; cd01152;FAD binding site [chemical binding];substrate binding site [chemical binding];catalytic base [active]
GOHSU27	CDS	GOHSU_27_00290	28781..29884		putative acyl-CoA dehydrogenase [nite]			RER_09540 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase		F1YIX7_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 09201 PE=4 SV=1;acyl-CoA dehydrogenase [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase, N-terminal domain; pfam02771;Acyl-CoA dehydrogenase; cd00567
GOHSU27	CDS	GOHSU_27_00300	complement(29907..30398)		hypothetical protein [nite]			KSE_04280 :hypothetical protein : : : Kitasatospora setae NBRC 14216	metal dependent phosphohydrolase		G6XBH1_MYCAB Uncharacterized protein OS=Mycobacteri um abscessus 47J26 GN=MAB47J26_18437 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];PFAM: metal-dependent phosphohydrolase, HD sub domain; KEGG: sma:SAV5438 hypothetical protein;Metal dependent phosphohydrolases with conserved 'HD' motif; cl00076
GOHSU27	CDS	GOHSU_27_00310	complement(30488..31672)		putative oxidoreductase [nite]			ROP_47890 :putative oxidoreductase : : : Rhodococcus opacus B4	2-nitropropane dioxygenase		D0L3X9_GORB4 2-nitropropane dioxygenase NPD OS=Gord onia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3958 PE=4 SV=1;2-nitropropane dioxygenase [Mycobacterium gilvum PY R- GCK];PFAM: 2-nitropropane dioxygenase, NPD; KEGG: mmc:Mmcs_4626 2-nitropropane dioxygenase, NPD;Nitronate monooxygenase; pfam03060;2-Nitropropane dioxygenase (NPD), one of the nitroalkane oxidizing enzyme families, catalyzes oxidative denitrification of nitroalkanes to their corresponding carbonyl compounds and nitrites. NDP is a member of the NAD(P)H-dependent flavin oxidoreductase...; cd04730;FMN binding site [chemical binding];substrate binding site [chemical binding];putative catalytic residue [active]
GOHSU27	CDS	GOHSU_27_00320	31724..32554		putative enoyl-CoA hydratase [nite]			RER_09350 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase/isomerase		F1YIY7_9ACTO Enoyl-CoA hydratase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_09251 PE=4 SV=1;enoyl-CoA hydratase/isomerase [Frankia sp. EuI1c];enoyl-CoA hydratase; Provisional;PFAM: enoyl-CoA hydratase/isomerase; KEGG: rer:RER_09350 enoyl-CoA hydratase;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU27	CDS	GOHSU_27_00330	complement(32541..34643)		Luciferase-like monooxygenase superfamily protein [migap]				Luciferase-like monooxygenase superfamily protein		H5UHI1_9ACTO Putative F420-dependent oxidoreductase OS=Gordonia terrae NBRC 100016 GN=GOTRE_109_00580 PE=4 SV =1;Luciferase-like monooxygenase superfamily protein [ marine gamma proteobacterium HTCC2148];[C] COG2141 Coenzyme F420-dependent N5,N10-methylene tetrahydromethanopterin reductase and related flavin-dependent oxidoreductases;probable F420-dependent oxidoreductase, Rv3520c family; TIGR03559;N5,N10-methylenetetrahydromethanopterin reductase (Mer) catalyzes the reduction of N5,N10-methylenetetrahydromethanopterin with reduced coenzyme F420 to N5-methyltetrahydromethanopterin and oxidized coenzyme F420; cd01097
GOHSU27	CDS	GOHSU_27_00340	34701..35717		hypothetical protein [nite]			ROP_47840 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5UHI2_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_109_00590 PE=4 SV=1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];PFAM: protein of unknown function DUF35; KEGG: afu:AF0965 hypothetical protein;Predicted nucleic-acid-binding protein containing a Zn-ribbon [General function prediction only];Rubredoxin-like zinc ribbon domain (DUF35_N); pfam12172;DUF35 OB-fold domain; pfam01796
GOHSU27	CDS	GOHSU_27_00350	35733..36791		conserved hypothetical protein [nite]			RER_09310 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	lipid-transfer protein		D0L3Y6_GORB4 Acetyl-CoA acetyltransferase-like prot ein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3965 PE=4 SV=1;lipid-transfer protein [Mycobacterium abscessus ATC C 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;lipid-transfer protein; Provisional;Similar to Q25W15_MYCVN e-157;Thiolase domain associated with sterol carrier protein (SCP)-x isoform and related proteins; SCP-2  has multiple roles in intracellular lipid circulation and metabolism. The N-terminal presequence in the SCP-x isoform represents a peroxisomal...; cd00829
GOHSU27	CDS	GOHSU_27_00360	36791..37960		hypothetical protein [nite]			ROP_47820 :hypothetical protein : : : Rhodococcus opacus B4	propanoyl-CoA C-acyltransferase		F9W180_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_111_00080 PE=4 SV= 1;unnamed protein product [Desulfatibacillum alkenivo rans AK- 01];acetyl-CoA acetyltransferase; Provisional;KEGG: reh:H16_A0868 acetyl-CoA acetyltransferase;Thiolase domain associated with sterol carrier protein (SCP)-x isoform and related proteins; SCP-2  has multiple roles in intracellular lipid circulation and metabolism. The N-terminal presequence in the SCP-x isoform represents a peroxisomal...; cd00829
GOHSU27	CDS	GOHSU_27_00370	38040..38411		hypothetical protein [nite]			ROP_38670 :hypothetical protein : : : Rhodococcus opacus B4	protein synthesis inhibitor		F1YK62_9ACTO Endoribonuclease OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_11895 PE=4 SV=1;protein synthesis inhibitor [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;YjgF, YER057c, and UK114 belong to a large family of proteins present in bacteria, archaea, and eukaryotes with no definitive function. The conserved domain is similar in structure to chorismate mutase but there is no sequence similarity and no...; cd00448;homotrimer interaction site [polypeptide binding];putative active site [active]
GOHSU27	CDS	GOHSU_27_00380	38562..38987		hypothetical membrane protein [nite]			ROP_57200 :hypothetical membrane protein : : : Rhodococcus opacus B4	DoxX family protein		F1YIZ7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09301 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: DoxX family protein; KEGG: rop:ROP_57280 hypothetical membrane protein;DoxX; cl00976
GOHSU27	CDS	GOHSU_27_00390	complement(39085..39552)		hypothetical protein [nite]			RER_09280 :hypothetical protein : : : Rhodococcus erythropolis PR4	ketosteroid isomerase-like protein		G7H2J7_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_051_00160 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109
GOHSU27	CDS	GOHSU_27_00400	complement(39539..40819)		cytochrome P450 [nite]			RER_09260 :cytochrome P450 : : : Rhodococcus erythropolis PR4	cytochrome P450	cyp125A7	G7H2J6_9ACTO Cytochrome P450 OS=Gordonia araii NBRC 100433 GN=GOARA_051_00150 PE=4 SV=1;cytochrome P450 [Mycobacterium sp. JDM601];Cytochrome P450 [Secondary metabolites biosynthesis, transport, and catabolism]; cl12078
GOHSU27	CDS	GOHSU_27_00410	40967..42130		putative acetyl-CoA acyltransferase [nite]			ROP_47770 :putative acetyl-CoA acyltransferase : : 2.3.1.- : Rhodococcus opacus B4	acetyl-CoA acetyltransferase		F1YJ00_9ACTO Acetyl-CoA acetyltransferase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_09316 PE=3 SV=1;acetyl-CoA acyltransferase [Rhodococcus erythropoli s PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;acetyl-CoA acetyltransferase; Provisional;Catalyzes the synthesis of acetoacetyl coenzyme A from two molecules of acetyl coenzyme A. It can also act as a thiolase, catalyzing the reverse reaction and generating two-carbon units from the four-carbon product of fatty acid oxidation;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU27	CDS	GOHSU_27_00420	42279..42743		hypothetical protein [nite]			YM304_03950 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	hypothetical protein		E3IW89_FRASU Putative uncharacterized protein OS=Fr ankia sp. (strain EuI1c) GN=FraEuI1c_0853 PE=4 SV=1;unnamed protein product [Pedobacter saltans DSM 121 45];KEGG: fjo:Fjoh_4054 hypothetical protein; SPTR:  uncharacterized protein;superfamily containing globins and truncated hemoglobins; cl00280;heme-binding site [chemical binding]
GOHSU27	CDS	GOHSU_27_00430	complement(42780..43247)		hypothetical protein [nite]			YM304_41450 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	hypothetical protein		B2HKV8_MYCMM Conserved protein OS=Mycobacterium mar inum (strain ATCC BAA-535 / M) GN=MMAR_1930 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];Also detected in the membrane fraction by proteomics.; cytoplasmic protein;START/RHO_alpha_C/PITP/Bet_v1/CoxG/CalC (SRPBCC) ligand-binding domain superfamily; cd07812;hydrophobic ligand binding site
GOHSU27	CDS	GOHSU_27_00440	complement(43257..44159)		hypothetical protein [nite]			KRH_20870 :hypothetical protein : : : Kocuria rhizophila DC2201	fructosamine kinase		G7H411_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_061_00250 PE=4 SV=1;fructosamine kinase [Chitinophaga pinensis DSM 2588 ];type strain of Chitinophaga pinensis;PFAM: fructosamine kinase; aminoglycoside phosphotransferase; KEGG: tcx:Tcr_2057 fructosamine kinase;Phosphotransferase enzyme family; pfam01636;Aminoglycoside 3'-phosphotransferase (APH) and Choline Kinase (ChoK) family. The APH/ChoK family is part of a larger superfamily that includes the catalytic domains of other kinases, such as the typical serine/threonine/tyrosine protein kinases (PKs); cd05120;ATP binding site [chemical binding];Protein Kinases, catalytic domain; cl09925;substrate binding site [chemical binding]
GOHSU27	CDS	GOHSU_27_00450	complement(44221..45849)		putative fatty-acid--CoA ligase [nite]			ROP_55830 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	acyl-CoA synthetase		H0QV50_9ACTO Putative fatty-acid--CoA ligase OS=Gor donia effusa NBRC 100432 GN=GOEFS_012_00020 PE=4 SV=1;fatty-acid--CoA ligase [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;activates fatty acids by binding to coenzyme A;acyl-CoA synthetase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU27	CDS	GOHSU_27_00460	45963..46622		conserved hypothetical protein [nite]			RER_09220 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJ03_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09331 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];DNA Topoisomerase, subtype IIA; domain A'; bacterial DNA topoisomerase IV (C subunit, ParC), bacterial DNA gyrases (A subunit, GyrA),mammalian DNA toposiomerases II. DNA topoisomerases are essential enzymes that regulate the conformational changes in DNA...; cl00148;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU27	CDS	GOHSU_27_00470	complement(46663..47028)		hypothetical protein [nite]			RER_04120 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F9W194_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_111_00220 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU27	CDS	GOHSU_27_00480	complement(47015..47911)		putative oxidoreductase [nite]			RER_09210 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		H6MUW3_GORPV Putative oxidoreductase, short-chain d ehydrogenase/reductase family OS=Gordonia polyisoprenivora ns (strain DSM 44266 / VH2) GN=GPOL_c43010 PE=3 SV=1;short chain dehydrogenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q743S3_MYCPA e-119;(3R)-hydroxyacyl-CoA dehydrogenase-like, classical(c)-like SDRs; cd05353;3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05653;homodimer interface [polypeptide binding];NAD binding site [chemical binding]
GOHSU27	CDS	GOHSU_27_00490	complement(47908..48735)		putative oxidoreductase [nite]			ROP_47510 :putative oxidoreductase : : : Rhodococcus opacus B4	short chain dehydrogenase		F1YJ08_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09356 PE=4 SV=1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU27	CDS	GOHSU_27_00500	48795..49544		putative enoyl-CoA hydratase [nite]			ROP_47500 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus opacus B4	enoyl-CoA hydratase		F9W197_9ACTO Putative enoyl-CoA hydratase OS=Gordon ia alkanivorans NBRC 16433 GN=GOALK_111_00250 PE=4 SV=1;enoyl-CoA hydratase [Rhodococcus jostii RHA1];Catalyzes the reversible hydration of unsaturated fatty acyl-CoA to beta-hydroxyacyl-CoA;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding];Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cl15762
GOHSU27	CDS	GOHSU_27_00510	49546..50451		putative CoA-transferase alpha subunit [nite]			RER_09190 :putative CoA-transferase alpha subunit : : 2.8.3.- : Rhodococcus erythropolis PR4	CoA-transferase alpha subunit		G7H2I5_9ACTO Putative CoA-transferase alpha subunit OS=Gordonia araii NBRC 100433 GN=GOARA_051_00040 PE=4 SV= 1;CoA-transferase alpha subunit [Rhodococcus equi 103 S];SugarP_isomerase: Sugar Phosphate Isomerase family; includes type A ribose 5-phosphate isomerase (RPI_A), glucosamine-6-phosphate (GlcN6P) deaminase, and 6-phosphogluconolactonase (6PGL). RPI catalyzes the reversible conversion of ribose-5-phosphate to...; cl00339
GOHSU27	CDS	GOHSU_27_00520	50448..51230		putative CoA-transferase beta subunit [nite]			ROP_47480 :putative CoA-transferase beta subunit : : 2.8.3.- : Rhodococcus opacus B4	CoA-transferase beta subunit		F1YJ11_9ACTO Putative CoA transferase beta subunit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_09371 PE=4 SV=1;CoA-transferase beta subunit [Rhodococcus equi 103S ];SugarP_isomerase: Sugar Phosphate Isomerase family; includes type A ribose 5-phosphate isomerase (RPI_A), glucosamine-6-phosphate (GlcN6P) deaminase, and 6-phosphogluconolactonase (6PGL). RPI catalyzes the reversible conversion of ribose-5-phosphate to...; cl00339
GOHSU27	CDS	GOHSU_27_00530	51227..52348		putative oxidoreductase [nite]			RER_09170 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	2-nitropropane dioxygenase		F1YJ12_9ACTO 2-nitropropane dioxygenase NPD OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_09376 PE=4 SV=1;2-nitropropane dioxygenase [Rhodococcus equi 103S];Nitronate monooxygenase; pfam03060;2-Nitropropane dioxygenase (NPD), one of the nitroalkane oxidizing enzyme families, catalyzes oxidative denitrification of nitroalkanes to their corresponding carbonyl compounds and nitrites. NDP is a member of the NAD(P)H-dependent flavin oxidoreductase...; cd04730;FMN binding site [chemical binding];substrate binding site [chemical binding];putative catalytic residue [active]
GOHSU27	CDS	GOHSU_27_00540	complement(52292..52789)		not assigned						
GOHSU27	CDS	GOHSU_27_00550	52879..54039		conserved hypothetical protein [nite]			RER_42710 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	16S RNA G1207 methylase RsmC		G7GZZ9_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_030_00140 PE=4 SV=1;16S RNA G1207 methylase RsmC [Microbacterium testac eum StLB037];16S RNA G1207 methylase RsmC [Translation, ribosomal structure and biogenesis]; COG2813;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU27	CDS	GOHSU_27_00560	complement(54020..55579)		putative fatty-acid--CoA ligase [nite]			RER_25580 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus erythropolis PR4	AMP-dependent synthetase and ligase		D0LAA6_GORB4 AMP-dependent synthetase and ligase OS =Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_0083 PE=4 SV=1;AMP-dependent synthetase and ligase [Frankia sp. Eu I1c];PFAM: AMP-dependent synthetase and ligase; KEGG: gbr:Gbro_0083 AMP-dependent synthetase and ligase;Acyl-coenzyme A synthetases/AMP-(fatty) acid ligases [Lipid metabolism]; COG0365;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU27	CDS	GOHSU_27_00570	complement(55663..56217)		putative TetR family transcriptional regulator [nite]			ROP_45430 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	tetr family transcriptional regulator		G7GZZ8_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_030_00130 PE=4 SV=1;tetr family transcriptional regulator [Rhodococcus equi 103S];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU27	CDS	GOHSU_27_00580	complement(56523..58028)		aldehyde dehydrogenase [nite]			RER_52140 :aldehyde dehydrogenase : : 1.2.1.- : Rhodococcus erythropolis PR4	aldehyde dehydrogenase (NAD) family protein		F1YLL4_9ACTO Aldehyde dehydrogenase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_14029 PE=3 SV=1;aldehyde dehydrogenase (NAD) family protein [marine gamma proteobacterium HTCC2148];[C] COG1012 NAD-dependent aldehyde dehydrogenases;Coniferyl aldehyde dehydrogenase-like; cd07133;NAD(P) binding site [chemical binding];catalytic residues [active]
GOHSU27	CDS	GOHSU_27_00590	complement(58168..59325)		putative acetyl-CoA acyltransferase [nite]			ROP_45440 :putative acetyl-CoA acyltransferase : : 2.3.1.- : Rhodococcus opacus B4	acetyl-CoA c-acyltransferase		H0QZ99_9ACTO Putative acetyl-CoA acyltransferase OS =Gordonia effusa NBRC 100432 GN=GOEFS_047_00090 PE=3 SV=1;acetyl-CoA c-acyltransferase [Rhodococcus equi 103S ];acetyl-CoA acetyltransferase; Provisional;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU27	CDS	GOHSU_27_00600	complement(59384..60196)		putative oxidoreductase [nite]			ROP_45420 :putative oxidoreductase : : : Rhodococcus opacus B4	short chain dehydrogenase		H0QZA0_9ACTO Putative oxidoreductase OS=Gordonia ef fusa NBRC 100432 GN=GOEFS_047_00100 PE=3 SV=1;short chain dehydrogenase [Mycobacterium avium 104] ;identified by match to protein family HMM PF00106;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU27	CDS	GOHSU_27_00610	complement(60193..61473)		putative acyl-CoA dehydrogenase [nite]			RER_08910 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase fadE30		F1YLL7_9ACTO Acyl-CoA dehydrogenase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_14044 PE=3 SV=1;unnamed protein product [Mycobacterium tuberculosis KZN 4207];drug-susceptible;pimeloyl-CoA dehydrogenase, large subunit; TIGR03204;Acyl-CoA dehydrogenase; cl09933
GOHSU27	CDS	GOHSU_27_00620	61516..63069		putative fatty-acid--CoA ligase [nite]			ROP_45400 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	acyl-CoA synthetase		F1YLL8_9ACTO Long-chain-fatty-acid--CoA ligase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_14049 PE=4 SV=1 ;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: AMP-binding enzyme;long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU27	CDS	GOHSU_27_00630	63136..63606		hypothetical membrane protein [nite]			ROP_03160 :hypothetical membrane protein : : : Rhodococcus opacus B4	integral membrane protein		H5TZY7_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_056_00280 PE=4 SV=1;integral membrane protein [Streptomyces sp. AA4];DoxX; cl00976
GOHSU27	CDS	GOHSU_27_00640	complement(63587..64474)		3-hydroxyisobutyrate dehydrogenase [nite]	 mmsB [nite]		RER_39520 :3-hydroxyisobutyrate dehydrogenase : mmsB : 1.1.1.31 : Rhodococcus erythropolis PR4	3-hydroxyisobutyrate dehydrogenase		I0S0I3_MYCPH 3-hydroxyisobutyrate dehydrogenase OS= Mycobacterium phlei RIVM601174 GN=MPHLEI_05192 PE=4 SV=1;3-hydroxyisobutyrate dehydrogenase [Brucella pinnip edialis B2/94];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;3-hydroxyisobutyrate dehydrogenase; TIGR01692
GOHSU27	CDS	GOHSU_27_00650	complement(64476..65642)		acyl-CoA dehydrogenase [nite]	 fadE [nite]		ROP_44980 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase	fadE	H5UIR0_9ACTO Acyl-CoA dehydrogenase OS=Gordonia ter rae NBRC 100016 GN=fadE PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus opacus B4];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Isobutyryl-CoA dehydrogenase; cd01162;substrate binding pocket [chemical binding];FAD binding site [chemical binding];catalytic base [active]
GOHSU27	CDS	GOHSU_27_00660	complement(65647..67179)		methylmalonate-semialdehyde dehydrogenase [nite]	 mmsA [nite]		KRH_16540 :methylmalonate-semialdehyde dehydrogenase : mmsA : 1.2.1.27 : Kocuria rhizophila DC2201	methylmalonate-semialdehyde dehydrogenase	mmsA	H6N3T8_GORPV Methylmalonate-semialdehyde dehydrogen ase MmsA OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=mmsA3 PE=3 SV=1;methylmalonate-semialdehyde dehydrogenase [Mycobact erium avium 104];identified by match to protein family HMM PF00171; match to protein family HMM TIGR01722;Methylmalonate semialdehyde dehydrogenase and ALDH family members 6A1 and 6B2; cd07085;Aldehyde dehydrogenase family; pfam00171;tetrameric interface [polypeptide binding];NAD binding site [chemical binding];catalytic residues [active]
GOHSU27	CDS	GOHSU_27_00670	67286..68236		putative LysR family transcriptional regulator [nite]			ROP_45000 :putative LysR family transcriptional regulator : : : Rhodococcus opacus B4	LysR family transcriptional regulator		Q0S0U7_RHOSR Probable transcriptional regulator, Ly sR family protein OS=Rhodococcus sp. (strain RHA1) GN=RHA1 _ro07075 PE=4 SV=1;LysR family transcriptional regulator [Rhodococcus opacus B4];Transcriptional regulator [Transcription]; COG0583;Helix-turn-helix domains; cl00088;The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cl11398;dimerization interface [polypeptide binding]
GOHSU27	CDS	GOHSU_27_00680	complement(68217..69749)		lysyl-tRNA synthetase [hamap]	lysS [hamap]	MF_00252:lysyl-tRNA synthetase:lysS 	ROP_43360 :lysyl-tRNA synthetase : lysS : 6.1.1.6 : Rhodococcus opacus B4	lysyl-tRNA synthetase	lysS	F1YLM5_9ACTO Lysine--tRNA ligase 1 OS=Gordonia neof elifaecis NRRL B-59395 GN=lysS PE=3 SV=1;lysyl-tRNA synthetase [Corynebacterium glutamicum A TCC 13032];IS fingerprint type 4-5;class II; LysRS2; catalyzes a two-step reaction, first charging a lysine molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA; in Methanosarcina barkeri, LysRS2 charges both tRNA molecules for lysine that exist in this organism and in addition can charge the tRNAPyl with lysine in the presence of LysRS1;lysyl-tRNA synthetase; Reviewed; PRK00484;LysRS_N: N-terminal, anticodon recognition domain of lysyl-tRNA synthetases (LysRS). These enzymes are homodimeric class 2b aminoacyl-tRNA synthetases (aaRSs). This domain is a beta-barrel domain (OB fold) involved in binding the tRNA anticodon stem-loop; cd04322;dimer interface [polypeptide binding];putative anticodon binding site;Lys_tRNA synthetase (LysRS) class II core domain. Class II LysRS is a dimer which attaches a lysine to the 3' OH group of ribose of the appropriate tRNA. Its assignment to class II aaRS is based upon its structure and the presence of three...; cd00775;motif 1;motif 2;motif 3
GOHSU27	CDS	GOHSU_27_00690	complement(69789..70253)		hypothetical protein [migap]				hypothetical protein		F1YLM6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14089 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU27	CDS	GOHSU_27_00700	70633..71244		hypothetical protein [nite]			RER_37430 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		C8NRF3_COREF Putative uncharacterized protein OS=Co rynebacterium efficiens (strain DSM 44549 / YS-314 / AJ 12 310 / JCM 11189 / NBRC 100395) GN=HMPREF0290_2578 PE=4 SV= 1;unnamed protein product [Corynebacterium diphtheria e 31A]
GOHSU27	CDS	GOHSU_27_00710	71307..73025		hypothetical protein [nite]			ROP_62480 :hypothetical protein : : : Rhodococcus opacus B4	metallophosphoesterase		F1YLM8_9ACTO Metallophosphoesterase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_14099 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: metallophosphoesterase; KEGG: rha:RHA1_ro06188 hypothetical protein;metallophosphatase superfamily, metallophosphatase domain; cl13995
GOHSU27	CDS	GOHSU_27_00720	complement(73030..73458)		conserved hypothetical protein [nite]			RER_17910 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	Acyl dehydratase		F1YLN3_9ACTO MaoC dehydratase OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_14124 PE=4 SV=1;Acyl dehydratase [Magnetospirillum magneticum AMB- 1];YdeM is a Bacillus subtilis protein that belongs to a family of prokaryotic proteins of unkown function.  YdeM has sequence similarity to the hot-dog fold of (R)-specific enoyl-CoA hydratase.   Other enzymes with this fold include the peroxisomal...; cd03454;putative active site [active];putative catalytic site [active]
GOHSU27	CDS	GOHSU_27_00730	complement(73455..74252)		type III pantothenate kinase [hamap]	coaX [hamap]	MF_01274:type III pantothenate kinase:coaX 	ROP_43330 :hypothetical protein : : : Rhodococcus opacus B4	transcriptional activator, Baf family		F1YLN4_9ACTO Type III pantothenate kinase OS=Gordon ia neofelifaecis NRRL B-59395 GN=coaX PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Type III pantothenate kinase; cl09130;TIGRFAM: transcriptional activator, Baf family; PFAM: Bvg accessory factor; KEGG: ank:AnaeK_1477 pantothenate kinase
GOHSU27	CDS	GOHSU_27_00740	complement(74264..74689)		aspartate 1-decarboxylase [hamap]	panD [hamap]	MF_00446:aspartate 1-decarboxylase:panD 	ROP_43320 :aspartate 1-decarboxylase : panD : 4.1.1.11 : Rhodococcus opacus B4	aspartate 1-decarboxylase	panD	F1YLN5_9ACTO Aspartate 1-decarboxylase OS=Gordonia neofelifaecis NRRL B-59395 GN=panD PE=3 SV=1;aspartate 1-decarboxylase [Amycolatopsis mediterran ei U32];Aspartate alpha-decarboxylase or L-aspartate 1-decarboxylase, a pyruvoyl group-dependent  decarboxylase in beta-alanine production; cd06919;tetramerization interface [polypeptide binding]
GOHSU27	CDS	GOHSU_27_00750	complement(74693..75658)		pantothenate synthetase [hamap]	panC [hamap]	MF_00158:pantothenate synthetase:panC 	ROP_43310 :pantothenate synthetase : panC : 6.3.2.1 : Rhodococcus opacus B4	pantothenate synthetase PanC	panC	F1YLN6_9ACTO Pantothenate synthetase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=panC PE=3 SV=1;panC gene product [Gordonia polyisoprenivorans VH2] ;Pantoate-beta-alanine ligase; cd00560;ATP-binding site [chemical binding];pantoate-binding site;HXXH motif
GOHSU27	CDS	GOHSU_27_00760	complement(75659..76567)		hypothetical protein [nite]			ROP_43300 :hypothetical protein : : : Rhodococcus opacus B4	putative oxidoreductase		F1YLN7_9ACTO Putative oxidoreductase/dehydrogenase, Rossmann-like domain-containing protein OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_14144 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Domain of unknown function (DUF2520); pfam10728
GOHSU27	CDS	GOHSU_27_00770	complement(76684..80436)		putative WD-40 repeat protein [nite]			RER_pREL1-02130 :putative WD-40 repeat protein : : : Rhodococcus erythropolis PR4	putative WD-40 repeat protein		D5G201_PODAS HET-E (Fragment) OS=Podospora anserina GN=het-e PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG2319 FOG: WD40 repeat;FOG: WD40 repeat [General function prediction only];WD40 domain, found in a number of eukaryotic proteins that cover a wide variety of functions including adaptor/regulatory modules in signal transduction, pre-mRNA processing and cytoskeleton assembly; typically contains a GH dipeptide 11-24 residues from...; cd00200;structural tetrad
GOHSU27	CDS	GOHSU_27_00780	complement(80674..81228)		flavin-containing amine oxidase [migap]	mao [migap]			flavin-containing amine oxidase	mao	D7GG76_PROFC Flavin-containing amine oxidase OS=Pro pionibacterium freudenreichii subsp. shermanii (strain ATC C 9614 / CIP 103027 / CIRM-BIA1) GN=mao PE=4 SV=1;flavin-containing amine oxidase [Propionibacterium freudenreichii subsp. shermanii CIRM- BIA1];Monoamine oxidase [Amino acid transport and metabolism];RCH2NH2 + H2O + O2 = RCHO + NH3 + H2O2
GOHSU27	CDS	GOHSU_27_00790	complement(81219..81596)		not assigned						
GOHSU27	CDS	GOHSU_27_00800	complement(81662..82336)		putative porin MspA [nite]	 mspA [nite]		ROP_04220 :putative porin MspA : mspA : : Rhodococcus opacus B4	hypothetical protein		F1YL08_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13563 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rer:RER_02300 porin MspA;MspA; pfam09203
GOHSU27	CDS	GOHSU_27_00810	complement(82579..83919)		hypothetical protein [migap]				hypothetical protein		F1YLN8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14149 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU27	CDS	GOHSU_27_00820	complement(83990..>84503)	3x codon	hypothetical membrane protein [nite]			ROP_43280 :hypothetical membrane protein : : : Rhodococcus opacus B4	putative secreted protein		F9VQG0_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_015_00190 PE=4 SV= 1;unnamed protein product [Corynebacterium diphtheria e HC04];Protein of unknown function (DUF3180); pfam11377
##GOHSU28	contig00131				72629
GOHSU28	CDS	GOHSU_28_00010	<1..1850	3x codon frame	putative LytR family regulatory protein [nite]			ROP_49490 :putative LytR family regulatory protein : : : Rhodococcus opacus B4	cell envelope-related transcriptional attenuator		F1YNZ4_9ACTO Cell envelope-like function transcript ional attenuator OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_18217 PE=4 SV=1;cell envelope-related transcriptional attenuator [S treptomyces violaceusniger Tu 4113];KEGG: cai:Caci_7618 cell envelope-related transcriptional attenuator; TIGRFAM: Cell envelope-related transcriptional attenuator; PFAM: Cell envelope-related transcriptional attenuator;Cell envelope-related transcriptional attenuator domain; cl00581;LytR cell envelope-related transcriptional attenuator; pfam13399
GOHSU28	CDS	GOHSU_28_00020	1910..2578		putative phosphate transport system regulatory protein [nite]	 phoU [nite]		RER_47930 :putative phosphate transport system regulatory protein : phoU : : Rhodococcus erythropolis PR4	phosphate transport system regulatory protein PhoU		F1YNZ5_9ACTO Phosphate transporter regulatory prote in PhoU OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_182 22 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: PhoU domain; TIGRFAM: phosphate transport system regulatory protein PhoU;phosphate transport system regulatory protein PhoU; TIGR02135;PhoU domain; pfam01895
GOHSU28	CDS	GOHSU_28_00030	complement(2664..3440)		phosphate ABC transporter ATP-binding protein [nite]	 pstB [nite]		ROP_49470 :phosphate ABC transporter ATP-binding protein : pstB : 3.6.3.27 : Rhodococcus opacus B4	phosphate ABC transporter ATP-binding protein		F1YNZ9_9ACTO Phosphate ABC transporter ATPase subun it OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18242 PE =3 SV=1;phosphate ABC transporter ATP-binding protein [Myco bacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;phosphate transporter ATP-binding protein; Provisional;Similar to PSTB_MYCSM e-123;Phosphate uptake is of fundamental importance in the cell physiology of bacteria because phosphate is required as a nutrient.  The Pst system of E. coli comprises four distinct subunits encoded by the pstS, pstA, pstB, and pstC genes.  The PstS protein...; cd03260;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU28	CDS	GOHSU_28_00040	complement(3443..4318)		phosphate ABC transporter permease protein [nite]	 pstA [nite]		RER_48020 :phosphate ABC transporter permease protein : pstA : : Rhodococcus erythropolis PR4	phosphate ABC transporter permease		F1YP00_9ACTO Phosphate ABC transport system, permea se protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 18247 PE=3 SV=1;phosphate ABC transporter permease [Mycobacterium a bscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q743D2_MYCPA 6e-84;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU28	CDS	GOHSU_28_00050	complement(4329..5369)		phosphate ABC transporter permease protein [nite]	 pstC [nite]		ROP_49450 :phosphate ABC transporter permease protein : pstC : : Rhodococcus opacus B4	phosphate ABC transporter permease		F1YP01_9ACTO Phosphate ABC transporter inner membra ne subunit PstC OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_18252 PE=3 SV=1;phosphate ABC transporter permease [Mycobacterium a bscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q743D3_MYCPA e-107;phosphate ABC transporter, permease protein PstC; TIGR02138;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU28	CDS	GOHSU_28_00060	complement(5366..6505)		phosphate ABC transporter phosphate-binding protein [nite]	 pstS [nite]		ROP_49440 :phosphate ABC transporter phosphate-binding protein : pstS : : Rhodococcus opacus B4	phosphate ABC transporter substrate-binding protein	phoS2_1	F1YP02_9ACTO Phosphate ABC transporter periplasmic phosphate--binding protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18257 PE=4 SV=1;phosphate ABC transporter substrate-binding protein [Mycobacterium sp. JDM601];The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cl11398
GOHSU28	CDS	GOHSU_28_00070	complement(6613..7533)		probable acetyltransferase MshD [nite]	 mshD [nite]		RER_48050 :probable acetyltransferase MshD : mshD : 2.3.1.- : Rhodococcus erythropolis PR4	mycothiol biosynthesis acetyltransferase		F1YP03_9ACTO Mycothiol biosynthesis acetyltransfera se OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18262 PE =3 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: nfa:nfa5990  acetyltransferase; TIGRFAM: mycothiol biosynthesis acetyltransferase; PFAM: GCN5-related N-acetyltransferase;mycothiol synthase; TIGR03448;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU28	CDS	GOHSU_28_00080	complement(7530..8300)		putative transcriptional regulator [nite]			ROP_49420 :putative transcriptional regulator : : : Rhodococcus opacus B4	transcriptional regulator	glnR	G7GTN2_9ACTO Putative transcriptional regulator OS= Gordonia amarae NBRC 15530 GN=GOAMR_61_00530 PE=4 SV=1;glnR gene product [Gordonia polyisoprenivorans VH2] ;Response regulators consisting of a CheY-like receiver domain and a winged-helix DNA-binding domain [Signal transduction mechanisms / Transcription]; COG0745;Effector domain of response regulator. Bacteria and certain eukaryotes like protozoa and higher plants use two-component signal transduction systems to detect and respond to changes in the environment. The system consists of a sensor histidine kinase and...; cd00383;DNA binding site [nucleotide binding]
GOHSU28	CDS	GOHSU_28_00090	8377..9258		hypothetical protein [nite]			ROP_49410 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YP05_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18272 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF2993); pfam11209
GOHSU28	CDS	GOHSU_28_00100	9334..10161		putative thiosulfate sulfurtransferase [nite]			RER_48100 :putative thiosulfate sulfurtransferase : : 2.8.1.1 : Rhodococcus erythropolis PR4	thiosulfate sulfurtransferase		F1YP06_9ACTO Sulfurtransferase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_18277 PE=4 SV=1;thiosulfate sulfurtransferase [Rhodococcus opacus B 4];Rhodanese-related sulfurtransferase [Inorganic ion transport and metabolism]; COG2897;protein synonym:rhodanese;Thiosulfate sulfurtransferase (TST), N-terminal, inactive domain. TST contains 2 copies of the Rhodanese Homology Domain; this is the 1st repeat, which does not contain the catalytically active Cys residue. The role of the 1st repeat is uncertain, but it...; cd01448;active site residue [active];Thiosulfate sulfurtransferase (TST), C-terminal, catalytic domain. TST contains 2 copies of the Rhodanese Homology Domain; this is the second repeat. Only the second repeat contains the catalytically active Cys residue; cd01449
GOHSU28	CDS	GOHSU_28_00110	10206..10511		conserved hypothetical protein [nite]			RER_48110 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein sseC1		F1YP07_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18282 PE=4 SV=1;unnamed protein product [Mycobacterium tuberculosis KZN 4207];drug-susceptible;Protein of unknown function (DUF1416); pfam07210
GOHSU28	CDS	GOHSU_28_00120	10789..11406		UPF0678 fatty acid-binding protein-like protein <locus_tag> [hamap]		MF_01297:UPF0678 fatty acid-binding protein-like protein <locus_tag>: 	ROP_49350 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YP12_9ACTO UPF0678 fatty acid-binding protein-lik e protein SCNU_18307 OS=Gordonia neofelifaecis NRRL B-5939 5 GN=SCNU_18307 PE=3 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];cytoplasmic protein; N-terminal truncated in relation to orthologs;nitrobindin heme-binding domain; cd07828;heme-binding site [chemical binding]
GOHSU28	CDS	GOHSU_28_00130	complement(11435..13135)		putative ABC transporter permease/ATP-binding protein CydC [nite]	 cydC [nite]		ROP_49330 :putative ABC transporter permease/ATP-binding protein CydC : cydC : : Rhodococcus opacus B4	ABC transport system, CydDC cysteine exporter (CydDC-E) family, ATP-binding/permease protein CydC	cydC	F1YN54_9ACTO ABC transporter CydDC cysteine exporte r (CydDC-E) family, permease/ATP-binding protein CydC OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_16398 PE=3 SV=1 ;cydC gene product [Gordonia polyisoprenivorans VH2] ;thiol reductant ABC exporter, CydC subunit; TIGR02868;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU28	CDS	GOHSU_28_00140	complement(13125..14888)		putative ABC transporter permease/ATP-binding protein CydD [nite]	 cydD [nite]		RER_48160 :putative ABC transporter permease/ATP-binding protein CydD : cydD : : Rhodococcus erythropolis PR4	cysteine ABC transporter permease/ATP-binding protein CydD		H0R8Z9_9ACTO ABC transporter permease/ATP-binding p rotein CydD OS=Gordonia polyisoprenivorans NBRC 16320 GN=c ydD PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: afe:Lferr_1109 ABC transporter CydDC cysteine exporter (CydDC-E) family, permease/ATP-binding protein CydD; TIGRFAM: ABC transporter CydDC cysteine exporter (CydDC-E) family, permease/ATP-binding protein CydD; PFAM: ABC transporter; SMART: ATPase AAA;ABC transporter transmembrane region; cl00549;The MRP (Mutidrug Resistance Protein)-like transporters are involved in drug, peptide, and lipid export.  They belong to the subfamily C of the ATP-binding cassette (ABC) superfamily of transport proteins.  The ABCC subfamily contains transporters with a...; cd03228;putative bacteriocin export ABC transporter, lactococcin 972 group; TIGR03608;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU28	CDS	GOHSU_28_00150	complement(14892..15971)		cytochrome bd-type menaquinol oxidase subunit II [nite]	 cydB [nite]		ROP_49310 :cytochrome bd-type menaquinol oxidase subunit II : cydB : 1.10.3.- : Rhodococcus opacus B4	cytochrome d ubiquinol oxidase, subunit 2	cydB	F1YN56_9ACTO Cytochrome bd-type menaquinol oxidase subunit II OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 16408 PE=4 SV=1;cydB gene product [Gordonia polyisoprenivorans VH2] ;cytochrome d ubiquinol oxidase subunit 2; Provisional; cl12219;Cytochrome oxidase subunit II; pfam02322
GOHSU28	CDS	GOHSU_28_00160	complement(15988..17592)		cytochrome bd-type menaquinol oxidase subunit I [nite]	 cydA [nite]		ROP_49300 :cytochrome bd-type menaquinol oxidase subunit I : cydA : 1.10.3.- : Rhodococcus opacus B4	cytochrome bd-I oxidase subunit I	cydA	F1YN57_9ACTO Cytochrome bd ubiquinol oxidase subuni t I OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16413 P E=4 SV=1;cytochrome bd-I oxidase subunit I [Amycolatopsis me diterranei U32];cytochrome d terminal oxidase subunit 1; Provisional;Bacterial Cytochrome Ubiquinol Oxidase; cl00562
GOHSU28	CDS	GOHSU_28_00170	complement(18147..18953)		putative aminotransferase [nite]			ROP_49290 :putative aminotransferase : : 2.6.1.- : Rhodococcus opacus B4	amino acid aminotransferase		H5TY68_9ACTO Putative aminotransferase OS=Gordonia sputi NBRC 100414 GN=GOSPT_045_00620 PE=3 SV=1;amino acid aminotransferase [Streptomyces sp. AA4];ADCL_like: 4-Amino-4-deoxychorismate lyase:  is a member of the fold-type IV of PLP dependent enzymes that converts 4-amino-4-deoxychorismate (ADC) to p-aminobenzoate and pyruvate.  Based on the information available from the crystal structure, most...; cd01559;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU28	CDS	GOHSU_28_00180	19029..20192		conserved hypothetical protein [nite]			RER_48200 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	folate-binding protein YgfZ		F1YN64_9ACTO Folate-binding protein YgfZ OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_16448 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: folate-binding protein YgfZ; PFAM: glycine cleavage T protein (aminomethyl transferase); KEGG: mxa:MXAN_6341 glycine cleavage system T protein;Predicted aminomethyltransferase related to GcvT [General function prediction only];folate-binding protein YgfZ; TIGR03317
GOHSU28	CDS	GOHSU_28_00190	20350..20535		conserved hypothetical protein [nite]			RER_48230 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0R4E9_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_098_00160 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];Protein of unknown function (DUF3073); pfam11273
GOHSU28	CDS	GOHSU_28_00200	complement(20954..22048)		aminoimidazole ribonucleotide synthetase [nite]	 purM [nite]		RER_48240 :aminoimidazole ribonucleotide synthetase : purM : 6.3.3.1 : Rhodococcus erythropolis PR4	phosphoribosylaminoimidazole synthetase	purM	F1YMY4_9ACTO Phosphoribosylformylglycinamidine cycl o-ligase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16 458 PE=3 SV=1;phosphoribosylaminoimidazole synthetase [Corynebact erium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 1-(5-phosphoribosyl)-5-aminoimidazole from 2-(formamido)-N1-(5-phosphoribosyl)acetamidine and ATP in purine biosynthesis;phosphoribosylaminoimidazole synthetase; Provisional;PurM (Aminoimidazole Ribonucleotide [AIR] synthetase), one of eleven enzymes required for purine biosynthesis, catalyzes the conversion of formylglycinamide ribonucleotide (FGAM) and ATP to AIR, ADP, and Pi, the fifth step in de novo purine biosynthesis; cd02196;dimerization interface [polypeptide binding];putative ATP binding site [chemical binding]
GOHSU28	CDS	GOHSU_28_00210	complement(22108..23664)		amidophosphoribosyltransferase [nite]	 purF [nite]		ROP_49230 :amidophosphoribosyltransferase : purF : 2.4.2.14 : Rhodococcus opacus B4	amidophosphoribosyltransferase		H0R4F1_9ACTO Amidophosphoribosyltransferase OS=Gord onia effusa NBRC 100432 GN=purF PE=3 SV=1;amidophosphoribosyltransferase [Corynebacterium glu tamicum ATCC 13032];Catalyzes first step of the de novo purine nucleotide biosynthetic pathway;amidophosphoribosyltransferase; Provisional;Glutamine amidotransferases class-II (GN-AT)_GPAT- type. This domain is found at the N-terminus of  glutamine phosphoribosylpyrophosphate (Prpp) amidotransferase (GPATase) . The glutaminase domain catalyzes amide nitrogen transfer from glutamine to the...; cd00715;tetramer interface [polypeptide binding];Phosphoribosyl transferase (PRT)-type I domain; cd06223
GOHSU28	CDS	GOHSU_28_00220	complement(23745..24143)		conserved hypothetical protein [nite]			RER_48290 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0L3Q5_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_0943 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU28	CDS	GOHSU_28_00230	24228..25088		hypothetical protein [nite]			KSE_48690 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		F1YMY7_9ACTO PHP domain-containing protein OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_16473 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 4 : Homologs of previously reported genes of unknown function;Predicted metal-dependent phosphoesterases (PHP family) [General function prediction only]
GOHSU28	CDS	GOHSU_28_00240	complement(25106..25354)	overlap	not assigned						H0RB08_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_028_00190 PE =4 SV=1
GOHSU28	CDS	GOHSU_28_00250	25269..25520	overlap	recombination protein F [migap]	recF [migap]			recombination protein F	recF	recF gene product [Anabaena variabilis ATCC 29413];recombination protein F; Reviewed; PRK00064;Required for DNA replication; binds preferentially to single-stranded, linear DNA;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099
GOHSU28	CDS	GOHSU_28_00260	25541..25825		not assigned						
GOHSU28	CDS	GOHSU_28_00270	complement(25923..26684)		hypothetical protein [migap]				hypothetical protein		H5TRC2_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_192_00180 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU28	CDS	GOHSU_28_00280	26843..28354		hypothetical protein [nite]			ROP_19750 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YMY8_9ACTO Glutamate--cysteine ligase, GCS2 OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16478 PE=4 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];KEGG: stp:Strop_2138 hypothetical protein;Glutamate-cysteine ligase family 2(GCS2); cl00954
GOHSU28	CDS	GOHSU_28_00290	28393..29190		glycerophosphoryl diester phosphodiesterase [migap]				glycerophosphoryl diester phosphodiesterase		H5UKJ9_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_149_00330 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: glycerophosphoryl diester phosphodiesterase; KEGG: avi:Avi_5609 hypothetical protein;Catalytic domain of phosphoinositide-specific phospholipase C-like phosphodiesterases superfamily; cl14615;catalytic site [active];Glycerophosphoryl diester phosphodiesterase family; pfam03009
GOHSU28	CDS	GOHSU_28_00300	complement(29239..29709)		hypothetical membrane protein [nite]			ROP_42650 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H0R4F7_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_098_00240 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF2752);Protein of unknown function (DUF2752); pfam10825
GOHSU28	CDS	GOHSU_28_00310	complement(29719..30228)		hypothetical protein [nite]			RER_42250 :hypothetical protein : : : Rhodococcus erythropolis PR4	interferon-induced transmembrane protein		H0R4F8_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_098_00250 PE=4 SV=1;interferon-induced transmembrane protein [Frankia s p. EuI1c];PFAM: interferon-induced transmembrane protein; KEGG: fal:FRAAL0658 hypothetical protein;Interferon-induced transmembrane protein; pfam04505
GOHSU28	CDS	GOHSU_28_00320	complement(30280..32574)		phosphoribosylformylglycinamidine synthase 2 [hamap]	purL [hamap]	MF_00420:phosphoribosylformylglycinamidine synthase 2:purL 	ROP_49190 :phosphoribosylformylglycinamidine synthase II : purL : 6.3.5.3 : Rhodococcus opacus B4	phosphoribosylformylglycinamidine synthase 2	purL	G7H357_9ACTO Phosphoribosylformylglycinamidine synt hase 2 OS=Gordonia araii NBRC 100433 GN=purL PE=3 SV=1;purL gene product [Gordonia polyisoprenivorans VH2] ;phosphoribosylformylglycinamidine synthase II; Provisional;PurL subunit of the formylglycinamide ribonucleotide amidotransferase (FGAR-AT), first repeat. FGAR-AT catalyzes the ATP-dependent conversion of formylglycinamide ribonucleotide (FGAR) and glutamine to formylglycinamidine ribonucleotide (FGAM), ADP; cd02203;dimerization interface [polypeptide binding];ATP binding site [chemical binding];PurL subunit of the formylglycinamide ribonucleotide amidotransferase (FGAR-AT), second repeat. FGAR-AT catalyzes the ATP-dependent conversion of formylglycinamide ribonucleotide (FGAR) and glutamine to formylglycinamidine ribonucleotide (FGAM), ADP; cd02204
GOHSU28	CDS	GOHSU_28_00330	32602..34386		ATPase central domain-containing protein [migap]				ATPase central domain-containing protein		F1YMZ6_9ACTO AAA ATPase central domain-containing p rotein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1651 8 PE=4 SV=1;ATPase central domain-containing protein [Mycobacte rium sp. KMS];PFAM: AAA ATPase, central domain protein; SMART: AAA ATPase; KEGG: mmc:Mmcs_0064 AAA ATPase, central region;type VII secretion AAA-ATPase EccA; TIGR03922;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger
GOHSU28	CDS	GOHSU_28_00340	complement(34396..34854)		not assigned						H0RMC0_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_093_00280 PE =4 SV=1
GOHSU28	CDS	GOHSU_28_00350	35083..36546		hypothetical protein [nite]			ROP_55160 :hypothetical protein : : : Rhodococcus opacus B4	putative HNH endonuclease		F1YN44_9ACTO HNH endonuclease OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_16758 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF222); pfam02720;HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and...; cd00085
GOHSU28	CDS	GOHSU_28_00360	complement(36837..38675)		adenylyl-sulfate kinase [hamap]	cysC [hamap]	MF_00065:adenylyl-sulfate kinase:cysC 	RER_39360 :bifunctional protein CysNC : cysNC : 2.7.7.4/2.7.1.25 : Rhodococcus erythropolis PR4	sulfate adenylyltransferase subunit 1 / adenylylsulfate kinase		F1YMZ9_9ACTO Bifunctional sulfate adenylyltransfera se subunit 1/adenylylsulfate kinase protein OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_16533 PE=3 SV=1;unnamed protein product [Pelagibacterium halotolera ns B2];bifunctional sulfate adenylyltransferase subunit 1/adenylylsulfate kinase protein; Provisional;CysN, together with protein CysD, forms the ATP sulfurylase (ATPS) complex; cd04166;CysD dimerization site [polypeptide binding];G1 box;putative GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;CysN_NodQ_II: This subfamily represents the domain II of the large subunit of ATP sulfurylase (ATPS): CysN or the N-terminal portion of NodQ, found mainly in proteobacteria and homologous to the domain II of EF-Tu. Escherichia coli ATPS consists of CysN...; cd03695;TCysN_NoDQ_II: This subfamily represents the domain II of the large subunit of ATP sulfurylase (ATPS): CysN or the N-terminal portion of NodQ, found mainly in proteobacteria and homologous to the domain II of EF-Tu. Escherichia coli ATPS consists of CysN...; cd04095;Adenosine 5'-phosphosulfate kinase (APSK) catalyzes the phosphorylation of adenosine 5'-phosphosulfate to form 3'-phosphoadenosine 5'-phosphosulfate (PAPS). The end-product PAPS is a biologically 'activated' sulfate form important for the assimilation of...; cd02027;ligand-binding site [chemical binding]
GOHSU28	CDS	GOHSU_28_00370	complement(38675..39601)		sulfate adenylyltransferase subunit 2 [nite]	 cysD [nite]		RER_39370 :sulfate adenylyltransferase subunit 2 : cysD : 2.7.7.4 : Rhodococcus erythropolis PR4	sulfate adenylyltransferase subunit 2		F1YN00_9ACTO Sulfate adenylyltransferase subunit 2 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16538 PE=4 SV=1;sulfate adenylyltransferase subunit 2 [Streptomyces sp. AA4];sulfate adenylyltransferase, small subunit; TIGR02039;This domain is found in phosphoadenosine phosphosulphate (PAPS) reductase enzymes or PAPS sulphotransferase. PAPS reductase is part of the adenine nucleotide alpha hydrolases superfamily also including N type ATP PPases and ATP sulphurylases. A highly...; cd01713;Active Sites [active]
GOHSU28	CDS	GOHSU_28_00380	39777..40358		hypothetical protein [nite]			MLP_32760 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YN02_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16548 PE=4 SV=1;unnamed protein product [Rhodopseudomonas palustris DX- 1];KEGG: rme:Rmet_4922 hypothetical protein;SirA, YedF, and YeeD. Two-layered alpha/beta sandwich domain.  SirA (also known as UvrY,  and YhhP) belongs to a family of bacterial two-component response regulators that controls secondary metabolism and virulence. The other member of this...; cl00436
GOHSU28	CDS	GOHSU_28_00390	40387..40713		7Fe ferredoxin [nite]			ROP_50850 :7Fe ferredoxin : : : Rhodococcus opacus B4	ferredoxin		H5TMB5_9ACTO Putative 7Fe ferredoxin OS=Gordonia ot itidis NBRC 100426 GN=GOOTI_114_00520 PE=4 SV=1;ferredoxin [Rhodococcus jostii RHA1];Ferredoxin [Energy production and conversion];4Fe-4S binding domain; cl02805
GOHSU28	CDS	GOHSU_28_00400	40808..41710		enoyl-CoA hydratase/isomerase family protein [nite]			ROP_45390 :enoyl-CoA hydratase/isomerase family protein : : : Rhodococcus opacus B4	enoyl-CoA hydratase/isomerase		H6MSY0_GORPV Putative enoyl-CoA hydratase OS=Gordon ia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c40 940 PE=4 SV=1;enoyl-CoA hydratase/isomerase [Frankia sp. EuI1c];PFAM: enoyl-CoA hydratase/isomerase; KEGG: tcu:Tcur_4614 enoyl-CoA hydratase/isomerase;enoyl-CoA hydratase; Provisional;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU28	CDS	GOHSU_28_00410	41710..42876		putative acyl-CoA dehydrogenase [nite]			ROP_45380 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		H6MSY1_GORPV Acyl-CoA dehydrogenase OS=Gordonia pol yisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c40950 PE =3 SV=1;acyl-CoA dehydrogenase [Amycolatopsis mediterranei U32];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU28	CDS	GOHSU_28_00420	42879..43823		putative acyl-CoA dehydrogenase [nite]			ROP_45370 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		H0R4H1_9ACTO Putative acyl-CoA dehydrogenase OS=Gor donia effusa NBRC 100432 GN=GOEFS_098_00380 PE=4 SV=1;acyl-CoA dehydrogenase [Rhodococcus jostii RHA1];Acyl-CoA dehydrogenase; cl09933
GOHSU28	CDS	GOHSU_28_00430	43838..44920		putative acyl-CoA dehydrogenase [nite]			ROP_45360 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		F1YN10_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 16588 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Acyl-CoA dehydrogenase, C-terminal domain; Acyl-CoA dehydrogenase, middle domain; Acyl-CoA dehydrogenase, N-terminal domain;Acyl-CoA dehydrogenase; cl09933;pimeloyl-CoA dehydrogenase, small subunit; TIGR03203
GOHSU28	CDS	GOHSU_28_00440	complement(44930..46084)		putative TetR family transcriptional regulator [nite]			ROP_23700 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		A1TC62_MYCVP Transcriptional regulator, TetR family OS=Mycobacterium vanbaalenii (strain DSM 7251 / PYR-1) GN =Mvan_3985 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m vanbaalenii PYR- 1];PFAM: regulatory protein, TetR; KEGG: fra:Francci3_2475 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU28	CDS	GOHSU_28_00450	complement(46113..47261)		acyl-CoA dehydrogenase [nite]	 fadE [nite]		RER_27750 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase	fadE	F4CPY1_PSEUX Long-chain-acyl-CoA dehydrogenase OS=P seudonocardia dioxanivorans (strain ATCC 55486 / DSM 44775 / JCM 13855 / CB1190) GN=Psed_3998 PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus opacus B4];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU28	CDS	GOHSU_28_00460	47494..49017		putative fatty-acid--CoA ligase [nite]			ROP_23620 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus opacus B4	fatty-acid--CoA ligase		H6QYB1_NOCCG Putative 4-coumarate--CoA ligase 2 (4C L 2) (4-coumaroyl-CoA synthase 2) OS=Nocardia cyriacigeorg ica (strain GUH-2) GN=NOCYR_1971 PE=4 SV=1;fatty-acid--CoA ligase [Rhodococcus erythropolis PR 4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;long-chain-fatty-acid--CoA ligase; Validated;protein synonym:acyl-CoA synthetase;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU28	CDS	GOHSU_28_00470	49046..50464		hypothetical protein [nite]			ROP_57120 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H6MSY9_GORPV Putative diacylglycerol O-acyltransfer ase OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2 ) GN=GPOL_c41030 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF1298); Uncharacterised protein family (UPF0089); TIGRFAM: acyltransferase, WS/DGAT/MGAT;acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007
GOHSU28	CDS	GOHSU_28_00480	complement(50499..51074)		putative oxidoreductase [nite]			RER_07580 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	oxidoreductase		H6MSZ3_GORPV Flavin-dependent monooxygenase, reduct ase subunit HsaB OS=Gordonia polyisoprenivorans (strain DS M 44266 / VH2) GN=hsaB PE=4 SV=1;oxidoreductase [Rhodococcus opacus B4];Flavin Reductases; cl00801
GOHSU28	CDS	GOHSU_28_00490	complement(51078..52028)		extradiol dioxygenase [nite]			ROP_44560 :extradiol dioxygenase : : 1.13.11.- : Rhodococcus opacus B4	glyoxalase/bleomycin resistance protein/dioxygenase		F1YN17_9ACTO Biphenyl-2,3-diol 1,2-dioxygenase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_16623 PE=3 SV=1 ;glyoxalase/bleomycin resistance protein/dioxygenase [Mycobacterium sp. KMS];PFAM: Glyoxalase/bleomycin resistance protein/dioxygenase; KEGG: mmc:Mmcs_4707 glyoxalase/bleomycin resistance protein/dioxygenase;2,3-dihydroxybiphenyl 1,2-dioxygenase; TIGR03213;N-terminal domain of 2,3-dihydroxybiphenyl 1,2-dioxygenase (BphC, EC 1.13.11.39) 1 from Rhodococcus globerulus P6 (BphC1-RGP6) and similar proteins; cd07252;C-terminal domain of 2,3-dihydroxybiphenyl 1,2-dioxygenase (BphC, EC 1.13.11.39) 1 from Rhodococcus globerulus P6 (BphC1-RGP6) and similar proteins; cd07237;Fe binding site [ion binding]
GOHSU28	CDS	GOHSU_28_00500	complement(52036..53217)		putative hydroxylase [nite]			RER_07550 :putative hydroxylase : : : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase		D0L3N3_GORB4 Acyl-CoA dehydrogenase type 2 domain p rotein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43 247 / JCM 3198 / NCTC 10667) GN=Gbro_0920 PE=4 SV=1;acyl-CoA dehydrogenase [Rhodococcus equi 103S];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Naphthocyclinone hydroxylase; cd01159;Flavin binding site [chemical binding]
GOHSU28	CDS	GOHSU_28_00510	53282..54523		3-ketosteroid 9alpha-hydroxylase component KshA [nite]	 kshA [nite]		ROP_44530 :3-ketosteroid 9alpha-hydroxylase component KshA : kshA : : Rhodococcus opacus B4	iron-sulfur binding oxidoreductase		G7H337_9ACTO 3-ketosteroid 9alpha-hydroxylase compo nent KshA OS=Gordonia araii NBRC 100433 GN=kshA PE=4 SV=1;iron-sulfur binding oxidoreductase [Rhodococcus equ i 103S];Phenylpropionate dioxygenase and related ring-hydroxylating dioxygenases, large terminal subunit [Inorganic ion transport and metabolism / General function prediction only]; COG4638;The alignment model represents the N-terminal rieske iron-sulfur domain of KshA, the oxygenase component of 3-ketosteroid 9-alpha-hydroxylase (KSH).  The terminal oxygenase component of KSH is a key enzyme in the microbial steroid degradation pathway; cd03531
GOHSU28	CDS	GOHSU_28_00520	54523..54903		hypothetical protein [nite]			ROP_44520 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YN20_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16638 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU28	CDS	GOHSU_28_00530	54986..56704		3-ketosteroid delta(1)-dehydrogenase [nite]			RER_29470 :3-ketosteroid delta(1)-dehydrogenase : : 1.3.99.4 : Rhodococcus erythropolis PR4	3-ketosteroid-delta-1-dehydrogenase	kstD	G7H0W1_9ACTO 3-keto-5alpha-steroid delta(1)-dehydro genase KstD OS=Gordonia araii NBRC 100433 GN=kstD PE=4 SV= 1;3-ketosteroid-delta-1-dehydrogenase [Rhodococcus jo stii RHA1];3-ketosteroid-delta-1-dehydrogenase; Reviewed;initiates steroid ring degradation; catalyzes the transhydrogenation of 3-keto-4-ene-steroid to 3-keto-1,4-diene-steroid e.g., progesterone to 1,4-androstadiene-3,17-dione
GOHSU28	CDS	GOHSU_28_00540	56707..57570		putative enoyl-CoA hydratase [nite]			RER_07360 :putative enoyl-CoA hydratase : : 4.2.1.- : Rhodococcus erythropolis PR4	dehydrogenase		F1YN28_9ACTO 3-alpha,7-alpha,12-alpha-trihydroxy-5- beta-chole st-24-enoyl-CoAhydratase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_16678 PE=4 SV=1;dehydrogenase [Rhodococcus equi 103S];enoyl-CoA hydratase;The hotdog fold was initially identified in the E. coli FabA (beta-hydroxydecanoyl-acyl carrier protein (ACP)-dehydratase) structure and subsequently in 4HBT (4-hydroxybenzoyl-CoA thioesterase) from Pseudomonas. A number of other seemingly unrelated...; cl00509;HDE_HSD  The R-hydratase-like hot dog fold of the 17-beta-hydroxysteriod dehydrogenase (HSD), and Hydratase-Dehydrogenase-Epimerase (HDE) proteins.  Other enzymes with this fold include MaoC dehydratase, and the fatty acid synthase beta subunit; cd03448;dimer interaction site [polypeptide binding];substrate-binding tunnel;catalytic site [active];substrate binding site [chemical binding]
GOHSU28	CDS	GOHSU_28_00550	57704..58042		conserved hypothetical protein [nite]			RER_17010 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YN29_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16683 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU28	CDS	GOHSU_28_00560	complement(58045..58191)	short	not assigned						F9VUK3_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_050_01460 PE=4 SV= 1
GOHSU28	CDS	GOHSU_28_00570	58357..58584		putative drug resistance efflux protein [nite]			RER_pREL1-02420 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	transporter		G8RPH2_MYCRN Sugar phosphate permease OS=Mycobacter ium rhodesiae (strain NBB3) GN=MycrhN_0703 PE=4 SV=1;transporter [Nocardia farcinica IFM 10152];methyl viologen resistance protein SmvA; Provisional;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU28	CDS	GOHSU_28_00580	58560..59900		putative drug resistance efflux protein [nite]			RER_50590 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	transporter		F1YN31_9ACTO Major facilitator superfamily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16693 PE=4 SV=1;transporter [Nocardia farcinica IFM 10152];methyl viologen resistance protein SmvA; Provisional;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU28	CDS	GOHSU_28_00590	60467..60745		not assigned						
GOHSU28	CDS	GOHSU_28_00600	61085..62026		putative LysR family transcriptional regulator [nite]			MLP_19700 :putative LysR family transcriptional regulator : : : Microlunatus phosphovorus NM-1	transcriptional regulator		D5UVK2_TSUPD Transcriptional regulator, LysR family OS=Tsukamurella paurometabola (strain ATCC 8368 / DSM 201 62 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_3197 PE= 4 SV=1;transcriptional regulator [Azospirillum sp. B510];LysR family;Transcriptional regulator [Transcription]; COG0583;Helix-turn-helix domains; cl00088;The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cd05466;dimerization interface [polypeptide binding]
GOHSU28	CDS	GOHSU_28_00610	62023..62664		hypothetical protein [nite]			MLP_49620 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YEJ0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_00560 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: sus:acid_7882 hypothetical protein;Conserved hypothetical protein (DUF2461); cl02374
GOHSU28	CDS	GOHSU_28_00620	62743..63162		pyrroline-5-carboxylate reductase [migap]				pyrroline-5-carboxylate reductase		F9VWD3_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_060_01550 PE=4 SV= 1;pyrroline-5-carboxylate reductase [Olsenella uli DS M 7084];COGs: COG0345 Pyrroline-5-carboxylate reductase; InterPro IPR016040:IPR000304:IPR004455; KEGG: hoh:Hoch_4718 pyrroline-5-carboxylate reductase; PFAM: NADP oxidoreductase coenzyme F420-dependent; PRIAM: Pyrroline-5-carboxylate reductase; SPTR: C2D7Z1 Possible pyrroline-5-carboxylate reductase; TIGRFAM: pyrroline-5-carboxylate reductase; PFAM: NADP oxidoreductase coenzyme F420-dependent; TIGRFAM: pyrroline-5-carboxylate reductase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;pyrroline-5-carboxylate reductase; TIGR00112
GOHSU28	CDS	GOHSU_28_00630	63315..64103		hypothetical protein [migap]				hypothetical protein		D0LFI0_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_4916 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];DnaB-like helicase N terminal domain; pfam00772
GOHSU28	CDS	GOHSU_28_00640	complement(64123..64563)	overlap	probable sulfate transporter [nite]			RER_25760 :probable sulfate transporter : : : Rhodococcus erythropolis PR4	sulfate transporter		H5UGQ8_9ACTO Putative sulfate transporter OS=Gordon ia terrae NBRC 100016 GN=GOTRE_076_00090 PE=4 SV=1;sulfate transporter [Microlunatus phosphovorus NM- 1];strain coidentity: NBRC 101784;high affinity sulphate transporter 1; TIGR00815;Sulphate Transporter and Anti-Sigma factor antagonist domain of SulP-like sulfate transporters, plays a role in the function and regulation of the transport activity, proposed general NTP binding function; cd07042
GOHSU28	CDS	GOHSU_28_00650	complement(64503..64712)	overlap	sulfate transporter [migap]				sulfate transporter		H5UGQ8_9ACTO Putative sulfate transporter OS=Gordon ia terrae NBRC 100016 GN=GOTRE_076_00090 PE=4 SV=1;sulfate transporter [Rhodococcus opacus B4];high affinity sulphate transporter 1; TIGR00815
GOHSU28	CDS	GOHSU_28_00660	complement(64794..65984)	overlap	sulfate transporter [nite]			ROP_36190 :sulfate transporter : : : Rhodococcus opacus B4	sulfate transporter		H5UGQ8_9ACTO Putative sulfate transporter OS=Gordon ia terrae NBRC 100016 GN=GOTRE_076_00090 PE=4 SV=1;sulfate transporter [Nakamurella multipartita DSM 4 4233];TIGRFAM: sulfate transporter; PFAM: sulfate transporter; sulfate transporter/antisigma-factor antagonist STAS; xanthine/uracil/vitamin C permease; KEGG: pna:Pnap_3364 sulfate transporter;high affinity sulphate transporter 1; TIGR00815;Sulfate transporter family; cl15842;Sulphate Transporter and Anti-Sigma factor antagonist domain of SulP-like sulfate transporters, plays a role in the function and regulation of the transport activity, proposed general NTP binding function; cd07042
GOHSU28	CDS	GOHSU_28_00670	complement(65875..66429)	overlap	sulfate transporter [nite]			MLP_34710 :sulfate transporter : : : Microlunatus phosphovorus NM-1	sulfate transporter		H0JY06_9NOCA Sulfate transporter OS=Rhodococcus pyr idinivorans AK37 GN=AK37_23248 PE=4 SV=1;sulfate transporter [Pseudomonas putida F1];TIGRFAM: sulfate transporter; PFAM: Sulfate transporter/antisigma-factor antagonist STAS; Xanthine/uracil/vitamin C permease; sulphate transporter;Sulfate transporter family; cl15842;high affinity sulphate transporter 1; TIGR00815
GOHSU28	CDS	GOHSU_28_00680	complement(66473..68080)		putative fatty-acid--CoA ligase [nite]			RER_53370 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus erythropolis PR4	acyl-CoA ligase		F1YN45_9ACTO AMP-dependent synthetase and ligase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16763 PE=4 SV =1;acyl-CoA ligase/synthetase [Rhodococcus equi 103S];acyl-CoA synthetase; Validated;AMP-binding enzyme; cl15778
GOHSU28	CDS	GOHSU_28_00690	complement(68156..69337)		hypothetical protein [nite]			ROP_44160 :hypothetical protein : : : Rhodococcus opacus B4	lipid-transfer protein		H0R0G3_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_058_00250 PE=4 SV=1;lipid-transfer protein [Rhodococcus opacus B4];lipid-transfer protein; Provisional;Thiolase domain associated with sterol carrier protein (SCP)-x isoform and related proteins; SCP-2  has multiple roles in intracellular lipid circulation and metabolism. The N-terminal presequence in the SCP-x isoform represents a peroxisomal...; cd00829
GOHSU28	CDS	GOHSU_28_00700	complement(69330..69743)		conserved hypothetical protein [nite]			RER_07110 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5UDN0_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_053_00190 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];SAV4209 is a Streptomyces avermitilis protein with a hot dog fold that is similar to those of (R)-specific enoyl-CoA hydratase, the peroxisomal Hydratase-Dehydrogenase-Epimerase (HDE) protein, and the fatty acid synthase beta subunit.  The alpha- and...; cd03455;putative active site [active];putative catalytic site [active]
GOHSU28	CDS	GOHSU_28_00710	complement(69740..70756)		conserved hypothetical protein [nite]			RER_07100 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative nucleic-acid-binding protein containing a Zn-ribbon		F1YN48_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16778 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: DUF35 OB-fold domain;(R)-hydratase [(R)-specific enoyl-CoA hydratase]. Catalyzes the hydration of trans-2-enoyl CoA to (R)-3-hydroxyacyl-CoA as part of the PHA (polyhydroxyalkanoate) biosynthetic pathway.  The structure of the monomer includes a five-strand antiparallel...; cd03441;catalytic site [active];Predicted nucleic-acid-binding protein containing a Zn-ribbon [General function prediction only];DUF35 OB-fold domain; pfam01796
GOHSU28	CDS	GOHSU_28_00720	complement(70753..71913)		putative acyl-CoA dehydrogenase [nite]			ROP_44130 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		F1YN49_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 16783 PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus jostii RHA1];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU28	CDS	GOHSU_28_00730	complement(71904..>72629)	codon	putative acyl-CoA dehydrogenase [nite]			RER_07080 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	putative acyl-CoA dehydrogenase		F1YN50_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 16788 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
##GOHSU29	contig00141				69239
GOHSU29	CDS	GOHSU_29_00010	complement(<1..1361)	3x codon	putative ATP-dependent RNA helicase [nite]			RER_23370 :putative ATP-dependent RNA helicase : : 3.6.1.- : Rhodococcus erythropolis PR4	ATP-dependent RNA helicase		F1YMF2_9ACTO DEAD/DEAH box helicase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 15184 PE=4 SV=1;ATP-dependent RNA helicase [Rhodococcus jostii RHA1 ];DEAD-box helicases. A diverse family of proteins involved in ATP-dependent RNA unwinding, needed in a variety of cellular processes including splicing, ribosome biogenesis and RNA degradation. The name derives from the sequence of the Walker  B motif; cd00268;DEAD-like helicases superfamily; smart00487;ATP binding site [chemical binding];Mg++ binding site [ion binding];motif III;Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding]
GOHSU29	CDS	GOHSU_29_00020	complement(1603..2067)		3-dehydroquinate dehydratase [hamap]	aroQ [hamap]	MF_00169:3-dehydroquinate dehydratase:aroQ 	ROP_10850 :3-dehydroquinate dehydratase : aroQ : 4.2.1.10 : Rhodococcus opacus B4	3-dehydroquinate dehydratase		H5U5M7_9ACTO 3-dehydroquinate dehydratase OS=Gordon ia sputi NBRC 100414 GN=aroQ PE=3 SV=1;3-dehydroquinate dehydratase [Rhodococcus jostii RH A1];catalyzes the formation of 3-dehydroshikimate from 3-dehydroquinate in chorismate biosynthesis;Dehydroquinase (DHQase), type II. Dehydroquinase (or 3-dehydroquinate dehydratase) catalyzes the reversible dehydration of 3-dehydroquinate to form 3-dehydroshikimate. This reaction is part of two metabolic pathways: the biosynthetic shikimate pathway...; cd00466;trimer interface [polypeptide binding];dimer interface [polypeptide binding]
GOHSU29	CDS	GOHSU_29_00030	2159..2587		hypothetical membrane protein [nite]			ROP_41980 :hypothetical membrane protein : : : Rhodococcus opacus B4	integral membrane protein		C1B9U2_RHOOB Hypothetical membrane protein OS=Rhodo coccus opacus (strain B4) GN=ROP_41980 PE=4 SV=1;integral membrane protein [Streptomyces sviceus ATC C 29083]
GOHSU29	CDS	GOHSU_29_00040	2661..5990		arabinosyltransferase [nite]			ROP_39460 :arabinosyltransferase : : 2.4.2.- : Rhodococcus opacus B4	arabinosyltransferase		F1YMF6_9ACTO Cell wall arabinan synthesis protein O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15204 PE=4 S V=1;arabinosyltransferase [Rhodococcus opacus B4];Mycobacterial cell wall arabinan synthesis protein; pfam04602
GOHSU29	CDS	GOHSU_29_00050	6097..9444		arabinosyltransferase [nite]			RER_02260 :arabinosyltransferase : : 2.4.2.- : Rhodococcus erythropolis PR4	cell wall arabinan synthesis protein		H5U5M9_9ACTO Putative arabinosyltransferase OS=Gord onia sputi NBRC 100414 GN=GOSPT_118_01150 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: cell wall arabinan synthesis protein; KEGG: pst:PSPTO_3539 membrane protein PslK;Mycobacterial cell wall arabinan synthesis protein; pfam04602
GOHSU29	CDS	GOHSU_29_00060	9441..12860		arabinosyltransferase [nite]			RER_02250 :arabinosyltransferase : : 2.4.2.- : Rhodococcus erythropolis PR4	cell wall arabinan synthesis protein		F1YMF8_9ACTO Cell wall arabinan synthesis protein O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15214 PE=4 S V=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: cell wall arabinan synthesis protein;Mycobacterial cell wall arabinan synthesis protein; pfam04602
GOHSU29	CDS	GOHSU_29_00070	complement(13098..14666)		acyl-CoA carboxylase beta chain [nite]	 accD [nite]		ROP_39440 :acyl-CoA carboxylase beta chain : accD : 6.4.1.- : Rhodococcus opacus B4	carboxyl transferase		G7H1V8_9ACTO Acyl-CoA carboxylase beta chain OS=Gor donia araii NBRC 100433 GN=accD PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: carboxyl transferase; KEGG: mxa:MXAN_1113 propionyl-CoA carboxylase subunit beta;Acetyl-CoA carboxylase, carboxyltransferase component (subunits alpha and beta) [Lipid metabolism];Acetyl co-enzyme A carboxylase carboxyltransferase alpha subunit; cl15772
GOHSU29	CDS	GOHSU_29_00080	complement(14730..19970)		polyketide synthase Pks13 [nite]	 pks13 [nite]		ROP_39430 :polyketide synthase Pks13 : pks13 : : Rhodococcus opacus B4	beta-ketoacyl synthase		F1YMG4_9ACTO Polyketide synthase Pks13 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15244 PE=4 SV=1;beta-ketoacyl synthase [Mycobacterium gilvum PYR- G CK];PFAM: beta-ketoacyl synthase; Thioesterase; acyl transferase domain protein; phosphopantetheine-binding; KEGG: mva:Mvan_5640 beta-ketoacyl synthase;Phosphopantetheine attachment site; cl09936;polyketide synthases (PKSs) polymerize simple fatty acids into a large variety of different products, called polyketides, by successive decarboxylating Claisen condensations. PKSs can be divided into 2 groups, modular type I PKSs consisting of one or...; cd00833;3-oxoacyl-(acyl carrier protein) synthase II; Reviewed;malonyl CoA-acyl carrier protein transacylase; TIGR00128;Acyl transferase domain; cl08282;Thioesterase domains of type I polyketide synthases or non-ribosomal peptide synthetases [Secondary metabolites biosynthesis, transport, and catabolism]
GOHSU29	CDS	GOHSU_29_00090	complement(20069..21973)		long-chain fatty acyl-AMP ligase FadD32 [nite]	 fadD32 [nite]		ROP_39420 :long-chain fatty acyl-AMP ligase FadD32 : fadD32 : : Rhodococcus opacus B4	long-chain-fatty-acid--CoA ligase		F1YMG5_9ACTO Long-chain-fatty-acid--CoA ligase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_15249 PE=4 SV=1 ;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];long-chain-fatty-acid--CoA ligase; Validated;activates fatty acids by binding to coenzyme A;AMP-binding enzyme; cl15778
GOHSU29	CDS	GOHSU_29_00100	complement(22121..23125)		putative oxidoreductase [nite]			ROP_39410 :putative oxidoreductase : : : Rhodococcus opacus B4	luciferase family protein		F1YMG6_9ACTO Putative F420-dependent oxidoreductase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15254 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];PFAM: luciferase family protein; KEGG: fal:FRAAL2800 hypothetical protein;probable F420-dependent oxidoreductase, MSMEG_2516 family; TIGR03621;Flavin-utilizing monoxygenases; cl07892
GOHSU29	CDS	GOHSU_29_00110	complement(23251..24213)		conserved hypothetical protein [nite]			RER_02180 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMG7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15259 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein;Cutinase; pfam01083
GOHSU29	CDS	GOHSU_29_00120	complement(24217..24774)		hypothetical protein [migap]				hypothetical protein		F1YMG8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15264 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU29	CDS	GOHSU_29_00130	complement(24884..26263)		mycolyltransferase [nite]	 csp1 [nite]		RER_02170 :mycolyltransferase : csp1 : 2.3.1.- : Rhodococcus erythropolis PR4	mycolyltransferase	csp1	F1YMG9_9ACTO Putative esterase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_15269 PE=4 SV=1;mycolyltransferase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Putative esterase; pfam00756;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;LGFP repeat; pfam08310
GOHSU29	CDS	GOHSU_29_00140	complement(26455..27462)		mycolyltransferase [nite]			ROP_50250 :mycolyltransferase : : 2.3.1.- : Rhodococcus opacus B4	putative esterase		F1YMH0_9ACTO Putative esterase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_15274 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted esterase [General function prediction only]
GOHSU29	CDS	GOHSU_29_00150	complement(27787..29751)		arabinosyltransferase AftB [nite]	 aftB [nite]		ROP_39350 :arabinosyltransferase AftB : aftB : 2.4.2.- : Rhodococcus opacus B4	putative transmembrane protein		H5U167_9ACTO Arabinosyltransferase AftB OS=Gordonia sputi NBRC 100414 GN=aftB PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU29	CDS	GOHSU_29_00160	complement(29870..30781)		conserved hypothetical membrane protein [nite]			RER_02140 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	phosphoribose diphosphate:decaprenyl-phosphate phosphoribosyltransferase		H0R149_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_064_00390 PE=4 SV=1;phosphoribose diphosphate:decaprenyl-phosphate phos phoribosyltransferase [Corynebacterium glutamicum ATCC 130 32];IS fingerprint type 4-5;catalyzes the formation of decaprenylphosphoryl-5-phosphoribose from phosphoribose diphosphate and decaprenyl phosphate;UbiA prenyltransferase family; cl00337
GOHSU29	CDS	GOHSU_29_00170	complement(30792..31394)		hypothetical protein [nite]			ROP_39330 :hypothetical protein : : : Rhodococcus opacus B4	phosphatase		D0LBC2_GORB4 Phosphoesterase PA-phosphatase related protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_0206 PE=4 SV=1;phosphatase [Rhodococcus equi 103S];PAP2_like proteins, a super-family of histidine phosphatases and vanadium haloperoxidases, includes type 2 phosphatidic acid phosphatase or lipid phosphate phosphatase (LPP), Glucose-6-phosphatase, Phosphatidylglycerophosphatase B and bacterial acid...; cl00474
GOHSU29	CDS	GOHSU_29_00180	complement(31375..33276)		galactofuranosyltransferase [nite]	 glfT [nite]		ROP_39320 :galactofuranosyltransferase : glfT : 2.4.1.- : Rhodococcus opacus B4	putative glycosyltransferase		H6MTB3_GORPV Putative glycosyltransferase OS=Gordon ia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c04 080 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted glycosyltransferases [General function prediction only];Glycosyltransferase family A (GT-A) includes diverse families of glycosyl transferases with a common GT-A type structural fold; cl11394
GOHSU29	CDS	GOHSU_29_00190	complement(33298..34506)		UDP-galactopyranose mutase [nite]	 glf [nite]		RER_02110 :UDP-galactopyranose mutase : glf : 5.4.99.9 : Rhodococcus erythropolis PR4	UDP-galactopyranose mutase	glf	F1YMH5_9ACTO UDP-galactopyranose mutase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15299 PE=4 SV=1;UDP-galactopyranose mutase [Corynebacterium glutami cum ATCC 13032];IS fingerprint type 4-5;UDP-galactopyranose mutase [Cell envelope biogenesis, outer membrane]; COG0562;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;UDP-galactopyranose mutase; pfam03275
GOHSU29	CDS	GOHSU_29_00200	complement(34579..36288)		SpoIID/LytB domain-containing protein [migap]				SpoIID/LytB domain-containing protein		H5UFL7_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_069_00130 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rop:ROP_39290 hypothetical protein; TIGRFAM: SpoIID/LytB domain protein; PFAM: stage II sporulation D domain protein;SpoIID/LytB domain; TIGR02669;Stage II sporulation protein; pfam08486
GOHSU29	CDS	GOHSU_29_00210	complement(36507..37355)		hypothetical protein [nite]			ROP_39270 :hypothetical protein : : : Rhodococcus opacus B4	hydrolase		F1YMH7_9ACTO Cof-like hydrolase OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_15309 PE=4 SV=1;hydrolase [Streptomyces sp. C];Haloacid Dehalogenase-like Hydrolases; cl11391;haloacid dehalogenase-like hydrolase; pfam08282;motif I;motif II;Haloacid dehalogenase-like hydrolases. The haloacid dehalogenase-like (HAD) superfamily includes L-2-haloacid dehalogenase, epoxide hydrolase, phosphoserine phosphatase, phosphomannomutase, phosphoglycolate phosphatase, P-type ATPase, and many others; cd01427
GOHSU29	CDS	GOHSU_29_00220	complement(37352..38131)		putative acyltransferase [nite]			ROP_39250 :putative acyltransferase : : 2.3.1.- : Rhodococcus opacus B4	acyltransferase		F1YMH8_9ACTO Phospholipid/glycerol acyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15314 PE=4 SV=1;acyltransferase [Mycobacterium sp. JDM601];Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: AGPAT-like; cd07989;putative acyl-acceptor binding pocket
GOHSU29	CDS	GOHSU_29_00230	complement(38133..38930)		putative acyltransferase [nite]			ROP_39250 :putative acyltransferase : : 2.3.1.- : Rhodococcus opacus B4	phospholipid/glycerol acyltransferase		H0QVJ0_9ACTO Putative acyltransferase OS=Gordonia e ffusa NBRC 100432 GN=GOEFS_015_00640 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Lysophospholipid Acyltransferases (LPLATs) of Glycerophospholipid Biosynthesis: AGPAT-like; cd07989;putative acyl-acceptor binding pocket
GOHSU29	CDS	GOHSU_29_00240	complement(38995..39714)		anti-sigma-K factor RskA [nite]	 rskA [nite]		RER_19250 :anti-sigma-K factor RskA : rskA : : Rhodococcus erythropolis PR4	hypothetical protein		F1YM11_9ACTO Anti-sigma-K factor RskA OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_14776 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Anti-sigma-K factor rskA;putative anti-sigmaE protein; Provisional;Anti-sigma-K factor rskA; pfam10099
GOHSU29	CDS	GOHSU_29_00250	complement(39707..40330)		RNA polymerase ECF-type sigma factor SigK [nite]	 sigK [nite]		RER_19260 :RNA polymerase ECF-type sigma factor SigK : sigK : : Rhodococcus erythropolis PR4	sigma-70 family RNA polymerase sigma factor		F1YM12_9ACTO RNA polymerase sigma factor SigK OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14781 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: RNA polymerase sigma factor, sigma-70 family; PFAM: sigma-70 region 2 domain protein; Sigma-70 region 4 type 2; sigma-70 region 4 domain protein; KEGG: bgl:bglu_2g18860 sigma-24 (FecI-like);RNA polymerase sigma factor SigK; Reviewed;Sigma-70 region 2; pfam04542;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU29	CDS	GOHSU_29_00260	complement(40353..41627)		putative cyclopropane fatty acid synthase [nite]	 cfa [nite]		ROP_22860 :putative cyclopropane fatty acid synthase : cfa : 2.1.1.79 : Rhodococcus opacus B4	cyclopropane-fatty-acyl-phospholipid synthase		F1YM13_9ACTO Cyclopropane-fatty-acyl-phospholipid s ynthase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_147 86 PE=4 SV=1;cyclopropane-fatty-acyl-phospholipid synthase [Cate nulispora acidiphila DSM 44928];PFAM: cyclopropane-fatty-acyl-phospholipid synthase; methyltransferase type 11; methyltransferase type 12; KEGG: nca:Noca_4538 cyclopropane-fatty-acyl- phospholipid synthase;Mycolic acid cyclopropane synthetase; pfam02353;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU29	CDS	GOHSU_29_00270	complement(41624..42463)		hypothetical protein [nite]			ROP_22850 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YM14_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14791 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF1365); pfam07103;Uncharacterized conserved protein [Function unknown]
GOHSU29	CDS	GOHSU_29_00280	complement(42460..43743)		hypothetical protein [nite]			ROP_22840 :hypothetical protein : : : Rhodococcus opacus B4	putative flavin containing amine oxidoreductase		F1YM15_9ACTO Amine oxidase OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_14796 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted NAD/FAD-binding protein [General function prediction only];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU29	CDS	GOHSU_29_00290	43842..45254		deoxyribodipyrimidine photo-lyase [nite]	 phr [nite]		ROP_50630 :deoxyribodipyrimidine photo-lyase : phr : 4.1.99.3 : Rhodococcus opacus B4	deoxyribodipyrimidine photolyase		F1YM16_9ACTO Deoxyribodipyrimidine photo-lyase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_14801 PE=3 SV=1 ;deoxyribodipyrimidine photolyase [Rhodococcus equi 103S];Deoxyribodipyrimidine photolyase [DNA replication, recombination, and repair]; COG0415;DNA photolyase; pfam00875;FAD binding domain of DNA photolyase; pfam03441
GOHSU29	CDS	GOHSU_29_00300	complement(45277..45924)		hypothetical protein [nite]			ROP_22810 :hypothetical protein : : : Rhodococcus opacus B4	outer membrane lipoprotein Blc	blc	H5RX83_9NOCA Putative uncharacterized protein OS=No cardia brasiliensis ATCC 700358 GN=O3I_23380 PE=4 SV=1;outer membrane lipoprotein Blc [Bdellovibrio bacter iovorus HD100];Lipocalin / cytosolic fatty-acid binding protein family; cl01150
GOHSU29	CDS	GOHSU_29_00310	complement(46024..46695)		hypothetical protein [nite]			ROP_22820 :hypothetical protein : : : Rhodococcus opacus B4	beta-Ig-H3/fasciclin		F1YM17_9ACTO Secreted/surface protein with fascicli n-like repeats OS=Gordonia neofelifaecis NRRL B-59395 GN=S CNU_14806 PE=4 SV=1;beta-Ig-H3/fasciclin [Mycobacterium gilvum PYR- GCK ];PFAM: beta-Ig-H3/fasciclin; KEGG: mmc:Mmcs_2041 beta-Ig-H3/fasciclin;Fasciclin domain; cl02663
GOHSU29	CDS	GOHSU_29_00320	complement(46861..47295)		hypothetical protein [nite]			KSE_06490 :hypothetical protein : : : Kitasatospora setae NBRC 14216	polyketide cyclase/dehydrase		F1YM19_9ACTO Polyketide cyclase/dehydrase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_14816 PE=4 SV=1;polyketide cyclase/dehydrase [Intrasporangium calvu m DSM 43043];InterPro IPR019587; KEGG: art:Arth_3265 hypothetical protein; PFAM: polyketide cyclase/dehydrase; SPTR:  uncharacterized protein; PFAM: polyketide cyclase / dehydrase and lipid transport;Ligand-binding SRPBCC domain of an uncharacterized subfamily of proteins; cd08865;putative hydrophobic ligand binding site [chemical binding]
GOHSU29	CDS	GOHSU_29_00330	complement(47292..48242)		putative oxidoreductase [nite]			ANT_09280 :putative oxidoreductase : : : Anaerolinea thermophila UNI-1	dehydrogenase/reductase		F1YM20_9ACTO Dehydrogenase/reductase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_14821 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];Detected in the cytoplamic fraction by LCMSMS; cytoplasmic protein; function unknown, probably involved in cellular metabolism.;3-ketoacyl-(acyl-carrier-protein) reductase; Reviewed;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU29	CDS	GOHSU_29_00340	48364..49971		conserved hypothetical protein [nite]			RER_57010 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	Rieske (2Fe-2S) domain-containing protein		G7H1Y5_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_048_00620 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Metallo-beta-lactamase superfamily; cl00446;Rieske domain; a [2Fe-2S] cluster binding domain commonly found in Rieske non-heme iron oxygenase (RO) systems such as naphthalene and biphenyl dioxygenases, as well as in plant/cyanobacterial chloroplast b6f and mitochondrial cytochrome bc(1) complexes; cl00938;iron-sulfur cluster [ion binding];[2Fe-2S] cluster binding site [ion binding]
GOHSU29	CDS	GOHSU_29_00350	50025..51173		CaiB/BaiF family protein [nite]			ROP_19950 :CaiB/BaiF family protein : : : Rhodococcus opacus B4	l-carnitine dehydratase/bile acid-inducible protein f		F1YLD4_9ACTO CaiB/BaiF family protein OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_13468 PE=4 SV=1;l-carnitine dehydratase/bile acid-inducible protein f [Alicycliphilus denitrificans BC];PFAM: L-carnitine dehydratase/bile acid-inducible protein F; KEGG: dac:Daci_0010 L-carnitine dehydratase/bile acid-inducible protein F;Predicted acyl-CoA transferases/carnitine dehydratase [Energy production and conversion]; COG1804;CoA-transferase family III; pfam02515
GOHSU29	CDS	GOHSU_29_00360	complement(51296..51919)		hypothetical protein [nite]			ROP_61030 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H6RJS0_BLASD Putative uncharacterized protein OS=Bl astococcus saxobsidens (strain DD2) GN=BLASA_2699 PE=4 SV= 1;unnamed protein product [Corynebacterium diphtheria e HC04]
GOHSU29	CDS	GOHSU_29_00370	complement(51949..52776)		hypothetical protein [nite]			RER_57720 :hypothetical protein : : : Rhodococcus erythropolis PR4	beta-lactamase-like protein		G7H412_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_061_00260 PE=4 SV=1;beta-lactamase-like protein [Rubrobacter xylanophil us DSM 9941];PFAM: beta-lactamase-like; KEGG: gsu:GSU1123 metallo-beta-lactamase family protein;Metallo-beta-lactamase superfamily; cl00446
GOHSU29	CDS	GOHSU_29_00380	complement(52843..53646)		hypothetical membrane protein [nite]			ROP_39080 :hypothetical membrane protein : : : Rhodococcus opacus B4	abortive infection protein		H5TR15_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_187_00580 PE=4 SV=1;abortive infection protein [Mycobacterium gilvum PY R- GCK];PFAM: Abortive infection protein; KEGG: mva:Mvan_5664 abortive infection protein;CAAX protease self-immunity; cl00558
GOHSU29	CDS	GOHSU_29_00390	complement(53753..54451)		hypothetical protein [nite]			YM304_24680 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	Methyltransferase type 11		G4CF43_9NEIS SAM-dependent methyltransferase OS=Nei sseria shayeganii 871 GN=HMPREF9371_0232 PE=4 SV=1;Methyltransferase type 11 [Leptonema illini DSM 215 28];Methylase involved in ubiquinone/menaquinone biosynthesis [Coenzyme metabolism]; COG2226;PFAM: Methyltransferase domain; COGs: COG2226 Methylase involved in ubiquinone/menaquinone biosynthesis; InterPro IPR013216; KEGG: psa:PST_2700 SAM-dependent methyltransferase; PFAM: Methyltransferase type 11; SPTR: SAM-dependent methyltransferase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU29	CDS	GOHSU_29_00400	complement(54448..55221)		hypothetical protein [nite]			ROP_39090 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GTF9_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_60_00170 PE=4 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];KEGG: kra:Krad_1109 hypothetical protein
GOHSU29	CDS	GOHSU_29_00410	55350..56282		prephenate dehydratase [nite]	 pheA [nite]		RER_01870 :prephenate dehydratase : pheA : 4.2.1.51 : Rhodococcus erythropolis PR4	prephenate dehydratase	pheA	G7H1Z3_9ACTO Prephenate dehydratase OS=Gordonia ara ii NBRC 100433 GN=pheA PE=4 SV=1;prephenate dehydratase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of phenylpyruvate from prephenate in phenylalanine biosynthesis;prephenate dehydratase; Provisional;Prephenate dehydratase; pfam00800;C-terminal ACT domain of the bifunctional chorismate mutase-prephenate dehydratase (CM-PDT) enzyme and the prephenate dehydratase (PDT) enzyme; cd04905;putative L-Phe binding site [chemical binding]
GOHSU29	CDS	GOHSU_29_00420	56284..56928		putative 2,3-bisphosphoglycerate-dependent phosphoglycerate mutase [nite]			ROP_39120 :putative 2,3-bisphosphoglycerate-dependent phosphoglycerate mutase : : 5.4.2.1 : Rhodococcus opacus B4	phosphoglycerate mutase		F1YMI9_9ACTO Phosphoglycerate mutase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_15369 PE=4 SV=1;phosphoglycerate mutase [Corynebacterium aurimucosu m ATCC 700975];Phosphoglycerate mutase 1;Histidine phosphatase domain found in phosphoglycerate mutases and related proteins, mostly phosphatases; contains a His residue which is phosphorylated during the reaction; cd07067;catalytic core [active]
GOHSU29	CDS	GOHSU_29_00430	57036..57680		hypothetical protein [migap]				hypothetical protein		A4QDW7_CORGB Putative uncharacterized protein OS=Co rynebacterium glutamicum (strain R) GN=cgR_1428 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314]
GOHSU29	CDS	GOHSU_29_00440	complement(57714..58052)		conserved hypothetical protein [nite]			RER_01930 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7H1Z1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_048_00680 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Possibl zinc metallo-peptidase; cl01577
GOHSU29	CDS	GOHSU_29_00450	complement(58067..59176)		conserved hypothetical protein [nite]			RER_01940 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMJ1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15379 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Septum formation; pfam13845
GOHSU29	CDS	GOHSU_29_00460	59233..60489		seryl-tRNA synthetase [hamap]	serS [hamap]	MF_00176:seryl-tRNA synthetase:serS 	ROP_39200 :seryl-tRNA synthetase : serS : 6.1.1.11 : Rhodococcus opacus B4	seryl-tRNA synthetase	serS	H5U138_9ACTO Serine--tRNA ligase OS=Gordonia sputi NBRC 100414 GN=serS PE=3 SV=1;seryl-tRNA synthetase [Corynebacterium glutamicum A TCC 13032];IS fingerprint type 4-5;seryl-tRNA synthetase; Provisional;Seryl-tRNA synthetase N-terminal domain; pfam02403;catalyzes a two-step reaction, first charging a serine molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA;Seryl-tRNA synthetase (SerRS) class II core catalytic domain. SerRS is responsible for the attachment of serine to the 3' OH group of ribose of the appropriate tRNA. This domain It is primarily responsible for ATP-dependent formation of the enzyme bound...; cd00770;dimer interface [polypeptide binding];motif 1;motif 2;motif 3
GOHSU29	CDS	GOHSU_29_00470	complement(60681..61226)		putative monooxygenase [nite]			KSE_62570 :putative monooxygenase : : : Kitasatospora setae NBRC 14216	monooxygenase		I0PD85_MYCAB Monooxygenase OS=Mycobacterium abscess us M94 GN=S7W_19898 PE=4 SV=1;monooxygenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to P71662_MYCTU 2e-90;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU29	CDS	GOHSU_29_00480	complement(61356..61658)		putative flavin-containing monooxygenase [nite]			RER_34410 :putative flavin-containing monooxygenase : : : Rhodococcus erythropolis PR4	monooxygenase		I0PD85_MYCAB Monooxygenase OS=Mycobacterium abscess us M94 GN=S7W_19898 PE=4 SV=1;monooxygenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to P71662_MYCTU 2e-90;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU29	CDS	GOHSU_29_00490	complement(61655..61999)		putative TetR family transcriptional regulator [nite]			RER_54500 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		D9VEN9_9ACTO Transcriptional regulator OS=Streptomy ces sp. AA4 GN=SSMG_05841 PE=4 SV=1;TetR family transcriptional regulator [Amycolatopsi s mediterranei U32];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU29	CDS	GOHSU_29_00500	62061..62633		3-oxoacyl-[acyl-carrier protein] reductase [nite]	 fabG [nite]		BBR47_36250 :3-oxoacyl-[acyl-carrier protein] reductase : fabG : 1.1.1.100 : Brevibacillus brevis NBRC 100599	short chain dehydrogenase		Q5Y9I6_9ACTO Putative dehydrogenase/reductase OS=Ae romicrobium erythreum PE=4 SV=1;short chain dehydrogenase [Gluconacetobacter diazot rophicus PAl 5];short chain dehydrogenase; Provisional;PFAM: short-chain dehydrogenase/reductase SDR; KR domain protein; KEGG: gdi:GDI3786 short-chain dehydrogenase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU29	CDS	GOHSU_29_00510	62630..62815		short chain dehydrogenase [migap]				short chain dehydrogenase		G7GXS7_9ACTO Putative short-chain dehydrogenase/red uctase OS=Gordonia araii NBRC 100433 GN=GOARA_011_00180 PE =4 SV=1;short chain dehydrogenase [Gluconacetobacter diazot rophicus PAl 5];short chain dehydrogenase; Provisional;PFAM: short-chain dehydrogenase/reductase SDR; KR domain protein; KEGG: gdi:GDI3786 short-chain dehydrogenase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU29	CDS	GOHSU_29_00520	62998..63897		putative amino acid ABC transporter amino acid-binding protein [nite]			RER_33160 :putative amino acid ABC transporter amino acid-binding protein : : : Rhodococcus erythropolis PR4	amino acid ABC transporter		C3JQM2_RHOER ABC transporter, substrate-binding com ponent OS=Rhodococcus erythropolis SK121 GN=RHOER0001_0518 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00497;Bacterial extracellular solute-binding proteins, family 3; pfam00497;Bacterial periplasmic transport systems use membrane-bound complexes and substrate-bound, membrane-associated, periplasmic binding proteins (PBPs) to transport a wide variety of  substrates, such as, amino acids, peptides, sugars, vitamins and inorganic...; cd00134;substrate binding pocket [chemical binding];membrane-bound complex binding site;hinge residues
GOHSU29	CDS	GOHSU_29_00530	63916..64896		hypothetical protein [nite]			YM304_28150 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	dipeptidylaminopeptidase/acylaminoacyl- peptidase-like protein		H5TLJ0_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_105_00130 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: msm:MSMEG_5171 hypothetical protein;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697;substrate binding pocket [chemical binding]
GOHSU29	CDS	GOHSU_29_00540	64927..65811		hypothetical membrane protein [nite]			ROP_02040 :hypothetical membrane protein : : : Rhodococcus opacus B4			G4HYL6_MYCRH Putative uncharacterized protein (Prec ursor) OS=Mycobacterium rhodesiae JS60 GN=MycrhDRAFT_2596 PE=4 SV=1
GOHSU29	CDS	GOHSU_29_00550	complement(65826..66995)		hypothetical protein [nite]			KSE_59850 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		Q5YVB3_NOCFA Putative uncharacterized protein OS=No cardia farcinica (strain IFM 10152) GN=NFA_30310 PE=4 SV=1 ;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Catalytic NodB homology domain of the carbohydrate esterase 4 superfamily; cl15692
GOHSU29	CDS	GOHSU_29_00560	complement(67013..67282)		predicted protein [migap]				predicted protein		C6J5S1_9BACL Predicted protein OS=Paenibacillus sp. oral taxon 786 str. D14 GN=POTG_03748 PE=4 SV=1;predicted protein [Paenibacillus sp. oral taxon 786 str. D14];Domain of unknown function (DUF4190); pfam13828
GOHSU29	CDS	GOHSU_29_00570	complement(67279..68100)		hypothetical membrane protein [nite]			RER_32320 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	putative integral membrane protein		H6R921_NOCCG Uncharacterized protein OS=Nocardia cy riacigeorgica (strain GUH-2) GN=NOCYR_2827 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU29	CDS	GOHSU_29_00580	complement(68097..69029)		putative LysR family transcriptional regulator [nite]			ROP_14600 :putative LysR family transcriptional regulator : : : Rhodococcus opacus B4	LysR family transcriptional regulator		D0LF66_GORB4 Regulatory protein LysR OS=Gordonia br onchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3569 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM:  LysR family transcriptional regulator; LysR substrate- binding; KEGG: lhk:LHK_02174 probable transcriptional regulator;DNA-binding transcriptional activator XapR; Provisional;Helix-turn-helix domains; cl00088;The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cl11398;dimerization interface [polypeptide binding]
##GOHSU30	contig00151				60542
GOHSU30	CDS	GOHSU_30_00010	<1..136	short	fructose 1,6-bisphosphatase class II [nite]	 glpX [nite]		YM304_10700 :fructose 1,6-bisphosphatase class II : glpX : 3.1.3.11 : Acidimicrobidae bacterium YM16-304	fructose 1,6-bisphosphatase II	glpX	F1YFG9_9ACTO Fructose-1,6-bisphosphatase OS=Gordoni a neofelifaecis NRRL B-59395 GN=glpX PE=3 SV=1;fructose 1,6-bisphosphatase II [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;type II fructose 1,6-bisphosphatae; in Escherichia coli this protein forms a dimer and binds manganese;Bacterial fructose-1,6-bisphosphatase, glpX-encoded. A dimeric enzyme dependent on Mg(2+). glpX-encoded FPBase (FBPase class II) differs from other members of the inositol-phosphatase superfamily by permutation of secondary structure elements. The core...; cd01516
GOHSU30	CDS	GOHSU_30_00020	complement(150..839)		putative ABC transporter ATP-binding protein [nite]			KSE_66850 :putative ABC transporter ATP-binding protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		F1YFG8_9ACTO Abc transporter related protein OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_02847 PE=3 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: rpb:RPB_2235 hypothetical protein;ABC-type antimicrobial peptide transport system, ATPase component [Defense mechanisms]; COG1136;This family is comprised of MJ0796 ATP-binding cassette, macrolide-specific ABC-type efflux carrier (MacAB), and proteins involved in cell division (FtsE), and release of liporoteins from the cytoplasmic membrane (LolCDE).  They are clustered together...; cd03255;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU30	CDS	GOHSU_30_00030	complement(848..1810)		putative ABC transporter permease protein [nite]			RER_01000 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	ABC transporter permease		F1YFG7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02842 PE=4 SV=1;ABC transporter permease protein [Rhodococcus eryth ropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;FtsX-like permease family; cl15850
GOHSU30	CDS	GOHSU_30_00040	2098..2676		two-component histidine kinase [nite]			KRH_16320 :two-component histidine kinase : : 2.7.13.- : Kocuria rhizophila DC2201	integral membrane sensor signal transduction histidine kinase		F1YFG6_9ACTO Histidine kinase OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_02837 PE=4 SV=1;integral membrane sensor signal transduction histid ine kinase [Streptomyces griseus XylebKG- 1];KEGG: sgr:SGR_875 putative two-component system sensor kinase; PFAM: ATPase-like, ATP-binding domain; Signal transduction histidine kinase, subgroup 3, dimerisation and phosphoacceptor domain; SMART: ATPase-like, ATP-binding domain;Signal transduction histidine kinase [Signal transduction mechanisms];Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU30	CDS	GOHSU_30_00050	2673..3305		probable NarL family two-component response regulator [nite]			RER_00980 :probable NarL family two-component response regulator : : : Rhodococcus erythropolis PR4	response regulator receiver protein		F1YFG5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02832 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197;KEGG: xac:XAC2168 two-component system regulatory protein;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding]
GOHSU30	CDS	GOHSU_30_00060	complement(3344..4312)		hypothetical protein [nite]			KSE_61150 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		D6ZD42_SEGRD Putative uncharacterized protein OS=Se gniliparus rotundus (strain ATCC BAA-972 / CDC 1076 / CIP 108378 / DSM 44985 / JCM 13578) GN=Srot_2797 PE=4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Uncharacterized protein conserved in bacteria (DUF2252); cl12088
GOHSU30	CDS	GOHSU_30_00070	complement(4743..5753)		putative oxidoreductase [nite]			RER_19720 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short-chain dehydrogenase		G7GXZ3_9ACTO Putative oxidoreductase OS=Gordonia ar aii NBRC 100433 GN=GOARA_012_00180 PE=3 SV=1;short-chain dehydrogenase [Amycolatopsis mediterran ei U32];short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU30	CDS	GOHSU_30_00080	complement(5750..6124)		membrane protein [migap]				membrane protein		H0QWN0_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_022_00110 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU30	CDS	GOHSU_30_00090	6252..6815		not assigned						H0R3K6_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_092_00820 PE=4 SV=1
GOHSU30	CDS	GOHSU_30_00100	6829..7560		putative TetR family transcriptional regulator [nite]			ROP_51270 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	transcriptional regulatory protein		G7H1B8_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_044_00160 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00440;DNA-binding transcriptional repressor FabR; Provisional;Helix-turn-helix domains; cl00088
GOHSU30	CDS	GOHSU_30_00110	complement(7616..8098)		cyclase/dehydrase [migap]				cyclase/dehydrase		G7H1B5_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_044_00130 PE=4 SV=1;cyclase/dehydrase [Mycobacterium gilvum PYR- GCK];PFAM: cyclase/dehydrase; KEGG: mmc:Mmcs_3280 cyclase/dehydrase;Ligand-binding SRPBCC domain of an uncharacterized subfamily of proteins; cd07819;putative hydrophobic ligand binding site [chemical binding]
GOHSU30	CDS	GOHSU_30_00120	complement(8142..8825)		hypothetical protein [migap]				hypothetical protein		G7GXB6_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_005_00360 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU30	CDS	GOHSU_30_00130	complement(8846..9793)		putative Mce family protein [nite]	 mce2F [nite]		ROP_18060 :putative Mce family protein : mce2F : : Rhodococcus opacus B4	organic solvent resistance ABC transporter periplasmic protein		G7GXB5_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceF PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: mce related protein; TIGRFAM: virulence factor Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00140	complement(9790..10809)		putative Mce family protein [nite]	 mce1E [nite]		RER_06730 :putative Mce family protein : mce1E : : Rhodococcus erythropolis PR4	putative Mce family protein		G7GXB4_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceE PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00150	complement(10806..11876)		putative Mce family protein [nite]	 mce1D [nite]		ROP_16460 :putative Mce family protein : mce1D : : Rhodococcus opacus B4	putative Mce family protein		G7GXB3_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceD PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00160	complement(11877..12872)		putative Mce family protein [nite]	 mce2C [nite]		ROP_18090 :putative Mce family protein : mce2C : : Rhodococcus opacus B4	hypothetical protein		G7GXB2_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceC PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: hypothetical protein; KEGG: nfa:nfa11180  Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00170	complement(12869..13882)		putative Mce family protein [nite]	 mce2B [nite]		ROP_18100 :putative Mce family protein : mce2B : : Rhodococcus opacus B4	virulence factor Mce family protein		G7GXB1_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceB PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: mce related protein; TIGRFAM: virulence factor Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00180	complement(13879..14907)		putative Mce family protein [nite]	 mce5A [nite]		ROP_47970 :putative Mce family protein : mce5A : : Rhodococcus opacus B4	virulence factor Mce family protein;hypothetical protein		G7GXB0_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceA PE=4 SV=1;virulence factor Mce family protein [Mycobacterium gilvum PYR- GCK];Signal predicted by SignalP 2.0 HMM (Signal peptide probability1.000) with cleavage site probability 0.311 at residue 55;TIGRFAM: virulence factor Mce family protein; PFAM: Mammalian cell entry related domain protein; KEGG: mmc:Mmcs_0275 mammalian cell entry;mce related protein; pfam02470;Protein of unknown function (DUF3407); pfam11887
GOHSU30	CDS	GOHSU_30_00190	complement(14912..15781)		putative YrbE family protein [nite]	 yrbE1B [nite]		RER_06780 :putative YrbE family protein : yrbE1B : : Rhodococcus erythropolis PR4	organic solvents resistance ABC transporter permease		G7GXA9_9ACTO YrbE family protein OS=Gordonia araii NBRC 100433 GN=yrbEB PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF140; TIGRFAM: conserved hypothetical integral membrane protein;Permease; cl00510
GOHSU30	CDS	GOHSU_30_00200	complement(15786..16592)		putative YrbE family protein [nite]	 yrbE2A [nite]		ROP_18130 :putative YrbE family protein : yrbE2A : : Rhodococcus opacus B4	hypothetical protein		G7GXA8_9ACTO YrbE family protein OS=Gordonia araii NBRC 100433 GN=yrbEA PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Permease; cl00510
GOHSU30	CDS	GOHSU_30_00210	complement(16673..17602)		putative hydrolase [nite]			RER_19150 :putative hydrolase : : : Rhodococcus erythropolis PR4	alpha/beta hydrolase		G7GXA7_9ACTO Putative hydrolase OS=Gordonia araii N BRC 100433 GN=GOARA_005_00270 PE=4 SV=1;alpha/beta hydrolase [Amycolatopsis mediterranei U3 2];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697
GOHSU30	CDS	GOHSU_30_00220	complement(17715..18779)		3-ketosteroid 9alpha-hydroxylase component KshB [nite]	 kshB [nite]		RER_17750 :3-ketosteroid 9alpha-hydroxylase component KshB : kshB : : Rhodococcus erythropolis PR4	oxidoreductase, electron transfer component		H0QXF4_9ACTO 3-ketosteroid 9alpha-hydroxylase compo nent KshB OS=Gordonia effusa NBRC 100432 GN=kshB PE=4 SV=1 ;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00111; match to protein family HMM PF00175; match to protein family HMM PF00970;phenylacetate-CoA oxygenase/reductase, PaaK subunit; TIGR02160;NAD(P) binding domain of ferredoxin reductase like phenylacetic acid (PA) degradation oxidoreductase. PA oxidoreductases of E. coli hydroxylate PA-CoA in the second step of PA degradation. Members of this group typically fuse a ferredoxin reductase-like...; cd06214;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding];beta-alpha-beta structure motif;NAD(p) ribose binding residues [chemical binding];NAD binding pocket [chemical binding];NAD(P)-pyrophosphate-nicotinamide binding residues [chemical binding];2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cd00207;catalytic loop [active];iron binding site [ion binding]
GOHSU30	CDS	GOHSU_30_00230	complement(18776..19624)		putative alcohol dehydrogenase [nite]			RER_41250 :putative alcohol dehydrogenase : : 1.1.1.1 : Rhodococcus erythropolis PR4	short-chain dehydrogenase		G7GXA5_9ACTO Putative oxidoreductase OS=Gordonia ar aii NBRC 100433 GN=GOARA_005_00250 PE=3 SV=1;unnamed protein product [gamma proteobacterium HdN1 ];silverDB:HdN1F1353;short chain dehydrogenase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU30	CDS	GOHSU_30_00240	complement(19660..21207)		putative oxidoreductase [nite]			ROP_18280 :putative oxidoreductase : : : Rhodococcus opacus B4	monooxygenase		G7GXA4_9ACTO Putative oxidoreductase OS=Gordonia ar aii NBRC 100433 GN=GOARA_005_00240 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU30	CDS	GOHSU_30_00250	complement(21265..22086)		putative peptidase S33 family protein [nite]			KSE_62580 :putative peptidase S33 family protein : : : Kitasatospora setae NBRC 14216	hydrolase		G7GXA3_9ACTO Putative hydrolase OS=Gordonia araii N BRC 100433 GN=GOARA_005_00230 PE=4 SV=1;hydrolase [Rhodococcus opacus B4];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697
GOHSU30	CDS	GOHSU_30_00260	complement(22158..23174)		hypothetical protein [nite]			ROP_51420 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GXA2_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_005_00220 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rha:RHA1_ro05227 hypothetical protein;Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cl00264
GOHSU30	CDS	GOHSU_30_00270	complement(23257..24252)		hypothetical protein [nite]			KSE_04040 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		G7GX99_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_005_00190 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];Evidence 4 : Homologs of previously reported genes of unknown function;Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cl00264
GOHSU30	CDS	GOHSU_30_00280	complement(24285..25523)		hypothetical protein [nite]			ROP_06320 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GX98_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_005_00180 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];Detected in the membrane fraction by proteomics (2D- LC-MS/MS) Also Detected in the extracellular matrix and the cytoplasmic fractions by proteomics.; membrane protein; function unknown but contains potential saccharopine-like dehydrogenase domain;Uncharacterized conserved protein [Function unknown];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU30	CDS	GOHSU_30_00290	25645..26022		flavin-containing monooxygenase [nite]			ROP_52750 :flavin-containing monooxygenase : : : Rhodococcus opacus B4	flavin binding monooxygenase		H0QXG3_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_035_00460 PE=4 SV=1;flavin binding monooxygenase [Amycolatopsis mediter ranei U32]
GOHSU30	CDS	GOHSU_30_00300	complement(26127..27041)		putative oxidoreductase [nite]			RER_47430 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	retinol dehydrogenase 13		H6R3M9_NOCCG Putative short chain dehydrogenase OS= Nocardia cyriacigeorgica (strain GUH-2) GN=NOCYR_1186 PE=3 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];short chain dehydrogenase; Provisional;identified by match to protein family HMM PF00106;retinol dehydrogenase (retinol-DH), Light dependent Protochlorophyllide (Pchlide) OxidoReductase (LPOR) and related proteins, classical (c) SDRs; cd05327;putative NAD(P) binding site [chemical binding]
GOHSU30	CDS	GOHSU_30_00310	complement(27079..28923)		putative peptidase S33 family protein [nite]			KSE_63730 :putative peptidase S33 family protein : : : Kitasatospora setae NBRC 14216	short chain dehydrogenase	ephD	I0PJQ3_MYCAB Short chain dehydrogenase OS=Mycobacte rium abscessus M93 GN=OUW_04053 PE=4 SV=1;short chain dehydrogenase [Mycobacterium marinum M] ;thought to be involved in detoxification reactions following oxidative damage to lipids;short chain dehydrogenase; Validated;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU30	CDS	GOHSU_30_00320	complement(29005..30405)		hypothetical protein [nite]			YM304_12950 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	ABC transporter ATP-binding protein		H0QXF1_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_035_00340 PE=4 SV=1;ABC transporter ATP-binding protein [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q5YT28_NOCFA 2e-82;Protein Kinases, catalytic domain; cl09925;Aminoglycoside 3'-phosphotransferase (APH) and Choline Kinase (ChoK) family. The APH/ChoK family is part of a larger superfamily that includes the catalytic domains of other kinases, such as the typical serine/threonine/tyrosine protein kinases (PKs); cd05120;ATP binding site [chemical binding]
GOHSU30	CDS	GOHSU_30_00330	30673..30948		hypothetical protein [nite]			ROP_34880 :hypothetical protein : : : Rhodococcus opacus B4	metal-dependent hydrolase		I0P5E5_MYCAB Uncharacterized protein OS=Mycobacteri um abscessus M94 GN=S7W_24985 PE=4 SV=1;metal-dependent hydrolase [Burkholderia multivorans ATCC 17616];COG3687:  metal-dependent hydrolase, Pseudomonas aeruginosa; KEGG, K07044;Predicted metal-dependent hydrolase [General function prediction only];Predicted metal-dependent hydrolase; pfam10118
GOHSU30	CDS	GOHSU_30_00340	30941..32071		putative oxidoreductase [nite]			RER_00070 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	oxidoreductase		I0PJQ2_MYCAB Oxidoreductase OS=Mycobacterium absces sus M93 GN=OUW_04048 PE=4 SV=1;oxidoreductase [Rhodococcus opacus B4];Flavodoxin reductases (ferredoxin-NADPH reductases) family 1 [Energy production and conversion]; COG1018;Phthalate dioxygenase reductase (PDR) is an FMN-dependent reductase that mediates electron transfer from NADH to FMN to an iron sulfur cluster. PDR has an an N-terminal  ferrredoxin reductase (FNR)-like NAD(H) binding domain and a C-terminal iron-sulfur; cd06185;FMN-binding pocket [chemical binding];flavin binding motif;phosphate binding motif [ion binding];beta-alpha-beta structure motif;NAD binding pocket [chemical binding];2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cd00207;catalytic loop [active];iron binding site [ion binding]
GOHSU30	CDS	GOHSU_30_00350	32109..33680		putative flavin-containing monooxygenase [nite]			RER_34410 :putative flavin-containing monooxygenase : : : Rhodococcus erythropolis PR4	monooxygenase		E5XPV8_9ACTO Putative uncharacterized protein OS=Se gniliparus rugosus ATCC BAA-974 GN=HMPREF9336_01530 PE=4 S V=1;monooxygenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to P71662_MYCTU e-135;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU30	CDS	GOHSU_30_00360	complement(33704..34177)		hypothetical protein [migap]				hypothetical protein		E2SFB6_9ACTO Putative uncharacterized protein OS=Ae romicrobium marinum DSM 15272 GN=HMPREF0063_12725 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mlo:mll2787 hypothetical protein;Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109
GOHSU30	CDS	GOHSU_30_00370	complement(34191..34469)		hypothetical protein [migap]				hypothetical protein		D5UPN5_TSUPD Putative uncharacterized protein OS=Ts ukamurella paurometabola (strain ATCC 8368 / DSM 20162 / J CM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_2181 PE=4 SV=1 ;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mab:MAB_1108c hypothetical protein
GOHSU30	CDS	GOHSU_30_00380	complement(34497..34925)		putative oligoketide cyclase [nite]			SLG_18020 :putative oligoketide cyclase : : : Sphingobium sp. SYK-6	hypothetical protein		G7H3P1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_056_02140 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Pyrabactin resistance 1 (PYR1), PYR1-like (PYL), regulatory component of abscisic acid receptors (RCARs), and related proteins; cd07821;putative hydrophobic ligand binding site [chemical binding];protein interface [polypeptide binding];gate
GOHSU30	CDS	GOHSU_30_00390	complement(35036..38029)		transmembrane transporter mmpL10 [migap]				transmembrane transporter mmpL10		G7H1B6_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_044_00140 PE=4 SV=1;unnamed protein product [Mycobacterium tuberculosis KZN 4207];drug-susceptible;Predicted drug exporters of the RND superfamily [General function prediction only]
GOHSU30	CDS	GOHSU_30_00400	complement(38033..38419)		radical SAM domain-containing protein [migap]				radical SAM domain-containing protein		H0QXE9_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_035_00320 PE=4 SV=1;unnamed protein product [Candidatus Solibacter usit atus Ellin6076];PFAM: Radical SAM domain protein; SMART: Elongator protein 3/MiaB/NifB; KEGG: ade:Adeh_2438 radical SAM;Elongator protein 3, MiaB family, Radical SAM; smart00729;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site
GOHSU30	CDS	GOHSU_30_00410	38688..40262		putative flavin-containing monooxygenase [nite]			RER_34410 :putative flavin-containing monooxygenase : : : Rhodococcus erythropolis PR4	monooxygenase		G7GXZ4_9ACTO Putative flavin-containing monooxygena se OS=Gordonia araii NBRC 100433 GN=GOARA_012_00190 PE=4 S V=1;monooxygenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Similar to Q27G14_MYCFV 3e-91
GOHSU30	CDS	GOHSU_30_00420	40262..40852		hypothetical protein [nite]			RER_46650 :hypothetical protein : : : Rhodococcus erythropolis PR4	polyketide cyclase/dehydrase		G7GXZ5_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_012_00200 PE=4 SV=1;polyketide cyclase/dehydrase [Segniliparus rotundus DSM 44985];type strain of Segniliparus rotundus;PFAM: polyketide cyclase/dehydrase; KEGG: mab:MAB_4041c hypothetical protein;Pyrabactin resistance 1 (PYR1), PYR1-like (PYL), regulatory component of abscisic acid receptors (RCARs), and related proteins; cd07821;putative hydrophobic ligand binding site [chemical binding];protein interface [polypeptide binding];gate
GOHSU30	CDS	GOHSU_30_00430	40863..41192		hypothetical protein [nite]			SLG_04940 :hypothetical protein : : : Sphingobium sp. SYK-6	hypothetical protein		D9VEN7_9ACTO Putative uncharacterized protein OS=St reptomyces sp. AA4 GN=SSMG_05839 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mlo:mll2787 hypothetical protein;Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109
GOHSU30	CDS	GOHSU_30_00440	41308..42225		putative transposase [nite]			KSE_22300 :putative transposase : : : Kitasatospora setae NBRC 14216	transposase		E4NA21_KITSK Putative transposase OS=Kitasatospora setae (strain ATCC 33774 / DSM 43861 / JCM 3304 / KCC A-03 04 / NBRC 14216 / KM-6054) GN=KSE_22300 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];cluster 0025 8: SCAB3371, SCAB4481, SCAB7601, SCAB11701, SCAB20281, SCAB40511, SCAB68181;Transposase domain (DUF772); cl15789
GOHSU30	CDS	GOHSU_30_00450	complement(42243..42881)		putative TetR family transcriptional regulator [nite]			RER_34420 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		H0ITP5_MYCAB TetR family transcriptional regulator OS=Mycobacterium abscessus subsp. bolletii BD GN=MBOL_3928 0 PE=4 SV=1;TetR family transcriptional regulator [Frankia sp. EuI1c];PFAM: regulatory protein TetR; KEGG: sna:Snas_2904 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU30	CDS	GOHSU_30_00460	complement(42871..43596)		TetR family transcriptional regulator [migap]				TetR family transcriptional regulator		G7GXZ7_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_012_00220 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q277X7_MYCFV 4e-11;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU30	CDS	GOHSU_30_00470	complement(43670..45256)		putative flavin-containing monooxygenase [nite]			RER_34410 :putative flavin-containing monooxygenase : : : Rhodococcus erythropolis PR4	monooxygenase		G7GXZ9_9ACTO Putative flavin-containing monooxygena se OS=Gordonia araii NBRC 100433 GN=GOARA_012_00240 PE=4 S V=1;monooxygenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q5Z112_NOCFA e-108;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072
GOHSU30	CDS	GOHSU_30_00480	complement(45273..46148)		putative oxidoreductase [nite]			RER_55960 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		H0QWN6_9ACTO Putative oxidoreductase OS=Gordonia ef fusa NBRC 100432 GN=GOEFS_022_00170 PE=3 SV=1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU30	CDS	GOHSU_30_00490	complement(46145..47686)		putative flavin-containing monooxygenase [nite]			RER_34410 :putative flavin-containing monooxygenase : : : Rhodococcus erythropolis PR4	monooxygenase		G7GY01_9ACTO Putative flavin-containing monooxygena se OS=Gordonia araii NBRC 100433 GN=GOARA_012_00260 PE=4 S V=1;monooxygenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Similar to Q27G14_MYCFV 3e-91
GOHSU30	CDS	GOHSU_30_00500	complement(47748..48224)		hypothetical protein [migap]				hypothetical protein		H0QWN0_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_022_00110 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Predicted membrane protein [Function unknown]
GOHSU30	CDS	GOHSU_30_00510	complement(48267..48995)		hypothetical protein [migap]				hypothetical protein		H0QWN8_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_022_00190 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU30	CDS	GOHSU_30_00520	complement(49105..50058)		putative Mce family protein [nite]	 mce1D [nite]		ROP_16460 :putative Mce family protein : mce1D : : Rhodococcus opacus B4	virulence factor mce family protein		G7GY03_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceF PE=4 SV=1;virulence factor mce family protein, partial [Strep tomyces griseoflavus Tu4000];overlaps another CDS with the same product name;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00530	complement(50060..51013)		putative Mce family protein [nite]	 mce5E [nite]		ROP_48010 :putative Mce family protein : mce5E : : Rhodococcus opacus B4	Mce family protein	mce3E	G7GY04_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceE PE=4 SV=1;Mce family protein [Nocardia farcinica IFM 10152];virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00540	complement(51067..52191)		putative Mce family protein [nite]	 mce2D [nite]		ROP_18080 :putative Mce family protein : mce2D : : Rhodococcus opacus B4	virulence factor Mce family protein		G7GY05_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceD PE=4 SV=1;virulence factor Mce family protein [Saccharomonosp ora viridis DSM 43017];type strain of Saccharomonospora viridis DSM 43017;PFAM: mce related protein; TIGRFAM: virulence factor Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00550	complement(52195..53232)		putative Mce family protein [nite]	 mce2C [nite]		ROP_18090 :putative Mce family protein : mce2C : : Rhodococcus opacus B4	MCE-family protein Mce6C	mce6C	G7GY06_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceC PE=4 SV=1;MCE-family protein Mce6C [Mycobacterium sp. JDM601] ;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00560	complement(53229..54233)		putative Mce family protein [nite]	 mce2B [nite]		ROP_18100 :putative Mce family protein : mce2B : : Rhodococcus opacus B4	Mce family protein	mce2B	G7GY07_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceB PE=4 SV=1;Mce family protein [Nocardia farcinica IFM 10152];virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00570	complement(54230..55282)		putative Mce family protein [nite]	 mce2A [nite]		ROP_18110 :putative Mce family protein : mce2A : : Rhodococcus opacus B4	organic solvent resistance ABC transporter periplasmic protein		G7GY08_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceA PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: mce related protein;ABC-type transport system involved in resistance to organic solvents, periplasmic component [Secondary metabolites biosynthesis, transport, and catabolism]; COG1463;mce related protein; pfam02470
GOHSU30	CDS	GOHSU_30_00580	complement(55287..56153)		putative YrbE family protein [nite]	 yrbE4B [nite]		RER_17240 :putative YrbE family protein : yrbE4B : : Rhodococcus erythropolis PR4	YrbE family protein	yrbE2B	G7GY09_9ACTO YrbE family protein OS=Gordonia araii NBRC 100433 GN=yrbEB PE=4 SV=1;YrbE family protein [Nocardia farcinica IFM 10152];Permease; cl00510
GOHSU30	CDS	GOHSU_30_00590	complement(56153..56980)		putative YrbE family protein [nite]	 yrbE2A [nite]		ROP_18130 :putative YrbE family protein : yrbE2A : : Rhodococcus opacus B4	YrbE family protein	yrbE3A	G7GY10_9ACTO YrbE family protein OS=Gordonia araii NBRC 100433 GN=yrbEA PE=4 SV=1;YrbE family protein [Nocardia farcinica IFM 10152];Permease; cl00510
GOHSU30	CDS	GOHSU_30_00600	complement(56977..57954)		carboxylesterase [nite]			ROP_13060 :carboxylesterase : : 3.1.1.1 : Rhodococcus opacus B4	carboxylesterase		H0QWP8_9ACTO Putative carboxylesterase OS=Gordonia effusa NBRC 100432 GN=GOEFS_022_00290 PE=4 SV=1;carboxylesterase [Rhodococcus opacus B4];alpha/beta hydrolase fold; pfam07859;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU30	CDS	GOHSU_30_00610	complement(58065..59477)		hypothetical protein [nite]			YM304_12950 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	ATP-binding protein		G7GY12_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_012_00370 PE=4 SV=1;ATP-binding protein [Nocardia farcinica IFM 10152];Protein Kinases, catalytic domain; cl09925
GOHSU30	CDS	GOHSU_30_00620	complement(59474..60151)		not assigned						H5RTJ2_9NOCA Putative uncharacterized protein OS=No cardia brasiliensis ATCC 700358 GN=O3I_16903 PE=4 SV=1
GOHSU30	CDS	GOHSU_30_00630	60363..>60542	codon	fructose 1,6-bisphosphatase class II [nite]	 fbp [nite]		ROP_59270 :fructose 1,6-bisphosphatase class II : fbp : 3.1.3.11 : Rhodococcus opacus B4	fructose 1,6-bisphosphatase II	glpX	F1YFG9_9ACTO Fructose-1,6-bisphosphatase OS=Gordoni a neofelifaecis NRRL B-59395 GN=glpX PE=3 SV=1;fructose 1,6-bisphosphatase II [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;type II fructose 1,6-bisphosphatae; in Escherichia coli this protein forms a dimer and binds manganese;Bacterial fructose-1,6-bisphosphatase, glpX-encoded. A dimeric enzyme dependent on Mg(2+). glpX-encoded FPBase (FBPase class II) differs from other members of the inositol-phosphatase superfamily by permutation of secondary structure elements. The core...; cd01516;putative active site [active]
##GOHSU31	contig00161				41227
GOHSU31	CDS	GOHSU_31_00010	<1..1426	3x codon frame	not assigned						I0PVS4_MYCAB DNA polymerase III subunit gamma/tau ( Fragment) OS=Mycobacterium abscessus M94 GN=S7W_04802 PE=4 SV=1
GOHSU31	CDS	GOHSU_31_00020	complement(1489..2757)		putative cyclopropane fatty acid synthase [nite]	 cfa [nite]		ROP_41340 :putative cyclopropane fatty acid synthase : cfa : 2.1.1.79 : Rhodococcus opacus B4	cyclopropane-fatty-acyl-phospholipid synthase		G7GMA4_9ACTO Cyclopropane fatty acid synthase OS=Go rdonia amarae NBRC 15530 GN=cfa PE=4 SV=1;cyclopropane-fatty-acyl-phospholipid synthase [Myco bacterium sp. JDM601];Mycolic acid cyclopropane synthetase; pfam02353;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU31	CDS	GOHSU_31_00030	complement(2754..4172)		hypothetical protein [nite]			ROP_41350 :hypothetical protein : : : Rhodococcus opacus B4	FAD/FMN-containing dehydrogenase		F1YKB5_9ACTO FAD linked oxidase domain-containing p rotein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1135 0 PE=4 SV=1;FAD/FMN-containing dehydrogenase [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;FAD/FMN-containing dehydrogenases [Energy production and conversion]; COG0277;FAD binding domain; pfam01565
GOHSU31	CDS	GOHSU_31_00040	4288..4725		hypothetical protein [nite]			ROP_41360 :hypothetical protein : : : Rhodococcus opacus B4	polyketide cyclase/dehydrase		F1YKB4_9ACTO Polyketide cyclase/dehydrase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_11345 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: polyketide cyclase/dehydrase;START/RHO_alpha_C/PITP/Bet_v1/CoxG/CalC (SRPBCC) ligand-binding domain superfamily; cl14643
GOHSU31	CDS	GOHSU_31_00050	4788..5684		putative pseudouridine synthase [nite]			RER_47620 :putative pseudouridine synthase : : : Rhodococcus erythropolis PR4	tRNA-pseudouridine synthase i		H5UG71_9ACTO Pseudouridine synthase OS=Gordonia ter rae NBRC 100016 GN=GOTRE_071_00560 PE=3 SV=1;tRNA-pseudouridine synthase i [Rhodococcus equi 103 S];Pseudouridylate synthases, 23S RNA-specific [Translation, ribosomal structure and biogenesis]; COG0564;PSRA_1: Pseudouridine synthase, a subgroup of the RluA family. This group is comprised of bacterial proteins assigned to the RluA family of pseudouridine synthases. Pseudouridine synthases catalyze the isomerization of specific uridines in an RNA...; cd02558;probable active site [active]
GOHSU31	CDS	GOHSU_31_00060	5703..6050		hypothetical protein [migap]				hypothetical protein		F9VW44_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_060_00660 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU31	CDS	GOHSU_31_00070	6079..6552		hypothetical protein [migap]				hypothetical protein		H0R1F9_9ACTO UPF0133 protein GOEFS_069_00360 OS=Gor donia effusa NBRC 100432 GN=GOEFS_069_00360 PE=3 SV=1;unnamed protein product [Corynebacterium diphtheria e HC02];YbaB/EbfC DNA-binding family; cl00494
GOHSU31	CDS	GOHSU_31_00080	6606..6956	case	UPF0133 protein ybaB [hamap]	ybaB [hamap]	MF_00274:UPF0133 protein ybaB:ybaB 	KSE_36670 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		H5TK62_9ACTO UPF0133 protein GOOTI_086_00070 OS=Gor donia otitidis NBRC 100426 GN=GOOTI_086_00070 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];YbaB/EbfC DNA-binding family; cl00494
GOHSU31	CDS	GOHSU_31_00090	6996..7607		recombination protein RecR [hamap]	recR [hamap]	MF_00017:recombination protein RecR:recR 	ROP_41380 :recombination protein RecR : recR : : Rhodococcus opacus B4	recombination protein RecR		H5U5D2_9ACTO Recombination protein RecR OS=Gordonia sputi NBRC 100414 GN=recR PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];recombination protein RecR; Reviewed; PRK00076;KEGG: afw:Anae109_3760 recombination protein RecR; TIGRFAM: recombination protein RecR; PFAM: TOPRIM domain protein; Zinc finger C4-type, RecR; SMART: Toprim sub domain protein;RecR protein; pfam02132;TOPRIM_recR: topoisomerase-primase (TOPRIM) nucleotidyl transferase/hydrolase domain of the type found in Escherichia coli RecR.  RecR participates in the RecFOR pathway of homologous recombinational repair in prokaryotes. This pathway provides a...; cd01025;putative active site [active];putative metal-binding site [ion binding];tetramer interface [polypeptide binding]
GOHSU31	CDS	GOHSU_31_00100	7634..8215		hypothetical protein [nite]			KSE_11510 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		H0R1F7_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_069_00340 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];P-loop containing Nucleoside Triphosphate Hydrolases; cl09099
GOHSU31	CDS	GOHSU_31_00110	8305..8718		hypothetical protein [nite]			YM304_40350 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	Rhodanese domain-containing protein		A4AHV1_9ACTN Phage shock protein OS=marine actinoba cterium PHSC20C1 GN=A20C1_01926 PE=4 SV=1;unnamed protein product [Brachyspira murdochii DSM 12563];type strain of Brachyspira murdochii;KEGG: bhy:BHWA1_01020 rhodanese-like domain protein; PFAM: Rhodanese domain protein; SMART: Rhodanese domain protein;Rhodanese Homology Domain (RHOD); an alpha beta fold domain found duplicated in the rhodanese protein. The cysteine containing enzymatically active version of the domain is also found in the Cdc25 class of protein phosphatases and a variety of proteins...; cd00158;active site residue [active]
GOHSU31	CDS	GOHSU_31_00120	complement(8734..9447)		conserved hypothetical protein [nite]			RER_04060 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	CobB/CobQ domain-containing protein glutamine amidotransferase		D0LDF5_GORB4 CobB/CobQ domain protein glutamine ami dotransferase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_0441 PE=4 SV=1 ;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: CobB/CobQ domain protein glutamine amidotransferase; KEGG: gem:GM21_3605 cobyric acid synthase CobQ;Type 1 glutamine amidotransferase (GATase1) domain found in Cobyric Acid Synthase (CobQ); cd01750;catalytic triad [active]
GOHSU31	CDS	GOHSU_31_00130	complement(9444..10631)		conserved hypothetical protein [nite]			RER_04070 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	UDP-N-acetylmuramyl tripeptide synthase-like protein		F9VW51_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_060_00730 PE=4 SV= 1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: aeh:Mlg_2195 UDP-N-acetylmuramoylalanine--D- glutamate ligase;Mur ligase middle domain; pfam08245;Domain of unknown function (DUF1727); pfam08353
GOHSU31	CDS	GOHSU_31_00140	10801..11847		probable esterase [nite]			RER_56940 :probable esterase : : : Rhodococcus erythropolis PR4	alpha/beta hydrolase domain-containing protein		F1YKA2_9ACTO Alpha/beta hydrolase fold-3 domain-con taining protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_11285 PE=4 SV=1;alpha/beta hydrolase domain-containing protein [Myc obacterium vanbaalenii PYR- 1];PFAM: Alpha/beta hydrolase fold-3 domain protein; KEGG: mmc:Mmcs_1940 alpha/beta hydrolase fold-3;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;alpha/beta hydrolase fold; pfam07859;substrate binding pocket [chemical binding]
GOHSU31	CDS	GOHSU_31_00150	11990..13999		putative BCCT family transporter [nite]			RER_05440 :putative BCCT family transporter : : : Rhodococcus erythropolis PR4	glycine betaine transport integral membrane protein BetP		E1VZ71_ARTAR Glycine betaine transporter OS=Arthrob acter arilaitensis (strain DSM 16368 / CIP 108037 / JCM 13 566 / Re117) GN=opuD PE=4 SV=1;glycine betaine transport integral membrane protein BetP [Amycolicicoccus subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;BCCT family transporter; cl00569
GOHSU31	CDS	GOHSU_31_00160	complement(14095..14394)		not assigned						
GOHSU31	CDS	GOHSU_31_00170	complement(14342..16195)		2-isopropylmalate synthase [hamap]	leuA [hamap]	MF_00572:2-isopropylmalate synthase:leuA 	ROP_41510 :2-isopropylmalate synthase : leuA : 2.3.3.13 : Rhodococcus opacus B4	2-isopropylmalate synthase	leuA	F1YK21_9ACTO 2-isopropylmalate synthase OS=Gordonia neofelifaecis NRRL B-59395 GN=leuA PE=3 SV=1;2-isopropylmalate synthase [Corynebacterium glutami cum ATCC 13032];IS fingerprint type 4-5;2-isopropylmalate synthase; Validated;Mycobacterium tuberculosis LeuA3 and related proteins, N-terminal catalytic TIM barrel domain; cd07942;catalytic residues [active];metal binding site [ion binding];LeuA allosteric (dimerisation) domain; cl07212
GOHSU31	CDS	GOHSU_31_00180	complement(16375..16965)		putative nitroreductase [nite]			SLG_12320 :putative nitroreductase : : : Sphingobium sp. SYK-6	nitroreductase		H5U7K9_9ACTO Putative nitroreductase OS=Gordonia sp uti NBRC 100414 GN=GOSPT_152_00160 PE=4 SV=1;nitroreductase [Thermovirga lienii DSM 17291];PFAM: Nitroreductase; COGs: COG0778 Nitroreductase; InterPro IPR000415; KEGG: hoh:Hoch_2261 nitroreductase; PFAM: nitroreductase; SPTR: Nitroreductase;Nitroreductase-like family 2. A subfamily of the nitroreductase family containing uncharacterized proteins that are similar to nitroreductase. Nitroreductase catalyzes the reduction of nitroaromatic compounds such as nitrotoluenes, nitrofurans and...; cd02138;putative FMN binding site [chemical binding]
GOHSU31	CDS	GOHSU_31_00190	complement(16991..18529)		putative drug resistance efflux protein [nite]			RER_35250 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	major facilitator superfamily protein		H5UGA2_9ACTO Putative drug resistance transporter O S=Gordonia terrae NBRC 100016 GN=GOTRE_071_00870 PE=4 SV=1 ;major facilitator superfamily protein [Stackebrandt ia nassauensis DSM 44728];PFAM: major facilitator superfamily MFS_1; KEGG: pol:Bpro_1369 major facilitator superfamily MFS_1;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;methyl viologen resistance protein SmvA; Provisional
GOHSU31	CDS	GOHSU_31_00200	18561..19826		aspartokinase [nite]	 ask [nite]		ROP_42050 :aspartokinase : ask : 2.7.2.4 : Rhodococcus opacus B4	aspartate kinase	lysC	F1YK17_9ACTO Aspartokinase OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_11236 PE=3 SV=1;aspartate kinase [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;aspartate kinase; Reviewed;catalyzes the formation of 4-phospho-L-aspartate from L-aspartate and ATP, in Bacillus, lysine sensitive; regulated by response to starvation.;AAK_AKii-LysC-BS: Amino Acid Kinase Superfamily (AAK), AKii; this CD includes the N-terminal catalytic aspartokinase (AK) domain of the lysine-sensitive aspartokinase isoenzyme AKII of Bacillus subtilis 168, and the lysine plus threonine-sensitive...; cd04261;putative nucleotide binding site [chemical binding];putative catalytic residues [active];putative Mg ion binding site [ion binding];putative aspartate binding site [chemical binding];ACT domains of the lysine-sensitive aspartokinase isoenzyme AKII of Bacillus subtilis (BS) strain 168 and related proteins; cd04913;putative allosteric regulatory site;ACT domains of the lysine-sensitive, aspartokinase (AK) isoenzyme AKII of Bacillus subtilis (BS) strain 168 and related domains; cd04936;putative allosteric regulatory residue
GOHSU31	CDS	GOHSU_31_00210	19829..20851		aspartate-semialdehyde dehydrogenase [nite]	 asd [nite]		ROP_42060 :aspartate-semialdehyde dehydrogenase : asd : 1.2.1.11 : Rhodococcus opacus B4	aspartate-semialdehyde dehydrogenase	asd	F1YK16_9ACTO Aspartate-semialdehyde dehydrogenase O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11231 PE=3 S V=1;aspartate-semialdehyde dehydrogenase [Corynebacteri um glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 4-aspartyl phosphate from aspartate 4-semialdehyde;aspartate-semialdehyde dehydrogenase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Glyceraldehyde 3-phosphate dehydrogenase, C-terminal domain; cl15856
GOHSU31	CDS	GOHSU_31_00220	20965..22236		putative lipase [nite]			RER_34630 :putative lipase : : 3.1.1.- : Rhodococcus erythropolis PR4	triacylglycerol lipase		F1YK15_9ACTO Secretory lipase OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_11226 PE=4 SV=1;triacylglycerol lipase [Amycolicicoccus subflavus D QS3- 9A1];type strain of Amycolicicoccus subflavus;Secretory lipase; pfam03583
GOHSU31	CDS	GOHSU_31_00230	22409..23071		hypothetical protein [migap]				hypothetical protein		F1YK13_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11216 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU31	CDS	GOHSU_31_00240	23155..23586		putative organic hydroperoxide resistance protein [nite]	 ohr [nite]		ROP_49730 :putative organic hydroperoxide resistance protein : ohr : : Rhodococcus opacus B4	peroxiredoxin, OsmC family protein		D0LDI2_GORB4 Peroxiredoxin, Ohr subfamily OS=Gordon ia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_0469 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];OsmC-like protein; cl00767
GOHSU31	CDS	GOHSU_31_00250	23636..24106		hypothetical protein [migap]				hypothetical protein		F1YK11_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11206 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.768) with cleavage site probability 0.655 at residue 20
GOHSU31	CDS	GOHSU_31_00260	24130..25506		conserved hypothetical protein [nite]			RER_29760 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YK10_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11201 PE=4 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Uncharacterized protein conserved in bacteria (DUF2252); cl12088
GOHSU31	CDS	GOHSU_31_00270	25750..26079		not assigned						H1DL39_9PORP Putative uncharacterized protein OS=Od oribacter laneus YIT 12061 GN=HMPREF9449_02975 PE=4 SV=1
GOHSU31	tRNA	GOHSU_31_t0010	complement(26316..26389)		not assigned						identified by tRNAscan-SE; putative
GOHSU31	CDS	GOHSU_31_00280	complement(26477..27472)		hypothetical protein [nite]			ROP_42220 :hypothetical protein : : : Rhodococcus opacus B4	phosphohydrolase		F1YK08_9ACTO Metallophosphoesterase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_11191 PE=4 SV=1;phosphohydrolase [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;Bacillus subtilis YkuE and related proteins, C-terminal metallophosphatase domain; cd07385;Calcineurin-like phosphoesterase; pfam00149;putative active site [active];putative metal binding site [ion binding]
GOHSU31	CDS	GOHSU_31_00290	complement(27492..29984)		penicillin-binding protein 1 [nite]	 pon1 [nite]		RER_04630 :penicillin-binding protein 1 : pon1 : : Rhodococcus erythropolis PR4	peptidoglycan glycosyltransferase		F1YK07_9ACTO Peptidoglycan glycosyltransferase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_11186 PE=4 SV=1 ;peptidoglycan glycosyltransferase [Pseudonocardia d ioxanivorans CB1190];PFAM: Glycosyl transferase, family 51; Penicillin-binding protein, transpeptidase; PASTA; KEGG: sen:SACE_0314 penicillin binding protein (class A); SMART: PASTA;Membrane carboxypeptidase (penicillin-binding protein) [Cell envelope biogenesis, outer membrane]; COG0744;Transglycosylase; cl07896;Penicillin binding protein transpeptidase domain; cl01009;PASTA domain of bacterial serine/threonine kinase pknB-like proteins. PknB is a member of a group of related transmembrane sensor kinases present in many gram positive bacteria, which has been shown to regulate cell shape in Mycobacterium tubercolosis; cd06577
GOHSU31	CDS	GOHSU_31_00300	30237..30572		WhiB family regulatory protein [nite]			ROP_42250 :WhiB family regulatory protein : : : Rhodococcus opacus B4	WhiB family transcriptional regulator		G7H317_9ACTO WhiB family regulatory protein OS=Gord onia araii NBRC 100433 GN=GOARA_055_00020 PE=4 SV=1;WhiB family transcriptional regulator [Rhodococcus jostii RHA1];Transcription factor WhiB; pfam02467
GOHSU31	CDS	GOHSU_31_00310	30642..31166		hypothetical protein [migap]				hypothetical protein		C7NIR2_KYTSD Putative uncharacterized protein OS=Ky tococcus sedentarius (strain ATCC 14392 / DSM 20547 / CCM 314 / 541) GN=Ksed_16850 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG0469 Pyruvate kinase
GOHSU31	CDS	GOHSU_31_00320	complement(31217..32314)		putative anion-transporting ATPase [nite]			ROP_42260 :putative anion-transporting ATPase : : 3.6.3.- : Rhodococcus opacus B4	putative anion-transporting ATPase		F1YK05_9ACTO Anion-transporting ATPase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11176 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted ATPase involved in chromosome partitioning [Cell division and chromosome partitioning]; COG0003;ArsA ATPase functionas as an efflux pump located on the inner membrane of the cell. This ATP-driven oxyanion pump catalyzes the extrusion of arsenite, antimonite and arsenate. Maintenance of a low intracellular concentration of oxyanion produces...; cd02035;DTAP/Switch II;Switch I
GOHSU31	CDS	GOHSU_31_00330	complement(32314..33330)		putative anion-transporting ATPase [nite]			ROP_42270 :putative anion-transporting ATPase : : 3.6.3.- : Rhodococcus opacus B4	anion-transporting ATPase		D0LEA9_GORB4 Anion-transporting ATPase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NC TC 10667) GN=Gbro_0497 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: Anion-transporting ATPase; KEGG: scl:sce8296 anion-transporting ATPase;Predicted ATPase involved in chromosome partitioning [Cell division and chromosome partitioning]; COG0003;ArsA ATPase functionas as an efflux pump located on the inner membrane of the cell. This ATP-driven oxyanion pump catalyzes the extrusion of arsenite, antimonite and arsenate. Maintenance of a low intracellular concentration of oxyanion produces...; cd02035;DTAP/Switch II;Switch I
GOHSU31	CDS	GOHSU_31_00340	33414..33578		hypothetical protein [nite]			ROP_42280 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		D0LEB0_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_0498 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Domain of unknown function (DUF4177); pfam13783
GOHSU31	CDS	GOHSU_31_00350	33581..34081		hypothetical protein [nite]			ROP_42290 :hypothetical protein : : : Rhodococcus opacus B4	endoribonuclease L-PSP		H5UHK5_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_121_00090 PE=4 SV=1;endoribonuclease L-PSP [Heliobacterium modesticaldu m Ice1];type strain of Heliobacterium modesticaldum;This group of proteins belong to a large family of YjgF/YER057c/UK114-like proteins present in bacteria, archaea, and eukaryotes with no definitive function.  The conserved domain is similar in structure to chorismate mutase but there is no sequence...; cd02199;homotrimer interaction site [polypeptide binding];putative active site [active]
GOHSU31	CDS	GOHSU_31_00360	34078..34893		putative beta-lactamase [nite]			RER_04690 :putative beta-lactamase : : 3.5.2.6 : Rhodococcus erythropolis PR4	beta-lactamase		F1YK00_9ACTO Beta-lactamase domain-containing prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11151 PE =4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: beta-lactamase; KEGG: pla:Plav_0801 beta-lactamase domain- containing protein;Metallo-beta-lactamase superfamily; cl00446
GOHSU31	CDS	GOHSU_31_00370	complement(35022..35696)		CRP/FNR family transcriptional regulator [nite]			RER_04800 :CRP/FNR family transcriptional regulator : : : Rhodococcus erythropolis PR4	Crp/Fnr family transcriptional regulator protein		F1YJZ9_9ACTO Cyclic nucleotide-binding protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_11146 PE=4 SV=1 ;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00027; match to protein family HMM PF00325;cAMP-binding proteins - catabolite gene activator and regulatory subunit of cAMP-dependent protein kinases [Signal transduction mechanisms]; COG0664;effector domain of the CAP family of transcription factors; members include CAP (or cAMP receptor protein (CRP)), which binds cAMP, FNR (fumarate and nitrate reduction), which uses an iron-sulfur cluster to sense oxygen) and CooA, a heme containing CO...; cd00038;ligand binding site [chemical binding];flexible hinge region;Helix-turn-helix domains; cl00088
GOHSU31	CDS	GOHSU_31_00380	complement(35817..36050)		hypothetical protein [nite]			ROP_42320 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YJZ8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11141 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.984) with cleavage site probability 0.966 at residue 33
GOHSU31	CDS	GOHSU_31_00390	36210..36911		endonuclease III [nite]	 nth [nite]		RER_04820 :endonuclease III : nth : 4.2.99.18 : Rhodococcus erythropolis PR4	endonuclease III		G7GN06_9ACTO Endonuclease III OS=Gordonia amarae NB RC 15530 GN=nth PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: sfu:Sfum_1062 endonuclease III; TIGRFAM: endonuclease III; PFAM: HhH-GPD family protein; iron-sulfur cluster loop; SMART: HhH-GPD family protein; iron-sulfur cluster loop;Predicted EndoIII-related endonuclease [DNA replication, recombination, and repair]; COG0177;endonuclease III; includes endonuclease III (DNA-(apurinic or apyrimidinic site) lyase), alkylbase DNA glycosidases (Alka-family) and other DNA glycosidases; cd00056;minor groove reading motif;helix-hairpin-helix signature motif;substrate binding pocket [chemical binding];Iron-sulfur binding domain of endonuclease III; cl15304
GOHSU31	CDS	GOHSU_31_00400	36908..37591		putative thiol-disulfide oxidoreductase [nite]			RER_04830 :putative thiol-disulfide oxidoreductase : : 1.8.-.- : Rhodococcus erythropolis PR4	alkyl hydroperoxide reductase/ Thiol specific antioxidant/ Mal allergen		F1YJZ6_9ACTO Alkyl hydroperoxide reductase/ Thiol s pecific antioxidant/ Mal allergen OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_11131 PE=4 SV=1;alkyl hydroperoxide reductase/ Thiol specific antio xidant/ Mal allergen [Mycobacterium sp. JLS];PFAM: alkyl hydroperoxide reductase/ Thiol specific antioxidant/ Mal allergen; Redoxin domain protein; KEGG: mmc:Mmcs_4815 redoxin;TlpA-like family; composed of  TlpA, ResA, DsbE and similar proteins. TlpA, ResA and DsbE are bacterial protein disulfide reductases with important roles in cytochrome maturation. They are membrane-anchored proteins with a soluble TRX domain containing a...; cd02966;catalytic residues [active]
GOHSU31	CDS	GOHSU_31_00410	37609..38379		hypothetical protein [nite]			ROP_42350 :hypothetical protein : : : Rhodococcus opacus B4	NUDIX hydrolase		F1YJZ5_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_11126 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: NUDIX hydrolase; KEGG: pnu:Pnuc_0521 NUDIX hydrolase;Coenzyme A pyrophosphatase (CoAse), a member of the Nudix hydrolase superfamily, functions to catalyze the elimination of oxidized inactive CoA, which can inhibit CoA-utilizing enzymes. The need of CoAses mainly arises under conditions of oxidative...; cd03426;putative active site [active];putative CoA binding site [chemical binding];nudix motif;metal binding site [ion binding]
GOHSU31	CDS	GOHSU_31_00420	38376..39578		putative S1C family peptidase [nite]			ROP_42360 :putative S1C family peptidase : : 3.4.21.- : Rhodococcus opacus B4	serine peptidase		F1YJZ4_9ACTO Colicin V production protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_11121 PE=4 SV=1;serine peptidase [Rhodococcus equi 103S];Trypsin-like serine protease; Many of these are synthesized as inactive precursor zymogens that are cleaved during limited proteolysis to generate their active forms. Alignment contains also inactive enzymes that have substitutions of the catalytic triad...; cl00149
GOHSU31	CDS	GOHSU_31_00430	complement(39600..40445)		putative hydrolase [nite]			RER_04860 :putative hydrolase : : : Rhodococcus erythropolis PR4	alpha/beta hydrolase		F1YJZ2_9ACTO Alpha/beta hydrolase fold protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_11111 PE=4 SV=1 ;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: alpha/beta hydrolase fold; KEGG: ank:AnaeK_0555 alpha/beta hydrolase fold;Alpha/beta hydrolase family; pfam12697;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU31	CDS	GOHSU_31_00440	complement(40548..41096)		conserved hypothetical protein [nite]			RER_04870 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F9VWC2_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_060_01440 PE=4 SV= 1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF07332;Protein of unknown function (DUF1469); cl12095
##GOHSU32	contig00162				18372
GOHSU32	CDS	GOHSU_32_00010	complement(50..1987)	case	acetyl-coenzyme A synthetase [hamap]	acs [hamap]	MF_01123:acetyl-coenzyme A synthetase:acs 	ROP_42420 :acetate--CoA ligase : acs : 6.2.1.1 : Rhodococcus opacus B4	acetyl-CoA synthetase	acsA	F1YJZ0_9ACTO Acetyl-coenzyme A synthetase OS=Gordon ia neofelifaecis NRRL B-59395 GN=acsA PE=3 SV=1;acetyl-CoA synthetase [Saccharopolyspora erythraea NRRL 2338];Acs; catalyzes the conversion of acetate and CoA to acetyl-CoA;acetyl-CoA synthetase; Provisional;Domain of unknown function (DUF3448); pfam11930;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU32	CDS	GOHSU_32_00020	2187..2933		conserved hypothetical protein [nite]			RER_05010 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJY9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11096 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU32	CDS	GOHSU_32_00030	complement(3339..4145)		hypothetical protein [nite]			ROP_42480 :hypothetical protein : : : Rhodococcus opacus B4	HAD-superfamily hydrolase		F1YJY8_9ACTO HAD-superfamily hydrolase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11091 PE=4 SV=1;HAD-superfamily hydrolase [Catenulispora acidiphila DSM 44928];TIGRFAM: HAD-superfamily subfamily IB hydrolase, TIGR01490; HAD-superfamily hydrolase, subfamily IB (PSPase- like); PFAM: haloacid dehalogenase; KEGG: sma:SAV_4604 morphological differentiation- associated protein;Phosphoserine phosphatase [Amino acid transport and metabolism]; COG0560;Haloacid Dehalogenase-like Hydrolases; cl11391
GOHSU32	CDS	GOHSU_32_00040	4551..5486		putative triacylglycerol lipase [nite]			ROP_43380 :putative triacylglycerol lipase : : 3.1.1.3 : Rhodococcus opacus B4	lipase		F1YJY7_9ACTO Lipase class 2 OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_11086 PE=4 SV=1;lipase [Rhodococcus jostii RHA1];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU32	CDS	GOHSU_32_00050	5577..6635		hypothetical protein [nite]			ROP_42490 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F9VWC5_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_060_01470 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: rpa:pRPA2 partition protein;helicase/secretion neighborhood CpaE-like protein; TIGR03815
GOHSU32	CDS	GOHSU_32_00060	6632..7789		conserved hypothetical protein [nite]			RER_05040 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	type II secretion system protein E		F9VWC6_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_060_01480 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: type II secretion system protein E; KEGG: sml:Smlt2873  type II/IV secretion system pilus related ATP-binding protein;helicase/secretion neighborhood ATPase; TIGR03819;Type IV secretory pathway component VirB11, and related ATPases. The homohexamer, VirB11 is one of eleven Vir proteins, which are required for T-pilus biogenesis and virulence in the transfer of T-DNA from the Ti (tumor-inducing) plasmid of bacterial to...; cd01130;ATP binding site [chemical binding];Walker A motif;hexamer interface [polypeptide binding];Walker B motif
GOHSU32	CDS	GOHSU_32_00070	7777..8580		conserved hypothetical membrane protein [nite]			RER_05050 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	type II secretion system protein F		F1YJY4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11071 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00482;Flp pilus assembly protein TadB [Intracellular trafficking and secretion]; COG4965
GOHSU32	CDS	GOHSU_32_00080	8577..9176		hypothetical membrane protein [nite]			ROP_42520 :hypothetical membrane protein : : : Rhodococcus opacus B4	type II secretion system F domain-containing protein		D0LED2_GORB4 Type II secretion system F domain prot ein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_0520 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: type II secretion system F domain; KEGG: bpd:BURPS668_A3093 type II secretion system protein
GOHSU32	CDS	GOHSU_32_00090	9190..9492		hypothetical protein [migap]				hypothetical protein		F9VWD0_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_060_01520 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF4244); pfam14029
GOHSU32	CDS	GOHSU_32_00100	9534..9869		hypothetical membrane protein [nite]			ROP_42540 :hypothetical membrane protein : : : Rhodococcus opacus B4	predicted protein		G7H068_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_034_00180 PE=4 SV=1;predicted protein [Streptomyces sp. AA4]
GOHSU32	CDS	GOHSU_32_00110	9911..10270		conserved hypothetical protein [nite]			RER_05090 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0LEI1_GORB4 Putative uncharacterized protein (Prec ursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43 247 / JCM 3198 / NCTC 10667) GN=Gbro_0570 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.636) with cleavage site probability 0.591 at residue 28;helicase/secretion neighborhood TadE-like protein; TIGR03816
GOHSU32	CDS	GOHSU_32_00120	complement(10314..11159)		putative 3-alpha-(or 20-beta)-hydroxysteroid dehydrogenase [nite]			MLP_06710 :putative 3-alpha-(or 20-beta)-hydroxysteroid dehydrogenase : : 1.1.1.53 : Microlunatus phosphovorus NM-1	short-chain dehydrogenase/reductase SDR;hypothetical protein		D3DE86_9ACTO Short-chain dehydrogenase/reductase SD R (Precursor) OS=Frankia sp. EUN1f GN=FrEUN1fDRAFT_8108 PE =4 SV=1;unnamed protein product [Frankia sp. EAN1pec];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.644) with cleavage site probability 0.358 at residue 28;PFAM: short-chain dehydrogenase/reductase SDR; KR domain protein; KEGG: sth:STH1875 short chain dehydrogenase;short chain dehydrogenase; Provisional;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU32	CDS	GOHSU_32_00130	complement(11220..13697)		probable serine/threonine protein kinase PknG [nite]	 pknG [nite]		RER_14740 :probable serine/threonine protein kinase PknG : pknG : 2.7.11.1 : Rhodococcus erythropolis PR4	serine/threonine-protein kinase		F1YN76_9ACTO Serine/threonine protein kinase-like p rotein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1686 3 PE=4 SV=1;serine/threonine-protein kinase [Rhodococcus jostii RHA1];Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
GOHSU32	CDS	GOHSU_32_00140	complement(13701..14687)		putative ABC transporter substrate-binding protein [nite]			RER_14750 :putative ABC transporter substrate-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter substrate-binding protein		F1YN77_9ACTO Extracellular solute-binding protein f amily 3 protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_16868 PE=4 SV=1;ABC transporter substrate-binding protein [Rhodococ cus opacus B4];Bacterial periplasmic transport systems use membrane-bound complexes and substrate-bound, membrane-associated, periplasmic binding proteins (PBPs) to transport a wide variety of  substrates, such as, amino acids, peptides, sugars, vitamins and inorganic...; cd00134;Bacterial extracellular solute-binding proteins, family 3; pfam00497;substrate binding pocket [chemical binding];membrane-bound complex binding site;hinge residues
GOHSU32	CDS	GOHSU_32_00150	complement(14684..16132)		hypothetical membrane protein [nite]			ROP_19080 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YN78_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16873 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU32	CDS	GOHSU_32_00160	16256..16717		conserved hypothetical protein [nite]			RER_14770 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	NUDIX hydrolase		H0QXU4_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_036_00840 PE=3 SV=1;NUDIX hydrolase [Arthrobacter chlorophenolicus A6];PFAM: NUDIX hydrolase; KEGG: art:Arth_2308 NUDIX hydrolase;Nudix hydrolase is a superfamily of enzymes found in all three kingdoms of life, and it catalyzes the hydrolysis of NUcleoside DIphosphates linked to other moieties, X. Enzymes belonging to this superfamily require a divalent cation, such as Mg2+ or Mn2+...; cl00447
GOHSU32	CDS	GOHSU_32_00170	complement(16753..17241)		hypothetical protein [nite]			ROP_68540 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YN80_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16883 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];LeuA allosteric (dimerisation) domain; cl07212
GOHSU32	CDS	GOHSU_32_00180	complement(17391..18080)		thiamine-phosphate pyrophosphorylase [hamap]	thiE [hamap]	MF_00097:thiamine-phosphate pyrophosphorylase:thiE 	ROP_19060 :thiamine-phosphate pyrophosphorylase : thiE : 2.5.1.3 : Rhodococcus opacus B4	thiamine-phosphate pyrophosphorylase		F9VQT3_9ACTO Thiamine-phosphate pyrophosphorylase O S=Gordonia alkanivorans NBRC 16433 GN=thiE PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: gsu:GSU0587 thiamine-phosphate pyrophosphorylase; TIGRFAM: thiamine-phosphate pyrophosphorylase; PFAM: thiamine monophosphate synthase;Thiamine monophosphate synthase (TMP synthase)/TenI. TMP synthase catalyzes an important step in the thiamine biosynthesis pathway, the substitution of the pyrophosphate of 2-methyl-4-amino-5- hydroxymethylpyrimidine pyrophosphate by 4-methyl-5-; cd00564;thiamine phosphate binding site [chemical binding];pyrophosphate binding site [ion binding]
GOHSU32	CDS	GOHSU_32_00190	18179..>18372	3x codon	glycine oxidase ThiO [migap]				glycine oxidase ThiO		F1YN82_9ACTO Glycine oxidase ThiO OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_16893 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.870) with cleavage site probability 0.468 at residue 29;TIGRFAM: glycine oxidase ThiO; PFAM: FAD dependent oxidoreductase; KEGG: dac:Daci_3293 FAD dependent oxidoreductase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;glycine oxidase ThiO; TIGR02352
##GOHSU33	contig00171				1653
GOHSU33	CDS	GOHSU_33_00010	complement(<1..1484)	3x codon	selenocysteine-specific elongation factor [nite]	 selB [nite]		KRH_07840 :selenocysteine-specific elongation factor : selB : : Kocuria rhizophila DC2201	selenocysteine-specific elongation factor	selB	F1YEK1_9ACTO Selenocysteine-specific translation el ongation factor OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_00615 PE=4 SV=1;selenocysteine-specific elongation factor [Kocuria rhizophila DC2201];Selenocysteine-specific translation elongation factor [Translation, ribosomal structure and biogenesis]; COG3276;SelB, the dedicated elongation factor for delivery of selenocysteinyl-tRNA to the ribosome; cd04171;G1 box;putative GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;selB_II: this subfamily represents the domain of elongation factor SelB, homologous to domain II of EF-Tu. SelB may function by replacing EF-Tu. In prokaryotes, the incorporation of selenocysteine as the 21st amino acid, encoded by TGA, requires several...; cd03696
GOHSU33	CDS	GOHSU_33_00020	complement(1487..>1653)	3x codon frame	selenocysteine synthase [nite]	 selA [nite]		KRH_07830 :selenocysteine synthase : selA : 2.9.1.1 : Kocuria rhizophila DC2201	L-seryl-tRNA(sec) selenium transferase SelA	selA	F1YEK0_9ACTO L-seryl-tRNA(Sec) selenium transferase OS=Gordonia neofelifaecis NRRL B-59395 GN=selA PE=3 SV=1;selA gene product [Gordonia polyisoprenivorans VH2] ;selenocysteine synthase; Provisional
##GOHSU34	contig00172				2340
GOHSU34	CDS	GOHSU_34_00010	complement(<1..1160)	3x codon	L-seryl-tRNA(Sec) selenium transferase [nite]	 selA [nite]		MLP_05340 :L-seryl-tRNA(Sec) selenium transferase : selA : 2.9.1.1 : Microlunatus phosphovorus NM-1	L-seryl-tRNA(Sec) selenium transferase	selA	H5TYD1_9ACTO L-seryl-tRNA(Sec) selenium transferase OS=Gordonia sputi NBRC 100414 GN=selA PE=3 SV=1;L-seryl-tRNA(Sec) selenium transferase [Microlunatu s phosphovorus NM- 1];strain coidentity: NBRC 101784;selenocysteine synthase; Provisional;protein synonym:selenocysteine synthase; protein synonym:selenocysteinyl-tRNA(Sec) synthase;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321
GOHSU34	tRNA	GOHSU_34_t0010	complement(1194..1285)		not assigned						identified by tRNAscan-SE; putative
GOHSU34	CDS	GOHSU_34_00020	1318..2310		selenide, water dikinase [hamap]	selD [hamap]	MF_00625:selenide, water dikinase:selD 	MLP_05330 :selenide, water dikinase : selD : 2.7.9.3 : Microlunatus phosphovorus NM-1	selenide, water dikinase		F1YEJ9_9ACTO Selenide, water dikinase OS=Gordonia n eofelifaecis NRRL B-59395 GN=selD PE=3 SV=1;selenide, water dikinase [Catenulispora acidiphila DSM 44928];KEGG: stp:Strop_0583 selenide, water dikinase; TIGRFAM: selenide, water dikinase; PFAM: AIR synthase related protein; AIR synthase related protein domain protein;Selenophosphate synthetase  (SelD) catalyzes the conversion of selenium to selenophosphate which is required by a number of bacterial, archaeal and eukaryotic organisms for synthesis of Secys-tRNA, the precursor of selenocysteine in selenoenzymes. The...; cd02195;dimerization interface [polypeptide binding];putative ATP binding site [chemical binding]
##GOHSU35	contig00173				55304
GOHSU35	CDS	GOHSU_35_00010	complement(<1..1093)	3x codon	hypothetical protein [nite]			KRH_07800 :hypothetical protein : : : Kocuria rhizophila DC2201	polysulphide reductase NrfD		H5TU85_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_256_00110 PE=4 SV=1;polysulphide reductase NrfD [Amycolatopsis mediterr anei U32];Polysulphide reductase, NrfD; cl01295
GOHSU35	CDS	GOHSU_35_00020	complement(1090..2103)		putative formate dehydrogenase iron-sulfur subunit [nite]			MLP_05270 :putative formate dehydrogenase iron-sulfur subunit : : : Microlunatus phosphovorus NM-1	Fe-S-cluster-containing hydrogenase		F1YEJ7_9ACTO 4Fe-4S ferredoxin iron-sulfur-binding domain-containing protein OS=Gordonia neofelifaecis NRRL B -59395 GN=SCNU_00595 PE=4 SV=1;Fe-S-cluster-containing hydrogenase [Amycolatopsis mediterranei U32];Fe-S-cluster-containing hydrogenase components 1 [Energy production and conversion]; COG0437;formate dehydrogenase, beta subunit, Fe-S containing; TIGR01582;4Fe-4S binding domain; cl02805
GOHSU35	CDS	GOHSU_35_00030	complement(2103..4631)		putative formate dehydrogenase major subunit [nite]			MLP_05260 :putative formate dehydrogenase major subunit : : : Microlunatus phosphovorus NM-1	formate dehydrogenase alpha subunit, selenocysteine-containing protein		F1YEJ6_9ACTO Molybdopterin dinucleotide-binding reg ion OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_00590 P E=4 SV=1;formate dehydrogenase alpha subunit, selenocysteine -containing protein [Amycolatopsis mediterranei U32];formate dehydrogenase, alpha subunit, proteobacterial-type; TIGR01553;Molybdopterin-Binding (MopB) domain of the MopB superfamily of proteins, a  large, diverse, heterogeneous superfamily of enzymes that, in general, bind molybdopterin as a cofactor. The MopB domain is found in a wide variety of molybdenum- and...; cl09928;Formate dehydrogenase N, alpha subunit (Formate-Dh-Na) is a major component of nitrate respiration in bacteria such as in the E. coli formate dehydrogenase N (Fdh-N). Fdh-N is a membrane protein that is a complex of three different subunits and is the...; cd02792;molybdopterin cofactor binding site
GOHSU35	CDS	GOHSU_35_00040	complement(4701..5267)		putative formate dehydrogenase alpha subunit [nite]			KRH_07770 :putative formate dehydrogenase alpha subunit : : : Kocuria rhizophila DC2201	formate dehydrogenase alpha subunit, selenocysteine-containing protein		H5TYC6_9ACTO Putative formate dehydrogenase major s ubunit OS=Gordonia sputi NBRC 100414 GN=GOSPT_045_01200 PE =4 SV=1;formate dehydrogenase alpha subunit, selenocysteine -containing protein [Amycolatopsis mediterranei U32];formate dehydrogenase, alpha subunit, proteobacterial-type; TIGR01553;Molybdopterin-Binding (MopB) domain of the MopB superfamily of proteins, a  large, diverse, heterogeneous superfamily of enzymes that, in general, bind molybdopterin as a cofactor. The MopB domain is found in a wide variety of molybdenum- and...; cl09928;molybdopterin cofactor binding site
GOHSU35	CDS	GOHSU_35_00050	5292..5969		hypothetical protein [nite]			ROP_67460 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5UA48_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_019_00310 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: smd:Smed_1698 LmbE family protein;GlcNAc-PI de-N-acetylase; cl00929
GOHSU35	CDS	GOHSU_35_00060	6053..6454		permease [migap]				permease		H0R4H6_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_098_00430 PE=4 SV=1;permease [Marinobacter aquaeolei VT8];PFAM: permease; KEGG: ilo:IL2591 uncharacterized conserved membrane protein, probable transporter;Predicted permeases [General function prediction only];Predicted permease; pfam03773
GOHSU35	CDS	GOHSU_35_00070	6659..7711		hypothetical protein [nite]			ROP_16860 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GWE0_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_75_00750 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];KEGG: fal:FRAAL4952 hypothetical protein;ADP-specific Phosphofructokinase/Glucokinase conserved region; cl14060;Superfamily of nucleases including Short Patch Repair (Vsr) Endonucleases, archaeal Holliday junction resolvases, MutH methy-directed DNA mismatch-repair endonucleases, and catalytic domains of many restriction endonucleases, such as EcoRI, BamHI, and...; cl00277;putative active site [active]
GOHSU35	CDS	GOHSU_35_00080	complement(7799..8671)		hypothetical protein [nite]			KSE_33050 :hypothetical protein : : : Kitasatospora setae NBRC 14216	ErfK/YbiS/YcfS/YnhG family protein		F1YFG3_9ACTO ErfK/YbiS/YcfS/YnhG family protein OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02822 PE=4 SV= 1;ErfK/YbiS/YcfS/YnhG family protein [Segniliparus ro tundus DSM 44985];type strain of Segniliparus rotundus;PFAM: ErfK/YbiS/YcfS/YnhG family protein; KEGG: mab:MAB_4299c hypothetical protein;L,D-transpeptidase catalytic domain; cl15405
GOHSU35	CDS	GOHSU_35_00090	8790..9404		hypothetical protein [nite]			YM304_18070 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	type 11 methyltransferase		G7H6F8_9ACTO Putative methyltransferase OS=Gordonia araii NBRC 100433 GN=GOARA_068_00940 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];PFAM: Methyltransferase type 11; Methyltransferase type 12; KEGG: stp:Strop_3347 methyltransferase type 12;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU35	CDS	GOHSU_35_00100	complement(9412..10713)		putative ATP-binding protein [nite]			ROP_59220 :putative ATP-binding protein : : : Rhodococcus opacus B4	ATP-binding protein		F1YFG0_9ACTO PhoH family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_02807 PE=4 SV=1;ATP-binding protein [Streptomyces clavuligerus ATCC 27064];NYN ribonuclease and ATPase of PhoH family domains [General    function prediction only];PIN domain of bacterial Smg6-like homologs with PhoH-like ATPase domains; cd09883;putative active site [active];P-loop containing Nucleoside Triphosphate Hydrolases; cl09099
GOHSU35	CDS	GOHSU_35_00110	complement(11146..12015)		acyl-[acyl-carrier-protein] desaturase DesA1 [nite]	 desA1 [nite]		ROP_49510 :acyl-[acyl-carrier-protein] desaturase DesA1 : desA1 : 1.14.19.2 : Rhodococcus opacus B4	fatty acid desaturase type 2		F1YFF9_9ACTO Fatty acid desaturase type 2 OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_02802 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: fatty acid desaturase type 2; KEGG: acyl-(acyl-carrier-protein) desaturase,  / stearoyl-ACP desaturase,  ; K03921 acyl-;Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cl00264;dinuclear metal binding motif [ion binding]
GOHSU35	CDS	GOHSU_35_00120	complement(12037..13329)		serine hydroxymethyltransferase [nite]	 glyA [nite]		RER_42520 :serine hydroxymethyltransferase : glyA : 2.1.2.1 : Rhodococcus erythropolis PR4	serine hydroxymethyltransferase	glyA	F1YFF8_9ACTO Serine hydroxymethyltransferase 2 OS=G ordonia neofelifaecis NRRL B-59395 GN=glyA PE=3 SV=1;serine hydroxymethyltransferase [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the reaction of glycine with 5,10-methylenetetrahydrofolate to form L-serine and tetrahydrofolate;serine hydroxymethyltransferase; Reviewed; PRK00011;Serine-glycine hydroxymethyltransferase (SHMT). This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). SHMT carries out interconversion of serine and glycine; it catalyzes the transfer of hydroxymethyl...; cd00378;dimer interface [polypeptide binding];glycine-pyridoxal phosphate binding site [chemical binding];folate binding site [chemical binding]
GOHSU35	CDS	GOHSU_35_00130	complement(13340..14563)		conserved hypothetical protein [nite]			RER_42530 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YFF7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02792 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: swd:Swoo_0250 hypothetical protein
GOHSU35	CDS	GOHSU_35_00140	14687..15541		pantothenate kinase [nite]	 coaA [nite]		RER_42540 :pantothenate kinase : coaA : 2.7.1.33 : Rhodococcus erythropolis PR4	pantothenate kinase	coaA	F1YFF6_9ACTO Pantothenate kinase OS=Gordonia neofel ifaecis NRRL B-59395 GN=coaA PE=3 SV=1;pantothenate kinase [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;catalyzes the formation of (R)-4'-phosphopantothenate in coenzyme A biosynthesis;pantothenate kinase; Provisional;Pantothenate kinase (PanK) catalyzes the phosphorylation of pantothenic acid to form 4'-phosphopantothenic, which is the first of five steps in coenzyme A (CoA) biosynthetic pathway. The reaction carried out by this enzyme is a key regulatory point in...; cd02025;ATP-binding site [chemical binding];CoA-binding site [chemical binding];Mg2+-binding site [ion binding]
GOHSU35	CDS	GOHSU_35_00150	complement(15663..16148)		putative aminoacyl-tRNA deacylase [nite]			ROP_64010 :putative aminoacyl-tRNA deacylase : : : Rhodococcus opacus B4	putative YbaK protein		F1YFF5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02782 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];This CD includes cysteinyl-tRNA(Pro) deacylases from Haemophilus influenzae and Escherichia coli and other related bacterial proteins. These trans-acting, single-domain proteins are homologs of ProX and also the cis-acting prolyl-tRNA synthetase (ProRS)...; cd00002;putative deacylase active site [active]
GOHSU35	CDS	GOHSU_35_00160	complement(16160..16945)		undecaprenyl pyrophosphate synthase [hamap]	uppS [hamap]	MF_01139:undecaprenyl pyrophosphate synthase:uppS 	RER_42550 :short-chain Z-isoprenyl diphosphate synthase : : 2.5.1.- : Rhodococcus erythropolis PR4	undecaprenyl pyrophosphate synthase	uppS1	F1YFF3_9ACTO Undecaprenyl diphosphate synthase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_02772 PE=3 SV=1 ;undecaprenyl pyrophosphate synthase [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of undecaprenyl pyrophosphate from isopentenyl pyrophosphate;Cis (Z)-Isoprenyl Diphosphate Synthases (cis-IPPS); homodimers which catalyze the successive 1'-4 condensation of the isopentenyl diphosphate (IPP) molecule to trans,trans-farnesyl diphosphate (FPP) or to cis,trans-FPP to form long-chain polyprenyl...; cl00230;Putative undecaprenyl diphosphate synthase; pfam01255;catalytic residue [active];putative FPP diphosphate  binding site;putative FPP binding hydrophobic cleft;dimer interface [polypeptide binding];putative IPP diphosphate binding site
GOHSU35	CDS	GOHSU_35_00170	17006..17686		conserved hypothetical membrane protein [nite]			RER_42560 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hemolysin III family channel protein		F1YFF2_9ACTO Channel protein, hemolysin III family OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02767 PE=4 SV=1;hemolysin III family channel protein [Mycobacterium sp. Spyr1];PFAM: Haemolysin-III related; TIGRFAM: channel protein, hemolysin III family;Haemolysin-III related; cl03831
GOHSU35	CDS	GOHSU_35_00180	17772..18284		hypothetical protein [migap]				hypothetical protein		F1YFF1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02762 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU35	CDS	GOHSU_35_00190	18486..19922	overlap	conserved hypothetical membrane protein [nite]			RER_11640 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	propeptide PepSY amd peptidase M4		D0L7R1_GORB4 PepSY-associated TM helix domain prote in (Precursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1654 PE=4 SV =1;propeptide PepSY amd peptidase M4 [Segniliparus rot undus DSM 44985];type strain of Segniliparus rotundus;PFAM: propeptide PepSY amd peptidase M4; PepSY- associated TM helix domain protein; KEGG: msm:MSMEG_6570 integral membrane protein;Uncharacterized iron-regulated membrane protein [Function unknown]; COG3182;PepSY-associated TM helix; cl15813;Peptidase propeptide and YPEB domain; cl15815
GOHSU35	CDS	GOHSU_35_00200	complement(19857..20417)	overlap	protein-L-isoaspartate O-methyltransferase (PIMT) [nite]	 pcm [nite]		APE_1011.1 :protein-L-isoaspartate O-methyltransferase (PIMT) : pcm : 2.1.1.77 : Aeropyrum pernix K1	protein-L-isoaspartate O-methyltransferase	pcm-2	E9UQ67_9ACTO Protein-L-isoaspartate(D-aspartate) O- methyltransferase OS=Nocardioidaceae bacterium Broad-1 GN= NBCG_00868 PE=4 SV=1;protein-L-isoaspartate O-methyltransferase [Archaeo globus fulgidus DSM 4304];catalyzes the methyl esterification of L-isoaspartyl residues that are formed in damaged proteins;Protein-L-isoaspartate carboxylmethyltransferase [Posttranslational modification, protein turnover, chaperones]; COG2518;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU35	CDS	GOHSU_35_00210	complement(20439..20843)		hypothetical protein [migap]				hypothetical protein		F5YTF4_MYCSD Putative uncharacterized protein OS=My cobacterium sp. (strain JDM601) GN=JDM601_2149 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF04075; match to protein family HMM TIGR00026;Domain of unknown function (DUF385); cl04387
GOHSU35	CDS	GOHSU_35_00220	complement(20916..22010)		3-deoxy-D-arabino-heptulosonate 7-phosphate synthase [nite]	 aroG [nite]		RER_46990 :3-deoxy-D-arabino-heptulosonate 7-phosphate synthase : aroG : 2.5.1.54 : Rhodococcus erythropolis PR4	phospho-2-dehydro-3-deoxyheptonate aldolase	aroF	F1YFE6_9ACTO Phospho-2-dehydro-3-deoxyheptonate ald olase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02737 PE=3 SV=1;phospho-2-dehydro-3-deoxyheptonate aldolase [Coryne bacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 3-deoxy-D-arabino-hept-2-ulosonate 7 phosphate from phosphoenolpyruvate and D-erythrose 4-phosphate, phenylalanine sensitive;NeuB family; cl00496;DAHP synthetase I family; pfam00793
GOHSU35	CDS	GOHSU_35_00230	complement(22315..24321)		hypothetical protein [nite]			ROP_59160 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YFE4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02727 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF255; KEGG: reh:H16_A1279 highly conserved protein containing a thioredoxin domain;Highly conserved protein containing a thioredoxin domain [Posttranslational modification, protein turnover, chaperones];TRX domain, SSP411 protein family; members of this family are highly conserved proteins present in eukaryotes, bacteria and archaea, about 600-800 amino acids in length, which contain a TRX domain with a redox active CXXC motif. The human/rat protein; cd02955;catalytic residues [active]
GOHSU35	CDS	GOHSU_35_00240	complement(24392..24751)		hypothetical membrane protein [nite]			ROP_59150 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H5TVC0_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_007_00390 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU35	CDS	GOHSU_35_00250	complement(24748..25629)		mycothiol S-conjugate amidase Mca [nite]	 mca [nite]		ROP_59140 :mycothiol S-conjugate amidase Mca : mca : : Rhodococcus opacus B4	mycothiol conjugate amidase Mca		F1YFE2_9ACTO Mycothiol conjugate amidase Mca OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_02717 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: mycothiol conjugate amidase Mca; PFAM: LmbE family protein; KEGG: pla:Plav_1416 LmbE family protein;GlcNAc-PI de-N-acetylase; cl00929
GOHSU35	CDS	GOHSU_35_00260	25813..26136		hypothetical membrane protein [nite]			ROP_59130 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H0R5X0_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_115_01110 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF4307); pfam14155
GOHSU35	CDS	GOHSU_35_00270	26147..26899		conserved hypothetical membrane protein [nite]			RER_03450 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	protein of unknown function DUF165		D0L7P4_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_1637 PE=4 SV=1;protein of unknown function DUF165 [Bacteroides cop rosuis DSM 18011];Signal predicted by SignalP 3.0 HMM; IMG reference gene:2504107503_SP;COGs: COG1738 conserved hypothetical protein; InterPro IPR003744; KEGG: bfs:BF1395 putative transmembrane protein; PFAM: Protein of unknown function DUF165; SPTR: Putative membrane protein; TIGRFAM: Protein of unknown function DUF165; IMG reference gene:2504107503; PFAM: Uncharacterized ACR, YhhQ family COG1738; TIGRFAM: conserved hypothetical integral membrane protein;Uncharacterized ACR, YhhQ family COG1738; cl00756
GOHSU35	CDS	GOHSU_35_00280	27079..27570		transcription elongation factor GreA [nite]	 greA [nite]		ROP_59120 :transcription elongation factor GreA : greA : : Rhodococcus opacus B4	transcription elongation factor GreA		F1YF59_9ACTO Transcription elongation factor OS=Gor donia neofelifaecis NRRL B-59395 GN=greA PE=3 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rop:ROP_59120 transcription elongation factor GreA; TIGRFAM: transcription elongation factor GreA; PFAM: transcription elongation factor GreA/GreB domain protein;transcription elongation factor GreA; Reviewed; PRK00226;Transcription elongation factor, N-terminal; pfam03449;Transcription elongation factor, GreA/GreB, C-term; pfam01272
GOHSU35	CDS	GOHSU_35_00290	complement(27685..28050)		Xre family DNA-binding protein [nite]			DMR_10290 :Xre family DNA-binding protein : : : Desulfovibrio magneticus RS-1	XRE family transcriptional regulator		F1YF58_9ACTO Transcriptional regulator, XRE family protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_026 97 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: cdi:DIP0643  DNA-binding protein; PFAM: helix-turn-helix domain protein; SMART: helix-turn-helix domain protein;Helix-turn-helix XRE-family like proteins. Prokaryotic DNA binding proteins belonging to the xenobiotic response element family of transcriptional regulators; cd00093;non-specific DNA binding site [nucleotide binding];salt bridge;sequence-specific DNA binding site [nucleotide binding]
GOHSU35	CDS	GOHSU_35_00300	complement(28471..29097)		hypothetical protein [migap]				hypothetical protein		H0R0N9_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_060_00330 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: mab:MAB_0011c hypothetical protein;Protein of unknown function (DUF3558); pfam12079
GOHSU35	CDS	GOHSU_35_00310	complement(29105..29407)		hypothetical protein [migap]				hypothetical protein		D0LCE8_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_0335 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU35	CDS	GOHSU_35_00320	complement(29879..31036)		cystathionine gamma-synthase/cystathionine gamma-lyase [nite]	 metB [nite]		ROP_59060 :cystathionine gamma-synthase/cystathionine gamma-lyase : metB : 2.5.1.48/4.4.1.1 : Rhodococcus opacus B4	cystathionine gamma-synthase	metB	H5THS6_9ACTO Cystathionine gamma-synthase/cystathio nine gamma-lyase OS=Gordonia otitidis NBRC 100426 GN=metB PE=3 SV=1;cystathionine gamma-synthase [Corynebacterium gluta micum ATCC 13032];IS fingerprint type 4-5;cystathionine gamma-synthase; Provisional;catalyzes the formation of cystathionine from L-cysteine and O-succinyl-L-homoserine;CGS_like: Cystathionine gamma-synthase is a PLP dependent enzyme and catalyzes the committed step of methionine biosynthesis. This pathway is unique to microorganisms and plants, rendering the enzyme an attractive target for the development of...; cd00614;homodimer interface [polypeptide binding];substrate-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU35	CDS	GOHSU_35_00330	complement(31033..32442)		cystathionine beta-synthase [nite]	 cbs [nite]		ROP_59040 :cystathionine beta-synthase : cbs : 4.2.1.22 : Rhodococcus opacus B4	cystathionine beta-synthase		H5TVD5_9ACTO Cystathionine beta-synthase OS=Gordoni a sputi NBRC 100414 GN=cbs PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mxa:MXAN_2041  cystathionine beta- synthase; TIGRFAM: cystathionine beta-synthase; PFAM: pyridoxal-5'-phosphate-dependent protein subunit beta; CBS domain containing protein; SMART: CBS domain containing protein;cystathionine beta-synthase; TIGR01137;CBS_like: This subgroup includes Cystathionine beta-synthase (CBS) and Cysteine synthase. CBS is a unique heme-containing enzyme that catalyzes a pyridoxal 5'-phosphate (PLP)-dependent condensation of serine and homocysteine to give cystathionine; cd01561;dimer interface [polypeptide binding];pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active];This cd contains two tandem repeats of the cystathionine beta-synthase (CBS pair) domains associated with the pyridoxal-phosphate (PALP) dependent enzyme domain upstream.   The vitamin B6 complex comprises pyridoxine, pyridoxal, and pyridoxamine, as well...; cd04609
GOHSU35	CDS	GOHSU_35_00340	32485..33708		putative beta-ketoadipyl-CoA thiolase [nite]			RER_42680 :putative beta-ketoadipyl-CoA thiolase : : 2.3.1.174 : Rhodococcus erythropolis PR4	acetyl-CoA acetyltransferase		H0RKJ9_9ACTO Putative acetyl-CoA acyltransferase OS =Gordonia polyisoprenivorans NBRC 16320 GN=GOPIP_085_01010 PE=3 SV=1;acetyl-CoA acetyltransferase [Rhodococcus jostii RH A1];acetyl-CoA acetyltransferase; Provisional;Catalyzes the synthesis of acetoacetyl coenzyme A from two molecules of acetyl coenzyme A. It can also act as a thiolase, catalyzing the reverse reaction and generating two-carbon units from the four-carbon product of fatty acid oxidation;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU35	CDS	GOHSU_35_00350	33932..34930		not assigned						
GOHSU35	CDS	GOHSU_35_00360	complement(35109..35990)		hypothetical membrane protein [nite]			ROP_59010 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YF50_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02657 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF1112; KEGG: xfa:XF2760 hypothetical protein;BAX inhibitor (BI)-1/YccA-like protein family; cl00473
GOHSU35	CDS	GOHSU_35_00370	complement(36034..36669)		putative TetR family transcriptional regulator [nite]			YM304_26110 :putative TetR family transcriptional regulator : : : Acidimicrobidae bacterium YM16-304	TetR family transcriptional regulator		F1YF49_9ACTO Regulatory protein, TetR OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_02652 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m gilvum PYR- GCK];PFAM: regulatory protein, TetR; KEGG: mva:Mvan_4572 transcriptional regulator, TetR family;Helix-turn-helix domains; cl00088;WHG domain; pfam13305
GOHSU35	CDS	GOHSU_35_00380	complement(36706..38919)		hypothetical protein [nite]			KSE_22940 :hypothetical protein : : : Kitasatospora setae NBRC 14216	MMPL domain-containing protein		F1YF48_9ACTO MMPL domain-containing protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_02647 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];Predicted drug exporters of the RND superfamily [General function prediction only];COG2409 Predicted drug exporters of the RND superfamily
GOHSU35	CDS	GOHSU_35_00390	complement(39003..39680)		conserved hypothetical protein [nite]			BBR47_09630 :conserved hypothetical protein : : : Brevibacillus brevis NBRC 100599	hypothetical protein		F1YF45_9ACTO DSBA oxidoreductase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_02632 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;DsbA family, FrnE subfamily; FrnE is a DsbA-like protein containing a CXXC motif. It is presumed to be a thiol oxidoreductase involved in polyketide biosynthesis, specifically in the production of the aromatic antibiotics frenolicin and nanaomycins; cd03024;Evidence 4 : Homologs of previously reported genes of unknown function;Predicted dithiol-disulfide isomerase involved in polyketide biosynthesis [Secondary metabolites biosynthesis, transport, and catabolism]; COG2761;catalytic residues [active]
GOHSU35	tRNA	GOHSU_35_t0010	39726..39802		not assigned						identified by tRNAscan-SE; putative
GOHSU35	CDS	GOHSU_35_00400	39885..40658		hypothetical protein [nite]			ROP_65980 :hypothetical protein : : : Rhodococcus opacus B4	putative esterase of the alpha-beta hydrolase superfamily		G8RS63_MYCRN Putative esterase of the alpha-beta hy drolase superfamily OS=Mycobacterium rhodesiae (strain NBB 3) GN=MycrhN_3501 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Predicted esterase of the alpha-beta hydrolase superfamily [General function prediction only]; COG1752;PFAM: Patatin-like phospholipase;Bacterial patatin-like phospholipase domain containing protein 6; cd07228;nucleophile elbow
GOHSU35	CDS	GOHSU_35_00410	complement(40682..41947)		putative hydrolase [nite]			YM304_36490 :putative hydrolase : : : Acidimicrobidae bacterium YM16-304	alpha/beta hydrolase		C7Q579_CATAD OsmC family protein OS=Catenulispora a cidiphila (strain DSM 44928 / NRRL B-24433 / NBRC 102108 / JCM 14897) GN=Caci_7018 PE=4 SV=1;alpha/beta fold family hydrolase [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;Alpha/beta hydrolase family; pfam12695;OsmC-like protein; cl00767
GOHSU35	CDS	GOHSU_35_00420	complement(42270..43184)		conserved hypothetical protein [nite]			BBR47_11130 :conserved hypothetical protein : : : Brevibacillus brevis NBRC 100599	aminoglycoside/hydroxyurea antibiotic resistance kinase		G7GX78_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_004_00430 PE=4 SV=1;aminoglycoside/hydroxyurea antibiotic resistance ki nase [Kribbella flavida DSM 17836];PFAM: aminoglycoside/hydroxyurea antibiotic resistance kinase; KEGG: pfo:Pfl01_3891 aminoglycoside/hydroxyurea antibiotic resistance kinase;Protein Kinases, catalytic domain; cl09925
GOHSU35	CDS	GOHSU_35_00430	43183..43920		hypothetical membrane protein [nite]			RER_32470 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		C1A020_RHOE4 Hypothetical membrane protein OS=Rhodo coccus erythropolis (strain PR4 / NBRC 100887) GN=RER_3247 0 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU35	CDS	GOHSU_35_00440	44150..44764		N-(5'-phosphoribosyl)anthranilate isomerase [nite]	 trpF [nite]		YM304_24300 :N-(5'-phosphoribosyl)anthranilate isomerase : trpF : 5.3.1.24 : Acidimicrobidae bacterium YM16-304	Phosphoribosylanthranilate isomerase		F1YFK6_9ACTO N-(5'-phosphoribosyl)anthranilate isom erase OS=Gordonia neofelifaecis NRRL B-59395 GN=trpF PE=3 SV=1;Phosphoribosylanthranilate isomerase [Jonesia denit rificans DSM 20603];type strain of Jonesia denitrificans;PFAM: N-(5'phosphoribosyl)anthranilate isomerase (PRAI); KEGG: msl:Msil_0146 phosphoribosylanthranilate isomerase;Phosphoribosylanthranilate isomerase (PRAI) catalyzes the fourth step of the tryptophan biosynthesis, the conversion of N-(5'- phosphoribosyl)-anthranilate (PRA) to 1-(o-carboxyphenylamino)- 1-deoxyribulose 5-phosphate (CdRP). Most PRAIs are monomeric; cd00405
GOHSU35	CDS	GOHSU_35_00450	44781..46886		beta-glucoside-specific phosphotransferase system enzyme II [nite]	 bglP [nite]		TEH_04790 :beta-glucoside-specific phosphotransferase system enzyme II : bglP : 2.7.1.69 : Tetragenococcus halophilus NBRC 12172	PTS system beta-glucoside-specific EIIBCA component		D0L9W0_GORB4 PTS system, glucose subfamily, IIA sub unit OS=Gordonia bronchialis (strain ATCC 25592 / DSM 4324 7 / JCM 3198 / NCTC 10667) GN=Gbro_2915 PE=4 SV=1;PTS system beta-glucoside-specific EIIBCA component [Paenibacillus sp. Aloe- 11];PTS system, beta-glucoside-specific IIABC component; TIGR01995;PTS_IIB, PTS system, glucose/sucrose specific IIB subunit. The bacterial phosphoenolpyruvate: sugar phosphotransferase system (PTS) is a multi-protein system involved in the regulation of a variety of metabolic and transcriptional processes. This family...; cd00212;active site turn [active];phosphorylation site [posttranslational modification];Phosphotransferase system IIC components, glucose/maltose/N-acetylglucosamine-specific [Carbohydrate transport and metabolism]; cl00557;PTS_IIA, PTS system, glucose/sucrose specific IIA subunit. The bacterial phosphoenolpyruvate: sugar phosphotransferase system (PTS) is a multi-protein system involved in the regulation of a variety of metabolic and transcriptional processes. This family...; cd00210;HPr interaction site;glycerol kinase (GK) interaction site [polypeptide binding]
GOHSU35	CDS	GOHSU_35_00460	46983..49223		conserved hypothetical membrane protein [nite]			RER_18230 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	integral membrane protein		H5TUA1_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_256_00270 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG2409 Predicted drug exporters of the RND superfamily;Predicted drug exporters of the RND superfamily [General function prediction only]
GOHSU35	CDS	GOHSU_35_00470	49220..49804		putative TetR family transcriptional regulator [nite]			KSE_47590 :putative TetR family transcriptional regulator : : : Kitasatospora setae NBRC 14216	TetR family transcriptional regulator		G7GWR3_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia amarae NBRC 15530 GN=GOAMR_78_00150 P E=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];PFAM: regulatory protein TetR; KEGG: mpa:MAP3711c hypothetical protein;mycofactocin system transcriptional regulator; TIGR03968;Helix-turn-helix domains; cl00088
GOHSU35	CDS	GOHSU_35_00480	complement(49847..51007)		hypothetical protein [nite]			ROP_23560 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YFJ9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02482 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rop:ROP_23560 hypothetical protein;Protein of unknown function (DUF3089); pfam11288
GOHSU35	CDS	GOHSU_35_00490	51162..52727		serine/threonine protein kinase [migap]				serine/threonine protein kinase		D0LB19_GORB4 Serine/threonine protein kinase-relate d protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3106 PE=3 SV=1;serine/threonine protein kinase [Mycobacterium aviu m 104];identified by match to protein family HMM PF00069; match to protein family HMM PF01436;Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
GOHSU35	CDS	GOHSU_35_00500	52808..52969		not assigned						F1YFJ7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02472 PE=4 SV=1
GOHSU35	CDS	GOHSU_35_00510	52966..53823		putative acyl-CoA reductase [nite]			RER_56290 :putative acyl-CoA reductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		F1YFJ6_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02467 PE=3 SV=1;short chain dehydrogenase [Rhodococcus jostii RHA1] ;short chain dehydrogenase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU35	CDS	GOHSU_35_00520	53858..>55304	3x codon	putative acyl-CoA dehydrogenase [nite]			ACMV_16190 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Acidiphilium multivorum AIU301	acyl-CoA dehydrogenase domain-containing protein		F1YFJ5_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 02462 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: acyl-CoA dehydrogenase domain protein; KEGG: rpi:Rpic_1444 acyl-CoA dehydrogenase domain protein;Acyl-CoA dehydrogenase N terminal; pfam12418;acyl-CoA dehydrogenase; Provisional;Acyl-CoA dehydrogenase; cl09933;Acetyl-CoA dehydrogenase C-terminal like; pfam12806
##GOHSU36	contig00181				58622
GOHSU36	CDS	GOHSU_36_00010	128..745		TM2 domain-containing protein [migap]				TM2 domain-containing protein		Q47RP3_THEFY Putative uncharacterized protein OS=Th ermobifida fusca (strain YX) GN=Tfu_0836 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU36	CDS	GOHSU_36_00020	complement(832..1926)		not assigned						F1YNA6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17023 PE=4 SV=1
GOHSU36	CDS	GOHSU_36_00030	complement(1954..2517)	case	deoxycytidine triphosphate deaminase [hamap]	dcd [hamap]	MF_00146:deoxycytidine triphosphate deaminase:dcd 	ROP_54420 :deoxycytidine triphosphate deaminase : dcd : 3.5.4.13 : Rhodococcus opacus B4	deoxycytidine triphosphate deaminase	dcd	H5UIJ8_9ACTO Deoxycytidine triphosphate deaminase O S=Gordonia terrae NBRC 100016 GN=dcd PE=3 SV=1;deoxycytidine triphosphate deaminase [Corynebacteri um glutamicum ATCC 13032];IS fingerprint type 4-5;Catalyzes the formation of dUTP from dCTP in thymidylate biosynthesis;Trimeric dUTP diphosphatases; cd07557;trimer interface [polypeptide binding]
GOHSU36	tRNA	GOHSU_36_t0010	complement(2606..2679)		not assigned						identified by tRNAscan-SE; putative
GOHSU36	CDS	GOHSU_36_00040	complement(2732..4120)		putative lipase [nite]			ROP_15770 :putative lipase : : 3.1.1.- : Rhodococcus opacus B4	lipase		G7H3E4_9ACTO Putative lipase OS=Gordonia araii NBRC 100433 GN=GOARA_056_01170 PE=4 SV=1;lipase [Rhodococcus opacus B4];Secretory lipase; pfam03583
GOHSU36	CDS	GOHSU_36_00050	4302..5192		conserved hypothetical membrane protein [nite]			RER_08990 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	bile acid:sodium symporter		C9Z730_STRSW Putative sodium transporter OS=Strepto myces scabies (strain 87.22) GN=SCAB_43591 PE=4 SV=1;bile acid:sodium symporter [Acinetobacter johnsonii SH046];Membrane transport protein; cl09117
GOHSU36	CDS	GOHSU_36_00060	complement(5210..6748)		putative drug resistance protein [nite]			KSE_71160 :putative drug resistance protein : : : Kitasatospora setae NBRC 14216	major facilitator superfamily protein		G7H314_9ACTO Putative drug resistance protein OS=Go rdonia araii NBRC 100433 GN=GOARA_054_00150 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: pol:Bpro_1369 major facilitator superfamily MFS_1;methyl viologen resistance protein SmvA; Provisional;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU36	CDS	GOHSU_36_00070	complement(6745..7335)		putative TetR family transcriptional regulator [nite]			MLP_26700 :putative TetR family transcriptional regulator : : : Microlunatus phosphovorus NM-1	transcriptional regulator, TetR family		G7H315_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_054_00160 PE=4 SV=1;transcriptional regulator, TetR family [Streptomyce s griseus XylebKG- 1];PFAM: Transcription regulator, TetR-like, DNA-binding, bacterial/archaeal; KEGG: sgr:SGR_3387 TetR family transcriptional regulator;transcriptional repressor BetI; TIGR03384;Helix-turn-helix domains; cl00088;Bacterial transcriptional repressor; cl07106
GOHSU36	CDS	GOHSU_36_00080	complement(7523..8320)		putative MerR family transcriptional regulator [nite]			KSE_62560 :putative MerR family transcriptional regulator : : : Kitasatospora setae NBRC 14216	MerR family transcriptional regulator		F1YNA7_9ACTO Regulatory protein MerR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_17028 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: MerR family transcriptional regulator; SMART: MerR family transcriptional regulator; KEGG: cti:RALTA_A3203  MerR family transcriptional regulator;Helix-Turn-Helix DNA binding domain of transcription regulators from the MerR superfamily; cl02600;MerR HTH family regulatory protein; pfam13411;DNA binding residues [nucleotide binding]
GOHSU36	CDS	GOHSU_36_00090	complement(8486..9613)		oxidoreductase [nite]			ROP_34890 :oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase		F1YNA8_9ACTO Ferredoxin OS=Gordonia neofelifaecis N RRL B-59395 GN=SCNU_17033 PE=4 SV=1;oxidoreductase [Rhodococcus opacus B4];Flavodoxin reductases (ferredoxin-NADPH reductases) family 1 [Energy production and conversion]; COG1018;Phthalate dioxygenase reductase (PDR) is an FMN-dependent reductase that mediates electron transfer from NADH to FMN to an iron sulfur cluster. PDR has an an N-terminal  ferrredoxin reductase (FNR)-like NAD(H) binding domain and a C-terminal iron-sulfur; cd06185;FMN-binding pocket [chemical binding];flavin binding motif;phosphate binding motif [ion binding];beta-alpha-beta structure motif;NAD binding pocket [chemical binding];2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cd00207;catalytic loop [active];iron binding site [ion binding]
GOHSU36	CDS	GOHSU_36_00100	complement(9615..10511)		hypothetical protein [nite]			KSE_63700 :hypothetical protein : : : Kitasatospora setae NBRC 14216	metal-dependent hydrolase		F1YNA9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_17038 PE=4 SV=1;metal-dependent hydrolase [Burkholderia cenocepacia MC0- 3];Predicted metal-dependent hydrolase [General function prediction only];KEGG: bur:Bcep18194_B0708 metal-dependent hydrolase;Predicted metal-dependent hydrolase; pfam10118
GOHSU36	CDS	GOHSU_36_00110	complement(10521..12290)		putative peptidase S33 family protein [nite]			KSE_63730 :putative peptidase S33 family protein : : : Kitasatospora setae NBRC 14216	oxidoreductase, short chain dehydrogenase/reductase family		F1YNB0_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_17043 PE=3 SV=1;oxidoreductase, short chain dehydrogenase/reductase family [Alcanivorax sp. DG881];[IQR] COG1028 Dehydrogenases with different specificities (related to short-chain alcohol dehydrogenases);short chain dehydrogenase; Validated;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU36	CDS	GOHSU_36_00120	complement(12439..13692)		putative alanine aminotransferase [nite]			ROP_55340 :putative alanine aminotransferase : : 2.6.1.2 : Rhodococcus opacus B4	aminotransferase		F1YNB1_9ACTO Aminotransferase AlaT OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_17048 PE=4 SV=1;aminotransferase AlaT [Corynebacterium glutamicum A TCC 13032];IS fingerprint type 4-5;broad specificity; family IV; in Corynebacterium glutamicum this protein can use glutamate, 2-aminobutyrate, and aspartate as amino donors and pyruvate as the acceptor;Aspartate/tyrosine/aromatic aminotransferase [Amino acid transport and metabolism];Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU36	CDS	GOHSU_36_00130	13751..14644		putative exodeoxyribonuclease [nite]			ROP_04150 :putative exodeoxyribonuclease : : 3.1.11.2 : Rhodococcus opacus B4	exodeoxyribonuclease III Xth		F9VVH6_9ACTO Putative exodeoxyribonuclease OS=Gordo nia alkanivorans NBRC 16433 GN=GOALK_056_00380 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;Exonuclease-Endonuclease-Phosphatase (EEP) domain superfamily; cl00490;KEGG: rha:RHA1_ro00291 exodeoxyribonuclease III; TIGRFAM: exodeoxyribonuclease III Xth; PFAM: endonuclease/exonuclease/phosphatase;putative catalytic site [active];putative metal binding site [ion binding];putative phosphate binding site [ion binding]
GOHSU36	CDS	GOHSU_36_00140	complement(14662..15555)		hypothetical protein [nite]			MLP_02110 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		G8RLE2_MYCRN Putative uncharacterized protein OS=My cobacterium rhodesiae (strain NBB3) GN=MycrhN_1793 PE=4 SV =1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF559);Domain of unknown function (DUF4095); pfam13338;Superfamily of nucleases including Short Patch Repair (Vsr) Endonucleases, archaeal Holliday junction resolvases, MutH methy-directed DNA mismatch-repair endonucleases, and catalytic domains of many restriction endonucleases, such as EcoRI, BamHI, and...; cl00277;putative active site [active]
GOHSU36	CDS	GOHSU_36_00150	complement(15832..19095)		putative iron-sulfur binding reductase [nite]			ROP_55420 :putative iron-sulfur binding reductase : : : Rhodococcus opacus B4	ferredoxin, 4Fe-4S		H5TV51_9ACTO Putative iron-sulfur binding reductase OS=Gordonia sputi NBRC 100414 GN=GOSPT_006_00500 PE=4 SV= 1;ferredoxin, 4Fe-4S [Streptomyces sp. e14];Fe-S oxidoreductase [Energy production and conversion]; COG0247;4Fe-4S binding domain; cl02805;Cysteine-rich domain; pfam02754
GOHSU36	CDS	GOHSU_36_00160	complement(19333..20730)		wax ester synthase/diacylglycerol acyltransferase [nite]			ROP_13050 :wax ester synthase/diacylglycerol acyltransferase : : 2.3.1.20 : Rhodococcus opacus B4	putative diacylglycerol O-acyltransferase		F1YJR3_9ACTO Acyltransferase, WS/DGAT/MGAT OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_10716 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007
GOHSU36	CDS	GOHSU_36_00170	complement(20797..22236)		wax ester synthase/diacylglycerol acyltransferase [nite]			ROP_13050 :wax ester synthase/diacylglycerol acyltransferase : : 2.3.1.20 : Rhodococcus opacus B4	hypothetical protein		F1YJR3_9ACTO Acyltransferase, WS/DGAT/MGAT OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_10716 PE=4 SV=1;unnamed protein product [Mycobacterium tuberculosis H37Rv];acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007
GOHSU36	CDS	GOHSU_36_00180	22512..24269		not assigned						F1YJR5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10726 PE=4 SV=1
GOHSU36	CDS	GOHSU_36_00190	24406..25017		hypothetical membrane protein [nite]			ROP_21530 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YJR2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10711 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: protein of unknown function DUF218; KEGG: rha:RHA1_ro02428 hypothetical protein;YdcF-like. YdcF-like is a large family of mainly bacterial proteins, with a few members found in fungi, plants, and archaea. Escherichia coli YdcF has been shown to bind S-adenosyl-L-methionine (AdoMet), but a biochemical function has not been...; cd06259;putative active site [active]
GOHSU36	CDS	GOHSU_36_00200	complement(25001..27151)		conserved hypothetical membrane protein [nite]			RER_01890 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	acyltransferase 3		F1YJQ9_9ACTO Acyltransferase 3 OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_10696 PE=4 SV=1;acyltransferase 3 [Mycobacterium gilvum PYR- GCK];PFAM: acyltransferase 3; KEGG: mbo:Mb1592c hypothetical protein;OpgC protein; cl00792
GOHSU36	CDS	GOHSU_36_00210	27319..28446		putative zinc-containing alcohol dehydrogenase [nite]			RER_36460 :putative zinc-containing alcohol dehydrogenase : : 1.1.1.- : Rhodococcus erythropolis PR4	alcohol dehydrogenase		G7H732_9ACTO Putative zinc-containing alcohol dehyd rogenase OS=Gordonia araii NBRC 100433 GN=GOARA_082_00450 PE=3 SV=1;alcohol dehydrogenase [Mycobacterium sp. KMS];PFAM: Alcohol dehydrogenase, zinc-binding domain protein; Alcohol dehydrogenase GroES domain protein; KEGG: mmc:Mmcs_0137 alcohol dehydrogenase, zinc-binding protein;oxidoreductase, Rxyl_3153 family; TIGR03989;Class III alcohol dehydrogenase; cd08279;NAD binding site [chemical binding];catalytic Zn binding site [ion binding];substrate binding site [chemical binding];structural Zn binding site [ion binding]
GOHSU36	CDS	GOHSU_36_00220	28628..30229		acyl-CoA carboxylase beta chain [nite]	 accD [nite]		ROP_16140 :acyl-CoA carboxylase beta chain : accD : 6.4.1.- : Rhodococcus opacus B4	acetyl-CoA carboxylase carboxyltransferase		D9V0S4_9ACTO Acetyl/propionyl CoA carboxylase OS=St reptomyces sp. AA4 GN=SSMG_02354 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF01039;3-methylcrotonyl-CoA carboxylase, beta chain;Acetyl co-enzyme A carboxylase carboxyltransferase alpha subunit; cl15772
GOHSU36	CDS	GOHSU_36_00230	30234..32228		acyl-CoA carboxylase alpha chain [nite]	 accA [nite]		ROP_16130 :acyl-CoA carboxylase alpha chain : accA : 6.4.1.- : Rhodococcus opacus B4	acetyl/propionyl-CoA carboxylase, alpha subunit		G6XBC1_MYCAB Acetyl-/propionyl-CoA carboxylase subu nit alpha AccA2 OS=Mycobacterium abscessus 47J26 GN=MAB47J 26_18187 PE=4 SV=1;acetyl/propionyl-CoA carboxylase, alpha subunit [Sa ccharomonospora marina XMU15];PFAM: Carbamoyl-phosphate synthase L chain, ATP binding domain; Biotin carboxylase C-terminal domain; Carbamoyl-phosphate synthase L chain, N-terminal domain; Biotin-requiring enzyme;Acetyl/propionyl-CoA carboxylase, alpha subunit [Lipid metabolism];Carbamoyl-phosphate synthase L chain, N-terminal domain; pfam00289;ATP-grasp domain; cl03087;Biotin carboxylase C-terminal domain; cl08365;The biotinyl-domain or biotin carboxyl carrier protein (BCCP) domain is present in all biotin-dependent enzymes, such as acetyl-CoA carboxylase, pyruvate carboxylase, propionyl-CoA carboxylase, methylcrotonyl-CoA carboxylase, geranyl-CoA carboxylase; cd06850;carboxyltransferase (CT) interaction site;biotinylation site [posttranslational modification]
GOHSU36	CDS	GOHSU_36_00240	32225..33895		putative fatty-acid--CoA ligase [nite]			RER_46680 :putative fatty-acid--CoA ligase : : 6.2.1.- : Rhodococcus erythropolis PR4	long-chain-fatty-acid--CoA ligase		F5YTH8_MYCSD Fatty-acid-CoA ligase FadD11 OS=Mycoba cterium sp. (strain JDM601) GN=fadD11 PE=4 SV=1;long-chain-fatty-acid--CoA ligase [Rhodococcus jost ii RHA1];Long-chain acyl-CoA synthetases (AMP-forming) [Lipid metabolism]; COG1022;AMP-binding enzyme; cl15778
GOHSU36	CDS	GOHSU_36_00250	33892..34701		putative enoyl-CoA hydratase [nite]			ROP_16110 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus opacus B4	enoyl-CoA hydratase/isomerase family protein		Q4K8Z3_PSEF5 Putative isohexenylglutaconyl-CoA hydr atase AtuE OS=Pseudomonas fluorescens (strain Pf-5 / ATCC BAA-477) GN=atuE PE=4 SV=1;enoyl-CoA hydratase/isomerase family protein [Alcan ivorax sp. DG881];[I] COG1024 Enoyl-CoA hydratase/carnithine racemase;enoyl-CoA hydratase; Provisional;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU36	CDS	GOHSU_36_00260	34698..35504		putative enoyl-CoA hydratase PaaB [nite]	 paaB [nite]		ROP_25830 :putative enoyl-CoA hydratase PaaB : paaB : 4.2.1.17 : Rhodococcus opacus B4	enoyl-CoA hydratase		Q1NAB7_9SPHN Putative enoyl-CoA hydratase/isomerase OS=Sphingomonas sp. SKA58 GN=SKA58_04651 PE=4 SV=1;unnamed protein product [Bordetella petrii DSM 1280 4];enoyl-CoA hydratase; Provisional;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU36	CDS	GOHSU_36_00270	35616..36260		TetR family transcriptional regulator [nite]			ANT_17620 :TetR family transcriptional regulator : : : Anaerolinea thermophila UNI-1	transcriptional regulator		F8AY78_FRADG Transcriptional regulator, TetR family OS=Frankia symbiont subsp. Datisca glomerata GN=FsymDg_20 86 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Bacterial regulatory proteins, tetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU36	CDS	GOHSU_36_00280	complement(36250..37992)		hypothetical protein [nite]			ROP_16160 :hypothetical protein : : : Rhodococcus opacus B4	Protein of unknown function (DUF1446)		Q5YNQ7_NOCFA Putative uncharacterized protein OS=No cardia farcinica (strain IFM 10152) GN=NFA_53320 PE=4 SV=1 ;Protein of unknown function (DUF1446) [Saccharomono spora marina XMU15];PFAM: Protein of unknown function (DUF1446);Protein of unknown function (DUF1446); pfam07287
GOHSU36	CDS	GOHSU_36_00290	complement(37989..38822)		oxidoreductase [nite]			ROP_34130 :oxidoreductase : : : Rhodococcus opacus B4	2,4-dienoyl-CoA reductase FadH	fadH3	F6EUI2_SPHCR 2,4-dienoyl-CoA reductase (NADPH) OS=S phingobium chlorophenolicum L-1 GN=Sphch_0176 PE=4 SV=1;2,4-dienoyl-CoA reductase FadH [Cupriavidus necator N- 1];type strain of Cupriavidus necator;short chain dehydrogenase; Provisional;Trans-2-enoyl-CoA reductase (TER) and 2,4-dienoyl-CoA reductase (DECR), atypical (a) SDR; cd05369;NAD(P) binding site [chemical binding];substrate binding site [chemical binding];homotetramer interface [polypeptide binding];homodimer interface [polypeptide binding]
GOHSU36	CDS	GOHSU_36_00300	complement(38854..40032)		acyl-CoA dehydrogenase [nite]	 fadE [nite]		ROP_46630 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase domain-containing protein		H0E9Z6_9ACTN Acyl-CoA dehydrogenase domain protein OS=Patulibacter sp. I11 GN=PAI11_36690 PE=3 SV=1;acyl-CoA dehydrogenase domain-containing protein [M ycobacterium sp. JLS];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;PFAM: acyl-CoA dehydrogenase domain protein; Acyl-CoA dehydrogenase, type 2, C-terminal domain; KEGG: mmc:Mmcs_3203 acyl-CoA dehydrogenase-like protein;Acyl-CoA dehydrogenase, N-terminal domain; pfam02771;Acyl-CoA dehydrogenase; cd00567
GOHSU36	CDS	GOHSU_36_00310	complement(40285..42327)		hypothetical protein [migap]				hypothetical protein		D0L5W3_GORB4 Membrane protein-like protein OS=Gordo nia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_4305 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: bvi:Bcep1808_3473 membrane protein-like protein
GOHSU36	CDS	GOHSU_36_00320	complement(42416..43063)		putative NarL family two-component response regulator [nite]			RER_10780 :putative NarL family two-component response regulator : : : Rhodococcus erythropolis PR4	response regulator receiver protein		H6MYE8_GORPV Putative two-component system response regulator OS=Gordonia polyisoprenivorans (strain DSM 4426 6 / VH2) GN=GPOL_c45800 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: eba:ebA6278 nitrate/nitrite response regulator;Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding]
GOHSU36	CDS	GOHSU_36_00330	complement(43060..44274)		putative two-component histidine kinase [nite]			ROP_52200 :putative two-component histidine kinase : : 2.7.13.- : Rhodococcus opacus B4	signal transduction histidine kinase		D0L5W1_GORB4 ATP-binding region ATPase domain prote in OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_4303 PE=4 SV=1;signal transduction histidine kinase [Saccharomonos pora marina XMU15];Signal predicted by SignalP 3.0 HMM; IMG reference gene:2508559596_SP;PFAM: Histidine kinase-, DNA gyrase B-, and HSP90-like ATPase;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU36	CDS	GOHSU_36_00340	complement(44308..45120)		putative enoyl-CoA hydratase [nite]			RER_44790 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase		G7H1L1_9ACTO Putative enoyl-CoA hydratase OS=Gordon ia araii NBRC 100433 GN=GOARA_046_00050 PE=3 SV=1;enoyl-CoA hydratase [Bradyrhizobium sp. BTAi1];enoyl-CoA hydratase; Provisional;Catalyzes the reversible hydration of unsaturated fatty acyl-CoA to beta-hydroxyacyl-CoA;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU36	CDS	GOHSU_36_00350	complement(45125..45952)		fumarylacetoacetate hydrolase family protein [nite]			ROP_15430 :fumarylacetoacetate hydrolase family protein : : : Rhodococcus opacus B4	fumarylacetoacetate hydrolase family protein		D6KG12_9ACTO Fumarylacetoacetate hydrolase domain-c ontaining protein 2A OS=Streptomyces sp. e14 GN=SSTG_05887 PE=4 SV=1;fumarylacetoacetate hydrolase family protein [Rhodo coccus opacus B4];Fumarylacetoacetate (FAA) hydrolase family; cl11421
GOHSU36	CDS	GOHSU_36_00360	46402..48249		chaperone protein DnaK [hamap]	dnaK [hamap]	MF_00332:chaperone protein DnaK:dnaK 	ROP_55720 :chaperone protein DnaK : dnaK : : Rhodococcus opacus B4	molecular chaperone DnaK	dnaK	H5TV12_9ACTO Chaperone protein DnaK 1 OS=Gordonia s puti NBRC 100414 GN=dnaK PE=3 SV=1;molecular chaperone DnaK [Rhodococcus jostii RHA1];molecular chaperone DnaK; Provisional; PRK00290;heat shock protein 70; assists in folding of nascent polypeptide chains; refolding of misfolded proteins; utilizes ATPase activity to help fold; co-chaperones are DnaJ and GrpE; multiple copies in some bacteria
GOHSU36	CDS	GOHSU_36_00370	48246..48818		protein GrpE [hamap]	grpE [hamap]	MF_01151:protein GrpE:grpE 	ROP_55730 :GrpE protein : grpE : : Rhodococcus opacus B4	heat shock protein GrpE	grpE1	F1YJN7_9ACTO Protein grpE OS=Gordonia neofelifaecis NRRL B-59395 GN=grpE PE=3 SV=1;heat shock protein GrpE [Rhodococcus jostii RHA1];Molecular chaperone GrpE (heat shock protein) [Posttranslational modification, protein turnover, chaperones]; COG0576;GrpE is the adenine nucleotide exchange factor of DnaK (Hsp70)-type ATPases. The GrpE dimer binds to the ATPase domain of Hsp70 catalyzing the dissociation of ADP, which enables rebinding of ATP, one step in the Hsp70 reaction cycle in protein folding; cd00446;dimer interface [polypeptide binding];hsp70 (ATPase domain) interactions [polypeptide binding]
GOHSU36	CDS	GOHSU_36_00380	48838..49044		not assigned						
GOHSU36	CDS	GOHSU_36_00390	48996..50213		chaperone protein DnaJ [hamap]	dnaJ [hamap]	MF_01152:chaperone protein DnaJ:dnaJ 	RER_13530 :chaperone protein DnaJ : dnaJ : : Rhodococcus erythropolis PR4	chaperone protein DnaJ	dnaJ	D0L5F6_GORB4 Chaperone protein DnaJ 2 OS=Gordonia b ronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCT C 10667) GN=dnaJ2 PE=3 SV=1;chaperone protein DnaJ [Rhodococcus opacus B4];chaperone Hsp40; co-chaperone with DnaK; Participates actively in the response to hyperosmotic and heat shock by preventing the aggregation of stress-denatured proteins and by disaggregating proteins, also in an autonomous, dnaK-independent fashion;chaperone protein DnaJ; Provisional;DnaJ domain or J-domain.  DnaJ/Hsp40 (heat shock protein 40) proteins are highly conserved and play crucial roles in protein translation, folding, unfolding, translocation, and degradation. They act primarily by stimulating the ATPase activity of Hsp70s; cd06257;HSP70 interaction site [polypeptide binding];Zinc finger domain of DnaJ and HSP40; cd10719;Zn binding sites [ion binding];C-terminal substrate binding domain of DnaJ and HSP40; cd10747;dimer interface [polypeptide binding]
GOHSU36	CDS	GOHSU_36_00400	50220..50600		putative heat shock protein HspR [nite]	 hspR [nite]		ROP_55750 :putative heat shock protein HspR : hspR : : Rhodococcus opacus B4	putative heat shock protein HspR	hspR	F1YJN5_9ACTO Regulatory protein MerR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_10566 PE=4 SV=1;hspR gene product [Gordonia polyisoprenivorans VH2] ;Helix-Turn-Helix DNA binding domain of the HspR transcription regulator; cd04766;MerR HTH family regulatory protein; pfam13411;DNA binding residues [nucleotide binding];putative dimer interface [polypeptide binding]
GOHSU36	CDS	GOHSU_36_00410	50743..51360	case	probable malonic semialdehyde reductase RutE [hamap]	rutE [hamap]	MF_01204:probable malonic semialdehyde reductase RutE:rutE 	KSE_47850 :putative nitroreductase : : : Kitasatospora setae NBRC 14216	hypothetical protein		Q8FLM1_COREF NADH dehydrogenase/NAD(P)H nitroreduct ase OS=Corynebacterium efficiens (strain DSM 44549 / YS-31 4 / AJ 12310 / JCM 11189 / NBRC 100395) GN=CE2847 PE=4 SV= 1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG0778 Nitroreductase;Nitroreductase-like family 5.  A subfamily of the nitroreductase family containing uncharacterized proteins that are similar to nitroreductase. Nitroreductase catalyzes the reduction of nitroaromatic compounds such as nitrotoluenes, nitrofurans and...; cd02148;putative FMN binding site [chemical binding]
GOHSU36	CDS	GOHSU_36_00420	complement(51540..52697)		putative oxidoreductase [nite]			ROP_55840 :putative oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase		F1YJN3_9ACTO Oxidoreductase FAD/NAD(P)-binding doma in-containing protein OS=Gordonia neofelifaecis NRRL B-593 95 GN=SCNU_10556 PE=3 SV=1;oxidoreductase [Rhodococcus opacus B4];bifunctional nitric oxide dioxygenase/dihydropteridine reductase 2; Provisional;Globins are heme proteins, which bind and transport oxygen. This family summarizes a diverse set of homologous protein domains, including: (1) tetrameric vertebrate hemoglobins, which are the major protein component of erythrocytes and transport oxygen...; cd01040;heme-binding site [chemical binding];The oxygenase reductase FAD/NADH binding domain acts as part of the multi-component bacterial oxygenases which oxidize hydrocarbons using oxygen as the oxidant. Electron transfer is from NADH via FAD (in the oxygenase reductase) and an [2FE-2S]...; cd06187;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding];beta-alpha-beta structure motif;NAD binding pocket [chemical binding]
GOHSU36	CDS	GOHSU_36_00430	52826..53986		amidohydrolase 2 [migap]				amidohydrolase 2		F1YJN2_9ACTO Amidohydrolase 2 OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_10551 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: amidohydrolase 2; KEGG: pla:Plav_1983 amidohydrolase 2;Superfamily of metallo-dependent hydrolases (also called amidohydrolase superfamily) is a large group of proteins that show conservation in their 3-dimensional fold (TIM barrel) and in details of their active site. The vast majority of the members have a...; cl00281
GOHSU36	CDS	GOHSU_36_00440	54122..54997		putative oxidoreductase [nite]			KSE_54840 :putative oxidoreductase : : : Kitasatospora setae NBRC 14216	short-chain dehydrogenase/reductase SDR		G7GKQ4_9ACTO Putative oxidoreductase OS=Gordonia am arae NBRC 15530 GN=GOAMR_14_00230 PE=4 SV=1;short-chain dehydrogenase/reductase SDR [Frankia sp . EuI1c];3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05653;PFAM: short-chain dehydrogenase/reductase SDR; KEGG: fal:FRAAL0345 ketoreductase;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU36	CDS	GOHSU_36_00450	complement(55035..55628)		putative TetR family transcriptional regulator [nite]			RER_07890 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		G7GX76_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_004_00410 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m vanbaalenii PYR- 1];PFAM: regulatory protein, TetR; KEGG: mmc:Mmcs_0086 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU36	CDS	GOHSU_36_00460	55733..56623		conserved hypothetical protein [nite]			RER_00810 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GX77_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_004_00420 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mxa:MXAN_0699 hypothetical protein;Uncharacterized protein conserved in bacteria (DUF2236); cl01472
GOHSU36	CDS	GOHSU_36_00470	56620..57141		conserved hypothetical protein [nite]			RER_47810 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GXX2_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_011_00630 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: spe:Spro_1171 hypothetical protein;GXWXG protein; pfam14231;Domain of unknown function (DUF4334); pfam14232
GOHSU36	CDS	GOHSU_36_00480	complement(57457..>58622)	3x codon frame	hypothetical protein [nite]			ROP_37430 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YHW7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_07408 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF222;Domain of unknown function (DUF222); pfam02720;HNH nucleases; HNH endonuclease signature which is found in viral, prokaryotic, and eukaryotic proteins. The alignment includes members of the large group of homing endonucleases, yeast intron 1 protein, MutS, as well as bacterial colicins, pyocins, and...; cd00085
##GOHSU37	contig00191				58339
GOHSU37	CDS	GOHSU_37_00010	complement(<1..338)	3x codon	glutamate ABC transporter ATP-binding protein [nite]	 gluA [nite]		RER_27580 :glutamate ABC transporter ATP-binding protein : gluA : : Rhodococcus erythropolis PR4	glutamate uptake system ATP-binding protein		H6MZ02_GORPV Putative glutamate ABC transporter ATP -binding protein OS=Gordonia polyisoprenivorans (strain DS M 44266 / VH2) GN=GPOL_c21040 PE=3 SV=1;glutamate uptake system ATP-binding protein [Strept omyces pristinaespiralis ATCC 25486];ABC-type polar amino acid transport system, ATPase component [Amino acid transport and metabolism]; COG1126;HisP and GlnQ are the ATP-binding components of the bacterial periplasmic histidine and glutamine permeases, repectively.  Histidine permease is a multisubunit complex containing the HisQ and HisM integral membrane subunits and two copies of HisP.  HisP...; cd03262;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid
GOHSU37	CDS	GOHSU_37_00020	453..2003		(dimethylallyl)adenosine tRNA methylthiotransferase MiaB [hamap]	miaB [hamap]	MF_01864:(dimethylallyl)adenosine tRNA methylthiotransferase MiaB:miaB 	ROP_67570 :putative tRNA-thiotransferase : miaB : : Rhodococcus opacus B4	(dimethylallyl)adenosine tRNA methylthiotransferase	miaB	F1YMU7_9ACTO (Dimethylallyl)adenosine tRNA methylth iotransferase MiaB OS=Gordonia neofelifaecis NRRL B-59395 GN=miaB PE=3 SV=1;(dimethylallyl)adenosine tRNA methylthiotransferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 2-methylthio-N6-(dimethylallyl)adenosine (ms(2)i(6)A) at position 37 in tRNAs that read codons beginning with uridine from N6-(dimethylallyl)adenosine (i(6)A);(dimethylallyl)adenosine tRNA methylthiotransferase; Provisional;Uncharacterized protein family UPF0004; pfam00919;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site;TRAM domain; cl01282
GOHSU37	CDS	GOHSU_37_00030	2070..2855		hypothetical membrane protein [nite]			RER_27600 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMU6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16209 PE=4 SV=1;integral membrane protein [Rhodococcus equi 103S]
GOHSU37	CDS	GOHSU_37_00040	complement(2852..4201)		hypothetical protein [nite]			ROP_67590 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YMU5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16204 PE=4 SV=1;unnamed protein product [Corynebacterium diphtheria e CDCE 8392];Domain of Unknown Function (DUF349); pfam03993
GOHSU37	CDS	GOHSU_37_00050	4371..5093		hypothetical protein [nite]			ROP_67600 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5TKC0_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_087_00520 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];The N-terminal domain, an extradiol dioxygenase class III subunit B-like domain, of unknown proteins containing a C-terminal AMMECR1 domain; cd07951
GOHSU37	CDS	GOHSU_37_00060	5090..6028		tRNA dimethylallyltransferase [hamap]	miaA [hamap]	MF_00185:tRNA dimethylallyltransferase:miaA 	ROP_67610 :tRNA delta(2)-isopentenylpyrophosphate transferase : miaA : 2.5.1.8 : Rhodococcus opacus B4	tRNA delta(2)-isopentenylpyrophosphate transferase	miaA	H6MZ07_GORPV tRNA dimethylallyltransferase OS=Gordo nia polyisoprenivorans (strain DSM 44266 / VH2) GN=miaA PE =3 SV=1;tRNA delta(2)-isopentenylpyrophosphate transferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;IPP transferase; isopentenyltransferase; involved in tRNA modification; in Escherichia coli this enzyme catalyzes the addition of a delta2-isopentenyl group from dimethylallyl diphosphate to the N6-nitrogen of adenosine adjacent to the anticodon of tRNA species that read codons starting with uracil; further tRNA modifications may occur; mutations in miaA result in defects in translation efficiency and fidelity;tRNA delta(2)-isopentenylpyrophosphate transferase [Translation, ribosomal structure and biogenesis]; COG0324
GOHSU37	CDS	GOHSU_37_00070	6038..6937		diaminopimelate epimerase [hamap]	dapF [hamap]	MF_00197:diaminopimelate epimerase:dapF 	RER_27650 :diaminopimelate epimerase : dapF : 5.1.1.7 : Rhodococcus erythropolis PR4	diaminopimelate epimerase	dapF	F1YMU2_9ACTO Diaminopimelate epimerase OS=Gordonia neofelifaecis NRRL B-59395 GN=dapF PE=3 SV=1;diaminopimelate epimerase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;involved in lysine biosynthesis; DAP epimerase; produces DL-diaminopimelate from LL-diaminopimelate;diaminopimelate epimerase; Provisional; PRK00450;Diaminopimelate epimerase; cl14668
GOHSU37	CDS	GOHSU_37_00080	7020..8444		putative GTP-binding protein HflX [nite]	 hflX [nite]		RER_27660 :putative GTP-binding protein HflX : hflX : : Rhodococcus erythropolis PR4	GTPase		F1YMU1_9ACTO GTPase HflX OS=Gordonia neofelifaecis NRRL B-59395 GN=hflX PE=3 SV=1;GTPase [Corynebacterium glutamicum ATCC 13032];COG2262:GTPases;GTPases [General function prediction only]; COG2262;GTP-binding GTPase N-terminal; pfam13167;HflX GTPase family; cd01878;G1 box;GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box
GOHSU37	CDS	GOHSU_37_00090	complement(8511..8771)		phosphocarrier protein HPr [nite]	 ptsH [nite]		RER_27690 :phosphocarrier protein HPr : ptsH : : Rhodococcus erythropolis PR4	phosphocarrier protein HPr	ptsH	F1YMU0_9ACTO Phosphotransferase system HPr (HPr) fa mily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCN U_16179 PE=4 SV=1;phosphocarrier protein HPr [Rhodococcus opacus B4];Histidine-containing phosphocarrier protein (HPr)-like proteins. HPr is a central component of the bacterial phosphoenolpyruvate sugar phosphotransferase system (PTS). The PTS catalyses the phosphorylation of sugar substrates during their translocation...; cl00206;protein synonym:histidine-containing protein
GOHSU37	CDS	GOHSU_37_00100	complement(8842..11022)		phosphotransferase system enzyme II [nite]			RER_27700 :phosphotransferase system enzyme II : : 2.7.1.69 : Rhodococcus erythropolis PR4	putative PTS fructose-specific enzyme IIABC		F1YMT9_9ACTO PTS system, fructose subfamily, IIC su bunit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16174 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];PTS_IIA, PTS system, fructose/mannitol specific IIA subunit. The bacterial phosphoenolpyruvate: sugar phosphotransferase system (PTS) is a multi-protein system involved in the regulation of a variety of metabolic and transcriptional processes. This...; cd00211;PTS system 2-O-a-mannosyl-D-glycerate specific transporter subunit IIABC; Provisional;phosphorylation site [posttranslational modification];PTS_IIB_fructose: subunit IIB of enzyme II (EII) of the fructose-specific phosphoenolpyruvate:carbohydrate phosphotransferase system (PTS). In this system, EII (also referred to as FruAB) is a fructose-specific permease made up of two proteins (FruA and...; cd05569;P-loop;Phosphotransferase system IIC components, glucose/maltose/N-acetylglucosamine-specific [Carbohydrate transport and metabolism]; cl00557
GOHSU37	CDS	GOHSU_37_00110	complement(11048..11995)		putative 1-phosphofructokinase [nite]	 fruK [nite]		RER_27710 :putative 1-phosphofructokinase : fruK : 2.7.1.56 : Rhodococcus erythropolis PR4	1-phosphofructokinase		D0LBQ2_GORB4 1-phosphofructokinase OS=Gordonia bron chialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 1 0667) GN=Gbro_2218 PE=3 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];1-phosphofructokinase (FruK), minor 6-phosphofructokinase (pfkB) and related sugar kinases. FruK plays an important role in the predominant pathway for fructose utilisation.This group also contains tagatose-6-phophate kinase, an enzyme of the tagatose...; cd01164;identified by match to protein family HMM PF00294;putative substrate binding site [chemical binding];putative ATP binding site [chemical binding]
GOHSU37	CDS	GOHSU_37_00120	complement(11992..12762)		putative DeoR family transcriptional regulator [nite]			RER_27720 :putative DeoR family transcriptional regulator : : : Rhodococcus erythropolis PR4	glucitol operon repressor		F1YMT7_9ACTO Regulatory protein DeoR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_16164 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Transcriptional regulators of sugar metabolism [Transcription / Carbohydrate transport and metabolism]; COG1349;identified by match to protein family HMM PF00455; match to protein family HMM PF08220;Helix-turn-helix domains; cl00088;SugarP_isomerase: Sugar Phosphate Isomerase family; includes type A ribose 5-phosphate isomerase (RPI_A), glucosamine-6-phosphate (GlcN6P) deaminase, and 6-phosphogluconolactonase (6PGL). RPI catalyzes the reversible conversion of ribose-5-phosphate to...; cl00339
GOHSU37	CDS	GOHSU_37_00130	12883..14583		phosphoenolpyruvate--protein phosphotransferase [nite]	 ptsI [nite]		ROP_67710 :phosphoenolpyruvate--protein phosphotransferase : ptsI : 2.7.3.9 : Rhodococcus opacus B4	phosphoenolpyruvate--protein phosphotransferase		F1YMT6_9ACTO Phosphoenolpyruvate-protein phosphotra nsferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16 159 PE=3 SV=1;phosphoenolpyruvate--protein phosphotransferase [Rh odococcus jostii RHA1];Phosphoenolpyruvate-protein kinase (PTS system EI component in bacteria) [Carbohydrate transport and metabolism]; COG1080;PEP-utilising enzyme, N-terminal; pfam05524;PEP-utilising enzyme, mobile domain; cl01586;Pyruvate kinase (PK):  Large allosteric enzyme that regulates glycolysis through binding of the substrate, phosphoenolpyruvate, and one or more allosteric effectors. Like other allosteric enzymes, PK has a high substrate affinity R state and a low...; cl09155
GOHSU37	CDS	GOHSU_37_00140	complement(14714..15421)		LexA repressor [hamap]	lexA [hamap]	MF_00015:LexA repressor:lexA 	RER_27760 :LexA repressor : lexA : 3.4.21.88 : Rhodococcus erythropolis PR4	LexA repressor	lexA	F1YMT5_9ACTO LexA repressor OS=Gordonia neofelifaec is NRRL B-59395 GN=lexA PE=3 SV=1;LexA repressor [Streptomyces clavuligerus ATCC 2706 4];COG:COG1974 SOS-response transcriptional repressors RecA-mediated autopeptidases.;Helix-turn-helix domains; cl00088;LexA repressor; Validated;Peptidase S24 LexA-like proteins are involved in the SOS response leading to the repair of single-stranded DNA within the bacterial cell. This family includes: the lambda repressor CI/C2 family and related bacterial prophage repressor proteins; LexA (EC...; cd06529;Catalytic site [active]
GOHSU37	CDS	GOHSU_37_00150	complement(15539..15859)		not assigned						
GOHSU37	CDS	GOHSU_37_00160	15858..16148		hypothetical protein [nite]			RER_27770 :hypothetical protein : : : Rhodococcus erythropolis PR4	cell wall hydrolase SleB		H6MZQ8_GORPV Putative peptidoglycan-binding LysM do main protein OS=Gordonia polyisoprenivorans (strain DSM 44 266 / VH2) GN=GPOL_c21200 PE=4 SV=1;cell wall hydrolase SleB [Bacillus cellulosilyticus DSM 2522];KEGG: hor:Hore_06410 cell wall hydrolase SleB; PFAM: cell wall hydrolase SleB; peptidoglycan-binding lysin domain; SMART: peptidoglycan-binding LysM;Lysin domain, found in a variety of enzymes involved in bacterial cell wall degradation. This domain may have a general peptidoglycan binding function; cd00118;putative peptidoglycan binding site;FOG: LysM repeat [Cell envelope biogenesis, outer membrane]; COG1388
GOHSU37	CDS	GOHSU_37_00170	16244..16702		transcriptional repressor NrdR [hamap]	nrdR [hamap]	MF_00440:transcriptional repressor NrdR:nrdR 	ROP_67760 :hypothetical protein : : : Rhodococcus opacus B4	ATP-cone domain-containing protein		H5UA86_9ACTO Transcriptional repressor NrdR OS=Gord onia terrae NBRC 100016 GN=nrdR PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];transcriptional regulator NrdR; Validated; PRK00464;PFAM: ATP-cone domain protein; KEGG: aap:NT05HA_1103 transcriptional regulator NrdR;ATP cone domain; pfam03477
GOHSU37	CDS	GOHSU_37_00180	complement(16704..20375)		ATP-dependent RNA helicase HrpA [nite]	 hrpA [nite]		ROP_67770 :ATP-dependent RNA helicase HrpA : hrpA : 3.6.1.- : Rhodococcus opacus B4	ATP-dependent helicase HrpA		F1YMT2_9ACTO ATP-dependent helicase HrpA OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_16139 PE=4 SV=1;ATP-dependent helicase HrpA [Vibrio sp. Ex25];ATP-dependent RNA helicase HrpA; Provisional;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];Helicase associated domain (HA2); cl04503;Oligonucleotide/oligosaccharide-binding (OB)-fold; pfam07717;Domain of unknown function (DUF3418); pfam11898
GOHSU37	CDS	GOHSU_37_00190	20452..21681		putative aminotransferase [nite]			RER_27820 :putative aminotransferase : : 2.6.1.- : Rhodococcus erythropolis PR4	adenosylmethionine-8-amino-7-oxononanoate transaminase		F9VXR2_9ACTO Putative aminotransferase OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_077_00110 PE=3 SV=1;adenosylmethionine-8-amino-7-oxononanoate transamin ase [Streptomyces sp. AA4];Adenosylmethionine-8-amino-7-oxononanoate aminotransferase [Coenzyme metabolism]; COG0161;Acetyl ornithine aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). The major groups in this CD correspond to ornithine aminotransferase, acetylornithine aminotransferase; cd00610;inhibitor-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU37	CDS	GOHSU_37_00200	complement(21748..22287)		alkyl hydroperoxide reductase AhpD [hamap]	ahpD [hamap]	MF_01676:alkyl hydroperoxide reductase AhpD:ahpD 	RER_27860 :alkyl hydroperoxide reductase AhpD : ahpD : : Rhodococcus erythropolis PR4	alkyl hydroperoxide reductase AhpD	ahpD2	H0RHC8_9ACTO Alkyl hydroperoxide reductase AhpD OS= Gordonia polyisoprenivorans NBRC 16320 GN=ahpD PE=3 SV=1;ahpD2 gene product [Gordonia polyisoprenivorans VH2 ];alkylhydroperoxidase, AhpD family; TIGR00777;Carboxymuconolactone decarboxylase family; cl00460
GOHSU37	CDS	GOHSU_37_00210	complement(22355..22942)		alkyl hydroperoxide reductase AhpC [nite]	 ahpC [nite]		RER_27870 :alkyl hydroperoxide reductase AhpC : ahpC : 1.11.1.15 : Rhodococcus erythropolis PR4	putative alkylhydroperoxide reductase		F1YMS6_9ACTO Alkyl hydroperoxide reductase AhpC OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16109 PE=4 SV= 1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Peroxiredoxin [Posttranslational modification, protein turnover, chaperones]; COG0450;Peroxiredoxin (PRX) family, Typical 2-Cys PRX subfamily; PRXs are thiol-specific antioxidant (TSA) proteins, which confer a protective role in cells through its peroxidase activity by reducing hydrogen peroxide, peroxynitrite, and organic hydroperoxides; cd03015;dimer interface [polypeptide binding];decamer (pentamer of dimers) interface [polypeptide binding];catalytic triad [active];peroxidatic and resolving cysteines [active]
GOHSU37	CDS	GOHSU_37_00220	23083..24015		putative oxidative stress response regulatory protein OxyR [nite]	 oxyR [nite]		RER_27880 :putative oxidative stress response regulatory protein OxyR : oxyR : : Rhodococcus erythropolis PR4	putative hydrogen peroxide-inducible genes activator OxyR		F1YMS5_9ACTO LysR substrate-binding protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_16104 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Helix-turn-helix domains; cl00088;DNA-binding transcriptional regulator OxyR; Provisional;The C-terminal substrate-binding domain of the LysR-type transcriptional regulator OxyR, a member of the type 2 periplasmic binding fold protein superfamily; cd08411;dimerization interface [polypeptide binding]
GOHSU37	CDS	GOHSU_37_00230	complement(24064..26016)		putative helicase [nite]			ROP_67880 :putative helicase : : 3.6.1.- : Rhodococcus opacus B4	ATP-dependent helicase		F1YMS4_9ACTO DEAD/DEAH box helicase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 16099 PE=4 SV=1;ATP-dependent helicase [Streptomyces clavuligerus A TCC 27064];COG function: Replication, recombination and repair. COG: COG4581 Superfamily II RNA helicase.;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];Domain of unknown function (DUF3516); pfam12029
GOHSU37	CDS	GOHSU_37_00240	complement(26178..27161)		hypothetical protein [nite]			ROP_67900 :hypothetical protein : : : Rhodococcus opacus B4	ATP-grasp superfamily enzyme		F1YMS3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16094 PE=4 SV=1;ATP-grasp superfamily enzyme [Saccharomonospora cya nea NA- 134];PFAM: PAC2 family;PAC2 family; cl00847
GOHSU37	CDS	GOHSU_37_00250	27329..27970		hypothetical protein [nite]			RER_27980 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMS2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16089 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.992) with cleavage site probability 0.884 at residue 21
GOHSU37	CDS	GOHSU_37_00260	28127..29269		hypothetical protein [nite]			ROP_67930 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YMS1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16084 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Domain of unknown function (DUF4192); pfam13830
GOHSU37	CDS	GOHSU_37_00270	complement(29295..29975)		iron-dependent regulator IdeR [nite]	 ideR [nite]		RER_28010 :iron-dependent regulator IdeR : ideR : : Rhodococcus erythropolis PR4	iron-dependent repressor IdeR	ideR1	F1YMS0_9ACTO Iron dependent repressor OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_16079 PE=4 SV=1;ideR1 gene product [Gordonia polyisoprenivorans VH2 ];Helix-turn-helix domains; cl00088;Helix-turn-helix diphteria tox regulatory element; smart00529;Iron dependent repressor, metal binding and dimerisation domain; pfam02742;FeoA domain; cl00838
GOHSU37	CDS	GOHSU_37_00280	30072..31316		putative acetoin utilization protein [nite]			RER_28020 :putative acetoin utilization protein : : : Rhodococcus erythropolis PR4	histone deacetylase		F1YMR9_9ACTO Histone deacetylase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_16074 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Deacetylases, including yeast histone deacetylase and acetoin utilization protein [Chromatin structure and dynamics / Secondary metabolites biosynthesis, transport, and catabolism]; COG0123;PFAM: histone deacetylase superfamily; KEGG: gsu:GSU1222 histone deacetylase/AcuC/AphA family protein;Histone deacetylase domain; cl02986
GOHSU37	CDS	GOHSU_37_00290	31309..34113		putative acetate--CoA ligase [nite]			ROP_67970 :putative acetate--CoA ligase : : 6.2.1.13 : Rhodococcus opacus B4	N-acetyltransferase GCN5		F1YMR8_9ACTO CoA-binding domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16069 PE=4 SV=1;N-acetyltransferase GCN5 [Nakamurella multipartita DSM 44233];PFAM: GCN5-related N-acetyltransferase; CoA-binding domain protein; KEGG: dal:Dalk_1024 acetyl coenzyme A synthetase (ADP forming), alpha domain protein;Acetyltransferase (GNAT) domain; pfam13420;N-Acyltransferase superfamily: Various enzymes that characteristically catalyze the transfer of an acyl group to a substrate; cd04301;Coenzyme A binding pocket [chemical binding];acetyl coenzyme A synthetase (ADP forming), alpha domain; TIGR02717;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;CoA-ligase; cl02894;ATP-grasp domain; cl03087
GOHSU37	CDS	GOHSU_37_00300	complement(34187..35155)		RNA polymerase sigma factor SigB [nite]	 sigB [nite]		ROP_67980 :RNA polymerase sigma factor SigB : sigB : : Rhodococcus opacus B4	sigma-70 family RNA polymerase sigma factor		F1YMR7_9ACTO RNA polymerase sigma factor OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_16064 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: RNA polymerase sigma factor, sigma-70 family; PFAM: sigma-70 region 3 domain protein; sigma-70 region 2 domain protein; sigma-70 region 1.2; sigma-70 region 4 domain protein; KEGG: PpSig2; plastid sigma facor SIG2;RNA polymerase sigma factor SigB; Reviewed;Sigma-70 factor, region 1.2; pfam00140;Sigma-70 region 2; pfam04542;Sigma-70 region 3; pfam04539;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU37	CDS	GOHSU_37_00310	complement(35284..35772)		hypothetical protein [migap]				hypothetical protein		H0RHB7_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_068_00390 PE =4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF3099); pfam11298
GOHSU37	CDS	GOHSU_37_00320	35860..36108		conserved hypothetical protein [nite]			RER_28070 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMR1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16034 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF3039);Protein of unknown function (DUF3039); pfam11238
GOHSU37	CDS	GOHSU_37_00330	complement(36113..37165)		conserved hypothetical membrane protein [nite]			RER_28080 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	ribonuclease BN		F1YMR0_9ACTO Ribonuclease BN OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_16029 PE=4 SV=1;ribonuclease BN [Nakamurella multipartita DSM 44233 ];PFAM: ribonuclease BN; KEGG: cak:Caul_0983 ribonuclease BN;Virulence factor BrkB; cl07918
GOHSU37	CDS	GOHSU_37_00340	37250..39034		conserved hypothetical protein [nite]			RER_28100 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	type III restriction protein res subunit		F1YMQ9_9ACTO Type III restriction protein res subun it OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16024 PE =4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: type III restriction protein res subunit; SMART: DEAD/DEAH box helicase; KEGG: sde:Sde_1925 GGDEF family protein;DNA or RNA helicases of superfamily II [Transcription / DNA replication, recombination, and repair]; COG1061;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding]
GOHSU37	CDS	GOHSU_37_00350	complement(39110..39298)		hypothetical protein [nite]			ROP_68040 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GVU5_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_71_00730 PE=4 SV=1;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;KEGG: hypothetical protein
GOHSU37	CDS	GOHSU_37_00360	complement(39432..40037)		putative TetR family transcriptional regulator [nite]			ROP_20620 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	regulatory protein TetR		F1YMQ7_9ACTO Transcriptional regulator, TetR family protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16 014 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];PFAM: regulatory protein, TetR; KEGG: nfa:nfa34140 putative transcriptional regulator;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU37	CDS	GOHSU_37_00370	40200..41003		putative oxidoreductase [nite]			ROP_20580 :putative oxidoreductase : : : Rhodococcus opacus B4	short-chain dehydrogenase/reductase SDR		F1YMQ5_9ACTO Short-chain dehydrogenase/reductase SD R OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16004 PE= 4 SV=1;unnamed protein product [Nocardioides sp. JS614];PFAM: short-chain dehydrogenase/reductase SDR; KR; KEGG: sco:SCO1692 oxidoreductase;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU37	CDS	GOHSU_37_00380	41051..41518		putative 3-hydroxyacyl-thioester dehydratase [nite]			ROP_27200 :putative 3-hydroxyacyl-thioester dehydratase : : 4.2.1.- : Rhodococcus opacus B4	enoyl-CoA hydratase		F1YMQ2_9ACTO Enoyl-CoA hydratase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_15989 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: sma:SAV_6603 hypothetical protein; PFAM: MaoC domain protein dehydratase;NodN (nodulation factor N) contains a single hot dog fold similar to those of the peroxisomal Hydratase-Dehydrogenase-Epimerase (HDE) protein, and the fatty acid synthase beta subunit.  Rhizobium and related species form nodules on the roots of their...; cd03450;putative active site [active];putative catalytic site [active]
GOHSU37	CDS	GOHSU_37_00390	41518..42567		hypothetical protein [nite]			KSE_18170 :hypothetical protein : : : Kitasatospora setae NBRC 14216	phosphotransferase		F1YMQ1_9ACTO Aminoglycoside phosphotransferase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_15984 PE=4 SV=1 ;phosphotransferase [Rhodococcus jostii RHA1];Predicted aminoglycoside phosphotransferase [General function prediction only];Acyl-CoA dehydrogenase (ACAD) 10 and 11, N-terminal domain, and similar proteins. This subfamily is part of a larger superfamily that includes the catalytic domains of other kinases, such as the typical serine/threonine/tyrosine protein kinases (PKs); cd05154;putative active site [active];putative substrate binding site [chemical binding];ATP binding site [chemical binding]
GOHSU37	CDS	GOHSU_37_00400	42586..43812		hypothetical protein [nite]			KSE_18180 :hypothetical protein : : : Kitasatospora setae NBRC 14216	acyl-CoA dehydrogenase		F1YMQ0_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 15979 PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus jostii RHA1];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU37	CDS	GOHSU_37_00410	complement(43919..45388)		RNA polymerase sigma factor SigA [nite]	 sigA [nite]		RER_28160 :RNA polymerase sigma factor SigA : sigA : : Rhodococcus erythropolis PR4	RNA polymerase sigma factor		F1YMP2_9ACTO RNA polymerase sigma factor OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_15939 PE=3 SV=1;RNA polymerase sigma factor [Corynebacterium glutam icum ATCC 13032];sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released;RNA polymerase sigma factor SigB; Reviewed;Sigma-70 factor, region 1.2; pfam00140;Sigma-70 region 2; pfam04542;Sigma-70 region 3; pfam04539;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU37	CDS	GOHSU_37_00420	complement(45595..46392)		polyphosphate glucokinase [nite]	 ppgK [nite]		RER_28170 :polyphosphate glucokinase : ppgK : 2.7.1.63 : Rhodococcus erythropolis PR4	polyphosphate--glucose phosphotransferase		F1YMP1_9ACTO Polyphosphate--glucose phosphotransfer ase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15934 P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ROK family protein; KEGG: asa:ASA_3376 ROK family protein;FGGY family of carbohydrate kinases, N-terminal domain; cl09121
GOHSU37	CDS	GOHSU_37_00430	46483..47364		inositol monophosphatase SuhB [nite]	 suhB [nite]		RER_28180 :inositol monophosphatase SuhB : suhB : 3.1.3.25 : Rhodococcus erythropolis PR4	inositol monophosphate phosphatase SuhB	suhB1	H0RHA9_9ACTO Inositol monophosphatase SuhB OS=Gordo nia polyisoprenivorans NBRC 16320 GN=suhB PE=4 SV=1;suhB1 gene product [Gordonia polyisoprenivorans VH2 ];IMPase, inositol monophosphatase and related domains. A family of Mg++ dependent phosphatases, inhibited by lithium, many of which may act on inositol monophosphate substrate. They dephosphorylate inositol phosphate to generate inositol, which may be...; cd01639;dimerization interface [polypeptide binding]
GOHSU37	CDS	GOHSU_37_00440	complement(47371..48024)		hypothetical membrane protein [nite]			ROP_68110 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H6MZT2_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c21440 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];LytR cell envelope-related transcriptional attenuator; pfam13399
GOHSU37	CDS	GOHSU_37_00450	48260..48559		conserved hypothetical protein [nite]			RER_28200 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMN8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15919 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];Domain of unknown function (DUF4193); pfam13834;KEGG: tfu:Tfu_1939 hypothetical protein
GOHSU37	CDS	GOHSU_37_00460	complement(48908..49429)		conserved hypothetical protein [nite]			RER_28210 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMN7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15914 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3093); pfam11292
GOHSU37	CDS	GOHSU_37_00470	49480..49962		deoxyuridine 5'-triphosphate nucleotidohydrolase [hamap]	dut [hamap]	MF_00116:deoxyuridine 5'-triphosphate nucleotidohydrolase:dut 	RER_28220 :deoxyuridine 5'-triphosphate nucleotidohydrolase : dut : 3.6.1.23 : Rhodococcus erythropolis PR4	deoxyuridine 5'-triphosphate nucleotidohydrolase	dut	F1YMN6_9ACTO Deoxyuridine 5'-triphosphate nucleotid ohydrolase OS=Gordonia neofelifaecis NRRL B-59395 GN=dut P E=3 SV=1;deoxyuridine 5'-triphosphate nucleotidohydrolase [C orynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of dUMP from dUTP;Trimeric dUTP diphosphatases; cd07557;trimer interface [polypeptide binding]
GOHSU37	CDS	GOHSU_37_00480	49985..50839		hypothetical protein [migap]				hypothetical protein		F1YMN5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15904 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 5 : No homology to any previously reported sequences;Protein of unknown function (DUF3710); pfam12502
GOHSU37	CDS	GOHSU_37_00490	complement(50840..51544)		hypothetical protein [nite]			ROP_68160 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H5TUX9_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_005_00250 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: sme:SMc03810 hypothetical protein;Alpha/beta hydrolase family; pfam12695;Alpha/beta hydrolase family; pfam12697
GOHSU37	CDS	GOHSU_37_00500	51624..51995		hypothetical protein [nite]			ROP_68170 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YMN3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15894 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];RecG_wedge_OBF: A subfamily of OB folds corresponding to the OB fold found in the N-terminal (wedge) domain of Escherichia coli RecG. RecG is a branched-DNA-specific helicase, which catalyzes the interconversion of a DNA replication fork to a...; cd04488;ssDNA binding site;generic binding surface II
GOHSU37	CDS	GOHSU_37_00510	complement(52032..52463)		hypothetical membrane protein [nite]			ROP_00850 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YMN2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15889 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU37	CDS	GOHSU_37_00520	complement(52463..53674)		flavohemoprotein [nite]	 hmp [nite]		RER_55430 :flavohemoprotein : hmp : 1.14.12.17 : Rhodococcus erythropolis PR4	flavohemoprotein		F1YMN1_9ACTO Oxidoreductase FAD-binding domain-cont aining protein OS=Gordonia neofelifaecis NRRL B-59395 GN=S CNU_15884 PE=3 SV=1;flavohemoprotein [Streptomyces coelicolor A3(2)];bifunctional nitric oxide dioxygenase/dihydropteridine reductase 2; Provisional;SC6D11.24, hmpA1, possible flavohemoprotein, len: 398 aa. Highly similar to many protein of the flavohemoprotein subfamily, including: Escherichia coli SW:HMPA_ECOLI(EMBL:X58872) flavohemoprotein (hemoglobin-like protein) (396 aa), fasta scores: opt: 826 z-score: 935.2 E():0 39.4% identity in 404 aa overlap and Erwinia chrysanthemi SW:HMPA_ERWCH (EMBL:X75893) flavohemoprotein (395 aa), fasta scores: opt: 761 z-score: 862.0 E():0 35.9% identity in 404 aa overlap. Contains Pfam matches to entries Pfam match to entry PF00042 globin, Globin and Pfam match to entry PF00175 oxidored_fad, Oxidoreductase FAD/NAD-binding domain.;Globins are heme proteins, which bind and transport oxygen. This family summarizes a diverse set of homologous protein domains, including: (1) tetrameric vertebrate hemoglobins, which are the major protein component of erythrocytes and transport oxygen...; cd01040;heme-binding site [chemical binding];FAD_NAD(P)H binding domain of flavohemoglobin. Flavohemoglobins have a globin domain containing a B-type heme fused with a ferredoxin reductase-like FAD/NAD-binding domain. Flavohemoglobins detoxify nitric oxide (NO) via an NO dioxygenase reaction. The...; cd06184;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding];beta-alpha-beta structure motif;NAD binding pocket [chemical binding];Heme binding  pocket [chemical binding]
GOHSU37	CDS	GOHSU_37_00530	complement(53671..54207)		putative Rrf2 family DNA-binding protein [nite]			ROP_19730 :putative Rrf2 family DNA-binding protein : : : Rhodococcus opacus B4	Rrf2 DNA-binding transcription regulator	rrf2A	G7GMV0_9ACTO Putative Rrf2 family DNA-binding prote in OS=Gordonia amarae NBRC 15530 GN=GOAMR_24_00490 PE=4 SV =1;Rrf2 DNA-binding transcription regulator [Corynebac terium resistens DSM 45100];Predicted transcriptional regulator;Helix-turn-helix domains; cl00088
GOHSU37	CDS	GOHSU_37_00540	complement(54303..54989)		Trk system potassium uptake protein TrkA [nite]	 trkA [nite]		RER_28280 :Trk system potassium uptake protein TrkA : trkA : : Rhodococcus erythropolis PR4	putative potassium transporter		F1YMM9_9ACTO TrkA-N domain-containing protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15874 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];K+ transport systems, NAD-binding component [Inorganic ion transport and metabolism]; COG0569;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Anion permease ArsB/NhaD.  These permeases have been shown to translocate sodium, arsenate, antimonite, sulfate and organic anions across biological membranes in all three kingdoms of life.  A typical anion permease contains 8-13 transmembrane helices...; cl15357
GOHSU37	CDS	GOHSU_37_00550	complement(54986..55657)		Trk system potassium uptake protein TrkA [nite]	 trkA [nite]		ROP_68200 :Trk system potassium uptake protein TrkA : trkA : : Rhodococcus opacus B4	putative potassium transporter		F9VR73_9ACTO Trk system potassium uptake protein Tr kA OS=Gordonia alkanivorans NBRC 16433 GN=trkA PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];K+ transport systems, NAD-binding component [Inorganic ion transport and metabolism]; COG0569;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Anion permease ArsB/NhaD.  These permeases have been shown to translocate sodium, arsenate, antimonite, sulfate and organic anions across biological membranes in all three kingdoms of life.  A typical anion permease contains 8-13 transmembrane helices...; cl15357
GOHSU37	CDS	GOHSU_37_00560	55661..57718		hypothetical membrane protein [nite]			ROP_68210 :hypothetical membrane protein : : : Rhodococcus opacus B4	amino acid transporter		F1YMM7_9ACTO Amino acid transporter OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_15864 PE=4 SV=1;amino acid transporter [Rhodococcus jostii RHA1];tc: 2.A.3;Amino acid transporters [Amino acid transport and metabolism]; COG0531
GOHSU37	CDS	GOHSU_37_00570	57732..>58339	3x codon	putative RNA methyltransferase [nite]			ROP_68220 :putative RNA methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	putative RNA methyltransferase		F1YMM6_9ACTO SAM-dependent methyltransferase OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_15859 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];SAM-dependent methyltransferases related to tRNA (uracil-5-)-methyltransferase [Translation, ribosomal structure and biogenesis]; COG2265;TRAM domain; cl01282
##GOHSU38	contig00201				52660
GOHSU38	CDS	GOHSU_38_00010	<1..357	codon	putative acyl-CoA dehydrogenase [nite]			ACMV_16190 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Acidiphilium multivorum AIU301	acyl-CoA dehydrogenase domain-containing protein		H5ULI3_9ACTO Putative acyl-CoA dehydrogenase OS=Gor donia terrae NBRC 100016 GN=GOTRE_175_00710 PE=3 SV=1;acyl-CoA dehydrogenase domain-containing protein [S treptomyces sp. SPB78];acyl-CoA dehydrogenase; Provisional
GOHSU38	CDS	GOHSU_38_00020	complement(388..1212)		probable monoacylglycerol lipase [nite]			RER_53090 :probable monoacylglycerol lipase : : 3.1.1.23 : Rhodococcus erythropolis PR4	putative monoglyceride lipase		F1YF38_9ACTO Acylglycerol lipase OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_02457 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Lysophospholipase [Lipid metabolism]; COG2267;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU38	CDS	GOHSU_38_00030	complement(1209..1988)		aquaporin Z [nite]	 aqpZ [nite]		MLP_38760 :aquaporin Z : aqpZ : : Microlunatus phosphovorus NM-1	aquaporin Z		H0RIV9_9ACTO Aquaporin Z OS=Gordonia polyisoprenivo rans NBRC 16320 GN=aqpZ PE=3 SV=1;aquaporin Z [Pusillimonas sp. T7- 7];Major intrinsic protein (MIP) superfamily. Members of the MIP superfamily function as membrane channels that selectively transport water, small neutral molecules, and ions out of and between cells. The channel proteins share a common fold: the N-terminal...; cd00333;amphipathic channel;Asn-Pro-Ala signature motifs
GOHSU38	CDS	GOHSU_38_00040	complement(2039..2434)		hypothetical protein [migap]				hypothetical protein		F1YF36_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02447 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Reovirus sigma C capsid protein; pfam04582
GOHSU38	CDS	GOHSU_38_00050	2571..3548		conserved hypothetical protein [nite]			RER_49550 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H0RGT8_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_064_01270 PE =3 SV=1;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Similar to Pirin in Homo sapiens (O00625).; Similar to Pirin-like protein in Solanum lycopersicum (Q9SEE4).;Pirin-related protein [General function prediction only];Cupin domain; cl09118;Pirin C-terminal cupin domain; pfam05726
GOHSU38	CDS	GOHSU_38_00060	3545..4027		putative MarR family transcriptional regulator [nite]			ROP_pROB01-03760 :putative MarR family transcriptional regulator : : : Rhodococcus opacus B4	MarR family transcriptional regulator		H6MRP3_GORPV Putative transcriptional regulator, Ma rR-family OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c14580 PE=4 SV=1;MarR family transcriptional regulator [Amycolatopsi s mediterranei U32];Helix-turn-helix domains; cl00088
GOHSU38	CDS	GOHSU_38_00070	4024..4779		conserved hypothetical protein [nite]			RER_49560 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	pirin		F1YF33_9ACTO Pirin domain-containing protein OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_02432 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: pirin; KEGG: bph:Bphy_2162 pirin domain-containing protein;Pirin-related protein [General function prediction only];Cupin domain; cl09118
GOHSU38	CDS	GOHSU_38_00080	4818..6152		putative M20A family peptidase [nite]			ROP_35530 :putative M20A family peptidase : : 3.4.-.- : Rhodococcus opacus B4	beta-Ala-His dipeptidase		F1YF32_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02427 PE=4 SV=1;beta-Ala-His dipeptidase [Pseudonocardia dioxanivor ans CB1190];Acetylornithine deacetylase/Succinyl-diaminopimelate desuccinylase and related deacylases [Amino acid transport and metabolism]; COG0624;KEGG: cai:Caci_6602 hypothetical protein; PFAM: Peptidase M20; Peptidase M20, dimerisation;M20 Dipeptidases; cd03893;metal binding site [ion binding];dimer interface [polypeptide binding]
GOHSU38	CDS	GOHSU_38_00090	complement(6168..7028)		hypothetical protein [migap]				hypothetical protein		H5UCH3_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_040_00060 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.999 at residue 28;KEGG: reu:Reut_A1174 porin
GOHSU38	CDS	GOHSU_38_00100	complement(7197..8153)		hypothetical protein [migap]				hypothetical protein		F1YF30_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02417 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: similar to nucleoporin 62kDa
GOHSU38	CDS	GOHSU_38_00110	complement(8209..9660)		serine/threonine protein kinase-like protein [migap]				serine/threonine protein kinase-like protein		D0L771_GORB4 Serine/threonine protein kinase-relate d protein (Precursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1582 P E=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: serine/threonine protein kinase-related; tyrosine protein kinase; SMART: serine/threonine protein kinase; tyrosine protein kinase; KEGG: mxa:MXAN_3338 serine/threonine protein kinase;Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
GOHSU38	CDS	GOHSU_38_00120	complement(9662..10279)		putative TetR family transcriptional regulator [nite]			RER_38530 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		F1YF28_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_02407 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;DNA-binding transcriptional repressor FabR; Provisional;Similar to Q5Z0A3_NOCFA 5e-35;Helix-turn-helix domains; cl00088
GOHSU38	CDS	GOHSU_38_00130	10423..11499		putative NADPH oxidoreductase [nite]			ROP_11200 :putative NADPH oxidoreductase : : : Rhodococcus opacus B4	putative FAD/NAD(P)-binding domain-containing oxidoreductase		F1YF27_9ACTO Oxidoreductase FAD/NAD(P)-binding doma in-containing protein OS=Gordonia neofelifaecis NRRL B-593 95 GN=SCNU_02402 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Iron-sulfur binding ferredoxin reductase (FNR) proteins combine the FAD and NAD(P) binding regions of FNR with an iron-sulfur binding cluster domain. Ferredoxin-NADP+ (oxido)reductase is an FAD-containing enzyme that catalyzes the reversible electron...; cd06216;Flavodoxin reductases (ferredoxin-NADPH reductases) family 1 [Energy production and conversion]; COG1018;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding];beta-alpha-beta structure motif;NAD binding pocket [chemical binding];2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cl00159
GOHSU38	CDS	GOHSU_38_00140	11529..12728		acyl-CoA desaturase [nite]			RER_38510 :acyl-CoA desaturase : : 1.14.19.1 : Rhodococcus erythropolis PR4	fatty acid desaturase		F1YF26_9ACTO Fatty acid desaturase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_02397 PE=4 SV=1;fatty acid desaturase [Rhodococcus jostii RHA1];Fatty acid desaturase [Lipid metabolism]; COG3239;The Delta6 Fatty Acid Desaturase (Delta6-FADS)-like CD includes the integral-membrane enzymes: delta-4, delta-5, delta-6, delta-8, delta-8-sphingolipid, and delta-11 desaturases found in vertebrates, higher plants, fungi, and bacteria. These desaturases...; cd03506;putative di-iron ligands [ion binding]
GOHSU38	CDS	GOHSU_38_00150	12867..13097		hypothetical protein [nite]			ROP_11180 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F6EKT4_AMYSD Putative uncharacterized protein OS=Am ycolicicoccus subflavus (strain DSM 45089 / DQS3-9A1) GN=A S9A_2967 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU38	CDS	GOHSU_38_00160	13172..14920		ATP-dependent DNA helicase RecQ [nite]	 recQ [nite]		KRH_23310 :ATP-dependent DNA helicase RecQ : recQ : 3.6.1.- : Kocuria rhizophila DC2201	ATP-dependent DNA helicase RecQ		F1YF22_9ACTO Superfamily II DNA helicase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_02377 PE=4 SV=1;unnamed protein product [Cellulomonas flavigena DSM 20109];type strain of Cellulomonas flavigena;TIGRFAM: ATP-dependent DNA helicase RecQ; ATP- dependent DNA helicase, RecQ family; PFAM: RQC domain; DEAD/DEAH box helicase domain protein; helicase domain protein; HRDC domain protein; KEGG: sen:SACE_3331 ATP-dependent DNA helicase; SMART: DEAD-like helicase ; helicase domain protein; HRDC domain protein;ATP-dependent DNA helicase RecQ; TIGR01389;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];RQC domain; cl09632;HRDC domain; cl02578
GOHSU38	CDS	GOHSU_38_00170	15028..15555		hypothetical protein [nite]			MLP_27660 :hypothetical protein : : : Microlunatus phosphovorus NM-1	integral membrane protein		G7H614_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_067_00780 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];membrane protein;Domain of unknown function (DUF4395); pfam14340
GOHSU38	CDS	GOHSU_38_00180	15622..16956		flavin-containing monooxygenase [migap]				flavin-containing monooxygenase		D6A0J1_9ACTO Flavin-containing monooxygenase 3 OS=S treptomyces ghanaensis ATCC 14672 GN=SSFG_02661 PE=4 SV=1;flavin-containing monooxygenase [Citreicella sp. SE 45];identified by match to protein family HMM PF00743;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU38	CDS	GOHSU_38_00190	complement(16949..17881)		putative phosphatase [nite]			RER_42800 :putative phosphatase : : : Rhodococcus erythropolis PR4	exopolyphosphatase	ppx2	D0L762_GORB4 Ppx/GppA phosphatase OS=Gordonia bronc hialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10 667) GN=Gbro_1573 PE=4 SV=1;exopolyphosphatase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Ppx/GppA phosphatase family; cl15449
GOHSU38	CDS	GOHSU_38_00200	complement(17878..18393)		conserved hypothetical protein [nite]			RER_42810 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YF19_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02362 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF501); cl00652
GOHSU38	CDS	GOHSU_38_00210	complement(18390..18875)		conserved hypothetical protein [nite]			RER_42820 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	septum formation initiator		H0R5Q2_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_115_00430 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];Septum formation initiator; cl11433
GOHSU38	CDS	GOHSU_38_00220	complement(19039..20316)		enolase [hamap]	eno [hamap]	MF_00318:enolase:eno 	RER_42830 :enolase : eno : 4.2.1.11 : Rhodococcus erythropolis PR4	enolase	eno	F1YF17_9ACTO Enolase OS=Gordonia neofelifaecis NRRL B-59395 GN=eno PE=3 SV=1;eno gene product [Gordonia polyisoprenivorans VH2];enolase; Provisional; PRK00077;Enolase: Enolases are homodimeric enzymes that catalyse the reversible dehydration of 2-phospho-D-glycerate to phosphoenolpyruvate as part of the glycolytic and gluconeogenesis pathways. The reaction is facilitated by the presence of metal ions; cd03313;dimer interface [polypeptide binding];metal binding site [ion binding];substrate binding pocket [chemical binding]
GOHSU38	CDS	GOHSU_38_00230	complement(20380..21129)		hypothetical protein [nite]			ROP_58400 :hypothetical protein : : : Rhodococcus opacus B4	membrane-bound lytic murein transglycosylase B- like protein		F9VXN3_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_075_00020 PE=4 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.972) with cleavage site probability 0.918 at residue 29;KEGG: aav:Aave_3237 lytic murein transglycosylase;Membrane-bound lytic murein transglycosylase B [Cell envelope biogenesis, outer membrane]; COG2951;Lytic Transglycosylase (LT)  and Goose Egg White Lysozyme (GEWL) domain. Members include the soluble and insoluble membrane-bound LTs in bacteria, the LTs in bacteriophage lambda, as well as, the eukaryotic 'goose-type' lysozymes (GEWL).  LTs catalyze...; cd00254;N-acetyl-D-glucosamine binding site [chemical binding]
GOHSU38	CDS	GOHSU_38_00240	21282..21839		putative TetR family transcriptional regulator [nite]			RER_15860 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	Transcriptional regulator, TetR family		H0QX17_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia effusa NBRC 100432 GN=GOEFS_028_00320 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU38	CDS	GOHSU_38_00250	21929..23092		hypothetical protein [nite]			ROP_25350 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0QX16_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_028_00310 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: scl:sce5500 hypothetical protein;Uncharacterized protein conserved in bacteria (DUF2236); cl01472
GOHSU38	CDS	GOHSU_38_00260	23089..24213		polyketide cyclase/dehydrase [migap]				polyketide cyclase/dehydrase		H0QX15_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_028_00300 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: polyketide cyclase/dehydrase;START/RHO_alpha_C/PITP/Bet_v1/CoxG/CalC (SRPBCC) ligand-binding domain superfamily; cl14643;hydrophobic ligand binding site
GOHSU38	CDS	GOHSU_38_00270	24275..25147		putative amino acid ABC transporter amino acid-binding protein [nite]			ROP_07130 :putative amino acid ABC transporter amino acid-binding protein : : : Rhodococcus opacus B4	amino acid ABC transporter		Q5YYK6_NOCFA Putative transporter substrate-binding protein OS=Nocardia farcinica (strain IFM 10152) GN=NFA_1 8890 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00497;Bacterial extracellular solute-binding proteins, family 3; pfam00497;Bacterial periplasmic transport systems use membrane-bound complexes and substrate-bound, membrane-associated, periplasmic binding proteins (PBPs) to transport a wide variety of  substrates, such as, amino acids, peptides, sugars, vitamins and inorganic...; cd00134;substrate binding pocket [chemical binding];membrane-bound complex binding site;hinge residues
GOHSU38	CDS	GOHSU_38_00280	25147..26121		hypothetical protein [nite]			YM304_28150 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	dipeptidylaminopeptidase/acylaminoacyl- peptidase-like protein		H5TV41_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_006_00400 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: msm:MSMEG_5171 hypothetical protein;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697;substrate binding pocket [chemical binding]
GOHSU38	CDS	GOHSU_38_00290	complement(26226..27491)		nucleoside triphosphate pyrophosphohydrolase [migap]				nucleoside triphosphate pyrophosphohydrolase		F1YPS3_9ACTO Nucleoside triphosphate pyrophosphohyd rolase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1967 2 PE=4 SV=1;nucleoside triphosphate pyrophosphohydrolase [Mycob acterium marinum M];functions in degradation of stringent response intracellular messenger ppGpp; in Escherichia coli this gene is co-transcribed with the toxin/antitoxin genes mazEF; activity of MazG is inhibited by MazEF in vitro; ppGpp inhibits mazEF expression; MazG thus works in limiting the toxic activity of the MazF toxin induced during starvation; MazG also interacts with the GTPase protein Era;nucleoside triphosphate pyrophosphohydrolase; Reviewed;MazG nucleotide pyrophosphohydrolase domain; cl00345
GOHSU38	CDS	GOHSU_38_00300	27643..27894		epoxide hydrolase [migap]				epoxide hydrolase		A6WH51_KINRD Putative uncharacterized protein OS=Ki neococcus radiotolerans (strain ATCC BAA-149 / DSM 14245 / SRS30216) GN=Krad_4682 PE=4 SV=1;epoxide hydrolase [Agrobacterium radiobacter K84];Alpha/beta hydrolase family; pfam12697
GOHSU38	CDS	GOHSU_38_00310	complement(27905..28150)		not assigned						I1RST3_GIBZE Uncharacterized protein OS=Gibberella zeae (strain PH-1 / ATCC MYA-4620 / FGSC 9075 / NRRL 31084 ) GN=FG07217.1 PE=4 SV=1
GOHSU38	CDS	GOHSU_38_00320	28337..31033		type III restriction enzyme res subunit [migap]				type III restriction enzyme res subunit		A1SNC4_NOCSJ Type III restriction enzyme, res subun it OS=Nocardioides sp. (strain BAA-499 / JS614) GN=Noca_38 09 PE=4 SV=1;type III restriction enzyme res subunit [Mycobacter ium sp. JDM601];N-terminal putative catalytic domain of uncharacterized prokaryotic and archeal HKD family nucleases fused to a DEAD/DEAH box helicase domain; cd09203;putative homodimer interface [polypeptide binding];putative active site [active];catalytic site [active];DNA or RNA helicases of superfamily II [Transcription / DNA replication, recombination, and repair]; COG1061;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];Domain of unknown function (DUF3427); pfam11907
GOHSU38	CDS	GOHSU_38_00330	31196..33598		hypothetical protein [nite]			KRH_23020 :hypothetical protein : : : Kocuria rhizophila DC2201	hypothetical protein		E4WFM3_RHOE1 Putative uncharacterized protein OS=Rh odococcus equi (strain 103S) GN=REQ_16540 PE=4 SV=1;unnamed protein product [Ruegeria sp. TM1040];Protein of unknown function DUF262; cl14890;PFAM: protein of unknown function DUF262: (5.4e-07) protein of unknown function DUF1524 RloF: (3.3e-42); KEGG: mac:MA2417 hypothetical protein, ev=0.0, 61% identity
GOHSU38	CDS	GOHSU_38_00340	complement(33630..37205)		transcription-repair coupling factor [nite]	 mfd [nite]		RER_43010 :transcription-repair coupling factor : mfd : : Rhodococcus erythropolis PR4	transcription-repair coupling factor	mfd	F1YPS7_9ACTO Transcription-repair coupling factor O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_19692 PE=4 S V=1;transcription-repair coupling factor [Corynebacteri um glutamicum ATCC 13032];IS fingerprint type 4-5;Transcription-repair coupling factor (superfamily II helicase) [DNA replication, recombination, and repair / Transcription]; COG1197;CarD-like/TRCF domain; cl00588;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding];TRCF domain; cl04088
GOHSU38	CDS	GOHSU_38_00350	37289..38011		putative TetR family transcriptional regulator [nite]			YM304_32770 :putative TetR family transcriptional regulator : : : Acidimicrobidae bacterium YM16-304	TetR family transcriptional regulator		D2SD03_GEOOG Transcriptional regulator, TetR family OS=Geodermatophilus obscurus (strain ATCC 25078 / DSM 431 60 / JCM 3152 / G-20) GN=Gobs_3775 PE=4 SV=1;TetR family transcriptional regulator [Frankia sp. EuI1c];PFAM: regulatory protein TetR; KEGG: sna:Snas_2904 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU38	CDS	GOHSU_38_00360	complement(38040..38861)		putative YrbE family protein [nite]	 yrbE2A [nite]		ROP_18130 :putative YrbE family protein : yrbE2A : : Rhodococcus opacus B4	YrbE family protein	yrbE3A	H0QYR7_9ACTO YrbE family protein OS=Gordonia effusa NBRC 100432 GN=yrbEA PE=4 SV=1;YrbE family protein [Nocardia farcinica IFM 10152];Permease; cl00510
GOHSU38	CDS	GOHSU_38_00370	complement(38858..39559)		hypothetical membrane protein [nite]			ROP_18050 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H0R5M8_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_115_00190 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: nfa:nfa6070 hypothetical protein
GOHSU38	CDS	GOHSU_38_00380	complement(39628..40569)		putative Mce family protein [nite]	 mce2F [nite]		ROP_18060 :putative Mce family protein : mce2F : : Rhodococcus opacus B4	hypothetical protein		D0L5I8_GORB4 Mammalian cell entry related domain pr otein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 432 47 / JCM 3198 / NCTC 10667) GN=Gbro_1219 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: hypothetical protein; KEGG: nfa:nfa11200  Mce family protein;ABC-type transport system involved in resistance to organic solvents, periplasmic component [Secondary metabolites biosynthesis, transport, and catabolism]; COG1463;mce related protein; pfam02470
GOHSU38	CDS	GOHSU_38_00390	complement(40572..41597)		putative Mce family protein [nite]	 mce2E [nite]		ROP_18070 :putative Mce family protein : mce2E : : Rhodococcus opacus B4	hypothetical protein		D0L5I7_GORB4 Mammalian cell entry related domain pr otein (Precursor) OS=Gordonia bronchialis (strain ATCC 255 92 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1218 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: hypothetical protein; KEGG: nfa:nfa11190  Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU38	CDS	GOHSU_38_00400	complement(41594..42649)		putative Mce family protein [nite]	 mce2D [nite]		ROP_18080 :putative Mce family protein : mce2D : : Rhodococcus opacus B4	hypothetical protein		G7H7G1_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceD PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: hypothetical protein; KEGG: nfa:nfa11180  Mce family protein;virulence factor Mce family protein; TIGR00996
GOHSU38	CDS	GOHSU_38_00410	complement(42643..43617)		putative Mce family protein [nite]	 mce2C [nite]		ROP_18090 :putative Mce family protein : mce2C : : Rhodococcus opacus B4	hypothetical protein		H0R5M4_9ACTO Mce family protein OS=Gordonia effusa NBRC 100432 GN=mceC PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: hypothetical protein; KEGG: nfa:nfa11170  Mce family protein;virulence factor Mce family protein; TIGR00996
GOHSU38	CDS	GOHSU_38_00420	complement(43605..44624)		putative Mce family protein [nite]	 mce2B [nite]		ROP_18100 :putative Mce family protein : mce2B : : Rhodococcus opacus B4	virulence factor Mce family protein		G7H7F9_9ACTO Mce family protein OS=Gordonia araii N BRC 100433 GN=mceB PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: mce related protein; TIGRFAM: virulence factor Mce family protein;virulence factor Mce family protein; TIGR00996;mce related protein; pfam02470
GOHSU38	CDS	GOHSU_38_00430	complement(44621..45643)		not assigned						H0QYS4_9ACTO Mce family protein OS=Gordonia effusa NBRC 100432 GN=mceA PE=4 SV=1
GOHSU38	CDS	GOHSU_38_00440	complement(45648..46505)		putative YrbE family protein [nite]	 yrbE2B [nite]		ROP_18120 :putative YrbE family protein : yrbE2B : : Rhodococcus opacus B4	organic solvents resistance ABC transporter permease		G7H7F7_9ACTO YrbE family protein OS=Gordonia araii NBRC 100433 GN=yrbEB PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Domain of unknown function DUF140; TIGRFAM: conserved hypothetical integral membrane protein;Permease; cl00510
GOHSU38	CDS	GOHSU_38_00450	complement(46564..47550)		conserved hypothetical protein [nite]			RER_20960 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLV7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14506 PE=4 SV=1;unnamed protein product [Mycobacterium ulcerans Agy 99];cytoplasmic protein;Uncharacterized protein conserved in bacteria (DUF2236); cl01472
GOHSU38	CDS	GOHSU_38_00460	47687..48184		hypothetical membrane protein [nite]			ROP_38730 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		G7GYY3_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_021_00450 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU38	CDS	GOHSU_38_00470	48227..48742		hypothetical protein [nite]			ROP_30540 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GYY4_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_021_00460 PE=4 SV=1;unnamed protein product [Bradyrhizobium japonicum U SDA 110];Protein of unknown function (DUF461); cl01071
GOHSU38	CDS	GOHSU_38_00480	48739..49962	overlap	putative peroxidase [nite]			RER_11620 :putative peroxidase : : 1.11.1.- : Rhodococcus erythropolis PR4	peroxidase		G7GYY5_9ACTO Putative peroxidase OS=Gordonia araii NBRC 100433 GN=GOARA_021_00470 PE=4 SV=1;peroxidase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Tat-translocated enzyme; TIGR01412;Dyp-type peroxidase family; cl01067
GOHSU38	CDS	GOHSU_38_00490	complement(49866..50621)	overlap	putative TetR family transcriptional regulator [nite]			RER_43090 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR-family transcriptional regulator		F1YPT5_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_19732 PE=4 SV=1;TetR-family transcriptional regulator [Streptomyces clavuligerus ATCC 27064];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU38	CDS	GOHSU_38_00500	complement(50965..51864)		not assigned						
GOHSU38	CDS	GOHSU_38_00510	complement(51861..52163)		not assigned						
GOHSU38	rRNA	GOHSU_38_r0010	complement(52509..52624)		not assigned						identified by RNAmmer; putative
##GOHSU39	contig00211				1744
GOHSU39	CDS	GOHSU_39_00010	<1..268	3x codon frame	not assigned						
GOHSU39	CDS	GOHSU_39_00020	375..695		ESAT-6-like protein [migap]				ESAT-6-like protein		F1YJF8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09974 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Proteins of 100 residues with WXG; cl02005
GOHSU39	CDS	GOHSU_39_00030	763..1059		hypothetical protein [nite]			ROP_62260 :hypothetical protein : : : Rhodococcus opacus B4	ESAT-6-like protein esxL		F1YJF9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09979 PE=4 SV=1;ESAT-6-like protein esxL [Mycobacterium tuberculosi s 02_1987];Proteins of 100 residues with WXG; cl02005
GOHSU39	CDS	GOHSU_39_00040	1186..>1744	3x codon	putative CDF family transporter [nite]			RER_21600 :putative CDF family transporter : : : Rhodococcus erythropolis PR4	cation diffusion facilitator family transporter		H6N062_GORPV Putative cation diffusion facilitator family transporter OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c35020 PE=4 SV=1;cation diffusion facilitator family transporter [No cardiopsis dassonvillei subsp. dassonvillei DSM 43111];COGs: COG0053 Co/Zn/Cd cation transporter; InterPro IPR002524; KEGG: saq:Sare_0901 cation diffusion facilitator family transporter; PFAM: cation efflux protein; SPTR: C1YIH7 Cation diffusion facilitator family transporter; TIGRFAM: cation diffusion facilitator family transporter; PFAM: Cation efflux family; TIGRFAM: cation diffusion facilitator family transporter;Predicted Co/Zn/Cd cation transporters [Inorganic ion transport and metabolism]; COG0053;Cation efflux family; cl00316
##GOHSU40	contig00212				48392
GOHSU40	CDS	GOHSU_40_00010	<1..216	codon	putative CDF family transporter [nite]			RER_21600 :putative CDF family transporter : : : Rhodococcus erythropolis PR4	cation diffusion facilitator family transporter		F1YJG2_9ACTO Cation diffusion facilitator family tr ansporter OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_0 9994 PE=4 SV=1;cation diffusion facilitator family transporter [St ackebrandtia nassauensis DSM 44728];TIGRFAM: cation diffusion facilitator family transporter; PFAM: cation efflux protein; KEGG: mrd:Mrad2831_3754 cation diffusion facilitator family transporter;Cation efflux family; cl00316
GOHSU40	CDS	GOHSU_40_00020	361..1914		NSS family transporter [nite]			KRH_05450 :NSS family transporter : : : Kocuria rhizophila DC2201	transporter		F9VZN2_9ACTO Transporter OS=Gordonia alkanivorans N BRC 16433 GN=GOALK_097_00990 PE=3 SV=1;transporter [Amycolicicoccus subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;Sodium:neurotransmitter symporter family; cl11976
GOHSU40	CDS	GOHSU_40_00030	1911..2099		predicted protein [migap]				predicted protein		F9VZN1_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_097_00980 PE=4 SV= 1;predicted protein [Streptomyces ghanaensis ATCC 146 72]
GOHSU40	CDS	GOHSU_40_00040	2137..3780		putative amino acid transporter [nite]			ROP_65360 :putative amino acid transporter : : : Rhodococcus opacus B4	amino acid transporter		Q0S2H8_RHOSR Cationic amino acid transport protein OS=Rhodococcus sp. (strain RHA1) GN=RHA1_ro06483 PE=4 SV=1 ;amino acid transporter [Rhodococcus opacus B4];amino acid transporter; TIGR00909;Spore germination protein; cl15802
GOHSU40	CDS	GOHSU_40_00050	3922..5391		adenosylhomocysteinase [nite]	 ahcY [nite]		RER_21660 :adenosylhomocysteinase : ahcY : 3.3.1.1 : Rhodococcus erythropolis PR4	S-adenosyl-L-homocysteine hydrolase		F1YJG4_9ACTO Adenosylhomocysteinase OS=Gordonia neo felifaecis NRRL B-59395 GN=ahcY PE=3 SV=1;S-adenosyl-L-homocysteine hydrolase [Actinosynnema mirum DSM 43827];catalyzes the formation of L-homocysteine from S-adenosyl-L-homocysteine;Adenosylhomocysteinase; Provisional;S-adenosyl-L-homocysteine hydrolase (AdoHycase) catalyzes the hydrolysis of S-adenosyl-L-homocysteine (AdoHyc) to form adenosine (Ado) and homocysteine (Hcy). The equilibrium lies far on the side of AdoHyc synthesis, but in nature the removal of Ado and...; cd00401;oligomerization interface [polypeptide binding];NAD+ binding site [chemical binding]
GOHSU40	CDS	GOHSU_40_00060	complement(5618..6250)	case	thymidylate kinase [hamap]	tmk [hamap]	MF_00165:thymidylate kinase:tmk 	ROP_63840 :thymidylate kinase : tmk : 2.7.4.9 : Rhodococcus opacus B4	thymidylate kinase	tmk	F1YJG5_9ACTO Thymidylate kinase OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_10009 PE=3 SV=1;thymidylate kinase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Thymidylate kinase [Nucleotide transport and metabolism]; COG0125;Thymidine monophosphate kinase (TMPK), also known as thymidylate kinase, catalyzes the phosphorylation of thymidine monophosphate (TMP) to thymidine diphosphate (TDP) utilizing ATP as its preferred phophoryl donor. TMPK represents the rate-limiting step...; cd01672;catalyzes the reversible phosphoryl transfer from adenosine triphosphate (ATP) to thymidine monophosphate (dTMP) to form thymidine diphosphate (dTDP);TMP-binding site;ATP-binding site [chemical binding]
GOHSU40	CDS	GOHSU_40_00070	complement(6307..6849)		RNA polymerase ECF subfamily sigma factor [nite]			OBV_10080 :RNA polymerase ECF subfamily sigma factor : : : Oscillibacter valericigenes Sjm18-20	RNA polymerase ECF-type sigma factor		H5UL88_9ACTO Putative RNA polymerase ECF-type sigma factor OS=Gordonia terrae NBRC 100016 GN=GOTRE_170_00270 PE=4 SV=1;RNA polymerase ECF-type sigma factor [Symbiobacteri um thermophilum IAM 14863];RNA polymerase sigma factor RpoE; Provisional;Sigma-70 region 2; pfam04542;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU40	CDS	GOHSU_40_00080	complement(6851..7444)		not assigned						H0JYV4_9NOCA Co-chaperone GrpE OS=Rhodococcus pyrid inivorans AK37 GN=AK37_24766 PE=3 SV=1
GOHSU40	CDS	GOHSU_40_00090	complement(7509..7889)		RNA polymerase sigma factor SigF [migap]				RNA polymerase sigma factor SigF		RNA polymerase sigma factor SigF [Cyanothece sp. PC C 7425];sigma factors are initiation factors that promote the attachment of RNA polymerase to specific initiation sites and are then released; this sigma factor controls the expression of genes coding cell surface proteins involved in motility and growth on nitrogen;RNA polymerase sigma factor SigF; Reviewed;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU40	CDS	GOHSU_40_00100	8097..9524		conserved hypothetical protein [nite]			RER_13460 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	isoniazid inductible protein IniC		F9VU22_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_048_00730 PE=4 SV= 1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;G1 box;GTP/Mg2+ binding site [chemical binding];G2 box;Switch I region;G3 box;Switch II region;G4 box;G5 box
GOHSU40	CDS	GOHSU_40_00110	9521..11440		hypothetical protein [migap]				hypothetical protein		D0L505_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_1149 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mca:MCA1193 hypothetical protein;Dynamin family; pfam00350;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;G3 box;Switch II region;GTP/Mg2+ binding site [chemical binding];G4 box;G5 box
GOHSU40	CDS	GOHSU_40_00120	complement(11465..11860)		ATP-dependent helicase [migap]				ATP-dependent helicase		G7MH32_MACMU Putative uncharacterized protein (Frag ment) OS=Macaca mulatta GN=EGK_00263 PE=4 SV=1;unnamed protein product [Herminiimonas arsenicoxyda ns];Genoscope sequence ID : Herminiimonas;Evidence 3 : Function proposed based on presence of conserved amino acid motif, structural feature or limited homology; Product type pe :  enzyme;Rad3-related DNA helicases [Transcription / DNA replication, recombination, and repair]; COG1199
GOHSU40	CDS	GOHSU_40_00130	12049..12477		hypothetical protein [migap]				hypothetical protein		F4FBB4_VERMA Putative uncharacterized protein OS=Ve rrucosispora maris (strain AB-18-032) GN=VAB18032_05420 PE =4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GA21679 gene product from transcript GA21679- RA
GOHSU40	CDS	GOHSU_40_00140	12600..13286		two-component response regulator MtrA [nite]	 mtrA [nite]		RER_21680 :two-component response regulator MtrA : mtrA : : Rhodococcus erythropolis PR4	two component system response regulator MtrA	mtrA	G7GQI2_9ACTO Two-component response regulator MtrA OS=Gordonia amarae NBRC 15530 GN=mtrA PE=4 SV=1;mtrA gene product [Gordonia polyisoprenivorans VH2] ;Response regulators consisting of a CheY-like receiver domain and a winged-helix DNA-binding domain [Signal transduction mechanisms / Transcription]; COG0745;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];Effector domain of response regulator. Bacteria and certain eukaryotes like protozoa and higher plants use two-component signal transduction systems to detect and respond to changes in the environment. The system consists of a sensor histidine kinase and...; cd00383;DNA binding site [nucleotide binding]
GOHSU40	CDS	GOHSU_40_00150	13358..15100		two-component histidine kinase MtrB [nite]	 mtrB [nite]		RER_21690 :two-component histidine kinase MtrB : mtrB : 2.7.13.- : Rhodococcus erythropolis PR4	ATP-binding protein		H5U5A7_9ACTO Two-component histidine kinase MtrB OS =Gordonia sputi NBRC 100414 GN=mtrB PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ATP-binding region ATPase domain protein; histidine kinase HAMP region domain protein; histidine kinase A domain protein; SMART: ATP-binding region ATPase domain protein; histidine kinase A domain protein; histidine kinase HAMP region domain protein; KEGG: gbm:Gbem_2673 heavy metal sensor signal transduction histidine kinase;Histidine kinase, Adenylyl cyclase, Methyl-accepting protein, and Phosphatase (HAMP) domain. HAMP is a signaling domain which occurs in a wide variety of signaling proteins, many of which are bacterial. The HAMP domain consists of two alpha helices...; cd06225;dimerization interface [polypeptide binding];Histidine Kinase A (dimerization/phosphoacceptor) domain; Histidine Kinase A dimers are formed through parallel association of 2 domains creating 4-helix bundles; usually these domains contain a conserved His residue and are activated via...; cd00082;dimer interface [polypeptide binding];phosphorylation site [posttranslational modification];Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU40	CDS	GOHSU_40_00160	15097..16824		hypothetical protein [nite]			ROP_63870 :hypothetical protein : : : Rhodococcus opacus B4	lipoprotein LpqB, beta-propeller domain-like protein		F1YJG8_9ACTO Lipoprotein LpqB OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_10024 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.733) with cleavage site probability 0.452 at residue 37;PFAM: Lipoprotein LpqB, beta-propeller domain-like; Lipoprotein LpqB, GerMN domain; KEGG: dol:Dole_2286 YadA domain-containing protein;lipoprotein LpqB; Provisional;Sporulation and spore germination; cl11253
GOHSU40	CDS	GOHSU_40_00170	16917..17621		conserved hypothetical protein [nite]			RER_21710 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJG9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10029 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Predicted amidophosphoribosyltransferases [General function prediction only]; COG1040
GOHSU40	CDS	GOHSU_40_00180	17906..18643		conserved hypothetical protein [nite]			RER_21720 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	S30AE family protein		F1YJH0_9ACTO Ribosomal subunit interface protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10034 PE=4 SV =1;S30AE family protein [Mycobacterium tuberculosis T4 6];RaiA ('ribosome-associated inhibitor A', also known as Protein Y (PY), YfiA, and SpotY,  is a stress-response protein that binds the ribosomal subunit interface and arrests translation by interfering with aminoacyl-tRNA binding to the ribosomal A site; cd00552;30S subunit binding site
GOHSU40	CDS	GOHSU_40_00190	18788..21544		protein translocase subunit SecA [hamap]	secA [hamap]	MF_01382:protein translocase subunit SecA:secA 	RER_21730 :preprotein translocase SecA subunit : secA : : Rhodococcus erythropolis PR4	preprotein translocase subunit SecA	secA	F1YJH1_9ACTO Protein translocase subunit SecA 1 OS= Gordonia neofelifaecis NRRL B-59395 GN=secA1 PE=3 SV=1;preprotein translocase subunit SecA [Corynebacteriu m glutamicum ATCC 13032];IS fingerprint type 4-5;preprotein translocase subunit SecA; Reviewed;functions in protein export; can interact with acidic membrane phospholipids and the SecYEG protein complex; binds to preproteins; binds to ATP and undergoes a conformational change to promote membrane insertion of SecA/bound preprotein; ATP hydrolysis appears to drive release of the preprotein from SecA and deinsertion of SecA from the membrane; additional proteins SecD/F/YajC aid SecA recycling; exists in an equilibrium between monomers and dimers; may possibly form higher order oligomers; proteins in this cluster correspond SecA1; SecA2 is not essential and seems to play a role in secretion of a subset of proteins;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;SecA preprotein cross-linking domain; cl03078;Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding]
GOHSU40	CDS	GOHSU_40_00200	21911..22495		hypothetical protein [migap]				hypothetical protein		F1YJH2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10044 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU40	CDS	GOHSU_40_00210	22504..24060		putative acyltransferase [nite]			ROP_63930 :putative acyltransferase : : 2.3.1.- : Rhodococcus opacus B4	acyltransferase		F1YJH3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10049 PE=4 SV=1;acyltransferase [Pseudonocardia dioxanivorans CB119 0];KEGG: ami:Amir_6300 acyltransferase, WS/DGAT/MGAT; TIGRFAM: Acyltransferase, WS/DGAT/MGAT; PFAM: O-acyltransferase, WSD1, N-terminal;acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007
GOHSU40	CDS	GOHSU_40_00220	24060..24557		hypothetical protein [nite]			ROP_63940 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7CNA9_MYCTH Putative uncharacterized protein OS=My cobacterium thermoresistibile ATCC 19527 GN=KEK_22534 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU40	CDS	GOHSU_40_00230	24669..25790		NADPH oxidoreductase [nite]			ROP_63950 :NADPH oxidoreductase : : : Rhodococcus opacus B4	oxidoreductase FAD-binding domain-containing protein		F1YJH6_9ACTO Oxidoreductase FAD-binding domain-cont aining protein OS=Gordonia neofelifaecis NRRL B-59395 GN=S CNU_10064 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: oxidoreductase FAD-binding domain protein; ferredoxin; oxidoreductase FAD/NAD(P)-binding domain protein; KEGG: abo:ABO_1717 flavodoxin reductases (ferredoxin-NADPH reductase);Iron-sulfur binding ferredoxin reductase (FNR) proteins combine the FAD and NAD(P) binding regions of FNR with an iron-sulfur binding cluster domain. Ferredoxin-NADP+ (oxido)reductase is an FAD-containing enzyme that catalyzes the reversible electron...; cd06216;Flavodoxin reductases (ferredoxin-NADPH reductases) family 1 [Energy production and conversion]; COG1018;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding];beta-alpha-beta structure motif;NAD binding pocket [chemical binding];2Fe-2S iron-sulfur cluster binding domain. Iron-sulfur proteins play an important role in electron transfer processes and in various enzymatic reactions. The family includes plant and algal ferredoxins, which act as electron carriers in photosynthesis...; cd00207;catalytic loop [active];iron binding site [ion binding]
GOHSU40	CDS	GOHSU_40_00240	25874..27166		acyl-CoA desaturase DesA3 [nite]	 desA3 [nite]		ROP_63970 :acyl-CoA desaturase DesA3 : desA3 : 1.14.19.1 : Rhodococcus opacus B4	fatty acid desaturase		F1YJH7_9ACTO Fatty acid desaturase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_10069 PE=4 SV=1;fatty acid desaturase [Rhodococcus jostii RHA1];Fatty acid desaturase [Lipid metabolism]; COG3239;The Delta6 Fatty Acid Desaturase (Delta6-FADS)-like CD includes the integral-membrane enzymes: delta-4, delta-5, delta-6, delta-8, delta-8-sphingolipid, and delta-11 desaturases found in vertebrates, higher plants, fungi, and bacteria. These desaturases...; cd03506;putative di-iron ligands [ion binding]
GOHSU40	CDS	GOHSU_40_00250	27206..28447		acyl-CoA desaturase [nite]			ROP_63960 :acyl-CoA desaturase : : 1.14.19.1 : Rhodococcus opacus B4	fatty acid desaturase		F1YJH8_9ACTO Fatty acid desaturase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_10074 PE=4 SV=1;fatty acid desaturase [Rhodococcus jostii RHA1];Fatty acid desaturase [Lipid metabolism]; COG3239;The Delta6 Fatty Acid Desaturase (Delta6-FADS)-like CD includes the integral-membrane enzymes: delta-4, delta-5, delta-6, delta-8, delta-8-sphingolipid, and delta-11 desaturases found in vertebrates, higher plants, fungi, and bacteria. These desaturases...; cd03506;putative di-iron ligands [ion binding]
GOHSU40	CDS	GOHSU_40_00260	complement(28461..29519)		putative GTPase [nite]			ROP_63980 :putative GTPase : : 3.6.1.- : Rhodococcus opacus B4	hypothetical protein		D0LF84_GORB4 GTPase EngC OS=Gordonia bronchialis (s train ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=G bro_3589 PE=3 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Differs from Rv3228, Mb3257 and BCG_3351 by 1aa, D169G in Mycobacterium bovis BCG Pasteur and BCG Mexico;Predicted GTPases [General function prediction only];Ribosomal interacting GTPase YjeQ/EngC, a circularly permuted subfamily of the Ras GTPases; cd01854;GTPase/Zn-binding domain interface [polypeptide binding];GTP/Mg2+ binding site [chemical binding];G4 box;G5 box;G1 box;Switch I region;G2 box;G3 box;Switch II region
GOHSU40	CDS	GOHSU_40_00270	complement(29516..30787)	case	3-phosphoshikimate 1-carboxyvinyltransferase [hamap]	aroA [hamap]	MF_00210:3-phosphoshikimate 1-carboxyvinyltransferase:aroA 	RER_21820 :3-phosphoshikimate 1-carboxyvinyltransferase : aroA : 2.5.1.19 : Rhodococcus erythropolis PR4	3-phosphoshikimate 1-carboxyvinyltransferase	aroA	F1YJI0_9ACTO 3-phosphoshikimate 1-carboxyvinyltrans ferase OS=Gordonia neofelifaecis NRRL B-59395 GN=aroA PE=3 SV=1;3-phosphoshikimate 1-carboxyvinyltransferase [Coryn ebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 5-O-(1-carboxyvinyl)-3-phosphoshikimate from phosphoenolpyruvate and 3-phosphoshikimate in tryptophan biosynthesis;3-phosphoshikimate 1-carboxyvinyltransferase; TIGR01356;EPSP synthase domain. 3-phosphoshikimate 1-carboxyvinyltransferase (5-enolpyruvylshikimate-3-phosphate synthase) (EC 2.5.1.19) catalyses the reaction between shikimate-3-phosphate (S3P) and phosphoenolpyruvate (PEP) to form...; cd01556;hinge
GOHSU40	CDS	GOHSU_40_00280	30973..31443		hypothetical protein [nite]			ROP_44250 :hypothetical protein : : : Rhodococcus opacus B4	NUDIX hydrolase		F1YJI2_9ACTO NUDIX hydrolase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_10094 PE=3 SV=1;NUDIX hydrolase [Chelativorans sp. BNC1];PFAM: NUDIX hydrolase; KEGG: sme:SMb20988 hypothetical protein;Nudix hydrolase is a superfamily of enzymes found in all three kingdoms of life, and it catalyzes the hydrolysis of NUcleoside DIphosphates linked to other moieties, X. Enzymes belonging to this superfamily require a divalent cation, such as Mg2+ or Mn2+...; cl00447;nudix motif
GOHSU40	CDS	GOHSU_40_00290	31479..32870		hypothetical membrane protein [nite]			DMR_16510 :hypothetical membrane protein : : : Desulfovibrio magneticus RS-1	amino acid transporter		H5TZF1_9ACTO Putative amino acid transporter OS=Gor donia sputi NBRC 100414 GN=GOSPT_052_00130 PE=4 SV=1;amino acid transporter [Methanococcoides burtonii D SM 6242];amino acid transporter; TIGR00909
GOHSU40	CDS	GOHSU_40_00300	complement(32877..34046)		putative valine--pyruvate transaminase [nite]	 avtA [nite]		MLP_04120 :putative valine--pyruvate transaminase : avtA : 2.6.1.66 : Microlunatus phosphovorus NM-1	pyridoxal-5'-phosphate dependent aminotransferase		F1YJI4_9ACTO Pyridoxal-5'-phosphate dependent amino transferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _10104 PE=3 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];aminotransferase; Validated;Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU40	CDS	GOHSU_40_00310	complement(34122..34955)		aldo/keto reductase [nite]			MLP_25250 :aldo/keto reductase : : : Microlunatus phosphovorus NM-1	2,5-didehydrogluconate reductase		F1YJI5_9ACTO 2,5-didehydrogluconate reductase OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10109 PE=4 SV=1;2,5-didehydrogluconate reductase [Streptomyces sp. SirexAA- E];KEGG: gbr:Gbro_3547 2,5-didehydrogluconate reductase; PFAM: aldo/keto reductase;Aldo-keto reductases (AKRs) are a superfamily of soluble NAD(P)(H) oxidoreductases whose chief purpose is to reduce aldehydes and ketones to primary and secondary alcohols. AKRs are present in all phyla and are of importance to both health and industrial...; cd06660;Aldo/keto reductase family; pfam00248;catalytic tetrad [active]
GOHSU40	CDS	GOHSU_40_00320	35058..35249		hypothetical protein [migap]				hypothetical protein		H0QUV2_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_008_00120 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.663) with cleavage site probability 0.419 at residue 29
GOHSU40	CDS	GOHSU_40_00330	complement(35227..36294)		hypothetical membrane protein [nite]			ROP_40820 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		E4WBY3_RHOE1 Putative integral membrane protein OS= Rhodococcus equi (strain 103S) GN=REQ_02930 PE=4 SV=1;integral membrane protein [Rhodococcus equi 103S]
GOHSU40	CDS	GOHSU_40_00340	complement(36559..37482)		putative oxidoreductase [nite]			RER_35090 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	alcohol dehydrogenase GroES		F1YJI8_9ACTO Alcohol dehydrogenase GroES domain-con taining protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_10124 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: alcohol dehydrogenase GroES domain protein; KEGG: maq:Maqu_0510 alcohol dehydrogenase;Zn-dependent alcohol dehydrogenases [General function prediction only]; COG1064;alcohol dehydrogenase and quinone reductase-like medium chain degydrogenases/reductases; cd05289;putative NAD(P) binding site [chemical binding]
GOHSU40	CDS	GOHSU_40_00350	37586..37843		hypothetical membrane protein [nite]			ROP_02210 :hypothetical membrane protein : : : Rhodococcus opacus B4	transglycosylase-associated protein		E2MW57_9CORY Integral membrane protein OS=Corynebac terium amycolatum SK46 GN=CORAM0001_2084 PE=4 SV=1;transglycosylase-associated protein [Herpetosiphon aurantiacus DSM 785];PFAM: Transglycosylase-associated protein; KEGG: xom:XOO_4288 transglycosylase associated protein;Transglycosylase associated protein; cl00978
GOHSU40	CDS	GOHSU_40_00360	37958..39493		putative flavin-containing monooxygenase [nite]			RER_34410 :putative flavin-containing monooxygenase : : : Rhodococcus erythropolis PR4	monooxygenase		F1YJI9_9ACTO FAD-dependent pyridine nucleotide-disu lfide oxidoreductase OS=Gordonia neofelifaecis NRRL B-5939 5 GN=SCNU_10129 PE=4 SV=1;monooxygenase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Predicted flavoprotein involved in K+ transport [Inorganic ion transport and metabolism]; COG2072;Similar to Q73YD8_MYCPA 0.0;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU40	CDS	GOHSU_40_00370	complement(39514..40098)		hypothetical protein [migap]				hypothetical protein		F1YJJ0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10134 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: msm:MSMEG_2697 hypothetical protein
GOHSU40	CDS	GOHSU_40_00380	complement(40095..40835)		conserved hypothetical protein [nite]			RER_21830 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJJ1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10139 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Uncharacterized ACR, COG2135; cl03646
GOHSU40	CDS	GOHSU_40_00390	40986..41705		RNA polymerase ECF-type sigma factor SigH [nite]	 sigH [nite]		ROP_64020 :RNA polymerase ECF-type sigma factor SigH : sigH : : Rhodococcus opacus B4	RNA polymerase sigma factor RpoE	sigH_1	F1YJJ2_9ACTO RNA polymerase sigma factor OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_10144 PE=3 SV=1;RNA polymerase sigma factor RpoE [Mycobacterium bov is BCG str. Pasteur 1173P2];Member of the extracytoplasmic function sigma factors which are active under specific conditions; binds with the catalytic core of RNA polymerase to produce the holoenzyme and directs bacterial core RNA polymerase to specific promoter elements to initiate transcription; this sigma factor is involved in heat shock and oxidative stress response;RNA polymerase sigma-70 factor, TIGR02947 family;Sigma-70 region 2; pfam04542;Sigma70, region (SR) 4 refers to the most C-terminal of four conserved domains found in Escherichia coli (Ec) sigma70, the main housekeeping sigma, and related sigma-factors (SFs). A SF is a dissociable subunit of RNA polymerase, it directs bacterial or...; cd06171;DNA binding residues [nucleotide binding]
GOHSU40	CDS	GOHSU_40_00400	41702..42004		putative anti-sigma factor [nite]			ROP_64030 :putative anti-sigma factor : : : Rhodococcus opacus B4	putative anti-sigma factor		H0RAF6_9ACTO Putative anti-sigma factor OS=Gordonia polyisoprenivorans NBRC 16320 GN=GOPIP_021_00150 PE=4 SV= 1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Putative zinc-finger; cl15806
GOHSU40	CDS	GOHSU_40_00410	42252..42488		probable biotinylated protein [nite]			RER_21880 :probable biotinylated protein : : : Rhodococcus erythropolis PR4	biotin/lipoyl attachment domain-containing protein		F1YJJ4_9ACTO Putative acetyl-CoA carboxylase biotin carboxyl carrier protein subunit OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_10154 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];PFAM: biotin/lipoyl attachment domain-containing protein; KEGG: fra:Francci3_3765 biotin/lipoyl attachment;The biotinyl-domain or biotin carboxyl carrier protein (BCCP) domain is present in all biotin-dependent enzymes, such as acetyl-CoA carboxylase, pyruvate carboxylase, propionyl-CoA carboxylase, methylcrotonyl-CoA carboxylase, geranyl-CoA carboxylase; cd06850;carboxyltransferase (CT) interaction site;biotinylation site [posttranslational modification]
GOHSU40	CDS	GOHSU_40_00420	42488..44020		putative two-component histidine kinase [nite]			RER_21890 :putative two-component histidine kinase : : 2.7.13.- : Rhodococcus erythropolis PR4	putative two component sensor kinase pdtaS		F1YJJ5_9ACTO Histidine kinase dimerization/phosphoa cceptor OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_101 59 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Differs from Rv3220c by 1aa, N47D in Mycobacterium bovis BCG Pasteur and BCG Mexico;Signal transduction histidine kinase; pfam12282;PAS domain S-box; TIGR00229;Signal transduction histidine kinase [Signal transduction mechanisms];Histidine kinase; cl06527;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cd00075;ATP binding site [chemical binding];Mg2+ binding site [ion binding];G-X-G motif
GOHSU40	CDS	GOHSU_40_00430	complement(44131..44385)		WhiB family regulatory protein [nite]			ROP_64060 :WhiB family regulatory protein : : : Rhodococcus opacus B4	transcriptional regulatory protein whiB-like whiB1	whiB1_1	F1YJJ6_9ACTO Transcription factor WhiB OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10164 PE=4 SV=1;transcriptional regulatory protein whiB-like whiB1 [Mycobacterium bovis BCG str. Pasteur 1173P2];Transcription factor WhiB; pfam02467
GOHSU40	CDS	GOHSU_40_00440	complement(44687..45643)		conserved hypothetical protein [nite]			RER_21910 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJJ7_9ACTO Diacylglycerol kinase catalytic region OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10169 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Sphingosine kinase and enzymes related to eukaryotic diacylglycerol kinase [Lipid metabolism / General function prediction only]; COG1597;Diacylglycerol kinase catalytic domain; cl01255
GOHSU40	CDS	GOHSU_40_00450	45661..46134		conserved hypothetical membrane protein [nite]			RER_21920 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	putative integral membrane protein		F1YJJ8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10174 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico]
GOHSU40	CDS	GOHSU_40_00460	46242..47033		putative Soj/ParA-related protein [nite]			ROP_64110 :putative Soj/ParA-related protein : : : Rhodococcus opacus B4	putative soj/parA-related protein		F1YJJ9_9ACTO Cobyrinic acid ac-diamide synthase OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_10179 PE=4 SV= 1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Differs from Rv3213c by 1aa, E144K in Mycobacterium tuberculosis CDC1551, Mycobacterium bovis, Mycobacterium bovis BCG Pasteur, BCG Tokyo and BCG Mexico;ATPases involved in chromosome partitioning [Cell division and chromosome partitioning]; COG1192;ParA and ParB of Caulobacter crescentus belong to a conserved family of bacterial proteins implicated in chromosome segregation. ParB binds to DNA sequences adjacent to the origin of replication and localizes to opposite cell poles shortly following the...; cd02042;P-loop;Magnesium ion binding site [ion binding]
GOHSU40	CDS	GOHSU_40_00470	complement(47099..48376)		conserved hypothetical protein [nite]			RER_21960 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YJK0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_10184 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
##GOHSU41	contig00221				49590
GOHSU41	CDS	GOHSU_41_00010	complement(<1..1466)	3x codon	phosphoribosylglycinamide formyltransferase 2 [hamap]	purT [hamap]	MF_01643:phosphoribosylglycinamide formyltransferase 2:purT 	ROP_19590 :glycinamide ribonucleotide transformylase PurT : purT : 2.1.2.- : Rhodococcus opacus B4	phosphoribosylglycinamide formyltransferase 2	purT	F1YLR4_9ACTO Phosphoribosylglycinamide formyltransf erase 2 OS=Gordonia neofelifaecis NRRL B-59395 GN=purT PE= 3 SV=1;phosphoribosylglycinamide formyltransferase 2 [Cory nebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;non-folate utilizing enzyme, catalyzes the production of beta-formyl glycinamide ribonucleotide from formate, ATP, and beta-GAR and a side reaction producing acetyl phosphate and ADP from acetate and ATP; involved in de novo purine biosynthesis;phosphoribosylglycinamide formyltransferase 2; Validated; PRK09288;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;ATP-grasp domain; cl03087
GOHSU41	CDS	GOHSU_41_00020	complement(1477..3078)		adenylate cyclase [nite]	 cya [nite]		ROP_11720 :adenylate cyclase : cya : 4.6.1.1 : Rhodococcus opacus B4	adenylate cyclase		F1YLR5_9ACTO Histidine kinase HAMP region domain-co ntaining protein OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_14294 PE=4 SV=1;adenylate cyclase [Mycobacterium marinum M];thought to play a role in regulation of cellular metabolism by catalysing the synthesis of a second messenger, camp [catalytic activity: ATP = 3',5'-cyclic AMP + pyrophosphate];Methyl-accepting chemotaxis protein [Cell motility and secretion / Signal transduction mechanisms]; COG0840;Histidine kinase, Adenylyl cyclase, Methyl-accepting protein, and Phosphatase (HAMP) domain. HAMP is a signaling domain which occurs in a wide variety of signaling proteins, many of which are bacterial. The HAMP domain consists of two alpha helices...; cd06225;dimerization interface [polypeptide binding];Adenylate cyclase, family 3 (some proteins contain HAMP domain) [Signal transduction mechanisms]; COG2114;cyclase homology domain; cd07302;nucleotidyl binding site;metal binding site [ion binding];dimer interface [polypeptide binding]
GOHSU41	CDS	GOHSU_41_00030	complement(3093..4379)		adenylosuccinate synthetase [hamap]	purA [hamap]	MF_00011:adenylosuccinate synthetase:purA 	ROP_56210 :adenylosuccinate synthetase : purA : 6.3.4.4 : Rhodococcus opacus B4	adenylosuccinate synthetase PurA	purA2	F1YLR6_9ACTO Adenylosuccinate synthetase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_14299 PE=3 SV=1;purA2 gene product [Gordonia polyisoprenivorans VH2 ];Adenylosuccinate synthetase; smart00788;Adenylosuccinate synthetase (AdSS) catalyzes the first step in the de novo biosynthesis of AMP. IMP and L-aspartate are conjugated in a two-step reaction accompanied by the hydrolysis of GTP to GDP in the presence of Mg2+. In the first step, the...; cd03108;GDP-binding site [chemical binding];ACT binding site;IMP binding site
GOHSU41	CDS	GOHSU_41_00040	4443..5096		conserved hypothetical protein [nite]			RER_14270 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5U467_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_103_01020 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU41	CDS	GOHSU_41_00050	5093..5890		conserved hypothetical membrane protein [nite]			RER_14260 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	membrane protein		G7GZM6_9ACTO Putative peptidase M50 family protein OS=Gordonia araii NBRC 100433 GN=GOARA_027_00140 PE=4 SV=1 ;membrane protein [Streptomyces clavuligerus ATCC 27 064];Uncharacterized homologs of Site-2 protease (S2P), zinc metalloproteases (MEROPS family M50) which cleave transmembrane domains of substrate proteins, regulating intramembrane proteolysis (RIP) of diverse signal transduction mechanisms. Members of the...; cd06158;putative substrate binding region [chemical binding]
GOHSU41	CDS	GOHSU_41_00060	complement(5942..6613)		conserved hypothetical protein [nite]			RER_06130 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	oligosaccharide deacetylase		G7H6W6_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_078_00400 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];Catalytic NodB homology domain of rhizobial NodB-like proteins; cd10917;NodB motif;catalytic site [active];metal binding site [ion binding]
GOHSU41	CDS	GOHSU_41_00070	6727..7044		Na+/solute symporter [migap]				Na+/solute symporter		A1BFE7_CHLPD Na+/solute symporter OS=Chlorobium pha eobacteroides (strain DSM 266) GN=Cpha266_1083 PE=3 SV=1;Na+/solute symporter [Chlorobium phaeobacteroides D SM 266];PFAM: Na+/solute symporter; KEGG: plt:Plut_1178 sodium:solute symporter family protein;Predicted symporter [General function prediction only]; COG4147;Sodium:solute symporter family; cl00456
GOHSU41	CDS	GOHSU_41_00080	complement(7060..8265)		putative acetyl-CoA acyltransferase [nite]			RER_01420 :putative acetyl-CoA acyltransferase : : 2.3.1.- : Rhodococcus erythropolis PR4	lipid-transfer protein		G7H3C0_9ACTO Putative acetyl-CoA acyltransferase OS =Gordonia araii NBRC 100433 GN=GOARA_056_00920 PE=3 SV=1;acetyl-CoA acyltransferase [Rhodococcus erythropoli s PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;protein synonym:acyl-CoA thiolase;lipid-transfer protein; Provisional;Thiolase domain associated with sterol carrier protein (SCP)-x isoform and related proteins; SCP-2  has multiple roles in intracellular lipid circulation and metabolism. The N-terminal presequence in the SCP-x isoform represents a peroxisomal...; cd00829
GOHSU41	CDS	GOHSU_41_00090	complement(8388..9038)		putative TetR family transcriptional regulator [nite]			ROP_42120 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	transcriptional regulator		G7H3C1_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_056_00930 PE=4 SV=1;transcriptional regulator [Saccharomonospora marina XMU15];PFAM: Bacterial regulatory proteins, tetR family;transcriptional repressor BetI; TIGR03384;Helix-turn-helix domains; cl00088
GOHSU41	CDS	GOHSU_41_00100	complement(9113..10054)		probable cation efflux protein [nite]			RER_14160 :probable cation efflux protein : : : Rhodococcus erythropolis PR4	cation efflux protein		H0RJB7_9ACTO Putative cation efflux protein OS=Gord onia polyisoprenivorans NBRC 16320 GN=GOPIP_077_00940 PE=4 SV=1;cation efflux protein [Rhodococcus erythropolis PR4 ];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Cation efflux family; cl00316
GOHSU41	CDS	GOHSU_41_00110	complement(10104..11252)		conserved hypothetical protein [nite]			RER_14150 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLS0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14319 PE=4 SV=1;unnamed protein product [Corynebacterium pseudotube rculosis FRC41];Predicted membrane protein [Function unknown];Fusaric acid resistance protein-like; cl15844
GOHSU41	CDS	GOHSU_41_00120	complement(11296..11709)		conserved hypothetical protein [nite]			RER_14140 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GKN2_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_13_00130 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];Protein of unknown function (DUF3151); pfam11349
GOHSU41	CDS	GOHSU_41_00130	11856..13199		not assigned						H5UFI3_9ACTO Putative uncharacterized protein (Frag ment) OS=Gordonia terrae NBRC 100016 GN=GOTRE_066_00640 PE =4 SV=1
GOHSU41	CDS	GOHSU_41_00140	complement(13282..14316)		fructose-bisphosphate aldolase class II [nite]	 fbaA [nite]		ROP_56110 :fructose-bisphosphate aldolase class II : fbaA : 4.1.2.13 : Rhodococcus opacus B4	fructose-bisphosphate aldolase	fda	G7H1P0_9ACTO Fructose-bisphosphate aldolase class I I OS=Gordonia araii NBRC 100433 GN=fbaA PE=4 SV=1;fructose-bisphosphate aldolase [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;fructose-bisphosphate aldolase; Provisional;catalyzes the formation of glycerone phosphate and glyceraldehyde 3-phosphate from fructose 1,6, bisphosphate;Class II Type A, Fructose-1,6-bisphosphate (FBP) aldolases. The enzyme catalyses the zinc-dependent, reversible aldol condensation of dihydroxyacetone phosphate with glyceraldehyde-3-phosphate to form fructose-1,6-bisphosphate. FBP aldolase is...; cd00946;intersubunit interface [polypeptide binding];zinc binding site [ion binding];Na+ binding site [ion binding]
GOHSU41	CDS	GOHSU_41_00150	14495..15232		conserved hypothetical membrane protein [nite]			RER_14110 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLS4_9ACTO Membrane-associated protein OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_14339 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];SNARE associated Golgi protein; cl00429
GOHSU41	CDS	GOHSU_41_00160	complement(15251..16906)		putative oxidoreductase [nite]			RER_28410 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	cholesterol oxidase		F1YLS7_9ACTO Oxidoreductase OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_14354 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Choline dehydrogenase and related flavoproteins [Amino acid transport and metabolism]; COG2303;GMC oxidoreductase; pfam05199
GOHSU41	CDS	GOHSU_41_00170	17015..17650		putative TetR family transcriptional regulator [nite]			RER_05640 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		F1YLS8_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14 359 PE=4 SV=1;TetR family transcriptional regulator [Streptomyces violaceusniger Tu 4113];PFAM: Transcriptional regulator, TetR-like, DNA-binding, bacterial/archaeal; KEGG: sro:Sros_7072  TetR family transcriptional regulator;Helix-turn-helix domains; cl00088
GOHSU41	CDS	GOHSU_41_00180	complement(17830..18681)		putative oxidoreductase [nite]			RER_53180 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short chain dehydrogenase		H0QZT8_9ACTO Putative oxidoreductase OS=Gordonia ef fusa NBRC 100432 GN=GOEFS_052_00120 PE=4 SV=1;short chain dehydrogenase [Myxococcus fulvus HW- 1] ;short chain dehydrogenase; Provisional;COG1028 Dehydrogenases with different specificities (related to short-chain alcohol dehydrogenases);human hydroxysteroid dehydrogenase-like protein 2 (HSDL2), classical (c) SDRs; cd09762;NAD(P) binding site [chemical binding];homodimer interface [polypeptide binding]
GOHSU41	CDS	GOHSU_41_00190	complement(18743..19504)		putative 3-hydroxyacyl-CoA dehydrogenase [nite]			RER_56230 :putative 3-hydroxyacyl-CoA dehydrogenase : : 1.1.1.35 : Rhodococcus erythropolis PR4	3-hydroxyacyl-CoA dehydrogenase		Q0SB91_RHOSR Probable 3-hydroxyacyl-CoA dehydrogena se OS=Rhodococcus sp. (strain RHA1) GN=RHA1_ro03392 PE=3 S V=1;3-hydroxyacyl-CoA dehydrogenase [Rhodococcus jostii RHA1];17hydroxysteroid dehydrogenase type 10 (HSD10)-like, classical (c) SDRs; cd05371;3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05557;NAD binding site [chemical binding];homotetramer interface [polypeptide binding];homodimer interface [polypeptide binding]
GOHSU41	CDS	GOHSU_41_00200	complement(19553..20695)		acyl-CoA dehydrogenase [nite]	 fadE [nite]		ROP_30610 :acyl-CoA dehydrogenase : fadE : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase domain-containing protein		I0WUA0_9NOCA Acyl-CoA dehydrogenase domain-containi ng protein OS=Rhodococcus imtechensis RKJ300 GN=W59_10684 PE=4 SV=1;acyl-CoA dehydrogenase domain-containing protein [M ycobacterium sp. JLS];PFAM: acyl-CoA dehydrogenase domain protein; Acyl-CoA dehydrogenase, type 2, C-terminal domain; KEGG: mmc:Mmcs_0099 acyl-CoA dehydrogenase-like protein;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU41	CDS	GOHSU_41_00210	20825..21406		putative TetR family transcriptional regulator [nite]			ROP_30600 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		Q5YTU6_NOCFA Putative transcriptional regulator OS= Nocardia farcinica (strain IFM 10152) GN=NFA_35470 PE=4 SV =1;TetR family transcriptional regulator [Rhodococcus opacus B4];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU41	CDS	GOHSU_41_00220	21450..21776		hypothetical protein [nite]			TEH_18590 :hypothetical protein : : : Tetragenococcus halophilus NBRC 12172	hypothetical protein	yrhK	F9EJ26_9ACTO Putative uncharacterized protein yrhK OS=Actinomyces sp. oral taxon 448 str. F0400 GN=yrhK PE=4 SV=1;yrhK gene product [Halomonas elongata DSM 2581];Uncharacterized protein yrhK - Bacillus subtilis, High confidence in function and specificity;YrhK-like protein; pfam14145
GOHSU41	CDS	GOHSU_41_00230	complement(21833..22492)		putative RNA methyltransferase [nite]			ROP_56060 :putative RNA methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	tRNA/rRNA methyltransferase SpoU		F1YLS9_9ACTO tRNA/rRNA methyltransferase SpoU OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14364 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: tRNA/rRNA methyltransferase (SpoU); KEGG: rop:ROP_56060  RNA methyltransferase;SpoU rRNA Methylase family; cl00362
GOHSU41	CDS	GOHSU_41_00240	complement(22615..23178)		orotate phosphoribosyltransferase [nite]	 pyrE [nite]		ROP_55920 :orotate phosphoribosyltransferase : pyrE : 2.4.2.10 : Rhodococcus opacus B4	orotate phosphoribosyltransferase	pyrE	H6MVU7_GORPV Orotate phosphoribosyltransferase OS=G ordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=pyr E PE=3 SV=1;orotate phosphoribosyltransferase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;involved in fifth step of pyrimidine biosynthesis; converts orotidine 5'-phosphate and diphosphate to orotate and 5-phospho-alpha-D-ribose 1-diphosphate;Phosphoribosyl transferase (PRT)-type I domain; cd06223
GOHSU41	CDS	GOHSU_41_00250	complement(23193..24374)		conserved hypothetical protein [nite]			RER_13930 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLT1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14374 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU41	CDS	GOHSU_41_00260	24456..25667		conserved hypothetical protein [nite]			BBR47_25710 :conserved hypothetical protein : : : Brevibacillus brevis NBRC 100599	acetylornithine deacetylase or succinyl- diaminopimelate desuccinylase		F1YLT2_9ACTO Acetylornithine deacetylase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_14379 PE=4 SV=1;acetylornithine deacetylase or succinyl- diaminopim elate desuccinylase [Sphaerobacter thermophilus DSM 20745] ;succinyl-diaminopimelate desuccinylase; Reviewed;TIGRFAM: acetylornithine deacetylase or succinyl- diaminopimelate desuccinylase; PFAM: peptidase M20; peptidase dimerisation domain protein; KEGG: ote:Oter_2918 peptidase M20;Peptidase M20 acetylornithine deacetylase/succinyl-diaminopimelate desuccinylase (ArgE/DapE)-like; cd08659;metal binding site [ion binding];dimer interface [polypeptide binding]
GOHSU41	CDS	GOHSU_41_00270	complement(25625..26017)	overlap	not assigned						
GOHSU41	CDS	GOHSU_41_00280	complement(25945..26670)	overlap	probable peroxiredoxin [hamap]		MF_00401:probable peroxiredoxin: 	ANT_26270 :putative peroxiredoxin : : 1.11.1.15 : Anaerolinea thermophila UNI-1	3-Cys thioredoxin peroxidase		A1T504_MYCVP 3-Cys thioredoxin peroxidase OS=Mycoba cterium vanbaalenii (strain DSM 7251 / PYR-1) GN=Mvan_1420 PE=4 SV=1;3-Cys thioredoxin peroxidase [Methanohalophilus mah ii DSM 5219];COGs: COG0450 peroxiredoxin; InterPro IPR019479:IPR000866:IPR013740; KEGG: aeh:Mlg_0829 1-Cys peroxiredoxin; PFAM: alkyl hydroperoxide reductase/ thiol specific antioxidant/ Mal allergen; peroxiredoxin-like; redoxin; PRIAM: peroxiredoxin; SPTR: Q12XC5 3-Cys thioredoxin peroxidase; PFAM: C-terminal domain of 1-Cys peroxiredoxin; AhpC/TSA family;peroxiredoxin; Provisional;Peroxiredoxin (PRX) family, 1-cys PRX subfamily; composed of PRXs containing only one conserved cysteine, which serves as the peroxidatic cysteine. They are homodimeric thiol-specific antioxidant (TSA) proteins that confer a protective role in cells by...; cd03016;dimer interface [polypeptide binding];decamer (pentamer of dimers) interface [polypeptide binding];catalytic triad [active]
GOHSU41	CDS	GOHSU_41_00290	26966..27490		predicted protein [migap]				predicted protein		I0V338_9PSEU Uncharacterized protein (Precursor) OS =Saccharomonospora xinjiangensis XJ-54 GN=SacxiDRAFT_2312 PE=4 SV=1;predicted protein [Streptomyces sp. AA4]
GOHSU41	CDS	GOHSU_41_00300	complement(27569..27946)		conserved hypothetical protein [nite]			RER_31390 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	pyridoxamine 5'-phosphate oxidase-related protein		H5UFE7_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_066_00280 PE=4 SV=1;pyridoxamine 5'-phosphate oxidase-related protein [ Streptomyces griseus subsp. griseus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;Pyridoxine 5'-phosphate (PNP) oxidase-like proteins; cl00381
GOHSU41	CDS	GOHSU_41_00310	complement(28042..30600)		chaperone ClpB [nite]	 clpB [nite]		ROP_55880 :chaperone ClpB : clpB : : Rhodococcus opacus B4	ATP-dependent chaperone ClpB		H0QWI3_9ACTO Chaperone ClpB OS=Gordonia effusa NBRC 100432 GN=clpB PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: gme:Gmet_2851 ATPase AAA; TIGRFAM: ATP-dependent chaperone ClpB; PFAM: ATPase AAA; Clp ATPase-like; ATPase AAA central domain protein; ATPase associated with various cellular activities AAA_5; Clp domain protein; SMART: ATPase AAA;ATP-dependent chaperone ClpB; TIGR03346;Clp amino terminal domain; pfam02861;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger;Protein of unknown function (DUF2730); pfam10805;C-terminal, D2-small domain, of ClpB protein; cl11083
GOHSU41	CDS	GOHSU_41_00320	31031..31918		not assigned						
GOHSU41	CDS	GOHSU_41_00330	32018..33763		type I restriction enzyme M protein [nite]	 hsdM [nite]		KRH_05230 :type I restriction enzyme M protein : hsdM : 2.1.1.- : Kocuria rhizophila DC2201	type I restriction-modification system methyltransferase subunit		Q0RV88_RHOSR Type I restriction-modification system methyltransferase subunit OS=Rhodococcus sp. (strain RHA1 ) GN=hsdM PE=4 SV=1;type I restriction-modification system methyltransf erase subunit [Saccharomonospora marina XMU15];Type I restriction-modification system methyltransferase subunit [Defense mechanisms]; COG0286;PFAM: N-6 DNA Methylase; HsdM N-terminal domain;HsdM N-terminal domain; pfam12161;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU41	CDS	GOHSU_41_00340	33760..35061		restriction endonuclease S subunit [migap]				restriction endonuclease S subunit		G2NFE6_9ACTO Restriction endonuclease S subunits-li ke protein OS=Streptomyces sp. SirexAA-E GN=SACTE_4309 PE= 4 SV=1;restriction endonuclease S subunit [Saccharomonospo ra marina XMU15];PFAM: Type I restriction modification DNA specificity domain;Type I restriction modification DNA specificity domain; pfam01420
GOHSU41	CDS	GOHSU_41_00350	35065..38172		type I restriction-modification system endonuclease [migap]				type I restriction-modification system endonuclease		E6TAV8_MYCSR Helicase, type I site-specific restric tion-modification system restriction subunit OS=Mycobacter ium sp. (strain Spyr1) GN=Mspyr1_52850 PE=4 SV=1;type I restriction-modification system endonuclease [Pseudomonas stutzeri A1501];Code: V; COG: COG0610;Type I site-specific restriction-modification system, R (restriction) subunit and related helicases [Defense mechanisms];Type I restriction enzyme R protein N terminus (HSDR_N); cl14958;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;ATP binding site [chemical binding];putative Mg++ binding site [ion binding]
GOHSU41	CDS	GOHSU_41_00360	complement(38216..38887)		hypothetical protein [nite]			RER_01380 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TVM6_9ACTO Putative uncharacterized protein (Frag ment) OS=Gordonia sputi NBRC 100414 GN=GOSPT_011_00060 PE= 4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ]
GOHSU41	CDS	GOHSU_41_00370	complement(39144..40034)		transposase IS3/IS911 family protein [migap]				transposase IS3/IS911 family protein		A4QGG2_CORGB Putative uncharacterized protein OS=Co rynebacterium glutamicum (strain R) GN=cgR_2323 PE=4 SV=1;unnamed protein product [Frankia symbiont of Datisc a glomerata];KEGG: nml:Namu_3665 transposase IS3/IS911 family protein; manually curated; PFAM: Transposase IS3/IS911;putative transposase OrfB; Reviewed;Integrase core domain; cl01316;Integrase core domain; cl15866
GOHSU41	CDS	GOHSU_41_00380	40071..40400		acetyl-CoA synthetase subunit beta [migap]				acetyl-CoA synthetase subunit beta		acetyl-CoA synthetase subunit beta [Syntrophus acid itrophicus SB];acetyl coenzyme A synthetase (ADP forming), alpha domain; TIGR02717
GOHSU41	CDS	GOHSU_41_00390	complement(40359..40574)		arginine 2-monooxygenase [migap]				arginine 2-monooxygenase		arginine 2-monooxygenase [Prevotella buccae D17];arginine decarboxylase; Provisional;Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzyme Arginine Decarboxylase; cd06830;dimer interface [polypeptide binding];pyridoxal 5'-phosphate (PLP) binding site [chemical binding]
GOHSU41	CDS	GOHSU_41_00400	40593..40754		not assigned						
GOHSU41	CDS	GOHSU_41_00410	complement(41213..42523)		putative transposase [nite]			RER_pREL1-02710 :putative transposase : : : Rhodococcus erythropolis PR4	IS1096, tnpA protein		H0R9V7_9ACTO Putative transposase OS=Gordonia polyi soprenivorans NBRC 16320 GN=GOPIP_014_00010 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by similarity to GP:150005; match to protein family HMM PF01610;Helix-turn-helix domain of transposase family ISL3; pfam13542;Transposase; pfam01610
GOHSU41	CDS	GOHSU_41_00420	42718..43908		hypothetical protein [migap]				hypothetical protein		A8LZQ9_SALAI Putative uncharacterized protein OS=Sa linispora arenicola (strain CNS-205) GN=Sare_2055 PE=4 SV= 1;unnamed protein product [Salinispora arenicola CNS- 205];KEGG: gur:Gura_2210 hypothetical protein
GOHSU41	CDS	GOHSU_41_00430	43905..45260		Hypothetical protein [migap]				Hypothetical protein		B6ENJ8_ALISL Putative uncharacterized protein OS=Al iivibrio salmonicida (strain LFI1238) GN=VSAL_I1803 PE=4 S V=1;Hypothetical protein SCLAV_4396 [Streptomyces clavu ligerus ATCC 27064];Nucleotidyltransferase (NT) domain of 2'5'-oligoadenylate (2-5A)synthetase (2-5OAS) and class I CCA-adding enzyme; cd05400;NTP binding site [chemical binding];metal binding triad [ion binding]
GOHSU41	CDS	GOHSU_41_00440	complement(45437..46186)		putative transposase [nite]			MLP_25550 :putative transposase : : : Microlunatus phosphovorus NM-1	putative transposase		H5TZZ0_9ACTO Putative transposase OS=Gordonia sputi NBRC 100414 GN=GOSPT_057_00010 PE=4 SV=1;unnamed protein product [Frankia symbiont of Datisc a glomerata];manually curated; KEGG: fre:Franean1_4461 putative transposase;Integrase core domain; cl01316;Transposase and inactivated derivatives [DNA replication, recombination, and repair]
GOHSU41	CDS	GOHSU_41_00450	complement(46690..47445)		not assigned						
GOHSU41	CDS	GOHSU_41_00460	47542..48813		putative transposase orfA for insertion sequence element [nite]			DMR_29110 :putative transposase orfA for insertion sequence element : : : Desulfovibrio magneticus RS-1	integrase catalytic protein		H5UGN5_9ACTO Putative transposase OS=Gordonia terra e NBRC 100016 GN=GOTRE_074_00010 PE=4 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];Transposase and inactivated derivatives [DNA replication, recombination, and repair];PFAM: Integrase catalytic region; Resolvase helix-turn-helix domain protein; KEGG: sen:SACE_2605 IS21 family transposase;Helix-turn-helix domain of Hin and related proteins, a family of DNA-binding domains unique to bacteria and represented by the Hin protein of Salmonella. The basic HTH domain is a simple fold comprised of three core helices that form a right-handed...; cd00569;DNA-binding interface [nucleotide binding];Integrase core domain; cl01316;Integrase core domain; cl15866
GOHSU41	CDS	GOHSU_41_00470	48810..49577		putative transposition helper protein [nite]			RER_pREL1-02030 :putative transposition helper protein : : : Rhodococcus erythropolis PR4	IstB helper protein	istB	H5UGN6_9ACTO Putative transposition helper protein OS=Gordonia terrae NBRC 100016 GN=GOTRE_074_00020 PE=4 SV= 1;IstB helper protein [Methanosarcina acetivorans C2A ];transposase/IS protein; Provisional;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif
##GOHSU42	contig00231				48752
GOHSU42	CDS	GOHSU_42_00010	<1..1977	codon	translation initiation factor IF-2 [nite]	 infB [nite]		ROP_66830 :translation initiation factor IF-2 : infB : : Rhodococcus opacus B4	translation initiation factor IF-2	infB	F1YIG1_9ACTO Translation initiation factor IF-2 OS= Gordonia neofelifaecis NRRL B-59395 GN=infB PE=3 SV=1;translation initiation factor IF-2 [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Protects formylmethionyl-tRNA from spontaneous hydrolysis and promotes its binding to the 30S ribosomal subunits during initiation of protein synthesis. Also involved in the hydrolysis of GTP during the formation of the 70S ribosomal complex;translation initiation factor IF-2; TIGR00487;Translation initiation factor IF-2, N-terminal region; pfam04760;Initiation Factor 2 (IF2)/ eukaryotic Initiation Factor 5B (eIF5B) family; cd01887;G1 box;putative GEF interaction site [polypeptide binding];GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;This family represents the domain II of bacterial Initiation Factor 2 (IF2) and its eukaryotic mitochondrial homologue mtIF2. IF2, the largest initiation factor is an essential GTP binding protein. In E. coli three natural forms of IF2 exist in the cell; cd03702;Translation-initiation factor 2; pfam11987;mtIF2_IVc: this family represents the C2 subdomain of domain IV of mitochondrial translation initiation factor 2 (mtIF2) which adopts a beta-barrel fold displaying a high degree of structural similarity with domain II of the translation elongation factor...; cd03692
GOHSU42	CDS	GOHSU_42_00020	1979..2431		ribosome-binding factor A [hamap]	rbfA [hamap]	MF_00003:ribosome-binding factor A:rbfA 	RER_26340 :ribosome-binding factor A : rbfA : : Rhodococcus erythropolis PR4	ribosome-binding factor A	rbfA	F1YIG2_9ACTO Ribosome-binding factor A OS=Gordonia neofelifaecis NRRL B-59395 GN=rbfA PE=3 SV=1;ribosome-binding factor A [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosome-binding factor A; cl00542;associates with free 30S ribosomal subunits; essential for efficient processing of 16S rRNA; in Escherichia coli rbfA is induced by cold shock
GOHSU42	CDS	GOHSU_42_00030	2428..3384		DHH family protein [nite]			ROP_66860 :DHH family protein : : : Rhodococcus opacus B4	putative phosphoesterase		F1YIG3_9ACTO Phosphoesterase RecJ domain-containing protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08 388 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Exopolyphosphatase-related proteins [General function prediction only];DHH family; pfam01368
GOHSU42	CDS	GOHSU_42_00040	3434..4681		conserved hypothetical membrane protein [nite]			RER_26360 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	putative DNA-damage-inducible protein F		F1YIG4_9ACTO MATE efflux family protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08393 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Na+-driven multidrug efflux pump [Defense mechanisms]; COG0534
GOHSU42	CDS	GOHSU_42_00050	complement(4718..5047)		hypothetical protein [nite]			KRH_05670 :hypothetical protein : : : Kocuria rhizophila DC2201	hypothetical protein		F1YIG6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08403 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU42	CDS	GOHSU_42_00060	complement(5085..6287)		ferredoxin reductase [nite]			ROP_10950 :ferredoxin reductase : : : Rhodococcus opacus B4	ferredoxin reductase		G7H0B4_9ACTO Ferredoxin reductase OS=Gordonia araii NBRC 100433 GN=GOARA_036_00210 PE=4 SV=1;ferredoxin reductase [Rhodococcus opacus B4];phenylpropionate dioxygenase ferredoxin reductase subunit; Provisional;Pyridine nucleotide-disulphide oxidoreductase; pfam00070
GOHSU42	CDS	GOHSU_42_00070	6359..7318		hypothetical protein [nite]			ROP_66880 :hypothetical protein : : : Rhodococcus opacus B4	metallophosphoesterase		F1YIG9_9ACTO Metallophosphoesterase OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_08418 PE=4 SV=1;metallophosphoesterase [Micromonospora sp. L5];PFAM: metallophosphoesterase; KEGG: saq:Sare_2235 metallophosphoesterase;Predicted phosphohydrolases [General function prediction only]; COG1409;metallophosphatase superfamily, metallophosphatase domain; cl13995;metal binding site [ion binding]
GOHSU42	CDS	GOHSU_42_00080	7556..8245		probable phosphopantetheinyl transferase [nite]			RER_26380 :probable phosphopantetheinyl transferase : : : Rhodococcus erythropolis PR4	4'-phosphopantetheinyl transferase		F1YIH0_9ACTO 4'-phosphopantetheinyl transferase OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08423 PE=4 SV= 1;4'-phosphopantetheinyl transferase [Micromonospora sp. L5];PFAM: 4'-phosphopantetheinyl transferase; KEGG: ami:Amir_5286 4'-phosphopantetheinyl transferase;Phosphopantetheinyl transferase component of siderophore synthetase [Secondary metabolites biosynthesis, transport, and catabolism]; COG2977;4'-phosphopantetheinyl transferase superfamily; cl00500
GOHSU42	CDS	GOHSU_42_00090	8249..10375		putative acyltransferase [nite]			ROP_22650 :putative acyltransferase : : 2.3.1.- : Rhodococcus opacus B4	acyltransferase 3		F1YHQ5_9ACTO Acyltransferase 3 OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_06615 PE=4 SV=1;acyltransferase 3 [Mycobacterium gilvum PYR- GCK];PFAM: acyltransferase 3; KEGG: mbo:Mb1592c hypothetical protein;OpgC protein; cl00792
GOHSU42	CDS	GOHSU_42_00100	10426..11325		tRNA pseudouridine synthase B [hamap]	truB [hamap]	MF_01080:tRNA pseudouridine synthase B:truB 	RER_26390 :tRNA pseudouridine synthase TruB : truB : 5.4.99.12 : Rhodococcus erythropolis PR4	tRNA pseudouridine synthase B	truB	F1YIH1_9ACTO tRNA pseudouridine synthase B OS=Gordo nia neofelifaecis NRRL B-59395 GN=truB PE=3 SV=1;tRNA pseudouridine synthase B [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;tRNA pseudouridine synthase B; Provisional; PRK03287;catalyzes isomerization of specific uridines in RNA to pseudouridine; responsible for residues in T loops of many tRNAs;PseudoU_synth_EcTruB: Pseudouridine synthase, Escherichia coli TruB like. This group consists of bacterial pseudouridine synthases similar to E. coli TruB and Mycobacterium tuberculosis TruB. Pseudouridine synthases catalyze the isomerization of specific...; cd02573;RNA binding site [nucleotide binding]
GOHSU42	CDS	GOHSU_42_00110	complement(11342..12025)		putative DtxR family transcriptional regulator [nite]			RER_26410 :putative DtxR family transcriptional regulator : : : Rhodococcus erythropolis PR4	Mn-dependent transcriptional regulator		F1YIH3_9ACTO Iron dependent repressor OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_08438 PE=4 SV=1;Mn-dependent transcriptional regulator [Arthrobacte r phenanthrenivorans Sphe3];PFAM: FeoA domain; Iron dependent repressor, metal binding and dimerisation domain; Iron dependent repressor, N-terminal DNA binding domain;Helix-turn-helix domains; cl00088;Helix-turn-helix diphteria tox regulatory element; smart00529;Iron dependent repressor, metal binding and dimerisation domain; pfam02742;FeoA domain; cl00838
GOHSU42	CDS	GOHSU_42_00120	12119..13030		flavokinase/FAD synthetase [nite]	 ribC [nite]		RER_26420 :flavokinase/FAD synthetase : ribC : 2.7.1.26/2.7.7.2 : Rhodococcus erythropolis PR4	riboflavin biosynthesis protein RibF		F1YIH4_9ACTO Bifunctional riboflavin kinase/FMN ade nylyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_08443 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: riboflavin biosynthesis protein RibF; PFAM: FAD synthetase; riboflavin kinase; KEGG: glo:Glov_1649 riboflavin biosynthesis protein RibF;bifunctional riboflavin kinase/FMN adenylyltransferase; Reviewed;FAD synthetase, N-terminal domain of the bifunctional enzyme; cd02064;Riboflavin kinase; cl03312
GOHSU42	CDS	GOHSU_42_00130	13180..13449		30S ribosomal protein S15 [nite]	 rpsO [nite]		RER_26430 :30S ribosomal protein S15 : rpsO : : Rhodococcus erythropolis PR4	30S ribosomal protein S15	rpsO	F4D124_PSEUX 30S ribosomal protein S15 OS=Pseudonoc ardia dioxanivorans (strain ATCC 55486 / DSM 44775 / JCM 1 3855 / CB1190) GN=rpsO PE=3 SV=1;30S ribosomal protein S15 [Corynebacterium glutamic um ATCC 13032];primary rRNA binding protein; helps nucleate assembly of 30S; binds directly to the 16S rRNA and an intersubunit bridge to the 23S rRNA; autoregulates translation through interactions with the mRNA leader sequence;Ribosomal protein S15 (prokaryotic)_S13 (eukaryotic) binds the central domain of 16S rRNA and is required for assembly of the small ribosomal subunit and for intersubunit association, thus representing a key element in the assembly of the whole ribosome; cd00353;16S/18S rRNA binding site [nucleotide binding];S13e-L30e interaction site [polypeptide binding];25S rRNA binding site [nucleotide binding]
GOHSU42	CDS	GOHSU_42_00140	13868..16117		polyribonucleotide nucleotidyltransferase [hamap]	pnp [hamap]	MF_01595:polyribonucleotide nucleotidyltransferase:pnp 	ROP_66940 :guanosine pentaphosphate synthetase/polynucleotide phosphorylase : gpsI : 2.7.7.8 : Rhodococcus opacus B4	polyribonucleotide nucleotidyltransferase		F1YIH7_9ACTO Polyribonucleotide nucleotidyltransfer ase OS=Gordonia neofelifaecis NRRL B-59395 GN=pnp PE=3 SV= 1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: dar:Daro_2448 polynucleotide phosphorylase/polyadenylase; TIGRFAM: polyribonucleotide nucleotidyltransferase; guanosine pentaphosphate synthetase I/polyribonucleotide nucleotidyltransferase; PFAM: 3' exoribonuclease; polynucleotide phosphorylase, phosphorolytic RNA-binding, bacterial/organelle-type; RNA binding S1 domain protein; Exoribonuclease, phosphorolytic domain 2; K Homology, type 1, subgroup; SMART: KH domain protein;guanosine pentaphosphate synthetase I/polynucleotide phosphorylase; TIGR02696;RNase PH-like 3'-5' exoribonucleases; cl03114;oligomer interface [polypeptide binding];RNA binding site [nucleotide binding];Polyribonucleotide nucleotidyltransferase, RNA binding domain; pfam03726;Polyribonucleotide nucleotidyltransferase, repeat 2; cd11364;RNase E interface [polypeptide binding];trimer interface [polypeptide binding];Polynucleotide phosphorylase (PNPase) K homology RNA-binding domain (KH). PNPase is a polyribonucleotide nucleotidyl transferase that degrades mRNA in prokaryotes and plant chloroplasts. The C-terminal region of PNPase contains domains homologous to...; cd02393;putative nucleic acid binding region [nucleotide binding];G-X-X-G motif;S1_PNPase: Polynucleotide phosphorylase (PNPase), ), S1-like RNA-binding domain. PNPase  is a polyribonucleotide nucleotidyl transferase that degrades mRNA. It is a trimeric multidomain protein. The C-terminus contains the S1 domain which binds ssRNA; cd04472;domain interface
GOHSU42	CDS	GOHSU_42_00150	complement(16241..18304)		probable ceramidase [nite]			RER_07640 :probable ceramidase : : 3.5.1.23 : Rhodococcus erythropolis PR4	Neutral/alkaline nonlysosomal ceramidase		F1YII0_9ACTO Ceramidase OS=Gordonia neofelifaecis N RRL B-59395 GN=SCNU_08473 PE=4 SV=1;Neutral/alkaline nonlysosomal ceramidase [Leptonema illini DSM 21528];PFAM: Neutral/alkaline non-lysosomal ceramidase; InterPro IPR006823; KEGG: lbf:LBF_2410 putative lipoprotein; PFAM: Neutral/alkaline nonlysosomal ceramidase; SPTR: Putative neutral/alkaline ceramidase; putative signal peptide;Neutral/alkaline non-lysosomal ceramidase; cl04712
GOHSU42	CDS	GOHSU_42_00160	18439..19923		putative amidase [nite]			ACMV_32720 :putative amidase : : 3.5.1.- : Acidiphilium multivorum AIU301	amidase		F1YIH9_9ACTO Amidase, AmiC_2 OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_08468 PE=4 SV=1;amidase [Pseudonocardia dioxanivorans CB1190];PFAM: Amidase; KEGG: svi:Svir_03450 amidase, Asp-tRNAAsn/Glu-tRNAGln amidotransferase A subunit;amidase; Provisional;Amidase; cl11426
GOHSU42	CDS	GOHSU_42_00170	20029..20787		dihydrodipicolinate reductase [hamap]	dapB [hamap]	MF_00102:dihydrodipicolinate reductase:dapB 	ROP_67260 :dihydrodipicolinate reductase : dapB : 1.3.1.26 : Rhodococcus opacus B4	dihydrodipicolinate reductase		F1YII1_9ACTO Dihydrodipicolinate reductase OS=Gordo nia neofelifaecis NRRL B-59395 GN=dapB PE=3 SV=1;dihydrodipicolinate reductase [Mycobacterium gilvum PYR- GCK];catalyzes the reduction of 2,3-dihydrodipicolinate to 2,3,4,5-tetrahydrodipicolinate in lysine and diaminopimelate biosynthesis;dihydrodipicolinate reductase; Provisional;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Dihydrodipicolinate reductase, C-terminus; pfam05173
GOHSU42	CDS	GOHSU_42_00180	20787..21266		hypothetical membrane protein [nite]			ROP_67270 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YHT2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08483 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU42	CDS	GOHSU_42_00190	21384..21836		hypothetical protein [nite]			RER_27000 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		Q5YSW4_NOCFA Putative uncharacterized protein OS=No cardia farcinica (strain IFM 10152) GN=NFA_38790 PE=4 SV=1 ;unnamed protein product [Frankia sp. EAN1pec];KEGG: tfu:Tfu_0321 hypothetical protein;NADPH-dependent FMN reductase; cl00438
GOHSU42	CDS	GOHSU_42_00200	21856..23898		putative ABC transporter permease/ATP-binding protein [nite]			ROP_57070 :putative ABC transporter permease/ATP-binding protein : : : Rhodococcus opacus B4	xenobiotic-transporting ATPase		H8E8E9_9MICO ABC transporter related protein OS=Mic robacterium laevaniformans OR221 GN=OR221_3123 PE=4 SV=1;xenobiotic-transporting ATPase [Pseudonocardia diox anivorans CB1190];PFAM: ABC transporter-like; ABC transporter, transmembrane domain; KEGG: rha:RHA1_ro05645 bifunctional ABC lipid A transporter: ATP-binding/permease; SMART: ATPase, AAA+ type, core;ABC-type multidrug transport system, ATPase and permease components [Defense mechanisms]; COG1132;ABC transporter transmembrane region; cl00549;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU42	CDS	GOHSU_42_00210	23930..24736		thymidylate synthase [hamap]	thyA [hamap]	MF_00008:thymidylate synthase:thyA 	ROP_19620 :thymidylate synthase : thyA : 2.1.1.45 : Rhodococcus opacus B4	thymidylate synthase	thyA	F1YMX8_9ACTO Thymidylate synthase OS=Gordonia neofe lifaecis NRRL B-59395 GN=thyA PE=3 SV=1;thymidylate synthase [Corynebacterium glutamicum AT CC 13032];IS fingerprint type 4-5;ThyA; catalyzes formation of dTMP and 7,8-dihydrofolate from 5,10-methylenetetrahydrofolate and dUMP; involved in deoxyribonucleotide biosynthesis; there are 2 copies in some Bacilli, one of which appears to be phage-derived;Thymidylate synthase and pyrimidine hydroxymethylase: Thymidylate synthase (TS) and deoxycytidylate hydroxymethylase (dCMP-HMase) are homologs that catalyze analogous alkylation of C5 of pyrimidine nucleotides. Both enzymes are involved in the...; cd00351;dimerization interface [polypeptide binding]
GOHSU42	CDS	GOHSU_42_00220	24797..25300		dihydrofolate reductase [nite]	 folA [nite]		ROP_19630 :dihydrofolate reductase : folA : 1.5.1.3 : Rhodococcus opacus B4	dihydrofolate reductase		F1YMX7_9ACTO Dihydrofolate reductase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_16364 PE=3 SV=1;dihydrofolate reductase [Methylibium petroleiphilum PM1];Dihydrofolate reductase (DHFR). Reduces 7,8-dihydrofolate to 5,6,7,8-tetrahydrofolate with NADPH as a cofactor. This is an essential step in the biosynthesis of deoxythymidine phosphate since 5,6,7,8-tetrahydrofolate is required to regenerate 5; cd00209;folate binding site [chemical binding];NADP+ binding site [chemical binding]
GOHSU42	CDS	GOHSU_42_00230	25333..26520		hypothetical protein [nite]			ROP_67290 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YMX2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16339 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF1006); cl15826
GOHSU42	CDS	GOHSU_42_00240	26517..27290		hypothetical protein [migap]				hypothetical protein		F1YMX6_9ACTO Methyltransferase type 11 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16359 PE=4 SV=1;unnamed protein product [Pseudomonas aeruginosa LES B58];SAM-dependent methyltransferases [Secondary metabolites biosynthesis, transport, and catabolism / General function prediction only]; COG0500;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU42	CDS	GOHSU_42_00250	27422..27961		not assigned						C5CAS8_MICLC Putative uncharacterized protein OS=Mi crococcus luteus (strain ATCC 4698 / DSM 20030 / JCM 1464 / NBRC 3333 / NCIMB 9278 / NCTC 2665 / VKM Ac-2230) GN=Ml ut_09410 PE=4 SV=1
GOHSU42	CDS	GOHSU_42_00260	28033..28416		hypothetical protein [nite]			RER_12320 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YMX3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16344 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: nfa:nfa34980 hypothetical protein
GOHSU42	CDS	GOHSU_42_00270	28451..29371		putative hydroxymethylglutaryl-CoA lyase [nite]			ROP_49950 :putative hydroxymethylglutaryl-CoA lyase : : 4.1.3.4 : Rhodococcus opacus B4	hydroxymethylglutaryl-CoA lyase		H0JWB4_9NOCA Hydroxymethylglutaryl-CoA lyase OS=Rho dococcus pyridinivorans AK37 GN=AK37_20114 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00682;3-hydroxy-3-methylglutaryl-CoA lyase, catalytic TIM barrel domain; cd07938;catalytic residues [active];metal binding site [ion binding]
GOHSU42	CDS	GOHSU_42_00280	29713..30483		hypothetical protein [nite]			ROP_51020 :hypothetical protein : : : Rhodococcus opacus B4	secretory lipase		H6N1F4_GORPV Putative secretory lipase OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c11030 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: secretory lipase; KEGG: rop:ROP_51020 hypothetical protein
GOHSU42	CDS	GOHSU_42_00290	30529..31299		phosphomannomutase [nite]			SLG_p_00450 :phosphomannomutase : : : Sphingobium sp. SYK-6	HAD-superfamily hydrolase		F1YMW9_9ACTO HAD-superfamily hydrolase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16324 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: HAD-superfamily hydrolase, subfamily IIB; PFAM: phosphomannomutase; KEGG: hypothetical protein;Predicted hydrolases of the HAD superfamily [General function prediction only]; COG0561;Haloacid Dehalogenase-like Hydrolases; cl11391
GOHSU42	CDS	GOHSU_42_00300	complement(31335..31850)		protein tyrosine/serine phosphatase-like protein [migap]				protein tyrosine/serine phosphatase-like protein		G7H0C9_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_036_00360 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: ppw:PputW619_1877 protein tyrosine/serine phosphatase;Protein tyrosine/serine phosphatase [Signal transduction mechanisms];Protein tyrosine phosphatases (PTP) catalyze the dephosphorylation of phosphotyrosine peptides; they regulate phosphotyrosine levels in signal transduction pathways. The depth of the active site cleft renders the enzyme specific for phosphorylated Tyr; cl00053
GOHSU42	CDS	GOHSU_42_00310	32013..32762		thymidylate synthase ThyX [hamap]	thyX [hamap]	MF_01408:thymidylate synthase ThyX:thyX 	ROP_67320 :FAD-dependent thymidylate synthase ThyX : thyX : 2.1.1.148 : Rhodococcus opacus B4	FAD-dependent thymidylate synthase	thyX	G7H0D0_9ACTO Thymidylate synthase thyX OS=Gordonia araii NBRC 100433 GN=thyX PE=3 SV=1;FAD-dependent thymidylate synthase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Thymidylate synthase complementing protein; cl03630;flavin dependent thymidylate synthase; ThyX; thymidylate synthase complementing protein; catalyzes the formation of dTMP and tetrahydrofolate from dUMP and methylenetetrahydrofolate; the enzyme from Mycobacterium tuberculosis forms homotetramers; uses FAD as a cofactor
GOHSU42	CDS	GOHSU_42_00320	32811..33716		dihydrodipicolinate synthase [hamap]	dapA [hamap]	MF_00418:dihydrodipicolinate synthase:dapA 	RER_27320 :dihydrodipicolinate synthase : dapA : 4.2.1.52 : Rhodococcus erythropolis PR4	dihydrodipicolinate synthase	dapA	F1YMW6_9ACTO Dihydrodipicolinate synthase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_16309 PE=3 SV=1;dihydrodipicolinate synthase [Corynebacterium gluta micum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of dihydrodipicolinate from L-aspartate 4-semialdehyde and pyruvate in lysine and diaminopimelate biosynthesis;Dihydrodipicolinate synthase/N-acetylneuraminate lyase [Amino acid transport and metabolism / Cell envelope biogenesis, outer membrane]; COG0329;Dihydrodipicolinate synthase (DHDPS); cd00950;dimer interface [polypeptide binding];catalytic residue [active]
GOHSU42	CDS	GOHSU_42_00330	33716..35569		hypothetical protein [nite]			ROP_67340 :hypothetical protein : : : Rhodococcus opacus B4	metallo-beta-lactamase		F1YMW5_9ACTO Beta-lactamase domain-containing prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16304 PE =4 SV=1;metallo-beta-lactamase [Streptomyces sp. e14];mRNA degradation ribonucleases J1/J2 (metallo-beta-lactamase superfamily) [Translation, ribosomal structure and biogenesis; Replication, recombination and repair];Metallo-beta-lactamase superfamily; cl00446;RNA-metabolising metallo-beta-lactamase; pfam07521
GOHSU42	CDS	GOHSU_42_00340	35928..38600		cell division protein FtsK [nite]	 ftsK [nite]		RER_27350 :cell division protein FtsK : ftsK : : Rhodococcus erythropolis PR4	cell division protein FtsK/SpoIIIE		H5UGZ1_9ACTO Cell division protein FtsK OS=Gordonia terrae NBRC 100016 GN=ftsK PE=4 SV=1;cell division protein FtsK/SpoIIIE [Verrucosispora maris AB-18- 032];COG1674 DNA segregation ATPase FtsK/SpoIIIE and related proteins;DNA segregation ATPase FtsK/SpoIIIE and related proteins [Cell division and chromosome partitioning]; COG1674;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Ftsk gamma domain; cl09645
GOHSU42	CDS	GOHSU_42_00350	38808..39803		putative TerC family integral membrane protein [nite]			ROP_67370 :putative TerC family integral membrane protein : : : Rhodococcus opacus B4	TerC family integral membrane protein		G7H0D6_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_036_00430 PE=4 SV=1;TerC family integral membrane protein [Rhodococcus opacus B4];Integral membrane protein TerC family; cl10468
GOHSU42	CDS	GOHSU_42_00360	complement(39763..40047)		hypothetical protein [nite]			ROP_67380 :hypothetical protein : : : Rhodococcus opacus B4	YCII--like protein		G7H243_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_048_01200 PE=4 SV=1;YCII--like protein [Nakamurella multipartita DSM 44 233];YCII-related domain; cl00999;PFAM: YCII-related; KEGG: mpt:Mpe_A2445 hypothetical protein
GOHSU42	CDS	GOHSU_42_00370	complement(40100..40633)		putative acetyltransferase [nite]			ROP_67390 :putative acetyltransferase : : 2.3.1.- : Rhodococcus opacus B4	N-acetylglutamate synthase		F9VXU0_9ACTO Putative acetyltransferase OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_077_00390 PE=4 SV=1;N-acetylglutamate synthase [Saccharopolyspora eryth raea NRRL 2338];catalyzes the conversion of l-glutamate to a-N-acetyl-l-glutamate in arginine biosynthesis;bifunctional argininosuccinate lyase/N-acetylglutamate synthase; Provisional;N-Acyltransferase superfamily: Various enzymes that characteristically catalyze the transfer of an acyl group to a substrate; cd04301;Coenzyme A binding pocket [chemical binding]
GOHSU42	CDS	GOHSU_42_00380	40707..41300		phosphatidylglycerophosphate synthase [nite]	 pgsA [nite]		ROP_67400 :phosphatidylglycerophosphate synthase : pgsA : 2.7.8.5 : Rhodococcus opacus B4	CDP-diacylglycerol--glycerol-3-phosphate 3-phosphatidyl-transferase PgsA	pgsA	H5TYM6_9ACTO Phosphatidylglycerophosphate synthase OS=Gordonia sputi NBRC 100414 GN=pgsA PE=3 SV=1;pgsA gene product [Gordonia polyisoprenivorans VH2] ;CDP-alcohol phosphatidyltransferase; cl00453
GOHSU42	CDS	GOHSU_42_00390	41400..41687		putative Xre family DNA-binding protein [nite]			ROP_67420 :putative Xre family DNA-binding protein : : : Rhodococcus opacus B4	XRE family transcriptional regulator		F9VXT8_9ACTO Putative Xre family DNA-binding protei n OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_077_00370 P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: helix-turn-helix domain protein; SMART: helix-turn-helix domain protein; KEGG: rlt:Rleg2_4257 transcriptional regulator, XRE family;Helix-turn-helix XRE-family like proteins. Prokaryotic DNA binding proteins belonging to the xenobiotic response element family of transcriptional regulators; cd00093;non-specific DNA binding site [nucleotide binding];salt bridge;sequence-specific DNA binding site [nucleotide binding]
GOHSU42	CDS	GOHSU_42_00400	41764..42552		putative PspA family protein [nite]			ROP_67430 :putative PspA family protein : : : Rhodococcus opacus B4	hypothetical protein		G7H0E1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_036_00480 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF04012;PspA/IM30 family; pfam04012
GOHSU42	CDS	GOHSU_42_00410	42567..43724		putative glycosyltransferase [nite]			RER_27510 :putative glycosyltransferase : : 2.4.-.- : Rhodococcus erythropolis PR4	hypothetical protein		F1YMV5_9ACTO Glycosyl transferase related to UDP-gl ucuronosyltransferase-like protein OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_16254 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];Glycosyl transferases, related to UDP-glucuronosyltransferase [Carbohydrate transport and metabolism / Signal transduction mechanisms];Glycosyltransferases catalyze the transfer of sugar moieties from activated donor molecules to specific acceptor molecules, forming glycosidic bonds. The acceptor molecule can be a lipid, a protein, a heterocyclic compound, or another carbohydrate...; cl10013
GOHSU42	CDS	GOHSU_42_00420	43750..43944		hypothetical protein [nite]			ROP_67500 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		Q5YSZ4_NOCFA Putative uncharacterized protein OS=No cardia farcinica (strain IFM 10152) GN=NFA_38490 PE=4 SV=1 ;unnamed protein product [Kitasatospora setae KM- 60 54];strain coidentity: NBRC 14216; type strain of Kitasatospora setae;Protein of unknown function (DUF3046); pfam11248
GOHSU42	CDS	GOHSU_42_00430	44138..45181		protein RecA [hamap]	recA [hamap]	MF_00268:protein RecA:recA 	RER_27530 :RecA protein : recA : : Rhodococcus erythropolis PR4	recombinase A	recA	F1YMV3_9ACTO Protein RecA OS=Gordonia neofelifaecis NRRL B-59395 GN=recA PE=3 SV=1;recA gene product [Frankia sp. EAN1pec];recombinase A; Provisional; PRK09354;catalyzes the hydrolysis of ATP in the presence of single-stranded DNA, the ATP-dependent uptake of single-stranded DNA by duplex DNA, and the ATP-dependent hybridization of homologous single-stranded DNAs;RecA is a  bacterial enzyme which has roles in homologous recombination, DNA repair, and the induction of the SOS response.  RecA couples ATP hydrolysis to DNA strand exchange; cd00983;hexamer interface [polypeptide binding];Walker A motif;ATP binding site [chemical binding];Walker B motif
GOHSU42	CDS	GOHSU_42_00440	45207..45695		regulatory protein RecX [hamap]	recX [hamap]	MF_01114:regulatory protein RecX:recX 	ROP_67520 :regulatory protein RecX : recX : : Rhodococcus opacus B4	recombination regulator RecX	recX	G7GVQ4_9ACTO Regulatory protein RecX OS=Gordonia am arae NBRC 15530 GN=recX PE=3 SV=1;recombination regulator RecX [Corynebacterium gluta micum ATCC 13032];IS fingerprint type 4-5;binds RecA and inhibits RecA-mediated DNA strand exchange and ATP hydrolysis and coprotease activities;RecX family; cl00936
GOHSU42	CDS	GOHSU_42_00450	complement(45719..46651)		glutamate ABC transporter permease protein [nite]	 gluD [nite]		RER_27550 :glutamate ABC transporter permease protein : gluD : : Rhodococcus erythropolis PR4	glutamate transporter permease		H6MYZ9_GORPV Putative glutamate ABC transporter per mease protein OS=Gordonia polyisoprenivorans (strain DSM 4 4266 / VH2) GN=GPOL_c21010 PE=3 SV=1;glutamate transporter permease [Streptomyces albus J1074];overlaps another CDS with the same product name;ABC-type arginine transport system, permease component [Amino acid transport and metabolism]; COG4215;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU42	CDS	GOHSU_42_00460	complement(46648..47331)		glutamate ABC transporter permease protein [nite]	 gluC [nite]		RER_27560 :glutamate ABC transporter permease protein : gluC : : Rhodococcus erythropolis PR4	putative glutamate ABC transporter permease		F1YMV0_9ACTO Polar amino acid ABC transporter inner membrane subunit OS=Gordonia neofelifaecis NRRL B-59395 G N=SCNU_16229 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cd06261;dimer interface [polypeptide binding];conserved gate region;putative PBP binding loops;ABC-ATPase subunit interface
GOHSU42	CDS	GOHSU_42_00470	complement(47332..48345)		glutamate ABC transporter glutamate-binding protein [nite]	 gluB [nite]		ROP_67550 :glutamate ABC transporter glutamate-binding protein : gluB : : Rhodococcus opacus B4	glutamate ABC transporter glutamate-binding protein	gluB	F1YMU9_9ACTO Extracellular solute-binding protein f amily 3 protein OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_16224 PE=4 SV=1;glutamate ABC transporter glutamate-binding protein [Microlunatus phosphovorus NM- 1];strain coidentity: NBRC 101784;Bacterial periplasmic transport systems use membrane-bound complexes and substrate-bound, membrane-associated, periplasmic binding proteins (PBPs) to transport a wide variety of  substrates, such as, amino acids, peptides, sugars, vitamins and inorganic...; cd00134;Bacterial extracellular solute-binding proteins, family 3; pfam00497;substrate binding pocket [chemical binding];membrane-bound complex binding site;hinge residues
GOHSU42	CDS	GOHSU_42_00480	complement(48348..>48752)	codon	glutamate ABC transporter ATP-binding protein [nite]	 gluA [nite]		ROP_67560 :glutamate ABC transporter ATP-binding protein : gluA : : Rhodococcus opacus B4	glutamate uptake system ATP-binding protein		G7GVQ8_9ACTO Glutamate ABC transporter ATP-binding protein OS=Gordonia amarae NBRC 15530 GN=gluA PE=3 SV=1;glutamate uptake system ATP-binding protein [Strept omyces clavuligerus ATCC 27064];ABC-type polar amino acid transport system, ATPase component [Amino acid transport and metabolism]; COG1126;HisP and GlnQ are the ATP-binding components of the bacterial periplasmic histidine and glutamine permeases, repectively.  Histidine permease is a multisubunit complex containing the HisQ and HisM integral membrane subunits and two copies of HisP.  HisP...; cd03262;ATP binding site [chemical binding];ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
##GOHSU43	contig00241				45282
GOHSU43	CDS	GOHSU_43_00010	complement(<1..1186)	3x codon	DNA polymerase III gamma/tau subunit [nite]	 dnaX [nite]		ROP_41260 :DNA polymerase III gamma/tau subunit : dnaX : 2.7.7.7 : Rhodococcus opacus B4	DNA polymerase III subunits gamma/tau		F1YKB7_9ACTO DNA polymerase III subunits gamma and tau OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11360 P E=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];DNA polymerase III, subunit gamma and tau; TIGR02397;KEGG: dal:Dalk_1998 DNA polymerase III subunits gamma/tau; TIGRFAM: DNA polymerase III subunits gamma/tau; SMART: AAA ATPase;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger;DNA polymerase III subunits gamma and tau domain III; pfam12169
GOHSU43	CDS	GOHSU_43_00020	complement(1233..2498)		putative glycosyltransferase [nite]			RER_03630 :putative glycosyltransferase : : 2.4.-.- : Rhodococcus erythropolis PR4	hypothetical protein		H5UG65_9ACTO Putative glycosyltransferase OS=Gordon ia terrae NBRC 100016 GN=GOTRE_071_00500 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF2029; KEGG: dol:Dole_2388 hypothetical protein;Protein of unknown function (DUF2029); cl09812
GOHSU43	CDS	GOHSU_43_00030	complement(2567..3835)		aspartate aminotransferase AspT [nite]	 aspT [nite]		RER_03620 :aspartate aminotransferase AspT : aspT : 2.6.1.1 : Rhodococcus erythropolis PR4	class I/II aminotransferase		F1YKB9_9ACTO Class I/II aminotransferase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_11370 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: aminotransferase class I and II; KEGG: scl:sce6239 hypothetical protein;Aminotransferase class I and II; pfam00155;Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU43	CDS	GOHSU_43_00040	complement(4074..5519)		catalase [nite]	 katA [nite]		ROP_59990 :catalase : katA : 1.11.1.6 : Rhodococcus opacus B4	catalase		H0R0U6_9ACTO Catalase OS=Gordonia effusa NBRC 10043 2 GN=katA PE=3 SV=1;catalase [Rhodococcus jostii RHA1];Catalase [Inorganic ion transport and metabolism]; COG0753;Clade 3 of the heme-binding enzyme catalase; cd08156;tetramer interface [polypeptide binding];heme binding pocket [chemical binding];NADPH binding site [chemical binding]
GOHSU43	tRNA	GOHSU_43_t0010	complement(5733..5817)		not assigned						identified by tRNAscan-SE; putative
GOHSU43	CDS	GOHSU_43_00050	complement(5910..7181)		hypothetical membrane protein [nite]			ROP_52190 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YKC2_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11385 PE=4 SV=1;protein [Clavibacter michiganensis subsp. sepedonic us];Predicted permease, member of the PurR regulon [General function prediction only]; COG0628;pheromone autoinducer 2 transporter; Reviewed; cl00465
GOHSU43	CDS	GOHSU_43_00060	complement(7266..8351)		hypothetical protein [migap]				hypothetical protein		F1YKC3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11390 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.946 at residue 35;Htaa; pfam04213
GOHSU43	CDS	GOHSU_43_00070	8446..8829		hypothetical protein [nite]			MLP_49780 :hypothetical protein : : : Microlunatus phosphovorus NM-1	glyoxalase/bleomycin resistance protein/dioxygenase		G7GX45_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_004_00100 PE=4 SV=1;glyoxalase/bleomycin resistance protein/dioxygenase [Streptomyces roseosporus NRRL 15998];This conserved domain belongs to a superfamily including the bleomycin resistance protein, glyoxalase I, and type I ring-cleaving dioxygenases; cd08355;Glyoxalase/Bleomycin resistance protein/Dioxygenase superfamily; pfam00903
GOHSU43	CDS	GOHSU_43_00080	complement(8858..9922)		hypothetical protein [nite]			ROP_51690 :hypothetical protein : : : Rhodococcus opacus B4	ATP-dependent DNA ligase	lig	F1YKC6_9ACTO DNA primase small subunit OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_11405 PE=4 SV=1;ATP-dependent DNA ligase [Amycolatopsis mediterrane i U32];LigD_Pol_like_2: Polymerase (Pol) domain of bacterial LigD proteins similar to Pseudomonas aeruginosa (Pae) LigD, subgroup 2. The LigD Pol domain belongs to the archaeal/eukaryal primase (AEP) superfamily. In prokaryotes, LigD along with Ku is required...; cd04865;nucleotide binding site [chemical binding]
GOHSU43	CDS	GOHSU_43_00090	complement(9925..10977)		ATP-dependent DNA ligase LigC [nite]	 ligC [nite]		RER_49760 :ATP-dependent DNA ligase LigC : ligC : 6.5.1.1 : Rhodococcus erythropolis PR4	ATP dependent DNA ligase-like protein,ATP dependent DNA ligase family protein		F9VW33_9ACTO ATP-dependent DNA ligase LigC OS=Gordo nia alkanivorans NBRC 16433 GN=ligC PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: ATP dependent DNA ligase domain; ATP dependent DNA ligase C terminal region;DNA polymerase LigD, ligase domain; TIGR02779;Adenylation domain of Mycobacterium tuberculosis LigC-like ATP-dependent DNA ligases; cd07905;DNA binding site [nucleotide binding];The Oligonucleotide/oligosaccharide binding (OB)-fold domain of ATP-dependent DNA ligase LigC is a DNA-binding module that is part of the catalytic core unit; cd07970
GOHSU43	CDS	GOHSU_43_00100	11018..11926		glutamyl-Q tRNA(Asp) synthetase [hamap]	gluQ [hamap]	MF_01428:glutamyl-Q tRNA(Asp) synthetase:gluQ 	ROP_41070 :glutamyl-Q-tRNA(Asp) synthetase : gluQ : 6.1.1.- : Rhodococcus opacus B4	glutamyl-Q tRNA(Asp) synthetase	gltX	G7GM90_9ACTO Glutamyl-Q-tRNA(Asp) synthetase OS=Gor donia amarae NBRC 15530 GN=gluQ PE=3 SV=1;glutamyl-Q tRNA(Asp) synthetase [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;this tRNA synthetase lacks the tRNA anticodon interaction domain; instead this enzyme modifies tRNA(Asp) with glutamate by esterifying glutamate to the 2-amino-5-(4,5-dihydroxy-2-cyclopenten-1-yl) moiety of queosine generating a modified nucleoside at the first anticodon position of tRNAAsp; the modified tRNA does not bind elongation factor Tu;tRNA synthetases class I (E and Q), catalytic domain; pfam00749;nucleotidyl transferase superfamily; cl00015;HIGH motif;nucleotide binding site [chemical binding];KMSKS motif
GOHSU43	CDS	GOHSU_43_00110	complement(12038..13285)		tRNA-guanine transglycosylase [nite]	 tgt [nite]		MLP_34160 :tRNA-guanine transglycosylase : tgt : 2.4.2.29 : Microlunatus phosphovorus NM-1	queuine tRNA-ribosyltransferase		F1YPK7_9ACTO Queuine tRNA-ribosyltransferase OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_19310 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];tRNA-guanine transglycosylase;Queuine/archaeosine tRNA-ribosyltransferase [Translation, ribosomal structure and biogenesis]; COG0343;queuine tRNA-ribosyltransferase; Provisional; cl00409
GOHSU43	CDS	GOHSU_43_00120	13314..13778		hypothetical protein [migap]				hypothetical protein		F1YPK8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19315 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.996) with cleavage site probability 0.719 at residue 26;KEGG: hypothetical protein LOC100077275
GOHSU43	CDS	GOHSU_43_00130	complement(13786..15102)		UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase [nite]	 murF [nite]		YM304_22300 :UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase : murF : 6.3.2.10 : Acidimicrobidae bacterium YM16-304	UDP-N-acetylmuramoylalanyl-D-glutamyl-2, 6-diaminopimelate/ D-alanyl-D-alanyl ligase		F1YPK9_9ACTO UDP-N-acetylmuramoyl-tripeptide--D-ala nyl-D-alanine ligase OS=Gordonia neofelifaecis NRRL B-5939 5 GN=SCNU_19320 PE=3 SV=1;UDP-N-acetylmuramoylalanyl-D-glutamyl-2, 6-diaminop imelate/ D-alanyl-D-alanyl ligase [Nocardiopsis dassonvill ei subsp. dassonvillei DSM 43111];COGs: COG0770 UDP-N-acetylmuramyl pentapeptide synthase; InterPro IPR013221:IPR004101:IPR005863:IPR000713; KEGG: sco:SCO2088 UDP-N-acetylmuramoylalanyl-D-glutamyl-2,6-diaminopimelate- D-alanyl-a lanyl ligase; PFAM: Mur ligase middle domain protein; cytoplasmic peptidoglycan synthetase domain protein; SPTR: C1YNT8 UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase; TIGRFAM:UDP-N-acetylmuramoylalanyl-D-glutamyl-2,6-d iaminopimelate/D-alanyl-D-alanylligase; PFAM: Mur ligase family, glutamate ligase domain; Mur ligase family, catalytic domain; Mur ligase middle domain; TIGRFAM: UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase;Mur ligase family, catalytic domain; pfam01225;UDP-N-acetylmuramoyl-tripeptide--D-alanyl-D-alanine ligase; TIGR01143;Mur ligase middle domain; pfam08245;Mur ligase family, glutamate ligase domain; pfam02875
GOHSU43	CDS	GOHSU_43_00140	15170..16249		D-alanine--D-alanine ligase [hamap]	ddl [hamap]	MF_00047:D-alanine--D-alanine ligase:ddl 	YM304_22290 :D-alanine--D-alanine ligase : ddl : 6.3.2.4 : Acidimicrobidae bacterium YM16-304	D-alanine--D-alanine ligase		F1YPL0_9ACTO D-alanine--D-alanine ligase OS=Gordoni a neofelifaecis NRRL B-59395 GN=ddl PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: sgl:SG0899 D-alanine-D-alanine ligase A; TIGRFAM: D-alanine/D-alanine ligase; PFAM: D-alanine--D-alanine ligase domain protein; ATP-dependent carboxylate-amine ligase domain protein ATP- grasp;D-alanyl-alanine synthetase A; Reviewed; PRK01966;D-ala D-ala ligase N-terminus; pfam01820;ATP-grasp domain; cl03087
GOHSU43	tRNA	GOHSU_43_t0020	complement(16306..16389)		not assigned						identified by tRNAscan-SE; putative
GOHSU43	CDS	GOHSU_43_00150	complement(16459..16668)		hypothetical protein [migap]				hypothetical protein		G7GM50_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_20_02250 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU43	CDS	GOHSU_43_00160	complement(16744..17199)		putative tRNA-specific adenosine deaminase [nite]	 tadA [nite]		RER_03300 :putative tRNA-specific adenosine deaminase : tadA : 3.5.4.- : Rhodococcus erythropolis PR4	cytidine and deoxycytidylate deaminase		G4I0A1_MYCRH CMP/dCMP deaminase zinc-binding OS=Myc obacterium rhodesiae JS60 GN=MycrhDRAFT_1982 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00383;Nucleoside deaminases include adenosine, guanine and cytosine deaminases. These enzymes are Zn dependent and catalyze the deamination of nucleosides. The zinc ion in the active site plays a central role in the proposed catalytic mechanism, activating a...; cd01285;nucleoside/Zn binding site;dimer interface [polypeptide binding];catalytic motif [active]
GOHSU43	CDS	GOHSU_43_00170	complement(17204..17743)		hypothetical protein [nite]			ROP_40850 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F9VVF9_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_056_00210 PE=4 SV= 1;unnamed protein product [Streptosporangium roseum D SM 43021];putative tRNA adenosine deaminase-associated protein; TIGR03941
GOHSU43	CDS	GOHSU_43_00180	17718..18755		prephenate dehydrogenase [nite]	 tyrA [nite]		RER_03280 :prephenate dehydrogenase : tyrA : 1.3.1.12 : Rhodococcus erythropolis PR4	prephenate dehydrogenase	tyrA	F1YPL4_9ACTO Prephenate dehydrogenase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_19345 PE=4 SV=1;prephenate dehydrogenase [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 4-hydroxyphenylpyruvate from prephenate;prephenate dehydrogenase; Validated;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU43	CDS	GOHSU_43_00190	complement(18765..19355)		conserved hypothetical protein [nite]			RER_03270 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YPL5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19350 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Putative glycolipid-binding; cl01425
GOHSU43	CDS	GOHSU_43_00200	complement(19375..20106)		not assigned						F1YPL6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19355 PE=4 SV=1
GOHSU43	CDS	GOHSU_43_00210	complement(20156..21148)		putative AraC family transcriptional regulator [nite]			ROP_41840 :putative AraC family transcriptional regulator : : : Rhodococcus opacus B4	AraC family transcriptional regulator		H5UL82_9ACTO Putative AraC family transcriptional r egulator OS=Gordonia terrae NBRC 100016 GN=GOTRE_170_00210 PE=4 SV=1;AraC family transcriptional regulator [Pseudomonas mendocina ymp];PFAM: helix-turn-helix- domain containing protein, AraC type;Arabinose-binding domain of AraC transcription regulator, N-term; pfam12625;helix_turn_helix, arabinose operon control protein; smart00342;Bacterial regulatory helix-turn-helix proteins, AraC family; pfam00165
GOHSU43	CDS	GOHSU_43_00220	21232..21489		not assigned						
GOHSU43	CDS	GOHSU_43_00230	21518..23086		probable fatty-acid--CoA ligase [nite]	 fadD [nite]		RER_49140 :probable fatty-acid--CoA ligase : fadD : 6.2.1.- : Rhodococcus erythropolis PR4	AMP-dependent synthetase/ligase		D5P1W7_9MYCO Feruloyl-CoA synthetase OS=Mycobacteri um parascrofulaceum ATCC BAA-614 GN=fcs3 PE=4 SV=1;AMP-dependent synthetase and ligase [Cupriavidus me tallidurans CH34];long-chain-fatty-acid--CoA ligase; Validated;AMP-binding enzyme; cl15778;Protein Kinases, catalytic domain; cl09925
GOHSU43	CDS	GOHSU_43_00240	23115..23837		putative oxidoreductase [nite]			YM304_16980 :putative oxidoreductase : : : Acidimicrobidae bacterium YM16-304	short-chain dehydrogenase/reductase SDR		E2MHS7_PSEUB Oxidoreductase, short chain dehydrogen ase/reductase family OS=Pseudomonas syringae pv. tomato T1 GN=PSPTOT1_1429 PE=4 SV=1;short-chain dehydrogenase/reductase SDR [Burkholder ia ambifaria MC40- 6];PFAM: NAD-dependent epimerase/dehydratase; short-chain dehydrogenase/reductase SDR; KR domain protein; KEGG: bur:Bcep18194_B3149 short-chain dehydrogenase/reductase SDR;3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05653;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU43	CDS	GOHSU_43_00250	complement(23842..25425)		hypothetical protein [nite]			RER_12470 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GLZ5_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_20_01680 PE=4 SV=1;unnamed protein product [Propionibacterium acnes Ty peIA2 P.acn33]
GOHSU43	tRNA	GOHSU_43_t0030	complement(25584..25659)		not assigned						identified by tRNAscan-SE; putative
GOHSU43	tRNA	GOHSU_43_t0040	complement(25731..25819)		not assigned						identified by tRNAscan-SE; putative
GOHSU43	CDS	GOHSU_43_00260	25938..26996		histidinol-phosphate aminotransferase [hamap]	hisC [hamap]	MF_01023:histidinol-phosphate aminotransferase:hisC 	RER_03130 :putative aromatic amino acid aminotransferase : aroT : 2.6.1.57 : Rhodococcus erythropolis PR4	aminotransferase	pat	H5UGS6_9ACTO Putative phenylalanine aminotransferas e OS=Gordonia terrae NBRC 100016 GN=pat PE=3 SV=1;aminotransferase [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;histidinol-phosphate aminotransferase; Validated;Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU43	CDS	GOHSU_43_00270	complement(26997..27980)		putative mycolyltransferase [nite]			ROP_40020 :putative mycolyltransferase : : 2.3.1.- : Rhodococcus opacus B4	mycolyltransferase		F1YP70_9ACTO Putative esterase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_18597 PE=4 SV=1;mycolyltransferase [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Predicted esterase [General function prediction only];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU43	CDS	GOHSU_43_00280	28075..28512		hypothetical protein [nite]			ROP_35380 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0RIW2_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_075_01110 PE =4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: rop:ROP_35380 hypothetical protein;4-hydroxybenzoyl-CoA thioesterase (4HBT). Catalyzes the final step in the 4-chlorobenzoate degradation pathway in which 4-chlorobenzoate is converted to 4-hydroxybenzoate in certain soil-dwelling bacteria. 4HBT forms a homotetramer with four active sites; cd00586
GOHSU43	CDS	GOHSU_43_00290	28653..28892		putative transcriptional regulator [nite]			CSE_04440 :putative transcriptional regulator : : : Caldisericum exile AZM16c01	FeoA family protein		D4H981_PROAS FeoA domain protein OS=Propionibacteri um acnes (strain SK137) GN=feoA PE=4 SV=1;FeoA family protein [Tepidanaerobacter sp. Re1];collected before 2009;KEGG: cpr:CPR_0972 ferrous iron repressor; PFAM: Ferrous iron transporter, FeoA subunit; SMART: Ferrous iron transporter, FeoA subunit;FeoA domain; cl00838
GOHSU43	CDS	GOHSU_43_00300	29059..31017		ferrous iron transport protein B [nite]	 feoB [nite]		SLG_36840 :ferrous iron transport protein B : feoB : : Sphingobium sp. SYK-6	ferrous iron transport protein B		H5UVX0_9MICO Ferrous iron transport protein B OS=Mo bilicoccus pelagius NBRC 104925 GN=feoB PE=4 SV=1;ferrous iron transport protein B [Actinomyces visco sus C505];Ferrous iron transport protein B (FeoB) family; cd01879;GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;Nucleoside recognition; cl00486;Ferrous iron transport protein B C terminus; pfam07664;Fe2+ transport system protein B [Inorganic ion transport and metabolism]; COG0370
GOHSU43	CDS	GOHSU_43_00310	31019..31663		NifU domain-containing protein [migap]				NifU domain-containing protein		C0W559_9ACTO NifU domain protein OS=Actinomyces uro genitalis DSM 15434 GN=HMPREF0058_1003 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];PFAM: nitrogen-fixing NifU domain protein; KEGG: lxx:Lxx21000 thioredoxin-like protein
GOHSU43	CDS	GOHSU_43_00320	complement(31716..32930)		hypothetical protein [nite]			RER_40830 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YP72_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18607 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU43	CDS	GOHSU_43_00330	complement(33021..33803)		hypothetical protein [nite]			KSE_60840 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		F9VQW2_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_016_01240 PE=4 SV= 1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: scl:sce1423 LysR family transcriptional regulator;The substrate binding domain of LysR-type transcriptional regulators (LTTRs), a member of the type 2 periplasmic binding fold protein superfamily; cl11398;dimerization interface [polypeptide binding]
GOHSU43	CDS	GOHSU_43_00340	33849..34616		conserved hypothetical protein [nite]			RER_38170 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		C3JRS8_RHOER Putative uncharacterized protein OS=Rh odococcus erythropolis SK121 GN=RHOER0001_0021 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314]
GOHSU43	CDS	GOHSU_43_00350	34683..35933		conserved hypothetical protein [nite]			RER_18090 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative regulatory protein		F1YP78_9ACTO GAF domain-containing protein OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_18637 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transcriptional activator of acetoin/glycerol metabolism [Secondary metabolites biosynthesis, transport, and catabolism / Transcription]; COG3284
GOHSU43	CDS	GOHSU_43_00360	36214..37737		aldehyde dehydrogenase [nite]			ROP_61390 :aldehyde dehydrogenase : : 1.2.1.3 : Rhodococcus opacus B4	aldehyde dehydrogenase		D0LAD0_GORB4 Aldehyde dehydrogenase (NAD(+)) OS=Gor donia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 319 8 / NCTC 10667) GN=Gbro_0109 PE=3 SV=1;aldehyde dehydrogenase [Arthrobacter aurescens TC1] ;identified by match to protein family HMM PF00171;Ralstonia eutrophus NAD+-dependent acetaldehyde dehydrogenase II-like; cd07116;Aldehyde dehydrogenase family; pfam00171;NAD(P) binding site [chemical binding];catalytic residues [active]
GOHSU43	CDS	GOHSU_43_00370	37818..38240		conserved hypothetical protein [nite]			RER_18110 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GX49_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_004_00140 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF779);Protein of unknown function (DUF779); cl01432
GOHSU43	CDS	GOHSU_43_00380	38397..39926		hypothetical protein [nite]			ROP_02890 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7H5P4_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_064_01710 PE=4 SV=1;unnamed protein product [Acaryochloris marina MBIC1 1017];type strain of Acaryochloris marina;2-polyprenyl-6-methoxyphenol hydroxylase and related FAD-dependent oxidoreductases [Coenzyme metabolism / Energy production and conversion]; COG0654
GOHSU43	CDS	GOHSU_43_00390	39930..41123		hypothetical membrane protein [nite]			RER_26840 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	predicted protein		G7GL91_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_19_00530 PE=4 SV=1;predicted protein [Streptomyces sp. AA4]
GOHSU43	CDS	GOHSU_43_00400	41137..41727		putative TetR family transcriptional regulator [nite]			ROP_04660 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		G7H5P2_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia araii NBRC 100433 GN=GOARA_064_01690 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m gilvum PYR- GCK];PFAM: regulatory protein, TetR; KEGG: mva:Mvan_5152 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU43	CDS	GOHSU_43_00410	complement(41760..42833)		hypothetical protein [nite]			MLP_21650 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F5XE06_MICPN Putative uncharacterized protein OS=Mi crolunatus phosphovorus (strain ATCC 700054 / DSM 10555 / JCM 9379 / NBRC 101784 / NCIMB 13414 / VKM Ac-1990 / NM-1) GN=MLP_21650 PE=3 SV=1;integral membrane protein [Rhodococcus equi 103S];Conserved hypothetical protein 698; cl01075
GOHSU43	CDS	GOHSU_43_00420	42890..43813		putative LysR family transcriptional regulator [nite]			RER_03650 :putative LysR family transcriptional regulator : : : Rhodococcus erythropolis PR4	LysR family transcriptional regulator		H0R046_9ACTO Putative LysR family transcriptional r egulator OS=Gordonia effusa NBRC 100432 GN=GOEFS_054_00610 PE=4 SV=1;LysR family transcriptional regulator [Rhodococcus jostii RHA1];Transcriptional regulator [Transcription]; COG0583;Helix-turn-helix domains; cl00088;C-terminal substrate binding domain of LysR-type transcriptional regulator CysL, which activates the transcription of the cysJI operon encoding sulfite reductase, contains the type 2 periplasmic binding fold; cd08420;putative dimerization interface [polypeptide binding]
GOHSU43	CDS	GOHSU_43_00430	complement(43810..44763)		putative dehydrogenase [nite]			RER_02980 :putative dehydrogenase : : : Rhodococcus erythropolis PR4	oxidoreductase		H6MU65_GORPV Putative dehydrogenase OS=Gordonia pol yisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c04720 PE =4 SV=1;oxidoreductase [Rhodococcus opacus B4];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU43	CDS	GOHSU_43_00440	complement(44732..45148)		conserved hypothetical protein [nite]			PH0267 :conserved hypothetical protein : : : Pyrococcus horikoshii OT3	signal-transduction protein		Q1BA80_MYCSS Putative signal-transduction protein w ith CBS domains OS=Mycobacterium sp. (strain MCS) GN=Mmcs_ 2096 PE=4 SV=1;signal-transduction protein [Thermoproteus uzoniens is 768- 20];FOG: CBS domain [General function prediction only];The CBS domain, named after human CBS, is a small domain originally identified in cystathionine beta-synthase and is subsequently found in a wide range of different proteins. CBS domains usually occur in tandem repeats. They associate to form a so-called...; cd04623
##GOHSU44	contig00251				44484
GOHSU44	CDS	GOHSU_44_00010	complement(48..1772)		putative iron-siderophore ABC transporter permease/ATP-binding protein [nite]			RER_43340 :putative iron-siderophore ABC transporter permease/ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter		G7H575_9ACTO Putative iron-siderophore ABC transpor ter permease/ATP-binding protein OS=Gordonia araii NBRC 10 0433 GN=GOARA_064_00020 PE=3 SV=1;ABC transporter [Nocardia farcinica IFM 10152];ABC transporter transmembrane region; cl00549;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;ABC-type antimicrobial peptide transport system, ATPase component [Defense mechanisms]; COG1136;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU44	CDS	GOHSU_44_00020	complement(1769..4336)		ABC transporter [migap]				ABC transporter		F1YPB9_9ACTO Transmembrane ATP-binding protein ABC transporter OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _18868 PE=3 SV=1;ABC transporter [Nocardia farcinica IFM 10152];ABC-type multidrug transport system, ATPase and permease components [Defense mechanisms]; COG1132;Predicted ATPase involved in cell division [Cell division and chromosome partitioning]; COG2884;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU44	CDS	GOHSU_44_00030	4475..4840		conserved hypothetical protein [nite]			RER_38760 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	glucitol operon activator-like protein		F1YPC0_9ACTO Glucitol operon activator-like protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18873 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Glucitol operon activator protein (GutM); cl01890
GOHSU44	CDS	GOHSU_44_00040	4837..5241		hypothetical membrane protein [nite]			ROP_11390 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H0R6E2_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_121_00450 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: bba:Bd3290 hypothetical protein;Domain of unknown function (DUF3817); cl14844
GOHSU44	CDS	GOHSU_44_00050	complement(5264..5878)	case	dITP/XTP pyrophosphatase [hamap]	rdgB [hamap]	MF_01405:dITP/XTP pyrophosphatase:rdgB 	ROP_11400 :nucleoside-triphosphatase : : 3.6.1.15 : Rhodococcus opacus B4	deoxyribonucleotide triphosphate pyrophosphatase		F1YPC3_9ACTO Non-canonical purine NTP pyrophosphata se, rdgB/HAM1 family protein OS=Gordonia neofelifaecis NRR L B-59395 GN=SCNU_18888 PE=3 SV=1;deoxyribonucleotide triphosphate pyrophosphatase [C orynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;HAM1-like protein; Rec-dependent growth; RgdB; yggV; it is suspected that this protein functions to remove misincorporated bases such as xanthine or hypoxanthine;NTPase/HAM1.  This family consists of the HAM1 protein and pyrophosphate-releasing xanthosine/ inosine triphosphatase. HAM1 protects the cell against mutagenesis by the base analog 6-N-hydroxylaminopurine (HAP) in E. Coli and S. cerevisiae. A...; cd00515;dimerization interface [polypeptide binding]
GOHSU44	CDS	GOHSU_44_00060	complement(5886..6641)		ribonuclease PH [hamap]	rph [hamap]	MF_00564:ribonuclease PH:rph 	RER_38790 :ribonuclease PH : rph : 2.7.7.56 : Rhodococcus erythropolis PR4	ribonuclease PH	rph	H5U4S3_9ACTO Ribonuclease PH OS=Gordonia sputi NBRC 100414 GN=rph PE=3 SV=1;ribonuclease PH [Corynebacterium glutamicum ATCC 13 032];IS fingerprint type 4-5;RNase PH; tRNA nucleotidyltransferase; forms hexamers in Bacillus subtilis; phosphoroltic 3'-5' exoribonuclease; involved in maturation of tRNA precursors and removes terminal nucleotides near CCA acceptor arms of mature tRNAs;ribonuclease PH; Reviewed; PRK00173;Ribonuclease PH; cd11362;hexamer interface [polypeptide binding]
GOHSU44	CDS	GOHSU_44_00070	complement(6711..7490)		conserved hypothetical protein [nite]			RER_38800 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	beta-lactamase domain-containing protein		F1YPC5_9ACTO Beta-lactamase domain-containing prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18898 PE =4 SV=1;beta-lactamase domain-containing protein [Actinosyn nema mirum DSM 43827];PFAM: beta-lactamase domain protein; KEGG: dal:Dalk_3435 beta-lactamase domain protein;Metallo-beta-lactamase superfamily; cl00446
GOHSU44	CDS	GOHSU_44_00080	7643..8728		GCN5-like N-acetyltransferase [migap]				GCN5-like N-acetyltransferase		A6BJQ4_9FIRM Putative uncharacterized protein OS=Do rea longicatena DSM 13814 GN=DORLON_02549 PE=4 SV=1;GCN5-like N-acetyltransferase [Halopiger xanaduensi s SH- 6];PFAM: GCN5-related N-acetyltransferase; KEGG: htu:Htur_0544 GCN5-related N-acetyltransferase;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357;N-Acyltransferase superfamily: Various enzymes that characteristically catalyze the transfer of an acyl group to a substrate; cd04301;Coenzyme A binding pocket [chemical binding]
GOHSU44	CDS	GOHSU_44_00090	complement(8754..10106)		UDP-N-acetylmuramoylalanine--D-glutamate ligase [nite]	 murD [nite]		CSE_11830 :UDP-N-acetylmuramoylalanine--D-glutamate ligase : murD : 6.3.2.9 : Caldisericum exile AZM16c01	UDP-N-acetylmuramoylalanine/d-glutamate ligase		D3CTD0_9ACTO UDP-N-acetylmuramoylalanine/D-glutamat e ligase OS=Frankia sp. EUN1f GN=FrEUN1fDRAFT_0820 PE=3 SV =1;UDP-N-acetylmuramoylalanine/d-glutamate ligase [Mic romonospora sp. L5];UDP-N-acetylmuramoyl-L-alanyl-D-glutamate synthetase; Provisional; PRK04690;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;KEGG: saq:Sare_3229 UDP-N-acetylmuramoyl-L-alanyl-D-glutamate synthetase; TIGRFAM: UDP-N-acetylmuramoylalanine/D-glutamate ligase; PFAM: Mur ligase middle domain protein; cytoplasmic peptidoglycan synthetase domain protein;Mur ligase middle domain; pfam08245
GOHSU44	CDS	GOHSU_44_00100	complement(10087..11460)		hypothetical protein [migap]				hypothetical protein		D3CTC9_9ACTO Putative uncharacterized protein OS=Fr ankia sp. EUN1f GN=FrEUN1fDRAFT_0819 PE=4 SV=1;unnamed protein product [Salinispora arenicola CNS- 205];KEGG: stp:Strop_3006 hypothetical protein
GOHSU44	CDS	GOHSU_44_00110	complement(11469..12107)		conserved hypothetical membrane protein [nite]			RER_38810 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	rhomboid family protein		H5U4S6_9ACTO Peptidase S54 family protein OS=Gordon ia sputi NBRC 100414 GN=GOSPT_114_00780 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: rhomboid family protein; KEGG: dat:HRM2_28470 hypothetical protein;Rhomboid family; cl11446
GOHSU44	CDS	GOHSU_44_00120	complement(12104..13213)		putative hydrolase [nite]			RER_38820 :putative hydrolase : : : Rhodococcus erythropolis PR4	L-aminopeptidase/D-esterase		F1YPC8_9ACTO Peptidase S58 DmpA OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_18913 PE=4 SV=1;L-aminopeptidase/D-esterase [Corynebacterium glutam icum ATCC 13032];IS fingerprint type 4-5;nylC-like family; composed of proteins with similarity to Flavobacterium endo-type 6-aminohexanoate-oligomer hydrolase (EIII), the product of the nylon oligomer degradation gene, nylC. EIII is an amide hydrolase that catalyzes the degradation of...; cd02252;putative active site pocket [active]
GOHSU44	CDS	GOHSU_44_00130	complement(13210..13794)		hypothetical protein [nite]			ROP_11490 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YPC9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18918 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF2017); pfam09438
GOHSU44	CDS	GOHSU_44_00140	complement(13791..14093)		ATP-dependent Clp protease adapter protein ClpS [hamap]	clpS [hamap]	MF_00302:ATP-dependent Clp protease adapter protein ClpS:clpS 	RER_38840 :putative ATP-dependent Clp protease adaptor protein ClpS : clpS : : Rhodococcus erythropolis PR4	ATP-dependent Clp protease adaptor protein ClpS	clpS	F1YPD0_9ACTO ATP-dependent Clp protease adapter pro tein ClpS OS=Gordonia neofelifaecis NRRL B-59395 GN=clpS P E=3 SV=1;ATP-dependent Clp protease adaptor protein ClpS [Co rynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;involved in the modulation of the specificity of the ClpAP-mediated ATP-dependent protein degradation; binds to the N-terminal domain of the chaperone ClpA;ATP-dependent Clp protease adaptor protein ClpS; cl00933
GOHSU44	CDS	GOHSU_44_00150	14113..15423		probable nicotinate phosphoribosyltransferase [nite]	 pncB [nite]		RER_38850 :probable nicotinate phosphoribosyltransferase : pncB : 2.4.2.11 : Rhodococcus erythropolis PR4	nicotinate phosphoribosyltransferase		H5TLT8_9ACTO Nicotinate phosphoribosyltransferase O S=Gordonia otitidis NBRC 100426 GN=pncB PE=3 SV=1;nicotinate phosphoribosyltransferase [Corynebacteri um glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of 5-phospho-alpha-D-ribose 1-diphosphate and nicotinate from nicotinate D-ribonucleotide and diphosphate;nicotinate phosphoribosyltransferase; Validated;Nicotinate phosphoribosyltransferase (NAPRTase), subgroup A. Nicotinate phosphoribosyltransferase catalyses the formation of NAMN and PPi from 5-phosphoribosy -1-pyrophosphate (PRPP) and nicotinic acid, this is the first, and also rate limiting, reaction...; cd01570
GOHSU44	CDS	GOHSU_44_00160	15459..16049		pyrazinamidase/nicotinamidase [nite]	 pncA [nite]		ROP_11520 :pyrazinamidase/nicotinamidase : pncA : 3.5.1.-/3.5.1.19 : Rhodococcus opacus B4	nicotinamidase		H5UAK3_9ACTO Pyrazinamidase/nicotinamidase OS=Gordo nia terrae NBRC 100016 GN=pncA PE=4 SV=1;nicotinamidase [Streptomyces griseus subsp. griseus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;Nicotinamidase/pyrazinamidase (PZase). Nicotinamidase, a ubiquitous enzyme in prokaryotes, converts nicotinamide to nicotinic acid (niacin) and ammonia, which in turn can be recycled to make nicotinamide adenine dinucleotide (NAD). The same enzyme is...; cd01011;Isochorismatase family; pfam00857;catalytic triad [active];metal binding site [ion binding];conserved cis-peptide bond
GOHSU44	CDS	GOHSU_44_00170	16046..18088		probable ATP-dependent DNA helicase DinG [nite]	 dinG [nite]		RER_38870 :probable ATP-dependent DNA helicase DinG : dinG : 3.6.1.- : Rhodococcus erythropolis PR4	ATP-dependent helicase		F1YPD3_9ACTO Helicase c2 OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18938 PE=4 SV=1;ATP-dependent helicase [Streptomyces clavuligerus A TCC 27064];Rad3-related DNA helicases [Transcription / DNA replication, recombination, and repair]; COG1199;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099
GOHSU44	CDS	GOHSU_44_00180	18085..19956		1,4-alpha-glucan-branching enzyme [hamap]	glgB [hamap]	MF_00685:1,4-alpha-glucan-branching enzyme:glgB 	RER_38900 :1,4-alpha-glucan branching enzyme : glgB : 2.4.1.18 : Rhodococcus erythropolis PR4	1,4-alpha-glucan branching enzyme	glgB	F1YPD4_9ACTO 1,4-alpha-glucan branching enzyme GlgB OS=Gordonia neofelifaecis NRRL B-59395 GN=glgB PE=3 SV=1;1,4-alpha-glucan branching protein [Mycobacterium s p. JDM601];glycogen branching enzyme; Provisional;Alpha amylase catalytic domain found in the Glycogen branching enzyme (also called 1,4-alpha-glucan branching enzyme); cd11322;catalytic site [active];Alpha amylase, C-terminal all-beta domain; cl02706
GOHSU44	CDS	GOHSU_44_00190	complement(20002..20931)		conserved hypothetical protein [nite]			RER_38920 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	thioredoxin domain-containing protein		F9VXZ3_9ACTO Putative thioredoxin OS=Gordonia alkan ivorans NBRC 16433 GN=GOALK_078_00150 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: thioredoxin domain; KEGG: avi:Avi_4245 thioredoxin;Thioredoxin domain-containing protein [Posttranslational modification, protein turnover, chaperones];ybbN protein family; ybbN is a hypothetical protein containing a redox-inactive TRX-like domain. Its gene has been sequenced from several gammaproteobacteria and actinobacteria; cd02956
GOHSU44	CDS	GOHSU_44_00200	complement(20928..21329)		hypothetical protein [nite]			RER_38930 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YPD7_9ACTO Integral membrane protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18958 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Domain of unknown function (DUF3817); cl14844
GOHSU44	CDS	GOHSU_44_00210	complement(21407..22600)		acetyl-CoA acetyltransferase [nite]			ROP_30620 :acetyl-CoA acetyltransferase : : 2.3.1.9 : Rhodococcus opacus B4	acetyl-CoA acetyltransferase		F1YPD8_9ACTO Putative acyltransferase OS=Gordonia n eofelifaecis NRRL B-59395 GN=SCNU_18963 PE=3 SV=1;acetyl-CoA acetyltransferase [Rhodococcus opacus B4 ];putative acyltransferase; Provisional;Catalyzes the synthesis of acetoacetyl coenzyme A from two molecules of acetyl coenzyme A. It can also act as a thiolase, catalyzing the reverse reaction and generating two-carbon units from the four-carbon product of fatty acid oxidation; in Rhizobia and Ralstonia is involved in PHB biosynthesis;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU44	CDS	GOHSU_44_00220	22728..23207		putative methylmalonyl-CoA epimerase [nite]			ROP_11680 :putative methylmalonyl-CoA epimerase : : 5.1.99.1 : Rhodococcus opacus B4	putative methylmalonyl-CoA epimerase		H6MW71_GORPV Putative methylmalonyl-CoA epimerase O S=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN= GPOL_c18400 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Methylmalonyl-CoA epimerase (MMCE); cd07249;Glyoxalase/Bleomycin resistance protein/Dioxygenase superfamily; pfam00903;dimer interface [polypeptide binding];substrate binding site [chemical binding];metal binding site [ion binding]
GOHSU44	CDS	GOHSU_44_00230	23340..24269		hypothetical protein [nite]			ROP_11690 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YPE0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18973 PE=4 SV=1;unnamed protein product [Salinispora tropica CNB- 4 40]
GOHSU44	CDS	GOHSU_44_00240	complement(24325..24999)		UPF0286 protein <locus_tag> [hamap]		MF_00722:UPF0286 protein <locus_tag>: 	ROP_11710 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		G7GX28_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_001_00230 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function DUF91; cl00709;PFAM: protein of unknown function DUF91; KEGG: bvi:Bcep1808_6934 hypothetical protein
GOHSU44	CDS	GOHSU_44_00250	complement(25069..26355)		L-ornithine N5-oxygenase [nite]			ROP_23320 :L-ornithine N5-oxygenase : : 1.13.12.- : Rhodococcus opacus B4	putative lysine/ornithine N-monooxygenase		H6N2H0_GORPV Putative lysine/ornithine N-monooxygen ase OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2 ) GN=GPOL_c23650 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];L-lysine 6-monooxygenase (NADPH-requiring); pfam13434;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU44	CDS	GOHSU_44_00260	26505..27674		isochorismate synthase [nite]			ROP_48860 :isochorismate synthase : : 5.4.4.2 : Rhodococcus opacus B4	isochorismate synthase		H0REJ4_9ACTO Putative isochorismate synthase OS=Gor donia polyisoprenivorans NBRC 16320 GN=GOPIP_054_00600 PE= 4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];chorismate binding enzyme; cl10555
GOHSU44	CDS	GOHSU_44_00270	complement(27706..27939)		hypothetical protein [nite]			ROP_48900 :hypothetical protein : : : Rhodococcus opacus B4	isochorismatase	entB	H0R693_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_120_00540 PE=4 SV=1;isochorismatase [Escherichia coli str. K-12 substr. DH10B]
GOHSU44	CDS	GOHSU_44_00280	28006..29349		putative siderophore export protein [nite]			ROP_48910 :putative siderophore export protein : : : Rhodococcus opacus B4	enterobactin exporter EntS		H0R697_9ACTO Putative siderophore export protein OS =Gordonia effusa NBRC 100432 GN=GOEFS_120_00580 PE=4 SV=1;enterobactin exporter EntS [Rhodococcus opacus B4];enterobactin exporter EntS; Provisional;protein p43; inner membrane protein that exports enterobactin to the periplasmic space; member of the major facilitator superfamily (MFS) of transporters;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174
GOHSU44	CDS	GOHSU_44_00290	29421..30581		hypothetical protein [nite]			ROP_50950 :hypothetical protein : : : Rhodococcus opacus B4	putative secretory lipase family protein		G7GSC6_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_54_00110 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Secretory lipase; pfam03583
GOHSU44	CDS	GOHSU_44_00300	complement(30587..31420)		putative ABC transporter ATP-binding protein [nite]			RER_43410 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ferric enterobactin transport ATP-binding protein	fepC	C3JWV1_RHOER Ferric enterobactin transport ATP-bind ing protein FepC OS=Rhodococcus erythropolis SK121 GN=RHOE R0001_3572 PE=3 SV=1;ferric enterobactin transport ATP-binding protein [ Arthrobacter aurescens TC1];identified by match to protein family HMM PF00005; match to protein family HMM PF02463;iron-enterobactin transporter ATP-binding protein; Provisional;ABC transporters, involved in the uptake of siderophores, heme, and vitamin B12, are widely conserved in bacteria and archaea.  Only very few species lack representatives of the siderophore family transporters. The E. coli BtuCD protein is an ABC...; cd03214;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU44	CDS	GOHSU_44_00310	complement(31420..32484)		putative ABC transporter permease protein [nite]			RER_43420 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	ferric enterobactin transport system, permease component	fepG	E9SZQ4_COREQ Ferric siderophore ABC superfamily ATP binding cassette transporter, membrane protein OS=Rhodoco ccus equi ATCC 33707 GN=fepG PE=4 SV=1;ferric enterobactin transport system, permease comp onent [Arthrobacter aurescens TC1];identified by match to protein family HMM PF01032;ABC-type enterobactin transport system, permease component [Inorganic ion transport and metabolism]; COG4779;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;ABC-ATPase subunit  interface;dimer interface [polypeptide binding];putative PBP binding regions
GOHSU44	CDS	GOHSU_44_00320	complement(32481..33581)		putative ABC transporter permease protein [nite]			RER_43430 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	iron ABC transporter		C3JWU9_RHOER Ferric enterobactin transport system p ermease protein FepD OS=Rhodococcus erythropolis SK121 GN= RHOER0001_3570 PE=4 SV=1;iron ABC transporter [Streptomyces griseus subsp. g riseus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;ABC-ATPase subunit  interface;dimer interface [polypeptide binding];putative PBP binding regions
GOHSU44	CDS	GOHSU_44_00330	33776..41425		non-ribosomal peptide synthetase [nite]			ROP_02580 :non-ribosomal peptide synthetase : : : Rhodococcus opacus B4	non-ribosomal peptide synthetase		H6R1H4_NOCCG Non-ribosomal peptide synthetase OS=No cardia cyriacigeorgica (strain GUH-2) GN=NOCYR_4800 PE=4 S V=1;non-ribosomal peptide synthetase [Rhodococcus opacu s B4];peptide synthase; Provisional;AMP-binding enzyme; cl15778;Phosphopantetheine attachment site; cl09936;HxxPF-repeated domain; pfam13745;non-ribosomal peptide synthase domain TIGR01720
GOHSU44	CDS	GOHSU_44_00340	41515..42564		putative ABC transporter substrate-binding protein [nite]			RER_54590 :putative ABC transporter substrate-binding protein : : : Rhodococcus erythropolis PR4	iron-siderophore ABC transporter substrate-binding protein		G7GSC4_9ACTO Putative ABC transporter substrate bin ding protein OS=Gordonia amarae NBRC 15530 GN=GOAMR_54_000 90 PE=4 SV=1;iron-siderophore ABC transporter substrate-binding protein [Streptomyces griseus subsp. griseus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;iron-dicitrate transporter substrate-binding subunit; Provisional; PRK11411;Fe3+-siderophore binding domain FhuD.  These proteins have been shown to function as initial receptors in ABC transport of Fe3+-siderophores in many eubacterial species. They belong to the TroA-like superfamily of helical backbone metal receptor proteins...; cd01146;siderophore binding site
GOHSU44	CDS	GOHSU_44_00350	42564..>44484	3x codon	non-ribosomal peptide synthetase [migap]				non-ribosomal peptide synthetase		G7GSC3_9ACTO Putative non-ribosomal peptide synthet ase OS=Gordonia amarae NBRC 15530 GN=GOAMR_54_00080 PE=4 S V=1;non-ribosomal peptide synthetase [Rhodococcus josti i RHA1];peptide synthase; Provisional
##GOHSU45	contig00261				43946
GOHSU45	CDS	GOHSU_45_00010	<1..286	3x codon	tyrosine recombinase XerC [nite]	 xerC [nite]		RER_25390 :tyrosine recombinase XerC : xerC : : Rhodococcus erythropolis PR4	integrase family protein		G7H522_9ACTO Tyrosine recombinase xerC OS=Gordonia araii NBRC 100433 GN=xerC PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: integrase family protein; integrase domain protein SAM domain protein; KEGG: hha:Hhal_1202 tyrosine recombinase XerC;site-specific tyrosine recombinase XerC; Reviewed; PRK00236;DNA breaking-rejoining enzymes, C-terminal catalytic domain. The DNA breaking-rejoining enzyme superfamily includes type IB topoisomerases and tyrosine recombinases that share the same fold in their catalytic domain containing six conserved active site...; cl00213
GOHSU45	CDS	GOHSU_45_00020	complement(291..839)		conserved hypothetical protein [nite]			RER_25410 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	peptidase M23		H0QYZ6_9ACTO Peptidase M23 family protein OS=Gordon ia effusa NBRC 100432 GN=GOEFS_046_00030 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: peptidase M23; KEGG: ade:Adeh_3386 peptidase M23B;Peptidase family M23; pfam01551
GOHSU45	CDS	GOHSU_45_00030	1072..1920		30S ribosomal protein S2 [nite]	 rpsB [nite]		ROP_66120 :30S ribosomal protein S2 : rpsB : : Rhodococcus opacus B4	30S ribosomal protein S2		F1YIB9_9ACTO 30S ribosomal protein S2 OS=Gordonia n eofelifaecis NRRL B-59395 GN=rpsB PE=3 SV=1;30S ribosomal protein S2 [Streptomyces clavuligerus ATCC 27064];Ribosomal protein S2 (RPS2), involved in formation of the translation initiation complex, where it might contact the messenger RNA and several components of the ribosome. It has been shown that in Escherichia coli RPS2 is essential for the binding of...; cd01425;rRNA interaction site [nucleotide binding];S8 interaction site;putative laminin-1 binding site
GOHSU45	CDS	GOHSU_45_00040	1979..2803	case	elongation factor Ts [hamap]	tsf [hamap]	MF_00050:elongation factor Ts:tsf 	RER_25430 :elongation factor Ts : tsf : : Rhodococcus erythropolis PR4	elongation factor Ts	tsf	F1YIC0_9ACTO Elongation factor Ts OS=Gordonia neofe lifaecis NRRL B-59395 GN=tsf PE=3 SV=1;elongation factor Ts [Corynebacterium glutamicum AT CC 13032];IS fingerprint type 4-5;elongation factor Ts; Provisional; PRK09377;EF-Ts; functions during elongation stage of protein translation; forms a dimer; associates with EF-Tu-GDP complex and promotes exchange of GDP to GTP resulting in regeneration of the active form of EF-Tu;Ubiquitin Associated domain. The UBA domain is a commonly occurring sequence motif in some members of the ubiquitination pathway, UV excision repair proteins, and certain protein kinases. Although its specific role is so far unknown, it has been...; cl00153;Elongation factor TS; pfam00889
GOHSU45	CDS	GOHSU_45_00050	2922..3656		uridylate kinase [nite]	 pyrH [nite]		RER_25440 :uridylate kinase : pyrH : 2.7.4.22 : Rhodococcus erythropolis PR4	uridylate kinase	pyrH	F1YIC1_9ACTO Uridylate kinase OS=Gordonia neofelifa ecis NRRL B-59395 GN=pyrH PE=3 SV=1;uridylate kinase [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;Catalyzes the phosphorylation of UMP to UDP;UMP kinase (UMPK)-Ec, the microbial/chloroplast uridine monophosphate kinase (uridylate kinase) enzyme that catalyzes UMP phosphorylation and plays a key role in pyrimidine nucleotide biosynthesis; regulation of this process is via feed-back control and...; cd04254;putative nucleotide binding site [chemical binding];uridine monophosphate binding site [chemical binding];homohexameric interface [polypeptide binding]
GOHSU45	CDS	GOHSU_45_00060	3723..4280		ribosome-recycling factor [hamap]	frr [hamap]	MF_00040:ribosome-recycling factor:frr 	ROP_66150 :ribosome recycling factor : frr : : Rhodococcus opacus B4	ribosome recycling factor	frr	F1YIC2_9ACTO Ribosome-recycling factor OS=Gordonia neofelifaecis NRRL B-59395 GN=frr PE=3 SV=1;ribosome recycling factor [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;ribosome recycling factor; Reviewed; PRK00083;Rrf; Frr; ribosome-recycling factor; release factor 4; RF4; recycles ribosomes upon translation termination along with release factor RF-3 and elongation factor EF-G; A GTPase-dependent process results in release of 50S from 70S; inhibited by release factor RF-1; essential for viability; structurally similar to tRNAs;Ribosome recycling factor (RRF). Ribosome recycling factor dissociates the posttermination complex, composed of the ribosome, deacylated tRNA, and mRNA, after termination of translation.  Thus ribosomes are 'recycled' and ready for another round of...; cd00520;hinge region
GOHSU45	CDS	GOHSU_45_00070	4277..5119		putative phosphatidate cytidylyltransferase [nite]	 cdsA [nite]		RER_25460 :putative phosphatidate cytidylyltransferase : cdsA : 2.7.7.41 : Rhodococcus erythropolis PR4	phosphatidate cytidylyltransferase	cdsA	H5U957_9ACTO Phosphatidate cytidylyltransferase OS= Gordonia terrae NBRC 100016 GN=cdsA PE=3 SV=1;cdsA gene product [Gordonia polyisoprenivorans VH2] ;Cytidylyltransferase family; cl00347
GOHSU45	CDS	GOHSU_45_00080	5119..5406		not assigned						
GOHSU45	CDS	GOHSU_45_00090	complement(5410..5835)		hypothetical membrane protein [nite]			ROP_66190 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		H5TLD2_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_100_00120 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU45	CDS	GOHSU_45_00100	5867..6973		ribosomal RNA large subunit methyltransferase N [hamap]	rlmN [hamap]	MF_01849:ribosomal RNA large subunit methyltransferase N:rlmN 	RER_25500 :putative rRNA methyltransferase : : 2.1.1.- : Rhodococcus erythropolis PR4	ribosomal RNA large subunit methyltransferase N		F1YIC5_9ACTO Ribosomal RNA large subunit methyltran sferase N OS=Gordonia neofelifaecis NRRL B-59395 GN=rlmN P E=3 SV=1;ribosomal RNA large subunit methyltransferase N [Co rynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;ribosomal RNA large subunit methyltransferase N; Provisional;23S rRNA m2A2503 methyltransferase; methylates the C2 position of the A2530 nucleotide in 23S rRNA; may be involved in antibiotic resistance;Radical SAM superfamily. Enzymes of this family generate radicals by combining a 4Fe-4S cluster and S-adenosylmethionine (SAM) in close proximity. They are characterized by a conserved CxxxCxxC motif, which coordinates the conserved iron-sulfur cluster; cd01335;FeS/SAM binding site
GOHSU45	CDS	GOHSU_45_00110	complement(7057..7323)		hypothetical protein [migap]				hypothetical protein		F1YIC6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08203 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF2631); pfam10939
GOHSU45	CDS	GOHSU_45_00120	7410..8597		1-deoxy-D-xylulose 5-phosphate reductoisomerase [hamap]	dxr [hamap]	MF_00183:1-deoxy-D-xylulose 5-phosphate reductoisomerase:dxr 	ROP_66220 :1-deoxy-D-xylulose 5-phosphate reductoisomerase : dxr : 1.1.1.267 : Rhodococcus opacus B4	1-deoxy-D-xylulose 5-phosphate reductoisomerase	dxr	F1YIC7_9ACTO 1-deoxy-D-xylulose 5-phosphate reducto isomerase OS=Gordonia neofelifaecis NRRL B-59395 GN=dxr PE =3 SV=1;1-deoxy-D-xylulose 5-phosphate reductoisomerase [Co rynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the NADP-dependent rearrangement and reduction of 1-deoxy-D-xylulose-5-phosphate (DXP) to 2-C-methyl-D-erythritol 4-phosphate;1-deoxy-D-xylulose 5-phosphate reductoisomerase; Provisional;1-deoxy-D-xylulose 5-phosphate reductoisomerase; pfam02670;1-deoxy-D-xylulose 5-phosphate reductoisomerase C-terminal; pfam08436;DXP reductoisomerase C-terminal domain; pfam13288
GOHSU45	CDS	GOHSU_45_00130	8614..9822		putative M50B family peptidase [nite]			ROP_66230 :putative M50B family peptidase : : 3.4.24.- : Rhodococcus opacus B4	peptidase M50		F1YIC8_9ACTO Peptidase M50 OS=Gordonia neofelifaeci s NRRL B-59395 GN=SCNU_08213 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: peptidase M50; SMART: PDZ/DHR/GLGF domain protein; KEGG: ank:AnaeK_1181 membrane-associated zinc metalloprotease;Predicted membrane-associated Zn-dependent proteases 1 [Cell envelope biogenesis, outer membrane];RseP-like Site-2 proteases (S2P), zinc metalloproteases (MEROPS family M50A), cleave transmembrane domains of substrate proteins, regulating intramembrane proteolysis (RIP) of diverse signal transduction mechanisms. In Escherichia coli, the S2P homolog...; cd06163;PDZ domain of bacterial and plant zinc metalloprotases, presumably membrane-associated or integral membrane proteases, which may be involved in signalling and regulatory mechanisms. May be responsible for substrate recognition and/or binding, as most PDZ...; cd00989;Site-2 protease (S2P) class of zinc metalloproteases (MEROPS family M50) cleaves transmembrane domains of substrate proteins, regulating intramembrane proteolysis (RIP) of diverse signal transduction mechanisms. Members of this family use proteolytic...; cl10020
GOHSU45	CDS	GOHSU_45_00140	9847..10998		4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase [hamap]	ispG [hamap]	MF_00159:4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase:ispG 	ROP_66240 :4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase : ispG : 1.17.4.3 : Rhodococcus opacus B4	4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase		F1YID0_9ACTO 4-hydroxy-3-methylbut-2-en-1-yl diphos phate synthase OS=Gordonia neofelifaecis NRRL B-59395 GN=i spG PE=3 SV=1;4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthas e [Streptomyces clavuligerus ATCC 27064];4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase; Validated;4-hydroxy-3-methylbut-2-en-1-yl diphosphate synthase; Reviewed; PRK00366
GOHSU45	CDS	GOHSU_45_00150	11024..11860		conserved hypothetical protein [nite]			RER_25550 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	N-acetyltransferase GCN5		F1YID1_9ACTO GCN5-like N-acetyltransferase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_08228 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: GCN5-related N-acetyltransferase; KEGG: mxa:MXAN_1267 hypothetical protein;Predicted acetyltransferase [General function prediction only];Domain of unknown function (DUF4081); pfam13312;N-Acyltransferase superfamily: Various enyzmes that characteristicly catalyze the transfer of an acyl group to a substrate; cl00357
GOHSU45	CDS	GOHSU_45_00160	11974..13734		putative penicillin-binding protein [nite]			RER_25560 :putative penicillin-binding protein : : : Rhodococcus erythropolis PR4	penicillin-binding protein		F1YID2_9ACTO Penicillin-binding protein transpeptid ase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08233 P E=4 SV=1;penicillin-binding protein [Rhodococcus erythropoli s PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;NTF2-like N-terminal transpeptidase domain; pfam05223;Cell division protein FtsI/penicillin-binding protein 2 [Cell envelope biogenesis, outer membrane]; COG0768;Penicillin binding protein transpeptidase domain; cl01009
GOHSU45	CDS	GOHSU_45_00170	complement(13754..14551)		putative hemin ABC transporter ATP-binding protein [nite]	 hmuV [nite]		KSE_73700 :putative hemin ABC transporter ATP-binding protein : hmuV : : Kitasatospora setae NBRC 14216	ABC transporter-like protein		F1YID4_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_08243 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ABC transporter; SMART: ATPase AAA; KEGG: csa:Csal_2702 ABC transporter;hemin importer ATP-binding subunit; Provisional; PRK13548;ABC transporters, involved in the uptake of siderophores, heme, and vitamin B12, are widely conserved in bacteria and archaea.  Only very few species lack representatives of the siderophore family transporters. The E. coli BtuCD protein is an ABC...; cd03214;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU45	CDS	GOHSU_45_00180	complement(14553..15608)		putative hemin ABC transporter permease protein [nite]	 hmuU [nite]		ROP_06210 :putative hemin ABC transporter permease protein : hmuU : : Rhodococcus opacus B4	transporter permease		F1YID5_9ACTO Transporter permease OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_08248 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: transporter permease; KEGG: sml:Smlt2356  FecCD-family transmembrane transporter;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;dimer interface [polypeptide binding];putative PBP binding regions;ABC-ATPase subunit  interface
GOHSU45	CDS	GOHSU_45_00190	complement(15605..16702)		putative hemin ABC transporter hemin-binding protein [nite]	 hmuT [nite]		RER_32050 :putative hemin ABC transporter hemin-binding protein : hmuT : : Rhodococcus erythropolis PR4	periplasmic binding protein		F1YID6_9ACTO Periplasmic-binding protein OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_08253 PE=4 SV=1;periplasmic binding protein [Pectobacterium carotov orum subsp. carotovorum PC1];PFAM: periplasmic binding protein; KEGG: ecf:ECH74115_4861 periplasmic-binding protein;Helical backbone metal receptor (TroA-like domain). These proteins have been shown to function in the ABC transport of ferric siderophores and metal ions such as Mn2+, Fe3+, Cu2+ and/or Zn2+.  Their ligand binding site is formed in the interface between...; cl00262;Periplasmic binding protein; pfam01497;intersubunit interface [polypeptide binding]
GOHSU45	CDS	GOHSU_45_00200	16701..17561		methionine aminopeptidase [nite]	 map [nite]		RER_25890 :methionine aminopeptidase : map : 3.4.11.18 : Rhodococcus erythropolis PR4	methionine aminopeptidase	map2	F1YID7_9ACTO Methionine aminopeptidase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08258 PE=3 SV=1;methionine aminopeptidase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;catalyzes the removal of N-terminal amino acids from peptides and arylamides; generally Co(II) however activity has been shown for some methionine aminopeptidases with Zn, Fe, or Mnin Bacillus subtilis the protein in this cluster is considered non-essential;Methionine Aminopeptidase 1. E.C. 3.4.11.18. Also known as methionyl aminopeptidase and Peptidase M. Catalyzes release of N-terminal amino acids, preferentially methionine, from peptides and arylamides; cd01086
GOHSU45	CDS	GOHSU_45_00210	17570..19111	case	cobyric acid synthase [hamap]	cobQ [hamap]	MF_00028:cobyric acid synthase:cobQ 	ROP_66390 :cobyric acid synthase : cobQ : 6.3.5.10 : Rhodococcus opacus B4	cobyric acid synthase CobQ		F1YID8_9ACTO Cobyric acid synthase OS=Gordonia neof elifaecis NRRL B-59395 GN=cobQ PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: cobyric acid synthase CobQ; PFAM: CobB/CobQ domain protein glutamine amidotransferase; cobyrinic acid ac-diamide synthase; KEGG: azo:azo3517 cobyric acid synthase;cobyric acid synthase; Provisional;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Type 1 glutamine amidotransferase (GATase1) domain found in Cobyric Acid Synthase (CobQ); cd01750;catalytic triad [active]
GOHSU45	CDS	GOHSU_45_00220	19093..19920		putative epoxide hydrolase [nite]			ROP_11220 :putative epoxide hydrolase : : 3.3.2.- : Rhodococcus opacus B4	alpha/beta hydrolase fold protein		F1YID9_9ACTO Alpha/beta hydrolase fold protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_08268 PE=4 SV=1 ;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: alpha/beta hydrolase fold; KEGG: rha:RHA1_ro01411 haloalkane dehalogenase;Alpha/beta hydrolase family; pfam12697
GOHSU45	CDS	GOHSU_45_00230	20035..21387		putative lipase [nite]			ROP_15770 :putative lipase : : 3.1.1.- : Rhodococcus opacus B4	lipase		G7H3E4_9ACTO Putative lipase OS=Gordonia araii NBRC 100433 GN=GOARA_056_01170 PE=4 SV=1;lipase [Rhodococcus opacus B4];Secretory lipase; pfam03583
GOHSU45	CDS	GOHSU_45_00240	complement(21401..22813)		mycothione reductase [nite]	 mtr [nite]		RER_26020 :mycothione reductase : mtr : 1.8.1.15 : Rhodococcus erythropolis PR4	mycothione reductase		F1YIE1_9ACTO Mycothione reductase OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_08278 PE=4 SV=1;mycothione reductase [Mycobacterium gilvum PYR- GCK ];catalyzes the reduction of mycothione or glutathione to mycothione or glutathione disulfide;mycothione reductase; Reviewed;Pyridine nucleotide-disulphide oxidoreductase; pfam00070;Pyridine nucleotide-disulphide oxidoreductase, dimerisation domain; pfam02852
GOHSU45	CDS	GOHSU_45_00250	complement(22810..23886)		hypothetical protein [nite]			ROP_66460 :hypothetical protein : : : Rhodococcus opacus B4	hydrolase alpha/beta fold family protein		F1YIE2_9ACTO Alpha/beta hydrolase fold family prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08283 PE =4 SV=1;hydrolase alpha/beta fold family protein [Arthrobac ter aurescens TC1];identified by match to protein family HMM PF00561;Lysophospholipase [Lipid metabolism]; COG2267;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU45	CDS	GOHSU_45_00260	24020..25546		probable malate:quinone oxidoreductase [hamap]	mqo [hamap]	MF_00212:probable malate:quinone oxidoreductase:mqo 	ROP_66470 :putative malate--quinone oxidoreductase : mqo : 1.1.99.16 : Rhodococcus opacus B4	malate:quinone oxidoreductase	mqo	F1YIE3_9ACTO Probable malate:quinone oxidoreductase OS=Gordonia neofelifaecis NRRL B-59395 GN=mqo PE=3 SV=1;malate:quinone oxidoreductase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;malate dehydrogenase; catalyzes the oxidation of malate to oxaloacetate;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Predicted dehydrogenase [General function prediction only]
GOHSU45	CDS	GOHSU_45_00270	25561..27450		putative magnesium chelatase subunit [nite]			RER_26060 :putative magnesium chelatase subunit : : : Rhodococcus erythropolis PR4	putative magnesium chelatase subunit		F1YIE4_9ACTO ATPase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08293 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];cobaltochelatase subunit; TIGR02442;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Von Willebrand factor type A (vWA) domain was originally found in the blood coagulation protein von Willebrand factor (vWF). Typically, the vWA domain is made up of approximately 200 amino acid residues folded into a classic a/b para-rossmann type of...; cl00057;metal ion-dependent adhesion site (MIDAS)
GOHSU45	CDS	GOHSU_45_00280	complement(27495..29759)		hypothetical protein [migap]				hypothetical protein		G5JJ19_9STAP Accumulation-associated protein (Fragm ent) OS=Staphylococcus simiae CCM 7213 GN=SS7213T_07338 PE =4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GH24424 gene product from transcript GH24424- RA;Domain of unknown function (DUF222); pfam02720
GOHSU45	CDS	GOHSU_45_00290	29984..30130	short	not assigned						
GOHSU45	CDS	GOHSU_45_00300	30135..30752		cob(I)alamin adenosyltransferase [nite]	 cobO [nite]		RER_26070 :cob(I)alamin adenosyltransferase : cobO : 2.5.1.17 : Rhodococcus erythropolis PR4	cob(I)alamin adenosyltransferase		F1YIE5_9ACTO Cob(I)alamin adenolsyltransferase/cobi namide ATP-dependent adenolsyltransferase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_08298 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;ATP:corrinoid adenosyltransferase [Coenzyme metabolism]; COG2109;TIGRFAM: cob(I)alamin adenosyltransferase; KEGG: rer:RER_26070 cob(I)alamin adenosyltransferase; PFAM: ATP:corrinoid adenosyltransferase BtuR/CobO/CobP;ATP:corrinoid adenosyltransferase BtuR/CobO/CobP. This family consists of the BtuR, CobO, CobP proteins all of which are Cob(I)alamin (vitamin B12) adenosyltransferase, which is involved in cobalamin (vitamin B12) biosynthesis. This enzyme is a homodimer; cd00561;homodimer interface [polypeptide binding];Walker A motif;ATP binding site [chemical binding];hydroxycobalamin binding site [chemical binding];Walker B motif
GOHSU45	CDS	GOHSU_45_00310	30746..32131	case	cobyrinic acid a,c-diamide synthase [hamap]	cobB [hamap]	MF_00027:cobyrinic acid a,c-diamide synthase:cobB 	ROP_66510 :cobyrinic acid a,c-diamide synthase : cobB : 6.3.5.9 : Rhodococcus opacus B4	cobyrinic acid a,c-diamide synthase		F1YIE6_9ACTO Cobyrinic acid a,c-diamide synthase OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08303 PE=3 SV =1;cobyrinic acid a,c-diamide synthase [Thermobispora bispora DSM 43833];type strain of Thermobispora bispora;TIGRFAM: cobyrinic acid a,c-diamide synthase; PFAM: Cobyrinic acid ac-diamide synthase; CobB/CobQ domain protein glutamine amidotransferase; KEGG: scl:sce0213 hydrogenobyrinic acid a,c-diamide synthase (glutamine-hydrolyzing);cobyrinic acid a,c-diamide synthase; Validated;Type 1 glutamine amidotransferase (GATase1) domain found in Cobyrinic Acid a,c-Diamide Synthase; cd03130;catalytic triad [active]
GOHSU45	CDS	GOHSU_45_00320	32134..33876		two-component histidine kinase [nite]			ROP_19680 :two-component histidine kinase : : 2.7.13.- : Rhodococcus opacus B4	sensor kinase, two-component system		D0LBH2_GORB4 GAF domain protein OS=Gordonia bronchi alis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 1066 7) GN=Gbro_2137 PE=4 SV=1;sensor kinase, two-component system [Rhodococcus jo stii RHA1];FOG: GAF domain [Signal transduction mechanisms]; COG2203;GAF domain; cl15785;GAF domain; pfam13185;Histidine kinase; pfam07730;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU45	CDS	GOHSU_45_00330	complement(33928..34431)		hypothetical protein [migap]				hypothetical protein		F6EI48_AMYSD Putative uncharacterized protein OS=Am ycolicicoccus subflavus (strain DSM 45089 / DQS3-9A1) GN=A S9A_2708 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: ami:Amir_1171 hypothetical protein
GOHSU45	CDS	GOHSU_45_00340	complement(34437..35111)		probable NarL family two-component response regulator [nite]			RER_44660 :probable NarL family two-component response regulator : : : Rhodococcus erythropolis PR4	response regulator, two-component system		F9VX41_9ACTO Putative NarL family two-component res ponse regulator OS=Gordonia alkanivorans NBRC 16433 GN=GOA LK_068_00380 PE=4 SV=1;response regulator, two-component system [Rhodococc us jostii RHA1];Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding]
GOHSU45	CDS	GOHSU_45_00350	35242..36192		acyl-[acyl-carrier-protein] desaturase DesA1 [nite]	 desA1 [nite]		ROP_49510 :acyl-[acyl-carrier-protein] desaturase DesA1 : desA1 : 1.14.19.2 : Rhodococcus opacus B4	fatty acid desaturase type 2		H5TZ45_9ACTO Acyl-[acyl-carrier-protein] desaturase DesA OS=Gordonia sputi NBRC 100414 GN=desA PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: fatty acid desaturase type 2; KEGG: Os08g0199400; hypothetical protein;Acyl ACP desaturase, ferritin-like diiron-binding domain; cd01050;homodimer interface [polypeptide binding];putative substrate binding pocket [chemical binding];diiron center [ion binding]
GOHSU45	CDS	GOHSU_45_00360	36218..37453		siroheme synthase CysG [nite]	 cysG [nite]		RER_26100 :siroheme synthase CysG : cysG : 1.3.1.76/2.1.1.107/4.99.1.4 : Rhodococcus erythropolis PR4	putative uroporphyrinogen-III C-methyltransferase		F1YIF0_9ACTO Uroporphyrin-III C-methyltransferase O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08323 PE=4 S V=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Precorrin-3B methylase [Coenzyme metabolism]; cl00304;Tetrapyrrole (Corrin/Porphyrin) Methylases; pfam00590
GOHSU45	CDS	GOHSU_45_00370	37453..37944		not assigned						F1YIF1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08328 PE=4 SV=1
GOHSU45	CDS	GOHSU_45_00380	complement(37983..38723)		hypothetical protein [nite]			ROP_66630 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YIF3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08338 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF328); cl01143;PFAM: protein of unknown function DUF328; KEGG: sun:SUN_0122 hypothetical protein
GOHSU45	CDS	GOHSU_45_00390	38749..40512		prolyl-tRNA synthetase [hamap]	proS [hamap]	MF_01569:prolyl-tRNA synthetase:proS 	RER_26150 :prolyl-tRNA synthetase : proS : 6.1.1.15 : Rhodococcus erythropolis PR4	prolyl-tRNA synthetase	proS	F1YIF4_9ACTO Proline--tRNA ligase OS=Gordonia neofe lifaecis NRRL B-59395 GN=proS PE=3 SV=1;prolyl-tRNA synthetase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the formation of prolyl-tRNA(Pro) from proline and tRNA(Pro);prolyl-tRNA synthetase; Provisional;Prolyl-tRNA synthetase (ProRS) class II core catalytic domain. ProRS is a homodimer. It is responsible for the attachment of proline to the 3' OH group of ribose of the appropriate tRNA. This domain is primarily responsible for ATP-dependent formation of...; cd00779;dimer interface [polypeptide binding];motif 1;motif 2;INS is an amino acid-editing domain inserted (INS) into the bacterial class II prolyl-tRNA synthetase (ProRS) however, this CD is not exclusively bacterial. It is also found at the N-terminus of the eukaryotic/archaea-like ProRS's of yeasts and...; cd04334;putative deacylase active site [active];Class II tRNA amino-acyl synthetase-like catalytic core domain. Class II amino acyl-tRNA synthetases (aaRS) share a common fold and generally attach an amino acid to the 3' OH of ribose of the appropriate tRNA.   PheRS is an exception in that it attaches...; cl00268;motif 3;ProRS Prolyl-anticodon binding domain, short version found predominantly in bacteria. ProRS belongs to class II aminoacyl-tRNA synthetases (aaRS). This alignment contains the anticodon binding domain, which is responsible for specificity in tRNA-binding; cd00861;anticodon binding site
GOHSU45	CDS	GOHSU_45_00400	complement(40509..40943)		hypothetical protein [nite]			RER_26270 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TKX2_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_095_00060 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Ferritin-like superfamily of diiron-containing four-helix-bundle proteins; cd00657;dinuclear metal binding motif [ion binding]
GOHSU45	CDS	GOHSU_45_00410	complement(40940..41419)		hypothetical protein [nite]			ROP_66790 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YIF7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08358 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.989) with cleavage site probability 0.518 at residue 28;PFAM: Twin-arginine translocation pathway, signal sequence
GOHSU45	CDS	GOHSU_45_00420	41502..42059		ribosome maturation factor RimP [hamap]	rimP [hamap]	MF_01077:ribosome maturation factor RimP:rimP 	RER_26290 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	ribosome maturation factor RimP	rimP	F1YIF8_9ACTO Ribosome maturation factor RimP OS=Gor donia neofelifaecis NRRL B-59395 GN=rimP PE=3 SV=1;rimP gene product [Gordonia polyisoprenivorans VH2] ;ribosome maturation protein RimP; Reviewed;The eukaryotic Sm and Sm-like (LSm) proteins associate with RNA to form the core domain of the ribonucleoprotein particles involved in a variety of RNA processing events including pre-mRNA splicing, telomere replication, and mRNA degradation.  Members of...; cl00259;Sm1 motif;D3 - B interaction site;D1 - D2 interaction site;Hfq - Hfq interaction site;RNA binding pocket [nucleotide binding];Sm2 motif
GOHSU45	CDS	GOHSU_45_00430	42145..43200		transcription elongation protein NusA [nite]	 nusA [nite]		ROP_66810 :transcription elongation protein NusA : nusA : : Rhodococcus opacus B4	transcription elongation factor NusA	nusA	F1YIF9_9ACTO Transcription elongation factor NusA O S=Gordonia neofelifaecis NRRL B-59395 GN=nusA PE=4 SV=1;transcription elongation factor NusA [Corynebacteri um glutamicum ATCC 13032];IS fingerprint type 4-5;transcription elongation factor NusA; Provisional; PRK12327;modifies transcription through interactions with RNA polymerase affecting elongation, readthrough, termination, and antitermination;NusA N-terminal domain; pfam08529;S1_NusA: N-utilizing substance A protein (NusA), S1-like RNA-binding domain. S1-like RNA-binding domains are found in a wide variety of RNA-associated proteins. NusA is a transcription elongation factor containing an N-terminal catalytic domain and three...; cd04455;RNA binding site [nucleotide binding];homodimer interface [polypeptide binding];K homology RNA-binding domain, type I.  KH binds single-stranded RNA or DNA. It is found in a wide variety of proteins including ribosomal proteins, transcription factors and post-transcriptional modifiers of mRNA. There are two different KH domains that...; cl00098;NusA_K homology RNA-binding domain (KH). NusA is an essential multifunctional transcription elongation factor that is universally conserved among prokaryotes and archaea. NusA anti-termination function plays an important role in the expression of...; cd02134;G-X-X-G motif
GOHSU45	CDS	GOHSU_45_00440	43285..43536		hypothetical protein [nite]			KSE_53060 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		G7H4X9_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_063_01030 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Ylxr homologs; group of conserved hypothetical bacterial proteins of unknown function; structure revealed putative RNA binding cleft; proteins are encoded by an operon that includes other proteins involved in transcription and/or translation; cl00189;putative RNA binding cleft [nucleotide binding]
GOHSU45	CDS	GOHSU_45_00450	43636..>43946	3x codon	translation initiation factor IF-2 [nite]	 infB [nite]		MLP_15740 :translation initiation factor IF-2 : infB : : Microlunatus phosphovorus NM-1	translation initiation factor IF-2		H5TKW7_9ACTO Translation initiation factor IF-2 (Fr agment) OS=Gordonia otitidis NBRC 100426 GN=infB PE=4 SV=1 ;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: translation initiation factor IF-2; small GTP-binding protein; PFAM: protein synthesis factor GTP-binding; translation initiation factor IF-2 domain protein; GTP- binding protein HSR1-related; elongation factor Tu domain 2 protein; KEGG: gbm:Gbem_1302 translation initiation factor IF-2;Translation initiation factor IF-2, N-terminal region; pfam04760
##GOHSU46	contig00271				38136
GOHSU46	CDS	GOHSU_46_00010	complement(<1..394)	3x	not assigned						G7H5U6_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_065_00140 PE=4 SV=1
GOHSU46	CDS	GOHSU_46_00020	636..1394		not assigned						
GOHSU46	CDS	GOHSU_46_00030	1438..2274		putative 3-mercaptopyruvate sulfurtransferase [nite]			RER_39860 :putative 3-mercaptopyruvate sulfurtransferase : : 2.8.1.2 : Rhodococcus erythropolis PR4	rhodanese domain-containing protein		H0R8L3_9ACTO Sulfurtransferase OS=Gordonia polyisop renivorans NBRC 16320 GN=GOPIP_007_00450 PE=4 SV=1;rhodanese domain-containing protein [Actinosynnema mirum DSM 43827];Rhodanese-related sulfurtransferase [Inorganic ion transport and metabolism]; COG2897;PFAM: rhodanese domain protein; SMART: rhodanese domain protein; KEGG: scl:sce6912 thiosulfate sulfurtransferase;Thiosulfate sulfurtransferase (TST), N-terminal, inactive domain. TST contains 2 copies of the Rhodanese Homology Domain; this is the 1st repeat, which does not contain the catalytically active Cys residue. The role of the 1st repeat is uncertain, but it...; cd01448;active site residue [active];Thiosulfate sulfurtransferase (TST), C-terminal, catalytic domain. TST contains 2 copies of the Rhodanese Homology Domain; this is the second repeat. Only the second repeat contains the catalytically active Cys residue; cd01449
GOHSU46	CDS	GOHSU_46_00040	complement(2352..3023)		hypothetical membrane protein [nite]			RER_47270 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	integral membrane protein		G7CIK8_MYCTH Putative uncharacterized protein OS=My cobacterium thermoresistibile ATCC 19527 GN=KEK_10598 PE=4 SV=1;integral membrane protein [Saccharopolyspora erythr aea NRRL 2338];Protein of unknown function (DUF1211); cl01421
GOHSU46	CDS	GOHSU_46_00050	complement(3028..3948)		putative F420-dependent oxidoreductase [nite]			YM304_13900 :putative F420-dependent oxidoreductase : : : Acidimicrobidae bacterium YM16-304	luciferase family protein		F1YNM9_9ACTO Luciferase family protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_17642 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];N5,N10-methylenetetrahydromethanopterin reductase (Mer) catalyzes the reduction of N5,N10-methylenetetrahydromethanopterin with reduced coenzyme F420 to N5-methyltetrahydromethanopterin and oxidized coenzyme F420; cd01097;PFAM: luciferase family protein; KEGG: fal:FRAAL4421 putative FMN reductase;F420-dependent oxidoreductase, MSMEG_4879 family; TIGR03564
GOHSU46	CDS	GOHSU_46_00060	complement(4019..4675)		hypothetical protein [nite]			KRH_03320 :hypothetical protein : : : Kocuria rhizophila DC2201	nucleoside-diphosphate-sugar epimerase		D5UQL4_TSUPD NAD-dependent epimerase/dehydratase OS =Tsukamurella paurometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_0193 PE=4 SV=1;nucleoside-diphosphate-sugar epimerase [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NADH(P)-binding; pfam13460;NAD(P) binding site [chemical binding]
GOHSU46	CDS	GOHSU_46_00070	complement(4694..5107)		not assigned						G7H6D3_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_068_00670 PE=4 SV=1
GOHSU46	CDS	GOHSU_46_00080	5244..5453		putative copper chaperone CopZ [nite]	 copZ [nite]		MLP_43150 :putative copper chaperone CopZ : copZ : : Microlunatus phosphovorus NM-1	copper chaperone		D4YNB8_9MICO Heavy metal-associated domain protein OS=Brevibacterium mcbrellneri ATCC 49030 GN=HMPREF0183_142 8 PE=4 SV=1;copper chaperone [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;Heavy-metal-associated domain (HMA) is a conserved domain of approximately 30 amino acid residues found in a number of proteins that transport or detoxify heavy metals, for example, the CPx-type heavy metal ATPases and copper chaperones. HMA domain...; cl00207
GOHSU46	CDS	GOHSU_46_00090	5453..6352		hypothetical protein [nite]			MLP_43160 :hypothetical protein : : : Microlunatus phosphovorus NM-1	secreted protein		A4ALX9_9ACTN Putative uncharacterized protein OS=ma rine actinobacterium PHSC20C1 GN=A20C1_00015 PE=4 SV=1;secreted protein [Micromonospora sp. ATCC 39149];Domain of Unknown Function (DUF748); pfam05359
GOHSU46	CDS	GOHSU_46_00100	6368..8647		putative copper-transporting ATPase CopA [nite]	 copA [nite]		MLP_43170 :putative copper-transporting ATPase CopA : copA : 3.6.3.4 : Microlunatus phosphovorus NM-1	heavy metal translocating P-type ATPase		A4T1Y3_MYCGI Heavy metal translocating P-type ATPas e OS=Mycobacterium gilvum (strain PYR-GCK) GN=Mflv_2849 PE =3 SV=1;heavy metal translocating P-type ATPase [Mycobacter ium gilvum PYR- GCK];TIGRFAM: ATPase, P-type (transporting), HAD superfamily, subfamily IC; copper-translocating P-type ATPase; heavy metal translocating P-type ATPase; PFAM: Haloacid dehalogenase domain protein hydrolase; Heavy metal transport/detoxification protein; E1-E2 ATPase-associated domain protein; KEGG: mmc:Mmcs_1452 heavy metal translocating P-type ATPase;Cation transport ATPase [Inorganic ion transport and metabolism]; COG2217;Heavy-metal-associated domain (HMA) is a conserved domain of approximately 30 amino acid residues found in a number of proteins that transport or detoxify heavy metals, for example, the CPx-type heavy metal ATPases and copper chaperones. HMA domain...; cd00371;metal-binding site [ion binding];E1-E2 ATPase; pfam00122;Haloacid Dehalogenase-like Hydrolases; cl11391
GOHSU46	CDS	GOHSU_46_00110	8664..9017		hypothetical protein [nite]			ROP_pROB01-02370 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		E3B9B9_9MICO Putative copper-sensing transcriptiona l repressor CsoR OS=Dermacoccus sp. Ellin185 GN=HMPREF0321 _0888 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF156; KEGG: bbt:BBta_p0120 hypothetical protein;Transcriptional regulators CsoR (copper-sensitive operon repressor), RcnR, and FrmR, and related domains; this domain superfamily was previously known as DUF156; cd10148;putative homotetramer interface [polypeptide binding];putative homodimer interface [polypeptide binding];putative allosteric switch controlling residues;putative metal binding site [ion binding];putative homodimer-homodimer interface [polypeptide binding]
GOHSU46	CDS	GOHSU_46_00120	complement(9074..9733)		putative TetR family transcriptional regulator [nite]			RER_05640 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		G2NDC6_9ACTO Transcriptional regulator, TetR family OS=Streptomyces sp. SirexAA-E GN=SACTE_0415 PE=4 SV=1;TetR family transcriptional regulator [Streptomyces violaceusniger Tu 4113];PFAM: Transcriptional regulator, TetR-like, DNA-binding, bacterial/archaeal; KEGG: sro:Sros_7072  TetR family transcriptional regulator;Helix-turn-helix domains; cl00088
GOHSU46	CDS	GOHSU_46_00130	9828..11984		hypothetical protein [migap]				hypothetical protein		E2PXA1_STRCL Putative membrane protein OS=Streptomy ces clavuligerus ATCC 27064 GN=SCLAV_0946 PE=4 SV=1;unnamed protein product [Streptomyces cattleya NRRL 8057];Evidence 5 : No homology to any previously reported sequences;ABC-2 type transporter; cl11417
GOHSU46	CDS	GOHSU_46_00140	complement(11992..12651)		enoyl-CoA hydratase/isomerase family protein [nite]			ROP_37610 :enoyl-CoA hydratase/isomerase family protein : : : Rhodococcus opacus B4	enoyl-CoA hydratase		I0WXH2_9NOCA Enoyl-CoA hydratase OS=Rhodococcus imt echensis RKJ300 GN=W59_04371 PE=4 SV=1;enoyl-CoA hydratase [Rhodococcus jostii RHA1];Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU46	CDS	GOHSU_46_00150	complement(12651..13337)		putative TetR family transcriptional regulator [nite]			ROP_37600 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		F9VS00_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_029_ 00440 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m avium 104];identified by match to protein family HMM PF00440;Helix-turn-helix domains; cl00088
GOHSU46	CDS	GOHSU_46_00160	complement(13423..13830)		conserved hypothetical protein [nite]			RER_01160 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H6N0J9_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c10410 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU46	CDS	GOHSU_46_00170	complement(14321..15256)		putative enoyl-CoA hydratase [nite]			YM304_28300 :putative enoyl-CoA hydratase : : 4.2.1.17 : Acidimicrobidae bacterium YM16-304	enoyl-CoA hydratase		Q0SD93_RHOSR Possible enoyl-CoA hydratase, C-termin al OS=Rhodococcus sp. (strain RHA1) GN=RHA1_ro02688 PE=4 S V=1;enoyl-CoA hydratase [Rhodococcus jostii RHA1];enoyl-CoA hydratase; Provisional;Catalyzes the reversible hydration of unsaturated fatty acyl-CoA to beta-hydroxyacyl-CoA;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU46	CDS	GOHSU_46_00180	15548..15961		conserved hypothetical protein [nite]			RER_48430 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	NADH:flavin oxidoreductase/NADH oxidase		Q08WG8_STIAD NADH:flavin oxidoreductase/nadh oxidas e OS=Stigmatella aurantiaca (strain DW4/3-1) GN=STAUR_2009 PE=4 SV=1;NADH:flavin oxidoreductase/NADH oxidase [Streptomyc es viridochromogenes DSM 40736]
GOHSU46	CDS	GOHSU_46_00190	16315..18105		potassium-transporting ATPase A chain [hamap]	kdpA [hamap]	MF_00275:potassium-transporting ATPase A chain:kdpA 	RER_37260 :potassium-transporting ATPase subunit A : kdpA : 3.6.3.12 : Rhodococcus erythropolis PR4	potassium-transporting ATPase subunit A		D0L4N2_GORB4 Potassium-transporting ATPase A chain (Precursor) OS=Gordonia bronchialis (strain ATCC 25592 / D SM 43247 / JCM 3198 / NCTC 10667) GN=kdpA PE=3 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG2060 K+-transporting ATPase, A chain;Potassium-transporting ATPase A subunit; cl00903
GOHSU46	CDS	GOHSU_46_00200	18225..20192		potassium-transporting ATPase B chain [hamap]	kdpB [hamap]	MF_00285:potassium-transporting ATPase B chain:kdpB 	ROP_09540 :potassium-transporting ATPase subunit B : kdpB : 3.6.3.12 : Rhodococcus opacus B4	potassium-transporting ATPase subunit B		G7H1N1_9ACTO Potassium-transporting ATPase B chain OS=Gordonia araii NBRC 100433 GN=kdpB PE=3 SV=1;potassium-transporting ATPase subunit B [Streptomyc es sp. Mg1];potassium-transporting ATPase subunit B; Provisional;E1-E2 ATPase; pfam00122;Haloacid Dehalogenase-like Hydrolases; cl11391
GOHSU46	CDS	GOHSU_46_00210	20199..20849		potassium-transporting ATPase subunit C [nite]	 kdpC [nite]		ROP_09530 :potassium-transporting ATPase subunit C : kdpC : 3.6.3.12 : Rhodococcus opacus B4	potassium-transporting ATPase subunit KdpC	kdpC	H6MWK4_GORPV Potassium-transporting ATPase C chain KdpC OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH 2) GN=kdpC PE=3 SV=1;kdpC gene product [Gordonia polyisoprenivorans VH2] ;K+-transporting ATPase, c chain; cl00944
GOHSU46	CDS	GOHSU_46_00220	20859..23414		turgor pressure sensor kinase KdpD [nite]	 kdpD [nite]		ROP_09520 :turgor pressure sensor kinase KdpD : kdpD : 2.7.13.- : Rhodococcus opacus B4	turgor pressure sensor		G7H1N3_9ACTO Turgor pressure sensor kinase KdpD OS= Gordonia araii NBRC 100433 GN=kdpD PE=4 SV=1;turgor pressure sensor [Streptomyces sp. Mg1];Osmosensitive K+ channel His kinase sensor domain; pfam02702;Osmosensitive K+ channel histidine kinase [Signal transduction mechanisms]; COG2205;USP domain is located between the N-terminal sensor domain and C-terminal catalytic domain of this Osmosensitive K+ channel histidine kinase family. The family of KdpD sensor kinase proteins regulates the kdpFABC operon responsible for potassium...; cd01987;Ligand Binding Site [chemical binding];Histidine Kinase A (dimerization/phosphoacceptor) domain; Histidine Kinase A dimers are formed through parallel association of 2 domains creating 4-helix bundles; usually these domains contain a conserved His residue and are activated via...; cd00082;dimer interface [polypeptide binding];phosphorylation site [posttranslational modification];Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU46	CDS	GOHSU_46_00230	23362..24087		turgor pressure transcriptional regulator [nite]	 kdpE [nite]		RER_37220 :turgor pressure transcriptional regulator : kdpE : : Rhodococcus erythropolis PR4	turgor pressure regulator		F1YLT4_9ACTO Response regulator receiver OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_14389 PE=4 SV=1;turgor pressure regulator [Streptomyces sp. Mg1];DNA-binding transcriptional activator KdpE; Provisional;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];Effector domain of response regulator. Bacteria and certain eukaryotes like protozoa and higher plants use two-component signal transduction systems to detect and respond to changes in the environment. The system consists of a sensor histidine kinase and...; cd00383;DNA binding site [nucleotide binding]
GOHSU46	CDS	GOHSU_46_00240	complement(24088..25797)		dihydroxy-acid dehydratase [hamap]	ilvD [hamap]	MF_00012:dihydroxy-acid dehydratase:ilvD 	RER_50540 :probable dihydroxy-acid dehydratase : ilvD : 4.2.1.9 : Rhodococcus erythropolis PR4	dihydroxy-acid dehydratase		H6N3P3_GORPV Dihydroxy-acid dehydratase 1 OS=Gordon ia polyisoprenivorans (strain DSM 44266 / VH2) GN=ilvD1 PE =3 SV=1;dihydroxy-acid dehydratase [Saccharomonospora cyane a NA- 134];PFAM: Dehydratase family; TIGRFAM: dihydroxy-acid dehydratase;Dehydratase family; cl00340
GOHSU46	CDS	GOHSU_46_00250	26117..26548		tautomerase enzyme superfamily protein [migap]				tautomerase enzyme superfamily protein		I0RDR3_MYCPH Uncharacterized protein OS=Mycobacteri um phlei RIVM601174 GN=MPHLEI_24991 PE=4 SV=1;tautomerase enzyme superfamily protein [Streptomyce s himastatinicus ATCC 53653];truncated CDS;4-Oxalocrotonate Tautomerase:  Catalyzes the isomerization of unsaturated ketones. The structure is a homohexamer that is arranged as a trimer of dimers. The hexamer contains six active sites, each formed by residues from three monomers, two from one...; cl00235
GOHSU46	CDS	GOHSU_46_00260	26894..27466		hypothetical protein [nite]			YM304_17150 :hypothetical protein : : : Acidimicrobidae bacterium YM16-304	alkylhydroperoxidase		B8FLK9_DESAA Carboxymuconolactone decarboxylase OS= Desulfatibacillum alkenivorans (strain AK-01) GN=Dalk_3675 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];TIGRFAM: alkylhydroperoxidase like protein, AhpD family; PFAM: Carboxymuconolactone decarboxylase; KEGG: sma:SAV429 hypothetical protein;Uncharacterized conserved protein [Function unknown];Carboxymuconolactone decarboxylase family; cl00460
GOHSU46	CDS	GOHSU_46_00270	complement(27519..28379)		putative enoyl-CoA hydratase [nite]			RER_46880 :putative enoyl-CoA hydratase : : 4.2.1.17 : Rhodococcus erythropolis PR4	enoyl-CoA hydratase/carnithine racemase		D3F865_CONWI Enoyl-CoA hydratase/isomerase OS=Conex ibacter woesei (strain DSM 14684 / JCM 11494 / NBRC 100937 / ID131577) GN=Cwoe_0500 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Enoyl-CoA hydratase/isomerase family;enoyl-CoA hydratase; Provisional;Crotonase/Enoyl-Coenzyme A (CoA) hydratase superfamily. This superfamily contains a diverse set of enzymes including enoyl-CoA hydratase, napthoate synthase, methylmalonyl-CoA decarboxylase, 3-hydoxybutyryl-CoA dehydratase, and dienoyl-CoA isomerase; cd06558;substrate binding site [chemical binding];oxyanion hole (OAH) forming residues;trimer interface [polypeptide binding]
GOHSU46	CDS	GOHSU_46_00280	complement(28553..29182)		TetR family transcriptional regulator [migap]				TetR family transcriptional regulator		D3F866_CONWI Transcriptional regulator, TetR family OS=Conexibacter woesei (strain DSM 14684 / JCM 11494 / NB RC 100937 / ID131577) GN=Cwoe_0501 PE=4 SV=1;TetR family transcriptional regulator [Conexibacter woesei DSM 14684];Transcriptional regulator [Transcription]; COG1309;PFAM: TetR family transcriptional regulator; KEGG: pat:Patl_3771 TetR family transcriptional regulator;Helix-turn-helix domains; cl00088
GOHSU46	CDS	GOHSU_46_00290	complement(29310..29864)		putative TetR family transcriptional regulator [nite]			RER_08710 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	transcriptional regulatory protein		C3JUY3_RHOER Transcriptional regulatory protein OS= Rhodococcus erythropolis SK121 GN=RHOER0001_0787 PE=4 SV=1 ;transcriptional regulatory protein [Mycobacterium m arinum M];possibly involved in transcriptional mechanism;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU46	CDS	GOHSU_46_00300	complement(29975..30739)		putative oxidoreductase [nite]			RER_52570 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	oxidoreductase		Q0SD44_RHOSR Probable 3-oxoacyl-[acyl-carrier-prote in] reductase OS=Rhodococcus sp. (strain RHA1) GN=RHA1_ro0 2737 PE=3 SV=1;oxidoreductase [Rhodococcus opacus B4];3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05557;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU46	CDS	GOHSU_46_00310	complement(30767..31780)		alcohol dehydrogenase [nite]			MLP_02790 :alcohol dehydrogenase : : 1.1.1.- : Microlunatus phosphovorus NM-1	alcohol dehydrogenase		C9YX11_STRSW Putative zinc-containing dehydrogenase OS=Streptomyces scabies (strain 87.22) GN=SCAB_83131 PE=4 SV=1;alcohol dehydrogenase [Leptospira interrogans serov ar Copenhageni str. Fiocruz L1- 130];Threonine dehydrogenase and related Zn-dependent dehydrogenases [Amino acid transport and metabolism / General function prediction only]; COG1063;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU46	CDS	GOHSU_46_00320	complement(31919..33352)		putative dioxygenase [nite]			ROP_55350 :putative dioxygenase : : : Rhodococcus opacus B4	lignostilbene-alpha,beta-dioxygenase		F9VWI1_9ACTO Putative dioxygenase OS=Gordonia alkan ivorans NBRC 16433 GN=GOALK_061_00260 PE=4 SV=1;unnamed protein product [Acaryochloris marina MBIC1 1017];type strain of Acaryochloris marina;Lignostilbene-alpha,beta-dioxygenase and related enzymes [Secondary metabolites biosynthesis, transport, and catabolism];Retinal pigment epithelial membrane protein; cl10080
GOHSU46	CDS	GOHSU_46_00330	33488..33973		TetR family transcriptional regulator [migap]				TetR family transcriptional regulator		F9VWI0_9ACTO Putative TetR family transcriptional r egulator OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_061_ 00250 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU46	CDS	GOHSU_46_00340	34685..35482		not assigned						
GOHSU46	CDS	GOHSU_46_00350	36005..37009		putative transposase [nite]			KSE_41190 :putative transposase : : : Kitasatospora setae NBRC 14216	insertion element ISCmi2 transposase		G7H780_9ACTO Putative transposase (Fragment) OS=Gor donia araii NBRC 100433 GN=GOARA_083_00010 PE=4 SV=1;insertion element ISCmi2 transposase [Clavibacter m ichiganensis subsp. sepedonicus];N/R/C;Helix-turn-helix domains; cl00088;Winged helix-turn helix; pfam13551;Integrase core domain; cl01316;Integrase core domain; cl15866
GOHSU46	CDS	GOHSU_46_00360	37274..37417	short	not assigned						
##GOHSU47	contig00281				32927
GOHSU47	CDS	GOHSU_47_00010	66..1328		hypothetical protein [nite]			RER_12470 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TQA5_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_176_00050 PE=4 SV=1;unnamed protein product [Propionibacterium acnes Ty peIA2 P.acn33]
GOHSU47	CDS	GOHSU_47_00020	1363..2616		putative MFS transporter [nite]			ROP_50640 :putative MFS transporter : : : Rhodococcus opacus B4	major facilitator superfamily protein		D5UPX5_TSUPD Major facilitator superfamily MFS_1 OS =Tsukamurella paurometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBRC 16120 / NCTC 13040) GN=Tpau_0089 PE=4 SV=1;major facilitator superfamily protein [Streptomyces violaceusniger Tu 4113];Signal predicted by SignalP 3.0 HMM (Signal peptide probabilty 0.769) with cleavage site probability 0.745 at residue 23;PFAM: Major facilitator superfamily MFS-1; KEGG: saq:Sare_1712 major facilitator transporter;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU47	CDS	GOHSU_47_00030	2597..3823		cystathionine beta-lyase [nite]	 metC [nite]		MLP_04900 :cystathionine beta-lyase : metC : 4.4.1.8 : Microlunatus phosphovorus NM-1	aminotransferase		G7CIA6_MYCTH Putative uncharacterized protein OS=My cobacterium thermoresistibile ATCC 19527 GN=KEK_10493 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG1168 Bifunctional PLP-dependent enzyme with beta-cystathionase and maltose regulon repressor activities;Bifunctional PLP-dependent enzyme with beta-cystathionase and maltose regulon repressor activities [Amino acid transport and metabolism]; COG1168;Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU47	CDS	GOHSU_47_00040	complement(3979..4755)		3-hydroxyacyl-CoA dehydrogenase [nite]			YM304_29750 :3-hydroxyacyl-CoA dehydrogenase : : 1.1.1.35 : Acidimicrobidae bacterium YM16-304	short-chain dehydrogenase/reductase SDR		F1YPU6_9ACTO 3-hydroxyacyl-CoA dehydrogenase OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_19802 PE=3 SV=1;short-chain dehydrogenase/reductase SDR [Rhodoferax ferrireducens T118];type strain of Rhodoferax ferrireducens T118;3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05557;17hydroxysteroid dehydrogenase type 10 (HSD10)-like, classical (c) SDRs; cd05371;NAD binding site [chemical binding];homotetramer interface [polypeptide binding];homodimer interface [polypeptide binding]
GOHSU47	CDS	GOHSU_47_00050	complement(4824..6446)		hypothetical protein [nite]			ROP_52510 :hypothetical protein : : : Rhodococcus opacus B4	VanW-like protein		F1YPU5_9ACTO VanW family protein OS=Gordonia neofel ifaecis NRRL B-59395 GN=SCNU_19797 PE=4 SV=1;VanW-like protein [Amycolatopsis mediterranei U32];Putative peptidoglycan binding domain; pfam12229;Uncharacterized vancomycin resistance protein [Defense mechanisms];VanW like protein; pfam04294
GOHSU47	CDS	GOHSU_47_00060	complement(6767..8074)		acetyl-CoA acyltransferase [nite]	 fadA [nite]		ROP_52500 :acetyl-CoA acyltransferase : fadA : 2.3.1.16 : Rhodococcus opacus B4	acetyl-CoA acetyltransferase		F1YPU4_9ACTO Acetyl-CoA acetyltransferase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_19792 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: abb:ABBFA_001414 acetyl-CoA acetyltransferase; TIGRFAM: acetyl-CoA acetyltransferase; PFAM: thiolase;acetyl-CoA acetyltransferase; Provisional;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU47	CDS	GOHSU_47_00070	8235..9596		putative 3-oxoacyl-[acyl-carrier-protein] reductase [nite]			ROP_52490 :putative 3-oxoacyl-[acyl-carrier-protein] reductase : : 1.1.1.100 : Rhodococcus opacus B4	short-chain dehydrogenase/reductase family oxidoreductase		F1YPU3_9ACTO 3-ketoacyl-(Acyl-carrier-protein) redu ctase OS=Gordonia neofelifaecis NRRL B-59395 GN=fabG PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];3-ketoacyl-(acyl-carrier-protein) reductase; Provisional; PRK08261;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU47	CDS	GOHSU_47_00080	9593..10483		conserved hypothetical protein [nite]			RER_11190 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	MaoC domain-containing protein dehydratase		F1YPU2_9ACTO MaoC domain-containing protein dehydra tase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_19782 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: MaoC domain protein dehydratase; KEGG: prw:PsycPRwf_1388 dehydratase;The hotdog fold was initially identified in the E. coli FabA (beta-hydroxydecanoyl-acyl carrier protein (ACP)-dehydratase) structure and subsequently in 4HBT (4-hydroxybenzoyl-CoA thioesterase) from Pseudomonas. A number of other seemingly unrelated...; cl00509;active site 2 [active];active site 1 [active]
GOHSU47	CDS	GOHSU_47_00090	complement(10578..11198)		putative TetR family transcriptional regulator [nite]			ROP_52470 :putative TetR family transcriptional regulator : : : Rhodococcus opacus B4	TetR family transcriptional regulator		F1YPX1_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_19912 PE=4 SV=1;TetR family transcriptional regulator [Amycolatopsi s mediterranei U32];Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU47	CDS	GOHSU_47_00100	complement(11242..12267)		putative glycosidase [nite]			ROP_52460 :putative glycosidase : : 3.2.1.- : Rhodococcus opacus B4	beta-glucosidase-like glycosyl hydrolase		F1YPX0_9ACTO Glycoside hydrolase family 3 domain-co ntaining protein OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_19907 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Glycosyl hydrolase family 3 N terminal domain;Glycosyl hydrolase family 3 N terminal domain; cl07971
GOHSU47	CDS	GOHSU_47_00110	12233..12496		not assigned						
GOHSU47	CDS	GOHSU_47_00120	12478..12678		hypothetical protein [migap]				hypothetical protein		F1YPW9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19902 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU47	CDS	GOHSU_47_00130	12860..16990		conserved hypothetical membrane protein [nite]			RER_11130 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	transmembrane protein		F1YPW8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19897 PE=4 SV=1;transmembrane protein, partial [Mycobacterium tuber culosis T92];Domain of unknown function (DUF3367); pfam11847
GOHSU47	CDS	GOHSU_47_00140	17014..17421	overlap	hypothetical protein [migap]				hypothetical protein		H0R4L4_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_105_00260 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU47	CDS	GOHSU_47_00150	complement(17353..18642)	overlap	conserved hypothetical membrane protein [nite]			RER_11120 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	acyltransferase 3		F1YPW7_9ACTO Acyltransferase 3 OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_19892 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;PFAM: acyltransferase 3; KEGG: rop:ROP_52410  acyltransferase;OpgC protein; cl00792
GOHSU47	CDS	GOHSU_47_00160	18584..19741		hypothetical membrane protein [nite]			ROP_52400 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YPW6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19887 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3068); pfam11271
GOHSU47	CDS	GOHSU_47_00170	complement(19757..20974)		hypothetical membrane protein [nite]			RER_11090 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GWK1_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_76_00460 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU47	CDS	GOHSU_47_00180	21103..22023		putative citrate lyase beta chain [nite]			KRH_05000 :putative citrate lyase beta chain : : : Kocuria rhizophila DC2201	citrate lyase beta chain		G7H6W4_9ACTO Putative citrate lyase beta chain OS=G ordonia araii NBRC 100433 GN=GOARA_078_00380 PE=4 SV=1;citrate lyase beta chain [Rhodococcus opacus B4];Pyruvate kinase (PK):  Large allosteric enzyme that regulates glycolysis through binding of the substrate, phosphoenolpyruvate, and one or more allosteric effectors. Like other allosteric enzymes, PK has a high substrate affinity R state and a low...; cl09155
GOHSU47	CDS	GOHSU_47_00190	22035..23774		conserved hypothetical membrane protein [nite]			RER_11080 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	putative transmembrane protein		F1YPW1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19862 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU47	CDS	GOHSU_47_00200	complement(23717..24847)		putative glycosyltransferase [nite]			ROP_52370 :putative glycosyltransferase : : 2.4.-.- : Rhodococcus opacus B4	glycosyltransferase		F1YPK2_9ACTO Glycosyl transferase group 1 protein O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_19285 PE=4 S V=1;glycosyltransferase [Corynebacterium glutamicum ATC C 13032];IS fingerprint type 4-5;Glycosyltransferase [Cell envelope biogenesis, outer membrane]; COG0438;This family is most closely related to the GT1 family of glycosyltransferases and named after YqgM in Bacillus licheniformis about which little is known. Glycosyltransferases catalyze the transfer of sugar moieties from activated donor molecules to...; cd03801
GOHSU47	CDS	GOHSU_47_00210	24878..25660		hypothetical protein [nite]			ROP_52360 :hypothetical protein : : : Rhodococcus opacus B4	SAM-dependent methyltransferase		F1YPK3_9ACTO Type 11 methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_19290 PE=4 SV=1;SAM-dependent methyltransferase [Corynebacterium gl utamicum ATCC 13032];IS fingerprint type 4-5;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU47	CDS	GOHSU_47_00220	complement(25690..26391)		hypothetical protein [migap]				hypothetical protein		H0QW14_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_018_00470 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: CG34220 gene product from transcript CG34220- RA
GOHSU47	CDS	GOHSU_47_00230	complement(26441..29011)		putative ABC transporter permease protein [nite]			RER_40880 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	ABC transporter		F1YPK5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_19300 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];MacB-like periplasmic core domain; pfam12704;Acidobacterial duplicated orphan permease; TIGR03434;FtsX-like permease family; cl15850;ABC-type transport system, involved in lipoprotein release, permease component [Cell envelope biogenesis, outer membrane]; COG4591
GOHSU47	CDS	GOHSU_47_00240	complement(29014..29772)		putative ABC transporter ATP-binding protein [nite]			RER_40890 :putative ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ABC transporter ATP-binding protein		F1YPK6_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_19305 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ];ABC-type antimicrobial peptide transport system, ATPase component [Defense mechanisms]; COG1136;This family is comprised of MJ0796 ATP-binding cassette, macrolide-specific ABC-type efflux carrier (MacAB), and proteins involved in cell division (FtsE), and release of liporoteins from the cytoplasmic membrane (LolCDE).  They are clustered together...; cd03255;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU47	CDS	GOHSU_47_00250	complement(29943..31769)		GTP-dependent phosphoenolpyruvate carboxykinase [hamap]	pckG [hamap]	MF_00452:GTP-dependent phosphoenolpyruvate carboxykinase:pckG 	RER_10900 :GTP-dependent phosphoenolpyruvate carboxykinase : pckG : 4.1.1.32 : Rhodococcus erythropolis PR4	phosphoenolpyruvate carboxykinase		F1YKD4_9ACTO Phosphoenolpyruvate carboxykinase [GTP ] OS=Gordonia neofelifaecis NRRL B-59395 GN=pckG PE=3 SV=1 ;phosphoenolpyruvate carboxykinase [Mycobacterium av ium 104];catalyzes the phosphorylation and decarboxylation of oxaloacetate to form phosphoenolpyruvate using GTP;Phosphoenolpyruvate carboxykinase (PEPCK), a critical gluconeogenic enzyme, catalyzes the first committed step in the diversion of tricarboxylic acid cycle intermediates toward gluconeogenesis. It catalyzes the reversible decarboxylation and...; cd00819;Phosphoenolpyruvate carboxykinase; pfam00821;substrate-binding site [chemical binding];metal-binding site [ion binding];GTP binding site [chemical binding]
GOHSU47	CDS	GOHSU_47_00260	complement(31879..32211)		pyruvate ferredoxin/flavodoxin oxidoreductase [migap]				pyruvate ferredoxin/flavodoxin oxidoreductase		G7GND5_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_26_00840 PE=4 SV=1;unnamed protein product [Desulfovibrio desulfurican s ND132];KEGG: dvl:Dvul_0348 pyruvate flavodoxin/ferredoxin oxidoreductase domain-containing protein; TIGRFAM: pyruvate ferredoxin/flavodoxin oxidoreductase; PFAM: pyruvate flavodoxin/ferredoxin oxidoreductase domain protein; Pyruvate/ketoisovalerate oxidoreductase, catalytic domain; Pyruvate-flavodoxin oxidoreductase, EKR domain; thiamine pyrophosphate TPP-binding domain-containing protein;pyruvate:ferredoxin (flavodoxin) oxidoreductase, homodimeric; TIGR02176
GOHSU47	CDS	GOHSU_47_00270	complement(32252..>32927)	3x codon	hypothetical protein [migap]				hypothetical protein		F1YKD6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11465 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
##GOHSU48	contig00291				32397
GOHSU48	CDS	GOHSU_48_00010	complement(<1..451)	3x	not assigned						G7H5U6_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_065_00140 PE=4 SV=1
GOHSU48	CDS	GOHSU_48_00020	complement(1147..1728)		putative TetR family transcriptional regulator [nite]			RER_pREL1-01020 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		H8GA24_9PSEU Uncharacterized protein (Precursor) OS =Saccharomonospora azurea NA-128 GN=SacazDRAFT_01626 PE=4 SV=1;TetR family transcriptional regulator [Frankia sp. EuI1c];PFAM: regulatory protein TetR; KEGG: sro:Sros_5790 hypothetical protein;Helix-turn-helix domains; cl00088;Bacterial transcriptional repressor; cl07106
GOHSU48	CDS	GOHSU_48_00030	1798..3177		putative drug resistance efflux protein [nite]			RER_pREL1-01030 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	EmrB/QacA subfamily drug resistance transporter		H5TNY9_9ACTO Putative drug resistance transporter O S=Gordonia otitidis NBRC 100426 GN=GOOTI_145_00160 PE=4 SV =1;EmrB/QacA subfamily drug resistance transporter [Ca tenulispora acidiphila DSM 44928];TIGRFAM: drug resistance transporter EmrB/QacA subfamily; PFAM: major facilitator superfamily MFS_1; KEGG: fre:Franean1_0731 EmrB/QacA family drug resistance transporter;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;Major Facilitator Superfamily; pfam07690;putative substrate translocation pore
GOHSU48	CDS	GOHSU_48_00040	3501..3941		putative MerR family transcriptional regulator [nite]			RER_11470 :putative MerR family transcriptional regulator : : : Rhodococcus erythropolis PR4	regulatory protein		E6SBH1_INTC7 Transcriptional regulator, MerR family OS=Intrasporangium calvum (strain ATCC 23552 / DSM 43043 / JCM 3097 / NBRC 12989 / 7 KIP) GN=Intca_3009 PE=4 SV=1;unnamed protein product [Frankia alni ACN14a];Genoscope sequence ID : FRAAL;Evidence 3 : Function proposed based on presence of conserved amino acid motif, structural feature or limited homology; Product type r : regulator;Predicted transcriptional regulators [Transcription]; COG0789;Helix-Turn-Helix DNA binding domain of transcription regulators from the MerR superfamily; cl02600;DNA binding residues [nucleotide binding]
GOHSU48	CDS	GOHSU_48_00050	4189..5667		putative SulP family transporter [nite]			RER_pREL1-00990 :putative SulP family transporter : : : Rhodococcus erythropolis PR4	integral membrane transporter		C3JS57_RHOER Integral membrane transporter OS=Rhodo coccus erythropolis SK121 GN=RHOER0001_5645 PE=4 SV=1;integral membrane transporter [Streptomyces roseosp orus NRRL 15998];Sulfate transporter family; cl15842;Sulphate Transporter and Anti-Sigma factor antagonist domain of SulP-like sulfate transporters, plays a role in the function and regulation of the transport activity, proposed general NTP binding function; cd07042
GOHSU48	CDS	GOHSU_48_00060	6073..6531		putative transposase [nite]			RER_pREL1-02710 :putative transposase : : : Rhodococcus erythropolis PR4	transposase for insertion sequence element		F9VU57_9ACTO Putative transposase OS=Gordonia alkan ivorans NBRC 16433 GN=GOALK_049_00010 PE=4 SV=1;transposase for insertion sequence element [Coryneb acterium variabile DSM 44702];Transposase and inactivated derivatives;Helix-turn-helix domain of transposase family ISL3; pfam13542
GOHSU48	CDS	GOHSU_48_00070	6590..7390		putative resolvase [nite]			ACMV_P6_00040 :putative resolvase : : : Acidiphilium multivorum AIU301	resolvase domain-containing protein		E6JD47_9ACTO Putative uncharacterized protein OS=Di etzia cinnamea P4 GN=ES5_15841 PE=4 SV=1;resolvase domain-containing protein [Pelobacter pro pionicus DSM 2379];PFAM: Resolvase, N-terminal domain; Resolvase helix-turn-helix domain protein; KEGG: neu:NE0751 site-specific recombinase;Serine Recombinase (SR) family, Resolvase and Invertase subfamily, catalytic domain; members contain a C-terminal DNA binding domain. Serine recombinases catalyze site-specific recombination of DNA molecules by a concerted, four-strand cleavage and...; cd03768;multiple promoter invertase; Provisional; PRK13413;catalytic residues [active];catalytic nucleophile [active];Presynaptic Site I dimer interface [polypeptide binding];Synaptic Antiparallel dimer interface [polypeptide binding];Synaptic Flat tetramer interface [polypeptide binding];Synaptic Site I dimer interface [polypeptide binding];DNA binding site [nucleotide binding];Helix-turn-helix domain of Hin and related proteins, a family of DNA-binding domains unique to bacteria and represented by the Hin protein of Salmonella. The basic HTH domain is a simple fold comprised of three core helices that form a right-handed...; cd00569;DNA-binding interface [nucleotide binding]
GOHSU48	CDS	GOHSU_48_00080	complement(7534..8307)		putative MFS transporter [nite]			ROP_pROB01-04260 :putative MFS transporter : : : Rhodococcus opacus B4	transporter		C3JWD8_RHOER Transmembrane efflux protein (Fragment ) OS=Rhodococcus erythropolis SK121 GN=RHOER0001_0152 PE=4 SV=1;transporter [Nocardia farcinica IFM 10152];methyl viologen resistance protein SmvA; Provisional
GOHSU48	CDS	GOHSU_48_00090	9196..9909		putative TetR family transcriptional regulator [nite]			RER_pREL1-02430 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		F6EL60_AMYSD Putative transcriptional regulator OS= Amycolicicoccus subflavus (strain DSM 45089 / DQS3-9A1) GN =AS9A_4290 PE=4 SV=1;TetR family transcriptional regulator [Mycobacteriu m sp. JLS];PFAM: regulatory protein, TetR; KEGG: mmc:Mmcs_4837 transcriptional regulator, TetR family;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU48	CDS	GOHSU_48_00100	10181..10435		putative transposase [nite]			KRH_13150 :putative transposase : : : Kocuria rhizophila DC2201	transposase IS3/IS911 family protein		A4FCR7_SACEN Putative transposase OS=Saccharopolysp ora erythraea (strain NRRL 23338) GN=SACE_2551 PE=4 SV=1;unnamed protein product [Frankia symbiont of Datisc a glomerata];PFAM: Transposase IS3/IS911; KEGG: fre:Franean1_3088 transposase IS3/IS911 family protein;Helix-turn-helix domains; cl00088
GOHSU48	CDS	GOHSU_48_00110	10497..10964		IS1096, tnpA protein [migap]				IS1096, tnpA protein		H0RD16_9ACTO Putative transposase OS=Gordonia polyi soprenivorans NBRC 16320 GN=GOPIP_032_00740 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by similarity to GP:150005; match to protein family HMM PF01610
GOHSU48	CDS	GOHSU_48_00120	complement(10917..11084)		diaminopimelate decarboxylase [migap]	lysA1 [migap]			diaminopimelate decarboxylase	lysA1	lysA1 gene product [Halomonas elongata DSM 2581];diaminopimelate decarboxylase; TIGR01048;Type III Pyridoxal 5-phosphate (PLP)-Dependent Enzyme Diaminopimelate Decarboxylase; cd06828;pyridoxal 5'-phosphate (PLP) binding site [chemical binding];substrate binding site [chemical binding];catalytic residues [active];dimer interface [polypeptide binding]
GOHSU48	CDS	GOHSU_48_00130	complement(11099..12139)		hypothetical protein [nite]			MLP_37600 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YGK8_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05791 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU48	CDS	GOHSU_48_00140	complement(12620..12955)		hypothetical protein [nite]			MLP_37610 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		F1YGY7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05526 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Helix-turn-helix XRE-family like proteins. Prokaryotic DNA binding proteins belonging to the xenobiotic response element family of transcriptional regulators; cl15761
GOHSU48	CDS	GOHSU_48_00150	complement(12952..14067)		putative recombinase [nite]			MLP_37620 :putative recombinase : : : Microlunatus phosphovorus NM-1	site-specific recombinase XerD		F1YGK6_9ACTO Phage integrase family protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_05781 PE=4 SV=1;site-specific recombinase XerD [Arthrobacter phenan threnivorans Sphe3];PFAM: Phage integrase family;DNA breaking-rejoining enzymes, C-terminal catalytic domain. The DNA breaking-rejoining enzyme superfamily includes type IB topoisomerases and tyrosine recombinases that share the same fold in their catalytic domain containing six conserved active site...; cl00213;DNA binding site [nucleotide binding];Int/Topo IB signature motif
GOHSU48	CDS	GOHSU_48_00160	14747..16039		putative competence-damage inducible protein [nite]			OBV_45450 :putative competence-damage inducible protein : : : Oscillibacter valericigenes Sjm18-20	competence damage-inducible protein A	cinA	F1YFM1_9ACTO CinA-like protein OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_02592 PE=3 SV=1;competence damage-inducible protein A [Bacillus lic heniformis ATCC 14580];RBL05695;competence damage-inducible protein A; Provisional;Competence-damaged protein. CinA is the first gene in the competence- inducible (cin) operon and is thought to be specifically required at some stage in the process of transformation. This domain is closely related to a domain, found in a variety of...; cd00885;putative MPT binding site;Competence-damaged protein; cl00666
GOHSU48	CDS	GOHSU_48_00170	complement(16077..16556)		flagellar basal-body rod protein FlgF [migap]				flagellar basal-body rod protein FlgF		F1YGY4_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05511 PE=4 SV=1;flagellar basal-body rod protein FlgF [Thauera sp. MZ1T];flagellar basal body rod protein FlgF; Reviewed; PRK12640;TIGRFAM: flagellar basal-body rod protein FlgF; PFAM: flagellar basal body rod protein; protein of unknown function DUF1078 domain protein; KEGG: azo:azo2736 flagellar basal-body rod protein FlgF;Flagellar basal body rod FlgEFG protein C-terminal; cl15855
GOHSU48	CDS	GOHSU_48_00180	16979..17542		conserved hypothetical protein [nite]			RER_52820 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	peroxidase-like protein		F1YGU5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_05296 PE=3 SV=1;peroxidase-like protein [Methylobacterium nodulans ORS 2060];TIGRFAM: alkylhydroperoxidase-like protein, AhpD family; uncharacterized peroxidase-related enzyme; PFAM: Carboxymuconolactone decarboxylase; KEGG: met:M446_3669 uncharacterized peroxidase-related enzyme;Carboxymuconolactone decarboxylase family; cl00460
GOHSU48	CDS	GOHSU_48_00190	17637..18269		propeptide PepSY amd peptidase M4 [migap]				propeptide PepSY amd peptidase M4		G2Z8S4_LISIP Putative uncharacterized protein OS=Li steria ivanovii (strain ATCC BAA-678 / PAM 55) GN=LIV_0045 PE=4 SV=1;propeptide PepSY amd peptidase M4 [Lactobacillus cr ispatus MV-3A- US];Peptidase propeptide and YPEB domain; cl15815
GOHSU48	CDS	GOHSU_48_00200	18397..19611		hypothetical protein [nite]			ROP_27780 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F9VTZ5_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_048_00460 PE=4 SV= 1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;KEGG: msm:MSMEG_6811 hypothetical protein;Uncharacterized protein conserved in bacteria (DUF2236); cl01472
GOHSU48	CDS	GOHSU_48_00210	19688..20404		hypothetical protein [migap]				hypothetical protein		D8K6L5_NITWC Putative uncharacterized protein OS=Ni trosococcus watsoni (strain C-113) GN=Nwat_1664 PE=4 SV=1;unnamed protein product [Granulicella mallensis MP5 ACTX8];KEGG: nwa:Nwat_1664 hypothetical protein
GOHSU48	CDS	GOHSU_48_00220	complement(20445..22442)		protein kinase [migap]				protein kinase		A4TA60_MYCGI Protein kinase OS=Mycobacterium gilvum (strain PYR-GCK) GN=Mflv_3427 PE=4 SV=1;protein kinase [Mycobacterium gilvum PYR- GCK];PFAM: protein kinase; SMART: tyrosine protein kinase; serine/threonine protein kinase; KEGG: mbo:Mb1775 anchored-membrane serine/threonine-protein kinase PknF (protein kinase F) (StpK F);Serine/Threonine protein kinases, catalytic domain; smart00220;Catalytic domain of Protein Kinases; cd00180;ATP binding site [chemical binding];substrate binding site [chemical binding];activation loop (A-loop)
GOHSU48	CDS	GOHSU_48_00230	22557..23597		probable N-acylhomoserine lactonase [nite]	 qsdA [nite]		RER_55000 :probable N-acylhomoserine lactonase : qsdA : : Rhodococcus erythropolis PR4	phosphotriesterase family protein		F1YGS8_9ACTO N-acylhomoserine lactonase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_05211 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Phosphotriesterase (PTE) catalyzes the hydrolysis of organophosphate nerve agents, including the chemical warfare agents VX, soman, and sarin as well as the insecticide paraoxon. PTE exists as a homodimer with one active site per monomer. The active site...; cd00530;substrate binding pocket [chemical binding];homodimer interaction site [polypeptide binding]
GOHSU48	CDS	GOHSU_48_00240	23605..24081		ferric uptake regulator [nite]	 furA [nite]		ROP_53180 :ferric uptake regulator : furA : : Rhodococcus opacus B4	transcription regulator FurA		F1YGS7_9ACTO Ferric-uptake regulator OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_05206 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF01475;Ferric uptake regulator(Fur) and related metalloregulatory proteins; typically iron-dependent, DNA-binding repressors and activators; cd07153;metal binding site 2 [ion binding];putative DNA binding helix;metal binding site 1 [ion binding];dimer interface [polypeptide binding];structural Zn2+ binding site [ion binding]
GOHSU48	CDS	GOHSU_48_00250	24180..26411		catalase-peroxidase [hamap]	katG [hamap]	MF_01961:catalase-peroxidase:katG 	ROP_53190 :catalase-peroxidase : katG : 1.11.1.6 : Rhodococcus opacus B4	catalase/peroxidase HPI		F1YGS6_9ACTO Catalase-peroxidase OS=Gordonia neofel ifaecis NRRL B-59395 GN=katG PE=3 SV=1;catalase/peroxidase HPI [Desulfitobacterium hafnien se DCB- 2];type strain of Desulfitobacterium hafniense;catalase/hydroperoxidase HPI(I); Provisional;TIGRFAM: catalase/peroxidase HPI; PFAM: Haem peroxidase; KEGG: gsu:GSU2100 catalase/peroxidase;N-terminal catalytic domain of catalase-peroxidases; cd00649;dimer interface [polypeptide binding];heme binding site [chemical binding];C-terminal non-catalytic domain of catalase-peroxidases; cd08200
GOHSU48	CDS	GOHSU_48_00260	26550..27584		putative F420-dependent oxidoreductase [nite]			ROP_47850 :putative F420-dependent oxidoreductase : : : Rhodococcus opacus B4	Luciferase-like monooxygenase superfamily protein		H0R053_9ACTO Putative F420-dependent oxidoreductase OS=Gordonia effusa NBRC 100432 GN=GOEFS_054_00680 PE=4 SV =1;Luciferase-like monooxygenase superfamily protein [ marine gamma proteobacterium HTCC2148];[C] COG2141 Coenzyme F420-dependent N5,N10-methylene tetrahydromethanopterin reductase and related flavin-dependent oxidoreductases;probable F420-dependent oxidoreductase, Rv3520c family; TIGR03559;N5,N10-methylenetetrahydromethanopterin reductase (Mer) catalyzes the reduction of N5,N10-methylenetetrahydromethanopterin with reduced coenzyme F420 to N5-methyltetrahydromethanopterin and oxidized coenzyme F420; cd01097
GOHSU48	CDS	GOHSU_48_00270	27599..28771		periplasmic-binding protein [migap]				periplasmic-binding protein		D1A2C6_THECD Periplasmic binding protein (Precursor ) OS=Thermomonospora curvata (strain ATCC 19995 / DSM 4318 3 / JCM 3096 / NCIMB 10081) GN=Tcur_0344 PE=4 SV=1;unnamed protein product [Propionibacterium acnes Ty peIA2 P.acn33];COG0614 ABC-type Fe3+-hydroxamate transport system, periplasmic component;Periplasmic binding protein TroA_d.  These proteins are predicted to function as initial receptors in the ABC metal ion uptake in eubacteria and archaea.  They belong to the TroA superfamily of helical backbone metal receptor proteins that share a...; cd01141;Periplasmic binding protein; pfam01497;putative ligand binding site [chemical binding]
GOHSU48	CDS	GOHSU_48_00280	28768..29805		putative ABC transporter permease protein [nite]			RER_43430 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	transport system permease		I0RWK3_MYCXE Transport system permease OS=Mycobacte rium xenopi RIVM700367 GN=MXEN_06686 PE=4 SV=1;transport system permease [Odoribacter splanchnicus DSM 20712];Signal predicted by SignalP 3.0 HMM;COGs: COG0609 ABC-type Fe3+-siderophore transport system permease component; InterPro IPR000522; KEGG: pdi:BDI_1018 iron(III) ABC transporter  permease; PFAM: Bacterial transport system permease; SPTR:  uncharacterized protein; PFAM: FecCD transport family;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;ABC-ATPase subunit  interface;dimer interface [polypeptide binding];putative PBP binding regions
GOHSU48	CDS	GOHSU_48_00290	29802..30629		putative iron-siderophore ABC transporter ATP-binding protein [nite]			RER_03190 :putative iron-siderophore ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	Iron-chelate-transporting ATPase		F5Z3V5_MYCSD ABC transporter related protein OS=Myc obacterium sp. (strain JDM601) GN=JDM601_3183 PE=4 SV=1;Iron-chelate-transporting ATPase [Odoribacter splan chnicus DSM 20712];COGs: COG1120 ABC-type cobalamin/Fe3+-siderophores transport systems ATPase components; InterPro IPR003439: IPR003593; KEGG: bfs:BF2248  iron transport related ATP-binding protein; PFAM: ABC transporter-like; PRIAM: Iron-chelate-transporting ATPase; SMART: ATPase, AAA+ type, core; SPTR:  uncharacterized protein; PFAM: ABC transporter;ABC-type cobalamin/Fe3+-siderophores transport systems, ATPase components [Inorganic ion transport and metabolism / Coenzyme metabolism]; COG1120;ABC transporters, involved in the uptake of siderophores, heme, and vitamin B12, are widely conserved in bacteria and archaea.  Only very few species lack representatives of the siderophore family transporters. The E. coli BtuCD protein is an ABC...; cd03214;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU48	CDS	GOHSU_48_00300	complement(30662..32278)		protease [migap]				protease		E9V216_9ACTO Serine/arginine repetitive matrix prot ein 1 OS=Nocardioidaceae bacterium Broad-1 GN=NBCG_05111 P E=4 SV=1;protease [Clostridium kluyveri DSM 555];Trypsin-like serine protease; Many of these are synthesized as inactive precursor zymogens that are cleaved during limited proteolysis to generate their active forms. Alignment contains also inactive enzymes that have substitutions of the catalytic triad...; cl00149;PDZ domain of tryspin-like serine proteases, such as DegP/HtrA, which are oligomeric proteins involved in heat-shock response, chaperone function, and apoptosis. May be responsible for substrate recognition and/or binding, as most PDZ domains bind...; cd00987;protein binding site [polypeptide binding]
##GOHSU49	contig00301				2144
GOHSU49	CDS	GOHSU_49_00010	complement(<1..1207)	3x codon	hypothetical protein [nite]			ROP_50360 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLX6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14601 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: scl:sce3972 hypothetical protein
GOHSU49	CDS	GOHSU_49_00020	complement(1182..1955)		hypothetical protein [nite]			ROP_50350 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLX7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14606 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF3027); pfam11228
##GOHSU50	contig00302				16387
GOHSU50	CDS	GOHSU_50_00010	<1..1770	codon	conserved hypothetical membrane protein [nite]			RER_46330 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	major facilitator superfamily protein		F1YLX8_9ACTO Major facilitator superfamily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14611 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: major facilitator superfamily MFS_1; KEGG: hypothetical protein;H+ Antiporter protein; TIGR00900
GOHSU50	CDS	GOHSU_50_00020	1795..2403		hypothetical protein [nite]			ROP_50320 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLX9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14616 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.901) with cleavage site probability 0.362 at residue 28;Protein of unknown function (DUF2771); pfam10969
GOHSU50	CDS	GOHSU_50_00030	complement(2440..2853)		cold shock protein [nite]			ROP_50310 :cold shock protein : : : Rhodococcus opacus B4	cold-shock protein DNA-binding protein		D0L5K3_GORB4 Cold-shock protein DNA-binding protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_1236 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: Cold-shock protein DNA-binding; SMART: Cold shock protein; KEGG: vok:COSY_0845 cold shock protein;Cold-Shock Protein (CSP) contains an S1-like cold-shock domain (CSD) that is found in eukaryotes, prokaryotes, and archaea.  CSP's include the major cold-shock proteins CspA and CspB in bacteria and the eukaryotic gene regulatory factor Y-box protein; cd04458;DNA-binding site [nucleotide binding];RNA-binding motif
GOHSU50	CDS	GOHSU_50_00040	2958..3176		hypothetical membrane protein [nite]			ROP_50300 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLY1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14626 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU50	CDS	GOHSU_50_00050	3173..3715		putative oxidoreductase [nite]			ROP_14290 :putative oxidoreductase : : : Rhodococcus opacus B4	flavin reductase like domain-containing protein		F1YLY2_9ACTO Flavin reductase domain-containing pro tein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14631 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Flavin Reductases; cl00801
GOHSU50	CDS	GOHSU_50_00060	4180..4776		resuscitation-promoting factor [nite]	 rpf [nite]		ROP_50280 :resuscitation-promoting factor : rpf : : Rhodococcus opacus B4	secreted protein		H0QY01_9ACTO Resuscitation-promoting factor OS=Gord onia effusa NBRC 100432 GN=rpf PE=4 SV=1;secreted protein [Streptomyces viridochromogenes DS M 40736];lysozyme_like domain.  This contains several members including Soluble Lytic Transglycosylases (SLT), Goose Egg-White Lysozymes (GEWL), Hen Egg-White Lysozymes (HEWL), chitinases, bacteriophage lambda lysozymes, endolysins, autolysins, and chitosanases; cl00222;Lysin domain, found in a variety of enzymes involved in bacterial cell wall degradation. This domain may have a general peptidoglycan binding function; cd00118;putative peptidoglycan binding site
GOHSU50	CDS	GOHSU_50_00070	complement(4994..5176)		conserved hypothetical protein [nite]			RER_46400 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5S0V4_9NOCA Putative uncharacterized protein OS=No cardia brasiliensis ATCC 700358 GN=O3I_30579 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU50	CDS	GOHSU_50_00080	5272..7593		hypothetical protein [nite]			ROP_50260 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLY5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14646 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];Helicase conserved C-terminal domain; pfam13625;WYL domain; cl14852
GOHSU50	CDS	GOHSU_50_00090	7607..9250		putative ATP-dependent DNA helicase [nite]			RER_46470 :putative ATP-dependent DNA helicase : : 3.6.1.- : Rhodococcus erythropolis PR4	DNA helicase		F9W121_9ACTO Putative ATP-dependent DNA helicase OS =Gordonia alkanivorans NBRC 16433 GN=GOALK_110_00200 PE=4 SV=1;DNA helicase [Mycobacterium abscessus ATCC 19977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Similar to Q269P6_MYCVN 0.0;Helicase conserved C-terminal domain; pfam13625;DNA or RNA helicases of superfamily II [Transcription / DNA replication, recombination, and repair]; COG1061;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding];putative Mg++ binding site [ion binding];Helicase superfamily c-terminal domain; associated with DEXDc-, DEAD-, and DEAH-box proteins, yeast initiation factor 4A, Ski2p, and Hepatitis C virus NS3 helicases; this domain is found in a wide variety of helicases and helicase related proteins; may...; cd00079;nucleotide binding region [chemical binding];ATP-binding site [chemical binding]
GOHSU50	CDS	GOHSU_50_00100	9254..10420		aminotransferase [nite]			RER_46740 :aminotransferase : : 2.6.1.- : Rhodococcus erythropolis PR4	class I/II aminotransferase		G7GS50_9ACTO Putative aminotransferase OS=Gordonia amarae NBRC 15530 GN=GOAMR_52_00110 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.876) with cleavage site probability 0.862 at residue 25;PFAM: aminotransferase class I and II; KEGG: mxa:MXAN_3751 aminotransferase, classes I and II;aminotransferase; Validated;Aspartate aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). Pyridoxal phosphate combines with an alpha-amino acid to form a compound called a Schiff base or aldimine...; cd00609;pyridoxal 5'-phosphate binding site [chemical binding];homodimer interface [polypeptide binding];catalytic residue [active]
GOHSU50	CDS	GOHSU_50_00110	10445..11080		hypothetical protein [nite]			ROP_49990 :hypothetical protein : : : Rhodococcus opacus B4	alkylated DNA repair protein		D0L5H5_GORB4 Putative alkylated DNA repair protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / J CM 3198 / NCTC 10667) GN=Gbro_1203 PE=4 SV=1;alkylated DNA repair protein [Frankia sp. EuI1c];KEGG: fal:FRAAL3273  alkylated DNA repair protein;2OG-Fe(II) oxygenase superfamily; cl15773
GOHSU50	CDS	GOHSU_50_00120	complement(11105..11986)		putative oxidoreductase [nite]			ROP_49720 :putative oxidoreductase : : : Rhodococcus opacus B4	dehydrogenase/reductase		H5UFI4_9ACTO Putative oxidoreductase OS=Gordonia te rrae NBRC 100016 GN=GOTRE_067_00010 PE=4 SV=1;dehydrogenase/reductase [Mycobacterium sp. JDM601];short chain dehydrogenase; Provisional;retinol dehydrogenase (retinol-DH), Light dependent Protochlorophyllide (Pchlide) OxidoReductase (LPOR) and related proteins, classical (c) SDRs; cd05327;putative NAD(P) binding site [chemical binding]
GOHSU50	CDS	GOHSU_50_00130	complement(11996..13696)		adenylate cyclase [nite]	 cya [nite]		ROP_11720 :adenylate cyclase : cya : 4.6.1.1 : Rhodococcus opacus B4	adenylate cyclase		F1YLZ0_9ACTO Histidine kinase HAMP region domain-co ntaining protein OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_14671 PE=4 SV=1;adenylate cyclase [Amycolicicoccus subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;Histidine kinase, Adenylyl cyclase, Methyl-accepting protein, and Phosphatase (HAMP) domain. HAMP is a signaling domain which occurs in a wide variety of signaling proteins, many of which are bacterial. The HAMP domain consists of two alpha helices...; cd06225;dimerization interface [polypeptide binding];Adenylyl- / guanylyl cyclase, catalytic domain; smart00044;cyclase homology domain; cd07302;nucleotidyl binding site;metal binding site [ion binding];dimer interface [polypeptide binding]
GOHSU50	CDS	GOHSU_50_00140	complement(13771..14322)		hypothetical protein [nite]			ROP_37670 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H0R2V4_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_083_00370 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: gme:Gmet_3449 YceI;YceI-like domain; cl01001
GOHSU50	CDS	GOHSU_50_00150	complement(14540..15784)		LGFP repeat-containing protein [migap]				LGFP repeat-containing protein		A4F663_SACEN Putative uncharacterized protein OS=Sa ccharopolyspora erythraea (strain NRRL 23338) GN=SACE_0185 PE=4 SV=1;LGFP repeat-containing protein [Streptomyces pristi naespiralis ATCC 25486];LGFP repeat; pfam08310
GOHSU50	CDS	GOHSU_50_00160	complement(15787..16386)		hypothetical protein [migap]				hypothetical protein		F1YLZ3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14686 PE=4 SV=1;unnamed protein product [Caulobacter crescentus NA1 000];CC_0675
##GOHSU51	contig00303				5387
GOHSU51	CDS	GOHSU_51_00010	complement(<1..2027)	3x codon	glutamate/valine-rich protein [migap]				glutamate/valine-rich protein		D8TF47_SELML Putative uncharacterized protein (Frag ment) OS=Selaginella moellendorffii GN=SELMODRAFT_138347 P E=4 SV=1;glutamate/valine-rich protein [Halobacterium salina rum R1]
GOHSU51	tRNA	GOHSU_51_t0010	complement(2241..2314)		not assigned						identified by tRNAscan-SE; putative
GOHSU51	tRNA	GOHSU_51_t0020	complement(2396..2469)		not assigned						identified by tRNAscan-SE; putative
GOHSU51	tRNA	GOHSU_51_t0030	complement(2506..2582)		not assigned						identified by tRNAscan-SE; putative
GOHSU51	tRNA	GOHSU_51_t0040	complement(2619..2691)		not assigned						identified by tRNAscan-SE; putative
GOHSU51	CDS	GOHSU_51_00020	3023..3568		putative TetR family transcriptional regulator [nite]			YM304_01210 :putative TetR family transcriptional regulator : : : Acidimicrobidae bacterium YM16-304	TetR/AcrR family transcriptional regulator		G8PTW5_PSEUV Transcriptional regulator, TetR family OS=Pseudovibrio sp. (strain FO-BEG1) GN=PSE_4213 PE=4 SV= 1;unnamed protein product [Pseudoxanthomonas spadix B D- a59];COG1309 Transcriptional regulator;HTH-type transcriptional regulator RutR; Provisional;Helix-turn-helix domains; cl00088;AefR-like transcriptional repressor, C-terminal region; pfam14246
GOHSU51	tRNA	GOHSU_51_t0050	complement(3620..3695)		not assigned						identified by tRNAscan-SE; putative
GOHSU51	CDS	GOHSU_51_00030	complement(3739..4644)		putative hydrolase [nite]			YM304_12960 :putative hydrolase : : : Acidimicrobidae bacterium YM16-304	3-oxoadipate enol-lactonase		D0L519_GORB4 Alpha/beta hydrolase fold protein OS=G ordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3 198 / NCTC 10667) GN=Gbro_1163 PE=4 SV=1;3-oxoadipate enol-lactonase [Nocardiopsis dassonvil lei subsp. dassonvillei DSM 43111];COGs: COG0596 hydrolase or acyltransferase (alpha/beta hydrolase superfamily); InterPro IPR012790:IPR000073; KEGG: rxy:Rxyl_1578 3-oxoadipate enol-lactonase; PFAM: alpha/beta hydrolase fold; SPTR: C1YUY2 3-oxoadipate enol-lactonase; TIGRFAM: 3-oxoadipate enol-lactonase; PFAM: alpha/beta hydrolase fold; TIGRFAM: 3-oxoadipate enol-lactonase;3-oxoadipate enol-lactonase; TIGR02427;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU51	CDS	GOHSU_51_00040	complement(4691..>5387)	3x codon	hypothetical membrane protein [nite]			RER_01290 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	copper resistance D domain-containing protein		F9VU40_9ACTO Putative uncharacterized protein OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_048_00910 PE=4 SV= 1;copper resistance D domain-containing protein [Rals tonia pickettii 12J];PFAM: copper resistance D domain protein; KEGG: rso:RSp0659 probable copper resistance D transmembrane protein;Copper resistance protein D; cl00563
##GOHSU52	contig00304				7548
GOHSU52	CDS	GOHSU_52_00010	complement(148..720)		putative copper resistance protein [nite]			RER_01300 :putative copper resistance protein : : : Rhodococcus erythropolis PR4	copper resistance protein CopC;hypothetical protein		H6R6C4_NOCCG Putative copper resistance protein OS= Nocardia cyriacigeorgica (strain GUH-2) GN=NOCYR_0073 PE=4 SV=1;copper resistance protein CopC [Mycobacterium gilvu m PYR- GCK];Signal predicted by SignalP 2.0 HMM (Signal peptide probability1.000) with cleavage site probability 0.731 at residue 33;PFAM: copper resistance protein CopC; KEGG: mva:Mvan_5679 copper resistance protein CopC;Bacterial Ig-like domain; cl01012
GOHSU52	CDS	GOHSU_52_00020	748..2748		putative glutamine-dependent NAD(+) synthetase [nite]	 nadE [nite]		MLP_10360 :putative glutamine-dependent NAD(+) synthetase : nadE : 6.3.5.1 : Microlunatus phosphovorus NM-1	NAD synthetase	nadE	F1YM02_9ACTO NAD synthetase OS=Gordonia neofelifaec is NRRL B-59395 GN=nadE PE=4 SV=1;NAD synthetase [Gluconacetobacter diazotrophicus PA l 5];NAD synthetase; Reviewed; PRK02628;catalyzes the formation of nicotinamide adenine dinucleotide (NAD) from nicotinic acid adenine dinucleotide (NAAD) using either ammonia or glutamine as the amide donor and ATP; ammonia-utilizing enzymes include the ones from Bacillus and Escherichia coli while glutamine-utilizing enzymes include the Mycobacterial one; forms homodimers;Glutamine aminotransferase (GAT, glutaminase) domain of glutamine-dependent NAD synthetases (class 7 and 8 nitrilases); cd07570;multimer interface [polypeptide binding];catalytic triad [active];protein interface 1 [polypeptide binding];NAD+ synthase is a homodimer, which catalyzes the final step in de novo nicotinamide adenine dinucleotide (NAD+) biosynthesis, an amide transfer from either ammonia or glutamine to nicotinic acid adenine dinucleotide (NaAD). The conversion of NaAD to NAD...; cd00553;homodimer interface [polypeptide binding];NAD binding pocket [chemical binding];ATP binding pocket [chemical binding];Mg binding site [ion binding];active-site loop [active]
GOHSU52	CDS	GOHSU_52_00030	complement(2771..4306)		hypothetical protein [nite]			ROP_68400 :hypothetical protein : : : Rhodococcus opacus B4	acyltransferase, WS/DGAT/MGAT		F1YNY7_9ACTO Acyltransferase, WS/DGAT/MGAT OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_18182 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Uncharacterised protein family (UPF0089); TIGRFAM: acyltransferase, WS/DGAT/MGAT;acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007
GOHSU52	CDS	GOHSU_52_00040	4444..4959		conserved hypothetical protein [nite]			RER_42000 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YNZ1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18202 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109
GOHSU52	CDS	GOHSU_52_00050	5103..6092		probable acyl-[acyl-carrier-protein] desaturase DesA1 [nite]	 desA1 [nite]		RER_47900 :probable acyl-[acyl-carrier-protein] desaturase DesA1 : desA1 : 1.14.19.2 : Rhodococcus erythropolis PR4	fatty acid desaturase type 2		H5UHC9_9ACTO Acyl-[acyl-carrier-protein] desaturase OS=Gordonia terrae NBRC 100016 GN=desA PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: fatty acid desaturase type 2; KEGG: Os01g0919900; hypothetical protein;Fatty acid desaturase; pfam03405;Acyl ACP desaturase, ferritin-like diiron-binding domain; cd01050;homodimer interface [polypeptide binding];putative substrate binding pocket [chemical binding];diiron center [ion binding]
GOHSU52	CDS	GOHSU_52_00060	6226..7326		putative tRNA-dihydrouridine synthase [nite]			ROP_49500 :putative tRNA-dihydrouridine synthase : : : Rhodococcus opacus B4	putative tRNA-dihydrouridine synthase		F1YNZ3_9ACTO tRNA-dihydrouridine synthase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_18212 PE=3 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Dihydrouridine synthase-like (DUS-like) FMN-binding domain. Members of this family catalyze the reduction of the 5,6-double bond of a uridine residue on tRNA. Dihydrouridine modification of tRNA is widely observed in prokaryotes and eukaryotes, and also...; cd02801;putative TIM-barrel protein, nifR3 family; TIGR00737;FMN binding site [chemical binding];catalytic residues [active];substrate binding site [chemical binding]
GOHSU52	CDS	GOHSU_52_00070	7396..>7548	codon	not assigned						F1YNZ4_9ACTO Cell envelope-like function transcript ional attenuator OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_18217 PE=4 SV=1
##GOHSU53	contig00311				24453
GOHSU53	CDS	GOHSU_53_00010	complement(56..508)	case	50S ribosomal protein L9 [hamap]	rplI [hamap]	MF_00503:50S ribosomal protein L9:rplI 	ROP_32220 :50S ribosomal protein L9 : rplI : : Rhodococcus opacus B4	50S ribosomal protein L9	rplI	F1YL37_9ACTO 50S ribosomal protein L9 OS=Gordonia n eofelifaecis NRRL B-59395 GN=rplI PE=3 SV=1;50S ribosomal protein L9 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;50S ribosomal protein L9; Reviewed; PRK00137;Ribosomal protein L9, N-terminal domain; pfam01281;in Escherichia coli this protein is wrapped around the base of the L1 stalk;Ribosomal protein L9, C-terminal domain; pfam03948
GOHSU53	CDS	GOHSU_53_00020	complement(529..786)	case	30S ribosomal protein S18 [hamap]	rpsR [hamap]	MF_00270:30S ribosomal protein S18:rpsR 	RER_58290 :probable 30S ribosomal protein S18 : rpsR : : Rhodococcus erythropolis PR4	30S ribosomal protein S18		H0R3L4_9ACTO 30S ribosomal protein S18 2 OS=Gordoni a effusa NBRC 100432 GN=rpsR PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: ribosomal protein S18; PFAM: ribosomal protein S18; KEGG: dps:DP2596 30S ribosomal protein S18;Ribosomal protein S18; cl00373
GOHSU53	CDS	GOHSU_53_00030	complement(829..1389)		single-stranded DNA-binding protein [nite]	 ssb [nite]		RER_58300 :single-stranded DNA-binding protein : ssb : : Rhodococcus erythropolis PR4	single-strand binding protein		H5TS31_9ACTO Single-stranded DNA-binding protein OS =Gordonia otitidis NBRC 100426 GN=ssb PE=4 SV=1;single-strand binding protein [Geodermatophilus obs curus DSM 43160];TIGRFAM: single-strand binding protein; PFAM: single-strand binding protein/Primosomal replication protein n; KEGG: mmi:MMAR_0073 single-stranded DNA-binding protein;SSB_OBF: A subfamily of OB folds similar to the OB fold of ssDNA-binding protein (SSB). SSBs bind with high affinity to ssDNA. They bind to and protect ssDNA intermediates during DNA metabolic pathways. All bacterial and eukaryotic SSBs studied to date...; cd04496;dimer interface [polypeptide binding];ssDNA binding site [nucleotide binding];tetramer (dimer of dimers) interface [polypeptide binding]
GOHSU53	CDS	GOHSU_53_00040	complement(1444..1725)	case	30S ribosomal protein S6 [hamap]	rpsF [hamap]	MF_00360:30S ribosomal protein S6:rpsF 	ROP_32250 :30S ribosomal protein S6 : rpsF : : Rhodococcus opacus B4	30S ribosomal protein S6	rpsF	G7GXF0_9ACTO 30S ribosomal protein S6 OS=Gordonia a raii NBRC 100433 GN=rpsF PE=3 SV=1;rpsF gene product [Gordonia polyisoprenivorans VH2] ;Ribosomal protein S6; cl00414
GOHSU53	CDS	GOHSU_53_00050	1926..3953		hypothetical protein [migap]				hypothetical protein		F1YL41_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_12993 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Protein of unknown function (DUF2029); cl09812
GOHSU53	CDS	GOHSU_53_00060	complement(3919..5421)		conserved hypothetical membrane protein [nite]			RER_58370 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	integral membrane protein-like protein		F1YL42_9ACTO Integral membrane protein-like protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_12998 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: ade:Adeh_0166 integral membrane protein-like;Protein of unknown function (DUF2029); cl09812
GOHSU53	CDS	GOHSU_53_00070	complement(5428..7629)		penicillin-binding protein [nite]			RER_58990 :penicillin-binding protein : : : Rhodococcus erythropolis PR4	penicillin-binding protein		F1YL43_9ACTO Peptidoglycan glycosyltransferase OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_13003 PE=4 SV=1 ;penicillin-binding protein [Rhodococcus equi 103S];Transglycosylase; cl07896;penicillin-binding protein, 1A family; TIGR02074;Penicillin binding protein transpeptidase domain; cl01009
GOHSU53	CDS	GOHSU_53_00080	complement(7795..8169)	overlap	conserved hypothetical protein [nite]			RER_58400 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H6N243_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c49210 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU53	CDS	GOHSU_53_00090	8056..8289	overlap	not assigned						
GOHSU53	CDS	GOHSU_53_00100	8369..9046		putative PadR family transcriptional regulator [nite]			ROP_32390 :putative PadR family transcriptional regulator : : : Rhodococcus opacus B4	ParR family transcriptional regulator		F1YL45_9ACTO Transcriptional regulator PadR family protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_130 13 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG1695 Predicted transcriptional regulators;Helix-turn-helix domains; cl00088
GOHSU53	CDS	GOHSU_53_00110	9039..10133		myo-inositol-1-phosphate synthase [nite]	 ino1 [nite]		RER_58450 :myo-inositol-1-phosphate synthase : ino1 : 5.5.1.4 : Rhodococcus erythropolis PR4	1L-myo-inositol-1-phosphate synthase		F1YL46_9ACTO Inositol 1-phosphate synthase OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_13018 PE=4 SV=1;1L-myo-inositol-1-phosphate synthase [Streptomyces clavuligerus ATCC 27064];Myo-inositol-1-phosphate synthase; cl00554
GOHSU53	CDS	GOHSU_53_00120	complement(10461..11339)		replicative DNA helicase [nite]	 dnaB [nite]		CSE_03330 :replicative DNA helicase : dnaB : 3.6.1.- : Caldisericum exile AZM16c01	replicative DNA helicase		D3Q400_STANL Replicative DNA helicase OS=Stackebran dtia nassauensis (strain DSM 44728 / NRRL B-16338 / NBRC 1 02104 / LLR-40K-21) GN=Snas_6265 PE=4 SV=1;replicative DNA helicase [Stackebrandtia nassauensi s DSM 44728];KEGG: dds:Ddes_2021 replicative DNA helicase; TIGRFAM: replicative DNA helicase; PFAM: DnaB domain protein helicase domain protein; SMART: AAA ATPase;replicative DNA helicase; TIGR00665;DnaB helicase C terminal domain. The hexameric helicase DnaB unwinds the DNA duplex at the  chromosome replication fork. Although the mechanism by which DnaB both couples ATP hydrolysis to translocation along DNA and denatures the duplex is unknown, a...; cd00984;Walker A motif;ATP binding site [chemical binding];Walker B motif;DNA binding loops [nucleotide binding]
GOHSU53	CDS	GOHSU_53_00130	complement(11465..11770)		chorismate mutase [migap]	aroQ [migap]			chorismate mutase	aroQ	G7GY85_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_013_00040 PE=4 SV=1;chorismate mutase [Salmonella bongori NCTC 12419];Chorismate mutase type II; cl00693
GOHSU53	CDS	GOHSU_53_00140	complement(11922..12542)		LuxR family transcriptional regulator [migap]				LuxR family transcriptional regulator		F9W299_9ACTO Putative LuxR family transcriptional r egulator OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_120_ 00450 PE=4 SV=1;LuxR family transcriptional regulator [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;Helix-turn-helix domains; cl00088;DNA binding residues [nucleotide binding];dimerization interface [polypeptide binding]
GOHSU53	CDS	GOHSU_53_00150	12701..13474		hypothetical protein [migap]				hypothetical protein		H0R3N5_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_093_00150 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197
GOHSU53	CDS	GOHSU_53_00160	14003..15199		putative solute/hydrogen antiporter [nite]			MLP_16870 :putative solute/hydrogen antiporter : : : Microlunatus phosphovorus NM-1	sodium/hydrogen exchanger		F1YL48_9ACTO Sodium/hydrogen exchanger OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_13028 PE=4 SV=1;sodium/hydrogen exchanger [Nakamurella multipartita DSM 44233];PFAM: sodium/hydrogen exchanger; KEGG: smt:Smal_1281 sodium/hydrogen exchanger;Kef-type K+ transport systems, membrane components [Inorganic ion transport and metabolism]; cl10482
GOHSU53	CDS	GOHSU_53_00170	complement(15215..16249)		hypothetical membrane protein [nite]			RER_04270 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	hypothetical protein		C0ZN83_RHOE4 Hypothetical membrane protein OS=Rhodo coccus erythropolis (strain PR4 / NBRC 100887) GN=RER_0427 0 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU53	CDS	GOHSU_53_00180	complement(16246..17463)		putative two-component histidine kinase [nite]			RER_04280 :putative two-component histidine kinase : : 2.7.13.- : Rhodococcus erythropolis PR4	ATPase domain-containing protein		C3JP01_RHOER ATPase domain protein OS=Rhodococcus e rythropolis SK121 GN=RHOER0001_3725 PE=4 SV=1;ATPase domain-containing protein [Amycolicicoccus s ubflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;Signal transduction histidine kinase [Signal transduction mechanisms];Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU53	CDS	GOHSU_53_00190	complement(17523..18941)		argininosuccinate synthase [hamap]	argG [hamap]	MF_00581:argininosuccinate synthase:argG 	MLP_47280 :argininosuccinate synthase : argG : 6.3.4.5 : Microlunatus phosphovorus NM-1	argininosuccinate synthase	argG	F1YL51_9ACTO Argininosuccinate synthase OS=Gordonia neofelifaecis NRRL B-59395 GN=argG PE=3 SV=1;argininosuccinate synthase [Microlunatus phosphovor us NM- 1];strain coidentity: NBRC 101784;protein synonym:citrulline--aspartate ligase;Argininosuccinate synthase [Amino acid transport and metabolism]; COG0137;Adenine nucleotide alpha hydrolases superfamily including N type ATP PPases, ATP sulphurylases Universal Stress Response protein and electron transfer flavoprotein (ETF). The domain forms a apha/beta/apha fold which  binds to Adenosine nucleotide; cl00292;Ligand Binding Site [chemical binding]
GOHSU53	CDS	GOHSU_53_00200	19166..20704		putative drug resistance efflux protein [nite]			RER_33930 :putative drug resistance efflux protein : : : Rhodococcus erythropolis PR4	drug resistance efflux protein		C1A0G6_RHOE4 Putative drug resistance efflux protei n OS=Rhodococcus erythropolis (strain PR4 / NBRC 100887) G N=RER_33930 PE=4 SV=1;drug resistance efflux protein [Rhodococcus erythro polis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174;putative substrate translocation pore
GOHSU53	CDS	GOHSU_53_00210	complement(20784..22085)		putative tryptophan synthase beta chain [nite]			RER_08690 :putative tryptophan synthase beta chain : : 4.2.1.20 : Rhodococcus erythropolis PR4	pyridoxal-phosphate dependent TrpB-like protein		F1YL54_9ACTO Tryptophan synthase subunit beta OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13058 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: GSVIVT00038137001; hypothetical protein LOC100261704; TIGRFAM: pyridoxal-phosphate dependent TrpB-like enzyme; PFAM: pyridoxal-5'-phosphate-dependent protein subunit beta;Tryptophan synthase beta superfamily (fold type II); this family of pyridoxal phosphate (PLP)-dependent enzymes catalyzes beta-replacement and beta-elimination reactions. This CD corresponds to aminocyclopropane-1-carboxylate deaminase (ACCD), tryptophan...; cl00342;Pyridoxal-phosphate dependent enzyme; pfam00291;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU53	CDS	GOHSU_53_00220	complement(22154..22792)		putative methyltransferase [nite]			ROP_13890 :putative methyltransferase : : 2.1.1.- : Rhodococcus opacus B4	methyltransferase		H5TH84_9ACTO Putative methyltransferase OS=Gordonia otitidis NBRC 100426 GN=GOOTI_029_00180 PE=4 SV=1;methyltransferase [Rhodococcus opacus B4];S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU53	CDS	GOHSU_53_00230	22870..23658		conserved hypothetical protein [nite]			RER_59100 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	putative F420-dependent oxidoreductase, MSMEG_2906 family		F1YL58_9ACTO Coenzyme F420-dependent N5 N10-methyle ne tetrahydromethanopterin reductase-like protein OS=Gordo nia neofelifaecis NRRL B-59395 GN=SCNU_13078 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Luciferase-like monooxygenase; TIGRFAM: probable F420-dependent oxidoreductase, MSMEG_2906 family;probable F420-dependent oxidoreductase, MSMEG_2906 family; TIGR03856;Flavin-utilizing monoxygenases; cl07892;N5,N10-methylenetetrahydromethanopterin reductase (Mer) catalyzes the reduction of N5,N10-methylenetetrahydromethanopterin with reduced coenzyme F420 to N5-methyltetrahydromethanopterin and oxidized coenzyme F420; cd01097
GOHSU53	CDS	GOHSU_53_00240	complement(23607..>24453)	3x codon frame	putative oxidoreductase [nite]			KSE_23040 :putative oxidoreductase : : : Kitasatospora setae NBRC 14216	phytoene dehydrogenase		F1YL60_9ACTO FAD dependent oxidoreductase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_13088 PE=4 SV=1;phytoene dehydrogenase [Microbacterium testaceum St LB037];Phytoene dehydrogenase and related proteins [Secondary metabolites biosynthesis, transport, and catabolism]
##GOHSU54	contig00321				22051
GOHSU54	CDS	GOHSU_54_00010	<1..2408	3x codon frame	putative ribonuclease [nite]			RER_37950 :putative ribonuclease : : 3.1.-.- : Rhodococcus erythropolis PR4	ribonuclease, Rne/Rng family		F1YI59_9ACTO Ribonuclease, Rne/Rng family OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_07868 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: eba:ebA3964 ribonuclease G (RNase G); TIGRFAM: ribonuclease, Rne/Rng family; PFAM: RNA-binding protein AU-1/ribonuclease E/G; RNA binding S1 domain protein; translation initiation factor IF-2 domain protein;ribonuclease, Rne/Rng family; TIGR00757;S1_RNase_E: RNase E and RNase G, S1-like RNA-binding domain. RNase E is an essential endoribonuclease in the processing and degradation of RNA. In addition to its role in mRNA degradation, RNase E has also been implicated in the processing of rRNA, and...; cd04453;homodimer interface [polypeptide binding];oligonucleotide binding site [chemical binding]
GOHSU54	CDS	GOHSU_54_00020	2606..2917	case	50S ribosomal protein L21 [hamap]	rplU [hamap]	MF_01363:50S ribosomal protein L21:rplU 	ROP_10350 :50S ribosomal protein L21 : rplU : : Rhodococcus opacus B4	50S ribosomal protein L21	rplU	H5TZC9_9ACTO 50S ribosomal protein L21 OS=Gordonia sputi NBRC 100414 GN=rplU PE=3 SV=1;50S ribosomal protein L21 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosomal prokaryotic L21 protein; cl00382
GOHSU54	CDS	GOHSU_54_00030	2937..3200	case	50S ribosomal protein L27 [hamap]	rpmA [hamap]	MF_00539:50S ribosomal protein L27:rpmA 	ROP_10340 :50S ribosomal protein L27 : rpmA : : Rhodococcus opacus B4	50S ribosomal protein L27		H5UAV5_9ACTO 50S ribosomal protein L27 OS=Gordonia terrae NBRC 100016 GN=rpmA PE=3 SV=1;50S ribosomal protein L27 [Streptomyces clavuligeru s ATCC 27064];Ribosomal L27 protein; cl00359
GOHSU54	CDS	GOHSU_54_00040	3315..4796		probable GTP-binding protein Obg [nite]	 obg [nite]		RER_37920 :probable GTP-binding protein Obg : obg : : Rhodococcus erythropolis PR4	GTPase ObgE	obgE	F1YI62_9ACTO GTPase obg OS=Gordonia neofelifaecis N RRL B-59395 GN=obgE PE=3 SV=1;GTPase ObgE [Corynebacterium glutamicum ATCC 13032] ;IS fingerprint type 4-5;GTPase CgtA; Reviewed; PRK12296;essential GTPase; exhibits high exchange rate for GTP/GDP; associates with 50S ribosomal subunit; involved in regulation of chromosomal replication;GTP1/OBG; pfam01018;Obg GTPase; cd01898;G1 box;GTP/Mg2+ binding site [chemical binding];Switch I region;G2 box;G3 box;Switch II region;G4 box;G5 box;Domain of unknown function (DUF1967); cl07779
GOHSU54	CDS	GOHSU_54_00050	4789..5910		glutamate 5-kinase [hamap]	proB [hamap]	MF_00456:glutamate 5-kinase:proB 	ROP_10320 :glutamate 5-kinase : proB : 2.7.2.11 : Rhodococcus opacus B4	gamma-glutamyl kinase	proB	F1YI63_9ACTO Glutamate 5-kinase OS=Gordonia neofeli faecis NRRL B-59395 GN=proB PE=3 SV=1;gamma-glutamyl kinase [Corynebacterium glutamicum A TCC 13032];IS fingerprint type 4-5;catalyzes the formation of glutamate 5-phosphate from glutamate in proline biosynthesis;glutamate 5-kinase; TIGR01027;AAK_G5K_ProB: Glutamate-5-kinase (G5K) catalyzes glutamate-dependent ATP cleavage; G5K transfers the terminal phosphoryl group of ATP to the gamma-carboxyl group of glutamate, in the first and controlling step of proline (and, in mammals, ornithine)...; cd04242;nucleotide binding site [chemical binding];homotetrameric interface [polypeptide binding];putative phosphate binding site [ion binding];putative allosteric binding site;PUA domain; cl00607
GOHSU54	CDS	GOHSU_54_00060	5946..7664		putative acyl-CoA dehydrogenase [nite]			ROP_66650 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase		H5TWQ6_9ACTO Putative acyl-CoA dehydrogenase OS=Gor donia sputi NBRC 100414 GN=GOSPT_022_02610 PE=3 SV=1;acyl-CoA dehydrogenase [Nocardia farcinica IFM 1015 2];Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU54	CDS	GOHSU_54_00070	complement(7646..8572)		NAD-dependent deacetylase [hamap]	cobB [hamap]	MF_01121:NAD-dependent deacetylase:cobB 	RER_37900 :putative NAD-dependent deacetylase : : 3.5.1.- : Rhodococcus erythropolis PR4	silent information regulator protein sir2		G7H105_9ACTO NAD-dependent deacetylase OS=Gordonia araii NBRC 100433 GN=cobB PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: silent information regulator protein Sir2; KEGG: scl:sce3874 hypothetical protein;SIR2 superfamily of proteins includes silent information regulator 2 (Sir2) enzymes which catalyze NAD+-dependent protein/histone deacetylation, where the acetyl group from the lysine epsilon-amino group is transferred to the ADP-ribose moiety of NAD+; cl00195;NAD-dependent deacetylase; Provisional
GOHSU54	CDS	GOHSU_54_00080	complement(8632..9030)		L-ectoine synthase [hamap]	ectC [hamap]	MF_01255:L-ectoine synthase:ectC 	RER_37870 :probable ectoine synthase : ectC : 4.2.1.108 : Rhodococcus erythropolis PR4	L-ectoine synthase	ectC	D5UY69_TSUPD L-ectoine synthase OS=Tsukamurella pau rometabola (strain ATCC 8368 / DSM 20162 / JCM 10117 / NBR C 16120 / NCTC 13040) GN=ectC PE=3 SV=1;L-ectoine synthase [Mycobacterium abscessus ATCC 19 977];Genoscope sequence ID : MYCOBACTERIUM_ABSCESSUS_CHROMOSOME;Cupin domain; cl09118;N-acetyldiaminobutyrate dehydratase; catalyzes the formation of the osmoprotectant ecotoine from gamma-N-acetyl-alpha,gamma-diaminobutyric acid
GOHSU54	CDS	GOHSU_54_00090	complement(9027..10319)		probable diaminobutyrate--2-oxoglutarate transaminase [nite]	 ectB [nite]		RER_37860 :probable diaminobutyrate--2-oxoglutarate transaminase : ectB : 2.6.1.76 : Rhodococcus erythropolis PR4	diaminobutyrate--2-oxoglutarate aminotransferase	ectB	F1YI65_9ACTO Diaminobutyrate--2-oxoglutarate aminot ransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 07898 PE=3 SV=1;diaminobutyrate--2-oxoglutarate aminotransferase [N ocardia farcinica IFM 10152];catalyzes the reversible formation of diaminobutyrate and 2-oxoglutarate from glutamate and L-aspartic beta-semialdehyde;diaminobutyrate--2-oxoglutarate aminotransferase; Validated;Acetyl ornithine aminotransferase family. This family belongs to pyridoxal phosphate (PLP)-dependent aspartate aminotransferase superfamily (fold I). The major groups in this CD correspond to ornithine aminotransferase, acetylornithine aminotransferase; cd00610;inhibitor-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU54	CDS	GOHSU_54_00100	complement(10329..10856)		probable L-2,4-diaminobutyric acid acetyltransferase [nite]	 ectA [nite]		RER_37850 :probable L-2,4-diaminobutyric acid acetyltransferase : ectA : 2.3.1.178 : Rhodococcus erythropolis PR4	L-2,4-diaminobutyric acid acetyltransferase	ectA2	H6MXX4_GORPV L-2,4-diaminobutyric acid acetyltransf erase EctA OS=Gordonia polyisoprenivorans (strain DSM 4426 6 / VH2) GN=ectA PE=4 SV=1;L-2,4-diaminobutyric acid acetyltransferase [Strept omyces clavuligerus ATCC 27064];KEGG: Glycine, serine and threonine metabolism Metabolic pathways. COG: COG0454 Histone acetyltransferase HPA2 and related acetyltransferases.;ribosomal-protein-alanine acetyltransferase; TIGR01575;N-Acyltransferase superfamily: Various enzymes that characteristically catalyze the transfer of an acyl group to a substrate; cd04301;Coenzyme A binding pocket [chemical binding]
GOHSU54	CDS	GOHSU_54_00110	11015..11581		hypothetical protein [nite]			ROP_65690 :hypothetical protein : : : Rhodococcus opacus B4	N-6 adenine-specific DNA methylase		F1YI68_9ACTO Methyltransferase OS=Gordonia neofelif aecis NRRL B-59395 GN=SCNU_07913 PE=4 SV=1;N6-adenine-specific methylase [Corynebacterium glut amicum ATCC 13032];IS fingerprint type 4-5;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU54	CDS	GOHSU_54_00120	11594..12082		phosphopantetheine adenylyltransferase [hamap]	coaD [hamap]	MF_00151:phosphopantetheine adenylyltransferase:coaD 	ROP_65700 :phosphopantetheine adenylyltransferase : coaD : 2.7.7.3 : Rhodococcus opacus B4	phosphopantetheine adenylyltransferase	coaD	F1YI69_9ACTO Phosphopantetheine adenylyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=coaD PE=3 SV=1;phosphopantetheine adenylyltransferase [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;phosphopantetheine adenylyltransferase; Provisional; PRK00168;Catalyzes the conversion of ATP and pantetheine 4'-phosphate to diphosphate and 3'-dephospho-coA;Phosphopantetheine adenylyltransferase; cd02163;(T/H)XGH motif
GOHSU54	CDS	GOHSU_54_00130	12202..13023		conserved hypothetical protein [nite]			RER_24280 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YI70_9ACTO Cell division initiation protein-like protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_079 23 PE=4 SV=1;unnamed protein product [Streptomyces bingchenggens is BCW- 1];COG3599 Cell division initiation protein;Plant ATP synthase F0; cl07975
GOHSU54	CDS	GOHSU_54_00140	13020..13616		conserved hypothetical protein [nite]			RER_24290 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H5TSH8_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_217_00470 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Uncharacterized ACR, COG1399; cl00616
GOHSU54	CDS	GOHSU_54_00150	13613..14335		ribonuclease III [hamap]	rnc [hamap]	MF_00104:ribonuclease III:rnc 	ROP_65740 :ribonuclease III : rnc : 3.1.26.3 : Rhodococcus opacus B4	ribonuclease 3	rnc	F1YI72_9ACTO Ribonuclease 3 OS=Gordonia neofelifaec is NRRL B-59395 GN=rnc PE=3 SV=1;rnc gene product [Gordonia polyisoprenivorans VH2];RIBOc. Ribonuclease III C terminal domain. This group consists of eukaryotic, bacterial and archeal ribonuclease III (RNAse III) proteins. RNAse III is a double stranded RNA-specific endonuclease. Prokaryotic RNAse III is important in...; cd00593;dimerization interface [polypeptide binding];metal binding site [ion binding];Double-stranded RNA binding motif. Binding is not sequence specific but is highly specific for double stranded RNA. Found in a variety of proteins including dsRNA dependent protein kinase PKR, RNA helicases, Drosophila staufen protein, E. coli RNase III; cd00048
GOHSU54	CDS	GOHSU_54_00160	14328..15206		formamidopyrimidine-DNA glycosylase [hamap]	mutM [hamap]	MF_00103:formamidopyrimidine-DNA glycosylase:mutM 	RER_24320 :formamidopyrimidine-DNA glycosylase/AP lyase : mutM : 3.2.2.23/4.2.99.18 : Rhodococcus erythropolis PR4	formamidopyrimidine-DNA glycosylase		F1YI73_9ACTO Formamidopyrimidine-DNA glycosylase OS =Gordonia neofelifaecis NRRL B-59395 GN=mutM PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];formamidopyrimidine/5-formyluracil/ 5-hydroxymethyluracil DNA glycosylase; Validated;KEGG: csa:Csal_2969 DNA-(apurinic or apyrimidinic site) lyase / formamidopyrimidine-DNA glycosylase; TIGRFAM: formamidopyrimidine-DNA glycosylase; PFAM: formamidopyrimidine-DNA glycosylase catalytic domain protein; zinc finger Fpg domain protein; DNA glycosylase/AP lyase, H2TH DNA-binding;N-terminal domain of Escherichia coli Fpg1/MutM and related bacterial DNA glycosylases; cd08966;DNA binding site [nucleotide binding];catalytic residue [active];H2TH interface [polypeptide binding];putative catalytic residues [active];turnover-facilitating residue;intercalation triad [nucleotide binding];8OG recognition residue [nucleotide binding];putative reading head residues;Formamidopyrimidine-DNA glycosylase H2TH domain; pfam06831;Zinc finger found in FPG and IleRS; pfam06827
GOHSU54	CDS	GOHSU_54_00170	complement(15358..15654)		hypothetical protein [migap]				hypothetical protein		D0L607_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_4351 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU54	CDS	GOHSU_54_00180	16304..17161		oxidoreductase [nite]			MLP_01990 :oxidoreductase : : : Microlunatus phosphovorus NM-1	extradiol ring-cleavage dioxygenase, class III enzyme subunit B		F9VRG8_9ACTO Putative oxidoreductase OS=Gordonia al kanivorans NBRC 16433 GN=GOALK_026_00130 PE=4 SV=1;extradiol ring-cleavage dioxygenase, class III enzy me subunit B [Amycolicicoccus subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;The Class III extradiol dioxygenase, 4,5-DOPA Dioxygenase, catalyzes the incorporation of both atoms of molecular oxygen into 4,5-dihydroxy-phenylalanine; cd07363;putative active site [active];metal binding site [ion binding]
GOHSU54	CDS	GOHSU_54_00190	17170..18006		hypothetical protein [nite]			ROP_70460 :hypothetical protein : : : Rhodococcus opacus B4	hydrolase		F6EEQ2_AMYSD Beta-lactamase domain protein OS=Amyco licicoccus subflavus (strain DSM 45089 / DQS3-9A1) GN=AS9A _2405 PE=4 SV=1;hydrolase [Bacillus cereus ATCC 14579];Metallo-beta-lactamase superfamily; cl00446
GOHSU54	CDS	GOHSU_54_00200	complement(18014..18745)		hypothetical protein [nite]			KRH_19750 :hypothetical protein : : : Kocuria rhizophila DC2201	GMP synthase - Glutamine amidotransferase domain		D0L372_GORB4 GMP synthase-Glutamine amidotransferas e domain-like protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_3824 P E=4 SV=1;GMP synthase - Glutamine amidotransferase domain [M icrobacterium testaceum StLB037];Subgroup of proteins having the Type 1 glutamine amidotransferase (GATase1) domain; cd01741;catalytic triad [active]
GOHSU54	CDS	GOHSU_54_00210	18817..19254		conserved hypothetical protein [nite]			RER_24330 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	OsmC family protein		D0LAL3_GORB4 OsmC family protein OS=Gordonia bronch ialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 106 67) GN=Gbro_2074 PE=4 SV=1;OsmC family protein [Isoptericola variabilis 225];PFAM: Peroxiredoxin, OsmC-like protein; KEGG: xce:Xcel_2839 OsmC family protein;OsmC-like protein; cl00767
GOHSU54	CDS	GOHSU_54_00220	19282..19551	case	acylphosphatase [hamap]	yccX [hamap]	MF_01450:acylphosphatase:yccX 	RER_24360 :acylphosphatase : acyP : 3.6.1.7 : Rhodococcus erythropolis PR4	acylphosphatase		F1YI80_9ACTO Acylphosphatase OS=Gordonia neofelifae cis NRRL B-59395 GN=SCNU_07973 PE=3 SV=1;acylphosphatase [Corynebacterium glutamicum ATCC 13 032];IS fingerprint type 4-5;Acylphosphatase; cl00551;catalyzes the hydrolysis of acylphosphate
GOHSU54	CDS	GOHSU_54_00230	19563..>22051	3x codon	chromosome partition protein SMC [nite]	 smc [nite]		ROP_65800 :chromosome partition protein SMC : smc : : Rhodococcus opacus B4	chromosome partition protein Smc	smc	F1YI81_9ACTO Chromosome partition protein Smc OS=Go rdonia neofelifaecis NRRL B-59395 GN=smc PE=3 SV=1;smc gene product [Gordonia polyisoprenivorans VH2];chromosome segregation protein SMC, common bacterial type; TIGR02168;Barmotin is a tight junction-associated protein expressed in rat epithelial cells which is thought to have an important regulatory role in tight junction barrier function.  Barmotin belongs to the SMC protein family. SMC proteins are large; cd03278;Walker A/P-loop;ATP binding site [chemical binding];SMC proteins Flexible Hinge Domain; cl05797
##GOHSU55	contig00331				19649
GOHSU55	CDS	GOHSU_55_00010	454..1152		hypothetical protein [nite]			TEH_10200 :hypothetical protein : : : Tetragenococcus halophilus NBRC 12172	ThiJ/PfpI domain-containing protein		B8NMI0_ASPFN Putative uncharacterized protein OS=As pergillus flavus (strain ATCC 200026 / FGSC A1120 / NRRL 3 357 / JCM 12722 / SRRC 167) GN=AFLA_124160 PE=4 SV=1;unnamed protein product [Tsukamurella paurometabola DSM 20162];type strain of Tsukamurella paurometabola;Putative intracellular protease/amidase [General function prediction only]; COG0693;PFAM: ThiJ/PfpI domain protein; KEGG:  intracellular protease/amidase;Type 1 glutamine amidotransferase (GATase1)-like domain found in proteins similar to Escherichia coli Hsp31 protein; cd03141;potential catalytic triad [active];conserved cys residue [active]
GOHSU55	CDS	GOHSU_55_00020	complement(1176..1331)		conserved hypothetical protein [nite]			RER_12000 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		A1TCH0_MYCVP Putative uncharacterized protein OS=My cobacterium vanbaalenii (strain DSM 7251 / PYR-1) GN=Mvan_ 4093 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3]
GOHSU55	CDS	GOHSU_55_00030	complement(1382..1594)		hypothetical protein [nite]			ROP_51950 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		Q745Q9_MYCPA Putative uncharacterized protein OS=My cobacterium paratuberculosis GN=MAP_0257 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU55	CDS	GOHSU_55_00040	complement(1690..2775)		hypothetical protein [nite]			KRH_13230 :hypothetical protein : : : Kocuria rhizophila DC2201	acyl-CoA dehydrogenase		H0JVQ2_9NOCA Acyl-CoA dehydrogenase OS=Rhodococcus pyridinivorans AK37 GN=AK37_18828 PE=4 SV=1;acyl-CoA dehydrogenase [Oligotropha carboxidovorans OM5];type strain of Oligotropha carboxidovorans;Acyl-CoA dehydrogenase; cl09933
GOHSU55	CDS	GOHSU_55_00050	complement(2772..3533)		putative ABC transporter permease protein [nite]			KRH_13200 :putative ABC transporter permease protein : : : Kocuria rhizophila DC2201	binding-protein-dependent transporters inner membrane component		E4W9D1_RHOE1 Putative ABC transporter integral memb rane subunit OS=Rhodococcus equi (strain 103S) GN=REQ_4084 0 PE=3 SV=1;binding-protein-dependent transporters inner membra ne component [Methylobacterium chloromethanicum CM4];type strain of methylobacterium chloromethanicum;PFAM: binding-protein-dependent transporters inner membrane component; KEGG: mex:Mext_3486 binding-protein-dependent transport systems inner membrane component;Transmembrane subunit (TM) found in Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters which generally bind type 2 PBPs. These types of transporters consist of a PBP, two TMs, and two cytoplasmic ABC ATPase subunits, and...; cl00427
GOHSU55	CDS	GOHSU_55_00060	3688..3882		not assigned						
GOHSU55	tRNA	GOHSU_55_t0010	complement(3938..4022)		not assigned						identified by tRNAscan-SE; putative
GOHSU55	CDS	GOHSU_55_00070	4106..5071		putative oxidoreductase [nite]			KSE_38740 :putative oxidoreductase : : : Kitasatospora setae NBRC 14216	putative quinone oxidoreductase		H5U5K2_9ACTO Putative oxidoreductase OS=Gordonia sp uti NBRC 100414 GN=GOSPT_118_00870 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];putative NAD(P)H quinone oxidoreductase, PIG3 family; TIGR02824;PIG3 p53-inducible quinone oxidoreductase; cd05276;NAD(P) binding site [chemical binding]
GOHSU55	CDS	GOHSU_55_00080	complement(5081..5572)		putative MarR family transcriptional regulator [nite]			RER_48720 :putative MarR family transcriptional regulator : : : Rhodococcus erythropolis PR4	MarR family transcriptional regulator		F1YPA4_9ACTO Regulatory protein MarR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_18767 PE=4 SV=1;MarR family transcriptional regulator [Thermobispor a bispora DSM 43833];type strain of Thermobispora bispora;PFAM: regulatory protein MarR; SMART: regulatory protein MarR; KEGG: scl:sce6390 MarR family transcriptional regulator;helix_turn_helix multiple antibiotic resistance protein; smart00347;Helix-turn-helix domains; cl00088
GOHSU55	CDS	GOHSU_55_00090	complement(5689..6924)		conserved hypothetical protein [nite]			RER_02620 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	cysteine desulfurase		F1YPA5_9ACTO Cysteine desulfurase family protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18772 PE=3 SV =1;cysteine desulfurase [Amycolatopsis mediterranei U3 2];Selenocysteine lyase/Cysteine desulfurase [Posttranslational modification, protein turnover, chaperones]; COG0520;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU55	CDS	GOHSU_55_00100	6923..7504		hypothetical protein [nite]			ROP_39790 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YPA6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_18777 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];Protein of unknown function (DUF2587); pfam10759
GOHSU55	CDS	GOHSU_55_00110	7536..8393		putative ABC transporter permease protein [nite]			ROP_39780 :putative ABC transporter permease protein : : : Rhodococcus opacus B4	ABC transporter		F1YPA7_9ACTO ABC-2 type transporter OS=Gordonia neo felifaecis NRRL B-59395 GN=SCNU_18782 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: ABC transporter; KEGG: dda:Dd703_2684 ABC-2 type transporter;ABC-2 type transporter; cl11417
GOHSU55	CDS	GOHSU_55_00120	8403..9212		putative ABC transporter ATP-binding protein [nite]			ROP_39770 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	teichoic acids export ATP-binding protein TagH		F1YPA8_9ACTO ABC transporter-like protein OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_18787 PE=4 SV=1;teichoic acids export ATP-binding protein TagH [Str eptomyces sp. AA4];ABC-type polysaccharide/polyol phosphate transport system, ATPase component [Carbohydrate transport and metabolism / Cell envelope biogenesis, outer membrane]; COG1134;ABC_KpsT_Wzt   The KpsT/Wzt ABC transporter subfamily is involved in extracellular polysaccharide export.  Among the variety of membrane-linked or extracellular polysaccharides excreted by bacteria, only capsular polysaccharides, lipopolysaccharides, and...; cd03220;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU55	CDS	GOHSU_55_00130	9209..10129		galactofuranosyltransferase [nite]	 glfT [nite]		ROP_39760 :galactofuranosyltransferase : glfT : 2.4.1.- : Rhodococcus opacus B4	glycosyl transferase family protein		F1YPA9_9ACTO Glycosyl transferase family 2 protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_18792 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];Predicted glycosyltransferases [General function prediction only];PFAM: glycosyl transferase, family 2; KEGG: nfa:nfa2170 putative glycosyltransferase;Glycosyltransferase family A (GT-A) includes diverse families of glycosyl transferases with a common GT-A type structural fold; cl11394
GOHSU55	CDS	GOHSU_55_00140	10146..11030		hypothetical protein [nite]			ROP_47810 :hypothetical protein : : : Rhodococcus opacus B4	putative F420-dependent oxidoreductase, Rv2161c family		H5UGU8_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_076_00490 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Luciferase-like monooxygenase; TIGRFAM: probable F420-dependent oxidoreductase, Rv2161c family;probable F420-dependent oxidoreductase, Rv2161c family; TIGR03619;Flavin-utilizing monoxygenases; cl07892
GOHSU55	CDS	GOHSU_55_00150	complement(11301..11819)		hypothetical membrane protein [nite]			ROP_39540 :hypothetical membrane protein : : : Rhodococcus opacus B4	GtrA family protein		H5UGV3_9ACTO Putative uncharacterized protein OS=Go rdonia terrae NBRC 100016 GN=GOTRE_076_00550 PE=4 SV=1;GtrA family protein [Mesorhizobium opportunistum WS M2075];Signal predicted by SignalP 3.0 HMM;COGs: COG2246 membrane protein; InterPro IPR007267; KEGG: mlo:mlr4738 hypothetical protein; PFAM: GtrA family protein; SPTR: GtrA family protein; PFAM: GtrA-like protein;GtrA-like protein; cl00971
GOHSU55	CDS	GOHSU_55_00160	11923..13125		hypothetical protein [nite]			ROP_22260 :hypothetical protein : : : Rhodococcus opacus B4	lipase		F1YME6_9ACTO Secretory lipase OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_15154 PE=4 SV=1;lipase [Rhodococcus equi 103S];Secretory lipase; pfam03583;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU55	CDS	GOHSU_55_00170	complement(13128..13592)		putative Usp family protein [nite]			ROP_66620 :putative Usp family protein : : : Rhodococcus opacus B4	universal stress protein UspA-like protein		F1YME7_9ACTO Universal stress protein family protei n OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15159 PE= 4 SV=1;universal stress protein UspA-like protein [Arthrob acter phenanthrenivorans Sphe3];PFAM: Universal stress protein family;Usp: Universal stress protein family. The universal stress protein Usp is a small cytoplasmic bacterial protein whose expression is enhanced when the cell is exposed to stress agents. Usp enhances the rate of cell survival during prolonged exposure to...; cd00293;Ligand Binding Site [chemical binding]
GOHSU55	CDS	GOHSU_55_00180	13791..15209		putative oxidoreductase [nite]			RER_02350 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	FAD linked oxidoreductase		F1YME8_9ACTO Oxidoreductase OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_15164 PE=4 SV=1;FAD linked oxidoreductase [Rhodococcus jostii RHA1] ;FAD/FMN-containing dehydrogenases [Energy production and conversion]; COG0277;FAD binding domain; pfam01565;D-arabinono-1,4-lactone oxidase; cl04370
GOHSU55	CDS	GOHSU_55_00190	15267..16028		putative oxidoreductase [nite]			ROP_39520 :putative oxidoreductase : : : Rhodococcus opacus B4	short-chain dehydrogenase of unknown substrate specificity		F1YME9_9ACTO Short chain dehydrogenase OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15169 PE=4 SV=1;short-chain dehydrogenase of unknown substrate spec ificity [Saccharomonospora marina XMU15];PFAM: short chain dehydrogenase;Short-chain dehydrogenases of various substrate specificities [General function prediction only]; COG0300;classical (c) SDRs; cd05233;NAD(P) binding site [chemical binding]
GOHSU55	CDS	GOHSU_55_00200	16029..17990		arabinosyltransferase AftA [nite]	 aftA [nite]		RER_02330 :arabinosyltransferase AftA : aftA : 2.4.2.- : Rhodococcus erythropolis PR4	arabinofuranosyltransferase AftA	aftA	F1YMF0_9ACTO Putative transmembrane protein OS=Gord onia neofelifaecis NRRL B-59395 GN=SCNU_15174 PE=4 SV=1;aftA gene product [Gordonia polyisoprenivorans VH2] ;Arabinofuranosyltransferase N terminal; pfam12250;Arabinofuranosyltransferase A C terminal; pfam12249
GOHSU55	CDS	GOHSU_55_00210	17996..18742		putative beta-lactamase [nite]			RER_48510 :putative beta-lactamase : : 3.5.2.6 : Rhodococcus erythropolis PR4	beta-lactamase domain-containing protein		F1YMF1_9ACTO Beta-lactamase domain-containing prote in OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_15179 PE =4 SV=1;beta-lactamase domain-containing protein [Streptomy ces sp. SPB78];Metallo-beta-lactamase superfamily; cl00446
GOHSU55	CDS	GOHSU_55_00220	18977..19366		not assigned						B2WIF5_PYRTR Glycosyl hydrolase OS=Pyrenophora trit ici-repentis (strain Pt-1C-BFP) GN=PTRG_09764 PE=4 SV=1
##GOHSU56	contig00341				18636
GOHSU56	CDS	GOHSU_56_00010	<1..773	3x codon frame	hypothetical protein [migap]				hypothetical protein		F1YM93_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15509 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: hypothetical protein
GOHSU56	CDS	GOHSU_56_00020	770..1555		putative glycerophosphoryl diester phosphodiesterase [nite]	 glpQ [nite]		RER_01710 :putative glycerophosphoryl diester phosphodiesterase : glpQ : 3.1.4.46 : Rhodococcus erythropolis PR4	glycerophosphoryl diester phosphodiesterase		G7H202_9ACTO Glycerophosphoryl diester phosphodiest erase OS=Gordonia araii NBRC 100433 GN=glpQ PE=4 SV=1;glycerophosphoryl diester phosphodiesterase [Strept omyces sp. Mg1];Glycerophosphodiester phosphodiesterase domain of uncharacterized bacterial glycerophosphodiester phosphodiesterases; cd08582;Glycerophosphoryl diester phosphodiesterase family; pfam03009;putative active site [active];catalytic site [active];putative metal binding site [ion binding]
GOHSU56	CDS	GOHSU_56_00030	1597..2565		hypothetical protein [nite]			ROP_39030 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YM91_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_15499 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU56	CDS	GOHSU_56_00040	complement(2581..3519)		putative LytR family regulatory protein [nite]			RER_01830 :putative LytR family regulatory protein : : : Rhodococcus erythropolis PR4	LytR family transcriptional regulator		F1YM89_9ACTO Cell envelope-like function transcript ional attenuator OS=Gordonia neofelifaecis NRRL B-59395 GN =SCNU_15489 PE=4 SV=1;LytR family transcriptional regulator [Saccharopoly spora erythraea NRRL 2338];Cell envelope-related transcriptional attenuator domain; cl00581
GOHSU56	CDS	GOHSU_56_00050	complement(4798..5496)		putative 2,3-dihydroxybenzoate-AMP ligase [nite]			ROP_34340 :putative 2,3-dihydroxybenzoate-AMP ligase : : 6.3.2.- : Rhodococcus opacus B4	2,3-dihydroxybenzoate-AMP ligase		G7H6U2_9ACTO Putative 2,3-dihydroxybenzoate-AMP lig ase OS=Gordonia araii NBRC 100433 GN=GOARA_078_00160 PE=4 SV=1;2,3-dihydroxybenzoate-AMP ligase [Rhodococcus opacu s B4];Peptide arylation enzymes [Secondary metabolites biosynthesis, transport, and catabolism]; COG1021;AMP-binding enzyme; cl15778
GOHSU56	CDS	GOHSU_56_00060	complement(5647..6942)		conserved hypothetical membrane protein [nite]			RER_32330 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	esterase		C3JWN8_RHOER Integral membrane protein OS=Rhodococc us erythropolis SK121 GN=RHOER0001_6532 PE=4 SV=1;esterase [Rhodococcus jostii RHA1];Putative esterase; pfam00756;Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031
GOHSU56	CDS	GOHSU_56_00070	complement(7186..8589)		alanyl-tRNA synthetase [nite]	 alaS [nite]		BBR47_19020 :alanyl-tRNA synthetase : alaS : 6.1.1.7 : Brevibacillus brevis NBRC 100599	alanyl-tRNA synthetase	alaS	C8XC46_NAKMY Alanine--tRNA ligase OS=Nakamurella mu ltipartita (strain ATCC 700099 / DSM 44233 / JCM 9543 / Y- 104) GN=Namu_5171 PE=4 SV=1;alanyl-tRNA synthetase [Bacillus licheniformis ATCC 14580];Catalyzes a two-step reaction, first charging an alanyl molecule by linking its carboxyl group to the alpha-phosphate of ATP, followed by transfer of the aminoacyl-adenylate to its tRNA;alanyl-tRNA synthetase; Reviewed; PRK00252;Alanyl-tRNA synthetase (AlaRS) class II core catalytic domain. AlaRS is a homodimer. It is responsible for the attachment of alanine to the 3' OH group of ribose of the appropriate tRNA. This domain is primarily responsible for ATP-dependent formation of...; cd00673;motif 1;motif 2;motif 3
GOHSU56	CDS	GOHSU_56_00080	8752..10896		hypothetical membrane protein [nite]			RER_32310 :hypothetical membrane protein : : : Rhodococcus erythropolis PR4	predicted protein		C3JWN6_RHOER Putative membrane protein OS=Rhodococc us erythropolis SK121 GN=RHOER0001_6531 PE=4 SV=1;predicted protein [Streptomyces sp. AA4];Uncharacterized conserved protein (DUF2156); cl12090
GOHSU56	CDS	GOHSU_56_00090	10896..11963		hypothetical membrane protein [nite]			ROP_40820 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		Q1B9T8_MYCSS Putative uncharacterized protein OS=My cobacterium sp. (strain MCS) GN=Mmcs_2238 PE=4 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155]
GOHSU56	CDS	GOHSU_56_00100	12139..12609		not assigned						F7CG30_HORSE Uncharacterized protein (Fragment) OS= Equus caballus GN=CCDC88C PE=4 SV=1
GOHSU56	CDS	GOHSU_56_00110	12758..13954		putative CPA1 family transporter [nite]			YM304_18570 :putative CPA1 family transporter : : : Acidimicrobidae bacterium YM16-304	sodium/hydrogen exchanger		F9VS16_9ACTO Putative CPA1 family transporter OS=Go rdonia alkanivorans NBRC 16433 GN=GOALK_030_00020 PE=4 SV= 1;sodium/hydrogen exchanger [Pseudonocardia dioxanivo rans CB1190];PFAM: Cation/H+ exchanger; KEGG: mjl:Mjls_4930 sodium/hydrogen exchanger
GOHSU56	CDS	GOHSU_56_00120	13954..14757		hypothetical protein [migap]				hypothetical protein		D3PW69_STANL Membrane protein-like protein OS=Stack ebrandtia nassauensis (strain DSM 44728 / NRRL B-16338 / N BRC 102104 / LLR-40K-21) GN=Snas_1522 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];Protease prsW family; cl15823
GOHSU56	CDS	GOHSU_56_00130	14754..15614		hypothetical protein [migap]				hypothetical protein		E8S4T2_MICSL Putative uncharacterized protein OS=Mi cromonospora sp. (strain L5) GN=ML5_4968 PE=4 SV=1;unnamed protein product [Salinispora tropica CNB- 4 40]
GOHSU56	CDS	GOHSU_56_00140	15619..17190		asparagine synthetase [glutamine-hydrolyzing] [nite]			PH1102 :asparagine synthetase [glutamine-hydrolyzing] : : 6.3.5.4 : Pyrococcus horikoshii OT3	asparagine synthetase B	asnB	A4X6Q9_SALTO Asparagine synthetase OS=Salinispora t ropica (strain ATCC BAA-916 / DSM 44818 / CNB-440) GN=Stro p_2108 PE=4 SV=1;asnB gene product [Flavobacterium johnsoniae UW101] ;asparagine synthetase B; Provisional; PRK09431;functions in asparagine biosynthesis; converts glutamine, aspartate, ATP, and water to glutamate, asparagine, pyrophosphate and AMP;Glutamine amidotransferases class-II (GATase) asparagine synthase_B type.  Asparagine synthetase B catalyses the ATP-dependent conversion of aspartate to asparagine. This enzyme is a homodimer, with each monomer composed of a  glutaminase domain and a...; cd00712;dimer interface [polypeptide binding];The C-terminal domain of Asparagine Synthase B. This domain is always found associated n-terminal amidotransferase domain. Family members that contain this domain catalyse the conversion of aspartate to asparagine. Asparagine synthetase B  catalyzes the...; cd01991;Ligand Binding Site [chemical binding];Molecular Tunnel
GOHSU56	CDS	GOHSU_56_00150	17278..18159		AraC family transcriptional regulator [nite]			SLG_10630 :AraC family transcriptional regulator : : : Sphingobium sp. SYK-6	AraC family transcriptional regulator		G6X1Q8_MYCAB AraC family transcriptional regulator OS=Mycobacterium abscessus 47J26 GN=MAB47J26_06890 PE=4 SV =1;AraC family transcriptional regulator [Pseudomonas putida W619];PFAM: helix-turn-helix- domain containing protein AraC type; KEGG: rle:RL1967 putative AraC family transcriptional regulator;AraC-type DNA-binding domain-containing proteins [Transcription]; COG2207;Bacterial regulatory helix-turn-helix proteins, AraC family; pfam00165;helix_turn_helix, arabinose operon control protein; smart00342
GOHSU56	CDS	GOHSU_56_00160	complement(18180..>18636)	3x codon frame	hypothetical protein [migap]	lpqJ [migap]			hypothetical protein	lpqJ	D5PB84_9MYCO LpqJ protein OS=Mycobacterium parascro fulaceum ATCC BAA-614 GN=lpqJ PE=4 SV=1;LpqJ [Mycobacterium avium subsp. paratuberculosis K - 10]
##GOHSU57	contig00351				17238
GOHSU57	CDS	GOHSU_57_00010	complement(<1..851)	3x codon	fructose 1,6-bisphosphatase class II [nite]	 fbp [nite]		RER_42390 :fructose 1,6-bisphosphatase class II : fbp : 3.1.3.11 : Rhodococcus erythropolis PR4	fructose 1,6-bisphosphatase II	glpX	F1YFG9_9ACTO Fructose-1,6-bisphosphatase OS=Gordoni a neofelifaecis NRRL B-59395 GN=glpX PE=3 SV=1;fructose 1,6-bisphosphatase II [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;type II fructose 1,6-bisphosphatae; in Escherichia coli this protein forms a dimer and binds manganese;Bacterial fructose-1,6-bisphosphatase, glpX-encoded. A dimeric enzyme dependent on Mg(2+). glpX-encoded FPBase (FBPase class II) differs from other members of the inositol-phosphatase superfamily by permutation of secondary structure elements. The core...; cd01516;putative active site [active]
GOHSU57	CDS	GOHSU_57_00020	892..1518		hypothetical protein [nite]			ROP_59280 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YFH0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02857 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.602) with cleavage site probability 0.261 at residue 38;Protein of unknown function (DUF4245); pfam14030
GOHSU57	CDS	GOHSU_57_00030	complement(1547..1774)		exodeoxyribonuclease VII small subunit [hamap]	xseB [hamap]	MF_00337:exodeoxyribonuclease VII small subunit:xseB 	ROP_59290 :exodeoxyribonuclease VII small subunit : xseB : 3.1.11.6 : Rhodococcus opacus B4	exodeoxyribonuclease VII small subunit	xseB	F1YFH1_9ACTO Exodeoxyribonuclease 7 small subunit O S=Gordonia neofelifaecis NRRL B-59395 GN=xseB PE=3 SV=1;exodeoxyribonuclease VII small subunit [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;catalyzes the bidirectional exonucleolytic cleavage of DNA;Exonuclease VII small subunit; cl00750
GOHSU57	CDS	GOHSU_57_00040	complement(1771..3015)		exodeoxyribonuclease VII large subunit [hamap]	xseA [hamap]	MF_00378:exodeoxyribonuclease VII large subunit:xseA 	ROP_59300 :exodeoxyribonuclease VII large subunit : xseA : 3.1.11.6 : Rhodococcus opacus B4	exodeoxyribonuclease VII large subunit	xseA	F1YFH2_9ACTO Exodeoxyribonuclease 7 large subunit O S=Gordonia neofelifaecis NRRL B-59395 GN=xseA PE=3 SV=1;exodeoxyribonuclease VII large subunit [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;bidirectionally degrades single-stranded DNA into large acid-insoluble oligonucleotides;exodeoxyribonuclease VII large subunit; Reviewed; PRK00286;ExoVII_LU_OBF: A subfamily of OB folds corresponding to the N-terminal OB-fold domain of Escherichia coli exodeoxyribonuclease VII (ExoVII) large subunit. E. coli ExoVII is composed of two non-identical subunits. E. coli ExoVII is a...; cd04489;generic binding surface II;generic binding surface I
GOHSU57	CDS	GOHSU_57_00050	complement(3033..3767)		conserved hypothetical protein [nite]			RER_42350 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YFH3_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02872 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: cvi:CV_3617 hypothetical protein
GOHSU57	CDS	GOHSU_57_00060	complement(3853..4086)	overlap	hypothetical protein [migap]				hypothetical protein		Q3JJ11_BURP1 Putative uncharacterized protein OS=Bu rkholderia pseudomallei (strain 1710b) GN=BURPS1710b_A1286 PE=4 SV=1;hypothetical protein BURPS1710b_A1286 [Burkholderia pseudomallei 1710b]
GOHSU57	CDS	GOHSU_57_00070	3982..4806	overlap	4-hydroxy-3-methylbut-2-enyl diphosphate reductase [hamap]	ispH [hamap]	MF_00191:4-hydroxy-3-methylbut-2-enyl diphosphate reductase:ispH 	RER_42340 :4-hydroxy-3-methylbut-2-enyl diphosphate reductase : ispH : 1.17.1.2 : Rhodococcus erythropolis PR4	4-hydroxy-3-methylbut-2-enyl diphosphate reductase	ispH	F9VSI9_9ACTO 4-hydroxy-3-methylbut-2-enyl diphospha te reductase OS=Gordonia alkanivorans NBRC 16433 GN=GOALK_ 033_01110 PE=3 SV=1;4-hydroxy-3-methylbut-2-enyl diphosphate reductase [Streptomyces clavuligerus ATCC 27064];KEGG: Metabolic pathways Terpenoid backbone biosynthesis. COG: COG0761 Penicillin tolerance protein.;LytB protein; cl00507;(E)-4-hydroxy-3-methyl-but-2-enyl pyrophosphate reductase (IPP and DMAPP forming); TIGR00216
GOHSU57	CDS	GOHSU_57_00080	complement(4871..6208)		probable S8 family peptidase [nite]			RER_18890 :probable S8 family peptidase : : 3.4.21.- : Rhodococcus erythropolis PR4	subtilase		F1YFH5_9ACTO Peptidase S8/S53 subtilisin kexin sedo lisin OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_02882 PE=3 SV=1;unnamed protein product [Mycobacterium smegmatis st r. MC2 155];identified by match to protein family HMM PF00082;Peptidase S8 family domain in Autotransporter serine proteases; cd04848;putative active site [active];catalytic triad [active]
GOHSU57	CDS	GOHSU_57_00090	complement(6208..7641)		hypothetical membrane protein [nite]			ROP_62220 :hypothetical membrane protein : : : Rhodococcus opacus B4	secretion protein snm4		F1YFH6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02887 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: secretion protein snm4; PFAM: protein of unknown function DUF571; KEGG: ret:RHE_CH02708 hypothetical protein;Ubiquitin-like proteins; cl00155
GOHSU57	CDS	GOHSU_57_00100	7755..11696		putative FtsK/SpoIIIE family protein [nite]			ROP_62230 :putative FtsK/SpoIIIE family protein : : : Rhodococcus opacus B4	cell division protein FtsK		D0L7U0_GORB4 Cell divisionFtsK/SpoIIIE OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NC TC 10667) GN=Gbro_1683 PE=4 SV=1;cell division protein FtsK [Catenulispora acidiphil a DSM 44928];PFAM: cell divisionFtsK/SpoIIIE; SMART: ATPase AAA; KEGG: stp:Strop_0474 cell divisionFtsK/SpoIIIE;type VII secretion protein EccCa; TIGR03924;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;type VII secretion protein EccCb; TIGR03925
GOHSU57	CDS	GOHSU_57_00110	11738..12892		hypothetical protein [migap]				hypothetical protein		F1YFH7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02902 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];type VII secretion-associated protein, Rv3446c family, C-terminal domain; TIGR03931
GOHSU57	CDS	GOHSU_57_00120	complement(12917..13450)		conserved hypothetical protein [nite]			RER_10710 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	YceI domain-containing protein		H5TTG4_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_239_00190 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];YceI-like domain; cl01001
GOHSU57	CDS	GOHSU_57_00130	13640..14083		50S ribosomal protein L13 [nite]	 rplM [nite]		ROP_62300 :50S ribosomal protein L13 : rplM : : Rhodococcus opacus B4	50S ribosomal protein L13		H0R6L9_9ACTO 50S ribosomal protein L13 OS=Gordonia effusa NBRC 100432 GN=rplM PE=3 SV=1;50S ribosomal protein L13 [Streptomyces clavuligeru s ATCC 27064];Ribosomal protein L13.  Protein L13, a large ribosomal subunit protein, is one of five proteins required for an early folding intermediate of 23S rRNA in the assembly of the large subunit. L13 is situated on the bottom of the large subunit, near the...; cd00392;23S rRNA interface [nucleotide binding];L3 interface [polypeptide binding]
GOHSU57	CDS	GOHSU_57_00140	14080..14601		30S ribosomal protein S9 [nite]	 rpsI [nite]		RER_19050 :30S ribosomal protein S9 : rpsI : : Rhodococcus erythropolis PR4	30S ribosomal protein S9	rpsI	F1YFI3_9ACTO 30S ribosomal protein S9 OS=Gordonia n eofelifaecis NRRL B-59395 GN=rpsI PE=3 SV=1;30S ribosomal protein S9 [Corynebacterium glutamicu m ATCC 13032];IS fingerprint type 4-5;forms a direct contact with the tRNA during translation;Ribosomal protein S9/S16; cl00334
GOHSU57	CDS	GOHSU_57_00150	14617..15957		phosphoglucosamine mutase [nite]	 glmM [nite]		RER_19060 :phosphoglucosamine mutase : glmM : 5.4.2.10 : Rhodococcus erythropolis PR4	phosphoglucosamine mutase	glmM	D0L7U4_GORB4 Phosphoglucosamine mutase OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NC TC 10667) GN=glmM PE=3 SV=1;phosphoglucosamine mutase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;phosphoglucosamine mutase; Provisional; PRK14318;catalyzes the conversion of glucosamine-6-phosphate to glucosamine-1-phosphate;GlmM is a bacterial phosphoglucosamine mutase (PNGM) that belongs to the alpha-D-phosphohexomutase superfamily. It is required for the interconversion of glucosamine-6-phosphate and glucosamine-1-phosphate in the biosynthetic pathway of...; cd05802;substrate binding site [chemical binding];metal binding site [ion binding]
GOHSU57	CDS	GOHSU_57_00160	complement(16006..16500)		conserved hypothetical protein [nite]			RER_08720 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		G7GZ19_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_021_00810 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];START/RHO_alpha_C/PITP/Bet_v1/CoxG/CalC (SRPBCC) ligand-binding domain superfamily; cl14643
GOHSU57	CDS	GOHSU_57_00170	16561..16974		not assigned						G4FYL1_9GAMM Cointegrate resolution protein T OS=Rh odanobacter sp. 2APBS1 GN=R2APBS1DRAFT_2768 PE=4 SV=1
GOHSU57	CDS	GOHSU_57_00180	16992..>17238	3x codon	primosome assembly protein PriA [migap]	priA [migap]			primosome assembly protein PriA	priA	priA gene product [Roseobacter denitrificans OCh 11 4];type strain of Roseobacter denitrificans;binding of PriA to forked DNA starts the assembly of the primosome, also possesses 3'-5' helicase activity;primosome assembly protein PriA; Validated;DEAD-like helicases superfamily. A diverse family of proteins involved in ATP-dependent RNA or DNA unwinding. This domain contains the ATP-binding region; cd00046;ATP binding site [chemical binding]
##GOHSU58	contig00361				16719
GOHSU58	CDS	GOHSU_58_00010	<1..4610	3x codon frame	non-ribosomal peptide synthetase [nite]			ROP_48130 :non-ribosomal peptide synthetase : : : Rhodococcus opacus B4	non-ribosomal peptide synthetase		A0QNF1_MYCS2 Amino acid adenylation OS=Mycobacteriu m smegmatis (strain ATCC 700084 / mc(2)155) GN=MSMEG_0019 PE=4 SV=1;non-ribosomal peptide synthetase [Rhodococcus opacu s B4];peptide synthase; Provisional;AMP-binding enzyme; cl15778;Phosphopantetheine attachment site; cl09936
GOHSU58	CDS	GOHSU_58_00020	4607..9187		putative non-ribosomal peptide synthetase [nite]			ROP_48890 :putative non-ribosomal peptide synthetase : : : Rhodococcus opacus B4	non-ribosomal peptide synthetase		G8NSW7_GRAMM Amino acid adenylation domain protein OS=Granulicella mallensis (strain ATCC BAA-1857 / DSM 2313 7 / MP5ACTX8) GN=AciX8_1980 PE=4 SV=1;non-ribosomal peptide synthetase [Rhodococcus opacu s B4];peptide synthase; Provisional;AMP-binding enzyme; cl15778;Phosphopantetheine attachment site; cl09936;non-ribosomal peptide synthase domain TIGR01720
GOHSU58	CDS	GOHSU_58_00030	9255..10145		putative siderophore-interacting protein [nite]			RER_56770 :putative siderophore-interacting protein : : : Rhodococcus erythropolis PR4	siderophore-interacting protein		G7GSC1_9ACTO Putative siderophore-interacting prote in OS=Gordonia amarae NBRC 15530 GN=GOAMR_54_00060 PE=4 SV =1;siderophore-interacting protein [Arthrobacter arila itensis Re117];Siderophore-interacting protein [Inorganic ion transport and metabolism]; COG2375;identified by match to protein family PF04954. Match to protein domain PF08021. Possibly involved in removal of iron from iron-siderophore complexes;Siderophore interacting proteins share the domain structure of the ferredoxin reductase like family. Siderophores are produced in various bacteria (and some plants) to extract iron from hosts. Binding constants are high, so iron can be pilfered from...; cd06193;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding];NAD binding pocket [chemical binding]
GOHSU58	CDS	GOHSU_58_00040	complement(10142..11728)		2,3-dihydroxybenzoate-AMP ligase [nite]			ROP_48840 :2,3-dihydroxybenzoate-AMP ligase : : 6.3.2.- : Rhodococcus opacus B4	putative 2,3-dihydroxybenzoate-AMP ligase		H0REI6_9ACTO Putative 2,3-dihydroxybenzoate-AMP lig ase OS=Gordonia polyisoprenivorans NBRC 16320 GN=GOPIP_054 _00490 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Peptide arylation enzymes [Secondary metabolites biosynthesis, transport, and catabolism]; COG1021;AMP-binding enzyme; cl15778
GOHSU58	CDS	GOHSU_58_00050	11752..12411		isochorismatase [nite]			ROP_20360 :isochorismatase : : 3.3.2.1 : Rhodococcus opacus B4	2,3-dihydro-2,3-dihydroxybenzoate synthetasecomplex component)	entB	H6N1R7_GORPV Isochorismatase OS=Gordonia polyisopre nivorans (strain DSM 44266 / VH2) GN=GPOL_c23560 PE=4 SV=1 ;entB gene product [Xenorhabdus bovienii SS- 2004];Isochorismatase, also known as 2,3 dihydro-2,3 dihydroxybenzoate synthase, catalyses the conversion of isochorismate, in the presence of water, to 2,3-dihydroxybenzoate and pyruvate, via the hydrolysis of a vinyl ether, an uncommon reaction in biological...; cd01013;hydrophobic substrate binding pocket;Isochorismatase family; pfam00857;conserved cis-peptide bond
GOHSU58	CDS	GOHSU_58_00060	12408..13139		putative 2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase [nite]			KSE_59060 :putative 2,3-dihydro-2,3-dihydroxybenzoate dehydrogenase : : : Kitasatospora setae NBRC 14216	2,3-dihydroxybenzoate-2,3-dehydrogenase		H0REI4_9ACTO Putative 2,3-dihydro-2,3-dihydroxybenz oate dehydrogenase OS=Gordonia polyisoprenivorans NBRC 163 20 GN=GOPIP_054_00470 PE=4 SV=1;2,3-dihydroxybenzoate-2,3-dehydrogenase [Dickeya da dantii Ech703];catalyzes the formation of 2,3-dihydroxybenzoate from 2,3-dihydro-2,3-dihydroxybenzoate; involved in the biosynthesis of siderophores, enterobactin, bacillibactin or vibriobactin;3-ketoacyl-(acyl-carrier-protein) reductase; Validated; PRK05653;2,3 dihydro-2,3 dihydrozybenzoate dehydrogenases, classical (c) SDRs; cd05331;putative NAD(P) binding site [chemical binding]
GOHSU58	CDS	GOHSU_58_00070	13295..15055		hypothetical protein [migap]				hypothetical protein		H0RIF1_9ACTO Putative uncharacterized protein OS=Go rdonia polyisoprenivorans NBRC 16320 GN=GOPIP_074_00130 PE =4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU58	CDS	GOHSU_58_00080	15191..16456		putative glycosyltransferase [nite]			RER_17740 :putative glycosyltransferase : : 2.4.-.- : Rhodococcus erythropolis PR4	hypothetical protein		C0ZVU7_RHOE4 Putative glycosyltransferase OS=Rhodoc occus erythropolis (strain PR4 / NBRC 100887) GN=RER_17740 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF2029; KEGG: bja:blr3063 hypothetical protein;Protein of unknown function (DUF2029); cl09812
##GOHSU59	contig00371				15357
GOHSU59	CDS	GOHSU_59_00010	complement(<1..372)	codon	hypothetical protein [migap]				hypothetical protein		H5TZU7_9ACTO Putative uncharacterized protein (Frag ment) OS=Gordonia sputi NBRC 100414 GN=GOSPT_055_00010 PE= 4 SV=1;unnamed protein product [Propionibacterium acnes Ty peIA2 P.acn33]
GOHSU59	CDS	GOHSU_59_00020	complement(424..1482)		alcohol dehydrogenase (NADP+) [nite]	 adh [nite]		SLG_34750 :alcohol dehydrogenase (NADP+) : adh : 1.1.1.2 : Sphingobium sp. SYK-6	alcohol dehydrogenase GroES domain-containing protein		F1YPV4_9ACTO Alcohol dehydrogenase OS=Gordonia neof elifaecis NRRL B-59395 GN=SCNU_19842 PE=3 SV=1;alcohol dehydrogenase GroES domain-containing prote in [Xylanimonas cellulosilytica DSM 15894];Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Threonine dehydrogenase and related Zn-dependent dehydrogenases [Amino acid transport and metabolism / General function prediction only]; COG1063;PFAM: Alcohol dehydrogenase GroES domain protein; Alcohol dehydrogenase zinc-binding domain protein; KEGG: afw:Anae109_0322 alcohol dehydrogenase;NAD(P) binding site [chemical binding]
GOHSU59	CDS	GOHSU_59_00030	complement(1694..3523)		putative acyl-CoA dehydrogenase [nite]			ROP_52530 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus opacus B4	acyl-CoA dehydrogenase domain-containing protein		F1YPV5_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 19847 PE=3 SV=1;acyl-CoA dehydrogenase domain-containing protein [M ycobacterium vanbaalenii PYR- 1];PFAM: acyl-CoA dehydrogenase domain protein; KEGG: mpa:MAP3694c FadE5;Acyl-CoA dehydrogenase N terminal; pfam12418;acyl-CoA dehydrogenase; Provisional;Putative acyl-CoA dehydrogenases similar to fadE5; cd01153;FAD binding site [chemical binding];substrate binding site [chemical binding];catalytic residues [active];Acetyl-CoA dehydrogenase C-terminal like; pfam12806
GOHSU59	CDS	GOHSU_59_00040	3742..5130		wax ester synthase/diacylglycerol acyltransferase [nite]			YM304_02820 :wax ester synthase/diacylglycerol acyltransferase : : 2.3.1.20 : Acidimicrobidae bacterium YM16-304	acyltransferase		G7GLY9_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_20_01620 PE=4 SV=1;acyltransferase [Roseiflexus castenholzii DSM 13941 ];TIGRFAM: Acyltransferase WS/DGAT/MGAT family protein; PFAM: protein of unknown function UPF0089; KEGG: rrs:RoseRS_2761 protein of unknown function UPF0089;acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007;Protein of unknown function (DUF1298); pfam06974
GOHSU59	CDS	GOHSU_59_00050	complement(5143..5910)		putative endonuclease IV [nite]	 nfo [nite]		ROP_15710 :putative endonuclease IV : nfo : 3.1.21.2 : Rhodococcus opacus B4	apurinic endonuclease Apn1		H5U6Z7_9ACTO Endonuclease IV OS=Gordonia sputi NBRC 100414 GN=nfo PE=3 SV=1;apurinic endonuclease Apn1 [Thermobispora bispora D SM 43833];type strain of Thermobispora bispora;KEGG: pca:Pcar_3071 endonuclease IV; TIGRFAM: apurinic endonuclease Apn1; PFAM: Xylose isomerase domain protein TIM barrel; SMART: AP endonuclease family 2;endonuclease IV; Provisional;AP endonuclease family 2; These endonucleases play a role in DNA repair. Cleave phosphodiester bonds at apurinic or apyrimidinic sites; the alignment also contains hexulose-6-phosphate isomerases, enzymes that catalyze the epimerization of...; cd00019;AP (apurinic/apyrimidinic) site pocket;DNA interaction;Metal-binding active site
GOHSU59	CDS	GOHSU_59_00060	5968..7215		hypothetical membrane protein [nite]			ROP_41690 :hypothetical membrane protein : : : Rhodococcus opacus B4	phosphoesterase PA-phosphatase-like protein		F1YN66_9ACTO Phosphoesterase PA-phosphatase related protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16 813 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: phosphoesterase PA-phosphatase related; KEGG: mxa:MXAN_0451 PAP2 family protein;PAP2_like proteins, Aur1_like subfamily. Yeast Aur1p or Ipc1p is necessary for the addition of inositol phosphate to ceramide, an essential step in yeast sphingolipid synthesis, and is the target of several antifungal compounds such as aureobasidin; cd03386
GOHSU59	CDS	GOHSU_59_00070	7212..7832		conserved hypothetical protein [nite]			RER_10550 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F4F2A5_VERMA Putative uncharacterized protein OS=Ve rrucosispora maris (strain AB-18-032) GN=VAB18032_08380 PE =4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.991) with cleavage site probability 0.570 at residue 30;KEGG: sus:acid_3130 hypothetical protein;Domain of unknown function (DUF305); cl15795
GOHSU59	CDS	GOHSU_59_00080	complement(7876..8709)		hypothetical protein [nite]			RER_08510 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YN68_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16823 PE=4 SV=1;unnamed protein product [Mycobacterium rhodesiae NB B3];PFAM: Protein of unknown function (DUF3705);Thioesterase-like superfamily; pfam13622
GOHSU59	CDS	GOHSU_59_00090	8733..9308		hypothetical protein [nite]			KSE_61710 :hypothetical protein : : : Kitasatospora setae NBRC 14216	integral membrane protein		D0L666_GORB4 Putative uncharacterized protein OS=Go rdonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 31 98 / NCTC 10667) GN=Gbro_4412 PE=4 SV=1;integral membrane protein [Streptomyces sviceus ATC C 29083];Short repeat of unknown function (DUF308); cl15828
GOHSU59	CDS	GOHSU_59_00100	complement(9321..9866)		hypothetical protein [migap]				hypothetical protein		F1YN70_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16833 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247]
GOHSU59	CDS	GOHSU_59_00110	9995..10897		cation efflux protein [nite]			ACMV_15820 :cation efflux protein : : : Acidiphilium multivorum AIU301	cation efflux protein		F1YN71_9ACTO Cation diffusion facilitator family tr ansporter OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_1 6838 PE=4 SV=1;cation efflux protein [Enterobacter cloacae];clinical isolate;Predicted Co/Zn/Cd cation transporters [Inorganic ion transport and metabolism]; COG0053;Cation efflux family; cl00316
GOHSU59	CDS	GOHSU_59_00120	10998..11387		hypothetical protein [nite]			MLP_34140 :hypothetical protein : : : Microlunatus phosphovorus NM-1	hypothetical protein		G7H1K8_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_046_00020 PE=4 SV=1;unnamed protein product [Streptomyces scabiei 87.22 ]
GOHSU59	CDS	GOHSU_59_00130	11390..11938		putative TetR family transcriptional regulator [nite]			RER_11440 :putative TetR family transcriptional regulator : : : Rhodococcus erythropolis PR4	TetR family transcriptional regulator		F1YN73_9ACTO Regulatory protein TetR OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_16848 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: TetR family transcriptional regulator; KEGG: bpe:BP1202 TetR family transcriptional regulator;Transcriptional regulator [Transcription]; COG1309;Helix-turn-helix domains; cl00088
GOHSU59	CDS	GOHSU_59_00140	11987..14080		phosphate acetyltransferase [nite]	 pta [nite]		ROP_19120 :phosphate acetyltransferase : pta : 2.3.1.8 : Rhodococcus opacus B4	phosphate acetyltransferase		F1YN74_9ACTO Phosphate acetyltransferase OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_16853 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: phosphate acetyltransferase; PFAM: phosphate acetyl/butaryl transferase; DRTGG domain protein; KEGG: net:Neut_1995 phosphate acetyltransferase;phosphate acetyltransferase; Reviewed;DRTGG domain; cl12147;Phosphate acetyl/butaryl transferase; cl00390
GOHSU59	CDS	GOHSU_59_00150	14077..15300		acetate kinase [hamap]	ackA [hamap]	MF_00020:acetate kinase:ackA 	ROP_19110 :acetate kinase : ackA : 2.7.2.1 : Rhodococcus opacus B4	acetate kinase	ackA	F1YN75_9ACTO Acetate kinase A/propionate kinase 2 O S=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_16858 PE=3 S V=1;acetate kinase [Corynebacterium glutamicum ATCC 130 32];IS fingerprint type 4-5;AckA utilizes acetate and can acetylate CheY which increases signal strength during flagellar rotation; utilizes magnesium and ATP; also involved in conversion of acetate to aceyl-CoA;propionate/acetate kinase; Provisional;Acetokinase family; cl01029
##GOHSU60	contig00381				13525
GOHSU60	CDS	GOHSU_60_00010	complement(<1..918)	codon	integral membrane protein [migap]				integral membrane protein		H5U6N3_9ACTO Putative uncharacterized protein OS=Go rdonia sputi NBRC 100414 GN=GOSPT_129_00380 PE=4 SV=1;integral membrane protein [Rhodococcus jostii RHA1] 
GOHSU60	CDS	GOHSU_60_00020	complement(915..3065)		integral membrane protein [migap]				integral membrane protein		Q0SJ28_RHOSR Possible integral membrane protein OS= Rhodococcus sp. (strain RHA1) GN=RHA1_ro00623 PE=4 SV=1;integral membrane protein [Rhodococcus jostii RHA1] ;Predicted membrane protein [Function unknown];Fusaric acid resistance protein-like; cl15844
GOHSU60	CDS	GOHSU_60_00030	3064..4257		putative acyl-CoA dehydrogenase [nite]			RER_23000 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase domain-containing protein		D0L994_GORB4 Acyl-CoA dehydrogenase domain protein OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / J CM 3198 / NCTC 10667) GN=Gbro_4826 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: acyl-CoA dehydrogenase domain protein; acyl- CoA dehydrogenase type 2 domain; KEGG: pap:PSPA7_5729  acyl-CoA dehydrogenase;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase; cl09933
GOHSU60	CDS	GOHSU_60_00040	4266..4781		not assigned						
GOHSU60	CDS	GOHSU_60_00050	4862..6901		adenine-specific methyltransferase [migap]				adenine-specific methyltransferase		F1YZE9_9STRE DNA adenine methylase OS=Streptococcus parauberis NCFD 2020 GN=dam PE=4 SV=1;adenine-specific methyltransferase [Helicobacter bi zzozeronii CIII- 1];Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU60	CDS	GOHSU_60_00060	complement(6870..8828)		N-terminal truncated GCATC--recognizing Type II restriction modification system (MmyCIII) endonuclease subunit [migap]				N-terminal truncated GCATC--recognizing Type II restriction modification system (MmyCIII) endonuclease subunit		F7Z9C8_MYCBH N-terminal truncated GCATC--recognizin g Type II restriction modification system (MmyCIII) endonu clease subunit OS=Mycoplasma bovis (strain Hubei-1) GN=MMB _0665 PE=4 SV=1;N-terminal truncated GCATC--recognizing Type II res triction modification system (MmyCIII) endonuclease subuni t [Mycoplasma bovis Hubei- 1];AlwI restriction endonuclease; pfam09491
GOHSU60	CDS	GOHSU_60_00070	8932..9303		putative delta(5)-3-ketosteroid isomerase [nite]			RER_59940 :putative delta(5)-3-ketosteroid isomerase : : 5.3.3.1 : Rhodococcus erythropolis PR4	delta(5)-3-ketosteroid isomerase		F1YL66_9ACTO Steroid delta-isomerase OS=Gordonia ne ofelifaecis NRRL B-59395 GN=SCNU_13118 PE=4 SV=1;delta(5)-3-ketosteroid isomerase [Rhodococcus eryth ropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;protein synonym:steroid delta-isomerase;Nuclear transport factor 2 (NTF2-like) superfamily. This family includes members of the NTF2 family, Delta-5-3-ketosteroid isomerases, Scytalone Dehydratases, and the beta subunit of Ring hydroxylating dioxygenases. This family is a classic example of...; cl09109
GOHSU60	CDS	GOHSU_60_00080	9440..10618		putative SulP family transporter [nite]			ROP_00250 :putative SulP family transporter : : : Rhodococcus opacus B4	integral membrane transporter		H5TGI9_9ACTO Putative SulP family transporter (Frag ment) OS=Gordonia otitidis NBRC 100426 GN=GOOTI_018_00010 PE=4 SV=1;integral membrane transporter [Streptomyces roseosp orus NRRL 15998];Sulfate transporter family; cl15842
GOHSU60	CDS	GOHSU_60_00090	complement(10650..11333)		putative oxidoreductase [nite]			RER_59780 :putative oxidoreductase : : : Rhodococcus erythropolis PR4	short-chain dehydrogenase/reductase SDR		H5UEN5_9ACTO Putative oxidoreductase OS=Gordonia te rrae NBRC 100016 GN=GOTRE_060_01020 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: short-chain dehydrogenase/reductase SDR; KR domain protein; KEGG: ent:Ent638_4222 short-chain dehydrogenase/reductase SDR;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU60	CDS	GOHSU_60_00100	complement(11330..13006)		conserved hypothetical protein [nite]			RER_37020 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YL61_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_13093 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Repair protein; cl01535
GOHSU60	CDS	GOHSU_60_00110	13060..>13525	3x codon	conserved hypothetical protein [nite]			RER_00500 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	FAD dependent oxidoreductase		F1YL60_9ACTO FAD dependent oxidoreductase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_13088 PE=4 SV=1;FAD dependent oxidoreductase [Streptomyces violaceu sniger Tu 4113];Phytoene dehydrogenase and related proteins [Secondary metabolites biosynthesis, transport, and catabolism];PFAM: FAD dependent oxidoreductase; KEGG: tcu:Tcur_3794 FAD dependent oxidoreductase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
##GOHSU61	contig00391				13227
GOHSU61	CDS	GOHSU_61_00010	complement(<1..259)	3x codon	isochorismate synthase [nite]			RER_11150 :isochorismate synthase : : 5.4.4.2 : Rhodococcus erythropolis PR4	isochorismate synthase	dhbC	Q5YNC1_NOCFA Putative isochorismate synthase OS=Noc ardia farcinica (strain IFM 10152) GN=NFA_54680 PE=4 SV=1;isochorismate synthase [Rhodococcus jostii RHA1];isochorismate synthase DhbC; Validated
GOHSU61	CDS	GOHSU_61_00020	483..1781		putative cysteine desulfurase [nite]			RER_05540 :putative cysteine desulfurase : : 2.8.1.7 : Rhodococcus erythropolis PR4	aminotransferase		G7GVZ4_9ACTO Putative cysteine desulfurase OS=Gordo nia amarae NBRC 15530 GN=GOAMR_72_00090 PE=3 SV=1;aminotransferase [Streptomyces albus J1074];Selenocysteine lyase/Cysteine desulfurase [Posttranslational modification, protein turnover, chaperones]; COG0520;Aspartate aminotransferase (AAT) superfamily (fold type I) of pyridoxal phosphate (PLP)-dependent enzymes. PLP combines with an alpha-amino acid to form a compound called a Schiff base or aldimine intermediate, which depending on the reaction, is the...; cl00321;pyridoxal 5'-phosphate binding pocket [chemical binding];catalytic residue [active]
GOHSU61	CDS	GOHSU_61_00030	complement(1866..2345)		inorganic pyrophosphatase [hamap]	ppa [hamap]	MF_00209:inorganic pyrophosphatase:ppa 	ROP_43160 :inorganic pyrophosphatase : ppa : 3.6.1.1 : Rhodococcus opacus B4	inorganic pyrophosphatase	ppa	H5U432_9ACTO Inorganic pyrophosphatase OS=Gordonia sputi NBRC 100414 GN=ppa PE=3 SV=1;inorganic pyrophosphatase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Inorganic pyrophosphatase. These enzymes hydrolyze inorganic pyrophosphate (PPi) to two molecules of orthophosphates (Pi). The reaction requires bivalent cations. The enzymes in general exist as homooligomers; cd00412;Catalyzes the hydrolysis of pyrophosphate;dimer interface [polypeptide binding];substrate binding site [chemical binding];metal binding sites [ion binding]
GOHSU61	CDS	GOHSU_61_00040	2461..3858		putative D-alanyl-D-alanine carboxypeptidase [nite]	 dac [nite]		RER_06180 :putative D-alanyl-D-alanine carboxypeptidase : dac : 3.4.16.4 : Rhodococcus erythropolis PR4	D-Ala D-Ala carboxypeptidase		F1YLQ0_9ACTO D-alanyl-D-alaninecarboxypeptidase/D-a lanyl-D-al anine-endopeptidase OS=Gordonia neofelifaecis N RRL B-59395 GN=SCNU_14219 PE=4 SV=1;unnamed protein product [Corynebacterium diphtheria e HC04];Penicillin binding protein transpeptidase domain; cl01009;D-alanyl-D-alanine carboxypeptidase, serine-type, PBP4 family; TIGR00666
GOHSU61	CDS	GOHSU_61_00050	3943..5031		conserved hypothetical protein [nite]			RER_06190 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLP9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14214 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: conserved hypothetical protein; PFAM: protein of unknown function DUF2342;Uncharacterized conserved protein (DUF2342); cl02183
GOHSU61	CDS	GOHSU_61_00060	5037..5990	case	tRNA(Ile)-lysidine synthase [hamap]	tilS [hamap]	MF_01161:tRNA(Ile)-lysidine synthase:tilS 	RER_06200 :putative tRNA(Ile)-lysidine synthase : tilS : 6.3.4.- : Rhodococcus erythropolis PR4	tRNA(Ile)-lysidine synthetase		F1YLP8_9ACTO tRNA(Ile)-lysidine synthase OS=Gordoni a neofelifaecis NRRL B-59395 GN=tilS PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: tRNA(Ile)-lysidine synthetase; PFAM: PP-loop domain protein; Queuosine synthesis- like; KEGG: afw:Anae109_2339 tRNA(Ile)-lysidine synthetase;N-terminal domain of predicted ATPase of the PP-loop faimly implicated in cell cycle control [Cell division and chromosome partitioning]. This is a subfamily of Adenine nucleotide alpha hydrolases superfamily.Adeninosine nucleotide alpha hydrolases...; cd01992;Ligand Binding Site [chemical binding]
GOHSU61	CDS	GOHSU_61_00070	complement(6030..6764)		hypothetical protein [migap]				hypothetical protein		F1YLP6_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14199 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 1.000) with cleavage site probability 0.997 at residue 24;KEGG: GH24282 gene product from transcript GH24282- RA
GOHSU61	CDS	GOHSU_61_00080	complement(6922..7539)		hypothetical membrane protein [nite]			ROP_55460 :hypothetical membrane protein : : : Rhodococcus opacus B4	hypothetical protein		F1YLP7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14204 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU61	CDS	GOHSU_61_00090	7769..8335		hypoxanthine phosphoribosyltransferase [nite]	 hpt [nite]		RER_06210 :hypoxanthine phosphoribosyltransferase : hpt : 2.4.2.8 : Rhodococcus erythropolis PR4	hypoxanthine-guanine phosphoribosyltransferase	hpt	H6MVN9_GORPV Hypoxanthine-guanine phosphoribosyltra nsferase Hpt OS=Gordonia polyisoprenivorans (strain DSM 44 266 / VH2) GN=hpt PE=4 SV=1;hypoxanthine-guanine phosphoribosyltransferase [Cor ynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;Catalyzes the salvage synthesis of inosine-5'-monophosphate (IMP) and guanosine-5'-monophosphate (GMP) from the purine bases hypoxanthine and guanine, respectively;Phosphoribosyl transferase (PRT)-type I domain; cd06223
GOHSU61	CDS	GOHSU_61_00100	8490..10862		ATP-dependent protease FtsH [nite]	 ftsH [nite]		ROP_43230 :ATP-dependent protease FtsH : ftsH : 3.4.24.- : Rhodococcus opacus B4	ATP-dependent metalloprotease FtsH		D0L4G2_GORB4 ATP-dependent zinc metalloprotease Fts H (Precursor) OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=ftsH PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.897) with cleavage site probability 0.438 at residue 23;KEGG: avi:Avi_3627 metalloprotease; TIGRFAM: ATP-dependent metalloprotease FtsH; PFAM: peptidase M41; peptidase M41 FtsH extracellular; ATPase AAA; ATPase associated with various cellular activities AAA_5; SMART: ATPase AAA;ATP-dependent metalloprotease FtsH; TIGR01241;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger;Peptidase family M41; pfam01434
GOHSU61	CDS	GOHSU_61_00110	10885..11487		GTP cyclohydrolase I [hamap]	folE [hamap]	MF_00223:GTP cyclohydrolase I:folE 	ROP_53730 :GTP cyclohydrolase I : folE : 3.5.4.16 : Rhodococcus opacus B4	GTP cyclohydrolase I	folE	H5U425_9ACTO GTP cyclohydrolase 1 OS=Gordonia sputi NBRC 100414 GN=folE PE=3 SV=1;GTP cyclohydrolase I [Rhodococcus jostii RHA1];involved in the first step of tetrahydrofolate biosynthesis; catalyzes the formation of formate and 2-amino-4-hydroxy-6-(erythro-1,2, 3-trihydroxypropyl)dihydropteridine triphosphate from GTP and water; forms a homopolymer;GTP cyclohydrolase I (GTP-CH-I) catalyzes the conversion of GTP into dihydroneopterin triphosphate.  The enzyme product is the precursor of tetrahydrofolate in eubacteria, fungi, and plants and of the folate analogs in methanogenic bacteria.  In...; cd00642;homodecamer interface [polypeptide binding];GTP cyclohydrolase I; Provisional;putative catalytic site residues [active];zinc binding site [ion binding];GTP-CH-I/GFRP interaction surface
GOHSU61	CDS	GOHSU_61_00120	11489..12352		dihydropteroate synthase [nite]	 folP [nite]		ROP_43250 :dihydropteroate synthase : folP : 2.5.1.15 : Rhodococcus opacus B4	dihydropteroate synthase		H5U424_9ACTO Dihydropteroate synthase OS=Gordonia s puti NBRC 100414 GN=folP PE=4 SV=1;dihydropteroate synthase [Amycolicicoccus subflavus DQS3- 9A1];type strain of Amycolicicoccus subflavus;DHPS subgroup of Pterin binding enzymes. DHPS (dihydropteroate synthase), a functional homodimer, catalyzes the condensation of p-aminobenzoic acid (pABA) in the de novo biosynthesis of folate, which is an essential cofactor in both nucleic acid and...; cd00739;dihydropteroate synthase; TIGR01496;substrate binding pocket [chemical binding];dimer interface [polypeptide binding];inhibitor binding site
GOHSU61	CDS	GOHSU_61_00130	12345..12728		dihydroneopterin aldolase [nite]	 folB [nite]		ROP_43260 :dihydroneopterin aldolase : folB : 4.1.2.25 : Rhodococcus opacus B4	dihydroneopterin aldolase	folX	H5UDZ7_9ACTO Dihydroneopterin aldolase OS=Gordonia terrae NBRC 100016 GN=folB PE=4 SV=1;dihydroneopterin aldolase [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Dihydroneopterin aldolase (DHNA) and 7,8-dihydroneopterin triphosphate epimerase domain (DHNTPE); these enzymes have been designated folB and folX, respectively. Folate derivatives are essential cofactors in the biosynthesis of purines, pyrimidines, and...; cd00534;homooctamer interface [polypeptide binding]
GOHSU61	CDS	GOHSU_61_00140	12725..>13227	3x codon	2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase [nite]	 folK [nite]		RER_06290 :2-amino-4-hydroxy-6-hydroxymethyldihydropteridine pyrophosphokinase : folK : 2.7.6.3 : Rhodococcus erythropolis PR4	2-amino-4-hydroxy-6- hydroxymethyldihydropteridine pyrophosphokinase		F9VQG1_9ACTO 2-amino-4-hydroxy-6-hydroxymethyldihyd ropteridine pyrophosphokinase OS=Gordonia alkanivorans NBR C 16433 GN=folK PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: rpa:RPA1838 2-amino-4-hydroxy-6- hydroxymethyldihydropteridine pyrophosphokinase; TIGRFAM:2-amino-4-hydroxy-6-hydroxymethyldihydropter idinepyrophosphokinase; PFAM: 78-dihydro-6-hydroxymethylpterin- pyrophosphokinase HPPK;7,8-dihydro-6-hydroxymethylpterin-pyrophosphokinase (HPPK). Folate derivatives are essential cofactors in the biosynthesis of purines, pyrimidines, and amino acids as well as formyl-tRNA. Mammalian cells are able to utilize pre-formed folates after...; cd00483;catalytic center binding site [active];ATP binding site [chemical binding]
##GOHSU62	contig00401				8678
GOHSU62	CDS	GOHSU_62_00010	<1..838	3x codon	putative thiamine biosynthesis oxidoreductase ThiO [nite]	 thiO [nite]		RER_14800 :putative thiamine biosynthesis oxidoreductase ThiO : thiO : : Rhodococcus erythropolis PR4	glycine oxidase ThiO		F1YN82_9ACTO Glycine oxidase ThiO OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_16893 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: glycine oxidase ThiO; PFAM: FAD dependent oxidoreductase; KEGG: dac:Daci_3293 FAD dependent oxidoreductase;glycine oxidase ThiO; TIGR02352
GOHSU62	CDS	GOHSU_62_00020	835..1053		putative thiamine biosynthesis protein ThiS [nite]	 thiS [nite]		RER_14810 :putative thiamine biosynthesis protein ThiS : thiS : : Rhodococcus erythropolis PR4	thiamine biosynthesis protein ThiS	thiS	F1YN83_9ACTO Sulfur carrier protein ThiS OS=Gordoni a neofelifaecis NRRL B-59395 GN=SCNU_16898 PE=4 SV=1;thiamine biosynthesis protein ThiS [Rhodococcus ery thropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;ThiaminS ubiquitin-like sulfur carrier protein; cd00565;thiS-thiF/thiG interaction site
GOHSU62	CDS	GOHSU_62_00030	1046..1825		thiazole synthase [hamap]	thiG [hamap]	MF_00443:thiazole synthase:thiG 	RER_14820 :thiazole biosynthesis protein ThiG : thiG : : Rhodococcus erythropolis PR4	thiazole synthase	thiG	H0QXT1_9ACTO Thiazole synthase OS=Gordonia effusa N BRC 100432 GN=thiG PE=3 SV=1;thiazole synthase [Rhodococcus opacus B4];functions in thiamine (vitamin B1) biosynthesis; in Bacillus subtilis this enzyme catalyzes the formation of thiazole from dehydroxyglycine and 1-deoxy-D-xylulose-5-phosphate and ThiS-thiocarboxylate;Thiazole synthase (ThiG) is the tetrameric enzyme that is involved in the formation of the thiazole moiety of thiamin pyrophosphate, an essential ubiquitous cofactor that plays an important role in carbohydrate and amino acid metabolism. ThiG catalyzes...; cd04728;ThiS interaction site;putative active site [active];tetramer interface [polypeptide binding]
GOHSU62	CDS	GOHSU_62_00040	complement(1836..2759)		hypothetical protein [nite]			KSE_51680 :hypothetical protein : : : Kitasatospora setae NBRC 14216	hypothetical protein		H6MWS0_GORPV Putative membrane protein OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c06570 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];exopolysaccharide biosynthesis polyprenyl glycosylphosphotransferase; TIGR03025
GOHSU62	CDS	GOHSU_62_00050	2847..4043		conserved hypothetical protein [nite]			BBR47_46760 :conserved hypothetical protein : : : Brevibacillus brevis NBRC 100599	hypothetical protein		F1YN86_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_16913 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function DUF201; KEGG: pfl:PFL_6136 hypothetical protein;Glycosyl transferase 4-like domain; cl15759;ATP-grasp domain; cl03087
GOHSU62	CDS	GOHSU_62_00060	complement(4006..4947)		putative hydrolase [nite]			RER_09730 :putative hydrolase : : : Rhodococcus erythropolis PR4	alpha/beta hydrolase		F1YN87_9ACTO Alpha/beta hydrolase fold protein OS=G ordonia neofelifaecis NRRL B-59395 GN=SCNU_16918 PE=4 SV=1 ;alpha/beta hydrolase [Frankia sp. CcI3];Esterases and lipases (includes fungal lipases, cholinesterases, etc.)  These enzymes act on carboxylic esters (EC: 3.1.1.-). The catalytic apparatus involves three residues (catalytic triad): a serine, a glutamate or aspartate and a histidine.These...; cl12031;Alpha/beta hydrolase family; pfam12697
GOHSU62	CDS	GOHSU_62_00070	4957..6012		putative ABC transporter ATP-binding protein [nite]			KRH_01750 :putative ABC transporter ATP-binding protein : : : Kocuria rhizophila DC2201	ABC transporter-like protein		H6MWH7_GORPV Putative ABC transporter ATP-binding p rotein OS=Gordonia polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c31670 PE=3 SV=1;unnamed protein product [Nocardioides sp. JS614];ABC-type multidrug transport system, ATPase component [Defense mechanisms]; COG1131;PFAM: ABC transporter related; SMART: AAA ATPase; KEGG: tfu:Tfu_1436 ABC-type multidrug transport system ATPase component;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU62	CDS	GOHSU_62_00080	6009..6803		hypothetical membrane protein [nite]			KRH_01740 :hypothetical membrane protein : : : Kocuria rhizophila DC2201	hypothetical protein		G7GT80_9ACTO Putative uncharacterized protein OS=Go rdonia amarae NBRC 15530 GN=GOAMR_59_01060 PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];KEGG: fal:FRAAL6618 hypothetical protein;ABC-2 type transporter; cl11417
GOHSU62	CDS	GOHSU_62_00090	complement(6835..8127)		acetyl-CoA acyltransferase [nite]	 fadA [nite]		ROP_52500 :acetyl-CoA acyltransferase : fadA : 2.3.1.16 : Rhodococcus opacus B4	acetyl-CoA acetyltransferase		G7H3P6_9ACTO Acetyl-CoA acyltransferase OS=Gordonia araii NBRC 100433 GN=fadA PE=3 SV=1;acetyl-CoA acetyltransferase [Pseudomonas fluoresce ns SBW25];acetyl-CoA acetyltransferase; Provisional;Catalyzes the synthesis of acetoacetyl coenzyme A from two molecules of acetyl coenzyme A. It can also act as a thiolase, catalyzing the reverse reaction and generating two-carbon units from the four-carbon product of fatty acid oxidation;Thiolase are ubiquitous enzymes that catalyze the reversible thiolytic cleavage of 3-ketoacyl-CoA into acyl-CoA and acetyl-CoA, a 2-step reaction involving a covalent intermediate formed with a catalytic cysteine. They are found in prokaryotes and...; cd00751;dimer interface [polypeptide binding]
GOHSU62	CDS	GOHSU_62_00100	8218..>8678	3x codon	phosphotransferase system enzyme IIAB component [nite]			KRH_22560 :phosphotransferase system enzyme IIAB component : : 2.7.1.69 : Kocuria rhizophila DC2201	PTS transporter subunit IIA-like nitrogen-regulatory protein PtsN		G2P3V1_STRVO Putative PTS IIA-like nitrogen-regulat ory protein PtsN OS=Streptomyces violaceusniger Tu 4113 GN =Strvi_5120 PE=4 SV=1;PTS transporter subunit IIA-like nitrogen-regulator y protein PtsN [Thermaerobacter marianensis DSM 12885];COGs: COG1762 phosphotransferase system mannitol/fructose-specific IIA domain (Ntr-type); InterPro IPR002178; KEGG: cpo:COPRO5265_0027 PtsN protein; PFAM: phosphoenolpyruvate-dependent sugar phosphotransferase system EIIA 2; SPTR: PtsN protein; PFAM: phosphoenolpyruvate-dependent sugar phosphotransferase system, EIIA 2;PTS_IIA, PTS system, fructose/mannitol specific IIA subunit. The bacterial phosphoenolpyruvate: sugar phosphotransferase system (PTS) is a multi-protein system involved in the regulation of a variety of metabolic and transcriptional processes. This...; cd00211;phosphorylation site [posttranslational modification]
##GOHSU63	contig00402				4186
GOHSU63	CDS	GOHSU_63_00010	<1..370	3x codon frame	phosphotransferase system enzyme IIAB component [nite]			KRH_22560 :phosphotransferase system enzyme IIAB component : : 2.7.1.69 : Kocuria rhizophila DC2201	PTS system lactose/cellobiose-specific transporter subunit IIB		Q8NLB1_CORGL Phosphotransferase system mannitol/fru ctose-specific IIA domain (Ntr-type) OS=Corynebacterium gl utamicum (strain ATCC 13032 / DSM 20300 / JCM 1318 / LMG 3 730 / NCIMB 10025) GN=rmpA PE=4 SV=1;PTS system lactose/cellobiose-specific transporter subunit IIB [Thermaerobacter marianensis DSM 12885];COGs: COG3414 phosphotransferase system galactitol-specific IIB component; InterPro IPR013011: IPR003501; KEGG: lin:lin2799 hypothetical protein; PFAM: phosphotransferase system lactose/cellobiose-specific IIsubunit beta; SPTR: phosphotransferase system lactose/cellobiose-specific IIsubunit beta; PFAM: PTS system, Lactose/cellobiose specific IIsubunit beta;PTS_IIB_ascorbate: subunit IIB of enzyme II (EII) of the L-ascorbate-specific phosphoenolpyruvate:carbohydrate phosphotransferase system (PTS). In this system, EII is an L-ascorbate-specific permease with two cytoplasmic subunits (IIA and IIB) and a...; cd05563;P-loop;phosphorylation site [posttranslational modification]
GOHSU63	CDS	GOHSU_63_00020	422..2002		phosphotransferase system enzyme IIC component [nite]			KRH_22570 :phosphotransferase system enzyme IIC component : : : Kocuria rhizophila DC2201	PTS system ascorbate-specific transporter subunit IIC	ulaA	D5P0Y4_CORAM Transport protein, SgaT family OS=Cory nebacterium ammoniagenes DSM 20306 GN=HMPREF0281_02272 PE= 4 SV=1;PTS system ascorbate-specific transporter subunit I IC [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;membrane component; functions with enzymes IIB (sgaB; ulaB) and IIA (sgaA; ulaC) enzyme I and HPr for anaerobic utilization and uptake of L-ascorbate; sgaTBA are regulated by yifQ as well as Crp and Fnr;PTS system sugar-specific permease component; cl04451
GOHSU63	CDS	GOHSU_63_00030	2144..2635		not assigned						E8S0X9_MICSL Putative uncharacterized protein (Prec ursor) OS=Micromonospora sp. (strain L5) GN=ML5_3529 PE=4 SV=1
GOHSU63	CDS	GOHSU_63_00040	complement(2788..4149)		putative FAD-linked oxidase [nite]			ROP_12930 :putative FAD-linked oxidase : : : Rhodococcus opacus B4	glycolate oxidase FAD-linked subunit		G7H0Z7_9ACTO Putative FAD-linked oxidase OS=Gordoni a araii NBRC 100433 GN=GOARA_042_00290 PE=4 SV=1;glycolate oxidase FAD-linked subunit [Rhodococcus j ostii RHA1];FAD binding domain; pfam01565;glycolate oxidase, subunit GlcD; TIGR00387
##GOHSU64	contig00411				12261
GOHSU64	CDS	GOHSU_64_00010	complement(<1..471)	codon	hybrid cluster protein [migap]				hybrid cluster protein		hybrid cluster protein [Thermobispora bispora DSM 4 3833];type strain of Thermobispora bispora;hybrid cluster protein; Provisional;The HCP family of iron-sulfur proteins includes hybrid cluster protein (HCP), acetyl-CoA synthase (ACS), and carbon monoxide dehydrogenase (CODH), all of which contain [Fe4-S4] metal clusters at their active sites. These proteins have a conserved...; cl14655;TIGRFAM: hybrid cluster protein; PFAM: Prismane; KEGG: bvi:Bcep1808_5223 hydroxylamine reductase;ACS interaction site;CODH interaction site
GOHSU64	CDS	GOHSU_64_00020	complement(549..1235)		tRNA (guanine-N(1)-)-methyltransferase [hamap]	trmD [hamap]	MF_00605:tRNA (guanine-N(1)-)-methyltransferase:trmD 	ROP_65900 :tRNA (guanine-N(1)-)-methyltransferase : trmD : 2.1.1.31 : Rhodococcus opacus B4	tRNA (guanine-N(1)-)-methyltransferase	trmD	F1YI95_9ACTO tRNA (guanine-N(1)-)-methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=trmD PE=3 SV=1;tRNA (guanine-N(1)-)-methyltransferase [Corynebacte rium glutamicum ATCC 13032];methylates guanosine-37 in various tRNAs; uses S-adenosyl-L-methionine to transfer methyl group to tRNA;tRNA (Guanine-1)-methyltransferase; cl00407
GOHSU64	CDS	GOHSU_64_00030	complement(1238..1765)		ribosome maturation factor RimM [hamap]	rimM [hamap]	MF_00014:ribosome maturation factor RimM:rimM 	RER_24460 :probable 16S rRNA processing protein RimM : rimM : : Rhodococcus erythropolis PR4	16S rRNA-processing protein RimM	rimM	H5TGD3_9ACTO Ribosome maturation factor RimM OS=Gor donia otitidis NBRC 100426 GN=rimM PE=3 SV=1;16S rRNA-processing protein RimM [Corynebacterium g lutamicum ATCC 13032];IS fingerprint type 4-5;16S rRNA-processing protein RimM; Provisional; PRK00122;Essential for efficient processing of 16S rRNA;RimM N-terminal domain; pfam01782;Photosynthetic reaction center (RC) complex, subunit H;  RC is an integral membrane protein-pigment complex which catalyzes light-induced reduction of ubiquinone to ubiquinol, generating a transmembrane electrochemical gradient of protons used to produce...; cl09959
GOHSU64	CDS	GOHSU_64_00040	complement(1765..2007)		UPF0109 protein <locus_tag> [hamap]		MF_00088:UPF0109 protein <locus_tag>: 	ROP_65880 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		H6MXZ5_GORPV UPF0109 protein GPOL_c19880 OS=Gordoni a polyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c198 80 PE=3 SV=1;unnamed protein product [Desulfovibrio africanus st r. Walvis Bay];HAMAP: Uncharacterized protein family UPF0109, bacterial; KEGG: dma:DMR_34690 hypothetical protein;Predicted RNA-binding protein (contains KH domain) [General function prediction only];K homology RNA-binding domain, type I.  KH binds single-stranded RNA or DNA. It is found in a wide variety of proteins including ribosomal proteins, transcription factors and post-transcriptional modifiers of mRNA. There are two different KH domains that...; cl00098
GOHSU64	CDS	GOHSU_64_00050	complement(2004..2582)	case	30S ribosomal protein S16 [hamap]	rpsP [hamap]	MF_00385:30S ribosomal protein S16:rpsP 	KSE_51880 :putative ribosomal protein S16 : rpsP : : Kitasatospora setae NBRC 14216	30S ribosomal protein S16	rpsP	G7H540_9ACTO 30S ribosomal protein S16 OS=Gordonia araii NBRC 100433 GN=rpsP PE=3 SV=1;30S ribosomal protein S16 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosomal protein S16; cl00368;binds to lower part of 30S body where it stabilizes two domains; required for efficient assembly of 30S; in Escherichia coli this protein has nuclease activity
GOHSU64	CDS	GOHSU_64_00060	2886..3539		hypothetical protein [migap]				hypothetical protein		G7H4Y6_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_063_01100 PE=4 SV=1;unnamed protein product [Corynebacterium efficiens YS- 314]
GOHSU64	CDS	GOHSU_64_00070	complement(3565..4647)		putative M38 family peptidase [nite]			ROP_65860 :putative M38 family peptidase : : 3.4.-.- : Rhodococcus opacus B4	amidohydrolase		F1YI90_9ACTO Amidohydrolase OS=Gordonia neofelifaec is NRRL B-59395 GN=SCNU_08023 PE=4 SV=1;amidohydrolase [Amycolatopsis mediterranei U32];Imidazolonepropionase and related amidohydrolases [Secondary metabolites biosynthesis, transport, and catabolism]; COG1228;Superfamily of metallo-dependent hydrolases (also called amidohydrolase superfamily) is a large group of proteins that show conservation in their 3-dimensional fold (TIM barrel) and in details of their active site. The vast majority of the members have a...; cl00281
GOHSU64	CDS	GOHSU_64_00080	complement(4650..6239)		signal recognition particle protein [nite]	 ffh [nite]		RER_24420 :signal recognition particle protein : ffh : : Rhodococcus erythropolis PR4	signal recognition particle protein		F1YI89_9ACTO Signal recognition particle protein OS =Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08018 PE=4 SV =1;unnamed protein product [Gordonia bronchialis DSM 4 3247];signal recognition particle protein; Provisional;KEGG: dds:Ddes_1804 signal recognition particle protein; TIGRFAM: signal recognition particle protein; PFAM: GTP-binding signal recognition particle SRP54 G- domain; Signal peptide binding (SRP54) M- domain protein; GTP-binding signal recognition particle SRP54 helical bundle; SMART: ATPase AAA;SRP54-type protein, helical bundle domain; cl03758;The signal recognition particle (SRP) mediates the transport to or across the plasma membrane in bacteria and the endoplasmic reticulum in eukaryotes. SRP recognizes N-terminal sighnal sequences of newly synthesized polypeptides at the ribosome. The...; cd03115;P loop;GTP binding site [chemical binding];Signal peptide binding domain; pfam02978
GOHSU64	CDS	GOHSU_64_00090	complement(6322..6660)		putative nitrogen regulatory protein P-II [nite]	 glnK2 [nite]		KSE_23320 :putative nitrogen regulatory protein P-II : glnK2 : : Kitasatospora setae NBRC 14216	nitrogen regulatory protein P-II		G7H544_9ACTO Nitrogen regulatory protein P-II OS=Go rdonia araii NBRC 100433 GN=glnK2 PE=3 SV=1;nitrogen regulatory protein P-II [Streptomyces svic eus ATCC 29083];Nitrogen regulatory protein P-II; cl00412
GOHSU64	CDS	GOHSU_64_00100	complement(6707..8041)		ammonium transporter AmtB [nite]	 amtB [nite]		ROP_65820 :ammonium transporter AmtB : amtB : : Rhodococcus opacus B4	ammonium transporter		F1YI86_9ACTO Ammonium transporter OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_08003 PE=4 SV=1;unnamed protein product [Arthrobacter sp. FB24];TIGRFAM: ammonium transporter; PFAM: Rh family protein/ammonium transporter; KEGG: nfa:nfa41640 putative ammonium transporter;Ammonium Transporter Family; cl03012
GOHSU64	CDS	GOHSU_64_00110	complement(8181..9803)		putative acyl-CoA dehydrogenase [nite]			RER_59150 :putative acyl-CoA dehydrogenase : : 1.3.99.- : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase		F1YI84_9ACTO Isovaleryl CoA dehydrogenase OS=Gordon ia neofelifaecis NRRL B-59395 GN=SCNU_07993 PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus opacus B4];isovaleryl CoA dehydrogenase; Provisional;Acyl-CoA dehydrogenase; cl09933
GOHSU64	CDS	GOHSU_64_00120	complement(9850..11217)		signal recognition particle receptor [nite]	 ftsY [nite]		ROP_65810 :signal recognition particle receptor : ftsY : : Rhodococcus opacus B4	signal recognition particle-docking protein FtsY	ftsY	F1YI82_9ACTO Signal recognition particle-docking pr otein FtsY OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 07983 PE=4 SV=1;ftsY gene product [Gordonia polyisoprenivorans VH2] ;signal recognition particle-docking protein FtsY; TIGR00064;SRP54-type protein, helical bundle domain; cl03758;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099
GOHSU64	CDS	GOHSU_64_00130	complement(11229..>12261)	3x codon frame	chromosome partition protein SMC [nite]	 smc [nite]		ROP_65800 :chromosome partition protein SMC : smc : : Rhodococcus opacus B4	chromosome segregation protein SMC		F1YI81_9ACTO Chromosome partition protein Smc OS=Go rdonia neofelifaecis NRRL B-59395 GN=smc PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: chromosome segregation protein SMC; PFAM: SMC domain protein; SMCs flexible hinge domain protein; KEGG: ppd:Ppro_3319 chromosome segregation protein SMC;Barmotin is a tight junction-associated protein expressed in rat epithelial cells which is thought to have an important regulatory role in tight junction barrier function.  Barmotin belongs to the SMC protein family. SMC proteins are large; cd03278;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
##GOHSU65	contig00421				11028
GOHSU65	CDS	GOHSU_65_00010	<1..284	3x codon frame	selenocysteine-specific elongation factor [nite]	 selB [nite]		MLP_05350 :selenocysteine-specific elongation factor : selB : : Microlunatus phosphovorus NM-1	selenocysteine-specific elongation factor SelB	selB	F1YEK1_9ACTO Selenocysteine-specific translation el ongation factor OS=Gordonia neofelifaecis NRRL B-59395 GN= SCNU_00615 PE=4 SV=1;selB gene product [Gordonia polyisoprenivorans VH2] ;Elongation factor SelB, winged helix; pfam09107
GOHSU65	CDS	GOHSU_65_00020	complement(295..2163)		heat shock protein 70 [migap]				heat shock protein 70		D0L506_GORB4 Heat shock protein 70 OS=Gordonia bron chialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 1 0667) GN=Gbro_1150 PE=3 SV=1;heat shock protein 70 [Stackebrandtia nassauensis D SM 44728];KEGG: hha:Hhal_0145 heat shock protein 70;Molecular chaperone [Posttranslational modification, protein turnover, chaperones]; COG0443
GOHSU65	CDS	GOHSU_65_00030	complement(2166..2672)		not assigned						H0JYV4_9NOCA Co-chaperone GrpE OS=Rhodococcus pyrid inivorans AK37 GN=AK37_24766 PE=3 SV=1
GOHSU65	CDS	GOHSU_65_00040	complement(2725..3006)		not assigned						A2DIS8_TRIVA Putative uncharacterized protein OS=Tr ichomonas vaginalis GN=TVAG_432870 PE=4 SV=1
GOHSU65	CDS	GOHSU_65_00050	complement(3169..4578)		fumarate hydratase class II [hamap]	fumC [hamap]	MF_00743:fumarate hydratase class II:fumC 	RER_42400 :fumarate hydratase class II : fumC : 4.2.1.2 : Rhodococcus erythropolis PR4	fumarate hydratase	fumC	H5UA47_9ACTO Fumarate hydratase class II OS=Gordoni a terrae NBRC 100016 GN=fumC PE=3 SV=1;fumarate hydratase [Corynebacterium glutamicum ATCC 13032];IS fingerprint type 4-5;class II family (does not require metal); tetrameric enzyme; fumarase C; reversibly converts (S)-malate to fumarate and water; functions in the TCA cycle;fumarate hydratase; Reviewed; PRK00485;Class II fumarases; cd01362;tetramer interface [polypeptide binding]
GOHSU65	CDS	GOHSU_65_00060	complement(4643..6343)		putative fumarate hydratase class I [nite]	 fum [nite]		KSE_47800 :putative fumarate hydratase class I : fum : 4.2.1.2 : Kitasatospora setae NBRC 14216	fumarate hydratase, class I	fumA	G7GZ61_9ACTO Fumarate hydratase class I OS=Gordonia araii NBRC 100433 GN=fum PE=4 SV=1;fumA gene product [Gordonia polyisoprenivorans VH2] ;fumarate hydratase; Provisional;Fumarate hydratase (Fumerase); cl00851;Fumarase C-terminus; cl00795
GOHSU65	CDS	GOHSU_65_00070	6435..7103		hypothetical protein [nite]			KSE_50990 :hypothetical protein : : : Kitasatospora setae NBRC 14216	integral membrane protein		F1YJ93_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_09649 PE=4 SV=1;integral membrane protein [Streptomyces lividans TK 24];Predicted membrane protein [Function unknown];UPF0126 domain; pfam03458
GOHSU65	CDS	GOHSU_65_00080	complement(7128..7763)		probable NarL family two-component response regulator [nite]			RER_00980 :probable NarL family two-component response regulator : : : Rhodococcus erythropolis PR4	response regulator receiver protein		F1YFG5_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02832 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];Response regulator containing a CheY-like receiver domain and an HTH DNA-binding domain [Signal transduction mechanisms / Transcription]; COG2197;KEGG: xac:XAC2168 two-component system regulatory protein;Signal receiver domain; originally thought to be unique to bacteria (CheY, OmpR, NtrC, and PhoB), now recently identified in eukaroytes ETR1 Arabidopsis thaliana; this domain receives the signal from the sensor partner in a two-component systems; cd00156;phosphorylation site [posttranslational modification];intermolecular recognition site;dimerization interface [polypeptide binding];C-terminal DNA-binding domain of LuxR-like proteins. This domain contains a helix-turn-helix motif and binds DNA. Proteins belonging to this group are response regulators; some act as transcriptional activators, others as transcriptional repressors. Many...; cd06170;DNA binding residues [nucleotide binding]
GOHSU65	CDS	GOHSU_65_00090	complement(7760..8899)		probable two-component histidine kinase [nite]			RER_00990 :probable two-component histidine kinase : : 2.7.13.- : Rhodococcus erythropolis PR4	histidine kinase		F1YFG6_9ACTO Histidine kinase OS=Gordonia neofelifa ecis NRRL B-59395 GN=SCNU_02837 PE=4 SV=1;unnamed protein product [Streptosporangium roseum D SM 43021];KEGG: reh:H16_A1985 signal transduction histidine kinase containing PAS/PAC sensor domain;Histidine kinase; pfam07730;Histidine kinase-like ATPases; This family includes several ATP-binding proteins for example: histidine kinase, DNA gyrase B, topoisomerases, heat shock protein HSP90, phytochrome-like ATPases and DNA mismatch repair proteins; cl00075
GOHSU65	CDS	GOHSU_65_00100	9063..10061		putative ABC transporter permease protein [nite]			RER_01000 :putative ABC transporter permease protein : : : Rhodococcus erythropolis PR4	protein of unknown function DUF214		F1YFG7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_02842 PE=4 SV=1;protein of unknown function DUF214 [Streptomyces gr iseus XylebKG- 1];PFAM: Domain of unknown function DUF214, ABC transporter permease; KEGG: sgr:SGR_1292 putative ABC transporter permease protein;MacB-like periplasmic core domain; pfam12704;ABC-type antimicrobial peptide transport system, permease component [Defense mechanisms]; COG0577
GOHSU65	CDS	GOHSU_65_00110	10133..10834		putative ABC transporter ATP-binding protein [nite]			ROP_38420 :putative ABC transporter ATP-binding protein : : : Rhodococcus opacus B4	ABC transporter-like protein		F1YFG8_9ACTO Abc transporter related protein OS=Gor donia neofelifaecis NRRL B-59395 GN=SCNU_02847 PE=3 SV=1;ABC transporter-like protein [Micromonospora sp. L5 ];KEGG: sco:SCO1367 ABC transport system ATP-binding protein; PFAM: ABC transporter related; SMART: AAA ATPase;ABC-type antimicrobial peptide transport system, ATPase component [Defense mechanisms]; COG1136;This family is comprised of MJ0796 ATP-binding cassette, macrolide-specific ABC-type efflux carrier (MacAB), and proteins involved in cell division (FtsE), and release of liporoteins from the cytoplasmic membrane (LolCDE).  They are clustered together...; cd03255;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU65	CDS	GOHSU_65_00120	complement(10893..>11028)	short	fructose 1,6-bisphosphatase class II [nite]	 glpX [nite]		YM304_10700 :fructose 1,6-bisphosphatase class II : glpX : 3.1.3.11 : Acidimicrobidae bacterium YM16-304	fructose 1,6-bisphosphatase II	glpX	F1YFG9_9ACTO Fructose-1,6-bisphosphatase OS=Gordoni a neofelifaecis NRRL B-59395 GN=glpX PE=3 SV=1;fructose 1,6-bisphosphatase II [Corynebacterium glu tamicum ATCC 13032];IS fingerprint type 4-5;type II fructose 1,6-bisphosphatae; in Escherichia coli this protein forms a dimer and binds manganese;Bacterial fructose-1,6-bisphosphatase, glpX-encoded. A dimeric enzyme dependent on Mg(2+). glpX-encoded FPBase (FBPase class II) differs from other members of the inositol-phosphatase superfamily by permutation of secondary structure elements. The core...; cd01516
##GOHSU66	contig00431				10118
GOHSU66	CDS	GOHSU_66_00010	complement(<1..324)	codon	tyrosine recombinase XerC [nite]	 xerC [nite]		RER_25390 :tyrosine recombinase XerC : xerC : : Rhodococcus erythropolis PR4	tyrosine recombinase XerC	xerC	F1YIB7_9ACTO Tyrosine recombinase xerC OS=Gordonia neofelifaecis NRRL B-59395 GN=xerC PE=3 SV=1;xerC gene product [Gordonia polyisoprenivorans VH2] ;site-specific tyrosine recombinase XerC; Reviewed; PRK00236;DNA breaking-rejoining enzymes, C-terminal catalytic domain. The DNA breaking-rejoining enzyme superfamily includes type IB topoisomerases and tyrosine recombinases that share the same fold in their catalytic domain containing six conserved active site...; cl00213
GOHSU66	CDS	GOHSU_66_00020	complement(404..652)	overlap	not assigned						
GOHSU66	CDS	GOHSU_66_00030	534..1940	overlap	putative major facilitator superfamily transporter [nite]			KSE_15900 :putative major facilitator superfamily transporter : : : Kitasatospora setae NBRC 14216	major facilitator transporter		F1YIB6_9ACTO Major facilitator superfamily protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_08153 PE=4 SV=1;unnamed protein product [Frankia sp. EAN1pec];PFAM: major facilitator superfamily MFS_1; KEGG: fal:FRAAL6615 putative membrane transport protein;The Major Facilitator Superfamily (MFS) is a large and diverse group of secondary transporters that includes uniporters, symporters, and antiporters. MFS proteins facilitate the transport across cytoplasmic or internal membranes of a variety of...; cd06174
GOHSU66	CDS	GOHSU_66_00040	complement(1963..2829)		putative siderophore-interacting protein [nite]			RER_25370 :putative siderophore-interacting protein : : : Rhodococcus erythropolis PR4	Siderophore-interacting protein		F1YIB5_9ACTO FAD-binding 9 siderophore-interacting domain-containing protein OS=Gordonia neofelifaecis NRRL B -59395 GN=SCNU_08148 PE=4 SV=1;Siderophore-interacting protein [Catenulispora acid iphila DSM 44928];PFAM: Siderophore-interacting protein; FAD-binding 9 siderophore-interacting domain protein; KEGG: sma:SAV_6713 siderophore-interacting protein;Siderophore-interacting protein [Inorganic ion transport and metabolism]; COG2375;Siderophore interacting proteins share the domain structure of the ferredoxin reductase like family. Siderophores are produced in various bacteria (and some plants) to extract iron from hosts. Binding constants are high, so iron can be pilfered from...; cd06193;FAD binding pocket [chemical binding];FAD binding motif [chemical binding];phosphate binding motif [ion binding];NAD binding pocket [chemical binding]
GOHSU66	CDS	GOHSU_66_00050	2949..4298		putative wax ester synthase/diacylglycerol acyltransferase [nite]			RER_11860 :putative wax ester synthase/diacylglycerol acyltransferase : : 2.3.1.20 : Rhodococcus erythropolis PR4	putative diacylglycerol O-acyltransferase		H8ESR0_MYCTE Uncharacterized protein OS=Mycobacteri um tuberculosis (strain ATCC 35801 / TMC 107 / Erdman) GN= ERDMAN_2500 PE=4 SV=1;unnamed protein product [Mycobacterium bovis BCG st r. Mexico];acyltransferase, WS/DGAT/MGAT; TIGR02946;Wax ester synthase-like Acyl-CoA acyltransferase domain; pfam03007
GOHSU66	CDS	GOHSU_66_00060	complement(4284..5420)		putative DNA processing protein [nite]	 dprA [nite]		ROP_66050 :putative DNA processing protein : dprA : : Rhodococcus opacus B4	DNA protecting protein DprA		F9VT10_9ACTO DNA processing protein OS=Gordonia alk anivorans NBRC 16433 GN=dprA PE=4 SV=1;unnamed protein product [Nocardioides sp. JS614];TIGRFAM: DNA protecting protein DprA; PFAM: SMF family protein; KEGG: sma:SAV2631 putative DNA processing Smf-family protein;DNA recombination-mediator protein A; pfam02481;Possible lysine decarboxylase; cl00695
GOHSU66	CDS	GOHSU_66_00070	complement(5417..6937)		hypothetical protein [nite]			ROP_66040 :hypothetical protein : : : Rhodococcus opacus B4	Mg chelatase subunit ChlI		H5TLE7_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_100_00280 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: maq:Maqu_0525 fis family transcriptional regulator; TIGRFAM: Mg chelatase, subunit ChlI; PFAM: magnesium chelatase ChlI subunit; ATPase AAA; SMART: AAA ATPase;Predicted ATPase with chaperone activity [Posttranslational modification, protein turnover, chaperones];Subunit ChlI of Mg-chelatase; cl12214;The AAA+ (ATPases Associated with a wide variety of cellular Activities) superfamily represents an ancient group of ATPases belonging to the ASCE (for additional strand, catalytic E) division of the P-loop NTPase fold. The ASCE division also includes ABC; cd00009;Walker A motif;ATP binding site [chemical binding];Walker B motif;arginine finger;Magnesium chelatase, subunit ChlI; pfam13335
GOHSU66	CDS	GOHSU_66_00080	complement(6945..7364)	case	UPF0102 protein yraN [hamap]	yraN [hamap]	MF_00048:UPF0102 protein yraN:yraN 	RER_24690 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		D0LAN4_GORB4 UPF0102 protein Gbro_2095 OS=Gordonia bronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NC TC 10667) GN=Gbro_2095 PE=3 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: protein of unknown function UPF0102; KEGG: gem:GM21_3485 protein of unknown function UPF0102;Restriction endonuclease; cl00516
GOHSU66	CDS	GOHSU_66_00090	complement(7468..7779)		hypothetical protein [nite]			ROP_65970 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YIA0_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_08073 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Protein of unknown function (DUF2469); pfam10611;PFAM: protein of unknown function DUF2469
GOHSU66	CDS	GOHSU_66_00100	complement(7776..8438)		putative ribonuclease HII [nite]	 rnhB [nite]		RER_24590 :putative ribonuclease HII : rnhB : 3.1.26.4 : Rhodococcus erythropolis PR4	ribonuclease HII	rnhB	F1YI99_9ACTO Ribonuclease HII OS=Gordonia neofelifa ecis NRRL B-59395 GN=rnhB PE=3 SV=1;ribonuclease HII [Corynebacterium glutamicum ATCC 1 3032];IS fingerprint type 4-5;RNH2; RNase HII; binds manganese; endonuclease which specifically degrades the RNA of RNA-DNA hybrids;bacterial Ribonuclease HII-like; cd07182;RNA/DNA hybrid binding site [nucleotide binding]
GOHSU66	CDS	GOHSU_66_00110	complement(8451..9299)		probable signal peptidase I [nite]	 lepB [nite]		RER_24580 :probable signal peptidase I : lepB : 3.4.21.89 : Rhodococcus erythropolis PR4	signal peptidase I	lepB	F1YI98_9ACTO Signal peptidase I OS=Gordonia neofeli faecis NRRL B-59395 GN=SCNU_08063 PE=3 SV=1;signal peptidase I [Rhodococcus erythropolis PR4];This strain is also named as MBIC01337, PR4.; synonym: Rhodococcus erythropolis NBRC 100887;protein synonym:leader peptidase I;signal peptidase I, bacterial type; TIGR02227;The S26 Type I signal peptidase (SPase; LepB; leader peptidase B; leader peptidase I; EC 3.4.21.89) family members are essential membrane-bound serine proteases that function to cleave the amino-terminal signal peptide extension from proteins that are...; cd06530;Catalytic site [active]
GOHSU66	CDS	GOHSU_66_00120	complement(9328..9669)	case	50S ribosomal protein L19 [hamap]	rplS [hamap]	MF_00402:50S ribosomal protein L19:rplS 	ROP_65940 :50S ribosomal protein L19 : rplS : : Rhodococcus opacus B4	50S ribosomal protein L19	rplS	G7GQ25_9ACTO 50S ribosomal protein L19 OS=Gordonia amarae NBRC 15530 GN=rplS PE=3 SV=1;50S ribosomal protein L19 [Corynebacterium glutamic um ATCC 13032];IS fingerprint type 4-5;Ribosomal protein L19; cl00406;this protein is located at the 30S-50S ribosomal subunit interface and may play a role in the structure and function of the aminoacyl-tRNA binding site
GOHSU66	CDS	GOHSU_66_00130	complement(9838..10086)		transmembrane efflux protein [migap]				transmembrane efflux protein		transmembrane efflux protein [Streptomyces sp. AA4] ;Peptidase associated domain: C-terminal domain of M14 N/E carboxypeptidase; putative folding, regulation, or interaction domain; cl01383
##GOHSU67	contig00441				9069
GOHSU67	CDS	GOHSU_67_00010	complement(95..931)		acetoin(diacetyl) reductase [nite]			ROP_53210 :acetoin(diacetyl) reductase : : 1.1.1.5 : Rhodococcus opacus B4	short-chain dehydrogenase		H0RJI4_9ACTO Putative oxidoreductase OS=Gordonia po lyisoprenivorans NBRC 16320 GN=GOPIP_077_01600 PE=3 SV=1;short-chain dehydrogenase [Erwinia billingiae Eb661 ];Short-chain alcohol dehydrogenase of unknown specificity [General function prediction only];silverDB:661Eb00748;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;NAD(P) binding site [chemical binding]
GOHSU67	CDS	GOHSU_67_00020	complement(1016..1804)		putative iron-siderophore ABC transporter ATP-binding protein [nite]			RER_31060 :putative iron-siderophore ABC transporter ATP-binding protein : : : Rhodococcus erythropolis PR4	ferric enterobactin transport ATP-binding protein		H6MVR2_GORPV Putative ABC transport system, ATP-bin ding protein OS=Gordonia polyisoprenivorans (strain DSM 44 266 / VH2) GN=GPOL_c43590 PE=3 SV=1;ferric enterobactin transport ATP-binding protein [ Streptomyces himastatinicus ATCC 53653];ABC-type cobalamin/Fe3+-siderophores transport systems, ATPase components [Inorganic ion transport and metabolism / Coenzyme metabolism]; COG1120;ABC transporters, involved in the uptake of siderophores, heme, and vitamin B12, are widely conserved in bacteria and archaea.  Only very few species lack representatives of the siderophore family transporters. The E. coli BtuCD protein is an ABC...; cd03214;Walker A/P-loop;ATP binding site [chemical binding];Q-loop/lid;ABC transporter signature motif;Walker B;D-loop;H-loop/switch region
GOHSU67	CDS	GOHSU_67_00030	complement(1801..2844)		putative iron-siderophore ABC transporter permease protein [nite]			ROP_70360 :putative iron-siderophore ABC transporter permease protein : : : Rhodococcus opacus B4	iron ABC transporter		F1YLR3_9ACTO Transporter permease OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_14284 PE=4 SV=1;iron ABC transporter [Streptomyces griseus subsp. g riseus NBRC 13350];synonym: Streptomyces griseus subsp. griseus IFO 13350;ABC-type enterobactin transport system, permease component [Inorganic ion transport and metabolism]; COG4779;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;ABC-ATPase subunit  interface;dimer interface [polypeptide binding];putative PBP binding regions
GOHSU67	CDS	GOHSU_67_00040	complement(2841..3839)		putative iron-siderophore ABC transporter permease protein [nite]			RER_31080 :putative iron-siderophore ABC transporter permease protein : : : Rhodococcus erythropolis PR4	ABC-type transporter, integral membrane subunit		F1YLR2_9ACTO Transporter permease OS=Gordonia neofe lifaecis NRRL B-59395 GN=SCNU_14279 PE=4 SV=1;ABC-type transporter, integral membrane subunit [St reptomyces griseus XylebKG- 1];PFAM: ABC transporter permease protein; KEGG: sgr:SGR_6711 iron ABC transporter;Transmembrane subunit (TM), of Periplasmic Binding Protein (PBP)-dependent ATP-Binding Cassette (ABC) transporters involved in the uptake of siderophores, heme, vitamin B12, or the divalent cations Mg2+ and Zn2+. PBP-dependent ABC transporters consist of...; cd06550;ABC-ATPase subunit  interface;dimer interface [polypeptide binding];putative PBP binding regions
GOHSU67	CDS	GOHSU_67_00050	4077..4541		hypothetical protein [migap]				hypothetical protein		F1YLR1_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14274 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU67	CDS	GOHSU_67_00060	complement(4625..6007)		putative ferredoxin--NADP(+) reductase [nite]			RER_14410 :putative ferredoxin--NADP(+) reductase : : 1.18.1.2 : Rhodococcus erythropolis PR4	ferredoxin/ferredoxin-NADP reductase	fprA	H5UDW5_9ACTO Putative ferredoxin--NADP(+) reductase OS=Gordonia terrae NBRC 100016 GN=GOTRE_054_00290 PE=4 SV =1;ferredoxin/ferredoxin-NADP reductase [Corynebacteri um glutamicum ATCC 13032];IS fingerprint type 4-5;ferredoxin-NADP+ reductase;Rossmann-fold NAD(P)(+)-binding proteins; cl09931
GOHSU67	CDS	GOHSU_67_00070	6207..6725		hypothetical protein [nite]			RER_14420 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YLQ9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_14264 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2]
GOHSU67	CDS	GOHSU_67_00080	6727..7140		conserved hypothetical protein [nite]			RER_14430 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	rhodanese domain-containing protein		D0L4H7_GORB4 Rhodanese domain protein OS=Gordonia b ronchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCT C 10667) GN=Gbro_4035 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];SMART: rhodanese; KEGG: pla:Plav_0002 rhodanese;Member of the Rhodanese Homology Domain superfamily, subgroup 1. This CD includes the putative rhodanese-related sulfurtransferases of several uncharacterized proteins; cd01522;active site residue [active]
GOHSU67	CDS	GOHSU_67_00090	7154..8353		probable O-succinylhomoserine sulfhydrylase [nite]	 metZ [nite]		RER_14440 :probable O-succinylhomoserine sulfhydrylase : metZ : 2.5.1.- : Rhodococcus erythropolis PR4	O-succinylhomoserine sulfhydrylase	metZ	F1YLQ7_9ACTO O-succinylhomoserine sulfhydrylase OS= Gordonia neofelifaecis NRRL B-59395 GN=SCNU_14254 PE=3 SV= 1;O-succinylhomoserine sulfhydrylase [Brucella pinnip edialis B2/94];O-succinylhomoserine sulfhydrylase; Reviewed;CGS_like: Cystathionine gamma-synthase is a PLP dependent enzyme and catalyzes the committed step of methionine biosynthesis. This pathway is unique to microorganisms and plants, rendering the enzyme an attractive target for the development of...; cd00614;homodimer interface [polypeptide binding];substrate-cofactor binding pocket;pyridoxal 5'-phosphate binding site [chemical binding];catalytic residue [active]
GOHSU67	CDS	GOHSU_67_00100	complement(8396..>9069)	3x codon frame	isochorismate synthase [nite]			ROP_48860 :isochorismate synthase : : 5.4.4.2 : Rhodococcus opacus B4	putative isochorismate synthase		D0L4H4_GORB4 Isochorismate synthase OS=Gordonia bro nchialis (strain ATCC 25592 / DSM 43247 / JCM 3198 / NCTC 10667) GN=Gbro_4032 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Isochorismate synthase [Coenzyme metabolism / Secondary metabolites biosynthesis, transport, and catabolism]; COG1169;chorismate binding enzyme; cl10555
##GOHSU68	contig00451				8270
GOHSU68	CDS	GOHSU_68_00010	<1..464	3x codon frame	membrane protein, suppressor for copper-sensitivity B [migap]				membrane protein, suppressor for copper-sensitivity B		B4TSM5_SALSV Membrane protein, suppressor for coppe r-sensitivity B OS=Salmonella schwarzengrund (strain CVM19 633) GN=SeSA_A1178 PE=4 SV=1;membrane protein, suppressor for copper-sensitivity B [Salmonella enterica subsp. enterica serovar Schwarzeng rund str. CVM19633];Disulphide bond corrector protein DsbC; cl15830;identified by match to protein family HMM PF02683
GOHSU68	CDS	GOHSU_68_00020	complement(451..945)		hypothetical protein [migap]				hypothetical protein		H6MTT5_GORPV Uncharacterized protein OS=Gordonia po lyisoprenivorans (strain DSM 44266 / VH2) GN=GPOL_c41600 P E=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Rieske domain; a [2Fe-2S] cluster binding domain commonly found in Rieske non-heme iron oxygenase (RO) systems such as naphthalene and biphenyl dioxygenases, as well as in plant/cyanobacterial chloroplast b6f and mitochondrial cytochrome bc(1) complexes; cl00938;iron-sulfur cluster [ion binding];[2Fe-2S] cluster binding site [ion binding]
GOHSU68	CDS	GOHSU_68_00030	1090..1428		thioredoxin [nite]	 trxA [nite]		OBV_06460 :thioredoxin : trxA : : Oscillibacter valericigenes Sjm18-20	thioredoxin		Q46TW1_CUPPJ Probable thioredoxin OS=Cupriavidus pi natubonensis (strain JMP134 / LMG 1197) GN=Reut_B4067 PE=4 SV=1;unnamed protein product [Ralstonia eutropha JMP134] ;TRX family; composed of two groups: Group I, which includes proteins that exclusively encode a TRX domain; and Group II, which are composed of fusion proteins of TRX and additional domains. Group I TRX is a small ancient protein that alter the redox...; cd02947;catalytic residues [active]
GOHSU68	CDS	GOHSU_68_00040	1681..2832		putative integrase [nite]			MLP_30540 :putative integrase : : : Microlunatus phosphovorus NM-1	phage integrase family protein		H5TTZ4_9ACTO Putative integrase OS=Gordonia otitidi s NBRC 100426 GN=GOOTI_245_00030 PE=4 SV=1;phage integrase family protein [Bifidobacterium lon gum subsp. longum JDM301];DNA breaking-rejoining enzymes, intergrase/recombinases, C-terminal catalytic domain. The tyrosine recombinase/integrase family share the same catalytic domain containing six conserved active site residues. The best-studied members of this diverse family...; cd01182;DNA binding site [nucleotide binding];Int/Topo IB signature motif;catalytic residues [active]
GOHSU68	CDS	GOHSU_68_00050	2825..3235		not assigned						F9PMZ6_9ACTO Putative uncharacterized protein OS=Ac tinomyces sp. oral taxon 175 str. F0384 GN=HMPREF9058_0878 PE=4 SV=1
GOHSU68	CDS	GOHSU_68_00060	3422..3610		putative phage excisionase [nite]			RER_45080 :putative phage excisionase : : : Rhodococcus erythropolis PR4	excisionase family DNA binding domain		I0P863_MYCAB Uncharacterized protein OS=Mycobacteri um abscessus M94 GN=S7W_22740 PE=4 SV=1;unnamed protein product [Gordonia polyisoprenivoran s VH2];Helix-Turn-Helix DNA binding domain of transcription regulators from the MerR superfamily; cl02600;DNA binding residues [nucleotide binding]
GOHSU68	CDS	GOHSU_68_00070	3607..3915		2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol hydroxylase [migap]				2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol hydroxylase		H0JQ35_9NOCA Putative uncharacterized protein OS=Rh odococcus pyridinivorans AK37 GN=AK37_08052 PE=4 SV=1;2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol hyd roxylase [Shewanella sp. MR- 7];TIGRFAM: Ubiquinone biosynthesis hydroxylase, UbiH/UbiF/VisC/COQ6 family; PFAM: monooxygenase, FAD-binding; FAD dependent oxidoreductase; KEGG: son:SO0778 oxidoreductase, FAD-binding, UbiH/Coq6 family;Rossmann-fold NAD(P)(+)-binding proteins; cl09931;Ubiquinone biosynthesis hydroxylase, UbiH/UbiF/VisC/COQ6 family; TIGR01988
GOHSU68	CDS	GOHSU_68_00080	3912..5165		hypothetical protein [nite]			RER_20520 :hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		H8IV16_MYCIA Putative uncharacterized protein OS=My cobacterium intracellulare (strain ATCC 13950 / DSM 43223 / JCM 6384 / NCTC 13025 / 3600) GN=OCU_33670 PE=4 SV=1;unnamed protein product [Candidatus Chloracidobacte rium thermophilum B];P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A motif;ATP binding site [chemical binding];Walker B motif
GOHSU68	CDS	GOHSU_68_00090	5162..5770		myosin N-terminal SH3-like domain-containing protein [migap]				myosin N-terminal SH3-like domain-containing protein		A5G5I7_GEOUR Myosin, N-terminal, SH3 domain protein OS=Geobacter uraniireducens (strain Rf4) GN=Gura_2883 PE= 4 SV=1;myosin N-terminal SH3-like domain-containing protei n [Geobacter uraniireducens Rf4];PFAM: Myosin, N-terminal, SH3 domain protein;Competence protein CoiA-like family; cl11541
GOHSU68	CDS	GOHSU_68_00100	5767..6114		Macrolide export ATP-binding/permease macB [migap]	macB [migap]			Macrolide export ATP-binding/permease macB	macB	D5PIE3_9MYCO Putative uncharacterized protein OS=My cobacterium parascrofulaceum ATCC BAA-614 GN=HMPREF0591_59 37 PE=4 SV=1;macB gene product [Hyphomicrobium sp. MC1];macrolide transporter ATP-binding /permease protein; Provisional;Evidence 2a : Function of homologous gene experimentally demonstrated in an other organism; PubMedId : 21429237, 21450803; Product type t : transporter;This family is comprised of MJ0796 ATP-binding cassette, macrolide-specific ABC-type efflux carrier (MacAB), and proteins involved in cell division (FtsE), and release of liporoteins from the cytoplasmic membrane (LolCDE).  They are clustered together...; cd03255;ATP binding site [chemical binding];ABC transporter signature motif
GOHSU68	CDS	GOHSU_68_00110	6111..6383		sensor protein RstB [migap]				sensor protein RstB		Q7NRD8_CHRVO Probable two-component sensor protein OS=Chromobacterium violaceum (strain ATCC 12472 / DSM 3019 1 / JCM 1249 / NBRC 12614 / NCIMB 9131 / NCTC 9757) GN=CV_ 3844 PE=4 SV=1;sensor protein RstB [Chromobacterium violaceum ATCC 12472];sensor protein RstB; Provisional;identified by sequence similarity; ORF located using Glimmer/GeneMark/Blastx/COG0642;Histidine kinase, Adenylyl cyclase, Methyl-accepting protein, and Phosphatase (HAMP) domain. HAMP is a signaling domain which occurs in a wide variety of signaling proteins, many of which are bacterial. The HAMP domain consists of two alpha helices...; cl01054
GOHSU68	CDS	GOHSU_68_00120	6380..6691		not assigned						A1TD89_MYCVP Putative uncharacterized protein OS=My cobacterium vanbaalenii (strain DSM 7251 / PYR-1) GN=Mvan_ 4362 PE=4 SV=1
GOHSU68	CDS	GOHSU_68_00130	6684..7076		DNA-binding domain-containing protein [migap]	orfY19 [migap]			DNA-binding domain-containing protein	orfY19	H5TIW5_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_065_00280 PE=4 SV=1;DNA-binding domain-containing protein [Sulfobacillu s thermotolerans];Helix-Turn-Helix DNA binding domain of transcription regulators from the MerR superfamily; cl02600
GOHSU68	CDS	GOHSU_68_00140	7095..7985		not assigned						H5TIW2_9ACTO Putative uncharacterized protein OS=Go rdonia otitidis NBRC 100426 GN=GOOTI_065_00250 PE=4 SV=1
GOHSU68	CDS	GOHSU_68_00150	complement(7995..8171)		not assigned						
##GOHSU69	contig00461				5951
GOHSU69	rRNA	GOHSU_69_r0010	523..2034		not assigned						identified by RNAmmer; putative
GOHSU69	rRNA	GOHSU_69_r0020	2563..5706	overlap	not assigned						identified by RNAmmer; putative
GOHSU69	CDS	GOHSU_69_00010	3693..3869	overlap	not assigned						E2S3A4_9CORY Putative uncharacterized protein OS=Co rynebacterium pseudogenitalium ATCC 33035 GN=HMPREF0305_11 006 PE=4 SV=1
GOHSU69	rRNA	GOHSU_69_r0030	5833..5948		not assigned						identified by RNAmmer; putative
##GOHSU70	contig00471				5403
GOHSU70	CDS	GOHSU_70_00010	complement(<1..297)	codon	dipeptidase [migap]				dipeptidase		F7DPT0_ORNAN Uncharacterized protein (Fragment) OS= Ornithorhynchus anatinus GN=EPHB3 PE=4 SV=1;dipeptidase [Phenylobacterium zucineum HLK1];renal dipeptidase (rDP), best studied in mammals and also called membrane or microsomal dipeptidase, is a membrane-bound glycoprotein hydrolyzing dipeptides and is involved in hydrolytic metabolism of penem and carbapenem beta-lactam antibiotics; cd01301
GOHSU70	CDS	GOHSU_70_00020	complement(294..1088)		hypothetical protein [migap]				hypothetical protein		F1YKD7_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11470 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: xop:PXO_03548 D-3-phosphoglycerate dehydrogenase
GOHSU70	CDS	GOHSU_70_00030	1183..2022		tRNA (guanine-N(7)-)-methyltransferase [hamap]	trmB [hamap]	MF_01057:tRNA (guanine-N(7)-)-methyltransferase:trmB 	RER_10860 :tRNA (guanine-N(7)-)-methyltransferase : trmB : 2.1.1.33 : Rhodococcus erythropolis PR4	tRNA (guanine-N(7)-)-methyltransferase	trmB	F1YKD8_9ACTO tRNA (guanine-N(7)-)-methyltransferase OS=Gordonia neofelifaecis NRRL B-59395 GN=trmB PE=3 SV=1;tRNA (guanine-N(7)-)-methyltransferase [Corynebacte rium glutamicum ATCC 13032];IS fingerprint type 4-5;tRNA (guanine-N(7)-)-methyltransferase; catalyzes the formation of N(7)-methylguanine at position 46 (m7G46) in tRNA by transferring the methyl residue from S-adenosyl-L-methionine;S-adenosylmethionine-dependent methyltransferases (SAM or AdoMet-MTase), class I;  AdoMet-MTases are enzymes that use S-adenosyl-L-methionine (SAM or AdoMet) as a substrate for methyltransfer, creating the product S-adenosyl-L-homocysteine (AdoHcy); cd02440;S-adenosylmethionine binding site [chemical binding]
GOHSU70	CDS	GOHSU_70_00040	2019..2681		hypothetical protein [nite]			ROP_52250 :hypothetical protein : : : Rhodococcus opacus B4	hypothetical protein		F1YKD9_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_11480 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: mxa:MXAN_6701 hypothetical protein;LabA_like proteins; cl10034;putative metal binding site [ion binding]
GOHSU70	CDS	GOHSU_70_00050	2777..>5403	3x codon	conserved hypothetical membrane protein [nite]			RER_10840 :conserved hypothetical membrane protein : : : Rhodococcus erythropolis PR4	RND superfamily drug exporter-like protein		F1YKE0_9ACTO Drug exporter of the RND superfamily-l ike protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU _11485 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Predicted drug exporters of the RND superfamily [General function prediction only];KEGG: MmpL efflux pump
##GOHSU71	contig00481				3304
GOHSU71	CDS	GOHSU_71_00010	complement(<1..82)	short	not assigned						
GOHSU71	CDS	GOHSU_71_00020	complement(135..2345)		replicative DNA helicase [nite]	 dnaB [nite]		ROP_32210 :replicative DNA helicase : dnaB : 3.6.1.- : Rhodococcus opacus B4	replicative DNA helicase		H0R3K7_9ACTO Replicative DNA helicase OS=Gordonia e ffusa NBRC 100432 GN=dnaB PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];TIGRFAM: replicative DNA helicase; PFAM: DnaB domain protein helicase; KEGG: bcm:Bcenmc03_1893 DnaB domain-containing protein;DnaB-like helicase N terminal domain; pfam00772;P-loop containing Nucleoside Triphosphate Hydrolases; cl09099;Walker A motif;Hedgehog/Intein domain, found in Hedgehog proteins as well as proteins which contain inteins and undergo protein splicing (e.g. DnaB, RIR1-2, GyrA and Pol). In protein splicing an intervening polypeptide sequence - the intein - is excised from a protein; cl15780;DnaB helicase C terminal domain. The hexameric helicase DnaB unwinds the DNA duplex at the  chromosome replication fork. Although the mechanism by which DnaB both couples ATP hydrolysis to translocation along DNA and denatures the duplex is unknown, a...; cd00984;ATP binding site [chemical binding];Walker B motif;DNA binding loops [nucleotide binding]
GOHSU71	CDS	GOHSU_71_00030	2706..3284		hypothetical protein [migap]				hypothetical protein		H0R3K8_9ACTO Putative uncharacterized protein OS=Go rdonia effusa NBRC 100432 GN=GOEFS_092_00840 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];Signal predicted by SignalP 3.0 HMM (Signal peptide probability 0.907) with cleavage site probability 0.858 at residue 55
##GOHSU72	contig00491				2130
GOHSU72	CDS	GOHSU_72_00010	complement(<1..211)	3x codon	acyl-CoA dehydrogenase domain-containing protein [migap]				acyl-CoA dehydrogenase domain-containing protein		F1YN50_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 16788 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];PFAM: acyl-CoA dehydrogenase domain protein; KEGG: bpt:Bpet0816  acyl-CoA dehydrogenase;Acyl-CoA dehydrogenases [Lipid metabolism]; COG1960;Acyl-CoA dehydrogenase, N-terminal domain; pfam02771;Acyl-CoA dehydrogenase; cl09933
GOHSU72	CDS	GOHSU_72_00020	complement(285..>2130)	3x codon frame	conserved hypothetical protein [nite]			RER_07070 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	acyl-CoA dehydrogenase		F1YN51_9ACTO Acyl-CoA dehydrogenase domain-containi ng protein OS=Gordonia neofelifaecis NRRL B-59395 GN=SCNU_ 16793 PE=3 SV=1;acyl-CoA dehydrogenase [Rhodococcus equi 103S];Acyl-CoA dehydrogenase; cl09933;pimeloyl-CoA dehydrogenase, large subunit; TIGR03204
##GOHSU73	contig90093	length=1908	numreads=9428		1908
GOHSU73	CDS	GOHSU_73_00010	complement(78..1415)		putative groupII intron-encoded protein [nite]			BBR47_58280 :putative groupII intron-encoded protein : : : Brevibacillus brevis NBRC 100599	group II intron-encoding maturase		E1VUA0_ARTAR Reverse transcriptase/maturase OS=Arth robacter arilaitensis (strain DSM 16368 / CIP 108037 / JCM 13566 / Re117) GN=trt PE=4 SV=1;group II intron-encoding maturase [Symbiobacterium thermophilum IAM 14863];RT_G2_intron: Reverse transcriptases (RTs) with group II intron origin. RT transcribes DNA using RNA as template. Proteins in this subfamily are found in bacterial and mitochondrial group II introns. Their most probable ancestor was a retrotransposable...; cd01651;Reverse transcriptase (RNA-dependent DNA polymerase); pfam00078;putative active site [active];putative NTP binding site [chemical binding];putative nucleic acid binding site [nucleotide binding];Group II intron, maturase-specific domain; pfam08388
GOHSU73	CDS	GOHSU_73_00020	complement(1840..>1908)	short	not assigned						
##GOHSU74	contig90094	length=1694	numreads=7126		1694
GOHSU74	CDS	GOHSU_74_00010	complement(179..1510)		putative transposase [nite]			ROP_45460 :putative transposase : : : Rhodococcus opacus B4	transposase, IS4 family protein		G7H7V1_9ACTO Putative uncharacterized protein OS=Go rdonia araii NBRC 100433 GN=GOARA_097_00010 PE=4 SV=1;transposase, IS4 family protein [Mycobacterium gilv um PYR- GCK];PFAM: transposase, IS4 family protein; KEGG: rha:RHA1_ro08197 possible transposase;Transposase DDE domain; pfam01609
##GOHSU75	contig90095	length=1426	numreads=10407		1426
GOHSU75	CDS	GOHSU_75_00010	complement(45..1220)		putative transposase for insertion sequence element [nite]			ACMV_20550 :putative transposase for insertion sequence element : : : Acidiphilium multivorum AIU301	transposase, Mutator family protein		G7H0W8_9ACTO Putative transposase OS=Gordonia araii NBRC 100433 GN=GOARA_041_00010 PE=4 SV=1;transposase, Mutator family protein [Mycobacterium avium 104];identified by match to protein family HMM PF00872;Transposase, Mutator family; pfam00872;MULE transposase domain; pfam10551
GOHSU75	CDS	GOHSU_75_00020	1345..>1426	short	not assigned						
##GOHSU76	contig90096	length=1194	numreads=71107		1194
GOHSU76	CDS	GOHSU_76_00010	complement(<1..1160)	3x codon	conserved hypothetical protein [nite]			RER_50950 :conserved hypothetical protein : : : Rhodococcus erythropolis PR4	hypothetical protein		F1YE04_9ACTO Putative uncharacterized protein OS=Go rdonia neofelifaecis NRRL B-59395 GN=SCNU_01930 PE=4 SV=1;unnamed protein product [Gordonia bronchialis DSM 4 3247];KEGG: KIAA0284;Domain of unknown function (DUF222); pfam02720
##GOHSU77	contig90097	length=1148	numreads=10031		1148
GOHSU77	CDS	GOHSU_77_00010	complement(<1..785)	3x codon	putative transposase [nite]			KSE_42080 :putative transposase : : : Kitasatospora setae NBRC 14216	integrase catalytic subunit		D3DEB9_9ACTO Integrase catalytic region OS=Frankia sp. EUN1f GN=FrEUN1fDRAFT_8141 PE=4 SV=1;unnamed protein product [Frankia symbiont of Datisc a glomerata];PFAM: Integrase, catalytic core; KEGG: fre:Franean1_3089 integrase catalytic region;putative transposase OrfB; Reviewed;HTH-like domain; pfam13276;Integrase core domain; cl01316;Integrase core domain; cl15866
GOHSU77	CDS	GOHSU_77_00020	complement(797..1081)		putative transposase orfA for insertion sequence element [nite]			MLP_50150 :putative transposase orfA for insertion sequence element : : : Microlunatus phosphovorus NM-1	transposase IS3/IS911 family protein		E0N0S2_9CORY IS3/IS911 family transposase OS=Coryne bacterium accolens ATCC 49726 GN=HMPREF0277_2356 PE=4 SV=1 ;unnamed protein product [Frankia symbiont of Datisc a glomerata];PFAM: Transposase IS3/IS911; KEGG: fre:Franean1_3088 transposase IS3/IS911 family protein;Helix-turn-helix domains; cl00088
GOHSU77	CDS	GOHSU_77_00030	complement(1053..>1148)	short	not assigned						
